<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATMI</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natmi">NATMI - 21</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires. <em>NATMI</em>, <em>7</em>(9), 1587. (<a href='https://doi.org/10.1038/s42256-025-01128-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Jiang, Yuepeng and Zhang, Pingping and Huo, Miaozhe and Li, Shuai Cheng},
  doi          = {10.1038/s42256-025-01128-1},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1587},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A framework to evaluate machine learning crystal stability predictions. <em>NATMI</em>, <em>7</em>(9), 1586. (<a href='https://doi.org/10.1038/s42256-025-01117-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Riebesell, Janosh and Goodall, Rhys E. A. and Benner, Philipp and Chiang, Yuan and Deng, Bowen and Ceder, Gerbrand and Asta, Mark and Lee, Alpha A. and Jain, Anubhav and Persson, Kristin A.},
  doi          = {10.1038/s42256-025-01117-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1586},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: A framework to evaluate machine learning crystal stability predictions},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanistic understanding and validation of large AI models with SemanticLens. <em>NATMI</em>, <em>7</em>(9), 1572-1585. (<a href='https://doi.org/10.1038/s42256-025-01084-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike human-engineered systems, such as aeroplanes, for which the role and dependencies of each component are well understood, the inner workings of artificial intelligence models remain largely opaque, which hinders verifiability and undermines trust. Current approaches to neural network interpretability, including input attribution methods, probe-based analysis and activation visualization techniques, typically provide limited insights about the role of individual components or require extensive manual interpretation that cannot scale with model complexity. This paper introduces SemanticLens, a universal explanation method for neural networks that maps hidden knowledge encoded by components (for example, individual neurons) into the semantically structured, multimodal space of a foundation model such as CLIP. In this space, unique operations become possible, including (1) textual searches to identify neurons encoding specific concepts, (2) systematic analysis and comparison of model representations, (3) automated labelling of neurons and explanation of their functional roles, and (4) audits to validate decision-making against requirements. Fully scalable and operating without human input, SemanticLens is shown to be effective for debugging and validation, summarizing model knowledge, aligning reasoning with expectations (for example, adherence to the ABCDE rule in melanoma classification) and detecting components tied to spurious correlations and their associated training data. By enabling component-level understanding and validation, the proposed approach helps mitigate the opacity that limits confidence in artificial intelligence systems compared to traditional engineered systems, enabling more reliable deployment in critical applications. SemanticLens is a tool that embeds artificial intelligence model components (such as neurons) into a searchable, human-understandable space. This enables automated auditing, validation of decisions and detection of problematic behaviours with minimal human oversight.},
  archive      = {J_NATMI},
  author       = {Dreyer, Maximilian and Berend, Jim and Labarta, Tobias and Vielhaben, Johanna and Wiegand, Thomas and Lapuschkin, Sebastian and Samek, Wojciech},
  doi          = {10.1038/s42256-025-01084-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1572-1585},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Mechanistic understanding and validation of large AI models with SemanticLens},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning. <em>NATMI</em>, <em>7</em>(9), 1561-1571. (<a href='https://doi.org/10.1038/s42256-025-01098-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence has transformed the field of precise organic synthesis. Data-driven methods, including machine learning and deep learning, have shown great promise in predicting reaction performance and synthesis planning. However, the inherent methodological divergence between numerical regression-driven reaction performance prediction and sequence generation-based synthesis planning creates formidable challenges in constructing a unified deep learning architecture. Here we present RXNGraphormer, a framework to jointly address these tasks through a unified pre-training approach. By synergizing graph neural networks for intramolecular pattern recognition with Transformer-based models for intermolecular interaction modelling, and training on 13 million reactions via a carefully designed strategy, RXNGraphormer achieves state-of-the-art performance across eight benchmark datasets for reactivity or selectivity prediction and forward-synthesis or retrosynthesis planning, as well as three external realistic datasets for reactivity and selectivity prediction. Notably, the model generates chemically meaningful embeddings that spontaneously cluster reactions by type without explicit supervision. This work bridges the critical gap between performance prediction and synthesis planning tasks in chemical AI, offering a versatile tool for accurate reaction prediction and synthesis design. Xu et al. present RXNGraphormer, a pre-trained model that learns bond transformation patterns from over 13 million reactions, achieving state-of-the-art accuracy in reaction performance prediction and synthesis planning.},
  archive      = {J_NATMI},
  author       = {Xu, Li-Cheng and Tang, Miao-Jiong and An, Junyi and Cao, Fenglei and Qi, Yuan},
  doi          = {10.1038/s42256-025-01098-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1561-1571},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces. <em>NATMI</em>, <em>7</em>(9), 1555-1560. (<a href='https://doi.org/10.1038/s42256-025-01091-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular docking plays a crucial role in structure-based drug discovery, enabling the prediction of how small molecules interact with protein targets. Traditional docking methods rely on scoring functions and search heuristics, whereas recent generative approaches, such as DiffDock, leverage deep learning for pose prediction. However, blind-diffusion-based docking often struggles with binding site localization and pose accuracy, particularly in complex protein–ligand systems. This work introduces GeoDirDock (GDD), a guided diffusion approach to molecular docking that enhances the accuracy and physical plausibility of ligand docking predictions. GDD guides the denoising process of a diffusion model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. Our method leverages expert knowledge to direct the generative modelling process, specifically targeting desired protein–ligand interaction regions. We demonstrate that GDD outperforms existing blind docking methods in terms of root mean squared distance accuracy and physicochemical pose realism. Our results indicate that incorporating domain expertise into the diffusion process leads to more biologically relevant docking predictions. Additionally, we explore the potential of GDD as a template-based modelling tool for lead optimization in drug discovery through angle transfer in maximum common substructure docking, showcasing its capability to accurately predict ligand orientations for chemically similar compounds. Future applications in real-world drug discovery campaigns will naturally continue to refine and extend the utility of prior-informed diffusion docking methods. GeoDirDock is a framework that guides the denoising process of a generative diffusion docking model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. This approach enhances the accuracy and physical plausibility of ligand docking predictions.},
  archive      = {J_NATMI},
  author       = {Miñán, Raúl and Gallardo, Javier and Ciudad, Álvaro and Molina, Alexis},
  doi          = {10.1038/s42256-025-01091-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1555-1560},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Error-controlled non-additive interaction discovery in machine learning models. <em>NATMI</em>, <em>7</em>(9), 1541-1554. (<a href='https://doi.org/10.1038/s42256-025-01086-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) models are powerful tools for detecting complex patterns, yet their ‘black-box’ nature limits their interpretability, hindering their use in critical domains like healthcare and finance. Interpretable ML methods aim to explain how features influence model predictions but often focus on univariate feature importance, overlooking complex feature interactions. Although recent efforts extend interpretability to feature interactions, existing approaches struggle with robustness and error control, especially under data perturbations. In this study, we introduce Diamond, a method for trustworthy feature interaction discovery. Diamond uniquely integrates the model-X knockoffs framework to control the false discovery rate, ensuring a low proportion of falsely detected interactions. Diamond includes a non-additivity distillation procedure that refines existing interaction importance measures to isolate non-additive interaction effects and preserve false discovery rate control. This approach addresses the limitations of off-the-shelf interaction measures, which, when used naively, can lead to inaccurate discoveries. Diamond’s applicability spans a broad class of ML models, including deep neural networks, transformers, tree-based models and factorization-based models. Empirical evaluations on both simulated and real datasets across various biomedical studies demonstrate its utility in enabling reliable data-driven scientific discoveries. Diamond represents a significant step forward in leveraging ML for scientific innovation and hypothesis generation. Diamond, a statistically rigorous method, is capable of finding meaningful feature interactions within machine learning models, making black-box models more interpretable for science and medicine.},
  archive      = {J_NATMI},
  author       = {Chen, Winston and Jiang, Yifan and Noble, William Stafford and Lu, Yang Young},
  doi          = {10.1038/s42256-025-01086-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1541-1554},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Error-controlled non-additive interaction discovery in machine learning models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks. <em>NATMI</em>, <em>7</em>(9), 1524-1540. (<a href='https://doi.org/10.1038/s42256-025-01082-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering novel drug candidate molecules is a fundamental step in drug development. Generative deep learning models can sample new molecular structures from learned probability distributions; however, their practical use in drug discovery hinges on generating compounds tailored to a specific target molecule. Here we introduce DrugGEN, an end-to-end generative system for the de novo design of drug candidate molecules that interact with a selected protein. The proposed method represents molecules as graphs and processes them using a generative adversarial network that comprises graph transformer layers. Trained on large datasets of drug-like compounds and target-specific bioactive molecules, DrugGEN designed candidate inhibitors for AKT1, a kinase crucial in many cancers. Docking and molecular dynamics simulations suggest that the generated compounds effectively bind to AKT1, and attention maps provide insights into the model’s reasoning. Furthermore, selected de novo molecules were synthesized and shown to inhibit AKT1 at low micromolar concentrations in the context of in vitro enzymatic assays. These results demonstrate the potential of DrugGEN for designing target-specific molecules. Using the open-access DrugGEN codebase, researchers can retrain the model for other druggable proteins, provided a dataset of known bioactive molecules is available. Inhibiting AKT1 kinase can have potentially positive uses against many types of cancer. To find novel molecules targeting this protein, a graph adversarial network is trained as a generative model.},
  archive      = {J_NATMI},
  author       = {Ünlü, Atabey and Çevrim, Elif and Yiğit, Melih Gökay and Sarıgün, Ahmet and Çelikbilek, Hayriye and Bayram, Osman and Kahraman, Deniz Cansen and Olğaç, Abdurrahman and Rifaioglu, Ahmet Sureyya and Banoğlu, Erden and Doğan, Tunca},
  doi          = {10.1038/s42256-025-01082-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1524-1540},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Brain–computer interface control with artificial intelligence copilots. <em>NATMI</em>, <em>7</em>(9), 1510-1523. (<a href='https://doi.org/10.1038/s42256-025-01090-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motor brain–computer interfaces (BCIs) decode neural signals to help people with paralysis move and communicate. Even with important advances in the past two decades, BCIs face a key obstacle to clinical viability: BCI performance should strongly outweigh costs and risks. To significantly increase the BCI performance, we use shared autonomy, where artificial intelligence (AI) copilots collaborate with BCI users to achieve task goals. We demonstrate this AI-BCI in a non-invasive BCI system decoding electroencephalography signals. We first contribute a hybrid adaptive decoding approach using a convolutional neural network and ReFIT-like Kalman filter, enabling healthy users and a participant with paralysis to control computer cursors and robotic arms via decoded electroencephalography signals. We then design two AI copilots to aid BCI users in a cursor control task and a robotic arm pick-and-place task. We demonstrate AI-BCIs that enable a participant with paralysis to achieve 3.9-times-higher performance in target hit rate during cursor control and control a robotic arm to sequentially move random blocks to random locations, a task they could not do without an AI copilot. As AI copilots improve, BCIs designed with shared autonomy may achieve higher performance. AI copilots are integrated into brain–computer interfaces, enabling a paralysed participant to achieve improved control of computer cursors and robotic arms. This shared autonomy approach offers a promising path to increase BCI performance and clinical viability.},
  archive      = {J_NATMI},
  author       = {Lee, Johannes Y. and Lee, Sangjoon and Mishra, Abhishek and Yan, Xu and McMahan, Brandon and Gaisford, Brent and Kobashigawa, Charles and Qu, Mike and Xie, Chang and Kao, Jonathan C.},
  doi          = {10.1038/s42256-025-01090-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1510-1523},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Brain–computer interface control with artificial intelligence copilots},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Conditional generation of real antigen-specific t cell receptor sequences. <em>NATMI</em>, <em>7</em>(9), 1494-1509. (<a href='https://doi.org/10.1038/s42256-025-01096-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent advances in T cell receptor (TCR) engineering, designing functional TCRs against arbitrary targets remains challenging due to complex rules governing cross-reactivity and limited paired data. Here we present TCR-TRANSLATE, a sequence-to-sequence framework that adapts low-resource machine translation techniques to generate antigen-specific TCR sequences against unseen epitopes. By evaluating 12 model variants of the BART and T5 model architectures, we identified key factors affecting performance and utility, revealing discordances between these objectives. Our flagship model, TCRT5, outperforms existing approaches on computational benchmarks, prioritizing functionally relevant sequences at higher ranks. Most significantly, we experimentally validated a computationally designed TCR against Wilms’ tumour antigen, a therapeutically relevant target in leukaemia, excluded from our training and validation sets. Although the identified TCR shows cross-reactivity with pathogen-derived peptides, highlighting limitations in specificity, our work represents the successful computational design of a functional TCR construct against a non-viral epitope from the target sequence alone. Our findings establish a foundation for computational TCR design and reveal current limitations in data availability and methodology, providing a framework for accelerating personalized immunotherapy by reducing the search space for novel targets. TCR-TRANSLATE, a deep learning framework adapting machine translation to immune design, demonstrates the successful generation of a functional T cell receptor sequence for a cancer epitope from the target sequence alone.},
  archive      = {J_NATMI},
  author       = {Karthikeyan, Dhuvarakesh and Bennett, Sarah N. and Reynolds, Amy G. and Vincent, Benjamin G. and Rubinsteyn, Alex},
  doi          = {10.1038/s42256-025-01096-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1494-1509},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Conditional generation of real antigen-specific t cell receptor sequences},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modelling neural coding in the auditory midbrain with high resolution and accuracy. <em>NATMI</em>, <em>7</em>(9), 1478-1493. (<a href='https://doi.org/10.1038/s42256-025-01104-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational models of auditory processing can be valuable tools for research and technology development. Models of the cochlea are highly accurate and widely used, but models of the auditory brain lag far behind in both performance and penetration. Here we present ICNet, a convolutional encoder–decoder model of neural coding in the inferior colliculus. We developed ICNet using large-scale intracranial recordings from anaesthetized gerbils, addressing three key modelling challenges that are common across all sensory systems: capturing the full statistical structure of neuronal response patterns; accounting for physiological and experimental non-stationarity; and extracting features of sensory processing that are shared across different brains. ICNet provides highly accurate simulation of multi-unit neural responses to a wide range of complex sounds, including near-perfect responses to speech. It also reproduces key neurophysiological phenomena such as forward masking and dynamic range adaptation. ICNet can be used to simulate activity from thousands of neural units or to provide a compact representation of early central auditory processing through its latent dynamics, facilitating a wide range of hearing and audio applications. It can also serve as a foundation core, providing a baseline neural representation for models of active listening or higher-level auditory processing. Drakopoulos et al. present a model that captures the transformation from sound waves to neural activity patterns underlying early auditory processing. The model reproduces neural responses to a range of complex sounds and key neurophysiological phenomena.},
  archive      = {J_NATMI},
  author       = {Drakopoulos, Fotios and Pellatt, Lloyd and Sabesan, Shievanie and Xia, Yiqing and Fragner, Andreas and Lesica, Nicholas A.},
  doi          = {10.1038/s42256-025-01104-9},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1478-1493},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Modelling neural coding in the auditory midbrain with high resolution and accuracy},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping. <em>NATMI</em>, <em>7</em>(9), 1461-1477. (<a href='https://doi.org/10.1038/s42256-025-01099-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) deciphers genome-wide chromatin accessibility, providing profound insights into gene regulation mechanisms. With the rapid advance of sequencing technologies, scATAC-seq data typically encompass numerous samples from various conditions, resulting in complex batch effects, thus necessitating reliable integration tools. While numerous batch integration tools exist for single-cell RNA sequencing data, inherent data characteristic differences limit their effectiveness on scATAC-seq data. Existing integration methods for scATAC-seq data suffer from several fundamental limitations, such as disrupting the biological heterogeneity and focusing solely on low-dimensional correction, which may distort data and hinder downstream analysis. Here we propose Fountain, a deep learning framework for scATAC-seq data integration via rigorous barycentric mapping. Barycentric mapping transforms one data distribution to another in a principled and effective manner through optimal transport. By regularizing barycentric mapping with geometric data information, Fountain achieves accurate batch alignment while preserving biological heterogeneity. Comprehensive experiments across diverse real-world datasets demonstrate the advantages of Fountain over existing methods in batch correction and biological conservation. In addition, the trained Fountain model can integrate data from new batches alongside already integrated data without retraining, enabling continuous online data integration. Moreover, Fountain’s reconstruction strategy generates batch-corrected ATAC profiles, improving the capture of cellular heterogeneity and revealing cell-type-specific implications such as expression enrichment analysis and partitioned heritability analysis. Zhu, Hua and Chen propose Fountain, a deep learning framework for batch integration of scATAC-seq data that utilizes regularized barycentric mapping. It preserves biological heterogeneity, enabling online and original dimensionality integration.},
  archive      = {J_NATMI},
  author       = {Zhu, Shuchen and Hua, Heyang and Chen, Shengquan},
  doi          = {10.1038/s42256-025-01099-3},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1461-1477},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerating protein engineering with fitness landscape modelling and reinforcement learning. <em>NATMI</em>, <em>7</em>(9), 1446-1460. (<a href='https://doi.org/10.1038/s42256-025-01103-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein engineering holds substantial promise for designing proteins with customized functions, yet the vast landscape of potential mutations versus limited laboratory capacity constrains the discovery of optimal sequences. Here, to address this, we present the μProtein framework, which accelerates protein engineering by combining μFormer, a deep learning model for accurate mutational effect prediction, with μSearch, a reinforcement learning algorithm designed to efficiently navigate the protein fitness landscape using μFormer as an oracle. μProtein leverages single-mutation data to predict optimal sequences with complex, multi-amino-acid mutations through its modelling of epistatic interactions and a multi-step search strategy. In addition to strong performance on benchmark datasets, μProtein identified high-gain-of-function multi-point mutants for the enzyme β-lactamase, surpassing one of the highest-known activity levels, in wet laboratory, trained solely on single-mutation data. These results demonstrate μProtein’s capability to discover impactful mutations across the vast protein sequence space, offering a robust, efficient approach for protein optimization. μProtein, combining deep learning and reinforcement learning, is developed to design high-function proteins. This framework, trained only on single-mutation data, discovers multi-site β-lactamase mutants with up to 2,000× growth rates.},
  archive      = {J_NATMI},
  author       = {Sun, Haoran and He, Liang and Deng, Pan and Liu, Guoqing and Zhao, Zhiyu and Jiang, Yuliang and Cao, Chuan and Ju, Fusong and Wu, Lijun and Liu, Haiguang and Qin, Tao and Liu, Tie-Yan},
  doi          = {10.1038/s42256-025-01103-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1446-1460},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accelerating protein engineering with fitness landscape modelling and reinforcement learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient protein structure generation with sparse denoising models. <em>NATMI</em>, <em>7</em>(9), 1429-1445. (<a href='https://doi.org/10.1038/s42256-025-01100-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteins play diverse roles in all domains of life and are extensively harnessed as biomolecules in biotechnology, with applications spanning from fundamental research to biomedicine. Therefore, there is considerable interest in computationally designing proteins with specified properties. Protein structure generative models provide a means to design protein structures in a controllable manner and have been successfully applied to address various protein design tasks. Such models are paired with protein sequence and structure predictors to produce and select protein sequences for experimental testing. However, current protein structure generators face important limitations for proteins with more than 400 amino acids and require retraining for protein design tasks unseen during model training. To address the first issue, we introduce salad, a family of sparse all-atom denoising models for protein structure generation. Our models are smaller and faster than the state of the art and matching or improving design quality, successfully generating structures for protein lengths up to 1,000 amino acids. To address the second issue, we combine salad with structure editing, a sampling strategy for expanding the capability of protein denoising models to unseen tasks. We apply our approach to a variety of challenging protein design tasks, from generating protein scaffolds containing functional protein motifs (motif scaffolding) to designing proteins capable of adopting multiple distinct folds under different conditions (multi-state protein design), demonstrating the flexibility of salad and structure editing. A small and fast diffusion model is presented, which is able to efficiently generate long protein backbones.},
  archive      = {J_NATMI},
  author       = {Jendrusch, Michael and Korbel, Jan O.},
  doi          = {10.1038/s42256-025-01100-z},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1429-1445},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Efficient protein structure generation with sparse denoising models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics. <em>NATMI</em>, <em>7</em>(9), 1414-1428. (<a href='https://doi.org/10.1038/s42256-025-01097-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Self-supervised learning (SSL) has emerged as a powerful approach for learning meaningful representations from large-scale unlabelled datasets in single-cell genomics. Richter et al. evaluated SSL pretext tasks on modelling single-cell RNA sequencing (scRNA-seq) data, demonstrating the effective use of SSL models. However, the transferability of these pretrained SSL models to the spatial transcriptomics domain remains unexplored. Here we assess the performance of three SSL models (random mask, gene programme mask and Barlow Twins) pretrained on scRNA-seq data with spatial transcriptomics datasets, focusing on cell-type prediction and spatial clustering. Our experiments demonstrate that the SSL model with random mask strategy exhibits the best overall performance among evaluated SSL models. Moreover, the models trained from scratch on spatial transcriptomics data outperform the fine-tuned SSL models on cell-type prediction, highlighting a domain gap between scRNA-seq and spatial transcriptomics data whose underlying causes remain an open question. Through expanded analyses of multiple imputation methods and data degradation scenarios, we demonstrate that gene imputation would degrade SSL model performance on cell-type prediction, an effect that is exacerbated by increasing data sparsity. Finally, integrating zero-shot random mask embeddings into chosen spatial clustering methods significantly enhanced their accuracy. Overall, our findings provide valuable insights into the limitations and potential of transferring SSL models to spatial transcriptomics and offer practical guidance for researchers leveraging pretrained models for spatial transcriptomics data analysis. Self-supervised learning models for single-cell RNA sequencing data exhibit poor transferability to spatial transcriptomics for cell-type prediction, although their learned features may enhance spatial analysis.},
  archive      = {J_NATMI},
  author       = {Han, Chuangyi and Lin, Senlin and Wang, Zhikang and Cui, Yan and Zou, Qi and Yuan, Zhiyuan},
  doi          = {10.1038/s42256-025-01097-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1414-1428},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosting the predictive power of protein representations with a corpus of text annotations. <em>NATMI</em>, <em>7</em>(9), 1403-1413. (<a href='https://doi.org/10.1038/s42256-025-01088-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models are trained to predict amino acid sequences from vast protein databases and learn to represent proteins as feature vectors. These vector representations have enabled impressive applications, from predicting mutation effects to protein folding. One of the reasons offered for the success of these models is that conserved sequence motifs tend to be important for protein fitness. Yet, the relationship between sequence conservation and fitness can be confounded by the evolutionary and environmental context. Should we, therefore, look to other data sources that may contain more direct functional information? In this work, we conduct a comprehensive study examining the effects of training protein models to predict 19 types of text annotation from UniProt. Our results show that fine-tuning protein models on a subset of these annotations enhances the models’ predictive capabilities on a variety of function prediction tasks. In particular, when evaluated on our tasks, our model outperforms the basic local alignment search tool, which none of the pretrained protein models accomplished. Our results suggest that a much wider array of data modalities, such as text annotations, may be tapped to improve protein language models. Although protein language models have enabled major advances, they often rely on indirect signals that may not fully capture functional relevance. Fine-tuning these models on textual annotations is shown to improve their performance on function prediction tasks.},
  archive      = {J_NATMI},
  author       = {Duan, Haonan and Skreta, Marta and Cotta, Leonardo and Rajaonson, Ella Miray and Dhawan, Nikita and Aspuru-Guzik, Alán and Maddison, Chris J.},
  doi          = {10.1038/s42256-025-01088-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1403-1413},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Boosting the predictive power of protein representations with a corpus of text annotations},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury. <em>NATMI</em>, <em>7</em>(9), 1390-1402. (<a href='https://doi.org/10.1038/s42256-025-01105-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spinal cord injury (SCI) disrupts neuromuscular control, severely affecting independence and quality of life. Although upper limb wearable robots hold considerable promise for functional restoration, most existing prototypes have been validated minimally in people with SCI and target almost exclusively hand opening and closing. We introduce a lightweight, modular assistive soft exosuit that simultaneously and automatically supports shoulder abduction and elbow flexion or extension movements using lightweight fabric-based pneumatic actuators, controlled through inertial sensors. The individual elbow modules were first validated in 11 healthy volunteers, and subsequently tested, together with the shoulder module, in 15 individuals with cervical SCI (C4–C7, AIS A–D). In the SCI participants, exosuits assistance resulted in increased static endurance time (by more than 250%), and lower activity of the primary muscles involved in dynamic tasks (by up to 50%). The two SCI participants retaining prehensile capability also improved their scores in the box and block test when assisted. Moreover, the soft actuation provided a safe, comfortable and easy-to-use solution that was positively appreciated by the participants. Collectively, these results provide encouraging evidence that exosuits can augment upper limb motor performance, and may ultimately translate into greater functional independence and quality of life for the SCI population. A lightweight, modular assistive soft exosuit is introduced, which supports shoulder and elbow movement in individuals with cervical spinal cord injury. The device enhances endurance and range of motion, reduces muscle effort and improves clinical test scores.},
  archive      = {J_NATMI},
  author       = {Ferroni, Roberto and D’Avola, Gaetano and Sciarrone, Giorgia and Righi, Gabriele and De Santis, Claudia and Carpaneto, Jacopo and Gandolla, Marta and Del Popolo, Giulio and Micera, Silvestro and Proietti, Tommaso},
  doi          = {10.1038/s42256-025-01105-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1390-1402},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Aligning generalization between humans and machines. <em>NATMI</em>, <em>7</em>(9), 1378-1389. (<a href='https://doi.org/10.1038/s42256-025-01109-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in artificial intelligence (AI)—including generative approaches—have resulted in technology that can support humans in scientific discovery and forming decisions, but may also disrupt democracies and target individuals. The responsible use of AI and its participation in human–AI teams increasingly shows the need for AI alignment, that is, to make AI systems act according to our preferences. A crucial yet often overlooked aspect of these interactions is the different ways in which humans and machines generalize. In cognitive science, human generalization commonly involves abstraction and concept learning. By contrast, AI generalization encompasses out-of-domain generalization in machine learning, rule-based reasoning in symbolic AI, and abstraction in neurosymbolic AI. Here we combine insights from AI and cognitive science to identify key commonalities and differences across three dimensions: notions of, methods for, and evaluation of generalization. We map the different conceptualizations of generalization in AI and cognitive science along these three dimensions and consider their role for alignment in human–AI teaming. This results in interdisciplinary challenges across AI and cognitive science that must be tackled to support effective and cognitively supported alignment in human–AI teaming scenarios. Ilievski et al. examine differences and similarities in the various ways human and AI systems generalize. The insights are important for effectively supporting alignment in human–AI teams.},
  archive      = {J_NATMI},
  author       = {Ilievski, Filip and Hammer, Barbara and van Harmelen, Frank and Paassen, Benjamin and Saralajew, Sascha and Schmid, Ute and Biehl, Michael and Bolognesi, Marianna and Dong, Xin Luna and Gashteovski, Kiril and Hitzler, Pascal and Marra, Giuseppe and Minervini, Pasquale and Mundt, Martin and Ngomo, Axel-Cyrille Ngonga and Oltramari, Alessandro and Pasi, Gabriella and Saribatur, Zeynep G. and Serafini, Luciano and Shawe-Taylor, John and Shwartz, Vered and Skitalinskaya, Gabriella and Stachl, Clemens and van de Ven, Gido M. and Villmann, Thomas},
  doi          = {10.1038/s42256-025-01109-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1378-1389},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Aligning generalization between humans and machines},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Real-world validation of a structure-aware pipeline for molecular design. <em>NATMI</em>, <em>7</em>(9), 1376-1377. (<a href='https://doi.org/10.1038/s42256-025-01102-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The next major challenge for artificial intelligence in drug development lies in proving its value in real-world settings. A new technology not only supports the generation of novel chemical entities but also accelerates a range of real-world molecular design tasks.},
  archive      = {J_NATMI},
  author       = {Dias, Ana Laura and Rodrigues, Tiago},
  doi          = {10.1038/s42256-025-01102-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1376-1377},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Real-world validation of a structure-aware pipeline for molecular design},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards agentic science for advancing scientific discovery. <em>NATMI</em>, <em>7</em>(9), 1373-1375. (<a href='https://doi.org/10.1038/s42256-025-01110-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence is transforming scientific discovery through (semi-)autonomous agents capable of reasoning, planning, and interacting with digital and physical environments. This Comment explores the foundations and frontiers of agentic science, outlining its emerging directions, current limitations, and the pathways for responsible integration into scientific practice.},
  archive      = {J_NATMI},
  author       = {Xin, Hongliang and Kitchin, John R. and Kulik, Heather J.},
  doi          = {10.1038/s42256-025-01110-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1373-1375},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards agentic science for advancing scientific discovery},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LLMs as all-in-one tools to easily generate publication-ready citation diversity reports. <em>NATMI</em>, <em>7</em>(9), 1371-1372. (<a href='https://doi.org/10.1038/s42256-025-01101-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Cantú, Melissa S. and King, Michael R.},
  doi          = {10.1038/s42256-025-01101-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1371-1372},
  shortjournal = {Nat. Mach. Intell.},
  title        = {LLMs as all-in-one tools to easily generate publication-ready citation diversity reports},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Applying genomic AI to combat antibiotic resistance in low-income countries. <em>NATMI</em>, <em>7</em>(9), 1369-1370. (<a href='https://doi.org/10.1038/s42256-025-01108-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Aruhomukama, Dickson},
  doi          = {10.1038/s42256-025-01108-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1369-1370},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Applying genomic AI to combat antibiotic resistance in low-income countries},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
