<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth">NMETH - 35</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: IPipet: Sample handling using a tablet. <em>NMETH</em>, <em>22</em>(9), 1996. (<a href='https://doi.org/10.1038/s41592-025-02804-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zielinski, Dina and Gordon, Assaf and Zaks, Benjamin L. and Erlich, Yaniv},
  doi          = {10.1038/s41592-025-02804-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1996},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: IPipet: Sample handling using a tablet},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A universal language for finding mass spectrometry data patterns. <em>NMETH</em>, <em>22</em>(9), 1995. (<a href='https://doi.org/10.1038/s41592-025-02785-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Damiani, Tito and Jarmusch, Alan K. and Aron, Allegra T. and Petras, Daniel and Phelan, Vanessa V. and Zhao, Haoqi Nina and Bittremieux, Wout and Acharya, Deepa D. and Ahmed, Mohammed M. A. and Bauermeister, Anelize and Bertin, Matthew J. and Boudreau, Paul D. and Borges, Ricardo M. and Bowen, Benjamin P. and Brown, Christopher J. and Chagas, Fernanda O. and Clevenger, Kenneth D. and Correia, Mario S. P. and Crandall, William J. and Crüsemann, Max and Fahy, Eoin and Fiehn, Oliver and Garg, Neha and Gerwick, William H. and Gilbert, Jeffrey R. and Globisch, Daniel and Gomes, Paulo Wender P. and Heuckeroth, Steffen and James, C. Andrew and Jarmusch, Scott A. and Kakhkhorov, Sarvar A. and Kang, Kyo Bin and Kessler, Nikolas and Kersten, Roland D. and Kim, Hyunwoo and Kirk, Riley D. and Kohlbacher, Oliver and Kontou, Eftychia E. and Liu, Ken and Lizama-Chamu, Itzel and Luu, Gordon T. and Luzzatto Knaan, Tal and Mannochio-Russo, Helena and Marty, Michael T. and Matsuzawa, Yuki and McAvoy, Andrew C. and McCall, Laura-Isobel and Mohamed, Osama G. and Nahor, Omri and Neuweger, Heiko and Niedermeyer, Timo H. J. and Nishida, Kozo and Northen, Trent R. and Overdahl, Kirsten E. and Rainer, Johannes and Reher, Raphael and Rodriguez, Elys and Sachsenberg, Timo T. and Sanchez, Laura M. and Schmid, Robin and Stevens, Cole and Subramaniam, Shankar and Tian, Zhenyu and Tripathi, Ashootosh and Tsugawa, Hiroshi and van der Hooft, Justin J. J. and Vicini, Andrea and Walter, Axel and Weber, Tilmann and Xiong, Quanbo and Xu, Tao and Pluskal, Tomáš and Dorrestein, Pieter C. and Wang, Mingxun},
  doi          = {10.1038/s41592-025-02785-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1995},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A universal language for finding mass spectrometry data patterns},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unified mass imaging maps the lipidome of vertebrate development. <em>NMETH</em>, <em>22</em>(9), 1981-1994. (<a href='https://doi.org/10.1038/s41592-025-02771-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryo development entails the formation of anatomical structures with distinct biochemical compositions. Compared with the wealth of knowledge on gene regulation, our understanding of metabolic programs operating during embryogenesis is limited. Mass spectrometry imaging (MSI) has the potential to map the distribution of metabolites across embryo development. Here we established uMAIA, an analytical framework for the joint analysis of large MSI datasets, which enables the construction of multidimensional metabolomic atlases. Employing this framework, we mapped the four-dimensional (4D) distribution of over a hundred lipids at micrometric resolution in Danio rerio embryos. We discovered metabolic trajectories that unfold in concert with morphogenesis and revealed spatially organized biochemical coordination overlooked by bulk measurements. Interestingly, lipid mapping revealed unexpected distributions of sphingolipid and triglyceride species, suggesting their involvement in pattern establishment and organ development. Our approach empowers a new generation of whole-organism metabolomic atlases and enables the discovery of spatially organized metabolic circuits. uMAIA is an analytical framework designed to enable the construction of metabolic atlases at high resolution using mass spectrometry imaging data.},
  archive      = {J_NMETH},
  author       = {Schede, Halima Hannah and Alieh, Leila Haj Abdullah and Rohde, Laurel Ann and Herrera, Antonio and Schlaeppi, Anjalie and Valentin, Guillaume and Gargoori Motlagh, Alireza and Dominguez Mantes, Albert and Jollivet, Chloe and Paz-Montoya, Jonathan and Capolupo, Laura and Khven, Irina and Oates, Andrew C. and D’Angelo, Giovanni and La Manno, Gioele},
  doi          = {10.1038/s41592-025-02771-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1981-1994},
  shortjournal = {Nat. Meth.},
  title        = {Unified mass imaging maps the lipidome of vertebrate development},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions. <em>NMETH</em>, <em>22</em>(9), 1964-1980. (<a href='https://doi.org/10.1038/s41592-025-02793-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues. TCAT is a pipeline that can simultaneously capture gene expression programs related to T cell subsets and activation states for accurate T cell characterization.},
  archive      = {J_NMETH},
  author       = {Kotliar, Dylan and Curtis, Michelle and Agnew, Ryan and Weinand, Kathryn and Nathan, Aparna and Baglaenko, Yuriy and Slowikowski, Kamil and Zhao, Yu and Sabeti, Pardis C. and Rao, Deepak A. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41592-025-02793-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1964-1980},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput profiling of chemical-induced gene expression across 93,644 perturbations. <em>NMETH</em>, <em>22</em>(9), 1954-1963. (<a href='https://doi.org/10.1038/s41592-025-02781-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this Resource, we present an extensive dataset of chemical-induced gene signatures (CIGS), encompassing expression patterns of 3,407 genes regulating key biological processes in 2 human cell lines exposed to 13,221 compounds across 93,664 perturbations. This dataset encompasses 319,045,108 gene expression events, generated through 2 high-throughput technologies: the previously documented high-throughput sequencing-based high-throughput screening (HTS2) and the newly developed highly multiplexed and parallel sequencing (HiMAP-seq). Our results show that HiMAP-seq is comparable to RNA sequencing, but can profile the expression of thousands of genes across thousands of samples in one single test by utilizing a pooled-sample strategy. We further illustrate CIGS’s utility in elucidating the mechanism of action of unannotated small molecules, like ligustroflavone and 2,4-dihydroxybenzaldehyde, and to identify perturbation-induced cell states, such as those resistant to ferroptosis. The full dataset is publicly accessible at https://cigs.iomicscloud.com/ . CIGS is a high-resolution dataset comprising chemical perturbation-induced 319,045,108 gene expression events.},
  archive      = {J_NMETH},
  author       = {Xiang, Lei and Wang, Yumei and Shao, Wei and Li, Qingzhou and Yu, Xiankuo and Wei, Mingming and Gui, Yu and Li, Shengrong and Qin, Pan and Hu, Chao and Zhang, Guochen and Zhang, Xianwen and Wang, Jiawen and Li, Yingying and An, Jun and Luo, Yan and Liao, Yile and Deng, Jinghong and Tai, Xinran and Xu, Richard Y. and Huang, Lijun and Guo, Dale and Zhang, Guanbin and Xie, Zhi and Deng, Yun and Xu, Junquan and Wang, Dong},
  doi          = {10.1038/s41592-025-02781-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1954-1963},
  shortjournal = {Nat. Meth.},
  title        = {High-throughput profiling of chemical-induced gene expression across 93,644 perturbations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI. <em>NMETH</em>, <em>22</em>(9), 1944-1953. (<a href='https://doi.org/10.1038/s41592-025-02798-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concurrent recording of electroencephalogram (EEG) and functional magnetic resonance imaging (fMRI) signals reveals cross-scale neurovascular dynamics crucial for explaining fundamental linkages between function and behaviors. However, MRI scanners generate artifacts for EEG detection. Despite existing denoising methods, cabled connections to EEG receivers are susceptible to environmental fluctuations inside MRI scanners, creating baseline drifts that complicate EEG signal retrieval from the noisy background. Here we show that a wireless integrated sensing detector for simultaneous EEG and MRI can encode fMRI and EEG signals on distinct sidebands of the detector’s oscillation wave for detection by a standard MRI console over the entire duration of the fMRI sequence. Local field potential and fMRI maps are retrieved through low-pass and high-pass filtering of frequency-demodulated signals. From optogenetically stimulated somatosensory cortex in ChR2-transfected Sprague Dawley rats, positive correlation between evoked local field potential and fMRI signals validates strong neurovascular coupling, enabling cross-scale brain mapping with this two-in-one transducer. WISDEM is a hybrid detector for simultaneous EEG and fMRI recordings without artifacts or crosstalk, which allows access to neural activity at high temporal and spatial resolution, respectively, as demonstrated in rats.},
  archive      = {J_NMETH},
  author       = {Chen, Yi and Qian, Wei and Razansky, Daniel and Yu, Xin and Qian, Chunqi},
  doi          = {10.1038/s41592-025-02798-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1944-1953},
  shortjournal = {Nat. Meth.},
  title        = {WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A versatile miniature two-photon microscope enabling multicolor deep-brain imaging. <em>NMETH</em>, <em>22</em>(9), 1935-1943. (<a href='https://doi.org/10.1038/s41592-025-02780-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present FHIRM-TPM 3.0, a 2.6 g miniature two-photon microscope capable of multicolor deep-brain imaging in freely behaving mice. The system was integrated with a broadband anti-resonant hollow-core fiber featuring low transmission loss, minimal dispersion from 700 nm to 1,060 nm and high tolerance of laser power. By correcting chromatic and spherical aberrations and optimizing the fluorescence collection aperture, we achieved cortical neuronal imaging at depths exceeding 820 μm and, using a GRIN lens, hippocampal Ca2+ imaging at single dendritic spine resolution. Moreover, we engineered three interchangeable parfocal objectives, allowing for a tenfold scalable field of view up to 1 × 0.8 mm², with lateral resolutions ranging from 0.68 μm to 1.46 μm. By multicolor imaging at excitation wavelengths of 780 nm, 920 nm and 1,030 nm, we investigated mitochondrial and cytosolic Ca2+ activities relative to the deposition of amyloid plaques in the cortex of awake APP/PS1 transgenic mice. Thus, FHIRM-TPM 3.0 provides a versatile imaging system suitable for diverse brain imaging scenarios. FHIRM-TPM 3.0 is a miniature microscope for multicolor two-photon imaging in freely moving mice. In addition to the multicolor imaging abilities achieved with the help of a specially designed optical fiber, the microscope is also compatible with multiple lenses for a choice of field of view and resolution.},
  archive      = {J_NMETH},
  author       = {Wu, Runlong and Zhao, Chunzhu and Qiu, Shan and Zhu, Yufei and Zhang, Lifeng and Fu, Qiang and Hu, Yanhui and Wu, Dakun and Yu, Fei and Zhou, Fangxu and Huang, Haipeng and Zhang, Yunfeng and Wang, Xianhua and Wang, Aimin and Cheng, Heping},
  doi          = {10.1038/s41592-025-02780-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1935-1943},
  shortjournal = {Nat. Meth.},
  title        = {A versatile miniature two-photon microscope enabling multicolor deep-brain imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy. <em>NMETH</em>, <em>22</em>(9), 1923-1934. (<a href='https://doi.org/10.1038/s41592-025-02794-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volume correlative light and electron microscopy (vCLEM) is a powerful imaging technique that enables the visualization of fluorescently labeled proteins within their ultrastructural context. Currently, vCLEM alignment relies on time-consuming and subjective manual methods. This paper presents CLEM-Reg, an algorithm that automates the three-dimensional alignment of vCLEM datasets by leveraging probabilistic point cloud registration techniques. Point clouds are derived from segmentations of common structures in each modality, created by state-of-the-art open-source methods. CLEM-Reg drastically reduces the registration time of vCLEM datasets to a few minutes and achieves correlation of fluorescent signal to submicron target structures in electron microscopy on three newly acquired vCLEM benchmark datasets. CLEM-Reg was then used to automatically obtain vCLEM overlays to unambiguously identify TGN46-positive transport carriers involved in protein trafficking between the trans-Golgi network and plasma membrane. Datasets are available on EMPIAR and BioStudies, and a napari plugin is provided to aid end-user adoption. CLEM-Reg automates the three-dimensional alignment of volume correlative light and electron microscopy datasets by leveraging probabilistic point cloud registration techniques for fast and accurate results across diverse datasets.},
  archive      = {J_NMETH},
  author       = {Krentzel, Daniel and Elphick, Matouš and Domart, Marie-Charlotte and Peddie, Christopher J. and Laine, Romain F. and Shand, Cameron and Henriques, Ricardo and Collinson, Lucy M. and Jones, Martin L.},
  doi          = {10.1038/s41592-025-02794-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1923-1934},
  shortjournal = {Nat. Meth.},
  title        = {CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE. <em>NMETH</em>, <em>22</em>(9), 1911-1922. (<a href='https://doi.org/10.1038/s41592-025-02770-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while preserving crucial spatial context within tissues. However, existing ST platforms are constrained by high costs, long turnaround times, low resolution, limited gene coverage and inherently small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that reconstructs large-scale, super-resolution gene expression landscapes and automatically annotates cellular-level tissue architecture in samples exceeding capture areas of current ST platforms. The performance of iSCALE was assessed by comprehensive evaluations involving benchmarking experiments, immunohistochemistry staining and manual annotations by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion-associated cellular characteristics undetectable by conventional ST experiments. Our results demonstrate the utility of iSCALE in analyzing large tissues by enabling unbiased annotation, resolving cell type composition, mapping cellular microenvironments and revealing spatial features beyond the reach of standard ST analysis or routine histopathological assessment. iSCALE leverages histology and spatial transcriptomics to infer gene expression at super resolution in large tissues.},
  archive      = {J_NMETH},
  author       = {Schroeder, Amelia and Loth, Melanie L. and Luo, Chunyu and Yao, Sicong and Yan, Hanying and Zhang, Daiwei and Piya, Sarbottam and Plowey, Edward and Hu, Wenxing and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Chan, Su Jing and Reynolds, Taylor L. and Carlile, Thomas and Cullen, Patrick and Sung, Ji-Youn and Tsai, Hui-Hsin and Park, Jeong Hwan and Hwang, Tae Hyun and Zhang, Baohong and Li, Mingyao},
  doi          = {10.1038/s41592-025-02770-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1911-1922},
  shortjournal = {Nat. Meth.},
  title        = {Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial gene expression at single-cell resolution from histology using deep learning with GHIST. <em>NMETH</em>, <em>22</em>(9), 1900-1910. (<a href='https://doi.org/10.1038/s41592-025-02795-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The increased use of spatially resolved transcriptomics provides new biological insights into disease mechanisms. However, the high cost and complexity of these methods are barriers to broader application. Consequently, methods have been created to predict spot-based gene expression from routinely collected histology images. Recent benchmarking showed that current methodologies have limited accuracy and spatial resolution, constraining translational capacity. Here, we introduce GHIST, a deep learning-based framework that predicts spatial gene expression at single-cell resolution by leveraging subcellular spatial transcriptomics and synergistic relationships between multiple layers of biological information. We validated GHIST using public datasets and The Cancer Genome Atlas data, demonstrating its flexibility across different spatial resolutions and superior performance. Our results underscore the utility of in silico generation of single-cell spatial gene expression measurements and the capacity to enrich existing datasets with a spatially resolved omics modality, paving the way for scalable multi-omics analysis and biomarker identification. GHIST is a deep learning-based method that can predict spatial gene expression at high resolution using histology data.},
  archive      = {J_NMETH},
  author       = {Fu, Xiaohang and Cao, Yue and Bian, Beilei and Wang, Chuhan and Graham, Dinny and Pathmanathan, Nirmala and Patrick, Ellis and Kim, Jinman and Yang, Jean Yee Hwa},
  doi          = {10.1038/s41592-025-02795-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1900-1910},
  shortjournal = {Nat. Meth.},
  title        = {Spatial gene expression at single-cell resolution from histology using deep learning with GHIST},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ultra-high-scale cytometry-based cellular interaction mapping. <em>NMETH</em>, <em>22</em>(9), 1887-1899. (<a href='https://doi.org/10.1038/s41592-025-02744-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular interactions are of fundamental importance, orchestrating organismal development, tissue homeostasis and immunity. Recently, powerful methods that use single-cell genomic technologies to dissect physically interacting cells have been developed. However, these approaches are characterized by low cellular throughput, long processing times and high costs and are typically restricted to predefined cell types. Here we introduce Interact-omics, a cytometry-based framework to accurately map cellular landscapes and cellular interactions across all immune cell types at ultra-high resolution and scale. We demonstrate the utility of our approach to study kinetics, mode of action and personalized response prediction of immunotherapies, and organism-wide shifts in cellular composition and cellular interaction dynamics following infection in vivo. Our scalable framework can be applied a posteriori to existing cytometry datasets or incorporated into newly designed cytometry-based studies to map cellular interactions with a broad range of applications from fundamental biology to applied biomedicine. Interact-omics, a high-throughput cytometry-based framework, resolves the cellular interaction landscape.},
  archive      = {J_NMETH},
  author       = {Vonficht, Dominik and Jopp-Saile, Lea and Yousefian, Schayan and Flore, Viktoria and Simó Vesperinas, Inés and Teuber, Ruth and Avanesyan, Bogdan and Luo, Yanjiang and Röthemeier, Caroline and Grünschläger, Florian and Fernandez-Vaquero, Mirian and Fregona, Vincent and Ordoñez-Rueda, Diana and Schmalbrock, Laura K. and Deininger, Luca and Yamachui Sitcheu, Angelo Jovin and Gu, Zuguang and Funk, Maja C. and Mikut, Ralf and Heikenwälder, Mathias and Eggert, Angelika and von Stackelberg, Arend and Kobold, Sebastian and Krönke, Jan and Keller, Ulrich and Trumpp, Andreas and Hegazy, Ahmed N. and Eckert, Cornelia and Hübschmann, Daniel and Haas, Simon},
  doi          = {10.1038/s41592-025-02744-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1887-1899},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-scale cytometry-based cellular interaction mapping},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Laser flash melting cryo-EM samples to overcome preferred orientation. <em>NMETH</em>, <em>22</em>(9), 1880-1886. (<a href='https://doi.org/10.1038/s41592-025-02796-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sample preparation remains a bottleneck for protein structure determination by cryo-electron microscopy. A frequently encountered issue is that proteins adsorb to the air–water interface of the sample in a limited number of orientations. This makes it challenging to obtain high-resolution reconstructions, or may even cause projects to fail altogether. We have previously observed that laser flash melting and revitrification of cryo-EM samples reduces preferred orientation for large, symmetric particles. Here we demonstrate that our method can in fact be used to scramble the orientation of proteins of a range of sizes and symmetries. The effect can be enhanced for some proteins by increasing the heating rate during flash melting or by depositing amorphous ice onto the sample prior to revitrification. This also allows us to shed light onto the underlying mechanism. Our experiments establish a set of tools for overcoming preferred orientation that can be easily integrated into existing workflows. Individual proteins tend to adopt preferred orientations when subjected to vitrification for cryo-electron microscopy analysis. A laser flash melting procedure followed by rapid revitrification provides a simple approach to mitigate this issue, reducing the number of micrographs required for successful structure determination at high-resolution.},
  archive      = {J_NMETH},
  author       = {Straub, Monique S. and Harder, Oliver F. and Mowry, Nathan J. and Barrass, Sarah V. and Hruby, Jakub and Drabbels, Marcel and Lorenz, Ulrich J.},
  doi          = {10.1038/s41592-025-02796-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1880-1886},
  shortjournal = {Nat. Meth.},
  title        = {Laser flash melting cryo-EM samples to overcome preferred orientation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biophysics-based protein language models for protein engineering. <em>NMETH</em>, <em>22</em>(9), 1868-1879. (<a href='https://doi.org/10.1038/s41592-025-02776-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models trained on evolutionary data have emerged as powerful tools for predictive problems involving protein sequence, structure and function. However, these models overlook decades of research into biophysical factors governing protein function. We propose mutational effect transfer learning (METL), a protein language model framework that unites advanced machine learning and biophysical modeling. Using the METL framework, we pretrain transformer-based neural networks on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics. We fine-tune METL on experimental sequence–function data to harness these biophysical signals and apply them when predicting protein properties like thermostability, catalytic activity and fluorescence. METL excels in challenging protein engineering tasks like generalizing from small training sets and position extrapolation, although existing methods that train on evolutionary signals remain powerful for many types of experimental assays. We demonstrate METL’s ability to design functional green fluorescent protein variants when trained on only 64 examples, showcasing the potential of biophysics-based protein language models for protein engineering. Mutational effect transfer learning (METL) is a protein language model framework that unites machine learning and biophysical modeling. Transformer-based neural networks are pretrained on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics.},
  archive      = {J_NMETH},
  author       = {Gelman, Sam and Johnson, Bryce and Freschlin, Chase R. and Sharma, Arnav and D’Costa, Sameer and Peters, John and Gitter, Anthony and Romero, Philip A.},
  doi          = {10.1038/s41592-025-02776-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1868-1879},
  shortjournal = {Nat. Meth.},
  title        = {Biophysics-based protein language models for protein engineering},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations. <em>NMETH</em>, <em>22</em>(9), 1857-1867. (<a href='https://doi.org/10.1038/s41592-025-02797-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modifications (PTMs) are critical regulators of protein function, and their disruption is a key mechanism by which missense variants contribute to disease. Accurate PTM site prediction using deep learning can help identify PTM-altering variants, but progress has been limited by the lack of large, high-quality training datasets. Here, we introduce PTMAtlas, a curated compendium of 397,524 PTM sites generated through systematic reprocessing of 241 public mass-spectrometry datasets, and DeepMVP, a deep learning framework trained on PTMAtlas to predict PTM sites for phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation. DeepMVP substantially outperforms existing tools across all six PTM types. Its application to predicting PTM-altering missense variants shows strong concordance with experimental results, validated using literature-curated variants and cancer proteogenomic datasets. Together, PTMAtlas and DeepMVP provide a robust platform for PTM research and a scalable framework for assessing the functional consequences of coding variants through the lens of PTMs. DeepMVP is a deep learning framework for predicting PTM sites and variant-induced alterations across six modification types, including phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation.},
  archive      = {J_NMETH},
  author       = {Wen, Bo and Wang, Chenwei and Li, Kai and Han, Ping and Holt, Matthew V. and Savage, Sara R. and Lei, Jonathan T. and Dou, Yongchao and Shi, Zhiao and Li, Yi and Zhang, Bing},
  doi          = {10.1038/s41592-025-02797-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1857-1867},
  shortjournal = {Nat. Meth.},
  title        = {DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data. <em>NMETH</em>, <em>22</em>(9), 1846-1856. (<a href='https://doi.org/10.1038/s41592-025-02773-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations such as copy number alterations accumulate during cancer progression, driving intratumor heterogeneity that impacts therapy effectiveness. Understanding the characteristics and spatial distribution of genetically distinct subclones is essential for unraveling tumor evolution and improving cancer treatment. Here we present Clonalscope, a subclone detection method using copy number profiles, applicable to spatial transcriptomics and single-cell sequencing data. Clonalscope implements a nested Chinese Restaurant Process to identify de novo tumor subclones, which can incorporate prior information from matched bulk DNA sequencing data for improved subclone detection and malignant cell labeling. On single-cell RNA sequencing and single-cell assay for transposase-accessible chromatin using sequencing data from gastrointestinal tumors, Clonalscope successfully labeled malignant cells and identified genetically different subclones with thorough validations. On spatial transcriptomics data from various primary and metastasized tumors, Clonalscope labeled malignant spots, traced subclones and identified spatially segregated subclones with distinct differentiation levels and expression of genes associated with drug resistance and survival. Clonalscope is a method for cancer subclone detection leveraging copy number profiles estimated using spatial and single-cell sequencing data.},
  archive      = {J_NMETH},
  author       = {Wu, Chi-Yun and Rong, Jiazhen and Sathe, Anuja and Hess, Paul R. and Lau, Billy T. and Grimes, Susan M. and Huang, Sijia and Ji, Hanlee P. and Zhang, Nancy R.},
  doi          = {10.1038/s41592-025-02773-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1846-1856},
  shortjournal = {Nat. Meth.},
  title        = {Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scvi-hub: An actionable repository for model-driven single-cell analysis. <em>NMETH</em>, <em>22</em>(9), 1836-1845. (<a href='https://doi.org/10.1038/s41592-025-02799-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing availability of single-cell omics datasets presents new opportunities for reuse, while challenges in data transfer, normalization and integration remain a barrier. Here we present scvi-hub: a platform for efficiently sharing and accessing single-cell omics datasets using pretrained probabilistic models. It enables immediate execution of fundamental tasks like visualization, imputation, annotation and deconvolution on new query datasets using state-of-the-art methods, with massively reduced storage and compute requirements. We show that pretrained models support efficient analysis of large references, including the CZI CELLxGENE Discover Census. Scvi-hub is built within the scvi-tools open-source environment and integrated into scverse. Scvi-hub offers a scalable and user-friendly framework for accessing and contributing to a growing ecosystem of ready-to-use models and datasets, thus putting the power of atlas-level analysis at the fingertips of a broad community of users. Scvi-hub is a versatile and efficient platform for model-based analysis of single-cell sequencing studies with access to a diverse array of datasets and downstream analysis.},
  archive      = {J_NMETH},
  author       = {Ergen, Can and Pour Amiri, Valeh Valiollah and Kim, Martin and Kronfeld, Ori and Streets, Aaron and Gayoso, Adam and Yosef, Nir},
  doi          = {10.1038/s41592-025-02799-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1836-1845},
  shortjournal = {Nat. Meth.},
  title        = {Scvi-hub: An actionable repository for model-driven single-cell analysis},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues. <em>NMETH</em>, <em>22</em>(9), 1824-1835. (<a href='https://doi.org/10.1038/s41592-025-02774-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) are essential regulators of RNA fate and function. A long-standing challenge in studying RBP regulation has been mapping RNA interactomes within the dynamic transcriptomic landscape, especially in single-cell contexts and primary tissues. Here we introduce MAPIT-seq (modification added to RBP interacting transcript-sequencing), which uses an antibody-directed editing strategy to map genome-wide in situ RBP–RNA interactions and gene expression concurrently. We demonstrate MAPIT-seq’s robustness across multiple RBPs and systematically analyze RNA substrates associated with core polycomb repressive complex 2 (PRC2) components. MAPIT-seq is also applicable to frozen tissue sections, enabling the mapping of RBP roles during brain development. Importantly, we develop high-throughput single-cell MAPIT-seq (scMAPIT-seq) to reveal cell stage-specific RBP regulation. In summary, MAPIT-seq expands multi-omics profiling, providing an effective framework to study post-transcriptional regulation in dynamic biological processes and clinically relevant scenarios. This work presents MAPIT-seq, an antibody-guided RNA editing method that enables co-profiling of RBP-RNA interactions and gene expression at single-cell resolution and in tissue contexts.},
  archive      = {J_NMETH},
  author       = {Cheng, Qi-Xuan and Xie, Gang and Zhang, Xiangyu and Wang, Jie and Ding, Shuangjin and Wu, Yi-Xia and Shi, Ming and Duan, Fei-Fei and Wan, Zi-Li and Wei, Jing-Jia and Xiao, Junyu and Wang, Yangming},
  doi          = {10.1038/s41592-025-02774-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1824-1835},
  shortjournal = {Nat. Meth.},
  title        = {Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A realistic phantom dataset for benchmarking cryo-ET data annotation. <em>NMETH</em>, <em>22</em>(9), 1819-1823. (<a href='https://doi.org/10.1038/s41592-025-02800-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) is a powerful technique for imaging molecular complexes in their native cellular environments. However, identifying the vast majority of molecular species in cellular tomograms remains prohibitively difficult. Machine learning (ML) methods provide an opportunity to automate the annotation process, but algorithm development has been hindered by the lack of large, standardized datasets. Here we present an experimental phantom dataset with comprehensive ground-truth annotations for six molecular species to spur new algorithm development and benchmark existing tools. This annotated dataset is available on the CryoET Data Portal with infrastructure to streamline access for methods developers across fields. A standardized, realistic phantom dataset consisting of ground-truth annotations for six diverse molecular species is provided as a community resource for cryo-electron-tomography algorithm benchmarking.},
  archive      = {J_NMETH},
  author       = {Peck, Ariana and Yu, Yue and Schwartz, Jonathan and Cheng, Anchi and Ermel, Utz Heinrich and Hutchings, Joshua and Kandel, Saugat and Kimanius, Dari and Montabana, Elizabeth A. and Serwas, Daniel and Siems, Hannah and Wang, Feng and Zhao, Zhuowen and Zheng, Shawn and Haury, Matthias and Agard, David A. and Potter, Clinton S. and Carragher, Bridget and Harrington, Kyle and Paraan, Mohammadreza},
  doi          = {10.1038/s41592-025-02800-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1819-1823},
  shortjournal = {Nat. Meth.},
  title        = {A realistic phantom dataset for benchmarking cryo-ET data annotation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the physical hallmarks of cancer. <em>NMETH</em>, <em>22</em>(9), 1800-1818. (<a href='https://doi.org/10.1038/s41592-024-02564-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The physical microenvironment plays a crucial role in tumor development, progression, metastasis and treatment. Recently, we proposed four physical hallmarks of cancer, with distinct origins and consequences, to characterize abnormalities in the physical tumor microenvironment: (1) elevated compressive–tensile solid stresses, (2) elevated interstitial fluid pressure and the resulting interstitial fluid flow, (3) altered material properties (for example, increased tissue stiffness) and (4) altered physical micro-architecture. As this emerging field of physical oncology is being advanced by tumor biologists, cell and developmental biologists, engineers, physicists and oncologists, there is a critical need for model systems and measurement tools to mechanistically probe these physical hallmarks. Here, after briefly defining these physical hallmarks, we discuss the tools and model systems available for probing each hallmark in vitro, ex vivo, in vivo and in clinical settings. We finally review the unmet needs for mechanistic probing of the physical hallmarks of tumors and discuss the challenges and unanswered questions associated with each hallmark. This Review discusses the current challenges and the available methods to study the four physical hallmarks of cancer.},
  archive      = {J_NMETH},
  author       = {Nia, Hadi T. and Munn, Lance L. and Jain, Rakesh K.},
  doi          = {10.1038/s41592-024-02564-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1800-1818},
  shortjournal = {Nat. Meth.},
  title        = {Probing the physical hallmarks of cancer},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes. <em>NMETH</em>, <em>22</em>(9), 1788-1799. (<a href='https://doi.org/10.1038/s41592-025-02807-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbes within tumors have been recognized and experimentally related to oncogenesis, tumor growth, metastasis and therapeutic responsiveness. Studying the tumor microbiome presents difficulties, as early indications suggest that microbe populations are low in abundance, sparse and highly heterogeneous. Disparate results from computational profiling of the tumor microbiome have cast doubt on the premise of microbes in tumors. Yet decades of experimental evidence support the presence of tumor microbes, at least in a limited number of tumor types. In this Perspective, we discuss the importance of iteratively improving microbe-targeted sequencing techniques, established analytical pipelines, robust computational tools and solid validations to address current challenges and fill existing knowledge gaps. The vast amount of human tumor sequencing data available could greatly enhance systematic investigations of microbiome–tumor interactions with methods to quantify the composition of the tumor microbiome accurately. This Perspective explores the challenges and future directions in the world of human tumor microbiome research.},
  archive      = {J_NMETH},
  author       = {Li, Yingjie and Ma, Anjun and Johnson, Evan and Eng, Charis and De, Subhajyoti and Jiang, Sizun and Li, Zihai and Spakowicz, Daniel and Ma, Qin},
  doi          = {10.1038/s41592-025-02807-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1788-1799},
  shortjournal = {Nat. Meth.},
  title        = {The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research. <em>NMETH</em>, <em>22</em>(9), 1776-1787. (<a href='https://doi.org/10.1038/s41592-025-02769-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vitro tumor models are essential tools for cancer research, offering key insights into not only tumor biology but also therapeutic responses. The transition from traditional two-dimensional to three-dimensional organoid systems marks a paradigm shift in cancer modeling. Although two-dimensional models have been instrumental in elucidating fundamental molecular and genetic mechanisms, they fail to accurately replicate the intricate three-dimensional architecture and dynamic microenvironment characteristic of human tumors. Here we outline how advanced organoid technologies now enable more faithful recapitulation of tumor heterogeneity that better mimic native tissue mechanics and biochemistry. We discuss emerging methods, including air–liquid interface cultures, microfluidic tumor-on-a-chip devices and high-content imaging integrated with machine learning, which collectively address longstanding challenges such as matrix variability and the limited incorporation of immune and vascular elements. These innovations promise to enhance reproducibility and scalability while providing unprecedented insights into tumor biology, cancer progression and therapeutic strategies. This Perspective discusses recent progresses in development of in vitro tumor models and the current challenges in this field.},
  archive      = {J_NMETH},
  author       = {Rauner, Gat and Gupta, Piyush B. and Kuperwasser, Charlotte},
  doi          = {10.1038/s41592-025-02769-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1776-1787},
  shortjournal = {Nat. Meth.},
  title        = {From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible annotation of t cell subsets and activation states with gene expression programs. <em>NMETH</em>, <em>22</em>(9), 1774-1775. (<a href='https://doi.org/10.1038/s41592-025-02792-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created T-CellAnnoTator (TCAT), a computational method that helps to identify T cell subsets, activation states and functions. It does this using reproducible gene expression programs found across many disease contexts and tissues. TCAT outperforms conventional approaches for T cell subset prediction, is easy to use programmatically or through a website, and can be adapted for other cell types.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02792-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1774-1775},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible annotation of t cell subsets and activation states with gene expression programs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A miniature two-photon microscope detects multiple signals deep in the brains of moving mice. <em>NMETH</em>, <em>22</em>(9), 1772-1773. (<a href='https://doi.org/10.1038/s41592-025-02779-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A lightweight miniature two-photon microscope features multi-wavelength excitation, correction of aberrations, and interchangeable objectives for scalable fields of view. It enables multicolor, deep-brain and scalable neural imaging in freely moving mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02779-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1772-1773},
  shortjournal = {Nat. Meth.},
  title        = {A miniature two-photon microscope detects multiple signals deep in the brains of moving mice},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unravelling cellular interactions using flow cytometry. <em>NMETH</em>, <em>22</em>(9), 1770-1771. (<a href='https://doi.org/10.1038/s41592-025-02743-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a cost-effective ultra-high-throughput cytometry-based framework for the detection of physical interactions between cells, along with the characterization of complex cellular landscapes. Application of our approach can offer a systems-level understanding of immunity and facilitate study of the kinetics, mode of action and personalized response prediction of immunotherapies.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02743-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1770-1771},
  shortjournal = {Nat. Meth.},
  title        = {Unravelling cellular interactions using flow cytometry},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles. <em>NMETH</em>, <em>22</em>(9), 1768-1769. (<a href='https://doi.org/10.1038/s41592-025-02775-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present MAPIT-seq, a method that uses antibody-directed RNA editing to concurrently profile in situ RNA-binding protein (RBP)–RNA interactions and transcriptome-wide gene expression in limited input material, including single cells and frozen tissues. This dual-omic strategy streamlines mechanistic analyses of post-transcriptional regulation in dynamic biological processes and clinically relevant samples.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02775-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1768-1769},
  shortjournal = {Nat. Meth.},
  title        = {MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bridging histology and spatial gene expression across scales. <em>NMETH</em>, <em>22</em>(9), 1765-1767. (<a href='https://doi.org/10.1038/s41592-025-02806-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two deep-learning frameworks — GHIST and iSCALE — turn routine histology images into a rich molecular resource, and predict spatial gene expression at single-cell resolution (GHIST) and at super-resolution across large tissue sections (iSCALE), for scalable, data-driven tissue biology.},
  archive      = {J_NMETH},
  author       = {Ma, Ying},
  doi          = {10.1038/s41592-025-02806-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1765-1767},
  shortjournal = {Nat. Meth.},
  title        = {Bridging histology and spatial gene expression across scales},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Taming cancer evolution surprises. <em>NMETH</em>, <em>22</em>(9), 1760-1764. (<a href='https://doi.org/10.1038/s41592-025-02802-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a focus on the evolutionary forces that shape cancer cell behavior, some researchers work out ways to outmaneuver cancer.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02802-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1760-1764},
  shortjournal = {Nat. Meth.},
  title        = {Taming cancer evolution surprises},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SNAP-tag2 improves live-cell imaging. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02830-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02830-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {SNAP-tag2 improves live-cell imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optical pooled screens shed light on ebola infection. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02831-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kamenova, Ivanka},
  doi          = {10.1038/s41592-025-02831-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {Optical pooled screens shed light on ebola infection},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ViSOR puts the periphery in focus. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02828-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Willson, Joseph},
  doi          = {10.1038/s41592-025-02828-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {ViSOR puts the periphery in focus},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Benchmarking genomic language models. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02829-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-025-02829-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking genomic language models},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deciphering regulatory sequence grammar. <em>NMETH</em>, <em>22</em>(9), 1757. (<a href='https://doi.org/10.1038/s41592-025-02827-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leveraging data generated by UUATAC-seq, a method for cross-species chromatin accessibility profiling, the deep learning model NvwaCE deciphers cis-regulatory grammar from DNA sequence.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-025-02827-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1757},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering regulatory sequence grammar},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cichlid fishes. <em>NMETH</em>, <em>22</em>(9), 1755-1756. (<a href='https://doi.org/10.1038/s41592-025-02809-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cichlid fishes are a family of thousands of recently evolved species. As charismatic laboratory models, they are useful for studying anatomical, physiological and behavioral traits that vary across these closely related species.},
  archive      = {J_NMETH},
  author       = {Juntti, Scott A.},
  doi          = {10.1038/s41592-025-02809-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1755-1756},
  shortjournal = {Nat. Meth.},
  title        = {Cichlid fishes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Road trip home to start a lab. <em>NMETH</em>, <em>22</em>(9), 1754. (<a href='https://doi.org/10.1038/s41592-025-02801-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As a postdoctoral fellowship ends, for some, next is a trek to their new lab.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02801-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1754},
  shortjournal = {Nat. Meth.},
  title        = {Road trip home to start a lab},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A tale of tumors. <em>NMETH</em>, <em>22</em>(9), 1753. (<a href='https://doi.org/10.1038/s41592-025-02838-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advances for studying the biology of tumors have expanded our understanding of cancer.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02838-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1753},
  shortjournal = {Nat. Meth.},
  title        = {A tale of tumors},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
