<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth">NMETH - 33</h2>
<ul>
<li><details>
<summary>
(2025). HippoMaps: Multiscale cartography of human hippocampal organization. <em>NMETH</em>, <em>22</em>(10), 2211-2222. (<a href='https://doi.org/10.1038/s41592-025-02783-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The hippocampus has a specialized microarchitecture, is situated at the nexus of multiple macroscale functional networks, contributes to numerous cognitive as well as affective processes and is highly susceptible to brain pathology across common disorders. These features make the hippocampus a model to understand how brain structure covaries with function, in both health and disease. Here we introduce HippoMaps, an open access toolbox and online data warehouse for the mapping and contextualization of subregional hippocampal data in the human brain ( http://hippomaps.readthedocs.io ). HippoMaps capitalizes on a unified hippocampal unfolding approach as well as shape intrinsic registration capabilities to allow for cross-participant and cross-modal data aggregation. We initialize this repository with a combination of hippocampal data spanning three-dimensional ex vivo histology, ex vivo 9.4-Tesla magnetic resonance imaging (MRI), as well as in vivo structural MRI and resting-state functional MRI obtained at 3 Tesla and 7 Tesla, together with intracranial encephalography recordings in patients with epilepsy. All code, data and tools are openly available online, with the aim of fostering further community contributions. HippoMaps provides an open-source resource for studying the human hippocampus at different scales and with different modalities such as histology, fMRI, structural MRI and EEG.},
  archive      = {J_NMETH},
  author       = {DeKraker, Jordan and Cabalo, Donna Gift and Royer, Jessica and Ngo, Alexander and Khan, Ali R. and Karat, Bradley G. and Benkarim, Oualid and Rodriguez-Cruces, Raul and Frauscher, Birgit and Pana, Raluca and Hansen, Justine Y. and Misic, Bratislav and Valk, Sofie L. and Lau, Jonathan C. and Kirschner, Matthias and Bernasconi, Andrea and Bernasconi, Neda and Muenzing, Sascha E. A. and Axer, Markus and Amunts, Katrin and Evans, Alan C. and Bernhardt, Boris C.},
  doi          = {10.1038/s41592-025-02783-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2211-2222},
  shortjournal = {Nat. Meth.},
  title        = {HippoMaps: Multiscale cartography of human hippocampal organization},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatiotemporal focusing enables all-optical in situ histology of heterogeneous tissue. <em>NMETH</em>, <em>22</em>(10), 2205-2210. (<a href='https://doi.org/10.1038/s41592-025-02849-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Living systems embody heterogeneous tissues with complex opto-mechanical properties. Achieving organ-scale, diffraction-limited volumetric imaging that faithfully captures in vivo architecture requires minimizing sample deformation and preserving vascular and neuronal continuity across delicate tissue interfaces. As a solution to this problem, we developed a robotic nonlinear optical system for iterative multiphoton microscopy and opto-micromachining. Adaptive control enabled days-long autonomous operation, while spatiotemporal line-focused ablation increased the machining efficiency by 100-fold over prior configurations. Using the intact murine craniocerebral system as a test bed, our approach demonstrates the potential for whole-body submicrometer resolution imaging and anatomical reconstruction. Laser ablation of imaged surface layers under adaptive control enables volumetric imaging of samples consisting of heterogeneous tissues, such as the skull and brain.},
  archive      = {J_NMETH},
  author       = {Ji, Xiang and Huang, Sincheng and Friedman, Beth and Kleinfeld, David},
  doi          = {10.1038/s41592-025-02849-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2205-2210},
  shortjournal = {Nat. Meth.},
  title        = {Spatiotemporal focusing enables all-optical in situ histology of heterogeneous tissue},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Kilohertz volumetric imaging of in vivo dynamics using squeezed light field microscopy. <em>NMETH</em>, <em>22</em>(10), 2194-2204. (<a href='https://doi.org/10.1038/s41592-025-02843-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volumetric functional imaging of transient cellular signaling and motion dynamics is often limited by hardware bandwidth and the scarcity of photons under short exposures. To overcome these challenges, we introduce squeezed light field microscopy (SLIM), a computational imaging approach that rapidly captures high-resolution three-dimensional light signals using only a single, low-format camera sensor. SLIM records over 1,000 volumes per second across a 550-µm diameter field of view and 300-µm depth, achieving 3.6-µm lateral and 6-µm axial resolution. Here we demonstrate its utility in blood cell velocimetry within the embryonic zebrafish brain and in freely moving tails undergoing high-frequency swings. Millisecond-scale temporal resolution further enables precise voltage imaging of neural membrane potentials in the leech ganglion and hippocampus of behaving mice. Together, these results establish SLIM as a versatile and robust tool for high-speed volumetric microscopy across diverse biological systems. Squeezed light field microscopy (SLIM) combines ideas from tomography and compressed sensing with light field microscopy to enable volumetric imaging at kilohertz rates, as demonstrated in blood flow imaging in zebrafish and voltage imaging in leeches and mice.},
  archive      = {J_NMETH},
  author       = {Wang, Zhaoqiang and Zhao, Ruixuan and Wagenaar, Daniel A. and Espino, Diego and Sheintuch, Liron and Benshlomo, Ohr and Kang, Wenjun and Zhu, Enbo and Lee, Calvin K. and Schmidt, William C. and Pammar, Aryan and Wang, Jing and Wong, Gerard C. L. and Liang, Rongguang and Golshani, Peyman and Hsiai, Tzung K. and Gao, Liang},
  doi          = {10.1038/s41592-025-02843-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2194-2204},
  shortjournal = {Nat. Meth.},
  title        = {Kilohertz volumetric imaging of in vivo dynamics using squeezed light field microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Highly multiplexed 3D profiling of cell states and immune niches in human tumors. <em>NMETH</em>, <em>22</em>(10), 2180-2193. (<a href='https://doi.org/10.1038/s41592-025-02824-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diseases such as cancer involve alterations in cell proportions, states and interactions, as well as complex changes in tissue morphology and architecture. Histopathological diagnosis of disease and most multiplexed spatial profiling relies on inspecting thin (4–5 µm) specimens. Here we describe a high-plex cyclic immunofluorescence method for three-dimensional tissue imaging and use it to show that few, if any, cells are intact in conventional thin tissue sections, reducing the accuracy of cell phenotyping and interaction analysis. However, three-dimensional cyclic immunofluorescence of sections eightfold to tenfold thicker enables accurate morphological assessment of diverse protein markers in intact tumor, immune and stromal cells. Moreover, the high resolution of this confocal approach generates images of cells in a preserved tissue environment at a level of detail previously limited to cell culture. Precise imaging of cell membranes also makes it possible to detect and map cell–cell contacts and juxtracrine signaling complexes in immune cell niches. Confocal microscopy enables high-resolution, high-plex 3D cyclic immunofluorescence of 30- to 50-µm-thick tissue sections. The approach allows for rich phenotypic assessments of intact cells and intercellular interactions with subcellular resolution.},
  archive      = {J_NMETH},
  author       = {Yapp, Clarence and Nirmal, Ajit J. and Zhou, Felix and Wong, Alex Y. H. and Tefft, Juliann B. and Lu, Yi Daniel and Shang, Zhiguo and Maliga, Zoltan and Llopis, Paula Montero and Murphy, George F. and Lian, Christine G. and Danuser, Gaudenz and Santagata, Sandro and Sorger, Peter K.},
  doi          = {10.1038/s41592-025-02824-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2180-2193},
  shortjournal = {Nat. Meth.},
  title        = {Highly multiplexed 3D profiling of cell states and immune niches in human tumors},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fourier-based three-dimensional multistage transformer for aberration correction in multicellular specimens. <em>NMETH</em>, <em>22</em>(10), 2171-2179. (<a href='https://doi.org/10.1038/s41592-025-02844-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-resolution tissue imaging is often compromised by sample-induced optical aberrations that degrade resolution and contrast. Although wavefront sensor-based adaptive optics (AO) can measure these aberrations, such hardware solutions are typically complex, expensive to implement and slow when serially mapping spatially varying aberrations across large fields of view. Here we introduce AOViFT (adaptive optical vision Fourier transformer)—a machine learning-based aberration sensing framework built around a three-dimensional multistage vision transformer that operates on Fourier domain embeddings. AOViFT infers aberrations and restores diffraction-limited performance in puncta-labeled specimens with substantially reduced computational cost, training time and memory footprint compared to conventional architectures or real-space networks. We validated AOViFT on live gene-edited zebrafish embryos, demonstrating its ability to correct spatially varying aberrations using either a deformable mirror or postacquisition deconvolution. By eliminating the need for the guide star and wavefront sensing hardware and simplifying the experimental workflow, AOViFT lowers technical barriers for high-resolution volumetric microscopy across diverse biological samples. Adaptive optical vision Fourier transformer (AOViFT) is a machine learning-based framework for accurately inferring aberrations and restoring diffraction-limited performance in diverse biological specimens.},
  archive      = {J_NMETH},
  author       = {Alshaabi, Thayer and Milkie, Daniel E. and Liu, Gaoxiang and Shirazinejad, Cyna and Hong, Jason L. and Achour, Kemal and Görlitz, Frederik and Milunovic-Jevtic, Ana and Simmons, Cat and Abuzahriyeh, Ibrahim S. and Hong, Erin and Williams, Samara Erin and Harrison, Nathanael and Huang, Evan and Bae, Eun Seok and Killilea, Alison N. and Swinburne, Ian A. and Drubin, David G. and Upadhyayula, Srigokul and Betzig, Eric},
  doi          = {10.1038/s41592-025-02844-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2171-2179},
  shortjournal = {Nat. Meth.},
  title        = {Fourier-based three-dimensional multistage transformer for aberration correction in multicellular specimens},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Automated classification of cellular expression in multiplexed imaging data with nimbus. <em>NMETH</em>, <em>22</em>(10), 2161-2170. (<a href='https://doi.org/10.1038/s41592-025-02826-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed imaging offers a powerful approach to characterize the spatial topography of tissues in both health and disease. To analyze such data, the specific combination of markers that are present in each cell must be enumerated to enable accurate phenotyping, a process that often relies on unsupervised clustering. We constructed the Pan-Multiplex (Pan-M) dataset containing 197 million distinct annotations of marker expression across 15 different cell types. We used Pan-M to create Nimbus, a deep learning model to predict marker positivity from multiplexed image data. Nimbus is a pretrained model that uses the underlying images to classify marker expression of individual cells as positive or negative across distinct cell types, from different tissues, acquired using different microscope platforms, without requiring any retraining. We demonstrate that Nimbus predictions capture the underlying staining patterns of the full diversity of markers present in Pan-M, and that Nimbus matches or exceeds the accuracy of previous approaches that must be retrained on each dataset. We then show how Nimbus predictions can be integrated with downstream clustering algorithms to robustly identify cell subtypes in image data. We have open-sourced Nimbus and Pan-M to enable community use at https://github.com/angelolab/Nimbus-Inference . Nimbus, a deep learning model, uses a large multiplexed imaging dataset to predict the likelihood of marker positivity in single cells.},
  archive      = {J_NMETH},
  author       = {Rumberger, Josef Lorenz and Greenwald, Noah F. and Ranek, Jolene S. and Boonrat, Potchara and Walker, Cameron and Franzen, Jannik and Varra, Sricharan Reddy and Kong, Alex and Sowers, Cameron and Liu, Candace C. and Averbukh, Inna and Piyadasa, Hadeesha and Vanguri, Rami and Nederlof, Iris and Wang, Xuefei Julie and Van Valen, David and Kok, Marleen and Bendall, Sean C. and Hollmann, Travis J. and Kainmueller, Dagmar and Angelo, Michael},
  doi          = {10.1038/s41592-025-02826-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2161-2170},
  shortjournal = {Nat. Meth.},
  title        = {Automated classification of cellular expression in multiplexed imaging data with nimbus},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Foundation model for efficient biological discovery in single-molecule time traces. <em>NMETH</em>, <em>22</em>(10), 2149-2160. (<a href='https://doi.org/10.1038/s41592-025-02839-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule fluorescence microscopy (SMFM) can reveal important biological insights. However, uncovering rare but critical intermediates often demands manual inspection of time traces and iterative ad hoc approaches. To facilitate systematic and efficient discovery from SMFM time traces, we introduce META-SiM, a transformer-based foundation model pretrained on diverse SMFM analysis tasks. META-SiM rivals best-in-class algorithms on a broad range of tasks including trace classification, segmentation, idealization and stepwise photobleaching analysis. Additionally, the model produces embeddings that encapsulate detailed information about each trace, which the web-based META-SiM Projector ( https://www.simol-projector.org ) casts into lower-dimensional space for efficient whole-dataset visualization, labeling, comparison and sharing. Combining this Projector with the objective metric of local Shannon entropy enables rapid identification of condition-specific behaviors, even if rare or subtle. Applying META-SiM to an existing single-molecule Förster resonance energy transfer dataset, we discover a previously undetected intermediate state in pre-mRNA splicing. META-SiM removes bottlenecks, improves objectivity and both systematizes and accelerates biological discovery in single-molecule data. META-SiM brings foundation model power to single-molecule time traces, excelling across diverse analysis tasks. Paired with the web-based META-SiM Projector and entropy mapping, it rapidly reveals hidden molecular behaviors inaccessible by other means.},
  archive      = {J_NMETH},
  author       = {Li, Jieming and Zhang, Leyou and Johnson-Buck, Alexander and Walter, Nils G.},
  doi          = {10.1038/s41592-025-02839-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2149-2160},
  shortjournal = {Nat. Meth.},
  title        = {Foundation model for efficient biological discovery in single-molecule time traces},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dose-efficient cryo-electron microscopy for thick samples using tilt- corrected scanning transmission electron microscopy. <em>NMETH</em>, <em>22</em>(10), 2138-2148. (<a href='https://doi.org/10.1038/s41592-025-02834-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron microscopy is a powerful tool in structural biology. In thick specimens, challenges arise as an exponentially larger fraction of the transmitted electrons lose energy from inelastic scattering and can no longer be properly focused as a result of chromatic aberrations in the post-specimen optics. Rather than filtering out the inelastic scattering at the price of reducing potential signal, as is done in energy-filtered transmission electron microscopy, we show how a dose-efficient and unfiltered image can be rapidly obtained using tilt-corrected bright-field scanning transmission electron microscopy data collected on a pixelated detector. Enhanced contrast and a 3–5× improvement in dose efficiency are observed for two-dimensional images of intact bacterial cells and large organelles using tilt-corrected bright-field scanning transmission electron microscopy compared to energy-filtered transmission electron microscopy for thicknesses beyond 500 nm. As a proof of concept for the technique’s performance in structural determination, we present a single-particle analysis map at sub-nanometer resolution for a highly symmetric virus-like particle determined from 789 particles. Tilt-corrected bright-field scanning transmission electron microscopy offers enhanced cryogenic electron microscopy contrast and substantial improvement in dose efficiency for thick samples such as bacterial cells and large organelles, while still being able to perform single-particle analysis.},
  archive      = {J_NMETH},
  author       = {Yu, Yue and Spoth, Katherine A. and Colletta, Michael and Nguyen, Kayla X. and Zeltmann, Steven E. and Zhang, Xiyue S. and Paraan, Mohammadreza and Kopylov, Mykhailo and Dubbeldam, Charlie and Serwas, Daniel and Siems, Hannah and Muller, David A. and Kourkoutis, Lena F.},
  doi          = {10.1038/s41592-025-02834-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2138-2148},
  shortjournal = {Nat. Meth.},
  title        = {Dose-efficient cryo-electron microscopy for thick samples using tilt- corrected scanning transmission electron microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Uncovering hidden protein modifications with native top-down mass spectrometry. <em>NMETH</em>, <em>22</em>(10), 2127-2137. (<a href='https://doi.org/10.1038/s41592-025-02846-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein modifications drive dynamic cellular processes by modulating biomolecular interactions, yet capturing these modifications within their native structural context remains a significant challenge. Native top-down mass spectrometry promises to preserve the critical link between modifications and interactions. However, current methods often fail to detect uncharacterized or low-abundance modifications, limiting insights into proteoform diversity. To address this gap, we introduce precise and accurate Identification Of Native proteoforms (precisION), an interactive end-to-end software package that leverages a robust, data-driven fragment-level open search to detect, localize and quantify ‘hidden’ modifications within intact protein complexes. Applying precisION to four therapeutically relevant targets—PDE6, ACE2, osteopontin (SPP1) and a GABA transporter (GAT1)—we discover undocumented phosphorylation, glycosylation and lipidation, and resolve previously uninterpretable density in an electron cryo-microscopy map of GAT1. As an open-source software package, precisION offers an intuitive means for interpreting complex protein fragmentation data. This tool will empower the community to unlock the potential of native top-down mass spectrometry, advancing integrative structural biology, molecular pathology and drug development. precisION discovers, localizes and quantifies protein modifications within complex proteoform assemblies through data-driven analysis of native top-down mass spectra.},
  archive      = {J_NMETH},
  author       = {Bennett, Jack L. and El-Baba, Tarick J. and Zouboulis, Konstantin C. and Kirschbaum, Carla and Song, Haigang and Butroid, Frances I. and Benesch, Justin L. P. and Lutomski, Corinne A. and Robinson, Carol V.},
  doi          = {10.1038/s41592-025-02846-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2127-2137},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering hidden protein modifications with native top-down mass spectrometry},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Merging conformational landscapes in a single consensus space with FlexConsensus algorithm. <em>NMETH</em>, <em>22</em>(10), 2118-2126. (<a href='https://doi.org/10.1038/s41592-025-02841-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural heterogeneity analysis in cryogenic electron microscopy is experiencing a breakthrough in estimating more accurate, richer and interpretable conformational landscapes derived from experimental data. The emergence of new methods designed to tackle the heterogeneity challenge reflects this new paradigm, enabling users to gain a better understanding of protein dynamics. However, the question of how intrinsically different heterogeneity algorithms compare remains unsolved, which is crucial for determining the reliability, stability and correctness of the estimated conformational landscapes. Here, to overcome the previous challenge, we introduce FlexConsenus: a multi-autoencoder neural network able to learn the commonalities and differences among several conformational landscapes, enabling them to be placed in a shared consensus space with enhanced reliability. The consensus space enables the measurement of reproducibility in heterogeneity estimations, allowing users to either focus their analysis on particles with a stable estimation of their structural variability or concentrate on specific particle subsets detected by only certain methods. FlexConsensus is a multi-autoencoder-based algorithm for merging different conformational landscapes from cryogenic electron microscopy heterogeneity analysis into a common latent space for the identification of similarities and differences among various methods. This helps in the validation of estimated conformational landscape and provides tools to streamline the heterogeneity workflow.},
  archive      = {J_NMETH},
  author       = {Herreros, David and Perez Mata, Carlos and Sanchez Sorzano, Carlos Oscar and Carazo, Jose Maria},
  doi          = {10.1038/s41592-025-02841-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2118-2126},
  shortjournal = {Nat. Meth.},
  title        = {Merging conformational landscapes in a single consensus space with FlexConsensus algorithm},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). InterPLM: Discovering interpretable features in protein language models via sparse autoencoders. <em>NMETH</em>, <em>22</em>(10), 2107-2117. (<a href='https://doi.org/10.1038/s41592-025-02836-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite their success in protein modeling and design, the internal mechanisms of protein language models (PLMs) are poorly understood. Here we present a systematic framework to extract and analyze interpretable features from PLMs using sparse autoencoders. Training sparse autoencoders on ESM-2 embeddings, we identify thousands of interpretable features highlighting biological concepts including binding sites, structural motifs and functional domains. Individual neurons show considerably less conceptual alignment, suggesting PLMs store concepts in superposition. This superposition persists across model scales and larger PLMs capture more interpretable concepts. Beyond known annotations, ESM-2 learns coherent patterns across evolutionarily distinct protein families. To systematically analyze these numerous features, we developed an automated interpretation approach using large language models for feature description and validation. As practical applications, these features can accurately identify missing database annotations and enable targeted steering of sequence generation. Our results show PLM representations can be decomposed into interpretable components, demonstrating the feasibility and utility of mechanistically interpreting these models. InterPLM is a computational framework to extract and analyze interpretable features from protein language models using sparse autoencoders. By training sparse autoencoders on ESM-2 embeddings, this study identifies thousands of interpretable biological features learned by the different layers of the ESM-2 model.},
  archive      = {J_NMETH},
  author       = {Simon, Elana and Zou, James},
  doi          = {10.1038/s41592-025-02836-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2107-2117},
  shortjournal = {Nat. Meth.},
  title        = {InterPLM: Discovering interpretable features in protein language models via sparse autoencoders},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). All-at-once RNA folding with 3D motif prediction framed by evolutionary information. <em>NMETH</em>, <em>22</em>(10), 2094-2106. (<a href='https://doi.org/10.1038/s41592-025-02833-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural RNAs exhibit a vast array of recurrent short three-dimensional (3D) elements found in loop regions involving non-Watson–Crick interactions that help arrange canonical double helices into tertiary structures. Here we present CaCoFold-R3D, a probabilistic grammar that predicts these RNA 3D motifs (also termed modules) jointly with RNA secondary structure over a sequence or alignment. CaCoFold-R3D uses evolutionary information present in an RNA alignment to reliably identify canonical helices (including pseudoknots) by covariation. Here we further introduce the R3D grammars, which also exploit helix covariation that constrains the positioning of the mostly noncovarying RNA 3D motifs. Our method runs predictions over an almost-exhaustive list of over 50 known RNA motifs (‘everything’). Motifs can appear in any nonhelical loop region (including three-way, four-way and higher junctions) (‘everywhere’). All structural motifs as well as the canonical helices are arranged into one single structure predicted by one single joint probabilistic grammar (‘all-at-once’). Our results demonstrate that CaCoFold-R3D is a valid alternative for predicting the all-residue interactions present in a RNA 3D structure. CaCoFold-R3D is fast and easily customizable for novel motif discovery and shows promising value both as a strong input for deep learning approaches to all-atom structure prediction as well as toward guiding RNA design as drug targets for therapeutic small molecules. CaCoFold-R3D is a probabilistic model that simultaneously predicts the RNA 3D motifs jointly with the secondary structure in a structural RNA using evolutionary information.},
  archive      = {J_NMETH},
  author       = {Karan, Aayush and Rivas, Elena},
  doi          = {10.1038/s41592-025-02833-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2094-2106},
  shortjournal = {Nat. Meth.},
  title        = {All-at-once RNA folding with 3D motif prediction framed by evolutionary information},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Coupling CRISPR scanning with targeted chromatin accessibility profiling using a double-stranded DNA deaminase. <em>NMETH</em>, <em>22</em>(10), 2083-2093. (<a href='https://doi.org/10.1038/s41592-025-02811-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing enables sequence-function profiling of endogenous cis-regulatory elements, driving understanding of their mechanisms. However, these approaches lack direct, scalable readouts of chromatin accessibility across long single-molecule chromatin fibers. Here we leverage double-stranded DNA cytidine deaminases to profile chromatin accessibility at endogenous loci of interest through targeted PCR and long-read sequencing, a method we term targeted deaminase-accessible chromatin sequencing (TDAC-seq). With high sequence coverage at targeted loci, TDAC-seq can be integrated with CRISPR perturbations to link genetic edits and their effects on chromatin accessibility on the same single chromatin fiber at single-nucleotide resolution. We employed TDAC-seq to parse CRISPR edits that activate fetal hemoglobin in human CD34+ hematopoietic stem and progenitor cells (HSPCs) during erythroid differentiation as well as in pooled CRISPR and base-editing screens tiling an enhancer controlling the globin locus. We further scaled the method to interrogate 947 variants in a GFI1B-linked enhancer associated with myeloproliferative neoplasm risk in a single pooled CRISPR experiment in CD34+ HSPCs. Together, TDAC-seq enables high-resolution sequence-function mapping of single-molecule chromatin fibers by genome editing. This paper presents TDAC-seq, a targeted chromatin-accessibility-profiling method using cytidine deaminases and long-read sequencing, to resolve the effects of CRISPR edits on single chromatin fibers.},
  archive      = {J_NMETH},
  author       = {Roh, Heejin and Shen, Simon P. and Hu, Yan and Kwok, Hui Si and Siegenfeld, Allison P. and Lee, Ceejay and Zepeda, Marcanthony U. and Guo, Chun-Jie and Roseman, Shelby A. and Comenho, Caroline and Sankaran, Vijay G. and Buenrostro, Jason D. and Liau, Brian B.},
  doi          = {10.1038/s41592-025-02811-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2083-2093},
  shortjournal = {Nat. Meth.},
  title        = {Coupling CRISPR scanning with targeted chromatin accessibility profiling using a double-stranded DNA deaminase},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Common to rare transfer learning (CORAL) enables inference and prediction for a quarter million rare malagasy arthropods. <em>NMETH</em>, <em>22</em>(10), 2074-2082. (<a href='https://doi.org/10.1038/s41592-025-02823-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA-based biodiversity surveys result in massive-scale data, including up to millions of species—of which, most are rare. Making the most of such data for inference and prediction requires modeling approaches that can relate species occurrences to environmental and spatial predictors, while incorporating information about their taxonomic or phylogenetic placement. Even if the scalability of joint species distribution models to large communities has greatly advanced, incorporating hundreds of thousands of species has not been feasible to date, leading to compromised analyses. Here we present a ‘common to rare transfer learning’ (CORAL) approach, based on borrowing information from the common species to enable statistically and computationally efficient modeling of both common and rare species. We illustrate that CORAL leads to much improved prediction and inference in the context of DNA metabarcoding data from Madagascar, comprising 255,188 arthropod species detected in 2,874 samples. CORAL can infer the occurrence of rare species based on common species, using DNA metabarcoding data or other high-dimensional biodiversity data. The approach is illustrated on a large-scale biodiversity survey from Madagascar.},
  archive      = {J_NMETH},
  author       = {Ovaskainen, Otso and Winter, Steven and Tikhonov, Gleb and Abrego, Nerea and Anslan, Sten and deWaard, Jeremy R. and deWaard, Stephanie L. and Fisher, Brian L. and Furneaux, Brendan and Hardwick, Bess and Kerdraon, Deirdre and Pentinsaari, Mikko and Raharinjanahary, Dimby and Rajoelison, Eric Tsiriniaina and Ratnasingham, Sujeevan and Somervuo, Panu and Sones, Jayme E. and Zakharov, Evgeny V. and Hebert, Paul D. N. and Roslin, Tomas and Dunson, David},
  doi          = {10.1038/s41592-025-02823-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2074-2082},
  shortjournal = {Nat. Meth.},
  title        = {Common to rare transfer learning (CORAL) enables inference and prediction for a quarter million rare malagasy arthropods},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). De novo discovery of conserved gene clusters in microbial genomes with spacedust. <em>NMETH</em>, <em>22</em>(10), 2065-2073. (<a href='https://doi.org/10.1038/s41592-025-02816-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metagenomics has revolutionized environmental and human-associated microbiome studies. However, the limited fraction of proteins with known biological processes and molecular functions presents a major bottleneck. In prokaryotes and viruses, evolution favors keeping genes participating in the same biological processes colocalized as conserved gene clusters. Conversely, conservation of gene neighborhood indicates functional association. Here we present Spacedust, a tool for systematic, de novo discovery of conserved gene clusters. To find homologous protein matches, Spacedust uses fast and sensitive structure comparison with Foldseek. Partially conserved clusters are detected using novel clustering and order conservation P values. We demonstrate Spacedust’s sensitivity with an all-versus-all analysis of 1,308 bacterial genomes, identifying 72,843 conserved gene clusters containing 58% of the 4.2 million genes. It recovered 95% of antiviral defense system clusters annotated by the specialized tool PADLOC. Spacedust’s high sensitivity and speed will facilitate the annotation of large numbers of sequenced bacterial, archaeal and viral genomes. This work presents Spacedust, a tool for de novo identification of conserved gene clusters from metagenomic data.},
  archive      = {J_NMETH},
  author       = {Zhang, Ruoshi and Mirdita, Milot and Söding, Johannes},
  doi          = {10.1038/s41592-025-02816-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2065-2073},
  shortjournal = {Nat. Meth.},
  title        = {De novo discovery of conserved gene clusters in microbial genomes with spacedust},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Giotto suite: A multiscale and technology-agnostic spatial multiomics analysis ecosystem. <em>NMETH</em>, <em>22</em>(10), 2052-2064. (<a href='https://doi.org/10.1038/s41592-025-02817-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emerging spatial multiomics technologies provide an increasingly large amount of information content at multiple scales. However, it remains challenging to efficiently represent and harmonize diverse spatial datasets. Here we present Giotto Suite, a suite of modular packages that provides scalable and extensible end-to-end solutions for multiscale and multiomic data analysis, integration and visualization. At its core, Giotto Suite is centered around an innovative data framework, allowing the representation and integration of spatial omics data in a technology-agnostic manner. Giotto Suite integrates molecular, morphology, spatial and annotated feature information to create a responsive and flexible workflow, as demonstrated by applications to several state-of-the-art spatial technologies. Furthermore, Giotto Suite builds upon interoperable interfaces and data structures that bridge the established fields of genomics and spatial data science in R, thereby enabling independent developers to create custom-engineered pipelines. As such, Giotto Suite creates an immersive and multiscale ecosystem for spatial multiomic data analysis. Giotto Suite provides a comprehensive, flexible and scalable platform for technology-agnostic spatial omics analysis using R.},
  archive      = {J_NMETH},
  author       = {Chen, Jiaji G. and Chávez-Fuentes, Joselyn C. and O’Brien, Matthew and Xu, Junxiang and Ruiz, Edward C. and Wang, Wen and Amin, Iqra and Sheridan, Jeffrey P. and Shin, Sujung C. and Hasyagar, Sanjana V. and Sarfraz, Irzam and Guckhool, Pratishtha and Sistig, Adriana and Jarzabek, Veronica and Yuan, Guo-Cheng and Dries, Ruben},
  doi          = {10.1038/s41592-025-02817-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2052-2064},
  shortjournal = {Nat. Meth.},
  title        = {Giotto suite: A multiscale and technology-agnostic spatial multiomics analysis ecosystem},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell multi-omic detection of DNA methylation and histone modifications reconstructs the dynamics of epigenomic maintenance. <em>NMETH</em>, <em>22</em>(10), 2042-2051. (<a href='https://doi.org/10.1038/s41592-025-02847-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA methylation and histone modifications encode epigenetic information. Recently, major progress was made to measure either mark at a single-cell resolution; however, a method for simultaneous detection is lacking, preventing study of their interactions. Here, to bridge this gap, we developed scEpi2-seq. Our technique provides a readout of histone modifications and DNA methylation at the single-cell and single-molecule level. Application in a cell line with the FUCCI cell cycle reporter system reveals how DNA methylation maintenance is influenced by the local chromatin context. In addition, profiling of H3K27me3 and DNA methylation in the mouse intestine yields insights into epigenetic interactions during cell type specification. Differentially methylated regions also demonstrated independent cell-type regulation in addition to H3K27me3 regulation, which reinforces that CpG methylation acts as an additional layer of control in facultative heterochromatin. This work presents scEpi2-seq, a method for simultaneous single-cell profiling of DNA methylation and histone modifications, enabling direct investigation of the interplay between these two epigenomic marks.},
  archive      = {J_NMETH},
  author       = {Geisenberger, Christoph and van den Berg, Jeroen and van Batenburg, Vincent and de Barbanson, Buys and Lyubimova, Anna and Verity-Legg, Joe and Chen, Xiufei and Liu, Yibin and Song, Chun-Xiao and de Ridder, Jeroen and van Oudenaarden, Alexander},
  doi          = {10.1038/s41592-025-02847-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2042-2051},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell multi-omic detection of DNA methylation and histone modifications reconstructs the dynamics of epigenomic maintenance},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing. <em>NMETH</em>, <em>22</em>(10), 2032-2041. (<a href='https://doi.org/10.1038/s41592-025-02805-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variants (both coding and noncoding) can impact gene function and expression, driving disease mechanisms such as cancer progression. The systematic study of endogenous genetic variants is hindered by inefficient precision editing tools, combined with technical limitations in confidently linking genotypes to gene expression at single-cell resolution. We developed single-cell DNA–RNA sequencing (SDR-seq) to simultaneously profile up to 480 genomic DNA loci and genes in thousands of single cells, enabling accurate determination of coding and noncoding variant zygosity alongside associated gene expression changes. Using SDR-seq, we associate coding and noncoding variants with distinct gene expression in human induced pluripotent stem cells. Furthermore, we demonstrate that in primary B cell lymphoma samples, cells with a higher mutational burden exhibit elevated B cell receptor signaling and tumorigenic gene expression. SDR-seq provides a powerful platform to dissect regulatory mechanisms encoded by genetic variants, advancing our understanding of gene expression regulation and its implications for disease. This study introduces SDR-seq, a droplet-based single-cell DNA–RNA sequencing platform, enabling the study of gene expression profiles linked to both noncoding and coding variants.},
  archive      = {J_NMETH},
  author       = {Lindenhofer, Dominik and Bauman, Julia R. and Hawkins, John A. and Fitzgerald, Donnacha and Yildiz, Umut and Jung, Haeyeon and Korosteleva, Anastasiia and Marttinen, Mikael and Kueblbeck, Moritz and Zaugg, Judith B. and Noh, Kyung-Min and Dietrich, Sascha and Huber, Wolfgang and Stegle, Oliver and Steinmetz, Lars M.},
  doi          = {10.1038/s41592-025-02805-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2032-2041},
  shortjournal = {Nat. Meth.},
  title        = {Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MSnLib: Efficient generation of open multi-stage fragmentation mass spectral libraries. <em>NMETH</em>, <em>22</em>(10), 2028-2031. (<a href='https://doi.org/10.1038/s41592-025-02813-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Untargeted high-resolution mass spectrometry is a key tool in clinical metabolomics, natural product discovery and exposomics, with compound identification remaining the major bottleneck. Currently, the standard workflow applies spectral library matching against tandem mass spectrometry (MS2) fragmentation data. Multi-stage fragmentation (MSn) yields more profound insights into substructures, enabling validation of fragmentation pathways; however, the community lacks open MSn reference data of diverse natural products and other chemicals. Here we describe MSnLib, a machine learning-ready open resource of &gt;2 million spectra in MSn trees of 30,008 unique small molecules, built with a high-throughput data acquisition and processing pipeline in the open-source software mzmine. MSnLib is a large-scale, open MSn spectral library featuring &gt;2.3 million MSn and &gt;357,000 MS2 spectra for 30,008 unique small molecules.},
  archive      = {J_NMETH},
  author       = {Brungs, Corinna and Schmid, Robin and Heuckeroth, Steffen and Mazumdar, Aninda and Drexler, Matúš and Šácha, Pavel and Dorrestein, Pieter C. and Petras, Daniel and Nothias, Louis-Felix and Veverka, Václav and Nencka, Radim and Kameník, Zdeněk and Pluskal, Tomáš},
  doi          = {10.1038/s41592-025-02813-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2028-2031},
  shortjournal = {Nat. Meth.},
  title        = {MSnLib: Efficient generation of open multi-stage fragmentation mass spectral libraries},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). GPU-accelerated homology search with MMseqs2. <em>NMETH</em>, <em>22</em>(10), 2024-2027. (<a href='https://doi.org/10.1038/s41592-025-02819-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapidly growing protein databases demand faster sensitive search tools. Here the graphics processing unit (GPU)-accelerated MMseqs2 delivers 6× faster single-protein searches than CPU methods on 2 × 64 cores, speeds previously requiring large protein batches. For larger query batches, it is the most cost-effective solution, outperforming the fastest alternative method by 2.4-fold with eight GPUs. It accelerates protein structure prediction with ColabFold 31.8× over the standard AlphaFold2 pipeline and protein structure search with Foldseek by 4–27×. MMseqs2-GPU is available under an open-source license at https://mmseqs.com/ . Graphics processing unit-accelerated MMseqs2 offers tremendous speedups for homology retrieval from metagenomic databases, query-centered multiple sequence alignment generation for structure prediction, and structural searches with Foldseek.},
  archive      = {J_NMETH},
  author       = {Kallenborn, Felix and Chacon, Alejandro and Hundt, Christian and Sirelkhatim, Hassan and Didi, Kieran and Cha, Sooyoung and Dallago, Christian and Mirdita, Milot and Schmidt, Bertil and Steinegger, Martin},
  doi          = {10.1038/s41592-025-02819-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2024-2027},
  shortjournal = {Nat. Meth.},
  title        = {GPU-accelerated homology search with MMseqs2},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Machine learning for accelerating discovery from single-molecule data. <em>NMETH</em>, <em>22</em>(10), 2022-2023. (<a href='https://doi.org/10.1038/s41592-025-02840-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Manual analysis of single-molecule time traces is slow and subjective. Now, a transformer-based foundation model — META-SiM —automates key analysis tasks across diverse datasets and enables rapid, systematic discovery of subtle single-molecule behaviors. Application of this approach reveals a previously undetected pre-mRNA splicing intermediate, highlighting its potential to streamline biological discovery.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02840-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2022-2023},
  shortjournal = {Nat. Meth.},
  title        = {Machine learning for accelerating discovery from single-molecule data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Joint profiling of chromatin accessibility and CRISPR edits via double-stranded DNA deaminases. <em>NMETH</em>, <em>22</em>(10), 2020-2021. (<a href='https://doi.org/10.1038/s41592-025-02812-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted deaminase-accessible chromatin sequencing (TDAC-seq) measures chromatin accessibility across long chromatin fibers at targeted loci using double-stranded DNA cytidine deaminases. When combined with pooled CRISPR mutational screening, TDAC-seq enables the high-throughput detection of changes in chromatin accessibility following CRISPR perturbations, allowing fine mapping of sequence–function relationships within endogenous cis-regulatory elements.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02812-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2020-2021},
  shortjournal = {Nat. Meth.},
  title        = {Joint profiling of chromatin accessibility and CRISPR edits via double-stranded DNA deaminases},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What does a language model know about proteins?. <em>NMETH</em>, <em>22</em>(10), 2017-2019. (<a href='https://doi.org/10.1038/s41592-025-02837-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new approach sheds light on the biological features learned by protein language models, promising greater interpretability for unsupervised sequence learning.},
  archive      = {J_NMETH},
  author       = {Ruffolo, Jeffrey A.},
  doi          = {10.1038/s41592-025-02837-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2017-2019},
  shortjournal = {Nat. Meth.},
  title        = {What does a language model know about proteins?},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cryogenic STEM of thick biological specimens. <em>NMETH</em>, <em>22</em>(10), 2015-2016. (<a href='https://doi.org/10.1038/s41592-025-02818-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A cryogenic scanning transmission electron microscopy (STEM) approach for analyzing thick biological specimens expands the reach of cryo-electron microscopy.},
  archive      = {J_NMETH},
  author       = {Sachse, Carsten},
  doi          = {10.1038/s41592-025-02818-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2015-2016},
  shortjournal = {Nat. Meth.},
  title        = {Cryogenic STEM of thick biological specimens},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Choose your human genome reference wisely. <em>NMETH</em>, <em>22</em>(10), 2010-2014. (<a href='https://doi.org/10.1038/s41592-025-02850-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists can choose between multiple human genome references, and a pangenome reference is coming. Deciding what to use when is not quite straightforward.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02850-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2010-2014},
  shortjournal = {Nat. Meth.},
  title        = {Choose your human genome reference wisely},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep-sea imaging of octopus locomotion. <em>NMETH</em>, <em>22</em>(10), 2009. (<a href='https://doi.org/10.1038/s41592-025-02875-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Economou, Andrew},
  doi          = {10.1038/s41592-025-02875-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2009},
  shortjournal = {Nat. Meth.},
  title        = {Deep-sea imaging of octopus locomotion},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Retracing tumor evolution. <em>NMETH</em>, <em>22</em>(10), 2009. (<a href='https://doi.org/10.1038/s41592-025-02876-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-025-02876-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2009},
  shortjournal = {Nat. Meth.},
  title        = {Retracing tumor evolution},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ionic liquids clear tissue without distortion. <em>NMETH</em>, <em>22</em>(10), 2008. (<a href='https://doi.org/10.1038/s41592-025-02873-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Whalley, Katherine},
  doi          = {10.1038/s41592-025-02873-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2008},
  shortjournal = {Nat. Meth.},
  title        = {Ionic liquids clear tissue without distortion},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). BioEmu is a biomolecular emulator for sampling protein structure ensembles. <em>NMETH</em>, <em>22</em>(10), 2008. (<a href='https://doi.org/10.1038/s41592-025-02874-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02874-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2008},
  shortjournal = {Nat. Meth.},
  title        = {BioEmu is a biomolecular emulator for sampling protein structure ensembles},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancing prime editing via AI-guided binder design. <em>NMETH</em>, <em>22</em>(10), 2007. (<a href='https://doi.org/10.1038/s41592-025-02872-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AI-designed binders can enhance prime editing performance by inhibiting the DNA mismatch repair pathway.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-025-02872-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2007},
  shortjournal = {Nat. Meth.},
  title        = {Enhancing prime editing via AI-guided binder design},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Translon: A single term for translated regions. <em>NMETH</em>, <em>22</em>(10), 2002-2006. (<a href='https://doi.org/10.1038/s41592-025-02810-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Świrski, Michał I. and Tierney, Jack A. S. and Albà, M. Mar and Andreev, Dmitry E. and Aspden, Julie L. and Atkins, John F. and Bassani-Sternberg, Michal and Berry, Marla J. and Biffo, Stefano and Boris-Lawrie, Kathleen and Borodovsky, Mark and Brierley, Ian and Brook, Matthew and Brunet, Marie A. and Bujnicki, Janusz M. and Caliskan, Neva and Calviello, Lorenzo and Carvunis, Anne-Ruxandra and Cate, Jamie H. D. and Cenik, Can and Chang, Kung Yao and Chen, Yiwen and Chothani, Sonia and Choudhary, Jyoti S. and Clark, Patricia L. and Clauwaert, Jim and Cooley, Lynn and Dassi, Erik and Dean, Kellie and Diaz, Jean-Jacques and Dieterich, Christoph and Dikstein, Rivka and Dinman, Jonathan D. and Dmitriev, Sergey E. and Dontsova, Olga A. and Dunham, Christine M. and Eswarappa, Sandeep M. and Farabaugh, Philip J. and Faridi, Pouya and Fierro-Monti, Ivo and Firth, Andrew E. and Gatfield, David and Gebauer, Fátima and Gelfand, Mikhail S. and Gray, Nicola K. and Green, Rachel and Hill, Chris H. and Hou, Ya-Ming and Hübner, Norbert and Ignatova, Zoya and Ivanov, Pavel and Iwasaki, Shintaro and Johnson, Rory and Jomaa, Ahmad and Jovanovic, Marko and Jungreis, Irwin and Kellis, Manolis and Kieft, Jeffrey S. and Kochetov, Alex V. and Koonin, Eugene V. and Korostelev, Andrei A. and Kufel, Joanna and Kulakovskiy, Ivan V. and Kurian, Leo and Lafontaine, Denis L. J. and Larsson, Ola and Loughran, Gary and Lukeš, Julius and Mariotti, Marco and Martens-Uzunova, Elena S. and Martinez, Thomas F. and Matsumoto, Akinobu and McManus, Joel and Medenbach, Jan and Melnikov, Sergey V. and Menschaert, Gerben and Merchante, Catharina and Mikl, Martin and Miller, W. Allen and Mühlemann, Oliver and Namy, Olivier and Nedialkova, Danny D. and Nosek, Jozef and Orchard, Sandra and Ozretić, Petar and Pertea, Mihaela and Pervouchine, Dmitri D. and Romão, Luísa and Ron, David and Roucou, Xavier and Rubtsova, Maria P. and Ruiz-Orera, Jorge and Saghatelian, Alan and Salzberg, Steven L. and Seale, Lucia A. and Seoighe, Cathal and Sergiev, Petr V. and Shah, Premal and Shirokikh, Nikolay and Slavoff, Sarah A. and Sonenberg, Nahum and Stasevich, Timothy J. and Szczesny, Roman J. and Tamm, Tiina and Tchórzewski, Marek and Topisirovic, Ivan and Tremblay, Michel L. and Tuller, Tamir and Ulitsky, Igor and Valášek, Leoš Shivaya and Van Damme, Petra and Viero, Gabriella and Vizcaino, Juan Antonio and Vogel, Christine and Wallace, Edward W. J. and Weissman, Jonathan S. and Westhof, Eric and Whiffin, Nicola and Wilson, Daniel N. and Xie, Zhi and Yewdell, Jonathan W. and Yordanova, Martina M. and Yu, Chien-Hung and Yurchenko, Vyacheslav and Zagrovic, Bojan and Valen, Eivind and Baranov, Pavel V.},
  doi          = {10.1038/s41592-025-02810-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2002-2006},
  shortjournal = {Nat. Meth.},
  title        = {Translon: A single term for translated regions},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mouse lemurs. <em>NMETH</em>, <em>22</em>(10), 1999-2001. (<a href='https://doi.org/10.1038/s41592-025-02852-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studying the gray mouse lemur (Microcebus murinus), one of the world’s smallest primates, in its natural habitat and in the laboratory provides unique perspectives on primate brain evolution, cognition, aging and neurodegenerative diseases, on an accelerated timescale and at a substantially lower cost as compared with larger primate models.},
  archive      = {J_NMETH},
  author       = {Fichtel, Claudia and Huber, Daniel},
  doi          = {10.1038/s41592-025-02852-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1999-2001},
  shortjournal = {Nat. Meth.},
  title        = {Mouse lemurs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What’s new at nature methods. <em>NMETH</em>, <em>22</em>(10), 1997. (<a href='https://doi.org/10.1038/s41592-025-02883-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Team news, editorial projects and initiatives, plus a preview of what’s to come in 2026.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02883-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1997},
  shortjournal = {Nat. Meth.},
  title        = {What’s new at nature methods},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
