<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng">NG - 34</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48. <em>NG</em>, <em>57</em>(9), 2340-2342. (<a href='https://doi.org/10.1038/s41588-025-02332-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Riazuddin, Saima and Belyantseva, Inna A. and Giese, Arnaud P. J. and Lee, Kwanghyuk and Indzhykulian, Artur A. and Nandamuri, Sri Pratima and Yousaf, Rizwan and Sinha, Ghanshyam P. and Lee, Sue and Terrell, David and Hegde, Rashmi S. and Ali, Rana A. and Anwar, Saima and Andrade-Elizondo, Paula B. and Sirmaci, Asli and Parise, Leslie V. and Basit, Sulman and Wali, Abdul and Ayub, Muhammad and Ansar, Muhammad and Ahmad, Wasim and Khan, Shaheen N. and Akram, Javed and Tekin, Mustafa and Riazuddin, Sheikh and Cook, Tiffany and Buschbeck, Elke K. and Frolenkov, Gregory I. and Leal, Suzanne M. and Friedman, Thomas B. and Ahmed, Zubair M.},
  doi          = {10.1038/s41588-025-02332-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2340-2342},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation. <em>NG</em>, <em>57</em>(9), 2339. (<a href='https://doi.org/10.1038/s41588-025-02328-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Uryu, Hidetaka and Saeki, Koichi and Haeno, Hiroshi and Kapadia, Chiraag Deepak and Furudate, Ken and Nangalia, Jyoti and Spencer Chapman, Michael and Zhang, Linda and Padilla, Jennifer and Zhao, Li and Hsu, Joanne I. and Zhao, Chong and Chen, Shujuan and Tanaka, Tomoyuki and Li, Zongrui and Ogata, Satoko and Hanache, Sarah and Yang, Hui and DiNardo, Courtney and Daver, Naval and Pemmaraju, Naveen and Jain, Nitin and Ravandi, Farhad and Zhang, Jianhua and Song, Xingzhi and Thompson, Erika and Tang, Hongli and Little, Latasha and Gumbs, Curtis and Orlowski, Robert Z. and Qazilbash, Muzaffar and Bhalla, Kapil and Colla, Simona and Kantarjian, Hagop and Kanagal-Shamanna, Rashmi and Bueso-Ramos, Carlos and Nakada, Daisuke and Al-Atrash, Gheath and Molldrem, Jeffery and Futreal, P. Andrew and Shpall, Elizabeth and Goodell, Margaret and Garcia-Manero, Guillermo and Takahashi, Koichi},
  doi          = {10.1038/s41588-025-02328-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2339},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma. <em>NG</em>, <em>57</em>(9), 2338. (<a href='https://doi.org/10.1038/s41588-025-02320-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gundem, Gunes and Levine, Max F. and Roberts, Stephen S. and Cheung, Irene Y. and Medina-Martínez, Juan S. and Feng, Yi and Arango-Ossa, Juan E. and Chadoutaud, Loic and Rita, Mathieu and Asimomitis, Georgios and Zhou, Joe and You, Daoqi and Bouvier, Nancy and Spitzer, Barbara and Solit, David B. and Dela Cruz, Filemon and LaQuaglia, Michael P. and Kushner, Brian H. and Modak, Shakeel and Shukla, Neerav and Iacobuzio-Donahue, Christine A. and Kung, Andrew L. and Cheung, Nai-Kong V. and Papaemmanuil, Elli},
  doi          = {10.1038/s41588-025-02320-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2338},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states. <em>NG</em>, <em>57</em>(9), 2323-2334. (<a href='https://doi.org/10.1038/s41588-025-02284-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell atlas projects have revealed that common cell types often comprise distinct, recurrent transcriptional states, but the function and regulation of these states remain poorly understood. Here, we show that systematic activation of transcription factors can recreate such states in vitro, providing tractable models for mechanistic and functional studies. Using a scalable CRISPR activation (CRISPRa) Perturb-seq platform, we activated 1,836 transcription factors in two cell types. CRISPRa induced gene expression within physiological ranges, with chromatin features predicting responsiveness. Comparisons with atlas datasets showed that transcription factor perturbations recapitulated key fibroblast states and identified their regulators, including KLF2 and KLF4 for a universal state present in many tissues, and PLAGL1 for a disease-associated inflammatory state. Inducing the universal state suppressed the inflammatory state, suggesting therapeutic potential. These findings position CRISPRa as a nuanced tool for perturbing differentiated cells and establish a general strategy for studying clinically relevant transcriptional states ex vivo. CRISPR activation of 1,836 human transcription factors recapitulates fibroblast transcriptional states observed in vivo and identifies regulators that can revert inflammatory states.},
  archive      = {J_NG},
  author       = {Southard, Kaden M. and Ardy, Rico C. and Tang, Anran and O’Sullivan, Deirdre D. and Metzner, Eli and Guruvayurappan, Karthik and Norman, Thomas M.},
  doi          = {10.1038/s41588-025-02284-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2323-2334},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize. <em>NG</em>, <em>57</em>(9), 2313-2322. (<a href='https://doi.org/10.1038/s41588-025-02246-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive maps of functional variation at transcription factor (TF) binding sites (cis-elements) are crucial for elucidating how genotype shapes phenotype. Here, we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped over 200,000 variants, genetic, epigenetic, or both (termed binding quantitative trait loci (bQTL)), linked to cis-element occupancy. Three lines of evidence support the functional significance of bQTL: (1) coincidence with causative loci that regulate traits, including vgt1, ZmTRE1 and the MITE transposon near ZmNAC111 under drought; (2) bQTL allelic bias is shared between inbred parents and matches chromatin immunoprecipitation sequencing results; and (3) partitioning genetic variation across genomic regions demonstrates that bQTL capture the majority of heritable trait variation across ~72% of 143 phenotypes. Our study provides an auspicious approach to make functional cis-variation accessible at scale for genetic studies and targeted engineering of complex traits. Pan-cistrome of the maize leaf under well-watered and drought conditions profiled by haplotype-specific MOA-seq highlights the relevance of transcription factor binding QTLs for understanding phenotypic diversity in maize.},
  archive      = {J_NG},
  author       = {Engelhorn, Julia and Snodgrass, Samantha J. and Kok, Amelie and Seetharam, Arun S. and Schneider, Michael and Kiwit, Tatjana and Singh, Ayush and Banf, Michael and Doan, Duong Thi Hai and Khaipho-Burch, Merritt and Runcie, Daniel E. and Sánchez-Camargo, Victor A. and Bader, Rechien and Vladimir Torres-Rodriguez, J. and Sun, Guangchao and Stam, Maike and Fiorani, Fabio and Beier, Sebastian and Schnable, James C. and Bass, Hank W. and Hufford, Matthew B. and Stich, Benjamin and Frommer, Wolf B. and Ross-Ibarra, Jeffrey and Hartwig, Thomas},
  doi          = {10.1038/s41588-025-02246-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2313-2322},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants. <em>NG</em>, <em>57</em>(9), 2302-2312. (<a href='https://doi.org/10.1038/s41588-025-02306-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clubroot disease, caused by the obligate intracellular rhizarian protist Plasmodiophora brassicae, is devastating to cruciferous crops worldwide. Widespread field P. brassicae pathotypes frequently overcome the pathotype-specific resistance of modern varieties, posing a challenge for durable control of this disease. Here a genome-wide association study of 3 years of data comprising field clubroot phenotyping of 244 genome-resequenced Brassica napus accessions identified a strong association of β-1,3-glucan synthase-like 5 (GSL5) with clubroot susceptibility. GSL5 was evolutionarily conserved, and inactivation of GSL5 by genome editing in Arabidopsis, B. napus, Brassica rapa and Brassica oleracea conferred broad-spectrum, high-level resistance to P. brassicae pathotypes without yield penalties in B. napus. GSL5 inactivation derepressed the jasmonic acid-mediated immunity during P. brassicae secondary infection, and this immune repression was possibly reinforced through stabilization of GSL5 by a P. brassicae effector, facilitating clubroot susceptibility. Our study provides durable resistance resources for cruciferous clubroot disease control and insights into plant resistance against intracellular eukaryotic phytopathogens. This study implicates GSL5 inactivation in high, broad-spectrum resistance to the clubroot pathogen Plasmodiophora brassicae in Arabidopsis thaliana, Brassica napus, Brassica oleracea and Brassica rapa.},
  archive      = {J_NG},
  author       = {Wu, Yupo and Zhao, Chuanji and Zhang, Yi and Shen, Cuicui and Zhang, Yuanyuan and Zhang, Xiong and Gao, Lixia and Zeng, Lingyi and Ke, Qinglin and Qin, Li and Liu, Fan and Huang, Junyan and Ren, Li and Liu, Yueying and Cheng, Hongtao and Tong, Chaobo and Hu, Qiong and Cheng, Xiaohui and Wei, Yangdou and Liu, Shengyi and Liu, Lijiang},
  doi          = {10.1038/s41588-025-02306-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2302-2312},
  shortjournal = {Nat. Genet.},
  title        = {Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism. <em>NG</em>, <em>57</em>(9), 2289-2301. (<a href='https://doi.org/10.1038/s41588-025-02293-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Making sense of whole-genome polymorphism data is challenging, but it is essential for overcoming the biases in SNP data. Here we analyze 27 genomes of Arabidopsis thaliana to illustrate these issues. Genome size variation is mostly due to tandem repeat regions that are difficult to assemble. However, while the rest of the genome varies little in length, it is full of structural variants, mostly due to transposon insertions. Because of this, the pangenome coordinate system grows rapidly with sample size and ultimately becomes 70% larger than the size of any single genome, even for n = 27. Finally, we show how short-read data are biased by read mapping. SNP calling is biased by the choice of reference genome, and both transcriptome and methylome profiling results are affected by mapping reads to a reference genome rather than to the genome of the assayed individual. New concepts for comparing the genomes of 27 naturally inbred Arabidopsis thaliana accessions provide essential insights into obtaining a less biased view of whole-genome polymorphism.},
  archive      = {J_NG},
  author       = {Igolkina, Anna A. and Vorbrugg, Sebastian and Rabanal, Fernando A. and Liu, Hai-Jun and Ashkenazy, Haim and Kornienko, Aleksandra E. and Fitz, Joffrey and Collenberg, Max and Kubica, Christian and Mollá Morales, Almudena and Jaegle, Benjamin and Wrightsman, Travis and Voloshin, Vitaly and Bezlepsky, Alexander D. and Llaca, Victor and Nizhynska, Viktoria and Reichardt, Ilka and Bezrukov, Ilja and Lanz, Christa and Bemm, Felix and Flood, Pádraic J. and Nemomissa, Sileshi and Hancock, Angela and Guo, Ya-Long and Kersey, Paul and Weigel, Detlef and Nordborg, Magnus},
  doi          = {10.1038/s41588-025-02293-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2289-2301},
  shortjournal = {Nat. Genet.},
  title        = {A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity. <em>NG</em>, <em>57</em>(9), 2276-2288. (<a href='https://doi.org/10.1038/s41588-025-02294-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common oat, belonging to the genus Avena with 30 recognized species, is a nutritionally important cereal crop and high-quality forage worldwide. Here, we construct a genus-level super-pangenome of Avena comprising 35 high-quality genomes from 14 cultivated oat accessions and 21 wild species. The fully resolved phylogenomic analysis unveils the origin and evolutionary scenario of Avena species, and the super-pangenome analysis identifies 26.62% and 59.93% specific genes and haplotypes in wild species. We delineate the landscape of structural variations (SVs) and the transcriptome profile based 1,401 RNA-sequencing (RNA-seq) samples from diverse abiotic stress treatments in oat. We highlight the crucial role of SVs in modulating gene expression and shaping adaptation to diverse stresses. Further combining SV-based genome-wide association studies (GWASs), we characterize 13 candidate genes associated with drought resistance such as AsARF7, validated by transgenic oat lines. Our study provides unprecedented genomic resources to facilitate genomic, evolution and molecular breeding research in oat. A genus-level super-pangenome of Avena comprising 35 high-quality genomes from 23 cultivated or wild oat species highlights the evolution of Avena and the landscape of structural variations related to abiotic stress resistance.},
  archive      = {J_NG},
  author       = {Zhang, Hongyu and Liu, Ningkun and Wang, Yaru and Zheng, Xinyuan and Li, Wei and Liu, Ze and Liu, Jianan and Wang, Yu and Xing, Longsheng and Li, Tao and Yun, Yange and Zhou, Qinghong and Wang, Meijia and Qin, Yujie and Yan, Jinjiang and Gong, Zhizhong and He, Qiang and Du, Huilong},
  doi          = {10.1038/s41588-025-02294-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2276-2288},
  shortjournal = {Nat. Genet.},
  title        = {Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts. <em>NG</em>, <em>57</em>(9), 2264-2275. (<a href='https://doi.org/10.1038/s41588-025-02330-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Locusts are characterized by a large genome size, polyphenism and an X0 sex determination system. Here we generated chromosome-level genomes for both desert and migratory locusts, as well as a comprehensive chromatin map for the latter. We found that genome enlargement is associated with an increased number of enhancers in expanded intronic regions. To explore the function of expanded enhancers, we identified a distal enhancer that contributes to behavioral differences between solitary and gregarious locusts. In the X0 sex system, H4K16ac enrichment and H4K20me1 depletion maintain balanced X-linked expression in male soma. Notably, the distance-dependent H4K16ac enrichment diminishes gradually in intergenic regions, revealing a special dosage compensation mechanism in large genomes. Furthermore, the distance-dependent H4K16ac results in a lag in the evolution of dosage compensation for the X-linked genes translocated recently from autosomes. Therefore, expanded intronic and intergenic regions shape a distinct chromatin regulation landscape in large genomes. Chromosome-level genome assemblies of migratory and desert locusts, coupled with epigenomic profiling of migratory locusts, reveal chromatin dynamics underlying polyphenism and X-linked dosage compensation following autosomal gene translocation.},
  archive      = {J_NG},
  author       = {Liu, Qing and Jiang, Feng and Li, Ran and Liu, Shanlin and Fang, Wenjing and Li, Xiao and Hu, Ting and Feng, Lianyun and Zhang, Jie and Liu, Zhikang and He, Jing and Guo, Wei and Wang, Xianhui and Sun, Zongyi and Li, Jingjing and Gao, Yunan and Yi, Jiacheng and Li, Qiye and Wang, Xiaoxiao and Wei, Liya and Kang, Le},
  doi          = {10.1038/s41588-025-02330-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2264-2275},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. <em>NG</em>, <em>57</em>(9), 2250-2263. (<a href='https://doi.org/10.1038/s41588-025-02305-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian switch/sucrose nonfermentable (mSWI/SNF) complex regulates chromatin accessibility and frequently shows alterations due to mutation in cancer and neurological diseases. Inadequate expression of mSWI/SNF in heterozygous mice can lead to developmental defects, indicating dosage-sensitive effects of mSWI/SNF. However, how its dosage affects function has remained unclear. Using a targeted protein degradation system, we investigated its dosage-sensitive effects by precisely controlling protein levels of BRG1, the ATPase subunit of the mSWI/SNF complex. We found that binding of BRG1 to chromatin exhibited a linear response to the BRG1 protein level. Although chromatin accessibility at most promoters and insulators was largely unaffected by BRG1 depletion, 44% of enhancers, including 84% of defined superenhancers, showed reduced accessibility. Notably, half of the BRG1-regulated enhancers, particularly superenhancers, exhibited a buffered response to BRG1 loss. Consistently, transcription exhibited a predominantly buffered response to changes in BRG1 levels. Collectively, our findings demonstrate a genomic feature-specific response to BRG1 dosage, shedding light on the dosage-sensitive effects of mSWI/SNF complex defects in cancer and other diseases. This study investigates the dosage-sensitive effects of BRG1, a core SWI/SNF subunit, on chromatin binding, accessibility and transcriptional regulation by precisely controlling its protein level.},
  archive      = {J_NG},
  author       = {Hagihara, Yota and Zhang, Chao and Zhang, Yi},
  doi          = {10.1038/s41588-025-02305-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2250-2263},
  shortjournal = {Nat. Genet.},
  title        = {Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue. <em>NG</em>, <em>57</em>(9), 2238-2249. (<a href='https://doi.org/10.1038/s41588-025-02300-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, TET1 and DNMT3A are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome. This multiomic study, including single-nucleus DNA methylation and chromatin conformation matched with single-nuclei RNA sequencing, provides insights into the epigenomic landscape of human subcutaneous adipose tissue.},
  archive      = {J_NG},
  author       = {Chen, Zeyuan Johnson and Das, Sankha Subhra and Kar, Asha and Lee, Seung Hyuk T. and Abuhanna, Kevin D. and Alvarez, Marcus and Sukhatme, Mihir G. and Wang, Zitian and Gelev, Kyla Z. and Heffel, Matthew G. and Zhang, Yi and Avram, Oren and Rahmani, Elior and Sankararaman, Sriram and Laakso, Markku and Heinonen, Sini and Peltoniemi, Hilkka and Halperin, Eran and Pietiläinen, Kirsi H. and Luo, Chongyuan and Pajukanta, Päivi},
  doi          = {10.1038/s41588-025-02300-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2238-2249},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution. <em>NG</em>, <em>57</em>(9), 2226-2237. (<a href='https://doi.org/10.1038/s41588-025-02307-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients. Integrated multi-omic analyses using samples from the TRACERx study highlight cross-talk between DNA hypermethylation and genomic lesions in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Gimeno-Valiente, Francisco and Castignani, Carla and Larose Cadieux, Elizabeth and Mensah, Nana E. and Liu, Xiaohong and Chen, Kezhong and Chervova, Olga and Karasaki, Takahiro and Weeden, Clare E. and Richard, Corentin and Lai, Siqi and Martínez-Ruiz, Carlos and Lim, Emilia L. and Frankell, Alexander M. and Watkins, Thomas B. K. and Stavrou, Georgia and Usaite, Ieva and Lu, Wei-Ting and Marinelli, Daniele and Saghafinia, Sadegh and Wilson, Gareth A. and Dhami, Pawan and Vaikkinen, Heli and Steif, Jonathan and Veeriah, Selvaraju and Hynds, Robert E. and Hirst, Martin and Hiley, Crispin and Feber, Andrew and Deniz, Özgen and Jamal-Hanjani, Mariam and McGranahan, Nicholas and Beck, Stephan and Demeulemeester, Jonas and Tanić, Miljana and Swanton, Charles and Van Loo, Peter and Kanu, Nnennaya},
  doi          = {10.1038/s41588-025-02307-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2226-2237},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis. <em>NG</em>, <em>57</em>(9), 2215-2225. (<a href='https://doi.org/10.1038/s41588-025-02296-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in PPM1D and the TERT gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers. This study explores the relationship between telomere length and clonal hematopoiesis. Splicing factor and PPM1D gene mutations are more frequent in people with genetically predicted shorter telomere lengths, suggesting that these mutations protect against the consequences of telomere attrition.},
  archive      = {J_NG},
  author       = {McLoughlin, Matthew A. and Cheloor Kovilakam, Sruthi and Dunn, William G. and Gu, Muxin and Tobin, Jake and Pershad, Yash and Williams, Nicholas and Leongamornlert, Daniel and Dawson, Kevin and Bond, Laura and Marando, Ludovica and Wen, Sean and Wilson, Rachael and Valenzano, Giampiero and Symeonidou, Vasiliki and Rak, Justyna and Damaskou, Aristi and Gozdecka, Malgorzata and Liu, Xiaoxuan and Barcena, Clea and Nomdedeu, Josep and Costeas, Paul and Dimitriou, Ioannis D. and Fiorillo, Edoardo and Orrù, Valeria and de Almeida, Jose Guilherme and McKerrell, Thomas and Cullen, Matthew and Mohorianu, Irina and Foukaneli, Theodora and Warren, Alan J. and Wong, Chi and Follows, George and Godfrey, Anna L. and Gudgin, Emma and Cucca, Francesco and McKinney, Eoin and Baxter, E. Joanna and Gerstung, Moritz and Mitchell, Jonathan and Wiseman, Daniel and Bick, Alexander G. and Fabre, Margarete and Quiros, Pedro M. and Nangalia, Jyoti and Kar, Siddhartha and Vassiliou, George S.},
  doi          = {10.1038/s41588-025-02296-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2215-2225},
  shortjournal = {Nat. Genet.},
  title        = {Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Temporal genomic dynamics shape clinical trajectory in multiple myeloma. <em>NG</em>, <em>57</em>(9), 2203-2214. (<a href='https://doi.org/10.1038/s41588-025-02292-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple myeloma evolution is characterized by the accumulation of genomic drivers over time. To unravel this timeline and its impact on clinical outcomes, we analyzed 421 whole-genome sequences from 382 patients. Using clock-like mutational signatures, we estimated a time lag of two to four decades between the initiation of events and diagnosis. We demonstrate that odd-numbered chromosome trisomies in patients with hyperdiploidy can be acquired simultaneously with other chromosomal gains (for example, 1q gain). We show that hyperdiploidy is acquired after immunoglobulin heavy chain translocation when both events co-occur. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (&gt;1 extra copy), but fared worse than those with late 1q gain. This finding underscores that the 1q gain prognostic impact depends more on the timing of acquisition than on the number of copies gained. Overall, this study contributes to a better understanding of the life history of myeloma and may have prognostic implications. This analysis of whole-genome sequencing data from 421 multiple myeloma samples elucidates the timing of key genomic events and shows associations between the timing of 1q gain and clinical outcome.},
  archive      = {J_NG},
  author       = {Maura, Francesco and Kaddoura, Marcella and Poos, Alexandra M. and Baughn, Linda B. and Ziccheddu, Bachisio and Bärtsch, Marc-Andrea and Cirrincione, Anthony and Maclachlan, Kylee and Chojnacka, Monika and Diamond, Benjamin and Papadimitriou, Marios and Blaney, Patrick and John, Lukas and Reichert, Philipp and Huhn, Stefanie and Gagler, Dylan and Zhang, Yanming and Dogan, Ahmet and Lesokhin, Alexander M. and Davies, Faith and Goldschmidt, Hartmut and Fenk, Roland and Weisel, Katja C. and Mai, Elias K. and Korde, Neha and Morgan, Gareth J. and Rajkumar, S. Vincent and Kumar, Shaji and Usmani, Saad and Landgren, Ola and Raab, Marc S. and Weinhold, Niels},
  doi          = {10.1038/s41588-025-02292-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2203-2214},
  shortjournal = {Nat. Genet.},
  title        = {Temporal genomic dynamics shape clinical trajectory in multiple myeloma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tamoxifen induces PI3K activation in uterine cancer. <em>NG</em>, <em>57</em>(9), 2192-2202. (<a href='https://doi.org/10.1038/s41588-025-02308-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutagenic processes and clonal selection contribute to the development of therapy-associated secondary neoplasms, a known complication of cancer treatment. The association between tamoxifen therapy and secondary uterine cancers is uncommon but well established; however, the genetic mechanisms underlying tamoxifen-driven tumorigenesis remain unclear. We find that oncogenic PIK3CA mutations, common in spontaneously arising estrogen-associated de novo uterine cancer, are significantly less frequent in tamoxifen-associated tumors. In vivo, tamoxifen-induced estrogen receptor stimulation activates phosphoinositide 3-kinase (PI3K) signaling in normal mouse uterine tissue, potentially eliminating the selective benefit of PI3K-activating mutations in tamoxifen-associated uterine cancer. Together, we present a unique pathway of therapy-associated carcinogenesis in which tamoxifen-induced activation of the PI3K pathway acts as a non-genetic driver event, contributing to the multistep model of uterine carcinogenesis. While this PI3K mechanism is specific to tamoxifen-associated uterine cancer, the concept of treatment-induced signaling events may have broader applicability to other routes of tumorigenesis. Sequencing analysis of tamoxifen-associated uterine cancers and further in vivo analyses suggest that the drug tamoxifen can activate the PI3K pathway in the absence of oncogenic mutations.},
  archive      = {J_NG},
  author       = {Kübler, Kirsten and Nardone, Agostina and Anand, Shankara and Gurevich, Daniel and Gao, Jianjiong and Droog, Marjolein and Hermida-Prado, Francisco and Akhshi, Tara and Feiglin, Ariel and Feit, Avery S. and Cohen Feit, Gabriella and Dackus, Gwen and Pun, Matthew and Kuang, Yanan and Cha, Justin and Miller, Mendy and Gregoricchio, Sebastian and Lanfermeijer, Mirthe and Cornelissen, Sten and Gibson, William J. and Paweletz, Cloud P. and Van Allen, Eliezer M. and van Leeuwen, Flora E. and Nederlof, Petra M. and Nguyen, Quang-Dé and Mourits, Marian J. E. and Radovich, Milan and Leshchiner, Ignaty and Stewart, Chip and Matulonis, Ursula A. and Zwart, Wilbert and Maruvka, Yosef E. and Getz, Gad and Jeselsohn, Rinath},
  doi          = {10.1038/s41588-025-02308-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2192-2202},
  shortjournal = {Nat. Genet.},
  title        = {Tamoxifen induces PI3K activation in uterine cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L. <em>NG</em>, <em>57</em>(9), 2177-2191. (<a href='https://doi.org/10.1038/s41588-025-02289-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the high prevalence of ERG transcription factor translocations in prostate cancer, the mechanism of tumorigenicity remains poorly understood. Using lineage tracing, we find the tumor-initiating activity of ERG resides in a subpopulation of murine basal cells that coexpress luminal genes (BasalLum) and not in the larger population of ERG+ luminal cells. Upon ERG activation, BasalLum cells give rise to highly proliferative intermediate (IM) cells with stem-like features that coexpress basal, luminal, hillock and club marker genes, before transitioning to Krt8+ luminal cells. Transcriptomic analysis of ERG+ human prostate cancers confirms the presence of rare ERG+ BasalLum cells, as well as IM cells whose presence is associated with a worse prognosis. Single-cell analysis revealed a chromatin state in ERG+ IM cells enriched for STAT3 transcription factor binding sites and elevated expression of the KMT2A/MLL1 and DOT1L, all three of which are essential for ERG-driven tumorigenicity in vivo. In addition to providing translational opportunities, this work illustrates how single-cell approaches combined with lineage tracing can identify cancer vulnerabilities not evident from bulk analysis. Lineage tracing in mice identifies a subpopulation of basal cells that express Tmprss2 and Nkx3 as the origin of ERG-driven prostate cancer. Upon expansion, these cells show an enrichment for STAT3 chromatin binding and elevated expression of KMT2A and DOT1L as dependencies for ERG oncogenicity.},
  archive      = {J_NG},
  author       = {Feng, Weiran and Ladewig, Erik and Lange, Matthew and Salsabeel, Nazifa and Zhao, Huiyong and Lee, Young Sun and Gopalan, Anuradha and Luo, Hanzhi and Kang, Wenfei and Fan, Ning and Rosiek, Eric and de Stanchina, Elisa and Chen, Yu and Carver, Brett S. and Leslie, Christina S. and Sawyers, Charles L.},
  doi          = {10.1038/s41588-025-02289-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2177-2191},
  shortjournal = {Nat. Genet.},
  title        = {ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutational landscape of triple-negative breast cancer in african american women. <em>NG</em>, <em>57</em>(9), 2166-2176. (<a href='https://doi.org/10.1038/s41588-025-02322-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {African American (AA) women have the highest incidence of triple-negative breast cancer (TNBC) among all racial groups, but are underrepresented in cancer genomic studies. In 462 AA women with TNBC, we characterized the tumor mutational landscape by whole-exome sequencing and RNA sequencing. We unveiled a high-resolution mutational portrait of TNBC in AA women reminiscent of that in Asian and non-Hispanic white women, with no evidence of associations of mutational features with African ancestry. We also made some distinctive discoveries, including an almost complete dominance of TP53 mutations, low frequency of PIK3CA mutations and mutational signature-based subtypes with etiologic and prognostic significance. These findings do not support major racial differences in TNBC biology at the level of somatic mutations. Our study contributes considerably to diversifying the knowledge base of breast cancer genomics and provides insights into the disease etiology, disparities and therapeutic vulnerability of TNBC in AA women. This study explores the genomic and transcriptomic landscapes of triple-negative breast cancer in African American women. The authors show that the mutational profile is broadly similar to that observed in European and East Asian ancestry women while highlighting some interesting differences.},
  archive      = {J_NG},
  author       = {Yao, Song and Wei, Lei and Hu, Qiang and Liu, Song and Manojlovic, Zarko and Fiorica, Peter N. and Long, Mark and Zirpoli, Gary R. and Cai, Qiuyin and Long, Jirong and Ping, Jie and Barnard, Mollie E. and Jin, Yuxin and Murakami, Mitsuko and Wang, Jianmin and Zhu, Qianqian and Davis, Warren and Chen, Jianhong and Ondracek, Rochelle P. and Khoury, Thaer and Gandhi, Shipra and Takabe, Kazuaki and Ko, Naomi and Sanderson, Maureen and Hong, Chi-Chen and Bandera, Elisa V. and Craig, David W. and Ambrosone, Christine B. and Palmer, Julie R. and Zheng, Wei and Carpten, John D.},
  doi          = {10.1038/s41588-025-02322-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2166-2176},
  shortjournal = {Nat. Genet.},
  title        = {Mutational landscape of triple-negative breast cancer in african american women},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions. <em>NG</em>, <em>57</em>(9), 2156-2165. (<a href='https://doi.org/10.1038/s41588-025-02291-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand shared and ancestry-specific genetic control of brain protein expression and its ramifications for disease, we mapped protein quantitative trait loci (pQTLs) in 1,362 brain proteomes from African American, Hispanic/Latin American and non-Hispanic white donors. Among the pQTLs that multiancestry fine-mapping MESuSiE confidently assigned as putative causal pQTLs in a specific population, most were shared across the three studied populations and are referred to as multiancestry causal pQTLs. These multiancestry causal pQTLs were enriched for exonic and promoter regions. To investigate their effects on disease, we modeled the 858 multiancestry causal pQTLs as instrumental variables using Mendelian randomization and genome-wide association study results for neurologic and psychiatric conditions (21 traits in participants with European ancestry, 10 in those with African ancestry and 4 in Hispanic participants). We identified 119 multiancestry pQTL–protein pairs consistent with a causal role in these conditions. Remarkably, 29% of the multiancestry pQTLs in these pairs were coding variants. These results lay an important foundation for the creation of new molecular models of neurologic and psychiatric conditions that are likely to be relevant to individuals across different genetic ancestries. Multiancestry fine-mapping of brain protein quantitative trait loci coupled with Mendelian randomization analyses identifies protein–trait pairs consistent with causal effects across neurological and psychiatric conditions.},
  archive      = {J_NG},
  author       = {Wingo, Aliza P. and Liu, Yue and Vattathil, Selina M. and Gerasimov, Ekaterina S. and Mei, Zhen and Ravindran, Suda Parimala and Liu, Jiaqi and Shantaraman, Ananth and Seifar, Fatemeh and Wang, Erming and Zhang, Bin and Reddy, Joseph and Allen, Mariet and Ertekin-Taner, Nilüfer and De Jager, Philip L. and Fox, Edward J. and Duong, Duc M. and Epstein, Michael P. and Cutler, David J. and Levey, Allan I. and Bennett, David A. and Seyfried, Nicholas T. and Wingo, Thomas S.},
  doi          = {10.1038/s41588-025-02291-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2156-2165},
  shortjournal = {Nat. Genet.},
  title        = {Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes. <em>NG</em>, <em>57</em>(9), 2146-2155. (<a href='https://doi.org/10.1038/s41588-025-02288-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks with whole-genome sequencing (WGS) now enable the association of noncoding rare variants with complex human traits. Given that &gt;98% of the genome is available for exploration, the selection of noncoding variants remains a critical yet unresolved challenge in these analyses. Here we leverage knowledge of blood gene regulation and deleteriousness scores to select noncoding variants pertinent for association with blood-related traits. Integrating WGS and 42 blood cell count and biomarker measurements for 166,740 UK Biobank samples, we perform variant collapsing tests, identifying hundreds of gene–trait associations involving noncoding variants. However, we demonstrate that most of these noncoding rare variant associations (1) reproduce associations known from previous studies and (2) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting noncoding rare variant association results. Noncoding rare variant analyses using whole-genome sequencing data from the UK Biobank identify gene–trait associations for 42 blood cell traits but find that most signals are driven by linkage disequilibrium between common and rare variants.},
  archive      = {J_NG},
  author       = {Ribeiro, Diogo M. and Hofmeister, Robin J. and Rubinacci, Simone and Delaneau, Olivier},
  doi          = {10.1038/s41588-025-02288-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2146-2155},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores. <em>NG</em>, <em>57</em>(9), 2136-2145. (<a href='https://doi.org/10.1038/s41588-025-02298-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic health record (EHR)-based phenotype risk scores (PheRS) leverage individuals’ health trajectories to estimate disease risk, similar to how polygenic scores (PGS) use genetic information. While PGS generalizability has been studied, less is known about PheRS generalizability across healthcare systems and whether PheRS are complementary to PGS. We trained elastic-net-based PheRS to predict the onset of 13 common diseases for 845,929 individuals (age = 32–70 years) from three biobank-based studies in Finland (FinnGen), the UK (UKB) and Estonia (EstB). All PheRS were statistically significantly associated with the diseases of interest and most generalized well without retraining when applied to other studies. PheRS and PGS were only moderately correlated and models including both predictors improved onset prediction compared to PGS alone for 8 of 13 diseases. Our results indicate that EHR-based risk scores can transfer well between EHRs, capture largely independent information from PGS, and provide additive benefits for disease risk prediction. Comparison of electronic health record-based phenotype risk scores (PheRS) and polygenic scores (PGS) across 13 common diseases and three biobank-based studies indicates that PheRS and PGS may provide additive benefits for risk prediction.},
  archive      = {J_NG},
  author       = {Detrois, Kira E. and Hartonen, Tuomo and Teder-Laving, Maris and Jermy, Bradley and Läll, Kristi and Yang, Zhiyu and Mägi, Reedik and Ripatti, Samuli and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02298-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2136-2145},
  shortjournal = {Nat. Genet.},
  title        = {Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes. <em>NG</em>, <em>57</em>(9), 2124-2135. (<a href='https://doi.org/10.1038/s41588-025-02317-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Ancestral Recombination Graph (ARG), which describes the genealogical history of a sample of genomes, is a vital tool in population genomics and biomedical research. Recent advancements have substantially increased ARG reconstruction scalability, but they rely on approximations that can reduce accuracy, especially under model misspecification. Moreover, they reconstruct only a single ARG topology and cannot quantify the considerable uncertainty associated with ARG inferences. Here, to address these challenges, we introduce SINGER (sampling and inferring of genealogies with recombination), a method that accelerates ARG sampling from the posterior distribution by two orders of magnitude, enabling accurate inference and uncertainty quantification for hundreds of whole-genome sequences. Through extensive simulations, we demonstrate SINGER’s enhanced accuracy and robustness to model misspecification compared to existing methods. We demonstrate the utility of SINGER by applying it to individuals of British and African descent within the 1000 Genomes Project, identifying signals of population differentiation, archaic introgression and strong support for ancient polymorphism in the human leukocyte antigen region shared across primates. SINGER is a method for creating ancestral recombination graphs to understand the genealogical history of genomes. The method has increased speed, and thus scalability, without sacrificing accuracy.},
  archive      = {J_NG},
  author       = {Deng, Yun and Nielsen, Rasmus and Song, Yun S.},
  doi          = {10.1038/s41588-025-02317-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2124-2135},
  shortjournal = {Nat. Genet.},
  title        = {Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. <em>NG</em>, <em>57</em>(9), 2116-2123. (<a href='https://doi.org/10.1038/s41588-025-02286-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mixed-model association analysis (MMAA) is the preferred tool for performing genome-wide association studies. However, existing MMAA tools often have long runtimes and high memory requirements. Here we present LDAK-KVIK, an MMAA tool for analysis of quantitative and binary phenotypes. LDAK-KVIK is computationally efficient, requiring less than 10 CPU hours and 5 Gb memory to analyze genome-wide data for 350,000 individuals. Using simulated phenotypes, we show that LDAK-KVIK produces well-calibrated test statistics for both homogeneous and heterogeneous datasets. When applied to real phenotypes, LDAK-KVIK has the highest power among all tools considered. For example, across 40 quantitative UK Biobank phenotypes (average sample size 349,000), LDAK-KVIK finds 16% more independent, genome-wide significant loci than classical linear regression, whereas BOLT-LMM and REGENIE find 15% and 11% more, respectively. LDAK-KVIK can also be used to perform gene-based tests; across the 40 quantitative UK Biobank phenotypes, LDAK-KVIK finds 18% more significant genes than the leading existing tool. Last, LDAK-KVIK produces state-of-the-art polygenic scores. LDAK-KVIK is a mixed-model association method for genome-wide studies that optimizes computational performance and power. LDAK-KVIK can also perform gene-based tests and produces state-of-the-art polygenic scores.},
  archive      = {J_NG},
  author       = {Hof, Jasper P. and Speed, Doug},
  doi          = {10.1038/s41588-025-02286-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2116-2123},
  shortjournal = {Nat. Genet.},
  title        = {LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward improving multiomic research in understudied cereals. <em>NG</em>, <em>57</em>(9), 2106-2115. (<a href='https://doi.org/10.1038/s41588-025-02245-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the looming challenges of climate change and population pressure, understudied cereals hold notably untapped potential for future global food security and sustainable agriculture due to their unique nutritional profiles and notable stress resilience. Multiomics holds great promise in realizing the value of understudied cereals by understanding the genetic basis and molecular mechanisms regulating their unique traits; however, multiomic studies of understudied cereals still lag behind those of major cereals, hindering their genetic improvement and wide adoption in modern agriculture. Here, we discuss the limitations and challenges in multiomic studies of understudied cereals and how they can be overcome to enhance utilization of the traits unique to understudied cereals in improving both themselves and major cereals. We also offer a database specific to understudied cereals, including multiomics data from 12 understudied cereals and wild relatives for some of these, to serve the community. Understudied cereals offer untapped potential for food security, yet multiomic research substantially lags behind that of major cereals. This Perspective discusses these challenges and introduces a dedicated database to serve the community.},
  archive      = {J_NG},
  author       = {Yang, Fanjing and Gong, Xiaojiao and Zhao, Pan and Cheng, Lingtong and Zhang, Yujie and Luo, Yingting and Zhu, Qian-Hao and Ye, Chu-Yu},
  doi          = {10.1038/s41588-025-02245-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2106-2115},
  shortjournal = {Nat. Genet.},
  title        = {Toward improving multiomic research in understudied cereals},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The LISTEN principles for genetic sequence data governance and database engineering. <em>NG</em>, <em>57</em>(9), 2099-2105. (<a href='https://doi.org/10.1038/s41588-025-02270-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several international legal agreements include an ‘access and benefit-sharing’ (ABS) mechanism that attaches obligations to the use of genetic sequence data. These agreements are frequently subject to critique on the grounds that ABS is either fundamentally incompatible with the principles of open science, or technically challenging to implement in open scientific databases. Here, we argue that these critiques arise from a misinterpretation of the principles of open science and that both considerations can be addressed by a set of simple principles that link database engineering and governance. We introduce a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, which can be readily adopted by both new and existing platforms participating in ABS systems. We also highlight how these principles can act in concert with familiar principles of open science, such as findable, accessible, interoperable and reusable (FAIR) data sharing. This Perspective proposes a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, aimed at ensuring access and benefit-sharing principles in open science.},
  archive      = {J_NG},
  author       = {Carlson, Colin J. and Granados, Monica and Phelan, Alexandra and Ramakrishnan, Nithin and Poisot, Timothée},
  doi          = {10.1038/s41588-025-02270-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2099-2105},
  shortjournal = {Nat. Genet.},
  title        = {The LISTEN principles for genetic sequence data governance and database engineering},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Governance of cross-border genomic data sharing through a human rights approach. <em>NG</em>, <em>57</em>(9), 2090-2098. (<a href='https://doi.org/10.1038/s41588-025-02252-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cross-border genomic data sharing is crucial for advancing research progress, understanding of genetic diversity and international public health. However, this practice also leads to concerns regarding biosecurity, individual privacy and regulatory conflicts. Scientific communities often advocate data sovereignty approaches to address these challenges, yet related frameworks are often inadequate in their ability to address such challenges and have an overemphasis on national control and protectionism. They also may not fully consider the inherently transnational nature of genomic data and the need for a governance system that reflects the shared human heritage of the data and global interconnectivity. Instead, we propose here a human rights-based approach that surpasses territorial boundaries to promote shared responsibility and equitable research practices. A balanced governance framework that incorporates extraterritorial obligations and aligns data policy with human rights principles can safeguard individual privacy and biosecurity, encourage international collaboration and support ethically responsible genomic research. This Perspective proposes a human rights-based governance framework for cross-border genomic data sharing, addressing limitations of data sovereignty approaches while balancing privacy, security and global research collaboration.},
  archive      = {J_NG},
  author       = {Yu, Liang and Feng, Ruohan and Sun, Youhai and Peng, Yaojin},
  doi          = {10.1038/s41588-025-02252-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2090-2098},
  shortjournal = {Nat. Genet.},
  title        = {Governance of cross-border genomic data sharing through a human rights approach},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recreating fibroblast diversity in vitro by activating transcription factors. <em>NG</em>, <em>57</em>(9), 2088-2089. (<a href='https://doi.org/10.1038/s41588-025-02283-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We combined CRISPR activation of all 1,836 known human transcription factors with high-throughput Perturb-seq to recreate the diverse transcriptional states occupied by fibroblasts in vivo. Our study revealed regulators of key states and showed that inducing normal states can, in some cases, suppress those linked to disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02283-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2088-2089},
  shortjournal = {Nat. Genet.},
  title        = {Recreating fibroblast diversity in vitro by activating transcription factors},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improving broad-spectrum resistance to clubroot disease with one gene. <em>NG</em>, <em>57</em>(9), 2086-2087. (<a href='https://doi.org/10.1038/s41588-025-02304-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GSL5, a glucan synthase, acts as a suppressor of jasmonic acid-mediated immunity in cruciferous plants. Inactivation of GSL5 by genome editing confers high-level and broad-spectrum resistance to pathogens that cause clubroot disease in four cruciferous species.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02304-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2086-2087},
  shortjournal = {Nat. Genet.},
  title        = {Improving broad-spectrum resistance to clubroot disease with one gene},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mystery of tamoxifen-associated uterine cancer uncovered. <em>NG</em>, <em>57</em>(9), 2084-2085. (<a href='https://doi.org/10.1038/s41588-025-02316-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tamoxifen is an essential drug in breast cancer therapy. Unlike prevailing models of therapy-related tumorigenesis, tamoxifen acts by directly activating the PI3K pathway, bypassing the need for mutations in one of the most common driver genes in sporadic uterine cancer. These findings open avenues for investigating similar mechanisms in other drugs.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02316-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2084-2085},
  shortjournal = {Nat. Genet.},
  title        = {Mystery of tamoxifen-associated uterine cancer uncovered},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A bivalent chromatin state regulates the function of composite transposons. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02337-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02337-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {A bivalent chromatin state regulates the function of composite transposons},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting obesity from childhood. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02338-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02338-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Predicting obesity from childhood},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Building immune tolerance in the thymus. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02339-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02339-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Building immune tolerance in the thymus},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI identifies promoter variants that alter gene expression. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02340-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02340-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {AI identifies promoter variants that alter gene expression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease. <em>NG</em>, <em>57</em>(9), 2079-2082. (<a href='https://doi.org/10.1038/s41588-025-02299-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite Africa’s vast genetic diversity, its populations are underrepresented in global genomic datasets. Here we describe the vision of the KidneyGenAfrica, a pan-African initiative launched to address this inequity, and call for more inclusive genomics research that recognizes Africa’s key role in genetic variation and potential to generate insights in chronic kidney disease.},
  archive      = {J_NG},
  author       = {Fatumo, Segun and Ojewunmi, Oyesola and Camenzuli, Rebecca and Kintu, Christopher and Kamiza, Abram and Brandenburg, Jean-Tristan and Kalungi, Allan and Kalyesubula, Robert and Salako, Babatunde and Nash, Oyekanmi and Crampin, Amelia and Adeyemo, Adebowale and Nyirenda, Moffat and Ramsay, Michele and Fabian, June},
  doi          = {10.1038/s41588-025-02299-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2079-2082},
  shortjournal = {Nat. Genet.},
  title        = {KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The future of polygenic risk scores in direct-to-consumer genomics. <em>NG</em>, <em>57</em>(9), 2077-2078. (<a href='https://doi.org/10.1038/s41588-025-02313-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Sun, Yuhao and Vines, John and Tenesa, Albert},
  doi          = {10.1038/s41588-025-02313-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2077-2078},
  shortjournal = {Nat. Genet.},
  title        = {The future of polygenic risk scores in direct-to-consumer genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
