<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng">NG - 34</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes. <em>NG</em>, <em>57</em>(10), 2604. (<a href='https://doi.org/10.1038/s41588-025-02383-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {van der Laan, Camiel M. and Ip, Hill F. and Schipper, Marijn and Hottenga, Jouke-Jan and St Pourcain, Beate and Zayats, Tetyana and Pool, René and Krapohl, Eva M. L. and Brikell, Isabell and Soler Artigas, María and Cabana-Domínguez, Judit and Llonga, Natalia and Nolte, Ilja M. and Bolhuis, Koen and Palviainen, Teemu and Zafarmand, Hadi and Gordon, Scott and Aliev, Fazil and Burt, S. Alexandra and Wang, Carol A. and Saunders, Gretchen and Karhunen, Ville and Adkins, Daniel E. and Border, Richard and Peterson, Roseann E. and Prinz, Joseph A. and Thiering, Elisabeth and Vilor-Tejedor, Natàlia and Ahluwalia, Tarunveer S. and Allegrini, Andrea and Rimfeld, Kaili and Chen, Qi and Lu, Yi and Martin, Joanna and Bosch, Rosa and Ramos-Quiroga, Josep Antoni and Neumann, Alexander and Ensink, Judith and Grasby, Katrina L. and Morosoli, José J. and Tong, Xiaoran and Marrington, Shelby and Scott, James G. and Shabalin, Andrey A. and Corley, Robin and Evans, Luke M. and Sugden, Karen and Alemany, Silvia and Sass, Lærke and Vinding, Rebecca and Ehli, Erik A. and Hagenbeek, Fiona A. and Derks, Eske M. and Larsson, Henrik and Snieder, Harold and Cecil, Charlotte and Whipp, Alyce M. and Korhonen, Tellervo and Vuoksimaa, Eero and Rose, Richard J. and Uitterlinden, André G. and Haavik, Jan and Harris, Jennifer R. and Helgeland, Øyvind and Johansson, Stefan and Knudsen, Gun Peggy S. and Njolstad, Pal Rasmus and Lu, Qing and Rodriguez, Alina and Henders, Anjali K. and Mamun, Abdullah and Najman, Jackob M. and Brown, Sandy and Hopfer, Christian and Krauter, Kenneth and Reynolds, Chandra A. and Smolen, Andrew and Stallings, Michael and Wadsworth, Sally and Wall, Tamara L. and Eaves, Lindon and Silberg, Judy L. and Miller, Allison and Havdahl, Alexandra and Llop, Sabrina and Lopez-Espinosa, Maria-Jose and Bønnelykke, Klaus and Sunyer, Jordi and Arseneault, Louise and Standl, Marie and Heinrich, Joachim and Boden, Joseph and Pearson, John and Horwood, John and Kennedy, Martin and Poulton, Richie and Maes, Hermine H. and Hewitt, John and Copeland, William E. and Middeldorp, Christel M. and Williams, Gail M. and Wray, Naomi and Järvelin, Marjo-Riitta and McGue, Matt and Iacono, William and Caspi, Avshalom and Moffitt, Terrie E. and Whitehouse, Andrew J. O. and Pennell, Craig E. and Klump, Kelly L. and Jiang, Chang and Dick, Danielle M. and Reichborn-Kjennerud, Ted and Martin, Nicholas G. and Medland, Sarah E. and Vrijkotte, Tanja and Kaprio, Jaakko and Tiemeier, Henning and Davey Smith, George and Hartman, Catharina A. and Oldehinkel, Albertine J. and Casas, Miquel and Ribasés, Marta and Lichtenstein, Paul and Lundström, Sebastian and Plomin, Robert and Bartels, Meike and Nivard, Michel G. and Boomsma, Dorret I.},
  doi          = {10.1038/s41588-025-02383-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2604},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars. <em>NG</em>, <em>57</em>(10), 2603. (<a href='https://doi.org/10.1038/s41588-025-02356-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Yano, Ryoichi and Li, Feng and Hiraga, Susumu and Takeshima, Ryoma and Kobayashi, Michie and Toda, Kyoko and Umehara, Yosuke and Kajiya-Kanegae, Hiromi and Iwata, Hiroyoshi and Kaga, Akito and Ishimoto, Masao},
  doi          = {10.1038/s41588-025-02356-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2603},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Why genomic diversity should not be framed by census alone. <em>NG</em>, <em>57</em>(10), 2603. (<a href='https://doi.org/10.1038/s41588-025-02357-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Corpas, Manuel and Guio, Heinner and Lopez-Correa, Catalina and Fatumo, Segun},
  doi          = {10.1038/s41588-025-02357-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2603},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Why genomic diversity should not be framed by census alone},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nucleotide dependency analysis of genomic language models detects functional elements. <em>NG</em>, <em>57</em>(10), 2589-2602. (<a href='https://doi.org/10.1038/s41588-025-02347-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering how nucleotides in genomes encode regulatory instructions and molecular machines is a long-standing goal. Genomic language models (gLMs) implicitly capture functional elements and their organization from genomic sequences alone by modeling probabilities of each nucleotide given its sequence context. However, discovering functional genomic elements from gLMs has been challenging due to the lack of interpretable methods. Here we introduce nucleotide dependencies, which quantify how nucleotide substitutions at one genomic position affect the probabilities of nucleotides at other positions. We demonstrate that nucleotide dependencies are more effective at indicating the deleteriousness of genetic variants than alignment-based conservation and gLM reconstruction. Dependency analysis accurately detects regulatory motifs and highlights bases in contact within RNAs, including pseudoknots and tertiary structure contacts, revealing new, experimentally validated RNA structures. Finally, we leverage dependency maps to reveal critical limitations of several gLM architectures and training strategies. Altogether, nucleotide dependency analysis opens a new avenue for discovering and studying functional elements and their interactions in genomes. Mapping pairwise nucleotide dependencies by leveraging genomic language models highlights functional genomic elements and predicts deleterious genetic variants more effectively than alignment-based conservation metrics.},
  archive      = {J_NG},
  author       = {Tomaz da Silva, Pedro and Karollus, Alexander and Hingerl, Johannes and Galindez, Gihanna Sta. Teresa and Wagner, Nils and Hernandez-Alias, Xavier and Incarnato, Danny and Gagneur, Julien},
  doi          = {10.1038/s41588-025-02347-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2589-2602},
  shortjournal = {Nat. Genet.},
  title        = {Nucleotide dependency analysis of genomic language models detects functional elements},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variants affecting RNA stability influence complex traits and disease risk. <em>NG</em>, <em>57</em>(10), 2578-2588. (<a href='https://doi.org/10.1038/s41588-025-02326-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression is modulated jointly by transcriptional regulation and messenger RNA stability, yet the latter is often overlooked in studies on genetic variants. Here, leveraging metabolic labeling data (Bru/BruChase-seq) and a new computational pipeline, RNAtracker, we categorize genes as allele-specific RNA stability (asRS) or allele-specific RNA transcription events. We identify more than 5,000 asRS variants among 665 genes across a panel of 11 human cell lines. These variants directly overlap conserved microRNA target regions and allele-specific RNA-binding protein sites, illuminating mechanisms through which stability is mediated. Furthermore, we identified causal asRS variants using a massively parallel screen (MapUTR) for variants that affect post-transcriptional mRNA abundance, as well as through CRISPR prime editing approaches. Notably, asRS genes were enriched significantly among a multitude of immune-related pathways and contribute to the risk of several immune system diseases. This work highlights RNA stability as a critical, yet understudied mechanism linking genetic variation and disease. RNAtracker is a computational pipeline that distinguishes variants associated with allele-specific RNA stability from those associated with allele-specific RNA transcription. Variants affecting RNA stability are enriched in immune-related genes and contribute to disease risk.},
  archive      = {J_NG},
  author       = {Huang, Elaine and Fu, Ting and Zhang, Ling and Yan, Guanao and Yamamoto, Ryo and Terrazas, Sari and Nguyen, Thuy Linh and Gonzalez-Figueroa, Carlos and Khanbabaei, Armen and Bahn, Jae Hoon and Varada, Rajagopal and Amoah, Kofi and Hervoso, Jonatan and Paulsen, Michelle T. and Magnuson, Brian and Ljungman, Mats and Li, Jingyi Jessica and Xiao, Xinshu},
  doi          = {10.1038/s41588-025-02326-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2578-2588},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variants affecting RNA stability influence complex traits and disease risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerated bayesian inference of population size history from recombining sequence data. <em>NG</em>, <em>57</em>(10), 2570-2577. (<a href='https://doi.org/10.1038/s41588-025-02323-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study introduces population history learning by averaging sampled histories (PHLASH), a new method for inferring population history from whole-genome sequence data. It works by drawing random, low-dimensional projections of the coalescent intensity function from the posterior distribution of a pairwise sequentially Markovian coalescent-like model and averaging them together to form an accurate and adaptive estimator. On simulated data, PHLASH tends to be faster and have lower error than several competing methods, including SMC++, MSMC2 and FITCOAL. Moreover, it provides automatic uncertainty quantification and leads to new Bayesian testing procedures for detecting population structure and ancient bottlenecks. The key technical advance is a new algorithm for computing the score function (gradient of the log likelihood) of a coalescent hidden Markov model, which has the same computational cost as evaluating the log likelihood. PHLASH has been released as an easy-to-use Python software package and leverages graphics processing unit acceleration when available. Population history learning by averaging sampled histories is a new Bayesian method for estimating historical effective population size from recombining sequence data that offers improved speed and accuracy and enables uncertainty quantification and testing for ancient bottlenecks.},
  archive      = {J_NG},
  author       = {Terhorst, Jonathan},
  doi          = {10.1038/s41588-025-02323-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2570-2577},
  shortjournal = {Nat. Genet.},
  title        = {Accelerated bayesian inference of population size history from recombining sequence data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bryophytes hold a larger gene family space than vascular plants. <em>NG</em>, <em>57</em>(10), 2562-2569. (<a href='https://doi.org/10.1038/s41588-025-02325-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After 500 million years of evolution, extant land plants compose the following two sister groups: the bryophytes and the vascular plants. Despite their small size and simple structure, bryophytes thrive in a wide variety of habitats, including extreme conditions. However, the genetic basis for their ecological adaptability and long-term survival is not well understood. A comprehensive super-pangenome analysis, incorporating 123 newly sequenced bryophyte genomes, reveals that bryophytes possess a substantially greater diversity of gene families than vascular plants. This includes a higher number of unique and lineage-specific gene families, originating from extensive new gene formation and continuous horizontal transfer of microbial genes over their long evolutionary history. The evolution of bryophytes’ rich and diverse genetic toolkit, which includes new physiological innovations like unique immune receptors, likely facilitated their spread across different biomes. These newly sequenced bryophyte genomes offer a valuable resource for exploring alternative evolutionary strategies for terrestrial success. A super-pangenome analysis incorporating 123 newly sequenced bryophyte genomes reveals that bryophytes exhibit a larger number of unique and lineage-specific gene families than vascular plants.},
  archive      = {J_NG},
  author       = {Dong, Shanshan and Wang, Sibo and Li, Linzhou and Yu, Jin and Zhang, Yongxia and Xue, Jia-Yu and Chen, Hengchi and Ma, Jianchao and Zeng, Yuying and Cai, Yuqing and Huang, Wei and Zhou, Xuping and Wu, Jiayi and Li, Jianyou and Yao, Yifeng and Hu, Ruoyang and Zhao, Tao and Villarreal A, Juan Carlos and Dirick, Leon and Liu, Li and Ignatov, Michael and Jin, Minghui and Ruan, Jue and He, Yikun and Wang, Haifeng and Xu, Bo and Rozzi, Ricardo and Wegrzyn, Jill and Stevenson, Dennis William and Renzaglia, Karen S. and Chen, Hongfeng and Zhang, Li and Zhang, Shouzhou and Mackenzie, Roy and Moreno, Javier E. and Melkonian, Michael and Wei, Tong and Gu, Ying and Xu, Xun and Rensing, Stefan A. and Huang, Jinling and Long, Manyuan and Goffinet, Bernard and Bowman, John L. and Van de Peer, Yves and Liu, Huan and Liu, Yang},
  doi          = {10.1038/s41588-025-02325-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2562-2569},
  shortjournal = {Nat. Genet.},
  title        = {Bryophytes hold a larger gene family space than vascular plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multi-tissue single-cell expression atlas in cattle. <em>NG</em>, <em>57</em>(10), 2546-2561. (<a href='https://doi.org/10.1038/s41588-025-02329-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Systematic characterization of the molecular states of cells in livestock tissues is essential for understanding the cellular and genetic mechanisms underlying economically and ecologically important physiological traits. Here, as part of the Farm Animal Genotype-Tissue Expression (FarmGTEx) project, we describe a comprehensive reference map including 1,793,854 cells from 59 bovine tissues in calves and adult cattle, spanning both sexes, which reveals intra-tissue and inter-tissue cellular heterogeneity in gene expression, transcription factor regulation and intercellular communication. Integrative analysis with genetic variants that underpin bovine monogenic and complex traits uncovers cell types of relevance, such as spermatocytes, responsible for sperm motility and excitatory neurons for milk fat yield. Comparative analysis reveals similarities in gene expression between cattle and humans, allowing for the detection of relevant cell types to study human complex phenotypes. This Cattle Cell Atlas will serve as a key resource for cattle genetics and genomics, selective breeding and comparative biology. This study introduces the Cattle Cell Atlas, a single-cell expression resource including 1,793,854 cells from 59 tissues. Integrative analyses leveraging this atlas provide insights into the biology underlying bovine monogenic and complex traits.},
  archive      = {J_NG},
  author       = {Han, Bo and Li, Houcheng and Zheng, Weijie and Zhang, Qi and Chen, Ao and Zhu, Senlin and Shi, Tao and Wang, Fei and Zou, Dong and Song, Yu and Ye, Wen and Du, Aixia and Fu, Yihan and Jia, Minghui and Bai, Zhonghao and Yuan, Zhixiang and Liu, Wansheng and Tuo, Wenbin and Hope, Jayne C. and MacHugh, David E. and O’Grady, John F. and Madsen, Ole and Sahana, Goutam and Luo, Yonglun and Lin, Lin and Li, Congjun and Cai, Zexi and Li, Bingjie and Huang, Jinming and Liu, Lin and Zhang, Zhang and Ma, Zhu and Hou, Yali and Liu, George E. and Jiang, Yu and Sun, Hui-zeng and Fang, Lingzhao and Sun, Dongxiao},
  doi          = {10.1038/s41588-025-02329-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2546-2561},
  shortjournal = {Nat. Genet.},
  title        = {A multi-tissue single-cell expression atlas in cattle},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic and epigenetic screens in primary human t cells link candidate causal autoimmune variants to t cell networks. <em>NG</em>, <em>57</em>(10), 2536-2545. (<a href='https://doi.org/10.1038/s41588-025-02301-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variants associated with autoimmune diseases are highly enriched within putative cis-regulatory regions of CD4+ T cells, suggesting that they could alter disease risk through changes in gene regulation. However, very few genetic variants have been shown to affect T cell gene expression or function. Here we tested &gt;18,000 autoimmune disease-associated variants for allele-specific effects on expression using massively parallel reporter assays in primary human CD4+ T cells. We find 545 variants that modulate expression in an allele-specific manner (emVars). Primary T cell emVars greatly enrich for likely causal variants, are mediated by common upstream pathways and their putative target genes are highly enriched within a lymphocyte activation network. Using bulk and single-cell CRISPR-interference screens, we confirm that emVar-containing T cell cis-regulatory elements modulate both known and previously unappreciated target genes that regulate T cell proliferation, providing plausible mechanisms by which these variants alter autoimmune disease risk. Massively parallel reporter assay in primary human CD4+ T cells and bulk and single-cell CRISPR-interference screens identify candidate causal variants linked to autoimmune disease risk that modulate T cell gene expression and proliferation.},
  archive      = {J_NG},
  author       = {Ho, Ching-Huang and Dippel, Maxwell A. and McQuade, Meghan S. and Nguyen, LeAnn P. and Mishra, Arpit and Pribitzer, Stephan and Hardy, Samantha and Chandok, Harshpreet and Chardon, Florence M. and McDiarmid, Troy A. and DeBerg, Hannah A. and Buckner, Jane H. and Shendure, Jay and de Boer, Carl G. and Guo, Michael H. and Tewhey, Ryan and Ray, John P.},
  doi          = {10.1038/s41588-025-02301-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2536-2545},
  shortjournal = {Nat. Genet.},
  title        = {Genetic and epigenetic screens in primary human t cells link candidate causal autoimmune variants to t cell networks},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq. <em>NG</em>, <em>57</em>(10), 2522-2535. (<a href='https://doi.org/10.1038/s41588-025-02343-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies. This study introduces single-cell transcription factor (TF) sequencing, a single-cell barcoded and doxycycline-inducible TF overexpression approach that reveals dose-sensitive functional classes of TFs and cellular heterogeneity by mapping TF dose-dependent transcriptomic changes during the reprogramming of mouse embryonic multipotent stromal cells.},
  archive      = {J_NG},
  author       = {Liu, Wangjie and Saelens, Wouter and Rainer, Pernille and Biočanin, Marjan and Gardeux, Vincent and Gralak, Antoni Jakub and van Mierlo, Guido and Gebhart, Angelika and Russeil, Julie and Liu, Tingdang and Chen, Wanze and Deplancke, Bart},
  doi          = {10.1038/s41588-025-02343-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2522-2535},
  shortjournal = {Nat. Genet.},
  title        = {Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation influences human centromere positioning and function. <em>NG</em>, <em>57</em>(10), 2509-2521. (<a href='https://doi.org/10.1038/s41588-025-02324-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maintaining the epigenetic identity of centromeres is essential to prevent genome instability. Centromeres are epigenetically defined by the histone H3 variant CENP-A. Prior work in human centromeres has shown that CENP-A is associated with regions of hypomethylated DNA located within large arrays of hypermethylated repeats, but the functional importance of these DNA methylation (DNAme) patterns remains poorly understood. To address this, we developed tools to perturb centromeric DNAme, revealing that it causally influences CENP-A positioning. We show that rapid loss of methylation results in increased binding of centromeric proteins and alterations in centromere architecture, leading to aneuploidy and reduced cell viability. We also demonstrate that gradual centromeric DNA demethylation prompts a process of cellular adaptation. Altogether, we find that DNAme causally influences CENP-A localization and centromere function, offering mechanistic insights into pathological alterations of centromeric DNAme. Genome-wide and targeted perturbation of DNA methylation at centromeres affects CENP-A positioning and centromere structure, resulting in aneuploidy and reduced cell viability.},
  archive      = {J_NG},
  author       = {Salinas-Luypaert, Catalina and Dubocanin, Danilo and Lee, Rosa Jooyoung and Andrade Ruiz, Lorena and Gamba, Riccardo and Grison, Marine and Velikovsky, Leonid and Angrisani, Annapaola and Scelfo, Andrea and Xu, Yuan and Dumont, Marie and Barra, Viviana and Wilhelm, Therese and Velasco, Guillaume and Losito, Marialucrezia and Wardenaar, René and Francastel, Claire and Foijer, Floris and Kops, Geert J. P. L. and Miga, Karen H. and Altemose, Nicolas and Fachinetti, Daniele},
  doi          = {10.1038/s41588-025-02324-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2509-2521},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation influences human centromere positioning and function},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell multi-omic and spatial profiling of esophageal squamous cell carcinoma reveals the immunosuppressive role of GPR116+ pericytes in cancer metastasis. <em>NG</em>, <em>57</em>(10), 2494-2508. (<a href='https://doi.org/10.1038/s41588-025-02341-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor metastasis leads to most cancer deaths. However, how cellular diversity and dynamic cooperation within the tumor microenvironment contribute to metastasis remains poorly understood. Here we leverage single-cell multi-omics (16 samples, 117,169 cells) and spatial transcriptomics (five samples, 195,366 cells) to uncover the cellular and spatial architecture of esophageal squamous cell carcinoma (ESCC), and characterize an immunosuppressive GPR116+ pericyte subset promoting tumor metastasis and immunotherapy resistance. GPR116+ pericyte enrichment is transcriptionally regulated by PRRX1, evidenced by pericyte-specific Prrx1 knockout mice. Mechanistically, GPR116+ pericytes secrete EGFL6 to bind integrin β1 on cancer cells, activating the NF-κB pathway to facilitate metastasis. Serum EGFL6 serves as a noninvasive biomarker for the diagnosis and prognosis of several tumors. Blocking integrin β1 suppresses metastasis and improves immunotherapy response in animal models of ESCC. Collectively, we provide a spatially resolved landscape of the prometastatic tumor microenvironment in ESCC and highlight the biological and clinical importance of GPR116+ pericytes, proposing potential innovative therapeutic strategies for metastatic cancers. This study provides a single-cell and spatial atlas of the prometastatic tumor microenvironment in esophageal squamous cell carcinoma and characterizes the immunosuppressive function of GPR116+ pericytes in cancer metastasis and immune evasion.},
  archive      = {J_NG},
  author       = {Pei, Xiaoya and Liu, Zhichao and Tang, Lin and Zhang, Junpeng and He, Yan and Zhuang, Xiaowan and Song, Yuxiang and Peng, Shaocong and Zhao, Yahui and Huang, Xuanzhang and Xu, Yiwei and Li, Junyi and Peng, Liang and Shi, Xingyuan and Zhang, Fan and Liu, Chong and He, Mingliang and Dai, Wei and Wu, Liang and Law, Simon and Liu, Jinbao and Wang, Zhenning and Li, Zhigang and Xu, Wenwen and Liu, Zhihua and Li, Bin},
  doi          = {10.1038/s41588-025-02341-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2494-2508},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-omic and spatial profiling of esophageal squamous cell carcinoma reveals the immunosuppressive role of GPR116+ pericytes in cancer metastasis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial signatures for predicting immunotherapy outcomes using multi-omics in non-small cell lung cancer. <em>NG</em>, <em>57</em>(10), 2482-2493. (<a href='https://doi.org/10.1038/s41588-025-02351-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Non-small cell lung cancer (NSCLC) shows variable responses to immunotherapy, highlighting the need for biomarkers to guide patient selection. We applied a spatial multi-omics approach to 234 advanced NSCLC patients treated with programmed death 1-based immunotherapy across three cohorts to identify biomarkers associated with outcome. Spatial proteomics (n = 67) and spatial compartment-based transcriptomics (n = 131) enabled profiling of the tumor immune microenvironment (TIME). Using spatial proteomics, we identified a resistance cell-type signature including proliferating tumor cells, granulocytes, vessels (hazard ratio (HR) = 3.8, P = 0.004) and a response signature, including M1/M2 macrophages and CD4 T cells (HR = 0.4, P = 0.019). We then generated a cell-to-gene resistance signature using spatial transcriptomics, which was predictive of poor outcomes (HR = 5.3, 2.2, 1.7 across Yale, University of Queensland and University of Athens cohorts), while a cell-to-gene response signature predicted favorable outcomes (HR = 0.22, 0.38 and 0.56, respectively). This framework enables robust TIME modeling and identifies biomarkers to support precision immunotherapy in NSCLC. This study uses spatial proteomic and spatial compartment-based whole-transcriptome profiling to develop predictive models for immunotherapy outcomes in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Aung, Thazin N. and Monkman, James and Warrell, Jonathan and Vathiotis, Ioannis and Bates, Katherine M. and Gavrielatou, Niki and Trontzas, Ioannis P. and Tan, Chin Wee and Fernandez, Aileen I. and Moutafi, Myrto and O’ Byrne, Ken and Schalper, Kurt A. and Syrigos, Konstantinos and Herbst, Roy S. and Kulasinghe, Arutha and Rimm, David L.},
  doi          = {10.1038/s41588-025-02351-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2482-2493},
  shortjournal = {Nat. Genet.},
  title        = {Spatial signatures for predicting immunotherapy outcomes using multi-omics in non-small cell lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Targeting histone H2B acetylated enhanceosomes via p300/CBP degradation in prostate cancer. <em>NG</em>, <em>57</em>(10), 2468-2481. (<a href='https://doi.org/10.1038/s41588-025-02336-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prostate cancer is driven by oncogenic transcription factor enhanceosomes comprising chromatin and epigenetic regulators. The lysine acetyltransferases p300 and CREB-binding protein (CBP) are key cofactors that activate enhancers through histone acetylation. Here we identify p300/CBP-mediated multisite histone H2B N-terminal acetylation (H2BNTac) as a defining feature of oncogenic enhanceosomes in androgen receptor (AR)-positive prostate cancer. p300/CBP are essential for AR and ETS transcription factor ERG transcriptional activity, and their dual degradation eliminates H2BNTac and histone H3 lysine 27 acetylation at hyperactive enhancers, leading to stronger suppression of oncogenic transcription than targeting either paralog or bromodomain alone. Cytotoxicity profiling across &gt;900 cell lines revealed that tumors with high H2BNTac, including AR-positive prostate cancer, are selectively dependent on p300/CBP. In preclinical models, systemic p300/CBP degradation inhibited tumor growth, synergized with AR antagonists and showed no evident toxicity. These findings position H2BNTac as an epigenetic marker of enhancer addiction and establish dual p300/CBP degradation as a promising therapeutic strategy for enhancer-driven cancers. This study shows that p300/CBP-dependent H2B acetylation is crucial for maintaining transcription of oncogenic gene programs in androgen receptor-driven prostate cancer.},
  archive      = {J_NG},
  author       = {Luo, Jie and Chen, Zhixiang and Qiao, Yuanyuan and Tien, Jean Ching-Yi and Young, Eleanor and Mannan, Rahul and Mahapatra, Somnath and Bhattacharyya, Rupam and Xiao, Lanbo and He, Tongchen and Eyunni, Sanjana and Zhang, Yuping and Zheng, Yang and Su, Fengyun and Cao, Xuhong and Wang, Rui and Cheng, Yunhui and Seri, Rithvik and George, James and Shahine, Miriam and Miner, Stephanie J. and Rees, Matthew G. and Ronan, Melissa M. and Roth, Jennifer A. and Vaishampayan, Ulka and Wang, Mi and Wang, Shaomeng and Parolia, Abhijit and Chinnaiyan, Arul M.},
  doi          = {10.1038/s41588-025-02336-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2468-2481},
  shortjournal = {Nat. Genet.},
  title        = {Targeting histone H2B acetylated enhanceosomes via p300/CBP degradation in prostate cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid epigenomic classification of acute leukemia. <em>NG</em>, <em>57</em>(10), 2456-2467. (<a href='https://doi.org/10.1038/s41588-025-02321-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acute leukemia requires precise molecular classification and urgent treatment. However, standard-of-care diagnostic tests are time-intensive and do not capture the full spectrum of acute leukemia heterogeneity. Here, we developed a framework to classify acute leukemia using genome-wide DNA methylation profiling. We first assembled a comprehensive reference cohort (n = 2,540 samples) and defined 38 methylation classes. Methylation-based classification matched standard-pathology lineage classification in most cases and revealed heterogeneity in addition to that captured by genetic categories. Using this reference, we developed a neural network (MARLIN; methylation- and AI-guided rapid leukemia subtype inference) for acute leukemia classification from sparse DNA methylation profiles. In retrospective cohorts profiled by nanopore sequencing, high-confidence predictions were concordant with conventional diagnoses in 25 out of 26 cases. Real-time MARLIN classification in patients with suspected acute leukemia provided accurate predictions in five out of five cases, which were typically generated within 2 h of sample receipt. In summary, we present a framework for rapid acute leukemia classification that complements and enhances standard-of-care diagnostics. The authors present a molecular classification of acute leukemia using 5-methylcytosine signatures, together with a neural network-based classifier for clinical use.},
  archive      = {J_NG},
  author       = {Steinicke, Til L. and Benfatto, Salvatore and Capilla-Guerra, Maria R. and Monteleone, Andre B. and Young, Jonathan H. and Shankar, Subha and Michaels, Phillip D. and Tsai, Harrison K. and Good, Jonathan D. and Kreso, Antonia and van Galen, Peter and Schliemann, Christoph and Chen, Evan C. and Griffin, Gabriel K. and Hovestadt, Volker},
  doi          = {10.1038/s41588-025-02321-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2456-2467},
  shortjournal = {Nat. Genet.},
  title        = {Rapid epigenomic classification of acute leukemia},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A genetic map of human metabolism across the allele frequency spectrum. <em>NG</em>, <em>57</em>(10), 2445-2455. (<a href='https://doi.org/10.1038/s41588-025-02355-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies of human metabolism have been limited in scale and allelic breadth. Here we provide a data-driven map of the genetic regulation of circulating small molecules and lipoprotein characteristics (249 traits) measured using proton nuclear magnetic resonance spectroscopy across the allele frequency spectrum in ~450,000 individuals. Trans-ancestral meta-analyses identify 29,824 locus–metabolite associations mapping to 753 regions with effects largely consistent between men and women and large ancestral groups represented in UK Biobank. We observe and classify extreme genetic pleiotropy, identify regulators of lipid metabolism, and assign effector genes at &gt;100 loci through rare-to-common allelic series. We propose roles for genes less established in metabolic control (for example, SIDT2), genes characterized by phenotypic heterogeneity (for example, APOA1) and genes with specific disease relevance (for example, VEGFA). Our study demonstrates the value of broad, large-scale metabolomic phenotyping to identify and characterize regulators of human metabolism. Genome-wide association analyses of 249 circulating small molecules and lipoprotein characteristics across the allele frequency spectrum in UK Biobank advance understanding of genes and pathways regulating human metabolism.},
  archive      = {J_NG},
  author       = {Zoodsma, Martijn and Beuchel, Carl and Yasmeen, Summaira and Kohleick, Leonhard and Nepal, Aakash and Koprulu, Mine and Kronenberg, Florian and Mayr, Manuel and Williamson, Alice and Pietzner, Maik and Langenberg, Claudia},
  doi          = {10.1038/s41588-025-02355-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2445-2455},
  shortjournal = {Nat. Genet.},
  title        = {A genetic map of human metabolism across the allele frequency spectrum},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Population-scale gene-based analysis of whole-genome sequencing provides insights into metabolic health. <em>NG</em>, <em>57</em>(10), 2436-2444. (<a href='https://doi.org/10.1038/s41588-025-02364-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In addition to its coverage of the noncoding genome, whole-genome sequencing (WGS) may better capture the coding genome than exome sequencing. Here we sought to exploit this and identify new rare, protein-coding variants associated with metabolic health in WGS data (n = 708,956) from the UK Biobank and All of Us studies. Identified genes highlight new biological mechanisms, including protein-truncating variants (PTVs) in the DNA double-strand break repair gene RIF1 that have a substantial effect on body mass index (2.66 kg m−2, s.e. 0.43, P = 3.7 × 10−10). UBR3 is an intriguing example where PTVs independently increase body mass index and type 2 diabetes risk. Furthermore, PTVs in IRS2 have a substantial effect on type 2 diabetes (odds ratio 6.4 (3.7–11.3), P = 9.9 × 10−14, 34% case prevalence among carriers) and were also associated with chronic kidney disease independent of diabetes status, suggesting an important role for IRS2 in maintaining renal health. Our study demonstrates that large-scale WGS provides new mechanistic insights into human metabolic phenotypes through improved capture of coding sequences. Analyses of whole-genome sequencing data from UK Biobank and All of Us identify rare variant burden signals associated with metabolic health, including effects of protein-truncating variants in IRS2 on type 2 diabetes and chronic kidney disease risk.},
  archive      = {J_NG},
  author       = {Zhao, Yajie and Lockhart, Sam and Liu, Jimmy and Li, Xihao and Cortes, Adrian and Hua, Xing and Gardner, Eugene J. and Kentistou, Katherine A. and Cañadas-Garre, Marisa and Fabian, Laurie and Ho, Karen and Timpson, Nicholas and Lo, Yancy and Davitte, Jonathan and Savage, David B. and Buser-Doepner, Carolyn and Ong, Ken K. and Zhang, Haoyu and Scott, Robert and O’Rahilly, Stephen and Perry, John R. B.},
  doi          = {10.1038/s41588-025-02364-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2436-2444},
  shortjournal = {Nat. Genet.},
  title        = {Population-scale gene-based analysis of whole-genome sequencing provides insights into metabolic health},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes. <em>NG</em>, <em>57</em>(10), 2427-2435. (<a href='https://doi.org/10.1038/s41588-025-02295-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We performed a genome-wide association meta-analysis (GWAMA) of 290,134 attention-deficit/hyperactivity disorder (ADHD) symptom measures of 70,953 unique individuals from multiple raters, ages and instruments (ADHDSYMP). Next, we meta-analyzed the results with a study of ADHD diagnosis (ADHDOVERALL). ADHDSYMP returned no genome-wide significant variants. We show that the combined ADHDOVERALL GWAMA identified 39 independent loci, of which 17 were new. Using a recently developed gene-mapping method, Fine-mapped Locus Assessment Model of Effector genes, we identified 22 potential ADHD effector genes implicating several new biological processes and pathways. Moderate negative genetic correlations (rg &lt; −0.40) were observed with multiple cognitive traits. In three cohorts, polygenic scores (PGSs) based on ADHDOVERALL outperformed PGSs based on ADHD symptoms and diagnosis alone. Our findings support the notion that clinical ADHD is at the extreme end of a continuous liability that is indexed by ADHD symptoms. We show that including ADHD symptom counts helps to identify new genes implicated in ADHD. Genome-wide association meta-analyses of attention-deficit/hyperactivity disorder symptom measures and clinical diagnoses identify new risk loci, highlight putative effector genes and yield improved polygenic risk scores.},
  archive      = {J_NG},
  author       = {van der Laan, Camiel M. and Ip, Hill F. and Schipper, Marijn and Hottenga, Jouke-Jan and St Pourcain, Beate and Zayats, Tetyana and Pool, René and Krapohl, Eva M. L. and Brikell, Isabell and Soler Artigas, María and Cabana-Domínguez, Judit and Llonga, Natalia and Nolte, Ilja M. and Bolhuis, Koen and Palviainen, Teemu and Zafarmand, Hadi and Gordon, Scott and Aliev, Fazil and Burt, S. Alexandra and Wang, Carol A. and Saunders, Gretchen and Karhunen, Ville and Adkins, Daniel E. and Border, Richard and Peterson, Roseann E. and Prinz, Joseph A. and Thiering, Elisabeth and Vilor-Tejedor, Natàlia and Ahluwalia, Tarunveer S. and Allegrini, Andrea and Rimfeld, Kaili and Chen, Qi and Lu, Yi and Martin, Joanna and Bosch, Rosa and Ramos-Quiroga, Josep Antoni and Neumann, Alexander and Ensink, Judith and Grasby, Katrina L. and Morosoli, José J. and Tong, Xiaoran and Marrington, Shelby and Scott, James G. and Shabalin, Andrey A. and Corley, Robin and Evans, Luke M. and Sugden, Karen and Alemany, Silvia and Sass, Lærke and Vinding, Rebecca and Ehli, Erik A. and Hagenbeek, Fiona A. and Derks, Eske M. and Larsson, Henrik and Snieder, Harold and Cecil, Charlotte and Whipp, Alyce M. and Korhonen, Tellervo and Vuoksimaa, Eero and Rose, Richard J. and Uitterlinden, André G. and Haavik, Jan and Harris, Jennifer R. and Helgeland, Øyvind and Johansson, Stefan and Knudsen, Gun Peggy S. and Njolstad, Pal Rasmus and Lu, Qing and Rodriguez, Alina and Henders, Anjali K. and Mamun, Abdullah and Najman, Jackob M. and Brown, Sandy and Hopfer, Christian and Krauter, Kenneth and Reynolds, Chandra A. and Smolen, Andrew and Stallings, Michael and Wadsworth, Sally and Wall, Tamara L. and Eaves, Lindon and Silberg, Judy L. and Miller, Allison and Havdahl, Alexandra and Llop, Sabrina and Lopez-Espinosa, Maria-Jose and Bønnelykke, Klaus and Sunyer, Jordi and Arseneault, Louise and Standl, Marie and Heinrich, Joachim and Boden, Joseph and Pearson, John and Horwood, John and Kennedy, Martin and Poulton, Richie and Maes, Hermine H. and Hewitt, John and Copeland, William E. and Middeldorp, Christel M. and Williams, Gail M. and Wray, Naomi and Järvelin, Marjo-Riitta and McGue, Matt and Iacono, William and Caspi, Avshalom and Moffitt, Terrie E. and Whitehouse, Andrew J. O. and Pennell, Craig E. and Klump, Kelly L. and Jiang, Chang and Dick, Danielle M. and Reichborn-Kjennerud, Ted and Martin, Nicholas G. and Medland, Sarah E. and Vrijkotte, Tanja and Kaprio, Jaakko and Tiemeier, Henning and Davey Smith, George and Hartman, Catharina A. and Oldehinkel, Albertine J. and Casas, Miquel and Ribasés, Marta and Lichtenstein, Paul and Lundström, Sebastian and Plomin, Robert and Bartels, Meike and Nivard, Michel G. and Boomsma, Dorret I.},
  doi          = {10.1038/s41588-025-02295-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2427-2435},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Limited overlap between genetic effects on disease susceptibility and disease survival. <em>NG</em>, <em>57</em>(10), 2418-2426. (<a href='https://doi.org/10.1038/s41588-025-02342-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding disease progression is of high biological and clinical interest. Unlike disease susceptibility, whose genetic basis has been abundantly studied, less is known about the genetics of disease progression and its overlap with disease susceptibility. Considering nine common diseases (ncases ranging from 11,980 to 124,682) across seven biobanks, we systematically compared genetic architectures of susceptibility and progression, defined as disease-specific mortality. We identified only one locus substantially associated with disease-specific mortality and showed that, at a similar sample size, more genome-wide significant loci can be identified in a genome-wide association study of disease susceptibility. Variants substantially affecting disease susceptibility were weakly or not associated with disease-specific mortality. Moreover, susceptibility polygenic scores (PGSs) were weak predictors of disease-specific mortality, while a PGS for general lifespan was substantially associated with disease-specific mortality for seven of nine diseases. We explored alternative definitions of disease progression and found that genetic signals for macrovascular complications in type 2 diabetes overlap with similar phenotypes in the general population; however, these effects are attenuated. Overall, our findings indicate limited similarity in genetic effects between disease susceptibility and disease-specific mortality, suggesting that larger sample sizes, different measures of progression, or the integration of related phenotypes from the general population is needed to identify the genetic underpinnings of disease progression. Systematic comparison of genome-wide association results for disease risk and disease-specific mortality for nine common diseases across seven biobanks finds limited overlap between genetic effects on disease susceptibility and survival.},
  archive      = {J_NG},
  author       = {Yang, Zhiyu and Pajuste, Fanny-Dhelia and Zguro, Kristina and Cheng, Yipeng and Kurant, Danielle E. and Eoli, Andrea and Wanner, Julian and Jermy, Bradley and Rämö, Joel and Kanoni, Stavroula and van Heel, David A. and Hayward, Caroline and Marioni, Riccardo E. and McCartney, Daniel L. and Renieri, Alessandra and Furini, Simone and Mägi, Reedik and Gusev, Alexander and Drineas, Petros and Paschou, Peristera and Heyne, Henrike and Ripatti, Samuli and Mars, Nina and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02342-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2418-2426},
  shortjournal = {Nat. Genet.},
  title        = {Limited overlap between genetic effects on disease susceptibility and disease survival},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pan-UK biobank genome-wide association analyses enhance discovery and resolution of ancestry-enriched effects. <em>NG</em>, <em>57</em>(10), 2408-2417. (<a href='https://doi.org/10.1038/s41588-025-02335-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks, such as the UK Biobank (UKB), enable massive phenome by genome-wide association studies that elucidate genetic etiology of complex traits. However, people from diverse genetic ancestry groups are often excluded from association analyses due to concerns about population structure introducing false positive associations. Here we generate mixed model associations and meta-analyses across genetic ancestry groups, inclusive of a larger fraction of the UK Biobank than previous efforts, to produce freely available summary statistics for 7,266 traits. We build a quality control and analysis framework informed by genetic architecture. Overall, we identify 14,676 significant loci (P &lt; 5 × 10−8) in the meta-analysis that were not found in the EUR genetic ancestry group alone, including new associations, for example between CAMK2D and triglycerides. We also highlight associations from ancestry-enriched variation, including a known pleiotropic missense variant in G6PD associated with several biomarker traits. We release these results publicly alongside frequently asked questions that describe caveats for interpretation of results, enhancing available resources for interpretation of risk variants across diverse populations. Genome-wide analyses for 7,266 traits leveraging data from several genetic ancestry groups in UK Biobank identify new associations and enhance resources for interpreting risk variants across diverse populations.},
  archive      = {J_NG},
  author       = {Karczewski, Konrad J. and Gupta, Rahul and Kanai, Masahiro and Lu, Wenhan and Tsuo, Kristin and Wang, Ying and Walters, Raymond K. and Turley, Patrick and Callier, Shawneequa and Shah, Nirav N. and Baya, Nikolas and Palmer, Duncan S. and Goldstein, Jacqueline I. and Sarma, Gopal and Solomonson, Matthew and Cheng, Nathan and Bryant, Sam and Churchhouse, Claire and Cusick, Caroline M. and Poterba, Timothy and Compitello, John and King, Daniel and Zhou, Wei and Seed, Cotton and Finucane, Hilary K. and Daly, Mark J. and Neale, Benjamin M. and Atkinson, Elizabeth G. and Martin, Alicia R.},
  doi          = {10.1038/s41588-025-02335-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2408-2417},
  shortjournal = {Nat. Genet.},
  title        = {Pan-UK biobank genome-wide association analyses enhance discovery and resolution of ancestry-enriched effects},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Non-CG DNA methylation in animal genomes. <em>NG</em>, <em>57</em>(10), 2395-2407. (<a href='https://doi.org/10.1038/s41588-025-02303-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine DNA methylation is widespread in animal genomes and occurs predominantly at CG dinucleotides (mCG). While the roles of mCG, such as in genomic imprinting and genome stability, are well established, non-CG DNA methylation (mCH) remains poorly understood. In most vertebrate tissues, roughly 80% of CGs are methylated, whereas mCH levels are generally low, typically ranging from 1% to 3%. In vertebrates, mCH is most prevalent in neural tissue, oocytes and embryonic stem cells and has been linked to neurodevelopmental disorders. Moreover, mCH appears to have a conserved role in regulating vertebrate neural genomes, and recent studies suggest that it has functions in the embryogenesis of teleost fish. Overall, mCH represents an intriguing emerging aspect of gene regulation with potential implications for cellular identity, repeat silencing and neural function. In this Review, we provide a critical overview of the patterning, mechanisms and functional implications of mCH in animals. This Review discusses noncanonical DNA methylation (mCH) in animal genomes and highlights the remaining need to clarify whether mCH represents a conserved regulatory layer or a lineage-specific epigenetic feature with distinct biological roles.},
  archive      = {J_NG},
  author       = {Brethouwer, Thirsa and de Mendoza, Alex and Bogdanovic, Ozren},
  doi          = {10.1038/s41588-025-02303-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2395-2407},
  shortjournal = {Nat. Genet.},
  title        = {Non-CG DNA methylation in animal genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing biological understanding of cellular senescence with computational multiomics. <em>NG</em>, <em>57</em>(10), 2381-2394. (<a href='https://doi.org/10.1038/s41588-025-02314-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular senescence is a complex biological process that plays a pathophysiological role in aging and age-related diseases. The biological understanding of senescence at the cellular and tissue levels remains incomplete due to the lack of specific biomarkers as well as the relative rarity of senescent cells, their phenotypic heterogeneity and dynamic features. This Review provides a comprehensive overview of multiomic approaches for the characterization and biological understanding of cellular senescence. The technical capability and challenges of each approach are discussed, and practical guidelines are provided for selecting tools for identifying, characterizing and spatially mapping senescent cells. The importance of computational analyses in multiomics research, including senescent cell identification, signature detection and interactions of senescent cells with microenvironments, is highlighted. Moreover, tissue-specific case studies and experimental design considerations for individual organs are presented. Finally, future directions and the potential impact of multiomic approaches on the biological understanding of cellular senescence are discussed. This Review discusses multiomic approaches for the characterization and biological understanding of cellular senescence, including detailed case studies on skeletal muscle and adipose tissue that highlight current outstanding issues in the field.},
  archive      = {J_NG},
  author       = {Li, Sheng and Agudelo Garcia, Paula A. and Aliferis, Constantin and Becich, Michael J. and Calyeca, Jazmin and Cosgrove, Benjamin D. and Elisseeff, Jennifer and Farzad, Negin and Fertig, Elana J. and Glass, Carolyn and Gu, Liangcai and Hu, Qianjiang and Ji, Zhicheng and Königshoff, Melanie and LeBrasseur, Nathan K. and Li, Dongmei and Ma, Anjun and Ma, Qin and Menon, Vilas and Mitchell, Jacob T. and Mora, Ana L. and Nagaraj, Sushma and Nelson, Andrew C. and Niedernhofer, Laura J. and Rojas, Mauricio and Taha, Hash Brown and Wang, Jinhua and Wang, Siyuan and Wu, Pei-Hsun and Xie, Jichun and Xu, Ming and Yu, Miao and Zhang, Xu and Zhao, Yue and Adams, Peter D. and Aguayo-Mazzucato, Cristina and Baker, Darren J. and Benz, Christopher and Bernlohr, David A. and Bueno, Marta and Chen, Jin and Childs, Bennett G. and Chuang, Jeffrey H. and Chung, Dongjun and Dileepan, Mythili and Ding, Li and Dong, Mingze and Duncan, Francesca and Enninful, Archibald and Flynn, William F. and Franco, Ana Catarina and Furman, David and Garovic, Vesna and Halene, Stephanie and Herman, Allison B. and Hertzel, Ann V. and Iwasaki, Kanako and Jeon, Hyeongseon and Kang, Jeon Woong and Karmakar, Shilpita and Kirkland, James L. and Korstanje, Ron and Kummerfeld, Erich and Lee, Jun Hee and Liu, Yang and Lu, Yao and Lugo-Martinez, Jose and Martini, Helene and Melov, Simon and Musi, Nicolas and Passos, João F. and Peters, Samuel T. and Rahman, Irfan and Ramasamy, Ramalakshmi and Rindone, Alexandra N. and Robbins, Paul D. and Robson, Paul and Rodriguez-Lopez, Jhonny and Rosas, Lorena and Rosenthal, Nadia and Schafer, Marissa J. and Schilling, Birgit and Schmidt, Elizabeth L. and Schneider, Kevin and Sengupta, Kaustav and Shu, Jian and So, Peter T. C. and Sun, Li and Tchkonia, Tamar and Teneche, Marcos G. and Vanegas, Natalia and Wang, Julia and Xie, Juan and Yin, Shanshan and Zhang, Ke and Zhu, Quan and Fan, Rong},
  doi          = {10.1038/s41588-025-02314-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2381-2394},
  shortjournal = {Nat. Genet.},
  title        = {Advancing biological understanding of cellular senescence with computational multiomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A standardized framework for circulating blood proteomics. <em>NG</em>, <em>57</em>(10), 2371-2380. (<a href='https://doi.org/10.1038/s41588-025-02319-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The circulating blood proteome holds immense potential for biomarker discovery and understanding disease mechanisms. Notable advances in mass spectrometry and affinity-based technologies have been made, but data integration across studies and platforms is hindered by the absence of unified analytical standards. This limitation impedes comprehensive exploration of human biology across diverse phenotypes and cohorts as well as the translation of findings into clinical applications. The disparities between datasets, stemming from a combination of factors related to differences in sample collection, pre-analytical handling, measurement methods and instrumentation, further complicate data integration. In this Perspective, we outline key challenges in blood-based proteomics and propose actionable strategies. Central to our recommendations are high-quality, technology-agnostic reference samples, which can bridge disparate datasets and enable robust cross-study comparisons. By fostering interconnected investigations across proteomic technologies, blood sample collections, clinical phenotypes and different populations, these references will accelerate the field and its translation. The field of blood proteomics faces an upsurge of data with the challenge of cross-study comparisons. This Perspective offers an in-depth analysis and proposes reference materials to enhance data integration and accelerate clinical translation.},
  archive      = {J_NG},
  author       = {Cai, Xue and Geyer, Philipp E. and Perez-Riverol, Yasset and Omenn, Gilbert S. and Dong, Lianhua and Winkler, Robert and Ahadi, Sara and Lössl, Philip and Yu, Xiaobo and Chang, Cheng and Ralser, Markus and Jimenez, Connie R. and Zhao, Yang and Chen, Yu-Ju and Poon, Terence C. W. and Bache, Nicolai and Shi, Leming and Dai, Xinhua and Wang, Ziyue and Zhu, Yi and Fang, Xiang and Schwenk, Jochen M. and Van Eyk, Jennifer E. and Völker, Uwe and Guo, Tiannan},
  doi          = {10.1038/s41588-025-02319-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2371-2380},
  shortjournal = {Nat. Genet.},
  title        = {A standardized framework for circulating blood proteomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The solve-RD solvathons as a pan-european interdisciplinary collaboration to diagnose patients with rare disease. <em>NG</em>, <em>57</em>(10), 2361-2370. (<a href='https://doi.org/10.1038/s41588-025-02290-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in genomic diagnostics, the majority of individuals with rare diseases remain without a confirmed genetic diagnosis. The rapid emergence of advanced omics technologies, such as long-read genome sequencing, optical genome mapping and multiomic profiling, has improved diagnostic yield but also substantially increased analytical and interpretational complexity. Addressing this complexity requires systematic multidisciplinary collaboration, as recently demonstrated by targeted diagnostic workshops. Here, we highlight the experience of the Solve-RD consortium, a pan-European initiative, in implementing four structured workshops, termed ‘Solvathons’, as a regular and effective component of its operational workflow. We provide actionable insights, best practices and lessons learned for successful data integration, expert training and scalable collaborative diagnostics within large research consortia. This Perspective presents the Solve-RD Solvathon model, an innovative, pan-European framework uniting clinical and bioinformatics experts to diagnose rare diseases through integrative multiomics analysis and structured collaboration.},
  archive      = {J_NG},
  author       = {Yépez, Vicente A. and Demidov, German and Ellwanger, Kornelia and Laurie, Steven and Luknárová, Rebeka and Joseph Maran, Midhuna Immaculate and Hentrich, Thomas and Sagath, Lydia and van der Sanden, Bart and Astuti, Galuh and Neveling, Kornelia and Batlle-Masó, Laura and Beijer, Danique and Brechtmann, Felix and Caballero-Oteyza, Andrés and Dabad, Marc and Denommé-Pichon, Anne-Sophie and Doornbos, Cenna and Eddafir, Zakaria and Estévez-Arias, Berta and Kilicarslan, Ozge Aksel and Kolen, Ingrid H. M. and Kraß, Leon and Lohmann, Katja and Londhe, Shubhankar and López-Martín, Estrella and Maassen, Kars and Macken, William and Martínez-Delgado, Beatriz and Mei, Davide and Mertes, Christian and Minardi, Raffaella and Morsy, Heba and Mueller, Juliane S. and Natera-de Benito, Daniel and Nelson, Isabelle and Oud, Machteld M. and Paramonov, Ida and Picó, Daniel and Piscia, Davide and Polavarapu, Kiran and Raineri, Emanuele and Savarese, Marco and Smal, Noor and Steehouwer, Marloes and Steyaert, Wouter and Swertz, Morris A. and Thomsen, Mirja and Töpf, Ana and Van de Vondel, Liedewei and van der Vries, Gerben and Vitobello, Antonio and Wilke, Carlo and Zurek, Birte and t’ Hoen, Peter-Bram and Matalonga, Leslie and Vissers, Lisenka E. L. M. and Gilissen, Christian and Schulze-Hentrich, Julia and Beltran, Sergi and Esteve-Codina, Anna and Hoischen, Alexander and Gagneur, Julien and Graessner, Holm},
  doi          = {10.1038/s41588-025-02290-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2361-2370},
  shortjournal = {Nat. Genet.},
  title        = {The solve-RD solvathons as a pan-european interdisciplinary collaboration to diagnose patients with rare disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic perturbations link autoimmune risk variants to t cell function. <em>NG</em>, <em>57</em>(10), 2359-2360. (<a href='https://doi.org/10.1038/s41588-025-02302-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We used a massively parallel reporter assay (MPRA) to test thousands of autoimmune disease-associated genetic variants for allele-specific effects on gene expression in primary human T cells. Variants that altered gene expression within our MPRA were then linked to T cell regulatory networks and proliferation using bulk and single-cell CRISPR-interference screens.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02302-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2359-2360},
  shortjournal = {Nat. Genet.},
  title        = {Genetic perturbations link autoimmune risk variants to t cell function},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cracking the histone code for prostate cancer therapy. <em>NG</em>, <em>57</em>(10), 2357-2358. (<a href='https://doi.org/10.1038/s41588-025-02354-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, we uncover the critical role of p300/CBP-mediated histone H2B N terminus multisite lysine acetylation (H2BNTac) in defining oncogenic enhanceosomes in prostate cancer. Degradation of p300/CBP rapidly disables H2BNTac-marked oncogenic enhancers and represents a promising therapeutic strategy for enhancer-driven malignancies, including prostate cancer.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02354-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2357-2358},
  shortjournal = {Nat. Genet.},
  title        = {Cracking the histone code for prostate cancer therapy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The dawn of bryophyte genomics is here. <em>NG</em>, <em>57</em>(10), 2355-2356. (<a href='https://doi.org/10.1038/s41588-025-02240-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bryophytes are a widespread group of land plants that occupy nearly all biomes, yet their genetics and evolutionary history have long remained underexplored. Now, a study that generates extensive genomic data for bryophytes highlights de novo gene formation and horizontal gene transfer as key forces that shape bryophyte diversity and adaptation.},
  archive      = {J_NG},
  author       = {Potente, Giacomo and Yue, Yuling and Szövényi, Péter},
  doi          = {10.1038/s41588-025-02240-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2355-2356},
  shortjournal = {Nat. Genet.},
  title        = {The dawn of bryophyte genomics is here},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dipping into the function of DNA methylation at centromeres. <em>NG</em>, <em>57</em>(10), 2352-2354. (<a href='https://doi.org/10.1038/s41588-025-02334-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human centromeres contain a small chromatin region with low levels of DNA cytosine methylation that resides with CENP-A. Salinas-Luypaert et al. find a role of DNA methylation in maintaining the size and function of centromeres by controlling the binding affinity of key centromere components.},
  archive      = {J_NG},
  author       = {Erhardt, Sylvia},
  doi          = {10.1038/s41588-025-02334-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2352-2354},
  shortjournal = {Nat. Genet.},
  title        = {Dipping into the function of DNA methylation at centromeres},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A high-power spatial, single-cell view of pericytes in esophageal cancer metastasis. <em>NG</em>, <em>57</em>(10), 2350-2351. (<a href='https://doi.org/10.1038/s41588-025-02318-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using integrated single-cell multi-omics and spatial analysis, this study identifies a PRRX1-driven GPR116⁺ pericyte subpopulation that promotes tumor metastasis and immune evasion in esophageal squamous cell carcinoma, offering a diagnostic biomarker and potential therapeutic strategies.},
  archive      = {J_NG},
  author       = {Sinha, Uttam K. and Park, Young Min and Lin, De-Chen},
  doi          = {10.1038/s41588-025-02318-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2350-2351},
  shortjournal = {Nat. Genet.},
  title        = {A high-power spatial, single-cell view of pericytes in esophageal cancer metastasis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A plasma proteomic signature for ALS. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02384-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02384-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {A plasma proteomic signature for ALS},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recycling enhancers. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02385-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02385-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Recycling enhancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Detecting parent-of-origin genetic effects without parental genomes. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02386-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02386-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Detecting parent-of-origin genetic effects without parental genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Basal cells drive small cell lung cancer plasticity. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02387-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02387-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Basal cells drive small cell lung cancer plasticity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Realizing the full potential of our future health through data linkage and trans-biobank efforts. <em>NG</em>, <em>57</em>(10), 2341-2348. (<a href='https://doi.org/10.1038/s41588-025-02327-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scale and population coverage of Our Future Health, alongside other next-generation biobanks, offers unique opportunities to advance genomic medicine. Focusing on the UK context, we provide a researcher’s perspective of how this new resource could reach its full potential in a way that is impactful, user-friendly and informs related global efforts.},
  archive      = {J_NG},
  author       = {Straub, Vincent J. and Benonisdottir, Stefania and Kong, Augustine and Mills, Melinda C.},
  doi          = {10.1038/s41588-025-02327-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2341-2348},
  shortjournal = {Nat. Genet.},
  title        = {Realizing the full potential of our future health through data linkage and trans-biobank efforts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
