<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATCS</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs">NATCS - 18</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Probing the limitations of multimodal language models for chemistry and materials research. <em>NATCS</em>, <em>5</em>(10), 973. (<a href='https://doi.org/10.1038/s43588-025-00869-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Alampara, Nawaf and Schilling-Wilhelmi, Mara and Ríos-García, Martiño and Mandal, Indrajeet and Khetarpal, Pranav and Grover, Hargun Singh and Krishnan, N. M. Anoop and Jablonka, Kevin Maik},
  doi          = {10.1038/s43588-025-00869-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {973},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Probing the limitations of multimodal language models for chemistry and materials research},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SciToolAgent: A knowledge-graph-driven scientific agent for multitool integration. <em>NATCS</em>, <em>5</em>(10), 962-972. (<a href='https://doi.org/10.1038/s43588-025-00849-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientific research increasingly relies on specialized computational tools, yet effectively utilizing these tools requires substantial domain expertise. While large language models show promise in tool automation, they struggle to seamlessly integrate and orchestrate multiple tools for complex scientific workflows. Here we present SciToolAgent, a large language model-powered agent that automates hundreds of scientific tools across biology, chemistry and materials science. At its core, SciToolAgent leverages a scientific tool knowledge graph that enables intelligent tool selection and execution through graph-based retrieval-augmented generation. The agent also incorporates a comprehensive safety-checking module to ensure responsible and ethical tool usage. Extensive evaluations on a curated benchmark demonstrate that SciToolAgent outperforms existing approaches. Case studies in protein engineering, chemical reactivity prediction, chemical synthesis and metal–organic framework screening further demonstrate SciToolAgent’s capability to automate complex scientific workflows, making advanced research tools accessible to both experts and nonexperts. This study presents SciToolAgent, a large language model-based agent that orchestrates scientific tools via a knowledge graph, enabling automated and effective execution of scientific research workflows.},
  archive      = {J_NATCS},
  author       = {Ding, Keyan and Yu, Jing and Huang, Junjie and Yang, Yuchen and Zhang, Qiang and Chen, Huajun},
  doi          = {10.1038/s43588-025-00849-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {962-972},
  shortjournal = {Nat. Comput. Sci},
  title        = {SciToolAgent: A knowledge-graph-driven scientific agent for multitool integration},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the limitations of multimodal language models for chemistry and materials research. <em>NATCS</em>, <em>5</em>(10), 952-961. (<a href='https://doi.org/10.1038/s43588-025-00836-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advancements in artificial intelligence have sparked interest in scientific assistants that could support researchers across the full spectrum of scientific workflows, from literature review to experimental design and data analysis. A key capability for such systems is the ability to process and reason about scientific information in both visual and textual forms—from interpreting spectroscopic data to understanding laboratory set-ups. Here we introduce MaCBench, a comprehensive benchmark for evaluating how vision language models handle real-world chemistry and materials science tasks across three core aspects: data extraction, experimental execution and results interpretation. Through a systematic evaluation of leading models, we find that although these systems show promising capabilities in basic perception tasks—achieving near-perfect performance in equipment identification and standardized data extraction—they exhibit fundamental limitations in spatial reasoning, cross-modal information synthesis and multi-step logical inference. Our insights have implications beyond chemistry and materials science, suggesting that developing reliable multimodal AI scientific assistants may require advances in curating suitable training data and approaches to training those models. A comprehensive benchmark, called MaCBench, is developed to evaluate how vision language models handle different aspects of real-world chemistry and materials science tasks.},
  archive      = {J_NATCS},
  author       = {Alampara, Nawaf and Schilling-Wilhelmi, Mara and Ríos-García, Martiño and Mandal, Indrajeet and Khetarpal, Pranav and Grover, Hargun Singh and Krishnan, N. M. Anoop and Jablonka, Kevin Maik},
  doi          = {10.1038/s43588-025-00836-3},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {952-961},
  shortjournal = {Nat. Comput. Sci},
  title        = {Probing the limitations of multimodal language models for chemistry and materials research},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Real-time raw signal genomic analysis using fully integrated memristor hardware. <em>NATCS</em>, <em>5</em>(10), 940-951. (<a href='https://doi.org/10.1038/s43588-025-00867-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in third-generation sequencing have enabled portable and real-time genomic sequencing, but real-time data processing remains a bottleneck, hampering on-site genomic analysis. These technologies generate noisy analog signals that traditionally require basecalling and read mapping, both demanding costly data movement on von Neumann hardware. Here, to overcome this, we present a memristor-based hardware–software codesign that processes raw sequencer signals directly in analog memory, combining the two separated steps. By exploiting intrinsic device noise for locality-sensitive hashing and implementing parallel approximate searches in content-addressable memory, we experimentally showcase on-site applications, including infectious disease detection and metagenomic classification on a fully integrated memristor chip. Our experimentally validated analysis confirms the effectiveness of this approach on real-world tasks, achieving a 97.15% F1 score in virus raw signal mapping, with 51× speed-up and 477× energy saving over an application-specific integrated circuit. These results demonstrate that in-memory computing hardware provides a viable solution for integration with portable sequencers, enabling real-time and on-site genomic analysis. The authors report a memristor-based system that analyzes raw analog signals from a genomic sequencer directly in memory. By bypassing slow data conversion, the system achieves substantial improvements in speed and efficiency, enabling real-time, on-site genomic analysis.},
  archive      = {J_NATCS},
  author       = {He, Peiyi and Wang, Shengbo and Mao, Ruibin and Jiang, Mingrui and Siegel, Sebastian and Pedretti, Giacomo and Ignowski, Jim and Strachan, John Paul and Luo, Ruibang and Li, Can},
  doi          = {10.1038/s43588-025-00867-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {940-951},
  shortjournal = {Nat. Comput. Sci},
  title        = {Real-time raw signal genomic analysis using fully integrated memristor hardware},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes. <em>NATCS</em>, <em>5</em>(10), 927-939. (<a href='https://doi.org/10.1038/s43588-025-00878-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Noncoding mutations play a critical role in regulating gene expression, yet predicting their effects across diverse tissues and cell types remains a challenge. Here we present EMO, a transformer-based model that integrates DNA sequence with chromatin accessibility data (assay for transposase-accessible chromatin with sequencing) to predict the regulatory impact of noncoding single nucleotide polymorphisms on gene expression. A key component of EMO is its ability to incorporate personalized functional genomic profiles, enabling individual-level and disease-contextual predictions and addressing critical limitations of current approaches. EMO generalizes across tissues and cell types by modeling both short- and long-range regulatory interactions and capturing dynamic gene expression changes associated with disease progression. In benchmark evaluations, the pretraining-based EMO framework outperformed existing models, with fine-tuning small-sample tissues enhancing the model’s ability to fit target tissues. In single-cell contexts, EMO accurately identified cell-type-specific regulatory patterns and successfully captured the effects of disease-associated single nucleotide polymorphisms in conditions, linking genetic variation to disease-relevant pathways. EMO integrates DNA sequence and chromatin accessibility data to predict how noncoding variants regulate gene expression across tissues and single cells, enabling context-aware personalized insights into genetic effects for precision medicine.},
  archive      = {J_NATCS},
  author       = {Liu, Zhe and Bao, Yihang and Gu, An and Song, Weichen and Lin, Guan Ning},
  doi          = {10.1038/s43588-025-00878-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {927-939},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rhythm-based hierarchical predictive computations support acoustic−semantic transformation in speech processing. <em>NATCS</em>, <em>5</em>(10), 915-926. (<a href='https://doi.org/10.1038/s43588-025-00876-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unraveling how humans understand speech despite distortions has long intrigued researchers. A prominent hypothesis highlights the role of multiple endogenous brain rhythms in forming the computational context to predict speech structure and content. Yet how neural processes may implement rhythm-based context formation remains unclear. Here we propose the brain rhythm-based inference model (BRyBI) as a possible neural implementation of speech processing in the auditory cortex based on the interaction of endogenous brain rhythms in a predictive coding framework. BRyBI encodes key rhythmic processes for parsing spectro-temporal representations of the speech signal into phoneme sequences and to govern the formation of the phrasal context. BRyBI matches patterns of human performance in speech recognition tasks and explains contradictory experimental observations of rhythms during speech listening and their dependence on the informational aspect of speech (uncertainty and surprise). This work highlights the computational role of multiscale brain rhythms in predictive speech processing. This study presents a brain rhythm-based inference model (BRyBI) for speech processing in the auditory cortex. BRyBI shows how rhythmic neural activity enables robust speech processing by dynamically predicting context and elucidates mechanistic principles that allow robust speech parsing in the brain.},
  archive      = {J_NATCS},
  author       = {Dogonasheva, Olesia and Doelling, Keith B. and Zakharov, Denis and Giraud, Anne-Lise and Gutkin, Boris},
  doi          = {10.1038/s43588-025-00876-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {915-926},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rhythm-based hierarchical predictive computations support acoustic−semantic transformation in speech processing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pharmacophore-oriented 3D molecular generation toward efficient feature-customized drug discovery. <em>NATCS</em>, <em>5</em>(10), 898-914. (<a href='https://doi.org/10.1038/s43588-025-00850-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular generation is a cutting-edge technology with the potential to revolutionize intelligent drug discovery. However, currently reported ligand-based or structure-based molecular generation methods remain unpractical for real-world drug discovery. Here we propose an explicit pharmacophore-oriented 3D molecular generation method, termed PhoreGen. PhoreGen employs asynchronous perturbations and updates on both atomic and bond information, coupled with a message-passing mechanism that incorporates prior knowledge of ligand–pharmacophore mapping during the diffusion–denoising process. Evaluations revealed that PhoreGen efficiently generates 3D molecules well aligned with pharmacophores, maintaining good chemical reasonability, diversity, drug-likeness and binding affinity and, importantly, produces feature-customized molecules at high frequency. By using PhoreGen, we successfully identified new bicyclic boronate inhibitors of evolved metallo-β-lactamase and serine-β-lactamases, which potentiate meropenem against clinically isolated superbugs. Moreover, we identified inhibitors of metallo-nicotinamidases, emerging targets for insecticides. This work explores an explicitly constrained mode for molecular generation and demonstrates its potential in feature-customized drug discovery. A pharmacophore-oriented 3D molecular generation method, which efficiently generates diverse, drug-like molecules is proposed. It successfully identifies new inhibitors in real-world drug discovery.},
  archive      = {J_NATCS},
  author       = {Peng, Jian and Yu, Jun-Lin and Yang, Zeng-Bao and Chen, Yi-Ting and Wei, Si-Qi and Meng, Fan-Bo and Wang, Yao-Geng and Huang, Xiao-Tian and Li, Guo-Bo},
  doi          = {10.1038/s43588-025-00850-5},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {898-914},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pharmacophore-oriented 3D molecular generation toward efficient feature-customized drug discovery},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MetaKSSD: Boosting the scalability of the reference taxonomic marker database and the performance of metagenomic profiling using sketch operations. <em>NATCS</em>, <em>5</em>(10), 884-897. (<a href='https://doi.org/10.1038/s43588-025-00855-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The performance of metagenomic profiling is constrained by the diversity of taxa present in the reference taxonomic marker database (MarkerDB) used. However, continually updating MarkerDB to include newly determined taxa using existing approaches faces increasing difficulties and will soon become impractical. Here we introduce MetaKSSD, which redefines MarkerDB construction and metagenomic profiling using sketch operations, enhancing MarkerDB scalability and profiling performance. MetaKSSD encompasses 85,202 species in its MarkerDB using just 0.17 GB of storage and profiles 10 GB of data within seconds. Leveraging its comprehensive MarkerDB, MetaKSSD substantially improves profiling results. In a microbiome–phenotype association study, MetaKSSD identified more effective associations than MetaPhlAn4. We profiled 382,016 metagenomic runs using MetaKSSD, conducted extensive sample clustering analyses and suggested potential yet-to-be-discovered niches. MetaKSSD offers functionality for instantaneous searching of similar profiles. It enables the swift transmission of metagenome sketches over the network and real-time online metagenomic analysis, facilitating use by non-expert users. MetaKSSD accelerates metagenomic profiling by combining k-mer sketching with marker-based methods, enabling real-time online metagenomic profiling and scalable analysis of massive microbial datasets without sacrificing accuracy.},
  archive      = {J_NATCS},
  author       = {Yi, Huiguang and Lu, Xiaoxin and Chang, Qing},
  doi          = {10.1038/s43588-025-00855-0},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {884-897},
  shortjournal = {Nat. Comput. Sci},
  title        = {MetaKSSD: Boosting the scalability of the reference taxonomic marker database and the performance of metagenomic profiling using sketch operations},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering morphogenesis of cell clusters with differentiable programming. <em>NATCS</em>, <em>5</em>(10), 875-883. (<a href='https://doi.org/10.1038/s43588-025-00851-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the fundamental rules of organismal development is a central, unsolved problem in biology. These rules dictate how individual cellular actions coordinate over macroscopic numbers of cells to grow complex structures with exquisite functionality. We use recent advances in automatic differentiation to discover local interaction rules and genetic networks that yield emergent, systems-level characteristics in a model of development. We consider a growing tissue with cellular interactions mediated by morphogen diffusion, cell adhesion and mechanical stress. Each cell has an internal genetic network that is used to make decisions based on the cell’s local environment. Here we show that one can learn the parameters governing cell interactions in the form of interpretable genetic networks for complex developmental scenarios. When combined with recent experimental advances measuring spatio-temporal dynamics and gene expression of cells in a growing tissue, the methodology outlined here offers a promising path to unraveling the cellular bases of development. This work uses differentiable simulations and reinforcement learning to design interpretable genetic networks, enabling simulated cells to self-organize into emergent developmental patterns by responding to local chemical and mechanical cues.},
  archive      = {J_NATCS},
  author       = {Deshpande, Ramya and Mottes, Francesco and Vlad, Ariana-Dalia and Brenner, Michael P. and Dal Co, Alma},
  doi          = {10.1038/s43588-025-00851-4},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {875-883},
  shortjournal = {Nat. Comput. Sci},
  title        = {Engineering morphogenesis of cell clusters with differentiable programming},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards privacy-aware mental health AI models. <em>NATCS</em>, <em>5</em>(10), 863-874. (<a href='https://doi.org/10.1038/s43588-025-00875-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mental health disorders create profound personal and societal burdens, yet conventional diagnostics are resource-intensive and limit accessibility. Recent advances in artificial intelligence, particularly natural language processing and multimodal methods, offer promise for detecting and addressing mental disorders. However, these innovations also introduce privacy concerns. Here we examine these challenges and propose solutions, including anonymization, synthetic data and privacy-preserving training, while outlining frameworks for privacy–utility trade-offs, aiming to advance reliable, privacy-aware artificial-intelligence tools that support clinical decision-making and improve mental health outcomes. In this Perspective, the authors examine privacy risks in mental health AI, and explore solutions and evaluation frameworks to balance privacy–utility trade-offs. They suggest a pipeline for developing privacy-aware mental health AI systems.},
  archive      = {J_NATCS},
  author       = {Mandal, Aishik and Chakraborty, Tanmoy and Gurevych, Iryna},
  doi          = {10.1038/s43588-025-00875-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {863-874},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards privacy-aware mental health AI models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational and ethical considerations for using large language models in psychotherapy. <em>NATCS</em>, <em>5</em>(10), 854-862. (<a href='https://doi.org/10.1038/s43588-025-00874-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) hold great potential for augmenting psychotherapy by enhancing accessibility, personalization and engagement. However, a systematic understanding of the roles that LLMs can play in psychotherapy remains underexplored. In this Perspective, we propose a taxonomy of LLM roles in psychotherapy that delineates six specific roles of LLMs across two key dimensions: artificial intelligence autonomy and emotional engagement. We discuss key computational and ethical challenges, such as emotion recognition, memory retention, privacy and emotional dependency, and offer recommendations to address these challenges. Large language models (LLMs) offer promising ways to enhance psychotherapy through greater accessibility, personalization and engagement. This Perspective introduces a typology that categorizes the roles of LLMs in psychotherapy along two critical dimensions: autonomy and emotional engagement.},
  archive      = {J_NATCS},
  author       = {Zhang, Renwen and Meng, Han and Neubronner, Marion and Lee, Yi-Chieh},
  doi          = {10.1038/s43588-025-00874-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {854-862},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational and ethical considerations for using large language models in psychotherapy},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vision language models excel at perception but struggles with scientific reasoning. <em>NATCS</em>, <em>5</em>(10), 852-853. (<a href='https://doi.org/10.1038/s43588-025-00871-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A benchmark — MaCBench — is developed for evaluating the scientific knowledge of vision language models (VLMs). Evaluation of leading VLMs reveals that they excel at basic scientific tasks such as equipment identification, but struggle with spatial reasoning and multistep analysis — a limitation for autonomous scientific discovery.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00871-0},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {852-853},
  shortjournal = {Nat. Comput. Sci},
  title        = {Vision language models excel at perception but struggles with scientific reasoning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pioneering real-time genomic analysis by in-memory computing. <em>NATCS</em>, <em>5</em>(10), 850-851. (<a href='https://doi.org/10.1038/s43588-025-00883-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid identification of pathogenic viruses remains a critical challenge. A recent study advances this frontier by demonstrating a fully integrated memristor-based hardware system that accelerates genomic analysis by a factor of 51, while reducing energy consumption to just 0.2% of that required by conventional computational methods.},
  archive      = {J_NATCS},
  author       = {Zhu, Kaichen and Lanza, Mario},
  doi          = {10.1038/s43588-025-00883-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {850-851},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pioneering real-time genomic analysis by in-memory computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How neural rhythms can guide word recognition. <em>NATCS</em>, <em>5</em>(10), 848-849. (<a href='https://doi.org/10.1038/s43588-025-00888-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recent computational model ‘BRyBI’ proposes that gamma, theta, and delta neural oscillations can guide the process of word recognition by providing temporal windows for the integration of bottom-up input with top-down information.},
  archive      = {J_NATCS},
  author       = {Slaats, Sophie},
  doi          = {10.1038/s43588-025-00888-5},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {848-849},
  shortjournal = {Nat. Comput. Sci},
  title        = {How neural rhythms can guide word recognition},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transforming psychiatry with computational and brain-based methods. <em>NATCS</em>, <em>5</em>(10), 844-847. (<a href='https://doi.org/10.1038/s43588-025-00884-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrating computational methods with brain-based data presents a path to precision psychiatry by capturing individual neurobiological variation, improving diagnosis, prognosis, and personalized care. This Viewpoint highlights advances in normative and foundation models, the importance of clinically grounded principles, and the role of robust measurement and interpretability in progressing mental health care.},
  archive      = {J_NATCS},
  author       = {Akiki, Teddy J. and Williams, Leanne M. and Wolfers, Thomas and Yang, Yanwu and Stahl, Daniel and Gillan, Claire M.},
  doi          = {10.1038/s43588-025-00884-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {844-847},
  shortjournal = {Nat. Comput. Sci},
  title        = {Transforming psychiatry with computational and brain-based methods},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trials for computational psychiatry. <em>NATCS</em>, <em>5</em>(10), 841-843. (<a href='https://doi.org/10.1038/s43588-025-00879-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational psychiatry is increasingly delivering causal evidence by focusing on interventions research and clinical trials. Causal evidence could improve patient outcomes through improved precision, repurposing, novel interventions, scaling of psychotherapy and better translation to the clinic.},
  archive      = {J_NATCS},
  author       = {Huys, Quentin J. M. and Browning, Michael},
  doi          = {10.1038/s43588-025-00879-6},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {841-843},
  shortjournal = {Nat. Comput. Sci},
  title        = {Trials for computational psychiatry},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Developing mental health AI tools that improve care across different groups and contexts. <em>NATCS</em>, <em>5</em>(10), 839-840. (<a href='https://doi.org/10.1038/s43588-025-00882-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In order to realize the potential of mental health AI applications to deliver improved care, a multipronged approach is needed, including representative AI datasets, research practices that reflect and anticipate potential sources of bias, stakeholder engagement, and equitable design practices.},
  archive      = {J_NATCS},
  author       = {Martinez-Martin, Nicole},
  doi          = {10.1038/s43588-025-00882-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {839-840},
  shortjournal = {Nat. Comput. Sci},
  title        = {Developing mental health AI tools that improve care across different groups and contexts},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rethinking mental illness through a computational lens. <em>NATCS</em>, <em>5</em>(10), 837-838. (<a href='https://doi.org/10.1038/s43588-025-00894-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Computational Science presents a Focus that explores the field of computational psychiatry and its key challenges, from privacy concerns to the ethical use of artificial intelligence, offering new insights into the future of mental health care.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00894-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {837-838},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rethinking mental illness through a computational lens},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
