<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATEE</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natee">NATEE - 33</h2>
<ul>
<li><details>
<summary>
(2025). Citrus reticulata (mandarin). <em>NATEE</em>, <em>9</em>(9), 1752. (<a href='https://doi.org/10.1038/s41559-025-02822-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Qiang Xu has been captivated by citrus fruits since childhood.},
  archive      = {J_NATEE},
  author       = {Xu, Qiang},
  doi          = {10.1038/s41559-025-02822-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1752},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Citrus reticulata (mandarin)},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease. <em>NATEE</em>, <em>9</em>(9), 1739-1751. (<a href='https://doi.org/10.1038/s41559-025-02797-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More than 10 years following the onset of the sea star wasting disease (SSWD) epidemic, affecting over 20 asteroid species from Mexico to Alaska, the causative agent has been elusive. SSWD killed billions of the most susceptible species, sunflower sea stars (Pycnopodia helianthoides), initiating a trophic cascade involving unchecked urchin population growth and the widespread loss of kelp forests. Identifying the causative agent underpins the development of recovery strategies. Here we induced disease and subsequent mortality in exposure experiments using tissue extracts, coelomic fluid and effluent water from wasting sunflower sea stars, with no mortality in controls. Deep sequencing of diseased sea star coelomic fluid samples from experiments and field outbreaks revealed a dominant proportion of reads assigned to the bacterium Vibrio pectenicida. Fulfilling Koch’s postulates, V. pectenicida strain FHCF-3, cultured from the coelomic fluid of a diseased sunflower sea star, caused disease and mortality in exposed sunflower sea stars, demonstrating that it is a causative agent of SSWD. This discovery will enable recovery efforts for sea stars and the ecosystems affected by their decline by facilitating culture-based experimental research and broad-scale screening for pathogen presence and abundance in the laboratory and field. The causative agent of sea star wasting disease has been elusive. This study used genetic datasets and experimental exposures to demonstrate that a strain of the bacterium Vibrio pectenicida caused disease and mortality in sea stars.},
  archive      = {J_NATEE},
  author       = {Prentice, Melanie B. and Crandall, Grace A. and Chan, Amy M. and Davis, Katherine M. and Hershberger, Paul K. and Finke, Jan F. and Hodin, Jason and McCracken, Andrew and Kellogg, Colleen T. E. and Clemente-Carvalho, Rute B. G. and Prentice, Carolyn and Zhong, Kevin X. and Harvell, C. Drew and Suttle, Curtis A. and Gehman, Alyssa-Lois M.},
  doi          = {10.1038/s41559-025-02797-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1739-1751},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories. <em>NATEE</em>, <em>9</em>(9), 1731-1738. (<a href='https://doi.org/10.1038/s41559-025-02798-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Radiogenic strontium isotopes (87Sr/86Sr) and the alkaline earth ratios (AERs) Sr/Ca and Ba/Ca in fossil dental enamel can inform the habitat, residence and life histories of early hominins recovered from the Pleistocene cradle-of-humankind sites of Gauteng, South Africa. Key questions, which may be addressed with these indices, are the relative exploitation of wet versus dry botanic regimes and whether early hominins dispersed in a manner similar to that of chimpanzees (characterized by male philopatry and female dispersal at puberty) or to that of humans (who are not so characterized). Here we developed 28 new dental chronologies in 20 Paranthropus robustus teeth from Swartkrans and Kromdraai. Resulting geochemical time series demonstrate that, while maternal 87Sr/86Sr in earlier-forming teeth varies widely, third molar 87Sr/86Sr, derived from postweaning solid foods, progressively converges to 0.7306 ± 0.0035 (± 2 s.d.), which we express as the local isotopically delineated exploitation area (LIDEA). The spatial resolution of LIDEA is determined using a bioavailable 87Sr/86Sr isoscape. In this environmental context, we interpret LIDEA as a quantifiable signal indicating eurytopy (generalization), with some 30% of Sr deriving from riparian woodland habitats. With regard to residence, many individuals arrived at the site after second molar mineralization, while some matured locally, demonstrating both male and female dispersal as well as lifelong local residence. Analysis of both 87Sr/86Sr and the AERs further highlights concomitant patterns, as well as numerous periodicities that may be related to resource depletion, seasonality or lunar cycles. Strontium analysis of Paranthropus robustus teeth from two South African sites reveals that the Pleistocene hominin probably exploited both savanna and riparian woodland habitats, had a home-range size compatible with that of present-day hunter-gatherers and had periodic ranging patterns. While some individuals lived their lives in the same place, others dispersed during their lifetime.},
  archive      = {J_NATEE},
  author       = {Sillen, Andrew and Dean, Christopher and Balter, Vincent},
  doi          = {10.1038/s41559-025-02798-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1731-1738},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas. <em>NATEE</em>, <em>9</em>(9), 1719-1730. (<a href='https://doi.org/10.1038/s41559-025-02794-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most faunas from the Mesozoic era were dominated by sauropod dinosaurs, the largest terrestrial animals to ever exist. These megaherbivores were remarkably diverse and widely distributed. Here we study three Late Jurassic faunas from the USA, Portugal and Tanzania, each approximately 150 million years old, which are known for their extreme sauropod diversity. Whereas general taxonomic composition was similar in these three faunas, the major clades differed in relative abundance. Moreover, their depositional strata record distinct climatic regimes. Using dental microwear texture analysis, we investigated the impact of these climate regimes and the resulting food availability on the different sauropod taxa. Wear patterns in camarasaurid macronarians show minimal variation across different climate regimes, supporting previous studies suggesting that these animals migrated to follow their preferred climate niche and food source. North American camarasaurids show similar wear patterns to those of Portuguese turiasaurs, another broad-crowned taxon, which did not exist in the Jurassic of North America. By contrast, where camarasaurids and turiasaurs co-occurred in Portugal, their microwear patterns are distinct, suggesting niche differentiation to avoid ecological competition between these two clades. Flagellicaudatan diplodocoids display highly variable wear patterns, indicating limited migration (and therefore seasonal variation in diet), which aligns with observed biogeography patterns in the USA. Early-branching titanosauriforms show highly distinct wear patterns between different climate regimes, which can probably be attributed to different abrasive loads in the respective habitats. Our results demonstrate that dental microwear texture analysis not only records dietary preferences but also reveals behaviour such as competition and migration related to dietary niches in past ecosystems. The authors compare tooth wear patterns from three Late Jurassic sauropod faunas in North America, Portugal and Tanzania and find that different regions and climate regimes seem to have had distinctive impacts on dental wear of megaherbivores in these areas, revealing habitat and ecological niche information.},
  archive      = {J_NATEE},
  author       = {Winkler, Daniela E. and Tschopp, Emanuel and Saleiro, André and Wiesinger, Ria and Kaiser, Thomas M.},
  doi          = {10.1038/s41559-025-02794-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1719-1730},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enlargement of sternum traits facilitated the evolution of powered flight in birds. <em>NATEE</em>, <em>9</em>(9), 1705-1718. (<a href='https://doi.org/10.1038/s41559-025-02795-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An enlarged sternum with a prominent keel is a central feature of the flight apparatus of modern birds. However, sterna of near-bird dinosaurs (Pennaraptora) and early avialans are either substantially different from those of living birds or absent altogether, raising questions about how specialized sternal structures evolved in birds and how they are related to function. This remains poorly understood because of the fragmentary nature of the fossil record, and the challenges in inferring form and function from crushed fossils. We use ancestral character estimations to trace sternal trait acquisition through the bird stem group, and multivariate phylogenetic regressions to analyse relationships between sternum morphology, body mass and flight capabilities. We find that sternum evolution was episodic: basal members of Pennaraptora had proportionally small sterna, which became larger and more craniocaudally elongated in Avialae. This enlargement precedes the appearance of a midline ridge, a possible precursor of the sternal keel, in Pygostylia. Sternum size increased again in crownward Ornithuromorpha, alongside a fully formed sternal keel and enlarged caudal projections, both critical areas of flight muscle attachment. Sternal experimentation in relation to flight characteristics occurs several times throughout Pennaraptora, including within Paraves and Enantiornithes, indicating that powered flight may have evolved several times before proliferating in crown-group birds. The extant avian sternum is highly adapted to powered flight, but understanding of its early evolution is hindered by patchy fossil representation and the fact that where preserved, sterna in near-bird dinosaurs and early avialans are substantially different. To overcome this, the authors use ancestral character estimations and find that sternum evolution was episodic and experimental in relation to flight characteristics.},
  archive      = {J_NATEE},
  author       = {Lowi-Merri, Talia M. and Benson, Roger and Hu, Han and O’Connor, Jingmai and Claramunt, Santiago and Evans, David C.},
  doi          = {10.1038/s41559-025-02795-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1705-1718},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Enlargement of sternum traits facilitated the evolution of powered flight in birds},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The role of plasmid copy number and mutation rate in evolutionary outcomes. <em>NATEE</em>, <em>9</em>(9), 1694-1704. (<a href='https://doi.org/10.1038/s41559-025-02792-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multicopy plasmids are widespread in nature and compose a common strategy for spreading beneficial traits across microbes. However, the role of plasmids in supporting the evolution of encoded genes remains underexplored due to challenges in experimentally manipulating key parameters such as plasmid copy number and mutation rate. Here we developed a strategy for controlling copy number in the plasmid-based Saccharomyces cerevisiae continuous evolution system, OrthoRep, and used our resulting capabilities to investigate the evolution of a conditionally essential gene under varying copy number and mutation rate conditions. Our results show that low copy number facilitated the faster enrichment of beneficial alleles whereas high copy number promoted robustness through the maintenance of allelic diversity. High copy number also slowed the removal of deleterious mutations and increased the fraction of non-functional alleles that could hitchhike during evolution. This study highlights the nuanced relationships between plasmid copy number, mutation rate and evolutionary outcomes, providing insights into the adaptive dynamics of genes encoded on multicopy plasmids and nominating OrthoRep as a versatile tool for studying plasmid evolution. In an experimental multicopy plasmid evolution system that allows manipulation of mutation rate and copy number, low copy number promoted enrichment of beneficial alleles whereas high copy number maintained allelic diversity.},
  archive      = {J_NATEE},
  author       = {Pisera, Alexander ‘Olek’ and Liu, Chang C.},
  doi          = {10.1038/s41559-025-02792-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1694-1704},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The role of plasmid copy number and mutation rate in evolutionary outcomes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas. <em>NATEE</em>, <em>9</em>(9), 1685-1693. (<a href='https://doi.org/10.1038/s41559-025-02771-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis is a recently identified cause of Hansen’s disease, and is associated with the more severe and potentially lethal presentations of diffuse lepromatous leprosy and Lucio’s phenomenon. Detection of this infection has been limited to a small number of individuals, leaving much to be learned about its global distribution and transmissibility. Its discovery in wild rodent populations in the United Kingdom and Ireland also raises questions about its zoonotic potential. Here, we raise further awareness of this disease via analyses of two exceptionally well preserved M. lepromatosis genomes obtained from 4,000-year-old human remains of two adult males from the archaeological sites of El Cerrito and La Herradura in Northern Chile. This formed the basis of genomic comparisons between ancient and modern forms of the pathogen. We demonstrate an unexpected long history of M. lepromatosis in the Americas, which contrasts with the more recent Eurasian history of the closely related Mycobacterium leprae. We offer relevant perspectives on its evolution while providing an incentive for further disease monitoring in both humans and other potential reservoir species in the Americas and elsewhere. Recovery of ancient Mycobacterium lepromatosis genomes in 4,000-year-old human remains from Chile demonstrates that this causal agent of Hansen’s disease has a long history in the Americas, predating the colonial period by millennia.},
  archive      = {J_NATEE},
  author       = {Ramirez, Darío A. and Sitter, T. Lesley and Översti, Sanni and Herrera-Soto, María José and Pastor, Nicolás and Fontana-Silva, Oscar Eduardo and Kirkpatrick, Casey L. and Castelleti-Dellepiane, José and Nores, Rodrigo and Bos, Kirsten I.},
  doi          = {10.1038/s41559-025-02771-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1685-1693},
  shortjournal = {Nat. Ecol. Evol},
  title        = {4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pervasive but biome-dependent relationship between fragmentation and resilience in forests. <em>NATEE</em>, <em>9</em>(9), 1670-1684. (<a href='https://doi.org/10.1038/s41559-025-02776-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The relationship between landscape fragmentation and vegetation resilience is uncertain. Here we use multiple satellite-based tree cover data and vegetation indices to quantify the apparent effects of fragmentation on global forest resilience and potential mechanisms thereof. We measure fragmentation as edge density, patch density and mean patch area of tree cover patches, and measure resilience as one-lag temporal autocorrelation of vegetation indices. We find a statistically significant (P &lt; 0.05) fragmentation–resilience relationship in about 77% of fragmented forests, but the direction varies across biomes. In tropical and temperate forests, fragmentation is linked to increased local temperature and atmospheric dryness, resulting in a negative fragmentation–resilience relationship. Conversely, in boreal forests, fragmentation is associated with decreased atmospheric dryness and enhanced light resource, thereby increasing forest resilience. Our results reconcile competing hypotheses and highlight the importance of accounting for fragmentation when predicting shifts in ecosystem resilience under disturbances. These findings also suggest the necessity of biome-targeted forest management strategies for climate change mitigation and adaptation. Human-driven forest fragmentation could have major impacts on ecosystem functioning. This global analysis shows that the relationship between fragmentation and vegetation resilience in forests may differ depending on bioclimatic region and local environmental conditions.},
  archive      = {J_NATEE},
  author       = {Su, Yongxian and Zhang, Chaoqun and Cescatti, Alessandro and Yu, Kailiang and Ciais, Philippe and Smith, Taylor and Shang, Jiali and Carnicer, Jofre and Liu, Jane and Chen, Jing Ming and Green, Julia K. and Wu, Jianping and Ponce-Campos, Guillermo E. and Zhang, Yongguang and Zuo, Zhiyan and Liao, Jinbao and Lafortezza, Raffaele and Yan, Kai and Yang, Xueqin and Liu, Liyang and Ren, Jiashun and Yuan, Wenping and Chen, Xiuzhi and Wu, Chaoyang and Zhou, Weiqi},
  doi          = {10.1038/s41559-025-02776-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1670-1684},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pervasive but biome-dependent relationship between fragmentation and resilience in forests},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family. <em>NATEE</em>, <em>9</em>(9), 1656-1669. (<a href='https://doi.org/10.1038/s41559-025-02777-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations are more likely to produce some phenotypes than others, but the causal role of these production propensities in the evolution of phenotypic diversity remains unclear. There are two major challenges: it is difficult to separate the effect of the genotype–phenotype (GP) map from that of natural selection when analysing natural diversity, and most extant phenotypes evolved long ago in species whose GP maps cannot be recovered. Here, using two reconstructed ancestral transcription factors that are closely related but differ in function, we created libraries containing all possible amino acid combinations at historically variable sites in the proteins’ DNA binding interface (the genotypes) and measured their capacity to specifically bind DNA elements containing all possible combinations of nucleotides at historically variable sites (the phenotypes). The ancestral GP maps were strongly anisotropic (the distribution of phenotypes encoded by genotypes is highly non-uniform) and heterogeneous (the phenotypes accessible around each genotype vary greatly among genotypes), but the extent and direction of these properties differed substantially between the maps. In both cases, these properties steered evolution towards the lineage-specific phenotypes that evolved during history. Our findings establish that ancient properties of the GP relationship were causal factors in the evolutionary process that produced present-day patterns of functional conservation and diversity. A combination of ancestral reconstruction and deep mutational scanning to examine the genotype–phenotype maps of steroid receptors shows that the properties of ancestral genotype–phenotype maps determine the lineage-specific evolution of DNA specificity.},
  archive      = {J_NATEE},
  author       = {Herrera-Álvarez, Santiago and Patton, Jaeda E. J. and Thornton, Joseph W.},
  doi          = {10.1038/s41559-025-02777-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1656-1669},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments. <em>NATEE</em>, <em>9</em>(9), 1643-1655. (<a href='https://doi.org/10.1038/s41559-025-02779-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human impacts on nature span vast spatial scales that transcend abiotic gradients and biogeographic barriers, yet estimates of biodiversity loss from land-use change overwhelmingly derive from local-scale studies. Using a field dataset of 971 bird species sampled in forest and cattle pasture across 13 biogeographic regions of Colombia, we quantify biodiversity losses from local to near-national scales. Losses are on average 60% worse at the pan-Colombian scale than in individual regions, with underestimation remaining until six to seven biogeographic regions are sampled. Regional losses greatly exceed local losses when beta-diversity is high due to reduced species turnover in pasture across geographic space and elevation. Extrapolation from local-scale studies causes major underestimation of biodiversity loss, emphasizing the need to incorporate spatial structure into measures of change. Analysis of data on 971 bird species in natural habitat and cattle pasture in Colombia finds that near-national-scale losses of bird diversity greatly exceed losses recorded at the local scale, suggesting that extrapolations from local studies will severely underestimate biodiversity losses.},
  archive      = {J_NATEE},
  author       = {Socolar, Jacob B. and Mills, Simon C. and Gilroy, James J. and Martínez-Revelo, Diego E. and Medina-Uribe, Claudia A. and Parra-Sanchez, Edicson and Ramirez-Gutierrez, Marcela and Sand Sæbø, Jørgen and Meneses, Henry S. and Pérez, Giovanny and Barlow, Jos and Ochoa Quintero, Jose M. and Freckleton, Robert P. and Haugaasen, Torbjørn and Edwards, David P.},
  doi          = {10.1038/s41559-025-02779-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1643-1655},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development. <em>NATEE</em>, <em>9</em>(9), 1626-1642. (<a href='https://doi.org/10.1038/s41559-025-02780-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bats are the only mammals capable of self-powered flight, an evolutionary innovation based on the transformation of forelimbs into wings. The bat wing is characterized by an extreme elongation of the second to fifth digits with a wing membrane called the chiropatagium connecting them. Here we investigated the developmental and cellular origin of this structure by comparing bat and mouse limbs using omics tools and single-cell analyses. Despite the substantial morphological differences between the species, we observed an overall conservation of cell populations and gene expression patterns including interdigital apoptosis. Single-cell analyses of micro-dissected embryonic chiropatagium identified a specific fibroblast population, independent of apoptosis-associated interdigital cells, as the origin of this tissue. These distal cells express a conserved gene programme including the transcription factors MEIS2 and TBX3, which are commonly known to specify and pattern the early proximal limb. Transgenic ectopic expression of MEIS2 and TBX3 in mouse distal limb cells resulted in the activation of genes expressed during wing development and phenotypic changes related to wing morphology, such as the fusion of digits. Our results elucidate fundamental molecular mechanisms of bat wing development and illustrate how drastic morphological changes can be achieved through repurposing of existing developmental programmes during evolution. Single-cell comparison of developing bat and mouse limbs reveals conservation of cell populations and gene expression patterns, and suggests repurposing of genes involved in proximal limb development for wing evolution.},
  archive      = {J_NATEE},
  author       = {Schindler, Magdalena and Feregrino, Christian and Aldrovandi, Silvia and Lo, Bai-Wei and Monaco, Anna A. and Ringel, Alessa R. and Morales, Ariadna E. and Zehnder, Tobias and Behncke, Rose Yinghan and Glaser, Juliane and Barclay, Alexander and Andrey, Guillaume and Kragesteen, Bjørt K. and Hägerling, René and Haas, Stefan A. and Vingron, Martin and Ulitsky, Igor and Marti-Renom, Marc A. and Hechavarria, Julio and Fasel, Nicolas and Hiller, Michael and Lupiáñez, Darío G. and Mundlos, Stefan and Real, Francisca M.},
  doi          = {10.1038/s41559-025-02780-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1626-1642},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep learning reveals the complex genetic architecture of male guppy colouration. <em>NATEE</em>, <em>9</em>(9), 1614-1625. (<a href='https://doi.org/10.1038/s41559-025-02781-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The extraordinary variation in male guppy (Poecilia reticulata) colouration is a powerful model for studying the interplay of natural and sexual selection. However, the complexity of this variation has hampered the high-resolution characterization and determination of the genetic architecture underlying male guppy colour and clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male colour variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a genome-wide association study of colour traits. Our phenotypic and genomic results converge to show that colour patterning in guppies is a combination of many heritable features, each with a largely independent genetic architecture spanning the entire genome. Autosomally inherited ornaments are polygenic, with significant contributions from loci involved in neural crest cell migration. Unusually, the results of our genome-wide association study suggest that gene duplicates from the autosomes to the Y chromosome are responsible for much of the sex-linked variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait. Guppy colouration is a very diverse trait under natural and sexual selection. Using a combination of high-resolution phenotyping and genomic analysis, the authors reveal the genetic architecture of this complex trait.},
  archive      = {J_NATEE},
  author       = {van der Bijl, Wouter and Shu, Jacelyn J. and Goberdhan, Versara S. and Sherin, Linley M. and Jia, Changfu and Cortazar-Chinarro, Maria and Corral-Lopez, Alberto and Mank, Judith E.},
  doi          = {10.1038/s41559-025-02781-w},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1614-1625},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deep learning reveals the complex genetic architecture of male guppy colouration},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood. <em>NATEE</em>, <em>9</em>(9), 1599-1613. (<a href='https://doi.org/10.1038/s41559-025-02785-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deadwood is a major carbon source in forests, and yet the fate of this carbon remains a gap in our understanding of global carbon cycling. Lignin, the most recalcitrant biopolymer in wood, is mainly decayed through extracellular enzymatic and chemical processes initiated by white-rot fungi. However, the intracellular conversion of lignin decay products has been overlooked in the fungal kingdom. Here we integrate comparative genomic and phylogenetic analyses to understand the distribution and evolution of enzymes responsible for modifying lignin-related aromatic compounds—such as decarboxylases, hydroxylases, dioxygenases and other downstream ring-cleavage enzymes—that funnel carbon to central metabolism across the bacterial and the fungal kingdoms. We demonstrate that specific fungal lineages conserve these enzyme families, and that the abilities to enzymatically depolymerize lignin and catabolize lignin-related aromatic compounds are not necessarily coupled. Our analyses also reveal an expanded substrate specificity of aromatic ring-cleavage enzymes during fungal evolution, as well as a clade of extracellular enzymes among them, broadening the spatial range of these biochemical capabilities. Together, our results highlight a large diversity of fungal enzymes and hosts that warrant further investigation for inclusion into carbon cycling models and biotechnological applications for the conversion of aromatic compounds. This study uses comparative genomics and phylogenetics to analyse the distribution and evolution of key enzymes involved in the catabolism of lignin-related aromatic compounds in the bacterial and fungal kingdoms.},
  archive      = {J_NATEE},
  author       = {Kijpornyongpan, Teeratas and Kuatsjah, Eugene and Komp, Evan and Evans, James E. and Ruiz-Dueñas, Francisco Javier and Salvachúa, Davinia},
  doi          = {10.1038/s41559-025-02785-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1599-1613},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife. <em>NATEE</em>, <em>9</em>(9), 1585-1598. (<a href='https://doi.org/10.1038/s41559-025-02786-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across the world, human (anthropophonic) sounds add to sounds of biological (biophonic) and geophysical (geophonic) origin, with human contributions including both speech and technophony (sounds of technological devices). To characterize society’s contribution to the global soundscapes, we used passive acoustic recorders at 139 sites across 6 continents, sampling both urban green spaces and nearby pristine sites continuously for 3 years in a paired design. Recordings were characterized by bird species richness and by 14 complementary acoustic indices. By relating each index to seasonal, diurnal, climatic and anthropogenic factors, we show here that latitude, time of day and day of year each predict a substantial proportion of variation in key metrics of biophony—whereas anthropophony (speech and traffic) show less predictable patterns. Compared to pristine sites, the soundscape of urban green spaces is more dominated by technophony and less diverse in terms of acoustic energy across frequencies and time steps, with less instances of quiet. We conclude that the global soundscape is formed from a highly predictable rhythm in biophony, with added noise from geophony and anthropophony. At urban sites, animals experience an increasingly noisy background of sound, which poses challenges to efficient communication. Analysis of soundscape data from 139 globally distributed sites reveals that sounds of biological origin exhibit predictable rhythms depending on location and season, whereas sounds of anthropogenic origin are less predictable. Comparisons between paired urban–rural sites show that urban green spaces are noisier and dominated by sounds of technological origin.},
  archive      = {J_NATEE},
  author       = {Somervuo, Panu and Roslin, Tomas and Fisher, Brian L. and Hardwick, Bess and Kerdraon, Deirdre and Raharinjanahary, Dimby and Rajoelison, Eric Tsiriniaina and Lauha, Patrik and Griem, Lukas and Lehikoinen, Petteri and Niittynen, Pekka and Piirainen, Esko and Lumme, Markus and Riihikoski, Ville-Matti and Acevedo-Charry, Orlando and Adalsteinsson, Solny A. and Ahmad, Maaz and Alcobia, Sandra and Aldará, Jón and Andrew, Nigel R. and Anslan, Sten and Antonelli, Alexandre and Arena, Julieta Soledad and Arroyo Almeida, Santiago and Aster, Ines and Autto, Hannu and Aviles Gamboa, Anahi and Baixeras, Joaquín and Baldauf, Mario and Balestrieri, Rosario and Banelyte, Gaia Giedre and Barrett, Adrian and Beja, Pedro and Berg, Thomas Olof and Bergerot, Benjamin and Biro, Elizabeth G. and Blendinger, Pedro G. and Bollache, Loïc and Bou Dagher Kharrat, Magda and Boyer, Stephane and Bridell, Erika and Brotherson, Martyn and Brown, Leslie Robert and Buckley, Hannah L. and Buscardo, Erika and Buthelezi, Nokuphila and Cagnolo, Luciano and Calvente, Alice and Capobianco, Giovanni and Carreón-Palau, Laura and Carriere, Suzanne and Case, Bradley S. and Chang, Jenyu and Chaparro, Juan Matías and Chen, Chi-Ling and Chicoine, Christine and Christensson, Madeleine and Collado Rosique, Francisco and Colom Montero, William and da Fonseca, Ricardo do Sacramento and Da Silva, Luís P. and Dal Molin, Anamaria and Dallas, Tad and de Francesco, Maria Carla and Del Ángel-Rodríguez, Jorge Arturo and Díaz-Delgado, Ricardo and Dirnböck, Thomas and Djukic, Ika and Dladla, Philile and Domínguez Masciale, Jeremías and Dorigo, Thiago and Douwes, Errol and Ekrem, Torbjørn and Enderskog, Helena and Erefur, Charlotta and Fahad, Muhammad and Falahati-Anbaran, Mohsen and Farrell, Arielle and Ferland, Gabriel and Ferrari, Emanuele and Figueiredo, Axa and Forero, Fernando and Freiberga, Inga and Frosch-Radivo, Andrea and Ganchozo Intriago, Luis Alberto and Garzoli, Laura and Giacomotti, Paola and Gianuca, Andros T. and Gilg, Olivier and Gilg, Vladimir and Gísladóttir, Fanney and Glowacki, Ryan and Gottsberger, Brigitte and Gregoire, Jocelyn and Groner, Elli and Guedes, Patrícia and Guile, Aimee Michelle and Haase, Peter and Hadi, Fazal and Haidegger, Magdalena and Hansen, Leivur Janus and Hansen, Lars Holst and Harrop, Reid and Havnås, Harald and Herrera Báez, David and Ho, Chris C. Y. and Hohenbühel, Denise and Houska Tahadlova, Marketa and Hänninen, Jari and Höglund, Linda and Í Haraldsstovu, Kolbrún and Imbeau, Elise and Inkinen, Jasmin and Ishihara, Masae Iwamoto and Jackson, Abigail C. and Jansson, Gunnar and Jha, Rohit and Kager, Gerald and Kahale, Rhea and Kalttopää, Oula and Karai, Elizabeth Wanjiru and Karlsson, Dave and Kaus-Thiel, Andrea and Khan, Asghar and Khan, Qaisar and Kimoto, Keishi and Kipngetich, Shadrack Chumo and Klante, Clemens and Klemedtsson, Leif and Klinth, Mårten and Koskinen, Janne and Kotakorpi, Matti and Kreiling, Agnes-Katharina and Krisai-Greilhuber, Irmgard and Kristensen, Erik and König, Sebastian and Langenheder, Silke and Laurila, Kalevi and Le Gouar, Pascaline and Lecomte, Nicolas and Lecomte, Erin and Leitman, Paula Moraes and León-Cortés, Jorge L. and Li, Daijiang and Loehr, John and Lopez-Vaamonde, Carlos and Makari, Mehsen and Mangini, Gabriela Giselle and Maroschek, Michael and Mata, Vanessa A. and Matsuoka, Shunsuke and Mazzafera, Thais and McDonald, Paul G. and Meinert, Laura and Meléndez-González, Mayra and Mendoza-Henao, Angela M. and Moreau, Sebastien and Moreau, Jérôme and Mosbacher, Jesper and Moyer, Esteban and Mrazova, Anna and Mteshane, Samantha and Mungai, Nancy Wangari and Muñoz Herraiz, Gema and Murillo-Vázquez, Andrea and Musazzi, Simona and Mutanen, Marko and Müller, Jörg and Navarro Canales, Rebeca and Ndlovu, Monica and Nicolai, Annegret and Niessner, Armin and Nordén, Jenni and Nowak, Paweł and O’Connell, Erin and Orru, Arianna and Pagnon, Thomas and Pantoja-Diaz, Yurani Nayive and Pentinsaari, Mikko and Pilloni, Sebastian and Pinder, Adrian and Pinheiro, Thiago A. and Põlme, Sergei and Powell, Luke L. and Pröll, Gisela and Pulido-Santacruz, Paola and Queralt, Enrique and Quilantang, Mark Tristan and Quinlan, Kirsty and Ramirez, Ricardo and Rankinen, Juha and Rasino, Micaela Del Valle and Rebelo, Rui and Remmers, Wolfram and Retz, Franziska and Reyes, Evelin and Rivas Torres, Gonzalo and Rogers, Hanna M. K. and Rosário, Inês T. and Rosário Da Rosàrio da Costa, Sidney and Rütting, Tobias and Sahlstén, Johannes and Saliba, Carole and Salmirinne, Teppo and Sam, Katerina and Santos, Douglas and Santos-Reis, Margarida and Sawan, Michel and Schattanek-Wiesmair, Benjamin and Schiestl-Aalto, Pauliina and Schmidt, Niels Martin and Seibold, Sebastian and Seidl, Rupert and Seifert, Linda and Sithole, Malibongwe and Sivault, Elise and Smart, Jessica and Smerczyński, Ireneusz and Soda, Ayaka and Sousa-Lima, Renata S. and Stanisci, Angela and Stanley, Margaret C. and Steenkamp, Daleen and Stengel, Elisa and Stoll, Stefan and Strauss, Willem Maartin and Stur, Elisabeth and Sujala, Maija and Sundell, Janne and Svavarsdóttir, Jónína and Tedersoo, Leho and Tepsa, Saana and Tikochinsky, Maor Tiko and Tuominen, Esa-Pekka and Tweraser, Stefanie and Ulloa Espinosa, Catalina and Uusitalo, Joni and Vallinmäki, Mikko and Vannier, Fabrice and Varela, Abigail and Vatka, Emma and Veikkolainen, Silja and Vernes, Karl and Watts, Phillip C. and Weslien, Per and Wirth, Ciara and Wisniewski, Jana Helga and Young, Amanda B. and Övergaard, Robyn and Ovaskainen, Otso},
  doi          = {10.1038/s41559-025-02786-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1585-1598},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency. <em>NATEE</em>, <em>9</em>(9), 1571-1584. (<a href='https://doi.org/10.1038/s41559-025-02761-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerous leaf-level experiments suggest that plant intrinsic water-use efficiency (iWUE) increases under elevated CO2 because of reduced stomatal conductance and enhanced photosynthesis. However, it remains elusive whether this response can be extrapolated to the ecosystem scale, because confounding factors and compensating feedbacks are often involved in ecosystem iWUE variations. Here we develop a machine learning-based framework to disentangle the ecosystem-scale CO2 effects on iWUE and its two components, canopy conductance (Gc) and gross primary productivity (GPP), based on global networks of long-term eddy covariance observations. Our results show widespread CO2-induced enhancement of iWUE across diverse ecosystems, driven predominantly by Gc reduction rather than GPP stimulation. Moreover, three divergent response types are identified across the studied ecosystems, based on the strength and significance of CO2-driven Gc reduction and GPP enhancement, indicating spatially non-uniform responses to rising CO2. Nutrient supply, water availability and biome types are found to be critical factors regulating this spatial heterogeneity. Overall, our study provides observational insights into ecosystem-scale CO2 fertilization effects. Such understandings are essential to inform terrestrial biosphere models for better projections of carbon and water cycles given the intensified changing climate in a CO2-rich future. A machine learning approach using long-term observations of eddy covariance finds that the increase in plant intrinsic water-use efficiency under higher CO2 levels, across diverse ecosystems, is driven primarily by reduction in canopy conductance, rather than by stimulation of gross primary productivity.},
  archive      = {J_NATEE},
  author       = {Zhan, Weiwei and Lian, Xu and Liu, Jiangong and Han, Jisu and Huang, Yu and Yang, Hao and Zhan, Chunhui and Winkler, Alexander J. and Gentine, Pierre},
  doi          = {10.1038/s41559-025-02761-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1571-1584},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change. <em>NATEE</em>, <em>9</em>(9), 1562-1570. (<a href='https://doi.org/10.1038/s41559-025-02788-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plankton models form the core of marine ecosystem simulators, with uses from regional resource and ecosystem management to climate change projections. In this Perspective, we suggest that stronger alignment of models with empirical knowledge about plankton physiology, diversity and trophic roles will improve model utility and the reliability of their outputs regarding biodiversity, ecophysiology, trophic dynamics and biogeochemistry. We recommend key steps to resolve the disconnect between empirical research and simulation models accounting for well-established plankton processes with an aim to increase the utility of such models for applied uses. A central challenge is characterizing the complexity of plankton diversity and activity in ways that are amenable to model incorporation. We argue that experts in empirical science are best placed to advise the development of next-generation models to address these challenges, and we propose a series of actions to achieve that engagement, including involvement of these experts in the design and exploitation of plankton digital twins. Plankton models have crucial applications in ecosystem management and climate change projections. This Perspective suggests that stronger alignment of plankton models with empirical knowledge is needed and recommends steps to close the gap between empirical research and modelling.},
  archive      = {J_NATEE},
  author       = {Flynn, Kevin J. and Atkinson, Angus and Beardall, John and Berges, John A. and Boersma, Maarten and Brunet, Christophe and Calbet, Albert and Caron, Dave A. and Dam, Hans G. and Glibert, Patricia M. and Hansen, Per Juel and Jin, Peng and Lønborg, Christian and Mayor, Daniel J. and Menden-Deuer, Susanne and Mock, Thomas and Mulholland, Margaret R. and Needham, David M. and Polimene, Luca and Poulton, Alex J. and Robinson, Carol and Rokitta, Sebastian D. and Rost, Björn and Saiz, Enric and Scanlan, David J. and Schmidt, Katrin and Sherr, Evelyn and Stoecker, Diane K. and Svensen, Camilla and Thiele, Stefan and Thingstad, Tron F. and Våge, Selina},
  doi          = {10.1038/s41559-025-02788-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1562-1570},
  shortjournal = {Nat. Ecol. Evol},
  title        = {More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bird sternum evolution facilitated the origins of powered flight. <em>NATEE</em>, <em>9</em>(9), 1560-1561. (<a href='https://doi.org/10.1038/s41559-025-02817-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An in-depth analysis of the breastbone (sternum) of bird ancestors reveals its importance in the origin and evolution of flight in dinosaurs as they evolved into birds.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02817-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1560-1561},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Bird sternum evolution facilitated the origins of powered flight},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus. <em>NATEE</em>, <em>9</em>(9), 1558-1559. (<a href='https://doi.org/10.1038/s41559-025-02799-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geochemical chronologies in surface increments and exposed cross-sections of naturally fractured hominin enamel from the South African sites of Swartkrans and Kromdraai indicate that Paranthropus robustus exploited both forest and grassland habitats, and that individuals did not move on the landscape in a manner analogous to extant African apes.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02799-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1558-1559},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolving a bat wing by reusing an existing gene program. <em>NATEE</em>, <em>9</em>(9), 1556-1557. (<a href='https://doi.org/10.1038/s41559-025-02782-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interspecies comparative analyses of single-cell transcriptomic data reveal that a proximal limb gene program is reactivated during bat wing formation. Our results illustrate how evolutionary innovations can arise from the reutilization of an existing gene set, activated through species-specific regulatory repertoires.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02782-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1556-1557},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolving a bat wing by reusing an existing gene program},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The next generation of colour pattern genomics. <em>NATEE</em>, <em>9</em>(9), 1554-1555. (<a href='https://doi.org/10.1038/s41559-025-02816-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Convolutional neural networks and genetic association analysis decode the evolution of colour pattern diversity and its underlying complex genetic architecture in the Trinidadian guppy.},
  archive      = {J_NATEE},
  author       = {Karagic, Nidal and Kratochwil, Claudius F.},
  doi          = {10.1038/s41559-025-02816-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1554-1555},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The next generation of colour pattern genomics},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sea star wasting disease mystery finally solved. <em>NATEE</em>, <em>9</em>(9), 1552-1553. (<a href='https://doi.org/10.1038/s41559-025-02789-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A decade after a marine epidemic killed off sea stars and triggered ecosystem-wide effects along the Pacific Coast of North America, researchers have identified the bacterial pathogen that is responsible for sea star wasting disease.},
  archive      = {J_NATEE},
  author       = {Lafferty, Kevin D.},
  doi          = {10.1038/s41559-025-02789-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1552-1553},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sea star wasting disease mystery finally solved},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leprosy in the americas before columbus. <em>NATEE</em>, <em>9</em>(9), 1550-1551. (<a href='https://doi.org/10.1038/s41559-025-02790-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis genomes associated with 4,000-year-old human skeletons in Chile establish an American origin for this causal agent of leprosy (also known as Hansen’s disease), and point to different evolutionary trajectories and transmission pathways for M. lepromatosis and its sister pathogen Mycobacterium leprae.},
  archive      = {J_NATEE},
  author       = {Roberts, Charlotte A.},
  doi          = {10.1038/s41559-025-02790-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1550-1551},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Leprosy in the americas before columbus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Water-use efficiency driven by water savings. <em>NATEE</em>, <em>9</em>(9), 1548-1549. (<a href='https://doi.org/10.1038/s41559-025-02793-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of land–atmosphere water and CO2 fluxes suggests that reduced water use by vegetation, rather than increased carbon uptake, is the driving factor behind the well-documented increase in vegetation water-use efficiency in response to rising atmospheric CO2.},
  archive      = {J_NATEE},
  author       = {Knauer, Jürgen},
  doi          = {10.1038/s41559-025-02793-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1548-1549},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Water-use efficiency driven by water savings},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Independent origins of long snouts. <em>NATEE</em>, <em>9</em>(9), 1547. (<a href='https://doi.org/10.1038/s41559-025-02856-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-025-02856-8},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1547},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Independent origins of long snouts},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Seed dispersers wanted. <em>NATEE</em>, <em>9</em>(9), 1546. (<a href='https://doi.org/10.1038/s41559-025-02857-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02857-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1546},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Seed dispersers wanted},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fish without borders. <em>NATEE</em>, <em>9</em>(9), 1545. (<a href='https://doi.org/10.1038/s41559-025-02852-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02852-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1545},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fish without borders},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mines aren’t on the maps. <em>NATEE</em>, <em>9</em>(9), 1544. (<a href='https://doi.org/10.1038/s41559-025-02833-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-025-02833-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1544},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mines aren’t on the maps},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Museums in the age of extinction. <em>NATEE</em>, <em>9</em>(9), 1542-1543. (<a href='https://doi.org/10.1038/s41559-025-02772-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Browning, Claire},
  doi          = {10.1038/s41559-025-02772-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1542-1543},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Museums in the age of extinction},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding environmental impacts of plastic requires a palaeontological lens. <em>NATEE</em>, <em>9</em>(9), 1539-1541. (<a href='https://doi.org/10.1038/s41559-025-02813-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Growing evidence suggests that timescales for plastic degradation have been vastly underestimated. The fossil record of plastic-like biopolymers might provide a perspective on plastic fossilization in deep time.},
  archive      = {J_NATEE},
  author       = {Gabbott, Sarah and Boom, Arnoud and Zalasiewicz, Jan},
  doi          = {10.1038/s41559-025-02813-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1539-1541},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Understanding environmental impacts of plastic requires a palaeontological lens},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Restoration outcomes are compromised by a lack of diverse native seed supply. <em>NATEE</em>, <em>9</em>(9), 1537-1538. (<a href='https://doi.org/10.1038/s41559-025-02825-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Changes in seed banking practices and business incentives are needed to ensure restoration practitioners have access to plentiful and reliable supplies of diverse native seeds.},
  archive      = {J_NATEE},
  author       = {Gallagher, Rachael V.},
  doi          = {10.1038/s41559-025-02825-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1537-1538},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Restoration outcomes are compromised by a lack of diverse native seed supply},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Increase the regulation of biodiversity effects from private equity markets. <em>NATEE</em>, <em>9</em>(9), 1535-1536. (<a href='https://doi.org/10.1038/s41559-025-02828-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Carter, Harrison and Miles, Nell and Hawkins, Isobel and Hulett, Toscanie and zu Ermgassen, Sophus O. S. E.},
  doi          = {10.1038/s41559-025-02828-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1535-1536},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increase the regulation of biodiversity effects from private equity markets},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The amazon’s new oil frontier is a biodiversity bomb in the making. <em>NATEE</em>, <em>9</em>(9), 1532-1534. (<a href='https://doi.org/10.1038/s41559-025-02815-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Paletta, Julia and Callegari, Camila and Oliveira Fiorini, Ana Carolina and Dias, Filipe Gomes and Freitas Junior, Adirson and de Barros, Pedro Henrique Batista},
  doi          = {10.1038/s41559-025-02815-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1532-1534},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The amazon’s new oil frontier is a biodiversity bomb in the making},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sound science. <em>NATEE</em>, <em>9</em>(9), 1531. (<a href='https://doi.org/10.1038/s41559-025-02860-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Passive acoustic monitoring is opening up new opportunities for biodiversity monitoring, but there is still work to do to ensure that it represents a reliable method for biodiversity reporting.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02860-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1531},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sound science},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
