<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>BIOMTC</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="biomtc">BIOMTC - 39</h2>
<ul>
<li><details>
<summary>
(2025). Joint disease mapping for bivariate count data with residual correlation due to unknown number of common cases. <em>BIOMTC</em>, <em>81</em>(3), ujaf119. (<a href='https://doi.org/10.1093/biomtc/ujaf119'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The joint spatial distribution of two count outcomes (eg, counts of two diseases) is usually studied using a Poisson shared component model (P-SCM), which uses geographically structured latent variables to model spatial variations that are specific and shared by both outcomes. In this model, the correlation between the outcomes is assumed to be fully accounted for by the latent variables. However, in this article, we show that when the outcomes have an unknown number of cases in common, the bivariate counts exhibit a positive “residual” correlation, which the P-SCM wrongly attributes to the covariance of the latent variables, leading to biased inference and degraded predictive performance. Accordingly, we propose a new SCM based on the Bivariate-Poisson distribution (BP-SCM hereafter) to study such correlated bivariate data. The BP-SCM decomposes each count into counts of common and distinct cases, and then models each of these three counts (two distinct and one common) using Gaussian Markov Random Fields. The model is formulated in a Bayesian framework using Hamiltonian Monte Carlo inference. Simulations and a real-world application showed the good inferential and predictive performances of the BP-SCM and confirm the bias in P-SCM. BP-SCM provides rich epidemiological information, such as the mean levels of the unknown counts of common and distinct cases, and their shared and specific spatial variations.},
  archive      = {J_BIOMTC},
  author       = {Chatignoux, Edouard and Uhry, Zoé and Remontet, Laurent and Albert, Isabelle},
  doi          = {10.1093/biomtc/ujaf119},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf119},
  shortjournal = {Biometrics},
  title        = {Joint disease mapping for bivariate count data with residual correlation due to unknown number of common cases},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nonparametric bayesian approach for dynamic borrowing of historical control data. <em>BIOMTC</em>, <em>81</em>(3), ujaf118. (<a href='https://doi.org/10.1093/biomtc/ujaf118'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When incorporating historical control data into the analysis of current randomized controlled trial data, it is critical to account for differences between the datasets. When the cause of difference is an unmeasured factor and adjustment for only observed covariates is insufficient, it is desirable to use a dynamic borrowing method that reduces the impact of heterogeneous historical controls. We propose a nonparametric Bayesian approach that addresses between-trial heterogeneity and allows borrowing historical controls homogeneous with the current control. Additionally, to emphasize conflict resolution between historical controls and the current control, we introduce a method based on the dependent Dirichlet process (DP) mixture. The proposed methods can be implemented using the same procedure, regardless of whether the outcome data comprise aggregated study-level data or individual participant data. We also develop a novel index of similarity between the historical and current control data, based on the posterior distribution of the parameter of interest. We conduct a simulation study and analyze clinical trial examples to evaluate the performance of the proposed methods compared to existing methods. The proposed method, based on the dependent DP mixture, can accurately borrow from homogeneous historical controls while reducing the impact of heterogeneous historical controls compared to the typical DP mixture. The proposed methods outperform existing methods in scenarios with heterogeneous historical controls, in which the meta-analytic approach is ineffective.},
  archive      = {J_BIOMTC},
  author       = {Ohigashi, Tomohiro and Maruo, Kazushi and Sozu, Takashi and Gosho, Masahiko},
  doi          = {10.1093/biomtc/ujaf118},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf118},
  shortjournal = {Biometrics},
  title        = {Nonparametric bayesian approach for dynamic borrowing of historical control data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bayesian inference for copy number intra-tumoral heterogeneity from single-cell RNA-sequencing data. <em>BIOMTC</em>, <em>81</em>(3), ujaf115. (<a href='https://doi.org/10.1093/biomtc/ujaf115'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Copy number alterations (CNA) are important drivers and markers of clonal structures within tumors. Understanding these structures at single-cell resolution is crucial to advancing cancer treatments. The objective is to cluster single cells into clones and identify CNA events in each clone. Early attempts often sacrifice the intrinsic link between cell clustering and clonal CNA detection for simplicity and rely heavily on human input for critical parameters such as the number of clones. Here, we develop a Bayesian model to utilize single-cell RNA sequencing (scRNA-seq) data for automatic analysis of intra-tumoral clonal structure concerning CNAs, without reliance on prior knowledge. The model clusters cells into sub-tumoral clones, identifies the number of clones, and simultaneously infers the clonal CNA profiles. It synergistically incorporates input from gene expression and germline single-nucleotide polymorphisms. A Gibbs sampling algorithm has been implemented and is available as an R package Chloris. We demonstrate that our new method compares strongly against existing software tools in terms of both cell clustering and CNA profile identification accuracy. Application to human metastatic melanoma and anaplastic thyroid tumor data demonstrates accurate clustering of tumor and non-tumor cells and reveals clonal CNA profiles that highlight functional gene expression differences between clones from the same tumor.},
  archive      = {J_BIOMTC},
  author       = {Qiao, PuXue and Kwok, Chun Fung and Qian, Guoqi and McCarthy, Davis J},
  doi          = {10.1093/biomtc/ujaf115},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf115},
  shortjournal = {Biometrics},
  title        = {Bayesian inference for copy number intra-tumoral heterogeneity from single-cell RNA-sequencing data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Revisiting optimal allocations for binary responses: Insights from considering type-I error rate control. <em>BIOMTC</em>, <em>81</em>(3), ujaf114. (<a href='https://doi.org/10.1093/biomtc/ujaf114'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This work revisits optimal response-adaptive designs from a type-I error rate perspective, highlighting when and how much these allocations exacerbate type-I error rate inflation—an issue previously undocumented. We explore a range of approaches from the literature that can be applied to reduce type-I error rate inflation. However, we found that all of these approaches fail to give a robust solution to the problem. To address this, we derive 2 optimal allocation proportions, incorporating the more robust score test (instead of the Wald test) with finite sample estimators (instead of the unknown true values) in the formulation of the optimization problem. One proportion optimizes statistical power, and the other minimizes the total number of failures in a trial while maintaining a fixed variance level. Through simulations based on an early phase and a confirmatory trial, we provide crucial practical insight into how these new optimal proportion designs can offer substantial patient outcomes advantages while controlling type-I error rate. While we focused on binary outcomes, the framework offers valuable insights that naturally extend to other outcome types, multi-armed trials, and alternative measures of interest.},
  archive      = {J_BIOMTC},
  author       = {Pin, Lukas and Villar, Sofía S and Rosenberger, William F},
  doi          = {10.1093/biomtc/ujaf114},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf114},
  shortjournal = {Biometrics},
  title        = {Revisiting optimal allocations for binary responses: Insights from considering type-I error rate control},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Semi-supervised linear regression: Enhancing efficiency and robustness in high dimensions. <em>BIOMTC</em>, <em>81</em>(3), ujaf113. (<a href='https://doi.org/10.1093/biomtc/ujaf113'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In semi-supervised learning, the prevailing understanding suggests that observing additional unlabeled samples improves estimation accuracy for linear parameters only in the case of model misspecification. In this work, we challenge such a claim and show that additional unlabeled samples are beneficial in high-dimensional settings. Initially focusing on a dense scenario, we introduce robust semi-supervised estimators for the regression coefficient without relying on sparse structures in the population slope. Even when the true underlying model is linear, we show that leveraging information from large-scale unlabeled data helps reduce estimation bias, thereby improving both estimation accuracy and inference robustness. Moreover, we propose semi-supervised methods with further enhanced efficiency in scenarios with a sparse linear slope. The performance of the proposed methods is demonstrated through extensive numerical studies.},
  archive      = {J_BIOMTC},
  author       = {Chen, Kai and Zhang, Yuqian},
  doi          = {10.1093/biomtc/ujaf113},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf113},
  shortjournal = {Biometrics},
  title        = {Semi-supervised linear regression: Enhancing efficiency and robustness in high dimensions},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Model robust designs for dose-response models. <em>BIOMTC</em>, <em>81</em>(3), ujaf112. (<a href='https://doi.org/10.1093/biomtc/ujaf112'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An optimal experimental design is a structured data collection plan aimed at maximizing the amount of information gathered. Determining an optimal experimental design, however, relies on the assumption that a predetermined model structure, relating the response and covariates, is known a priori. In practical scenarios, such as dose-response modeling, the form of the model representing the “true” relationship is frequently unknown, although there exists a finite set or pool of potential alternative models. Designing experiments based on a single model from this set may lead to inefficiency or inadequacy if the “true” model differs from that assumed when calculating the design. One approach to minimize the impact of the uncertainty in the model on the experimental plan is known as model robust design . In this context, we systematically address the challenge of finding approximate optimal model robust experimental designs. Our focus is on locally optimal designs, so allowing some of the models in the pool to be nonlinear. We present three Semidefinite Programming-based formulations, each aligned with one of the classes of model robustness criteria introduced by Läuter. These formulations exploit the semidefinite representability of the robustness criteria, leading to the representation of the robust problem as a semidefinite program. To ensure comparability of information measures across various models, we employ standardized designs. To illustrate the application of our approach, we consider a dose-response study where, initially, seven models were postulated as potential candidates to describe the dose-response relationship.},
  archive      = {J_BIOMTC},
  author       = {Duarte, Belmiro P M and Atkinson, Anthony C and Oliveira, Nuno M C},
  doi          = {10.1093/biomtc/ujaf112},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf112},
  shortjournal = {Biometrics},
  title        = {Model robust designs for dose-response models},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Correction to: Evaluating the effects of high-throughput structural neuroimaging predictors on whole-brain functional connectome outcomes via network-based matrix-on-vector regression. <em>BIOMTC</em>, <em>81</em>(3), ujaf111. (<a href='https://doi.org/10.1093/biomtc/ujaf111'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BIOMTC},
  doi          = {10.1093/biomtc/ujaf111},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf111},
  shortjournal = {Biometrics},
  title        = {Correction to: Evaluating the effects of high-throughput structural neuroimaging predictors on whole-brain functional connectome outcomes via network-based matrix-on-vector regression},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mastering rare event analysis: Subsample-size determination in cox and logistic regressions. <em>BIOMTC</em>, <em>81</em>(3), ujaf110. (<a href='https://doi.org/10.1093/biomtc/ujaf110'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the realm of contemporary data analysis, the use of massive datasets has taken on heightened significance, albeit often entailing considerable demands on computational time and memory. While a multitude of existing works offer optimal subsampling methods for conducting analyses on subsamples with minimized efficiency loss, they notably lack tools for judiciously selecting the subsample size. To bridge this gap, our work introduces tools designed for choosing the subsample size. We focus on three settings: the Cox regression model for survival data with rare events, and logistic regression for both balanced and imbalanced datasets. Additionally, we present a new optimal subsampling procedure tailored to logistic regression with imbalanced data. The efficacy of these tools and procedures is demonstrated through an extensive simulation study and meticulous analyses of two sizable datasets: survival analysis of UK Biobank colorectal cancer data with about 350 million rows and logistic regression of linked birth and infant death data with about 28 million observations.},
  archive      = {J_BIOMTC},
  author       = {Agassi, Tal and Keret, Nir and Gorfine, Malka},
  doi          = {10.1093/biomtc/ujaf110},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf110},
  shortjournal = {Biometrics},
  title        = {Mastering rare event analysis: Subsample-size determination in cox and logistic regressions},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-dimensional multi-study multi-modality covariate-augmented generalized factor model. <em>BIOMTC</em>, <em>81</em>(3), ujaf107. (<a href='https://doi.org/10.1093/biomtc/ujaf107'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Latent factor models that integrate data from multiple sources/studies or modalities have garnered considerable attention across various disciplines. However, existing methods predominantly focus either on multi-study integration or multi-modality integration, rendering them insufficient for analyzing the diverse modalities measured across multiple studies. To address this limitation and cater to practical needs, we introduce a high-dimensional generalized factor model that seamlessly integrates multi-modality data from multiple studies, while also accommodating additional covariates. We conduct a thorough investigation of the identifiability conditions to enhance the model’s interpretability. To tackle the complexity of high-dimensional nonlinear integration caused by 4 large latent random matrices, we utilize a variational lower bound to approximate the observed log-likelihood by employing a variational posterior distribution. By profiling the variational parameters, we establish the asymptotical properties of estimators for model parameters using M-estimation theory. Furthermore, we devise a computationally efficient variational expectation maximization (EM) algorithm to execute the estimation process and a criterion to determine the optimal number of both study-shared and study-specific factors. Extensive simulation studies and a real-world application show that the proposed method significantly outperforms existing methods in terms of estimation accuracy and computational efficiency.},
  archive      = {J_BIOMTC},
  author       = {Liu, Wei and Zhong, Qingzhi},
  doi          = {10.1093/biomtc/ujaf107},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf107},
  shortjournal = {Biometrics},
  title        = {High-dimensional multi-study multi-modality covariate-augmented generalized factor model},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A monotone single index model for spatially referenced multistate current status data. <em>BIOMTC</em>, <em>81</em>(3), ujaf105. (<a href='https://doi.org/10.1093/biomtc/ujaf105'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assessment of multistate disease progression is commonplace in biomedical research, such as in periodontal disease (PD). However, the presence of multistate current status endpoints, where only a single snapshot of each subject’s progression through disease states is available at a random inspection time after a known starting state, complicates the inferential framework. In addition, these endpoints can be clustered, and spatially associated, where a group of proximally located teeth (within subjects) may experience similar PD status, compared to those distally located. Motivated by a clinical study recording PD progression, we propose a Bayesian semiparametric accelerated failure time model with an inverse-Wishart proposal for accommodating (spatial) random effects, and flexible errors that follow a Dirichlet process mixture of Gaussians. For clinical interpretability, the systematic component of the event times is modeled using a monotone single index model, with the (unknown) link function estimated via a novel integrated basis expansion and basis coefficients endowed with constrained Gaussian process priors. In addition to establishing parameter identifiability, we present scalable computing via a combination of elliptical slice sampling, fast circulant embedding techniques, and smoothing of hard constraints, leading to straightforward estimation of parameters, and state occupation and transition probabilities. Using synthetic data, we study the finite sample properties of our Bayesian estimates and their performance under model misspecification. We also illustrate our method via application to the real clinical PD dataset.},
  archive      = {J_BIOMTC},
  author       = {Das, Snigdha and Chae, Minwoo and Pati, Debdeep and Bandyopadhyay, Dipankar},
  doi          = {10.1093/biomtc/ujaf105},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf105},
  shortjournal = {Biometrics},
  title        = {A monotone single index model for spatially referenced multistate current status data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Semiparametric joint modeling to estimate the treatment effect on a longitudinal surrogate with application to chronic kidney disease trials. <em>BIOMTC</em>, <em>81</em>(3), ujaf104. (<a href='https://doi.org/10.1093/biomtc/ujaf104'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In clinical trials where long follow-up is required to measure the primary outcome of interest, there is substantial interest in using an accepted surrogate outcome that can be measured earlier in time or with less cost to estimate a treatment effect. For example, in clinical trials of chronic kidney disease, the effect of a treatment is often demonstrated on a longitudinal surrogate, the change of the longitudinal outcome (glomerular filtration rate, GFR) per year or GFR slope. However, estimating the effect of a treatment on the GFR slope is complicated by the fact that GFR measurement can be terminated by the occurrence of a terminal event, such as death or kidney failure. Thus, to estimate this effect, one must consider both the longitudinal GFR trajectory and the terminal event process. In this paper, we build a semiparametric framework to jointly model the longitudinal outcome and the terminal event, where the model for the longitudinal outcome is semiparametric, the relationship between the longitudinal outcome and the terminal event is nonparametric, and the terminal event is modeled via a semiparametric Cox model. The proposed semiparametric joint model is flexible and can be easily extended to include a nonlinear trajectory of the longitudinal outcome. An estimating equation based method is proposed to estimate the treatment effect on the longitudinal surrogate outcome (eg, GFR slope). Theoretical properties of the proposed estimators are derived, and finite sample performance is evaluated through simulation studies. We illustrate the proposed method using data from the Reduction of Endpoints in NIDDM with the Angiotensin II Antagonist Losartan (RENAAL) trial to examine the effect of Losartan on GFR slope.},
  archive      = {J_BIOMTC},
  author       = {Wang, Xuan and Zhou, Jie and Parast, Layla and Greene, Tom},
  doi          = {10.1093/biomtc/ujaf104},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf104},
  shortjournal = {Biometrics},
  title        = {Semiparametric joint modeling to estimate the treatment effect on a longitudinal surrogate with application to chronic kidney disease trials},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evaluating longitudinal treatment effects for duchenne muscular dystrophy using dynamically enriched bayesian small sample, sequential, multiple assignment randomized trial (snSMART). <em>BIOMTC</em>, <em>81</em>(3), ujaf103. (<a href='https://doi.org/10.1093/biomtc/ujaf103'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For progressive rare diseases like Duchenne muscular dystrophy (DMD), evaluating disease burden by measuring the totality of evidence from outcome data over time per patient can be highly informative, especially regarding how a new treatment impacts disease progression and functional outcomes. This paper focuses on new statistical approaches for analyzing data generated over time in a small sample, sequential, multiple assignment, randomized trial (snSMART), with an application to DMD. In addition, the use of external control data can enhance the statistical and operational efficiency in rare disease drug development by solving participant scarcity issues and ethical challenges. We employ a two-step robust meta-analytic approach to leverage external control data while adjusting for important baseline confounders and potential conflicts between external controls and trial data. Furthermore, our approach integrates important baseline covariates to account for patient heterogeneity and introduces a novel piecewise model to manage stage-wise treatment assignments. By applying this methodology to a case study in DMD research, we not only demonstrate the practical application and benefits of our approach but also highlight its potential to mitigate challenges in rare disease trials. Our findings advocate for a more nuanced and statistically robust analysis of treatment effects, thereby improving the reliability of clinical trial results.},
  archive      = {J_BIOMTC},
  author       = {Wang, Sidi and Roychoudhury, Satrajit and Kidwell, Kelley M},
  doi          = {10.1093/biomtc/ujaf103},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf103},
  shortjournal = {Biometrics},
  title        = {Evaluating longitudinal treatment effects for duchenne muscular dystrophy using dynamically enriched bayesian small sample, sequential, multiple assignment randomized trial (snSMART)},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sensitivity analysis for attributable effects in case2 studies. <em>BIOMTC</em>, <em>81</em>(3), ujaf102. (<a href='https://doi.org/10.1093/biomtc/ujaf102'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The case 2 study, also referred to as the case–case study design, is a valuable approach for conducting inference for treatment effects. Unlike traditional case-control studies, the case 2 design compares treatment in cases of concern (the first type of case) to other cases (the second type of case). One of the quantities of interest is the attributable effect for the first type of case—that is, the number of the first type of case that would not have occurred had the treatment been withheld from all units. In some case 2 studies, a key quantity of interest is the attributable effect for the first type of case. Two key assumptions that are usually made for making inferences about this attributable effect in case 2 studies are (1) treatment does not cause the second type of case, and (2) the treatment does not alter an individual’s case type. However, these assumptions are not realistic in many real-data applications. In this article, we present a sensitivity analysis framework to scrutinize the impact of deviations from these assumptions on inferences for the attributable effect. We also include sensitivity analyses related to the assumption of unmeasured confounding, recognizing the potential bias introduced by unobserved covariates. The proposed methodology is exemplified through an investigation into whether having violent behavior in the last year of life increases suicide risk using the 1993 National Mortality Followback Survey dataset.},
  archive      = {J_BIOMTC},
  author       = {Chen, Kan and Ye, Ting and Small, Dylan S},
  doi          = {10.1093/biomtc/ujaf102},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf102},
  shortjournal = {Biometrics},
  title        = {Sensitivity analysis for attributable effects in case2 studies},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Smooth and shape-constrained quantile distributed lag models. <em>BIOMTC</em>, <em>81</em>(3), ujaf101. (<a href='https://doi.org/10.1093/biomtc/ujaf101'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exposure to environmental pollutants during the gestational period can significantly impact infant health outcomes, such as birth weight and neurological development. Identifying critical windows of susceptibility, which are specific periods during pregnancy when exposure has the most profound effects, is essential for developing targeted interventions. Distributed lag models (DLMs) are widely used in environmental epidemiology to analyze the temporal patterns of exposure and their impact on health outcomes. However, traditional DLMs focus on modeling the conditional mean, which may fail to capture heterogeneity in the relationship between predictors and the outcome. Moreover, when modeling the distribution of health outcomes like gestational birth weight, it is the extreme quantiles that are of most clinical relevance. We introduce 2 new quantile distributed lag model (QDLM) estimators designed to address the limitations of existing methods by leveraging smoothness and shape constraints, such as unimodality and concavity, to enhance interpretability and efficiency. We apply our QDLM estimators to the Colorado birth cohort data, demonstrating their effectiveness in identifying critical windows of susceptibility and informing public health interventions.},
  archive      = {J_BIOMTC},
  author       = {Jin, Yisen and Molstad, Aaron J and Wilson, Ander and Antonelli, Joseph},
  doi          = {10.1093/biomtc/ujaf101},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf101},
  shortjournal = {Biometrics},
  title        = {Smooth and shape-constrained quantile distributed lag models},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Regression analysis of interval-censored failure time data with change points and a cured subgroup. <em>BIOMTC</em>, <em>81</em>(3), ujaf100. (<a href='https://doi.org/10.1093/biomtc/ujaf100'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There exists a substantial body of literature that discusses regression analysis of interval-censored failure time data and also many methods have been proposed for handling the presence of a cured subgroup. However, only limited research exists on the problems incorporating change points, with or without a cured subgroup, which can occur in various contexts such as clinical trials where disease risks may shift dramatically when certain biological indicators exceed specific thresholds. To fill this gap, we consider a class of partly linear transformation models within the mixture cure model framework and propose a sieve maximum likelihood estimation approach using Bernstein polynomials and piecewise linear functions for inference. Additionally, we provide a data-driven adaptive procedure to identify the number and locations of change points and establish the asymptotic properties of the proposed method. Extensive simulation studies demonstrate the effectiveness and practical utility of the proposed methods, which are applied to the real data from a breast cancer study that motivated this work.},
  archive      = {J_BIOMTC},
  author       = {Lou, Yichen and Du, Mingyue and Song, Xinyuan},
  doi          = {10.1093/biomtc/ujaf100},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf100},
  shortjournal = {Biometrics},
  title        = {Regression analysis of interval-censored failure time data with change points and a cured subgroup},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Negative binomial mixed effects location-scale models for intensive longitudinal count-type physical activity data provided by wearable devices. <em>BIOMTC</em>, <em>81</em>(3), ujaf099. (<a href='https://doi.org/10.1093/biomtc/ujaf099'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, the use of wearable devices, for example, accelerometers, have become increasingly prevalent. Wearable devices enable more accurate real-time tracking of a subject’s physical activity (PA) level, such as steps, number of activity bouts, or time in moderate-to-vigorous intensity PA (MVPA), which are important general health markers and can often be represented as counts. These intensive within-subject count data provided by wearable devices, for example, minutes in MVPA summarized per hour across days and even months, allow the possibility for modeling not only the mean PA level, but also the dispersion level for each subject. Especially in the context of daily PA, subjects’ dispersion levels are potentially informative in reflecting their exercise patterns: some subjects might exhibit consistent PA across time and can be considered “less dispersed” subjects; while others might have a large amount of PA at a particular time point, while being sedentary for most of the day, and can be considered “more dispersed” subjects. Thus, we propose a negative binomial mixed effects location-scale model to model these intensive longitudinal PA counts and to account for the heterogeneity in both the mean and dispersion level across subjects. Further, to handle the issue of inflated numbers of zeros in the PA data, we also propose a hurdle/zero-inflated version which additionally includes the modeling of the probability of having |$>$| 0 PA levels.},
  archive      = {J_BIOMTC},
  author       = {Ma, Qianheng and Dunton, Genevieve F and Hedeker, Donald},
  doi          = {10.1093/biomtc/ujaf099},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf099},
  shortjournal = {Biometrics},
  title        = {Negative binomial mixed effects location-scale models for intensive longitudinal count-type physical activity data provided by wearable devices},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Causal machine learning for heterogeneous treatment effects in the presence of missing outcome data. <em>BIOMTC</em>, <em>81</em>(3), ujaf098. (<a href='https://doi.org/10.1093/biomtc/ujaf098'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When estimating heterogeneous treatment effects, missing outcome data can complicate treatment effect estimation, causing certain subgroups of the population to be poorly represented. In this work, we discuss this commonly overlooked problem and consider the impact that missing at random outcome data has on causal machine learning estimators for the conditional average treatment effect (CATE). We propose 2 de-biased machine learning estimators for the CATE, the mDR-learner, and mEP-learner, which address the issue of under-representation by integrating inverse probability of censoring weights into the DR-learner and EP-learner, respectively. We show that under reasonable conditions, these estimators are oracle efficient and illustrate their favorable performance through simulated data settings, comparing them to existing CATE estimators, including comparison to estimators that use common missing data techniques. We present an example of their application using the GBSG2 trial, exploring treatment effect heterogeneity when comparing hormonal therapies to non-hormonal therapies among breast cancer patients post surgery, and offer guidance on the decisions a practitioner must make when implementing these estimators.},
  archive      = {J_BIOMTC},
  author       = {Pryce, Matthew and Diaz-Ordaz, Karla and Keogh, Ruth H and Vansteelandt, Stijn},
  doi          = {10.1093/biomtc/ujaf098},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf098},
  shortjournal = {Biometrics},
  title        = {Causal machine learning for heterogeneous treatment effects in the presence of missing outcome data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Covariance-on-covariance regression. <em>BIOMTC</em>, <em>81</em>(3), ujaf097. (<a href='https://doi.org/10.1093/biomtc/ujaf097'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A covariance-on-covariance regression model is introduced in this manuscript. It is assumed that there exists (at least) a pair of linear projections on outcome covariance matrices and predictor covariance matrices such that a log-linear model links the variances in the projection spaces, as well as additional covariates of interest. An ordinary least square type of estimator is proposed to simultaneously identify the projections and estimate model coefficients. Under regularity conditions, the proposed estimator is asymptotically consistent. The superior performance of the proposed approach over existing methods is demonstrated via simulation studies. Applying to data collected in the Human Connectome Project Aging study, the proposed approach identifies 3 pairs of brain networks, where functional connectivity within the resting-state network predicts functional connectivity within the corresponding task-state network. The 3 networks correspond to a global signal network, a task-related network, and a task-unrelated network. The findings are consistent with existing knowledge about brain function.},
  archive      = {J_BIOMTC},
  author       = {Zhao, Yi and Zhao, Yize},
  doi          = {10.1093/biomtc/ujaf097},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf097},
  shortjournal = {Biometrics},
  title        = {Covariance-on-covariance regression},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Valid and efficient inference for nonparametric variable importance in two-phase studies. <em>BIOMTC</em>, <em>81</em>(3), ujaf095. (<a href='https://doi.org/10.1093/biomtc/ujaf095'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We consider a common nonparametric regression setting, where the data consist of a response variable Y , some easily obtainable covariates X ⁠ , and a set of costly covariates Z ⁠ . Before establishing predictive models for Y , a natural question arises: Is it worthwhile to include Z as predictors, given the additional cost of collecting data on Z for both training the models and predicting Y for future individuals? Therefore, we aim to conduct preliminary investigations to infer importance of Z in predicting Y in the presence of X ⁠ . To achieve this goal, we propose a nonparametric variable importance measure for Z ⁠ . It is defined as a parameter that aggregates maximum potential contributions of Z in single or multiple predictive models, with contributions quantified by general loss functions. Considering two-phase data that provide a large number of observations for ( Y , X ) with the expensive Z measured only in a small subsample, we develop a novel approach to infer the proposed importance measure, accommodating missingness of Z in the sample by substituting functions of ( Y , X ) for each individual’s contribution to the predictive loss of models involving Z ⁠ . Our approach attains unified and efficient inference regardless of whether Z makes zero or positive contribution to predicting Y , a desirable yet surprising property owing to data incompleteness. As intermediate steps of our theoretical development, we establish novel results in two relevant research areas, semi-supervised inference and two-phase nonparametric estimation. Numerical results from both simulated and real data demonstrate superior performance of our approach.},
  archive      = {J_BIOMTC},
  author       = {Dai, Guorong and Carroll, Raymond J and Chen, Jinbo},
  doi          = {10.1093/biomtc/ujaf095},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf095},
  shortjournal = {Biometrics},
  title        = {Valid and efficient inference for nonparametric variable importance in two-phase studies},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improved prediction and flagging of extreme random effects for non-gaussian outcomes using weighted methods. <em>BIOMTC</em>, <em>81</em>(3), ujaf094. (<a href='https://doi.org/10.1093/biomtc/ujaf094'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Investigators often focus on predicting extreme random effects from mixed effects models fitted to longitudinal or clustered data, and on identifying or “flagging” outliers such as poorly performing hospitals or rapidly deteriorating patients. Our recent work with Gaussian outcomes showed that weighted prediction methods can substantially reduce mean square error of prediction for extremes and substantially increase correct flagging rates compared to previous methods, while controlling the incorrect flagging rates. This paper extends the weighted prediction methods to non-Gaussian outcomes such as binary and count data. Closed-form expressions for predicted random effects and probabilities of correct and incorrect flagging are not available for the usual non-Gaussian outcomes, and the computational challenges are substantial. Therefore, our results include the development of theory to support algorithms that tune predictors that we call “self-calibrated” (which control the incorrect flagging rate using very simple flagging rules) and innovative numerical methods to calculate weighted predictors as well as to evaluate their performance. Comprehensive numerical evaluations show that the novel weighted predictors for non-Gaussian outcomes have substantially lower mean square error of prediction at the extremes and considerably higher correct flagging rates than previously proposed methods, while controlling the incorrect flagging rates. We illustrate our new methods using data on emergency room readmissions for children with asthma.},
  archive      = {J_BIOMTC},
  author       = {Neuhaus, John and McCulloch, Charles and Boylan, Ross},
  doi          = {10.1093/biomtc/ujaf094},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf094},
  shortjournal = {Biometrics},
  title        = {Improved prediction and flagging of extreme random effects for non-gaussian outcomes using weighted methods},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Two-stage estimators for spatial confounding with point-referenced data. <em>BIOMTC</em>, <em>81</em>(3), ujaf093. (<a href='https://doi.org/10.1093/biomtc/ujaf093'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Public health data are often spatially dependent, but standard spatial regression methods can suffer from bias and invalid inference when the independent variable is associated with spatially correlated residuals. This could occur if, for example, there is an unmeasured environmental contaminant associated with the independent and outcome variables in a spatial regression analysis. Geoadditive structural equation modeling (gSEM), in which an estimated spatial trend is removed from both the explanatory and response variables before estimating the parameters of interest, has previously been proposed as a solution but there has been little investigation of gSEM’s properties with point-referenced data. We link gSEM to results on double machine learning and semiparametric regression based on two-stage procedures. We propose using these semiparametric estimators for spatial regression using Gaussian processes with Matèrn covariance to estimate the spatial trends and term this class of estimators double spatial regression (DSR). We derive regularity conditions for root- n asymptotic normality and consistency and closed-form variance estimation, and show that in simulations where standard spatial regression estimators are highly biased and have poor coverage, DSR can mitigate bias more effectively than competitors and obtain nominal coverage.},
  archive      = {J_BIOMTC},
  author       = {Wiecha, Nate and Hoppin, Jane A and Reich, Brian J},
  doi          = {10.1093/biomtc/ujaf093},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf093},
  shortjournal = {Biometrics},
  title        = {Two-stage estimators for spatial confounding with point-referenced data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Using model-assisted calibration methods to improve efficiency of regression analyses using two-phase samples or pooled samples under complex survey designs. <em>BIOMTC</em>, <em>81</em>(3), ujaf092. (<a href='https://doi.org/10.1093/biomtc/ujaf092'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two-phase sampling designs are frequently applied in epidemiological studies and large-scale health surveys. In such designs, certain variables are collected exclusively within a second-phase random subsample of the initial first-phase sample, often due to factors such as high costs, response burden, or constraints on data collection or assessment. Consequently, second-phase sample estimators can be inefficient due to the diminished sample size. Model-assisted calibration methods have been used to improve the efficiency of second-phase estimators in regression analysis. However, limited literature provides valid finite population inferences of the calibration estimators that use appropriate calibration auxiliary variables while simultaneously accounting for the complex sample designs in the first- and second-phase samples. Moreover, no literature considers the “pooled design” where some covariates are measured exclusively in certain repeated survey cycles. This paper proposes calibrating the sample weights for the second-phase sample to the weighted first-phase sample based on score functions of the regression model that uses predictions of the second-phase variable for the first-phase sample. We establish the consistency of estimation using calibrated weights and provide variance estimation for the regression coefficients under the two-phase design or the pooled design nested within complex survey designs. Empirical evidence highlights the efficiency and robustness of the proposed calibration compared to existing calibration and imputation methods. Data examples from the National Health and Nutrition Examination Survey are provided.},
  archive      = {J_BIOMTC},
  author       = {Wang, Lingxiao},
  doi          = {10.1093/biomtc/ujaf092},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf092},
  shortjournal = {Biometrics},
  title        = {Using model-assisted calibration methods to improve efficiency of regression analyses using two-phase samples or pooled samples under complex survey designs},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Correction to “Propensity weighting plus adjustment in proportional hazards model is not doubly robust,” by erin e. gabriel, michael c. sachs, ingeborg waernbaum, els goetghebeur, paul f. blanche, stijn vansteelandt, arvid sjölander, and thomas scheike; volume 80, issue 3, september 2024, https://doi.org/10.1093/biomtc/ujae069. <em>BIOMTC</em>, <em>81</em>(3), ujaf091. (<a href='https://doi.org/10.1093/biomtc/ujaf091'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BIOMTC},
  author       = {Gabriel, Erin E and Sachs, Michael C and Waernbaum, Ingeborg and Goetghebeur, Els and Blanche, Paul F and Vansteelandt, Stijn and Sjölander, Arvid and Scheike, Thomas},
  doi          = {10.1093/biomtc/ujaf091},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf091},
  shortjournal = {Biometrics},
  title        = {Correction to “Propensity weighting plus adjustment in proportional hazards model is not doubly robust,” by erin e. gabriel, michael c. sachs, ingeborg waernbaum, els goetghebeur, paul f. blanche, stijn vansteelandt, arvid sjölander, and thomas scheike; volume 80, issue 3, september 2024, https://doi.org/10.1093/biomtc/ujae069},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Adjusted predictions for generalized estimating equations. <em>BIOMTC</em>, <em>81</em>(3), ujaf090. (<a href='https://doi.org/10.1093/biomtc/ujaf090'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generalized estimating equations (GEEs) are a popular statistical method for longitudinal data analysis, requiring specification of the first 2 marginal moments of the response along with a working correlation matrix to capture temporal correlations within a cluster. When it comes to prediction at future/new time points using GEEs, a standard approach adopted by practitioners and software is to base it simply on the marginal mean model. In this article, we propose an alternative approach to prediction for independent cluster GEEs. By viewing the GEE as solving an iterative working linear model, we borrow ideas from universal kriging to construct an adjusted predictor that exploits working cross-correlations between the current and new observations within the same cluster. We establish theoretical conditions for the adjusted GEE predictor to outperform the standard GEE predictor. Simulations and an application to longitudinal data on the growth of sitka spruces demonstrate that, even when we misspecify the working correlation structure, adjusted GEE predictors can achieve better performance relative to standard GEE predictors, the so-called “oracle” GEE predictor using all time points, and potentially even cluster-specific predictions from a generalized linear mixed model.},
  archive      = {J_BIOMTC},
  author       = {Hui, Francis K C and Muller, Samuel and Welsh, Alan H},
  doi          = {10.1093/biomtc/ujaf090},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf090},
  shortjournal = {Biometrics},
  title        = {Adjusted predictions for generalized estimating equations},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tree-based additive noise directed acyclic graphical models for nonlinear causal discovery with interactions. <em>BIOMTC</em>, <em>81</em>(3), ujaf089. (<a href='https://doi.org/10.1093/biomtc/ujaf089'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed acyclic graphical models with additive noises are essential in nonlinear causal discovery and have numerous applications in various domains, such as social science and systems biology. Most such models further assume that structural causal functions are additive to ensure causal identifiability and computational feasibility, which may be too restrictive in the presence of causal interactions. Some methods consider general nonlinear causal functions represented by, for example, Gaussian processes and neural networks, to accommodate interactions. However, they are either computationally intensive or lack interpretability. We propose a highly interpretable and computationally feasible approach using trees to incorporate interactions in nonlinear causal discovery, termed tree-based additive noise models. The nature of the tree construction leads to piecewise constant causal functions, making existing causal identifiability results of additive noise models with continuous and smooth causal functions inapplicable. Therefore, we provide new conditions under which the proposed model is identifiable. We develop a recursive algorithm for source node identification and a score-based ordering search algorithm. Through extensive simulations, we demonstrate the utility of the proposed model and algorithms benchmarking against existing additive noise models, especially when there are strong causal interactions. Our method is applied to infer a protein–protein interaction network for breast cancer, where proteins may form protein complexes to perform their functions.},
  archive      = {J_BIOMTC},
  author       = {Zhou, Fangting and He, Kejun and Ni, Yang},
  doi          = {10.1093/biomtc/ujaf089},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf089},
  shortjournal = {Biometrics},
  title        = {Tree-based additive noise directed acyclic graphical models for nonlinear causal discovery with interactions},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Simple simulation based reconstruction of incidence rates from death data. <em>BIOMTC</em>, <em>81</em>(3), ujaf088. (<a href='https://doi.org/10.1093/biomtc/ujaf088'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Daily deaths from an infectious disease provide a means for retrospectively inferring daily incidence, given knowledge of the infection-to-death interval distribution. Existing methods for doing so rely either on fitting simplified non-linear epidemic models to the deaths data or on spline based deconvolution approaches. The former runs the risk of introducing unintended artefacts via the model formulation, while the latter may be viewed as technically obscure, impeding uptake by practitioners. This note proposes a simple simulation based approach to inferring fatal incidence from deaths that requires minimal assumptions, is easy to understand, and allows testing of alternative hypothesized incidence trajectories. The aim is that in any future situation similar to the COVID pandemic, the method can be easily, rapidly, transparently, and uncontroversially deployed as an input to management.},
  archive      = {J_BIOMTC},
  author       = {Wood, Simon N},
  doi          = {10.1093/biomtc/ujaf088},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf088},
  shortjournal = {Biometrics},
  title        = {Simple simulation based reconstruction of incidence rates from death data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A flexible framework for N-mixture occupancy models: Applications to breeding bird surveys. <em>BIOMTC</em>, <em>81</em>(3), ujaf087. (<a href='https://doi.org/10.1093/biomtc/ujaf087'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimating species abundance under imperfect detection is a key challenge in biodiversity conservation. The N -mixture model, widely recognized for its ability to distinguish between abundance and individual detection probability without marking individuals, is constrained by its stringent closure assumption, which leads to biased estimates when violated in real-world settings. To address this limitation, we propose an extended framework based on a development of the mixed Gamma-Poisson model, incorporating a community parameter that represents the proportion of individuals consistently present throughout the survey period. This flexible framework generalizes both the zero-inflated type occupancy model and the standard N -mixture model as special cases, corresponding to community parameter values of 0 and 1, respectively. The model’s effectiveness is validated through simulations and applications to real-world datasets, specifically with 5 species from the North American Breeding Bird Survey and 46 species from the Swiss Breeding Bird Survey, demonstrating its improved accuracy and adaptability in settings where strict closure may not hold.},
  archive      = {J_BIOMTC},
  author       = {Huynh, Huu-Dinh and Royle, J Andrew and Hwang, Wen-Han},
  doi          = {10.1093/biomtc/ujaf087},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf087},
  shortjournal = {Biometrics},
  title        = {A flexible framework for N-mixture occupancy models: Applications to breeding bird surveys},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiple tests for restricted mean time lost with competing risks data. <em>BIOMTC</em>, <em>81</em>(3), ujaf086. (<a href='https://doi.org/10.1093/biomtc/ujaf086'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Easy-to-interpret effect estimands are highly desirable in survival analysis. In the competing risks framework, one good candidate is the restricted mean time lost (RMTL). It is defined as the area under the cumulative incidence function up to a prespecified time point and, thus, it summarizes the cumulative incidence function into a meaningful estimand. While existing RMTL-based tests are limited to 2-sample comparisons and mostly to 2 event types, we aim to develop general contrast tests for factorial designs and an arbitrary number of event types based on a Wald-type test statistic. Furthermore, we avoid the often-made, rather restrictive continuity assumption on the event time distribution. This allows for ties in the data, which often occur in practical applications, for example, when event times are measured in whole days. In addition, we develop more reliable tests for RMTL comparisons that are based on a permutation approach to improve the small sample performance. In a second step, multiple tests for RMTL comparisons are developed to test several null hypotheses simultaneously. Here, we incorporate the asymptotically exact dependence structure between the local test statistics to gain more power. The small sample performance of the proposed testing procedures is analyzed in simulations and finally illustrated by analyzing a real-data example about leukemia patients who underwent bone marrow transplantation.},
  archive      = {J_BIOMTC},
  author       = {Munko, Merle and Dobler, Dennis and Ditzhaus, Marc},
  doi          = {10.1093/biomtc/ujaf086},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf086},
  shortjournal = {Biometrics},
  title        = {Multiple tests for restricted mean time lost with competing risks data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A positivity robust strategy to study effects of switching treatment. <em>BIOMTC</em>, <em>81</em>(3), ujaf085. (<a href='https://doi.org/10.1093/biomtc/ujaf085'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In studies of medical treatments, individuals often experience post-treatment events that predict their future outcomes. In this work, we study how to use initial observations of a recurrent event—a type of post-treatment event—to offer updated treatment recommendations in settings where no, or few, individuals are observed to switch between treatment arms. Specifically, we formulate an estimand quantifying the average effect of switching treatment on subsequent events. We derive bounds on the value of this estimand under plausible conditions and propose non-parametric estimators of the bounds. Furthermore, we define a value and regret function for a dynamic treatment-switching regime, and use these to determine 3 types of optimal regimes under partial identification: the pessimist (maximin value), optimist (maximax value), and opportunist (minimax regret) regimes. The pessimist regime is guaranteed to perform at least as well as the standard of care. We apply our methods to data from the Systolic Blood Pressure Intervention Trial.},
  archive      = {J_BIOMTC},
  author       = {Janvin, Matias and Ryalen, Pål C and Sarvet, Aaron L and Stensrud, Mats J},
  doi          = {10.1093/biomtc/ujaf085},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf085},
  shortjournal = {Biometrics},
  title        = {A positivity robust strategy to study effects of switching treatment},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Doubly robust nonparametric estimators of the predictive value of covariates for survival data. <em>BIOMTC</em>, <em>81</em>(3), ujaf084. (<a href='https://doi.org/10.1093/biomtc/ujaf084'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The predictive value of a covariate is often of interest in studies with a survival endpoint. A common situation is that there are some well established predictors and a potential valuable new marker. The challenge is how to judge the potentially added predictive value of this new marker. We propose to use the positive predictive value (PPV) curve based on a nonparametric scoring rule. The estimand of interest is viewed as a single transformation of the underlying data generating probability measure, which allows us to develop a robust nonparametric estimator of the PPV by first calculating the corresponding efficient influence function. We provide asymptotic results and illustrate the approach with numerical studies and with 2 cancer data studies.},
  archive      = {J_BIOMTC},
  author       = {Martinussen, Torben and van der Laan, Mark J},
  doi          = {10.1093/biomtc/ujaf084},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf084},
  shortjournal = {Biometrics},
  title        = {Doubly robust nonparametric estimators of the predictive value of covariates for survival data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Frequency band analysis of nonstationary multivariate time series. <em>BIOMTC</em>, <em>81</em>(3), ujaf083. (<a href='https://doi.org/10.1093/biomtc/ujaf083'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Information from frequency bands in biomedical time series provides useful summaries of the observed signal. Many existing methods consider summaries of the time series obtained over a few well-known, pre-defined frequency bands of interest. However, there is a dearth of data-driven methods for identifying frequency bands that optimally summarize frequency-domain information in the time series. A new method to identify partition points in the frequency space of a multivariate locally stationary time series is proposed. These partition points signify changes across frequencies in the time-varying behavior of the signal and provide frequency band summary measures that best preserve nonstationary dynamics of the observed series. An |$L_2$| -norm based discrepancy measure that finds differences in the time-varying spectral density matrix is constructed, and its asymptotic properties are derived. New nonparametric bootstrap tests are also provided to identify significant frequency partition points and to identify components and cross-components of the spectral matrix exhibiting changes over frequencies. Finite-sample performance of the proposed method is illustrated via simulations. The proposed method is used to develop optimal frequency band summary measures for characterizing time-varying behavior in resting-state electroencephalography time series, as well as identifying components and cross-components associated with each frequency partition point.},
  archive      = {J_BIOMTC},
  author       = {Sundararajan, Raanju R and Bruce, Scott A},
  doi          = {10.1093/biomtc/ujaf083},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf083},
  shortjournal = {Biometrics},
  title        = {Frequency band analysis of nonstationary multivariate time series},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sparse 2-stage bayesian meta-analysis for individualized treatments. <em>BIOMTC</em>, <em>81</em>(3), ujaf082. (<a href='https://doi.org/10.1093/biomtc/ujaf082'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Individualized treatment rules tailor treatments to patients based on clinical, demographic, and other characteristics. Estimation of individualized treatment rules requires the identification of individuals who benefit most from the particular treatments and thus the detection of variability in treatment effects. To develop an effective individualized treatment rule, data from multisite studies may be required due to the low power provided by smaller datasets for detecting the often small treatment-covariate interactions. However, sharing of individual-level data is sometimes constrained. Furthermore, sparsity may arise in 2 senses: different data sites may recruit from different populations, making it infeasible to estimate identical models or all parameters of interest at all sites, and the number of non-zero parameters in the model for the treatment rule may be small. To address these issues, we adopt a 2-stage Bayesian meta-analysis approach to estimate individualized treatment rules which optimize expected patient outcomes using multisite data without disclosing individual-level data beyond the sites. Simulation results demonstrate that our approach can provide consistent estimates of the parameters which fully characterize the optimal individualized treatment rule. We estimate the optimal Warfarin dose strategy using data from the International Warfarin Pharmacogenetics Consortium, where data sparsity and small treatment-covariate interaction effects pose additional statistical challenges.},
  archive      = {J_BIOMTC},
  author       = {Shen, Junwei and Moodie, Erica E M and Golchi, Shirin},
  doi          = {10.1093/biomtc/ujaf082},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf082},
  shortjournal = {Biometrics},
  title        = {Sparse 2-stage bayesian meta-analysis for individualized treatments},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inference on age-specific fertility in ecology and evolution. learning from other disciplines and improving the state of the art. <em>BIOMTC</em>, <em>81</em>(3), ujaf081. (<a href='https://doi.org/10.1093/biomtc/ujaf081'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the importance of age-specific fertility for ecology and evolution, the methods for modeling and inference have proven considerably limited. However, other disciplines have long focused on exploring and developing a vast number of models. Here, I provide an overview of the different models proposed since the 1940s by formal demographers, statisticians, and social scientists, most of which are unknown to the ecological and evolutionary communities. I describe how these fall into 2 main categories, namely polynomials and those based on probability density functions. I discuss their merits in terms of their overall behavior and how well they represent the different stages of fertility. Despite many alternative models, inference on age-specific fertility has usually been limited to simple least squares. Although this might be sufficient for human data, I hope to demonstrate that inference requires more sophisticated approaches for ecological and evolutionary datasets. To illustrate how inference and model choice can be achieved on different types of typical ecological and evolutionary data, I present the new R package Bayesian Fertility Trajectory Analysis, which I apply to published aggregated data for lions and baboons. I then conduct a simulation study to test its performance on individual-level data. I show that appropriate inference and model selection can be achieved even when a small number of parents are followed.},
  archive      = {J_BIOMTC},
  author       = {Colchero, Fernando},
  doi          = {10.1093/biomtc/ujaf081},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf081},
  shortjournal = {Biometrics},
  title        = {Inference on age-specific fertility in ecology and evolution. learning from other disciplines and improving the state of the art},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Introduction to environmental data science by jerry d. davis, chapman and Hall/CRC, 2023, ISBN: 9781003317821 https://www.routledge.com/Introduction-to-environmental-data-science/Davis/p/book/9781003317821. <em>BIOMTC</em>, <em>81</em>(3), ujaf079. (<a href='https://doi.org/10.1093/biomtc/ujaf079'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BIOMTC},
  author       = {Sheppard, Lianne and Blanco, Magali N},
  doi          = {10.1093/biomtc/ujaf079},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf079},
  shortjournal = {Biometrics},
  title        = {Introduction to environmental data science by jerry d. davis, chapman and Hall/CRC, 2023, ISBN: 9781003317821 https://www.routledge.com/Introduction-to-environmental-data-science/Davis/p/book/9781003317821},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Change point analysis for time series by lajos horváth and gregory rice, springer, 2024, ISBN: 9783031516085https://link.springer.com/book/10.1007/978-3-031-51609-2. <em>BIOMTC</em>, <em>81</em>(3), ujaf078. (<a href='https://doi.org/10.1093/biomtc/ujaf078'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BIOMTC},
  author       = {Han, Fang},
  doi          = {10.1093/biomtc/ujaf078},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf078},
  shortjournal = {Biometrics},
  title        = {Change point analysis for time series by lajos horváth and gregory rice, springer, 2024, ISBN: 9783031516085https://link.springer.com/book/10.1007/978-3-031-51609-2},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of gene therapies: Strategic, scientific, regulatory, and access considerations by avery McIntosh and oleksandr sverdlov, chapman and Hall/CRC, 2024, ISBN: 9781032136554 https://www.routledge.com/Development-of-gene-therapies-strategic-scientific-regulatory-and-access-Considerations/McIntosh-sverdlov/p/book/9781032136554. <em>BIOMTC</em>, <em>81</em>(3), ujaf070. (<a href='https://doi.org/10.1093/biomtc/ujaf070'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BIOMTC},
  author       = {Yuan, Ying},
  doi          = {10.1093/biomtc/ujaf070},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf070},
  shortjournal = {Biometrics},
  title        = {Development of gene therapies: Strategic, scientific, regulatory, and access considerations by avery McIntosh and oleksandr sverdlov, chapman and Hall/CRC, 2024, ISBN: 9781032136554 https://www.routledge.com/Development-of-gene-therapies-strategic-scientific-regulatory-and-access-Considerations/McIntosh-sverdlov/p/book/9781032136554},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cumulative incidence function estimation using population-based biobank data. <em>BIOMTC</em>, <em>81</em>(3), ujaf049. (<a href='https://doi.org/10.1093/biomtc/ujaf049'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many countries have established population-based biobanks, which are being used increasingly in epidemiological and clinical research. These biobanks offer opportunities for large-scale studies addressing questions beyond the scope of traditional clinical trials or cohort studies. However, using biobank data poses new challenges. Typically, biobank data are collected from a study cohort recruited over a defined calendar period, with subjects entering the study at various ages falling between c L and c U ⁠ . This work focuses on biobank data with individuals reporting disease-onset age upon recruitment, termed prevalent data, along with individuals initially recruited as healthy, and their disease onset observed during the follow-up period. We propose a novel cumulative incidence function (CIF) estimator that efficiently incorporates prevalent cases, in contrast to existing methods, providing two advantages: (1) increased efficiency and (2) CIF estimation for ages before the lower limit, c L ⁠ .},
  archive      = {J_BIOMTC},
  author       = {Gorfine, Malka and Zucker, David M and Shoham, Shoval},
  doi          = {10.1093/biomtc/ujaf049},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf049},
  shortjournal = {Biometrics},
  title        = {Cumulative incidence function estimation using population-based biobank data},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precision generalized phase I-II designs. <em>BIOMTC</em>, <em>81</em>(3), ujaf043. (<a href='https://doi.org/10.1093/biomtc/ujaf043'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new family of precision Bayesian dose optimization designs, PGen I-II, based on early efficacy, early toxicity, and long-term time to treatment failure is proposed. A PGen I-II design refines a Gen I-II design by accounting for patient heterogeneity characterized by subgroups that may be defined by prognostic levels, disease subtypes, or biomarker categories. The design makes subgroup-specific decisions, which may be to drop an unacceptably toxic or inefficacious dose, randomize patients among acceptable doses, or identify a best dose in terms of treatment success defined in terms of time to failure over long-term follow-up. A piecewise exponential distribution for failure time is assumed, including subgroup-specific effects of dose, response, and toxicity. Latent variables are used to adaptively cluster subgroups found to have similar dose-outcome distributions, with the model simplified to borrow strength between subgroups in the same cluster. Guidelines and user-friendly computer software for implementing the design are provided. A simulation study is reported that shows the PGen I-II design is superior to similarly structured designs that either assume patient homogeneity or conduct separate trials within subgroups.},
  archive      = {J_BIOMTC},
  author       = {Zhao, Saijun and Thall, Peter F and Yuan, Ying and Lee, Juhee and Msaouel, Pavlos and Zang, Yong},
  doi          = {10.1093/biomtc/ujaf043},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf043},
  shortjournal = {Biometrics},
  title        = {Precision generalized phase I-II designs},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mixture models: Parametric, semiparametric, and new directions by weixin yao and sijia xiang, chapman and Hall/CRC, 2024, ISBN: 9780367481827 https://www.routledge.com/Mixture-models-parametric-semiparametric-and-new-directions/Yao-xiang/p/book/9780367481827. <em>BIOMTC</em>, <em>81</em>(3), ujaf031. (<a href='https://doi.org/10.1093/biomtc/ujaf031'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BIOMTC},
  author       = {Chen, Li-Pang},
  doi          = {10.1093/biomtc/ujaf031},
  journal      = {Biometrics},
  month        = {9},
  number       = {3},
  pages        = {ujaf031},
  shortjournal = {Biometrics},
  title        = {Mixture models: Parametric, semiparametric, and new directions by weixin yao and sijia xiang, chapman and Hall/CRC, 2024, ISBN: 9780367481827 https://www.routledge.com/Mixture-models-parametric-semiparametric-and-new-directions/Yao-xiang/p/book/9780367481827},
  volume       = {81},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
