<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>JTB</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="jtb">JTB - 11</h2>
<ul>
<li><details>
<summary>
(2026). Adaptive dynamic resource allocation can cause tragedy of the commons in plants with nutrient competition. <em>JTB</em>, <em>616</em>, 112279. (<a href='https://doi.org/10.1016/j.jtbi.2025.112279'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plants exhibit plastic responses to the absence or presence of competitors. When competing for soil nutrients, plants often show root overproliferation compared to when they grow without competitors. This excessive investment in roots to acquire more nutrients can reduce reproductive yield (e.g., seed mass), a phenomenon known as the tragedy of the commons (TOC). The mechanisms of this phenomenon have been investigated theoretically, focusing on resource allocation strategies between the aboveground (shoot) and the belowground (roots) parts. The previous studies have primarily considered these strategies in terms of sizes of those parts or static allocation rates to those over the season, overlooking dynamic change of allocation within the season. In this study, we introduced a concept of dynamic resource allocation into the plant competition game and investigate the optimal resource allocation strategy using Pontryagin’s maximum principle. Based on the solutions of schedules, we explored the mechanism causing TOC in nutrient competition. Our findings reveal that plants adopt the singular control (i.e., simultaneous allocation to shoot and root), where the control trajectory is identical regardless of the presence or absence of competitors, although the period of simultaneous allocation become longer in the presence of competitors. This trend associates with increasing the root size and decreasing the shoot size at the end of season in the competitive case. Our analysis demonstrates that TOC in plant nutrient competition arises from differences in the allocation period to roots in the competitive scenario.},
  archive      = {J_JTB},
  author       = {Bo-Moon Kim and Atsushi Yamauchi},
  doi          = {10.1016/j.jtbi.2025.112279},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112279},
  shortjournal = {J. Theor. Biol},
  title        = {Adaptive dynamic resource allocation can cause tragedy of the commons in plants with nutrient competition},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Gaussian process modelling of infectious diseases using the greta software package and GPUs. <em>JTB</em>, <em>616</em>, 112278. (<a href='https://doi.org/10.1016/j.jtbi.2025.112278'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gaussian process are a widely-used statistical tool for conducting non-parametric inference in applied sciences, with many computational packages available to fit to data and predict future observations. We study the use of the Greta software for Bayesian inference to apply Gaussian process regression to spatio-temporal data of infectious disease outbreaks and predict future outbreaks. Greta builds on Tensorflow, making it comparatively easy to take advantage of the significant gain in speed offered by GPUs. In these complex spatio-temporal models, we show a reduction of up to 70% in computational time relative to fitting the same models on CPUs. We show how the choice of covariance kernel impacts the ability to infer spread and extrapolate to unobserved spatial and temporal units. The inference pipeline is applied to weekly incidence data on tuberculosis in the East and West Midlands regions of England over a period of two years.},
  archive      = {J_JTB},
  author       = {Eva Gunn and Nikhil Sengupta and Ben Swallow},
  doi          = {10.1016/j.jtbi.2025.112278},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112278},
  shortjournal = {J. Theor. Biol},
  title        = {Gaussian process modelling of infectious diseases using the greta software package and GPUs},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Effective decoupling of mutations and the resulting loss of biodiversity caused by environmental change. <em>JTB</em>, <em>616</em>, 112277. (<a href='https://doi.org/10.1016/j.jtbi.2025.112277'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many biological populations exhibit diversity in their strategy for survival and reproduction in a given environment, and microbes are an example. We explore the fate of different strategies under sustained environmental change by considering a mathematical model for a large population of asexual organisms. Fitness is a bimodal function of a quantitative trait, with two local optima, separated by a local minimum, i.e., a mixture of stabilising and disruptive selection. The optima represent two locally ‘best’ trait values. We consider regimes where, when the environment is unchanging, the equilibrium distribution of the trait is bimodal. A bimodal trait distribution generally requires, for its existence, mutational coupling between the two peaks, and it indicates two coexisting clones with distinct survival and reproduction strategies. When subject to persistent environmental change, the population adapts by utilising mutations that allow it to track the changing environment. The faster the rate of change of the environment, the larger the effect of the mutations that are utilised. Under persistent environmental change, the distribution of trait values takes two different forms. At low rates of change, the distribution remains bimodal. At higher rates, the distribution becomes unimodal. This loss of a clone/biodiversity is driven by a novel mechanism where environmental change decouples a class of mutations.},
  archive      = {J_JTB},
  author       = {Ruixi Huang and David Waxman},
  doi          = {10.1016/j.jtbi.2025.112277},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112277},
  shortjournal = {J. Theor. Biol},
  title        = {Effective decoupling of mutations and the resulting loss of biodiversity caused by environmental change},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). A kinetic study of multi-substrate uniporters. <em>JTB</em>, <em>616</em>, 112267. (<a href='https://doi.org/10.1016/j.jtbi.2025.112267'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transporters play key roles in regulating the movement of molecules into and out of cells. Uniporters, the simplest class of transporters, use facilitated diffusion to translocate molecules across membranes down their concentration gradient. This process can be affected by the presence of additional substrates in the intra- and extracellular environment, which can either increase the net transport rate of a molecule via trans acceleration or decrease it via competitive inhibition. In this study, we derived mathematical models to describe the net transport rate of uniporters in the presence of multiple extracellular substrates or inhibitors. Analyses of these models identified four possible states for the system when two substrates are present, with two states leading to trans acceleration and the other two states resulting in inhibition. Finally, we found that the relation between kinetic constants that controls the fraction of transporters in the inward-facing open state is responsible for these behaviors. Our theoretical results provide a mathematical framework for understanding the dynamic response of uniporters in the presence of multiple substrates and inhibitors, which could have implications for various processes, from nutrient utilization to metabolic engineering.},
  archive      = {J_JTB},
  author       = {Ana S. de Pereda and Jihyun Park and Lily S. Cheung},
  doi          = {10.1016/j.jtbi.2025.112267},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112267},
  shortjournal = {J. Theor. Biol},
  title        = {A kinetic study of multi-substrate uniporters},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Drug-loaded nanoparticles for cancer therapy: A high-throughput multicellular agent-based modeling study. <em>JTB</em>, <em>616</em>, 112266. (<a href='https://doi.org/10.1016/j.jtbi.2025.112266'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interactions between biological systems and engineered nanomaterials have become an important area of study due to their application in medicine. In particular, the opportunity to apply nanomaterials for cancer diagnosis and treatment presents a challenge due to the complex biology of this disease, which spans multiple time and spatial scales. A systems-level analysis from mathematical modeling and computational simulation to explore the interactions between anticancer drug-loaded nanoparticles (NPs), cells, and tissues, and the associated system parameters and patient response would be of benefit. Although a number of models have explored these interactions in the past, few have focused on simulating individual cell-NP interactions. This study develops a multicellular agent-based model of cancer nanotherapy that simulates NP internalization, drug release within the cell cytoplasm, inheritance of NPs by daughter cells at cell division, cell pharmacodynamic response to intracellular drug levels, and overall drug effect on tumor growth. A large-scale parallel computational framework is used to investigate the impact of pharmacokinetic design parameters (NP internalization rate, NP decay rate, anticancer drug release rate) and therapeutic strategies (NP doses and injection frequency) on tumor growth. In particular, through the exploration of NP inheritance at cell division, the results indicate that cancer treatment may be improved when NPs are inherited at cell division for cytotoxic chemotherapy. Moreover, smaller dose of cytostatic chemotherapy may also improve inhibition of tumor growth when cell division is not completely inhibited. This work suggests that slow delivery by heritable NPs can drive new dimensions of nanotherapy design for more sustained therapeutic response.},
  archive      = {J_JTB},
  author       = {Yafei Wang and John Metzcar and Elmar Bucher and Heber L. Rocha and Vikram Jadhao and Randy Heiland and Hermann B. Frieboes and Paul Macklin},
  doi          = {10.1016/j.jtbi.2025.112266},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112266},
  shortjournal = {J. Theor. Biol},
  title        = {Drug-loaded nanoparticles for cancer therapy: A high-throughput multicellular agent-based modeling study},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Stably encoding phylogenetic trees with folios of leaf addresses. <em>JTB</em>, <em>616</em>, 112265. (<a href='https://doi.org/10.1016/j.jtbi.2025.112265'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As genome sequencing data continue to expand, a persistent research challenge is to accommodate the growth of a phylogeny. This situation arises in molecular epidemiology, for example, where new taxonomic groups can appear in real time as pathogen isolates are sequenced. Efficient computational methods have been developed to place new leaves in existing trees, which removes the need to reconstruct trees from scratch. But for these tree extensions to be fully integrated with classification schemes requires a stable encoding of trees that keeps existing tree structures intact as new branches appear. Here, we propose a tree encoding, which we call a folio , that records the path from a reference vertex to each leaf, giving each leaf an address . We present a simple set of rules to assign new addresses to added leaves. The encoding is stable in the sense that it does not change as further leaf addresses are added to the folio. The tree can be uniquely recovered from a folio of addresses. We illustrate the methods using Salmonella genome data. Due to the properties of our encoding framework, we anticipate that it can be used for a range of different phylogenetic analyses.},
  archive      = {J_JTB},
  author       = {Mark M. Tanaka and Ruiting Lan and Andrew R. Francis},
  doi          = {10.1016/j.jtbi.2025.112265},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112265},
  shortjournal = {J. Theor. Biol},
  title        = {Stably encoding phylogenetic trees with folios of leaf addresses},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Phenomenological modeling of gene transcription by approximating cooperativity of transcription factors improves prediction and reduces complexity in gene regulatory network models. <em>JTB</em>, <em>616</em>, 112264. (<a href='https://doi.org/10.1016/j.jtbi.2025.112264'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several computational models are available for representing the gene expression process, with each having their advantages and disadvantages. Phenomenological models are widely used as they make appropriate simplifications that aim to find a middle ground between accuracy and complexity. The existing phenomenological models compete in terms of how the transcription initiation process is approximated, to achieve high accuracy while having the lowest complexity possible. However, most current models still suffer from high parameter complexity in the case of complex promoters. Herein, we formally derive a phenomenological approach to model RNA polymerase recruitment, stating approximations on cooperativity between transcription factors that are applicable to promoters requiring multifactorial input, which reduces parameter complexity. We then apply this method to biologically relevant networks of varying complexities to show that the approximations improved predictive ability compared to existing models. In summary, our reduced parameter model (RPM) had lower complexity while maintaining high accuracy, which leads to better scalability for complex networks.},
  archive      = {J_JTB},
  author       = {Thiruvickraman Jothiprakasam and Siddharth Jhunjhunwala},
  doi          = {10.1016/j.jtbi.2025.112264},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112264},
  shortjournal = {J. Theor. Biol},
  title        = {Phenomenological modeling of gene transcription by approximating cooperativity of transcription factors improves prediction and reduces complexity in gene regulatory network models},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Mathematical model suggests current CAR-macrophage dosage is efficient to low pre-infusion tumour burden but refractory to high tumour burden. <em>JTB</em>, <em>616</em>, 112263. (<a href='https://doi.org/10.1016/j.jtbi.2025.112263'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor (CAR)-macrophage therapy is a promising approach for tumour treatment due to antigen-specific phagocytosis and tumour clearance. However, the precise impact of tumour burden, dose and dosing regimens on therapeutic outcomes remains poorly understood. We developed ordinary differential equation (ODE) mathematical modelling and utilised parameter inference to analyse in vitro FACS-based phagocytosis assay data testing CD19-positive Raji tumour cell against CAR-macrophage, and revealed that phagocytosing efficiency of CAR-macrophage increases but saturates as both Raji cell and CAR-macrophage concentrations increase. This interaction resulted in bistable Raji cell kinetics; specifically, within a particular range of CAR-macrophage concentration, low tumour burdens are effectively inhibited, while high tumour burdens remain refractory. Furthermore, our model predicted that CAR-macrophage dosages typically suggested by current clinical trials yield favourable therapeutic outcomes only when tumour burden is low. For split CAR-macrophage infusion with fixed total dosage, the first infusion with high CAR-macrophage dose delivers superior treatment outcomes. Finally, we identified alternative infusion regimens: five billion cells administered monthly for three months, or seven billion cells every two months for six months, can efficiently suppress Raji cell replication irrespective of tumour burden. Our findings highlight CAR-macrophage therapeutic outcomes are strongly influenced by both tumour burden and different dosing regimens. This work underscores that reducing tumour burden, increasing CAR-macrophage dose in the first infusion and prolonging CAR-macrophage persistence are key strategies for achieving durable responses.},
  archive      = {J_JTB},
  author       = {Shilian Xu and Maoxuan Liu},
  doi          = {10.1016/j.jtbi.2025.112263},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112263},
  shortjournal = {J. Theor. Biol},
  title        = {Mathematical model suggests current CAR-macrophage dosage is efficient to low pre-infusion tumour burden but refractory to high tumour burden},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Emergent microtubule properties in a model of filament turnover and nucleation. <em>JTB</em>, <em>616</em>, 112254. (<a href='https://doi.org/10.1016/j.jtbi.2025.112254'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microtubules (MTs) are dynamic protein filaments essential for intracellular organization and transport, particularly in long-lived cells such as neurons. The plus and minus ends of neuronal MTs switch between growth and shrinking phases, and the nucleation of new filaments is believed to be regulated in both healthy and injury conditions. We propose stochastic and deterministic mathematical models to investigate the impact of filament nucleation and length-regulation mechanisms on emergent properties such as MT lengths and numbers in living cells. We expand our stochastic continuous-time Markov chain model of filament dynamics to incorporate MT nucleation and capture realistic stochastic fluctuations in MT numbers and tubulin availability. We also propose a simplified partial differential equation (PDE) model, which allows for tractable analytical investigation into steady-state MT distributions under different nucleation and length-regulating mechanisms. We find that the stochastic and PDE modeling approaches show good agreement in MT length distributions, and that both MT nucleation and the catastrophe rate of large-length MTs regulate MT length distributions. In both frameworks, multiple mechanistic combinations achieve the same average MT length. The models proposed can predict parameter regimes where the system is scarce in tubulin, the building block of MTs, and suggest that low filament nucleation regimes are characterized by high variation in MT lengths, while high nucleation regimes drive high variation in MT numbers. These mathematical frameworks have the potential to improve our understanding of MT regulation in both healthy and injured neurons.},
  archive      = {J_JTB},
  author       = {Anna C. Nelson and Scott A. McKinley and Melissa M. Rolls and Maria-Veronica Ciocanel},
  doi          = {10.1016/j.jtbi.2025.112254},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112254},
  shortjournal = {J. Theor. Biol},
  title        = {Emergent microtubule properties in a model of filament turnover and nucleation},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Modelling phylogeny in 16S rRNA gene sequencing datasets using string-based kernels. <em>JTB</em>, <em>616</em>, 112249. (<a href='https://doi.org/10.1016/j.jtbi.2025.112249'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The bacterial microbiome is increasingly being recognised as a key factor in human health, driven in large part by datasets collected using 16S rRNA (ribosomal ribonucleic acid) gene sequencing, which enable cost-effective quantification of the composition of an individual’s bacterial community. One of the defining characteristics of 16S rRNA datasets is the evolutionary relationships that exist between taxa (phylogeny). Here, we demonstrate the utility of modelling these phylogenetic relationships in two statistical tasks (the two sample test and host trait prediction) and propose a novel family of kernels for analysing microbiome datasets by leveraging string kernels from the natural language processing literature. We show via simulation studies that a kernel two-sample test using the proposed kernel is sensitive to the phylogenetic scale of the difference between the two populations. In a second set of simulations we also show how Gaussian process modelling with string kernels can infer the distribution of bacterial-host effects across the phylogenetic tree and apply this approach to a real host-trait prediction task. The results in the paper can be reproduced by running the code at https://github.com/jonathanishhorowicz/modelling_phylogeny_in_16srrna_using_string_kernels .},
  archive      = {J_JTB},
  author       = {Jonathan Ish-Horowicz and Sarah Filippi},
  doi          = {10.1016/j.jtbi.2025.112249},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112249},
  shortjournal = {J. Theor. Biol},
  title        = {Modelling phylogeny in 16S rRNA gene sequencing datasets using string-based kernels},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
<li><details>
<summary>
(2026). Approximate bayesian computation for markovian binary trees in phylogenetics. <em>JTB</em>, <em>616</em>, 112246. (<a href='https://doi.org/10.1016/j.jtbi.2025.112246'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phylogenetic trees describe the relationships between species in the evolutionary process, and provide information about the rates of diversification. To understand the mechanisms behind macroevolution, we consider a class of multitype branching processes called Markovian binary trees (MBTs). MBTs allow for trait-based variation in diversification rates, and provide a flexible and realistic probabilistic model for phylogenetic trees. We develop an approximate Bayesian computation (ABC) scheme to infer the rates of MBT parameters by exploiting the information in the shapes of phylogenetic trees. We evaluate the accuracy of this inference method using simulation studies, and find that our method is able to detect variation in the diversification rates, with accuracy comparable to, and generally better than, likelihood-based methods. In an application to a real-life phylogeny of squamata, we reinforce conclusions drawn from earlier studies, in particular supporting the existence of ovi-/viviparity transitions in both directions. Our method demonstrates the potential for more complex models of evolution to be employed in phylogenetic inference, in conjunction with likelihood-free schemes.},
  archive      = {J_JTB},
  author       = {Mingqi He and Sophie Hautphenne and Yao-ban Chan},
  doi          = {10.1016/j.jtbi.2025.112246},
  journal      = {Journal of Theoretical Biology},
  month        = {1},
  pages        = {112246},
  shortjournal = {J. Theor. Biol},
  title        = {Approximate bayesian computation for markovian binary trees in phylogenetics},
  volume       = {616},
  year         = {2026},
}
</textarea>
</details></li>
</ul>

</body>
</html>
