<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>nature</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs">NATCS - 20</h2>
<ul>
<li><details>
<summary>
(2025). Confidential computing for population-scale genome-wide association studies with SECRET-GWAS. <em>NATCS</em>, <em>5</em>(9), 825-835. (<a href='https://doi.org/10.1038/s43588-025-00856-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic data from a single institution lacks global diversity representation, especially for rare variants and diseases. Confidential computing can enable collaborative genome-wide association studies (GWAS) without compromising privacy or accuracy. However, due to limited secure memory space and performance overheads, previous solutions fail to support widely used regression methods. Here we present SECRET-GWAS—a rapid, privacy-preserving, population-scale, collaborative GWAS tool. We discuss several system optimizations, including streaming, batching, data parallelization and reducing trusted hardware overheads to efficiently scale linear and logistic regression to over a thousand processor cores on an Intel SGX-based cloud platform. In addition, we protect SECRET-GWAS against several hardware side-channel attacks. SECRET-GWAS is an open-source tool and works with the widely used Hail genomic analysis framework. Our experiments on Azure’s Confidential Computing platform demonstrate that SECRET-GWAS enables multivariate linear and logistic regression GWAS queries on population-scale datasets from ten independent sources in just 4.5 and 29 minutes, respectively. Secure collaborative genome-wide association studies (GWAS) with population-scale datasets address gaps in genomic data. This work proposes SECRET-GWAS and system optimizations that overcome resource constraints and exploit parallelism, while maintaining privacy and accuracy.},
  archive      = {J_NATCS},
  author       = {Rosenblum, Jonah and Dong, Juechu and Narayanasamy, Satish},
  doi          = {10.1038/s43588-025-00856-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {825-835},
  shortjournal = {Nat. Comput. Sci},
  title        = {Confidential computing for population-scale genome-wide association studies with SECRET-GWAS},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analog in-memory computing attention mechanism for fast and energy-efficient large language models. <em>NATCS</em>, <em>5</em>(9), 813-824. (<a href='https://doi.org/10.1038/s43588-025-00854-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transformer networks, driven by self-attention, are central to large language models. In generative transformers, self-attention uses cache memory to store token projections, avoiding recomputation at each time step. However, graphics processing unit (GPU)-stored projections must be loaded into static random-access memory for each new generation step, causing latency and energy bottlenecks. Here we present a custom self-attention in-memory computing architecture based on emerging charge-based memories called gain cells, which can be efficiently written to store new tokens during sequence generation and enable parallel analog dot-product computation required for self-attention. However, the analog gain-cell circuits introduce non-idealities and constraints preventing the direct mapping of pre-trained models. To circumvent this problem, we design an initialization algorithm achieving text-processing performance comparable to GPT-2 without training from scratch. Our architecture reduces attention latency and energy consumption by up to two and four orders of magnitude, respectively, compared with GPUs, marking a substantial step toward ultrafast, low-power generative transformers. Leveraging in-memory computing with emerging gain-cell devices, the authors accelerate attention—a core mechanism in large language models. They train a 1.5-billion-parameter model, achieving up to a 70,000-fold reduction in energy consumption and a 100-fold speed-up compared with GPUs.},
  archive      = {J_NATCS},
  author       = {Leroux, Nathan and Manea, Paul-Philipp and Sudarshan, Chirag and Finkbeiner, Jan and Siegel, Sebastian and Strachan, John Paul and Neftci, Emre},
  doi          = {10.1038/s43588-025-00854-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {813-824},
  shortjournal = {Nat. Comput. Sci},
  title        = {Analog in-memory computing attention mechanism for fast and energy-efficient large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep active optimization for complex systems. <em>NATCS</em>, <em>5</em>(9), 801-812. (<a href='https://doi.org/10.1038/s43588-025-00858-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inferring optimal solutions from limited data is considered the ultimate goal in scientific discovery. Artificial intelligence offers a promising avenue to greatly accelerate this process. Existing methods often depend on large datasets, strong assumptions about objective functions, and classic machine learning techniques, restricting their effectiveness to low-dimensional or data-rich problems. Here we introduce an optimization pipeline that can effectively tackle complex, high-dimensional problems with limited data. This approach utilizes a deep neural surrogate to iteratively find optimal solutions and introduces additional mechanisms to avoid local optima, thereby minimizing the required samples. Our method finds superior solutions in problems with up to 2,000 dimensions, whereas existing approaches are confined to 100 dimensions and need considerably more data. It excels across varied real-world systems, outperforming current algorithms and enabling efficient knowledge discovery. Although focused on scientific problems, its benefits extend to numerous quantitative fields, paving the way for advanced self-driving laboratories. This study introduces a deep active optimization pipeline that effectively tackles high-dimensional, complex problems with limited data. The approach minimizes sample size and surpasses existing methods, achieving optimal solutions in up to 2,000 dimensions.},
  archive      = {J_NATCS},
  author       = {Wei, Ye and Peng, Bo and Xie, Ruiwen and Chen, Yangtao and Qin, Yu and Wen, Peng and Bauer, Stefan and Tung, Po-Yen and Raabe, Dierk},
  doi          = {10.1038/s43588-025-00858-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {801-812},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep active optimization for complex systems},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Digital twin for chemical science: A case study on water interactions on the ag(111) surface. <em>NATCS</em>, <em>5</em>(9), 793-800. (<a href='https://doi.org/10.1038/s43588-025-00857-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directly visualizing chemical trajectories offers insights into catalysis, gas-phase reactions and photoinduced dynamics. Tracking the transformation of chemical species is best achieved by coupling theory and experiment. Here we developed Digital Twin for Chemical Science (DTCS) v.01, which integrates theory, experiment and their bidirectional feedback loops into a unified platform for chemical characterization. DTCS addresses a core question: given a set of experimental conditions, what is the expected outcome and why? It consists of a forward solver that takes a chemical reaction network and predicts spectra under experimental conditions, and an inverse solver that infers kinetics from measured spectra. We applied DTCS to ambient-pressure X-ray photoelectron spectroscopy measurements of the Ag–H2O interface as an example. This approach enables real-time knowledge extraction and guides experiments until a stopping condition is met based on accuracy and degeneracy. As a step toward autonomous chemical characterization, DTCS provides mechanistic knowledge in a verified, standardized manner. Interpreting spectroscopic data in real time remains a challenge in chemical characterization. Here a digital twin framework is developed that links first-principles theory and experimental data via a bidirectional feedback loop, enabling on-the-fly decision-making and insights into reaction mechanisms based on measured spectra during chemical experiments.},
  archive      = {J_NATCS},
  author       = {Qian, Jin and Jana, Asmita and Menon, Siddarth and Bogdan, Andrew E. and Hamlyn, Rebecca and Mahl, Johannes and Crumlin, Ethan J.},
  doi          = {10.1038/s43588-025-00857-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {793-800},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digital twin for chemical science: A case study on water interactions on the ag(111) surface},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SurFF: A foundation model for surface exposure and morphology across intermetallic crystals. <em>NATCS</em>, <em>5</em>(9), 782-792. (<a href='https://doi.org/10.1038/s43588-025-00839-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With approximately 90% of industrial reactions occurring on surfaces, the role of heterogeneous catalysts is paramount. Currently, accurate surface exposure prediction is vital for heterogeneous catalyst design, but it is hindered by the high costs of experimental and computational methods. Here we introduce a foundation force-field-based model for predicting surface exposure and synthesizability (SurFF) across intermetallic crystals, which are essential materials for heterogeneous catalysts. We created a comprehensive intermetallic surface database using an active learning method and high-throughput density functional theory calculations, encompassing 12,553 unique surfaces and 344,200 single points. SurFF achieves density-functional-theory-level precision with a prediction error of 3 meV Å−2 and enables large-scale surface exposure prediction with a 105-fold acceleration. Validation against computational and experimental data both show strong alignment. We applied SurFF for large-scale predictions of surface energy and Wulff shapes for over 6,000 intermetallic crystals, providing valuable data for the community. A foundation machine learning model, SurFF, enables DFT-accurate predictions of surface energies and morphologies in intermetallic catalysts, achieving over 105-fold acceleration for high-throughput materials screening.},
  archive      = {J_NATCS},
  author       = {Yin, Jun and Chen, Honghao and Qiu, Jiangjie and Li, Wentao and He, Peng and Li, Jiali and Karimi, Iftekhar A. and Lan, Xiaocheng and Wang, Tiefeng and Wang, Xiaonan},
  doi          = {10.1038/s43588-025-00839-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {782-792},
  shortjournal = {Nat. Comput. Sci},
  title        = {SurFF: A foundation model for surface exposure and morphology across intermetallic crystals},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Exploiting pleiotropy to enhance variant discovery with functional false discovery rates. <em>NATCS</em>, <em>5</em>(9), 769-781. (<a href='https://doi.org/10.1038/s43588-025-00852-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cost of recruiting participants for genome-wide association studies (GWASs) can limit sample sizes and hinder the discovery of genetic variants. Here we introduce the surrogate functional false discovery rate (sfFDR) framework that integrates summary statistics of related traits to increase power. The sfFDR framework provides estimates of FDR quantities such as the functional local FDR and q value, and uses these estimates to derive a functional P value for type I error rate control and a functional local Bayes’ factor for post-GWAS analyses. Compared with a standard analysis, sfFDR substantially increased power (equivalent to a 52% increase in sample size) in a study of obesity-related traits from the UK Biobank and discovered eight additional lead SNPs near genes linked to immune-related responses in a rare disease GWAS of eosinophilic granulomatosis with polyangiitis. Collectively, these results highlight the utility of exploiting related traits in both small and large studies. This study introduces a cost-effective strategy called surrogate functional false discovery rates to increase power in genome-wide association studies by leveraging genetic correlations (or pleiotropy) between related traits.},
  archive      = {J_NATCS},
  author       = {Bass, Andrew J. and Wallace, Chris},
  doi          = {10.1038/s43588-025-00852-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {769-781},
  shortjournal = {Nat. Comput. Sci},
  title        = {Exploiting pleiotropy to enhance variant discovery with functional false discovery rates},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Virtual brain twins for stimulation in epilepsy. <em>NATCS</em>, <em>5</em>(9), 754-768. (<a href='https://doi.org/10.1038/s43588-025-00841-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimating the epileptogenic zone network (EZN) is an important part of the diagnosis of drug-resistant focal epilepsy and has a pivotal role in treatment and intervention. Virtual brain twins provide a modeling method for personalized diagnosis and treatment. They integrate patient-specific brain topography with structural connectivity from anatomical neuroimaging such as magnetic resonance imaging, and dynamic activity from functional recordings such as electroencephalography (EEG) and stereo-EEG (SEEG). Seizures show rich spatial and temporal features in functional recordings, which can be exploited to estimate the EZN. Stimulation-induced seizures can provide important and complementary information. Here we consider invasive SEEG stimulation and non-invasive temporal interference stimulation as a complementary approach. This paper offers a high-resolution virtual brain twin framework for EZN diagnosis based on stimulation-induced seizures. It provides an important methodological and conceptual basis to make the transition from invasive to non-invasive diagnosis and treatment of drug-resistant focal epilepsy. A high-resolution virtual brain twin approach is proposed using stimulation-induced seizures to estimate the epileptogenic network, offering a step toward non-invasive diagnosis and treatment of drug-resistant focal epilepsy.},
  archive      = {J_NATCS},
  author       = {Wang, Huifang E. and Dollomaja, Borana and Triebkorn, Paul and Duma, Gian Marco and Williamson, Adam and Makhalova, Julia and Lemarechal, Jean-Didier and Bartolomei, Fabrice and Jirsa, Viktor},
  doi          = {10.1038/s43588-025-00841-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {754-768},
  shortjournal = {Nat. Comput. Sci},
  title        = {Virtual brain twins for stimulation in epilepsy},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On the compatibility of generative AI and generative linguistics. <em>NATCS</em>, <em>5</em>(9), 745-753. (<a href='https://doi.org/10.1038/s43588-025-00861-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chomsky’s generative linguistics has made substantial contributions to cognitive science and symbolic artificial intelligence. With the rise of neural language models, however, the compatibility between generative artificial intelligence and generative linguistics has come under debate. Here we outline three ways in which generative artificial intelligence aligns with and supports the core ideas of generative linguistics. In turn, generative linguistics can provide criteria to evaluate and improve neural language models as models of human language and cognition. This Perspective discusses that generative AI aligns with generative linguistics by showing that neural language models (NLMs) are formal generative models. Furthermore, generative linguistics offers a framework for evaluating and improving NLMs.},
  archive      = {J_NATCS},
  author       = {Portelance, Eva and Jasbi, Masoud},
  doi          = {10.1038/s43588-025-00861-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {745-753},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the compatibility of generative AI and generative linguistics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Arti-‘fickle’ intelligence: Using LLMs as a tool for inference in the political and social sciences. <em>NATCS</em>, <em>5</em>(9), 737-744. (<a href='https://doi.org/10.1038/s43588-025-00843-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To promote the scientific use of large language models (LLMs), we suggest that researchers in the political and social sciences refocus on the scientific goal of inference. We suggest that this refocus will improve the accumulation of shared scientific knowledge about these tools and their uses in the social sciences. We discuss the challenges and opportunities related to scientific inference with LLMs, using validation of model output as an illustrative case for discussion. We then propose a set of guidelines related to establishing the failure and success of LLMs when completing particular tasks and discuss how to make inferences from these observations. Large language models are increasingly important in social science research. The authors provide guidance on how best to validate and use these models as rigorous tools to further scientific inference.},
  archive      = {J_NATCS},
  author       = {Argyle, Lisa P. and Busby, Ethan C. and Gubler, Joshua R. and Hepner, Bryce and Lyman, Alex and Wingate, David},
  doi          = {10.1038/s43588-025-00843-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {737-744},
  shortjournal = {Nat. Comput. Sci},
  title        = {Arti-‘fickle’ intelligence: Using LLMs as a tool for inference in the political and social sciences},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Urban planning in the era of large language models. <em>NATCS</em>, <em>5</em>(9), 727-736. (<a href='https://doi.org/10.1038/s43588-025-00846-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {City plans are the product of integrating human creativity with emerging technologies, which continuously evolve and reshape urban morphology and environments. Here we argue that large language models hold large untapped potential in addressing the growing complexities of urban planning and enabling a more holistic, innovative and responsive approach to city design. By harnessing their advanced generation and simulation capabilities, large language models can contribute as an intelligent assistant for human planners in synthesizing conceptual ideas, generating urban designs and evaluating the outcomes of planning efforts. Large language models remain largely unexplored is the design of cities. In this Perspective, the authors discuss the potential opportunities brought by these models in assisting urban planning.},
  archive      = {J_NATCS},
  author       = {Zheng, Yu and Xu, Fengli and Lin, Yuming and Santi, Paolo and Ratti, Carlo and Wang, Qi R. and Li, Yong},
  doi          = {10.1038/s43588-025-00846-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {727-736},
  shortjournal = {Nat. Comput. Sci},
  title        = {Urban planning in the era of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rethinking chemical research in the age of large language models. <em>NATCS</em>, <em>5</em>(9), 715-726. (<a href='https://doi.org/10.1038/s43588-025-00811-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) offer opportunities for advancing chemical research, including planning, optimization, data analysis, automation and knowledge management. Deploying LLMs in active environments, where they interact with tools and data, can greatly enhance their capabilities. However, challenges remain in evaluating their performance and addressing ethical issues such as reproducibility, data privacy and bias. Here we discuss ongoing and potential integrations of LLMs in chemical research, highlighting existing challenges to guide the effective use of LLMs as active scientific partners. This Perspective highlights the potential integrations of large language models (LLMs) in chemical research and provides guidance on the effective use of LLMs as research partners, noting the ethical and performance-based challenges that must be addressed moving forward.},
  archive      = {J_NATCS},
  author       = {MacKnight, Robert and Boiko, Daniil A. and Regio, Jose Emilio and Gallegos, Liliana C. and Neukomm, Théo A. and Gomes, Gabe},
  doi          = {10.1038/s43588-025-00811-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {715-726},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rethinking chemical research in the age of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A digital twin that interprets and refines chemical mechanisms. <em>NATCS</em>, <em>5</em>(9), 713-714. (<a href='https://doi.org/10.1038/s43588-025-00859-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An integrated platform, Digital Twin for Chemical Science (DTCS), is developed to connect first-principles theory with spectroscopic measurements through a bidirectional feedback loop. By predicting and refining chemical reaction mechanisms before, during and after experiments, DTCS enables the interpretation of spectra and supports real-time decision-making in chemical characterization.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00859-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {713-714},
  shortjournal = {Nat. Comput. Sci},
  title        = {A digital twin that interprets and refines chemical mechanisms},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Overcoming computational bottlenecks in large language models through analog in-memory computing. <em>NATCS</em>, <em>5</em>(9), 711-712. (<a href='https://doi.org/10.1038/s43588-025-00860-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study demonstrates the potential of using in-memory computing architecture for implementing large language models for an improved computational efficiency in both time and energy while maintaining a high accuracy.},
  archive      = {J_NATCS},
  author       = {Lin, Yudeng and Tang, Jianshi},
  doi          = {10.1038/s43588-025-00860-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {711-712},
  shortjournal = {Nat. Comput. Sci},
  title        = {Overcoming computational bottlenecks in large language models through analog in-memory computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Neuromorphic principles in self-attention hardware for efficient transformers. <em>NATCS</em>, <em>5</em>(9), 708-710. (<a href='https://doi.org/10.1038/s43588-025-00868-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strong barriers remain between neuromorphic engineering and machine learning, especially with regard to recent large language models (LLMs) and transformers. This Comment makes the case that neuromorphic engineering may hold the keys to more efficient inference with transformer-like models.},
  archive      = {J_NATCS},
  author       = {Leroux, Nathan and Finkbeiner, Jan and Neftci, Emre},
  doi          = {10.1038/s43588-025-00868-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {708-710},
  shortjournal = {Nat. Comput. Sci},
  title        = {Neuromorphic principles in self-attention hardware for efficient transformers},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Using LLMs to advance the cognitive science of collectives. <em>NATCS</em>, <em>5</em>(9), 704-707. (<a href='https://doi.org/10.1038/s43588-025-00848-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) are already transforming the study of individual cognition, but their application to studying collective cognition has been underexplored. We lay out how LLMs may be able to address the complexity that has hindered the study of collectives and raise possible risks that warrant new methods.},
  archive      = {J_NATCS},
  author       = {Sucholutsky, Ilia and Collins, Katherine M. and Jacoby, Nori and Thompson, Bill D. and Hawkins, Robert D.},
  doi          = {10.1038/s43588-025-00848-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {704-707},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using LLMs to advance the cognitive science of collectives},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Threats to scientific software from over-reliance on AI code assistants. <em>NATCS</em>, <em>5</em>(9), 701-703. (<a href='https://doi.org/10.1038/s43588-025-00845-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The adoption of generative artificial intelligence (AI) code assistants in scientific software development is promising, but user studies across an array of programming contexts suggest that programmers are at risk of over-reliance on these tools, leading them to accept undetected errors in generated code. Scientific software may be particularly vulnerable to such errors because most research code is untested and scientists are undertrained in software development skills. This Comment outlines the factors that place scientific code at risk and suggests directions for research groups, educators, publishers and funders to counter these liabilities.},
  archive      = {J_NATCS},
  author       = {O’Brien, Gabrielle},
  doi          = {10.1038/s43588-025-00845-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {701-703},
  shortjournal = {Nat. Comput. Sci},
  title        = {Threats to scientific software from over-reliance on AI code assistants},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational challenges arising in algorithmic fairness and health equity with generative AI. <em>NATCS</em>, <em>5</em>(9), 698-700. (<a href='https://doi.org/10.1038/s43588-025-00806-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of generative artificial intelligence (AI) in healthcare is advancing, but understanding its potential challenges for fairness and health equity is still in its early stages. This Comment investigates how to define fairness and measure it, and highlights research that can help address challenges in the field.},
  archive      = {J_NATCS},
  author       = {Suriyakumar, Vinith M. and Zink, Anna and Hightower, Maia and Ghassemi, Marzyeh and Beaulieu-Jones, Brett},
  doi          = {10.1038/s43588-025-00806-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {698-700},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational challenges arising in algorithmic fairness and health equity with generative AI},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The impact of language models on the humanities and vice versa. <em>NATCS</em>, <em>5</em>(9), 695-697. (<a href='https://doi.org/10.1038/s43588-025-00819-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many humanists are skeptical of language models and concerned about their effects on universities. However, researchers with a background in the humanities are also actively engaging with artificial intelligence — seeking not only to adopt language models as tools, but to steer them toward a more flexible, contextual representation of written culture.},
  archive      = {J_NATCS},
  author       = {Underwood, Ted},
  doi          = {10.1038/s43588-025-00819-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {695-697},
  shortjournal = {Nat. Comput. Sci},
  title        = {The impact of language models on the humanities and vice versa},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The other AI revolution: How the global south is building and repurposing language models that speak to billions. <em>NATCS</em>, <em>5</em>(9), 691-694. (<a href='https://doi.org/10.1038/s43588-025-00865-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While leading tech companies race to build ever-larger models, researchers in Brazil, India and Africa are using clever tricks to remix big labs’ LLMs to bring AI to billions of users.},
  archive      = {J_NATCS},
  author       = {Burgos, Pedro},
  doi          = {10.1038/s43588-025-00865-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {691-694},
  shortjournal = {Nat. Comput. Sci},
  title        = {The other AI revolution: How the global south is building and repurposing language models that speak to billions},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The rise of large language models. <em>NATCS</em>, <em>5</em>(9), 689-690. (<a href='https://doi.org/10.1038/s43588-025-00890-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This issue of Nature Computational Science features a Focus that highlights both the promises and perils of large language models, their emerging applications across diverse scientific domains, and the opportunities to overcome the challenges that lie ahead.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00890-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {689-690},
  shortjournal = {Nat. Comput. Sci},
  title        = {The rise of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natee">NATEE - 33</h2>
<ul>
<li><details>
<summary>
(2025). Citrus reticulata (mandarin). <em>NATEE</em>, <em>9</em>(9), 1752. (<a href='https://doi.org/10.1038/s41559-025-02822-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Qiang Xu has been captivated by citrus fruits since childhood.},
  archive      = {J_NATEE},
  author       = {Xu, Qiang},
  doi          = {10.1038/s41559-025-02822-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1752},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Citrus reticulata (mandarin)},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease. <em>NATEE</em>, <em>9</em>(9), 1739-1751. (<a href='https://doi.org/10.1038/s41559-025-02797-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More than 10 years following the onset of the sea star wasting disease (SSWD) epidemic, affecting over 20 asteroid species from Mexico to Alaska, the causative agent has been elusive. SSWD killed billions of the most susceptible species, sunflower sea stars (Pycnopodia helianthoides), initiating a trophic cascade involving unchecked urchin population growth and the widespread loss of kelp forests. Identifying the causative agent underpins the development of recovery strategies. Here we induced disease and subsequent mortality in exposure experiments using tissue extracts, coelomic fluid and effluent water from wasting sunflower sea stars, with no mortality in controls. Deep sequencing of diseased sea star coelomic fluid samples from experiments and field outbreaks revealed a dominant proportion of reads assigned to the bacterium Vibrio pectenicida. Fulfilling Koch’s postulates, V. pectenicida strain FHCF-3, cultured from the coelomic fluid of a diseased sunflower sea star, caused disease and mortality in exposed sunflower sea stars, demonstrating that it is a causative agent of SSWD. This discovery will enable recovery efforts for sea stars and the ecosystems affected by their decline by facilitating culture-based experimental research and broad-scale screening for pathogen presence and abundance in the laboratory and field. The causative agent of sea star wasting disease has been elusive. This study used genetic datasets and experimental exposures to demonstrate that a strain of the bacterium Vibrio pectenicida caused disease and mortality in sea stars.},
  archive      = {J_NATEE},
  author       = {Prentice, Melanie B. and Crandall, Grace A. and Chan, Amy M. and Davis, Katherine M. and Hershberger, Paul K. and Finke, Jan F. and Hodin, Jason and McCracken, Andrew and Kellogg, Colleen T. E. and Clemente-Carvalho, Rute B. G. and Prentice, Carolyn and Zhong, Kevin X. and Harvell, C. Drew and Suttle, Curtis A. and Gehman, Alyssa-Lois M.},
  doi          = {10.1038/s41559-025-02797-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1739-1751},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories. <em>NATEE</em>, <em>9</em>(9), 1731-1738. (<a href='https://doi.org/10.1038/s41559-025-02798-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Radiogenic strontium isotopes (87Sr/86Sr) and the alkaline earth ratios (AERs) Sr/Ca and Ba/Ca in fossil dental enamel can inform the habitat, residence and life histories of early hominins recovered from the Pleistocene cradle-of-humankind sites of Gauteng, South Africa. Key questions, which may be addressed with these indices, are the relative exploitation of wet versus dry botanic regimes and whether early hominins dispersed in a manner similar to that of chimpanzees (characterized by male philopatry and female dispersal at puberty) or to that of humans (who are not so characterized). Here we developed 28 new dental chronologies in 20 Paranthropus robustus teeth from Swartkrans and Kromdraai. Resulting geochemical time series demonstrate that, while maternal 87Sr/86Sr in earlier-forming teeth varies widely, third molar 87Sr/86Sr, derived from postweaning solid foods, progressively converges to 0.7306 ± 0.0035 (± 2 s.d.), which we express as the local isotopically delineated exploitation area (LIDEA). The spatial resolution of LIDEA is determined using a bioavailable 87Sr/86Sr isoscape. In this environmental context, we interpret LIDEA as a quantifiable signal indicating eurytopy (generalization), with some 30% of Sr deriving from riparian woodland habitats. With regard to residence, many individuals arrived at the site after second molar mineralization, while some matured locally, demonstrating both male and female dispersal as well as lifelong local residence. Analysis of both 87Sr/86Sr and the AERs further highlights concomitant patterns, as well as numerous periodicities that may be related to resource depletion, seasonality or lunar cycles. Strontium analysis of Paranthropus robustus teeth from two South African sites reveals that the Pleistocene hominin probably exploited both savanna and riparian woodland habitats, had a home-range size compatible with that of present-day hunter-gatherers and had periodic ranging patterns. While some individuals lived their lives in the same place, others dispersed during their lifetime.},
  archive      = {J_NATEE},
  author       = {Sillen, Andrew and Dean, Christopher and Balter, Vincent},
  doi          = {10.1038/s41559-025-02798-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1731-1738},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas. <em>NATEE</em>, <em>9</em>(9), 1719-1730. (<a href='https://doi.org/10.1038/s41559-025-02794-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most faunas from the Mesozoic era were dominated by sauropod dinosaurs, the largest terrestrial animals to ever exist. These megaherbivores were remarkably diverse and widely distributed. Here we study three Late Jurassic faunas from the USA, Portugal and Tanzania, each approximately 150 million years old, which are known for their extreme sauropod diversity. Whereas general taxonomic composition was similar in these three faunas, the major clades differed in relative abundance. Moreover, their depositional strata record distinct climatic regimes. Using dental microwear texture analysis, we investigated the impact of these climate regimes and the resulting food availability on the different sauropod taxa. Wear patterns in camarasaurid macronarians show minimal variation across different climate regimes, supporting previous studies suggesting that these animals migrated to follow their preferred climate niche and food source. North American camarasaurids show similar wear patterns to those of Portuguese turiasaurs, another broad-crowned taxon, which did not exist in the Jurassic of North America. By contrast, where camarasaurids and turiasaurs co-occurred in Portugal, their microwear patterns are distinct, suggesting niche differentiation to avoid ecological competition between these two clades. Flagellicaudatan diplodocoids display highly variable wear patterns, indicating limited migration (and therefore seasonal variation in diet), which aligns with observed biogeography patterns in the USA. Early-branching titanosauriforms show highly distinct wear patterns between different climate regimes, which can probably be attributed to different abrasive loads in the respective habitats. Our results demonstrate that dental microwear texture analysis not only records dietary preferences but also reveals behaviour such as competition and migration related to dietary niches in past ecosystems. The authors compare tooth wear patterns from three Late Jurassic sauropod faunas in North America, Portugal and Tanzania and find that different regions and climate regimes seem to have had distinctive impacts on dental wear of megaherbivores in these areas, revealing habitat and ecological niche information.},
  archive      = {J_NATEE},
  author       = {Winkler, Daniela E. and Tschopp, Emanuel and Saleiro, André and Wiesinger, Ria and Kaiser, Thomas M.},
  doi          = {10.1038/s41559-025-02794-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1719-1730},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enlargement of sternum traits facilitated the evolution of powered flight in birds. <em>NATEE</em>, <em>9</em>(9), 1705-1718. (<a href='https://doi.org/10.1038/s41559-025-02795-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An enlarged sternum with a prominent keel is a central feature of the flight apparatus of modern birds. However, sterna of near-bird dinosaurs (Pennaraptora) and early avialans are either substantially different from those of living birds or absent altogether, raising questions about how specialized sternal structures evolved in birds and how they are related to function. This remains poorly understood because of the fragmentary nature of the fossil record, and the challenges in inferring form and function from crushed fossils. We use ancestral character estimations to trace sternal trait acquisition through the bird stem group, and multivariate phylogenetic regressions to analyse relationships between sternum morphology, body mass and flight capabilities. We find that sternum evolution was episodic: basal members of Pennaraptora had proportionally small sterna, which became larger and more craniocaudally elongated in Avialae. This enlargement precedes the appearance of a midline ridge, a possible precursor of the sternal keel, in Pygostylia. Sternum size increased again in crownward Ornithuromorpha, alongside a fully formed sternal keel and enlarged caudal projections, both critical areas of flight muscle attachment. Sternal experimentation in relation to flight characteristics occurs several times throughout Pennaraptora, including within Paraves and Enantiornithes, indicating that powered flight may have evolved several times before proliferating in crown-group birds. The extant avian sternum is highly adapted to powered flight, but understanding of its early evolution is hindered by patchy fossil representation and the fact that where preserved, sterna in near-bird dinosaurs and early avialans are substantially different. To overcome this, the authors use ancestral character estimations and find that sternum evolution was episodic and experimental in relation to flight characteristics.},
  archive      = {J_NATEE},
  author       = {Lowi-Merri, Talia M. and Benson, Roger and Hu, Han and O’Connor, Jingmai and Claramunt, Santiago and Evans, David C.},
  doi          = {10.1038/s41559-025-02795-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1705-1718},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Enlargement of sternum traits facilitated the evolution of powered flight in birds},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The role of plasmid copy number and mutation rate in evolutionary outcomes. <em>NATEE</em>, <em>9</em>(9), 1694-1704. (<a href='https://doi.org/10.1038/s41559-025-02792-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multicopy plasmids are widespread in nature and compose a common strategy for spreading beneficial traits across microbes. However, the role of plasmids in supporting the evolution of encoded genes remains underexplored due to challenges in experimentally manipulating key parameters such as plasmid copy number and mutation rate. Here we developed a strategy for controlling copy number in the plasmid-based Saccharomyces cerevisiae continuous evolution system, OrthoRep, and used our resulting capabilities to investigate the evolution of a conditionally essential gene under varying copy number and mutation rate conditions. Our results show that low copy number facilitated the faster enrichment of beneficial alleles whereas high copy number promoted robustness through the maintenance of allelic diversity. High copy number also slowed the removal of deleterious mutations and increased the fraction of non-functional alleles that could hitchhike during evolution. This study highlights the nuanced relationships between plasmid copy number, mutation rate and evolutionary outcomes, providing insights into the adaptive dynamics of genes encoded on multicopy plasmids and nominating OrthoRep as a versatile tool for studying plasmid evolution. In an experimental multicopy plasmid evolution system that allows manipulation of mutation rate and copy number, low copy number promoted enrichment of beneficial alleles whereas high copy number maintained allelic diversity.},
  archive      = {J_NATEE},
  author       = {Pisera, Alexander ‘Olek’ and Liu, Chang C.},
  doi          = {10.1038/s41559-025-02792-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1694-1704},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The role of plasmid copy number and mutation rate in evolutionary outcomes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas. <em>NATEE</em>, <em>9</em>(9), 1685-1693. (<a href='https://doi.org/10.1038/s41559-025-02771-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis is a recently identified cause of Hansen’s disease, and is associated with the more severe and potentially lethal presentations of diffuse lepromatous leprosy and Lucio’s phenomenon. Detection of this infection has been limited to a small number of individuals, leaving much to be learned about its global distribution and transmissibility. Its discovery in wild rodent populations in the United Kingdom and Ireland also raises questions about its zoonotic potential. Here, we raise further awareness of this disease via analyses of two exceptionally well preserved M. lepromatosis genomes obtained from 4,000-year-old human remains of two adult males from the archaeological sites of El Cerrito and La Herradura in Northern Chile. This formed the basis of genomic comparisons between ancient and modern forms of the pathogen. We demonstrate an unexpected long history of M. lepromatosis in the Americas, which contrasts with the more recent Eurasian history of the closely related Mycobacterium leprae. We offer relevant perspectives on its evolution while providing an incentive for further disease monitoring in both humans and other potential reservoir species in the Americas and elsewhere. Recovery of ancient Mycobacterium lepromatosis genomes in 4,000-year-old human remains from Chile demonstrates that this causal agent of Hansen’s disease has a long history in the Americas, predating the colonial period by millennia.},
  archive      = {J_NATEE},
  author       = {Ramirez, Darío A. and Sitter, T. Lesley and Översti, Sanni and Herrera-Soto, María José and Pastor, Nicolás and Fontana-Silva, Oscar Eduardo and Kirkpatrick, Casey L. and Castelleti-Dellepiane, José and Nores, Rodrigo and Bos, Kirsten I.},
  doi          = {10.1038/s41559-025-02771-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1685-1693},
  shortjournal = {Nat. Ecol. Evol},
  title        = {4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pervasive but biome-dependent relationship between fragmentation and resilience in forests. <em>NATEE</em>, <em>9</em>(9), 1670-1684. (<a href='https://doi.org/10.1038/s41559-025-02776-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The relationship between landscape fragmentation and vegetation resilience is uncertain. Here we use multiple satellite-based tree cover data and vegetation indices to quantify the apparent effects of fragmentation on global forest resilience and potential mechanisms thereof. We measure fragmentation as edge density, patch density and mean patch area of tree cover patches, and measure resilience as one-lag temporal autocorrelation of vegetation indices. We find a statistically significant (P &lt; 0.05) fragmentation–resilience relationship in about 77% of fragmented forests, but the direction varies across biomes. In tropical and temperate forests, fragmentation is linked to increased local temperature and atmospheric dryness, resulting in a negative fragmentation–resilience relationship. Conversely, in boreal forests, fragmentation is associated with decreased atmospheric dryness and enhanced light resource, thereby increasing forest resilience. Our results reconcile competing hypotheses and highlight the importance of accounting for fragmentation when predicting shifts in ecosystem resilience under disturbances. These findings also suggest the necessity of biome-targeted forest management strategies for climate change mitigation and adaptation. Human-driven forest fragmentation could have major impacts on ecosystem functioning. This global analysis shows that the relationship between fragmentation and vegetation resilience in forests may differ depending on bioclimatic region and local environmental conditions.},
  archive      = {J_NATEE},
  author       = {Su, Yongxian and Zhang, Chaoqun and Cescatti, Alessandro and Yu, Kailiang and Ciais, Philippe and Smith, Taylor and Shang, Jiali and Carnicer, Jofre and Liu, Jane and Chen, Jing Ming and Green, Julia K. and Wu, Jianping and Ponce-Campos, Guillermo E. and Zhang, Yongguang and Zuo, Zhiyan and Liao, Jinbao and Lafortezza, Raffaele and Yan, Kai and Yang, Xueqin and Liu, Liyang and Ren, Jiashun and Yuan, Wenping and Chen, Xiuzhi and Wu, Chaoyang and Zhou, Weiqi},
  doi          = {10.1038/s41559-025-02776-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1670-1684},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pervasive but biome-dependent relationship between fragmentation and resilience in forests},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family. <em>NATEE</em>, <em>9</em>(9), 1656-1669. (<a href='https://doi.org/10.1038/s41559-025-02777-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations are more likely to produce some phenotypes than others, but the causal role of these production propensities in the evolution of phenotypic diversity remains unclear. There are two major challenges: it is difficult to separate the effect of the genotype–phenotype (GP) map from that of natural selection when analysing natural diversity, and most extant phenotypes evolved long ago in species whose GP maps cannot be recovered. Here, using two reconstructed ancestral transcription factors that are closely related but differ in function, we created libraries containing all possible amino acid combinations at historically variable sites in the proteins’ DNA binding interface (the genotypes) and measured their capacity to specifically bind DNA elements containing all possible combinations of nucleotides at historically variable sites (the phenotypes). The ancestral GP maps were strongly anisotropic (the distribution of phenotypes encoded by genotypes is highly non-uniform) and heterogeneous (the phenotypes accessible around each genotype vary greatly among genotypes), but the extent and direction of these properties differed substantially between the maps. In both cases, these properties steered evolution towards the lineage-specific phenotypes that evolved during history. Our findings establish that ancient properties of the GP relationship were causal factors in the evolutionary process that produced present-day patterns of functional conservation and diversity. A combination of ancestral reconstruction and deep mutational scanning to examine the genotype–phenotype maps of steroid receptors shows that the properties of ancestral genotype–phenotype maps determine the lineage-specific evolution of DNA specificity.},
  archive      = {J_NATEE},
  author       = {Herrera-Álvarez, Santiago and Patton, Jaeda E. J. and Thornton, Joseph W.},
  doi          = {10.1038/s41559-025-02777-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1656-1669},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments. <em>NATEE</em>, <em>9</em>(9), 1643-1655. (<a href='https://doi.org/10.1038/s41559-025-02779-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human impacts on nature span vast spatial scales that transcend abiotic gradients and biogeographic barriers, yet estimates of biodiversity loss from land-use change overwhelmingly derive from local-scale studies. Using a field dataset of 971 bird species sampled in forest and cattle pasture across 13 biogeographic regions of Colombia, we quantify biodiversity losses from local to near-national scales. Losses are on average 60% worse at the pan-Colombian scale than in individual regions, with underestimation remaining until six to seven biogeographic regions are sampled. Regional losses greatly exceed local losses when beta-diversity is high due to reduced species turnover in pasture across geographic space and elevation. Extrapolation from local-scale studies causes major underestimation of biodiversity loss, emphasizing the need to incorporate spatial structure into measures of change. Analysis of data on 971 bird species in natural habitat and cattle pasture in Colombia finds that near-national-scale losses of bird diversity greatly exceed losses recorded at the local scale, suggesting that extrapolations from local studies will severely underestimate biodiversity losses.},
  archive      = {J_NATEE},
  author       = {Socolar, Jacob B. and Mills, Simon C. and Gilroy, James J. and Martínez-Revelo, Diego E. and Medina-Uribe, Claudia A. and Parra-Sanchez, Edicson and Ramirez-Gutierrez, Marcela and Sand Sæbø, Jørgen and Meneses, Henry S. and Pérez, Giovanny and Barlow, Jos and Ochoa Quintero, Jose M. and Freckleton, Robert P. and Haugaasen, Torbjørn and Edwards, David P.},
  doi          = {10.1038/s41559-025-02779-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1643-1655},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development. <em>NATEE</em>, <em>9</em>(9), 1626-1642. (<a href='https://doi.org/10.1038/s41559-025-02780-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bats are the only mammals capable of self-powered flight, an evolutionary innovation based on the transformation of forelimbs into wings. The bat wing is characterized by an extreme elongation of the second to fifth digits with a wing membrane called the chiropatagium connecting them. Here we investigated the developmental and cellular origin of this structure by comparing bat and mouse limbs using omics tools and single-cell analyses. Despite the substantial morphological differences between the species, we observed an overall conservation of cell populations and gene expression patterns including interdigital apoptosis. Single-cell analyses of micro-dissected embryonic chiropatagium identified a specific fibroblast population, independent of apoptosis-associated interdigital cells, as the origin of this tissue. These distal cells express a conserved gene programme including the transcription factors MEIS2 and TBX3, which are commonly known to specify and pattern the early proximal limb. Transgenic ectopic expression of MEIS2 and TBX3 in mouse distal limb cells resulted in the activation of genes expressed during wing development and phenotypic changes related to wing morphology, such as the fusion of digits. Our results elucidate fundamental molecular mechanisms of bat wing development and illustrate how drastic morphological changes can be achieved through repurposing of existing developmental programmes during evolution. Single-cell comparison of developing bat and mouse limbs reveals conservation of cell populations and gene expression patterns, and suggests repurposing of genes involved in proximal limb development for wing evolution.},
  archive      = {J_NATEE},
  author       = {Schindler, Magdalena and Feregrino, Christian and Aldrovandi, Silvia and Lo, Bai-Wei and Monaco, Anna A. and Ringel, Alessa R. and Morales, Ariadna E. and Zehnder, Tobias and Behncke, Rose Yinghan and Glaser, Juliane and Barclay, Alexander and Andrey, Guillaume and Kragesteen, Bjørt K. and Hägerling, René and Haas, Stefan A. and Vingron, Martin and Ulitsky, Igor and Marti-Renom, Marc A. and Hechavarria, Julio and Fasel, Nicolas and Hiller, Michael and Lupiáñez, Darío G. and Mundlos, Stefan and Real, Francisca M.},
  doi          = {10.1038/s41559-025-02780-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1626-1642},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep learning reveals the complex genetic architecture of male guppy colouration. <em>NATEE</em>, <em>9</em>(9), 1614-1625. (<a href='https://doi.org/10.1038/s41559-025-02781-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The extraordinary variation in male guppy (Poecilia reticulata) colouration is a powerful model for studying the interplay of natural and sexual selection. However, the complexity of this variation has hampered the high-resolution characterization and determination of the genetic architecture underlying male guppy colour and clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male colour variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a genome-wide association study of colour traits. Our phenotypic and genomic results converge to show that colour patterning in guppies is a combination of many heritable features, each with a largely independent genetic architecture spanning the entire genome. Autosomally inherited ornaments are polygenic, with significant contributions from loci involved in neural crest cell migration. Unusually, the results of our genome-wide association study suggest that gene duplicates from the autosomes to the Y chromosome are responsible for much of the sex-linked variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait. Guppy colouration is a very diverse trait under natural and sexual selection. Using a combination of high-resolution phenotyping and genomic analysis, the authors reveal the genetic architecture of this complex trait.},
  archive      = {J_NATEE},
  author       = {van der Bijl, Wouter and Shu, Jacelyn J. and Goberdhan, Versara S. and Sherin, Linley M. and Jia, Changfu and Cortazar-Chinarro, Maria and Corral-Lopez, Alberto and Mank, Judith E.},
  doi          = {10.1038/s41559-025-02781-w},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1614-1625},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deep learning reveals the complex genetic architecture of male guppy colouration},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood. <em>NATEE</em>, <em>9</em>(9), 1599-1613. (<a href='https://doi.org/10.1038/s41559-025-02785-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deadwood is a major carbon source in forests, and yet the fate of this carbon remains a gap in our understanding of global carbon cycling. Lignin, the most recalcitrant biopolymer in wood, is mainly decayed through extracellular enzymatic and chemical processes initiated by white-rot fungi. However, the intracellular conversion of lignin decay products has been overlooked in the fungal kingdom. Here we integrate comparative genomic and phylogenetic analyses to understand the distribution and evolution of enzymes responsible for modifying lignin-related aromatic compounds—such as decarboxylases, hydroxylases, dioxygenases and other downstream ring-cleavage enzymes—that funnel carbon to central metabolism across the bacterial and the fungal kingdoms. We demonstrate that specific fungal lineages conserve these enzyme families, and that the abilities to enzymatically depolymerize lignin and catabolize lignin-related aromatic compounds are not necessarily coupled. Our analyses also reveal an expanded substrate specificity of aromatic ring-cleavage enzymes during fungal evolution, as well as a clade of extracellular enzymes among them, broadening the spatial range of these biochemical capabilities. Together, our results highlight a large diversity of fungal enzymes and hosts that warrant further investigation for inclusion into carbon cycling models and biotechnological applications for the conversion of aromatic compounds. This study uses comparative genomics and phylogenetics to analyse the distribution and evolution of key enzymes involved in the catabolism of lignin-related aromatic compounds in the bacterial and fungal kingdoms.},
  archive      = {J_NATEE},
  author       = {Kijpornyongpan, Teeratas and Kuatsjah, Eugene and Komp, Evan and Evans, James E. and Ruiz-Dueñas, Francisco Javier and Salvachúa, Davinia},
  doi          = {10.1038/s41559-025-02785-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1599-1613},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife. <em>NATEE</em>, <em>9</em>(9), 1585-1598. (<a href='https://doi.org/10.1038/s41559-025-02786-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across the world, human (anthropophonic) sounds add to sounds of biological (biophonic) and geophysical (geophonic) origin, with human contributions including both speech and technophony (sounds of technological devices). To characterize society’s contribution to the global soundscapes, we used passive acoustic recorders at 139 sites across 6 continents, sampling both urban green spaces and nearby pristine sites continuously for 3 years in a paired design. Recordings were characterized by bird species richness and by 14 complementary acoustic indices. By relating each index to seasonal, diurnal, climatic and anthropogenic factors, we show here that latitude, time of day and day of year each predict a substantial proportion of variation in key metrics of biophony—whereas anthropophony (speech and traffic) show less predictable patterns. Compared to pristine sites, the soundscape of urban green spaces is more dominated by technophony and less diverse in terms of acoustic energy across frequencies and time steps, with less instances of quiet. We conclude that the global soundscape is formed from a highly predictable rhythm in biophony, with added noise from geophony and anthropophony. At urban sites, animals experience an increasingly noisy background of sound, which poses challenges to efficient communication. Analysis of soundscape data from 139 globally distributed sites reveals that sounds of biological origin exhibit predictable rhythms depending on location and season, whereas sounds of anthropogenic origin are less predictable. Comparisons between paired urban–rural sites show that urban green spaces are noisier and dominated by sounds of technological origin.},
  archive      = {J_NATEE},
  author       = {Somervuo, Panu and Roslin, Tomas and Fisher, Brian L. and Hardwick, Bess and Kerdraon, Deirdre and Raharinjanahary, Dimby and Rajoelison, Eric Tsiriniaina and Lauha, Patrik and Griem, Lukas and Lehikoinen, Petteri and Niittynen, Pekka and Piirainen, Esko and Lumme, Markus and Riihikoski, Ville-Matti and Acevedo-Charry, Orlando and Adalsteinsson, Solny A. and Ahmad, Maaz and Alcobia, Sandra and Aldará, Jón and Andrew, Nigel R. and Anslan, Sten and Antonelli, Alexandre and Arena, Julieta Soledad and Arroyo Almeida, Santiago and Aster, Ines and Autto, Hannu and Aviles Gamboa, Anahi and Baixeras, Joaquín and Baldauf, Mario and Balestrieri, Rosario and Banelyte, Gaia Giedre and Barrett, Adrian and Beja, Pedro and Berg, Thomas Olof and Bergerot, Benjamin and Biro, Elizabeth G. and Blendinger, Pedro G. and Bollache, Loïc and Bou Dagher Kharrat, Magda and Boyer, Stephane and Bridell, Erika and Brotherson, Martyn and Brown, Leslie Robert and Buckley, Hannah L. and Buscardo, Erika and Buthelezi, Nokuphila and Cagnolo, Luciano and Calvente, Alice and Capobianco, Giovanni and Carreón-Palau, Laura and Carriere, Suzanne and Case, Bradley S. and Chang, Jenyu and Chaparro, Juan Matías and Chen, Chi-Ling and Chicoine, Christine and Christensson, Madeleine and Collado Rosique, Francisco and Colom Montero, William and da Fonseca, Ricardo do Sacramento and Da Silva, Luís P. and Dal Molin, Anamaria and Dallas, Tad and de Francesco, Maria Carla and Del Ángel-Rodríguez, Jorge Arturo and Díaz-Delgado, Ricardo and Dirnböck, Thomas and Djukic, Ika and Dladla, Philile and Domínguez Masciale, Jeremías and Dorigo, Thiago and Douwes, Errol and Ekrem, Torbjørn and Enderskog, Helena and Erefur, Charlotta and Fahad, Muhammad and Falahati-Anbaran, Mohsen and Farrell, Arielle and Ferland, Gabriel and Ferrari, Emanuele and Figueiredo, Axa and Forero, Fernando and Freiberga, Inga and Frosch-Radivo, Andrea and Ganchozo Intriago, Luis Alberto and Garzoli, Laura and Giacomotti, Paola and Gianuca, Andros T. and Gilg, Olivier and Gilg, Vladimir and Gísladóttir, Fanney and Glowacki, Ryan and Gottsberger, Brigitte and Gregoire, Jocelyn and Groner, Elli and Guedes, Patrícia and Guile, Aimee Michelle and Haase, Peter and Hadi, Fazal and Haidegger, Magdalena and Hansen, Leivur Janus and Hansen, Lars Holst and Harrop, Reid and Havnås, Harald and Herrera Báez, David and Ho, Chris C. Y. and Hohenbühel, Denise and Houska Tahadlova, Marketa and Hänninen, Jari and Höglund, Linda and Í Haraldsstovu, Kolbrún and Imbeau, Elise and Inkinen, Jasmin and Ishihara, Masae Iwamoto and Jackson, Abigail C. and Jansson, Gunnar and Jha, Rohit and Kager, Gerald and Kahale, Rhea and Kalttopää, Oula and Karai, Elizabeth Wanjiru and Karlsson, Dave and Kaus-Thiel, Andrea and Khan, Asghar and Khan, Qaisar and Kimoto, Keishi and Kipngetich, Shadrack Chumo and Klante, Clemens and Klemedtsson, Leif and Klinth, Mårten and Koskinen, Janne and Kotakorpi, Matti and Kreiling, Agnes-Katharina and Krisai-Greilhuber, Irmgard and Kristensen, Erik and König, Sebastian and Langenheder, Silke and Laurila, Kalevi and Le Gouar, Pascaline and Lecomte, Nicolas and Lecomte, Erin and Leitman, Paula Moraes and León-Cortés, Jorge L. and Li, Daijiang and Loehr, John and Lopez-Vaamonde, Carlos and Makari, Mehsen and Mangini, Gabriela Giselle and Maroschek, Michael and Mata, Vanessa A. and Matsuoka, Shunsuke and Mazzafera, Thais and McDonald, Paul G. and Meinert, Laura and Meléndez-González, Mayra and Mendoza-Henao, Angela M. and Moreau, Sebastien and Moreau, Jérôme and Mosbacher, Jesper and Moyer, Esteban and Mrazova, Anna and Mteshane, Samantha and Mungai, Nancy Wangari and Muñoz Herraiz, Gema and Murillo-Vázquez, Andrea and Musazzi, Simona and Mutanen, Marko and Müller, Jörg and Navarro Canales, Rebeca and Ndlovu, Monica and Nicolai, Annegret and Niessner, Armin and Nordén, Jenni and Nowak, Paweł and O’Connell, Erin and Orru, Arianna and Pagnon, Thomas and Pantoja-Diaz, Yurani Nayive and Pentinsaari, Mikko and Pilloni, Sebastian and Pinder, Adrian and Pinheiro, Thiago A. and Põlme, Sergei and Powell, Luke L. and Pröll, Gisela and Pulido-Santacruz, Paola and Queralt, Enrique and Quilantang, Mark Tristan and Quinlan, Kirsty and Ramirez, Ricardo and Rankinen, Juha and Rasino, Micaela Del Valle and Rebelo, Rui and Remmers, Wolfram and Retz, Franziska and Reyes, Evelin and Rivas Torres, Gonzalo and Rogers, Hanna M. K. and Rosário, Inês T. and Rosário Da Rosàrio da Costa, Sidney and Rütting, Tobias and Sahlstén, Johannes and Saliba, Carole and Salmirinne, Teppo and Sam, Katerina and Santos, Douglas and Santos-Reis, Margarida and Sawan, Michel and Schattanek-Wiesmair, Benjamin and Schiestl-Aalto, Pauliina and Schmidt, Niels Martin and Seibold, Sebastian and Seidl, Rupert and Seifert, Linda and Sithole, Malibongwe and Sivault, Elise and Smart, Jessica and Smerczyński, Ireneusz and Soda, Ayaka and Sousa-Lima, Renata S. and Stanisci, Angela and Stanley, Margaret C. and Steenkamp, Daleen and Stengel, Elisa and Stoll, Stefan and Strauss, Willem Maartin and Stur, Elisabeth and Sujala, Maija and Sundell, Janne and Svavarsdóttir, Jónína and Tedersoo, Leho and Tepsa, Saana and Tikochinsky, Maor Tiko and Tuominen, Esa-Pekka and Tweraser, Stefanie and Ulloa Espinosa, Catalina and Uusitalo, Joni and Vallinmäki, Mikko and Vannier, Fabrice and Varela, Abigail and Vatka, Emma and Veikkolainen, Silja and Vernes, Karl and Watts, Phillip C. and Weslien, Per and Wirth, Ciara and Wisniewski, Jana Helga and Young, Amanda B. and Övergaard, Robyn and Ovaskainen, Otso},
  doi          = {10.1038/s41559-025-02786-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1585-1598},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency. <em>NATEE</em>, <em>9</em>(9), 1571-1584. (<a href='https://doi.org/10.1038/s41559-025-02761-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerous leaf-level experiments suggest that plant intrinsic water-use efficiency (iWUE) increases under elevated CO2 because of reduced stomatal conductance and enhanced photosynthesis. However, it remains elusive whether this response can be extrapolated to the ecosystem scale, because confounding factors and compensating feedbacks are often involved in ecosystem iWUE variations. Here we develop a machine learning-based framework to disentangle the ecosystem-scale CO2 effects on iWUE and its two components, canopy conductance (Gc) and gross primary productivity (GPP), based on global networks of long-term eddy covariance observations. Our results show widespread CO2-induced enhancement of iWUE across diverse ecosystems, driven predominantly by Gc reduction rather than GPP stimulation. Moreover, three divergent response types are identified across the studied ecosystems, based on the strength and significance of CO2-driven Gc reduction and GPP enhancement, indicating spatially non-uniform responses to rising CO2. Nutrient supply, water availability and biome types are found to be critical factors regulating this spatial heterogeneity. Overall, our study provides observational insights into ecosystem-scale CO2 fertilization effects. Such understandings are essential to inform terrestrial biosphere models for better projections of carbon and water cycles given the intensified changing climate in a CO2-rich future. A machine learning approach using long-term observations of eddy covariance finds that the increase in plant intrinsic water-use efficiency under higher CO2 levels, across diverse ecosystems, is driven primarily by reduction in canopy conductance, rather than by stimulation of gross primary productivity.},
  archive      = {J_NATEE},
  author       = {Zhan, Weiwei and Lian, Xu and Liu, Jiangong and Han, Jisu and Huang, Yu and Yang, Hao and Zhan, Chunhui and Winkler, Alexander J. and Gentine, Pierre},
  doi          = {10.1038/s41559-025-02761-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1571-1584},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change. <em>NATEE</em>, <em>9</em>(9), 1562-1570. (<a href='https://doi.org/10.1038/s41559-025-02788-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plankton models form the core of marine ecosystem simulators, with uses from regional resource and ecosystem management to climate change projections. In this Perspective, we suggest that stronger alignment of models with empirical knowledge about plankton physiology, diversity and trophic roles will improve model utility and the reliability of their outputs regarding biodiversity, ecophysiology, trophic dynamics and biogeochemistry. We recommend key steps to resolve the disconnect between empirical research and simulation models accounting for well-established plankton processes with an aim to increase the utility of such models for applied uses. A central challenge is characterizing the complexity of plankton diversity and activity in ways that are amenable to model incorporation. We argue that experts in empirical science are best placed to advise the development of next-generation models to address these challenges, and we propose a series of actions to achieve that engagement, including involvement of these experts in the design and exploitation of plankton digital twins. Plankton models have crucial applications in ecosystem management and climate change projections. This Perspective suggests that stronger alignment of plankton models with empirical knowledge is needed and recommends steps to close the gap between empirical research and modelling.},
  archive      = {J_NATEE},
  author       = {Flynn, Kevin J. and Atkinson, Angus and Beardall, John and Berges, John A. and Boersma, Maarten and Brunet, Christophe and Calbet, Albert and Caron, Dave A. and Dam, Hans G. and Glibert, Patricia M. and Hansen, Per Juel and Jin, Peng and Lønborg, Christian and Mayor, Daniel J. and Menden-Deuer, Susanne and Mock, Thomas and Mulholland, Margaret R. and Needham, David M. and Polimene, Luca and Poulton, Alex J. and Robinson, Carol and Rokitta, Sebastian D. and Rost, Björn and Saiz, Enric and Scanlan, David J. and Schmidt, Katrin and Sherr, Evelyn and Stoecker, Diane K. and Svensen, Camilla and Thiele, Stefan and Thingstad, Tron F. and Våge, Selina},
  doi          = {10.1038/s41559-025-02788-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1562-1570},
  shortjournal = {Nat. Ecol. Evol},
  title        = {More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bird sternum evolution facilitated the origins of powered flight. <em>NATEE</em>, <em>9</em>(9), 1560-1561. (<a href='https://doi.org/10.1038/s41559-025-02817-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An in-depth analysis of the breastbone (sternum) of bird ancestors reveals its importance in the origin and evolution of flight in dinosaurs as they evolved into birds.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02817-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1560-1561},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Bird sternum evolution facilitated the origins of powered flight},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus. <em>NATEE</em>, <em>9</em>(9), 1558-1559. (<a href='https://doi.org/10.1038/s41559-025-02799-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geochemical chronologies in surface increments and exposed cross-sections of naturally fractured hominin enamel from the South African sites of Swartkrans and Kromdraai indicate that Paranthropus robustus exploited both forest and grassland habitats, and that individuals did not move on the landscape in a manner analogous to extant African apes.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02799-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1558-1559},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolving a bat wing by reusing an existing gene program. <em>NATEE</em>, <em>9</em>(9), 1556-1557. (<a href='https://doi.org/10.1038/s41559-025-02782-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interspecies comparative analyses of single-cell transcriptomic data reveal that a proximal limb gene program is reactivated during bat wing formation. Our results illustrate how evolutionary innovations can arise from the reutilization of an existing gene set, activated through species-specific regulatory repertoires.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02782-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1556-1557},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolving a bat wing by reusing an existing gene program},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The next generation of colour pattern genomics. <em>NATEE</em>, <em>9</em>(9), 1554-1555. (<a href='https://doi.org/10.1038/s41559-025-02816-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Convolutional neural networks and genetic association analysis decode the evolution of colour pattern diversity and its underlying complex genetic architecture in the Trinidadian guppy.},
  archive      = {J_NATEE},
  author       = {Karagic, Nidal and Kratochwil, Claudius F.},
  doi          = {10.1038/s41559-025-02816-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1554-1555},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The next generation of colour pattern genomics},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sea star wasting disease mystery finally solved. <em>NATEE</em>, <em>9</em>(9), 1552-1553. (<a href='https://doi.org/10.1038/s41559-025-02789-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A decade after a marine epidemic killed off sea stars and triggered ecosystem-wide effects along the Pacific Coast of North America, researchers have identified the bacterial pathogen that is responsible for sea star wasting disease.},
  archive      = {J_NATEE},
  author       = {Lafferty, Kevin D.},
  doi          = {10.1038/s41559-025-02789-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1552-1553},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sea star wasting disease mystery finally solved},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leprosy in the americas before columbus. <em>NATEE</em>, <em>9</em>(9), 1550-1551. (<a href='https://doi.org/10.1038/s41559-025-02790-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis genomes associated with 4,000-year-old human skeletons in Chile establish an American origin for this causal agent of leprosy (also known as Hansen’s disease), and point to different evolutionary trajectories and transmission pathways for M. lepromatosis and its sister pathogen Mycobacterium leprae.},
  archive      = {J_NATEE},
  author       = {Roberts, Charlotte A.},
  doi          = {10.1038/s41559-025-02790-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1550-1551},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Leprosy in the americas before columbus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Water-use efficiency driven by water savings. <em>NATEE</em>, <em>9</em>(9), 1548-1549. (<a href='https://doi.org/10.1038/s41559-025-02793-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of land–atmosphere water and CO2 fluxes suggests that reduced water use by vegetation, rather than increased carbon uptake, is the driving factor behind the well-documented increase in vegetation water-use efficiency in response to rising atmospheric CO2.},
  archive      = {J_NATEE},
  author       = {Knauer, Jürgen},
  doi          = {10.1038/s41559-025-02793-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1548-1549},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Water-use efficiency driven by water savings},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Independent origins of long snouts. <em>NATEE</em>, <em>9</em>(9), 1547. (<a href='https://doi.org/10.1038/s41559-025-02856-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-025-02856-8},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1547},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Independent origins of long snouts},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Seed dispersers wanted. <em>NATEE</em>, <em>9</em>(9), 1546. (<a href='https://doi.org/10.1038/s41559-025-02857-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02857-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1546},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Seed dispersers wanted},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fish without borders. <em>NATEE</em>, <em>9</em>(9), 1545. (<a href='https://doi.org/10.1038/s41559-025-02852-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02852-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1545},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fish without borders},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mines aren’t on the maps. <em>NATEE</em>, <em>9</em>(9), 1544. (<a href='https://doi.org/10.1038/s41559-025-02833-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-025-02833-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1544},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mines aren’t on the maps},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Museums in the age of extinction. <em>NATEE</em>, <em>9</em>(9), 1542-1543. (<a href='https://doi.org/10.1038/s41559-025-02772-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Browning, Claire},
  doi          = {10.1038/s41559-025-02772-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1542-1543},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Museums in the age of extinction},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding environmental impacts of plastic requires a palaeontological lens. <em>NATEE</em>, <em>9</em>(9), 1539-1541. (<a href='https://doi.org/10.1038/s41559-025-02813-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Growing evidence suggests that timescales for plastic degradation have been vastly underestimated. The fossil record of plastic-like biopolymers might provide a perspective on plastic fossilization in deep time.},
  archive      = {J_NATEE},
  author       = {Gabbott, Sarah and Boom, Arnoud and Zalasiewicz, Jan},
  doi          = {10.1038/s41559-025-02813-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1539-1541},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Understanding environmental impacts of plastic requires a palaeontological lens},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Restoration outcomes are compromised by a lack of diverse native seed supply. <em>NATEE</em>, <em>9</em>(9), 1537-1538. (<a href='https://doi.org/10.1038/s41559-025-02825-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Changes in seed banking practices and business incentives are needed to ensure restoration practitioners have access to plentiful and reliable supplies of diverse native seeds.},
  archive      = {J_NATEE},
  author       = {Gallagher, Rachael V.},
  doi          = {10.1038/s41559-025-02825-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1537-1538},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Restoration outcomes are compromised by a lack of diverse native seed supply},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Increase the regulation of biodiversity effects from private equity markets. <em>NATEE</em>, <em>9</em>(9), 1535-1536. (<a href='https://doi.org/10.1038/s41559-025-02828-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Carter, Harrison and Miles, Nell and Hawkins, Isobel and Hulett, Toscanie and zu Ermgassen, Sophus O. S. E.},
  doi          = {10.1038/s41559-025-02828-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1535-1536},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increase the regulation of biodiversity effects from private equity markets},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The amazon’s new oil frontier is a biodiversity bomb in the making. <em>NATEE</em>, <em>9</em>(9), 1532-1534. (<a href='https://doi.org/10.1038/s41559-025-02815-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Paletta, Julia and Callegari, Camila and Oliveira Fiorini, Ana Carolina and Dias, Filipe Gomes and Freitas Junior, Adirson and de Barros, Pedro Henrique Batista},
  doi          = {10.1038/s41559-025-02815-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1532-1534},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The amazon’s new oil frontier is a biodiversity bomb in the making},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sound science. <em>NATEE</em>, <em>9</em>(9), 1531. (<a href='https://doi.org/10.1038/s41559-025-02860-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Passive acoustic monitoring is opening up new opportunities for biodiversity monitoring, but there is still work to do to ensure that it represents a reliable method for biodiversity reporting.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02860-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1531},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sound science},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natmi">NATMI - 21</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires. <em>NATMI</em>, <em>7</em>(9), 1587. (<a href='https://doi.org/10.1038/s42256-025-01128-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Jiang, Yuepeng and Zhang, Pingping and Huo, Miaozhe and Li, Shuai Cheng},
  doi          = {10.1038/s42256-025-01128-1},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1587},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A framework to evaluate machine learning crystal stability predictions. <em>NATMI</em>, <em>7</em>(9), 1586. (<a href='https://doi.org/10.1038/s42256-025-01117-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Riebesell, Janosh and Goodall, Rhys E. A. and Benner, Philipp and Chiang, Yuan and Deng, Bowen and Ceder, Gerbrand and Asta, Mark and Lee, Alpha A. and Jain, Anubhav and Persson, Kristin A.},
  doi          = {10.1038/s42256-025-01117-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1586},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: A framework to evaluate machine learning crystal stability predictions},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanistic understanding and validation of large AI models with SemanticLens. <em>NATMI</em>, <em>7</em>(9), 1572-1585. (<a href='https://doi.org/10.1038/s42256-025-01084-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike human-engineered systems, such as aeroplanes, for which the role and dependencies of each component are well understood, the inner workings of artificial intelligence models remain largely opaque, which hinders verifiability and undermines trust. Current approaches to neural network interpretability, including input attribution methods, probe-based analysis and activation visualization techniques, typically provide limited insights about the role of individual components or require extensive manual interpretation that cannot scale with model complexity. This paper introduces SemanticLens, a universal explanation method for neural networks that maps hidden knowledge encoded by components (for example, individual neurons) into the semantically structured, multimodal space of a foundation model such as CLIP. In this space, unique operations become possible, including (1) textual searches to identify neurons encoding specific concepts, (2) systematic analysis and comparison of model representations, (3) automated labelling of neurons and explanation of their functional roles, and (4) audits to validate decision-making against requirements. Fully scalable and operating without human input, SemanticLens is shown to be effective for debugging and validation, summarizing model knowledge, aligning reasoning with expectations (for example, adherence to the ABCDE rule in melanoma classification) and detecting components tied to spurious correlations and their associated training data. By enabling component-level understanding and validation, the proposed approach helps mitigate the opacity that limits confidence in artificial intelligence systems compared to traditional engineered systems, enabling more reliable deployment in critical applications. SemanticLens is a tool that embeds artificial intelligence model components (such as neurons) into a searchable, human-understandable space. This enables automated auditing, validation of decisions and detection of problematic behaviours with minimal human oversight.},
  archive      = {J_NATMI},
  author       = {Dreyer, Maximilian and Berend, Jim and Labarta, Tobias and Vielhaben, Johanna and Wiegand, Thomas and Lapuschkin, Sebastian and Samek, Wojciech},
  doi          = {10.1038/s42256-025-01084-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1572-1585},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Mechanistic understanding and validation of large AI models with SemanticLens},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning. <em>NATMI</em>, <em>7</em>(9), 1561-1571. (<a href='https://doi.org/10.1038/s42256-025-01098-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence has transformed the field of precise organic synthesis. Data-driven methods, including machine learning and deep learning, have shown great promise in predicting reaction performance and synthesis planning. However, the inherent methodological divergence between numerical regression-driven reaction performance prediction and sequence generation-based synthesis planning creates formidable challenges in constructing a unified deep learning architecture. Here we present RXNGraphormer, a framework to jointly address these tasks through a unified pre-training approach. By synergizing graph neural networks for intramolecular pattern recognition with Transformer-based models for intermolecular interaction modelling, and training on 13 million reactions via a carefully designed strategy, RXNGraphormer achieves state-of-the-art performance across eight benchmark datasets for reactivity or selectivity prediction and forward-synthesis or retrosynthesis planning, as well as three external realistic datasets for reactivity and selectivity prediction. Notably, the model generates chemically meaningful embeddings that spontaneously cluster reactions by type without explicit supervision. This work bridges the critical gap between performance prediction and synthesis planning tasks in chemical AI, offering a versatile tool for accurate reaction prediction and synthesis design. Xu et al. present RXNGraphormer, a pre-trained model that learns bond transformation patterns from over 13 million reactions, achieving state-of-the-art accuracy in reaction performance prediction and synthesis planning.},
  archive      = {J_NATMI},
  author       = {Xu, Li-Cheng and Tang, Miao-Jiong and An, Junyi and Cao, Fenglei and Qi, Yuan},
  doi          = {10.1038/s42256-025-01098-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1561-1571},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces. <em>NATMI</em>, <em>7</em>(9), 1555-1560. (<a href='https://doi.org/10.1038/s42256-025-01091-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular docking plays a crucial role in structure-based drug discovery, enabling the prediction of how small molecules interact with protein targets. Traditional docking methods rely on scoring functions and search heuristics, whereas recent generative approaches, such as DiffDock, leverage deep learning for pose prediction. However, blind-diffusion-based docking often struggles with binding site localization and pose accuracy, particularly in complex protein–ligand systems. This work introduces GeoDirDock (GDD), a guided diffusion approach to molecular docking that enhances the accuracy and physical plausibility of ligand docking predictions. GDD guides the denoising process of a diffusion model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. Our method leverages expert knowledge to direct the generative modelling process, specifically targeting desired protein–ligand interaction regions. We demonstrate that GDD outperforms existing blind docking methods in terms of root mean squared distance accuracy and physicochemical pose realism. Our results indicate that incorporating domain expertise into the diffusion process leads to more biologically relevant docking predictions. Additionally, we explore the potential of GDD as a template-based modelling tool for lead optimization in drug discovery through angle transfer in maximum common substructure docking, showcasing its capability to accurately predict ligand orientations for chemically similar compounds. Future applications in real-world drug discovery campaigns will naturally continue to refine and extend the utility of prior-informed diffusion docking methods. GeoDirDock is a framework that guides the denoising process of a generative diffusion docking model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. This approach enhances the accuracy and physical plausibility of ligand docking predictions.},
  archive      = {J_NATMI},
  author       = {Miñán, Raúl and Gallardo, Javier and Ciudad, Álvaro and Molina, Alexis},
  doi          = {10.1038/s42256-025-01091-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1555-1560},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Error-controlled non-additive interaction discovery in machine learning models. <em>NATMI</em>, <em>7</em>(9), 1541-1554. (<a href='https://doi.org/10.1038/s42256-025-01086-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) models are powerful tools for detecting complex patterns, yet their ‘black-box’ nature limits their interpretability, hindering their use in critical domains like healthcare and finance. Interpretable ML methods aim to explain how features influence model predictions but often focus on univariate feature importance, overlooking complex feature interactions. Although recent efforts extend interpretability to feature interactions, existing approaches struggle with robustness and error control, especially under data perturbations. In this study, we introduce Diamond, a method for trustworthy feature interaction discovery. Diamond uniquely integrates the model-X knockoffs framework to control the false discovery rate, ensuring a low proportion of falsely detected interactions. Diamond includes a non-additivity distillation procedure that refines existing interaction importance measures to isolate non-additive interaction effects and preserve false discovery rate control. This approach addresses the limitations of off-the-shelf interaction measures, which, when used naively, can lead to inaccurate discoveries. Diamond’s applicability spans a broad class of ML models, including deep neural networks, transformers, tree-based models and factorization-based models. Empirical evaluations on both simulated and real datasets across various biomedical studies demonstrate its utility in enabling reliable data-driven scientific discoveries. Diamond represents a significant step forward in leveraging ML for scientific innovation and hypothesis generation. Diamond, a statistically rigorous method, is capable of finding meaningful feature interactions within machine learning models, making black-box models more interpretable for science and medicine.},
  archive      = {J_NATMI},
  author       = {Chen, Winston and Jiang, Yifan and Noble, William Stafford and Lu, Yang Young},
  doi          = {10.1038/s42256-025-01086-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1541-1554},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Error-controlled non-additive interaction discovery in machine learning models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks. <em>NATMI</em>, <em>7</em>(9), 1524-1540. (<a href='https://doi.org/10.1038/s42256-025-01082-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering novel drug candidate molecules is a fundamental step in drug development. Generative deep learning models can sample new molecular structures from learned probability distributions; however, their practical use in drug discovery hinges on generating compounds tailored to a specific target molecule. Here we introduce DrugGEN, an end-to-end generative system for the de novo design of drug candidate molecules that interact with a selected protein. The proposed method represents molecules as graphs and processes them using a generative adversarial network that comprises graph transformer layers. Trained on large datasets of drug-like compounds and target-specific bioactive molecules, DrugGEN designed candidate inhibitors for AKT1, a kinase crucial in many cancers. Docking and molecular dynamics simulations suggest that the generated compounds effectively bind to AKT1, and attention maps provide insights into the model’s reasoning. Furthermore, selected de novo molecules were synthesized and shown to inhibit AKT1 at low micromolar concentrations in the context of in vitro enzymatic assays. These results demonstrate the potential of DrugGEN for designing target-specific molecules. Using the open-access DrugGEN codebase, researchers can retrain the model for other druggable proteins, provided a dataset of known bioactive molecules is available. Inhibiting AKT1 kinase can have potentially positive uses against many types of cancer. To find novel molecules targeting this protein, a graph adversarial network is trained as a generative model.},
  archive      = {J_NATMI},
  author       = {Ünlü, Atabey and Çevrim, Elif and Yiğit, Melih Gökay and Sarıgün, Ahmet and Çelikbilek, Hayriye and Bayram, Osman and Kahraman, Deniz Cansen and Olğaç, Abdurrahman and Rifaioglu, Ahmet Sureyya and Banoğlu, Erden and Doğan, Tunca},
  doi          = {10.1038/s42256-025-01082-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1524-1540},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Brain–computer interface control with artificial intelligence copilots. <em>NATMI</em>, <em>7</em>(9), 1510-1523. (<a href='https://doi.org/10.1038/s42256-025-01090-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motor brain–computer interfaces (BCIs) decode neural signals to help people with paralysis move and communicate. Even with important advances in the past two decades, BCIs face a key obstacle to clinical viability: BCI performance should strongly outweigh costs and risks. To significantly increase the BCI performance, we use shared autonomy, where artificial intelligence (AI) copilots collaborate with BCI users to achieve task goals. We demonstrate this AI-BCI in a non-invasive BCI system decoding electroencephalography signals. We first contribute a hybrid adaptive decoding approach using a convolutional neural network and ReFIT-like Kalman filter, enabling healthy users and a participant with paralysis to control computer cursors and robotic arms via decoded electroencephalography signals. We then design two AI copilots to aid BCI users in a cursor control task and a robotic arm pick-and-place task. We demonstrate AI-BCIs that enable a participant with paralysis to achieve 3.9-times-higher performance in target hit rate during cursor control and control a robotic arm to sequentially move random blocks to random locations, a task they could not do without an AI copilot. As AI copilots improve, BCIs designed with shared autonomy may achieve higher performance. AI copilots are integrated into brain–computer interfaces, enabling a paralysed participant to achieve improved control of computer cursors and robotic arms. This shared autonomy approach offers a promising path to increase BCI performance and clinical viability.},
  archive      = {J_NATMI},
  author       = {Lee, Johannes Y. and Lee, Sangjoon and Mishra, Abhishek and Yan, Xu and McMahan, Brandon and Gaisford, Brent and Kobashigawa, Charles and Qu, Mike and Xie, Chang and Kao, Jonathan C.},
  doi          = {10.1038/s42256-025-01090-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1510-1523},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Brain–computer interface control with artificial intelligence copilots},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Conditional generation of real antigen-specific t cell receptor sequences. <em>NATMI</em>, <em>7</em>(9), 1494-1509. (<a href='https://doi.org/10.1038/s42256-025-01096-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent advances in T cell receptor (TCR) engineering, designing functional TCRs against arbitrary targets remains challenging due to complex rules governing cross-reactivity and limited paired data. Here we present TCR-TRANSLATE, a sequence-to-sequence framework that adapts low-resource machine translation techniques to generate antigen-specific TCR sequences against unseen epitopes. By evaluating 12 model variants of the BART and T5 model architectures, we identified key factors affecting performance and utility, revealing discordances between these objectives. Our flagship model, TCRT5, outperforms existing approaches on computational benchmarks, prioritizing functionally relevant sequences at higher ranks. Most significantly, we experimentally validated a computationally designed TCR against Wilms’ tumour antigen, a therapeutically relevant target in leukaemia, excluded from our training and validation sets. Although the identified TCR shows cross-reactivity with pathogen-derived peptides, highlighting limitations in specificity, our work represents the successful computational design of a functional TCR construct against a non-viral epitope from the target sequence alone. Our findings establish a foundation for computational TCR design and reveal current limitations in data availability and methodology, providing a framework for accelerating personalized immunotherapy by reducing the search space for novel targets. TCR-TRANSLATE, a deep learning framework adapting machine translation to immune design, demonstrates the successful generation of a functional T cell receptor sequence for a cancer epitope from the target sequence alone.},
  archive      = {J_NATMI},
  author       = {Karthikeyan, Dhuvarakesh and Bennett, Sarah N. and Reynolds, Amy G. and Vincent, Benjamin G. and Rubinsteyn, Alex},
  doi          = {10.1038/s42256-025-01096-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1494-1509},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Conditional generation of real antigen-specific t cell receptor sequences},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modelling neural coding in the auditory midbrain with high resolution and accuracy. <em>NATMI</em>, <em>7</em>(9), 1478-1493. (<a href='https://doi.org/10.1038/s42256-025-01104-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational models of auditory processing can be valuable tools for research and technology development. Models of the cochlea are highly accurate and widely used, but models of the auditory brain lag far behind in both performance and penetration. Here we present ICNet, a convolutional encoder–decoder model of neural coding in the inferior colliculus. We developed ICNet using large-scale intracranial recordings from anaesthetized gerbils, addressing three key modelling challenges that are common across all sensory systems: capturing the full statistical structure of neuronal response patterns; accounting for physiological and experimental non-stationarity; and extracting features of sensory processing that are shared across different brains. ICNet provides highly accurate simulation of multi-unit neural responses to a wide range of complex sounds, including near-perfect responses to speech. It also reproduces key neurophysiological phenomena such as forward masking and dynamic range adaptation. ICNet can be used to simulate activity from thousands of neural units or to provide a compact representation of early central auditory processing through its latent dynamics, facilitating a wide range of hearing and audio applications. It can also serve as a foundation core, providing a baseline neural representation for models of active listening or higher-level auditory processing. Drakopoulos et al. present a model that captures the transformation from sound waves to neural activity patterns underlying early auditory processing. The model reproduces neural responses to a range of complex sounds and key neurophysiological phenomena.},
  archive      = {J_NATMI},
  author       = {Drakopoulos, Fotios and Pellatt, Lloyd and Sabesan, Shievanie and Xia, Yiqing and Fragner, Andreas and Lesica, Nicholas A.},
  doi          = {10.1038/s42256-025-01104-9},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1478-1493},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Modelling neural coding in the auditory midbrain with high resolution and accuracy},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping. <em>NATMI</em>, <em>7</em>(9), 1461-1477. (<a href='https://doi.org/10.1038/s42256-025-01099-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) deciphers genome-wide chromatin accessibility, providing profound insights into gene regulation mechanisms. With the rapid advance of sequencing technologies, scATAC-seq data typically encompass numerous samples from various conditions, resulting in complex batch effects, thus necessitating reliable integration tools. While numerous batch integration tools exist for single-cell RNA sequencing data, inherent data characteristic differences limit their effectiveness on scATAC-seq data. Existing integration methods for scATAC-seq data suffer from several fundamental limitations, such as disrupting the biological heterogeneity and focusing solely on low-dimensional correction, which may distort data and hinder downstream analysis. Here we propose Fountain, a deep learning framework for scATAC-seq data integration via rigorous barycentric mapping. Barycentric mapping transforms one data distribution to another in a principled and effective manner through optimal transport. By regularizing barycentric mapping with geometric data information, Fountain achieves accurate batch alignment while preserving biological heterogeneity. Comprehensive experiments across diverse real-world datasets demonstrate the advantages of Fountain over existing methods in batch correction and biological conservation. In addition, the trained Fountain model can integrate data from new batches alongside already integrated data without retraining, enabling continuous online data integration. Moreover, Fountain’s reconstruction strategy generates batch-corrected ATAC profiles, improving the capture of cellular heterogeneity and revealing cell-type-specific implications such as expression enrichment analysis and partitioned heritability analysis. Zhu, Hua and Chen propose Fountain, a deep learning framework for batch integration of scATAC-seq data that utilizes regularized barycentric mapping. It preserves biological heterogeneity, enabling online and original dimensionality integration.},
  archive      = {J_NATMI},
  author       = {Zhu, Shuchen and Hua, Heyang and Chen, Shengquan},
  doi          = {10.1038/s42256-025-01099-3},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1461-1477},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerating protein engineering with fitness landscape modelling and reinforcement learning. <em>NATMI</em>, <em>7</em>(9), 1446-1460. (<a href='https://doi.org/10.1038/s42256-025-01103-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein engineering holds substantial promise for designing proteins with customized functions, yet the vast landscape of potential mutations versus limited laboratory capacity constrains the discovery of optimal sequences. Here, to address this, we present the μProtein framework, which accelerates protein engineering by combining μFormer, a deep learning model for accurate mutational effect prediction, with μSearch, a reinforcement learning algorithm designed to efficiently navigate the protein fitness landscape using μFormer as an oracle. μProtein leverages single-mutation data to predict optimal sequences with complex, multi-amino-acid mutations through its modelling of epistatic interactions and a multi-step search strategy. In addition to strong performance on benchmark datasets, μProtein identified high-gain-of-function multi-point mutants for the enzyme β-lactamase, surpassing one of the highest-known activity levels, in wet laboratory, trained solely on single-mutation data. These results demonstrate μProtein’s capability to discover impactful mutations across the vast protein sequence space, offering a robust, efficient approach for protein optimization. μProtein, combining deep learning and reinforcement learning, is developed to design high-function proteins. This framework, trained only on single-mutation data, discovers multi-site β-lactamase mutants with up to 2,000× growth rates.},
  archive      = {J_NATMI},
  author       = {Sun, Haoran and He, Liang and Deng, Pan and Liu, Guoqing and Zhao, Zhiyu and Jiang, Yuliang and Cao, Chuan and Ju, Fusong and Wu, Lijun and Liu, Haiguang and Qin, Tao and Liu, Tie-Yan},
  doi          = {10.1038/s42256-025-01103-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1446-1460},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accelerating protein engineering with fitness landscape modelling and reinforcement learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient protein structure generation with sparse denoising models. <em>NATMI</em>, <em>7</em>(9), 1429-1445. (<a href='https://doi.org/10.1038/s42256-025-01100-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteins play diverse roles in all domains of life and are extensively harnessed as biomolecules in biotechnology, with applications spanning from fundamental research to biomedicine. Therefore, there is considerable interest in computationally designing proteins with specified properties. Protein structure generative models provide a means to design protein structures in a controllable manner and have been successfully applied to address various protein design tasks. Such models are paired with protein sequence and structure predictors to produce and select protein sequences for experimental testing. However, current protein structure generators face important limitations for proteins with more than 400 amino acids and require retraining for protein design tasks unseen during model training. To address the first issue, we introduce salad, a family of sparse all-atom denoising models for protein structure generation. Our models are smaller and faster than the state of the art and matching or improving design quality, successfully generating structures for protein lengths up to 1,000 amino acids. To address the second issue, we combine salad with structure editing, a sampling strategy for expanding the capability of protein denoising models to unseen tasks. We apply our approach to a variety of challenging protein design tasks, from generating protein scaffolds containing functional protein motifs (motif scaffolding) to designing proteins capable of adopting multiple distinct folds under different conditions (multi-state protein design), demonstrating the flexibility of salad and structure editing. A small and fast diffusion model is presented, which is able to efficiently generate long protein backbones.},
  archive      = {J_NATMI},
  author       = {Jendrusch, Michael and Korbel, Jan O.},
  doi          = {10.1038/s42256-025-01100-z},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1429-1445},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Efficient protein structure generation with sparse denoising models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics. <em>NATMI</em>, <em>7</em>(9), 1414-1428. (<a href='https://doi.org/10.1038/s42256-025-01097-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Self-supervised learning (SSL) has emerged as a powerful approach for learning meaningful representations from large-scale unlabelled datasets in single-cell genomics. Richter et al. evaluated SSL pretext tasks on modelling single-cell RNA sequencing (scRNA-seq) data, demonstrating the effective use of SSL models. However, the transferability of these pretrained SSL models to the spatial transcriptomics domain remains unexplored. Here we assess the performance of three SSL models (random mask, gene programme mask and Barlow Twins) pretrained on scRNA-seq data with spatial transcriptomics datasets, focusing on cell-type prediction and spatial clustering. Our experiments demonstrate that the SSL model with random mask strategy exhibits the best overall performance among evaluated SSL models. Moreover, the models trained from scratch on spatial transcriptomics data outperform the fine-tuned SSL models on cell-type prediction, highlighting a domain gap between scRNA-seq and spatial transcriptomics data whose underlying causes remain an open question. Through expanded analyses of multiple imputation methods and data degradation scenarios, we demonstrate that gene imputation would degrade SSL model performance on cell-type prediction, an effect that is exacerbated by increasing data sparsity. Finally, integrating zero-shot random mask embeddings into chosen spatial clustering methods significantly enhanced their accuracy. Overall, our findings provide valuable insights into the limitations and potential of transferring SSL models to spatial transcriptomics and offer practical guidance for researchers leveraging pretrained models for spatial transcriptomics data analysis. Self-supervised learning models for single-cell RNA sequencing data exhibit poor transferability to spatial transcriptomics for cell-type prediction, although their learned features may enhance spatial analysis.},
  archive      = {J_NATMI},
  author       = {Han, Chuangyi and Lin, Senlin and Wang, Zhikang and Cui, Yan and Zou, Qi and Yuan, Zhiyuan},
  doi          = {10.1038/s42256-025-01097-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1414-1428},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosting the predictive power of protein representations with a corpus of text annotations. <em>NATMI</em>, <em>7</em>(9), 1403-1413. (<a href='https://doi.org/10.1038/s42256-025-01088-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models are trained to predict amino acid sequences from vast protein databases and learn to represent proteins as feature vectors. These vector representations have enabled impressive applications, from predicting mutation effects to protein folding. One of the reasons offered for the success of these models is that conserved sequence motifs tend to be important for protein fitness. Yet, the relationship between sequence conservation and fitness can be confounded by the evolutionary and environmental context. Should we, therefore, look to other data sources that may contain more direct functional information? In this work, we conduct a comprehensive study examining the effects of training protein models to predict 19 types of text annotation from UniProt. Our results show that fine-tuning protein models on a subset of these annotations enhances the models’ predictive capabilities on a variety of function prediction tasks. In particular, when evaluated on our tasks, our model outperforms the basic local alignment search tool, which none of the pretrained protein models accomplished. Our results suggest that a much wider array of data modalities, such as text annotations, may be tapped to improve protein language models. Although protein language models have enabled major advances, they often rely on indirect signals that may not fully capture functional relevance. Fine-tuning these models on textual annotations is shown to improve their performance on function prediction tasks.},
  archive      = {J_NATMI},
  author       = {Duan, Haonan and Skreta, Marta and Cotta, Leonardo and Rajaonson, Ella Miray and Dhawan, Nikita and Aspuru-Guzik, Alán and Maddison, Chris J.},
  doi          = {10.1038/s42256-025-01088-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1403-1413},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Boosting the predictive power of protein representations with a corpus of text annotations},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury. <em>NATMI</em>, <em>7</em>(9), 1390-1402. (<a href='https://doi.org/10.1038/s42256-025-01105-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spinal cord injury (SCI) disrupts neuromuscular control, severely affecting independence and quality of life. Although upper limb wearable robots hold considerable promise for functional restoration, most existing prototypes have been validated minimally in people with SCI and target almost exclusively hand opening and closing. We introduce a lightweight, modular assistive soft exosuit that simultaneously and automatically supports shoulder abduction and elbow flexion or extension movements using lightweight fabric-based pneumatic actuators, controlled through inertial sensors. The individual elbow modules were first validated in 11 healthy volunteers, and subsequently tested, together with the shoulder module, in 15 individuals with cervical SCI (C4–C7, AIS A–D). In the SCI participants, exosuits assistance resulted in increased static endurance time (by more than 250%), and lower activity of the primary muscles involved in dynamic tasks (by up to 50%). The two SCI participants retaining prehensile capability also improved their scores in the box and block test when assisted. Moreover, the soft actuation provided a safe, comfortable and easy-to-use solution that was positively appreciated by the participants. Collectively, these results provide encouraging evidence that exosuits can augment upper limb motor performance, and may ultimately translate into greater functional independence and quality of life for the SCI population. A lightweight, modular assistive soft exosuit is introduced, which supports shoulder and elbow movement in individuals with cervical spinal cord injury. The device enhances endurance and range of motion, reduces muscle effort and improves clinical test scores.},
  archive      = {J_NATMI},
  author       = {Ferroni, Roberto and D’Avola, Gaetano and Sciarrone, Giorgia and Righi, Gabriele and De Santis, Claudia and Carpaneto, Jacopo and Gandolla, Marta and Del Popolo, Giulio and Micera, Silvestro and Proietti, Tommaso},
  doi          = {10.1038/s42256-025-01105-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1390-1402},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Aligning generalization between humans and machines. <em>NATMI</em>, <em>7</em>(9), 1378-1389. (<a href='https://doi.org/10.1038/s42256-025-01109-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in artificial intelligence (AI)—including generative approaches—have resulted in technology that can support humans in scientific discovery and forming decisions, but may also disrupt democracies and target individuals. The responsible use of AI and its participation in human–AI teams increasingly shows the need for AI alignment, that is, to make AI systems act according to our preferences. A crucial yet often overlooked aspect of these interactions is the different ways in which humans and machines generalize. In cognitive science, human generalization commonly involves abstraction and concept learning. By contrast, AI generalization encompasses out-of-domain generalization in machine learning, rule-based reasoning in symbolic AI, and abstraction in neurosymbolic AI. Here we combine insights from AI and cognitive science to identify key commonalities and differences across three dimensions: notions of, methods for, and evaluation of generalization. We map the different conceptualizations of generalization in AI and cognitive science along these three dimensions and consider their role for alignment in human–AI teaming. This results in interdisciplinary challenges across AI and cognitive science that must be tackled to support effective and cognitively supported alignment in human–AI teaming scenarios. Ilievski et al. examine differences and similarities in the various ways human and AI systems generalize. The insights are important for effectively supporting alignment in human–AI teams.},
  archive      = {J_NATMI},
  author       = {Ilievski, Filip and Hammer, Barbara and van Harmelen, Frank and Paassen, Benjamin and Saralajew, Sascha and Schmid, Ute and Biehl, Michael and Bolognesi, Marianna and Dong, Xin Luna and Gashteovski, Kiril and Hitzler, Pascal and Marra, Giuseppe and Minervini, Pasquale and Mundt, Martin and Ngomo, Axel-Cyrille Ngonga and Oltramari, Alessandro and Pasi, Gabriella and Saribatur, Zeynep G. and Serafini, Luciano and Shawe-Taylor, John and Shwartz, Vered and Skitalinskaya, Gabriella and Stachl, Clemens and van de Ven, Gido M. and Villmann, Thomas},
  doi          = {10.1038/s42256-025-01109-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1378-1389},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Aligning generalization between humans and machines},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Real-world validation of a structure-aware pipeline for molecular design. <em>NATMI</em>, <em>7</em>(9), 1376-1377. (<a href='https://doi.org/10.1038/s42256-025-01102-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The next major challenge for artificial intelligence in drug development lies in proving its value in real-world settings. A new technology not only supports the generation of novel chemical entities but also accelerates a range of real-world molecular design tasks.},
  archive      = {J_NATMI},
  author       = {Dias, Ana Laura and Rodrigues, Tiago},
  doi          = {10.1038/s42256-025-01102-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1376-1377},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Real-world validation of a structure-aware pipeline for molecular design},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards agentic science for advancing scientific discovery. <em>NATMI</em>, <em>7</em>(9), 1373-1375. (<a href='https://doi.org/10.1038/s42256-025-01110-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence is transforming scientific discovery through (semi-)autonomous agents capable of reasoning, planning, and interacting with digital and physical environments. This Comment explores the foundations and frontiers of agentic science, outlining its emerging directions, current limitations, and the pathways for responsible integration into scientific practice.},
  archive      = {J_NATMI},
  author       = {Xin, Hongliang and Kitchin, John R. and Kulik, Heather J.},
  doi          = {10.1038/s42256-025-01110-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1373-1375},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards agentic science for advancing scientific discovery},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LLMs as all-in-one tools to easily generate publication-ready citation diversity reports. <em>NATMI</em>, <em>7</em>(9), 1371-1372. (<a href='https://doi.org/10.1038/s42256-025-01101-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Cantú, Melissa S. and King, Michael R.},
  doi          = {10.1038/s42256-025-01101-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1371-1372},
  shortjournal = {Nat. Mach. Intell.},
  title        = {LLMs as all-in-one tools to easily generate publication-ready citation diversity reports},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Applying genomic AI to combat antibiotic resistance in low-income countries. <em>NATMI</em>, <em>7</em>(9), 1369-1370. (<a href='https://doi.org/10.1038/s42256-025-01108-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Aruhomukama, Dickson},
  doi          = {10.1038/s42256-025-01108-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1369-1370},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Applying genomic AI to combat antibiotic resistance in low-income countries},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nature">NATURE - 38</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: TCF1 and LEF1 promote B-1a cell homeostasis and regulatory function. <em>NATURE</em>, <em>646</em>(8083), E3. (<a href='https://doi.org/10.1038/s41586-025-09609-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Shen, Qian and Wang, Hao and Roco, Jonathan A. and Meng, Xiangpeng and Bosticardo, Marita and Hodges, Marie and Battaglia, Michael and Feng, Zhi-Ping and Talks, Benjamin James and Powell, Jason and Shanmugiah, Vijaya Baskar Mahalingam and Chu, Julia and Rahman, Najib M. and Elsheikh, Alguili and Chakravarty, Probir and Grenov, Amalie and Emmerich, Max and Delmonte, Ottavia M. and Freeman, Alexandra F. and Keller, Michael D. and Belaid, Brahim and Papa, Ilenia and Lee, James C. and Cañete, Pablo F. and Gonzalez-Figueroa, Paula and Zhang, Yaoyuan and Xue, Hai-Hui and Turajlic, Samra and Notarangelo, Luigi D. and Haniffa, Muzlifah and Garrett-Sinha, Lee Ann and Parry, Helen M. and Kanellakis, Nikolaos I. and Vinuesa, Carola G.},
  doi          = {10.1038/s41586-025-09609-4},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {E3},
  shortjournal = {Nature},
  title        = {Publisher correction: TCF1 and LEF1 promote B-1a cell homeostasis and regulatory function},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Experimental determination of partial charges with electron diffraction. <em>NATURE</em>, <em>646</em>(8083), E2. (<a href='https://doi.org/10.1038/s41586-025-09608-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Mahmoudi, Soheil and Gruene, Tim and Schröder, Christian and Ferjaoui, Khalil D. and Fröjdh, Erik and Mozzanica, Aldo and Takaba, Kiyofumi and Volkov, Anatoliy and Maisriml, Julian and Paunović, Vladimir and van Bokhoven, Jeroen A. and Keppler, Bernhard K.},
  doi          = {10.1038/s41586-025-09608-5},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {E2},
  shortjournal = {Nature},
  title        = {Publisher correction: Experimental determination of partial charges with electron diffraction},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A ctf4 trimer couples the CMG helicase to DNA polymerase α in the eukaryotic replisome. <em>NATURE</em>, <em>646</em>(8083), E1. (<a href='https://doi.org/10.1038/s41586-025-09606-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Simon, Aline C. and Zhou, Jin C. and Perera, Rajika L. and van Deursen, Frederick and Evrin, Cecile and Ivanova, Marina E. and Kilkenny, Mairi L. and Renault, Ludovic and Kjaer, Svend and Matak-Vinković, Dijana and Labib, Karim and Costa, Alessandro and Pellegrini, Luca},
  doi          = {10.1038/s41586-025-09606-7},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {E1},
  shortjournal = {Nature},
  title        = {Author correction: A ctf4 trimer couples the CMG helicase to DNA polymerase α in the eukaryotic replisome},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Molecular mechanism of ultrafast transport by plasma membrane ca2+-ATPases. <em>NATURE</em>, <em>646</em>(8083), 236-245. (<a href='https://doi.org/10.1038/s41586-025-09402-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tight control of intracellular Ca2+ levels is fundamental as they are used to control numerous signal transduction pathways1. Plasma membrane Ca2+-ATPases (PMCAs) have a crucial role in this process by extruding Ca2+ against a steep concentration gradient from the cytosol to the extracellular space2. Although new details of PMCA biology are constantly being uncovered, the structural basis of the most distinguishing features of these pumps, namely, transport rates in the kilohertz range and regulation of activity by the plasma membrane phospholipid PtdIns(4,5)P2, has so far remained elusive. Here we present the structures of mouse PMCA2 in the presence and absence of its accessory subunit neuroplastin in eight different stages of its transport cycle. Combined with whole-cell recordings that accurately track PMCA-mediated Ca2+ extrusion in intact cells, these structures enable us to establish the first comprehensive transport model for a PMCA, reveal the role of disease-causing mutations and uncover the structural underpinnings of regulatory PMCA–phospholipid interaction. The transport cycle-dependent dynamics of PtdIns(4,5)P2 are fundamental for its role as a ‘latch’ promoting the fast release of Ca2+ and opening a passageway for counter-ions. These actions are required for maintaining the ultra-fast transport cycle. Moreover, we identify the PtdIns(4,5)P2-binding site as an unanticipated target for drug-mediated manipulation of intracellular Ca2+ levels. Our work provides detailed structural insights into the uniquely fast operation of native PMCA-type Ca2+ pumps and its control by membrane lipids and drugs. Structural insights into ultrafast Ca2+ transport by plasma membrane Ca2+-ATPases are provided, highlighting an essential role for PtdIns(4,5)P2 dynamics.},
  archive      = {J_NATURE},
  author       = {Vinayagam, Deivanayagabarathy and Sitsel, Oleg and Schulte, Uwe and Constantin, Cristina E. and Oosterheert, Wout and Prumbaum, Daniel and Zolles, Gerd and Fakler, Bernd and Raunser, Stefan},
  doi          = {10.1038/s41586-025-09402-3},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {236-245},
  shortjournal = {Nature},
  title        = {Molecular mechanism of ultrafast transport by plasma membrane ca2+-ATPases},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pre-rRNA spatial distribution and functional organization of the nucleolus. <em>NATURE</em>, <em>646</em>(8083), 227-235. (<a href='https://doi.org/10.1038/s41586-025-09412-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The multilayered nucleolus is the primary site of ribosome biogenesis1,2, where successive maturation of small (SSU)3,4 and large (LSU)5 ribosomal subunit precursors occurs. However, the spatiofunctional relationship between pre-rRNA processing and nucleolar substructures and how this adapts to changing cellular physiological demands have remained incompletely understood6,7. Here our spatiotemporal analyses revealed a compartment-specific ribosomal subunit processing in human nucleoli, with SSU processomes maintained in fibrillar centre (FC)–dense fibrillar component (DFC)–periphery dense fibrillar component (PDFC) domains while LSU pre-rRNAs largely transited to PDFC–granular component regions. Slowly proliferating cells exhibited unexpected 5′ external transcribed spacer (5′&nbsp;ETS)-centred SSU&nbsp;processing impairment, accompanied by structural remodelling of FC–DFC units&nbsp;and retarded SSU outflux. Direct 5′&nbsp;ETS processing perturbation at least partially recapitulated these FC–DFC unit&nbsp;alterations, supporting the functional interdependence between SSU processing and nucleolar architecture. Notably, anamniote bipartite nucleoli with merged FC–DFC compartments8,9 exhibited distinct 5′&nbsp;ETS distribution and slower pre-rRNA flux compared with multilayered nucleoli in amniotes. Introducing a FC/DFC interface to bipartite nucleoli enhanced processing efficiency, indicating that the evolutionary emergence of nested FC–DFC units&nbsp;may have optimized pre-rRNA processing. Collectively, depicting the spatiotemporal distribution of pre-rRNAs reveals an essential role of 5′&nbsp;ETS-centred processing in maintaining nucleolar substructures and suggests a possible evolutionary advantage of the multilayered structure in amniotes. The evolutionary emergence of a&nbsp;nested fibrillar centre–dense fibrillar component may have optimized pre-rRNA processing.},
  archive      = {J_NATURE},
  author       = {Pan, Yu-Hang and Shan, Lin and Zhang, Yu-Yao and Yang, Zheng-Hu and Zhang, Yuan and Cao, Shi-Meng and Liu, Xiao-Qi and Zhang, Jun and Yang, Li and Chen, Ling-Ling},
  doi          = {10.1038/s41586-025-09412-1},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {227-235},
  shortjournal = {Nature},
  title        = {Pre-rRNA spatial distribution and functional organization of the nucleolus},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A missing enzyme-rescue metabolite as cause of a rare skeletal dysplasia. <em>NATURE</em>, <em>646</em>(8083), 218-226. (<a href='https://doi.org/10.1038/s41586-025-09397-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Living cells depend on an intricate network of chemical reactions catalysed by enzymes, which sometimes make mistakes that lead to their inactivation. Here we report a metabolite-based mechanism for preserving enzyme function in an unfavourable environment. We found that the enzyme&nbsp;TGDS produces UDP-4-keto-6-deoxyglucose, a mimic of the reaction intermediate of the enzyme UXS1, which regenerates the essential cofactor NAD+ within the catalytic pocket of UXS1 by completing its catalytic cycle. Thus, the production of an ‘enzyme-rescue metabolite’ by TGDS represents a mechanism for maintaining the activity of an enzyme in a subcellular compartment where NAD+ is scarce. Using a combination of in vitro and in vivo studies, we demonstrate that the inability to produce sufficient amounts of this enzyme-rescue metabolite leads to the inactivation of UXS1, impairing the synthesis of specific glycans that are crucial for skeletal development. This provides an explanation for the development of the hereditary skeletal disorder Catel–Manzke syndrome in individuals with TGDS deficiency. Defects in similar protective layers might contribute to metabolic changes in other diseases that cannot be explained with common concepts in metabolic biochemistry. In mammals, the enzyme TGDS&nbsp;produces UDP-4-keto-6-deoxyglucose, which binds to the catalytic pocket of UDP-xylose synthase, thereby regenerating the essential NAD+ cofactor of UDP-xylose synthase in conditions of low NAD+.},
  archive      = {J_NATURE},
  author       = {Jacobs, Jean and Lyubenova, Hristiana and Potelle, Sven and Kopp, Johannes and Gerin, Isabelle and Chan, Wing Lee and Rodriguez de los Santos, Miguel and Hülsemann, Wiebke and Mensah, Martin A. and Cormier-Daire, Valérie and Joosten, Marieke and Bruggenwirth, Hennie T. and Stuurman, Kyra E. and Miranda, Valancy and Campeau, Philippe M. and Wittler, Lars and Graff, Julie and Mundlos, Stefan and Ibrahim, Daniel M. and Van Schaftingen, Emile and Fischer-Zirnsak, Björn and Kornak, Uwe and Ehmke, Nadja and Bommer, Guido T.},
  doi          = {10.1038/s41586-025-09397-x},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {218-226},
  shortjournal = {Nature},
  title        = {A missing enzyme-rescue metabolite as cause of a rare skeletal dysplasia},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Establishment of chromatin architecture interplays with embryo hypertranscription. <em>NATURE</em>, <em>646</em>(8083), 208-217. (<a href='https://doi.org/10.1038/s41586-025-09400-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After fertilization, early embryos undergo dissolution of conventional chromatin organization, including topologically associating domains (TADs)1,2. Zygotic genome activation then commences amid unusually slow de novo establishment of three-dimensional chromatin architecture2. How chromatin organization is established and how it interplays with transcription in early mammalian embryos remain elusive. Here we show that CTCF occupies chromatin throughout mouse early development. By contrast, cohesin poorly binds chromatin in one-cell embryos, coinciding with TAD dissolution. Cohesin binding then progressively increases from two- to eight-cell embryos, accompanying TAD establishment. Unexpectedly, strong ‘genic cohesin islands’ (GCIs) emerge across gene bodies of active genes in this period. GCI genes enrich for cell identity and regulatory genes, display broad H3K4me3 at promoters, and exhibit strong binding of transcription factors and the cohesin loader NIPBL at nearby enhancers. We show that transcription is hyperactive in two- to eight-cell embryos and is required for GCI formation. Conversely, induced transcription can also create GCIs. Finally, GCIs can function as insulation boundaries and form contact domains with nearby CTCF sites, enhancing both the transcription levels and stability of GCI genes. These data reveal a hypertranscription state in early embryos that both shapes and is fostered by the three-dimensional genome organization, revealing an intimate interplay between chromatin structure and transcription. A study reveals progressive cohesin loading and&nbsp;a hypertranscription state responsible for the formation of genic&nbsp;cohesin islands at&nbsp;active regulatory&nbsp;genes in mouse embryos, highlighting an interplay between chromatin architecture and transcription as they are established for the first time.},
  archive      = {J_NATURE},
  author       = {Yu, Guang and Xu, Kai and Xia, Weikun and Zhang, Ke and Xu, Qianhua and Li, Lijia and Lin, Zili and Liu, Ling and Liu, Bofeng and Du, Zhenhai and Chen, Xia and Fan, Qiang and Lai, Fangnong and Wang, Wenying and Wang, Lijuan and Kong, Feng and Wang, Chao and Dai, Haiqiang and Wang, Huili and Xie, Wei},
  doi          = {10.1038/s41586-025-09400-5},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {208-217},
  shortjournal = {Nature},
  title        = {Establishment of chromatin architecture interplays with embryo hypertranscription},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transient APC/C inactivation by mTOR boosts glycolysis during cell cycle entry. <em>NATURE</em>, <em>646</em>(8083), 198-207. (<a href='https://doi.org/10.1038/s41586-025-09328-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian cells entering the cell cycle favour glycolysis to rapidly generate ATP and produce the biosynthetic intermediates that are required for rapid biomass accumulation1. Simultaneously, the ubiquitin-ligase anaphase-promoting complex/cyclosome and its coactivator CDH1 (APC/CCDH1) remains active, allowing origin licensing and blocking premature DNA replication. Paradoxically, glycolysis is reduced by APC/CCDH1 through the degradation of key glycolytic enzymes2, raising the question of how cells coordinate these mutually exclusive events to ensure proper cell division. Here we show that cells resolve this paradox by transiently inactivating the APC/C during cell cycle entry, which allows a transient metabolic shift favouring glycolysis. After mitogen stimulation, rapid mTOR-mediated phosphorylation of the APC/C adapter protein CDH1 at the amino terminus causes it to partially dissociate from the APC/C. This partial inactivation of the APC/C leads to the accumulation of PFKFB3, a rate-limiting enzyme for glycolysis, promoting a metabolic shift towards glycolysis. Delayed accumulation of phosphatase activity later removes CDH1 phosphorylation, restoring full APC/C activity, and shifting cells back to favouring oxidative phosphorylation. Thus, cells coordinate the simultaneous demands of cell cycle progression and metabolism through an incoherent feedforward loop, which transiently inhibits APC/C activity to generate a pulse of glycolysis that is required for mammalian cell cycle entry. APC/C activity is&nbsp;transiently inhibited to generate a pulse of glycolysis that is required for mammalian cell cycle entry.},
  archive      = {J_NATURE},
  author       = {Paul, Debasish and Bolhuis, Derek L. and Yan, Hualong and Das, Sudipto and Xu, Xia and Abbate, Christina C. and Jenkins, Lisa M. M. and Emanuele, Michael J. and Andresson, Thorkell and Huang, Jing and Albeck, John G. and Brown, Nicholas G. and Cappell, Steven D.},
  doi          = {10.1038/s41586-025-09328-w},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {198-207},
  shortjournal = {Nature},
  title        = {Transient APC/C inactivation by mTOR boosts glycolysis during cell cycle entry},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Clone copy number diversity is linked to survival in lung cancer. <em>NATURE</em>, <em>646</em>(8083), 190-197. (<a href='https://doi.org/10.1038/s41586-025-09398-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Both single nucleotide variants (SNVs) and somatic copy number alterations (SCNAs) accumulate in cancer cells during tumour development, fuelling clonal evolution. However, accurate estimation of clone-specific copy numbers from bulk DNA-sequencing data is challenging. Here we present allele-specific phylogenetic analysis of copy number alterations (ALPACA), a method to infer SNV and SCNA coevolution by leveraging phylogenetic trees reconstructed from multi-sample bulk tumour sequencing data using SNV frequencies. ALPACA estimates the SCNA evolution of simulated tumours with a higher accuracy than current state-of-the-art methods1–4. ALPACA uncovers loss-of-heterozygosity and amplification events in minor clones that may be missed using standard approaches and reveals the temporal order of somatic alterations. Analysing clone-specific copy numbers in TRACERx421 lung tumours5,6, we find evidence of increased chromosomal instability in metastasis-seeding clones and enrichment for losses affecting tumour suppressor genes and amplification affecting CCND1. Furthermore, we identify increased SCNA rates in both tumours with polyclonal metastatic dissemination and tumours with extrathoracic metastases, and an association between higher clone copy number diversity and reduced disease-free survival in patients with lung cancer. A study presents ALPACA, a computational method for inferring clone- and allele-specific copy numbers of individual clones from multi-sample bulk DNA-sequencing data, and demonstrates its use to study metastasis trajectories.},
  archive      = {J_NATURE},
  author       = {Pawlik, Piotr and Grigoriadis, Kristiana and Bunkum, Abigail and Coggan, Helena and Frankell, Alexander M. and Martinez-Ruiz, Carlos and Karasaki, Takahiro and Huebner, Ariana and Rowan, Andrew and Fisher, Jasmin and Hackshaw, Allan and Swanton, Charles and Zaccaria, Simone and McGranahan, Nicholas},
  doi          = {10.1038/s41586-025-09398-w},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {190-197},
  shortjournal = {Nature},
  title        = {Clone copy number diversity is linked to survival in lung cancer},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MAChR4 suppresses liver disease via GAP-induced antimicrobial immunity. <em>NATURE</em>, <em>646</em>(8083), 180-189. (<a href='https://doi.org/10.1038/s41586-025-09395-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alcohol-use disorder and alcohol-associated liver disease (ALD) are major causes of death and liver transplantation1. The gut–liver axis has a crucial yet poorly understood role in ALD pathogenesis, which depends on microbial translocation. Intestinal goblet cells (GCs) educate the immune system by forming GC-associated antigen passages (GAPs) on activation of muscarinic acetylcholine receptor M4 (mAChR4, also known as M4), enabling sampling of luminal antigens by lamina&nbsp;propria antigen-presenting cells. Here we show that chronic alcohol use in humans and mice downregulates small intestinal mAChR4 and reduces GAP formation, disrupting antimicrobial immunity. This is reversed on activation of intestinal IL-6 signal transducer (IL6ST, also known as&nbsp;glycoprotein 130;&nbsp;gp130), which restores mAChR4 expression and GAP formation, enabling induction of downstream type-3 innate lymphoid cell-derived IL-22 and antimicrobial REG3 proteins. This blunts translocation of enteric bacteria to the liver, thereby conferring ALD resistance. GAP induction by GC-specific mAChR4 activation was essential and sufficient for prevention of ethanol-induced steatohepatitis. These results lay the foundation for a therapeutic approach using mAChR4 or IL6ST agonists to promote GAP formation and prevent ALD by inhibiting microbial translocation. Chronic alcohol use in humans and mice downregulates small intestinal mAChR4 and reduces goblet-cell-associated antigen passage formation, disrupting antimicrobial immunity.},
  archive      = {J_NATURE},
  author       = {Llorente, Cristina and Raya Tonetti, Fernanda and Bruellman, Ryan and Brea, Rocío and Pell, Nuria and Hartmann, Phillipp and Maccioni, Luca and Han, Hui and Cabré, Noemí and Liu, Junlai and Eguileor, Alvaro and Fondevila, Marcos F. and Meijnikman, Abraham S. and Hsu, Cynthia L. and Alghafri, Ameera and Zhou, Rongrong and Gao, Bei and Duan, Yi and Zhang, Peng and Febbraio, Mark A. and Taniguchi, Koji and Newberry, Rodney D. and Fouts, Derrick E. and Brenner, David A. and Stärkel, Peter and Karin, Michael and Schnabl, Bernd},
  doi          = {10.1038/s41586-025-09395-z},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {180-189},
  shortjournal = {Nature},
  title        = {MAChR4 suppresses liver disease via GAP-induced antimicrobial immunity},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). EBV induces CNS homing of b cells attracting inflammatory t cells. <em>NATURE</em>, <em>646</em>(8083), 171-179. (<a href='https://doi.org/10.1038/s41586-025-09378-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epidemiological data have identified Epstein–Barr virus (EBV) infection as the main environmental risk factor for multiple sclerosis, the predominant autoimmune disease of the central nervous system (CNS)1. However, how EBV infection initiates multiple sclerosis pathogenesis remains unclear. Here we demonstrate that EBV expands oligoclonal T-bet+CXCR3+ B cells that home to the CNS in humanized mice. Effector memory CD8+ T cells and CD4+ TH1 cells as well as CD4+ TH17 cells co-migrate to the brain of EBV-infected humanized mice. T-bet+CXCR3+ B cells can colonize submeningeal brain regions in the absence of other lymphocytes and attract T cells. Depletion of B cells with rituximab or blocking of CXCR3 significantly decreases lymphocyte infiltration into the CNS. Thus, we suggest that symptomatic primary EBV infection generates B cell subsets that gain access to the CNS, attract T cells and thereby initiate multiple sclerosis. Epstein–Barr virus infection generates a neuroinvasive B cell subset, which recruits activated T cells to the central nervous system, promoting multiple sclerosis.},
  archive      = {J_NATURE},
  author       = {Läderach, Fabienne and Piteros, Ioannis and Fennell, Éanna and Bremer, Elena and Last, Mette and Schmid, Sandra and Rieble, Lisa and Campbell, Caroline and Ludwig-Portugall, Isis and Bornemann, Lea and Gruhl, Alexander and Eulitz, Klaus and Gueguen, Paul and Mietz, Juliane and Müller, Anne and Pezzino, Gaetana and Schmitz, Jürgen and Ferlazzo, Guido and Mautner, Josef and Münz, Christian},
  doi          = {10.1038/s41586-025-09378-0},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {171-179},
  shortjournal = {Nature},
  title        = {EBV induces CNS homing of b cells attracting inflammatory t cells},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Maternal stress triggers early-life eczema through fetal mast cell programming. <em>NATURE</em>, <em>646</em>(8083), 161-170. (<a href='https://doi.org/10.1038/s41586-025-09419-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prenatal stress (PS) is a repeated exposure to aversive situations during pregnancy, including high emotional strain, which is suspected to affect homeostatic systems in infants. Paediatric eczema develops quickly after birth at flexural sites subjected to continuous mechanical constraints1,2. Although epidemiological studies have suggested an association between PS and a higher risk of eczema in children3–6, no causative biological link has yet been identified. Here we show that eczema at birth originates from molecular dysregulations of neuroimmune circuits in utero, triggered by fluctuations in the maternal hypothalamic–pituitary–adrenal axis. We found that offspring of stressed pregnant dams have dysregulated mast cells and skin-projecting neurons and quickly develop eczema in response to harmless mechanical friction. We demonstrated that PS transiently modulates amniotic fluid corticosterone concentrations, which directly alters the activation program of skin mast cells expressing the glucocorticoid receptor Nr3c1 and the adjacent sensory neurons conveying mechanosensation. Therapeutic normalization of maternal corticosterone concentrations or genetic depletion of Mcpt5+ mast cells during stressed gestation prevents fetal immune dysregulation and protects against eczema development after birth. Our findings support a new model in which early-onset paediatric eczema originates from dysregulations in the fetal immune system, caused by fluctuations in maternal glucocorticoids induced by stress. Prenatal stress triggers molecular dysregulations in fetal neuroimmune circuits, leading to altered mast cell and sensory neuron function, which predisposes offspring to develop eczema in response to otherwise harmless mechanical friction after birth.},
  archive      = {J_NATURE},
  author       = {Serhan, Nadine and Abdullah, Nasser S. and Gheziel, Nadine and Loste, Alexia and Ekren, Rüçhan and Labit, Elodie and Gonzalez, Anne-Alicia and Oliva, Giulia and Tarot, Pauline and Petitfils, Camille and Payros, Gaëlle and D’Avino, Paolo and Voisin, Allison and Tinsley, Holly Freya Grace and Gentek, Rebecca and Brosseau, Carole and Bodinier, Marie and Reber, Laurent and Val, Pierre and Akdis, Cezmi A. and Mitamura, Yasutaka and Andiappan, Anand Kumar and Chan, Jerry Kok Yen and Ginhoux, Florent and François, Amaury and Cénac, Nicolas and Basso, Lilian and Gaudenzio, Nicolas},
  doi          = {10.1038/s41586-025-09419-8},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {161-170},
  shortjournal = {Nature},
  title        = {Maternal stress triggers early-life eczema through fetal mast cell programming},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A novel bacterial protein family that catalyses nitrous oxide reduction. <em>NATURE</em>, <em>646</em>(8083), 152-160. (<a href='https://doi.org/10.1038/s41586-025-09401-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nitrous oxide (N2O), a driver of global warming and climate change, has reached unprecedented concentrations in Earth’s atmosphere1. Current N2O sources outpace N2O sinks, emphasizing the need for comprehensive understanding of processes that consume N2O. Microbes that express the enzyme N2O reductase (N2OR) convert N2O to climate change-neutral dinitrogen (N2). Known N2ORs belong to the canonical clade I and clade II NosZ reductases and are considered key enzymes for N2O reduction2–4. Here we report a previously unrecognized protein family with a role in N2O reduction, clade III lactonase-type N2OR (L-N2OR), which diverges in sequence from canonical NosZ but conserves three-dimensional protein structural features. Integrated physiological, metagenomic, proteomic and structural modelling studies demonstrate that L-N2ORs catalyse N2O reduction. L-N2OR genes occur in several phyla, predominantly in uncultured taxa with broad geographic distribution. Our findings expand the known diversity of N2ORs and implicate previously unrecognized taxa (for example, Nitrospinota) in N2O consumption. The expansion of N2OR diversity and the identification of a novel type of catalyst for N2O reduction advances the understanding of N2O sinks, has implications for greenhouse gas emission and climate change modelling, and expands opportunities for innovative biotechnologies aimed at curbing N2O emissions5,6. Cultivation of tropical soil microorganisms combined with physiological experiments and bioinformatics analyses identify a family of clade III lactonase-type nitrous oxide reductases with low sequence identity but high 3D structural similarity to known nitrous oxide reductases.},
  archive      = {J_NATURE},
  author       = {He, Guang and Wang, Weijiao and Chen, Gao and Xie, Yongchao and Parks, Jerry M. and Davin, Megan E. and Hettich, Robert L. and Konstantinidis, Konstantinos T. and Löffler, Frank E.},
  doi          = {10.1038/s41586-025-09401-4},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {152-160},
  shortjournal = {Nature},
  title        = {A novel bacterial protein family that catalyses nitrous oxide reduction},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Remodelling of corticostriatal axonal boutons during motor learning. <em>NATURE</em>, <em>646</em>(8083), 143-151. (<a href='https://doi.org/10.1038/s41586-025-09336-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motor skill learning induces long-lasting synaptic plasticity at dendritic spines1–4 and at the outputs of motor cortical neurons to the striatum5,6. However, little is known about corticostriatal axon activity and structural plasticity during learning in the adult brain. Here, using longitudinal in vivo two-photon imaging, we tracked thousands of corticostriatal axonal boutons in the dorsolateral striatum of awake mice. We found that learning a new motor skill dynamically regulated these boutons. The activities of motor corticostriatal axonal boutons exhibited selectivity for rewarded movements (RM) and unrewarded movements (UM). Notably, boutons on the same axonal branches showed diverse responses during behaviour. Motor learning significantly increased the proportion of RM boutons and reduced the heterogeneity of bouton activities. Moreover, motor learning induced profound structural dynamism in boutons. By combining structural and functional imaging, we saw that newly formed axonal boutons were more likely to exhibit selectivity for RM and were stabilized during motor learning, whereas UM boutons were selectively eliminated. These findings reveal a novel form of plasticity in corticostriatal axons and show that motor learning drives dynamic bouton reorganization to support motor skill acquisition and execution. Individual boutons on mouse corticostriatal axons exhibit differential selectivity for rewarded and unrewarded movements during motor learning, and the proportion of reward-responsive boutons increases.},
  archive      = {J_NATURE},
  author       = {Sheng, Mengjun and Lu, Di and Roth, Richard H. and Hwang, Fuu-Jiun and Sheng, Kaiwen and Ding, Jun B.},
  doi          = {10.1038/s41586-025-09336-w},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {143-151},
  shortjournal = {Nature},
  title        = {Remodelling of corticostriatal axonal boutons during motor learning},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Eye structure shapes neuron function in drosophila motion vision. <em>NATURE</em>, <em>646</em>(8083), 135-142. (<a href='https://doi.org/10.1038/s41586-025-09276-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many animals use vision to navigate their environment. The pattern of changes that self-motion induces in the visual scene, referred to as optic flow1, is first estimated in local patches by directionally selective neurons2–4. However, how arrays of directionally selective neurons, each responsive to motion in a preferred direction at specific retinal positions, are organized to support robust decoding of optic flow by downstream circuits is unclear. Understanding this global organization requires mapping fine, local features of neurons across an animal’s field of view3. In Drosophila, the asymmetrical dendrites of the T4 and T5 directionally selective neurons establish their preferred direction, which makes it possible to predict directional tuning from anatomy4,5. Here we show that the organization of the compound eye shapes the systematic variation in the preferred directions of directionally selective neurons across the entire visual field. To estimate the preferred directions across the visual field, we reconstructed hundreds of T4 neurons in an electron-microscopy volume of the full adult fly brain6, and discovered unexpectedly stereotypical dendritic arborizations. We then used whole-head micro-computed-tomography scans to map the viewing directions of all compound eye facets, and found a non-uniform sampling of visual space that explains the spatial variation in preferred directions. Our findings show that the global organization of the directionally selective neurons’ preferred directions is determined mainly by the fly’s compound eye, revealing the intimate connections between eye structure, functional properties of neurons and locomotion control. In Drosophila, the physical structure of the eye has a key role in the directional tuning of motion-sensitive neurons, showing how navigational behaviour is tightly associated with anatomy.},
  archive      = {J_NATURE},
  author       = {Zhao, Arthur and Gruntman, Eyal and Nern, Aljoscha and Iyer, Nirmala and Rogers, Edward M. and Koskela, Sanna and Siwanowicz, Igor and Dreher, Marisa and Flynn, Miriam A. and Laughland, Connor and Ludwig, Henrique and Thomson, Alexander and Moran, Cullen and Gezahegn, Bruck and Bock, Davi D. and Reiser, Michael B.},
  doi          = {10.1038/s41586-025-09276-5},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {135-142},
  shortjournal = {Nature},
  title        = {Eye structure shapes neuron function in drosophila motion vision},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Delegation to artificial intelligence can increase dishonest behaviour. <em>NATURE</em>, <em>646</em>(8083), 126-134. (<a href='https://doi.org/10.1038/s41586-025-09505-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although artificial intelligence enables productivity gains from delegating tasks to machines1, it may facilitate the delegation of unethical behaviour2. This risk is highly relevant amid the rapid rise of ‘agentic’ artificial intelligence systems3,4. Here we demonstrate this risk by having human principals instruct machine agents to perform tasks with incentives to cheat. Requests for cheating increased when principals could induce machine dishonesty without telling the machine precisely what to do, through supervised learning or high-level goal setting. These effects held whether delegation was voluntary or mandatory. We also examined delegation via natural language to large language models5. Although the cheating requests by principals were not always higher for machine&nbsp;agents than for human agents, compliance diverged sharply: machines were far more likely than human agents to carry out fully unethical instructions. This compliance could be curbed, but usually not eliminated, with the injection of prohibitive, task-specific guardrails. Our results highlight ethical risks in the context of increasingly accessible and powerful machine delegation, and suggest design and policy strategies to mitigate them. People cheat more when they delegate&nbsp;tasks to artificial intelligence, and large language models are more likely than humans to comply with unethical instructions—a risk that can be minimized by introducing prohibitive, task-specific guardrails.},
  archive      = {J_NATURE},
  author       = {Köbis, Nils and Rahwan, Zoe and Rilla, Raluca and Supriyatno, Bramantyo Ibrahim and Bersch, Clara and Ajaj, Tamer and Bonnefon, Jean-François and Rahwan, Iyad},
  doi          = {10.1038/s41586-025-09505-x},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {126-134},
  shortjournal = {Nature},
  title        = {Delegation to artificial intelligence can increase dishonest behaviour},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precisely defining disease variant effects in CRISPR-edited single cells. <em>NATURE</em>, <em>646</em>(8083), 117-125. (<a href='https://doi.org/10.1038/s41586-025-09313-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies have identified thousands of individual disease-associated non-coding alleles, but the identification of the causal alleles and their functions remains a critical bottleneck1. CRISPR–Cas editing has enabled targeted modification of DNA to introduce and test disease alleles. However, the combination of inefficient editing, heterogeneous editing outcomes in individual cells and nonspecific transcriptional changes caused by editing and culturing conditions limits the ability to detect the functional consequences of disease alleles2,3. To overcome these challenges, we present a multi-omic single-cell sequencing approach that directly identifies genomic DNA edits, assays the transcriptome and measures cell-surface protein expression. We apply this approach to investigate the effects of gene disruption, deletions in regulatory regions, non-coding single-nucleotide polymorphism alleles and multiplexed editing. We identify the effects of individual single-nucleotide polymorphisms, including the state-specific effects of an IL2RA autoimmune variant in primary human T cells. Multimodal functional genomic single-cell assays, including DNA sequencing, enable the identification of causal variation in primary human cells and bridge a crucial gap in our understanding of complex human diseases. A plate-based assay called CRAFTseq has been developed that uses ‘multi-omic’ single-cell RNA sequencing and direct genotyping of&nbsp;CRISPR edits to test the functional effects of genetic variants on cell state and function.},
  archive      = {J_NATURE},
  author       = {Baglaenko, Yuriy and Mu, Zepeng and Curtis, Michelle and Mire, Hafsa M. and Jayanthi, Vidyashree and Al Suqri, Majd and Liu, Cassidy and Agnew, Ryan and Nathan, Aparna and Mah-Som, Annelise Yoo and Liu, David R. and Newby, Gregory A. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41586-025-09313-3},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {117-125},
  shortjournal = {Nature},
  title        = {Precisely defining disease variant effects in CRISPR-edited single cells},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Connecting chemical and protein sequence space to predict biocatalytic reactions. <em>NATURE</em>, <em>646</em>(8083), 108-116. (<a href='https://doi.org/10.1038/s41586-025-09519-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The application of biocatalysis in synthesis has the potential to offer streamlined routes towards target molecules1, tunable catalyst-controlled selectivity2, as well as processes with improved sustainability3. Despite these advantages, biocatalysis is often a high-risk strategy to implement, as identifying an enzyme capable of performing chemistry on a specific intermediate required for a synthesis can be a roadblock that requires extensive screening of enzymes and protein engineering to overcome4. Strategies for predicting which enzyme and small molecule are compatible have been hindered by the lack of well-studied biocatalytic reaction datasets5. The underexploration of connections between chemical and protein sequence space constrains navigation between these two landscapes. Here we report a two-phase effort relying on high-throughput experimentation to populate connections between productive substrate and enzyme pairs and the subsequent development of a tool, CATNIP, for predicting compatible α-ketoglutarate (α-KG)/Fe(ii)-dependent enzymes for a given substrate or, conversely, for ranking potential substrates for a given&nbsp;α-KG/Fe(ii)-dependent enzyme sequence. We anticipate that our approach can be readily expanded to further enzyme and transformation classes and will derisk the investigation and application of biocatalytic methods. A two-phase machine-learning-based tool making use of high-throughput experimentation is introduced to examine the connections between chemical and protein sequence space and predict productive biocatalytic reactions among substrate and enzyme pairs.},
  archive      = {J_NATURE},
  author       = {Paton, Alexandra E. and Boiko, Daniil A. and Perkins, Jonathan C. and Cemalovic, Nicholas I. and Reschützegger, Thiago and Gomes, Gabe and Narayan, Alison R. H.},
  doi          = {10.1038/s41586-025-09519-5},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {108-116},
  shortjournal = {Nature},
  title        = {Connecting chemical and protein sequence space to predict biocatalytic reactions},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the heterogeneous nature of LiF in solid–electrolyte interphases. <em>NATURE</em>, <em>646</em>(8083), 102-107. (<a href='https://doi.org/10.1038/s41586-025-09498-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The electrolyte–electrode interface serves as the foundation for a myriad of chemical and physical processes. In battery chemistry, the formation of a well-known solid–electrolyte interphase (SEI) plays a pivotal role in ensuring the reversible operations of rechargeable lithium-ion batteries (LIBs)1,2. However, characterizing the precise chemical composition of the low crystallinity and highly sensitive SEI presents a formidable challenge3. Here, taking lithium fluoride (LiF)—a widely studied and considered crucial SEI component4–7—as an example, we use 19F solid-state nuclear magnetic resonance (NMR) and identify that LiF formed in SEI (LiFSEI) has fruitful spectroscopy features that originated from the formation of limited LiF–LiH solid solutions: H-rich phase (LiH1−yFy) and F-rich phase (LiF1−xHx), which is further validated by 6Li isotope NMR, synchrotron X-ray diffraction and cryo-electron microscopy (cryo-EM). By characterizing SEI formed in various electrolytes, we confirm the dominance of LiH1−yFy in high-coulombic-efficiency electrolyte, which can be rationalized by the fact that LiF–LiH solid solution shows improved ionic conductivity over LiF. As a proof of concept, we demonstrate that LiH1−yFy-rich coating layer presents obvious advantages compared with LiF-rich coating layer in lithium-metal batteries. This revised understanding of the heterogeneous nature of SEI components would provide new insights for electrode–electrolyte interface design. Using 19F nuclear magnetic resonance to study LiF–LiH solid solutions revealed that it is present in the solid–electrolyte interphase of lithium metal batteries and confirms the heterogeneous nature of LiF in such interphases.},
  archive      = {J_NATURE},
  author       = {Liu, Xiangsi and Li, Shuyang and Yuan, Chen and Zheng, Bizhu and Cheng, Gangya and Chen, Yufan and Lu, Xingyu and Gu, Danyu and Lv, Baijiang and Li, Hao and Yan, Zihan and Qian, Hui and Zhu, Yizhou and Sun, Dalin and Song, Yun and Xiang, Yuxuan},
  doi          = {10.1038/s41586-025-09498-7},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {102-107},
  shortjournal = {Nature},
  title        = {Probing the heterogeneous nature of LiF in solid–electrolyte interphases},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toughened self-assembled monolayers for durable perovskite solar cells. <em>NATURE</em>, <em>646</em>(8083), 95-101. (<a href='https://doi.org/10.1038/s41586-025-09509-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hole-selective self-assembled monolayers (SAMs)1,2 have played a key role in driving the certified power conversion efficiency (PCE) of inverted perovskite solar cells3–5 to 26.7% (ref. 6). However, their instability often compromises the operational performance of devices, strongly hindering their practical applications7,8. Here we employ a cross-linkable co-SAM to enhance the conformational stability of hole-selective SAMs against external stresses, while suppressing the formation of defects and voids in SAM during self-assembly. The azide-containing SAM can be thermally activated to form a cross-linked and densely assembled co-SAM with a thermally stable conformation and preferred orientation. This effectively minimizes substrate surface exposure caused by wiggling of loose SAMs under thermal stress, preventing perovskite decomposition. This enables a certified PCE of 26.92% to be achieved for the best-performing cell, which also possesses excellent thermal stability with negligible decay under maximum-power-point tracking at 85 °C for 1,000 h. It also retains &gt;98% of initial PCE after 700 repetitive thermal cycles between −40 °C and 85 °C, representing the state of the art of the field. This work offers an in-depth understanding of SAM degradation mechanisms to guide the design of a&nbsp;more robust buried interface for SAM-based devices adopting high-roughness substrates to realize highly efficient and durable perovskite solar cells. Cross-linkable co-SAMs improve hole-selective SAM stability, preventing defects and thermal degredation in perovskite solar cells, enabling 26.92% efficiency with high heat durability, and guiding the design of more efficient and durable solar cells.},
  archive      = {J_NATURE},
  author       = {Jiang, Wenlin and Qu, Geping and Huang, Xiaofeng and Chen, Xia and Chi, Linyuan and Wang, Tonghui and Wong, Chun-To and Lin, Francis R. and Yang, Chunlei and Jiang, Qing and Wu, Shengfan and Zhang, Jie and Jen, Alex K.-Y.},
  doi          = {10.1038/s41586-025-09509-7},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {95-101},
  shortjournal = {Nature},
  title        = {Toughened self-assembled monolayers for durable perovskite solar cells},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Kirigami-inspired parachutes with programmable reconfiguration. <em>NATURE</em>, <em>646</em>(8083), 88-94. (<a href='https://doi.org/10.1038/s41586-025-09515-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The art of kirigami allows programming a sheet to deform into a particular manner with a pattern of cuts, endowing it with exotic mechanical properties and behaviours1–17. Here we program discs to deform into stably falling parachutes as they deploy under fluid–structure interaction. Parachutes are expensive and delicate to manufacture, which limits their use for humanitarian airdrops or drone delivery. Laser cutting a closed-loop kirigami pattern18 in a disc induces porosity and flexibility into an easily fabricated parachute. By performing wind tunnel testing and numerical simulations using a custom flow-induced reconfiguration model19, we develop a design tool to realize kirigami-inspired parachutes. Guided by these results, we fabricate parachutes from the centimetre to the metre scale and test them in realistic conditions. We show that at low load-to-area ratios, kirigami-inspired parachutes exhibit a comparable terminal velocity to conventional ones. However, unlike conventional parachutes that require a gliding angle for vertical stability20 and fall at random far from a target, our kirigami-inspired parachutes always fall near the target, regardless of their initial release angle. These kinds of parachutes could limit material losses during airdropping as well as decrease manufacturing costs and complexity. A thin planar disc designed with appropriately patterned cuts transforms itself, due to air flow effects, into an effective parachute exhibiting good positional stability, regardless of its initial orientation.},
  archive      = {J_NATURE},
  author       = {Lamoureux, Danick and Fillion, Jérémi and Ramananarivo, Sophie and Gosselin, Frédérick P. and Melancon, David},
  doi          = {10.1038/s41586-025-09515-9},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {88-94},
  shortjournal = {Nature},
  title        = {Kirigami-inspired parachutes with programmable reconfiguration},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Industry-compatible silicon spin-qubit unit cells exceeding 99% fidelity. <em>NATURE</em>, <em>646</em>(8083), 81-87. (<a href='https://doi.org/10.1038/s41586-025-09531-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Among the many types of qubit presently being investigated for a future quantum computer, silicon spin qubits with millions of qubits on a single chip are uniquely positioned to enable quantum computing. However, it has not been clear whether the outstanding high-fidelity operations and long coherence times shown by silicon spin qubits fabricated in academic settings1–8 can be reliably reproduced when the qubits are manufactured in a semiconductor foundry9–11. Here we show precise qubit operation of silicon two-qubit devices made with standard semiconductor tooling in a 300-mm foundry environment. Of the key metrics, single- and two-qubit control fidelities exceed 99% for all four devices, and the state preparation and measurement fidelities reach up to 99.9%, as evidenced by gate set tomography. We report spin lifetime and coherence up to T1 = 9.5 s, $${T}_{2}^{* }=40.6\,{\rm{\mu }}{\rm{s}}$$ and $${T}_{2}^{{\rm{Hahn}}}=1.9\,{\rm{ms}}$$ . We determine that residual nuclear spin-carrying isotopes contribute substantially to operational errors, identifying further isotopic purification as a clear pathway to even higher performance. Two-qubit operations exceeding 99% fidelity have been demonstrated by silicon devices made with standard semiconductor tooling in a 300-mm foundry environment.},
  archive      = {J_NATURE},
  author       = {Steinacker, Paul and Dumoulin Stuyck, Nard and Lim, Wee Han and Tanttu, Tuomo and Feng, MengKe and Serrano, Santiago and Nickl, Andreas and Candido, Marco and Cifuentes, Jesus D. and Vahapoglu, Ensar and Bartee, Samuel K. and Hudson, Fay E. and Chan, Kok Wai and Kubicek, Stefan and Jussot, Julien and Canvel, Yann and Beyne, Sofie and Shimura, Yosuke and Loo, Roger and Godfrin, Clement and Raes, Bart and Baudot, Sylvain and Wan, Danny and Laucht, Arne and Yang, Chih Hwan and Saraiva, Andre and Escott, Christopher C. and De Greve, Kristiaan and Dzurak, Andrew S.},
  doi          = {10.1038/s41586-025-09531-9},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {81-87},
  shortjournal = {Nature},
  title        = {Industry-compatible silicon spin-qubit unit cells exceeding 99% fidelity},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spin squeezing in an ensemble of nitrogen–vacancy centres in diamond. <em>NATURE</em>, <em>646</em>(8083), 74-80. (<a href='https://doi.org/10.1038/s41586-025-09524-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spin-squeezed states provide a seminal example of how the structure of quantum mechanical correlations can be controlled to produce metrologically useful entanglement1–7. These squeezed states have been demonstrated in a wide variety of quantum systems ranging from atoms in optical cavities to trapped ion crystals8–16. By contrast, despite their numerous advantages as practical sensors, spin ensembles in solid-state materials have yet to be controlled with sufficient precision to generate targeted entanglement such as spin squeezing. Here we report the experimental demonstration of spin squeezing in a solid-state spin system. Our experiments are performed on a strongly interacting ensemble of nitrogen–vacancy colour centres in diamond at room temperature, and squeezing (−0.50 ± 0.13 dB) below the noise of uncorrelated spins is generated by the native magnetic dipole–dipole interaction between nitrogen–vacancy centres. To generate and detect squeezing in a solid-state spin system, we overcome several challenges. First, we develop an approach, using interaction-enabled noise spectroscopy, to characterize the quantum projection noise in our system without directly resolving the spin probability distribution. Second, noting that the random positioning of spin defects severely limits the generation of spin squeezing, we implement a pair of strategies aimed at isolating the dynamics of a relatively ordered sub-ensemble of nitrogen–vacancy centres. Our results open the door to entanglement-enhanced metrology using macroscopic ensembles of optically active spins in solids. An ensemble of spins from nitrogen–vacancy centres in diamonds is prepared in a spin-squeezed state.},
  archive      = {J_NATURE},
  author       = {Wu, Weijie and Davis, Emily J. and Hughes, Lillian B. and Ye, Bingtian and Wang, Zilin and Kufel, Dominik and Ono, Tasuku and Meynell, Simon A. and Block, Maxwell and Liu, Che and Yang, Haopu and Bleszynski Jayich, Ania C. and Yao, Norman Y.},
  doi          = {10.1038/s41586-025-09524-8},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {74-80},
  shortjournal = {Nature},
  title        = {Spin squeezing in an ensemble of nitrogen–vacancy centres in diamond},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Signal amplification in a solid-state sensor through asymmetric many-body echo. <em>NATURE</em>, <em>646</em>(8083), 68-73. (<a href='https://doi.org/10.1038/s41586-025-09452-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic spins of nitrogen–vacancy centres in diamond constitute a promising system for micro- and nanoscale magnetic sensing1–4, because of their operation under ambient conditions5, ease of placement in close proximity to sensing targets6 and biological compatibility7. At high densities, the electronic spins interact through dipolar coupling, which typically limits8 but can also potentially enhance9 sensing performance. Here we report the experimental demonstration of many-body signal amplification in a solid-state, room-temperature quantum sensor. Our approach uses time-reversed two-axis-twisting interactions, engineered through dynamical control of the quantization axis and Floquet engineering10 in a two-dimensional ensemble of nitrogen–vacancy centres. We observe that optimal amplification occurs when the backward evolution time equals twice the forward evolution time, in sharp contrast to the conventional Loschmidt echo11,12. These observations can be understood as resulting from an underlying time-reversed mirror symmetry of the microscopic dynamics, providing key insights into signal amplification and opportunities for practical entanglement-enhanced quantum sensing. The experimental demonstration of many-body signal amplification in a solid-state, room-temperature quantum sensor is reported.},
  archive      = {J_NATURE},
  author       = {Gao, Haoyang and Martin, Leigh S. and Hughes, Lillian B. and Leitao, Nathaniel T. and Put, Piotr and Zhou, Hengyun and Koyluoglu, Nazli U. and Meynell, Simon A. and Jayich, Ania C. Bleszynski and Park, Hongkun and Lukin, Mikhail D.},
  doi          = {10.1038/s41586-025-09452-7},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {68-73},
  shortjournal = {Nature},
  title        = {Signal amplification in a solid-state sensor through asymmetric many-body echo},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Late fluid flow in a primitive asteroid revealed by Lu–Hf isotopes in ryugu. <em>NATURE</em>, <em>646</em>(8083), 62-67. (<a href='https://doi.org/10.1038/s41586-025-09483-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Carbonaceous asteroids are the source of the most primitive meteorites1 and represent leftover planetesimals that formed from ice and dust in the outer Solar System and may have delivered volatiles to the terrestrial planets2–5. Understanding the aqueous activity of asteroids is key to deciphering their thermal, chemical and orbital evolution, with implications for the origin of water on the terrestrial planets. Analyses of the objects, in particular pristine samples returned from asteroid Ryugu, have provided detailed information on fluid–rock interactions within a few million years after parent-body formation6–11. However, the long-term fate of asteroidal water remains poorly understood. Here we present evidence for fluid flow in a carbonaceous asteroid more than 1 billion years after formation, based on the 176Lu–176Hf decay systematics of Ryugu samples, which reflect late lutetium mobilization. Such late fluid flow was probably triggered by an impact that generated heat for ice melting and opened rock fractures for fluid migration. This contrasts the early aqueous activity powered by short-lived radioactive decay, with limited fluid flow and little elemental fractionation12. Our results imply that carbonaceous planetesimals accreted by the terrestrial planets could have retained not only hydrous minerals but also aqueous water, leading to an upwards revision of the inventory of their water delivery by a factor of two to three. Analysis of samples from the asteroid Ryugu provide evidence of late fluid flow in a carbonaceous asteroid, indicating that such bodies may have retained two to three times more water than previously thought.},
  archive      = {J_NATURE},
  author       = {Iizuka, Tsuyoshi and Shibuya, Takazo and Hayakawa, Takehito and Yokoyama, Tetsuya and Gautam, Ikshu and Haba, Makiko K. and Ito, Kengo T. M. and Hibiya, Yuki and Yamaguchi, Akira and Abe, Yoshinari and Aléon, Jérôme and Alexander, Conel M. O’D. and Amari, Sachiko and Amelin, Yuri and Bajo, Ken-ichi and Bizzarro, Martin and Bouvier, Audrey and Carlson, Richard W. and Chaussidon, Marc and Choi, Byeon-Gak and Dauphas, Nicolas and Davis, Andrew M. and Di Rocco, Tommaso and Fujiya, Wataru and Fukai, Ryota and Hidaka, Hiroshi and Homma, Hisashi and Huss, Gary R. and Ireland, Trevor R. and Ishikawa, Akira and Itoh, Shoichi and Kawasaki, Noriyuki and Kita, Noriko T. and Kitajima, Koki and Kleine, Thorsten and Komatani, Shintaro and Krot, Alexander N. and Liu, Ming-Chang and Masuda, Yuki and Motomura, Kazuko and Moynier, Frédéric and Nagashima, Kazuhide and Nakai, Izumi and Nguyen, Ann and Nittler, Larry and Pack, Andreas and Park, Changkun and Piani, Laurette and Qin, Liping and Russell, Sara and Sakamoto, Naoya and Schönbächler, Maria and Tafla, Lauren and Tang, Haolan and Terada, Kentaro and Terada, Yasuko and Usui, Tomohiro and Wada, Sohei and Wadhwa, Meenakshi and Walker, Richard J. and Yamashita, Katsuyuki and Yin, Qing-Zhu and Yoneda, Shigekazu and Yui, Hiroharu and Zhang, Ai-Cheng and Nakamura, Tomoki and Naraoka, Hiroshi and Noguchi, Takaaki and Okazaki, Ryuji and Sakamoto, Kanako and Yabuta, Hikaru and Abe, Masanao and Miyazaki, Akiko and Nakato, Aiko and Nishimura, Masahiro and Okada, Tatsuaki and Yada, Toru and Yogata, Kasumi and Nakazawa, Satoru and Saiki, Takanao and Tanaka, Satoshi and Terui, Fuyuto and Tsuda, Yuichi and Watanabe, Sei-ichiro and Yoshikawa, Makoto and Tachibana, Shogo and Yurimoto, Hisayoshi},
  doi          = {10.1038/s41586-025-09483-0},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {62-67},
  shortjournal = {Nature},
  title        = {Late fluid flow in a primitive asteroid revealed by Lu–Hf isotopes in ryugu},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stratified wind from a super-eddington X-ray binary is slower than expected. <em>NATURE</em>, <em>646</em>(8083), 57-61. (<a href='https://doi.org/10.1038/s41586-025-09495-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accretion disks in strong gravity ubiquitously produce winds, seen as blueshifted absorption lines in the X-ray band of both stellar mass X-ray binaries (black holes and neutron stars)1–4 and supermassive black holes5. Some of the most powerful winds (termed Eddington winds) are expected to arise from systems in which radiation pressure is sufficient to unbind material from the inner disk (L ≳ LEdd). These winds should be extremely fast and carry a large amount of kinetic power, which, when associated with supermassive black holes, would make them a prime contender for the feedback mechanism linking the growth of those black holes with their host galaxies6. Here we show the XRISM Resolve spectrum of the galactic neutron star X-ray binary, GX 13+1, which reveals one of the densest winds ever seen in absorption lines. This Compton-thick wind significantly attenuates the flux, making it appear faint, although it is intrinsically more luminous than usual (L ≳ LEdd). However, the wind is extremely slow, more consistent with the predictions of thermal-radiative winds launched by X-ray irradiation of the outer disk than with the expected Eddington wind driven by radiation pressure from the inner disk. This puts new constraints on the origin of winds from bright accretion flows in binaries, but also highlights the very different origin required for the ultrafast (v ~ 0.3c) winds seen in recent Resolve observations of a supermassive black hole at a similarly high Eddington ratio7. The XRISM Resolve spectrum of the galactic neutron star X-ray binary, GX 13+1, reveals one of the densest winds ever seen in absorption lines.},
  archive      = {J_NATURE},
  doi          = {10.1038/s41586-025-09495-w},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {57-61},
  shortjournal = {Nature},
  title        = {Stratified wind from a super-eddington X-ray binary is slower than expected},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Doughnut of social and planetary boundaries monitors a world out of balance. <em>NATURE</em>, <em>646</em>(8083), 47-56. (<a href='https://doi.org/10.1038/s41586-025-09385-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The doughnut-shaped framework of social and planetary boundaries (the ‘Doughnut’) provides a concise visual assessment of progress towards the goal of meeting the needs of all people within the means of the living planet1–3. Here we present a renewed Doughnut framework with a revised set of 35 indicators that monitor trends in social deprivation and ecological overshoot over the 2000–2022 period. Although global gross domestic product (GDP) has more than doubled, our median results show a modest achievement in reducing human deprivation that would have to accelerate fivefold to meet the needs of all people by 2030. Meanwhile, the increase in ecological overshoot would have to stop immediately and accelerate nearly two times faster towards planetary boundaries to safeguard Earth-system stability by 2050. Disaggregating these global findings shows that the richest 20% of nations, with 15% of the global population, contribute more than 40% of annual ecological overshoot, whereas the poorest 40% of countries, with 42% of the global population, experience more than 60% of the social shortfall. These trends and inequalities reaffirm the case for overcoming the dependence of nations on perpetual GDP growth4,5 and reorienting towards regenerative and distributive economic activity—within and between nations—that assigns priority to human needs and planetary integrity. A revised ‘Doughnut’ providing a visual assessment of trends in social deprivation and planetary degradation over the past two decades shows more than doubling of global GDP accompanied by accelerating ecological overshoot but only a modest reduction in human deprivation.},
  archive      = {J_NATURE},
  author       = {Fanning, Andrew L. and Raworth, Kate},
  doi          = {10.1038/s41586-025-09385-1},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {47-56},
  shortjournal = {Nature},
  title        = {Doughnut of social and planetary boundaries monitors a world out of balance},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rare skeletal condition caused by enzyme’s failure to rescue a catalytic cycle. <em>NATURE</em>, <em>646</em>(8083), 45-46. (<a href='https://doi.org/10.1038/d41586-025-02428-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When enzymes require rescue strategies to complete progression through their reaction, but natural safety nets fail, disease states can ensue. When enzymes require rescue strategies to complete progression through their reaction, but natural safety nets fail, disease states can ensue.},
  archive      = {J_NATURE},
  author       = {Allen, Karen N.},
  doi          = {10.1038/d41586-025-02428-7},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {45-46},
  shortjournal = {Nature},
  title        = {Rare skeletal condition caused by enzyme’s failure to rescue a catalytic cycle},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Parachutes inspired by the japanese art of kirigami. <em>NATURE</em>, <em>646</em>(8083), 43-45. (<a href='https://doi.org/10.1038/d41586-025-02969-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cutting a pattern into a flat disc can transform it into a parachute capable of carrying small payloads, which might be used to deliver humanitarian aid Cutting a pattern into a flat disc can transform it into a parachute capable of carrying small payloads, which might be used to deliver humanitarian aid},
  archive      = {J_NATURE},
  author       = {Brun, Pierre-Thomas},
  doi          = {10.1038/d41586-025-02969-x},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {43-45},
  shortjournal = {Nature},
  title        = {Parachutes inspired by the japanese art of kirigami},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Early efforts to understand the processes underlying bird migration. <em>NATURE</em>, <em>646</em>(8083), 43. (<a href='https://doi.org/10.1038/d41586-025-02977-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists try to explain bird behaviour, and a book delves into scientific paradoxes. Snippets from Nature’s past.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02977-x},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {43},
  shortjournal = {Nature},
  title        = {Early efforts to understand the processes underlying bird migration},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People are more likely to cheat when they delegate tasks to AI. <em>NATURE</em>, <em>646</em>(8083), 42-43. (<a href='https://doi.org/10.1038/d41586-025-02819-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence is moving from being a mere tool to an active partner in decision-making, but when we offload misconduct to machines, who bears the blame? Artificial intelligence is moving from being a mere tool to an active partner in decision-making, but when we offload misconduct to machines, who bears the blame?},
  archive      = {J_NATURE},
  author       = {Suzuki, Shoko},
  doi          = {10.1038/d41586-025-02819-w},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {42-43},
  shortjournal = {Nature},
  title        = {People are more likely to cheat when they delegate tasks to AI},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantum correlations enhance the sensitivity of room-temperature sensors. <em>NATURE</em>, <em>646</em>(8083), 40-42. (<a href='https://doi.org/10.1038/d41586-025-02968-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum sensors based on diamond promise robustness but have limited sensitivity. Generating quantum correlations in the material might be a solution. Quantum sensors based on diamond promise robustness but have limited sensitivity. Generating quantum correlations in the material might be a solution.},
  archive      = {J_NATURE},
  author       = {Scharnagl, Maja S. and Hammerer, Klemens},
  doi          = {10.1038/d41586-025-02968-y},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {40-42},
  shortjournal = {Nature},
  title        = {Quantum correlations enhance the sensitivity of room-temperature sensors},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Childhood eczema linked to mother’s stress during pregnancy. <em>NATURE</em>, <em>646</em>(8083), 39-40. (<a href='https://doi.org/10.1038/d41586-025-02427-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The causes of early-life eczema have been unclear, but evidence indicates that changes to fetal immune cells and sensory neurons during pregnancy play a key part. The causes of early-life eczema have been unclear, but evidence indicates that changes to fetal immune cells and sensory neurons during pregnancy play a key part.},
  archive      = {J_NATURE},
  author       = {Rathore, Abhay P. S. and Abraham, Soman N.},
  doi          = {10.1038/d41586-025-02427-8},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {39-40},
  shortjournal = {Nature},
  title        = {Childhood eczema linked to mother’s stress during pregnancy},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Technology leaders should ‘pay back’ society to support the common good. <em>NATURE</em>, <em>646</em>(8083), 34-37. (<a href='https://doi.org/10.1038/d41586-025-03152-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Media technologies shape individuals’ sensitivity to the world around them. But are algorithms promoting what’s best for people? Media technologies shape individuals’ sensitivity to the world around them. But are algorithms promoting what’s best for people?},
  archive      = {J_NATURE},
  author       = {Almeida, Abilio},
  doi          = {10.1038/d41586-025-03152-y},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {34-37},
  shortjournal = {Nature},
  title        = {Technology leaders should ‘pay back’ society to support the common good},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Public trust in science has declined since COVID — Virologists need to unite around safety standards. <em>NATURE</em>, <em>646</em>(8083), 31-34. (<a href='https://doi.org/10.1038/d41586-025-03151-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Creating international viral biosafety guidelines are key to clearing up confusion, regaining trust and ensuring that essential research continues. Creating international viral biosafety guidelines are key to clearing up confusion, regaining trust and ensuring that essential research continues.},
  archive      = {J_NATURE},
  author       = {Ott, Melanie and Schwartz, Olivier and Sigal, Alex},
  doi          = {10.1038/d41586-025-03151-z},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {31-34},
  shortjournal = {Nature},
  title        = {Public trust in science has declined since COVID — Virologists need to unite around safety standards},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI is dreaming up millions of new materials. are they any good?. <em>NATURE</em>, <em>646</em>(8083), 22-25. (<a href='https://doi.org/10.1038/d41586-025-03147-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Critics slammed attempts by Google, Microsoft and Meta to speed up materials discovery. But behind the hype, there is progress. Critics slammed attempts by Google, Microsoft and Meta to speed up materials discovery. But behind the hype, there is progress.},
  archive      = {J_NATURE},
  author       = {Peplow, Mark},
  doi          = {10.1038/d41586-025-03147-9},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {22-25},
  shortjournal = {Nature},
  title        = {AI is dreaming up millions of new materials. are they any good?},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Six journal rejections and a major rethink: Why i’m happy to admit to my research failures, and you should too. <em>NATURE</em>, <em>646</em>(8083), 10. (<a href='https://doi.org/10.1038/d41586-025-03146-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Outlining the trial-and-error processes that are involved in every research project could help others to become more efficient and paint a more honest picture of life as a researcher. Outlining the trial-and-error processes that are involved in every research project could help others to become more efficient and paint a more honest picture of life as a researcher.},
  archive      = {J_NATURE},
  author       = {Toussaert, Séverine},
  doi          = {10.1038/d41586-025-03146-w},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {10},
  shortjournal = {Nature},
  title        = {Six journal rejections and a major rethink: Why i’m happy to admit to my research failures, and you should too},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). End GDP mania: How the world should really measure prosperity. <em>NATURE</em>, <em>646</em>(8083), 7. (<a href='https://doi.org/10.1038/d41586-025-03144-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The obsession with economic output as a measure of human development puts sustainability on the back burner. Researchers can now help to devise better indicators. The obsession with economic output as a measure of human development puts sustainability on the back burner. Researchers can now help to devise better indicators.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-03144-y},
  journal      = {Nature},
  month        = {10},
  number       = {8083},
  pages        = {7},
  shortjournal = {Nature},
  title        = {End GDP mania: How the world should really measure prosperity},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nbt">NBT - 34</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(9), 1580. (<a href='https://doi.org/10.1038/s41587-025-02804-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02804-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry. <em>NBT</em>, <em>43</em>(9), 1577-1579. (<a href='https://doi.org/10.1038/s41587-025-02803-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breaking the cycle of institutional betrayal and the culture of silence requires decisive actions to promote transparency, accountability and supportive environments in both academia and industry.},
  archive      = {J_NBT},
  author       = {Vogelaar, Alison E. and Mahmoudi, Morteza},
  doi          = {10.1038/s41587-025-02803-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1577-1579},
  shortjournal = {Nature Biotech.},
  title        = {Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Global profiling of protein complex dynamics with an experimental library of protein interaction markers. <em>NBT</em>, <em>43</em>(9), 1562-1576. (<a href='https://doi.org/10.1038/s41587-024-02432-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to systematically monitor protein complex dynamics are needed. We introduce serial ultrafiltration combined with limited proteolysis-coupled mass spectrometry (FLiP–MS), a structural proteomics workflow that generates a library of peptide markers specific to changes in PPIs by probing differences in protease susceptibility between complex-bound and monomeric forms of proteins. The library includes markers mapping to protein-binding interfaces and markers reporting on structural changes that accompany PPI changes. Integrating the marker library with LiP–MS data allows for global profiling of protein–protein interactions (PPIs) from unfractionated lysates. We apply FLiP–MS to Saccharomyces cerevisiae and probe changes in protein complex dynamics after DNA replication stress, identifying links between Spt-Ada-Gcn5 acetyltransferase activity and the assembly state of several complexes. FLiP–MS enables protein complex dynamics to be probed on any perturbation, proteome-wide, at high throughput, with peptide-level structural resolution and informing on occupancy of binding interfaces, thus providing both global and molecular views of a system under study. FLiP–MS uses a library of protein–protein interaction markers to understand protein complex dynamics.},
  archive      = {J_NBT},
  author       = {Dörig, Christian and Marulli, Cathy and Peskett, Thomas and Volkmar, Norbert and Pantolini, Lorenzo and Studer, Gabriel and Paleari, Camilla and Frommelt, Fabian and Schwede, Torsten and de Souza, Natalie and Barral, Yves and Picotti, Paola},
  doi          = {10.1038/s41587-024-02432-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1562-1576},
  shortjournal = {Nature Biotech.},
  title        = {Global profiling of protein complex dynamics with an experimental library of protein interaction markers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A circularly permuted CasRx platform for efficient, site-specific RNA editing. <em>NBT</em>, <em>43</em>(9), 1548-1561. (<a href='https://doi.org/10.1038/s41587-024-02430-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inactive Cas13 orthologs have been fused to a mutant human ADAR2 deaminase domain at the C terminus to enable programmable adenosine-to-inosine (A-to-I) RNA editing in selected transcripts. Although promising, existing RNA-editing tools generally suffer from a trade-off between efficacy and specificity, and off-target editing remains an unsolved problem. Here we describe the development of an optimized RNA-editing platform by rational protein engineering, CasRx-based Programmable Editing of RNA Technology (xPERT). We demonstrate that the topological rearrangement of a CasRx K940L mutant by circular permutation results in a robust scaffold for the tethering of a deaminase domain. We benchmark our tool against the REPAIR system and show that xPERT exhibits strong on-target activity like REPAIRv1 but low off-target editing like REPAIRv2. Our xPERT platform can be used to alter RNA sequence information without risking genome damage, effect temporary cellular changes and customize protein function. Site-specific and efficient RNA editing is achieved with a CasRx editor.},
  archive      = {J_NBT},
  author       = {Wang, Yuanming and Liu, Kaiwen Ivy and Liu, Mengying Mandy and Ooi, Kean Hean and Nguyen, Tram Anh and Chee, Jiunn En and Teo, Shun Xiang Danny and He, Shan and Tay, Jie Wen Douglas and Teo, Seok Yee and Liew, Kai Shin and Ge, Xiao Yu and Ng, Zhi Jian and Avagyan, Hasmik and Liu, Hao and Yi, Zirong and Chang, Keziah and Kok, Eng Piew Louis and Chen, Runjia and Yau, Chun En and Koh, Jun Wei and Wan, Yue and Tan, Meng How},
  doi          = {10.1038/s41587-024-02430-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1548-1561},
  shortjournal = {Nature Biotech.},
  title        = {A circularly permuted CasRx platform for efficient, site-specific RNA editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One-step nanoscale expansion microscopy reveals individual protein shapes. <em>NBT</em>, <em>43</em>(9), 1539-1547. (<a href='https://doi.org/10.1038/s41587-024-02431-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The attainable resolution of fluorescence microscopy has reached the subnanometer range, but this technique still fails to image the morphology of single proteins or small molecular complexes. Here, we expand the specimens at least tenfold, label them with conventional fluorophores and image them with conventional light microscopes, acquiring videos in which we analyze fluorescence fluctuations. One-step nanoscale expansion (ONE) microscopy enables the visualization of the shapes of individual membrane and soluble proteins, achieving around 1-nm resolution. We show that conformational changes are readily observable, such as those undergone by the ~17-kDa protein calmodulin upon Ca2+ binding. ONE is also applied to clinical samples, analyzing the morphology of protein aggregates in cerebrospinal fluid from persons with Parkinson disease, potentially aiding disease diagnosis. This technology bridges the gap between high-resolution structural biology techniques and light microscopy, providing new avenues for discoveries in biology and medicine. Combining expansion microscopy with super-resolution radial fluctuations captures the morphology of single proteins.},
  archive      = {J_NBT},
  author       = {Shaib, Ali H. and Chouaib, Abed Alrahman and Chowdhury, Rajdeep and Altendorf, Jonas and Mihaylov, Daniel and Zhang, Chi and Krah, Donatus and Imani, Vanessa and Spencer, Russell K. W. and Georgiev, Svilen Veselinov and Mougios, Nikolaos and Monga, Mehar and Reshetniak, Sofiia and Mimoso, Tiago and Chen, Han and Fatehbasharzad, Parisa and Crzan, Dagmar and Saal, Kim-Ann and Alawieh, Mohamad Mahdi and Alawar, Nadia and Eilts, Janna and Kang, Jinyoung and Soleimani, Alireza and Müller, Marcus and Pape, Constantin and Alvarez, Luis and Trenkwalder, Claudia and Mollenhauer, Brit and Outeiro, Tiago F. and Köster, Sarah and Preobraschenski, Julia and Becherer, Ute and Moser, Tobias and Boyden, Edward S. and Aricescu, A. Radu and Sauer, Markus and Opazo, Felipe and Rizzoli, Silvio O.},
  doi          = {10.1038/s41587-024-02431-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1539-1547},
  shortjournal = {Nature Biotech.},
  title        = {One-step nanoscale expansion microscopy reveals individual protein shapes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of compact transcriptional effectors using high-throughput measurements in diverse contexts. <em>NBT</em>, <em>43</em>(9), 1525-1538. (<a href='https://doi.org/10.1038/s41587-024-02442-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional effectors are protein domains known to activate or repress gene expression; however, a systematic understanding of which effector domains regulate transcription across genomic, cell type and DNA-binding domain (DBD) contexts is lacking. Here we develop dCas9-mediated high-throughput recruitment (HT-recruit), a pooled screening method for quantifying effector function at endogenous target genes and test effector function for a library containing 5,092 nuclear protein Pfam domains across varied contexts. We also map context dependencies of effectors drawn from unannotated protein regions using a larger library tiling chromatin regulators and transcription factors. We find that many effectors depend on target and DBD contexts, such as HLH domains that can act as either activators or repressors. To enable efficient perturbations, we select context-robust domains, including ZNF705 KRAB, that improve CRISPRi tools to silence promoters and enhancers. We engineer a compact human activator called NFZ, by combining NCOA3, FOXO3 and ZNF473 domains, which enables efficient CRISPRa with better viral delivery and inducible control of chimeric antigen receptor T cells. Improved effectors for CRISPRi/CRISPRa are developed following high-throughput screening of transcriptional domains.},
  archive      = {J_NBT},
  author       = {Tycko, Josh and Van, Mike V. and Aradhana and DelRosso, Nicole and Ye, Hanrong and Yao, David and Valbuena, Raeline and Vaughan-Jackson, Alun and Xu, Xiaoshu and Ludwig, Connor and Spees, Kaitlyn and Liu, Katherine and Gu, Mingxin and Khare, Venya and Mukund, Adi Xiyal and Suzuki, Peter H. and Arana, Sophia and Zhang, Catherine and Du, Peter P. and Ornstein, Thea S. and Hess, Gaelen T. and Kamber, Roarke A. and Qi, Lei S. and Khalil, Ahmad S. and Bintu, Lacramioara and Bassik, Michael C.},
  doi          = {10.1038/s41587-024-02442-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1525-1538},
  shortjournal = {Nature Biotech.},
  title        = {Development of compact transcriptional effectors using high-throughput measurements in diverse contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations. <em>NBT</em>, <em>43</em>(9), 1510-1524. (<a href='https://doi.org/10.1038/s41587-024-02428-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To assist the translation of genetic findings to disease pathobiology and therapeutics discovery, we present an ensemble deep learning framework, termed PIONEER (Protein–protein InteractiOn iNtErfacE pRediction), that predicts protein-binding partner-specific interfaces for all known protein interactions in humans and seven other common model organisms to generate comprehensive structurally informed protein interactomes. We demonstrate that PIONEER outperforms existing state-of-the-art methods and experimentally validate its predictions. We show that disease-associated mutations are enriched in PIONEER-predicted protein–protein interfaces and explore their impact on disease prognosis and drug responses. We identify 586 significant protein–protein interactions (PPIs) enriched with PIONEER-predicted interface somatic mutations (termed oncoPPIs) from analysis of approximately 11,000 whole exomes across 33 cancer types and show significant associations of oncoPPIs with patient survival and drug responses. PIONEER, implemented as both a web server platform and a software package, identifies functional consequences of disease-associated alleles and offers a deep learning tool for precision medicine at multiscale interactome network levels. Protein–protein interactomes incorporating structural data predict the functional consequences of disease mutations.},
  archive      = {J_NBT},
  author       = {Xiong, Dapeng and Qiu, Yunguang and Zhao, Junfei and Zhou, Yadi and Lee, Dongjin and Gupta, Shobhita and Torres, Mateo and Lu, Weiqiang and Liang, Siqi and Kang, Jin Joo and Eng, Charis and Loscalzo, Joseph and Cheng, Feixiong and Yu, Haiyuan},
  doi          = {10.1038/s41587-024-02428-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1510-1524},
  shortjournal = {Nature Biotech.},
  title        = {A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors. <em>NBT</em>, <em>43</em>(9), 1496-1509. (<a href='https://doi.org/10.1038/s41587-024-02448-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies to enhance the anti-tumor immune response of stereotactic ablative radiotherapy (SABR) at primary tumors and abscopal sites are under intensive investigation. Here we report a metabolizable binary supracluster (BSCgal) that combines gold nanoclusters as radiosensitizing adjuvants with small interfering RNA (siRNA) targeting the immunosuppressive mediator galectin-1 (Gal-1). BSCgal comprises reversibly crosslinked cationic gold nanoclusters and siRNA complexes in a polymer matrix that biodegrades over weeks, facilitating clearance (90.3% in vivo clearance at 4 weeks) to reduce toxicity. The particle size well above the renal filtration threshold facilitates passive delivery to tumors. Using mouse models of head and neck cancer, we show that BSCgal augments the radiodynamic and immunotherapeutic effects of SABR at the primary and metastatic tumors by promoting tumor-inhibitory leukocytes, upregulating cytotoxic granzyme B and reducing immunosuppressive cell populations. It outperforms SABR plus Gal-1 antagonists, chemoradiation drug cisplatin or PD-1 inhibitor. This work presents a translatable strategy to converge focal radiosensitization with targeted immune checkpoint silencing for personalized radioimmunotherapy. Gold-siRNA clusters boost the immune response of radiotherapy against primary and distant tumors.},
  archive      = {J_NBT},
  author       = {Jiang, Yuyan and Cao, Hongbin and Deng, Huaping and Guan, Li and Langthasa, Jimpi and Colburg, Deana Rae Crystal and Melemenidis, Stavros and Cotton, Renee M. and Aleman, John and Wang, Xiao-Jing and Graves, Edward E. and Kalbasi, Anusha and Pu, Kanyi and Rao, Jianghong and Le, Quynh-Thu},
  doi          = {10.1038/s41587-024-02448-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1496-1509},
  shortjournal = {Nature Biotech.},
  title        = {Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE. <em>NBT</em>, <em>43</em>(9), 1485-1495. (<a href='https://doi.org/10.1038/s41587-024-02426-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing of messenger RNA (mRNA) found in extracellular vesicles (EVs) in liquid biopsies can provide clinical information such as somatic mutations, resistance profiles and tumor recurrence. Despite this, EV mRNA remains underused due to its low abundance in liquid biopsies, and large sample volumes or specialized techniques for analysis are required. Here we introduce Self-amplified and CRISPR-aided Operation to Profile EVs (SCOPE), a platform for EV mRNA detection. SCOPE leverages CRISPR-mediated recognition of target RNA using Cas13 to initiate replication and signal amplification, achieving a sub-attomolar detection limit while maintaining single-nucleotide resolution. As a proof of concept, we designed probes for key mutations in KRAS, BRAF, EGFR and IDH1 genes, optimized protocols for single-pot assays and implemented an automated device for multi-sample detection. We validated SCOPE’s ability to detect early-stage lung cancer in animal models, monitored tumor mutational burden in patients with colorectal cancer and stratified patients with glioblastoma. SCOPE can expedite readouts, augmenting the clinical use of EVs in precision oncology. SCOPE detects low-abundance extracellular vesicle mRNA with high sensitivity for clinical applications.},
  archive      = {J_NBT},
  author       = {Song, Jayeon and Cho, Mi Hyeon and Cho, Hayoung and Song, Younseong and Lee, Sung Woon and Nam, Ho Chul and Yoon, Tae Ho and Shin, Jong Cheol and Hong, Jae-Sang and Kim, Yejin and Ekanayake, Emil and Jeon, Jueun and You, Dong Gil and Im, Sung Gap and Choi, Gyu-Seog and Park, Jun Seok and Carter, Bob C. and Balaj, Leonora and Seo, An Na and Miller, Miles A. and Park, Soo Yeun and Kang, Taejoon and Castro, Cesar M. and Lee, Hakho},
  doi          = {10.1038/s41587-024-02426-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1485-1495},
  shortjournal = {Nature Biotech.},
  title        = {Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Saturation profiling of drug-resistant genetic variants using prime editing. <em>NBT</em>, <em>43</em>(9), 1471-1484. (<a href='https://doi.org/10.1038/s41587-024-02465-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to characterize the functional effects of genetic variants of uncertain significance (VUSs) have been limited by incomplete coverage of the mutational space. In clinical oncology, drug resistance arising from VUSs can prevent optimal treatment. Here we introduce PEER-seq, a high-throughput method based on prime editing that can evaluate the functional effects of single-nucleotide variants (SNVs). PEER-seq introduces both intended SNVs and synonymous marker mutations using prime editing and deep sequences the endogenous target regions to identify the introduced SNVs. We generate and functionally evaluate 2,476 SNVs in the epidermal growth factor receptor gene (EGFR), including 99% of all possible variants in the canonical tyrosine kinase domain. We determined resistance profiles of 95% of all possible EGFR protein variants encoded in the whole tyrosine kinase domain against the common tyrosine kinase inhibitors afatinib, osimertinib and osimertinib in the presence of the co-occurring substitution T790M, in PC-9 cells. Our study has the potential to substantially improve the precision of therapeutic choices in clinical settings. PEER-seq characterizes the drug resistance profiles of thousands of genetic variants.},
  archive      = {J_NBT},
  author       = {Kim, Younggwang and Oh, Hyeong-Cheol and Lee, Seungho and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-024-02465-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1471-1484},
  shortjournal = {Nature Biotech.},
  title        = {Saturation profiling of drug-resistant genetic variants using prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal scanning of genetic variants with base and prime editing. <em>NBT</em>, <em>43</em>(9), 1458-1470. (<a href='https://doi.org/10.1038/s41587-024-02439-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational scanning connects genetic variants to phenotype, enabling the interrogation of protein functions, interactions and variant pathogenicity. However, current methodologies cannot efficiently engineer customizable sets of diverse genetic variants in endogenous loci across cellular contexts in high throughput. Here, we combine cytosine and adenine base editors and a prime editor to assess the pathogenicity of a broad spectrum of variants in the epithelial growth factor receptor gene (EGFR). Using pooled base editing and prime editing guide RNA libraries, we install tens of thousands of variants spanning the full coding sequence of EGFR in multiple cell lines and assess the role of these variants in tumorigenesis and resistance to tyrosine kinase inhibitors. Our EGFR variant scan identifies important hits, supporting the robustness of the approach and revealing underappreciated routes to EGFR activation and drug response. We anticipate that multimodal precision mutational scanning can be applied broadly to characterize genetic variation in any genetic element of interest at high and single-nucleotide resolution. Thousands of rare oncogene variants are evaluated using multimodal gene editing screens.},
  archive      = {J_NBT},
  author       = {Belli, Olivier and Karava, Kyriaki and Farouni, Rick and Platt, Randall J.},
  doi          = {10.1038/s41587-024-02439-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1458-1470},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal scanning of genetic variants with base and prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein. <em>NBT</em>, <em>43</em>(9), 1445-1457. (<a href='https://doi.org/10.1038/s41587-024-02437-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticle (LNP) delivery of clustered regularly interspaced short palindromic repeat (CRISPR) ribonucleoproteins (RNPs) could enable high-efficiency, low-toxicity and scalable in vivo genome editing if efficacious RNP–LNP complexes can be reliably produced. Here we engineer a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) to generate iGeoCas9 variants capable of &gt;100× more genome editing of cells and organs compared with the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP–LNP complexes edit a variety of cell types and induce homology-directed repair in cells receiving codelivered single-stranded DNA templates. Using tissue-selective LNP formulations, we observe genome-editing levels of 16‒37% in the liver and lungs of reporter mice that receive single intravenous injections of iGeoCas9 RNP–LNPs. In addition, iGeoCas9 RNPs complexed to biodegradable LNPs edit the disease-causing SFTPC gene in lung tissue with 19% average efficiency, representing a major improvement over genome-editing levels observed previously using viral or nonviral delivery strategies. These results show that thermostable Cas9 RNP–LNP complexes can expand the therapeutic potential of genome editing. An engineered clustered regularly interspaced short palindromic repeat ribonucleoprotein delivered in lipid nanoparticles efficiently edits cells in vivo.},
  archive      = {J_NBT},
  author       = {Chen, Kai and Han, Hesong and Zhao, Sheng and Xu, Bryant and Yin, Boyan and Lawanprasert, Atip and Trinidad, Marena and Burgstone, Benjamin W. and Murthy, Niren and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-024-02437-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1445-1457},
  shortjournal = {Nature Biotech.},
  title        = {Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering b cells to treat and study human disease. <em>NBT</em>, <em>43</em>(9), 1431-1444. (<a href='https://doi.org/10.1038/s41587-025-02757-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern cell therapies are designed to harness natural biology to treat a range of complex diseases. The field of immunology has shown that B cells exhibit multiple unique features, including a natural propensity to interact with and regulate other immune cells, a high capacity to produce proteins, and a long cellular lifespan, which are being creatively applied in engineered B cell (eB cell) therapies. In recent years, advances in genome editing technologies and animal modeling have facilitated rapid progress in our ability to study eB cells and execute proof-of-concept studies, thus enabling the first clinical trials of eB cell therapies. In this review, we provide an overview of recent developments in eB cell therapies, including early clinical studies. We discuss challenges to clinical implementation, and promising directions for leveraging B cell biology in future applications for cancer and chronic disease. Recent developments in B cell engineering expand our ability to study human disease and drive progress toward new B cell therapies, balancing challenges in clinical implementation with emerging opportunities.},
  archive      = {J_NBT},
  author       = {Trivedi, Nikita and Pitner, Ragan A. and Rawlings, David J. and James, Richard G.},
  doi          = {10.1038/s41587-025-02757-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1431-1444},
  shortjournal = {Nature Biotech.},
  title        = {Engineering b cells to treat and study human disease},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A CRISPR-based method for detecting mRNA in extracellular vesicles. <em>NBT</em>, <em>43</em>(9), 1428-1429. (<a href='https://doi.org/10.1038/s41587-024-02440-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A CRISPR-based assay both recognizes and amplifies target mRNA, achieving sub-attomolar sensitivity with single-nucleotide resolution. This method enables the detection of low-abundance mRNA in extracellular vesicles, providing clinically relevant information for precision oncology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02440-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1428-1429},
  shortjournal = {Nature Biotech.},
  title        = {A CRISPR-based method for detecting mRNA in extracellular vesicles},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Bao and rogers. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02784-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02784-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bao and rogers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Ntziachristos and yao. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02785-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02785-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Ntziachristos and yao},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI designs de novo antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02822-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02822-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {AI designs de novo antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosted mitochondria reverse cognitive impairment in mice. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02823-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02823-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Boosted mitochondria reverse cognitive impairment in mice},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mining archaea for antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02824-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02824-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Mining archaea for antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MicroRNAs. <em>NBT</em>, <em>43</em>(9), 1424. (<a href='https://doi.org/10.1038/s41587-025-02800-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to pharmaceutical compositions and methods of treatment involving microRNAs.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02800-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1424},
  shortjournal = {Nature Biotech.},
  title        = {MicroRNAs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Intellectual property rights in the cultivated meat industry: Opportunities and challenges. <em>NBT</em>, <em>43</em>(9), 1419-1423. (<a href='https://doi.org/10.1038/s41587-025-02798-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivated meat faces escalating intellectual property (IP) issues as the industry expands, including conventional patent rights, the private IP rights of animal breeders and the Convention on Biological Diversity.},
  archive      = {J_NBT},
  author       = {Baek, Garam and Chung, Sukhyun and Yu, Peter and Park, Yooheon},
  doi          = {10.1038/s41587-025-02798-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1419-1423},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property rights in the cultivated meat industry: Opportunities and challenges},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining druggable targets with artificial intelligence. <em>NBT</em>, <em>43</em>(9), 1416-1418. (<a href='https://doi.org/10.1038/s41587-025-02770-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A vast landscape of ‘undruggable’ cancer targets remains beyond the reach of conventional therapeutic agents. Recent advances in artificial intelligence (AI), however, are challenging this paradigm. Synthesizing insights from a Cancer Moonshot workshop, we argue that systemically addressing the undruggable target space with AI requires a new conceptual framework. We highlight the failure of current target taxonomies and the need for benchmarking datasets, and re-evaluate clinical validation for novel AI-driven modalities.},
  archive      = {J_NBT},
  author       = {Akinsanya, Karen and AlQuraishi, Mohammed and Boija, Ann and Chodera, John and Cichońska, Anna and Ghassemi, Marzyeh and Head, Martha and Jin, Wengong and Kibbe, Warren A. and Krogan, Nevan and LeVine, Michael V. and Moult, John and Stuart, Lynda and Tourassi, Georgia and Zou, James and Barzilay, Regina and Elemento, Olivier},
  doi          = {10.1038/s41587-025-02770-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1416-1418},
  shortjournal = {Nature Biotech.},
  title        = {Redefining druggable targets with artificial intelligence},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Senolytics under scrutiny in the quest to slow aging. <em>NBT</em>, <em>43</em>(9), 1413-1415. (<a href='https://doi.org/10.1038/s41587-025-02740-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de Magalhães, João Pedro},
  doi          = {10.1038/s41587-025-02740-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1413-1415},
  shortjournal = {Nature Biotech.},
  title        = {Senolytics under scrutiny in the quest to slow aging},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Freya biosciences. <em>NBT</em>, <em>43</em>(9), 1411-1412. (<a href='https://doi.org/10.1038/s41587-025-02783-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Freya Biosciences is restoring balance in the vaginal microbiome to improve pregnancy outcomes.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02783-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1411-1412},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Freya biosciences},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Vilya therapeutics. <em>NBT</em>, <em>43</em>(9), 1409-1410. (<a href='https://doi.org/10.1038/s41587-025-02782-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Vilya Therapeutics is designing macrocycle drug candidates.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-025-02782-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1409-1410},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Vilya therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Seaport therapeutics. <em>NBT</em>, <em>43</em>(9), 1407-1408. (<a href='https://doi.org/10.1038/s41587-025-02792-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Seaport Therapeutics is modifying central nervous system drugs for delivery through the lymphatic system, bypassing first-pass liver metabolism.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-025-02792-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1407-1408},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Seaport therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(9), 1406. (<a href='https://doi.org/10.1038/s41587-025-02817-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02817-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1406},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blooms supersized. <em>NBT</em>, <em>43</em>(9), 1405. (<a href='https://doi.org/10.1038/s41587-025-02825-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02825-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1405},
  shortjournal = {Nature Biotech.},
  title        = {Blooms supersized},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wegovy approved for MASH. <em>NBT</em>, <em>43</em>(9), 1404. (<a href='https://doi.org/10.1038/s41587-025-02828-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02828-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1404},
  shortjournal = {Nature Biotech.},
  title        = {Wegovy approved for MASH},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA go-ahead for HPV immunotherapy. <em>NBT</em>, <em>43</em>(9), 1402. (<a href='https://doi.org/10.1038/s41587-025-02827-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02827-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1402},
  shortjournal = {Nature Biotech.},
  title        = {FDA go-ahead for HPV immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Regenerative cell therapy marches into corneal blindness. <em>NBT</em>, <em>43</em>(9), 1401-1404. (<a href='https://doi.org/10.1038/s41587-025-02808-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a newly marketed product in Japan and 10 years of clinical data, regenerative cell therapies promise new options for patients facing a choice between invasive surgery or blindness.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew},
  doi          = {10.1038/s41587-025-02808-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1401-1404},
  shortjournal = {Nature Biotech.},
  title        = {Regenerative cell therapy marches into corneal blindness},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). US government slams mRNA vaccine research. <em>NBT</em>, <em>43</em>(9), 1400. (<a href='https://doi.org/10.1038/s41587-025-02826-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02826-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1400},
  shortjournal = {Nature Biotech.},
  title        = {US government slams mRNA vaccine research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mitochondrial transplants make healthy babies. <em>NBT</em>, <em>43</em>(9), 1399-1400. (<a href='https://doi.org/10.1038/s41587-025-02799-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After decades of research and consultation with regulators, mitochondrial donation is taking off in the UK. Will other countries follow?},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-025-02799-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1399-1400},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial transplants make healthy babies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing antivirals. <em>NBT</em>, <em>43</em>(9), 1397-1398. (<a href='https://doi.org/10.1038/s41587-025-02829-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Funding and public support for vaccines in the United States is at an all-time low. Promising research is resulting in effective antivirals, but they should not take the place of vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02829-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1397-1398},
  shortjournal = {Nature Biotech.},
  title        = {Advancing antivirals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nchem">NCHEM - 24</h2>
<ul>
<li><details>
<summary>
(2025). Harnessing light the retinal way. <em>NCHEM</em>, <em>17</em>(10), 1618. (<a href='https://doi.org/10.1038/s41557-025-01924-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Yasuhisa Mizutani marvels at how effectively nature utilizes the π-conjugated system of retinal in proteins.},
  archive      = {J_NCHEM},
  author       = {Mizutani, Yasuhisa},
  doi          = {10.1038/s41557-025-01924-7},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1618},
  shortjournal = {Nat. Chem.},
  title        = {Harnessing light the retinal way},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lattice O–O ligands in fe-incorporated hydroxides enhance water oxidation electrocatalysis. <em>NCHEM</em>, <em>17</em>(10), 1607-1614. (<a href='https://doi.org/10.1038/s41557-025-01898-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the structural dynamics of ligands and their interaction with catalytic centres under reaction conditions remains a fundamental challenge, yet it is essential for catalyst design. Here we reveal an in situ transformation of Ni–Fe hydroxide into a stable superoxo-hydroxide phase, which is accompanied by the formation of lattice O–O (Olatt–Olatt) ligands, as demonstrated using operando 18O-labelling spectroelectrochemistry and machine-learning-assisted global optimization. By correlating the intrinsic activity of Fe with the Olatt–Olatt concentration across a series of Fe-incorporated transition-metal hydroxides and oxides, we demonstrate that Olatt–Olatt triggers Fe activation for oxygen evolution electrocatalysis—a finding further supported by first-principles calculations. Oxygen production proceeds via an adsorbate evolution mechanism, and the enhanced reaction kinetics stem from the lowered activation energy at surface Fe sites in the newly formed superoxo-hydroxide structure. This work offers a strategic framework for designing high-performance Fe-incorporated electrocatalysts and underscores the pivotal role of ligand dynamics in activating catalytic centres. While the ligand coordination microenvironment surrounding catalytic centres influences reactivity, dynamic oxygen reconstruction during water oxidation electrocatalysis complicates structure-based mechanistic insights. Now the in situ formation of lattice O–O ligands has been shown to activate Fe centres in metal oxides and hydroxides, thereby enhancing their oxygen evolution reaction activity.},
  archive      = {J_NCHEM},
  author       = {Shi, Guoshuai and Li, Jili and Lu, Tingyu and Yang, Chunlei and Xu, Qinshang and Gu, Huoliang and Tong, Haonan and Zhao, Siwen and Zhu, Chenyuan and Shen, Yuluo and Wu, Jing and Lao, Xianzhuo and Chen, Peng-Cheng and Li, Jiong and Zhang, Shuo and Shi, Jueli and Zhang, Kelvin H. L. and Li, Ye-Fei and Liu, Zhi-Pan and Zhang, Liming},
  doi          = {10.1038/s41557-025-01898-6},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1607-1614},
  shortjournal = {Nat. Chem.},
  title        = {Lattice O–O ligands in fe-incorporated hydroxides enhance water oxidation electrocatalysis},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Immobilized acyl-transfer molecular reactors enable the solid-phase synthesis of sterically hindered peptides. <em>NCHEM</em>, <em>17</em>(10), 1596-1606. (<a href='https://doi.org/10.1038/s41557-025-01896-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Incorporating unnatural amino acids, such as hindered N-methylated or α,α-disubstituted amino acid(s), into peptides can improve their properties for application in the pharmaceutical and biomedical fields. However, the current solid-phase peptide synthesis (SPPS) faces sluggish reaction rates and low yields when incorporating sterically hindered amino acids, owing to the poor kinetics of the two-phase acyl-transfer process from solution to solid. Here we introduce an immobilized ribosome-mimicking molecular reactor to facilitate on-resin proximity-induced intra(inter)-reactor acyl transfers. This strategy bypasses the two-phase acyl-transfer mechanism in conventional SPPS and boosts coupling efficiency in the solid-phase synthesis of N-methylated and/or α,α-disubstituted amino acid(s)-containing sterically hindered peptides, including cyclosporin A and alamethicin F analogues. The ribosome-mimicking molecular reactor SPPS can be integrated into existing SPPS platforms using commercially available resins and reagents, and displays high compatibility with standard synthesizers, enabling the automated synthesis of pharmaceutically important, sterically hindered difficult peptides. Current solid-phase peptide synthesis methods struggle to incorporate sterically hindered amino acids. Now researchers have developed a ribosome-mimicking molecular reactor that bypasses two-phase acyl transfer to boost the coupling efficiency of peptides containing N-methylated and/or α,α-disubstituted amino acids.},
  archive      = {J_NCHEM},
  author       = {Wei, Siyuan and Zhang, Xuchun and Yang, Xiaoliang and Liu, Fa and Yao, Zhu-Jun},
  doi          = {10.1038/s41557-025-01896-8},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1596-1606},
  shortjournal = {Nat. Chem.},
  title        = {Immobilized acyl-transfer molecular reactors enable the solid-phase synthesis of sterically hindered peptides},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stepwise and reversible assembly of [2Fe–2S] rhombs to [8Fe–8S] clusters and their topological interconversions. <em>NCHEM</em>, <em>17</em>(10), 1586-1595. (<a href='https://doi.org/10.1038/s41557-025-01895-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Among all enzymatic metallocofactors, those found in nitrogenases, the P and L or M clusters, stand out for their intricate structures. They are assembled by proteins of the Nif gene cluster from Fe2S2 rhombs—the smallest building blocks in FeS cluster chemistry—through a sequence of reactions constructing a Fe8S8 precursor. To advance our understanding of how enzymes selectively build such elaborate inorganic molecules, here we parallel the biosynthetic pathway by reporting the rational stepwise assembly of [Fe8S8]m+ (m = 2, 4, 6) clusters from [Fe2S2]2+ rhombs within an extensive cyclic synthetic network. A [Fe8S8]4+ cluster of unique topology is identified, for which we coin the term ‘interlocked’ double cubane. As a molecular analogue of the NifB K cluster, a proposed precursor to both the P and L or M clusters, its preparation and the characterization of all related intermediates, offers fundamental insights into the molecular mechanisms governing the assembly of both biogenic and synthetic FeS clusters. Elucidating the nature of the metallocofactors in nitrogenase enzymes, and preparing synthetic analogues of these clusters, is a classic target for bioinorganic chemists. Now the transformation of [Fe2S2]2+ rhombs to [Fe8S8]n+ clusters has been achieved through a series of redox- and ligand-substitution reactions.},
  archive      = {J_NCHEM},
  author       = {Grunwald, Liam and Weber, Micha L. and Seng, Henrik and Clémancey, Martin and Dubourdeaux, Patrick and Wang, Hongxin and Wörle, Michael and Yoda, Yoshitaka and Cramer, Stephen P. and Blondin, Geneviève and Mougel, Victor},
  doi          = {10.1038/s41557-025-01895-9},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1586-1595},
  shortjournal = {Nat. Chem.},
  title        = {Stepwise and reversible assembly of [2Fe–2S] rhombs to [8Fe–8S] clusters and their topological interconversions},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unlocking azole chemical space via modular and regioselective N-alkylation. <em>NCHEM</em>, <em>17</em>(10), 1576-1585. (<a href='https://doi.org/10.1038/s41557-025-01891-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Azoles are important synthetic targets due to their diverse applications in areas ranging from human health to food security. Accordingly, access to N-functionalized azoles is an essential goal in modern synthetic chemistry. Surprisingly, however, the relied-upon azole N-alkylation strategies fundamentally limit the structural diversity of these important compounds that can be synthesized and studied. Here we introduce an approach to prepare a broad array of important but difficult-to-access N-alkyl azole compounds. We accomplish this through the introduction of a base-catalysed hydroazolation of readily accessible alkenylthianthrenium electrophiles. This strategy circumvents the classical challenge of azole alkylation regiocontrol through an unusual reversible C–N-bond-forming step that exploits the thermodynamic differences between azole N-alkylation isomers. This reaction furnishes a class of versatile azolothianthrenium building blocks that provides a general platform to investigate diverse N-alkyl azole molecules. More broadly, the distinctive approach outlined through this project is poised to impact the design and development of diverse regioselective alkylation reactions. Azoles are important synthetic targets due to their diverse applications in areas ranging from human health to food security. Now it has been shown that the hydroazolation of alkenylthianthrenium salts provides a modular platform to access diverse, densely functionalized N-alkyl azole compounds with high N-regioselectivity.},
  archive      = {J_NCHEM},
  author       = {Dorval, Céline and Matthews, Adrian D. and Targos, Karina and Alektiar, Sara N. and Holst, Dylan E. and Tan, Zhifeng and Muuronen, Mikko and Diccianni, Justin B. and Gómez, José Enrique and Sanders, Kyana M. and Guzei, Ilia A. and Wickens, Zachary K.},
  doi          = {10.1038/s41557-025-01891-z},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1576-1585},
  shortjournal = {Nat. Chem.},
  title        = {Unlocking azole chemical space via modular and regioselective N-alkylation},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput discovery of fluoroprobes that recognize amyloid fibril polymorphs. <em>NCHEM</em>, <em>17</em>(10), 1565-1575. (<a href='https://doi.org/10.1038/s41557-025-01889-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aggregation of microtubule-associated protein tau into conformationally distinct fibrils underpins neurodegenerative tauopathies. Fluorescent probes (fluoroprobes) such as thioflavin T have been essential tools for studying tau aggregation; however, most of them do not discriminate between amyloid fibril conformations (polymorphs). This gap is due, in part, to a lack of high-throughput methods for screening large, diverse chemical collections. Here we leverage advances in protein-adaptive differential scanning fluorimetry to screen the Aurora collection of 300+ fluoroprobes against multiple synthetic fibril polymorphs, including those formed from tau, α-synuclein and islet amyloid polypeptide. This screen—coupled with excitation-multiplexed bright-emission recording (EMBER) imaging and orthogonal secondary assays—revealed pan-fibril-binding chemotypes, as well as fluoroprobes selective for fibril subsets. One fluoroprobe recognized tau pathology in ex vivo brain slices from Alzheimer’s disease and rodent models. We propose that these scaffolds represent entry points for developing fibril-selective ligands. Amyloid fibrils can adopt a range of distinct conformations, yet it is challenging to rapidly discriminate between these polymorphs. Now methods have been developed to screen large, diverse libraries of turn-on fluorescent dyes to rapidly identify probes that recognize fibril subsets.},
  archive      = {J_NCHEM},
  author       = {Carroll, Emma C. and Yang, Hyunjun and Powell, Wyatt C. and Charvat, Annemarie F. and Oehler, Abby and Jones, Julia G. and Montgomery, Kelly M. and Yung, Anthony and Millbern, Zoe and Taylor, Alexander I. P. and Wilkinson, Martin and Ranson, Neil A. and Radford, Sheena E. and Vinueza, Nelson R. and DeGrado, William F. and Mordes, Daniel A. and Condello, Carlo and Gestwicki, Jason E.},
  doi          = {10.1038/s41557-025-01889-7},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1565-1575},
  shortjournal = {Nat. Chem.},
  title        = {High-throughput discovery of fluoroprobes that recognize amyloid fibril polymorphs},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deaminative giese-type reaction. <em>NCHEM</em>, <em>17</em>(10), 1556-1564. (<a href='https://doi.org/10.1038/s41557-025-01888-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Primary aliphatic amines are essential components in numerous functional molecules and rank among the most readily available commercial building blocks. Although commonly utilized as nitrogen nucleophiles, their application as alkyl sources for constructing (sp3)C–C(sp3) bonds remains a notable challenge. Here we present an approach integrating nitrogen-atom deletion into the aza-Michael reaction, thereby redirecting the classical pathway from (sp3)C–N bond formation to (sp3)C–C(sp3) bond construction. Leveraging commercially available O-diphenylphosphinylhydroxylamine as an efficient nitrogen-deletion reagent, this method enables a wide variety of primary aliphatic amines to serve as alkyl sources in couplings with structurally diverse electron-deficient olefins. This Giese-type reaction proceeds under mild conditions, achieves completion within 10 min and exhibits broad functional-group compatibility. By bridging two foundational transformations—the aza-Michael reaction and the Giese-type reaction—this approach interlinks their product spaces through a unified precursor library, substantially enhancing synthetic utility. Primary aliphatic amines are abundant building blocks but underutilized as alkyl sources in C–C bond formation. Now it has been shown that integrating nitrogen-atom deletion into the aza-Michael reaction redirects the classical pathway from (sp3)C–N bond formation to Giese-type (sp3)C–C(sp3) bond construction.},
  archive      = {J_NCHEM},
  author       = {Ma, Panpan and Cui, Zhangkai and Lu, Hongjian},
  doi          = {10.1038/s41557-025-01888-8},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1556-1564},
  shortjournal = {Nat. Chem.},
  title        = {Deaminative giese-type reaction},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ultrafast charging of two-dimensional polymer cathodes enabled by cross-flow structure design. <em>NCHEM</em>, <em>17</em>(10), 1546-1555. (<a href='https://doi.org/10.1038/s41557-025-01899-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While ultrafast charging and discharging is highly desired for energy storage, the densely packed crystalline inorganic electrodes suffer from sluggish ion transport that limits this capability. Here we report two-dimensional vertical ladder polymer cathode materials that feature layered nanosheets with rich intralayer pores and structural defects alongside weak interlayer interactions. This structural design allows lithium ions to migrate vertically across the intrinsic pores and/or defects accompanied by horizontal intercalation, thus establishing a cross-flow pathway for lithium storage. Such an effective ion-transport method enables flash charging of an ultrahigh-power polymer cathode to ~70% state-of-charge within 30 s at a high current density. Even operated at −50 °C, the polymer cathode achieves 3-min charging to ~55% state-of-charge. Furthermore, we propose an organic–inorganic hybrid strategy that overall improves the electrode-level specific energy at high rates for meeting practical metrics. This work demonstrates the potential of organic electrodes for high-power output under extreme operational conditions. Electrochemical energy-storage devices require rapid charge–discharge profiles, but the dense electrode packing required to achieve high energy densities results in sluggish ion-transport kinetics. Now a two-dimensional vertical ladder polymer cathode has been shown to achieve promising performance and tolerance by using a cross-flow lithium migration pathway.},
  archive      = {J_NCHEM},
  author       = {Deng, Xianming and Liu, Li and Zhang, Shuchao and Chen, Qingxuan and Zhang, Xinming and Wu, Huimin and Chen, Yongqi and Hou, Tingzheng and Liu, Ming and Zhou, Dong and Peng, Lele and Chen, Zhen},
  doi          = {10.1038/s41557-025-01899-5},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1546-1555},
  shortjournal = {Nat. Chem.},
  title        = {Ultrafast charging of two-dimensional polymer cathodes enabled by cross-flow structure design},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantitative profiling of lipid transport between organelles enabled by subcellular photocatalytic labelling. <em>NCHEM</em>, <em>17</em>(10), 1534-1545. (<a href='https://doi.org/10.1038/s41557-025-01886-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Subcellular lipid composition and transport substantially influence the physiological and pathological functions of both cells and organelles. However, lipid transport and turnover between organelles remain poorly understood due to a lack of methods for selectively labelling lipids in organelles. Here we develop a subcellular photocatalytic labelling strategy that enables organelle-selective lipid analysis by mass spectrometry and the quantitative profiling of lipid transport between organelles. We use this approach to quantitatively characterize fatty-acyl-dependent transport of phosphatidylethanolamine and phosphatidylserine lipids between the endoplasmic reticulum and mitochondria, the nucleus or lysosomes. Further experiments revealed the relative contributions of various biosynthesis pathways to the phosphatidylethanolamine and phosphatidylserine lipid compositions in the mitochondria, nucleus and lysosomes. Lysosome-specific photocatalytic labelling revealed the impact of the mTOR kinase pathway on lysosomal lipid metabolism. Together, this subcellularly localized photocatalytic labelling of lipids quantitatively deciphers the subcellular lipid composition and transport, enhancing our understanding of lipid metabolism in living organisms. Subcellular lipid transport between organelles and turnover remain poorly explored due to the technical challenges associated with selective lipid labelling. Now a subcellular photocatalytic labelling strategy has been developed that allows organelle-specific lipid analysis and quantitative profiling of lipid transport.},
  archive      = {J_NCHEM},
  author       = {Chen, Xi and He, Ru and Xiong, Haolin and Wang, Ruohong and Yin, Yandong and Chen, Yiyun and Zhu, Zheng-Jiang},
  doi          = {10.1038/s41557-025-01886-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1534-1545},
  shortjournal = {Nat. Chem.},
  title        = {Quantitative profiling of lipid transport between organelles enabled by subcellular photocatalytic labelling},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diastereodivergent synthesis of multi-substituted cycloalkanes. <em>NCHEM</em>, <em>17</em>(10), 1524-1533. (<a href='https://doi.org/10.1038/s41557-025-01885-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The escape from flat aromatic architectures towards saturated structures enriched with an increased proportion of sp3 atoms (Fsp3) has been harnessed as a drug design strategy. Chemical synthesis of drug-like saturated cycloalkanes, particularly in terms of the precise control of substituents either on the same or opposing faces of the ring plane, poses greater complexities than the assembly of their two-dimensional aromatic analogues. Reliable and predictable divergent synthesis of all isomers of multi-substituted cycloalkanes continues to pose a formidable challenge. Here we show a cobalt-catalysed process for achieving diastereodivergent synthesis. By strategically manipulating ligands, we accomplished diastereodivergent hydroalkylation of methylenecyclohexanes. This cobalt-catalysed system manifests exceptional resilience and versatility in synthesizing all isomers of disubstituted cyclohexanes and piperidines, as well as multi-substituted cyclohexanes, transcending inherent limited selective tendencies in other traditional synthesis strategies. The synthesis of drug-like saturated cycloalkanes is more complex than their two-dimensional aromatic analogues. A formidable challenge lies in synthesizing all isomers of multi-substituted cycloalkanes. Now a cobalt-catalysed system enables diastereodivergent hydroalkylation of substituted methylenecyclohexanes with exceptional versatility. Strategic manipulation of ligands provides access to all isomers of disubstituted cyclohexanes, piperidines and multi-substituted cyclohexanes.},
  archive      = {J_NCHEM},
  author       = {Li, Zhen and Liu, Deguang and Gao, Gen-Wei and Chen, Pei-Wen and Li, Yan and Zeng, Xiongzhi and Li, Zhenyu and Lu, Xi and Fu, Yao},
  doi          = {10.1038/s41557-025-01885-x},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1524-1533},
  shortjournal = {Nat. Chem.},
  title        = {Diastereodivergent synthesis of multi-substituted cycloalkanes},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Introducing metal–sulfur active sites in metal–organic frameworks via post-synthetic modification for hydrogenation catalysis. <em>NCHEM</em>, <em>17</em>(10), 1514-1523. (<a href='https://doi.org/10.1038/s41557-025-01876-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metal–sulfur active sites play a central role in catalytic processes such as hydrogenation and dehydrogenation, yet the majority of active sites in these compounds reside on the surfaces and edges of catalyst particles, limiting overall efficiency. Here we present a strategy to embed metal–sulfur active sites into metal–organic frameworks (MOFs) by converting bridging or terminal chloride ligands into hydroxide and subsequently into sulfide groups through post-synthetic modification. We apply this method to two representative MOF families: one featuring one-dimensional metal–chloride chains and another containing discrete multinuclear metal clusters. Crystallographic and spectroscopic analyses confirm structural integrity and sulfide incorporation, and the transformation is monitored by in situ total scattering methods. The sulfided MOFs display enhanced catalytic activity in the selective hydrogenation of nitroarenes using molecular hydrogen. Density functional theory calculations indicate that sulfur incorporation promotes homolytic metal–ligand bond cleavage and facilitates H2 activation. This work establishes an approach to construct MOFs featuring accessible metal–sulfide sites. Metal–sulfur motifs are commonly found in enzymatic active sites and heterogeneous catalysis, but they remain underexplored in porous solids. Now, sulfur-based ligands have been incorporated into metal–organic frameworks through post-synthetic modifications. The resulting sulfide MOFs exhibit enhanced catalytic performance in the selective hydrogenation of nitroarenes compared with their parent MOFs containing terminal or bridging chloride and hydroxyl groups.},
  archive      = {J_NCHEM},
  author       = {Xie, Haomiao and Khoshooei, Milad Ahmadi and Mandal, Mukunda and Vornholt, Simon M. and Hofmann, Jan and Tufaro, Luke M. and Kirlikovali, Kent O. and Grimes, Dawson A. and Lee, Seryeong and Su, Shengyi and Reischauer, Susanne and Sengupta, Debabrata and Fahy, Kira and Ma, Kaikai and Wang, Xiaoliang and Sha, Fanrui and Gong, Wei and Chen, Yongwei and Vitillo, Jenny G. and Anderson, John S. and Notestein, Justin M. and Chapman, Karena W. and Gagliardi, Laura and Farha, Omar K.},
  doi          = {10.1038/s41557-025-01876-y},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1514-1523},
  shortjournal = {Nat. Chem.},
  title        = {Introducing metal–sulfur active sites in metal–organic frameworks via post-synthetic modification for hydrogenation catalysis},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Magnetic activation of spherical nucleic acids enables the remote control of synthetic cells. <em>NCHEM</em>, <em>17</em>(10), 1505-1513. (<a href='https://doi.org/10.1038/s41557-025-01909-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The flexible and modular design of synthetic cells, comprising lipid vesicles capable of imitating the structure and function of living cells, facilitates their application as drug delivery devices. The ability to control the synthesis of biomolecules within synthetic cells using a tissue-penetrating stimulus opens up additional levels of functionality that has the potential to improve biological potency and circumvent drug leakage from preloaded vesicles. To this end, we have designed spherical nucleic acids comprising DNA promoter sequences decorating magnetic nanoparticle cores. These spherical nucleic acids allowed us to harness the heat dissipated from magnetic hyperthermia (a clinically approved anticancer therapy) to regulate cell-free protein synthesis and release cargo on demand. Furthermore, this magnetic regulation of biosynthesis was achieved using clinically tolerable magnetic field strengths and frequencies. We then deployed an opaque blocking material that is impenetrable by current activation methods to highlight the potential of this technology for targeting and controlling the in situ synthesis of biomolecules using tissue-penetrating magnetic fields deep within the body. The programmability of synthetic cells, comprising lipid vesicles that are capable of imitating the structure and function of living cells, facilitates their application as drug delivery devices. Now, magnetic hyperthermia has been used to control the on-demand synthesis and release of biomolecules from within synthetic cells.},
  archive      = {J_NCHEM},
  author       = {Parkes, Ellen and Al Samad, Assala and Mazzotti, Giacomo and Newell, Charlie and Ng, Brian and Radford, Amy and Booth, Michael J.},
  doi          = {10.1038/s41557-025-01909-6},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1505-1513},
  shortjournal = {Nat. Chem.},
  title        = {Magnetic activation of spherical nucleic acids enables the remote control of synthetic cells},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flame synthesis achieves compositionally tailorable high-entropy metal-containing nanomaterials. <em>NCHEM</em>, <em>17</em>(10), 1497-1504. (<a href='https://doi.org/10.1038/s41557-025-01894-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-entropy metal-containing nanomaterials have garnered interest in diverse fields such as electrocatalysis and energy conversion. Their synthesis typically requires high temperatures (&gt;1,000 K) to facilitate homogeneous mixing and rapid transformation of metal precursors. However, current state-of-the-art approaches typically involve complex reaction environments and require specialized equipment and operations. Herein we demonstrate a versatile flame synthesis process to fabricate high-entropy metallic single atoms and/or nanoparticles supported on soot-like carbon via blending organometallic precursors into fuel (namely, paraffin wax) and subsequent burning. The high flame temperature (~1,800 K) enables strong metal–carbon association with tailorable chemistry and homogeneous bonding between dissimilar metallic elements (up to 25 metals studied), regardless of their thermodynamic compatibility. Additionally, we show high-performance electrosynthesis of hydrogen peroxide to highlight this approach as a promising method for electrocatalyst development. High-entropy nanomaterials, which incorporate multiple homogeneously dispersed elements, are promising for energy conversion, but typically require complex, high-temperature synthesis. Now a scalable flame synthesis method has been shown to produce tunable, high-entropy nanomaterials under facile conditions, enabling the design of advanced electrocatalysts for hydrogen peroxide production and other relevant applications.},
  archive      = {J_NCHEM},
  author       = {Liu, Zhen and Goudeli, Eirini and Guo, Rui and Xie, Houting and Luo, Qinjian and Zhao, Laichun and Xu, Wanjun and Richardson, Joseph J. and Xu, Weijian and Caruso, Frank and Pan, Shuaijun},
  doi          = {10.1038/s41557-025-01894-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1497-1504},
  shortjournal = {Nat. Chem.},
  title        = {Flame synthesis achieves compositionally tailorable high-entropy metal-containing nanomaterials},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stable single-site organonickel catalyst preferentially hydrogenolyses branched polyolefin C–C bonds. <em>NCHEM</em>, <em>17</em>(10), 1488-1496. (<a href='https://doi.org/10.1038/s41557-025-01892-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods of processing accumulated polyolefin waste typically require harsh conditions, precious metals or high metal loadings to achieve appreciable activities. Here we examined supported, single-site organonickel catalysts for polyolefin upcycling. Chemisorption of Ni(COD)2 (COD, 1,5-cyclooctadiene) onto Brønsted acidic sulfated alumina (AlS) yields a highly electrophilic Ni(I) precatalyst, AlS/Ni(COD)2, which is converted under H2 to the active AlS/NiIIH catalyst. This single-site system exhibits unique hydrogenolysis selectivity that favours cleaving branched polyolefin C–C linkages, enabling the hydrogenolytic separation of polyethylene and isotactic polypropylene (iPP) mixtures. Moreover, AlS/NiIIH remains highly selective and active for hydrogenolysis of iPP admixed with polyvinyl chloride, and the spent catalyst can be repeatedly regenerated by AlEt3 treatment. Experimental mechanistic analysis and density functional theory modelling reveal a turnover-limiting C–C scission pathway featuring β-alkyl transfer and strong olefin binding. These results highlight the potential of nickel-based systems for the selective upcycling of complex plastic waste streams. Selective chemical upcycling of polyolefin mixtures remains challenging due to the structural similarity of their backbones. Now it has been shown that a single-site nickel catalyst can preferentially and efficiently cleave branched C–C bonds, enabling the hydrogenolytic separation of isotactic polypropylene from mixtures containing both isotactic polypropylene and polyethylene.},
  archive      = {J_NCHEM},
  author       = {Lai, Qingheng and Zhang, Xinrui and Jiang, Shan and Krzyaniak, Matthew D. and Alayoglu, Selim and Agarwal, Amol and Liu, Yukun and Edenfield, Wilson C. and Kobayashi, Takeshi and Wang, Yuyang and Dravid, Vinayak and Wasielewski, Michael R. and Miller, Jeffery T. and Kratish, Yosi and Marks, Tobin J.},
  doi          = {10.1038/s41557-025-01892-y},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1488-1496},
  shortjournal = {Nat. Chem.},
  title        = {Stable single-site organonickel catalyst preferentially hydrogenolyses branched polyolefin C–C bonds},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanochemical pathway for converting fluoropolymers to fluorochemicals. <em>NCHEM</em>, <em>17</em>(10), 1480-1487. (<a href='https://doi.org/10.1038/s41557-025-01855-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing demand for fluorochemicals has increased the production of toxic hydrogen fluoride (HF)—a widely used fluorination agent—from mineral fluorspar (CaF2). Although potassium fluoride (KF) is a safer alternative fluorination agent, its production still involves HF as an intermediate. Simultaneously, the disposal of fluoropolymer waste, especially polyvinylidene fluoride (PVDF), poses environmental challenges. We present a mechanochemical protocol that decomposes PVDF to generate KF as a nucleophilic fluorinating agent. As an advantage over traditional batch reactions, mechanochemical protocols can be performed quickly, and with or without modest volumes of solvents. We show that KF produced using this method can be used to efficiently construct S–F, C(sp2)–F and C(sp3)–F bonds, providing a sustainable solution to fluorine resource depletion and hazardous HF usage. Our approach also addresses fluoropolymer waste disposal and reduces reliance on natural fluorspar resources, marking a notable advancement in fluorine chemistry and paving the way for safer and eco-friendly fluorination techniques. The production of fluorination agents generally involves the formation of toxic hydrogen fluoride. Now, a mechanochemical protocol to decompose the fluoropolymer polyvinylidene fluoride has been developed, generating potassium fluoride as a fluorinating agent. This approach provides a safer and sustainable fluorination strategy for efficient S–F, C(sp2)–F and C(sp3)–F bond formation.},
  archive      = {J_NCHEM},
  author       = {Hattori, Masashi and Saha, Debarshi and Bacho, Muhamad Zulfaqar and Shibata, Norio},
  doi          = {10.1038/s41557-025-01855-3},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1480-1487},
  shortjournal = {Nat. Chem.},
  title        = {Mechanochemical pathway for converting fluoropolymers to fluorochemicals},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tunable cobalt-catalysed hydrogenation of allenes enabled by multiple metal–ligand cooperative functionalities. <em>NCHEM</em>, <em>17</em>(10), 1469-1479. (<a href='https://doi.org/10.1038/s41557-025-01945-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Catalytic hydrogenation is essential in synthetic chemistry, with ongoing innovations aimed at enhancing the selectivity and efficiency of these reactions. The hydrogenation of multisubstituted allenes, however, presents a long-standing challenge due to the difficulty of controlling multiple selectivity factors simultaneously. Here we introduce a series of chiral pincer cobalt catalysts featuring multiple metal–ligand cooperative functionalities. These catalysts feature an ‘N–H’ moiety as an outer-sphere binding site and an N-heterocycle group as a hemilabile basic site, enabling the use of structurally diverse tridentate ligands for selective hydrogenation of functionalized allenes. This design liberates a coordination site for H2 activation and enhances selectivity control through the structural tuning of the N-heterocycle group. The catalysts exhibit exceptional chemo-, regio-, enantio- and Z/E-selectivities, along with broad functional group tolerance, enabling access to all possible semihydrogenation products of multisubstituted allenes. Mechanistic studies uncover a distinctive redox-neutral Co(I) catalytic cycle that facilitates heterolytic cleavage of H2, assisted by the basic N-heterocycle on the ligand. The hydrogenation of multisubstituted allenes presents a long-standing challenge due to the difficulty in simultaneously controlling multiple selectivity parameters. Now a class of pincer cobalt catalysts featuring multiple metal–ligand cooperation functionalities has been developed that enables the regio- and stereodivergent hydrogenation of allenes to provide all possible multisubstituted alkene products.},
  archive      = {J_NCHEM},
  author       = {Rong, Xianle and Ren, Yunjuan and Chen, Yongshun and Li, Chongyao and Wang, Jie and Chen, Zhaoqiang and Tao, Lizhi and Wei, Zhihong and Ma, Shengming and Liu, Qiang},
  doi          = {10.1038/s41557-025-01945-2},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1469-1479},
  shortjournal = {Nat. Chem.},
  title        = {Tunable cobalt-catalysed hydrogenation of allenes enabled by multiple metal–ligand cooperative functionalities},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dual dynamic helical poly(disulfide)s with conformational adaptivity and configurational recyclability. <em>NCHEM</em>, <em>17</em>(10), 1462-1468. (<a href='https://doi.org/10.1038/s41557-025-01947-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The structural dynamics of biopolymers, such as proteins and DNA, feature the unique combination of simultaneous conformational adaptivity and configurational reversibility. However, it remains challenging to design a synthetic system featuring dynamic covalent bonds with secondary structural folding and full recycling capabilities. Here we present a synthetic covalent polymer, constructed from biologically relevant units (amino acids and disulfides), that can reversibly adapt between disordered and helical conformations and can recycle itself to the original small molecules. This dual dynamic feature arises from the subtle synergy of noncovalent bonds (H bonds) and dynamic covalent bonds within a coupled chemical equilibrium, of which thermodynamics is found to follow a nonlinear van’t Hoff equation with nonzero heat capacity change. The covalent polymers show conformational dynamics between structured helices and disordered coils and the ability to convert back to the original monomers. This system demonstrates how dual dynamic function can control helical and recyclable polymer formation. Covalent polymers with helical conformations offer an adaptive scaffold for smart materials, but polymer-to-monomer deconstruction is inhibited by the covalent backbone. Now it has been shown that poly(disulfide)s can be folded into helices driven by side-chain hydrogen-bonding self-assembly, resulting in a synthetic helical polymer that can be fully recycled.},
  archive      = {J_NCHEM},
  author       = {Zhang, Qi and Nicu, Valentin P. and Buma, Wybren Jan and Tian, He and Qu, Da-Hui and Feringa, Ben L.},
  doi          = {10.1038/s41557-025-01947-0},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1462-1468},
  shortjournal = {Nat. Chem.},
  title        = {Dual dynamic helical poly(disulfide)s with conformational adaptivity and configurational recyclability},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding single-molecule reactions using nanopore-based techniques. <em>NCHEM</em>, <em>17</em>(10), 1450-1461. (<a href='https://doi.org/10.1038/s41557-025-01905-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The formation and cleavage of chemical bonds are two fundamental processes in chemistry, and the nature of these bonds determines the physical and chemical properties of a molecule. Real-time observation of chemical bonding at the single-molecule level offers insights into transient intermediates that are normally inaccessible via ensemble measurements. Protein nanopores, with their unique geometries, can be tailored into nanoreactors. Molecular bond-making and -cleavage at the reactive site of a protein nanopore’s interior wall can be visualized by monitoring ionic current changes. Therefore, nanopore-based techniques can enhance the understanding of complex binding kinetics and reaction mechanisms. In this Review we summarize recent advances in using biological nanopores as both single-molecule nanoreactors and single-molecule biosensors. The discussion covers the kinetics of single-molecule reactions under nanopore confinement, the strategies for designing biological nanopores and the latest progress in revealing reaction intermediates and pathways at the single-molecule level. Finally, we emphasize unresolved challenges and anticipate future developments in this rapidly evolving field. Real-time single-molecule observation provides insights into transient intermediates that are normally inaccessible using ensemble measurements. This Review discusses the use of nanopores as both single-molecule nanoreactors and single-molecule sensors for elucidating reaction pathways and kinetics, while highlighting their potential to control covalent reactions under confinement.},
  archive      = {J_NCHEM},
  author       = {Ying, Yi-Lun and Yang, Chao-Nan and Liu, Wei and Wu, Xue-Yuan and Long, Yi-Tao},
  doi          = {10.1038/s41557-025-01905-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1450-1461},
  shortjournal = {Nat. Chem.},
  title        = {Understanding single-molecule reactions using nanopore-based techniques},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Repurposing magnetic hyperthermia therapy to control biosynthesis in synthetic cells. <em>NCHEM</em>, <em>17</em>(10), 1448-1449. (<a href='https://doi.org/10.1038/s41557-025-01910-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The programmability of synthetic cells facilitates their application as drug delivery devices, but on-demand biosynthesis and release of cargo using a tissue-penetrating stimuli remains a challenge. Now, the heat generated from magnetic hyperthermia — a clinically approved anticancer therapy — is used to control the in situ synthesis and release of biomolecules from within synthetic cells.},
  archive      = {J_NCHEM},
  doi          = {10.1038/s41557-025-01910-z},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1448-1449},
  shortjournal = {Nat. Chem.},
  title        = {Repurposing magnetic hyperthermia therapy to control biosynthesis in synthetic cells},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-entropy nanomaterials by candlelight. <em>NCHEM</em>, <em>17</em>(10), 1445-1447. (<a href='https://doi.org/10.1038/s41557-025-01959-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-entropy materials offer an exciting realm of possibilities for material discovery and application; however, their nanoscale synthesis is limited by the need for specialized equipment. Now, a simple synthetic strategy employing wax candles enables production of these compositionally complex nanomaterials.},
  archive      = {J_NCHEM},
  author       = {Liu, Shuo and Dun, Chaochao and Swihart, Mark T.},
  doi          = {10.1038/s41557-025-01959-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1445-1447},
  shortjournal = {Nat. Chem.},
  title        = {High-entropy nanomaterials by candlelight},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-site catalyst cuts branched polyolefins to pieces. <em>NCHEM</em>, <em>17</em>(10), 1442-1444. (<a href='https://doi.org/10.1038/s41557-025-01927-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical upcycling of waste plastic is challenging because of its heterogeneous nature. Now, a newly designed single-site organo-nickel catalyst cuts through the mix by selectively snipping branched C–C linkages, allowing for the separation of polyolefin mixtures.},
  archive      = {J_NCHEM},
  author       = {Brody, Leo and Rorrer, Julie},
  doi          = {10.1038/s41557-025-01927-4},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1442-1444},
  shortjournal = {Nat. Chem.},
  title        = {Single-site catalyst cuts branched polyolefins to pieces},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanochemical upcycling technologies for sustainable circular fluorine economy. <em>NCHEM</em>, <em>17</em>(10), 1439-1441. (<a href='https://doi.org/10.1038/s41557-025-01956-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Per- and polyfluoroalkyl substances are widely utilized across various fields such as energy and electronics industries, but concerns have been raised about their environmental accumulation. Now, recent studies have revealed methods to decompose these compounds into reusable fluorine sources, enabling a circular economy.},
  archive      = {J_NCHEM},
  author       = {Matsunami, Asuka and Okazoe, Takashi},
  doi          = {10.1038/s41557-025-01956-z},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1439-1441},
  shortjournal = {Nat. Chem.},
  title        = {Mechanochemical upcycling technologies for sustainable circular fluorine economy},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cooperative ligands enable cobalt-catalysed hydrogenation. <em>NCHEM</em>, <em>17</em>(10), 1437-1438. (<a href='https://doi.org/10.1038/s41557-025-01944-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The partial hydrogenation of multi-substituted allenes is a highly useful reaction but impeded by difficult selectivity control. Now, pincer cobalt catalysts enable this transformation through metal–ligand cooperative interactions, achieving ligand-controlled selective allene semihydrogenation in a divergent manner.},
  archive      = {J_NCHEM},
  author       = {Zhong, Hongyu},
  doi          = {10.1038/s41557-025-01944-3},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1437-1438},
  shortjournal = {Nat. Chem.},
  title        = {Cooperative ligands enable cobalt-catalysed hydrogenation},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Opinionated science. <em>NCHEM</em>, <em>17</em>(10), 1435-1436. (<a href='https://doi.org/10.1038/s41557-025-01957-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When the facts can’t speak for themselves, scientists can give them a voice, argues Michelle Francl.},
  archive      = {J_NCHEM},
  author       = {Francl, Michelle},
  doi          = {10.1038/s41557-025-01957-y},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1435-1436},
  shortjournal = {Nat. Chem.},
  title        = {Opinionated science},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="ng">NG - 34</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48. <em>NG</em>, <em>57</em>(9), 2340-2342. (<a href='https://doi.org/10.1038/s41588-025-02332-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Riazuddin, Saima and Belyantseva, Inna A. and Giese, Arnaud P. J. and Lee, Kwanghyuk and Indzhykulian, Artur A. and Nandamuri, Sri Pratima and Yousaf, Rizwan and Sinha, Ghanshyam P. and Lee, Sue and Terrell, David and Hegde, Rashmi S. and Ali, Rana A. and Anwar, Saima and Andrade-Elizondo, Paula B. and Sirmaci, Asli and Parise, Leslie V. and Basit, Sulman and Wali, Abdul and Ayub, Muhammad and Ansar, Muhammad and Ahmad, Wasim and Khan, Shaheen N. and Akram, Javed and Tekin, Mustafa and Riazuddin, Sheikh and Cook, Tiffany and Buschbeck, Elke K. and Frolenkov, Gregory I. and Leal, Suzanne M. and Friedman, Thomas B. and Ahmed, Zubair M.},
  doi          = {10.1038/s41588-025-02332-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2340-2342},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation. <em>NG</em>, <em>57</em>(9), 2339. (<a href='https://doi.org/10.1038/s41588-025-02328-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Uryu, Hidetaka and Saeki, Koichi and Haeno, Hiroshi and Kapadia, Chiraag Deepak and Furudate, Ken and Nangalia, Jyoti and Spencer Chapman, Michael and Zhang, Linda and Padilla, Jennifer and Zhao, Li and Hsu, Joanne I. and Zhao, Chong and Chen, Shujuan and Tanaka, Tomoyuki and Li, Zongrui and Ogata, Satoko and Hanache, Sarah and Yang, Hui and DiNardo, Courtney and Daver, Naval and Pemmaraju, Naveen and Jain, Nitin and Ravandi, Farhad and Zhang, Jianhua and Song, Xingzhi and Thompson, Erika and Tang, Hongli and Little, Latasha and Gumbs, Curtis and Orlowski, Robert Z. and Qazilbash, Muzaffar and Bhalla, Kapil and Colla, Simona and Kantarjian, Hagop and Kanagal-Shamanna, Rashmi and Bueso-Ramos, Carlos and Nakada, Daisuke and Al-Atrash, Gheath and Molldrem, Jeffery and Futreal, P. Andrew and Shpall, Elizabeth and Goodell, Margaret and Garcia-Manero, Guillermo and Takahashi, Koichi},
  doi          = {10.1038/s41588-025-02328-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2339},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma. <em>NG</em>, <em>57</em>(9), 2338. (<a href='https://doi.org/10.1038/s41588-025-02320-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gundem, Gunes and Levine, Max F. and Roberts, Stephen S. and Cheung, Irene Y. and Medina-Martínez, Juan S. and Feng, Yi and Arango-Ossa, Juan E. and Chadoutaud, Loic and Rita, Mathieu and Asimomitis, Georgios and Zhou, Joe and You, Daoqi and Bouvier, Nancy and Spitzer, Barbara and Solit, David B. and Dela Cruz, Filemon and LaQuaglia, Michael P. and Kushner, Brian H. and Modak, Shakeel and Shukla, Neerav and Iacobuzio-Donahue, Christine A. and Kung, Andrew L. and Cheung, Nai-Kong V. and Papaemmanuil, Elli},
  doi          = {10.1038/s41588-025-02320-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2338},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states. <em>NG</em>, <em>57</em>(9), 2323-2334. (<a href='https://doi.org/10.1038/s41588-025-02284-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell atlas projects have revealed that common cell types often comprise distinct, recurrent transcriptional states, but the function and regulation of these states remain poorly understood. Here, we show that systematic activation of transcription factors can recreate such states in vitro, providing tractable models for mechanistic and functional studies. Using a scalable CRISPR activation (CRISPRa) Perturb-seq platform, we activated 1,836 transcription factors in two cell types. CRISPRa induced gene expression within physiological ranges, with chromatin features predicting responsiveness. Comparisons with atlas datasets showed that transcription factor perturbations recapitulated key fibroblast states and identified their regulators, including KLF2 and KLF4 for a universal state present in many tissues, and PLAGL1 for a disease-associated inflammatory state. Inducing the universal state suppressed the inflammatory state, suggesting therapeutic potential. These findings position CRISPRa as a nuanced tool for perturbing differentiated cells and establish a general strategy for studying clinically relevant transcriptional states ex vivo. CRISPR activation of 1,836 human transcription factors recapitulates fibroblast transcriptional states observed in vivo and identifies regulators that can revert inflammatory states.},
  archive      = {J_NG},
  author       = {Southard, Kaden M. and Ardy, Rico C. and Tang, Anran and O’Sullivan, Deirdre D. and Metzner, Eli and Guruvayurappan, Karthik and Norman, Thomas M.},
  doi          = {10.1038/s41588-025-02284-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2323-2334},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize. <em>NG</em>, <em>57</em>(9), 2313-2322. (<a href='https://doi.org/10.1038/s41588-025-02246-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive maps of functional variation at transcription factor (TF) binding sites (cis-elements) are crucial for elucidating how genotype shapes phenotype. Here, we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped over 200,000 variants, genetic, epigenetic, or both (termed binding quantitative trait loci (bQTL)), linked to cis-element occupancy. Three lines of evidence support the functional significance of bQTL: (1) coincidence with causative loci that regulate traits, including vgt1, ZmTRE1 and the MITE transposon near ZmNAC111 under drought; (2) bQTL allelic bias is shared between inbred parents and matches chromatin immunoprecipitation sequencing results; and (3) partitioning genetic variation across genomic regions demonstrates that bQTL capture the majority of heritable trait variation across ~72% of 143 phenotypes. Our study provides an auspicious approach to make functional cis-variation accessible at scale for genetic studies and targeted engineering of complex traits. Pan-cistrome of the maize leaf under well-watered and drought conditions profiled by haplotype-specific MOA-seq highlights the relevance of transcription factor binding QTLs for understanding phenotypic diversity in maize.},
  archive      = {J_NG},
  author       = {Engelhorn, Julia and Snodgrass, Samantha J. and Kok, Amelie and Seetharam, Arun S. and Schneider, Michael and Kiwit, Tatjana and Singh, Ayush and Banf, Michael and Doan, Duong Thi Hai and Khaipho-Burch, Merritt and Runcie, Daniel E. and Sánchez-Camargo, Victor A. and Bader, Rechien and Vladimir Torres-Rodriguez, J. and Sun, Guangchao and Stam, Maike and Fiorani, Fabio and Beier, Sebastian and Schnable, James C. and Bass, Hank W. and Hufford, Matthew B. and Stich, Benjamin and Frommer, Wolf B. and Ross-Ibarra, Jeffrey and Hartwig, Thomas},
  doi          = {10.1038/s41588-025-02246-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2313-2322},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants. <em>NG</em>, <em>57</em>(9), 2302-2312. (<a href='https://doi.org/10.1038/s41588-025-02306-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clubroot disease, caused by the obligate intracellular rhizarian protist Plasmodiophora brassicae, is devastating to cruciferous crops worldwide. Widespread field P. brassicae pathotypes frequently overcome the pathotype-specific resistance of modern varieties, posing a challenge for durable control of this disease. Here a genome-wide association study of 3 years of data comprising field clubroot phenotyping of 244 genome-resequenced Brassica napus accessions identified a strong association of β-1,3-glucan synthase-like 5 (GSL5) with clubroot susceptibility. GSL5 was evolutionarily conserved, and inactivation of GSL5 by genome editing in Arabidopsis, B. napus, Brassica rapa and Brassica oleracea conferred broad-spectrum, high-level resistance to P. brassicae pathotypes without yield penalties in B. napus. GSL5 inactivation derepressed the jasmonic acid-mediated immunity during P. brassicae secondary infection, and this immune repression was possibly reinforced through stabilization of GSL5 by a P. brassicae effector, facilitating clubroot susceptibility. Our study provides durable resistance resources for cruciferous clubroot disease control and insights into plant resistance against intracellular eukaryotic phytopathogens. This study implicates GSL5 inactivation in high, broad-spectrum resistance to the clubroot pathogen Plasmodiophora brassicae in Arabidopsis thaliana, Brassica napus, Brassica oleracea and Brassica rapa.},
  archive      = {J_NG},
  author       = {Wu, Yupo and Zhao, Chuanji and Zhang, Yi and Shen, Cuicui and Zhang, Yuanyuan and Zhang, Xiong and Gao, Lixia and Zeng, Lingyi and Ke, Qinglin and Qin, Li and Liu, Fan and Huang, Junyan and Ren, Li and Liu, Yueying and Cheng, Hongtao and Tong, Chaobo and Hu, Qiong and Cheng, Xiaohui and Wei, Yangdou and Liu, Shengyi and Liu, Lijiang},
  doi          = {10.1038/s41588-025-02306-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2302-2312},
  shortjournal = {Nat. Genet.},
  title        = {Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism. <em>NG</em>, <em>57</em>(9), 2289-2301. (<a href='https://doi.org/10.1038/s41588-025-02293-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Making sense of whole-genome polymorphism data is challenging, but it is essential for overcoming the biases in SNP data. Here we analyze 27 genomes of Arabidopsis thaliana to illustrate these issues. Genome size variation is mostly due to tandem repeat regions that are difficult to assemble. However, while the rest of the genome varies little in length, it is full of structural variants, mostly due to transposon insertions. Because of this, the pangenome coordinate system grows rapidly with sample size and ultimately becomes 70% larger than the size of any single genome, even for n = 27. Finally, we show how short-read data are biased by read mapping. SNP calling is biased by the choice of reference genome, and both transcriptome and methylome profiling results are affected by mapping reads to a reference genome rather than to the genome of the assayed individual. New concepts for comparing the genomes of 27 naturally inbred Arabidopsis thaliana accessions provide essential insights into obtaining a less biased view of whole-genome polymorphism.},
  archive      = {J_NG},
  author       = {Igolkina, Anna A. and Vorbrugg, Sebastian and Rabanal, Fernando A. and Liu, Hai-Jun and Ashkenazy, Haim and Kornienko, Aleksandra E. and Fitz, Joffrey and Collenberg, Max and Kubica, Christian and Mollá Morales, Almudena and Jaegle, Benjamin and Wrightsman, Travis and Voloshin, Vitaly and Bezlepsky, Alexander D. and Llaca, Victor and Nizhynska, Viktoria and Reichardt, Ilka and Bezrukov, Ilja and Lanz, Christa and Bemm, Felix and Flood, Pádraic J. and Nemomissa, Sileshi and Hancock, Angela and Guo, Ya-Long and Kersey, Paul and Weigel, Detlef and Nordborg, Magnus},
  doi          = {10.1038/s41588-025-02293-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2289-2301},
  shortjournal = {Nat. Genet.},
  title        = {A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity. <em>NG</em>, <em>57</em>(9), 2276-2288. (<a href='https://doi.org/10.1038/s41588-025-02294-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common oat, belonging to the genus Avena with 30 recognized species, is a nutritionally important cereal crop and high-quality forage worldwide. Here, we construct a genus-level super-pangenome of Avena comprising 35 high-quality genomes from 14 cultivated oat accessions and 21 wild species. The fully resolved phylogenomic analysis unveils the origin and evolutionary scenario of Avena species, and the super-pangenome analysis identifies 26.62% and 59.93% specific genes and haplotypes in wild species. We delineate the landscape of structural variations (SVs) and the transcriptome profile based 1,401 RNA-sequencing (RNA-seq) samples from diverse abiotic stress treatments in oat. We highlight the crucial role of SVs in modulating gene expression and shaping adaptation to diverse stresses. Further combining SV-based genome-wide association studies (GWASs), we characterize 13 candidate genes associated with drought resistance such as AsARF7, validated by transgenic oat lines. Our study provides unprecedented genomic resources to facilitate genomic, evolution and molecular breeding research in oat. A genus-level super-pangenome of Avena comprising 35 high-quality genomes from 23 cultivated or wild oat species highlights the evolution of Avena and the landscape of structural variations related to abiotic stress resistance.},
  archive      = {J_NG},
  author       = {Zhang, Hongyu and Liu, Ningkun and Wang, Yaru and Zheng, Xinyuan and Li, Wei and Liu, Ze and Liu, Jianan and Wang, Yu and Xing, Longsheng and Li, Tao and Yun, Yange and Zhou, Qinghong and Wang, Meijia and Qin, Yujie and Yan, Jinjiang and Gong, Zhizhong and He, Qiang and Du, Huilong},
  doi          = {10.1038/s41588-025-02294-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2276-2288},
  shortjournal = {Nat. Genet.},
  title        = {Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts. <em>NG</em>, <em>57</em>(9), 2264-2275. (<a href='https://doi.org/10.1038/s41588-025-02330-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Locusts are characterized by a large genome size, polyphenism and an X0 sex determination system. Here we generated chromosome-level genomes for both desert and migratory locusts, as well as a comprehensive chromatin map for the latter. We found that genome enlargement is associated with an increased number of enhancers in expanded intronic regions. To explore the function of expanded enhancers, we identified a distal enhancer that contributes to behavioral differences between solitary and gregarious locusts. In the X0 sex system, H4K16ac enrichment and H4K20me1 depletion maintain balanced X-linked expression in male soma. Notably, the distance-dependent H4K16ac enrichment diminishes gradually in intergenic regions, revealing a special dosage compensation mechanism in large genomes. Furthermore, the distance-dependent H4K16ac results in a lag in the evolution of dosage compensation for the X-linked genes translocated recently from autosomes. Therefore, expanded intronic and intergenic regions shape a distinct chromatin regulation landscape in large genomes. Chromosome-level genome assemblies of migratory and desert locusts, coupled with epigenomic profiling of migratory locusts, reveal chromatin dynamics underlying polyphenism and X-linked dosage compensation following autosomal gene translocation.},
  archive      = {J_NG},
  author       = {Liu, Qing and Jiang, Feng and Li, Ran and Liu, Shanlin and Fang, Wenjing and Li, Xiao and Hu, Ting and Feng, Lianyun and Zhang, Jie and Liu, Zhikang and He, Jing and Guo, Wei and Wang, Xianhui and Sun, Zongyi and Li, Jingjing and Gao, Yunan and Yi, Jiacheng and Li, Qiye and Wang, Xiaoxiao and Wei, Liya and Kang, Le},
  doi          = {10.1038/s41588-025-02330-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2264-2275},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. <em>NG</em>, <em>57</em>(9), 2250-2263. (<a href='https://doi.org/10.1038/s41588-025-02305-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian switch/sucrose nonfermentable (mSWI/SNF) complex regulates chromatin accessibility and frequently shows alterations due to mutation in cancer and neurological diseases. Inadequate expression of mSWI/SNF in heterozygous mice can lead to developmental defects, indicating dosage-sensitive effects of mSWI/SNF. However, how its dosage affects function has remained unclear. Using a targeted protein degradation system, we investigated its dosage-sensitive effects by precisely controlling protein levels of BRG1, the ATPase subunit of the mSWI/SNF complex. We found that binding of BRG1 to chromatin exhibited a linear response to the BRG1 protein level. Although chromatin accessibility at most promoters and insulators was largely unaffected by BRG1 depletion, 44% of enhancers, including 84% of defined superenhancers, showed reduced accessibility. Notably, half of the BRG1-regulated enhancers, particularly superenhancers, exhibited a buffered response to BRG1 loss. Consistently, transcription exhibited a predominantly buffered response to changes in BRG1 levels. Collectively, our findings demonstrate a genomic feature-specific response to BRG1 dosage, shedding light on the dosage-sensitive effects of mSWI/SNF complex defects in cancer and other diseases. This study investigates the dosage-sensitive effects of BRG1, a core SWI/SNF subunit, on chromatin binding, accessibility and transcriptional regulation by precisely controlling its protein level.},
  archive      = {J_NG},
  author       = {Hagihara, Yota and Zhang, Chao and Zhang, Yi},
  doi          = {10.1038/s41588-025-02305-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2250-2263},
  shortjournal = {Nat. Genet.},
  title        = {Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue. <em>NG</em>, <em>57</em>(9), 2238-2249. (<a href='https://doi.org/10.1038/s41588-025-02300-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, TET1 and DNMT3A are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome. This multiomic study, including single-nucleus DNA methylation and chromatin conformation matched with single-nuclei RNA sequencing, provides insights into the epigenomic landscape of human subcutaneous adipose tissue.},
  archive      = {J_NG},
  author       = {Chen, Zeyuan Johnson and Das, Sankha Subhra and Kar, Asha and Lee, Seung Hyuk T. and Abuhanna, Kevin D. and Alvarez, Marcus and Sukhatme, Mihir G. and Wang, Zitian and Gelev, Kyla Z. and Heffel, Matthew G. and Zhang, Yi and Avram, Oren and Rahmani, Elior and Sankararaman, Sriram and Laakso, Markku and Heinonen, Sini and Peltoniemi, Hilkka and Halperin, Eran and Pietiläinen, Kirsi H. and Luo, Chongyuan and Pajukanta, Päivi},
  doi          = {10.1038/s41588-025-02300-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2238-2249},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution. <em>NG</em>, <em>57</em>(9), 2226-2237. (<a href='https://doi.org/10.1038/s41588-025-02307-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients. Integrated multi-omic analyses using samples from the TRACERx study highlight cross-talk between DNA hypermethylation and genomic lesions in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Gimeno-Valiente, Francisco and Castignani, Carla and Larose Cadieux, Elizabeth and Mensah, Nana E. and Liu, Xiaohong and Chen, Kezhong and Chervova, Olga and Karasaki, Takahiro and Weeden, Clare E. and Richard, Corentin and Lai, Siqi and Martínez-Ruiz, Carlos and Lim, Emilia L. and Frankell, Alexander M. and Watkins, Thomas B. K. and Stavrou, Georgia and Usaite, Ieva and Lu, Wei-Ting and Marinelli, Daniele and Saghafinia, Sadegh and Wilson, Gareth A. and Dhami, Pawan and Vaikkinen, Heli and Steif, Jonathan and Veeriah, Selvaraju and Hynds, Robert E. and Hirst, Martin and Hiley, Crispin and Feber, Andrew and Deniz, Özgen and Jamal-Hanjani, Mariam and McGranahan, Nicholas and Beck, Stephan and Demeulemeester, Jonas and Tanić, Miljana and Swanton, Charles and Van Loo, Peter and Kanu, Nnennaya},
  doi          = {10.1038/s41588-025-02307-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2226-2237},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis. <em>NG</em>, <em>57</em>(9), 2215-2225. (<a href='https://doi.org/10.1038/s41588-025-02296-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in PPM1D and the TERT gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers. This study explores the relationship between telomere length and clonal hematopoiesis. Splicing factor and PPM1D gene mutations are more frequent in people with genetically predicted shorter telomere lengths, suggesting that these mutations protect against the consequences of telomere attrition.},
  archive      = {J_NG},
  author       = {McLoughlin, Matthew A. and Cheloor Kovilakam, Sruthi and Dunn, William G. and Gu, Muxin and Tobin, Jake and Pershad, Yash and Williams, Nicholas and Leongamornlert, Daniel and Dawson, Kevin and Bond, Laura and Marando, Ludovica and Wen, Sean and Wilson, Rachael and Valenzano, Giampiero and Symeonidou, Vasiliki and Rak, Justyna and Damaskou, Aristi and Gozdecka, Malgorzata and Liu, Xiaoxuan and Barcena, Clea and Nomdedeu, Josep and Costeas, Paul and Dimitriou, Ioannis D. and Fiorillo, Edoardo and Orrù, Valeria and de Almeida, Jose Guilherme and McKerrell, Thomas and Cullen, Matthew and Mohorianu, Irina and Foukaneli, Theodora and Warren, Alan J. and Wong, Chi and Follows, George and Godfrey, Anna L. and Gudgin, Emma and Cucca, Francesco and McKinney, Eoin and Baxter, E. Joanna and Gerstung, Moritz and Mitchell, Jonathan and Wiseman, Daniel and Bick, Alexander G. and Fabre, Margarete and Quiros, Pedro M. and Nangalia, Jyoti and Kar, Siddhartha and Vassiliou, George S.},
  doi          = {10.1038/s41588-025-02296-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2215-2225},
  shortjournal = {Nat. Genet.},
  title        = {Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Temporal genomic dynamics shape clinical trajectory in multiple myeloma. <em>NG</em>, <em>57</em>(9), 2203-2214. (<a href='https://doi.org/10.1038/s41588-025-02292-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple myeloma evolution is characterized by the accumulation of genomic drivers over time. To unravel this timeline and its impact on clinical outcomes, we analyzed 421 whole-genome sequences from 382 patients. Using clock-like mutational signatures, we estimated a time lag of two to four decades between the initiation of events and diagnosis. We demonstrate that odd-numbered chromosome trisomies in patients with hyperdiploidy can be acquired simultaneously with other chromosomal gains (for example, 1q gain). We show that hyperdiploidy is acquired after immunoglobulin heavy chain translocation when both events co-occur. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (&gt;1 extra copy), but fared worse than those with late 1q gain. This finding underscores that the 1q gain prognostic impact depends more on the timing of acquisition than on the number of copies gained. Overall, this study contributes to a better understanding of the life history of myeloma and may have prognostic implications. This analysis of whole-genome sequencing data from 421 multiple myeloma samples elucidates the timing of key genomic events and shows associations between the timing of 1q gain and clinical outcome.},
  archive      = {J_NG},
  author       = {Maura, Francesco and Kaddoura, Marcella and Poos, Alexandra M. and Baughn, Linda B. and Ziccheddu, Bachisio and Bärtsch, Marc-Andrea and Cirrincione, Anthony and Maclachlan, Kylee and Chojnacka, Monika and Diamond, Benjamin and Papadimitriou, Marios and Blaney, Patrick and John, Lukas and Reichert, Philipp and Huhn, Stefanie and Gagler, Dylan and Zhang, Yanming and Dogan, Ahmet and Lesokhin, Alexander M. and Davies, Faith and Goldschmidt, Hartmut and Fenk, Roland and Weisel, Katja C. and Mai, Elias K. and Korde, Neha and Morgan, Gareth J. and Rajkumar, S. Vincent and Kumar, Shaji and Usmani, Saad and Landgren, Ola and Raab, Marc S. and Weinhold, Niels},
  doi          = {10.1038/s41588-025-02292-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2203-2214},
  shortjournal = {Nat. Genet.},
  title        = {Temporal genomic dynamics shape clinical trajectory in multiple myeloma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tamoxifen induces PI3K activation in uterine cancer. <em>NG</em>, <em>57</em>(9), 2192-2202. (<a href='https://doi.org/10.1038/s41588-025-02308-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutagenic processes and clonal selection contribute to the development of therapy-associated secondary neoplasms, a known complication of cancer treatment. The association between tamoxifen therapy and secondary uterine cancers is uncommon but well established; however, the genetic mechanisms underlying tamoxifen-driven tumorigenesis remain unclear. We find that oncogenic PIK3CA mutations, common in spontaneously arising estrogen-associated de novo uterine cancer, are significantly less frequent in tamoxifen-associated tumors. In vivo, tamoxifen-induced estrogen receptor stimulation activates phosphoinositide 3-kinase (PI3K) signaling in normal mouse uterine tissue, potentially eliminating the selective benefit of PI3K-activating mutations in tamoxifen-associated uterine cancer. Together, we present a unique pathway of therapy-associated carcinogenesis in which tamoxifen-induced activation of the PI3K pathway acts as a non-genetic driver event, contributing to the multistep model of uterine carcinogenesis. While this PI3K mechanism is specific to tamoxifen-associated uterine cancer, the concept of treatment-induced signaling events may have broader applicability to other routes of tumorigenesis. Sequencing analysis of tamoxifen-associated uterine cancers and further in vivo analyses suggest that the drug tamoxifen can activate the PI3K pathway in the absence of oncogenic mutations.},
  archive      = {J_NG},
  author       = {Kübler, Kirsten and Nardone, Agostina and Anand, Shankara and Gurevich, Daniel and Gao, Jianjiong and Droog, Marjolein and Hermida-Prado, Francisco and Akhshi, Tara and Feiglin, Ariel and Feit, Avery S. and Cohen Feit, Gabriella and Dackus, Gwen and Pun, Matthew and Kuang, Yanan and Cha, Justin and Miller, Mendy and Gregoricchio, Sebastian and Lanfermeijer, Mirthe and Cornelissen, Sten and Gibson, William J. and Paweletz, Cloud P. and Van Allen, Eliezer M. and van Leeuwen, Flora E. and Nederlof, Petra M. and Nguyen, Quang-Dé and Mourits, Marian J. E. and Radovich, Milan and Leshchiner, Ignaty and Stewart, Chip and Matulonis, Ursula A. and Zwart, Wilbert and Maruvka, Yosef E. and Getz, Gad and Jeselsohn, Rinath},
  doi          = {10.1038/s41588-025-02308-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2192-2202},
  shortjournal = {Nat. Genet.},
  title        = {Tamoxifen induces PI3K activation in uterine cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L. <em>NG</em>, <em>57</em>(9), 2177-2191. (<a href='https://doi.org/10.1038/s41588-025-02289-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the high prevalence of ERG transcription factor translocations in prostate cancer, the mechanism of tumorigenicity remains poorly understood. Using lineage tracing, we find the tumor-initiating activity of ERG resides in a subpopulation of murine basal cells that coexpress luminal genes (BasalLum) and not in the larger population of ERG+ luminal cells. Upon ERG activation, BasalLum cells give rise to highly proliferative intermediate (IM) cells with stem-like features that coexpress basal, luminal, hillock and club marker genes, before transitioning to Krt8+ luminal cells. Transcriptomic analysis of ERG+ human prostate cancers confirms the presence of rare ERG+ BasalLum cells, as well as IM cells whose presence is associated with a worse prognosis. Single-cell analysis revealed a chromatin state in ERG+ IM cells enriched for STAT3 transcription factor binding sites and elevated expression of the KMT2A/MLL1 and DOT1L, all three of which are essential for ERG-driven tumorigenicity in vivo. In addition to providing translational opportunities, this work illustrates how single-cell approaches combined with lineage tracing can identify cancer vulnerabilities not evident from bulk analysis. Lineage tracing in mice identifies a subpopulation of basal cells that express Tmprss2 and Nkx3 as the origin of ERG-driven prostate cancer. Upon expansion, these cells show an enrichment for STAT3 chromatin binding and elevated expression of KMT2A and DOT1L as dependencies for ERG oncogenicity.},
  archive      = {J_NG},
  author       = {Feng, Weiran and Ladewig, Erik and Lange, Matthew and Salsabeel, Nazifa and Zhao, Huiyong and Lee, Young Sun and Gopalan, Anuradha and Luo, Hanzhi and Kang, Wenfei and Fan, Ning and Rosiek, Eric and de Stanchina, Elisa and Chen, Yu and Carver, Brett S. and Leslie, Christina S. and Sawyers, Charles L.},
  doi          = {10.1038/s41588-025-02289-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2177-2191},
  shortjournal = {Nat. Genet.},
  title        = {ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutational landscape of triple-negative breast cancer in african american women. <em>NG</em>, <em>57</em>(9), 2166-2176. (<a href='https://doi.org/10.1038/s41588-025-02322-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {African American (AA) women have the highest incidence of triple-negative breast cancer (TNBC) among all racial groups, but are underrepresented in cancer genomic studies. In 462 AA women with TNBC, we characterized the tumor mutational landscape by whole-exome sequencing and RNA sequencing. We unveiled a high-resolution mutational portrait of TNBC in AA women reminiscent of that in Asian and non-Hispanic white women, with no evidence of associations of mutational features with African ancestry. We also made some distinctive discoveries, including an almost complete dominance of TP53 mutations, low frequency of PIK3CA mutations and mutational signature-based subtypes with etiologic and prognostic significance. These findings do not support major racial differences in TNBC biology at the level of somatic mutations. Our study contributes considerably to diversifying the knowledge base of breast cancer genomics and provides insights into the disease etiology, disparities and therapeutic vulnerability of TNBC in AA women. This study explores the genomic and transcriptomic landscapes of triple-negative breast cancer in African American women. The authors show that the mutational profile is broadly similar to that observed in European and East Asian ancestry women while highlighting some interesting differences.},
  archive      = {J_NG},
  author       = {Yao, Song and Wei, Lei and Hu, Qiang and Liu, Song and Manojlovic, Zarko and Fiorica, Peter N. and Long, Mark and Zirpoli, Gary R. and Cai, Qiuyin and Long, Jirong and Ping, Jie and Barnard, Mollie E. and Jin, Yuxin and Murakami, Mitsuko and Wang, Jianmin and Zhu, Qianqian and Davis, Warren and Chen, Jianhong and Ondracek, Rochelle P. and Khoury, Thaer and Gandhi, Shipra and Takabe, Kazuaki and Ko, Naomi and Sanderson, Maureen and Hong, Chi-Chen and Bandera, Elisa V. and Craig, David W. and Ambrosone, Christine B. and Palmer, Julie R. and Zheng, Wei and Carpten, John D.},
  doi          = {10.1038/s41588-025-02322-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2166-2176},
  shortjournal = {Nat. Genet.},
  title        = {Mutational landscape of triple-negative breast cancer in african american women},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions. <em>NG</em>, <em>57</em>(9), 2156-2165. (<a href='https://doi.org/10.1038/s41588-025-02291-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand shared and ancestry-specific genetic control of brain protein expression and its ramifications for disease, we mapped protein quantitative trait loci (pQTLs) in 1,362 brain proteomes from African American, Hispanic/Latin American and non-Hispanic white donors. Among the pQTLs that multiancestry fine-mapping MESuSiE confidently assigned as putative causal pQTLs in a specific population, most were shared across the three studied populations and are referred to as multiancestry causal pQTLs. These multiancestry causal pQTLs were enriched for exonic and promoter regions. To investigate their effects on disease, we modeled the 858 multiancestry causal pQTLs as instrumental variables using Mendelian randomization and genome-wide association study results for neurologic and psychiatric conditions (21 traits in participants with European ancestry, 10 in those with African ancestry and 4 in Hispanic participants). We identified 119 multiancestry pQTL–protein pairs consistent with a causal role in these conditions. Remarkably, 29% of the multiancestry pQTLs in these pairs were coding variants. These results lay an important foundation for the creation of new molecular models of neurologic and psychiatric conditions that are likely to be relevant to individuals across different genetic ancestries. Multiancestry fine-mapping of brain protein quantitative trait loci coupled with Mendelian randomization analyses identifies protein–trait pairs consistent with causal effects across neurological and psychiatric conditions.},
  archive      = {J_NG},
  author       = {Wingo, Aliza P. and Liu, Yue and Vattathil, Selina M. and Gerasimov, Ekaterina S. and Mei, Zhen and Ravindran, Suda Parimala and Liu, Jiaqi and Shantaraman, Ananth and Seifar, Fatemeh and Wang, Erming and Zhang, Bin and Reddy, Joseph and Allen, Mariet and Ertekin-Taner, Nilüfer and De Jager, Philip L. and Fox, Edward J. and Duong, Duc M. and Epstein, Michael P. and Cutler, David J. and Levey, Allan I. and Bennett, David A. and Seyfried, Nicholas T. and Wingo, Thomas S.},
  doi          = {10.1038/s41588-025-02291-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2156-2165},
  shortjournal = {Nat. Genet.},
  title        = {Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes. <em>NG</em>, <em>57</em>(9), 2146-2155. (<a href='https://doi.org/10.1038/s41588-025-02288-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks with whole-genome sequencing (WGS) now enable the association of noncoding rare variants with complex human traits. Given that &gt;98% of the genome is available for exploration, the selection of noncoding variants remains a critical yet unresolved challenge in these analyses. Here we leverage knowledge of blood gene regulation and deleteriousness scores to select noncoding variants pertinent for association with blood-related traits. Integrating WGS and 42 blood cell count and biomarker measurements for 166,740 UK Biobank samples, we perform variant collapsing tests, identifying hundreds of gene–trait associations involving noncoding variants. However, we demonstrate that most of these noncoding rare variant associations (1) reproduce associations known from previous studies and (2) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting noncoding rare variant association results. Noncoding rare variant analyses using whole-genome sequencing data from the UK Biobank identify gene–trait associations for 42 blood cell traits but find that most signals are driven by linkage disequilibrium between common and rare variants.},
  archive      = {J_NG},
  author       = {Ribeiro, Diogo M. and Hofmeister, Robin J. and Rubinacci, Simone and Delaneau, Olivier},
  doi          = {10.1038/s41588-025-02288-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2146-2155},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores. <em>NG</em>, <em>57</em>(9), 2136-2145. (<a href='https://doi.org/10.1038/s41588-025-02298-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic health record (EHR)-based phenotype risk scores (PheRS) leverage individuals’ health trajectories to estimate disease risk, similar to how polygenic scores (PGS) use genetic information. While PGS generalizability has been studied, less is known about PheRS generalizability across healthcare systems and whether PheRS are complementary to PGS. We trained elastic-net-based PheRS to predict the onset of 13 common diseases for 845,929 individuals (age = 32–70 years) from three biobank-based studies in Finland (FinnGen), the UK (UKB) and Estonia (EstB). All PheRS were statistically significantly associated with the diseases of interest and most generalized well without retraining when applied to other studies. PheRS and PGS were only moderately correlated and models including both predictors improved onset prediction compared to PGS alone for 8 of 13 diseases. Our results indicate that EHR-based risk scores can transfer well between EHRs, capture largely independent information from PGS, and provide additive benefits for disease risk prediction. Comparison of electronic health record-based phenotype risk scores (PheRS) and polygenic scores (PGS) across 13 common diseases and three biobank-based studies indicates that PheRS and PGS may provide additive benefits for risk prediction.},
  archive      = {J_NG},
  author       = {Detrois, Kira E. and Hartonen, Tuomo and Teder-Laving, Maris and Jermy, Bradley and Läll, Kristi and Yang, Zhiyu and Mägi, Reedik and Ripatti, Samuli and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02298-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2136-2145},
  shortjournal = {Nat. Genet.},
  title        = {Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes. <em>NG</em>, <em>57</em>(9), 2124-2135. (<a href='https://doi.org/10.1038/s41588-025-02317-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Ancestral Recombination Graph (ARG), which describes the genealogical history of a sample of genomes, is a vital tool in population genomics and biomedical research. Recent advancements have substantially increased ARG reconstruction scalability, but they rely on approximations that can reduce accuracy, especially under model misspecification. Moreover, they reconstruct only a single ARG topology and cannot quantify the considerable uncertainty associated with ARG inferences. Here, to address these challenges, we introduce SINGER (sampling and inferring of genealogies with recombination), a method that accelerates ARG sampling from the posterior distribution by two orders of magnitude, enabling accurate inference and uncertainty quantification for hundreds of whole-genome sequences. Through extensive simulations, we demonstrate SINGER’s enhanced accuracy and robustness to model misspecification compared to existing methods. We demonstrate the utility of SINGER by applying it to individuals of British and African descent within the 1000 Genomes Project, identifying signals of population differentiation, archaic introgression and strong support for ancient polymorphism in the human leukocyte antigen region shared across primates. SINGER is a method for creating ancestral recombination graphs to understand the genealogical history of genomes. The method has increased speed, and thus scalability, without sacrificing accuracy.},
  archive      = {J_NG},
  author       = {Deng, Yun and Nielsen, Rasmus and Song, Yun S.},
  doi          = {10.1038/s41588-025-02317-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2124-2135},
  shortjournal = {Nat. Genet.},
  title        = {Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. <em>NG</em>, <em>57</em>(9), 2116-2123. (<a href='https://doi.org/10.1038/s41588-025-02286-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mixed-model association analysis (MMAA) is the preferred tool for performing genome-wide association studies. However, existing MMAA tools often have long runtimes and high memory requirements. Here we present LDAK-KVIK, an MMAA tool for analysis of quantitative and binary phenotypes. LDAK-KVIK is computationally efficient, requiring less than 10 CPU hours and 5 Gb memory to analyze genome-wide data for 350,000 individuals. Using simulated phenotypes, we show that LDAK-KVIK produces well-calibrated test statistics for both homogeneous and heterogeneous datasets. When applied to real phenotypes, LDAK-KVIK has the highest power among all tools considered. For example, across 40 quantitative UK Biobank phenotypes (average sample size 349,000), LDAK-KVIK finds 16% more independent, genome-wide significant loci than classical linear regression, whereas BOLT-LMM and REGENIE find 15% and 11% more, respectively. LDAK-KVIK can also be used to perform gene-based tests; across the 40 quantitative UK Biobank phenotypes, LDAK-KVIK finds 18% more significant genes than the leading existing tool. Last, LDAK-KVIK produces state-of-the-art polygenic scores. LDAK-KVIK is a mixed-model association method for genome-wide studies that optimizes computational performance and power. LDAK-KVIK can also perform gene-based tests and produces state-of-the-art polygenic scores.},
  archive      = {J_NG},
  author       = {Hof, Jasper P. and Speed, Doug},
  doi          = {10.1038/s41588-025-02286-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2116-2123},
  shortjournal = {Nat. Genet.},
  title        = {LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward improving multiomic research in understudied cereals. <em>NG</em>, <em>57</em>(9), 2106-2115. (<a href='https://doi.org/10.1038/s41588-025-02245-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the looming challenges of climate change and population pressure, understudied cereals hold notably untapped potential for future global food security and sustainable agriculture due to their unique nutritional profiles and notable stress resilience. Multiomics holds great promise in realizing the value of understudied cereals by understanding the genetic basis and molecular mechanisms regulating their unique traits; however, multiomic studies of understudied cereals still lag behind those of major cereals, hindering their genetic improvement and wide adoption in modern agriculture. Here, we discuss the limitations and challenges in multiomic studies of understudied cereals and how they can be overcome to enhance utilization of the traits unique to understudied cereals in improving both themselves and major cereals. We also offer a database specific to understudied cereals, including multiomics data from 12 understudied cereals and wild relatives for some of these, to serve the community. Understudied cereals offer untapped potential for food security, yet multiomic research substantially lags behind that of major cereals. This Perspective discusses these challenges and introduces a dedicated database to serve the community.},
  archive      = {J_NG},
  author       = {Yang, Fanjing and Gong, Xiaojiao and Zhao, Pan and Cheng, Lingtong and Zhang, Yujie and Luo, Yingting and Zhu, Qian-Hao and Ye, Chu-Yu},
  doi          = {10.1038/s41588-025-02245-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2106-2115},
  shortjournal = {Nat. Genet.},
  title        = {Toward improving multiomic research in understudied cereals},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The LISTEN principles for genetic sequence data governance and database engineering. <em>NG</em>, <em>57</em>(9), 2099-2105. (<a href='https://doi.org/10.1038/s41588-025-02270-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several international legal agreements include an ‘access and benefit-sharing’ (ABS) mechanism that attaches obligations to the use of genetic sequence data. These agreements are frequently subject to critique on the grounds that ABS is either fundamentally incompatible with the principles of open science, or technically challenging to implement in open scientific databases. Here, we argue that these critiques arise from a misinterpretation of the principles of open science and that both considerations can be addressed by a set of simple principles that link database engineering and governance. We introduce a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, which can be readily adopted by both new and existing platforms participating in ABS systems. We also highlight how these principles can act in concert with familiar principles of open science, such as findable, accessible, interoperable and reusable (FAIR) data sharing. This Perspective proposes a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, aimed at ensuring access and benefit-sharing principles in open science.},
  archive      = {J_NG},
  author       = {Carlson, Colin J. and Granados, Monica and Phelan, Alexandra and Ramakrishnan, Nithin and Poisot, Timothée},
  doi          = {10.1038/s41588-025-02270-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2099-2105},
  shortjournal = {Nat. Genet.},
  title        = {The LISTEN principles for genetic sequence data governance and database engineering},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Governance of cross-border genomic data sharing through a human rights approach. <em>NG</em>, <em>57</em>(9), 2090-2098. (<a href='https://doi.org/10.1038/s41588-025-02252-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cross-border genomic data sharing is crucial for advancing research progress, understanding of genetic diversity and international public health. However, this practice also leads to concerns regarding biosecurity, individual privacy and regulatory conflicts. Scientific communities often advocate data sovereignty approaches to address these challenges, yet related frameworks are often inadequate in their ability to address such challenges and have an overemphasis on national control and protectionism. They also may not fully consider the inherently transnational nature of genomic data and the need for a governance system that reflects the shared human heritage of the data and global interconnectivity. Instead, we propose here a human rights-based approach that surpasses territorial boundaries to promote shared responsibility and equitable research practices. A balanced governance framework that incorporates extraterritorial obligations and aligns data policy with human rights principles can safeguard individual privacy and biosecurity, encourage international collaboration and support ethically responsible genomic research. This Perspective proposes a human rights-based governance framework for cross-border genomic data sharing, addressing limitations of data sovereignty approaches while balancing privacy, security and global research collaboration.},
  archive      = {J_NG},
  author       = {Yu, Liang and Feng, Ruohan and Sun, Youhai and Peng, Yaojin},
  doi          = {10.1038/s41588-025-02252-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2090-2098},
  shortjournal = {Nat. Genet.},
  title        = {Governance of cross-border genomic data sharing through a human rights approach},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recreating fibroblast diversity in vitro by activating transcription factors. <em>NG</em>, <em>57</em>(9), 2088-2089. (<a href='https://doi.org/10.1038/s41588-025-02283-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We combined CRISPR activation of all 1,836 known human transcription factors with high-throughput Perturb-seq to recreate the diverse transcriptional states occupied by fibroblasts in vivo. Our study revealed regulators of key states and showed that inducing normal states can, in some cases, suppress those linked to disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02283-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2088-2089},
  shortjournal = {Nat. Genet.},
  title        = {Recreating fibroblast diversity in vitro by activating transcription factors},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improving broad-spectrum resistance to clubroot disease with one gene. <em>NG</em>, <em>57</em>(9), 2086-2087. (<a href='https://doi.org/10.1038/s41588-025-02304-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GSL5, a glucan synthase, acts as a suppressor of jasmonic acid-mediated immunity in cruciferous plants. Inactivation of GSL5 by genome editing confers high-level and broad-spectrum resistance to pathogens that cause clubroot disease in four cruciferous species.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02304-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2086-2087},
  shortjournal = {Nat. Genet.},
  title        = {Improving broad-spectrum resistance to clubroot disease with one gene},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mystery of tamoxifen-associated uterine cancer uncovered. <em>NG</em>, <em>57</em>(9), 2084-2085. (<a href='https://doi.org/10.1038/s41588-025-02316-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tamoxifen is an essential drug in breast cancer therapy. Unlike prevailing models of therapy-related tumorigenesis, tamoxifen acts by directly activating the PI3K pathway, bypassing the need for mutations in one of the most common driver genes in sporadic uterine cancer. These findings open avenues for investigating similar mechanisms in other drugs.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02316-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2084-2085},
  shortjournal = {Nat. Genet.},
  title        = {Mystery of tamoxifen-associated uterine cancer uncovered},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A bivalent chromatin state regulates the function of composite transposons. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02337-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02337-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {A bivalent chromatin state regulates the function of composite transposons},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting obesity from childhood. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02338-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02338-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Predicting obesity from childhood},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Building immune tolerance in the thymus. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02339-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02339-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Building immune tolerance in the thymus},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI identifies promoter variants that alter gene expression. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02340-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02340-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {AI identifies promoter variants that alter gene expression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease. <em>NG</em>, <em>57</em>(9), 2079-2082. (<a href='https://doi.org/10.1038/s41588-025-02299-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite Africa’s vast genetic diversity, its populations are underrepresented in global genomic datasets. Here we describe the vision of the KidneyGenAfrica, a pan-African initiative launched to address this inequity, and call for more inclusive genomics research that recognizes Africa’s key role in genetic variation and potential to generate insights in chronic kidney disease.},
  archive      = {J_NG},
  author       = {Fatumo, Segun and Ojewunmi, Oyesola and Camenzuli, Rebecca and Kintu, Christopher and Kamiza, Abram and Brandenburg, Jean-Tristan and Kalungi, Allan and Kalyesubula, Robert and Salako, Babatunde and Nash, Oyekanmi and Crampin, Amelia and Adeyemo, Adebowale and Nyirenda, Moffat and Ramsay, Michele and Fabian, June},
  doi          = {10.1038/s41588-025-02299-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2079-2082},
  shortjournal = {Nat. Genet.},
  title        = {KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The future of polygenic risk scores in direct-to-consumer genomics. <em>NG</em>, <em>57</em>(9), 2077-2078. (<a href='https://doi.org/10.1038/s41588-025-02313-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Sun, Yuhao and Vines, John and Tenesa, Albert},
  doi          = {10.1038/s41588-025-02313-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2077-2078},
  shortjournal = {Nat. Genet.},
  title        = {The future of polygenic risk scores in direct-to-consumer genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmat">NMAT - 28</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Highly reversible extrinsic electrocaloric effects over a wide temperature range in epitaxially strained SrTiO3 films. <em>NMAT</em>, <em>24</em>(10), 1664. (<a href='https://doi.org/10.1038/s41563-025-02321-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMAT},
  author       = {Zhang, S. and Deliyore-Ramírez, J. and Deng, S. and Nair, B. and Pesquera, D. and Jing, Q. and Vickers, M. E. and Crossley, S. and Ghidini, M. and Kar-Narayan, S. and Guzmán-Verri, G. G. and Moya, X. and Mathur, N. D.},
  doi          = {10.1038/s41563-025-02321-8},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1664},
  shortjournal = {Nat. Mater.},
  title        = {Publisher correction: Highly reversible extrinsic electrocaloric effects over a wide temperature range in epitaxially strained SrTiO3 films},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blood–brain-barrier-crossing lipid nanoparticles for mRNA delivery to the central nervous system. <em>NMAT</em>, <em>24</em>(10), 1653-1663. (<a href='https://doi.org/10.1038/s41563-024-02114-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The systemic delivery of mRNA molecules to the central nervous system is challenging as they need to cross the blood–brain barrier (BBB) to reach into the brain. Here we design and synthesize 72 BBB-crossing lipids fabricated by conjugating BBB-crossing modules and amino lipids, and use them to assemble BBB-crossing lipid nanoparticles for mRNA delivery. Screening and structure optimization studies resulted in a lead formulation that has substantially higher mRNA delivery efficiency into the brain than those exhibited by FDA-approved lipid nanoparticles. Studies in distinct mouse models show that these BBB-crossing lipid nanoparticles can transfect neurons and astrocytes of the whole brain after intravenous injections, being well tolerated across several dosage regimens. Moreover, these nanoparticles can deliver mRNA to human brain ex vivo samples. Overall, these BBB-crossing lipid nanoparticles deliver mRNA to neurons and astrocytes in broad brain regions, thereby being a promising platform to treat a range of central nervous system diseases. A lipid-nanoparticle-based formulation incorporating engineered lipids containing a blood–brain-barrier-crossing moiety effectively delivers mRNA to neurons and astrocytes following systemic administration in mice.},
  archive      = {J_NMAT},
  author       = {Wang, Chang and Xue, Yonger and Markovic, Tamara and Li, Haoyuan and Wang, Siyu and Zhong, Yichen and Du, Shi and Zhang, Yuebao and Hou, Xucheng and Yu, Yang and Liu, Zhengwei and Tian, Meng and Kang, Diana D. and Wang, Leiming and Guo, Kaiyuan and Cao, Dinglingge and Yan, Jingyue and Deng, Binbin and McComb, David W. and Parsons, Ramon E. and Minier-Toribio, Angelica M. and Holt, Leanne M. and Pan, Jiayi and Hashemi, Alice and Kopell, Brian H. and Charney, Alexander W. and Nestler, Eric J. and Peng, Paul C. and Dong, Yizhou},
  doi          = {10.1038/s41563-024-02114-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1653-1663},
  shortjournal = {Nat. Mater.},
  title        = {Blood–brain-barrier-crossing lipid nanoparticles for mRNA delivery to the central nervous system},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bond-centric modular design of protein assemblies. <em>NMAT</em>, <em>24</em>(10), 1644-1652. (<a href='https://doi.org/10.1038/s41563-025-02297-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directional interactions that generate regular coordination geometries are a powerful means of guiding molecular and colloidal self-assembly, but implementing such high-level interactions with proteins remains challenging due to their complex shapes and intricate interface properties. Here we describe a modular approach to protein nanomaterial design inspired by the rich chemical diversity that can be generated from the small number of atomic valencies. We design protein building blocks using deep learning-based generative tools, incorporating regular coordination geometries and tailorable bonding interactions that enable the assembly of diverse closed and open architectures guided by simple geometric principles. Experimental characterization confirms the successful formation of more than 20 multicomponent polyhedral protein cages, two-dimensional arrays and three-dimensional protein lattices, with a high (10%–50%) success rate and electron microscopy data closely matching the corresponding design models. Due to modularity, individual building blocks can assemble with different partners to generate distinct regular assemblies, resulting in an economy of parts and enabling the construction of reconfigurable networks for designer nanomaterials. Deep learning-based generative tools are used to design protein building blocks with well-defined directional bonding interactions, allowing the generation of a variety of scalable protein assemblies from a small set of reusable subunits.},
  archive      = {J_NMAT},
  author       = {Wang, Shunzhi and Favor, Andrew and Kibler, Ryan D. and Lubner, Joshua M. and Borst, Andrew J. and Coudray, Nicolas and Redler, Rachel L. and Chiang, Huat Thart and Sheffler, William and Hsia, Yang and Bethel, Neville P. and Li, Zhe and Ekiert, Damian C. and Bhabha, Gira and Pozzo, Lilo D. and Baker, David},
  doi          = {10.1038/s41563-025-02297-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1644-1652},
  shortjournal = {Nat. Mater.},
  title        = {Bond-centric modular design of protein assemblies},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational design of bifaceted protein nanomaterials. <em>NMAT</em>, <em>24</em>(10), 1635-1643. (<a href='https://doi.org/10.1038/s41563-025-02295-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in computational methods have led to considerable progress in the design of protein nanomaterials. However, nearly all nanoparticles designed so far exhibit strict point group symmetry, which limits structural diversity and precludes anisotropic functionalization. Here we describe a computational strategy for designing multicomponent bifaceted protein nanomaterials with two distinctly addressable sides. The method centres on docking pseudosymmetric hetero-oligomeric building blocks in architectures with dihedral symmetry and designing an asymmetric protein–protein interface between them. We obtain an initial 30-subunit assembly with pseudo-D5 symmetry and generate variants in which we alter the size and morphology of the bifaceted nanoparticles by designing extensions to one of the subunits. Functionalization of the two nanoparticle faces with protein minibinders enables the specific colocalization of two populations of polystyrene microparticles coated with the target protein receptors. The ability to accurately design anisotropic protein nanoparticles could be broadly useful in applications requiring the colocalization of distinct target moieties. Predictive protein design and experiments are combined to develop anisotropic bifaceted protein nanomaterials using pseudosymmetric hetero-oligomeric building blocks.},
  archive      = {J_NMAT},
  author       = {Rankovic, Sanela and Carr, Kenneth D. and Decarreau, Justin and Skotheim, Rebecca and Kibler, Ryan D. and Ols, Sebastian and Lee, Sangmin and Chun, Jung-Ho and Tooley, Marti R. and Dauparas, Justas and Eisenach, Helen E. and Glögl, Matthias and Weidle, Connor and Borst, Andrew J. and Baker, David and King, Neil P.},
  doi          = {10.1038/s41563-025-02295-7},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1635-1643},
  shortjournal = {Nat. Mater.},
  title        = {Computational design of bifaceted protein nanomaterials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Organic solar cells with 21% efficiency enabled by a hybrid interfacial layer with dual-component synergy. <em>NMAT</em>, <em>24</em>(10), 1626-1634. (<a href='https://doi.org/10.1038/s41563-025-02305-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cathode interfacial layer (CIL) critically influences electron extraction and charge recombination, thereby playing a pivotal role in organic solar cells (OSCs). However, most state-of-the-art CILs are constrained by limited conductivity, high recombination and poor morphology, which collectively hinder device efficiency and stability. Here we report an inorganic–organic hybrid CIL (AZnO-F3N), developed by a dual-component synergy strategy, which integrates organic material PNDIT-F3N with two-dimensional amorphous zinc oxide. This design leverages the synergistic interactions between two-dimensional amorphous zinc oxide and PNDIT-F3N, resulting in reduced interfacial defect, enhanced conductivity and improved film uniformity. OSCs incorporating the AZnO-F3N CIL exhibit more efficient charge extraction and transport, along with reduced recombination. Consequently, a D18:L8-BO-based binary OSC achieves an efficiency of 20.6%. The introduction of BTP-eC9 as the third component further elevates the efficiency to 21.0% (certified as 20.8%). Moreover, the CIL demonstrates versatility across various active layers, thick-film configuration and flexible devices, underscoring its great potential to advance OSC technology. A hybrid cathode interfacial layer (AZnO-F3N) is developed, delivering 21.0% efficiency along with excellent stability, mechanical robustness and broad versatility, highlighting its potential to advance organic solar cells.},
  archive      = {J_NMAT},
  author       = {Li, Congqi and Cai, Yunhao and Hu, Pengfei and Liu, Tao and Zhu, Lei and Zeng, Rui and Han, Fei and Zhang, Ming and Zhang, Meng and Lv, Jikai and Ma, Yuanxin and Han, Dexia and Lin, Qijie and Xu, Jingwen and Yu, Na and Qiao, Jiawei and Wang, Jiarui and Zhang, Xin and Xia, Jianlong and Tang, Zheng and Ye, Long and Li, Xiaoyi and Xu, Zihao and Hao, Xiaotao and Peng, Qian and Liu, Feng and Guo, Lin and Huang, Hui},
  doi          = {10.1038/s41563-025-02305-8},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1626-1634},
  shortjournal = {Nat. Mater.},
  title        = {Organic solar cells with 21% efficiency enabled by a hybrid interfacial layer with dual-component synergy},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanoscale phonon dynamics in self-assembled nanoparticle lattices. <em>NMAT</em>, <em>24</em>(10), 1616-1625. (<a href='https://doi.org/10.1038/s41563-025-02253-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geometry and topology endow mechanical frames with unusual properties from shape morphing to phonon wave manipulation, enabling emerging technologies. Despite important advances in macroscopic frames, the realization and phonon imaging of nanoscale mechanical metamaterials has remained challenging. Here we extend the principle of topologically engineered mechanical frames to self-assembled nanoparticle lattices, resolving phonon dynamics using liquid-phase transmission electron microscopy. The vibrations of nanoparticles in Maxwell lattices are used to measure properties that have been difficult to obtain, such as phonon band structures, nanoscale spring constants and nonlinear lattice deformation paths. Studies of five different lattices reveal that these properties are modulated by nanoscale colloidal interactions. Our discrete mechanical model and simulations capture these interactions and the critical role of effects beyond nearest neighbours, bridging mechanical metamaterials with nanoparticle self-assembly. Our study provides opportunities for understanding and manufacturing self-assembled nanostructures for phonon manipulation, offering solution processability, transformability and emergent functions at underexplored scales of length, frequency and energy density. The realization and phonon imaging of nanoscale mechanical metamaterials has remained challenging. Here the authors resolve the phonon dynamics and band structures of five different self-assembled nanoparticle lattices, revealing the role of nanoscale colloidal interactions in modulating the lattice properties.},
  archive      = {J_NMAT},
  author       = {Qian, Chang and Stanifer, Ethan and Ma, Zhan and Yao, Lehan and Luo, Binbin and Liu, Chang and Li, Jiahui and Pan, Puquan and Pan, Wenxiao and Mao, Xiaoming and Chen, Qian},
  doi          = {10.1038/s41563-025-02253-3},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1616-1625},
  shortjournal = {Nat. Mater.},
  title        = {Nanoscale phonon dynamics in self-assembled nanoparticle lattices},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flexible fibre-shaped fuel cells with gel-mediated internal pressure encapsulation. <em>NMAT</em>, <em>24</em>(10), 1608-1615. (<a href='https://doi.org/10.1038/s41563-025-02319-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of flexible fuel cells has been hindered by the rigid components and stringent requirements for pressure encapsulation and fuel sealing. Here we report an adaptive internal pressure encapsulation strategy that leverages the dynamic swelling behaviour of woven cotton fibres enclosed in a gel matrix in methanol. This strategy achieves simultaneous interfacial self-reinforcement and pressure modulation, enabling the fabrication of fibre-shaped direct methanol fuel cells. These flexible fuel cells operate across a broad temperature range, from –22 °C to 70 °C, showcasing cuttability, water resistance and fast refuelling capabilities, with full refuelling being achieved within 1 min. Furthermore, the fuel cells maintain consistent discharge performance, even after enduring 2,000 continuous flexing cycles. With an energy density of 161.36 Wh kg−1, these fibre-shaped direct methanol fuel cells surpass the energy densities of typical fibre-based power systems. This technology mitigates many of the technical challenges related to the lightweight and flexible application of fuel cells or fuel cell stacks for powering high-energy flexible devices. Cotton yarns encapsulated by a gel layer show dynamic swelling in methanol, creating interfacial self-reinforcement and pressure modulation mechanisms that enable the fabrication of fibre-shaped methanol fuel cells.},
  archive      = {J_NMAT},
  author       = {Yuan, Yongjiang and Liu, Ziyang and Zou, Xiuyang and Fang, Pengda and Zhang, Jiale and Zhang, Qiuhuan and Zhang, Hao and Yu, Qikun and Zhou, Tao and Li, Weizheng and Zheng, Sijie and Yang, Mingchen and Sun, Zhe and Zhu, Meifang and Yan, Feng},
  doi          = {10.1038/s41563-025-02319-2},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1608-1615},
  shortjournal = {Nat. Mater.},
  title        = {Flexible fibre-shaped fuel cells with gel-mediated internal pressure encapsulation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An amorphous Li–V–O–F cathode with tetrahedral coordination and O–O formal redox at low voltage. <em>NMAT</em>, <em>24</em>(10), 1600-1607. (<a href='https://doi.org/10.1038/s41563-025-02293-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ever-increasing demand for lithium-ion batteries has necessitated the development of high-performance cathode materials. However, previous studies have predominantly focused on crystal cathodes comprising the octahedral coordination of metal atoms and a well-ordered layered topology. This omits other cathode materials with other structures or coordination that could potentially surpass conventional counterparts in terms of performance. Here, using X-ray diffraction, resonant inelastic X-ray scattering and X-ray absorption near-edge spectra experiments, we investigated an amorphous Li–V–O–F cathode (a-LVOF) with tetrahedral coordination and elucidated an O–O formal redox mechanism at a moderate voltage of 4.1 V, without a conventional octahedral Li–O–Li configuration. The electrochemically amorphized material fosters randomly distributed VO4 units and scattered dangling oxygen bonds, which facilitated O–O binding. Moreover, a-LVOF demonstrates a high capacity of 230 mAh g−1. Our findings reveal a low-voltage O–O formal redox mechanism in an amorphized cathode material. Cathode materials for lithium-ion batteries typically possess octahedral coordination, which may exclude other possible solutions to degradation during deep cycling. A series of tetrahedral-framework-based amorphous Li–V–O–F materials are investigated, and shown to demonstrate O–O formal redox at 4.1 V.},
  archive      = {J_NMAT},
  author       = {Zhang, Kun and Yang, Tonghuan and Chen, Tao and Yang, Yali and Jiang, Zewen and Gao, Chuan and Zuo, Yuxuan and Xiao, Wukun and Xia, Dingguo},
  doi          = {10.1038/s41563-025-02293-9},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1600-1607},
  shortjournal = {Nat. Mater.},
  title        = {An amorphous Li–V–O–F cathode with tetrahedral coordination and O–O formal redox at low voltage},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Signatures of magnetism in zigzag graphene nanoribbons embedded in a hexagonal boron nitride lattice. <em>NMAT</em>, <em>24</em>(10), 1592-1599. (<a href='https://doi.org/10.1038/s41563-025-02317-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zigzag edges of graphene are predicted to host magnetic electronic states, critical for spintronics, but an experimental confirmation of these magnetic conduction channels remains elusive. Here we report the signatures of magnetism in zigzag graphene nanoribbons (zGNRs) embedded in hexagonal boron nitride. Hexagonal boron nitride provides crucial edge stabilization, enabling the direct probing of this intrinsic magnetism. Scanning nitrogen-vacancy-centre microscopy initially confirmed magnetism in zGNR. Subsequently, an ~9-nm-wide zGNR transistor was fabricated with a sub-50-nm channel length. Magnetotransport measurements at 4 K revealed distinct Fabry–Pérot-like interference patterns, indicating coherent transport. A large, anisotropic magnetoresistance (~175 Ω, ~1.3%) was observed, persisting well above room temperature. These findings strongly corroborate the existence of robust magnetic ordering in the zGNR edge state. This hexagonal-boron-nitride-embedded zGNR system offers an effective platform for future graphene-based spintronic devices. Nitrogen-vacancy-centre microscopy and anisotropic magnetoresistance measurements reveal the evidence of magnetism in zigzag graphene nanoribbons embedded in hexagonal boron nitride.},
  archive      = {J_NMAT},
  author       = {Jiang, Chengxin and Wang, Hui Shan and Liu, Chenxi and Chen, Chen and Chen, Lingxiu and Wang, Xiujun and Wang, Yibo and Kong, Ziqiang and Feng, Yuhan and Liu, Yixin and Feng, Yu and Zhang, Yu and Wei, Zhipeng and Guo, Maosen and Tong, Aomei and Mu, Gang and Yang, Yumeng and Watanabe, Kenji and Taniguchi, Takashi and Shi, Wangzhou and Wang, Haomin},
  doi          = {10.1038/s41563-025-02317-4},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1592-1599},
  shortjournal = {Nat. Mater.},
  title        = {Signatures of magnetism in zigzag graphene nanoribbons embedded in a hexagonal boron nitride lattice},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Symmetry-protected electronic metastability in an optically driven cuprate ladder. <em>NMAT</em>, <em>24</em>(10), 1584-1591. (<a href='https://doi.org/10.1038/s41563-025-02254-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optically excited quantum materials exhibit non-equilibrium states with remarkable emergent properties, but these phenomena are usually transient, decaying on picosecond timescales and limiting practical applications. Advancing the design and control of non-equilibrium phases requires the development of targeted strategies to achieve long-lived, metastable phases. Here we report the discovery of symmetry-protected electronic metastability in the model cuprate ladder Sr14Cu24O41. Using femtosecond resonant X-ray scattering and spectroscopy, we show that this metastability is driven by a transfer of holes from chain-like charge reservoirs into the ladders. This ultrafast charge redistribution arises from the optical dressing and activation of a hopping pathway that is forbidden by symmetry at equilibrium. Relaxation back to the ground state is, hence, suppressed after the pump coherence dissipates. Our findings highlight how dressing materials with electromagnetic fields can dynamically activate terms in the electronic Hamiltonian, and provide a rational design strategy for non-equilibrium phases of matter. A long-lived photoinduced metastable state in a quasi-one-dimensional cuprate, Sr14Cu24O41, is reported.},
  archive      = {J_NMAT},
  author       = {Padma, Hari and Glerean, Filippo and TenHuisen, Sophia F. R. and Shen, Zecheng and Wang, Haoxin and Xu, Luogen and Elliott, Joshua D. and Homes, Christopher C. and Skoropata, Elizabeth and Ueda, Hiroki and Liu, Biaolong and Paris, Eugenio and Romaguera, Arnau and Lee, Byungjune and He, Wei and Wang, Yu and Lee, Seng Huat and Choi, Hyeongi and Park, Sang-Youn and Mao, Zhiqiang and Calandra, Matteo and Jang, Hoyoung and Razzoli, Elia and Dean, Mark P. M. and Wang, Yao and Mitrano, Matteo},
  doi          = {10.1038/s41563-025-02254-2},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1584-1591},
  shortjournal = {Nat. Mater.},
  title        = {Symmetry-protected electronic metastability in an optically driven cuprate ladder},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Achieving small singlet–triplet energy gaps in polycyclic heteroaromatic emitters. <em>NMAT</em>, <em>24</em>(10), 1576-1583. (<a href='https://doi.org/10.1038/s41563-025-02309-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In polycyclic heteroaromatic (PHA) emitters, which possess great potential for application in ultrahigh-definition displays, the origin of a small singlet–triplet energy gap (ΔES1T1) and its relationship with the molecular structure still remain poorly established. Here we derive an effective expression for ΔES1T1, in which ΔES1T1 positively depends on 2KHL (where KHL is the exchange energy between the highest occupied molecular orbital and the lowest unoccupied molecular orbital (LUMO)) and on the energy gap between LUMO and LUMO + 1 (ΔELUMO–LUMO+1). This expression for ΔES1T1 is validated over a series of 100 reported PHA emitters. It allows us to easily identify various molecular design approaches for managing ΔES1T1 by synergistically regulating 2KHL and ΔELUMO–LUMO+1. The proof-of-concept PHA molecules were synthesized and characterized to further confirm the validity of this expression for ΔES1T1. Overall, our work provides a physical picture to not only modulate ΔES1T1 in emerging PHA emitters but also design and screen such materials with small ΔES1T1. Insight into the singlet–triplet energy gap and molecular structure relationship in polycyclic heteroaromatic molecules enables organic light-emitting diodes based on multiresonance thermally activated delayed fluorescence, with improved efficiency and roll-off.},
  archive      = {J_NMAT},
  author       = {Walia, Rajat and Xiong, Xin and Fan, Xiao-Chun and Chen, Ting-Feng and Wang, Hui and Wang, Kai and Shi, Yi-Zhong and Tang, Xun and Bredas, Jean-Luc and Adachi, Chihaya and Chen, Xian-Kai and Zhang, Xiao-Hong},
  doi          = {10.1038/s41563-025-02309-4},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1576-1583},
  shortjournal = {Nat. Mater.},
  title        = {Achieving small singlet–triplet energy gaps in polycyclic heteroaromatic emitters},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flatland wakes based on leaky hyperbolic polaritons. <em>NMAT</em>, <em>24</em>(10), 1569-1575. (<a href='https://doi.org/10.1038/s41563-025-02280-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hyperbolic polaritons facilitate nanoscale light manipulation, but strong field confinement limits their transmission across interfaces. Conversely, leaky waves can convert radiation from confined sources towards the far field. Here we combine hyperbolic polaritons and leaky wave radiation to demonstrate flatland leaky polaritonic wakes. We employ a mixed-dimensional van der Waals heterostructure consisting of a nanoscale waveguide strip on a van der Waals film. The waveguide mode, confined inside the hyperbolic light cone of the background film, enables efficient directional in-plane emission of fast phonon polaritons. The constructive interference of these leaky polaritons generates highly directional polaritonic wakes. Their spatial symmetry can be tailored through the orientation of the heterostructure with respect to the hyperbolic film dispersion. Leveraging van der Waals stacking, we also demonstrate effective acceleration and deceleration of polaritonic wakes by locally tailoring the leaky nano-waveguide dispersion through gradient thickness design. Our findings demonstrate that polaritonic wakes hold promise for integrated nanophotonic circuits. The authors combine the strong confinement of hyperbolic polaritons with leaky nano-waveguides to demonstrate directional in-plane emission of fast phonon polaritons and their acceleration and deceleration by tailoring waveguide dispersion.},
  archive      = {J_NMAT},
  author       = {Chen, Na and Teng, Hanchao and Hu, Hai and Liu, Min and Jiang, Chengyu and Xue, Zhuoxin and Zhu, Hualong and Gui, Jiayi and Li, Peining and Alù, Andrea and Dai, Qing},
  doi          = {10.1038/s41563-025-02280-0},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1569-1575},
  shortjournal = {Nat. Mater.},
  title        = {Flatland wakes based on leaky hyperbolic polaritons},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Twist-assisted intrinsic toughening in two-dimensional transition metal dichalcogenides. <em>NMAT</em>, <em>24</em>(10), 1561-1568. (<a href='https://doi.org/10.1038/s41563-025-02193-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Material fractures are typically irreversible, marking a one-time event leading to failure. Great efforts have been made to enhance both strength and fracture toughness of bulk materials for engineering applications, such as by introducing self-recovery and secondary breaking behaviours. In low-dimensional structures, two-dimensional materials often exhibit exceptional strength but accompanied by extreme brittleness. Here we discover that the toughness of two-dimensional materials can be enhanced without sacrificing strength—by simply twisting the layers. Through in situ scanning transmission electron microscopy, supported by nanoindentation and theoretical analysis, we reveal that twisted bilayer structures enable sequential fracture events: initial cracks heal to form stable grain boundaries, which then shield subsequent fracture tips from stress concentration. This process consumes additional energy compared with conventional fracture, with toughness enhancement tunable through twist angle adjustment. The intrinsic toughening mechanism via twisting, along with the emerging electronic properties of twistronics that are currently attracting substantial attention, presents an exciting opportunity for future devices. Intrinsic toughening in two-dimensional transition metal dichalcogenides can be achieved simply by twisting the layers. This twisting promotes cross-layer healing and grain boundary formation, which shield fracture tips from stress concentration.},
  archive      = {J_NMAT},
  author       = {Zheng, Xiaodong and Feng, Shizhe and Tsang, Chi Shing and Thi, Quoc Huy and Han, Wei and Wong, Lok Wing and Liu, Haijun and Lee, Chun-Sing and Lau, Shu Ping and Ly, Thuc Hue and Xu, Zhiping and Zhao, Jiong},
  doi          = {10.1038/s41563-025-02193-y},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1561-1568},
  shortjournal = {Nat. Mater.},
  title        = {Twist-assisted intrinsic toughening in two-dimensional transition metal dichalcogenides},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flaw-size-dependent mechanical interlayer coupling and edge-reconstruction embrittlement in van der waals materials. <em>NMAT</em>, <em>24</em>(10), 1554-1560. (<a href='https://doi.org/10.1038/s41563-025-02194-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Van der Waals (vdW) materials consisting of two-dimensional (2D) building blocks have strong in-plane covalent bonding and weak interlayer interactions. While monolayer 2D materials exhibit impressive fracture resistance, as demonstrated in hexagonal boron nitride (h-BN), preserving these remarkable properties in vdW materials remains a challenge. Here we reveal an anomalous mechanical interlayer coupling that involves interlayer-friction toughening and edge-reconstruction embrittlement during the fracture of multilayer h-BN. Both asynchronous and synchronous fracture modes and their flaw-size dependence are identified. Edge reconstruction in the synchronous fracture mode can eliminate a toughening mechanism induced by lattice asymmetry in monolayer h-BN, leading to embrittlement of the multilayer h-BN, while the asynchronous fracture mode results in greater fracture resistance. Such findings will provide fundamental guidelines for engineering interlayer interactions in vdW materials including heterostructures and layered architectures for better mechanical and functional performances. Fracture behaviours in multilayer h-BN, involving interlayer-friction toughening and edge-reconstruction embrittlement, are identified through in situ experiments and theoretical analyses.},
  archive      = {J_NMAT},
  author       = {Song, Zhigong and Zhang, Boyu and Yang, Yingchao and Gao, Guanhui and Tang, Daiming and Fang, Qiyi and Zhang, Youtian and Shin, Bongki and Steinbach, Doug and Ai, Qing and Zhao, Xuan and Han, Yimo and Padture, Nitin P. and Sheldon, Brian W. and Taniguchi, Takashi and Watanabe, Kenji and Gao, Huajian and Lou, Jun},
  doi          = {10.1038/s41563-025-02194-x},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1554-1560},
  shortjournal = {Nat. Mater.},
  title        = {Flaw-size-dependent mechanical interlayer coupling and edge-reconstruction embrittlement in van der waals materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Iterative sublattice amorphization facilitates exceptional processability in inorganic semiconductors. <em>NMAT</em>, <em>24</em>(10), 1545-1553. (<a href='https://doi.org/10.1038/s41563-024-02112-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cold-forming processing is a crucial means for the cost-effective production of metal and alloy products. However, this process often results in catastrophic fracture when applied to most inorganic semiconductors owing to their inherent brittleness. Here we report the unique room-temperature plastic deformation mechanism involving sublattice amorphization coupled with Ag-ion diffusion in inorganic semiconductors Ag2Te1–xSx (0.3 ≤ x ≤ 0.6), and an ultrahigh extensibility of up to 10,150%. Once subject to external stress, the crystalline Te/S sublattice undergoes a uniform transformation into an amorphous state, whereas the Ag cations continuously bond with Te/S anions, endowing bulk Ag2Te1–xSx with exceptional plastic deformability. Remarkably, even slight polishing can induce sublattice amorphization in the surface layers. Furthermore, this sublattice amorphization can be reversed to crystals through simple annealing, enlightening the iterative sublattice amorphization strategy, with which metal-like wire drawing, curving, forging and ultrahigh ductility have been obtained in bulk Ag2Te1–xSx at room temperature. These results highlight sublattice amorphization as a critical plastic deformation mechanism in silver chalcogenide inorganic semiconductors, which will facilitate their applications in flexible electronics and drive further exploration of more plastic inorganic semiconductors. Sublattice amorphization is revealed as the deformation mechanism of Ag2Te1–xSx (0.3 ≤ x ≤ 0.6), based on which an iterative crystalline–amorphous transition strategy is proposed to enable these bulk inorganic semiconductors with metal-like processability.},
  archive      = {J_NMAT},
  author       = {Wang, Yuechu and Li, Airan and Hong, Youran and Deng, Tianqi and Deng, Pan and Huang, Yi and Liu, Kai and Wang, Jiangwei and Fu, Chenguang and Zhu, Tiejun},
  doi          = {10.1038/s41563-024-02112-7},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1545-1553},
  shortjournal = {Nat. Mater.},
  title        = {Iterative sublattice amorphization facilitates exceptional processability in inorganic semiconductors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Warm metalworking for plastic manufacturing in brittle semiconductors. <em>NMAT</em>, <em>24</em>(10), 1538-1544. (<a href='https://doi.org/10.1038/s41563-025-02223-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Semiconductors are the core of modern electronics1. Because of their brittleness, semiconductors are usually processed by the complicated techniques of sputtering or deposition2–4, instead of the effective and versatile metalworking methods like rolling, extrusion and pressing used with metals5. Here we show that brittle semiconductors can be plastically manufactured with an extensibility as large as ~3,000% using warm metalworking, that is, plastic manufacturing at slightly elevated temperatures (empirically below 500 K). Many bulk brittle semiconductors, such as Cu2Se, Ag2Se and Bi90Sb10, can be processed like metals below 400–500 K into free-standing, large and high-quality films with a thickness from the macro-scale to the micrometre scale. A model based on temperature-dependent collective atomic displacement and thermal vibration is proposed to explain the superior plasticity. The warm-metalworked films can retain the excellent and tunable physical properties of the bulk versions, such as a high carrier mobility up to ~5,000 cm2 V−1 s−1 and tunable electrical conductivities over six orders of magnitude by adjusting the chemical composition. A case study in film thermoelectric devices demonstrates ultra-high normalized output power densities of 43–54 μW cm−2 K−2. This work suggests that brittle semiconductors can be manufactured by warm metalworking for applications in various electronics. The authors show that bulk brittle semiconductors can be plastically manufactured like metals by warm metalworking into free-standing, metre-scale films with decent physical properties.},
  archive      = {J_NMAT},
  author       = {Gao, Zhiqiang and Yang, Shiqi and Ma, Yupeng and Wei, Tian-Ran and Chen, Xiaohui and Zheng, Wenwen and Qiu, Pengfei and Zeng, Xiaoqin and Chen, Lidong and Shi, Xun},
  doi          = {10.1038/s41563-025-02223-9},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1538-1544},
  shortjournal = {Nat. Mater.},
  title        = {Warm metalworking for plastic manufacturing in brittle semiconductors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Streaming flexoelectricity in saline ice. <em>NMAT</em>, <em>24</em>(10), 1533-1537. (<a href='https://doi.org/10.1038/s41563-025-02332-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite 10% of the Earth’s surface being covered by ice, ice power remains untapped. Although ice is known to generate electricity upon bending via flexoelectricity1, the generated electric polarization per curvature, that is, the flexoelectric coefficient, is too small (~1–10 nC m−1) to be utilized for electromechanical devices. Here we demonstrate that doping ice with NaCl can enhance its flexoelectric coefficient 1,000-fold, to ~1–10 μC m−1. We find that this enhancement is due to the bending-induced streaming current along ice grain boundaries. On the basis of this mechanism, we fabricated flexural devices with an effective piezoelectric coefficient of ~4,000 pC N−1, which is comparable to that of the best piezoelectric materials. The high flexoelectricity of saline ice brings the vision of harnessing ice power one step closer to reality, and may also be relevant to the electrical activity of ice-covered terrestrial regions and icy ocean worlds such as Europa or Enceladus. In addition, the model for coupling between strain gradients and streaming currents is not limited to ice and provides a general framework for extracting electromechanical activity from liquid-infused porous solids. By doping ice with NaCl, it is shown that a flexoelectric coefficient of up to 10 μC m−1 is generated, enabling effective piezoelectric coefficients that are comparable to those of ceramics. This arises from the streaming current of quasi-liquid flow through grain boundaries from one side of the sample to the other.},
  archive      = {J_NMAT},
  author       = {Wen, X. and Ma, Q. and Liu, J. and Saeed, U. and Shen, S. and Catalan, G.},
  doi          = {10.1038/s41563-025-02332-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1533-1537},
  shortjournal = {Nat. Mater.},
  title        = {Streaming flexoelectricity in saline ice},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerated data-driven materials science with the materials project. <em>NMAT</em>, <em>24</em>(10), 1522-1532. (<a href='https://doi.org/10.1038/s41563-025-02272-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Materials Project was launched formally in 2011 to drive materials discovery forwards through high-throughput computation and open data. More than a decade later, the Materials Project has become an indispensable tool used by more than 600,000 materials researchers around the world. This Perspective describes how the Materials Project, as a data platform and a software ecosystem, has helped to shape research in data-driven materials science. We cover how sustainable software and computational methods have accelerated materials design while becoming more open source and collaborative in nature. Next, we present cases where the Materials Project was used to understand and discover functional materials. We then describe our efforts to meet the needs of an expanding user base, through technical infrastructure updates ranging from data architecture and cloud resources to interactive web applications. Finally, we discuss opportunities to better aid the research community, with the vision that more accessible and easy-to-understand materials data will result in democratized materials knowledge and an increasingly collaborative community. Materials design and informatics have become increasingly prominent over the past several decades. Using the Materials Project as an example, this Perspective discusses how properties are calculated and curated, how this knowledge can be used for materials discovery, and the challenges in modelling complex material systems or managing software architecture.},
  archive      = {J_NMAT},
  author       = {Horton, Matthew K. and Huck, Patrick and Yang, Ruo Xi and Munro, Jason M. and Dwaraknath, Shyam and Ganose, Alex M. and Kingsbury, Ryan S. and Wen, Mingjian and Shen, Jimmy X. and Mathis, Tyler S. and Kaplan, Aaron D. and Berket, Karlo and Riebesell, Janosh and George, Janine and Rosen, Andrew S. and Spotte-Smith, Evan W. C. and McDermott, Matthew J. and Cohen, Orion A. and Dunn, Alex and Kuner, Matthew C. and Rignanese, Gian-Marco and Petretto, Guido and Waroquiers, David and Griffin, Sinead M. and Neaton, Jeffrey B. and Chrzan, Daryl C. and Asta, Mark and Hautier, Geoffroy and Cholia, Shreyas and Ceder, Gerbrand and Ong, Shyue Ping and Jain, Anubhav and Persson, Kristin A.},
  doi          = {10.1038/s41563-025-02272-0},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1522-1532},
  shortjournal = {Nat. Mater.},
  title        = {Accelerated data-driven materials science with the materials project},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerating development of polycyclic heteroaromatic emitters with small singlet–triplet energy gaps. <em>NMAT</em>, <em>24</em>(10), 1520-1521. (<a href='https://doi.org/10.1038/s41563-025-02352-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A simple physical model based on electronic interactions between excited configurations enables the quick and reliable prediction of singlet–triplet energy gaps in polycyclic heteroaromatic emitters. Guided by this model, organic emitters with small singlet–triplet energy gaps can be designed for applications in organic light-emitting diodes with high efficiency and colour purity.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02352-1},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1520-1521},
  shortjournal = {Nat. Mater.},
  title        = {Accelerating development of polycyclic heteroaromatic emitters with small singlet–triplet energy gaps},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Designer lipids for delivering mRNA to the brain. <em>NMAT</em>, <em>24</em>(10), 1518-1519. (<a href='https://doi.org/10.1038/s41563-025-02184-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticles formulated with ionizable lipids inspired by brain-targeting small molecules facilitate the delivery of mRNA past the blood–brain barrier and into the brain.},
  archive      = {J_NMAT},
  author       = {Han, Emily L. and Safford, Hannah C. and Mitchell, Michael J.},
  doi          = {10.1038/s41563-025-02184-z},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1518-1519},
  shortjournal = {Nat. Mater.},
  title        = {Designer lipids for delivering mRNA to the brain},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Entering the age of designer protein materials. <em>NMAT</em>, <em>24</em>(10), 1515-1517. (<a href='https://doi.org/10.1038/s41563-025-02372-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning-based generative tools are used to design protein building blocks with well-defined directional protein bonding interactions, allowing for the generation of a variety of scalable protein assemblies from a small set of reusable subunits.},
  archive      = {J_NMAT},
  author       = {Zhang, Shuguang},
  doi          = {10.1038/s41563-025-02372-x},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1515-1517},
  shortjournal = {Nat. Mater.},
  title        = {Entering the age of designer protein materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capturing the dance of nanoparticles in crystals. <em>NMAT</em>, <em>24</em>(10), 1513-1514. (<a href='https://doi.org/10.1038/s41563-025-02342-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phonon dynamics of self-assembled two-dimensional nanoparticle crystals is resolved by liquid-phase transmission electron microscopy.},
  archive      = {J_NMAT},
  author       = {Shen, Eric and Gang, Oleg},
  doi          = {10.1038/s41563-025-02342-3},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1513-1514},
  shortjournal = {Nat. Mater.},
  title        = {Capturing the dance of nanoparticles in crystals},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Salt turns ice into a powerful flexoelectric material. <em>NMAT</em>, <em>24</em>(10), 1511-1512. (<a href='https://doi.org/10.1038/s41563-025-02328-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The brine network in common sea ice boosts its flexoelectric response by three orders of magnitude, matching high-quality piezoelectrics and pointing to cost-effective energy harvesters for cold environments.},
  archive      = {J_NMAT},
  author       = {Lee, Daesu},
  doi          = {10.1038/s41563-025-02328-1},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1511-1512},
  shortjournal = {Nat. Mater.},
  title        = {Salt turns ice into a powerful flexoelectric material},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering a light-induced metastable state. <em>NMAT</em>, <em>24</em>(10), 1509-1510. (<a href='https://doi.org/10.1038/s41563-025-02303-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Investigation of a long-lived metallic state in a low-dimensional cuprate demonstrates a strategy for on-demand light-driven metastability.},
  archive      = {J_NMAT},
  author       = {Lee, Wei-Sheng},
  doi          = {10.1038/s41563-025-02303-w},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1509-1510},
  shortjournal = {Nat. Mater.},
  title        = {Engineering a light-induced metastable state},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Interlayer mechanical coupling in van der waals materials. <em>NMAT</em>, <em>24</em>(10), 1507-1508. (<a href='https://doi.org/10.1038/s41563-025-02239-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fracture behaviours and related atomic-scale phenomena of van der Waals materials are elucidated through the use of electron microscopy, which highlights the important role of interlayer coupling and re-bonding, providing valuable insights for engineering mechanical properties of layered materials and structures.},
  archive      = {J_NMAT},
  author       = {Kim, Kangsik and Lee, Zonghoon},
  doi          = {10.1038/s41563-025-02239-1},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1507-1508},
  shortjournal = {Nat. Mater.},
  title        = {Interlayer mechanical coupling in van der waals materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Plastic processing of bulk semiconductors. <em>NMAT</em>, <em>24</em>(10), 1505-1506. (<a href='https://doi.org/10.1038/s41563-025-02283-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bulk inorganic semiconductors can show remarkable plasticity and extensibility, defying their inherent brittleness and enabling opportunities in advanced semiconductor manufacturing and processing.},
  archive      = {J_NMAT},
  author       = {Nakamura, Atsutomo and Li, Yan},
  doi          = {10.1038/s41563-025-02283-x},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1505-1506},
  shortjournal = {Nat. Mater.},
  title        = {Plastic processing of bulk semiconductors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mikhail eremets (1949–2024). <em>NMAT</em>, <em>24</em>(10), 1504. (<a href='https://doi.org/10.1038/s41563-025-02333-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pioneer of high-pressure physics and superconductivity.},
  archive      = {J_NMAT},
  author       = {Boeri, Lilia and Drozdov, Alexander P. and Ksenofontov, Vadim and Du, Feng and Minkov, Vasily S. and Pöschl, Ulrich},
  doi          = {10.1038/s41563-025-02333-4},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1504},
  shortjournal = {Nat. Mater.},
  title        = {Mikhail eremets (1949–2024)},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanical insights from functional materials. <em>NMAT</em>, <em>24</em>(10), 1503. (<a href='https://doi.org/10.1038/s41563-025-02378-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our knowledge of deformation behaviour in functional materials is pushing the frontiers of mechanics, informing design strategies and enabling scalable manufacturing.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02378-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1503},
  shortjournal = {Nat. Mater.},
  title        = {Mechanical insights from functional materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmeth">NMETH - 35</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: IPipet: Sample handling using a tablet. <em>NMETH</em>, <em>22</em>(9), 1996. (<a href='https://doi.org/10.1038/s41592-025-02804-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zielinski, Dina and Gordon, Assaf and Zaks, Benjamin L. and Erlich, Yaniv},
  doi          = {10.1038/s41592-025-02804-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1996},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: IPipet: Sample handling using a tablet},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A universal language for finding mass spectrometry data patterns. <em>NMETH</em>, <em>22</em>(9), 1995. (<a href='https://doi.org/10.1038/s41592-025-02785-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Damiani, Tito and Jarmusch, Alan K. and Aron, Allegra T. and Petras, Daniel and Phelan, Vanessa V. and Zhao, Haoqi Nina and Bittremieux, Wout and Acharya, Deepa D. and Ahmed, Mohammed M. A. and Bauermeister, Anelize and Bertin, Matthew J. and Boudreau, Paul D. and Borges, Ricardo M. and Bowen, Benjamin P. and Brown, Christopher J. and Chagas, Fernanda O. and Clevenger, Kenneth D. and Correia, Mario S. P. and Crandall, William J. and Crüsemann, Max and Fahy, Eoin and Fiehn, Oliver and Garg, Neha and Gerwick, William H. and Gilbert, Jeffrey R. and Globisch, Daniel and Gomes, Paulo Wender P. and Heuckeroth, Steffen and James, C. Andrew and Jarmusch, Scott A. and Kakhkhorov, Sarvar A. and Kang, Kyo Bin and Kessler, Nikolas and Kersten, Roland D. and Kim, Hyunwoo and Kirk, Riley D. and Kohlbacher, Oliver and Kontou, Eftychia E. and Liu, Ken and Lizama-Chamu, Itzel and Luu, Gordon T. and Luzzatto Knaan, Tal and Mannochio-Russo, Helena and Marty, Michael T. and Matsuzawa, Yuki and McAvoy, Andrew C. and McCall, Laura-Isobel and Mohamed, Osama G. and Nahor, Omri and Neuweger, Heiko and Niedermeyer, Timo H. J. and Nishida, Kozo and Northen, Trent R. and Overdahl, Kirsten E. and Rainer, Johannes and Reher, Raphael and Rodriguez, Elys and Sachsenberg, Timo T. and Sanchez, Laura M. and Schmid, Robin and Stevens, Cole and Subramaniam, Shankar and Tian, Zhenyu and Tripathi, Ashootosh and Tsugawa, Hiroshi and van der Hooft, Justin J. J. and Vicini, Andrea and Walter, Axel and Weber, Tilmann and Xiong, Quanbo and Xu, Tao and Pluskal, Tomáš and Dorrestein, Pieter C. and Wang, Mingxun},
  doi          = {10.1038/s41592-025-02785-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1995},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A universal language for finding mass spectrometry data patterns},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unified mass imaging maps the lipidome of vertebrate development. <em>NMETH</em>, <em>22</em>(9), 1981-1994. (<a href='https://doi.org/10.1038/s41592-025-02771-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryo development entails the formation of anatomical structures with distinct biochemical compositions. Compared with the wealth of knowledge on gene regulation, our understanding of metabolic programs operating during embryogenesis is limited. Mass spectrometry imaging (MSI) has the potential to map the distribution of metabolites across embryo development. Here we established uMAIA, an analytical framework for the joint analysis of large MSI datasets, which enables the construction of multidimensional metabolomic atlases. Employing this framework, we mapped the four-dimensional (4D) distribution of over a hundred lipids at micrometric resolution in Danio rerio embryos. We discovered metabolic trajectories that unfold in concert with morphogenesis and revealed spatially organized biochemical coordination overlooked by bulk measurements. Interestingly, lipid mapping revealed unexpected distributions of sphingolipid and triglyceride species, suggesting their involvement in pattern establishment and organ development. Our approach empowers a new generation of whole-organism metabolomic atlases and enables the discovery of spatially organized metabolic circuits. uMAIA is an analytical framework designed to enable the construction of metabolic atlases at high resolution using mass spectrometry imaging data.},
  archive      = {J_NMETH},
  author       = {Schede, Halima Hannah and Alieh, Leila Haj Abdullah and Rohde, Laurel Ann and Herrera, Antonio and Schlaeppi, Anjalie and Valentin, Guillaume and Gargoori Motlagh, Alireza and Dominguez Mantes, Albert and Jollivet, Chloe and Paz-Montoya, Jonathan and Capolupo, Laura and Khven, Irina and Oates, Andrew C. and D’Angelo, Giovanni and La Manno, Gioele},
  doi          = {10.1038/s41592-025-02771-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1981-1994},
  shortjournal = {Nat. Meth.},
  title        = {Unified mass imaging maps the lipidome of vertebrate development},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions. <em>NMETH</em>, <em>22</em>(9), 1964-1980. (<a href='https://doi.org/10.1038/s41592-025-02793-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues. TCAT is a pipeline that can simultaneously capture gene expression programs related to T cell subsets and activation states for accurate T cell characterization.},
  archive      = {J_NMETH},
  author       = {Kotliar, Dylan and Curtis, Michelle and Agnew, Ryan and Weinand, Kathryn and Nathan, Aparna and Baglaenko, Yuriy and Slowikowski, Kamil and Zhao, Yu and Sabeti, Pardis C. and Rao, Deepak A. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41592-025-02793-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1964-1980},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput profiling of chemical-induced gene expression across 93,644 perturbations. <em>NMETH</em>, <em>22</em>(9), 1954-1963. (<a href='https://doi.org/10.1038/s41592-025-02781-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this Resource, we present an extensive dataset of chemical-induced gene signatures (CIGS), encompassing expression patterns of 3,407 genes regulating key biological processes in 2 human cell lines exposed to 13,221 compounds across 93,664 perturbations. This dataset encompasses 319,045,108 gene expression events, generated through 2 high-throughput technologies: the previously documented high-throughput sequencing-based high-throughput screening (HTS2) and the newly developed highly multiplexed and parallel sequencing (HiMAP-seq). Our results show that HiMAP-seq is comparable to RNA sequencing, but can profile the expression of thousands of genes across thousands of samples in one single test by utilizing a pooled-sample strategy. We further illustrate CIGS’s utility in elucidating the mechanism of action of unannotated small molecules, like ligustroflavone and 2,4-dihydroxybenzaldehyde, and to identify perturbation-induced cell states, such as those resistant to ferroptosis. The full dataset is publicly accessible at https://cigs.iomicscloud.com/ . CIGS is a high-resolution dataset comprising chemical perturbation-induced 319,045,108 gene expression events.},
  archive      = {J_NMETH},
  author       = {Xiang, Lei and Wang, Yumei and Shao, Wei and Li, Qingzhou and Yu, Xiankuo and Wei, Mingming and Gui, Yu and Li, Shengrong and Qin, Pan and Hu, Chao and Zhang, Guochen and Zhang, Xianwen and Wang, Jiawen and Li, Yingying and An, Jun and Luo, Yan and Liao, Yile and Deng, Jinghong and Tai, Xinran and Xu, Richard Y. and Huang, Lijun and Guo, Dale and Zhang, Guanbin and Xie, Zhi and Deng, Yun and Xu, Junquan and Wang, Dong},
  doi          = {10.1038/s41592-025-02781-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1954-1963},
  shortjournal = {Nat. Meth.},
  title        = {High-throughput profiling of chemical-induced gene expression across 93,644 perturbations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI. <em>NMETH</em>, <em>22</em>(9), 1944-1953. (<a href='https://doi.org/10.1038/s41592-025-02798-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concurrent recording of electroencephalogram (EEG) and functional magnetic resonance imaging (fMRI) signals reveals cross-scale neurovascular dynamics crucial for explaining fundamental linkages between function and behaviors. However, MRI scanners generate artifacts for EEG detection. Despite existing denoising methods, cabled connections to EEG receivers are susceptible to environmental fluctuations inside MRI scanners, creating baseline drifts that complicate EEG signal retrieval from the noisy background. Here we show that a wireless integrated sensing detector for simultaneous EEG and MRI can encode fMRI and EEG signals on distinct sidebands of the detector’s oscillation wave for detection by a standard MRI console over the entire duration of the fMRI sequence. Local field potential and fMRI maps are retrieved through low-pass and high-pass filtering of frequency-demodulated signals. From optogenetically stimulated somatosensory cortex in ChR2-transfected Sprague Dawley rats, positive correlation between evoked local field potential and fMRI signals validates strong neurovascular coupling, enabling cross-scale brain mapping with this two-in-one transducer. WISDEM is a hybrid detector for simultaneous EEG and fMRI recordings without artifacts or crosstalk, which allows access to neural activity at high temporal and spatial resolution, respectively, as demonstrated in rats.},
  archive      = {J_NMETH},
  author       = {Chen, Yi and Qian, Wei and Razansky, Daniel and Yu, Xin and Qian, Chunqi},
  doi          = {10.1038/s41592-025-02798-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1944-1953},
  shortjournal = {Nat. Meth.},
  title        = {WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A versatile miniature two-photon microscope enabling multicolor deep-brain imaging. <em>NMETH</em>, <em>22</em>(9), 1935-1943. (<a href='https://doi.org/10.1038/s41592-025-02780-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present FHIRM-TPM 3.0, a 2.6 g miniature two-photon microscope capable of multicolor deep-brain imaging in freely behaving mice. The system was integrated with a broadband anti-resonant hollow-core fiber featuring low transmission loss, minimal dispersion from 700 nm to 1,060 nm and high tolerance of laser power. By correcting chromatic and spherical aberrations and optimizing the fluorescence collection aperture, we achieved cortical neuronal imaging at depths exceeding 820 μm and, using a GRIN lens, hippocampal Ca2+ imaging at single dendritic spine resolution. Moreover, we engineered three interchangeable parfocal objectives, allowing for a tenfold scalable field of view up to 1 × 0.8 mm², with lateral resolutions ranging from 0.68 μm to 1.46 μm. By multicolor imaging at excitation wavelengths of 780 nm, 920 nm and 1,030 nm, we investigated mitochondrial and cytosolic Ca2+ activities relative to the deposition of amyloid plaques in the cortex of awake APP/PS1 transgenic mice. Thus, FHIRM-TPM 3.0 provides a versatile imaging system suitable for diverse brain imaging scenarios. FHIRM-TPM 3.0 is a miniature microscope for multicolor two-photon imaging in freely moving mice. In addition to the multicolor imaging abilities achieved with the help of a specially designed optical fiber, the microscope is also compatible with multiple lenses for a choice of field of view and resolution.},
  archive      = {J_NMETH},
  author       = {Wu, Runlong and Zhao, Chunzhu and Qiu, Shan and Zhu, Yufei and Zhang, Lifeng and Fu, Qiang and Hu, Yanhui and Wu, Dakun and Yu, Fei and Zhou, Fangxu and Huang, Haipeng and Zhang, Yunfeng and Wang, Xianhua and Wang, Aimin and Cheng, Heping},
  doi          = {10.1038/s41592-025-02780-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1935-1943},
  shortjournal = {Nat. Meth.},
  title        = {A versatile miniature two-photon microscope enabling multicolor deep-brain imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy. <em>NMETH</em>, <em>22</em>(9), 1923-1934. (<a href='https://doi.org/10.1038/s41592-025-02794-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volume correlative light and electron microscopy (vCLEM) is a powerful imaging technique that enables the visualization of fluorescently labeled proteins within their ultrastructural context. Currently, vCLEM alignment relies on time-consuming and subjective manual methods. This paper presents CLEM-Reg, an algorithm that automates the three-dimensional alignment of vCLEM datasets by leveraging probabilistic point cloud registration techniques. Point clouds are derived from segmentations of common structures in each modality, created by state-of-the-art open-source methods. CLEM-Reg drastically reduces the registration time of vCLEM datasets to a few minutes and achieves correlation of fluorescent signal to submicron target structures in electron microscopy on three newly acquired vCLEM benchmark datasets. CLEM-Reg was then used to automatically obtain vCLEM overlays to unambiguously identify TGN46-positive transport carriers involved in protein trafficking between the trans-Golgi network and plasma membrane. Datasets are available on EMPIAR and BioStudies, and a napari plugin is provided to aid end-user adoption. CLEM-Reg automates the three-dimensional alignment of volume correlative light and electron microscopy datasets by leveraging probabilistic point cloud registration techniques for fast and accurate results across diverse datasets.},
  archive      = {J_NMETH},
  author       = {Krentzel, Daniel and Elphick, Matouš and Domart, Marie-Charlotte and Peddie, Christopher J. and Laine, Romain F. and Shand, Cameron and Henriques, Ricardo and Collinson, Lucy M. and Jones, Martin L.},
  doi          = {10.1038/s41592-025-02794-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1923-1934},
  shortjournal = {Nat. Meth.},
  title        = {CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE. <em>NMETH</em>, <em>22</em>(9), 1911-1922. (<a href='https://doi.org/10.1038/s41592-025-02770-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while preserving crucial spatial context within tissues. However, existing ST platforms are constrained by high costs, long turnaround times, low resolution, limited gene coverage and inherently small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that reconstructs large-scale, super-resolution gene expression landscapes and automatically annotates cellular-level tissue architecture in samples exceeding capture areas of current ST platforms. The performance of iSCALE was assessed by comprehensive evaluations involving benchmarking experiments, immunohistochemistry staining and manual annotations by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion-associated cellular characteristics undetectable by conventional ST experiments. Our results demonstrate the utility of iSCALE in analyzing large tissues by enabling unbiased annotation, resolving cell type composition, mapping cellular microenvironments and revealing spatial features beyond the reach of standard ST analysis or routine histopathological assessment. iSCALE leverages histology and spatial transcriptomics to infer gene expression at super resolution in large tissues.},
  archive      = {J_NMETH},
  author       = {Schroeder, Amelia and Loth, Melanie L. and Luo, Chunyu and Yao, Sicong and Yan, Hanying and Zhang, Daiwei and Piya, Sarbottam and Plowey, Edward and Hu, Wenxing and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Chan, Su Jing and Reynolds, Taylor L. and Carlile, Thomas and Cullen, Patrick and Sung, Ji-Youn and Tsai, Hui-Hsin and Park, Jeong Hwan and Hwang, Tae Hyun and Zhang, Baohong and Li, Mingyao},
  doi          = {10.1038/s41592-025-02770-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1911-1922},
  shortjournal = {Nat. Meth.},
  title        = {Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial gene expression at single-cell resolution from histology using deep learning with GHIST. <em>NMETH</em>, <em>22</em>(9), 1900-1910. (<a href='https://doi.org/10.1038/s41592-025-02795-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The increased use of spatially resolved transcriptomics provides new biological insights into disease mechanisms. However, the high cost and complexity of these methods are barriers to broader application. Consequently, methods have been created to predict spot-based gene expression from routinely collected histology images. Recent benchmarking showed that current methodologies have limited accuracy and spatial resolution, constraining translational capacity. Here, we introduce GHIST, a deep learning-based framework that predicts spatial gene expression at single-cell resolution by leveraging subcellular spatial transcriptomics and synergistic relationships between multiple layers of biological information. We validated GHIST using public datasets and The Cancer Genome Atlas data, demonstrating its flexibility across different spatial resolutions and superior performance. Our results underscore the utility of in silico generation of single-cell spatial gene expression measurements and the capacity to enrich existing datasets with a spatially resolved omics modality, paving the way for scalable multi-omics analysis and biomarker identification. GHIST is a deep learning-based method that can predict spatial gene expression at high resolution using histology data.},
  archive      = {J_NMETH},
  author       = {Fu, Xiaohang and Cao, Yue and Bian, Beilei and Wang, Chuhan and Graham, Dinny and Pathmanathan, Nirmala and Patrick, Ellis and Kim, Jinman and Yang, Jean Yee Hwa},
  doi          = {10.1038/s41592-025-02795-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1900-1910},
  shortjournal = {Nat. Meth.},
  title        = {Spatial gene expression at single-cell resolution from histology using deep learning with GHIST},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ultra-high-scale cytometry-based cellular interaction mapping. <em>NMETH</em>, <em>22</em>(9), 1887-1899. (<a href='https://doi.org/10.1038/s41592-025-02744-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular interactions are of fundamental importance, orchestrating organismal development, tissue homeostasis and immunity. Recently, powerful methods that use single-cell genomic technologies to dissect physically interacting cells have been developed. However, these approaches are characterized by low cellular throughput, long processing times and high costs and are typically restricted to predefined cell types. Here we introduce Interact-omics, a cytometry-based framework to accurately map cellular landscapes and cellular interactions across all immune cell types at ultra-high resolution and scale. We demonstrate the utility of our approach to study kinetics, mode of action and personalized response prediction of immunotherapies, and organism-wide shifts in cellular composition and cellular interaction dynamics following infection in vivo. Our scalable framework can be applied a posteriori to existing cytometry datasets or incorporated into newly designed cytometry-based studies to map cellular interactions with a broad range of applications from fundamental biology to applied biomedicine. Interact-omics, a high-throughput cytometry-based framework, resolves the cellular interaction landscape.},
  archive      = {J_NMETH},
  author       = {Vonficht, Dominik and Jopp-Saile, Lea and Yousefian, Schayan and Flore, Viktoria and Simó Vesperinas, Inés and Teuber, Ruth and Avanesyan, Bogdan and Luo, Yanjiang and Röthemeier, Caroline and Grünschläger, Florian and Fernandez-Vaquero, Mirian and Fregona, Vincent and Ordoñez-Rueda, Diana and Schmalbrock, Laura K. and Deininger, Luca and Yamachui Sitcheu, Angelo Jovin and Gu, Zuguang and Funk, Maja C. and Mikut, Ralf and Heikenwälder, Mathias and Eggert, Angelika and von Stackelberg, Arend and Kobold, Sebastian and Krönke, Jan and Keller, Ulrich and Trumpp, Andreas and Hegazy, Ahmed N. and Eckert, Cornelia and Hübschmann, Daniel and Haas, Simon},
  doi          = {10.1038/s41592-025-02744-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1887-1899},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-scale cytometry-based cellular interaction mapping},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Laser flash melting cryo-EM samples to overcome preferred orientation. <em>NMETH</em>, <em>22</em>(9), 1880-1886. (<a href='https://doi.org/10.1038/s41592-025-02796-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sample preparation remains a bottleneck for protein structure determination by cryo-electron microscopy. A frequently encountered issue is that proteins adsorb to the air–water interface of the sample in a limited number of orientations. This makes it challenging to obtain high-resolution reconstructions, or may even cause projects to fail altogether. We have previously observed that laser flash melting and revitrification of cryo-EM samples reduces preferred orientation for large, symmetric particles. Here we demonstrate that our method can in fact be used to scramble the orientation of proteins of a range of sizes and symmetries. The effect can be enhanced for some proteins by increasing the heating rate during flash melting or by depositing amorphous ice onto the sample prior to revitrification. This also allows us to shed light onto the underlying mechanism. Our experiments establish a set of tools for overcoming preferred orientation that can be easily integrated into existing workflows. Individual proteins tend to adopt preferred orientations when subjected to vitrification for cryo-electron microscopy analysis. A laser flash melting procedure followed by rapid revitrification provides a simple approach to mitigate this issue, reducing the number of micrographs required for successful structure determination at high-resolution.},
  archive      = {J_NMETH},
  author       = {Straub, Monique S. and Harder, Oliver F. and Mowry, Nathan J. and Barrass, Sarah V. and Hruby, Jakub and Drabbels, Marcel and Lorenz, Ulrich J.},
  doi          = {10.1038/s41592-025-02796-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1880-1886},
  shortjournal = {Nat. Meth.},
  title        = {Laser flash melting cryo-EM samples to overcome preferred orientation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biophysics-based protein language models for protein engineering. <em>NMETH</em>, <em>22</em>(9), 1868-1879. (<a href='https://doi.org/10.1038/s41592-025-02776-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models trained on evolutionary data have emerged as powerful tools for predictive problems involving protein sequence, structure and function. However, these models overlook decades of research into biophysical factors governing protein function. We propose mutational effect transfer learning (METL), a protein language model framework that unites advanced machine learning and biophysical modeling. Using the METL framework, we pretrain transformer-based neural networks on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics. We fine-tune METL on experimental sequence–function data to harness these biophysical signals and apply them when predicting protein properties like thermostability, catalytic activity and fluorescence. METL excels in challenging protein engineering tasks like generalizing from small training sets and position extrapolation, although existing methods that train on evolutionary signals remain powerful for many types of experimental assays. We demonstrate METL’s ability to design functional green fluorescent protein variants when trained on only 64 examples, showcasing the potential of biophysics-based protein language models for protein engineering. Mutational effect transfer learning (METL) is a protein language model framework that unites machine learning and biophysical modeling. Transformer-based neural networks are pretrained on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics.},
  archive      = {J_NMETH},
  author       = {Gelman, Sam and Johnson, Bryce and Freschlin, Chase R. and Sharma, Arnav and D’Costa, Sameer and Peters, John and Gitter, Anthony and Romero, Philip A.},
  doi          = {10.1038/s41592-025-02776-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1868-1879},
  shortjournal = {Nat. Meth.},
  title        = {Biophysics-based protein language models for protein engineering},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations. <em>NMETH</em>, <em>22</em>(9), 1857-1867. (<a href='https://doi.org/10.1038/s41592-025-02797-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modifications (PTMs) are critical regulators of protein function, and their disruption is a key mechanism by which missense variants contribute to disease. Accurate PTM site prediction using deep learning can help identify PTM-altering variants, but progress has been limited by the lack of large, high-quality training datasets. Here, we introduce PTMAtlas, a curated compendium of 397,524 PTM sites generated through systematic reprocessing of 241 public mass-spectrometry datasets, and DeepMVP, a deep learning framework trained on PTMAtlas to predict PTM sites for phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation. DeepMVP substantially outperforms existing tools across all six PTM types. Its application to predicting PTM-altering missense variants shows strong concordance with experimental results, validated using literature-curated variants and cancer proteogenomic datasets. Together, PTMAtlas and DeepMVP provide a robust platform for PTM research and a scalable framework for assessing the functional consequences of coding variants through the lens of PTMs. DeepMVP is a deep learning framework for predicting PTM sites and variant-induced alterations across six modification types, including phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation.},
  archive      = {J_NMETH},
  author       = {Wen, Bo and Wang, Chenwei and Li, Kai and Han, Ping and Holt, Matthew V. and Savage, Sara R. and Lei, Jonathan T. and Dou, Yongchao and Shi, Zhiao and Li, Yi and Zhang, Bing},
  doi          = {10.1038/s41592-025-02797-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1857-1867},
  shortjournal = {Nat. Meth.},
  title        = {DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data. <em>NMETH</em>, <em>22</em>(9), 1846-1856. (<a href='https://doi.org/10.1038/s41592-025-02773-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations such as copy number alterations accumulate during cancer progression, driving intratumor heterogeneity that impacts therapy effectiveness. Understanding the characteristics and spatial distribution of genetically distinct subclones is essential for unraveling tumor evolution and improving cancer treatment. Here we present Clonalscope, a subclone detection method using copy number profiles, applicable to spatial transcriptomics and single-cell sequencing data. Clonalscope implements a nested Chinese Restaurant Process to identify de novo tumor subclones, which can incorporate prior information from matched bulk DNA sequencing data for improved subclone detection and malignant cell labeling. On single-cell RNA sequencing and single-cell assay for transposase-accessible chromatin using sequencing data from gastrointestinal tumors, Clonalscope successfully labeled malignant cells and identified genetically different subclones with thorough validations. On spatial transcriptomics data from various primary and metastasized tumors, Clonalscope labeled malignant spots, traced subclones and identified spatially segregated subclones with distinct differentiation levels and expression of genes associated with drug resistance and survival. Clonalscope is a method for cancer subclone detection leveraging copy number profiles estimated using spatial and single-cell sequencing data.},
  archive      = {J_NMETH},
  author       = {Wu, Chi-Yun and Rong, Jiazhen and Sathe, Anuja and Hess, Paul R. and Lau, Billy T. and Grimes, Susan M. and Huang, Sijia and Ji, Hanlee P. and Zhang, Nancy R.},
  doi          = {10.1038/s41592-025-02773-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1846-1856},
  shortjournal = {Nat. Meth.},
  title        = {Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scvi-hub: An actionable repository for model-driven single-cell analysis. <em>NMETH</em>, <em>22</em>(9), 1836-1845. (<a href='https://doi.org/10.1038/s41592-025-02799-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing availability of single-cell omics datasets presents new opportunities for reuse, while challenges in data transfer, normalization and integration remain a barrier. Here we present scvi-hub: a platform for efficiently sharing and accessing single-cell omics datasets using pretrained probabilistic models. It enables immediate execution of fundamental tasks like visualization, imputation, annotation and deconvolution on new query datasets using state-of-the-art methods, with massively reduced storage and compute requirements. We show that pretrained models support efficient analysis of large references, including the CZI CELLxGENE Discover Census. Scvi-hub is built within the scvi-tools open-source environment and integrated into scverse. Scvi-hub offers a scalable and user-friendly framework for accessing and contributing to a growing ecosystem of ready-to-use models and datasets, thus putting the power of atlas-level analysis at the fingertips of a broad community of users. Scvi-hub is a versatile and efficient platform for model-based analysis of single-cell sequencing studies with access to a diverse array of datasets and downstream analysis.},
  archive      = {J_NMETH},
  author       = {Ergen, Can and Pour Amiri, Valeh Valiollah and Kim, Martin and Kronfeld, Ori and Streets, Aaron and Gayoso, Adam and Yosef, Nir},
  doi          = {10.1038/s41592-025-02799-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1836-1845},
  shortjournal = {Nat. Meth.},
  title        = {Scvi-hub: An actionable repository for model-driven single-cell analysis},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues. <em>NMETH</em>, <em>22</em>(9), 1824-1835. (<a href='https://doi.org/10.1038/s41592-025-02774-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) are essential regulators of RNA fate and function. A long-standing challenge in studying RBP regulation has been mapping RNA interactomes within the dynamic transcriptomic landscape, especially in single-cell contexts and primary tissues. Here we introduce MAPIT-seq (modification added to RBP interacting transcript-sequencing), which uses an antibody-directed editing strategy to map genome-wide in situ RBP–RNA interactions and gene expression concurrently. We demonstrate MAPIT-seq’s robustness across multiple RBPs and systematically analyze RNA substrates associated with core polycomb repressive complex 2 (PRC2) components. MAPIT-seq is also applicable to frozen tissue sections, enabling the mapping of RBP roles during brain development. Importantly, we develop high-throughput single-cell MAPIT-seq (scMAPIT-seq) to reveal cell stage-specific RBP regulation. In summary, MAPIT-seq expands multi-omics profiling, providing an effective framework to study post-transcriptional regulation in dynamic biological processes and clinically relevant scenarios. This work presents MAPIT-seq, an antibody-guided RNA editing method that enables co-profiling of RBP-RNA interactions and gene expression at single-cell resolution and in tissue contexts.},
  archive      = {J_NMETH},
  author       = {Cheng, Qi-Xuan and Xie, Gang and Zhang, Xiangyu and Wang, Jie and Ding, Shuangjin and Wu, Yi-Xia and Shi, Ming and Duan, Fei-Fei and Wan, Zi-Li and Wei, Jing-Jia and Xiao, Junyu and Wang, Yangming},
  doi          = {10.1038/s41592-025-02774-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1824-1835},
  shortjournal = {Nat. Meth.},
  title        = {Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A realistic phantom dataset for benchmarking cryo-ET data annotation. <em>NMETH</em>, <em>22</em>(9), 1819-1823. (<a href='https://doi.org/10.1038/s41592-025-02800-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) is a powerful technique for imaging molecular complexes in their native cellular environments. However, identifying the vast majority of molecular species in cellular tomograms remains prohibitively difficult. Machine learning (ML) methods provide an opportunity to automate the annotation process, but algorithm development has been hindered by the lack of large, standardized datasets. Here we present an experimental phantom dataset with comprehensive ground-truth annotations for six molecular species to spur new algorithm development and benchmark existing tools. This annotated dataset is available on the CryoET Data Portal with infrastructure to streamline access for methods developers across fields. A standardized, realistic phantom dataset consisting of ground-truth annotations for six diverse molecular species is provided as a community resource for cryo-electron-tomography algorithm benchmarking.},
  archive      = {J_NMETH},
  author       = {Peck, Ariana and Yu, Yue and Schwartz, Jonathan and Cheng, Anchi and Ermel, Utz Heinrich and Hutchings, Joshua and Kandel, Saugat and Kimanius, Dari and Montabana, Elizabeth A. and Serwas, Daniel and Siems, Hannah and Wang, Feng and Zhao, Zhuowen and Zheng, Shawn and Haury, Matthias and Agard, David A. and Potter, Clinton S. and Carragher, Bridget and Harrington, Kyle and Paraan, Mohammadreza},
  doi          = {10.1038/s41592-025-02800-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1819-1823},
  shortjournal = {Nat. Meth.},
  title        = {A realistic phantom dataset for benchmarking cryo-ET data annotation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the physical hallmarks of cancer. <em>NMETH</em>, <em>22</em>(9), 1800-1818. (<a href='https://doi.org/10.1038/s41592-024-02564-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The physical microenvironment plays a crucial role in tumor development, progression, metastasis and treatment. Recently, we proposed four physical hallmarks of cancer, with distinct origins and consequences, to characterize abnormalities in the physical tumor microenvironment: (1) elevated compressive–tensile solid stresses, (2) elevated interstitial fluid pressure and the resulting interstitial fluid flow, (3) altered material properties (for example, increased tissue stiffness) and (4) altered physical micro-architecture. As this emerging field of physical oncology is being advanced by tumor biologists, cell and developmental biologists, engineers, physicists and oncologists, there is a critical need for model systems and measurement tools to mechanistically probe these physical hallmarks. Here, after briefly defining these physical hallmarks, we discuss the tools and model systems available for probing each hallmark in vitro, ex vivo, in vivo and in clinical settings. We finally review the unmet needs for mechanistic probing of the physical hallmarks of tumors and discuss the challenges and unanswered questions associated with each hallmark. This Review discusses the current challenges and the available methods to study the four physical hallmarks of cancer.},
  archive      = {J_NMETH},
  author       = {Nia, Hadi T. and Munn, Lance L. and Jain, Rakesh K.},
  doi          = {10.1038/s41592-024-02564-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1800-1818},
  shortjournal = {Nat. Meth.},
  title        = {Probing the physical hallmarks of cancer},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes. <em>NMETH</em>, <em>22</em>(9), 1788-1799. (<a href='https://doi.org/10.1038/s41592-025-02807-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbes within tumors have been recognized and experimentally related to oncogenesis, tumor growth, metastasis and therapeutic responsiveness. Studying the tumor microbiome presents difficulties, as early indications suggest that microbe populations are low in abundance, sparse and highly heterogeneous. Disparate results from computational profiling of the tumor microbiome have cast doubt on the premise of microbes in tumors. Yet decades of experimental evidence support the presence of tumor microbes, at least in a limited number of tumor types. In this Perspective, we discuss the importance of iteratively improving microbe-targeted sequencing techniques, established analytical pipelines, robust computational tools and solid validations to address current challenges and fill existing knowledge gaps. The vast amount of human tumor sequencing data available could greatly enhance systematic investigations of microbiome–tumor interactions with methods to quantify the composition of the tumor microbiome accurately. This Perspective explores the challenges and future directions in the world of human tumor microbiome research.},
  archive      = {J_NMETH},
  author       = {Li, Yingjie and Ma, Anjun and Johnson, Evan and Eng, Charis and De, Subhajyoti and Jiang, Sizun and Li, Zihai and Spakowicz, Daniel and Ma, Qin},
  doi          = {10.1038/s41592-025-02807-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1788-1799},
  shortjournal = {Nat. Meth.},
  title        = {The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research. <em>NMETH</em>, <em>22</em>(9), 1776-1787. (<a href='https://doi.org/10.1038/s41592-025-02769-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vitro tumor models are essential tools for cancer research, offering key insights into not only tumor biology but also therapeutic responses. The transition from traditional two-dimensional to three-dimensional organoid systems marks a paradigm shift in cancer modeling. Although two-dimensional models have been instrumental in elucidating fundamental molecular and genetic mechanisms, they fail to accurately replicate the intricate three-dimensional architecture and dynamic microenvironment characteristic of human tumors. Here we outline how advanced organoid technologies now enable more faithful recapitulation of tumor heterogeneity that better mimic native tissue mechanics and biochemistry. We discuss emerging methods, including air–liquid interface cultures, microfluidic tumor-on-a-chip devices and high-content imaging integrated with machine learning, which collectively address longstanding challenges such as matrix variability and the limited incorporation of immune and vascular elements. These innovations promise to enhance reproducibility and scalability while providing unprecedented insights into tumor biology, cancer progression and therapeutic strategies. This Perspective discusses recent progresses in development of in vitro tumor models and the current challenges in this field.},
  archive      = {J_NMETH},
  author       = {Rauner, Gat and Gupta, Piyush B. and Kuperwasser, Charlotte},
  doi          = {10.1038/s41592-025-02769-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1776-1787},
  shortjournal = {Nat. Meth.},
  title        = {From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible annotation of t cell subsets and activation states with gene expression programs. <em>NMETH</em>, <em>22</em>(9), 1774-1775. (<a href='https://doi.org/10.1038/s41592-025-02792-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created T-CellAnnoTator (TCAT), a computational method that helps to identify T cell subsets, activation states and functions. It does this using reproducible gene expression programs found across many disease contexts and tissues. TCAT outperforms conventional approaches for T cell subset prediction, is easy to use programmatically or through a website, and can be adapted for other cell types.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02792-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1774-1775},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible annotation of t cell subsets and activation states with gene expression programs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A miniature two-photon microscope detects multiple signals deep in the brains of moving mice. <em>NMETH</em>, <em>22</em>(9), 1772-1773. (<a href='https://doi.org/10.1038/s41592-025-02779-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A lightweight miniature two-photon microscope features multi-wavelength excitation, correction of aberrations, and interchangeable objectives for scalable fields of view. It enables multicolor, deep-brain and scalable neural imaging in freely moving mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02779-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1772-1773},
  shortjournal = {Nat. Meth.},
  title        = {A miniature two-photon microscope detects multiple signals deep in the brains of moving mice},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unravelling cellular interactions using flow cytometry. <em>NMETH</em>, <em>22</em>(9), 1770-1771. (<a href='https://doi.org/10.1038/s41592-025-02743-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a cost-effective ultra-high-throughput cytometry-based framework for the detection of physical interactions between cells, along with the characterization of complex cellular landscapes. Application of our approach can offer a systems-level understanding of immunity and facilitate study of the kinetics, mode of action and personalized response prediction of immunotherapies.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02743-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1770-1771},
  shortjournal = {Nat. Meth.},
  title        = {Unravelling cellular interactions using flow cytometry},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles. <em>NMETH</em>, <em>22</em>(9), 1768-1769. (<a href='https://doi.org/10.1038/s41592-025-02775-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present MAPIT-seq, a method that uses antibody-directed RNA editing to concurrently profile in situ RNA-binding protein (RBP)–RNA interactions and transcriptome-wide gene expression in limited input material, including single cells and frozen tissues. This dual-omic strategy streamlines mechanistic analyses of post-transcriptional regulation in dynamic biological processes and clinically relevant samples.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02775-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1768-1769},
  shortjournal = {Nat. Meth.},
  title        = {MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bridging histology and spatial gene expression across scales. <em>NMETH</em>, <em>22</em>(9), 1765-1767. (<a href='https://doi.org/10.1038/s41592-025-02806-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two deep-learning frameworks — GHIST and iSCALE — turn routine histology images into a rich molecular resource, and predict spatial gene expression at single-cell resolution (GHIST) and at super-resolution across large tissue sections (iSCALE), for scalable, data-driven tissue biology.},
  archive      = {J_NMETH},
  author       = {Ma, Ying},
  doi          = {10.1038/s41592-025-02806-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1765-1767},
  shortjournal = {Nat. Meth.},
  title        = {Bridging histology and spatial gene expression across scales},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Taming cancer evolution surprises. <em>NMETH</em>, <em>22</em>(9), 1760-1764. (<a href='https://doi.org/10.1038/s41592-025-02802-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a focus on the evolutionary forces that shape cancer cell behavior, some researchers work out ways to outmaneuver cancer.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02802-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1760-1764},
  shortjournal = {Nat. Meth.},
  title        = {Taming cancer evolution surprises},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SNAP-tag2 improves live-cell imaging. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02830-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02830-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {SNAP-tag2 improves live-cell imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optical pooled screens shed light on ebola infection. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02831-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kamenova, Ivanka},
  doi          = {10.1038/s41592-025-02831-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {Optical pooled screens shed light on ebola infection},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ViSOR puts the periphery in focus. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02828-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Willson, Joseph},
  doi          = {10.1038/s41592-025-02828-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {ViSOR puts the periphery in focus},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Benchmarking genomic language models. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02829-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-025-02829-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking genomic language models},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deciphering regulatory sequence grammar. <em>NMETH</em>, <em>22</em>(9), 1757. (<a href='https://doi.org/10.1038/s41592-025-02827-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leveraging data generated by UUATAC-seq, a method for cross-species chromatin accessibility profiling, the deep learning model NvwaCE deciphers cis-regulatory grammar from DNA sequence.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-025-02827-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1757},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering regulatory sequence grammar},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cichlid fishes. <em>NMETH</em>, <em>22</em>(9), 1755-1756. (<a href='https://doi.org/10.1038/s41592-025-02809-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cichlid fishes are a family of thousands of recently evolved species. As charismatic laboratory models, they are useful for studying anatomical, physiological and behavioral traits that vary across these closely related species.},
  archive      = {J_NMETH},
  author       = {Juntti, Scott A.},
  doi          = {10.1038/s41592-025-02809-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1755-1756},
  shortjournal = {Nat. Meth.},
  title        = {Cichlid fishes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Road trip home to start a lab. <em>NMETH</em>, <em>22</em>(9), 1754. (<a href='https://doi.org/10.1038/s41592-025-02801-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As a postdoctoral fellowship ends, for some, next is a trek to their new lab.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02801-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1754},
  shortjournal = {Nat. Meth.},
  title        = {Road trip home to start a lab},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A tale of tumors. <em>NMETH</em>, <em>22</em>(9), 1753. (<a href='https://doi.org/10.1038/s41592-025-02838-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advances for studying the biology of tumors have expanded our understanding of cancer.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02838-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1753},
  shortjournal = {Nat. Meth.},
  title        = {A tale of tumors},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nphys">NPHYS - 29</h2>
<ul>
<li><details>
<summary>
(2025). Precision without parity. <em>NPHYS</em>, <em>21</em>(9), 1498. (<a href='https://doi.org/10.1038/s41567-025-03018-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metrology has long been defined by its pursuit of reproducibility, accuracy and traceability. Yet, when it comes to the representation of women and other marginalized communities, precision has not been matched by parity, finds Solmaz Nadiri.},
  archive      = {J_NPHYS},
  author       = {Nadiri, Solmaz},
  doi          = {10.1038/s41567-025-03018-0},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1498},
  shortjournal = {Nat. Phys.},
  title        = {Precision without parity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: High-efficiency optical training of itinerant two-dimensional magnets. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03013-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Xie, Ti and Liang, Jierui and Bhattacharya, Dhritiman and Arachchige, Hasitha Suriya and Yakovenko, Victor M. and Mandrus, David G. and Qiu, Zi Qiang and Liu, Kai and Gong, Cheng},
  doi          = {10.1038/s41567-025-03013-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Author correction: High-efficiency optical training of itinerant two-dimensional magnets},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03054-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-03054-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient implementation of arbitrary two-qubit gates using unified control. <em>NPHYS</em>, <em>21</em>(9), 1489-1496. (<a href='https://doi.org/10.1038/s41567-025-02990-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The set of quantum logic gates that can be easily implemented is fundamental to the performance of quantum computers, as it governs the accuracy of basic quantum operations and dictates the complexity of implementing quantum algorithms. Traditional approaches to extending gate sets often require operating devices outside the ideal parameter regimes used to realize qubits, leading to increased control complexity while offering only a limited set of gates. Here we experimentally demonstrate a unified and versatile gate scheme capable of generating arbitrary two-qubit gates using only an exchange interaction and qubit driving on a superconducting quantum processor. We achieve high fidelities averaging 99.38% across a wide range of commonly used two-qubit unitaries, enabling precise multipartite entangled state preparation. Furthermore, we successfully produce a B gate, which efficiently synthesizes the entire family of two-qubit gates. Our results establish that fully exploiting the capabilities of the exchange interaction can yield a comprehensive and highly accurate gate set. With maximum expressivity, optimal gate time, demonstrated high fidelity and easy adaption to other quantum platforms, our unified control scheme offers the prospect of improved performance in quantum hardware and algorithm development. The efficiency of a quantum computer depends on which basic operations it can implement. Now a scheme that can implement any two-qubit logic gate has been demonstrated on a superconducting architecture.},
  archive      = {J_NPHYS},
  author       = {Chen, Zhen and Liu, Weiyang and Ma, Yanjun and Sun, Weijie and Wang, Ruixia and Wang, He and Xu, Huikai and Xue, Guangming and Yan, Haisheng and Yang, Zhen and Ding, Jiayu and Gao, Yang and Li, Feiyu and Zhang, Yujia and Zhang, Zikang and Jin, Yirong and Yu, Haifeng and Chen, Jianxin and Yan, Fei},
  doi          = {10.1038/s41567-025-02990-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1489-1496},
  shortjournal = {Nat. Phys.},
  title        = {Efficient implementation of arbitrary two-qubit gates using unified control},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1482-1488. (<a href='https://doi.org/10.1038/s41567-025-02989-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-fidelity quantum optomechanical control of a mechanical oscillator requires the ability to perform efficient, low-noise operations on long-lived phononic excitations. Microfabricated high-overtone bulk acoustic wave resonators (μHBARs) support high-frequency mechanical modes above 10 GHz with coherence times exceeding one millisecond. Here we demonstrate a μHBAR-based cavity optomechanical system that permits quantum optomechanical control of individual high-coherence phonon modes. We perform laser cooling of the phonon modes from an occupation of approximately 22 phonons to fewer than 0.4, corresponding to laser-based ground-state cooling of a mechanical object with a mass of 7.5 μg. During the cooling process we do not observe any absorption-induced heating, demonstrating the resilience of the HBAR optomechanical systems against parasitic heating. Our work demonstrates that μHBARs are promising as the basis for quantum optomechanical systems with robustness to decoherence that is necessary for efficient, low-noise photon–phonon conversion. A massive phonon mode in a high-overtone bulk acoustic wave resonator has been laser cooled close to its ground state. Its robustness to decoherence establishes the potential of these devices for quantum technologies.},
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-02989-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1482-1488},
  shortjournal = {Nat. Phys.},
  title        = {Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fast optical control of a coherent hole spin in a microcavity. <em>NPHYS</em>, <em>21</em>(9), 1475-1481. (<a href='https://doi.org/10.1038/s41567-025-02988-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many of the most promising quantum information platforms store quantum information in electronic or atomic spins. To incorporate these devices into quantum networks, a spin–photon interface is required. Currently, the best on-demand single-photon sources use a semiconductor quantum dot in an engineered photonic environment. However, it is difficult to achieve coherent spin control in a high-performance single-photon source, and spin coherence is limited by magnetic noise from nuclear spins in the semiconductor host material. Here we combine all-optical spin control with a quantum dot in an open microcavity. We demonstrate fast coherent rotations of a hole spin around an arbitrary axis of the Bloch sphere with a maximum π-pulse fidelity of 98.6%. To suppress the slow magnetic noise, we laser cool the nuclear spins using the hole as a central spin. This extends the hole spin free-induction-decay time by more than an order of magnitude. It becomes much larger than both rotation time and radiative recombination time of the spin, enabling the creation of many spin–photon pairs before the loss of spin coherence. Spin–photon interfaces provide a connection between quantum information stored in atomic or electronic spins and optical communications networks. A quantum photon emitter with long-lived, controllable coherent spin has now been demonstrated.},
  archive      = {J_NPHYS},
  author       = {Hogg, Mark R. and Antoniadis, Nadia O. and Marczak, Malwina A. and Nguyen, Giang N. and Baltisberger, Timon L. and Javadi, Alisa and Schott, Rüdiger and Valentin, Sascha R. and Wieck, Andreas D. and Ludwig, Arne and Warburton, Richard J.},
  doi          = {10.1038/s41567-025-02988-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1475-1481},
  shortjournal = {Nat. Phys.},
  title        = {Fast optical control of a coherent hole spin in a microcavity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A mechanical quantum memory for microwave photons. <em>NPHYS</em>, <em>21</em>(9), 1469-1474. (<a href='https://doi.org/10.1038/s41567-025-02975-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Superconducting qubits possess outstanding capabilities for processing quantum information in the microwave domain; however they have limited coherence times. An interface between photons and phonons could allow quantum information to be stored in long-lived mechanical oscillators. Here, we introduce a platform that relies on electrostatic forces in nanoscale structures to achieve strong coupling between a superconducting qubit and a nanomechanical oscillator with an energy decay time (T1) of approximately 25 ms, well beyond those achieved in integrated superconducting circuits. We use quantum operations in this system to investigate the microscopic origins of mechanical decoherence and mitigate its impact. By using two-pulse dynamical decoupling sequences, we can extend the coherence time (T2) from 64 μs to 1 ms. These findings establish that mechanical oscillators can act as quantum memories for superconducting devices, with potential future applications in quantum computing, sensing and transduction. Superconducting qubits, a leading platform for quantum information processing, suffer from decoherence. Interfacing them with nanomechanical oscillators allows quantum information to be stored in motional states with longer lifetimes.},
  archive      = {J_NPHYS},
  author       = {Bozkurt, Alkım B. and Golami, Omid and Yu, Yue and Tian, Hao and Mirhosseini, Mohammad},
  doi          = {10.1038/s41567-025-02975-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1469-1474},
  shortjournal = {Nat. Phys.},
  title        = {A mechanical quantum memory for microwave photons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics. <em>NPHYS</em>, <em>21</em>(9), 1462-1468. (<a href='https://doi.org/10.1038/s41567-025-02973-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Skyrmions are topologically stable quasiparticles that have been investigated in contexts including particle physics, quantum field theory, acoustics and condensed-matter physics. Quantum optical skyrmions with local topological textures are expected to reshape the landscape of quantum photonic technology, although their experimental implementation has not yet been demonstrated. Here we present experimental realizations of nanophotonic quantum skyrmions using a semiconductor cavity quantum electrodynamics system. By manipulating the photonic spin–orbit coupling in a Gaussian microcavity, we obtained a confined optical mode whose polarizations feature skyrmionic topologies. With pronounced cavity quantum electrodynamics effects, we generated and detected single-photon skyrmions from a solid-state quantum emitter deterministically coupled to the Gaussian microcavity. The polarity associated with single-photon skyrmions can be swapped by flipping the polarization of the quantum emitter through the Zeeman effect. We also investigated the topological protection of quantum optical skyrmions under different perturbations. Our work opens an unexplored aspect of quantum light–matter interactions in the nanoscale and might advance resilient photonic quantum technology with high-dimensional qubits and high-capacity quantum memories. Quantum optical skyrmions are promising for quantum photonic applications but have not been experimentally realized. Now nanophotonic quantum skyrmions are generated using a semiconductor quantum dot–Gaussian microcavity quantum electrodynamics system.},
  archive      = {J_NPHYS},
  author       = {Ma, Jiantao and Yang, Jiawei and Liu, Shunfa and Chen, Bo and Li, Xueshi and Song, Changkun and Qiu, Guixin and Zou, Kai and Hu, Xiaolong and Li, Feng and Yu, Ying and Liu, Jin},
  doi          = {10.1038/s41567-025-02973-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1462-1468},
  shortjournal = {Nat. Phys.},
  title        = {Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The actin cortex acts as a mechanical memory of morphology in confined migrating cells. <em>NPHYS</em>, <em>21</em>(9), 1451-1461. (<a href='https://doi.org/10.1038/s41567-025-02980-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell migration in narrow microenvironments occurs in numerous physiological processes. It involves successive cycles of confinement and release that drive important morphological changes. However, it remains unclear whether migrating cells can retain a memory of their past morphological states that could potentially facilitate their navigation through confined spaces. We demonstrate that local geometry governs a switch between two cell morphologies, thereby facilitating cell passage through long and narrow gaps. We combined cell migration assays on standardized microsystems with biophysical modelling and biochemical perturbations to show that migrating cells have a long-term memory of past confinement events. The morphological cell states correlate across transitions through actin cortex remodelling. These findings indicate that mechanical memory in migrating cells plays an active role in their migratory potential in confined environments. Cells often navigate through confined spaces. Now it is shown that cells retain a mechanical memory of previous confinement events, which makes them more efficient at migrating through narrow microenvironments.},
  archive      = {J_NPHYS},
  author       = {Kalukula, Yohalie and Luciano, Marine and Simanov, Gleb and Charras, Guillaume and Brückner, David B. and Gabriele, Sylvain},
  doi          = {10.1038/s41567-025-02980-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1451-1461},
  shortjournal = {Nat. Phys.},
  title        = {The actin cortex acts as a mechanical memory of morphology in confined migrating cells},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capillary interactions drive the self-organization of bacterial colonies. <em>NPHYS</em>, <em>21</em>(9), 1444-1450. (<a href='https://doi.org/10.1038/s41567-025-02965-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many bacteria inhabit thin water layers on solid surfaces. These thin films occur both naturally—in soils, on hosts and on textiles—and in the laboratory on agar hydrogels. In these environments, cells experience capillary forces, but it is unclear how these forces shape bacterial collective behaviour. Here we show that the water menisci formed around bacteria lead to capillary attraction between cells while still allowing them to slide past one another. We develop an experimental apparatus that allows us to control bacterial collective behaviour by varying the strength and range of capillary forces. Combining three-dimensional imaging and cell tracking with agent-based modelling, we demonstrate that capillary attraction organizes rod-shaped bacteria into densely packed nematic groups and influences their collective dynamics and morphologies. Our results suggest that capillary forces may be a ubiquitous physical ingredient in shaping microbial communities in partially hydrated environments. Bacteria tend to live in thin layers of water on surfaces. Now the capillary forces in these layers are shown to help organize the bacteria into dense packs.},
  archive      = {J_NPHYS},
  author       = {Black, Matthew E. and Fei, Chenyi and Alert, Ricard and Wingreen, Ned S. and Shaevitz, Joshua W.},
  doi          = {10.1038/s41567-025-02965-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1444-1450},
  shortjournal = {Nat. Phys.},
  title        = {Capillary interactions drive the self-organization of bacterial colonies},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion. <em>NPHYS</em>, <em>21</em>(9), 1431-1443. (<a href='https://doi.org/10.1038/s41567-025-02964-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells can sense and respond to their environment. Central to this process is the formation of molecular clutches, which are dynamic linkages between the extracellular matrix and the actin cytoskeleton mediated by integrins and adaptor proteins. Although it is well known that force-dependent interactions between molecular-clutch components are essential for sensing substrate rigidity, the influence of nonlinear adaptor-protein elasticity is poorly understood. Here we show that adaptor-protein elasticity and local interactions between molecular clutches and the extracellular matrix are key to cellular perception of substrate stiffness. We present a semi-analytical theory that integrates experimentally measured force responses of adaptor proteins to describe cell-adhesion sensing. We propose that molecular clutches probe local mechanical substrate properties and collectively function as a differential that allows a retrograde actin flow and substrate movement to occur at different rates while maintaining a stable mechanical coupling between them. Our model reproduces experimentally measured force-loading rates of individual molecular clutches and correctly predicts cell-adhesion behaviour for a range of substrate stiffnesses. The framework presented can be extended to study complex phenomena such as focal-adhesion maturation and cell-type-specific mechanosensing. The cytoskeleton and extracellular matrix are linked by molecular clutches that enable cells to sense their environment. Now it is shown that the elastic properties of the clutches underpin stiffness sensing.},
  archive      = {J_NPHYS},
  author       = {Liu, Ping and Wang, Qiuyu and Dai, Xin and Pei, Lingxuan and Wang, Junfan and Zhao, Weiyi and Johnson, Heath E. and Yao, Mingxi and Efremov, Artem K.},
  doi          = {10.1038/s41567-025-02964-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1431-1443},
  shortjournal = {Nat. Phys.},
  title        = {Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility. <em>NPHYS</em>, <em>21</em>(9), 1420-1430. (<a href='https://doi.org/10.1038/s41567-025-02983-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microtubules are hollow cylindrical cytoskeletal polymers of laterally associated protofilaments that contain head-to-tail aligned ɑ/β-tubulin heterodimers. Although the exposed microtubule exterior is readily accessible to proteins, the mechanism governing the accessibility of the confined microtubule lumen to luminal particles remains unknown. Here we show that certain tubulin family proteins (isotypes) facilitate luminal accessibility because of the mechanical properties and lateral interactions that they confer to the microtubules. We characterized the microtubules reconstituted from defined compositions of Caenorhabditis elegans tubulin isotypes. These tubulin isotypes form microtubules with comparable protofilament numbers but different luminal accessibility. We further revealed the role of tubulin isotypes in regulating the strength of inter-protofilament lateral interactions, which determines luminal accessibility through the mechanosensitivity of reversible protofilament separation. Deformation of the microtubule lattice, which generates stresses exceeding the strength of the lateral interactions, creates gaps between adjacent protofilaments, enhancing the accessibility of the lumen. Together, our findings uncovered the tubulin isotype-dependent mechanical plasticity that confers force sensitivity to the microtubule lattice and modulates the energy barrier for luminal proteins to access the lumen. Proteins interact with both the exterior and interior of microtubules. Here the relationship between microtubule building blocks and the accessibility of the microtubule interior to proteins is clarified.},
  archive      = {J_NPHYS},
  author       = {Ye, Yucheng and Hao, Zheng and Luo, Jingyi and Lam, Wai Hei and Liu, Zheng and Li, Xiang David and Zhai, Yuanliang and Lin, Yuan and Ti, Shih-Chieh},
  doi          = {10.1038/s41567-025-02983-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1420-1430},
  shortjournal = {Nat. Phys.},
  title        = {Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Control of collective activity to crystallize an oscillator gas. <em>NPHYS</em>, <em>21</em>(9), 1412-1419. (<a href='https://doi.org/10.1038/s41567-025-02957-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motility-induced phase separation occurs in assemblies of self-propelled units when activity is coupled negatively to density. By contrast, the consequences of a positive coupling between density and activity on the collective behaviour of active matter remain unexplored. Here we show that collective activity can emerge from such a positive coupling among non-motile building blocks. We perform experiments with self-sustained oscillators powered by contact-charge electrophoresis. Although the oscillators are non-motile by design, they spontaneously form an active gas when confined together. The super-elastic nature of collisions constitutes a positive density–activity coupling and underlies the active gas properties. Elucidating the origin of binary collisions allows us to precisely control the structure of the active gas and its eventual crystallization. Beyond considering the overlooked positive coupling between density and activity, our work suggests that rich collective properties can emerge not only from the symmetry of interactions between active building blocks but also from their adaptable and responsive behaviour. In active matter systems, increasing density usually reduces activity. Now, in a system where density enhances activity, collective motion is shown to arise from non-motile oscillators when they are confined.},
  archive      = {J_NPHYS},
  author       = {Le Blay, Marine and Saldi, Joshua H. K. and Morin, Alexandre},
  doi          = {10.1038/s41567-025-02957-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1412-1419},
  shortjournal = {Nat. Phys.},
  title        = {Control of collective activity to crystallize an oscillator gas},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multistable polar textures in geometrically frustrated nematic liquid crystals. <em>NPHYS</em>, <em>21</em>(9), 1404-1411. (<a href='https://doi.org/10.1038/s41567-025-02966-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to manipulate polar entities with multiple external fields could enable functionalities and applications in spin systems, photonics, metamaterials and soft matter. Liquid crystals that exhibit both a crystalline structure and liquid fluidity represent a promising platform for manipulating phases with polar molecular order, notably ferroelectric ones. However, achieving a polar symmetry is challenging with rod-shaped liquid crystal molecules, which form predominantly apolar nematic phases. Here we report an approach in which a geometric lattice confinement of nematic liquid crystals is used to induce planar polar order on the scale of a mesoscopic metamaterial. We confine the nematic liquid crystal in a micropillar array, forming topological defect–pillar pairs of elastic dipoles with a free top interface in contact with an immiscible fluid. The resulting dipole lattice configurations can be programmed rheologically by flowing the top fluid and maintained even after flow cessation, a phenomenon attributed to orientational multistability of the dipoles. This multimemory effect enables the encoding and reconfiguration of directional information. Overall, these results advance our understanding of topological dipoles under confinement and shear flow, enabling the detection, tracking and recording of flow profiles and could facilitate the development of stimuli-responsive materials. Programmable topological dipoles could be used in stimuli-responsive materials. Now it is shown that confinement of an apolar liquid crystal results in controllable polar arrangements of elastic dipoles.},
  archive      = {J_NPHYS},
  author       = {Kara, Ufuoma I. and Chen, Boyuan and Čopar, Simon and Li, Shucong and Mamtani, Rajdeep and Xu, Yang and Weible, Alan H. and Boerner, Eric C. and Yang, Zhan and Yao, Yuxing and Selinger, Robin L. B. and Tkalec, Uroš and Wang, Xiaoguang},
  doi          = {10.1038/s41567-025-02966-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1404-1411},
  shortjournal = {Nat. Phys.},
  title        = {Multistable polar textures in geometrically frustrated nematic liquid crystals},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability. <em>NPHYS</em>, <em>21</em>(9), 1398-1403. (<a href='https://doi.org/10.1038/s41567-025-02982-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Topological defects give rise to non-trivial excitations—such as solitons, vortices and skyrmions—characterized by integer-valued topological charges. However, their classification becomes difficult when singularities are present in the order parameter field, complicating the computation of these charges. Although exotic nonlinear excitations have been theoretically proposed in the superfluid 3He-A phase and spinor Bose–Einstein condensates, they have not been experimentally observed and their stability has not been investigated. Here we demonstrate the presence of a singular skyrmion that goes beyond the framework of topology in a ferromagnetic superfluid. These skyrmions emerge from anomalous symmetry-breaking associated with an eccentric spin singularity and carry half the elementary charge—a feature that distinguishes them from conventional skyrmions and merons. We realize the universal regime of the quantum Kelvin–Helmholtz instability, and we identify the unconventional fractional skyrmions produced by emission from a magnetic domain wall and the spontaneous splitting of an integer skyrmion with spin singularities. The singular skyrmions are stable even after 2 s of hold time. Our results confirm the universality between classical and quantum Kelvin–Helmholtz instabilities and broaden our understanding of complex nonlinear dynamics for a non-trivial texture in topological quantum systems. The quantum Kelvin–Helmholtz instability can induce non-trivial topological excitations. Now a proof-of-principle experiment demonstrates the emergence of a stable singular fractional skyrmion spin texture originating from this quantum instability.},
  archive      = {J_NPHYS},
  author       = {Huh, SeungJung and Yun, Wooyoung and Yun, Gabin and Hwang, Samgyu and Kwon, Kiryang and Hur, Junhyeok and Lee, Seungho and Takeuchi, Hiromitsu and Kim, Se Kwon and Choi, Jae-yoon},
  doi          = {10.1038/s41567-025-02982-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1398-1403},
  shortjournal = {Nat. Phys.},
  title        = {Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Anyonic braiding in a chiral Mach–Zehnder interferometer. <em>NPHYS</em>, <em>21</em>(9), 1392-1397. (<a href='https://doi.org/10.1038/s41567-025-02960-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fractional quantum statistics are the defining characteristic of anyons. Measuring the phase generated by an exchange of anyons is challenging, as standard interferometry set-ups—such as the Fabry–Pérot interferometer—suffer from charging effects that obscure the interference signal. Here we present the observation of anyonic interference and exchange phases in an optical-like Mach–Zehnder interferometer based on co-propagating interface modes. By avoiding backscattering and deleterious charging effects, this set-up enables pristine and robust Aharonov–Bohm interference without any phase slips. At various fractional filling factors, the observed flux periodicities agree with the fundamental fractionally charged excitations that correspond to Jain states and depend only on the bulk topological order. To probe the anyonic statistics, we used a small, charged top gate in the interferometer bulk to induce localized quasiparticles without modifying the Aharonov–Bohm phase. The added quasiparticles introduce periodic phase slips. The sign and magnitude of the observed phase slips align with the expected value at filling 1/3, but their direction shows systematic deviations at fillings 2/5 and 3/7. Control over added individual quasiparticles in this design is essential for measuring the coveted non-abelian statistics in the future. An interferometer design allows systematic investigation of the anyonic statistics of bulk fractional quantum Hall states.},
  archive      = {J_NPHYS},
  author       = {Ghosh, Bikash and Labendik, Maria and Musina, Liliia and Umansky, Vladimir and Heiblum, Moty and Mross, David F.},
  doi          = {10.1038/s41567-025-02960-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1392-1397},
  shortjournal = {Nat. Phys.},
  title        = {Anyonic braiding in a chiral Mach–Zehnder interferometer},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Measurement of phonon angular momentum. <em>NPHYS</em>, <em>21</em>(9), 1387-1391. (<a href='https://doi.org/10.1038/s41567-025-02952-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In condensed matter, angular momentum is intimately related to the emergence of topological quantum states, including chiral superconductivity, quantum spin liquids and various chiral quasiparticles. Recently, it has been predicted that microscopic lattice excitations, known as phonons, can carry finite angular momentum, leading to specific physical properties of materials. However, phonon angular momentum has not yet been observed directly. Here we demonstrate that angular momentum conservation results in a macroscopic mechanical torque when applying a time-reversal symmetry-breaking thermal gradient along the chiral axis of single-crystal tellurium. We probe this torque using a cantilever-based device and establish that it changes sign by flipping the chirality or thermal gradient. This behavior disappears in polycrystalline samples that lack a preferred chirality. Our experimental results align well with theoretical calculations. We provide compelling evidence for phonon angular momentum, which might enable quantum states with potential applications in microelectronics. Theory predicts that phonons—quanta of lattice vibrations—can carry finite angular momentum and thus influence physical properties of materials. Now phonons with angular momentum have been seen in tellurium with a chiral crystal structure.},
  archive      = {J_NPHYS},
  author       = {Zhang, H. and Peshcherenko, N. and Yang, Fazhi and Ward, T. Z. and Raghuvanshi, P. and Lindsay, L. and Felser, Claudia and Zhang, Yang and Yan, J.-Q. and Miao, H.},
  doi          = {10.1038/s41567-025-02952-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1387-1391},
  shortjournal = {Nat. Phys.},
  title        = {Measurement of phonon angular momentum},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Strongly interacting hofstadter states in magic-angle twisted bilayer graphene. <em>NPHYS</em>, <em>21</em>(9), 1380-1386. (<a href='https://doi.org/10.1038/s41567-025-02997-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Magic-angle twisted bilayer graphene hosts a variety of strongly correlated states at partial fillings of its flat bands. In a magnetic field, these flat bands evolve into a Hofstadter spectrum renormalized by strong Coulomb interactions. Here we study the interacting Hofstadter states that spontaneously form within the topological magnetic sub-bands of an ultraclean magic-angle twisted bilayer graphene device, including symmetry-broken Chern insulator states and fractional quantum Hall states. The observed symmetry-broken Chern insulator states form a cascade, with their Chern numbers mimicking the main sequence of correlated Chern insulators. The fractional quantum Hall states form in a Jain sequence. However, they disappear at high magnetic field, in contrast to conventional fractional quantum Hall states that strengthen with increasing magnetic field. We reveal a magnetic-field-driven phase transition from composite fermion phases to a dissipative Fermi liquid. Our theoretical analysis of the magnetic sub-bands hosting the fractional quantum Hall states predicts non-uniform quantum geometric properties far from the lowest Landau level. This points towards a more natural interpretation of these states as in-field fractional Chern insulators of the magnetic sub-bands. Interacting electrons in Hofstadter bands can form symmetry-broken topological states. These are now revealed in magic-angle twisted bilayer graphene, and their properties are influenced by non-uniform quantum geometry.},
  archive      = {J_NPHYS},
  author       = {He, Minhao and Wang, Xiaoyu and Cai, Jiaqi and Herzog-Arbeitman, Jonah and Peng, Ran and Taniguchi, Takashi and Watanabe, Kenji and Stern, Ady and Bernevig, B. Andrei and Yankowitz, Matthew and Vafek, Oskar and Xu, Xiaodong},
  doi          = {10.1038/s41567-025-02997-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1380-1386},
  shortjournal = {Nat. Phys.},
  title        = {Strongly interacting hofstadter states in magic-angle twisted bilayer graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Universality in quantum critical flow of charge and heat in ultraclean graphene. <em>NPHYS</em>, <em>21</em>(9), 1374-1379. (<a href='https://doi.org/10.1038/s41567-025-02972-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Close to the Dirac point, graphene is expected to exist in a quantum critical Dirac fluid state, where the flow of both charge and heat can be described with a characteristic d.c. electrical conductivity and thermodynamic variables such as entropy and enthalpy densities. Although the fluid-like viscous flow of charge has been reported in state-of-the-art graphene devices, the value of conductivity, predicted to be quantized and determined only by the universality class of the critical point, has not been established experimentally so far. Here we have discerned the quantum critical universality in graphene transport by combining the electrical and thermal conductivities in very high-quality devices close to the Dirac point. We find that they are inversely related, as expected from relativistic hydrodynamics, and the characteristic conductivity converges to a quantized value. We also observe a giant violation of the Wiedemann–Franz law, where the Lorentz number exceeds the semiclassical value by more than 200 times close to the Dirac point at low temperatures. At high temperatures, the effective dynamic viscosity to entropy density ratio close to the Dirac point in the cleanest devices approaches that of a minimally viscous quantum fluid within a factor of four. Critical behaviour is expected in graphene when the carrier density is tuned to the Dirac point. Now, universality associated with the critical point is observed in electronic and thermal transport.},
  archive      = {J_NPHYS},
  author       = {Majumdar, Aniket and Chadha, Nisarg and Pal, Pritam and Gugnani, Akash and Ghawri, Bhaskar and Watanabe, Kenji and Taniguchi, Takashi and Mukerjee, Subroto and Ghosh, Arindam},
  doi          = {10.1038/s41567-025-02972-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1374-1379},
  shortjournal = {Nat. Phys.},
  title        = {Universality in quantum critical flow of charge and heat in ultraclean graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerator technologies for proton and ion beam therapy. <em>NPHYS</em>, <em>21</em>(9), 1363-1373. (<a href='https://doi.org/10.1038/s41567-025-02994-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past 75 years, proton beam therapy has emerged as a promising modality for cancer treatment, boasting precise targeting and reduced collateral damage to healthy tissue. Here we discuss the evolution of accelerator technology in proton therapy, examining advancements in cyclotron, synchrotron and linear accelerator technology, and their implications for modern treatment delivery. Additionally, we explore advances in delivering accelerated carbon or helium ions for therapeutic treatments. We also discuss the integration of advanced imaging modalities, such as multienergy X-ray, magnetic resonance imaging and ion-based imaging, for real-time monitoring and adaptive radiotherapy. These advancements position particle therapy to offer personalized and effective cancer treatment strategies, heralding improved patient outcomes. Proton and heavy-ion beam therapies are promising approaches for cancer treatment. This Review discusses the accelerator technologies that enable these therapies.},
  archive      = {J_NPHYS},
  author       = {Maradia, Vivek and Clasie, Benjamin and Snively, Emma and Parodi, Katia and Schwarz, Marco and Durante, Marco},
  doi          = {10.1038/s41567-025-02994-7},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1363-1373},
  shortjournal = {Nat. Phys.},
  title        = {Accelerator technologies for proton and ion beam therapy},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Extended native gate sets to unlock the performance of quantum processors. <em>NPHYS</em>, <em>21</em>(9), 1361-1362. (<a href='https://doi.org/10.1038/s41567-025-02991-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum operations are modelled as unitary matrices, yet experimental implementations have been restricted to only a few gate types. This fundamental limitation has now been overcome using a scheme that combines frequency control with microwave driving, enabling universal implementation of arbitrary two-qubit operations.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02991-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1361-1362},
  shortjournal = {Nat. Phys.},
  title        = {Extended native gate sets to unlock the performance of quantum processors},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How squeezed cells remember their shape to migrate efficiently. <em>NPHYS</em>, <em>21</em>(9), 1359-1360. (<a href='https://doi.org/10.1038/s41567-025-02984-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells migrating through narrow spaces in their environment undergo repeated shape changes to pass through tight constrictions. Epithelial cells retain a memory of past confinement, allowing them to maintain a polarized, compact morphology that enhances future migration through narrow gaps. This memory is mechanically encoded in the actin cortex.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02984-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1359-1360},
  shortjournal = {Nat. Phys.},
  title        = {How squeezed cells remember their shape to migrate efficiently},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantized limit of conductivity in near-ideal graphene. <em>NPHYS</em>, <em>21</em>(9), 1357-1358. (<a href='https://doi.org/10.1038/s41567-025-02974-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transport properties near the Dirac point in graphene are expected to be determined by quantum many-body interactions between relativistic electrons. Experiments now show that the flow of charge and heat in high-quality graphene close to charge neutrality can be described within a hydrodynamic framework, with universal intrinsic electrical conductivity that is quantized to a value close to the quantum of conductance.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02974-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1357-1358},
  shortjournal = {Nat. Phys.},
  title        = {Quantized limit of conductivity in near-ideal graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boron at the boundaries. <em>NPHYS</em>, <em>21</em>(9), 1356. (<a href='https://doi.org/10.1038/s41567-025-03041-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Brierley, Richard},
  doi          = {10.1038/s41567-025-03041-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1356},
  shortjournal = {Nat. Phys.},
  title        = {Boron at the boundaries},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Liquid dipole lattice. <em>NPHYS</em>, <em>21</em>(9), 1354-1355. (<a href='https://doi.org/10.1038/s41567-025-02992-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geometrically confined apolar nematic liquid crystals lose rotational symmetry and form a multifunctional polar lattice of fluid elastic dipoles with controllable orientation. The lattice is of interest from a fundamental science perspective as well as having potential applications.},
  archive      = {J_NPHYS},
  author       = {Lucchetti, Liana},
  doi          = {10.1038/s41567-025-02992-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1354-1355},
  shortjournal = {Nat. Phys.},
  title        = {Liquid dipole lattice},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How to twist anyons. <em>NPHYS</em>, <em>21</em>(9), 1352-1353. (<a href='https://doi.org/10.1038/s41567-025-03009-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The properties of quantum Hall states can be probed in experiments inspired by optical interferometers. Now, a device design that overcomes major drawbacks of existing interferometers allows the extraction of the quasiparticle statistics.},
  archive      = {J_NPHYS},
  author       = {Carrega, Matteo and Heun, Stefan},
  doi          = {10.1038/s41567-025-03009-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1352-1353},
  shortjournal = {Nat. Phys.},
  title        = {How to twist anyons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In plain sight. <em>NPHYS</em>, <em>21</em>(9), 1351. (<a href='https://doi.org/10.1038/s41567-025-03021-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Buchanan, Mark},
  doi          = {10.1038/s41567-025-03021-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1351},
  shortjournal = {Nat. Phys.},
  title        = {In plain sight},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Invest in science education research to make science open to all. <em>NPHYS</em>, <em>21</em>(9), 1350. (<a href='https://doi.org/10.1038/s41567-025-03019-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attacks on Diversity, Equity and Inclusion efforts expose the need for science education research.},
  archive      = {J_NPHYS},
  author       = {Holmes, N. G.},
  doi          = {10.1038/s41567-025-03019-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1350},
  shortjournal = {Nat. Phys.},
  title        = {Invest in science education research to make science open to all},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Serendipitous signal. <em>NPHYS</em>, <em>21</em>(9), 1349. (<a href='https://doi.org/10.1038/s41567-025-03052-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first direct observation of gravitational waves occurred on 14 September 2015. We look back at ten years of gravitational wave science.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-03052-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1349},
  shortjournal = {Nat. Phys.},
  title        = {Serendipitous signal},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
