<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>nature</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs">NATCS - 20</h2>
<ul>
<li><details>
<summary>
(2025). Confidential computing for population-scale genome-wide association studies with SECRET-GWAS. <em>NATCS</em>, <em>5</em>(9), 825-835. (<a href='https://doi.org/10.1038/s43588-025-00856-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic data from a single institution lacks global diversity representation, especially for rare variants and diseases. Confidential computing can enable collaborative genome-wide association studies (GWAS) without compromising privacy or accuracy. However, due to limited secure memory space and performance overheads, previous solutions fail to support widely used regression methods. Here we present SECRET-GWAS—a rapid, privacy-preserving, population-scale, collaborative GWAS tool. We discuss several system optimizations, including streaming, batching, data parallelization and reducing trusted hardware overheads to efficiently scale linear and logistic regression to over a thousand processor cores on an Intel SGX-based cloud platform. In addition, we protect SECRET-GWAS against several hardware side-channel attacks. SECRET-GWAS is an open-source tool and works with the widely used Hail genomic analysis framework. Our experiments on Azure’s Confidential Computing platform demonstrate that SECRET-GWAS enables multivariate linear and logistic regression GWAS queries on population-scale datasets from ten independent sources in just 4.5 and 29 minutes, respectively. Secure collaborative genome-wide association studies (GWAS) with population-scale datasets address gaps in genomic data. This work proposes SECRET-GWAS and system optimizations that overcome resource constraints and exploit parallelism, while maintaining privacy and accuracy.},
  archive      = {J_NATCS},
  author       = {Rosenblum, Jonah and Dong, Juechu and Narayanasamy, Satish},
  doi          = {10.1038/s43588-025-00856-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {825-835},
  shortjournal = {Nat. Comput. Sci},
  title        = {Confidential computing for population-scale genome-wide association studies with SECRET-GWAS},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analog in-memory computing attention mechanism for fast and energy-efficient large language models. <em>NATCS</em>, <em>5</em>(9), 813-824. (<a href='https://doi.org/10.1038/s43588-025-00854-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transformer networks, driven by self-attention, are central to large language models. In generative transformers, self-attention uses cache memory to store token projections, avoiding recomputation at each time step. However, graphics processing unit (GPU)-stored projections must be loaded into static random-access memory for each new generation step, causing latency and energy bottlenecks. Here we present a custom self-attention in-memory computing architecture based on emerging charge-based memories called gain cells, which can be efficiently written to store new tokens during sequence generation and enable parallel analog dot-product computation required for self-attention. However, the analog gain-cell circuits introduce non-idealities and constraints preventing the direct mapping of pre-trained models. To circumvent this problem, we design an initialization algorithm achieving text-processing performance comparable to GPT-2 without training from scratch. Our architecture reduces attention latency and energy consumption by up to two and four orders of magnitude, respectively, compared with GPUs, marking a substantial step toward ultrafast, low-power generative transformers. Leveraging in-memory computing with emerging gain-cell devices, the authors accelerate attention—a core mechanism in large language models. They train a 1.5-billion-parameter model, achieving up to a 70,000-fold reduction in energy consumption and a 100-fold speed-up compared with GPUs.},
  archive      = {J_NATCS},
  author       = {Leroux, Nathan and Manea, Paul-Philipp and Sudarshan, Chirag and Finkbeiner, Jan and Siegel, Sebastian and Strachan, John Paul and Neftci, Emre},
  doi          = {10.1038/s43588-025-00854-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {813-824},
  shortjournal = {Nat. Comput. Sci},
  title        = {Analog in-memory computing attention mechanism for fast and energy-efficient large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep active optimization for complex systems. <em>NATCS</em>, <em>5</em>(9), 801-812. (<a href='https://doi.org/10.1038/s43588-025-00858-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inferring optimal solutions from limited data is considered the ultimate goal in scientific discovery. Artificial intelligence offers a promising avenue to greatly accelerate this process. Existing methods often depend on large datasets, strong assumptions about objective functions, and classic machine learning techniques, restricting their effectiveness to low-dimensional or data-rich problems. Here we introduce an optimization pipeline that can effectively tackle complex, high-dimensional problems with limited data. This approach utilizes a deep neural surrogate to iteratively find optimal solutions and introduces additional mechanisms to avoid local optima, thereby minimizing the required samples. Our method finds superior solutions in problems with up to 2,000 dimensions, whereas existing approaches are confined to 100 dimensions and need considerably more data. It excels across varied real-world systems, outperforming current algorithms and enabling efficient knowledge discovery. Although focused on scientific problems, its benefits extend to numerous quantitative fields, paving the way for advanced self-driving laboratories. This study introduces a deep active optimization pipeline that effectively tackles high-dimensional, complex problems with limited data. The approach minimizes sample size and surpasses existing methods, achieving optimal solutions in up to 2,000 dimensions.},
  archive      = {J_NATCS},
  author       = {Wei, Ye and Peng, Bo and Xie, Ruiwen and Chen, Yangtao and Qin, Yu and Wen, Peng and Bauer, Stefan and Tung, Po-Yen and Raabe, Dierk},
  doi          = {10.1038/s43588-025-00858-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {801-812},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep active optimization for complex systems},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Digital twin for chemical science: A case study on water interactions on the ag(111) surface. <em>NATCS</em>, <em>5</em>(9), 793-800. (<a href='https://doi.org/10.1038/s43588-025-00857-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directly visualizing chemical trajectories offers insights into catalysis, gas-phase reactions and photoinduced dynamics. Tracking the transformation of chemical species is best achieved by coupling theory and experiment. Here we developed Digital Twin for Chemical Science (DTCS) v.01, which integrates theory, experiment and their bidirectional feedback loops into a unified platform for chemical characterization. DTCS addresses a core question: given a set of experimental conditions, what is the expected outcome and why? It consists of a forward solver that takes a chemical reaction network and predicts spectra under experimental conditions, and an inverse solver that infers kinetics from measured spectra. We applied DTCS to ambient-pressure X-ray photoelectron spectroscopy measurements of the Ag–H2O interface as an example. This approach enables real-time knowledge extraction and guides experiments until a stopping condition is met based on accuracy and degeneracy. As a step toward autonomous chemical characterization, DTCS provides mechanistic knowledge in a verified, standardized manner. Interpreting spectroscopic data in real time remains a challenge in chemical characterization. Here a digital twin framework is developed that links first-principles theory and experimental data via a bidirectional feedback loop, enabling on-the-fly decision-making and insights into reaction mechanisms based on measured spectra during chemical experiments.},
  archive      = {J_NATCS},
  author       = {Qian, Jin and Jana, Asmita and Menon, Siddarth and Bogdan, Andrew E. and Hamlyn, Rebecca and Mahl, Johannes and Crumlin, Ethan J.},
  doi          = {10.1038/s43588-025-00857-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {793-800},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digital twin for chemical science: A case study on water interactions on the ag(111) surface},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SurFF: A foundation model for surface exposure and morphology across intermetallic crystals. <em>NATCS</em>, <em>5</em>(9), 782-792. (<a href='https://doi.org/10.1038/s43588-025-00839-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With approximately 90% of industrial reactions occurring on surfaces, the role of heterogeneous catalysts is paramount. Currently, accurate surface exposure prediction is vital for heterogeneous catalyst design, but it is hindered by the high costs of experimental and computational methods. Here we introduce a foundation force-field-based model for predicting surface exposure and synthesizability (SurFF) across intermetallic crystals, which are essential materials for heterogeneous catalysts. We created a comprehensive intermetallic surface database using an active learning method and high-throughput density functional theory calculations, encompassing 12,553 unique surfaces and 344,200 single points. SurFF achieves density-functional-theory-level precision with a prediction error of 3 meV Å−2 and enables large-scale surface exposure prediction with a 105-fold acceleration. Validation against computational and experimental data both show strong alignment. We applied SurFF for large-scale predictions of surface energy and Wulff shapes for over 6,000 intermetallic crystals, providing valuable data for the community. A foundation machine learning model, SurFF, enables DFT-accurate predictions of surface energies and morphologies in intermetallic catalysts, achieving over 105-fold acceleration for high-throughput materials screening.},
  archive      = {J_NATCS},
  author       = {Yin, Jun and Chen, Honghao and Qiu, Jiangjie and Li, Wentao and He, Peng and Li, Jiali and Karimi, Iftekhar A. and Lan, Xiaocheng and Wang, Tiefeng and Wang, Xiaonan},
  doi          = {10.1038/s43588-025-00839-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {782-792},
  shortjournal = {Nat. Comput. Sci},
  title        = {SurFF: A foundation model for surface exposure and morphology across intermetallic crystals},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Exploiting pleiotropy to enhance variant discovery with functional false discovery rates. <em>NATCS</em>, <em>5</em>(9), 769-781. (<a href='https://doi.org/10.1038/s43588-025-00852-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cost of recruiting participants for genome-wide association studies (GWASs) can limit sample sizes and hinder the discovery of genetic variants. Here we introduce the surrogate functional false discovery rate (sfFDR) framework that integrates summary statistics of related traits to increase power. The sfFDR framework provides estimates of FDR quantities such as the functional local FDR and q value, and uses these estimates to derive a functional P value for type I error rate control and a functional local Bayes’ factor for post-GWAS analyses. Compared with a standard analysis, sfFDR substantially increased power (equivalent to a 52% increase in sample size) in a study of obesity-related traits from the UK Biobank and discovered eight additional lead SNPs near genes linked to immune-related responses in a rare disease GWAS of eosinophilic granulomatosis with polyangiitis. Collectively, these results highlight the utility of exploiting related traits in both small and large studies. This study introduces a cost-effective strategy called surrogate functional false discovery rates to increase power in genome-wide association studies by leveraging genetic correlations (or pleiotropy) between related traits.},
  archive      = {J_NATCS},
  author       = {Bass, Andrew J. and Wallace, Chris},
  doi          = {10.1038/s43588-025-00852-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {769-781},
  shortjournal = {Nat. Comput. Sci},
  title        = {Exploiting pleiotropy to enhance variant discovery with functional false discovery rates},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Virtual brain twins for stimulation in epilepsy. <em>NATCS</em>, <em>5</em>(9), 754-768. (<a href='https://doi.org/10.1038/s43588-025-00841-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimating the epileptogenic zone network (EZN) is an important part of the diagnosis of drug-resistant focal epilepsy and has a pivotal role in treatment and intervention. Virtual brain twins provide a modeling method for personalized diagnosis and treatment. They integrate patient-specific brain topography with structural connectivity from anatomical neuroimaging such as magnetic resonance imaging, and dynamic activity from functional recordings such as electroencephalography (EEG) and stereo-EEG (SEEG). Seizures show rich spatial and temporal features in functional recordings, which can be exploited to estimate the EZN. Stimulation-induced seizures can provide important and complementary information. Here we consider invasive SEEG stimulation and non-invasive temporal interference stimulation as a complementary approach. This paper offers a high-resolution virtual brain twin framework for EZN diagnosis based on stimulation-induced seizures. It provides an important methodological and conceptual basis to make the transition from invasive to non-invasive diagnosis and treatment of drug-resistant focal epilepsy. A high-resolution virtual brain twin approach is proposed using stimulation-induced seizures to estimate the epileptogenic network, offering a step toward non-invasive diagnosis and treatment of drug-resistant focal epilepsy.},
  archive      = {J_NATCS},
  author       = {Wang, Huifang E. and Dollomaja, Borana and Triebkorn, Paul and Duma, Gian Marco and Williamson, Adam and Makhalova, Julia and Lemarechal, Jean-Didier and Bartolomei, Fabrice and Jirsa, Viktor},
  doi          = {10.1038/s43588-025-00841-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {754-768},
  shortjournal = {Nat. Comput. Sci},
  title        = {Virtual brain twins for stimulation in epilepsy},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On the compatibility of generative AI and generative linguistics. <em>NATCS</em>, <em>5</em>(9), 745-753. (<a href='https://doi.org/10.1038/s43588-025-00861-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chomsky’s generative linguistics has made substantial contributions to cognitive science and symbolic artificial intelligence. With the rise of neural language models, however, the compatibility between generative artificial intelligence and generative linguistics has come under debate. Here we outline three ways in which generative artificial intelligence aligns with and supports the core ideas of generative linguistics. In turn, generative linguistics can provide criteria to evaluate and improve neural language models as models of human language and cognition. This Perspective discusses that generative AI aligns with generative linguistics by showing that neural language models (NLMs) are formal generative models. Furthermore, generative linguistics offers a framework for evaluating and improving NLMs.},
  archive      = {J_NATCS},
  author       = {Portelance, Eva and Jasbi, Masoud},
  doi          = {10.1038/s43588-025-00861-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {745-753},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the compatibility of generative AI and generative linguistics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Arti-‘fickle’ intelligence: Using LLMs as a tool for inference in the political and social sciences. <em>NATCS</em>, <em>5</em>(9), 737-744. (<a href='https://doi.org/10.1038/s43588-025-00843-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To promote the scientific use of large language models (LLMs), we suggest that researchers in the political and social sciences refocus on the scientific goal of inference. We suggest that this refocus will improve the accumulation of shared scientific knowledge about these tools and their uses in the social sciences. We discuss the challenges and opportunities related to scientific inference with LLMs, using validation of model output as an illustrative case for discussion. We then propose a set of guidelines related to establishing the failure and success of LLMs when completing particular tasks and discuss how to make inferences from these observations. Large language models are increasingly important in social science research. The authors provide guidance on how best to validate and use these models as rigorous tools to further scientific inference.},
  archive      = {J_NATCS},
  author       = {Argyle, Lisa P. and Busby, Ethan C. and Gubler, Joshua R. and Hepner, Bryce and Lyman, Alex and Wingate, David},
  doi          = {10.1038/s43588-025-00843-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {737-744},
  shortjournal = {Nat. Comput. Sci},
  title        = {Arti-‘fickle’ intelligence: Using LLMs as a tool for inference in the political and social sciences},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Urban planning in the era of large language models. <em>NATCS</em>, <em>5</em>(9), 727-736. (<a href='https://doi.org/10.1038/s43588-025-00846-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {City plans are the product of integrating human creativity with emerging technologies, which continuously evolve and reshape urban morphology and environments. Here we argue that large language models hold large untapped potential in addressing the growing complexities of urban planning and enabling a more holistic, innovative and responsive approach to city design. By harnessing their advanced generation and simulation capabilities, large language models can contribute as an intelligent assistant for human planners in synthesizing conceptual ideas, generating urban designs and evaluating the outcomes of planning efforts. Large language models remain largely unexplored is the design of cities. In this Perspective, the authors discuss the potential opportunities brought by these models in assisting urban planning.},
  archive      = {J_NATCS},
  author       = {Zheng, Yu and Xu, Fengli and Lin, Yuming and Santi, Paolo and Ratti, Carlo and Wang, Qi R. and Li, Yong},
  doi          = {10.1038/s43588-025-00846-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {727-736},
  shortjournal = {Nat. Comput. Sci},
  title        = {Urban planning in the era of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rethinking chemical research in the age of large language models. <em>NATCS</em>, <em>5</em>(9), 715-726. (<a href='https://doi.org/10.1038/s43588-025-00811-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) offer opportunities for advancing chemical research, including planning, optimization, data analysis, automation and knowledge management. Deploying LLMs in active environments, where they interact with tools and data, can greatly enhance their capabilities. However, challenges remain in evaluating their performance and addressing ethical issues such as reproducibility, data privacy and bias. Here we discuss ongoing and potential integrations of LLMs in chemical research, highlighting existing challenges to guide the effective use of LLMs as active scientific partners. This Perspective highlights the potential integrations of large language models (LLMs) in chemical research and provides guidance on the effective use of LLMs as research partners, noting the ethical and performance-based challenges that must be addressed moving forward.},
  archive      = {J_NATCS},
  author       = {MacKnight, Robert and Boiko, Daniil A. and Regio, Jose Emilio and Gallegos, Liliana C. and Neukomm, Théo A. and Gomes, Gabe},
  doi          = {10.1038/s43588-025-00811-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {715-726},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rethinking chemical research in the age of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A digital twin that interprets and refines chemical mechanisms. <em>NATCS</em>, <em>5</em>(9), 713-714. (<a href='https://doi.org/10.1038/s43588-025-00859-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An integrated platform, Digital Twin for Chemical Science (DTCS), is developed to connect first-principles theory with spectroscopic measurements through a bidirectional feedback loop. By predicting and refining chemical reaction mechanisms before, during and after experiments, DTCS enables the interpretation of spectra and supports real-time decision-making in chemical characterization.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00859-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {713-714},
  shortjournal = {Nat. Comput. Sci},
  title        = {A digital twin that interprets and refines chemical mechanisms},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Overcoming computational bottlenecks in large language models through analog in-memory computing. <em>NATCS</em>, <em>5</em>(9), 711-712. (<a href='https://doi.org/10.1038/s43588-025-00860-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study demonstrates the potential of using in-memory computing architecture for implementing large language models for an improved computational efficiency in both time and energy while maintaining a high accuracy.},
  archive      = {J_NATCS},
  author       = {Lin, Yudeng and Tang, Jianshi},
  doi          = {10.1038/s43588-025-00860-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {711-712},
  shortjournal = {Nat. Comput. Sci},
  title        = {Overcoming computational bottlenecks in large language models through analog in-memory computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Neuromorphic principles in self-attention hardware for efficient transformers. <em>NATCS</em>, <em>5</em>(9), 708-710. (<a href='https://doi.org/10.1038/s43588-025-00868-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strong barriers remain between neuromorphic engineering and machine learning, especially with regard to recent large language models (LLMs) and transformers. This Comment makes the case that neuromorphic engineering may hold the keys to more efficient inference with transformer-like models.},
  archive      = {J_NATCS},
  author       = {Leroux, Nathan and Finkbeiner, Jan and Neftci, Emre},
  doi          = {10.1038/s43588-025-00868-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {708-710},
  shortjournal = {Nat. Comput. Sci},
  title        = {Neuromorphic principles in self-attention hardware for efficient transformers},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Using LLMs to advance the cognitive science of collectives. <em>NATCS</em>, <em>5</em>(9), 704-707. (<a href='https://doi.org/10.1038/s43588-025-00848-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) are already transforming the study of individual cognition, but their application to studying collective cognition has been underexplored. We lay out how LLMs may be able to address the complexity that has hindered the study of collectives and raise possible risks that warrant new methods.},
  archive      = {J_NATCS},
  author       = {Sucholutsky, Ilia and Collins, Katherine M. and Jacoby, Nori and Thompson, Bill D. and Hawkins, Robert D.},
  doi          = {10.1038/s43588-025-00848-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {704-707},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using LLMs to advance the cognitive science of collectives},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Threats to scientific software from over-reliance on AI code assistants. <em>NATCS</em>, <em>5</em>(9), 701-703. (<a href='https://doi.org/10.1038/s43588-025-00845-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The adoption of generative artificial intelligence (AI) code assistants in scientific software development is promising, but user studies across an array of programming contexts suggest that programmers are at risk of over-reliance on these tools, leading them to accept undetected errors in generated code. Scientific software may be particularly vulnerable to such errors because most research code is untested and scientists are undertrained in software development skills. This Comment outlines the factors that place scientific code at risk and suggests directions for research groups, educators, publishers and funders to counter these liabilities.},
  archive      = {J_NATCS},
  author       = {O’Brien, Gabrielle},
  doi          = {10.1038/s43588-025-00845-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {701-703},
  shortjournal = {Nat. Comput. Sci},
  title        = {Threats to scientific software from over-reliance on AI code assistants},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational challenges arising in algorithmic fairness and health equity with generative AI. <em>NATCS</em>, <em>5</em>(9), 698-700. (<a href='https://doi.org/10.1038/s43588-025-00806-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of generative artificial intelligence (AI) in healthcare is advancing, but understanding its potential challenges for fairness and health equity is still in its early stages. This Comment investigates how to define fairness and measure it, and highlights research that can help address challenges in the field.},
  archive      = {J_NATCS},
  author       = {Suriyakumar, Vinith M. and Zink, Anna and Hightower, Maia and Ghassemi, Marzyeh and Beaulieu-Jones, Brett},
  doi          = {10.1038/s43588-025-00806-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {698-700},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational challenges arising in algorithmic fairness and health equity with generative AI},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The impact of language models on the humanities and vice versa. <em>NATCS</em>, <em>5</em>(9), 695-697. (<a href='https://doi.org/10.1038/s43588-025-00819-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many humanists are skeptical of language models and concerned about their effects on universities. However, researchers with a background in the humanities are also actively engaging with artificial intelligence — seeking not only to adopt language models as tools, but to steer them toward a more flexible, contextual representation of written culture.},
  archive      = {J_NATCS},
  author       = {Underwood, Ted},
  doi          = {10.1038/s43588-025-00819-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {695-697},
  shortjournal = {Nat. Comput. Sci},
  title        = {The impact of language models on the humanities and vice versa},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The other AI revolution: How the global south is building and repurposing language models that speak to billions. <em>NATCS</em>, <em>5</em>(9), 691-694. (<a href='https://doi.org/10.1038/s43588-025-00865-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While leading tech companies race to build ever-larger models, researchers in Brazil, India and Africa are using clever tricks to remix big labs’ LLMs to bring AI to billions of users.},
  archive      = {J_NATCS},
  author       = {Burgos, Pedro},
  doi          = {10.1038/s43588-025-00865-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {691-694},
  shortjournal = {Nat. Comput. Sci},
  title        = {The other AI revolution: How the global south is building and repurposing language models that speak to billions},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The rise of large language models. <em>NATCS</em>, <em>5</em>(9), 689-690. (<a href='https://doi.org/10.1038/s43588-025-00890-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This issue of Nature Computational Science features a Focus that highlights both the promises and perils of large language models, their emerging applications across diverse scientific domains, and the opportunities to overcome the challenges that lie ahead.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00890-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {689-690},
  shortjournal = {Nat. Comput. Sci},
  title        = {The rise of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natee">NATEE - 33</h2>
<ul>
<li><details>
<summary>
(2025). Citrus reticulata (mandarin). <em>NATEE</em>, <em>9</em>(9), 1752. (<a href='https://doi.org/10.1038/s41559-025-02822-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Qiang Xu has been captivated by citrus fruits since childhood.},
  archive      = {J_NATEE},
  author       = {Xu, Qiang},
  doi          = {10.1038/s41559-025-02822-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1752},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Citrus reticulata (mandarin)},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease. <em>NATEE</em>, <em>9</em>(9), 1739-1751. (<a href='https://doi.org/10.1038/s41559-025-02797-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More than 10 years following the onset of the sea star wasting disease (SSWD) epidemic, affecting over 20 asteroid species from Mexico to Alaska, the causative agent has been elusive. SSWD killed billions of the most susceptible species, sunflower sea stars (Pycnopodia helianthoides), initiating a trophic cascade involving unchecked urchin population growth and the widespread loss of kelp forests. Identifying the causative agent underpins the development of recovery strategies. Here we induced disease and subsequent mortality in exposure experiments using tissue extracts, coelomic fluid and effluent water from wasting sunflower sea stars, with no mortality in controls. Deep sequencing of diseased sea star coelomic fluid samples from experiments and field outbreaks revealed a dominant proportion of reads assigned to the bacterium Vibrio pectenicida. Fulfilling Koch’s postulates, V. pectenicida strain FHCF-3, cultured from the coelomic fluid of a diseased sunflower sea star, caused disease and mortality in exposed sunflower sea stars, demonstrating that it is a causative agent of SSWD. This discovery will enable recovery efforts for sea stars and the ecosystems affected by their decline by facilitating culture-based experimental research and broad-scale screening for pathogen presence and abundance in the laboratory and field. The causative agent of sea star wasting disease has been elusive. This study used genetic datasets and experimental exposures to demonstrate that a strain of the bacterium Vibrio pectenicida caused disease and mortality in sea stars.},
  archive      = {J_NATEE},
  author       = {Prentice, Melanie B. and Crandall, Grace A. and Chan, Amy M. and Davis, Katherine M. and Hershberger, Paul K. and Finke, Jan F. and Hodin, Jason and McCracken, Andrew and Kellogg, Colleen T. E. and Clemente-Carvalho, Rute B. G. and Prentice, Carolyn and Zhong, Kevin X. and Harvell, C. Drew and Suttle, Curtis A. and Gehman, Alyssa-Lois M.},
  doi          = {10.1038/s41559-025-02797-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1739-1751},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories. <em>NATEE</em>, <em>9</em>(9), 1731-1738. (<a href='https://doi.org/10.1038/s41559-025-02798-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Radiogenic strontium isotopes (87Sr/86Sr) and the alkaline earth ratios (AERs) Sr/Ca and Ba/Ca in fossil dental enamel can inform the habitat, residence and life histories of early hominins recovered from the Pleistocene cradle-of-humankind sites of Gauteng, South Africa. Key questions, which may be addressed with these indices, are the relative exploitation of wet versus dry botanic regimes and whether early hominins dispersed in a manner similar to that of chimpanzees (characterized by male philopatry and female dispersal at puberty) or to that of humans (who are not so characterized). Here we developed 28 new dental chronologies in 20 Paranthropus robustus teeth from Swartkrans and Kromdraai. Resulting geochemical time series demonstrate that, while maternal 87Sr/86Sr in earlier-forming teeth varies widely, third molar 87Sr/86Sr, derived from postweaning solid foods, progressively converges to 0.7306 ± 0.0035 (± 2 s.d.), which we express as the local isotopically delineated exploitation area (LIDEA). The spatial resolution of LIDEA is determined using a bioavailable 87Sr/86Sr isoscape. In this environmental context, we interpret LIDEA as a quantifiable signal indicating eurytopy (generalization), with some 30% of Sr deriving from riparian woodland habitats. With regard to residence, many individuals arrived at the site after second molar mineralization, while some matured locally, demonstrating both male and female dispersal as well as lifelong local residence. Analysis of both 87Sr/86Sr and the AERs further highlights concomitant patterns, as well as numerous periodicities that may be related to resource depletion, seasonality or lunar cycles. Strontium analysis of Paranthropus robustus teeth from two South African sites reveals that the Pleistocene hominin probably exploited both savanna and riparian woodland habitats, had a home-range size compatible with that of present-day hunter-gatherers and had periodic ranging patterns. While some individuals lived their lives in the same place, others dispersed during their lifetime.},
  archive      = {J_NATEE},
  author       = {Sillen, Andrew and Dean, Christopher and Balter, Vincent},
  doi          = {10.1038/s41559-025-02798-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1731-1738},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas. <em>NATEE</em>, <em>9</em>(9), 1719-1730. (<a href='https://doi.org/10.1038/s41559-025-02794-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most faunas from the Mesozoic era were dominated by sauropod dinosaurs, the largest terrestrial animals to ever exist. These megaherbivores were remarkably diverse and widely distributed. Here we study three Late Jurassic faunas from the USA, Portugal and Tanzania, each approximately 150 million years old, which are known for their extreme sauropod diversity. Whereas general taxonomic composition was similar in these three faunas, the major clades differed in relative abundance. Moreover, their depositional strata record distinct climatic regimes. Using dental microwear texture analysis, we investigated the impact of these climate regimes and the resulting food availability on the different sauropod taxa. Wear patterns in camarasaurid macronarians show minimal variation across different climate regimes, supporting previous studies suggesting that these animals migrated to follow their preferred climate niche and food source. North American camarasaurids show similar wear patterns to those of Portuguese turiasaurs, another broad-crowned taxon, which did not exist in the Jurassic of North America. By contrast, where camarasaurids and turiasaurs co-occurred in Portugal, their microwear patterns are distinct, suggesting niche differentiation to avoid ecological competition between these two clades. Flagellicaudatan diplodocoids display highly variable wear patterns, indicating limited migration (and therefore seasonal variation in diet), which aligns with observed biogeography patterns in the USA. Early-branching titanosauriforms show highly distinct wear patterns between different climate regimes, which can probably be attributed to different abrasive loads in the respective habitats. Our results demonstrate that dental microwear texture analysis not only records dietary preferences but also reveals behaviour such as competition and migration related to dietary niches in past ecosystems. The authors compare tooth wear patterns from three Late Jurassic sauropod faunas in North America, Portugal and Tanzania and find that different regions and climate regimes seem to have had distinctive impacts on dental wear of megaherbivores in these areas, revealing habitat and ecological niche information.},
  archive      = {J_NATEE},
  author       = {Winkler, Daniela E. and Tschopp, Emanuel and Saleiro, André and Wiesinger, Ria and Kaiser, Thomas M.},
  doi          = {10.1038/s41559-025-02794-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1719-1730},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enlargement of sternum traits facilitated the evolution of powered flight in birds. <em>NATEE</em>, <em>9</em>(9), 1705-1718. (<a href='https://doi.org/10.1038/s41559-025-02795-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An enlarged sternum with a prominent keel is a central feature of the flight apparatus of modern birds. However, sterna of near-bird dinosaurs (Pennaraptora) and early avialans are either substantially different from those of living birds or absent altogether, raising questions about how specialized sternal structures evolved in birds and how they are related to function. This remains poorly understood because of the fragmentary nature of the fossil record, and the challenges in inferring form and function from crushed fossils. We use ancestral character estimations to trace sternal trait acquisition through the bird stem group, and multivariate phylogenetic regressions to analyse relationships between sternum morphology, body mass and flight capabilities. We find that sternum evolution was episodic: basal members of Pennaraptora had proportionally small sterna, which became larger and more craniocaudally elongated in Avialae. This enlargement precedes the appearance of a midline ridge, a possible precursor of the sternal keel, in Pygostylia. Sternum size increased again in crownward Ornithuromorpha, alongside a fully formed sternal keel and enlarged caudal projections, both critical areas of flight muscle attachment. Sternal experimentation in relation to flight characteristics occurs several times throughout Pennaraptora, including within Paraves and Enantiornithes, indicating that powered flight may have evolved several times before proliferating in crown-group birds. The extant avian sternum is highly adapted to powered flight, but understanding of its early evolution is hindered by patchy fossil representation and the fact that where preserved, sterna in near-bird dinosaurs and early avialans are substantially different. To overcome this, the authors use ancestral character estimations and find that sternum evolution was episodic and experimental in relation to flight characteristics.},
  archive      = {J_NATEE},
  author       = {Lowi-Merri, Talia M. and Benson, Roger and Hu, Han and O’Connor, Jingmai and Claramunt, Santiago and Evans, David C.},
  doi          = {10.1038/s41559-025-02795-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1705-1718},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Enlargement of sternum traits facilitated the evolution of powered flight in birds},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The role of plasmid copy number and mutation rate in evolutionary outcomes. <em>NATEE</em>, <em>9</em>(9), 1694-1704. (<a href='https://doi.org/10.1038/s41559-025-02792-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multicopy plasmids are widespread in nature and compose a common strategy for spreading beneficial traits across microbes. However, the role of plasmids in supporting the evolution of encoded genes remains underexplored due to challenges in experimentally manipulating key parameters such as plasmid copy number and mutation rate. Here we developed a strategy for controlling copy number in the plasmid-based Saccharomyces cerevisiae continuous evolution system, OrthoRep, and used our resulting capabilities to investigate the evolution of a conditionally essential gene under varying copy number and mutation rate conditions. Our results show that low copy number facilitated the faster enrichment of beneficial alleles whereas high copy number promoted robustness through the maintenance of allelic diversity. High copy number also slowed the removal of deleterious mutations and increased the fraction of non-functional alleles that could hitchhike during evolution. This study highlights the nuanced relationships between plasmid copy number, mutation rate and evolutionary outcomes, providing insights into the adaptive dynamics of genes encoded on multicopy plasmids and nominating OrthoRep as a versatile tool for studying plasmid evolution. In an experimental multicopy plasmid evolution system that allows manipulation of mutation rate and copy number, low copy number promoted enrichment of beneficial alleles whereas high copy number maintained allelic diversity.},
  archive      = {J_NATEE},
  author       = {Pisera, Alexander ‘Olek’ and Liu, Chang C.},
  doi          = {10.1038/s41559-025-02792-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1694-1704},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The role of plasmid copy number and mutation rate in evolutionary outcomes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas. <em>NATEE</em>, <em>9</em>(9), 1685-1693. (<a href='https://doi.org/10.1038/s41559-025-02771-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis is a recently identified cause of Hansen’s disease, and is associated with the more severe and potentially lethal presentations of diffuse lepromatous leprosy and Lucio’s phenomenon. Detection of this infection has been limited to a small number of individuals, leaving much to be learned about its global distribution and transmissibility. Its discovery in wild rodent populations in the United Kingdom and Ireland also raises questions about its zoonotic potential. Here, we raise further awareness of this disease via analyses of two exceptionally well preserved M. lepromatosis genomes obtained from 4,000-year-old human remains of two adult males from the archaeological sites of El Cerrito and La Herradura in Northern Chile. This formed the basis of genomic comparisons between ancient and modern forms of the pathogen. We demonstrate an unexpected long history of M. lepromatosis in the Americas, which contrasts with the more recent Eurasian history of the closely related Mycobacterium leprae. We offer relevant perspectives on its evolution while providing an incentive for further disease monitoring in both humans and other potential reservoir species in the Americas and elsewhere. Recovery of ancient Mycobacterium lepromatosis genomes in 4,000-year-old human remains from Chile demonstrates that this causal agent of Hansen’s disease has a long history in the Americas, predating the colonial period by millennia.},
  archive      = {J_NATEE},
  author       = {Ramirez, Darío A. and Sitter, T. Lesley and Översti, Sanni and Herrera-Soto, María José and Pastor, Nicolás and Fontana-Silva, Oscar Eduardo and Kirkpatrick, Casey L. and Castelleti-Dellepiane, José and Nores, Rodrigo and Bos, Kirsten I.},
  doi          = {10.1038/s41559-025-02771-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1685-1693},
  shortjournal = {Nat. Ecol. Evol},
  title        = {4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pervasive but biome-dependent relationship between fragmentation and resilience in forests. <em>NATEE</em>, <em>9</em>(9), 1670-1684. (<a href='https://doi.org/10.1038/s41559-025-02776-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The relationship between landscape fragmentation and vegetation resilience is uncertain. Here we use multiple satellite-based tree cover data and vegetation indices to quantify the apparent effects of fragmentation on global forest resilience and potential mechanisms thereof. We measure fragmentation as edge density, patch density and mean patch area of tree cover patches, and measure resilience as one-lag temporal autocorrelation of vegetation indices. We find a statistically significant (P &lt; 0.05) fragmentation–resilience relationship in about 77% of fragmented forests, but the direction varies across biomes. In tropical and temperate forests, fragmentation is linked to increased local temperature and atmospheric dryness, resulting in a negative fragmentation–resilience relationship. Conversely, in boreal forests, fragmentation is associated with decreased atmospheric dryness and enhanced light resource, thereby increasing forest resilience. Our results reconcile competing hypotheses and highlight the importance of accounting for fragmentation when predicting shifts in ecosystem resilience under disturbances. These findings also suggest the necessity of biome-targeted forest management strategies for climate change mitigation and adaptation. Human-driven forest fragmentation could have major impacts on ecosystem functioning. This global analysis shows that the relationship between fragmentation and vegetation resilience in forests may differ depending on bioclimatic region and local environmental conditions.},
  archive      = {J_NATEE},
  author       = {Su, Yongxian and Zhang, Chaoqun and Cescatti, Alessandro and Yu, Kailiang and Ciais, Philippe and Smith, Taylor and Shang, Jiali and Carnicer, Jofre and Liu, Jane and Chen, Jing Ming and Green, Julia K. and Wu, Jianping and Ponce-Campos, Guillermo E. and Zhang, Yongguang and Zuo, Zhiyan and Liao, Jinbao and Lafortezza, Raffaele and Yan, Kai and Yang, Xueqin and Liu, Liyang and Ren, Jiashun and Yuan, Wenping and Chen, Xiuzhi and Wu, Chaoyang and Zhou, Weiqi},
  doi          = {10.1038/s41559-025-02776-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1670-1684},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pervasive but biome-dependent relationship between fragmentation and resilience in forests},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family. <em>NATEE</em>, <em>9</em>(9), 1656-1669. (<a href='https://doi.org/10.1038/s41559-025-02777-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations are more likely to produce some phenotypes than others, but the causal role of these production propensities in the evolution of phenotypic diversity remains unclear. There are two major challenges: it is difficult to separate the effect of the genotype–phenotype (GP) map from that of natural selection when analysing natural diversity, and most extant phenotypes evolved long ago in species whose GP maps cannot be recovered. Here, using two reconstructed ancestral transcription factors that are closely related but differ in function, we created libraries containing all possible amino acid combinations at historically variable sites in the proteins’ DNA binding interface (the genotypes) and measured their capacity to specifically bind DNA elements containing all possible combinations of nucleotides at historically variable sites (the phenotypes). The ancestral GP maps were strongly anisotropic (the distribution of phenotypes encoded by genotypes is highly non-uniform) and heterogeneous (the phenotypes accessible around each genotype vary greatly among genotypes), but the extent and direction of these properties differed substantially between the maps. In both cases, these properties steered evolution towards the lineage-specific phenotypes that evolved during history. Our findings establish that ancient properties of the GP relationship were causal factors in the evolutionary process that produced present-day patterns of functional conservation and diversity. A combination of ancestral reconstruction and deep mutational scanning to examine the genotype–phenotype maps of steroid receptors shows that the properties of ancestral genotype–phenotype maps determine the lineage-specific evolution of DNA specificity.},
  archive      = {J_NATEE},
  author       = {Herrera-Álvarez, Santiago and Patton, Jaeda E. J. and Thornton, Joseph W.},
  doi          = {10.1038/s41559-025-02777-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1656-1669},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments. <em>NATEE</em>, <em>9</em>(9), 1643-1655. (<a href='https://doi.org/10.1038/s41559-025-02779-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human impacts on nature span vast spatial scales that transcend abiotic gradients and biogeographic barriers, yet estimates of biodiversity loss from land-use change overwhelmingly derive from local-scale studies. Using a field dataset of 971 bird species sampled in forest and cattle pasture across 13 biogeographic regions of Colombia, we quantify biodiversity losses from local to near-national scales. Losses are on average 60% worse at the pan-Colombian scale than in individual regions, with underestimation remaining until six to seven biogeographic regions are sampled. Regional losses greatly exceed local losses when beta-diversity is high due to reduced species turnover in pasture across geographic space and elevation. Extrapolation from local-scale studies causes major underestimation of biodiversity loss, emphasizing the need to incorporate spatial structure into measures of change. Analysis of data on 971 bird species in natural habitat and cattle pasture in Colombia finds that near-national-scale losses of bird diversity greatly exceed losses recorded at the local scale, suggesting that extrapolations from local studies will severely underestimate biodiversity losses.},
  archive      = {J_NATEE},
  author       = {Socolar, Jacob B. and Mills, Simon C. and Gilroy, James J. and Martínez-Revelo, Diego E. and Medina-Uribe, Claudia A. and Parra-Sanchez, Edicson and Ramirez-Gutierrez, Marcela and Sand Sæbø, Jørgen and Meneses, Henry S. and Pérez, Giovanny and Barlow, Jos and Ochoa Quintero, Jose M. and Freckleton, Robert P. and Haugaasen, Torbjørn and Edwards, David P.},
  doi          = {10.1038/s41559-025-02779-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1643-1655},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development. <em>NATEE</em>, <em>9</em>(9), 1626-1642. (<a href='https://doi.org/10.1038/s41559-025-02780-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bats are the only mammals capable of self-powered flight, an evolutionary innovation based on the transformation of forelimbs into wings. The bat wing is characterized by an extreme elongation of the second to fifth digits with a wing membrane called the chiropatagium connecting them. Here we investigated the developmental and cellular origin of this structure by comparing bat and mouse limbs using omics tools and single-cell analyses. Despite the substantial morphological differences between the species, we observed an overall conservation of cell populations and gene expression patterns including interdigital apoptosis. Single-cell analyses of micro-dissected embryonic chiropatagium identified a specific fibroblast population, independent of apoptosis-associated interdigital cells, as the origin of this tissue. These distal cells express a conserved gene programme including the transcription factors MEIS2 and TBX3, which are commonly known to specify and pattern the early proximal limb. Transgenic ectopic expression of MEIS2 and TBX3 in mouse distal limb cells resulted in the activation of genes expressed during wing development and phenotypic changes related to wing morphology, such as the fusion of digits. Our results elucidate fundamental molecular mechanisms of bat wing development and illustrate how drastic morphological changes can be achieved through repurposing of existing developmental programmes during evolution. Single-cell comparison of developing bat and mouse limbs reveals conservation of cell populations and gene expression patterns, and suggests repurposing of genes involved in proximal limb development for wing evolution.},
  archive      = {J_NATEE},
  author       = {Schindler, Magdalena and Feregrino, Christian and Aldrovandi, Silvia and Lo, Bai-Wei and Monaco, Anna A. and Ringel, Alessa R. and Morales, Ariadna E. and Zehnder, Tobias and Behncke, Rose Yinghan and Glaser, Juliane and Barclay, Alexander and Andrey, Guillaume and Kragesteen, Bjørt K. and Hägerling, René and Haas, Stefan A. and Vingron, Martin and Ulitsky, Igor and Marti-Renom, Marc A. and Hechavarria, Julio and Fasel, Nicolas and Hiller, Michael and Lupiáñez, Darío G. and Mundlos, Stefan and Real, Francisca M.},
  doi          = {10.1038/s41559-025-02780-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1626-1642},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep learning reveals the complex genetic architecture of male guppy colouration. <em>NATEE</em>, <em>9</em>(9), 1614-1625. (<a href='https://doi.org/10.1038/s41559-025-02781-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The extraordinary variation in male guppy (Poecilia reticulata) colouration is a powerful model for studying the interplay of natural and sexual selection. However, the complexity of this variation has hampered the high-resolution characterization and determination of the genetic architecture underlying male guppy colour and clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male colour variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a genome-wide association study of colour traits. Our phenotypic and genomic results converge to show that colour patterning in guppies is a combination of many heritable features, each with a largely independent genetic architecture spanning the entire genome. Autosomally inherited ornaments are polygenic, with significant contributions from loci involved in neural crest cell migration. Unusually, the results of our genome-wide association study suggest that gene duplicates from the autosomes to the Y chromosome are responsible for much of the sex-linked variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait. Guppy colouration is a very diverse trait under natural and sexual selection. Using a combination of high-resolution phenotyping and genomic analysis, the authors reveal the genetic architecture of this complex trait.},
  archive      = {J_NATEE},
  author       = {van der Bijl, Wouter and Shu, Jacelyn J. and Goberdhan, Versara S. and Sherin, Linley M. and Jia, Changfu and Cortazar-Chinarro, Maria and Corral-Lopez, Alberto and Mank, Judith E.},
  doi          = {10.1038/s41559-025-02781-w},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1614-1625},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deep learning reveals the complex genetic architecture of male guppy colouration},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood. <em>NATEE</em>, <em>9</em>(9), 1599-1613. (<a href='https://doi.org/10.1038/s41559-025-02785-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deadwood is a major carbon source in forests, and yet the fate of this carbon remains a gap in our understanding of global carbon cycling. Lignin, the most recalcitrant biopolymer in wood, is mainly decayed through extracellular enzymatic and chemical processes initiated by white-rot fungi. However, the intracellular conversion of lignin decay products has been overlooked in the fungal kingdom. Here we integrate comparative genomic and phylogenetic analyses to understand the distribution and evolution of enzymes responsible for modifying lignin-related aromatic compounds—such as decarboxylases, hydroxylases, dioxygenases and other downstream ring-cleavage enzymes—that funnel carbon to central metabolism across the bacterial and the fungal kingdoms. We demonstrate that specific fungal lineages conserve these enzyme families, and that the abilities to enzymatically depolymerize lignin and catabolize lignin-related aromatic compounds are not necessarily coupled. Our analyses also reveal an expanded substrate specificity of aromatic ring-cleavage enzymes during fungal evolution, as well as a clade of extracellular enzymes among them, broadening the spatial range of these biochemical capabilities. Together, our results highlight a large diversity of fungal enzymes and hosts that warrant further investigation for inclusion into carbon cycling models and biotechnological applications for the conversion of aromatic compounds. This study uses comparative genomics and phylogenetics to analyse the distribution and evolution of key enzymes involved in the catabolism of lignin-related aromatic compounds in the bacterial and fungal kingdoms.},
  archive      = {J_NATEE},
  author       = {Kijpornyongpan, Teeratas and Kuatsjah, Eugene and Komp, Evan and Evans, James E. and Ruiz-Dueñas, Francisco Javier and Salvachúa, Davinia},
  doi          = {10.1038/s41559-025-02785-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1599-1613},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife. <em>NATEE</em>, <em>9</em>(9), 1585-1598. (<a href='https://doi.org/10.1038/s41559-025-02786-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across the world, human (anthropophonic) sounds add to sounds of biological (biophonic) and geophysical (geophonic) origin, with human contributions including both speech and technophony (sounds of technological devices). To characterize society’s contribution to the global soundscapes, we used passive acoustic recorders at 139 sites across 6 continents, sampling both urban green spaces and nearby pristine sites continuously for 3 years in a paired design. Recordings were characterized by bird species richness and by 14 complementary acoustic indices. By relating each index to seasonal, diurnal, climatic and anthropogenic factors, we show here that latitude, time of day and day of year each predict a substantial proportion of variation in key metrics of biophony—whereas anthropophony (speech and traffic) show less predictable patterns. Compared to pristine sites, the soundscape of urban green spaces is more dominated by technophony and less diverse in terms of acoustic energy across frequencies and time steps, with less instances of quiet. We conclude that the global soundscape is formed from a highly predictable rhythm in biophony, with added noise from geophony and anthropophony. At urban sites, animals experience an increasingly noisy background of sound, which poses challenges to efficient communication. Analysis of soundscape data from 139 globally distributed sites reveals that sounds of biological origin exhibit predictable rhythms depending on location and season, whereas sounds of anthropogenic origin are less predictable. Comparisons between paired urban–rural sites show that urban green spaces are noisier and dominated by sounds of technological origin.},
  archive      = {J_NATEE},
  author       = {Somervuo, Panu and Roslin, Tomas and Fisher, Brian L. and Hardwick, Bess and Kerdraon, Deirdre and Raharinjanahary, Dimby and Rajoelison, Eric Tsiriniaina and Lauha, Patrik and Griem, Lukas and Lehikoinen, Petteri and Niittynen, Pekka and Piirainen, Esko and Lumme, Markus and Riihikoski, Ville-Matti and Acevedo-Charry, Orlando and Adalsteinsson, Solny A. and Ahmad, Maaz and Alcobia, Sandra and Aldará, Jón and Andrew, Nigel R. and Anslan, Sten and Antonelli, Alexandre and Arena, Julieta Soledad and Arroyo Almeida, Santiago and Aster, Ines and Autto, Hannu and Aviles Gamboa, Anahi and Baixeras, Joaquín and Baldauf, Mario and Balestrieri, Rosario and Banelyte, Gaia Giedre and Barrett, Adrian and Beja, Pedro and Berg, Thomas Olof and Bergerot, Benjamin and Biro, Elizabeth G. and Blendinger, Pedro G. and Bollache, Loïc and Bou Dagher Kharrat, Magda and Boyer, Stephane and Bridell, Erika and Brotherson, Martyn and Brown, Leslie Robert and Buckley, Hannah L. and Buscardo, Erika and Buthelezi, Nokuphila and Cagnolo, Luciano and Calvente, Alice and Capobianco, Giovanni and Carreón-Palau, Laura and Carriere, Suzanne and Case, Bradley S. and Chang, Jenyu and Chaparro, Juan Matías and Chen, Chi-Ling and Chicoine, Christine and Christensson, Madeleine and Collado Rosique, Francisco and Colom Montero, William and da Fonseca, Ricardo do Sacramento and Da Silva, Luís P. and Dal Molin, Anamaria and Dallas, Tad and de Francesco, Maria Carla and Del Ángel-Rodríguez, Jorge Arturo and Díaz-Delgado, Ricardo and Dirnböck, Thomas and Djukic, Ika and Dladla, Philile and Domínguez Masciale, Jeremías and Dorigo, Thiago and Douwes, Errol and Ekrem, Torbjørn and Enderskog, Helena and Erefur, Charlotta and Fahad, Muhammad and Falahati-Anbaran, Mohsen and Farrell, Arielle and Ferland, Gabriel and Ferrari, Emanuele and Figueiredo, Axa and Forero, Fernando and Freiberga, Inga and Frosch-Radivo, Andrea and Ganchozo Intriago, Luis Alberto and Garzoli, Laura and Giacomotti, Paola and Gianuca, Andros T. and Gilg, Olivier and Gilg, Vladimir and Gísladóttir, Fanney and Glowacki, Ryan and Gottsberger, Brigitte and Gregoire, Jocelyn and Groner, Elli and Guedes, Patrícia and Guile, Aimee Michelle and Haase, Peter and Hadi, Fazal and Haidegger, Magdalena and Hansen, Leivur Janus and Hansen, Lars Holst and Harrop, Reid and Havnås, Harald and Herrera Báez, David and Ho, Chris C. Y. and Hohenbühel, Denise and Houska Tahadlova, Marketa and Hänninen, Jari and Höglund, Linda and Í Haraldsstovu, Kolbrún and Imbeau, Elise and Inkinen, Jasmin and Ishihara, Masae Iwamoto and Jackson, Abigail C. and Jansson, Gunnar and Jha, Rohit and Kager, Gerald and Kahale, Rhea and Kalttopää, Oula and Karai, Elizabeth Wanjiru and Karlsson, Dave and Kaus-Thiel, Andrea and Khan, Asghar and Khan, Qaisar and Kimoto, Keishi and Kipngetich, Shadrack Chumo and Klante, Clemens and Klemedtsson, Leif and Klinth, Mårten and Koskinen, Janne and Kotakorpi, Matti and Kreiling, Agnes-Katharina and Krisai-Greilhuber, Irmgard and Kristensen, Erik and König, Sebastian and Langenheder, Silke and Laurila, Kalevi and Le Gouar, Pascaline and Lecomte, Nicolas and Lecomte, Erin and Leitman, Paula Moraes and León-Cortés, Jorge L. and Li, Daijiang and Loehr, John and Lopez-Vaamonde, Carlos and Makari, Mehsen and Mangini, Gabriela Giselle and Maroschek, Michael and Mata, Vanessa A. and Matsuoka, Shunsuke and Mazzafera, Thais and McDonald, Paul G. and Meinert, Laura and Meléndez-González, Mayra and Mendoza-Henao, Angela M. and Moreau, Sebastien and Moreau, Jérôme and Mosbacher, Jesper and Moyer, Esteban and Mrazova, Anna and Mteshane, Samantha and Mungai, Nancy Wangari and Muñoz Herraiz, Gema and Murillo-Vázquez, Andrea and Musazzi, Simona and Mutanen, Marko and Müller, Jörg and Navarro Canales, Rebeca and Ndlovu, Monica and Nicolai, Annegret and Niessner, Armin and Nordén, Jenni and Nowak, Paweł and O’Connell, Erin and Orru, Arianna and Pagnon, Thomas and Pantoja-Diaz, Yurani Nayive and Pentinsaari, Mikko and Pilloni, Sebastian and Pinder, Adrian and Pinheiro, Thiago A. and Põlme, Sergei and Powell, Luke L. and Pröll, Gisela and Pulido-Santacruz, Paola and Queralt, Enrique and Quilantang, Mark Tristan and Quinlan, Kirsty and Ramirez, Ricardo and Rankinen, Juha and Rasino, Micaela Del Valle and Rebelo, Rui and Remmers, Wolfram and Retz, Franziska and Reyes, Evelin and Rivas Torres, Gonzalo and Rogers, Hanna M. K. and Rosário, Inês T. and Rosário Da Rosàrio da Costa, Sidney and Rütting, Tobias and Sahlstén, Johannes and Saliba, Carole and Salmirinne, Teppo and Sam, Katerina and Santos, Douglas and Santos-Reis, Margarida and Sawan, Michel and Schattanek-Wiesmair, Benjamin and Schiestl-Aalto, Pauliina and Schmidt, Niels Martin and Seibold, Sebastian and Seidl, Rupert and Seifert, Linda and Sithole, Malibongwe and Sivault, Elise and Smart, Jessica and Smerczyński, Ireneusz and Soda, Ayaka and Sousa-Lima, Renata S. and Stanisci, Angela and Stanley, Margaret C. and Steenkamp, Daleen and Stengel, Elisa and Stoll, Stefan and Strauss, Willem Maartin and Stur, Elisabeth and Sujala, Maija and Sundell, Janne and Svavarsdóttir, Jónína and Tedersoo, Leho and Tepsa, Saana and Tikochinsky, Maor Tiko and Tuominen, Esa-Pekka and Tweraser, Stefanie and Ulloa Espinosa, Catalina and Uusitalo, Joni and Vallinmäki, Mikko and Vannier, Fabrice and Varela, Abigail and Vatka, Emma and Veikkolainen, Silja and Vernes, Karl and Watts, Phillip C. and Weslien, Per and Wirth, Ciara and Wisniewski, Jana Helga and Young, Amanda B. and Övergaard, Robyn and Ovaskainen, Otso},
  doi          = {10.1038/s41559-025-02786-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1585-1598},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency. <em>NATEE</em>, <em>9</em>(9), 1571-1584. (<a href='https://doi.org/10.1038/s41559-025-02761-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerous leaf-level experiments suggest that plant intrinsic water-use efficiency (iWUE) increases under elevated CO2 because of reduced stomatal conductance and enhanced photosynthesis. However, it remains elusive whether this response can be extrapolated to the ecosystem scale, because confounding factors and compensating feedbacks are often involved in ecosystem iWUE variations. Here we develop a machine learning-based framework to disentangle the ecosystem-scale CO2 effects on iWUE and its two components, canopy conductance (Gc) and gross primary productivity (GPP), based on global networks of long-term eddy covariance observations. Our results show widespread CO2-induced enhancement of iWUE across diverse ecosystems, driven predominantly by Gc reduction rather than GPP stimulation. Moreover, three divergent response types are identified across the studied ecosystems, based on the strength and significance of CO2-driven Gc reduction and GPP enhancement, indicating spatially non-uniform responses to rising CO2. Nutrient supply, water availability and biome types are found to be critical factors regulating this spatial heterogeneity. Overall, our study provides observational insights into ecosystem-scale CO2 fertilization effects. Such understandings are essential to inform terrestrial biosphere models for better projections of carbon and water cycles given the intensified changing climate in a CO2-rich future. A machine learning approach using long-term observations of eddy covariance finds that the increase in plant intrinsic water-use efficiency under higher CO2 levels, across diverse ecosystems, is driven primarily by reduction in canopy conductance, rather than by stimulation of gross primary productivity.},
  archive      = {J_NATEE},
  author       = {Zhan, Weiwei and Lian, Xu and Liu, Jiangong and Han, Jisu and Huang, Yu and Yang, Hao and Zhan, Chunhui and Winkler, Alexander J. and Gentine, Pierre},
  doi          = {10.1038/s41559-025-02761-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1571-1584},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change. <em>NATEE</em>, <em>9</em>(9), 1562-1570. (<a href='https://doi.org/10.1038/s41559-025-02788-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plankton models form the core of marine ecosystem simulators, with uses from regional resource and ecosystem management to climate change projections. In this Perspective, we suggest that stronger alignment of models with empirical knowledge about plankton physiology, diversity and trophic roles will improve model utility and the reliability of their outputs regarding biodiversity, ecophysiology, trophic dynamics and biogeochemistry. We recommend key steps to resolve the disconnect between empirical research and simulation models accounting for well-established plankton processes with an aim to increase the utility of such models for applied uses. A central challenge is characterizing the complexity of plankton diversity and activity in ways that are amenable to model incorporation. We argue that experts in empirical science are best placed to advise the development of next-generation models to address these challenges, and we propose a series of actions to achieve that engagement, including involvement of these experts in the design and exploitation of plankton digital twins. Plankton models have crucial applications in ecosystem management and climate change projections. This Perspective suggests that stronger alignment of plankton models with empirical knowledge is needed and recommends steps to close the gap between empirical research and modelling.},
  archive      = {J_NATEE},
  author       = {Flynn, Kevin J. and Atkinson, Angus and Beardall, John and Berges, John A. and Boersma, Maarten and Brunet, Christophe and Calbet, Albert and Caron, Dave A. and Dam, Hans G. and Glibert, Patricia M. and Hansen, Per Juel and Jin, Peng and Lønborg, Christian and Mayor, Daniel J. and Menden-Deuer, Susanne and Mock, Thomas and Mulholland, Margaret R. and Needham, David M. and Polimene, Luca and Poulton, Alex J. and Robinson, Carol and Rokitta, Sebastian D. and Rost, Björn and Saiz, Enric and Scanlan, David J. and Schmidt, Katrin and Sherr, Evelyn and Stoecker, Diane K. and Svensen, Camilla and Thiele, Stefan and Thingstad, Tron F. and Våge, Selina},
  doi          = {10.1038/s41559-025-02788-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1562-1570},
  shortjournal = {Nat. Ecol. Evol},
  title        = {More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bird sternum evolution facilitated the origins of powered flight. <em>NATEE</em>, <em>9</em>(9), 1560-1561. (<a href='https://doi.org/10.1038/s41559-025-02817-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An in-depth analysis of the breastbone (sternum) of bird ancestors reveals its importance in the origin and evolution of flight in dinosaurs as they evolved into birds.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02817-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1560-1561},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Bird sternum evolution facilitated the origins of powered flight},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus. <em>NATEE</em>, <em>9</em>(9), 1558-1559. (<a href='https://doi.org/10.1038/s41559-025-02799-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geochemical chronologies in surface increments and exposed cross-sections of naturally fractured hominin enamel from the South African sites of Swartkrans and Kromdraai indicate that Paranthropus robustus exploited both forest and grassland habitats, and that individuals did not move on the landscape in a manner analogous to extant African apes.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02799-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1558-1559},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolving a bat wing by reusing an existing gene program. <em>NATEE</em>, <em>9</em>(9), 1556-1557. (<a href='https://doi.org/10.1038/s41559-025-02782-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interspecies comparative analyses of single-cell transcriptomic data reveal that a proximal limb gene program is reactivated during bat wing formation. Our results illustrate how evolutionary innovations can arise from the reutilization of an existing gene set, activated through species-specific regulatory repertoires.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02782-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1556-1557},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolving a bat wing by reusing an existing gene program},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The next generation of colour pattern genomics. <em>NATEE</em>, <em>9</em>(9), 1554-1555. (<a href='https://doi.org/10.1038/s41559-025-02816-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Convolutional neural networks and genetic association analysis decode the evolution of colour pattern diversity and its underlying complex genetic architecture in the Trinidadian guppy.},
  archive      = {J_NATEE},
  author       = {Karagic, Nidal and Kratochwil, Claudius F.},
  doi          = {10.1038/s41559-025-02816-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1554-1555},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The next generation of colour pattern genomics},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sea star wasting disease mystery finally solved. <em>NATEE</em>, <em>9</em>(9), 1552-1553. (<a href='https://doi.org/10.1038/s41559-025-02789-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A decade after a marine epidemic killed off sea stars and triggered ecosystem-wide effects along the Pacific Coast of North America, researchers have identified the bacterial pathogen that is responsible for sea star wasting disease.},
  archive      = {J_NATEE},
  author       = {Lafferty, Kevin D.},
  doi          = {10.1038/s41559-025-02789-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1552-1553},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sea star wasting disease mystery finally solved},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leprosy in the americas before columbus. <em>NATEE</em>, <em>9</em>(9), 1550-1551. (<a href='https://doi.org/10.1038/s41559-025-02790-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis genomes associated with 4,000-year-old human skeletons in Chile establish an American origin for this causal agent of leprosy (also known as Hansen’s disease), and point to different evolutionary trajectories and transmission pathways for M. lepromatosis and its sister pathogen Mycobacterium leprae.},
  archive      = {J_NATEE},
  author       = {Roberts, Charlotte A.},
  doi          = {10.1038/s41559-025-02790-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1550-1551},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Leprosy in the americas before columbus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Water-use efficiency driven by water savings. <em>NATEE</em>, <em>9</em>(9), 1548-1549. (<a href='https://doi.org/10.1038/s41559-025-02793-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of land–atmosphere water and CO2 fluxes suggests that reduced water use by vegetation, rather than increased carbon uptake, is the driving factor behind the well-documented increase in vegetation water-use efficiency in response to rising atmospheric CO2.},
  archive      = {J_NATEE},
  author       = {Knauer, Jürgen},
  doi          = {10.1038/s41559-025-02793-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1548-1549},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Water-use efficiency driven by water savings},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Independent origins of long snouts. <em>NATEE</em>, <em>9</em>(9), 1547. (<a href='https://doi.org/10.1038/s41559-025-02856-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-025-02856-8},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1547},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Independent origins of long snouts},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Seed dispersers wanted. <em>NATEE</em>, <em>9</em>(9), 1546. (<a href='https://doi.org/10.1038/s41559-025-02857-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02857-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1546},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Seed dispersers wanted},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fish without borders. <em>NATEE</em>, <em>9</em>(9), 1545. (<a href='https://doi.org/10.1038/s41559-025-02852-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02852-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1545},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fish without borders},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mines aren’t on the maps. <em>NATEE</em>, <em>9</em>(9), 1544. (<a href='https://doi.org/10.1038/s41559-025-02833-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-025-02833-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1544},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mines aren’t on the maps},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Museums in the age of extinction. <em>NATEE</em>, <em>9</em>(9), 1542-1543. (<a href='https://doi.org/10.1038/s41559-025-02772-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Browning, Claire},
  doi          = {10.1038/s41559-025-02772-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1542-1543},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Museums in the age of extinction},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding environmental impacts of plastic requires a palaeontological lens. <em>NATEE</em>, <em>9</em>(9), 1539-1541. (<a href='https://doi.org/10.1038/s41559-025-02813-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Growing evidence suggests that timescales for plastic degradation have been vastly underestimated. The fossil record of plastic-like biopolymers might provide a perspective on plastic fossilization in deep time.},
  archive      = {J_NATEE},
  author       = {Gabbott, Sarah and Boom, Arnoud and Zalasiewicz, Jan},
  doi          = {10.1038/s41559-025-02813-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1539-1541},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Understanding environmental impacts of plastic requires a palaeontological lens},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Restoration outcomes are compromised by a lack of diverse native seed supply. <em>NATEE</em>, <em>9</em>(9), 1537-1538. (<a href='https://doi.org/10.1038/s41559-025-02825-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Changes in seed banking practices and business incentives are needed to ensure restoration practitioners have access to plentiful and reliable supplies of diverse native seeds.},
  archive      = {J_NATEE},
  author       = {Gallagher, Rachael V.},
  doi          = {10.1038/s41559-025-02825-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1537-1538},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Restoration outcomes are compromised by a lack of diverse native seed supply},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Increase the regulation of biodiversity effects from private equity markets. <em>NATEE</em>, <em>9</em>(9), 1535-1536. (<a href='https://doi.org/10.1038/s41559-025-02828-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Carter, Harrison and Miles, Nell and Hawkins, Isobel and Hulett, Toscanie and zu Ermgassen, Sophus O. S. E.},
  doi          = {10.1038/s41559-025-02828-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1535-1536},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increase the regulation of biodiversity effects from private equity markets},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The amazon’s new oil frontier is a biodiversity bomb in the making. <em>NATEE</em>, <em>9</em>(9), 1532-1534. (<a href='https://doi.org/10.1038/s41559-025-02815-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Paletta, Julia and Callegari, Camila and Oliveira Fiorini, Ana Carolina and Dias, Filipe Gomes and Freitas Junior, Adirson and de Barros, Pedro Henrique Batista},
  doi          = {10.1038/s41559-025-02815-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1532-1534},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The amazon’s new oil frontier is a biodiversity bomb in the making},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sound science. <em>NATEE</em>, <em>9</em>(9), 1531. (<a href='https://doi.org/10.1038/s41559-025-02860-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Passive acoustic monitoring is opening up new opportunities for biodiversity monitoring, but there is still work to do to ensure that it represents a reliable method for biodiversity reporting.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02860-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1531},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sound science},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natmi">NATMI - 21</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires. <em>NATMI</em>, <em>7</em>(9), 1587. (<a href='https://doi.org/10.1038/s42256-025-01128-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Jiang, Yuepeng and Zhang, Pingping and Huo, Miaozhe and Li, Shuai Cheng},
  doi          = {10.1038/s42256-025-01128-1},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1587},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A framework to evaluate machine learning crystal stability predictions. <em>NATMI</em>, <em>7</em>(9), 1586. (<a href='https://doi.org/10.1038/s42256-025-01117-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Riebesell, Janosh and Goodall, Rhys E. A. and Benner, Philipp and Chiang, Yuan and Deng, Bowen and Ceder, Gerbrand and Asta, Mark and Lee, Alpha A. and Jain, Anubhav and Persson, Kristin A.},
  doi          = {10.1038/s42256-025-01117-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1586},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: A framework to evaluate machine learning crystal stability predictions},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanistic understanding and validation of large AI models with SemanticLens. <em>NATMI</em>, <em>7</em>(9), 1572-1585. (<a href='https://doi.org/10.1038/s42256-025-01084-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike human-engineered systems, such as aeroplanes, for which the role and dependencies of each component are well understood, the inner workings of artificial intelligence models remain largely opaque, which hinders verifiability and undermines trust. Current approaches to neural network interpretability, including input attribution methods, probe-based analysis and activation visualization techniques, typically provide limited insights about the role of individual components or require extensive manual interpretation that cannot scale with model complexity. This paper introduces SemanticLens, a universal explanation method for neural networks that maps hidden knowledge encoded by components (for example, individual neurons) into the semantically structured, multimodal space of a foundation model such as CLIP. In this space, unique operations become possible, including (1) textual searches to identify neurons encoding specific concepts, (2) systematic analysis and comparison of model representations, (3) automated labelling of neurons and explanation of their functional roles, and (4) audits to validate decision-making against requirements. Fully scalable and operating without human input, SemanticLens is shown to be effective for debugging and validation, summarizing model knowledge, aligning reasoning with expectations (for example, adherence to the ABCDE rule in melanoma classification) and detecting components tied to spurious correlations and their associated training data. By enabling component-level understanding and validation, the proposed approach helps mitigate the opacity that limits confidence in artificial intelligence systems compared to traditional engineered systems, enabling more reliable deployment in critical applications. SemanticLens is a tool that embeds artificial intelligence model components (such as neurons) into a searchable, human-understandable space. This enables automated auditing, validation of decisions and detection of problematic behaviours with minimal human oversight.},
  archive      = {J_NATMI},
  author       = {Dreyer, Maximilian and Berend, Jim and Labarta, Tobias and Vielhaben, Johanna and Wiegand, Thomas and Lapuschkin, Sebastian and Samek, Wojciech},
  doi          = {10.1038/s42256-025-01084-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1572-1585},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Mechanistic understanding and validation of large AI models with SemanticLens},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning. <em>NATMI</em>, <em>7</em>(9), 1561-1571. (<a href='https://doi.org/10.1038/s42256-025-01098-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence has transformed the field of precise organic synthesis. Data-driven methods, including machine learning and deep learning, have shown great promise in predicting reaction performance and synthesis planning. However, the inherent methodological divergence between numerical regression-driven reaction performance prediction and sequence generation-based synthesis planning creates formidable challenges in constructing a unified deep learning architecture. Here we present RXNGraphormer, a framework to jointly address these tasks through a unified pre-training approach. By synergizing graph neural networks for intramolecular pattern recognition with Transformer-based models for intermolecular interaction modelling, and training on 13 million reactions via a carefully designed strategy, RXNGraphormer achieves state-of-the-art performance across eight benchmark datasets for reactivity or selectivity prediction and forward-synthesis or retrosynthesis planning, as well as three external realistic datasets for reactivity and selectivity prediction. Notably, the model generates chemically meaningful embeddings that spontaneously cluster reactions by type without explicit supervision. This work bridges the critical gap between performance prediction and synthesis planning tasks in chemical AI, offering a versatile tool for accurate reaction prediction and synthesis design. Xu et al. present RXNGraphormer, a pre-trained model that learns bond transformation patterns from over 13 million reactions, achieving state-of-the-art accuracy in reaction performance prediction and synthesis planning.},
  archive      = {J_NATMI},
  author       = {Xu, Li-Cheng and Tang, Miao-Jiong and An, Junyi and Cao, Fenglei and Qi, Yuan},
  doi          = {10.1038/s42256-025-01098-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1561-1571},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces. <em>NATMI</em>, <em>7</em>(9), 1555-1560. (<a href='https://doi.org/10.1038/s42256-025-01091-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular docking plays a crucial role in structure-based drug discovery, enabling the prediction of how small molecules interact with protein targets. Traditional docking methods rely on scoring functions and search heuristics, whereas recent generative approaches, such as DiffDock, leverage deep learning for pose prediction. However, blind-diffusion-based docking often struggles with binding site localization and pose accuracy, particularly in complex protein–ligand systems. This work introduces GeoDirDock (GDD), a guided diffusion approach to molecular docking that enhances the accuracy and physical plausibility of ligand docking predictions. GDD guides the denoising process of a diffusion model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. Our method leverages expert knowledge to direct the generative modelling process, specifically targeting desired protein–ligand interaction regions. We demonstrate that GDD outperforms existing blind docking methods in terms of root mean squared distance accuracy and physicochemical pose realism. Our results indicate that incorporating domain expertise into the diffusion process leads to more biologically relevant docking predictions. Additionally, we explore the potential of GDD as a template-based modelling tool for lead optimization in drug discovery through angle transfer in maximum common substructure docking, showcasing its capability to accurately predict ligand orientations for chemically similar compounds. Future applications in real-world drug discovery campaigns will naturally continue to refine and extend the utility of prior-informed diffusion docking methods. GeoDirDock is a framework that guides the denoising process of a generative diffusion docking model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. This approach enhances the accuracy and physical plausibility of ligand docking predictions.},
  archive      = {J_NATMI},
  author       = {Miñán, Raúl and Gallardo, Javier and Ciudad, Álvaro and Molina, Alexis},
  doi          = {10.1038/s42256-025-01091-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1555-1560},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Error-controlled non-additive interaction discovery in machine learning models. <em>NATMI</em>, <em>7</em>(9), 1541-1554. (<a href='https://doi.org/10.1038/s42256-025-01086-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) models are powerful tools for detecting complex patterns, yet their ‘black-box’ nature limits their interpretability, hindering their use in critical domains like healthcare and finance. Interpretable ML methods aim to explain how features influence model predictions but often focus on univariate feature importance, overlooking complex feature interactions. Although recent efforts extend interpretability to feature interactions, existing approaches struggle with robustness and error control, especially under data perturbations. In this study, we introduce Diamond, a method for trustworthy feature interaction discovery. Diamond uniquely integrates the model-X knockoffs framework to control the false discovery rate, ensuring a low proportion of falsely detected interactions. Diamond includes a non-additivity distillation procedure that refines existing interaction importance measures to isolate non-additive interaction effects and preserve false discovery rate control. This approach addresses the limitations of off-the-shelf interaction measures, which, when used naively, can lead to inaccurate discoveries. Diamond’s applicability spans a broad class of ML models, including deep neural networks, transformers, tree-based models and factorization-based models. Empirical evaluations on both simulated and real datasets across various biomedical studies demonstrate its utility in enabling reliable data-driven scientific discoveries. Diamond represents a significant step forward in leveraging ML for scientific innovation and hypothesis generation. Diamond, a statistically rigorous method, is capable of finding meaningful feature interactions within machine learning models, making black-box models more interpretable for science and medicine.},
  archive      = {J_NATMI},
  author       = {Chen, Winston and Jiang, Yifan and Noble, William Stafford and Lu, Yang Young},
  doi          = {10.1038/s42256-025-01086-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1541-1554},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Error-controlled non-additive interaction discovery in machine learning models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks. <em>NATMI</em>, <em>7</em>(9), 1524-1540. (<a href='https://doi.org/10.1038/s42256-025-01082-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering novel drug candidate molecules is a fundamental step in drug development. Generative deep learning models can sample new molecular structures from learned probability distributions; however, their practical use in drug discovery hinges on generating compounds tailored to a specific target molecule. Here we introduce DrugGEN, an end-to-end generative system for the de novo design of drug candidate molecules that interact with a selected protein. The proposed method represents molecules as graphs and processes them using a generative adversarial network that comprises graph transformer layers. Trained on large datasets of drug-like compounds and target-specific bioactive molecules, DrugGEN designed candidate inhibitors for AKT1, a kinase crucial in many cancers. Docking and molecular dynamics simulations suggest that the generated compounds effectively bind to AKT1, and attention maps provide insights into the model’s reasoning. Furthermore, selected de novo molecules were synthesized and shown to inhibit AKT1 at low micromolar concentrations in the context of in vitro enzymatic assays. These results demonstrate the potential of DrugGEN for designing target-specific molecules. Using the open-access DrugGEN codebase, researchers can retrain the model for other druggable proteins, provided a dataset of known bioactive molecules is available. Inhibiting AKT1 kinase can have potentially positive uses against many types of cancer. To find novel molecules targeting this protein, a graph adversarial network is trained as a generative model.},
  archive      = {J_NATMI},
  author       = {Ünlü, Atabey and Çevrim, Elif and Yiğit, Melih Gökay and Sarıgün, Ahmet and Çelikbilek, Hayriye and Bayram, Osman and Kahraman, Deniz Cansen and Olğaç, Abdurrahman and Rifaioglu, Ahmet Sureyya and Banoğlu, Erden and Doğan, Tunca},
  doi          = {10.1038/s42256-025-01082-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1524-1540},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Brain–computer interface control with artificial intelligence copilots. <em>NATMI</em>, <em>7</em>(9), 1510-1523. (<a href='https://doi.org/10.1038/s42256-025-01090-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motor brain–computer interfaces (BCIs) decode neural signals to help people with paralysis move and communicate. Even with important advances in the past two decades, BCIs face a key obstacle to clinical viability: BCI performance should strongly outweigh costs and risks. To significantly increase the BCI performance, we use shared autonomy, where artificial intelligence (AI) copilots collaborate with BCI users to achieve task goals. We demonstrate this AI-BCI in a non-invasive BCI system decoding electroencephalography signals. We first contribute a hybrid adaptive decoding approach using a convolutional neural network and ReFIT-like Kalman filter, enabling healthy users and a participant with paralysis to control computer cursors and robotic arms via decoded electroencephalography signals. We then design two AI copilots to aid BCI users in a cursor control task and a robotic arm pick-and-place task. We demonstrate AI-BCIs that enable a participant with paralysis to achieve 3.9-times-higher performance in target hit rate during cursor control and control a robotic arm to sequentially move random blocks to random locations, a task they could not do without an AI copilot. As AI copilots improve, BCIs designed with shared autonomy may achieve higher performance. AI copilots are integrated into brain–computer interfaces, enabling a paralysed participant to achieve improved control of computer cursors and robotic arms. This shared autonomy approach offers a promising path to increase BCI performance and clinical viability.},
  archive      = {J_NATMI},
  author       = {Lee, Johannes Y. and Lee, Sangjoon and Mishra, Abhishek and Yan, Xu and McMahan, Brandon and Gaisford, Brent and Kobashigawa, Charles and Qu, Mike and Xie, Chang and Kao, Jonathan C.},
  doi          = {10.1038/s42256-025-01090-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1510-1523},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Brain–computer interface control with artificial intelligence copilots},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Conditional generation of real antigen-specific t cell receptor sequences. <em>NATMI</em>, <em>7</em>(9), 1494-1509. (<a href='https://doi.org/10.1038/s42256-025-01096-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent advances in T cell receptor (TCR) engineering, designing functional TCRs against arbitrary targets remains challenging due to complex rules governing cross-reactivity and limited paired data. Here we present TCR-TRANSLATE, a sequence-to-sequence framework that adapts low-resource machine translation techniques to generate antigen-specific TCR sequences against unseen epitopes. By evaluating 12 model variants of the BART and T5 model architectures, we identified key factors affecting performance and utility, revealing discordances between these objectives. Our flagship model, TCRT5, outperforms existing approaches on computational benchmarks, prioritizing functionally relevant sequences at higher ranks. Most significantly, we experimentally validated a computationally designed TCR against Wilms’ tumour antigen, a therapeutically relevant target in leukaemia, excluded from our training and validation sets. Although the identified TCR shows cross-reactivity with pathogen-derived peptides, highlighting limitations in specificity, our work represents the successful computational design of a functional TCR construct against a non-viral epitope from the target sequence alone. Our findings establish a foundation for computational TCR design and reveal current limitations in data availability and methodology, providing a framework for accelerating personalized immunotherapy by reducing the search space for novel targets. TCR-TRANSLATE, a deep learning framework adapting machine translation to immune design, demonstrates the successful generation of a functional T cell receptor sequence for a cancer epitope from the target sequence alone.},
  archive      = {J_NATMI},
  author       = {Karthikeyan, Dhuvarakesh and Bennett, Sarah N. and Reynolds, Amy G. and Vincent, Benjamin G. and Rubinsteyn, Alex},
  doi          = {10.1038/s42256-025-01096-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1494-1509},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Conditional generation of real antigen-specific t cell receptor sequences},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modelling neural coding in the auditory midbrain with high resolution and accuracy. <em>NATMI</em>, <em>7</em>(9), 1478-1493. (<a href='https://doi.org/10.1038/s42256-025-01104-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational models of auditory processing can be valuable tools for research and technology development. Models of the cochlea are highly accurate and widely used, but models of the auditory brain lag far behind in both performance and penetration. Here we present ICNet, a convolutional encoder–decoder model of neural coding in the inferior colliculus. We developed ICNet using large-scale intracranial recordings from anaesthetized gerbils, addressing three key modelling challenges that are common across all sensory systems: capturing the full statistical structure of neuronal response patterns; accounting for physiological and experimental non-stationarity; and extracting features of sensory processing that are shared across different brains. ICNet provides highly accurate simulation of multi-unit neural responses to a wide range of complex sounds, including near-perfect responses to speech. It also reproduces key neurophysiological phenomena such as forward masking and dynamic range adaptation. ICNet can be used to simulate activity from thousands of neural units or to provide a compact representation of early central auditory processing through its latent dynamics, facilitating a wide range of hearing and audio applications. It can also serve as a foundation core, providing a baseline neural representation for models of active listening or higher-level auditory processing. Drakopoulos et al. present a model that captures the transformation from sound waves to neural activity patterns underlying early auditory processing. The model reproduces neural responses to a range of complex sounds and key neurophysiological phenomena.},
  archive      = {J_NATMI},
  author       = {Drakopoulos, Fotios and Pellatt, Lloyd and Sabesan, Shievanie and Xia, Yiqing and Fragner, Andreas and Lesica, Nicholas A.},
  doi          = {10.1038/s42256-025-01104-9},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1478-1493},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Modelling neural coding in the auditory midbrain with high resolution and accuracy},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping. <em>NATMI</em>, <em>7</em>(9), 1461-1477. (<a href='https://doi.org/10.1038/s42256-025-01099-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) deciphers genome-wide chromatin accessibility, providing profound insights into gene regulation mechanisms. With the rapid advance of sequencing technologies, scATAC-seq data typically encompass numerous samples from various conditions, resulting in complex batch effects, thus necessitating reliable integration tools. While numerous batch integration tools exist for single-cell RNA sequencing data, inherent data characteristic differences limit their effectiveness on scATAC-seq data. Existing integration methods for scATAC-seq data suffer from several fundamental limitations, such as disrupting the biological heterogeneity and focusing solely on low-dimensional correction, which may distort data and hinder downstream analysis. Here we propose Fountain, a deep learning framework for scATAC-seq data integration via rigorous barycentric mapping. Barycentric mapping transforms one data distribution to another in a principled and effective manner through optimal transport. By regularizing barycentric mapping with geometric data information, Fountain achieves accurate batch alignment while preserving biological heterogeneity. Comprehensive experiments across diverse real-world datasets demonstrate the advantages of Fountain over existing methods in batch correction and biological conservation. In addition, the trained Fountain model can integrate data from new batches alongside already integrated data without retraining, enabling continuous online data integration. Moreover, Fountain’s reconstruction strategy generates batch-corrected ATAC profiles, improving the capture of cellular heterogeneity and revealing cell-type-specific implications such as expression enrichment analysis and partitioned heritability analysis. Zhu, Hua and Chen propose Fountain, a deep learning framework for batch integration of scATAC-seq data that utilizes regularized barycentric mapping. It preserves biological heterogeneity, enabling online and original dimensionality integration.},
  archive      = {J_NATMI},
  author       = {Zhu, Shuchen and Hua, Heyang and Chen, Shengquan},
  doi          = {10.1038/s42256-025-01099-3},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1461-1477},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerating protein engineering with fitness landscape modelling and reinforcement learning. <em>NATMI</em>, <em>7</em>(9), 1446-1460. (<a href='https://doi.org/10.1038/s42256-025-01103-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein engineering holds substantial promise for designing proteins with customized functions, yet the vast landscape of potential mutations versus limited laboratory capacity constrains the discovery of optimal sequences. Here, to address this, we present the μProtein framework, which accelerates protein engineering by combining μFormer, a deep learning model for accurate mutational effect prediction, with μSearch, a reinforcement learning algorithm designed to efficiently navigate the protein fitness landscape using μFormer as an oracle. μProtein leverages single-mutation data to predict optimal sequences with complex, multi-amino-acid mutations through its modelling of epistatic interactions and a multi-step search strategy. In addition to strong performance on benchmark datasets, μProtein identified high-gain-of-function multi-point mutants for the enzyme β-lactamase, surpassing one of the highest-known activity levels, in wet laboratory, trained solely on single-mutation data. These results demonstrate μProtein’s capability to discover impactful mutations across the vast protein sequence space, offering a robust, efficient approach for protein optimization. μProtein, combining deep learning and reinforcement learning, is developed to design high-function proteins. This framework, trained only on single-mutation data, discovers multi-site β-lactamase mutants with up to 2,000× growth rates.},
  archive      = {J_NATMI},
  author       = {Sun, Haoran and He, Liang and Deng, Pan and Liu, Guoqing and Zhao, Zhiyu and Jiang, Yuliang and Cao, Chuan and Ju, Fusong and Wu, Lijun and Liu, Haiguang and Qin, Tao and Liu, Tie-Yan},
  doi          = {10.1038/s42256-025-01103-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1446-1460},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accelerating protein engineering with fitness landscape modelling and reinforcement learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient protein structure generation with sparse denoising models. <em>NATMI</em>, <em>7</em>(9), 1429-1445. (<a href='https://doi.org/10.1038/s42256-025-01100-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteins play diverse roles in all domains of life and are extensively harnessed as biomolecules in biotechnology, with applications spanning from fundamental research to biomedicine. Therefore, there is considerable interest in computationally designing proteins with specified properties. Protein structure generative models provide a means to design protein structures in a controllable manner and have been successfully applied to address various protein design tasks. Such models are paired with protein sequence and structure predictors to produce and select protein sequences for experimental testing. However, current protein structure generators face important limitations for proteins with more than 400 amino acids and require retraining for protein design tasks unseen during model training. To address the first issue, we introduce salad, a family of sparse all-atom denoising models for protein structure generation. Our models are smaller and faster than the state of the art and matching or improving design quality, successfully generating structures for protein lengths up to 1,000 amino acids. To address the second issue, we combine salad with structure editing, a sampling strategy for expanding the capability of protein denoising models to unseen tasks. We apply our approach to a variety of challenging protein design tasks, from generating protein scaffolds containing functional protein motifs (motif scaffolding) to designing proteins capable of adopting multiple distinct folds under different conditions (multi-state protein design), demonstrating the flexibility of salad and structure editing. A small and fast diffusion model is presented, which is able to efficiently generate long protein backbones.},
  archive      = {J_NATMI},
  author       = {Jendrusch, Michael and Korbel, Jan O.},
  doi          = {10.1038/s42256-025-01100-z},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1429-1445},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Efficient protein structure generation with sparse denoising models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics. <em>NATMI</em>, <em>7</em>(9), 1414-1428. (<a href='https://doi.org/10.1038/s42256-025-01097-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Self-supervised learning (SSL) has emerged as a powerful approach for learning meaningful representations from large-scale unlabelled datasets in single-cell genomics. Richter et al. evaluated SSL pretext tasks on modelling single-cell RNA sequencing (scRNA-seq) data, demonstrating the effective use of SSL models. However, the transferability of these pretrained SSL models to the spatial transcriptomics domain remains unexplored. Here we assess the performance of three SSL models (random mask, gene programme mask and Barlow Twins) pretrained on scRNA-seq data with spatial transcriptomics datasets, focusing on cell-type prediction and spatial clustering. Our experiments demonstrate that the SSL model with random mask strategy exhibits the best overall performance among evaluated SSL models. Moreover, the models trained from scratch on spatial transcriptomics data outperform the fine-tuned SSL models on cell-type prediction, highlighting a domain gap between scRNA-seq and spatial transcriptomics data whose underlying causes remain an open question. Through expanded analyses of multiple imputation methods and data degradation scenarios, we demonstrate that gene imputation would degrade SSL model performance on cell-type prediction, an effect that is exacerbated by increasing data sparsity. Finally, integrating zero-shot random mask embeddings into chosen spatial clustering methods significantly enhanced their accuracy. Overall, our findings provide valuable insights into the limitations and potential of transferring SSL models to spatial transcriptomics and offer practical guidance for researchers leveraging pretrained models for spatial transcriptomics data analysis. Self-supervised learning models for single-cell RNA sequencing data exhibit poor transferability to spatial transcriptomics for cell-type prediction, although their learned features may enhance spatial analysis.},
  archive      = {J_NATMI},
  author       = {Han, Chuangyi and Lin, Senlin and Wang, Zhikang and Cui, Yan and Zou, Qi and Yuan, Zhiyuan},
  doi          = {10.1038/s42256-025-01097-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1414-1428},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosting the predictive power of protein representations with a corpus of text annotations. <em>NATMI</em>, <em>7</em>(9), 1403-1413. (<a href='https://doi.org/10.1038/s42256-025-01088-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models are trained to predict amino acid sequences from vast protein databases and learn to represent proteins as feature vectors. These vector representations have enabled impressive applications, from predicting mutation effects to protein folding. One of the reasons offered for the success of these models is that conserved sequence motifs tend to be important for protein fitness. Yet, the relationship between sequence conservation and fitness can be confounded by the evolutionary and environmental context. Should we, therefore, look to other data sources that may contain more direct functional information? In this work, we conduct a comprehensive study examining the effects of training protein models to predict 19 types of text annotation from UniProt. Our results show that fine-tuning protein models on a subset of these annotations enhances the models’ predictive capabilities on a variety of function prediction tasks. In particular, when evaluated on our tasks, our model outperforms the basic local alignment search tool, which none of the pretrained protein models accomplished. Our results suggest that a much wider array of data modalities, such as text annotations, may be tapped to improve protein language models. Although protein language models have enabled major advances, they often rely on indirect signals that may not fully capture functional relevance. Fine-tuning these models on textual annotations is shown to improve their performance on function prediction tasks.},
  archive      = {J_NATMI},
  author       = {Duan, Haonan and Skreta, Marta and Cotta, Leonardo and Rajaonson, Ella Miray and Dhawan, Nikita and Aspuru-Guzik, Alán and Maddison, Chris J.},
  doi          = {10.1038/s42256-025-01088-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1403-1413},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Boosting the predictive power of protein representations with a corpus of text annotations},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury. <em>NATMI</em>, <em>7</em>(9), 1390-1402. (<a href='https://doi.org/10.1038/s42256-025-01105-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spinal cord injury (SCI) disrupts neuromuscular control, severely affecting independence and quality of life. Although upper limb wearable robots hold considerable promise for functional restoration, most existing prototypes have been validated minimally in people with SCI and target almost exclusively hand opening and closing. We introduce a lightweight, modular assistive soft exosuit that simultaneously and automatically supports shoulder abduction and elbow flexion or extension movements using lightweight fabric-based pneumatic actuators, controlled through inertial sensors. The individual elbow modules were first validated in 11 healthy volunteers, and subsequently tested, together with the shoulder module, in 15 individuals with cervical SCI (C4–C7, AIS A–D). In the SCI participants, exosuits assistance resulted in increased static endurance time (by more than 250%), and lower activity of the primary muscles involved in dynamic tasks (by up to 50%). The two SCI participants retaining prehensile capability also improved their scores in the box and block test when assisted. Moreover, the soft actuation provided a safe, comfortable and easy-to-use solution that was positively appreciated by the participants. Collectively, these results provide encouraging evidence that exosuits can augment upper limb motor performance, and may ultimately translate into greater functional independence and quality of life for the SCI population. A lightweight, modular assistive soft exosuit is introduced, which supports shoulder and elbow movement in individuals with cervical spinal cord injury. The device enhances endurance and range of motion, reduces muscle effort and improves clinical test scores.},
  archive      = {J_NATMI},
  author       = {Ferroni, Roberto and D’Avola, Gaetano and Sciarrone, Giorgia and Righi, Gabriele and De Santis, Claudia and Carpaneto, Jacopo and Gandolla, Marta and Del Popolo, Giulio and Micera, Silvestro and Proietti, Tommaso},
  doi          = {10.1038/s42256-025-01105-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1390-1402},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Aligning generalization between humans and machines. <em>NATMI</em>, <em>7</em>(9), 1378-1389. (<a href='https://doi.org/10.1038/s42256-025-01109-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in artificial intelligence (AI)—including generative approaches—have resulted in technology that can support humans in scientific discovery and forming decisions, but may also disrupt democracies and target individuals. The responsible use of AI and its participation in human–AI teams increasingly shows the need for AI alignment, that is, to make AI systems act according to our preferences. A crucial yet often overlooked aspect of these interactions is the different ways in which humans and machines generalize. In cognitive science, human generalization commonly involves abstraction and concept learning. By contrast, AI generalization encompasses out-of-domain generalization in machine learning, rule-based reasoning in symbolic AI, and abstraction in neurosymbolic AI. Here we combine insights from AI and cognitive science to identify key commonalities and differences across three dimensions: notions of, methods for, and evaluation of generalization. We map the different conceptualizations of generalization in AI and cognitive science along these three dimensions and consider their role for alignment in human–AI teaming. This results in interdisciplinary challenges across AI and cognitive science that must be tackled to support effective and cognitively supported alignment in human–AI teaming scenarios. Ilievski et al. examine differences and similarities in the various ways human and AI systems generalize. The insights are important for effectively supporting alignment in human–AI teams.},
  archive      = {J_NATMI},
  author       = {Ilievski, Filip and Hammer, Barbara and van Harmelen, Frank and Paassen, Benjamin and Saralajew, Sascha and Schmid, Ute and Biehl, Michael and Bolognesi, Marianna and Dong, Xin Luna and Gashteovski, Kiril and Hitzler, Pascal and Marra, Giuseppe and Minervini, Pasquale and Mundt, Martin and Ngomo, Axel-Cyrille Ngonga and Oltramari, Alessandro and Pasi, Gabriella and Saribatur, Zeynep G. and Serafini, Luciano and Shawe-Taylor, John and Shwartz, Vered and Skitalinskaya, Gabriella and Stachl, Clemens and van de Ven, Gido M. and Villmann, Thomas},
  doi          = {10.1038/s42256-025-01109-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1378-1389},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Aligning generalization between humans and machines},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Real-world validation of a structure-aware pipeline for molecular design. <em>NATMI</em>, <em>7</em>(9), 1376-1377. (<a href='https://doi.org/10.1038/s42256-025-01102-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The next major challenge for artificial intelligence in drug development lies in proving its value in real-world settings. A new technology not only supports the generation of novel chemical entities but also accelerates a range of real-world molecular design tasks.},
  archive      = {J_NATMI},
  author       = {Dias, Ana Laura and Rodrigues, Tiago},
  doi          = {10.1038/s42256-025-01102-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1376-1377},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Real-world validation of a structure-aware pipeline for molecular design},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards agentic science for advancing scientific discovery. <em>NATMI</em>, <em>7</em>(9), 1373-1375. (<a href='https://doi.org/10.1038/s42256-025-01110-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence is transforming scientific discovery through (semi-)autonomous agents capable of reasoning, planning, and interacting with digital and physical environments. This Comment explores the foundations and frontiers of agentic science, outlining its emerging directions, current limitations, and the pathways for responsible integration into scientific practice.},
  archive      = {J_NATMI},
  author       = {Xin, Hongliang and Kitchin, John R. and Kulik, Heather J.},
  doi          = {10.1038/s42256-025-01110-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1373-1375},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards agentic science for advancing scientific discovery},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LLMs as all-in-one tools to easily generate publication-ready citation diversity reports. <em>NATMI</em>, <em>7</em>(9), 1371-1372. (<a href='https://doi.org/10.1038/s42256-025-01101-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Cantú, Melissa S. and King, Michael R.},
  doi          = {10.1038/s42256-025-01101-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1371-1372},
  shortjournal = {Nat. Mach. Intell.},
  title        = {LLMs as all-in-one tools to easily generate publication-ready citation diversity reports},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Applying genomic AI to combat antibiotic resistance in low-income countries. <em>NATMI</em>, <em>7</em>(9), 1369-1370. (<a href='https://doi.org/10.1038/s42256-025-01108-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Aruhomukama, Dickson},
  doi          = {10.1038/s42256-025-01108-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1369-1370},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Applying genomic AI to combat antibiotic resistance in low-income countries},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nature">NATURE - 64</h2>
<ul>
<li><details>
<summary>
(2025). Editorial expression of concern: Recovery of learning and memory is associated with chromatin remodelling. <em>NATURE</em>, <em>645</em>(8082), E13. (<a href='https://doi.org/10.1038/s41586-025-09590-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Fischer, Andre and Sananbenesi, Farahnaz and Wang, Xinyu and Dobbin, Matthew and Tsai, Li-Huei},
  doi          = {10.1038/s41586-025-09590-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {E13},
  shortjournal = {Nature},
  title        = {Editorial expression of concern: Recovery of learning and memory is associated with chromatin remodelling},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: In situ light-field imaging of octopus locomotion reveals simplified control. <em>NATURE</em>, <em>645</em>(8082), E12. (<a href='https://doi.org/10.1038/s41586-025-09512-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Katija, Kakani and Huffard, Christine L. and Roberts, Paul L. D. and Daniels, Joost and Erickson, Jon and Klimov, Denis and Ruhl, Henry A. and Sherman, Alana D.},
  doi          = {10.1038/s41586-025-09512-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {E12},
  shortjournal = {Nature},
  title        = {Publisher correction: In situ light-field imaging of octopus locomotion reveals simplified control},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A broad-spectrum lasso peptide antibiotic targeting the bacterial ribosome. <em>NATURE</em>, <em>645</em>(8082), E11. (<a href='https://doi.org/10.1038/s41586-025-09597-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Jangra, Manoj and Travin, Dmitrii Y. and Aleksandrova, Elena V. and Kaur, Manpreet and Darwish, Lena and Koteva, Kalinka and Klepacki, Dorota and Wang, Wenliang and Tiffany, Maya and Sokaribo, Akosiererem and Chen, Xuefei and Deng, Zixin and Tao, Meifeng and Coombes, Brian K. and Vázquez-Laslop, Nora and Polikanov, Yury S. and Mankin, Alexander S. and Wright, Gerard D.},
  doi          = {10.1038/s41586-025-09597-5},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {E11},
  shortjournal = {Nature},
  title        = {Author correction: A broad-spectrum lasso peptide antibiotic targeting the bacterial ribosome},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: PPP2R1A mutations portend improved survival after cancer immunotherapy. <em>NATURE</em>, <em>645</em>(8082), E10. (<a href='https://doi.org/10.1038/s41586-025-09586-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Dai, Yibo and Knisely, Anne and Yano, Mitsutake and Dang, Minghao and Hinchcliff, Emily M. and Lee, Sanghoon and Welp, Annalyn and Chelvanambi, Manoj and Lastrapes, Matthew and Liu, Heng and Yuan, Zhe and Wang, Chen and Nie, Hao and Jean, Stephanie and Montaner, Luis J. and Hou, Jiakai and Patel, Ami and Patel, Shrina and Fellman, Bryan and Yuan, Ying and Sun, Baohua and Pandurengan, Renganayaki Krishna and Cuentas, Edwin Roger Parra and Celestino, Joseph and Liu, Yan and Liu, Jinsong and Hillman, R. Tyler and Westin, Shannon N. and Sood, Anil K. and Soliman, Pamela T. and Shafer, Aaron and Meyer, Larissa A. and Gershenson, David M. and Vining, David and Ganeshan, Dhakshinamoorthy and Lu, Karen and Wargo, Jennifer A. and Peng, Weiyi and Zhang, Rugang and Wang, Linghua and Jazaeri, Amir A.},
  doi          = {10.1038/s41586-025-09586-8},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {E10},
  shortjournal = {Nature},
  title        = {Author correction: PPP2R1A mutations portend improved survival after cancer immunotherapy},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). I deeply scan timber to reduce wasting this precious resource. <em>NATURE</em>, <em>645</em>(8082), 1098. (<a href='https://doi.org/10.1038/d41586-025-03044-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Johannes Huber builds an understanding of logs using medical-imaging technology, to improve future wood-harvesting methods. Johannes Huber builds an understanding of logs using medical-imaging technology, to improve future wood-harvesting methods.},
  archive      = {J_NATURE},
  author       = {Dimitropoulos, Stav},
  doi          = {10.1038/d41586-025-03044-1},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1098},
  shortjournal = {Nature},
  title        = {I deeply scan timber to reduce wasting this precious resource},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Real-time molecular recorders expose the inner lives of cells. <em>NATURE</em>, <em>645</em>(8082), 1094-1096. (<a href='https://doi.org/10.1038/d41586-025-03035-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene-editing tools transform cells into record-keepers, capturing hidden timelines of health and disease. Gene-editing tools transform cells into record-keepers, capturing hidden timelines of health and disease.},
  archive      = {J_NATURE},
  author       = {Dolgin, Elie},
  doi          = {10.1038/d41586-025-03035-2},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1094-1096},
  shortjournal = {Nature},
  title        = {Real-time molecular recorders expose the inner lives of cells},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). No lectures, exams, essays: Inside a twenty-first-century university. <em>NATURE</em>, <em>645</em>(8082), 1091-1093. (<a href='https://doi.org/10.1038/d41586-025-03034-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Some innovative higher-education institutions are reimagining pedagogy by prioritizing local needs over research and international student recruitment. Anna McKie visits one of them. Some innovative higher-education institutions are reimagining pedagogy by prioritizing local needs over research and international student recruitment. Anna McKie visits one of them.},
  archive      = {J_NATURE},
  author       = {McKie, Anna},
  doi          = {10.1038/d41586-025-03034-3},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1091-1093},
  shortjournal = {Nature},
  title        = {No lectures, exams, essays: Inside a twenty-first-century university},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural basis of fast N-type inactivation in kv channels. <em>NATURE</em>, <em>645</em>(8082), 1081-1089. (<a href='https://doi.org/10.1038/s41586-025-09339-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Action potentials are generated by opening of voltage-activated sodium (Nav) and potassium (Kv) channels1, which can rapidly inactivate to shape the nerve impulse and contribute to synaptic facilitation and short-term memory1–4. The mechanism of fast inactivation was proposed to involve an intracellular domain that blocks the internal pore in both Nav5,6 and Kv7–9 channels; however, recent studies in Nav10,11 and Kv12,13 channels support a mechanism in which the internal pore closes during inactivation. Here we investigate the mechanism of fast inactivation in the Shaker Kv channel using cryo-electron microscopy, mass spectrometry and electrophysiology. We resolved structures of a fully inactivated state in which the non-polar end of the N terminus plugs the internal pore in an extended conformation. The N-terminal methionine is deleted, leaving an alanine that is acetylated and interacts with a pore-lining isoleucine residue where RNA editing regulates fast inactivation14. Opening of the internal activation gate is required for fast inactivation because it enables the plug domain to block the pore and repositions gate residues to interact with and stabilize that domain. We also show that external K+ destabilizes the inactivated state by altering the conformation of the ion selectivity filter rather than by electrostatic repulsion. These findings establish the mechanism of fast inactivation in Kv channels, revealing how it is regulated by RNA editing and N-terminal acetylation, and providing a framework for understanding related mechanisms in other voltage-activated channels. This paper explores how voltage-gated potassium channels can plug the pore to prevent the conductance of ions during inactivation.},
  archive      = {J_NATURE},
  author       = {Tan, Xiao-Feng and Fernández-Mariño, Ana I. and Li, Yan and Chang, Tsg-Hui and Swartz, Kenton J.},
  doi          = {10.1038/s41586-025-09339-7},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1081-1089},
  shortjournal = {Nature},
  title        = {Structural basis of fast N-type inactivation in kv channels},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). NASP modulates histone turnover to drive PARP inhibitor resistance. <em>NATURE</em>, <em>645</em>(8082), 1071-1080. (<a href='https://doi.org/10.1038/s41586-025-09414-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The&nbsp;poly(ADP-ribose) polymerase inhibitor (PARPi) class of drugs&nbsp;represents a remarkable advance in the treatment of patients with homologous recombination-deficient tumours, but resistance remains a challenge1–5. Although most research has focused on the downstream consequences of PARPi exposure to tackle resistance, the immediate effect of PARP inhibition on the chromatin environment and its contribution to PARPi toxicity remains elusive. Here we show that PARP inhibition induces histone release from the chromatin. This presents a vulnerability of PARPi-resistant cancer cells, which require histone homeostasis mechanisms to sustain elevated DNA replication rates and survival. Through functional genetic screens, we identified NASP as a key factor in maintaining the stability of evicted histones via its TPR motifs. Loss of NASP renders tumour cells hypersensitive to PARPi treatment in vitro and in vivo, impairs replication fork progression and elevates levels of replication-associated DNA damage. Moreover, NASP acts together with the INO80 complex and the chaperoning activity of PARP1 to ensure efficient histone turnover and prevent the accumulation of lethal DNA damage. Collectively, our work reports on histone eviction as an immediate cellular response to PARPi treatment and provides a promising avenue for targeting histone supply pathways to overcome PARPi resistance. PARP inhibitor treatment triggers&nbsp;histone release from the chromatin in cancer cells; consequently, targeting the histone chaperone NASP renders cells vulnerable to PARP inhibition.},
  archive      = {J_NATURE},
  author       = {Moser, Sarah C. and Khalizieva, Anna and Roehsner, Josef and Pottendorfer, Elisabeth and Kaptein, Milo L. and Ricci, Giulia and Bhardwaj, Vivek and Bleijerveld, Onno B. and Hoekman, Liesbeth and van der Heijden, Ingrid and di Sanzo, Simone and Fish, Alexander and Chikunova, Aleksandra and Haarhuis, Judith H. I. and Oldenkamp, Roel and Robbez-Masson, Luisa and Sprengers, Justin and Vis, Daniel J. and Wessels, Lodewyk F. A. and van de Ven, Marieke and Pettitt, Stephen J. and Tutt, Andrew N. J. and Lord, Christopher J. and Rowland, Benjamin D. and Völker-Albert, Moritz and Mattiroli, Francesca and Brummelkamp, Thijn R. and Mazouzi, Abdelghani and Jonkers, Jos},
  doi          = {10.1038/s41586-025-09414-z},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1071-1080},
  shortjournal = {Nature},
  title        = {NASP modulates histone turnover to drive PARP inhibitor resistance},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Elementary 3D organization of active and silenced e. coli genome. <em>NATURE</em>, <em>645</em>(8082), 1060-1070. (<a href='https://doi.org/10.1038/s41586-025-09396-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unravelling how genomes are spatially organized and how their three-dimensional (3D) architecture drives cellular functions remains a major challenge in biology1,2. In bacteria, genomic DNA is compacted into a highly ordered, condensed state called nucleoid3–5. Despite progress in characterizing bacterial 3D genome architecture over recent decades6–8, the fine structure and functional organization of the nucleoid remain elusive due to low-resolution contact maps from methods such as Hi-C9–11. Here we developed an enhanced Micro-C chromosome conformation capture, achieving 10-base&nbsp;pair (bp) resolution. This ultra-high-resolution analysis reveals elemental spatial structures in the Escherichia coli nucleoid, including chromosomal hairpins (CHINs) and chromosomal hairpin domains (CHIDs). These structures, organized by histone-like proteins H-NS and StpA, have key roles in repressing horizontally transferred genes. Disruption of H-NS causes drastic reorganization of the 3D genome, decreasing CHINs and CHIDs, whereas removing both H-NS and StpA results in their complete disassembly, increased transcription of horizontally transferred genes and delayed growth. Similar effects are observed with netropsin, which competes with H-NS and StpA for AT-rich DNA binding. Interactions between CHINs further organize the genome into isolated loops, potentially insulating active operons. Our Micro-C analysis reveals that all actively transcribed genes form distinct operon-sized chromosomal interaction domains (OPCIDs) in a transcription-dependent manner. These structures appear as square patterns on Micro-C maps, reflecting continuous contacts throughout transcribed regions. This work unveils the fundamental structural elements of the E. coli nucleoid, highlighting their connection to nucleoid-associated proteins and transcription machinery. An ultra-high-resolution chromatin organization map of E. coli, using Micro-C, reveals intricate chromatin structures involved in the silencing of horizontally transferred genes and those associated with active operons.},
  archive      = {J_NATURE},
  author       = {Gavrilov, Alexey A. and Shamovsky, Ilya and Zhegalova, Irina and Proshkin, Sergey and Shamovsky, Yosef and Evko, Grigory and Epshtein, Vitaly and Rasouly, Aviram and Blavatnik, Anna and Lahiri, Sudipta and Rothenberg, Eli and Razin, Sergey V. and Nudler, Evgeny},
  doi          = {10.1038/s41586-025-09396-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1060-1070},
  shortjournal = {Nature},
  title        = {Elementary 3D organization of active and silenced e. coli genome},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). NNMT inhibition in cancer-associated fibroblasts restores antitumour immunity. <em>NATURE</em>, <em>645</em>(8082), 1051-1059. (<a href='https://doi.org/10.1038/s41586-025-09303-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer-associated fibroblasts (CAFs) have a pivotal cancer-supportive role, yet CAF-targeted therapies are lacking1,2. Here, using spatial transcriptomics and single-cell RNA sequencing, we investigate the role of nicotinamide N-methyltransferase (NNMT) in high-grade serous ovarian cancer. Mechanistically, NNMT-induced H3K27me3 hypomethylation drives complement secretion from CAFs, attracting immunosuppressive myeloid-derived suppressor cells (MDSCs) to the tumour. Nnmt knockout in immunocompetent mice impairs tumour growth in syngeneic ovarian, breast and colon tumour models through enhanced CD8+ T cell activation. Using high-throughput screening, we develop a potent and specific NNMT inhibitor that reduces the tumour burden and metastasis in multiple mouse cancer models and restores immune checkpoint blockade efficacy by decreasing CAF-mediated recruitment of MDSCs and reinvigorating CD8+ T cell activation. Our findings establish NNMT as a central CAF regulator and a promising therapeutic target to mitigate immunosuppression in the tumour microenvironment. An NNMT inhibitor reduces tumour burden and metastasis in multiple mouse cancer models and restores immune checkpoint blockade efficacy by decreasing cancer-associated-fibroblast-mediated recruitment of myeloid-derived suppressor cells and reinvigorating CD8+ T cell activation.},
  archive      = {J_NATURE},
  author       = {Heide, Janna and Bilecz, Agnes J. and Patnaik, Samarjit and Allega, Maria Francesca and Donle, Leonhard and Yang, Kaiting and Teich, Ethan and Li, Yan and Lin, Qiaoshan and Kong, Ke and Liu, Li and Yang, Tae Gyun and Cheng, Ken Chih-Chien and Shrimp, Jonathan H. and Hanson, Quinlin M. and Shen, Min and Sun, Hongmao and Shah, Hardik and Schweizer, Lisa and Zawieracz, Katarzyna and Olland, Andrea and White, Andre and Suto, Robert K. and Alhunayan, Razzaq and Taşdemir, Medine and Longman, Noa and Liang, Hua and Mann, Matthias and Stott, Gordon M. and Hall, Matthew D. and Schwörer, Simon and Weichselbaum, Ralph R. and Piffkó, András and Lengyel, Ernst},
  doi          = {10.1038/s41586-025-09303-5},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1051-1059},
  shortjournal = {Nature},
  title        = {NNMT inhibition in cancer-associated fibroblasts restores antitumour immunity},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Expanding the cytokine receptor alphabet reprograms t cells into diverse states. <em>NATURE</em>, <em>645</em>(8082), 1039-1050. (<a href='https://doi.org/10.1038/s41586-025-09393-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells respond to cytokines through receptor dimers that have been selected over the course of evolution to activate canonical JAK–STAT signalling and gene expression programs1. However, the potential combinatorial diversity of JAK–STAT receptor pairings can be expanded by exploring the untapped biology of alternative non-natural pairings. Here we exploited the common γ chain (γc) receptor as a shared signalling hub on T cells and enforced the expression of both natural and non-natural heterodimeric JAK–STAT receptor pairings using an orthogonal cytokine receptor platform2–4 to expand the γc signalling code. We tested receptors from γc cytokines as well as interferon, IL-10 and homodimeric receptor families that do not normally pair with γc or are not naturally expressed on T cells. These receptors simulated their natural counterparts but also induced contextually unique transcriptional programs. This led to distinct T cell fates in tumours, including myeloid-like T cells with phagocytic capacity driven by orthogonal GSCFR (oGCSFR), and type 2 cytotoxic T&nbsp;(TC2) and helper T (TH2) cell differentiation driven by orthogonal IL-4R (o4R). T cells with orthogonal IL-22R (o22R) and oGCSFR, neither of which are natively expressed on T cells, exhibited stem-like and exhaustion-resistant transcriptional and chromatin landscapes, enhancing anti-tumour properties. Non-native receptor pairings and their resultant JAK–STAT signals open a path to diversifying T cell states beyond those induced by natural cytokines. Orthogonal chimeric receptors containing intracellular domains for IL-4R, IL-20R, IL-22R and GCSFR reprogram T cells to promote distinct natural and synthetic cell states with functional and therapeutic implications.},
  archive      = {J_NATURE},
  author       = {Zhao, Yang and Ogishi, Masato and Pal, Aastha and Su, Leon L. and Tao, Pingdong and Jiang, Hua and Rodriguez, Grayson E. and Chen, Xiaojing and Sun, Qinli and Rysavy, Lea Wenting and Limsuwannarot, Sam and Waghray, Deepa and Kalbasi, Anusha and Garcia, K. Christopher},
  doi          = {10.1038/s41586-025-09393-1},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1039-1050},
  shortjournal = {Nature},
  title        = {Expanding the cytokine receptor alphabet reprograms t cells into diverse states},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic suppression features ABHD18 as a barth syndrome therapeutic target. <em>NATURE</em>, <em>645</em>(8082), 1029-1038. (<a href='https://doi.org/10.1038/s41586-025-09373-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cardiolipin (CL) is the signature phospholipid of the inner mitochondrial membrane, where it stabilizes electron transport chain protein complexes1. The final step in CL biosynthesis relates to its remodelling: the exchange of nascent acyl chains with longer, unsaturated chains1. However, the enzyme responsible for cleaving nascent CL&nbsp;(nCL) has remained elusive. Here, we describe ABHD18 as a candidate deacylase in the CL biosynthesis pathway. Accordingly, ABHD18 converts CL into monolysocardiolipin (MLCL) in vitro, and its inactivation in cells and mice results in a shift to nCL in serum and tissues. Notably, ABHD18 deactivation rescues the mitochondrial defects in cells and the morbidity and mortality in mice associated with Barth syndrome. This rare genetic disease is characterized by the build-up of MLCL resulting from inactivating mutations in TAFAZZIN (TAZ), which encodes the final enzyme in the CL-remodelling cascade1. We also identified a selective, covalent, small-molecule inhibitor of ABHD18 that rescues TAZ mutant phenotypes in fibroblasts from human&nbsp;patients&nbsp;and in&nbsp;fish embryos. This study highlights a striking example of genetic suppression of a monogenic disease revealing a canonical enzyme in the CL biosynthesis pathway. The enzyme ABHD18 is shown to have a key role in cardiolipin metabolism in mitochondria, offering a potential route for a small-molecule treatment of the rare genetic disease Barth syndrome.},
  archive      = {J_NATURE},
  author       = {Masud, Sanna N. and Srivastava, Anchal and Mero, Patricia and Echezarreta, Victoria Saba and Anderson, Eve and van Buren, Lennard and Wei, Jiarun and Taylor, David Thomson and Farias, Adrian Granda and Mikolajewicz, Nicholas and Shaw, Angela and Murareanu, Brandon M. and Lohbihler, Michelle and Carney, Olivia Sniezek and van Heeringen, Simon and Clijsters, Linda and Sizova, Olga and van Ameijde, Jeroen and Nye, Freya and Habsid, Andrea and Nedyalkova, Lucy and McDonald, Laura and Simpson, Craig and Wybenga-Groot, Leanne and Brown, Kevin R. and Nho, Nhi and Suciu, Radu M. and Chan, Katherine and Tong, Amy H. Y. and Vaz, Frédéric M. and Evers, Bastiaan and Lesurf, Robert and Papaz, Tanya and Nutter, Lauryl M. J. and Protze, Stephanie and Billmann, Maximilian and Costanzo, Michael and Andrews, Brenda J. and Myers, Chad L. and Mital, Seema and Vernon, Hilary and Brummelkamp, Thijn R. and Boone, Charles and Scott, Ian C. and Niphakis, Micah J. and Strathdee, Douglas and Nijman, Sebastian M. B. and Blomen, Vincent A. and Moffat, Jason},
  doi          = {10.1038/s41586-025-09373-5},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1029-1038},
  shortjournal = {Nature},
  title        = {Genetic suppression features ABHD18 as a barth syndrome therapeutic target},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Determinants of successful AAV-vectored delivery of HIV-1 bNAbs in early life. <em>NATURE</em>, <em>645</em>(8082), 1020-1028. (<a href='https://doi.org/10.1038/s41586-025-09330-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in HIV-1 prophylaxis, vertical transmission remains a pressing problem in developing countries1. Given the promise of broadly neutralizing antibodies (bNAbs) for HIV-1 prevention2, we hypothesized that neonatal delivery of bNAbs using adeno-associated virus (AAV) could provide durable HIV-1 immunity during infancy. Here, using infant rhesus macaques (Macaca mulatta) as a model, we show that a one-time administration of an AAV vector encoding bNAb 3BNC117 at birth led to sustained bNAb expression for more than three years without redosing. This approach significantly protected both infant and pre-adolescent rhesus macaques from infection with simian–human immunodeficiency virus in mucosal challenge models that mimic HIV-1 transmission through breastfeeding and sexual intercourse. Age at the time of AAV-3BNC117 administration was a main determinant of success and was inversely correlated with the incidence of host anti-drug antibodies that restricted bNAb expression. Consistent with principles of neonatal tolerance3,4, newborn rhesus macaques exhibited higher levels of bNAb expression than older infants and juveniles following AAV-3BNC117 dosing. Furthermore, in utero exposure to recombinant 3BNC117 suppressed anti-drug antibodies and improved AAV-vectored delivery of this bNAb in older infants. Thus, our results suggest that neonatal and fetal immunological tolerance can be leveraged to improve postnatal AAV delivery of HIV-1 bNAbs in primates. Since years-long HIV-1 immunity can be generated in rhesus macaques from a one-time AAV vector administration at birth, future studies should evaluate the ability of this strategy to prevent perinatal and adolescent HIV-1 infections in humans. A single dose of an adeno-associated virus vector encoding an HIV-1 broadly neutralizing antibody given shortly after birth results in persistent antibody expression and protection from infection in rhesus macaque models of human HIV-1 transmission through breastfeeding and sexual intercourse.},
  archive      = {J_NATURE},
  author       = {Ardeshir, Amir and O’Hagan, Daniel and Mehta, Isha and Shandilya, Siddhartha and Hopkins, Lincoln L. J. and Adamson, Lourdes and Kuroda, Marcelo J. and Hahn, Patricia A. and da Costa, Lucas A. B. and Fuchs, Sebastian P. and Martinez-Navio, Jose M. and Gardner, Matthew R. and Van Rompay, Koen K. A. and Magnani, Diogo M. and Lifson, Jeffrey D. and Gao, Guangping and Farzan, Michael and Desrosiers, Ronald C. and Das, Jishnu and Martins, Mauricio A.},
  doi          = {10.1038/s41586-025-09330-2},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1020-1028},
  shortjournal = {Nature},
  title        = {Determinants of successful AAV-vectored delivery of HIV-1 bNAbs in early life},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiple oestradiol functions inhibit ferroptosis and acute kidney injury. <em>NATURE</em>, <em>645</em>(8082), 1011-1019. (<a href='https://doi.org/10.1038/s41586-025-09389-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acute tubular necrosis mediates acute kidney injury (AKI) and nephron loss1, the hallmark of end-stage renal disease2–4. For decades, it has been known that female kidneys are less sensitive to AKI5,6. Acute tubular necrosis involves dynamic cell death propagation by ferroptosis along the tubular compartment7,8. Here we demonstrate abrogated ferroptotic cell death propagation in female kidney tubules. 17β-oestradiol establishes an anti-ferroptotic state through non-genomic and genomic mechanisms. These include the potent direct inhibition of ferroptosis by hydroxyoestradiol derivatives, which function as radical trapping antioxidants, are present at high concentrations in kidney tubules and, when exogenously applied, protect male mice from AKI. In cells, the oxidized hydroxyoestradiols are recycled by FSP19,10, but FSP1-deficient female mice were not sensitive to AKI. At the genomic level, female ESR1-deficient kidney tubules partially lose their anti-ferroptotic capacity, similar to ovariectomized mice. While ESR1 promotes the anti-ferroptotic hydropersulfide system, male tubules express pro-ferroptotic proteins of the ether lipid pathway which are suppressed by ESR1 in female tissues until menopause. In summary, we identified non-genomic and genomic mechanisms that collectively explain ferroptosis resistance in female tubules and may function as therapeutic targets for male and postmenopausal female individuals. 17β-oestradiol establishes an anti-ferroptotic state in female kidney tubules.},
  archive      = {J_NATURE},
  author       = {Tonnus, Wulf and Maremonti, Francesca and Gavali, Shubhangi and Schlecht, Marlena Nastassja and Gembardt, Florian and Belavgeni, Alexia and Leinung, Nadja and Flade, Karolin and Bethe, Natalie and Traikov, Sofia and Haag, Anne and Schilling, Danny and Penkov, Sider and Mallais, Melodie and Gaillet, Christine and Meyer, Claudia and Katebi, Melika and Ray, Anushka and Gerhardt, Louisa M. S. and Brucker, Anne and Becker, Jorunn Naila and Tmava, Mirela and Schlicker, Lisa and Schulze, Almut and Himmerkus, Nina and Shevchenko, Andrej and Peitzsch, Mirko and Barayeu, Uladzimir and Nasi, Sonia and Putz, Juliane and Korach, Kenneth S. and Neugarten, Joel and Golestaneh, Ladan and Hugo, Christian and Becker, Jan Ulrich and Weinberg, Joel M. and Lorenz, Svenja and Proneth, Bettina and Conrad, Marcus and Wolf, Eckhard and Plietker, Bernd and Rodriguez, Raphaël and Pratt, Derek A. and Dick, Tobias P. and Fedorova, Maria and Bornstein, Stefan R. and Linkermann, Andreas},
  doi          = {10.1038/s41586-025-09389-x},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1011-1019},
  shortjournal = {Nature},
  title        = {Multiple oestradiol functions inhibit ferroptosis and acute kidney injury},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Commensal yeast promotes salmonella typhimurium virulence. <em>NATURE</em>, <em>645</em>(8082), 1002-1010. (<a href='https://doi.org/10.1038/s41586-025-09415-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enteric pathogens engage in complex interactions with the host and the resident microbiota to establish gut colonization1–3. Although mechanistic interactions between enteric pathogens and bacterial commensals have been extensively studied, whether and how commensal fungi affect enteric infections remain largely unknown1. Here we show that colonization with the common human gut commensal fungus Candida albicans worsened infections with the enteric pathogen Salmonella enterica subsp. enterica serovar Typhimurium. The presence of C. albicans in the mouse gut increased Salmonella caecal colonization and systemic dissemination. We investigated the underlying mechanism and found that Salmonella binds to C. albicans via type 1 fimbriae and uses its type 3 secretion system to deliver effector proteins into C. albicans. A specific effector, SopB, was sufficient to manipulate C. albicans metabolism and trigger the release of millimolar amounts of arginine into the extracellular environment. The released arginine, in turn, induced expression of the type 3 secretion system in Salmonella, increasing its invasion of epithelial cells. C. albicans deficient in arginine production was unable to increase Salmonella virulence. Arginine-producing C. albicans also dampened the inflammatory response during Salmonella infection. Arginine supplementation in the absence of C. albicans increased the systemic spread of Salmonella and decreased the inflammatory response, phenocopying the presence of C. albicans. In summary, we identified C. albicans colonization as a susceptibility factor for disseminated Salmonella infection and arginine as a central metabolite in the cross-kingdom interaction between fungi, bacteria and host. Commensal Candida albicans enhances the virulence and dissemination of Salmonella enterica subsp. enterica serovar Typhimurium.},
  archive      = {J_NATURE},
  author       = {Jaswal, Kanchan and Todd, Olivia A. and Flores Audelo, Roberto C. and Santus, William and Paul, Saikat and Duffy, Ciera M. and Eshoo, Edward T. and Singh, Manmeet and Miao, Jian and Underhill, David M. and Peters, Brian M. and Behnsen, Judith},
  doi          = {10.1038/s41586-025-09415-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {1002-1010},
  shortjournal = {Nature},
  title        = {Commensal yeast promotes salmonella typhimurium virulence},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inter-brain neural dynamics in biological and artificial intelligence systems. <em>NATURE</em>, <em>645</em>(8082), 991-1001. (<a href='https://doi.org/10.1038/s41586-025-09196-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Social interaction can be regarded as a dynamic feedback loop between interacting individuals as they act and react to each other1,2. Here, to understand the neural basis of these interactions, we investigated inter-brain neural dynamics across individuals in both mice and artificial intelligence systems. By measuring activities of molecularly defined neurons in the dorsomedial prefrontal cortex of socially interacting mice, we find that the multi-dimensional neural space within each individual can be partitioned into two distinct subspaces—a shared neural subspace that represents shared neural dynamics across animals and a unique neural subspace that represents activity unique to each animal. Notably, compared with glutamatergic neurons, GABAergic (γ-aminobutyric acid-producing) neurons in the dorsomedial prefrontal cortex contain a considerably larger shared neural subspace, which arises from behaviours of both self and others. We extended this framework to artificial intelligence agents and observed that, as social interactions emerged, so too did shared neural dynamics between interacting agents. Importantly, selectively disrupting the neural components that contribute to shared neural dynamics substantially reduces the agents’ social actions. Our findings suggest that shared neural dynamics represent a fundamental and generalizable feature of interacting neural systems present in both biological and artificial agents and highlight the functional significance of shared neural dynamics in driving social interactions. Shared inter-brain neural dynamics, reflecting aspects of social interaction including self and other’s behaviours, arise in GABAergic neurons of the dorsomedial prefrontal cortex of socially interacting mice, as well as in the neurons of socially interacting artificial intelligence agents.},
  archive      = {J_NATURE},
  author       = {Zhang, Xingjian and Phi, Nguyen and Li, Qin and Gorzek, Ryan and Zwingenberger, Niklas and Huang, Shan and Zhou, John L. and Kingsbury, Lyle and Raam, Tara and Wu, Ye Emily and Wei, Don and Kao, Jonathan C. and Hong, Weizhe},
  doi          = {10.1038/s41586-025-09196-4},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {991-1001},
  shortjournal = {Nature},
  title        = {Inter-brain neural dynamics in biological and artificial intelligence systems},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A hypothalamic circuit that modulates feeding and parenting behaviours. <em>NATURE</em>, <em>645</em>(8082), 981-990. (<a href='https://doi.org/10.1038/s41586-025-09268-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across mammalian species, new mothers undergo behavioural changes to nurture their offspring and meet the caloric demands of milk production1–5. Although many neural circuits underlying feeding and parenting behaviours are well characterized6–9, it is unclear how these different circuits interact and adapt during lactation. Here we performed transcriptomic profiling of the arcuate nucleus (ARC) and the medial preoptic area (MPOA) of the mouse hypothalamus in response&nbsp;to lactation and hunger. Furthermore, we showed that heightened appetite in lactating mice was accompanied by increased activity of hunger-promoting agouti-related peptide (AgRP) neurons in the ARC (ARCAgRP neurons). To assess the strength of hunger versus maternal drives, we designed a conflict assay in which female mice chose between a food source or pups and nesting material. Although food-deprived lactating mothers prioritized parenting over feeding, hunger reduced the duration and disrupted the sequences of parenting behaviours in both lactating and virgin females. We found that ARCAgRP neurons inhibit bombesin receptor subtype&nbsp;3 (BRS3) neurons in the MPOA (MPOABRS3 neurons), which become more active postpartum and govern parenting and satiety. Activation of this ARCAgRP-to-MPOABRS3 circuit shifted behaviours from parenting to food-seeking. Thus, hypothalamic networks are modulated by physiological states and work antagonistically during the prioritization of competing motivated behaviours. Single-cell transcriptomic analysis of mouse hypothalamus and behavioural experiments show that specific hypothalamic networks regulate conflicting feeding versus parenting behaviours of female mice.},
  archive      = {J_NATURE},
  author       = {Alcantara, Ivan C. and Li, Chia and Gao, Claire and Rodriguez González, Shakira and Mickelsen, Laura E. and Papas, Brian N. and Goldschmidt, Abigail I. and Cohen, Isabel M. and Mazzone, Christopher M. and de Araujo Salgado, Isabel and Piñol, Ramón A. and Xiao, Cuiying and Karolczak, Eva O. and Li, Jian-Liang and Cui, Guohong and Reitman, Marc L. and Krashes, Michael J.},
  doi          = {10.1038/s41586-025-09268-5},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {981-990},
  shortjournal = {Nature},
  title        = {A hypothalamic circuit that modulates feeding and parenting behaviours},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Replay and representation dynamics in the hippocampus of freely flying bats. <em>NATURE</em>, <em>645</em>(8082), 974-980. (<a href='https://doi.org/10.1038/s41586-025-09341-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cognitive functions for navigation and memory rely on emergent properties of neural ensembles in the hippocampus, such as activity replay1–5 and theta sequences6–9. However, whether and how these phenomena generalize across species with distinct navigational demands and neurophysiological properties remains unclear. Here we wirelessly recorded neural activity from large populations of cells and local field potentials from the hippocampus of freely flying Egyptian fruit bats (Rousettus aegyptiacus) engaged in free, spontaneous foraging behaviour. During rest, we identified time-compressed forward and reverse replays of multiple flight trajectories coinciding with sharp-wave ripples. Notably, replays occurred predominantly at locations that were both spatially and temporally distant from the replayed behaviour, and their speed scaled with trajectory length, challenging present models of replay mechanisms. During flight, neural ensembles exhibited fast representational sweeps, in which the decoded location moved ahead of the bat’s position cyclically. In contrast to reports in rodents, sweeps occurred in the absence of theta oscillations, and were instead phase locked to a prominent motor behavioural rhythm—the bat’s wing-beat cycle. This suggests that behaviourally relevant sensorimotor rhythms can interact with hippocampal ensemble dynamics in a highly structured manner. Combined, our findings challenge existing models of ensemble dynamics in the mammalian hippocampus, and highlight the importance of comparative studies in ethologically relevant conditions for elucidating brain function. Wireless large-scale&nbsp;recordings of neural activity in freely flying fruit bats reveal that hippocampal ensemble properties are associated with sensorimotor rhythms, and that previous models of replay and sequential dynamics, based on experiments in rodents, might not be applicable to other species.},
  archive      = {J_NATURE},
  author       = {Forli, Angelo and Fan, Wudi and Qi, Kevin K. and Yartsev, Michael M.},
  doi          = {10.1038/s41586-025-09341-z},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {974-980},
  shortjournal = {Nature},
  title        = {Replay and representation dynamics in the hippocampus of freely flying bats},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The genomic origin of the unique chaetognath body plan. <em>NATURE</em>, <em>645</em>(8082), 964-973. (<a href='https://doi.org/10.1038/s41586-025-09403-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of animal phyla, each with their unique body plan, was a rapid event in the history of animal life, yet its genomic underpinnings are still poorly understood1. Here we investigate at the genomic, regulatory and cellular levels, the origin of one of the most distinctive animal phyla, the chaetognaths, whose organismal characteristics have historically complicated their phylogenetic placement2,3. We show that these characteristics are reflected at the cell-type level by the expression of genes that originated in the chaetognath lineage, contributing to adaptation to planktonic life at the sensory and structural levels4. Similarly to other members of gnathiferans (which also include rotifers and several other microscopic phyla)5,6, chaetognaths have undergone accelerated genomic evolution with gene loss and chromosomal fusions7,8. Furthermore, they secondarily duplicated thousands of genes9,10, without evidence for a whole-genome duplication, yielding, for instance, tandemly expanded Hox genes, as well as many phylum-specific genes. We also detected repeat-rich highly methylated neocentromeres and a simplified DNA methylation toolkit that is involved in mobile element repression rather than transcriptional control. Consistent with fossil evidence11,12, our observations suggest that chaetognaths emerged after a phase of morphological simplification through a reinvention of organ systems paralleled by massive genomic reorganization, explaining the uniqueness of their body plan. Genomic, single-cell transcriptomic and epigenetic analyses show that chaetognaths, following extensive gene loss in the gnathiferan lineage, relied on newly evolved genes and lineage-specific tandem duplications, not caused by a whole-genome duplication event, to shape their distinctive body plan.},
  archive      = {J_NATURE},
  author       = {Piovani, Laura and Gavriouchkina, Daria and Parey, Elise and Sarre, Luke A. and Peijnenburg, Katja T. C. A. and Martín-Durán, José María and Rokhsar, Daniel S. and Satoh, Noriyuki and de Mendoza, Alex and Goto, Taichiro and Marlétaz, Ferdinand},
  doi          = {10.1038/s41586-025-09403-2},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {964-973},
  shortjournal = {Nature},
  title        = {The genomic origin of the unique chaetognath body plan},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The evolution of hominin bipedalism in two steps. <em>NATURE</em>, <em>645</em>(8082), 952-963. (<a href='https://doi.org/10.1038/s41586-025-09399-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bipedalism is a human-defining trait1–3. It is made possible by the familiar, bowl-shaped pelvis, whose short, wide iliac blades curve along the sides of the body to stabilize walking and support internal organs and a large-brained, broad-shouldered baby4–6. The ilium changes compared with living primates are an evolutionary novelty7. However, how this evolution came about remains unknown. Here, using a multifaceted histological, comparative genomic and functional genomic approach, we identified the developmental bases of the morphogenetic shifts in the human pelvis that made bipedalism possible. First, we observe that the human ilium cartilage growth plate underwent a heterotopic shift, residing perpendicular to the orientation present in other primate (and mouse) ilia. Second, we observe heterochronic and heterotopic shifts in ossification that are unlike those in non-human primate ilia or human long bones. Ossification initiates posteriorly, resides externally with fibroblast (and perichondral) cells contributing to osteoblasts, and is delayed compared with other bones in humans and with primate ilia. Underlying these two shifts are regulatory changes in an integrated chondrocyte–perichondral–osteoblast pathway, involving complex hierarchical interactions between SOX9–ZNF521–PTH1R and RUNX2–FOXP1/2. These innovations facilitated further growth of the human pelvis and the unique formation of the ilium among primates. The human pelvis exhibits distinct spatiotemporal ossification patterns and an ilium cartilage growth plate that is shifted perpendicularly compared with those of other mammals and non-human primates—two key adaptations that underlie bipedalism.},
  archive      = {J_NATURE},
  author       = {Senevirathne, Gayani and Fernandopulle, Serena C. and Richard, Daniel and Baumgart, Stephanie L. and Christensen, Anika Liv and Fabbri, Matteo and Höppner, Jakob and Jüppner, Harald and Li, Peishu and Bothe, Vivien and Fröbisch, Nadia and Simcock, Ian and Arthurs, Owen J. and Calder, Alistair and Freilich, Naomi and Nowlan, Niamh C. and Glass, Ian A. and Craft, April and Capellini, Terence D.},
  doi          = {10.1038/s41586-025-09399-9},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {952-963},
  shortjournal = {Nature},
  title        = {The evolution of hominin bipedalism in two steps},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stronger el niños reduce tropical forest arthropod diversity and function. <em>NATURE</em>, <em>645</em>(8082), 946-951. (<a href='https://doi.org/10.1038/s41586-025-09351-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is ongoing debate about the vulnerability of arthropods to climate change1,2. Long-term impacts of climate change on arthropod communities could manifest through short-term weather patterns3. Arthropods in the tropics are hyper-diverse4,5 and contribute many crucial ecosystem functions6,7, but are comparatively less studied than in temperate regions1,8,9. Tropical forest arthropods and the functions that they provide may be vulnerable to intensified El Niño events under climate change10–12. Here we perform time-series analysis of data from primary tropical forests, which reveal long-term declines in arthropod diversity and function that were linked to El Niño occurrence. In the Americas, species losses correlated with El Niño sensitivity, and abundant species fluctuated according to feeding traits and level of ecological specialization. Parallel declines in butterflies in Southeast Asia suggested that impacts spanned continents. Predicted arthropod diversity changes correlated with observed rates of invertebrate-mediated decomposition and leaf herbivory, which were oscillating and crashing, respectively, across the tropics. Our analyses suggest that an intensified El Niño immediately threatens tropical forest arthropods and the ecosystem functions that they provide. The broader consequences remain unknown, but such widespread changes could fundamentally alter tropical forest ecosystems13. Long-term monitoring of arthropod diversity and forest functioning across the tropics is paramount, as is researching the potential mechanisms that underly this novel threat. Time-series data from tropical forests tracking weather and declines in arthropod diversity and function show that fluctuations in species were largely dependent on their El Niño sensitivity and ecological specialization.},
  archive      = {J_NATURE},
  author       = {Sharp, Adam C. and Boyle, Michael J. W. and Bonebrake, Timothy C. and Guo, Yirong and Kitching, Roger L. and Stork, Nigel E. and Zeng, Xiaoyi and Ashton, Louise A.},
  doi          = {10.1038/s41586-025-09351-x},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {946-951},
  shortjournal = {Nature},
  title        = {Stronger el niños reduce tropical forest arthropod diversity and function},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Volcanic crisis reveals coupled magma system at santorini and kolumbo. <em>NATURE</em>, <em>645</em>(8082), 939-945. (<a href='https://doi.org/10.1038/s41586-025-09525-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volcanic crises, driven by renewed magma inflow and migration, result in surface deformation and seismicity that can provide unique insights into the structure of volcanic systems and magmatic processes. Although the highly explosive volcanoes of Santorini and Kolumbo1,2 in the Greek Aegean Sea are just 7 km apart, their potentially coupled deep magmatic feeding systems are only poorly understood3,4. The 2025 volcano–tectonic crisis of Santorini simultaneously affected both volcanic centres, providing insights into a complex, multistorage feeder system. Here we integrate onshore and marine seismological data with geodetic measurements to reconstruct magma migration before and during the crisis. Gradual inflation in the Santorini caldera, beginning in mid-2024, preceded the January 2025 intrusion of a magma-filled dike sourced from a mid-crustal reservoir beneath Kolumbo, indicating a link between the two volcanoes. Joint inversion of ground and satellite-based deformation data indicates that approximately 0.31 km3 of magma intruded as an approximately 13-km-long dike, reactivating principal regional faults and arresting 3–5 km below the seafloor. The 2024–2025 resurgence of magmatic activity beneath both volcanic centres and their apparent coupling provides insights into the dynamic interplay of magma storage, transport and reservoir failure beneath neighbouring volcanoes. Seismological and geodetic data are used together with a machine learning earthquake catalogue to reconstruct magma migration before and during the 2025 volcano–tectonic crisis at Santorini volcano, indicating a coupling between Santorini and Kolumbo.},
  archive      = {J_NATURE},
  author       = {Isken, Marius P. and Karstens, Jens and Nomikou, Paraskevi and Parks, Michelle Maree and Drouin, Vincent and Rivalta, Eleonora and Crutchley, Gareth J. and Haghighi, Mahmud Haghshenas and Hooft, Emilie E. E. and Cesca, Simone and Walter, Thomas R. and Hainzl, Sebastian and Saul, Joachim and Anastasiou, Dimitris and Raptakis, Kostas and Shapiro, Nikolai M. and Münchmeyer, Jannes and Higueret, Quentin and Soubestre, Jean and Brenguier, Florent and Hufstetler, Rebeckah S. and Autumn, Kaisa R. and Tsakiri, Maria and Lange, Dietrich and Kopp, Heidrun and Urlaub, Morelia and Blanch Jover, María and Preine, Jonas and Hübscher, Christian and Motagh, Mahdi and Müller, Daniel and Dahm, Torsten and Berndt, Christian},
  doi          = {10.1038/s41586-025-09525-7},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {939-945},
  shortjournal = {Nature},
  title        = {Volcanic crisis reveals coupled magma system at santorini and kolumbo},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Catalytic enantioselective synthesis of alkylidenecyclopropanes. <em>NATURE</em>, <em>645</em>(8082), 932-938. (<a href='https://doi.org/10.1038/s41586-025-09485-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The enantioselective construction of small-ring carbocycles provides organic chemists with an enduring challenge1. Despite their commercial importance, enantioselective synthetic routes towards alkylidenecyclopropanes, a class of small-ring carbocycles, remain underdeveloped2,3. Alkylidenecyclopropanes can be converted into cyclopropanes, a common feature in drug molecules (for example, Nirmatrelvir 1)4, as well as both naturally occurring and synthetic agrochemicals (for example, permethrin 2)5,6. Here we describe the facile synthesis of highly enantioenriched alkylidenecyclopropanes through the use of a bifunctional iminophosphorane catalysed, stereo-controlled, strain-relieving deconjugation. Small modifications to the basic catalyst system were used to broaden the scope of the reaction to substrates containing ester, amide, phosphine oxide and ketone functionalities. Through the design of a suitable substrate and retuning of the catalyst’s iminophosphorane moiety, the transformation was effectively applied to the synthesis of a single stereoisomer of the commonplace insecticide permethrin as well as a range of cyclopropane-based insecticide cores. State-of-the-art computational studies were performed to provide detailed insights into the mechanistic pathway and origin of both diastereoselectivities and enantioselectivities. &nbsp;A bifunctional iminophosphorane-catalysed, stereo-controlled deconjugation for the synthesis of highly enantioenriched alkylidenecyclopropanes is described, alongside computational studies elucidating the mechanistic pathway and origins of diastereoselectivity and enantioselectivity.},
  archive      = {J_NATURE},
  author       = {Golec, Jonathan C. and Tan, Dong-Hang and Yamazaki, Ken and Tiekink, Eveline H. and Christensen, Kirsten E. and Hamlin, Trevor A. and Dixon, Darren J.},
  doi          = {10.1038/s41586-025-09485-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {932-938},
  shortjournal = {Nature},
  title        = {Catalytic enantioselective synthesis of alkylidenecyclopropanes},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Robot-assisted mapping of chemical reaction hyperspaces and networks. <em>NATURE</em>, <em>645</em>(8082), 922-931. (<a href='https://doi.org/10.1038/s41586-025-09490-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite decades of investigation, it remains unclear (and hard to predict1–4) how the outcomes of chemical reactions change over multidimensional ‘hyperspaces’ defined by reaction conditions5. Whereas human chemists can explore only a limited subset of these manifolds, automated platforms6–12 can generate thousands of reactions in parallel. Yet, purification and yield quantification remain bottlenecks, constrained by time-consuming and resource-intensive analytical techniques. As a result, our understanding of reaction hyperspaces remains fragmentary7,9,13–16. Are yield distributions smooth or corrugated? Do they conceal mechanistically new reactions? Can major products vary across different regions? Here, to address these questions, we developed a low-cost robotic platform using primarily optical detection to quantify yields of products and by-products at unprecedented throughput and minimal cost per condition. Scanning hyperspaces across thousands of conditions, we find and prove mathematically that, for continuous variables (concentrations, temperatures), individual yield distributions are generally slow-varying. At the same time, we uncover hyperspace regions of unexpected reactivity as well as switchovers between major products. Moreover, by systematically surveying substrate proportions, we reconstruct underlying reaction networks and expose hidden intermediates and products—even in reactions studied for well over a century. This hyperspace-scanning approach provides a versatile and scalable framework for reaction optimization and discovery. Crucially, it can help identify conditions under which complex mixtures can be driven cleanly towards different major products, thereby expanding synthetic diversity while reducing chemical input requirements. A low-cost robotic platform using mainly optical detection to quantify yields of products and by-products allows the analysis of multidimensional chemical reaction hyperspaces and networks much faster than is possible by human chemists.},
  archive      = {J_NATURE},
  author       = {Jia, Yankai and Frydrych, Rafał and Sobolev, Yaroslav I. and Wong, Wai-Shing and Prajapati, Bibek and Matuszczyk, Daniel and Bilgi, Yasemin and Gadina, Louis and Ahumada, Juan Carlos and Moldagulov, Galymzhan and Kim, Namhun and Larsen, Eric S. and Deschamps, Maxence and Jiang, Yanqiu and Grzybowski, Bartosz A.},
  doi          = {10.1038/s41586-025-09490-1},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {922-931},
  shortjournal = {Nature},
  title        = {Robot-assisted mapping of chemical reaction hyperspaces and networks},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stabilizing effect of amino acids on protein and colloidal dispersions. <em>NATURE</em>, <em>645</em>(8082), 915-921. (<a href='https://doi.org/10.1038/s41586-025-09506-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amino acids (AAs) have a long history of being used as stabilizers for biological media1. For example, they are important components in biomedical formulations. The effect of AAs on biological systems is also starting to be appreciated. For example, it is believed that water-stressed cells increase the levels of AAs to prevent protein aggregation2. Several hypotheses have been put forward regarding their function, ranging from water-structuring3 to hydrotropic4 to specific effects such as&nbsp;stabilization against misfolding, yet it is not known whether their stabilizing function is protein specific or a generic colloidal property. Here we deduce that AAs possess a new and broad colloidal property: they stabilize patchy nanoscale colloids by adsorbing onto their surfaces through weak interactions. We demonstrate this general property by careful experimental evaluation of the stabilizing effect of AAs on dispersions of various proteins, plasmid DNA and non-biological nanoparticles. Furthermore, we develop a theoretical framework that captures this phenomenon and&nbsp;experimentally corroborate several new broad theoretical implications that apply beyond AAs. In vivo experiments further demonstrate that the addition of 1 M proline to insulin doubles its bioavailability in blood. Overall, our results indicate that the role of small molecules is as important as that of ionic strength and should always be reported in biophysics experiments. Amino acids possess a new and broad colloidal property: they stabilize patchy nanoscale colloids,&nbsp;such as proteins, by adsorbing onto their surfaces through weak interactions.},
  archive      = {J_NATURE},
  author       = {Mao, Ting and Xu, Xufeng and Winkler, Pamina M. and Siri, Cécilia and Poliukhina, Ekaterina and Silva, Paulo Jacob and Xu, Nan and Hu, Yu and Al Zahabi, Karim and La Polla, Rémi and Luo, Zhi and Ong, Quy and Alexander-Katz, Alfredo and Stellacci, Francesco},
  doi          = {10.1038/s41586-025-09506-w},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {915-921},
  shortjournal = {Nature},
  title        = {Stabilizing effect of amino acids on protein and colloidal dispersions},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electrostatic-repulsion-based transfer of van der waals materials. <em>NATURE</em>, <em>645</em>(8082), 906-914. (<a href='https://doi.org/10.1038/s41586-025-09510-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Van der Waals (vdW) materials offer unique opportunities for 3D integration1,2 of planar circuits towards higher-density transistors and energy-efficient computation3–7. Owing to the high thermal budget and special substrate requirement for the synthesis of high-quality vdW materials8–10, an advanced transfer technique is required that can simultaneously meet a broad range of industrial requirements, including high intactness, cleanliness and speed, large scale, low cost and versatility. However, previous efforts based on either etching or etching-free mechanisms typically only improve one or two of the aforementioned aspects11–13 and a comprehensive and systematic solution remains lacking. Here we demonstrate an electrostatic-repulsion-enabled advanced transfer technique that is etching free, high yield, fast, wafer scale, low cost and widely applicable, using ammonia solution compatible with the complementary metal–oxide–semiconductor (CMOS) industry. The high material intactness and interface cleanliness enable superior device performances in 2D field-effect transistors with 100% yield, near-zero hysteresis (7 mV) and near-ideal subthreshold swing (65.9 mV dec−1). The combination with bismuth contact further enables an ultrahigh on-current of 1.3 mA μm−1 under 1 V bias. This advanced transfer approach offers a facile and manufacturing-viable solution for vdW-materials-based electronics, paving the way for advanced 3D integration in the future. An electrostatic-repulsion-enabled advanced transfer technique based on ammonia solution is introduced for separating van der Waals thin-film materials from their substrates, demonstrating suitability for its use in the complementary metal–oxide–semiconductor (CMOS) industry.},
  archive      = {J_NATURE},
  author       = {Zheng, Xudong and Wang, Jiangtao and Jiang, Jianfeng and Zhang, Tianyi and Zhu, Jiadi and Dang, Tong and Wu, Peng and Lu, Ang-Yu and Chen, Ding-Rui and Yang, Tilo H. and Zhang, Xinyuan and Zhang, Kenan and Ma, Kyung Yeol and Wang, Zhien and Yao, Aijia and Liu, Haomin and Wan, Yi and Hsieh, Ya-Ping and Bulović, Vladimir and Palacios, Tomás and Kong, Jing},
  doi          = {10.1038/s41586-025-09510-0},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {906-914},
  shortjournal = {Nature},
  title        = {Electrostatic-repulsion-based transfer of van der waals materials},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Observing differential spin currents by resonant inelastic X-ray scattering. <em>NATURE</em>, <em>645</em>(8082), 900-905. (<a href='https://doi.org/10.1038/s41586-025-09488-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Controlling spin currents, that is, the flow of spin angular momentum, in small magnetic devices, is the principal objective of spin electronics, a main contender for future energy-efficient information technologies1,2. A pure spin current has never been measured directly because the associated electric stray fields and/or shifts in the non-equilibrium spin-dependent distribution functions are too small for conventional experimental detection methods optimized for charge transport3,4. Here we report that resonant inelastic X-ray scattering (RIXS) can bridge this gap by measuring the spin current carried by magnons—the quanta of the spin wave excitations of the magnetic order—in the presence of temperature gradients across a magnetic insulator. This is possible due to the sensitivity of the momentum- and energy-resolved RIXS intensity to minute changes in the magnon distribution under non-equilibrium conditions. We use the Boltzmann equation in the relaxation time approximation to extract transport parameters, such as the magnon lifetime at finite momentum, essential for the realization of magnon spintronics. Resonant inelastic X-ray scattering can be used to measure the spin current carried by magnons in the presence of temperature gradients across a magnetic insulator.},
  archive      = {J_NATURE},
  author       = {Gu, Yanhong and Barker, Joseph and Li, Jiemin and Kikkawa, Takashi and Camino, Fernando and Kisslinger, Kim and Sinsheimer, John and Lienhard, Lukas and Bauer, Jackson J. and Ross, Caroline A. and Basov, Dmitri N. and Saitoh, Eiji and Pelliciari, Jonathan and Bauer, Gerrit E. W. and Bisogni, Valentina},
  doi          = {10.1038/s41586-025-09488-9},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {900-905},
  shortjournal = {Nature},
  title        = {Observing differential spin currents by resonant inelastic X-ray scattering},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Atomic-scale imaging of frequency-dependent phonon anisotropy. <em>NATURE</em>, <em>645</em>(8082), 893-899. (<a href='https://doi.org/10.1038/s41586-025-09511-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directly visualizing vibrational anisotropy in individual phonon modes is essential for understanding a wide range of intriguing optical, thermal and elastic phenomena in materials1–5. Although conventional optical and diffraction techniques have been used to estimate vibrational anisotropies, they fall short in achieving the spatial and energy resolution necessary to provide detailed information4–7. Here, we introduce a new form of momentum-selective electron energy-loss spectroscopy, which enables the element-resolved imaging of frequency- and symmetry-dependent vibrational anisotropies with atomic resolution. Vibrational anisotropies manifest in different norms of orthogonal atomic displacements, known as thermal ellipsoids. Using the centrosymmetric strontium titanate as a model system, we observed two distinct types of oxygen vibrations with contrasting anisotropies: oblate thermal ellipsoids below 60 meV and prolate ones above 60 meV. In non-centrosymmetric barium titanate, our approach can detect subtle distortions of the oxygen octahedra by observing the unexpected modulation of q-selective signals between apical and equatorial oxygen sites near 55 meV, which originates from reduced crystal symmetry and may also be linked to ferroelectric polarization. These observations are quantitatively supported by theoretical modelling, which demonstrates the reliability of our approach. The measured frequency-dependent vibrational anisotropies shed new light on the dielectric and thermal behaviours governed by acoustic and optical phonons. The ability to visualize phonon eigenvectors at specific crystallographic sites with unprecedented spatial and energy resolution opens new avenues for exploring dielectric, optical, thermal and superconducting properties. A new form of momentum-selective electron energy-loss spectroscopy enables the element-resolved imaging of frequency- and symmetry-dependent vibrational anisotropies with atomic resolution.},
  archive      = {J_NATURE},
  author       = {Yan, Xingxu and Zeiger, Paul M. and Huang, Yifeng and Sun, Haoying and Li, Jie and Gadre, Chaitanya A. and Yang, Hongbin and He, Ri and Aoki, Toshihiro and Zhong, Zhicheng and Nie, Yuefeng and Wu, Ruqian and Rusz, Ján and Pan, Xiaoqing},
  doi          = {10.1038/s41586-025-09511-z},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {893-899},
  shortjournal = {Nature},
  title        = {Atomic-scale imaging of frequency-dependent phonon anisotropy},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analogue speech recognition based on physical computing. <em>NATURE</em>, <em>645</em>(8082), 886-892. (<a href='https://doi.org/10.1038/s41586-025-09501-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the rise of decentralized computing, such as in the Internet of Things, autonomous driving and personalized healthcare, it is increasingly important to process time-dependent signals ‘at the edge’ efficiently: right at the place where the temporal data are collected, avoiding time-consuming, insecure and costly communication with a centralized computing facility (or ‘cloud’). However, modern-day processors often cannot meet the restrained power and time budgets of edge systems because of intrinsic limitations imposed by their architecture (von Neumann bottleneck) or domain conversions (analogue to digital and time to frequency). Here we propose an edge temporal-signal processor based on two in-materia computing systems for both feature extraction and classification, reaching near-software accuracy for the TI-46-Word1 and Google Speech Commands2 datasets. First, a nonlinear, room-temperature reconfigurable-nonlinear-processing-unit3,4 layer realizes analogue, time-domain feature extraction from the raw audio signals, similar to the human cochlea. Second, an analogue in-memory computing chip5, consisting of memristive crossbar arrays, implements a compact neural network trained on the extracted features for classification. With submillisecond latency, reconfigurable-nonlinear-processing-unit-based feature extraction consuming roughly 300 nJ per inference, and the analogue in-memory computing-based classifier using around 78 µJ (with potential for roughly 10 µJ)6, our findings offer a promising avenue for advancing the compactness, efficiency and performance of heterogeneous smart edge processors through in materia computing hardware. A temporal-signal processor based on two in-materia computing hardware platforms—reconfigurable nonlinear-processing units (RNPUs) and analogue in-memory computing (AIMC)—is used for both feature extraction and classification, advancing compactness, efficiency, and performance of heterogeneous smart edge devices.},
  archive      = {J_NATURE},
  author       = {Zolfagharinejad, Mohamadreza and Büchel, Julian and Cassola, Lorenzo and Kinge, Sachin and Syed, Ghazi Sarwat and Sebastian, Abu and van der Wiel, Wilfred G.},
  doi          = {10.1038/s41586-025-09501-1},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {886-892},
  shortjournal = {Nature},
  title        = {Analogue speech recognition based on physical computing},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The biodiversity cell atlas: Mapping the tree of life at cellular resolution. <em>NATURE</em>, <em>645</em>(8082), 877-885. (<a href='https://doi.org/10.1038/s41586-025-09312-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell types are fundamental functional units that can be traced across the tree of life. Rapid advances in single-cell technologies, coupled with the phylogenetic expansion in genome sequencing, present opportunities for the molecular characterization of cells across a broad range of organisms. Despite these developments, our understanding of eukaryotic cell diversity remains limited and we are far from decoding this diversity from genome sequences. Here we introduce the Biodiversity Cell Atlas initiative, which aims to create comprehensive single-cell molecular atlases across the eukaryotic tree of life. This community effort will be phylogenetically informed, rely on high-quality genomes and use shared standards to facilitate comparisons across species. The Biodiversity Cell Atlas aspires to deepen our understanding of the evolution and diversity of life at the cellular level, encompassing gene regulatory programs, differentiation trajectories, cell-type-specific molecular profiles and inter-organismal interactions. The Biodiversity Cell Atlas aims to create comprehensive single-cell molecular atlases across the eukaryotic tree of life, which will be phylogenetically informed, rely on high-quality genomes and use shared standards to facilitate comparisons across species.},
  archive      = {J_NATURE},
  author       = {Sebé-Pedrós, Arnau and Tanay, Amos and Lawniczak, Mara K. N. and Arendt, Detlev and Aerts, Stein and Archibald, John and Arnone, Maria Ina and Blaxter, Mark and Cleves, Phillip and Coelho, Susana M. and Dias, Mafalda and Dunn, Casey and Elek, Anamaria and Frazer, Jonathan and Gabaldón, Toni and Gillis, Jesse and Grau-Bové, Xavier and Guigó, Roderic and Hobert, Oliver and Huerta-Cepas, Jaime and Irimia, Manuel and Klein, Allon and Lewin, Harris and Lowe, Christopher J. and Marlow, Heather and Musser, Jacob M. and Nagy, László G. and Najle, Sebastián R. and Pachter, Lior and Paez, Sadye and Papatheodorou, Irene and Passalacqua, Michael J. and Rajewsky, Nikolaus and Rhee, Seung Y. and Richards, Thomas A. and Sauka-Spengler, Tatjana and Saunders, Lauren M. and Seuntjens, Eve and Solana, Jordi and Song, Yuyao and Technau, Ulrich and Wang, Bo},
  doi          = {10.1038/s41586-025-09312-4},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {877-885},
  shortjournal = {Nature},
  title        = {The biodiversity cell atlas: Mapping the tree of life at cellular resolution},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The geoeconomic turn in decarbonization. <em>NATURE</em>, <em>645</em>(8082), 869-876. (<a href='https://doi.org/10.1038/s41586-025-09416-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rise of green industrial policy is transforming efforts to decarbonize the global economy and mitigate climate change. The first three decades of climate policy centred on international cooperation on dividing up the costs of mitigation. In the new era of green industrial policy, geoeconomic competition for the benefits of decarbonization has emerged alongside international cooperation on emissions reductions. Governments invest in the manufacturing and deployment of clean technologies to advance economic development, energy security and emissions cuts. Geoeconomic competition has the potential to accelerate global decarbonization by facilitating greater technology deployment, speeding up technology cost declines and, thus, lowering the barriers to climate action. However, it also creates major pitfalls by facilitating the rise of trade protectionism, creating international conflict, and reproducing economic divides between richer and poorer, yet growing, countries. It is thus uncertain how the geoeconomic turn will impact global decarbonization. Meanwhile, policymakers are asking fundamental questions about how to design industrial policy, manage politics, develop institutions, and deal with the trade-offs between economic, climate and security goals. This Perspective demonstrates the recent geoeconomic turn in decarbonization, lays out its implications for policymaking, identifies global spillovers and addresses research needs. The rise of green industrial policy is transforming decarbonization efforts; the implications for policymaking and global spillovers are discussed, as well as research needs.},
  archive      = {J_NATURE},
  author       = {Meckling, Jonas},
  doi          = {10.1038/s41586-025-09416-x},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {869-876},
  shortjournal = {Nature},
  title        = {The geoeconomic turn in decarbonization},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Exercise improves survival of some people with colon cancer, clinical trial shows. <em>NATURE</em>, <em>645</em>(8082), 866-867. (<a href='https://doi.org/10.1038/d41586-025-02591-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most studies on how exercise affects the survival of people with cancer have been observational. A clinical trial provides compelling evidence of the benefits of physical activity. Most studies on how exercise affects the survival of people with cancer have been observational. A clinical trial provides compelling evidence of the benefits of physical activity.},
  archive      = {J_NATURE},
  author       = {Chan, Doris S. M. and Gunter, Marc J.},
  doi          = {10.1038/d41586-025-02591-x},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {866-867},
  shortjournal = {Nature},
  title        = {Exercise improves survival of some people with colon cancer, clinical trial shows},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Head start: Fossil clues about how bodies evolved from two-fold to five-fold symmetry. <em>NATURE</em>, <em>645</em>(8082), 864-865. (<a href='https://doi.org/10.1038/d41586-025-02818-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How the five-fold symmetrical bodies of starfish and other echinoderms evolved is a mystery. New evidence fills in a key piece of the puzzle. How the five-fold symmetrical bodies of starfish and other echinoderms evolved is a mystery. New evidence fills in a key piece of the puzzle.},
  archive      = {J_NATURE},
  author       = {Wray, Gregory A.},
  doi          = {10.1038/d41586-025-02818-x},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {864-865},
  shortjournal = {Nature},
  title        = {Head start: Fossil clues about how bodies evolved from two-fold to five-fold symmetry},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oestrogen defends against kidney damage caused by iron-dependent cell death. <em>NATURE</em>, <em>645</em>(8082), 862-864. (<a href='https://doi.org/10.1038/d41586-025-02422-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Differences in protection against ferroptosis explain why men and postmenopausal women are more susceptible than premenopausal women to acute kidney injury. Differences in protection against ferroptosis explain why men and postmenopausal women are more susceptible than premenopausal women to acute kidney injury.},
  archive      = {J_NATURE},
  author       = {Vanden Berghe, Tom},
  doi          = {10.1038/d41586-025-02422-z},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {862-864},
  shortjournal = {Nature},
  title        = {Oestrogen defends against kidney damage caused by iron-dependent cell death},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Robotic system takes chemistry into hyperspace. <em>NATURE</em>, <em>645</em>(8082), 861-862. (<a href='https://doi.org/10.1038/d41586-025-02821-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exploring multidimensional ‘landscapes’ of chemical reactions provides rich insights into how they work, and might even enable the discovery of new forms of reactivity. Exploring multidimensional ‘landscapes’ of chemical reactions provides rich insights into how they work, and might even enable the discovery of new forms of reactivity.},
  archive      = {J_NATURE},
  author       = {Robinson, William E.},
  doi          = {10.1038/d41586-025-02821-2},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {861-862},
  shortjournal = {Nature},
  title        = {Robotic system takes chemistry into hyperspace},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The perplexing body plan of arrow worms decoded. <em>NATURE</em>, <em>645</em>(8082), 860-861. (<a href='https://doi.org/10.1038/d41586-025-02423-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A genome sequence and single-cell atlas of a marine worm species point towards bursts of gene emergence, duplication and loss as the drivers of lineage-specific body traits. A genome sequence and single-cell atlas of a marine worm species point towards bursts of gene emergence, duplication and loss as the drivers of lineage-specific body traits.},
  archive      = {J_NATURE},
  author       = {Lewin, Thomas D. and Luo, Yi-Jyun},
  doi          = {10.1038/d41586-025-02423-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {860-861},
  shortjournal = {Nature},
  title        = {The perplexing body plan of arrow worms decoded},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How the pelvis evolved to enable human bipedalism. <em>NATURE</em>, <em>645</em>(8082), 858-859. (<a href='https://doi.org/10.1038/d41586-025-02424-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analyses of pelvis development in humans and other primates reveal how changes in bone-patterning processes helped humans to gain the ability to walk upright. Analyses of pelvis development in humans and other primates reveal how changes in bone-patterning processes helped humans to gain the ability to walk upright.},
  archive      = {J_NATURE},
  author       = {Berthelot, Camille},
  doi          = {10.1038/d41586-025-02424-x},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {858-859},
  shortjournal = {Nature},
  title        = {How the pelvis evolved to enable human bipedalism},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electrostatic-based transfer keeps 2D materials ultra-clean. <em>NATURE</em>, <em>645</em>(8082), 857-858. (<a href='https://doi.org/10.1038/d41586-025-02822-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A technique that spontaneously detaches a 2D material from the substrate on which it was grown creates high-quality nanoscale electronic devices. A technique that spontaneously detaches a 2D material from the substrate on which it was grown creates high-quality nanoscale electronic devices.},
  archive      = {J_NATURE},
  author       = {Meng, Wanqing and Li, Lain-Jong},
  doi          = {10.1038/d41586-025-02822-1},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {857-858},
  shortjournal = {Nature},
  title        = {Electrostatic-based transfer keeps 2D materials ultra-clean},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). China’s private higher-education sector is at a turning point. <em>NATURE</em>, <em>645</em>(8082), 856. (<a href='https://doi.org/10.1038/d41586-025-03089-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Feng, Yanchao and Hu, Shilei},
  doi          = {10.1038/d41586-025-03089-2},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {856},
  shortjournal = {Nature},
  title        = {China’s private higher-education sector is at a turning point},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). US investment in african food systems will have global benefits. <em>NATURE</em>, <em>645</em>(8082), 856. (<a href='https://doi.org/10.1038/d41586-025-03090-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Barrett, Christopher B. and Herrero, Mario and Djikeng, Appolinaire and Fanzo, Jessica},
  doi          = {10.1038/d41586-025-03090-9},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {856},
  shortjournal = {Nature},
  title        = {US investment in african food systems will have global benefits},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Combat the human-made causes of spain’s wildfires. <em>NATURE</em>, <em>645</em>(8082), 856. (<a href='https://doi.org/10.1038/d41586-025-03091-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Avilés, Jesús Miguel},
  doi          = {10.1038/d41586-025-03091-8},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {856},
  shortjournal = {Nature},
  title        = {Combat the human-made causes of spain’s wildfires},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-standing marine reserves need continued support. <em>NATURE</em>, <em>645</em>(8082), 856. (<a href='https://doi.org/10.1038/d41586-025-03100-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Kersting, Diego K. and Ramos-Esplá, Alfonso A. and Templado, José and Sala, Enric},
  doi          = {10.1038/d41586-025-03100-w},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {856},
  shortjournal = {Nature},
  title        = {Long-standing marine reserves need continued support},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Universities must move with the times: How six scholars tackle AI, mental health and more. <em>NATURE</em>, <em>645</em>(8082), 852-855. (<a href='https://doi.org/10.1038/d41586-025-03031-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Societies are evolving, and so must higher education. Researchers describe initiatives that can help to create stimulating and nurturing environments fit for today’s learning landscape. Societies are evolving, and so must higher education. Researchers describe initiatives that can help to create stimulating and nurturing environments fit for today’s learning landscape.},
  archive      = {J_NATURE},
  author       = {Zhang, Ya-Qin and Jagannathan, Ramesh and Pires de Carvalho, Denise and Dougan, Lorna and Ganesh Pichamuthu, Bala and Ndlovu, Sibonokuhle},
  doi          = {10.1038/d41586-025-03031-6},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {852-855},
  shortjournal = {Nature},
  title        = {Universities must move with the times: How six scholars tackle AI, mental health and more},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How universities came to be — And why they are in trouble now. <em>NATURE</em>, <em>645</em>(8082), 849-851. (<a href='https://doi.org/10.1038/d41586-025-03030-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What and who universities are for has changed considerably since the Second World War, leaving long-standing institutions ill-equipped to cope with current financial and political challenges. What and who universities are for has changed considerably since the Second World War, leaving long-standing institutions ill-equipped to cope with current financial and political challenges.},
  archive      = {J_NATURE},
  author       = {Altbach, Philip G.},
  doi          = {10.1038/d41586-025-03030-7},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {849-851},
  shortjournal = {Nature},
  title        = {How universities came to be — And why they are in trouble now},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reviving the past and designing the future: Books in brief. <em>NATURE</em>, <em>645</em>(8082), 847. (<a href='https://doi.org/10.1038/d41586-025-03050-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Andrew Robinson reviews five of the best science picks. Andrew Robinson reviews five of the best science picks.},
  archive      = {J_NATURE},
  author       = {Robinson, Andrew},
  doi          = {10.1038/d41586-025-03050-3},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {847},
  shortjournal = {Nature},
  title        = {Reviving the past and designing the future: Books in brief},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What drove the rise of civilizations? a decades-long quest points to warfare. <em>NATURE</em>, <em>645</em>(8082), 845-847. (<a href='https://doi.org/10.1038/d41586-025-03033-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An ambitious analysis of a vast trove of data from 800 societies around the world concludes that there’s safety in numbers. An ambitious analysis of a vast trove of data from 800 societies around the world concludes that there’s safety in numbers.},
  archive      = {J_NATURE},
  author       = {Spinney, Laura},
  doi          = {10.1038/d41586-025-03033-4},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {845-847},
  shortjournal = {Nature},
  title        = {What drove the rise of civilizations? a decades-long quest points to warfare},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fighting climate change takes more than data — It needs wonder, love and hope. <em>NATURE</em>, <em>645</em>(8082), 843-844. (<a href='https://doi.org/10.1038/d41586-025-03032-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An informative and highly accessible book discusses climate change, its impacts and challenges to mitigating global warming through a compelling personal account. An informative and highly accessible book discusses climate change, its impacts and challenges to mitigating global warming through a compelling personal account.},
  archive      = {J_NATURE},
  author       = {Muttarak, Raya},
  doi          = {10.1038/d41586-025-03032-5},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {843-844},
  shortjournal = {Nature},
  title        = {Fighting climate change takes more than data — It needs wonder, love and hope},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The mini placentas and ovaries revealing the basics of women’s health. <em>NATURE</em>, <em>645</em>(8082), 840-842. (<a href='https://doi.org/10.1038/d41586-025-03029-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lab-made organoids that mimic reproductive tissues could point to treatments for common conditions such as pre-eclampsia and endometriosis. Lab-made organoids that mimic reproductive tissues could point to treatments for common conditions such as pre-eclampsia and endometriosis.},
  archive      = {J_NATURE},
  author       = {Willyard, Cassandra},
  doi          = {10.1038/d41586-025-03029-0},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {840-842},
  shortjournal = {Nature},
  title        = {The mini placentas and ovaries revealing the basics of women’s health},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The great university shake-up: Four charts show how global higher education is changing. <em>NATURE</em>, <em>645</em>(8082), 836-839. (<a href='https://doi.org/10.1038/d41586-025-03028-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More students than ever are studying across international borders, but where and what they learn is shifting. More students than ever are studying across international borders, but where and what they learn is shifting.},
  archive      = {J_NATURE},
  author       = {Garisto, Dan},
  doi          = {10.1038/d41586-025-03028-1},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {836-839},
  shortjournal = {Nature},
  title        = {The great university shake-up: Four charts show how global higher education is changing},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dinosaur egg dated directly for the first time. <em>NATURE</em>, <em>645</em>(8082), 834. (<a href='https://doi.org/10.1038/d41586-025-02904-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Laser method could help to pinpoint the age of many fossils, but some palaeontologists say further research is needed to verify the technique. Laser method could help to pinpoint the age of many fossils, but some palaeontologists say further research is needed to verify the technique.},
  archive      = {J_NATURE},
  author       = {Basu, Mohana},
  doi          = {10.1038/d41586-025-02904-0},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {834},
  shortjournal = {Nature},
  title        = {Dinosaur egg dated directly for the first time},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trump team disbands controversial US climate panel. <em>NATURE</em>, <em>645</em>(8082), 833-834. (<a href='https://doi.org/10.1038/d41586-025-02942-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A report by the panel downplays the ills of global warming and was key to White House efforts to revoke federal authority to regulate climate policies. A report by the panel downplays the ills of global warming and was key to White House efforts to revoke federal authority to regulate climate policies.},
  archive      = {J_NATURE},
  author       = {Tollefson, Jeff},
  doi          = {10.1038/d41586-025-02942-8},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {833-834},
  shortjournal = {Nature},
  title        = {Trump team disbands controversial US climate panel},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). When AI rejects your grant proposal: Algorithms are helping to make funding decisions. <em>NATURE</em>, <em>645</em>(8082), 832-833. (<a href='https://doi.org/10.1038/d41586-025-02852-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Spanish foundation using an artificial-intelligence tool to screen grant applications says it makes the peer-review process more efficient, but other researchers say it’s a breakdown in trust. A Spanish foundation using an artificial-intelligence tool to screen grant applications says it makes the peer-review process more efficient, but other researchers say it’s a breakdown in trust.},
  archive      = {J_NATURE},
  author       = {Adam, David},
  doi          = {10.1038/d41586-025-02852-9},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {832-833},
  shortjournal = {Nature},
  title        = {When AI rejects your grant proposal: Algorithms are helping to make funding decisions},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Weird ‘time crystals’ are made visible at last. <em>NATURE</em>, <em>645</em>(8082), 831-832. (<a href='https://doi.org/10.1038/d41586-025-02939-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Time crystals, a state of matter once thought physically impossible, could soon be on a banknote. Time crystals, a state of matter once thought physically impossible, could soon be on a banknote.},
  archive      = {J_NATURE},
  author       = {Gibney, Elizabeth},
  doi          = {10.1038/d41586-025-02939-3},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {831-832},
  shortjournal = {Nature},
  title        = {Weird ‘time crystals’ are made visible at last},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ‘Revolutionary’ AI tools rescue old weather data to improve climate models. <em>NATURE</em>, <em>645</em>(8082), 830-831. (<a href='https://doi.org/10.1038/d41586-025-02798-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Specialist machine-learning models are helping researchers to transcribe centuries-old handwritten records. Specialist machine-learning models are helping researchers to transcribe centuries-old handwritten records.},
  archive      = {J_NATURE},
  author       = {Castelvecchi, Davide},
  doi          = {10.1038/d41586-025-02798-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {830-831},
  shortjournal = {Nature},
  title        = {‘Revolutionary’ AI tools rescue old weather data to improve climate models},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mysterious changes near earth’s core revealed by satellites in space. <em>NATURE</em>, <em>645</em>(8082), 829. (<a href='https://doi.org/10.1038/d41586-025-03007-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Orbiters can pick up gravitational shifts thousands of kilometres below the surface. Orbiters can pick up gravitational shifts thousands of kilometres below the surface.},
  archive      = {J_NATURE},
  author       = {Witze, Alexandra},
  doi          = {10.1038/d41586-025-03007-6},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {829},
  shortjournal = {Nature},
  title        = {Mysterious changes near earth’s core revealed by satellites in space},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LIGO is 10 years old: Black-hole breakthroughs will ‘only get better’. <em>NATURE</em>, <em>645</em>(8082), 827-828. (<a href='https://doi.org/10.1038/d41586-025-02941-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A decade after the pair of observatories first detected gravitational waves, researchers are drawing up ambitious plans for its successors. A decade after the pair of observatories first detected gravitational waves, researchers are drawing up ambitious plans for its successors.},
  archive      = {J_NATURE},
  author       = {Castelvecchi, Davide},
  doi          = {10.1038/d41586-025-02941-9},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {827-828},
  shortjournal = {Nature},
  title        = {LIGO is 10 years old: Black-hole breakthroughs will ‘only get better’},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Smoke-dried mummies pre-date egypt’s embalmed bodies. <em>NATURE</em>, <em>645</em>(8082), 825. (<a href='https://doi.org/10.1038/d41586-025-02988-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Remains buried in Asia in a crouching position are thought to be oldest known examples of intentionally mummified burials. Remains buried in Asia in a crouching position are thought to be oldest known examples of intentionally mummified burials.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02988-8},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {825},
  shortjournal = {Nature},
  title        = {Smoke-dried mummies pre-date egypt’s embalmed bodies},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rare comet photobombs telescope’s view of the cosmos. <em>NATURE</em>, <em>645</em>(8082), 825. (<a href='https://doi.org/10.1038/d41586-025-02990-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The TESS exoplanet-hunting satellite accidentally captures images of the interstellar comet 3I/ATLAS. The TESS exoplanet-hunting satellite accidentally captures images of the interstellar comet 3I/ATLAS.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02990-0},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {825},
  shortjournal = {Nature},
  title        = {Rare comet photobombs telescope’s view of the cosmos},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Soft robot steers itself down the human airway. <em>NATURE</em>, <em>645</em>(8082), 825. (<a href='https://doi.org/10.1038/d41586-025-02991-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Device can place an emergency breathing tube in the trachea faster than medical personnel can, even those with extensive training. Device can place an emergency breathing tube in the trachea faster than medical personnel can, even those with extensive training.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02991-z},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {825},
  shortjournal = {Nature},
  title        = {Soft robot steers itself down the human airway},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A plant-forward menu is linked to lower diabetes risk. <em>NATURE</em>, <em>645</em>(8082), 825. (<a href='https://doi.org/10.1038/d41586-025-02992-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Regimens that sideline animal products are associated with both environmental and health benefits, according to study of nearly 24,000 people. Regimens that sideline animal products are associated with both environmental and health benefits, according to study of nearly 24,000 people.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02992-y},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {825},
  shortjournal = {Nature},
  title        = {A plant-forward menu is linked to lower diabetes risk},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How can universities train the skilled workers of tomorrow?. <em>NATURE</em>, <em>645</em>(8082), 824. (<a href='https://doi.org/10.1038/d41586-025-03070-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The high cost of running educational laboratories is hindering industrial progress. An innovative path forwards must be charted. The high cost of running educational laboratories is hindering industrial progress. An innovative path forwards must be charted.},
  archive      = {J_NATURE},
  author       = {Warren, Roseanne},
  doi          = {10.1038/d41586-025-03070-z},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {824},
  shortjournal = {Nature},
  title        = {How can universities train the skilled workers of tomorrow?},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Universities under fire must harness more of the financial value they create. <em>NATURE</em>, <em>645</em>(8082), 823. (<a href='https://doi.org/10.1038/d41586-025-03027-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attacks on US academic institutions miss the point that they generate wealth for society. Tapping even some of that could keep innovation going. Attacks on US academic institutions miss the point that they generate wealth for society. Tapping even some of that could keep innovation going.},
  archive      = {J_NATURE},
  author       = {Huang, Yasheng},
  doi          = {10.1038/d41586-025-03027-2},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {823},
  shortjournal = {Nature},
  title        = {Universities under fire must harness more of the financial value they create},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Universities are — And must continue to be — A force for good. <em>NATURE</em>, <em>645</em>(8082), 821. (<a href='https://doi.org/10.1038/d41586-025-03065-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {From funding squeezes to political attacks, higher education is in trouble in many parts of the world. Fresh ideas will allow the sector to thrive and deliver value for society. From funding squeezes to political attacks, higher education is in trouble in many parts of the world. Fresh ideas will allow the sector to thrive and deliver value for society.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-03065-w},
  journal      = {Nature},
  month        = {9},
  number       = {8082},
  pages        = {821},
  shortjournal = {Nature},
  title        = {Universities are — And must continue to be — A force for good},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nbt">NBT - 34</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(9), 1580. (<a href='https://doi.org/10.1038/s41587-025-02804-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02804-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry. <em>NBT</em>, <em>43</em>(9), 1577-1579. (<a href='https://doi.org/10.1038/s41587-025-02803-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breaking the cycle of institutional betrayal and the culture of silence requires decisive actions to promote transparency, accountability and supportive environments in both academia and industry.},
  archive      = {J_NBT},
  author       = {Vogelaar, Alison E. and Mahmoudi, Morteza},
  doi          = {10.1038/s41587-025-02803-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1577-1579},
  shortjournal = {Nature Biotech.},
  title        = {Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Global profiling of protein complex dynamics with an experimental library of protein interaction markers. <em>NBT</em>, <em>43</em>(9), 1562-1576. (<a href='https://doi.org/10.1038/s41587-024-02432-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to systematically monitor protein complex dynamics are needed. We introduce serial ultrafiltration combined with limited proteolysis-coupled mass spectrometry (FLiP–MS), a structural proteomics workflow that generates a library of peptide markers specific to changes in PPIs by probing differences in protease susceptibility between complex-bound and monomeric forms of proteins. The library includes markers mapping to protein-binding interfaces and markers reporting on structural changes that accompany PPI changes. Integrating the marker library with LiP–MS data allows for global profiling of protein–protein interactions (PPIs) from unfractionated lysates. We apply FLiP–MS to Saccharomyces cerevisiae and probe changes in protein complex dynamics after DNA replication stress, identifying links between Spt-Ada-Gcn5 acetyltransferase activity and the assembly state of several complexes. FLiP–MS enables protein complex dynamics to be probed on any perturbation, proteome-wide, at high throughput, with peptide-level structural resolution and informing on occupancy of binding interfaces, thus providing both global and molecular views of a system under study. FLiP–MS uses a library of protein–protein interaction markers to understand protein complex dynamics.},
  archive      = {J_NBT},
  author       = {Dörig, Christian and Marulli, Cathy and Peskett, Thomas and Volkmar, Norbert and Pantolini, Lorenzo and Studer, Gabriel and Paleari, Camilla and Frommelt, Fabian and Schwede, Torsten and de Souza, Natalie and Barral, Yves and Picotti, Paola},
  doi          = {10.1038/s41587-024-02432-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1562-1576},
  shortjournal = {Nature Biotech.},
  title        = {Global profiling of protein complex dynamics with an experimental library of protein interaction markers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A circularly permuted CasRx platform for efficient, site-specific RNA editing. <em>NBT</em>, <em>43</em>(9), 1548-1561. (<a href='https://doi.org/10.1038/s41587-024-02430-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inactive Cas13 orthologs have been fused to a mutant human ADAR2 deaminase domain at the C terminus to enable programmable adenosine-to-inosine (A-to-I) RNA editing in selected transcripts. Although promising, existing RNA-editing tools generally suffer from a trade-off between efficacy and specificity, and off-target editing remains an unsolved problem. Here we describe the development of an optimized RNA-editing platform by rational protein engineering, CasRx-based Programmable Editing of RNA Technology (xPERT). We demonstrate that the topological rearrangement of a CasRx K940L mutant by circular permutation results in a robust scaffold for the tethering of a deaminase domain. We benchmark our tool against the REPAIR system and show that xPERT exhibits strong on-target activity like REPAIRv1 but low off-target editing like REPAIRv2. Our xPERT platform can be used to alter RNA sequence information without risking genome damage, effect temporary cellular changes and customize protein function. Site-specific and efficient RNA editing is achieved with a CasRx editor.},
  archive      = {J_NBT},
  author       = {Wang, Yuanming and Liu, Kaiwen Ivy and Liu, Mengying Mandy and Ooi, Kean Hean and Nguyen, Tram Anh and Chee, Jiunn En and Teo, Shun Xiang Danny and He, Shan and Tay, Jie Wen Douglas and Teo, Seok Yee and Liew, Kai Shin and Ge, Xiao Yu and Ng, Zhi Jian and Avagyan, Hasmik and Liu, Hao and Yi, Zirong and Chang, Keziah and Kok, Eng Piew Louis and Chen, Runjia and Yau, Chun En and Koh, Jun Wei and Wan, Yue and Tan, Meng How},
  doi          = {10.1038/s41587-024-02430-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1548-1561},
  shortjournal = {Nature Biotech.},
  title        = {A circularly permuted CasRx platform for efficient, site-specific RNA editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One-step nanoscale expansion microscopy reveals individual protein shapes. <em>NBT</em>, <em>43</em>(9), 1539-1547. (<a href='https://doi.org/10.1038/s41587-024-02431-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The attainable resolution of fluorescence microscopy has reached the subnanometer range, but this technique still fails to image the morphology of single proteins or small molecular complexes. Here, we expand the specimens at least tenfold, label them with conventional fluorophores and image them with conventional light microscopes, acquiring videos in which we analyze fluorescence fluctuations. One-step nanoscale expansion (ONE) microscopy enables the visualization of the shapes of individual membrane and soluble proteins, achieving around 1-nm resolution. We show that conformational changes are readily observable, such as those undergone by the ~17-kDa protein calmodulin upon Ca2+ binding. ONE is also applied to clinical samples, analyzing the morphology of protein aggregates in cerebrospinal fluid from persons with Parkinson disease, potentially aiding disease diagnosis. This technology bridges the gap between high-resolution structural biology techniques and light microscopy, providing new avenues for discoveries in biology and medicine. Combining expansion microscopy with super-resolution radial fluctuations captures the morphology of single proteins.},
  archive      = {J_NBT},
  author       = {Shaib, Ali H. and Chouaib, Abed Alrahman and Chowdhury, Rajdeep and Altendorf, Jonas and Mihaylov, Daniel and Zhang, Chi and Krah, Donatus and Imani, Vanessa and Spencer, Russell K. W. and Georgiev, Svilen Veselinov and Mougios, Nikolaos and Monga, Mehar and Reshetniak, Sofiia and Mimoso, Tiago and Chen, Han and Fatehbasharzad, Parisa and Crzan, Dagmar and Saal, Kim-Ann and Alawieh, Mohamad Mahdi and Alawar, Nadia and Eilts, Janna and Kang, Jinyoung and Soleimani, Alireza and Müller, Marcus and Pape, Constantin and Alvarez, Luis and Trenkwalder, Claudia and Mollenhauer, Brit and Outeiro, Tiago F. and Köster, Sarah and Preobraschenski, Julia and Becherer, Ute and Moser, Tobias and Boyden, Edward S. and Aricescu, A. Radu and Sauer, Markus and Opazo, Felipe and Rizzoli, Silvio O.},
  doi          = {10.1038/s41587-024-02431-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1539-1547},
  shortjournal = {Nature Biotech.},
  title        = {One-step nanoscale expansion microscopy reveals individual protein shapes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of compact transcriptional effectors using high-throughput measurements in diverse contexts. <em>NBT</em>, <em>43</em>(9), 1525-1538. (<a href='https://doi.org/10.1038/s41587-024-02442-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional effectors are protein domains known to activate or repress gene expression; however, a systematic understanding of which effector domains regulate transcription across genomic, cell type and DNA-binding domain (DBD) contexts is lacking. Here we develop dCas9-mediated high-throughput recruitment (HT-recruit), a pooled screening method for quantifying effector function at endogenous target genes and test effector function for a library containing 5,092 nuclear protein Pfam domains across varied contexts. We also map context dependencies of effectors drawn from unannotated protein regions using a larger library tiling chromatin regulators and transcription factors. We find that many effectors depend on target and DBD contexts, such as HLH domains that can act as either activators or repressors. To enable efficient perturbations, we select context-robust domains, including ZNF705 KRAB, that improve CRISPRi tools to silence promoters and enhancers. We engineer a compact human activator called NFZ, by combining NCOA3, FOXO3 and ZNF473 domains, which enables efficient CRISPRa with better viral delivery and inducible control of chimeric antigen receptor T cells. Improved effectors for CRISPRi/CRISPRa are developed following high-throughput screening of transcriptional domains.},
  archive      = {J_NBT},
  author       = {Tycko, Josh and Van, Mike V. and Aradhana and DelRosso, Nicole and Ye, Hanrong and Yao, David and Valbuena, Raeline and Vaughan-Jackson, Alun and Xu, Xiaoshu and Ludwig, Connor and Spees, Kaitlyn and Liu, Katherine and Gu, Mingxin and Khare, Venya and Mukund, Adi Xiyal and Suzuki, Peter H. and Arana, Sophia and Zhang, Catherine and Du, Peter P. and Ornstein, Thea S. and Hess, Gaelen T. and Kamber, Roarke A. and Qi, Lei S. and Khalil, Ahmad S. and Bintu, Lacramioara and Bassik, Michael C.},
  doi          = {10.1038/s41587-024-02442-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1525-1538},
  shortjournal = {Nature Biotech.},
  title        = {Development of compact transcriptional effectors using high-throughput measurements in diverse contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations. <em>NBT</em>, <em>43</em>(9), 1510-1524. (<a href='https://doi.org/10.1038/s41587-024-02428-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To assist the translation of genetic findings to disease pathobiology and therapeutics discovery, we present an ensemble deep learning framework, termed PIONEER (Protein–protein InteractiOn iNtErfacE pRediction), that predicts protein-binding partner-specific interfaces for all known protein interactions in humans and seven other common model organisms to generate comprehensive structurally informed protein interactomes. We demonstrate that PIONEER outperforms existing state-of-the-art methods and experimentally validate its predictions. We show that disease-associated mutations are enriched in PIONEER-predicted protein–protein interfaces and explore their impact on disease prognosis and drug responses. We identify 586 significant protein–protein interactions (PPIs) enriched with PIONEER-predicted interface somatic mutations (termed oncoPPIs) from analysis of approximately 11,000 whole exomes across 33 cancer types and show significant associations of oncoPPIs with patient survival and drug responses. PIONEER, implemented as both a web server platform and a software package, identifies functional consequences of disease-associated alleles and offers a deep learning tool for precision medicine at multiscale interactome network levels. Protein–protein interactomes incorporating structural data predict the functional consequences of disease mutations.},
  archive      = {J_NBT},
  author       = {Xiong, Dapeng and Qiu, Yunguang and Zhao, Junfei and Zhou, Yadi and Lee, Dongjin and Gupta, Shobhita and Torres, Mateo and Lu, Weiqiang and Liang, Siqi and Kang, Jin Joo and Eng, Charis and Loscalzo, Joseph and Cheng, Feixiong and Yu, Haiyuan},
  doi          = {10.1038/s41587-024-02428-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1510-1524},
  shortjournal = {Nature Biotech.},
  title        = {A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors. <em>NBT</em>, <em>43</em>(9), 1496-1509. (<a href='https://doi.org/10.1038/s41587-024-02448-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies to enhance the anti-tumor immune response of stereotactic ablative radiotherapy (SABR) at primary tumors and abscopal sites are under intensive investigation. Here we report a metabolizable binary supracluster (BSCgal) that combines gold nanoclusters as radiosensitizing adjuvants with small interfering RNA (siRNA) targeting the immunosuppressive mediator galectin-1 (Gal-1). BSCgal comprises reversibly crosslinked cationic gold nanoclusters and siRNA complexes in a polymer matrix that biodegrades over weeks, facilitating clearance (90.3% in vivo clearance at 4 weeks) to reduce toxicity. The particle size well above the renal filtration threshold facilitates passive delivery to tumors. Using mouse models of head and neck cancer, we show that BSCgal augments the radiodynamic and immunotherapeutic effects of SABR at the primary and metastatic tumors by promoting tumor-inhibitory leukocytes, upregulating cytotoxic granzyme B and reducing immunosuppressive cell populations. It outperforms SABR plus Gal-1 antagonists, chemoradiation drug cisplatin or PD-1 inhibitor. This work presents a translatable strategy to converge focal radiosensitization with targeted immune checkpoint silencing for personalized radioimmunotherapy. Gold-siRNA clusters boost the immune response of radiotherapy against primary and distant tumors.},
  archive      = {J_NBT},
  author       = {Jiang, Yuyan and Cao, Hongbin and Deng, Huaping and Guan, Li and Langthasa, Jimpi and Colburg, Deana Rae Crystal and Melemenidis, Stavros and Cotton, Renee M. and Aleman, John and Wang, Xiao-Jing and Graves, Edward E. and Kalbasi, Anusha and Pu, Kanyi and Rao, Jianghong and Le, Quynh-Thu},
  doi          = {10.1038/s41587-024-02448-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1496-1509},
  shortjournal = {Nature Biotech.},
  title        = {Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE. <em>NBT</em>, <em>43</em>(9), 1485-1495. (<a href='https://doi.org/10.1038/s41587-024-02426-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing of messenger RNA (mRNA) found in extracellular vesicles (EVs) in liquid biopsies can provide clinical information such as somatic mutations, resistance profiles and tumor recurrence. Despite this, EV mRNA remains underused due to its low abundance in liquid biopsies, and large sample volumes or specialized techniques for analysis are required. Here we introduce Self-amplified and CRISPR-aided Operation to Profile EVs (SCOPE), a platform for EV mRNA detection. SCOPE leverages CRISPR-mediated recognition of target RNA using Cas13 to initiate replication and signal amplification, achieving a sub-attomolar detection limit while maintaining single-nucleotide resolution. As a proof of concept, we designed probes for key mutations in KRAS, BRAF, EGFR and IDH1 genes, optimized protocols for single-pot assays and implemented an automated device for multi-sample detection. We validated SCOPE’s ability to detect early-stage lung cancer in animal models, monitored tumor mutational burden in patients with colorectal cancer and stratified patients with glioblastoma. SCOPE can expedite readouts, augmenting the clinical use of EVs in precision oncology. SCOPE detects low-abundance extracellular vesicle mRNA with high sensitivity for clinical applications.},
  archive      = {J_NBT},
  author       = {Song, Jayeon and Cho, Mi Hyeon and Cho, Hayoung and Song, Younseong and Lee, Sung Woon and Nam, Ho Chul and Yoon, Tae Ho and Shin, Jong Cheol and Hong, Jae-Sang and Kim, Yejin and Ekanayake, Emil and Jeon, Jueun and You, Dong Gil and Im, Sung Gap and Choi, Gyu-Seog and Park, Jun Seok and Carter, Bob C. and Balaj, Leonora and Seo, An Na and Miller, Miles A. and Park, Soo Yeun and Kang, Taejoon and Castro, Cesar M. and Lee, Hakho},
  doi          = {10.1038/s41587-024-02426-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1485-1495},
  shortjournal = {Nature Biotech.},
  title        = {Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Saturation profiling of drug-resistant genetic variants using prime editing. <em>NBT</em>, <em>43</em>(9), 1471-1484. (<a href='https://doi.org/10.1038/s41587-024-02465-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to characterize the functional effects of genetic variants of uncertain significance (VUSs) have been limited by incomplete coverage of the mutational space. In clinical oncology, drug resistance arising from VUSs can prevent optimal treatment. Here we introduce PEER-seq, a high-throughput method based on prime editing that can evaluate the functional effects of single-nucleotide variants (SNVs). PEER-seq introduces both intended SNVs and synonymous marker mutations using prime editing and deep sequences the endogenous target regions to identify the introduced SNVs. We generate and functionally evaluate 2,476 SNVs in the epidermal growth factor receptor gene (EGFR), including 99% of all possible variants in the canonical tyrosine kinase domain. We determined resistance profiles of 95% of all possible EGFR protein variants encoded in the whole tyrosine kinase domain against the common tyrosine kinase inhibitors afatinib, osimertinib and osimertinib in the presence of the co-occurring substitution T790M, in PC-9 cells. Our study has the potential to substantially improve the precision of therapeutic choices in clinical settings. PEER-seq characterizes the drug resistance profiles of thousands of genetic variants.},
  archive      = {J_NBT},
  author       = {Kim, Younggwang and Oh, Hyeong-Cheol and Lee, Seungho and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-024-02465-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1471-1484},
  shortjournal = {Nature Biotech.},
  title        = {Saturation profiling of drug-resistant genetic variants using prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal scanning of genetic variants with base and prime editing. <em>NBT</em>, <em>43</em>(9), 1458-1470. (<a href='https://doi.org/10.1038/s41587-024-02439-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational scanning connects genetic variants to phenotype, enabling the interrogation of protein functions, interactions and variant pathogenicity. However, current methodologies cannot efficiently engineer customizable sets of diverse genetic variants in endogenous loci across cellular contexts in high throughput. Here, we combine cytosine and adenine base editors and a prime editor to assess the pathogenicity of a broad spectrum of variants in the epithelial growth factor receptor gene (EGFR). Using pooled base editing and prime editing guide RNA libraries, we install tens of thousands of variants spanning the full coding sequence of EGFR in multiple cell lines and assess the role of these variants in tumorigenesis and resistance to tyrosine kinase inhibitors. Our EGFR variant scan identifies important hits, supporting the robustness of the approach and revealing underappreciated routes to EGFR activation and drug response. We anticipate that multimodal precision mutational scanning can be applied broadly to characterize genetic variation in any genetic element of interest at high and single-nucleotide resolution. Thousands of rare oncogene variants are evaluated using multimodal gene editing screens.},
  archive      = {J_NBT},
  author       = {Belli, Olivier and Karava, Kyriaki and Farouni, Rick and Platt, Randall J.},
  doi          = {10.1038/s41587-024-02439-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1458-1470},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal scanning of genetic variants with base and prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein. <em>NBT</em>, <em>43</em>(9), 1445-1457. (<a href='https://doi.org/10.1038/s41587-024-02437-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticle (LNP) delivery of clustered regularly interspaced short palindromic repeat (CRISPR) ribonucleoproteins (RNPs) could enable high-efficiency, low-toxicity and scalable in vivo genome editing if efficacious RNP–LNP complexes can be reliably produced. Here we engineer a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) to generate iGeoCas9 variants capable of &gt;100× more genome editing of cells and organs compared with the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP–LNP complexes edit a variety of cell types and induce homology-directed repair in cells receiving codelivered single-stranded DNA templates. Using tissue-selective LNP formulations, we observe genome-editing levels of 16‒37% in the liver and lungs of reporter mice that receive single intravenous injections of iGeoCas9 RNP–LNPs. In addition, iGeoCas9 RNPs complexed to biodegradable LNPs edit the disease-causing SFTPC gene in lung tissue with 19% average efficiency, representing a major improvement over genome-editing levels observed previously using viral or nonviral delivery strategies. These results show that thermostable Cas9 RNP–LNP complexes can expand the therapeutic potential of genome editing. An engineered clustered regularly interspaced short palindromic repeat ribonucleoprotein delivered in lipid nanoparticles efficiently edits cells in vivo.},
  archive      = {J_NBT},
  author       = {Chen, Kai and Han, Hesong and Zhao, Sheng and Xu, Bryant and Yin, Boyan and Lawanprasert, Atip and Trinidad, Marena and Burgstone, Benjamin W. and Murthy, Niren and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-024-02437-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1445-1457},
  shortjournal = {Nature Biotech.},
  title        = {Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering b cells to treat and study human disease. <em>NBT</em>, <em>43</em>(9), 1431-1444. (<a href='https://doi.org/10.1038/s41587-025-02757-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern cell therapies are designed to harness natural biology to treat a range of complex diseases. The field of immunology has shown that B cells exhibit multiple unique features, including a natural propensity to interact with and regulate other immune cells, a high capacity to produce proteins, and a long cellular lifespan, which are being creatively applied in engineered B cell (eB cell) therapies. In recent years, advances in genome editing technologies and animal modeling have facilitated rapid progress in our ability to study eB cells and execute proof-of-concept studies, thus enabling the first clinical trials of eB cell therapies. In this review, we provide an overview of recent developments in eB cell therapies, including early clinical studies. We discuss challenges to clinical implementation, and promising directions for leveraging B cell biology in future applications for cancer and chronic disease. Recent developments in B cell engineering expand our ability to study human disease and drive progress toward new B cell therapies, balancing challenges in clinical implementation with emerging opportunities.},
  archive      = {J_NBT},
  author       = {Trivedi, Nikita and Pitner, Ragan A. and Rawlings, David J. and James, Richard G.},
  doi          = {10.1038/s41587-025-02757-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1431-1444},
  shortjournal = {Nature Biotech.},
  title        = {Engineering b cells to treat and study human disease},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A CRISPR-based method for detecting mRNA in extracellular vesicles. <em>NBT</em>, <em>43</em>(9), 1428-1429. (<a href='https://doi.org/10.1038/s41587-024-02440-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A CRISPR-based assay both recognizes and amplifies target mRNA, achieving sub-attomolar sensitivity with single-nucleotide resolution. This method enables the detection of low-abundance mRNA in extracellular vesicles, providing clinically relevant information for precision oncology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02440-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1428-1429},
  shortjournal = {Nature Biotech.},
  title        = {A CRISPR-based method for detecting mRNA in extracellular vesicles},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Bao and rogers. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02784-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02784-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bao and rogers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Ntziachristos and yao. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02785-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02785-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Ntziachristos and yao},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI designs de novo antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02822-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02822-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {AI designs de novo antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosted mitochondria reverse cognitive impairment in mice. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02823-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02823-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Boosted mitochondria reverse cognitive impairment in mice},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mining archaea for antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02824-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02824-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Mining archaea for antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MicroRNAs. <em>NBT</em>, <em>43</em>(9), 1424. (<a href='https://doi.org/10.1038/s41587-025-02800-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to pharmaceutical compositions and methods of treatment involving microRNAs.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02800-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1424},
  shortjournal = {Nature Biotech.},
  title        = {MicroRNAs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Intellectual property rights in the cultivated meat industry: Opportunities and challenges. <em>NBT</em>, <em>43</em>(9), 1419-1423. (<a href='https://doi.org/10.1038/s41587-025-02798-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivated meat faces escalating intellectual property (IP) issues as the industry expands, including conventional patent rights, the private IP rights of animal breeders and the Convention on Biological Diversity.},
  archive      = {J_NBT},
  author       = {Baek, Garam and Chung, Sukhyun and Yu, Peter and Park, Yooheon},
  doi          = {10.1038/s41587-025-02798-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1419-1423},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property rights in the cultivated meat industry: Opportunities and challenges},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining druggable targets with artificial intelligence. <em>NBT</em>, <em>43</em>(9), 1416-1418. (<a href='https://doi.org/10.1038/s41587-025-02770-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A vast landscape of ‘undruggable’ cancer targets remains beyond the reach of conventional therapeutic agents. Recent advances in artificial intelligence (AI), however, are challenging this paradigm. Synthesizing insights from a Cancer Moonshot workshop, we argue that systemically addressing the undruggable target space with AI requires a new conceptual framework. We highlight the failure of current target taxonomies and the need for benchmarking datasets, and re-evaluate clinical validation for novel AI-driven modalities.},
  archive      = {J_NBT},
  author       = {Akinsanya, Karen and AlQuraishi, Mohammed and Boija, Ann and Chodera, John and Cichońska, Anna and Ghassemi, Marzyeh and Head, Martha and Jin, Wengong and Kibbe, Warren A. and Krogan, Nevan and LeVine, Michael V. and Moult, John and Stuart, Lynda and Tourassi, Georgia and Zou, James and Barzilay, Regina and Elemento, Olivier},
  doi          = {10.1038/s41587-025-02770-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1416-1418},
  shortjournal = {Nature Biotech.},
  title        = {Redefining druggable targets with artificial intelligence},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Senolytics under scrutiny in the quest to slow aging. <em>NBT</em>, <em>43</em>(9), 1413-1415. (<a href='https://doi.org/10.1038/s41587-025-02740-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de Magalhães, João Pedro},
  doi          = {10.1038/s41587-025-02740-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1413-1415},
  shortjournal = {Nature Biotech.},
  title        = {Senolytics under scrutiny in the quest to slow aging},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Freya biosciences. <em>NBT</em>, <em>43</em>(9), 1411-1412. (<a href='https://doi.org/10.1038/s41587-025-02783-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Freya Biosciences is restoring balance in the vaginal microbiome to improve pregnancy outcomes.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02783-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1411-1412},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Freya biosciences},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Vilya therapeutics. <em>NBT</em>, <em>43</em>(9), 1409-1410. (<a href='https://doi.org/10.1038/s41587-025-02782-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Vilya Therapeutics is designing macrocycle drug candidates.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-025-02782-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1409-1410},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Vilya therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Seaport therapeutics. <em>NBT</em>, <em>43</em>(9), 1407-1408. (<a href='https://doi.org/10.1038/s41587-025-02792-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Seaport Therapeutics is modifying central nervous system drugs for delivery through the lymphatic system, bypassing first-pass liver metabolism.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-025-02792-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1407-1408},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Seaport therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(9), 1406. (<a href='https://doi.org/10.1038/s41587-025-02817-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02817-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1406},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blooms supersized. <em>NBT</em>, <em>43</em>(9), 1405. (<a href='https://doi.org/10.1038/s41587-025-02825-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02825-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1405},
  shortjournal = {Nature Biotech.},
  title        = {Blooms supersized},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wegovy approved for MASH. <em>NBT</em>, <em>43</em>(9), 1404. (<a href='https://doi.org/10.1038/s41587-025-02828-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02828-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1404},
  shortjournal = {Nature Biotech.},
  title        = {Wegovy approved for MASH},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA go-ahead for HPV immunotherapy. <em>NBT</em>, <em>43</em>(9), 1402. (<a href='https://doi.org/10.1038/s41587-025-02827-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02827-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1402},
  shortjournal = {Nature Biotech.},
  title        = {FDA go-ahead for HPV immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Regenerative cell therapy marches into corneal blindness. <em>NBT</em>, <em>43</em>(9), 1401-1404. (<a href='https://doi.org/10.1038/s41587-025-02808-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a newly marketed product in Japan and 10 years of clinical data, regenerative cell therapies promise new options for patients facing a choice between invasive surgery or blindness.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew},
  doi          = {10.1038/s41587-025-02808-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1401-1404},
  shortjournal = {Nature Biotech.},
  title        = {Regenerative cell therapy marches into corneal blindness},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). US government slams mRNA vaccine research. <em>NBT</em>, <em>43</em>(9), 1400. (<a href='https://doi.org/10.1038/s41587-025-02826-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02826-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1400},
  shortjournal = {Nature Biotech.},
  title        = {US government slams mRNA vaccine research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mitochondrial transplants make healthy babies. <em>NBT</em>, <em>43</em>(9), 1399-1400. (<a href='https://doi.org/10.1038/s41587-025-02799-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After decades of research and consultation with regulators, mitochondrial donation is taking off in the UK. Will other countries follow?},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-025-02799-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1399-1400},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial transplants make healthy babies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing antivirals. <em>NBT</em>, <em>43</em>(9), 1397-1398. (<a href='https://doi.org/10.1038/s41587-025-02829-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Funding and public support for vaccines in the United States is at an all-time low. Promising research is resulting in effective antivirals, but they should not take the place of vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02829-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1397-1398},
  shortjournal = {Nature Biotech.},
  title        = {Advancing antivirals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nchem">NCHEM - 22</h2>
<ul>
<li><details>
<summary>
(2025). Future-proofing calcium fluoride. <em>NCHEM</em>, <em>17</em>(9), 1434. (<a href='https://doi.org/10.1038/s41557-025-01922-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thomas Schlatzer and Véronique Gouverneur discuss why the chemistry of calcium fluoride is challenging, and how imaginative thinking based on fundamental principles can create positive change for the fluorochemical industry.},
  archive      = {J_NCHEM},
  author       = {Schlatzer, Thomas and Gouverneur, Véronique},
  doi          = {10.1038/s41557-025-01922-9},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1434},
  shortjournal = {Nat. Chem.},
  title        = {Future-proofing calcium fluoride},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Catalytic and stoichiometric stepwise conversion of side-on bound dinitrogen to ammonia mediated by a uranium complex. <em>NCHEM</em>, <em>17</em>(9), 1425-1433. (<a href='https://doi.org/10.1038/s41557-025-01867-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The well-defined catalytic conversion of dinitrogen (N2) to ammonia (NH3) by molecular complexes is of fundamental interest and important for providing an atomic-level understanding of reactivity that can be related to industrial and biological nitrogen-fixation processes. Molecular catalytic N2 to NH3 conversion currently involves the reduction and protonation of terminal or bridging end-on bound metal–N2 complexes. However, catalytic N2 to NH3 conversion by side-on bound metal–N2 molecular complexes is relevant to both the industrial and biological nitrogen-fixation processes. Here, using a uranium triamidoamine complex, we report catalytic N2 to NH3 conversion involving side-on bound N2 binding. Stoichiometric reactions reveal stepwise reduction of N2 from free N2 to bridging side-on bound forms and subsequently to bridging nitrides, uniquely accessing four different states of side-on bound N2 for the same molecular system. This reveals the roles of N2, N22−, N23−, N24− and N3− in the catalytic conversion of N2 to NH3 when involving side-on bridging N2. The catalytic conversion of nitrogen to ammonia by molecular complexes typically involves the reduction and protonation of end-on metal–N2 compounds. Now, a uranium complex has been shown to catalyse the conversion of N2 to ammonia through the formation of a side-on bound dinitrogen complex. Four different states of side-on bound dinitrogen and the cleavage to nitride have been observed.},
  archive      = {J_NCHEM},
  author       = {Batov, Mikhail S. and Partlow, Heather T. and Chatelain, Lucile and Seed, John A. and Scopelliti, Rosario and Zivkovic, Ivica and Adams, Ralph W. and Liddle, Stephen T. and Mazzanti, Marinella},
  doi          = {10.1038/s41557-025-01867-z},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1425-1433},
  shortjournal = {Nat. Chem.},
  title        = {Catalytic and stoichiometric stepwise conversion of side-on bound dinitrogen to ammonia mediated by a uranium complex},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Catalytic c(sp2) homologation of alkylboranes. <em>NCHEM</em>, <em>17</em>(9), 1418-1424. (<a href='https://doi.org/10.1038/s41557-025-01854-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoboron compounds are important intermediates in organic synthesis, commonly used in metal-catalysed cross-coupling reactions. Their unique reactivity also allows modifications of their carbon framework with preservation of the valuable boryl group. Traditionally, these homologation reactions have been confined to the formation of alkyl boron compounds via C(sp3) insertion into a C–B bond. However, recent advancements in C(sp2)-insertive homologation highlight the potential of these reactions in synthesizing complex alkenes, despite current limitations in scope and control of the alkene geometry. Here we demonstrate a catalytic C(sp2)-insertive homologation for the regio- and diastereoselective synthesis of complex trisubstituted diborylalkenes from simple alkylboranes and alkynyl boronic esters. Our work demonstrates a broad reaction scope and application of the resulting products in modular and stereoselective synthesis of highly substituted alkenes. Furthermore, we provide evidence supporting a unique mechanism responsible for the excellent stereoselectivity observed in the reaction. Homologation reactions of organoboron compounds, while common, have rarely been used in stereoselective synthesis of highly substituted alkenes. Now a catalytic diastereoselective homologation of alkylboranes into trisubstituted alkenes has been developed, with key catalytic intermediates elucidated in a detailed mechanistic study.},
  archive      = {J_NCHEM},
  author       = {Gardner, Bradley W. and Lalic, Gojko},
  doi          = {10.1038/s41557-025-01854-4},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1418-1424},
  shortjournal = {Nat. Chem.},
  title        = {Catalytic c(sp2) homologation of alkylboranes},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bright triplet and bright charge-separated singlet excitons in organic diradicals enable optical read-out and writing of spin states. <em>NCHEM</em>, <em>17</em>(9), 1410-1417. (<a href='https://doi.org/10.1038/s41557-025-01875-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical control of electron spin states is important for quantum sensing and computing applications, as developed with the diamond nitrogen vacancy centre. This requires electronic excitations, excitons, with net spin. Here we report a molecular diradical where two trityl radical groups are coupled via a meta-linked fluorene bridge. The singlet exciton is at lower energy than the triplet because electron transfer from one of the radical non-bonding orbitals to the other is spin allowed, set by the charging energy for the double occupancy of the non-bonding level, the Hubbard U. Both excitons give efficient photoluminescence at 640 and 700 nm with near unity efficiency. The ground state exchange energy is low, 60 µeV, allowing control of ground state spin populations. We demonstrate spin-selective intersystem crossing and show coherent microwave control. We report up to 8% photoluminescence contrast at microwave resonance. This tuning of the singlet Mott–Hubbard exciton against the ‘bandgap’ exciton provides a new design platform for spin–optical materials. Controlling electron spin states with light is vital for quantum technologies but requires electronic excitations with net spin. Now a molecular diradical with two trityl radical groups coupled via a meta-linked fluorene bridge has been developed that features luminescent singlet and triplet excitons. The spins in the ground state can be written and read using photons, giving rise to broad magnetic and microwave modulation of the photoluminescence.},
  archive      = {J_NCHEM},
  author       = {Chowdhury, Rituparno and Murto, Petri and Panjwani, Naitik A. and Sun, Yan and Ghosh, Pratyush and Boeije, Yorrick and Cordeiro, Chiara Delpiano and Derkach, Vadim and Woo, Seung-Je and Millington, Oliver and Congrave, Daniel G. and Fu, Yao and Mustafa, Tarig B. E. and Monteverde, Miguel and Cerdá, Jesús and Londi, Giacomo and Behrends, Jan and Rao, Akshay and Beljonne, David and Chepelianskii, Alexei and Bronstein, Hugo and Friend, Richard H.},
  doi          = {10.1038/s41557-025-01875-z},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1410-1417},
  shortjournal = {Nat. Chem.},
  title        = {Bright triplet and bright charge-separated singlet excitons in organic diradicals enable optical read-out and writing of spin states},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synthesis of zero-dimensional octahedral metal halides through solvent incorporation and their photophysical properties. <em>NCHEM</em>, <em>17</em>(9), 1401-1409. (<a href='https://doi.org/10.1038/s41557-025-01869-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zero-dimensional (0D) metal halides, which feature discrete metal halide octahedra interspersed with large organoammonium cations, are the building blocks of halide perovskites. The optical properties of these materials make them promising candidates in light-emitting devices. However, developing their general design principles remains challenging. Here we report an antisolvent incorporation approach that transforms a broad range of two-dimensional tin iodide perovskites into 0D structures. This approach accommodates diverse organic cations and antisolvent molecules and is extendable to germanium and lead analogues. We show that enhancing the structural rigidity in Sn-based octahedra—modulated by organic packing—increases the radiative recombination rate while decreasing the non-radiative recombination rate, achieving near-unity photoluminescence quantum yield. Ge- and Sn-based structures exhibit large Stokes shifts and microsecond-scale triplet lifetimes due to localized excited states, while their Pb-based counterpart shows faster recombination via triplet–singlet mixing, supported by theoretical calculations. This work offers a versatile synthetic platform for 0D metal halides and deepens our current understanding of excited-state dynamics in halide perovskites. Zero-dimensional metal halides exhibit intriguing photophysical properties; however, developing their general design principles remains challenging. Now it has been shown that two-dimensional perovskites can be converted into zero-dimensional octahedral metal halides through solvent incorporation. The resulting materials feature near-unity photoluminescence quantum yields. Structural rigidity and the choice of metal cation are key parameters to tune the exciton recombination dynamics.},
  archive      = {J_NCHEM},
  author       = {Zheng, Nanlong and Cao, Songqi and Zhang, Tianhao and Huo, Peihao and Hu, Conglan and Liu, Huanyu and Guo, Ruoyao and Yan, Wenchao and Zheng, Jiayin and Jiang, Xiaofan and Zhu, Zhenyu and Sun, Junliang and Jiang, Hong and Bian, Zuqiang and Fu, Yongping and Liu, Zhiwei},
  doi          = {10.1038/s41557-025-01869-x},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1401-1409},
  shortjournal = {Nat. Chem.},
  title        = {Synthesis of zero-dimensional octahedral metal halides through solvent incorporation and their photophysical properties},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Expedited access to polyunsaturated fatty acids and biofunctional analogues by full solid-phase synthesis. <em>NCHEM</em>, <em>17</em>(9), 1391-1400. (<a href='https://doi.org/10.1038/s41557-025-01853-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyunsaturated fatty acids (PUFAs) represent a fundamental and essential class of lipids that exhibit versatile biofunctions. Lipidomic analysis has identified a growing number of lipid species, including PUFAs with diverse structural variations and biofunctions, yet their structure–function relationships are still largely unknown. In this context, an efficient synthesis of PUFAs would be highly desirable. However, no practical methodology exists for their preparation, in contrast to peptides and nucleic acids, for which diverse molecules are accessible through a well-established solid-phase synthesis. To address this, we have developed an efficient and expedited method to access a wide array of PUFAs by full solid-phase synthesis. The method allows the synthesis of various PUFAs and analogues through rapid and facile operations. Moreover, within our PUFA library, we have discovered an artificial fatty acid, antiefin, that has a high anti-inflammatory effect in vivo. Therefore, our practical synthetic pathway to PUFAs, a crucial class of lipids, is expected to make an important contribution to lipid science. Polyunsaturated fatty acids (PUFAs) are attracting considerable interest because of their diverse structures and properties. Now a method to access a wide array of PUFAs by full solid-phase synthesis has been developed and used to discover an anti-inflammatory fatty acid.},
  archive      = {J_NCHEM},
  author       = {Saito, Yutaro and Akita, Mayuko and Saika, Azusa and Sano, Yusuke and Hotta, Masashi and Morimoto, Jumpei and Uwamizu, Akiharu and Aoki, Junken and Nagatake, Takahiro and Kunisawa, Jun and Sando, Shinsuke},
  doi          = {10.1038/s41557-025-01853-5},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1391-1400},
  shortjournal = {Nat. Chem.},
  title        = {Expedited access to polyunsaturated fatty acids and biofunctional analogues by full solid-phase synthesis},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Energy transfer-enabled enantioselective photocyclization using a privileged al–salen catalyst. <em>NCHEM</em>, <em>17</em>(9), 1383-1390. (<a href='https://doi.org/10.1038/s41557-025-01857-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chiral catalysts that can engage multiple substrates, via distinct ground-state activation modes, to deliver enantioenriched products with high levels of fidelity are often described as ‘privileged’. Achieving generality in excited-state processes remains challenging, and efforts to identify privileged chiral photocatalysts are being intensively pursued. Aluminium–salen complexes are emergent contenders on account of their well-defined photophysical properties. Here we report the development of an enantioselective energy transfer (EnT) catalysis-enabled photocyclization of acrylanilides to expand the activation repertoire of Al–salen photocatalysts. This approach allows reactivity and enantioselectivity to be simultaneously regulated by an inexpensive, commercial chiral Al–salen complex upon irradiation at λ = 400 nm. Diverse cyclic products can be forged with high levels of enantioselectivity (up to 96:4 e.r.). Establishing this dichotomy in excited-state activation modes serves to consolidate the privileged status of chiral Al–salen complexes in enantioselective photocatalysis and to complement their ubiquity in ground-state regimes. Photocatalysts that can engage a range of substrates and regulate reactivity and enantioselectivity are highly desirable. Now, an enantioselective energy transfer catalysis-enabled photocyclization of acrylanilides has been developed. Reactivity and enantioselectivity are simultaneously regulated by a commercial chiral Al–salen complex upon irradiation, enabling the formation of diverse cyclic products with high levels of enantioselectivity.},
  archive      = {J_NCHEM},
  author       = {Soika, Julia and Onneken, Carina and Wiegmann, Thorben and Stünkel, Timo and Morack, Tobias and Lindfeld, Leander and Hebenbrock, Marian and Mück-Lichtenfeld, Christian and Neugebauer, Johannes and Gilmour, Ryan},
  doi          = {10.1038/s41557-025-01857-1},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1383-1390},
  shortjournal = {Nat. Chem.},
  title        = {Energy transfer-enabled enantioselective photocyclization using a privileged al–salen catalyst},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered reactivity of a bacterial e1-like enzyme enables ATP-driven modification of protein and peptide c termini. <em>NCHEM</em>, <em>17</em>(9), 1371-1382. (<a href='https://doi.org/10.1038/s41557-025-01871-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In biological systems, adenosine triphosphate (ATP) provides an energetic driving force for peptide bond formation, but protein chemists lack tools that emulate this strategy. Here we develop an ATP-driven platform for C-terminal activation and peptide ligation based on MccB, a bacterial ancestor of ubiquitin-activating (E1) enzymes. We show that MccB can act on non-native substrates to generate an O-AMPylated electrophile that reacts with exogenous nucleophiles to form diverse C-terminal functional groups including thioesters, a versatile class of biological intermediates that have been exploited for protein C-terminal bioconjugation. By mining the natural diversity of the MccB family, we identify both epitope-specific and more promiscuous MccBs. We show that epitope-specific MccB activity can be directed toward specific proteins of interest to enable high-yield, ATP-driven protein bioconjugation, and promiscuous MccB activity can be deployed for the synthesis of peptide thioester substrates for bioconjugation. Our method mimics the chemical logic of biological peptide bond synthesis for high-yield in vitro manipulation of protein structure with molecular precision. In living systems, ATP provides an energetic driving force for protein synthesis and modification. Now, an engineered enzymatic tool has been developed for high-yield, ATP-driven protein bioconjugation through versatile thioester intermediates. This method mimics biological strategies for peptide bond formation to enable precision chemical modification of proteins in vitro.},
  archive      = {J_NCHEM},
  author       = {Frazier, Clara L. and Deb, Debashrito and Leiter, William E. and Mondal, Umasankar and Weeks, Amy M.},
  doi          = {10.1038/s41557-025-01871-3},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1371-1382},
  shortjournal = {Nat. Chem.},
  title        = {Engineered reactivity of a bacterial e1-like enzyme enables ATP-driven modification of protein and peptide c termini},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A lanthanide–carbon triple bond stabilized within a fullerene cage. <em>NCHEM</em>, <em>17</em>(9), 1364-1370. (<a href='https://doi.org/10.1038/s41557-025-01856-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metal–ligand multiple bonds are ubiquitous in coordination and organometallic chemistry. In contrast, lanthanide–carbon multiple bonds are difficult to form. The isolation of complexes containing lanthanide–carbon double bonds with terminal methyl carbene (=CH2) and lanthanide–carbon triple bonds remains challenging. Here we present the synthesis of a lanthanide–carbon triple bond contained inside an endohedral fullerene cage. More specifically, we report a cerium–carbide [Ce≡CSc2] cluster encapsulated inside a C80 fullerene cage. The molecular structure of Ce≡CSc2@C80 and the nature of the Ce≡C triple bond are studied through X-ray crystallography, spectroscopic analyses and quantum chemical calculations. Our data reveal a very short Ce≡C distance of 1.969(7) Å. Chemical bonding analysis suggests that the formation of the Ce≡C bond primarily arises from the stronger bonding affinity between carbon and cerium compared with scandium inside the encapsulated cluster. The fullerene cage plays a crucial role in stabilizing and protecting this trimetallic carbide cluster featuring a Ce≡C triple bond. The synthesis and isolation of compounds featuring lanthanide–carbon triple bonds remain challenging. Now a cluster featuring a cerium–carbon triple bond has been stabilized inside a C80 fullerene cage. The solid-state structure of the compound reveals a cerium–carbon bond distance of 1.969(7) Å.},
  archive      = {J_NCHEM},
  author       = {Jiang, Hongjie and Zhao, Jing and Meng, Qingyu and Zhao, Xiao-Kun and Guo, Min and Hu, Han-Shi and Li, Jun and Chen, Ning},
  doi          = {10.1038/s41557-025-01856-2},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1364-1370},
  shortjournal = {Nat. Chem.},
  title        = {A lanthanide–carbon triple bond stabilized within a fullerene cage},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Zigzag graphene nanoribbons with periodic porphyrin edge extensions. <em>NCHEM</em>, <em>17</em>(9), 1356-1363. (<a href='https://doi.org/10.1038/s41557-025-01887-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Graphene nanoribbons (GNRs) with zigzag edges are promising materials for spintronic devices due to tunable bandgaps and spin-polarized edge states. Porphyrins offer complementary benefits such as desirable optoelectronic properties. Here we combine these features in a hybrid system by means of the on-surface synthesis of zigzag-edge GNRs embedded with porphyrins laterally fused along the ribbon backbone. Using scanning probe methods, we show that this design achieves strong electronic coupling between the porphyrin and the GNR. For transition metal porphyrins, pronounced exchange coupling between distant metal centres is mediated by the π-electron system. Such a hybrid d and π electron ribbon system introduces spin–orbit coupling and magnetic anisotropy to carbon nanomaterials, and holds great promise for coherent electrical control of electron spins. Graphene nanoribbons with zigzag edges are key candidates for spintronic applications due to their tunable bandgaps and spin-polarized edge states. Now it has been shown that hybrid ribbons embedded with metalloporphyrins enable strong electronic and magnetic coupling between distant metal centres, positioning such hybrids as promising materials for quantum devices.},
  archive      = {J_NCHEM},
  author       = {Xiang, Feifei and Gu, Yanwei and Kinikar, Amogh and Bassi, Nicolò and Ortega-Guerrero, Andres and Qiu, Zijie and Gröning, Oliver and Ruffieux, Pascal and Pignedoli, Carlo A. and Müllen, Klaus and Fasel, Roman},
  doi          = {10.1038/s41557-025-01887-9},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1356-1363},
  shortjournal = {Nat. Chem.},
  title        = {Zigzag graphene nanoribbons with periodic porphyrin edge extensions},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Metal-centred states control carrier lifetimes in transition metal oxide photocatalysts. <em>NCHEM</em>, <em>17</em>(9), 1348-1355. (<a href='https://doi.org/10.1038/s41557-025-01868-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efficient sunlight-to-energy conversion requires materials that can generate long-lived charge carriers upon illumination. However, the targeted design of semiconductors possessing intrinsically long lifetimes remains a key challenge. Here using a series of transition metal oxides, we establish a link between carrier lifetime and electronic configuration in transition metal-based semiconductors. We identify a subpicosecond relaxation mechanism via metal-centred ligand field states that compromise quantum yields in open d-shell transition metal oxides (for example, Fe2O3, Co3O4, Cr2O3 and NiO), which is more reminiscent of molecular complexes than crystalline semiconductors. We found that materials with spin-forbidden ligand field transitions could partially mitigate this relaxation pathway, explaining why Fe2O3 achieves higher photoelectrochemical activity than other visible light-absorbing transition metal oxides. However, achieving high yields of long-lived charges requires transition metal oxides with d0 or d10 electronic configurations (for example, TiO2 and BiVO4), where ligand field states are absent. These trends translate to transition metal-containing semiconductors beyond oxides, enabling the design of photoabsorbers with better-controlled recombination channels in photovoltaics, photocatalysis and communication devices. Efficient sunlight-to-fuel conversion requires materials that produce long-lived charges, but increasing carrier lifetimes in visible-light-active photocatalysts has proven difficult. Now it has been shown that rapid charge deactivation via metal-centred states prevents long lifetimes and limits photocatalytic performance. These insights bridge solid-state and molecular photochemistry, informing strategies for improved photocatalyst design.},
  archive      = {J_NCHEM},
  author       = {Sachs, Michael and Harnett-Caulfield, Liam and Pastor, Ernest and Davies, Bernadette and Sowood, Daniel J. C. and Moss, Benjamin and Kafizas, Andreas and Nelson, Jenny and Walsh, Aron and Durrant, James R.},
  doi          = {10.1038/s41557-025-01868-y},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1348-1355},
  shortjournal = {Nat. Chem.},
  title        = {Metal-centred states control carrier lifetimes in transition metal oxide photocatalysts},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On-DNA C–H functionalization of electron-rich arenes for DNA-encoded libraries. <em>NCHEM</em>, <em>17</em>(9), 1340-1347. (<a href='https://doi.org/10.1038/s41557-025-01844-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA-encoded libraries (DELs) are useful for hit discovery in the pharmaceutical industry. Although a large number of individually coded molecules are accessible through DELs, their structural diversity is limited because few transformations are benign and chemoselective enough to be applied in the presence of DNA in aqueous environments. In particular, C–H functionalization chemistry that could be ideally suited to increase structural diversity through late-stage functionalization is currently absent from DEL synthesis. Here we present a general C–H functionalization of electron-rich arenes on DNA. The development of a selenoxide reagent is key to achieving the regio- and chemoselective formation of arylselenonium salts in aqueous media. The introduction of arylselenonium salts offers a versatile linchpin on DNA conjugates, which gives access to a multitude of analogues through diverse subsequent reactions, including transition-metal-mediated and photochemical transformations for the formation of C–C, C–I and C–S bonds. C–H functionalization is a key reaction in organic synthesis. Now a selenoxide reagent has been developed for the formation of arylselenonium salts via C–H functionalization of DNA conjugates with high selectivity. The arylselenonium salts participate in various transformations to forge new C–C and C–X bonds.},
  archive      = {J_NCHEM},
  author       = {de Pedro Beato, Eduardo and Torkowski, Luca and Hartmann, Philipp and Vogelsang, Lara and Dietz, Karl-Josef and Ritter, Tobias},
  doi          = {10.1038/s41557-025-01844-6},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1340-1347},
  shortjournal = {Nat. Chem.},
  title        = {On-DNA C–H functionalization of electron-rich arenes for DNA-encoded libraries},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A selenoxide for single-atom protein modification of tyrosine residues enabled by water-resistant chalcogen and hydrogen bonding. <em>NCHEM</em>, <em>17</em>(9), 1331-1339. (<a href='https://doi.org/10.1038/s41557-025-01842-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modifications such as phosphorylation and acetylation are often minor structural modifications that can have profound effects on protein structure and thus broaden protein functions. Nevertheless, studying these effects directly is often out of reach because no general chemistry exists to introduce small modifications selectively; either a large, stable linker structure is selectively installed on protein residues, or a small substituent is introduced at the risk of low selectivity due to the use of reactive, indiscriminate molecules. Here we report a C–H functionalization reaction of tyrosine residues to access peptides and proteins modified by small structural changes including single-atom substitutions. A rationally designed selenoxide introduces a versatile selenonium linchpin featuring a Ctyr–Se bond that can be used for further transformations at specific tyrosine residues. Key to the advance is the interplay of water-resistant, intramolecular chalcogen and hydrogen bonding of the selenoxide reagent, which allows chemo- and site-selective electrophilic aromatic substitution of tyrosine residues in aqueous solutions. Proteins with small structural modifications at specific sites are valuable, yet challenging to access by chemical methods. Now, tyrosine-selective single-atom modifications on proteins have been achieved by C–H functionalization using a rationally designed selenoxide to introduce a versatile selenonium linchpin for further transformations.},
  archive      = {J_NCHEM},
  author       = {Lin, Songyun and Hirao, Marina and Hartmann, Philipp and Leutzsch, Markus and Sterling, Marie Sophie and Vetere, Alessandro and Klimmek, Sandra and Hinrichs, Heike and Mengeler, Johanna Marie and Lehmann, Johannes and Samsonowicz-Gόrski, Jan and Berger, Florian and Ritter, Tobias},
  doi          = {10.1038/s41557-025-01842-8},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1331-1339},
  shortjournal = {Nat. Chem.},
  title        = {A selenoxide for single-atom protein modification of tyrosine residues enabled by water-resistant chalcogen and hydrogen bonding},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modular alkyl growth in amines via the selective insertion of alkynes into C–C bonds. <em>NCHEM</em>, <em>17</em>(9), 1323-1330. (<a href='https://doi.org/10.1038/s41557-025-01849-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The site-specific modification of amines has been a highly sought-after objective in organic synthesis. Despite the rapid advancement of carbon–hydrogen (C–H) bond functionalization methods, effective strategies for carbon–carbon (C–C) bond functionalization of amines remain elusive. Here we report a borane-catalysed method for the selective insertion of alkynes into alkyl C–C bonds of amines, resulting in the ring expansion of cyclic amines and chain elongation of acyclic amines. This approach begins with the cleavage of C–H bonds in amines, then transitioning to C–C bond functionalization upon reaction with alkynes. This method is effective with amines lacking an activating or leaving group and is suitable for late-stage functionalization of pharmaceuticals through C–C bond modification. Furthermore, by coupling this reaction with hydrolysis and hydrogenation steps, successive alkyne insertions are achieved, enabling modular and iterative alkyl growth of amines. The functionalization of carbon–carbon bonds in amines remains an important challenge in organic synthesis. Now a borane-catalysed method has been developed that enables the insertion of alkynes into the alkyl C–C bonds of amines, providing a versatile approach for the ring expansion of cyclic amines and chain elongation of acyclic amines.},
  archive      = {J_NCHEM},
  author       = {Zhou, Xin-Yue and Liu, Lu and Lyu, Hairong and Wang, Xiao-Chen},
  doi          = {10.1038/s41557-025-01849-1},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1323-1330},
  shortjournal = {Nat. Chem.},
  title        = {Modular alkyl growth in amines via the selective insertion of alkynes into C–C bonds},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Solid solvation structure design improves all-solid-state organic batteries. <em>NCHEM</em>, <em>17</em>(9), 1313-1322. (<a href='https://doi.org/10.1038/s41557-025-01866-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organic electrode materials offer a versatile, sustainable approach for next-generation lithium-ion batteries but are limited by low working voltages and poor cycling stability. Here we report a solid-solvation-structure design strategy to improve both the voltage and stability of organic electrode materials in all-solid-state batteries. As a proof of concept, we incorporate halide electrolytes as solid solutes and tetrachloro-o-benzoquinone as a solid solvent to form homogeneous solid cathode solutions. Systematic optimization of the inner solvation configuration enables tetrachloro-o-benzoquinone to achieve a high working voltage (3.6 V vs. Li+/Li) at room temperature within an asymmetric solid solvation sheath. Moreover, the equilibrium redox pathway and electrostatically driven self-healing interfaces revealed rapid redox kinetics and stable performance over 7,500 cycles in all-solid-state batteries under low stack pressures. This work demonstrates that organic electrode materials can serve as viable, durable and cost-effective alternatives to transition metal oxides in all-solid-state batteries. Organic electrode materials offer a versatile and sustainable route for lithium-ion batteries, but their application is hindered by low working voltages and poor cycling stability. Now, it has been shown that dissolving halide electrolytes into organic electrodes effectively tunes the working voltage and enhances the electrochemical performance of all-solid-state batteries.},
  archive      = {J_NCHEM},
  author       = {Hu, Yang and Su, Han and Fu, Jiamin and Luo, Jing and Yu, Qihang and Zhao, Feipeng and Li, Weihan and Deng, Sixu and Liu, Yu and Yuan, Yi and Gan, Yi and Wang, Yijia and Kim, Jung Tae and Chen, Ning and Shakouri, Mohsen and Hao, Xiaoge and Gao, Yingjie and Pang, Tianlu and Zhang, Nian and Jiang, Ming and Li, Xia and Zhao, Yang and Tu, Jiangping and Wang, Changhong and Sun, Xueliang},
  doi          = {10.1038/s41557-025-01866-0},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1313-1322},
  shortjournal = {Nat. Chem.},
  title        = {Solid solvation structure design improves all-solid-state organic batteries},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An enzyme for high-yield, ATP-driven C-terminal thioester generation. <em>NCHEM</em>, <em>17</em>(9), 1311-1312. (<a href='https://doi.org/10.1038/s41557-025-01872-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current tools for protein bioconjugation at the C-terminal have limited yields. Now, an enzymatic strategy for ATP-dependent activation of protein and peptide C termini has been developed. This versatile tool can be deployed for synthesis of C-terminal thioesters that enhance the yield and accessibility of diverse protein bioconjugation methods.},
  archive      = {J_NCHEM},
  doi          = {10.1038/s41557-025-01872-2},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1311-1312},
  shortjournal = {Nat. Chem.},
  title        = {An enzyme for high-yield, ATP-driven C-terminal thioester generation},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stabilizing a Ce≡C triple bond through fullerene confinement. <em>NCHEM</em>, <em>17</em>(9), 1309-1310. (<a href='https://doi.org/10.1038/s41557-025-01919-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The formation of lanthanide–carbon triple bonds is generally considered unfeasible. Now, a cerium–carbon triple bond is stabilized through encapsulation within an endohedral fullerene cage, suggesting a revision to the fundamental concepts of lanthanide bonding theory.},
  archive      = {J_NCHEM},
  doi          = {10.1038/s41557-025-01919-4},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1309-1310},
  shortjournal = {Nat. Chem.},
  title        = {Stabilizing a Ce≡C triple bond through fullerene confinement},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Merging d- and π-electron magnetism. <em>NCHEM</em>, <em>17</em>(9), 1307-1308. (<a href='https://doi.org/10.1038/s41557-025-01923-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of materials with tunable electronic and magnetic properties contributes to the advancement of spintronic technologies and optoelectronics. Now, the combination of π- and d-electron magnetism has been achieved through the lateral fusion of periodic metalloporphyrins along the zigzag edges of graphene nanoribbons.},
  archive      = {J_NCHEM},
  author       = {Song, Shaotang and Lu, Jiong},
  doi          = {10.1038/s41557-025-01923-8},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1307-1308},
  shortjournal = {Nat. Chem.},
  title        = {Merging d- and π-electron magnetism},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Selenoxides unlock C–H bonds of biomacromolecules. <em>NCHEM</em>, <em>17</em>(9), 1304-1306. (<a href='https://doi.org/10.1038/s41557-025-01921-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Direct C–H selenylation of biomacromolecules under biocompatible conditions has long posed a formidable challenge. Now, engineered selenoxide reagents have been shown to achieve site-specific incorporation of selenium into DNA and proteins within physiologically relevant environments.},
  archive      = {J_NCHEM},
  author       = {Fang, Xianfu and Li, Yizhou},
  doi          = {10.1038/s41557-025-01921-w},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1304-1306},
  shortjournal = {Nat. Chem.},
  title        = {Selenoxides unlock C–H bonds of biomacromolecules},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A rationale for charge carrier lifetimes in transition metal semiconductors. <em>NCHEM</em>, <em>17</em>(9), 1301-1303. (<a href='https://doi.org/10.1038/s41557-025-01913-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-lived charge carriers in semiconductors are essential for effective sunlight-to-energy conversion. Still, few studies have fundamentally rationalized the charge carrier lifetimes. Now, metal-centred ligand field states of transition metal oxides have been suggested to control carrier lifetimes, akin to those for molecular photosensitizers.},
  archive      = {J_NCHEM},
  author       = {Bourgois, Céline and Troian-Gautier, Ludovic},
  doi          = {10.1038/s41557-025-01913-w},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1301-1303},
  shortjournal = {Nat. Chem.},
  title        = {A rationale for charge carrier lifetimes in transition metal semiconductors},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ring expansion and chain extension of alkyl amines. <em>NCHEM</em>, <em>17</em>(9), 1299-1300. (<a href='https://doi.org/10.1038/s41557-025-01920-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amines are highly valued molecules, used to understand biological processes and in the discovery of new medicines. However, medium-sized cyclic and macrocyclic amines are extremely challenging to synthesize and are therefore underutilized. Now, up to 15-membered cyclic amines can be formed directly from smaller, more accessible rings via an organoborane-catalysed two-carbon-unit C–C bond insertion.},
  archive      = {J_NCHEM},
  author       = {Barysevich, Maryia and Pulis, Alexander P.},
  doi          = {10.1038/s41557-025-01920-x},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1299-1300},
  shortjournal = {Nat. Chem.},
  title        = {Ring expansion and chain extension of alkyl amines},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Agree to disagree. <em>NCHEM</em>, <em>17</em>(9), 1297-1298. (<a href='https://doi.org/10.1038/s41557-025-01918-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Science is about discovering new knowledge, so, logically, there will be disagreement. Shira Joudan contemplates how disagreements can be useful, and how to deal with them when they arise.},
  archive      = {J_NCHEM},
  author       = {Joudan, Shira},
  doi          = {10.1038/s41557-025-01918-5},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1297-1298},
  shortjournal = {Nat. Chem.},
  title        = {Agree to disagree},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="ng">NG - 34</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48. <em>NG</em>, <em>57</em>(9), 2340-2342. (<a href='https://doi.org/10.1038/s41588-025-02332-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Riazuddin, Saima and Belyantseva, Inna A. and Giese, Arnaud P. J. and Lee, Kwanghyuk and Indzhykulian, Artur A. and Nandamuri, Sri Pratima and Yousaf, Rizwan and Sinha, Ghanshyam P. and Lee, Sue and Terrell, David and Hegde, Rashmi S. and Ali, Rana A. and Anwar, Saima and Andrade-Elizondo, Paula B. and Sirmaci, Asli and Parise, Leslie V. and Basit, Sulman and Wali, Abdul and Ayub, Muhammad and Ansar, Muhammad and Ahmad, Wasim and Khan, Shaheen N. and Akram, Javed and Tekin, Mustafa and Riazuddin, Sheikh and Cook, Tiffany and Buschbeck, Elke K. and Frolenkov, Gregory I. and Leal, Suzanne M. and Friedman, Thomas B. and Ahmed, Zubair M.},
  doi          = {10.1038/s41588-025-02332-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2340-2342},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation. <em>NG</em>, <em>57</em>(9), 2339. (<a href='https://doi.org/10.1038/s41588-025-02328-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Uryu, Hidetaka and Saeki, Koichi and Haeno, Hiroshi and Kapadia, Chiraag Deepak and Furudate, Ken and Nangalia, Jyoti and Spencer Chapman, Michael and Zhang, Linda and Padilla, Jennifer and Zhao, Li and Hsu, Joanne I. and Zhao, Chong and Chen, Shujuan and Tanaka, Tomoyuki and Li, Zongrui and Ogata, Satoko and Hanache, Sarah and Yang, Hui and DiNardo, Courtney and Daver, Naval and Pemmaraju, Naveen and Jain, Nitin and Ravandi, Farhad and Zhang, Jianhua and Song, Xingzhi and Thompson, Erika and Tang, Hongli and Little, Latasha and Gumbs, Curtis and Orlowski, Robert Z. and Qazilbash, Muzaffar and Bhalla, Kapil and Colla, Simona and Kantarjian, Hagop and Kanagal-Shamanna, Rashmi and Bueso-Ramos, Carlos and Nakada, Daisuke and Al-Atrash, Gheath and Molldrem, Jeffery and Futreal, P. Andrew and Shpall, Elizabeth and Goodell, Margaret and Garcia-Manero, Guillermo and Takahashi, Koichi},
  doi          = {10.1038/s41588-025-02328-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2339},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma. <em>NG</em>, <em>57</em>(9), 2338. (<a href='https://doi.org/10.1038/s41588-025-02320-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gundem, Gunes and Levine, Max F. and Roberts, Stephen S. and Cheung, Irene Y. and Medina-Martínez, Juan S. and Feng, Yi and Arango-Ossa, Juan E. and Chadoutaud, Loic and Rita, Mathieu and Asimomitis, Georgios and Zhou, Joe and You, Daoqi and Bouvier, Nancy and Spitzer, Barbara and Solit, David B. and Dela Cruz, Filemon and LaQuaglia, Michael P. and Kushner, Brian H. and Modak, Shakeel and Shukla, Neerav and Iacobuzio-Donahue, Christine A. and Kung, Andrew L. and Cheung, Nai-Kong V. and Papaemmanuil, Elli},
  doi          = {10.1038/s41588-025-02320-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2338},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states. <em>NG</em>, <em>57</em>(9), 2323-2334. (<a href='https://doi.org/10.1038/s41588-025-02284-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell atlas projects have revealed that common cell types often comprise distinct, recurrent transcriptional states, but the function and regulation of these states remain poorly understood. Here, we show that systematic activation of transcription factors can recreate such states in vitro, providing tractable models for mechanistic and functional studies. Using a scalable CRISPR activation (CRISPRa) Perturb-seq platform, we activated 1,836 transcription factors in two cell types. CRISPRa induced gene expression within physiological ranges, with chromatin features predicting responsiveness. Comparisons with atlas datasets showed that transcription factor perturbations recapitulated key fibroblast states and identified their regulators, including KLF2 and KLF4 for a universal state present in many tissues, and PLAGL1 for a disease-associated inflammatory state. Inducing the universal state suppressed the inflammatory state, suggesting therapeutic potential. These findings position CRISPRa as a nuanced tool for perturbing differentiated cells and establish a general strategy for studying clinically relevant transcriptional states ex vivo. CRISPR activation of 1,836 human transcription factors recapitulates fibroblast transcriptional states observed in vivo and identifies regulators that can revert inflammatory states.},
  archive      = {J_NG},
  author       = {Southard, Kaden M. and Ardy, Rico C. and Tang, Anran and O’Sullivan, Deirdre D. and Metzner, Eli and Guruvayurappan, Karthik and Norman, Thomas M.},
  doi          = {10.1038/s41588-025-02284-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2323-2334},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize. <em>NG</em>, <em>57</em>(9), 2313-2322. (<a href='https://doi.org/10.1038/s41588-025-02246-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive maps of functional variation at transcription factor (TF) binding sites (cis-elements) are crucial for elucidating how genotype shapes phenotype. Here, we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped over 200,000 variants, genetic, epigenetic, or both (termed binding quantitative trait loci (bQTL)), linked to cis-element occupancy. Three lines of evidence support the functional significance of bQTL: (1) coincidence with causative loci that regulate traits, including vgt1, ZmTRE1 and the MITE transposon near ZmNAC111 under drought; (2) bQTL allelic bias is shared between inbred parents and matches chromatin immunoprecipitation sequencing results; and (3) partitioning genetic variation across genomic regions demonstrates that bQTL capture the majority of heritable trait variation across ~72% of 143 phenotypes. Our study provides an auspicious approach to make functional cis-variation accessible at scale for genetic studies and targeted engineering of complex traits. Pan-cistrome of the maize leaf under well-watered and drought conditions profiled by haplotype-specific MOA-seq highlights the relevance of transcription factor binding QTLs for understanding phenotypic diversity in maize.},
  archive      = {J_NG},
  author       = {Engelhorn, Julia and Snodgrass, Samantha J. and Kok, Amelie and Seetharam, Arun S. and Schneider, Michael and Kiwit, Tatjana and Singh, Ayush and Banf, Michael and Doan, Duong Thi Hai and Khaipho-Burch, Merritt and Runcie, Daniel E. and Sánchez-Camargo, Victor A. and Bader, Rechien and Vladimir Torres-Rodriguez, J. and Sun, Guangchao and Stam, Maike and Fiorani, Fabio and Beier, Sebastian and Schnable, James C. and Bass, Hank W. and Hufford, Matthew B. and Stich, Benjamin and Frommer, Wolf B. and Ross-Ibarra, Jeffrey and Hartwig, Thomas},
  doi          = {10.1038/s41588-025-02246-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2313-2322},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants. <em>NG</em>, <em>57</em>(9), 2302-2312. (<a href='https://doi.org/10.1038/s41588-025-02306-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clubroot disease, caused by the obligate intracellular rhizarian protist Plasmodiophora brassicae, is devastating to cruciferous crops worldwide. Widespread field P. brassicae pathotypes frequently overcome the pathotype-specific resistance of modern varieties, posing a challenge for durable control of this disease. Here a genome-wide association study of 3 years of data comprising field clubroot phenotyping of 244 genome-resequenced Brassica napus accessions identified a strong association of β-1,3-glucan synthase-like 5 (GSL5) with clubroot susceptibility. GSL5 was evolutionarily conserved, and inactivation of GSL5 by genome editing in Arabidopsis, B. napus, Brassica rapa and Brassica oleracea conferred broad-spectrum, high-level resistance to P. brassicae pathotypes without yield penalties in B. napus. GSL5 inactivation derepressed the jasmonic acid-mediated immunity during P. brassicae secondary infection, and this immune repression was possibly reinforced through stabilization of GSL5 by a P. brassicae effector, facilitating clubroot susceptibility. Our study provides durable resistance resources for cruciferous clubroot disease control and insights into plant resistance against intracellular eukaryotic phytopathogens. This study implicates GSL5 inactivation in high, broad-spectrum resistance to the clubroot pathogen Plasmodiophora brassicae in Arabidopsis thaliana, Brassica napus, Brassica oleracea and Brassica rapa.},
  archive      = {J_NG},
  author       = {Wu, Yupo and Zhao, Chuanji and Zhang, Yi and Shen, Cuicui and Zhang, Yuanyuan and Zhang, Xiong and Gao, Lixia and Zeng, Lingyi and Ke, Qinglin and Qin, Li and Liu, Fan and Huang, Junyan and Ren, Li and Liu, Yueying and Cheng, Hongtao and Tong, Chaobo and Hu, Qiong and Cheng, Xiaohui and Wei, Yangdou and Liu, Shengyi and Liu, Lijiang},
  doi          = {10.1038/s41588-025-02306-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2302-2312},
  shortjournal = {Nat. Genet.},
  title        = {Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism. <em>NG</em>, <em>57</em>(9), 2289-2301. (<a href='https://doi.org/10.1038/s41588-025-02293-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Making sense of whole-genome polymorphism data is challenging, but it is essential for overcoming the biases in SNP data. Here we analyze 27 genomes of Arabidopsis thaliana to illustrate these issues. Genome size variation is mostly due to tandem repeat regions that are difficult to assemble. However, while the rest of the genome varies little in length, it is full of structural variants, mostly due to transposon insertions. Because of this, the pangenome coordinate system grows rapidly with sample size and ultimately becomes 70% larger than the size of any single genome, even for n = 27. Finally, we show how short-read data are biased by read mapping. SNP calling is biased by the choice of reference genome, and both transcriptome and methylome profiling results are affected by mapping reads to a reference genome rather than to the genome of the assayed individual. New concepts for comparing the genomes of 27 naturally inbred Arabidopsis thaliana accessions provide essential insights into obtaining a less biased view of whole-genome polymorphism.},
  archive      = {J_NG},
  author       = {Igolkina, Anna A. and Vorbrugg, Sebastian and Rabanal, Fernando A. and Liu, Hai-Jun and Ashkenazy, Haim and Kornienko, Aleksandra E. and Fitz, Joffrey and Collenberg, Max and Kubica, Christian and Mollá Morales, Almudena and Jaegle, Benjamin and Wrightsman, Travis and Voloshin, Vitaly and Bezlepsky, Alexander D. and Llaca, Victor and Nizhynska, Viktoria and Reichardt, Ilka and Bezrukov, Ilja and Lanz, Christa and Bemm, Felix and Flood, Pádraic J. and Nemomissa, Sileshi and Hancock, Angela and Guo, Ya-Long and Kersey, Paul and Weigel, Detlef and Nordborg, Magnus},
  doi          = {10.1038/s41588-025-02293-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2289-2301},
  shortjournal = {Nat. Genet.},
  title        = {A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity. <em>NG</em>, <em>57</em>(9), 2276-2288. (<a href='https://doi.org/10.1038/s41588-025-02294-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common oat, belonging to the genus Avena with 30 recognized species, is a nutritionally important cereal crop and high-quality forage worldwide. Here, we construct a genus-level super-pangenome of Avena comprising 35 high-quality genomes from 14 cultivated oat accessions and 21 wild species. The fully resolved phylogenomic analysis unveils the origin and evolutionary scenario of Avena species, and the super-pangenome analysis identifies 26.62% and 59.93% specific genes and haplotypes in wild species. We delineate the landscape of structural variations (SVs) and the transcriptome profile based 1,401 RNA-sequencing (RNA-seq) samples from diverse abiotic stress treatments in oat. We highlight the crucial role of SVs in modulating gene expression and shaping adaptation to diverse stresses. Further combining SV-based genome-wide association studies (GWASs), we characterize 13 candidate genes associated with drought resistance such as AsARF7, validated by transgenic oat lines. Our study provides unprecedented genomic resources to facilitate genomic, evolution and molecular breeding research in oat. A genus-level super-pangenome of Avena comprising 35 high-quality genomes from 23 cultivated or wild oat species highlights the evolution of Avena and the landscape of structural variations related to abiotic stress resistance.},
  archive      = {J_NG},
  author       = {Zhang, Hongyu and Liu, Ningkun and Wang, Yaru and Zheng, Xinyuan and Li, Wei and Liu, Ze and Liu, Jianan and Wang, Yu and Xing, Longsheng and Li, Tao and Yun, Yange and Zhou, Qinghong and Wang, Meijia and Qin, Yujie and Yan, Jinjiang and Gong, Zhizhong and He, Qiang and Du, Huilong},
  doi          = {10.1038/s41588-025-02294-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2276-2288},
  shortjournal = {Nat. Genet.},
  title        = {Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts. <em>NG</em>, <em>57</em>(9), 2264-2275. (<a href='https://doi.org/10.1038/s41588-025-02330-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Locusts are characterized by a large genome size, polyphenism and an X0 sex determination system. Here we generated chromosome-level genomes for both desert and migratory locusts, as well as a comprehensive chromatin map for the latter. We found that genome enlargement is associated with an increased number of enhancers in expanded intronic regions. To explore the function of expanded enhancers, we identified a distal enhancer that contributes to behavioral differences between solitary and gregarious locusts. In the X0 sex system, H4K16ac enrichment and H4K20me1 depletion maintain balanced X-linked expression in male soma. Notably, the distance-dependent H4K16ac enrichment diminishes gradually in intergenic regions, revealing a special dosage compensation mechanism in large genomes. Furthermore, the distance-dependent H4K16ac results in a lag in the evolution of dosage compensation for the X-linked genes translocated recently from autosomes. Therefore, expanded intronic and intergenic regions shape a distinct chromatin regulation landscape in large genomes. Chromosome-level genome assemblies of migratory and desert locusts, coupled with epigenomic profiling of migratory locusts, reveal chromatin dynamics underlying polyphenism and X-linked dosage compensation following autosomal gene translocation.},
  archive      = {J_NG},
  author       = {Liu, Qing and Jiang, Feng and Li, Ran and Liu, Shanlin and Fang, Wenjing and Li, Xiao and Hu, Ting and Feng, Lianyun and Zhang, Jie and Liu, Zhikang and He, Jing and Guo, Wei and Wang, Xianhui and Sun, Zongyi and Li, Jingjing and Gao, Yunan and Yi, Jiacheng and Li, Qiye and Wang, Xiaoxiao and Wei, Liya and Kang, Le},
  doi          = {10.1038/s41588-025-02330-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2264-2275},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. <em>NG</em>, <em>57</em>(9), 2250-2263. (<a href='https://doi.org/10.1038/s41588-025-02305-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian switch/sucrose nonfermentable (mSWI/SNF) complex regulates chromatin accessibility and frequently shows alterations due to mutation in cancer and neurological diseases. Inadequate expression of mSWI/SNF in heterozygous mice can lead to developmental defects, indicating dosage-sensitive effects of mSWI/SNF. However, how its dosage affects function has remained unclear. Using a targeted protein degradation system, we investigated its dosage-sensitive effects by precisely controlling protein levels of BRG1, the ATPase subunit of the mSWI/SNF complex. We found that binding of BRG1 to chromatin exhibited a linear response to the BRG1 protein level. Although chromatin accessibility at most promoters and insulators was largely unaffected by BRG1 depletion, 44% of enhancers, including 84% of defined superenhancers, showed reduced accessibility. Notably, half of the BRG1-regulated enhancers, particularly superenhancers, exhibited a buffered response to BRG1 loss. Consistently, transcription exhibited a predominantly buffered response to changes in BRG1 levels. Collectively, our findings demonstrate a genomic feature-specific response to BRG1 dosage, shedding light on the dosage-sensitive effects of mSWI/SNF complex defects in cancer and other diseases. This study investigates the dosage-sensitive effects of BRG1, a core SWI/SNF subunit, on chromatin binding, accessibility and transcriptional regulation by precisely controlling its protein level.},
  archive      = {J_NG},
  author       = {Hagihara, Yota and Zhang, Chao and Zhang, Yi},
  doi          = {10.1038/s41588-025-02305-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2250-2263},
  shortjournal = {Nat. Genet.},
  title        = {Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue. <em>NG</em>, <em>57</em>(9), 2238-2249. (<a href='https://doi.org/10.1038/s41588-025-02300-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, TET1 and DNMT3A are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome. This multiomic study, including single-nucleus DNA methylation and chromatin conformation matched with single-nuclei RNA sequencing, provides insights into the epigenomic landscape of human subcutaneous adipose tissue.},
  archive      = {J_NG},
  author       = {Chen, Zeyuan Johnson and Das, Sankha Subhra and Kar, Asha and Lee, Seung Hyuk T. and Abuhanna, Kevin D. and Alvarez, Marcus and Sukhatme, Mihir G. and Wang, Zitian and Gelev, Kyla Z. and Heffel, Matthew G. and Zhang, Yi and Avram, Oren and Rahmani, Elior and Sankararaman, Sriram and Laakso, Markku and Heinonen, Sini and Peltoniemi, Hilkka and Halperin, Eran and Pietiläinen, Kirsi H. and Luo, Chongyuan and Pajukanta, Päivi},
  doi          = {10.1038/s41588-025-02300-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2238-2249},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution. <em>NG</em>, <em>57</em>(9), 2226-2237. (<a href='https://doi.org/10.1038/s41588-025-02307-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients. Integrated multi-omic analyses using samples from the TRACERx study highlight cross-talk between DNA hypermethylation and genomic lesions in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Gimeno-Valiente, Francisco and Castignani, Carla and Larose Cadieux, Elizabeth and Mensah, Nana E. and Liu, Xiaohong and Chen, Kezhong and Chervova, Olga and Karasaki, Takahiro and Weeden, Clare E. and Richard, Corentin and Lai, Siqi and Martínez-Ruiz, Carlos and Lim, Emilia L. and Frankell, Alexander M. and Watkins, Thomas B. K. and Stavrou, Georgia and Usaite, Ieva and Lu, Wei-Ting and Marinelli, Daniele and Saghafinia, Sadegh and Wilson, Gareth A. and Dhami, Pawan and Vaikkinen, Heli and Steif, Jonathan and Veeriah, Selvaraju and Hynds, Robert E. and Hirst, Martin and Hiley, Crispin and Feber, Andrew and Deniz, Özgen and Jamal-Hanjani, Mariam and McGranahan, Nicholas and Beck, Stephan and Demeulemeester, Jonas and Tanić, Miljana and Swanton, Charles and Van Loo, Peter and Kanu, Nnennaya},
  doi          = {10.1038/s41588-025-02307-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2226-2237},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis. <em>NG</em>, <em>57</em>(9), 2215-2225. (<a href='https://doi.org/10.1038/s41588-025-02296-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in PPM1D and the TERT gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers. This study explores the relationship between telomere length and clonal hematopoiesis. Splicing factor and PPM1D gene mutations are more frequent in people with genetically predicted shorter telomere lengths, suggesting that these mutations protect against the consequences of telomere attrition.},
  archive      = {J_NG},
  author       = {McLoughlin, Matthew A. and Cheloor Kovilakam, Sruthi and Dunn, William G. and Gu, Muxin and Tobin, Jake and Pershad, Yash and Williams, Nicholas and Leongamornlert, Daniel and Dawson, Kevin and Bond, Laura and Marando, Ludovica and Wen, Sean and Wilson, Rachael and Valenzano, Giampiero and Symeonidou, Vasiliki and Rak, Justyna and Damaskou, Aristi and Gozdecka, Malgorzata and Liu, Xiaoxuan and Barcena, Clea and Nomdedeu, Josep and Costeas, Paul and Dimitriou, Ioannis D. and Fiorillo, Edoardo and Orrù, Valeria and de Almeida, Jose Guilherme and McKerrell, Thomas and Cullen, Matthew and Mohorianu, Irina and Foukaneli, Theodora and Warren, Alan J. and Wong, Chi and Follows, George and Godfrey, Anna L. and Gudgin, Emma and Cucca, Francesco and McKinney, Eoin and Baxter, E. Joanna and Gerstung, Moritz and Mitchell, Jonathan and Wiseman, Daniel and Bick, Alexander G. and Fabre, Margarete and Quiros, Pedro M. and Nangalia, Jyoti and Kar, Siddhartha and Vassiliou, George S.},
  doi          = {10.1038/s41588-025-02296-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2215-2225},
  shortjournal = {Nat. Genet.},
  title        = {Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Temporal genomic dynamics shape clinical trajectory in multiple myeloma. <em>NG</em>, <em>57</em>(9), 2203-2214. (<a href='https://doi.org/10.1038/s41588-025-02292-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple myeloma evolution is characterized by the accumulation of genomic drivers over time. To unravel this timeline and its impact on clinical outcomes, we analyzed 421 whole-genome sequences from 382 patients. Using clock-like mutational signatures, we estimated a time lag of two to four decades between the initiation of events and diagnosis. We demonstrate that odd-numbered chromosome trisomies in patients with hyperdiploidy can be acquired simultaneously with other chromosomal gains (for example, 1q gain). We show that hyperdiploidy is acquired after immunoglobulin heavy chain translocation when both events co-occur. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (&gt;1 extra copy), but fared worse than those with late 1q gain. This finding underscores that the 1q gain prognostic impact depends more on the timing of acquisition than on the number of copies gained. Overall, this study contributes to a better understanding of the life history of myeloma and may have prognostic implications. This analysis of whole-genome sequencing data from 421 multiple myeloma samples elucidates the timing of key genomic events and shows associations between the timing of 1q gain and clinical outcome.},
  archive      = {J_NG},
  author       = {Maura, Francesco and Kaddoura, Marcella and Poos, Alexandra M. and Baughn, Linda B. and Ziccheddu, Bachisio and Bärtsch, Marc-Andrea and Cirrincione, Anthony and Maclachlan, Kylee and Chojnacka, Monika and Diamond, Benjamin and Papadimitriou, Marios and Blaney, Patrick and John, Lukas and Reichert, Philipp and Huhn, Stefanie and Gagler, Dylan and Zhang, Yanming and Dogan, Ahmet and Lesokhin, Alexander M. and Davies, Faith and Goldschmidt, Hartmut and Fenk, Roland and Weisel, Katja C. and Mai, Elias K. and Korde, Neha and Morgan, Gareth J. and Rajkumar, S. Vincent and Kumar, Shaji and Usmani, Saad and Landgren, Ola and Raab, Marc S. and Weinhold, Niels},
  doi          = {10.1038/s41588-025-02292-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2203-2214},
  shortjournal = {Nat. Genet.},
  title        = {Temporal genomic dynamics shape clinical trajectory in multiple myeloma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tamoxifen induces PI3K activation in uterine cancer. <em>NG</em>, <em>57</em>(9), 2192-2202. (<a href='https://doi.org/10.1038/s41588-025-02308-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutagenic processes and clonal selection contribute to the development of therapy-associated secondary neoplasms, a known complication of cancer treatment. The association between tamoxifen therapy and secondary uterine cancers is uncommon but well established; however, the genetic mechanisms underlying tamoxifen-driven tumorigenesis remain unclear. We find that oncogenic PIK3CA mutations, common in spontaneously arising estrogen-associated de novo uterine cancer, are significantly less frequent in tamoxifen-associated tumors. In vivo, tamoxifen-induced estrogen receptor stimulation activates phosphoinositide 3-kinase (PI3K) signaling in normal mouse uterine tissue, potentially eliminating the selective benefit of PI3K-activating mutations in tamoxifen-associated uterine cancer. Together, we present a unique pathway of therapy-associated carcinogenesis in which tamoxifen-induced activation of the PI3K pathway acts as a non-genetic driver event, contributing to the multistep model of uterine carcinogenesis. While this PI3K mechanism is specific to tamoxifen-associated uterine cancer, the concept of treatment-induced signaling events may have broader applicability to other routes of tumorigenesis. Sequencing analysis of tamoxifen-associated uterine cancers and further in vivo analyses suggest that the drug tamoxifen can activate the PI3K pathway in the absence of oncogenic mutations.},
  archive      = {J_NG},
  author       = {Kübler, Kirsten and Nardone, Agostina and Anand, Shankara and Gurevich, Daniel and Gao, Jianjiong and Droog, Marjolein and Hermida-Prado, Francisco and Akhshi, Tara and Feiglin, Ariel and Feit, Avery S. and Cohen Feit, Gabriella and Dackus, Gwen and Pun, Matthew and Kuang, Yanan and Cha, Justin and Miller, Mendy and Gregoricchio, Sebastian and Lanfermeijer, Mirthe and Cornelissen, Sten and Gibson, William J. and Paweletz, Cloud P. and Van Allen, Eliezer M. and van Leeuwen, Flora E. and Nederlof, Petra M. and Nguyen, Quang-Dé and Mourits, Marian J. E. and Radovich, Milan and Leshchiner, Ignaty and Stewart, Chip and Matulonis, Ursula A. and Zwart, Wilbert and Maruvka, Yosef E. and Getz, Gad and Jeselsohn, Rinath},
  doi          = {10.1038/s41588-025-02308-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2192-2202},
  shortjournal = {Nat. Genet.},
  title        = {Tamoxifen induces PI3K activation in uterine cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L. <em>NG</em>, <em>57</em>(9), 2177-2191. (<a href='https://doi.org/10.1038/s41588-025-02289-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the high prevalence of ERG transcription factor translocations in prostate cancer, the mechanism of tumorigenicity remains poorly understood. Using lineage tracing, we find the tumor-initiating activity of ERG resides in a subpopulation of murine basal cells that coexpress luminal genes (BasalLum) and not in the larger population of ERG+ luminal cells. Upon ERG activation, BasalLum cells give rise to highly proliferative intermediate (IM) cells with stem-like features that coexpress basal, luminal, hillock and club marker genes, before transitioning to Krt8+ luminal cells. Transcriptomic analysis of ERG+ human prostate cancers confirms the presence of rare ERG+ BasalLum cells, as well as IM cells whose presence is associated with a worse prognosis. Single-cell analysis revealed a chromatin state in ERG+ IM cells enriched for STAT3 transcription factor binding sites and elevated expression of the KMT2A/MLL1 and DOT1L, all three of which are essential for ERG-driven tumorigenicity in vivo. In addition to providing translational opportunities, this work illustrates how single-cell approaches combined with lineage tracing can identify cancer vulnerabilities not evident from bulk analysis. Lineage tracing in mice identifies a subpopulation of basal cells that express Tmprss2 and Nkx3 as the origin of ERG-driven prostate cancer. Upon expansion, these cells show an enrichment for STAT3 chromatin binding and elevated expression of KMT2A and DOT1L as dependencies for ERG oncogenicity.},
  archive      = {J_NG},
  author       = {Feng, Weiran and Ladewig, Erik and Lange, Matthew and Salsabeel, Nazifa and Zhao, Huiyong and Lee, Young Sun and Gopalan, Anuradha and Luo, Hanzhi and Kang, Wenfei and Fan, Ning and Rosiek, Eric and de Stanchina, Elisa and Chen, Yu and Carver, Brett S. and Leslie, Christina S. and Sawyers, Charles L.},
  doi          = {10.1038/s41588-025-02289-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2177-2191},
  shortjournal = {Nat. Genet.},
  title        = {ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutational landscape of triple-negative breast cancer in african american women. <em>NG</em>, <em>57</em>(9), 2166-2176. (<a href='https://doi.org/10.1038/s41588-025-02322-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {African American (AA) women have the highest incidence of triple-negative breast cancer (TNBC) among all racial groups, but are underrepresented in cancer genomic studies. In 462 AA women with TNBC, we characterized the tumor mutational landscape by whole-exome sequencing and RNA sequencing. We unveiled a high-resolution mutational portrait of TNBC in AA women reminiscent of that in Asian and non-Hispanic white women, with no evidence of associations of mutational features with African ancestry. We also made some distinctive discoveries, including an almost complete dominance of TP53 mutations, low frequency of PIK3CA mutations and mutational signature-based subtypes with etiologic and prognostic significance. These findings do not support major racial differences in TNBC biology at the level of somatic mutations. Our study contributes considerably to diversifying the knowledge base of breast cancer genomics and provides insights into the disease etiology, disparities and therapeutic vulnerability of TNBC in AA women. This study explores the genomic and transcriptomic landscapes of triple-negative breast cancer in African American women. The authors show that the mutational profile is broadly similar to that observed in European and East Asian ancestry women while highlighting some interesting differences.},
  archive      = {J_NG},
  author       = {Yao, Song and Wei, Lei and Hu, Qiang and Liu, Song and Manojlovic, Zarko and Fiorica, Peter N. and Long, Mark and Zirpoli, Gary R. and Cai, Qiuyin and Long, Jirong and Ping, Jie and Barnard, Mollie E. and Jin, Yuxin and Murakami, Mitsuko and Wang, Jianmin and Zhu, Qianqian and Davis, Warren and Chen, Jianhong and Ondracek, Rochelle P. and Khoury, Thaer and Gandhi, Shipra and Takabe, Kazuaki and Ko, Naomi and Sanderson, Maureen and Hong, Chi-Chen and Bandera, Elisa V. and Craig, David W. and Ambrosone, Christine B. and Palmer, Julie R. and Zheng, Wei and Carpten, John D.},
  doi          = {10.1038/s41588-025-02322-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2166-2176},
  shortjournal = {Nat. Genet.},
  title        = {Mutational landscape of triple-negative breast cancer in african american women},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions. <em>NG</em>, <em>57</em>(9), 2156-2165. (<a href='https://doi.org/10.1038/s41588-025-02291-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand shared and ancestry-specific genetic control of brain protein expression and its ramifications for disease, we mapped protein quantitative trait loci (pQTLs) in 1,362 brain proteomes from African American, Hispanic/Latin American and non-Hispanic white donors. Among the pQTLs that multiancestry fine-mapping MESuSiE confidently assigned as putative causal pQTLs in a specific population, most were shared across the three studied populations and are referred to as multiancestry causal pQTLs. These multiancestry causal pQTLs were enriched for exonic and promoter regions. To investigate their effects on disease, we modeled the 858 multiancestry causal pQTLs as instrumental variables using Mendelian randomization and genome-wide association study results for neurologic and psychiatric conditions (21 traits in participants with European ancestry, 10 in those with African ancestry and 4 in Hispanic participants). We identified 119 multiancestry pQTL–protein pairs consistent with a causal role in these conditions. Remarkably, 29% of the multiancestry pQTLs in these pairs were coding variants. These results lay an important foundation for the creation of new molecular models of neurologic and psychiatric conditions that are likely to be relevant to individuals across different genetic ancestries. Multiancestry fine-mapping of brain protein quantitative trait loci coupled with Mendelian randomization analyses identifies protein–trait pairs consistent with causal effects across neurological and psychiatric conditions.},
  archive      = {J_NG},
  author       = {Wingo, Aliza P. and Liu, Yue and Vattathil, Selina M. and Gerasimov, Ekaterina S. and Mei, Zhen and Ravindran, Suda Parimala and Liu, Jiaqi and Shantaraman, Ananth and Seifar, Fatemeh and Wang, Erming and Zhang, Bin and Reddy, Joseph and Allen, Mariet and Ertekin-Taner, Nilüfer and De Jager, Philip L. and Fox, Edward J. and Duong, Duc M. and Epstein, Michael P. and Cutler, David J. and Levey, Allan I. and Bennett, David A. and Seyfried, Nicholas T. and Wingo, Thomas S.},
  doi          = {10.1038/s41588-025-02291-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2156-2165},
  shortjournal = {Nat. Genet.},
  title        = {Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes. <em>NG</em>, <em>57</em>(9), 2146-2155. (<a href='https://doi.org/10.1038/s41588-025-02288-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks with whole-genome sequencing (WGS) now enable the association of noncoding rare variants with complex human traits. Given that &gt;98% of the genome is available for exploration, the selection of noncoding variants remains a critical yet unresolved challenge in these analyses. Here we leverage knowledge of blood gene regulation and deleteriousness scores to select noncoding variants pertinent for association with blood-related traits. Integrating WGS and 42 blood cell count and biomarker measurements for 166,740 UK Biobank samples, we perform variant collapsing tests, identifying hundreds of gene–trait associations involving noncoding variants. However, we demonstrate that most of these noncoding rare variant associations (1) reproduce associations known from previous studies and (2) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting noncoding rare variant association results. Noncoding rare variant analyses using whole-genome sequencing data from the UK Biobank identify gene–trait associations for 42 blood cell traits but find that most signals are driven by linkage disequilibrium between common and rare variants.},
  archive      = {J_NG},
  author       = {Ribeiro, Diogo M. and Hofmeister, Robin J. and Rubinacci, Simone and Delaneau, Olivier},
  doi          = {10.1038/s41588-025-02288-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2146-2155},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores. <em>NG</em>, <em>57</em>(9), 2136-2145. (<a href='https://doi.org/10.1038/s41588-025-02298-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic health record (EHR)-based phenotype risk scores (PheRS) leverage individuals’ health trajectories to estimate disease risk, similar to how polygenic scores (PGS) use genetic information. While PGS generalizability has been studied, less is known about PheRS generalizability across healthcare systems and whether PheRS are complementary to PGS. We trained elastic-net-based PheRS to predict the onset of 13 common diseases for 845,929 individuals (age = 32–70 years) from three biobank-based studies in Finland (FinnGen), the UK (UKB) and Estonia (EstB). All PheRS were statistically significantly associated with the diseases of interest and most generalized well without retraining when applied to other studies. PheRS and PGS were only moderately correlated and models including both predictors improved onset prediction compared to PGS alone for 8 of 13 diseases. Our results indicate that EHR-based risk scores can transfer well between EHRs, capture largely independent information from PGS, and provide additive benefits for disease risk prediction. Comparison of electronic health record-based phenotype risk scores (PheRS) and polygenic scores (PGS) across 13 common diseases and three biobank-based studies indicates that PheRS and PGS may provide additive benefits for risk prediction.},
  archive      = {J_NG},
  author       = {Detrois, Kira E. and Hartonen, Tuomo and Teder-Laving, Maris and Jermy, Bradley and Läll, Kristi and Yang, Zhiyu and Mägi, Reedik and Ripatti, Samuli and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02298-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2136-2145},
  shortjournal = {Nat. Genet.},
  title        = {Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes. <em>NG</em>, <em>57</em>(9), 2124-2135. (<a href='https://doi.org/10.1038/s41588-025-02317-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Ancestral Recombination Graph (ARG), which describes the genealogical history of a sample of genomes, is a vital tool in population genomics and biomedical research. Recent advancements have substantially increased ARG reconstruction scalability, but they rely on approximations that can reduce accuracy, especially under model misspecification. Moreover, they reconstruct only a single ARG topology and cannot quantify the considerable uncertainty associated with ARG inferences. Here, to address these challenges, we introduce SINGER (sampling and inferring of genealogies with recombination), a method that accelerates ARG sampling from the posterior distribution by two orders of magnitude, enabling accurate inference and uncertainty quantification for hundreds of whole-genome sequences. Through extensive simulations, we demonstrate SINGER’s enhanced accuracy and robustness to model misspecification compared to existing methods. We demonstrate the utility of SINGER by applying it to individuals of British and African descent within the 1000 Genomes Project, identifying signals of population differentiation, archaic introgression and strong support for ancient polymorphism in the human leukocyte antigen region shared across primates. SINGER is a method for creating ancestral recombination graphs to understand the genealogical history of genomes. The method has increased speed, and thus scalability, without sacrificing accuracy.},
  archive      = {J_NG},
  author       = {Deng, Yun and Nielsen, Rasmus and Song, Yun S.},
  doi          = {10.1038/s41588-025-02317-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2124-2135},
  shortjournal = {Nat. Genet.},
  title        = {Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. <em>NG</em>, <em>57</em>(9), 2116-2123. (<a href='https://doi.org/10.1038/s41588-025-02286-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mixed-model association analysis (MMAA) is the preferred tool for performing genome-wide association studies. However, existing MMAA tools often have long runtimes and high memory requirements. Here we present LDAK-KVIK, an MMAA tool for analysis of quantitative and binary phenotypes. LDAK-KVIK is computationally efficient, requiring less than 10 CPU hours and 5 Gb memory to analyze genome-wide data for 350,000 individuals. Using simulated phenotypes, we show that LDAK-KVIK produces well-calibrated test statistics for both homogeneous and heterogeneous datasets. When applied to real phenotypes, LDAK-KVIK has the highest power among all tools considered. For example, across 40 quantitative UK Biobank phenotypes (average sample size 349,000), LDAK-KVIK finds 16% more independent, genome-wide significant loci than classical linear regression, whereas BOLT-LMM and REGENIE find 15% and 11% more, respectively. LDAK-KVIK can also be used to perform gene-based tests; across the 40 quantitative UK Biobank phenotypes, LDAK-KVIK finds 18% more significant genes than the leading existing tool. Last, LDAK-KVIK produces state-of-the-art polygenic scores. LDAK-KVIK is a mixed-model association method for genome-wide studies that optimizes computational performance and power. LDAK-KVIK can also perform gene-based tests and produces state-of-the-art polygenic scores.},
  archive      = {J_NG},
  author       = {Hof, Jasper P. and Speed, Doug},
  doi          = {10.1038/s41588-025-02286-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2116-2123},
  shortjournal = {Nat. Genet.},
  title        = {LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward improving multiomic research in understudied cereals. <em>NG</em>, <em>57</em>(9), 2106-2115. (<a href='https://doi.org/10.1038/s41588-025-02245-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the looming challenges of climate change and population pressure, understudied cereals hold notably untapped potential for future global food security and sustainable agriculture due to their unique nutritional profiles and notable stress resilience. Multiomics holds great promise in realizing the value of understudied cereals by understanding the genetic basis and molecular mechanisms regulating their unique traits; however, multiomic studies of understudied cereals still lag behind those of major cereals, hindering their genetic improvement and wide adoption in modern agriculture. Here, we discuss the limitations and challenges in multiomic studies of understudied cereals and how they can be overcome to enhance utilization of the traits unique to understudied cereals in improving both themselves and major cereals. We also offer a database specific to understudied cereals, including multiomics data from 12 understudied cereals and wild relatives for some of these, to serve the community. Understudied cereals offer untapped potential for food security, yet multiomic research substantially lags behind that of major cereals. This Perspective discusses these challenges and introduces a dedicated database to serve the community.},
  archive      = {J_NG},
  author       = {Yang, Fanjing and Gong, Xiaojiao and Zhao, Pan and Cheng, Lingtong and Zhang, Yujie and Luo, Yingting and Zhu, Qian-Hao and Ye, Chu-Yu},
  doi          = {10.1038/s41588-025-02245-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2106-2115},
  shortjournal = {Nat. Genet.},
  title        = {Toward improving multiomic research in understudied cereals},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The LISTEN principles for genetic sequence data governance and database engineering. <em>NG</em>, <em>57</em>(9), 2099-2105. (<a href='https://doi.org/10.1038/s41588-025-02270-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several international legal agreements include an ‘access and benefit-sharing’ (ABS) mechanism that attaches obligations to the use of genetic sequence data. These agreements are frequently subject to critique on the grounds that ABS is either fundamentally incompatible with the principles of open science, or technically challenging to implement in open scientific databases. Here, we argue that these critiques arise from a misinterpretation of the principles of open science and that both considerations can be addressed by a set of simple principles that link database engineering and governance. We introduce a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, which can be readily adopted by both new and existing platforms participating in ABS systems. We also highlight how these principles can act in concert with familiar principles of open science, such as findable, accessible, interoperable and reusable (FAIR) data sharing. This Perspective proposes a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, aimed at ensuring access and benefit-sharing principles in open science.},
  archive      = {J_NG},
  author       = {Carlson, Colin J. and Granados, Monica and Phelan, Alexandra and Ramakrishnan, Nithin and Poisot, Timothée},
  doi          = {10.1038/s41588-025-02270-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2099-2105},
  shortjournal = {Nat. Genet.},
  title        = {The LISTEN principles for genetic sequence data governance and database engineering},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Governance of cross-border genomic data sharing through a human rights approach. <em>NG</em>, <em>57</em>(9), 2090-2098. (<a href='https://doi.org/10.1038/s41588-025-02252-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cross-border genomic data sharing is crucial for advancing research progress, understanding of genetic diversity and international public health. However, this practice also leads to concerns regarding biosecurity, individual privacy and regulatory conflicts. Scientific communities often advocate data sovereignty approaches to address these challenges, yet related frameworks are often inadequate in their ability to address such challenges and have an overemphasis on national control and protectionism. They also may not fully consider the inherently transnational nature of genomic data and the need for a governance system that reflects the shared human heritage of the data and global interconnectivity. Instead, we propose here a human rights-based approach that surpasses territorial boundaries to promote shared responsibility and equitable research practices. A balanced governance framework that incorporates extraterritorial obligations and aligns data policy with human rights principles can safeguard individual privacy and biosecurity, encourage international collaboration and support ethically responsible genomic research. This Perspective proposes a human rights-based governance framework for cross-border genomic data sharing, addressing limitations of data sovereignty approaches while balancing privacy, security and global research collaboration.},
  archive      = {J_NG},
  author       = {Yu, Liang and Feng, Ruohan and Sun, Youhai and Peng, Yaojin},
  doi          = {10.1038/s41588-025-02252-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2090-2098},
  shortjournal = {Nat. Genet.},
  title        = {Governance of cross-border genomic data sharing through a human rights approach},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recreating fibroblast diversity in vitro by activating transcription factors. <em>NG</em>, <em>57</em>(9), 2088-2089. (<a href='https://doi.org/10.1038/s41588-025-02283-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We combined CRISPR activation of all 1,836 known human transcription factors with high-throughput Perturb-seq to recreate the diverse transcriptional states occupied by fibroblasts in vivo. Our study revealed regulators of key states and showed that inducing normal states can, in some cases, suppress those linked to disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02283-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2088-2089},
  shortjournal = {Nat. Genet.},
  title        = {Recreating fibroblast diversity in vitro by activating transcription factors},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improving broad-spectrum resistance to clubroot disease with one gene. <em>NG</em>, <em>57</em>(9), 2086-2087. (<a href='https://doi.org/10.1038/s41588-025-02304-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GSL5, a glucan synthase, acts as a suppressor of jasmonic acid-mediated immunity in cruciferous plants. Inactivation of GSL5 by genome editing confers high-level and broad-spectrum resistance to pathogens that cause clubroot disease in four cruciferous species.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02304-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2086-2087},
  shortjournal = {Nat. Genet.},
  title        = {Improving broad-spectrum resistance to clubroot disease with one gene},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mystery of tamoxifen-associated uterine cancer uncovered. <em>NG</em>, <em>57</em>(9), 2084-2085. (<a href='https://doi.org/10.1038/s41588-025-02316-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tamoxifen is an essential drug in breast cancer therapy. Unlike prevailing models of therapy-related tumorigenesis, tamoxifen acts by directly activating the PI3K pathway, bypassing the need for mutations in one of the most common driver genes in sporadic uterine cancer. These findings open avenues for investigating similar mechanisms in other drugs.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02316-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2084-2085},
  shortjournal = {Nat. Genet.},
  title        = {Mystery of tamoxifen-associated uterine cancer uncovered},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A bivalent chromatin state regulates the function of composite transposons. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02337-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02337-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {A bivalent chromatin state regulates the function of composite transposons},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting obesity from childhood. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02338-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02338-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Predicting obesity from childhood},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Building immune tolerance in the thymus. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02339-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02339-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Building immune tolerance in the thymus},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI identifies promoter variants that alter gene expression. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02340-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02340-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {AI identifies promoter variants that alter gene expression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease. <em>NG</em>, <em>57</em>(9), 2079-2082. (<a href='https://doi.org/10.1038/s41588-025-02299-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite Africa’s vast genetic diversity, its populations are underrepresented in global genomic datasets. Here we describe the vision of the KidneyGenAfrica, a pan-African initiative launched to address this inequity, and call for more inclusive genomics research that recognizes Africa’s key role in genetic variation and potential to generate insights in chronic kidney disease.},
  archive      = {J_NG},
  author       = {Fatumo, Segun and Ojewunmi, Oyesola and Camenzuli, Rebecca and Kintu, Christopher and Kamiza, Abram and Brandenburg, Jean-Tristan and Kalungi, Allan and Kalyesubula, Robert and Salako, Babatunde and Nash, Oyekanmi and Crampin, Amelia and Adeyemo, Adebowale and Nyirenda, Moffat and Ramsay, Michele and Fabian, June},
  doi          = {10.1038/s41588-025-02299-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2079-2082},
  shortjournal = {Nat. Genet.},
  title        = {KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The future of polygenic risk scores in direct-to-consumer genomics. <em>NG</em>, <em>57</em>(9), 2077-2078. (<a href='https://doi.org/10.1038/s41588-025-02313-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Sun, Yuhao and Vines, John and Tenesa, Albert},
  doi          = {10.1038/s41588-025-02313-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2077-2078},
  shortjournal = {Nat. Genet.},
  title        = {The future of polygenic risk scores in direct-to-consumer genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmat">NMAT - 30</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Ab initio structure solutions from nanocrystalline powder diffraction data via diffusion models. <em>NMAT</em>, <em>24</em>(9), 1502. (<a href='https://doi.org/10.1038/s41563-025-02301-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMAT},
  author       = {Guo, Gabe and Saidi, Tristan Luca and Terban, Maxwell W. and Valsecchi, Michele and Billinge, Simon J. L. and Lipson, Hod},
  doi          = {10.1038/s41563-025-02301-y},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1502},
  shortjournal = {Nat. Mater.},
  title        = {Author correction: Ab initio structure solutions from nanocrystalline powder diffraction data via diffusion models},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hybrid hydrogel–extracellular matrix scaffolds identify biochemical and mechanical signatures of cardiac ageing. <em>NMAT</em>, <em>24</em>(9), 1489-1501. (<a href='https://doi.org/10.1038/s41563-025-02234-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extracellular matrix remodelling of cardiac tissue is a key contributor to age-related cardiovascular disease and dysfunction. Such remodelling is multifaceted including changes to the biochemical composition, architecture and mechanics, clouding our understanding of how and which extracellular matrix properties contribute to a dysfunctional state. Here we describe a decellularized extracellular matrix–synthetic hydrogel hybrid scaffold that independently confers two distinct matrix properties—ligand presentation and stiffness—to cultured cells in vitro, allowing for the identification of their specific roles in cardiac ageing. The hybrid scaffold maintains native matrix composition and organization of young or aged murine cardiac tissue, whereas its mechanical properties can be independently tuned to mimic young or aged tissue stiffness. Seeding these scaffolds with murine primary cardiac fibroblasts, we identify distinct age- and matrix-dependent mechanisms of cardiac fibroblast activation, matrix remodelling and senescence. Importantly, we show that the ligand presentation of a young extracellular matrix can outweigh the profibrotic stiffness cues typically present in an aged extracellular matrix in maintaining or driving cardiac fibroblast quiescence. Ultimately, these tunable scaffolds can enable the discovery of specific extracellular targets to prevent ageing dysfunction and promote rejuvenation. A hybrid extracellular matrix–hydrogel with tunable mechanical stiffness and biochemical composition of young or aged cardiac tissue is used to identify the specific contributions of extracellular matrix ligands and mechanics for fibroblast aging.},
  archive      = {J_NMAT},
  author       = {Sun, Avery Rui and Ramli, Md Faris H. and Shen, Xingyu and Kannivadi Ramakanth, Karthikbabu and Chen, Dixiao and Hu, Yang and Vidyasekar, Prasanna and Foo, Roger S. and Long, Yuchen and Zhu, Jin and Ackers-Johnson, Matthew and Young, Jennifer L.},
  doi          = {10.1038/s41563-025-02234-6},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1489-1501},
  shortjournal = {Nat. Mater.},
  title        = {Hybrid hydrogel–extracellular matrix scaffolds identify biochemical and mechanical signatures of cardiac ageing},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cytoplasmic anillin and ect2 promote RhoA/myosin II-dependent confined migration and invasion. <em>NMAT</em>, <em>24</em>(9), 1476-1488. (<a href='https://doi.org/10.1038/s41563-025-02269-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell migration in mechanically confined environments is a crucial step of metastatic cancer progression. Nonetheless, the molecular components and processes mediating such behaviour are still not fully understood. Here we demonstrate that a pool of the scaffolding protein anillin and its cofactor Ect2, which are both predominantly nuclear proteins and critical mediators of cytokinesis, is present in the cytoplasm of multiple interphase cell types that promote confined cell migration. Confined migration in biomimetic microfluidic models triggers the actomyosin-binding-dependent recruitment of anillin to the plasma membrane at the poles of migrating cells in a manner that scales with microenvironmental stiffness and confinement. The guanine nucleotide exchange activity of Ect2 is required for its RhoA-GTPase-mediated activation of myosin II at the cell poles, enhancing invasion, bleb-based migration and extravasation. Confinement-induced nuclear envelope rupture further amplifies this process due to the release of further anillin and Ect2 into the cytoplasm. Overall, these results show how Ect2 and anillin cooperate to mediate RhoA/ROCK/myosin II-dependent mechanoadaptation and invasive cancer progression. Cell migration in confined environments is initiated by a cytoplasmic pool of anillin and Ect2 that promotes RhoA/myosin II-mediated activation at the poles of migrating cells, in a process dependent on the extracellular environment stiffness.},
  archive      = {J_NMAT},
  author       = {Tran, Avery T. and Wisniewski, Emily O. and Mistriotis, Panagiotis and Stoletov, Konstantin and Parlani, Maria and Amitrano, Alice and Ifemembi, Brent and Lee, Se Jong and Bera, Kaustav and Zhang, Yuqi and Tuntithavornwat, Soontorn and Afthinos, Alexandros and Kiepas, Alexander and Agarwal, Bhawana and Nath, Sanjiban and Jamieson, John J. and Zuo, Yi and Habib, Daniel and Wu, Pei-Hsun and Martin, Stuart S. and Gerecht, Sharon and Gu, Luo and Lewis, John D. and Kalab, Petr and Friedl, Peter and Konstantopoulos, Konstantinos},
  doi          = {10.1038/s41563-025-02269-9},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1476-1488},
  shortjournal = {Nat. Mater.},
  title        = {Cytoplasmic anillin and ect2 promote RhoA/myosin II-dependent confined migration and invasion},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Adaptive peptide dispersions enable drying-induced biomolecule encapsulation. <em>NMAT</em>, <em>24</em>(9), 1465-1475. (<a href='https://doi.org/10.1038/s41563-025-02300-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peptides are promising building blocks of designer materials with wide-ranging applications. These materials are stabilized by directional hydrogen-bonding patterns, giving rise to one-dimensional or two-dimensional assembly. It remains a challenge to mimic biology’s context-adaptive and flexible structures. Here we introduce minimalistic tripeptide sequences that form highly soluble dynamic ensembles through multivalent side-chain interactions. We observe these supramolecular dispersions undergo drying-induced sequential liquid–liquid phase separation followed by solidification, resulting in the formation of films of stiff, densely packed and porous peptide microparticles that can be instantaneously redispersed upon the re-introduction of water. Air-drying of peptide dispersions in the presence of proteins or small-molecule payloads results in efficient encapsulation and the retention of protein stability after redispersion, showing promise for the emulsification, encapsulation, protection and storage of biomacromolecules. The mechanism resembles the protective strategies in natural systems during desiccation, which rely on liquid–liquid phase separation to survive extreme conditions. Here the authors design tripeptides that form dynamic soluble dispersions and undergo phase separation upon drying to assemble into porous particles. This evaporation-driven emulsification can be harnessed to encapsulate and stabilize biomolecules.},
  archive      = {J_NMAT},
  author       = {Dave, Dhwanit R. and Kassem, Salma and Coste, Maeva and Xu, Lele and Tayarani-Najjaran, Mona and Podbevšek, Darjan and Colon-De Leon, Paola and Zhang, Sheng and Ortuno Macias, Luis and Sementa, Deborah and Pérez-Ferreiro, María and Ayati, Nooshin Sadat and Choudhury, Muniyat A. and Veerasammy, Kelly and Doganata, Selma and Zhong, Tiffany and Weng, Cory and Morales, Jorge and Favaro, Denize C. and Marianski, Mateusz and Ahn, So Yeon and Obermeyer, Allie C. and Wang, Tong and Li, Tai-De and Chen, Xi and Tu, Raymond and He, Ye and Ulijn, Rein V.},
  doi          = {10.1038/s41563-025-02300-z},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1465-1475},
  shortjournal = {Nat. Mater.},
  title        = {Adaptive peptide dispersions enable drying-induced biomolecule encapsulation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unveiling high-mobility hot carriers in a two-dimensional conjugated coordination polymer. <em>NMAT</em>, <em>24</em>(9), 1457-1464. (<a href='https://doi.org/10.1038/s41563-025-02246-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hot carriers, inheriting excess kinetic energy from high-energy photons, drive numerous optoelectronic applications reliant on non-equilibrium transport processes. Although extensively studied in inorganic materials, their potential in organic-based systems remains largely unexplored. Here we demonstrate highly mobile hot carriers in crystalline two-dimensional conjugated coordination polymer Cu3BHT (BHT, benzenehexathiol) films. Leveraging a suite of ultrafast spectroscopic and imaging techniques, we map the microscopic charge transport landscape in Cu3BHT films following non-equilibrium photoexcitation across temporal, spatial and frequency domains, revealing two distinct high-mobility transport regimes. In the non-equilibrium regime, hot carriers achieve an ultrahigh mobility of ~2,000 cm2 V–1 s–1, traversing grain boundaries up to ~300 nm within a picosecond. In the quasi-equilibrium regime, free carriers exhibit Drude-type, band-like transport with a remarkable mobility of ~400 cm2 V–1 s–1 and an intrinsic diffusion length exceeding 1 μm. These findings position two-dimensional conjugated coordination polymers as versatile platforms for advancing organic-based hot carrier applications. Hot carrier transport in organic systems has remained elusive due to rapid energy relaxation and limited transport properties. Here highly mobile hot carriers and their relaxation dynamics are reported in a crystalline two-dimensional conjugated coordination polymer, revealing two distinct transport regimes.},
  archive      = {J_NMAT},
  author       = {Fu, Shuai and Huang, Xing and Gao, Guoquan and St. Petkov, Petko and Gao, Wenpei and Zhang, Jianjun and Gao, Lei and Zhang, Heng and Liu, Min and Hambsch, Mike and Zhang, Wenjie and Zhang, Jiaxu and Li, Keming and Kaiser, Ute and Parkin, Stuart S. P. and Mannsfeld, Stefan C. B. and Zhu, Tong and Wang, Hai I. and Wang, Zhiyong and Dong, Renhao and Feng, Xinliang and Bonn, Mischa},
  doi          = {10.1038/s41563-025-02246-2},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1457-1464},
  shortjournal = {Nat. Mater.},
  title        = {Unveiling high-mobility hot carriers in a two-dimensional conjugated coordination polymer},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise synthesis of advanced polyarylamines for efficient perovskite solar cells. <em>NMAT</em>, <em>24</em>(9), 1450-1456. (<a href='https://doi.org/10.1038/s41563-025-02199-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although being highly demanded in organic electronics, functional conjugated polymers face challenges on scalable synthesis with batch uniformities. Here a reactivity-regulated sequent cross-coupling carbon–nitrogen polycondensation method is developed to enable the precise synthesis of functional polyarylamines with excellent batch-to-batch uniformity. It is revealed that the stepwise regulation of intermediate reactivities is key to accomplish controllable polycondensation via two sequent palladium-promoted carbon–nitrogen coupling cycles, which is distinct to the unicyclic carbon–carbon coupling. A variety of polyarylamines are prepared to improve the material functionalities, where a ternary polymer consisting of polar substituents is shown to optimize the interfacial and bulk properties of perovskite layers fabricated on top. The corresponding inverted perovskite solar cells achieved remarkable power conversion efficiencies of 25.2% (active area, 5.97 mm2) and 23.2% (active area, 128 mm2), along with decent operational stabilities. Overall, this work provides an effective polymerization method for advanced conjugated polymers to enable high-performance optoelectronics. The precise synthesis of functional polyarylamines based on a reactivity-regulated sequent cross-coupling carbon–nitrogen polycondensation method has been reported, with excellent batch-to-batch uniformity for perovskite solar cells.},
  archive      = {J_NMAT},
  author       = {Shen, Ziqiu and Huang, Yanchun and Dong, Yuan and Yan, Kangrong and Chen, Hongzhen and Li, Chang-Zhi},
  doi          = {10.1038/s41563-025-02199-6},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1450-1456},
  shortjournal = {Nat. Mater.},
  title        = {Precise synthesis of advanced polyarylamines for efficient perovskite solar cells},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Solvent co-intercalation in layered cathode active materials for sodium-ion batteries. <em>NMAT</em>, <em>24</em>(9), 1441-1449. (<a href='https://doi.org/10.1038/s41563-025-02287-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Solvent co-intercalation, that is, the combined intercalation of ions and solvent molecules into electrode materials, is an additional but much less explored lever for modifying the properties of metal-ion battery electrodes (metal = Li, Na, Mg, etc.). Knowledge on solvent co-intercalation is relatively scarce and largely limited to graphite anodes, for which in sodium-ion batteries, the co-intercalation of glyme molecules is fast and highly reversible. The use of co-intercalation for cathode active materials (CAMs) remains much less explored. Here we investigate for a series of sodium-layered sulfide CAMs (NaxMS2, M = Ti, V, Cr and mixtures) under which conditions solvent co-intercalation occurs and how this process impacts the phase behaviour, electrode breathing, redox potential and cycle life compared to ‘Na+-only’ intercalation. Co-intercalation is a complex process that can, for example, cause opposing fluxes, meaning that solvents intercalate into the CAMs while sodium ions simultaneously deintercalate. Co-intercalation leads to layered structures that can include different amounts of confined solvated ions, ions and unbound solvent molecules. It is an approach to designing structurally diverse, layered materials with potential applications for batteries and beyond. Solvent co-intercalation into graphite anodes for sodium-ion batteries is common; however, intercalation into cathodes is much less explored. Here, using operando experiments as well as theory, solvent co-intercalation in a range of layered sulfides is investigated.},
  archive      = {J_NMAT},
  author       = {Sun, Yanan and Åvall, Gustav and Wu, Shu-Han and A. Ferrero, Guillermo and Freytag, Annica and Groszewicz, Pedro B. and Wang, Hui and Mazzio, Katherine A. and Bianchini, Matteo and Baran, Volodymyr and Risse, Sebastian and Adelhelm, Philipp},
  doi          = {10.1038/s41563-025-02287-7},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1441-1449},
  shortjournal = {Nat. Mater.},
  title        = {Solvent co-intercalation in layered cathode active materials for sodium-ion batteries},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electron ptychography reveals a ferroelectricity dominated by anion displacements. <em>NMAT</em>, <em>24</em>(9), 1433-1440. (<a href='https://doi.org/10.1038/s41563-025-02205-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sodium niobate, a lead-free ferroic material, hosts delicately balanced, competing order parameters, including ferroelectric states that can be stabilized by epitaxial strain. Here we show that the resulting macroscopic ferroelectricity exhibits an unconventional microscopic structure using multislice electron ptychography. This technique overcomes multiple scattering artefacts limiting conventional electron microscopy, enabling both lateral spatial resolution beyond the diffraction limit and recovery of three-dimensional structural information. These imaging capabilities allow us to separate the ferroelectric interior of the sample from the relaxed surface structure and identify the soft phonon mode and related structural distortions with picometre precision. Unlike conventional ferroelectric perovskites, we find that the polar distortion in this material involves minimal distortions of the cation sublattices and is instead dominated by anion displacements relative to the niobium sublattice. We establish limits on film thickness for interfacial octahedral rotation engineering and directly visualize a random octahedral rotation pattern, arising from the flat dispersion of the associated phonon mode. Limited by challenges in light-atom imaging, microscopic investigations of ferroelectricity have used cation–cation displacements as a proxy for the true cation–anion distortions. Using electron ptychography, oxygen anions can be tracked to observe an otherwise-hidden ferroelectric mechanism in thin-film NaNbO3, which would have appeared antiferroelectric from cations alone.},
  archive      = {J_NMAT},
  author       = {KP, Harikrishnan and Xu, Ruijuan and Patel, Kinnary and Crust, Kevin J. and Khandelwal, Aarushi and Zhang, Chenyu and Prosandeev, Sergey and Zhou, Hua and Shao, Yu-Tsun and Bellaiche, Laurent and Hwang, Harold Y. and Muller, David A.},
  doi          = {10.1038/s41563-025-02205-x},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1433-1440},
  shortjournal = {Nat. Mater.},
  title        = {Electron ptychography reveals a ferroelectricity dominated by anion displacements},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Skyrmion nanodomains in ferroelectric–antiferroelectric solid solutions. <em>NMAT</em>, <em>24</em>(9), 1424-1432. (<a href='https://doi.org/10.1038/s41563-025-02216-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polar skyrmions have demonstrated rich physics and exotic properties for developing novel functionalities. However, so far, skyrmion nanodomains exist only in a few material systems, such as ferroelectric/dielectric superlattices, free-standing PbTiO3/SrTiO3 epitaxial bilayers and ultrathin Pb(Zr,Ti)O3/SrTiO3/Pb(Zr,Ti)O3 sandwiches. These heterostructures are fabricated with elaborately designed boundary conditions to meet the delicate energy balance for stabilizing topological phases. This requirement limits the broad applications of skyrmions in electronic devices. Here we show widespread skyrmion nanodomains in ferroelectric–antiferroelectric solid solutions, composed of ferroelectric PbTiO3 and one antiferroelectric PbSnO3 (Pb(Ti1–xSnx)O3), PbHfO3 (Pb(Ti1–xHfx)O3) or PbZrO3 (Pb(Ti1–xZrx)O3). The skyrmionic textures are formed by engineering dipole–dipole and antiferrodistortive–dipole couplings in competition between ferroelectric and antiferroelectric polar orderings, allowing the stabilization of topological phases. A phase diagram is built for the three solid solution series, revealing the stabilization regions of skyrmion nanodomains. In addition, the non-trivial domains also exhibit improved switching character, reversible writing/erasure and long-term retention for the electrical manipulation of polar configurations. These findings open an avenue for the investigation and exploitation of polar skyrmions in ferroelectric-based materials, providing opportunities in topological electronics. Polar skyrmions are of interest for nanoelectronics due to their exotic properties. However, so far, these are metastable states requiring a delicate balance of boundary conditions to form in heterostructures. In ferroelectric and antiferroelectric solid solutions, skyrmionic textures are observed to form in both bulk and film.},
  archive      = {J_NMAT},
  author       = {Zheng, Weijie and Ma, Xingyue and Pang, Zhentao and Ren, Yifeng and Chen, Hongying and Xu, Jibo and Zheng, Chunyan and Liu, Jianyi and Liu, Xiaohui and Deng, Yu and Nie, Yuefeng and Wu, Di and Bellaiche, Laurent and Yang, Yurong and Wen, Zheng},
  doi          = {10.1038/s41563-025-02216-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1424-1432},
  shortjournal = {Nat. Mater.},
  title        = {Skyrmion nanodomains in ferroelectric–antiferroelectric solid solutions},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Configurable antiferromagnetic domains and lateral exchange bias in atomically thin CrPS4. <em>NMAT</em>, <em>24</em>(9), 1414-1423. (<a href='https://doi.org/10.1038/s41563-025-02259-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interfacial exchange coupling between antiferromagnets (AFMs) and ferromagnets (FMs) crucially makes it possible to shift the FM hysteresis, known as exchange bias, and to switch AFM states. Two-dimensional magnets unlock opportunities to combine AFM and FM materials; however, the buried AFM–FM interfaces obtained by stacking remains challenging to understand. Here we demonstrate interfacial control via intralayer exchange coupling in the layered AFM CrPS4, where connected even and odd layers realize pristine lateral interfaces between AFM-like and FM-like regions. We distinguish antiphase even-layer states by scanning nitrogen-vacancy centre magnetometry due to a weak surface magnetization. This surface magnetization enables control over the even-layer state, with different regions switching at distinct fields due to their own lateral couplings. We toggle three AFM domains adjacent to a FM-like region and demonstrate a tunable multilevel exchange bias. Our nanoscale visualization unveils the microscopic origins of exchange bias and advances single two-dimensional crystals for hybrid AFM–FM technologies. The authors use nitrogen-vacancy centre magnetometry to explore layer number and magnetic field evolution of ferromagnetic and antiferromagnetic domains in the A-type antiferromagnet CrPS4.},
  archive      = {J_NMAT},
  author       = {Wang, Yu-Xuan and Graham, Thomas K. M. and Rama-Eiroa, Ricardo and Islam, Md Ariful and Badarneh, Mohammad H. and Nunes Gontijo, Rafael and Tiwari, Ganesh Prasad and Adhikari, Tibendra and Zhang, Xin-Yue and Watanabe, Kenji and Taniguchi, Takashi and Besson, Claire and J. G. Santos, Elton and Lin, Zhong and Zhou, Brian B.},
  doi          = {10.1038/s41563-025-02259-x},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1414-1423},
  shortjournal = {Nat. Mater.},
  title        = {Configurable antiferromagnetic domains and lateral exchange bias in atomically thin CrPS4},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). All-electrical perpendicular switching of chiral antiferromagnetic order. <em>NMAT</em>, <em>24</em>(9), 1407-1413. (<a href='https://doi.org/10.1038/s41563-025-02228-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antiferromagnets as active components in devices have been attracting attention due to their negligible stray field and ultrafast magnetic dynamics, which are promising for high-density and fast memory devices. Although the switching of antiferromagnetic (AFM) order by current-induced spin–orbit torque (SOT) has already been realized, field-free SOT-induced bidirectional switching of AFM order in perpendicular geometry has been elusive. Here we experimentally demonstrate field-free perpendicular switching of the magnetic octupole moct in chiral AFM Mn3Sn by combining in-plane and out-of-plane SOTs generated by two-dimensional van der Vaals WTe2. The out-of-plane SOT breaks the in-plane inversion symmetry and leads to deterministic bidirectional switching of moct in polycrystalline Mn3Sn even without a fixed perpendicular moct easy axis. The switching ratio reaches up to 80% compared to 20–30% in polycrystalline Mn3Sn and the critical current density is reduced by an order of magnitude to around 1 MA cm−2. Our work promotes the development of chiral AFM devices toward practical application. By introducing an unconventional spin–orbit torque, the authors demonstrate an all-electrical way to bidirectionally switch the perpendicular chiral antiferromagnetic order in Mn3Sn, promoting the application of antiferromagnetic spintronics.},
  archive      = {J_NMAT},
  author       = {Zheng, Zhenyi and Jia, Lanxin and Zhang, Zhizhong and Shen, Qia and Zhou, Guowei and Cui, Zhiwei and Ren, Lizhu and Chen, Zhiteng and Jamaludin, Nur Fadilah and Zhao, Tieyang and Xiao, Rui and Zhang, Qihan and Du, Yi and Liu, Liang and Gradečak, Silvija and Novoselov, Kostya S. and Zhao, Weisheng and Xu, Xiaohong and Zhang, Yue and Chen, Jingsheng},
  doi          = {10.1038/s41563-025-02228-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1407-1413},
  shortjournal = {Nat. Mater.},
  title        = {All-electrical perpendicular switching of chiral antiferromagnetic order},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Superconducting magic-angle twisted trilayer graphene with competing magnetic order and moiré inhomogeneities. <em>NMAT</em>, <em>24</em>(9), 1400-1406. (<a href='https://doi.org/10.1038/s41563-025-02252-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The microscopic mechanism of unconventional superconductivity in magic-angle twisted trilayer graphene is poorly understood. We show direct evidence for an in-plane magnetic order competing with the superconducting state motivated by theoretical proposals. We use two complementary electrical transport measurements. First, in statistically significant switching events in the superconducting state of magic-angle twisted trilayer graphene, we observe non-monotonic and hysteretic responses in the switching distributions as a function of temperature and in-plane magnetic field. Additionally, the system behaves like a network of Josephson junctions due to lattice-relaxation-induced moiré inhomogeneity. Second, in normal regions doped slightly away from the superconducting regime, hysteretic and linear positive magnetoresistance with the in-plane magnetic field shows evidence for an in-plane magnetic order. Furthermore, we estimate superfluid stiffness Js ≈ 0.15 K with strong temperature dependence and show a broadened Berezinskii–Kosterlitz–Thouless transition. Our observations may constrain possible intervalley-coherent magnetic orders and the superconductivity arising from its fluctuations. Electrical transport measurements of magic-angle twisted trilayer graphene are presented.},
  archive      = {J_NMAT},
  author       = {Mukherjee, Ayshi and Layek, Surat and Sinha, Subhajit and Kundu, Ritajit and Marchawala, Alisha H. and Hingankar, Mahesh and Sarkar, Joydip and Sangani, L. D. Varma and Agarwal, Heena and Ghosh, Sanat and Tazi, Aya Batoul and Watanabe, Kenji and Taniguchi, Takashi and Pasupathy, Abhay N. and Kundu, Arijit and Deshmukh, Mandar M.},
  doi          = {10.1038/s41563-025-02252-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1400-1406},
  shortjournal = {Nat. Mater.},
  title        = {Superconducting magic-angle twisted trilayer graphene with competing magnetic order and moiré inhomogeneities},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Observation of non-hermitian topology from optical loss modulation. <em>NMAT</em>, <em>24</em>(9), 1393-1399. (<a href='https://doi.org/10.1038/s41563-025-02278-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the interplay of non-Hermiticity and topology is crucial given the intrinsic openness of most natural and engineered systems, and has important ramifications in topological lasers and sensors. Recently, it has been theoretically proposed that topological features could originate solely from a system’s non-Hermiticity in photonic platforms. Here we experimentally demonstrate the appearance of non-Hermitian topology exclusively from loss modulation in a photonic system that is topologically trivial in the absence of loss. We do this by implementing a non-Hermitian generalization of an Aubry–André–Harper model with purely imaginary potential in a programmable integrated photonics platform, which allows us to investigate different periodic and quasiperiodic configurations of the model. In both cases, we show the emergence of topological edge modes and explore their resilience to different kinds of disorder. Our work highlights loss engineering as a mechanism to generate topological properties. Using a reconfigurable integrated photonic platform, the authors reveal the appearance of non-Hermitian topology and the existence of edge modes emerging exclusively from optical loss modulation.},
  archive      = {J_NMAT},
  author       = {Hashemi, Amin and Pereira, Elizabeth Louis and Li, Hongwei and Lado, Jose L. and Blanco-Redondo, Andrea},
  doi          = {10.1038/s41563-025-02278-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1393-1399},
  shortjournal = {Nat. Mater.},
  title        = {Observation of non-hermitian topology from optical loss modulation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hyper-gap transparent conductor. <em>NMAT</em>, <em>24</em>(9), 1387-1392. (<a href='https://doi.org/10.1038/s41563-025-02248-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An elusive conductor with perfect optical transparency holds revolutionary potential for fields such as optoelectronics and nanophotonics. Such a hypothetical metal would possess a spectral gap1,2—a ‘hyper-gap’—in its absorption spectrum, separating the intraband and interband absorptions, in which optical losses could vanish. Currently, this property is achievable only within the bandgap of insulators. However, realizing such a hyper-gap metal demands an exotic electronic structure in which the conducting bands have a bandwidth narrower than their energy separations from the remaining electronic states. Here we present such a hyper-gap in a family of organic metals—the Fabre charge-transfer salts3—through first-principles predictions coupled with both electrical and optical measurements. A transparent window, spanning from red to near-infrared wavelengths, is identified in bulk single crystals that remain transmissive over a thickness of 30 µm. The corresponding absorption coefficient is the lowest among known stoichiometric metals, rivalling thin films of transparent conductive oxides. This finding introduces a path, beyond traditional doping strategies in insulators, to combine electronic conduction and optical transparency. A family of organic metals that behave as hyper-gap transparent conductors is discussed. Such an elusive combination of electronic conduction and optical transparency is highly attractive for plasmonics and photonics applications.},
  archive      = {J_NMAT},
  author       = {Wu, Zhengran and Li, Chunhong and Hu, Xiaolei and Chen, Kun and Guo, Xiang and Li, Yan and Lu, Ling},
  doi          = {10.1038/s41563-025-02248-0},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1387-1392},
  shortjournal = {Nat. Mater.},
  title        = {Hyper-gap transparent conductor},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-quality narrow black phosphorus nanoribbons with nearly atomically smooth edges and well-defined edge orientation. <em>NMAT</em>, <em>24</em>(9), 1375-1386. (<a href='https://doi.org/10.1038/s41563-025-02314-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Black phosphorus nanoribbons (BPNRs) with a tunable bandgap and intriguing electronic and optical properties hold strong potential for logic applications. However, efficiently producing high-quality BPNRs with precise control over their size and structure remains a great challenge. Here we achieved high-quality, narrow and clean BPNRs with nearly atomically smooth edges and well-defined edge orientation at high yield (up to ~95%) through the sonochemical exfoliation of the synthesized bulk BP crystals with a slightly enlarged lattice parameter along the armchair direction. The resulting BPNRs have widths centred at 32 nm and can be as narrow as 1.5 nm, with edges consistently aligned along the zigzag direction in measured BPNRs with widths ≤340 nm. The formation of one-dimensional BPNRs with zigzag edges is attributed to the introduction of pre-stress along the armchair direction of the grown bulk BP and the application of suitable sonication conditions. The BPNR bandgap increases as the BPNR width decreases from 83 nm to 13 nm, with a large bandgap of 0.64 eV for a 13-nm-wide BPNR. A typical graphene-contacted field-effect transistor fabricated with a 13-nm-wide and 10-nm-thick BPNR can achieve an on/off ratio of 1.7 × 106, mobility of 1,506 cm2 V−1 s−1 and on-state channel conductivity of 1,845 µS. The devices also exhibit excellent photodetection performance. Our method opens up a route to produce BPNRs with high material quality and defined edge chirality for fundamental studies and practical applications in electronic and optoelectronic fields. Through the sonochemical exfoliation of grown black phosphorus crystals, high-quality, narrow and clean black phosphorus nanoribbons with smooth edges and defined edge chirality are achieved, enabling the realization of high-performance transistors and photodetectors.},
  archive      = {J_NMAT},
  author       = {Zhang, Teng and Chen, Youxin and He, Zhiyan and Liu, Qinran and Chen, Le and Wang, Kai and Chen, Wenzhe and Wang, Kunchan and Ye, Xiaowo and He, Zhuoyang and Zhang, Yuyang and Zhang, Yanming and Liao, Chenghao and He, Yuan and Gao, Wenpei and Zhang, Tianyu and Hu, Bei and Jiang, Shenghao and Shi, Fangyuan and Gao, Shengguang and Li, Xinyue and Chen, Changxin},
  doi          = {10.1038/s41563-025-02314-7},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1375-1386},
  shortjournal = {Nat. Mater.},
  title        = {High-quality narrow black phosphorus nanoribbons with nearly atomically smooth edges and well-defined edge orientation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bioinstructive scaffolds enhance stem cell engraftment for functional tissue regeneration. <em>NMAT</em>, <em>24</em>(9), 1364-1374. (<a href='https://doi.org/10.1038/s41563-025-02212-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stem cell therapy is a promising approach for tissue regeneration after traumatic injury, yet current applications are limited by inadequate control over the fate of stem cells after transplantation. Here we introduce a bioconstruct engineered for the staged release of growth factors, tailored to direct different phases of muscle regeneration. The bioconstruct is composed of a decellularized extracellular matrix containing polymeric nanocapsules sequentially releasing basic fibroblast growth factor and insulin-like growth factor 1, which promote the proliferation and differentiation of muscle stem cells, respectively. When applied to a volumetric muscle loss defect in an animal model, the bioconstruct enhances myofibre formation, angiogenesis, innervation and functional restoration. Further, it promotes functional muscle formation with human or aged murine muscle stem cells, highlighting the translational potential of this bioconstruct. Overall, these results highlight the potential of bioconstructs with orchestrated growth factor release for stem cell therapies in traumatic injury. A decellularized extracellular matrix scaffold containing nanocapsules orchestrates timed growth factor release to promote muscle stem cell proliferation and differentiation and effective muscle regeneration after traumatic injury.},
  archive      = {J_NMAT},
  author       = {Wu, Di and Eugenis, Ioannis and Hu, Caroline and Kim, Soochi and Kanugovi, Abhijnya and Yue, Shouzheng and Wheeler, Joshua R. and Fathali, Iman and Feeley, Sonali and Shrager, Joseph B. and Huang, Ngan F. and Rando, Thomas A.},
  doi          = {10.1038/s41563-025-02212-y},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1364-1374},
  shortjournal = {Nat. Mater.},
  title        = {Bioinstructive scaffolds enhance stem cell engraftment for functional tissue regeneration},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vortices and antivortices in antiferroelectric PbZrO3. <em>NMAT</em>, <em>24</em>(9), 1359-1363. (<a href='https://doi.org/10.1038/s41563-025-02245-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ferroelectric materials are characterized by a parallel arrangement of electric dipoles, but at the nanoscale they can present vortices and other non-trivial topological structures1–9 that combine small size and topological protection, rendering them functionally attractive10–13. The driving force for the appearance of vortices in ferroelectrics is the need to minimize the depolarizing fields at interfaces3–5,14; by making the polarization rotate, depolarization fields vanish4,5,8,9. Antiferroelectrics, by contrast, are defined by an antiparallel arrangement of electric dipoles15. A priori, therefore, they lack the depolarization fields that drive the appearance of non-trivial topologies in ferroelectrics. At the atomic scale of the dipoles, however, we find that polar discontinuities can still happen, driving the appearance of topological singularities at ferroelastic domain walls. Topological phenomena in ferroelectrics such as vortices are of interest as they may be useful for high-density storage applications, but similar phenomena have not been seen in antiferroelectrics, which possess antipolar dipole arrangements. Here, using electron microscopy, topological antivortices and faint vortices are seen in the prototypical antiferroelectric PbZrO3.},
  archive      = {J_NMAT},
  author       = {Liu, Ying and Zhang, Huazhang and Shapovalov, Konstantin and Niu, Ranming and Cairney, Julie M. and Liao, Xiaozhou and Roleder, Krystian and Majchrowski, Andrzej and Arbiol, Jordi and Ghosez, Philippe and Catalan, Gustau},
  doi          = {10.1038/s41563-025-02245-3},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1359-1363},
  shortjournal = {Nat. Mater.},
  title        = {Vortices and antivortices in antiferroelectric PbZrO3},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanisms of resistive switching in two-dimensional monolayer and multilayer materials. <em>NMAT</em>, <em>24</em>(9), 1346-1358. (<a href='https://doi.org/10.1038/s41563-025-02170-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The power and energy consumption of resistive switching devices can be lowered by reducing the dimensions of their active layers. Efforts to push this low-energy switching property to its limits have led to the investigation of active regions made with two-dimensional (2D) layered materials. Despite their small dimensions, 2D layered materials exhibit a rich variety of switching mechanisms, each involving different types of atomic structure reconfiguration. In this Review, we highlight and classify the mechanisms of resistive switching in monolayer and bulk 2D layered materials, with a subsequent focus on those occurring in a monolayer and/or localized to point defects in the crystalline sheet. We discuss the complex energetics involved in these fundamentally defect-assisted processes, including the coexistence of multiple mechanisms and the effects of the contacts used. Examining the highly localized ‘atomristor’-type switching, we provide insights into atomic motions and electronic transport across the metal–2D interfaces underlying their operation. Finally, we discuss progress and our perspective on the challenges associated with the development of 2D resistive switching devices. Promising application areas and material systems are identified and suggested for further research. The authors review the mechanisms of resistive switching in monolayer and bulk forms of two-dimensional layered materials, providing insights into atomic motions and electronic transport across interfaces.},
  archive      = {J_NMAT},
  author       = {Kaniselvan, M. and Jeon, Y.-R. and Mladenović, M. and Luisier, M. and Akinwande, D.},
  doi          = {10.1038/s41563-025-02170-5},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1346-1358},
  shortjournal = {Nat. Mater.},
  title        = {Mechanisms of resistive switching in two-dimensional monolayer and multilayer materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modulation of optical loss leads to robust localization of light. <em>NMAT</em>, <em>24</em>(9), 1344-1345. (<a href='https://doi.org/10.1038/s41563-025-02298-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical loss usually leads to dissipation, but has been shown, counterintuitively, to create strong light localization. This demonstration — realized by implementing a photonic arrangement with a special topology in a programmable integrated photonic platform — highlights loss as a powerful control of the topological properties of electromagnetic modes.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02298-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1344-1345},
  shortjournal = {Nat. Mater.},
  title        = {Modulation of optical loss leads to robust localization of light},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Black phosphorus nanoribbons for high-performance transistors and photodetectors. <em>NMAT</em>, <em>24</em>(9), 1342-1343. (<a href='https://doi.org/10.1038/s41563-025-02313-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sonochemical exfoliation of black phosphorus crystals with a slightly enlarged lattice parameter along the armchair direction produces narrow black phosphorus nanoribbons (BPNRs) with well-defined edge chirality at a yield of up to 95%. The high performance of transistors and photodetectors fabricated with these BPNRs demonstrates the potential of BPNRs for electronic and optoelectronic applications.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02313-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1342-1343},
  shortjournal = {Nat. Mater.},
  title        = {Black phosphorus nanoribbons for high-performance transistors and photodetectors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Orchestrated growth factor release for muscle restoration. <em>NMAT</em>, <em>24</em>(9), 1340-1341. (<a href='https://doi.org/10.1038/s41563-025-02330-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Implantable bioconstructs with sequential release of growth factors promote enhanced myofibre formation, angiogenesis and innervation in a mouse model of volumetric muscle loss.},
  archive      = {J_NMAT},
  author       = {Fischer, Eliana O. and Tsukerman, Anna and Levenberg, Shulamit},
  doi          = {10.1038/s41563-025-02330-7},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1340-1341},
  shortjournal = {Nat. Mater.},
  title        = {Orchestrated growth factor release for muscle restoration},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Role of the extracellular matrix in cardiac ageing. <em>NMAT</em>, <em>24</em>(9), 1337-1339. (<a href='https://doi.org/10.1038/s41563-025-02325-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A hybrid decellularized extracellular matrix–polyacrylamide scaffold reveals how extracellular matrix properties influence cardiac ageing, offering insights for potential rejuvenation therapies.},
  archive      = {J_NMAT},
  author       = {Prakash, Aishwarya and Smith, Poppy O. and Iskratsch, Thomas},
  doi          = {10.1038/s41563-025-02325-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1337-1339},
  shortjournal = {Nat. Mater.},
  title        = {Role of the extracellular matrix in cardiac ageing},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tripeptide armour shields encapsulated proteins. <em>NMAT</em>, <em>24</em>(9), 1335-1336. (<a href='https://doi.org/10.1038/s41563-025-02302-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drying-driven assembly of tripeptides into porous, redispersible microparticles enables spontaneous protein encapsulation and protection.},
  archive      = {J_NMAT},
  author       = {Li, Xianbao and Li, Junbai},
  doi          = {10.1038/s41563-025-02302-x},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1335-1336},
  shortjournal = {Nat. Mater.},
  title        = {Tripeptide armour shields encapsulated proteins},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-lived hot carriers in a polymeric semiconductor. <em>NMAT</em>, <em>24</em>(9), 1333-1334. (<a href='https://doi.org/10.1038/s41563-025-02286-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hot carriers with high mobility and long diffusion length are observed in a two-dimensional conjugated coordination polymer.},
  archive      = {J_NMAT},
  author       = {Handa, Taketo and Zhu, Xiaoyang},
  doi          = {10.1038/s41563-025-02286-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1333-1334},
  shortjournal = {Nat. Mater.},
  title        = {Long-lived hot carriers in a polymeric semiconductor},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-performance polymers for perovskite photovoltaics. <em>NMAT</em>, <em>24</em>(9), 1331-1332. (<a href='https://doi.org/10.1038/s41563-025-02235-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A carbon–nitrogen cross-coupling polymerization method is developed to precisely synthesize functional polyarylamines at large scale, resulting in perovskite solar cells with promising efficiencies and stability.},
  archive      = {J_NMAT},
  author       = {Wakamiya, Atsushi and Murdey, Richard},
  doi          = {10.1038/s41563-025-02235-5},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1331-1332},
  shortjournal = {Nat. Mater.},
  title        = {High-performance polymers for perovskite photovoltaics},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unveiling ferroelectric 3D microstructures. <em>NMAT</em>, <em>24</em>(9), 1328-1330. (<a href='https://doi.org/10.1038/s41563-025-02206-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multislice electron ptychography reveals ferroelectric microstructures with sub-ångström lateral resolution and nanometre depth resolution, directly imaging a ferroelectricity generated by anion displacements relative to the Nb sublattice.},
  archive      = {J_NMAT},
  author       = {Gao, Xiaoyue and Gao, Peng},
  doi          = {10.1038/s41563-025-02206-w},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1328-1330},
  shortjournal = {Nat. Mater.},
  title        = {Unveiling ferroelectric 3D microstructures},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A tug-of-war recipe for nanoscale swirls. <em>NMAT</em>, <em>24</em>(9), 1326-1327. (<a href='https://doi.org/10.1038/s41563-025-02288-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mixed ceramics and thin films of well-known ferroelectric and antiferroelectric materials reveal dipolar skyrmions, providing a scalable platform to explore the properties and applications of non-collinear polarization textures.},
  archive      = {J_NMAT},
  author       = {Maksymovych, Petro},
  doi          = {10.1038/s41563-025-02288-6},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1326-1327},
  shortjournal = {Nat. Mater.},
  title        = {A tug-of-war recipe for nanoscale swirls},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reconfigurable magnetic order in 2D materials. <em>NMAT</em>, <em>24</em>(9), 1324-1325. (<a href='https://doi.org/10.1038/s41563-025-02304-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Layer-dependent magnetic behaviour is used to realize reconfigurable exchange bias in a 2D antiferromagnet.},
  archive      = {J_NMAT},
  author       = {Stevenson, Paul},
  doi          = {10.1038/s41563-025-02304-9},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1324-1325},
  shortjournal = {Nat. Mater.},
  title        = {Reconfigurable magnetic order in 2D materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Switching a chiral antiferromagnet. <em>NMAT</em>, <em>24</em>(9), 1322-1323. (<a href='https://doi.org/10.1038/s41563-025-02266-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unconventional spin–orbit torque generated by a low-symmetry topological semimetal enables purely electrical switching of the perpendicular magnetic octupole in a chiral antiferromagnet.},
  archive      = {J_NMAT},
  author       = {Singh, Simranjeet and Kao, I-Hsuan},
  doi          = {10.1038/s41563-025-02266-y},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1322-1323},
  shortjournal = {Nat. Mater.},
  title        = {Switching a chiral antiferromagnet},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The expanding world of topological ferroelectrics. <em>NMAT</em>, <em>24</em>(9), 1321. (<a href='https://doi.org/10.1038/s41563-025-02350-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue we present three papers on the topological properties or structure of ferroelectrics, which are interlinked.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02350-3},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1321},
  shortjournal = {Nat. Mater.},
  title        = {The expanding world of topological ferroelectrics},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmeth">NMETH - 35</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: IPipet: Sample handling using a tablet. <em>NMETH</em>, <em>22</em>(9), 1996. (<a href='https://doi.org/10.1038/s41592-025-02804-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zielinski, Dina and Gordon, Assaf and Zaks, Benjamin L. and Erlich, Yaniv},
  doi          = {10.1038/s41592-025-02804-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1996},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: IPipet: Sample handling using a tablet},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A universal language for finding mass spectrometry data patterns. <em>NMETH</em>, <em>22</em>(9), 1995. (<a href='https://doi.org/10.1038/s41592-025-02785-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Damiani, Tito and Jarmusch, Alan K. and Aron, Allegra T. and Petras, Daniel and Phelan, Vanessa V. and Zhao, Haoqi Nina and Bittremieux, Wout and Acharya, Deepa D. and Ahmed, Mohammed M. A. and Bauermeister, Anelize and Bertin, Matthew J. and Boudreau, Paul D. and Borges, Ricardo M. and Bowen, Benjamin P. and Brown, Christopher J. and Chagas, Fernanda O. and Clevenger, Kenneth D. and Correia, Mario S. P. and Crandall, William J. and Crüsemann, Max and Fahy, Eoin and Fiehn, Oliver and Garg, Neha and Gerwick, William H. and Gilbert, Jeffrey R. and Globisch, Daniel and Gomes, Paulo Wender P. and Heuckeroth, Steffen and James, C. Andrew and Jarmusch, Scott A. and Kakhkhorov, Sarvar A. and Kang, Kyo Bin and Kessler, Nikolas and Kersten, Roland D. and Kim, Hyunwoo and Kirk, Riley D. and Kohlbacher, Oliver and Kontou, Eftychia E. and Liu, Ken and Lizama-Chamu, Itzel and Luu, Gordon T. and Luzzatto Knaan, Tal and Mannochio-Russo, Helena and Marty, Michael T. and Matsuzawa, Yuki and McAvoy, Andrew C. and McCall, Laura-Isobel and Mohamed, Osama G. and Nahor, Omri and Neuweger, Heiko and Niedermeyer, Timo H. J. and Nishida, Kozo and Northen, Trent R. and Overdahl, Kirsten E. and Rainer, Johannes and Reher, Raphael and Rodriguez, Elys and Sachsenberg, Timo T. and Sanchez, Laura M. and Schmid, Robin and Stevens, Cole and Subramaniam, Shankar and Tian, Zhenyu and Tripathi, Ashootosh and Tsugawa, Hiroshi and van der Hooft, Justin J. J. and Vicini, Andrea and Walter, Axel and Weber, Tilmann and Xiong, Quanbo and Xu, Tao and Pluskal, Tomáš and Dorrestein, Pieter C. and Wang, Mingxun},
  doi          = {10.1038/s41592-025-02785-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1995},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A universal language for finding mass spectrometry data patterns},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unified mass imaging maps the lipidome of vertebrate development. <em>NMETH</em>, <em>22</em>(9), 1981-1994. (<a href='https://doi.org/10.1038/s41592-025-02771-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryo development entails the formation of anatomical structures with distinct biochemical compositions. Compared with the wealth of knowledge on gene regulation, our understanding of metabolic programs operating during embryogenesis is limited. Mass spectrometry imaging (MSI) has the potential to map the distribution of metabolites across embryo development. Here we established uMAIA, an analytical framework for the joint analysis of large MSI datasets, which enables the construction of multidimensional metabolomic atlases. Employing this framework, we mapped the four-dimensional (4D) distribution of over a hundred lipids at micrometric resolution in Danio rerio embryos. We discovered metabolic trajectories that unfold in concert with morphogenesis and revealed spatially organized biochemical coordination overlooked by bulk measurements. Interestingly, lipid mapping revealed unexpected distributions of sphingolipid and triglyceride species, suggesting their involvement in pattern establishment and organ development. Our approach empowers a new generation of whole-organism metabolomic atlases and enables the discovery of spatially organized metabolic circuits. uMAIA is an analytical framework designed to enable the construction of metabolic atlases at high resolution using mass spectrometry imaging data.},
  archive      = {J_NMETH},
  author       = {Schede, Halima Hannah and Alieh, Leila Haj Abdullah and Rohde, Laurel Ann and Herrera, Antonio and Schlaeppi, Anjalie and Valentin, Guillaume and Gargoori Motlagh, Alireza and Dominguez Mantes, Albert and Jollivet, Chloe and Paz-Montoya, Jonathan and Capolupo, Laura and Khven, Irina and Oates, Andrew C. and D’Angelo, Giovanni and La Manno, Gioele},
  doi          = {10.1038/s41592-025-02771-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1981-1994},
  shortjournal = {Nat. Meth.},
  title        = {Unified mass imaging maps the lipidome of vertebrate development},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions. <em>NMETH</em>, <em>22</em>(9), 1964-1980. (<a href='https://doi.org/10.1038/s41592-025-02793-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues. TCAT is a pipeline that can simultaneously capture gene expression programs related to T cell subsets and activation states for accurate T cell characterization.},
  archive      = {J_NMETH},
  author       = {Kotliar, Dylan and Curtis, Michelle and Agnew, Ryan and Weinand, Kathryn and Nathan, Aparna and Baglaenko, Yuriy and Slowikowski, Kamil and Zhao, Yu and Sabeti, Pardis C. and Rao, Deepak A. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41592-025-02793-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1964-1980},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput profiling of chemical-induced gene expression across 93,644 perturbations. <em>NMETH</em>, <em>22</em>(9), 1954-1963. (<a href='https://doi.org/10.1038/s41592-025-02781-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this Resource, we present an extensive dataset of chemical-induced gene signatures (CIGS), encompassing expression patterns of 3,407 genes regulating key biological processes in 2 human cell lines exposed to 13,221 compounds across 93,664 perturbations. This dataset encompasses 319,045,108 gene expression events, generated through 2 high-throughput technologies: the previously documented high-throughput sequencing-based high-throughput screening (HTS2) and the newly developed highly multiplexed and parallel sequencing (HiMAP-seq). Our results show that HiMAP-seq is comparable to RNA sequencing, but can profile the expression of thousands of genes across thousands of samples in one single test by utilizing a pooled-sample strategy. We further illustrate CIGS’s utility in elucidating the mechanism of action of unannotated small molecules, like ligustroflavone and 2,4-dihydroxybenzaldehyde, and to identify perturbation-induced cell states, such as those resistant to ferroptosis. The full dataset is publicly accessible at https://cigs.iomicscloud.com/ . CIGS is a high-resolution dataset comprising chemical perturbation-induced 319,045,108 gene expression events.},
  archive      = {J_NMETH},
  author       = {Xiang, Lei and Wang, Yumei and Shao, Wei and Li, Qingzhou and Yu, Xiankuo and Wei, Mingming and Gui, Yu and Li, Shengrong and Qin, Pan and Hu, Chao and Zhang, Guochen and Zhang, Xianwen and Wang, Jiawen and Li, Yingying and An, Jun and Luo, Yan and Liao, Yile and Deng, Jinghong and Tai, Xinran and Xu, Richard Y. and Huang, Lijun and Guo, Dale and Zhang, Guanbin and Xie, Zhi and Deng, Yun and Xu, Junquan and Wang, Dong},
  doi          = {10.1038/s41592-025-02781-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1954-1963},
  shortjournal = {Nat. Meth.},
  title        = {High-throughput profiling of chemical-induced gene expression across 93,644 perturbations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI. <em>NMETH</em>, <em>22</em>(9), 1944-1953. (<a href='https://doi.org/10.1038/s41592-025-02798-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concurrent recording of electroencephalogram (EEG) and functional magnetic resonance imaging (fMRI) signals reveals cross-scale neurovascular dynamics crucial for explaining fundamental linkages between function and behaviors. However, MRI scanners generate artifacts for EEG detection. Despite existing denoising methods, cabled connections to EEG receivers are susceptible to environmental fluctuations inside MRI scanners, creating baseline drifts that complicate EEG signal retrieval from the noisy background. Here we show that a wireless integrated sensing detector for simultaneous EEG and MRI can encode fMRI and EEG signals on distinct sidebands of the detector’s oscillation wave for detection by a standard MRI console over the entire duration of the fMRI sequence. Local field potential and fMRI maps are retrieved through low-pass and high-pass filtering of frequency-demodulated signals. From optogenetically stimulated somatosensory cortex in ChR2-transfected Sprague Dawley rats, positive correlation between evoked local field potential and fMRI signals validates strong neurovascular coupling, enabling cross-scale brain mapping with this two-in-one transducer. WISDEM is a hybrid detector for simultaneous EEG and fMRI recordings without artifacts or crosstalk, which allows access to neural activity at high temporal and spatial resolution, respectively, as demonstrated in rats.},
  archive      = {J_NMETH},
  author       = {Chen, Yi and Qian, Wei and Razansky, Daniel and Yu, Xin and Qian, Chunqi},
  doi          = {10.1038/s41592-025-02798-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1944-1953},
  shortjournal = {Nat. Meth.},
  title        = {WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A versatile miniature two-photon microscope enabling multicolor deep-brain imaging. <em>NMETH</em>, <em>22</em>(9), 1935-1943. (<a href='https://doi.org/10.1038/s41592-025-02780-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present FHIRM-TPM 3.0, a 2.6 g miniature two-photon microscope capable of multicolor deep-brain imaging in freely behaving mice. The system was integrated with a broadband anti-resonant hollow-core fiber featuring low transmission loss, minimal dispersion from 700 nm to 1,060 nm and high tolerance of laser power. By correcting chromatic and spherical aberrations and optimizing the fluorescence collection aperture, we achieved cortical neuronal imaging at depths exceeding 820 μm and, using a GRIN lens, hippocampal Ca2+ imaging at single dendritic spine resolution. Moreover, we engineered three interchangeable parfocal objectives, allowing for a tenfold scalable field of view up to 1 × 0.8 mm², with lateral resolutions ranging from 0.68 μm to 1.46 μm. By multicolor imaging at excitation wavelengths of 780 nm, 920 nm and 1,030 nm, we investigated mitochondrial and cytosolic Ca2+ activities relative to the deposition of amyloid plaques in the cortex of awake APP/PS1 transgenic mice. Thus, FHIRM-TPM 3.0 provides a versatile imaging system suitable for diverse brain imaging scenarios. FHIRM-TPM 3.0 is a miniature microscope for multicolor two-photon imaging in freely moving mice. In addition to the multicolor imaging abilities achieved with the help of a specially designed optical fiber, the microscope is also compatible with multiple lenses for a choice of field of view and resolution.},
  archive      = {J_NMETH},
  author       = {Wu, Runlong and Zhao, Chunzhu and Qiu, Shan and Zhu, Yufei and Zhang, Lifeng and Fu, Qiang and Hu, Yanhui and Wu, Dakun and Yu, Fei and Zhou, Fangxu and Huang, Haipeng and Zhang, Yunfeng and Wang, Xianhua and Wang, Aimin and Cheng, Heping},
  doi          = {10.1038/s41592-025-02780-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1935-1943},
  shortjournal = {Nat. Meth.},
  title        = {A versatile miniature two-photon microscope enabling multicolor deep-brain imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy. <em>NMETH</em>, <em>22</em>(9), 1923-1934. (<a href='https://doi.org/10.1038/s41592-025-02794-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volume correlative light and electron microscopy (vCLEM) is a powerful imaging technique that enables the visualization of fluorescently labeled proteins within their ultrastructural context. Currently, vCLEM alignment relies on time-consuming and subjective manual methods. This paper presents CLEM-Reg, an algorithm that automates the three-dimensional alignment of vCLEM datasets by leveraging probabilistic point cloud registration techniques. Point clouds are derived from segmentations of common structures in each modality, created by state-of-the-art open-source methods. CLEM-Reg drastically reduces the registration time of vCLEM datasets to a few minutes and achieves correlation of fluorescent signal to submicron target structures in electron microscopy on three newly acquired vCLEM benchmark datasets. CLEM-Reg was then used to automatically obtain vCLEM overlays to unambiguously identify TGN46-positive transport carriers involved in protein trafficking between the trans-Golgi network and plasma membrane. Datasets are available on EMPIAR and BioStudies, and a napari plugin is provided to aid end-user adoption. CLEM-Reg automates the three-dimensional alignment of volume correlative light and electron microscopy datasets by leveraging probabilistic point cloud registration techniques for fast and accurate results across diverse datasets.},
  archive      = {J_NMETH},
  author       = {Krentzel, Daniel and Elphick, Matouš and Domart, Marie-Charlotte and Peddie, Christopher J. and Laine, Romain F. and Shand, Cameron and Henriques, Ricardo and Collinson, Lucy M. and Jones, Martin L.},
  doi          = {10.1038/s41592-025-02794-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1923-1934},
  shortjournal = {Nat. Meth.},
  title        = {CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE. <em>NMETH</em>, <em>22</em>(9), 1911-1922. (<a href='https://doi.org/10.1038/s41592-025-02770-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while preserving crucial spatial context within tissues. However, existing ST platforms are constrained by high costs, long turnaround times, low resolution, limited gene coverage and inherently small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that reconstructs large-scale, super-resolution gene expression landscapes and automatically annotates cellular-level tissue architecture in samples exceeding capture areas of current ST platforms. The performance of iSCALE was assessed by comprehensive evaluations involving benchmarking experiments, immunohistochemistry staining and manual annotations by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion-associated cellular characteristics undetectable by conventional ST experiments. Our results demonstrate the utility of iSCALE in analyzing large tissues by enabling unbiased annotation, resolving cell type composition, mapping cellular microenvironments and revealing spatial features beyond the reach of standard ST analysis or routine histopathological assessment. iSCALE leverages histology and spatial transcriptomics to infer gene expression at super resolution in large tissues.},
  archive      = {J_NMETH},
  author       = {Schroeder, Amelia and Loth, Melanie L. and Luo, Chunyu and Yao, Sicong and Yan, Hanying and Zhang, Daiwei and Piya, Sarbottam and Plowey, Edward and Hu, Wenxing and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Chan, Su Jing and Reynolds, Taylor L. and Carlile, Thomas and Cullen, Patrick and Sung, Ji-Youn and Tsai, Hui-Hsin and Park, Jeong Hwan and Hwang, Tae Hyun and Zhang, Baohong and Li, Mingyao},
  doi          = {10.1038/s41592-025-02770-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1911-1922},
  shortjournal = {Nat. Meth.},
  title        = {Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial gene expression at single-cell resolution from histology using deep learning with GHIST. <em>NMETH</em>, <em>22</em>(9), 1900-1910. (<a href='https://doi.org/10.1038/s41592-025-02795-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The increased use of spatially resolved transcriptomics provides new biological insights into disease mechanisms. However, the high cost and complexity of these methods are barriers to broader application. Consequently, methods have been created to predict spot-based gene expression from routinely collected histology images. Recent benchmarking showed that current methodologies have limited accuracy and spatial resolution, constraining translational capacity. Here, we introduce GHIST, a deep learning-based framework that predicts spatial gene expression at single-cell resolution by leveraging subcellular spatial transcriptomics and synergistic relationships between multiple layers of biological information. We validated GHIST using public datasets and The Cancer Genome Atlas data, demonstrating its flexibility across different spatial resolutions and superior performance. Our results underscore the utility of in silico generation of single-cell spatial gene expression measurements and the capacity to enrich existing datasets with a spatially resolved omics modality, paving the way for scalable multi-omics analysis and biomarker identification. GHIST is a deep learning-based method that can predict spatial gene expression at high resolution using histology data.},
  archive      = {J_NMETH},
  author       = {Fu, Xiaohang and Cao, Yue and Bian, Beilei and Wang, Chuhan and Graham, Dinny and Pathmanathan, Nirmala and Patrick, Ellis and Kim, Jinman and Yang, Jean Yee Hwa},
  doi          = {10.1038/s41592-025-02795-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1900-1910},
  shortjournal = {Nat. Meth.},
  title        = {Spatial gene expression at single-cell resolution from histology using deep learning with GHIST},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ultra-high-scale cytometry-based cellular interaction mapping. <em>NMETH</em>, <em>22</em>(9), 1887-1899. (<a href='https://doi.org/10.1038/s41592-025-02744-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular interactions are of fundamental importance, orchestrating organismal development, tissue homeostasis and immunity. Recently, powerful methods that use single-cell genomic technologies to dissect physically interacting cells have been developed. However, these approaches are characterized by low cellular throughput, long processing times and high costs and are typically restricted to predefined cell types. Here we introduce Interact-omics, a cytometry-based framework to accurately map cellular landscapes and cellular interactions across all immune cell types at ultra-high resolution and scale. We demonstrate the utility of our approach to study kinetics, mode of action and personalized response prediction of immunotherapies, and organism-wide shifts in cellular composition and cellular interaction dynamics following infection in vivo. Our scalable framework can be applied a posteriori to existing cytometry datasets or incorporated into newly designed cytometry-based studies to map cellular interactions with a broad range of applications from fundamental biology to applied biomedicine. Interact-omics, a high-throughput cytometry-based framework, resolves the cellular interaction landscape.},
  archive      = {J_NMETH},
  author       = {Vonficht, Dominik and Jopp-Saile, Lea and Yousefian, Schayan and Flore, Viktoria and Simó Vesperinas, Inés and Teuber, Ruth and Avanesyan, Bogdan and Luo, Yanjiang and Röthemeier, Caroline and Grünschläger, Florian and Fernandez-Vaquero, Mirian and Fregona, Vincent and Ordoñez-Rueda, Diana and Schmalbrock, Laura K. and Deininger, Luca and Yamachui Sitcheu, Angelo Jovin and Gu, Zuguang and Funk, Maja C. and Mikut, Ralf and Heikenwälder, Mathias and Eggert, Angelika and von Stackelberg, Arend and Kobold, Sebastian and Krönke, Jan and Keller, Ulrich and Trumpp, Andreas and Hegazy, Ahmed N. and Eckert, Cornelia and Hübschmann, Daniel and Haas, Simon},
  doi          = {10.1038/s41592-025-02744-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1887-1899},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-scale cytometry-based cellular interaction mapping},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Laser flash melting cryo-EM samples to overcome preferred orientation. <em>NMETH</em>, <em>22</em>(9), 1880-1886. (<a href='https://doi.org/10.1038/s41592-025-02796-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sample preparation remains a bottleneck for protein structure determination by cryo-electron microscopy. A frequently encountered issue is that proteins adsorb to the air–water interface of the sample in a limited number of orientations. This makes it challenging to obtain high-resolution reconstructions, or may even cause projects to fail altogether. We have previously observed that laser flash melting and revitrification of cryo-EM samples reduces preferred orientation for large, symmetric particles. Here we demonstrate that our method can in fact be used to scramble the orientation of proteins of a range of sizes and symmetries. The effect can be enhanced for some proteins by increasing the heating rate during flash melting or by depositing amorphous ice onto the sample prior to revitrification. This also allows us to shed light onto the underlying mechanism. Our experiments establish a set of tools for overcoming preferred orientation that can be easily integrated into existing workflows. Individual proteins tend to adopt preferred orientations when subjected to vitrification for cryo-electron microscopy analysis. A laser flash melting procedure followed by rapid revitrification provides a simple approach to mitigate this issue, reducing the number of micrographs required for successful structure determination at high-resolution.},
  archive      = {J_NMETH},
  author       = {Straub, Monique S. and Harder, Oliver F. and Mowry, Nathan J. and Barrass, Sarah V. and Hruby, Jakub and Drabbels, Marcel and Lorenz, Ulrich J.},
  doi          = {10.1038/s41592-025-02796-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1880-1886},
  shortjournal = {Nat. Meth.},
  title        = {Laser flash melting cryo-EM samples to overcome preferred orientation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biophysics-based protein language models for protein engineering. <em>NMETH</em>, <em>22</em>(9), 1868-1879. (<a href='https://doi.org/10.1038/s41592-025-02776-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models trained on evolutionary data have emerged as powerful tools for predictive problems involving protein sequence, structure and function. However, these models overlook decades of research into biophysical factors governing protein function. We propose mutational effect transfer learning (METL), a protein language model framework that unites advanced machine learning and biophysical modeling. Using the METL framework, we pretrain transformer-based neural networks on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics. We fine-tune METL on experimental sequence–function data to harness these biophysical signals and apply them when predicting protein properties like thermostability, catalytic activity and fluorescence. METL excels in challenging protein engineering tasks like generalizing from small training sets and position extrapolation, although existing methods that train on evolutionary signals remain powerful for many types of experimental assays. We demonstrate METL’s ability to design functional green fluorescent protein variants when trained on only 64 examples, showcasing the potential of biophysics-based protein language models for protein engineering. Mutational effect transfer learning (METL) is a protein language model framework that unites machine learning and biophysical modeling. Transformer-based neural networks are pretrained on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics.},
  archive      = {J_NMETH},
  author       = {Gelman, Sam and Johnson, Bryce and Freschlin, Chase R. and Sharma, Arnav and D’Costa, Sameer and Peters, John and Gitter, Anthony and Romero, Philip A.},
  doi          = {10.1038/s41592-025-02776-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1868-1879},
  shortjournal = {Nat. Meth.},
  title        = {Biophysics-based protein language models for protein engineering},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations. <em>NMETH</em>, <em>22</em>(9), 1857-1867. (<a href='https://doi.org/10.1038/s41592-025-02797-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modifications (PTMs) are critical regulators of protein function, and their disruption is a key mechanism by which missense variants contribute to disease. Accurate PTM site prediction using deep learning can help identify PTM-altering variants, but progress has been limited by the lack of large, high-quality training datasets. Here, we introduce PTMAtlas, a curated compendium of 397,524 PTM sites generated through systematic reprocessing of 241 public mass-spectrometry datasets, and DeepMVP, a deep learning framework trained on PTMAtlas to predict PTM sites for phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation. DeepMVP substantially outperforms existing tools across all six PTM types. Its application to predicting PTM-altering missense variants shows strong concordance with experimental results, validated using literature-curated variants and cancer proteogenomic datasets. Together, PTMAtlas and DeepMVP provide a robust platform for PTM research and a scalable framework for assessing the functional consequences of coding variants through the lens of PTMs. DeepMVP is a deep learning framework for predicting PTM sites and variant-induced alterations across six modification types, including phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation.},
  archive      = {J_NMETH},
  author       = {Wen, Bo and Wang, Chenwei and Li, Kai and Han, Ping and Holt, Matthew V. and Savage, Sara R. and Lei, Jonathan T. and Dou, Yongchao and Shi, Zhiao and Li, Yi and Zhang, Bing},
  doi          = {10.1038/s41592-025-02797-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1857-1867},
  shortjournal = {Nat. Meth.},
  title        = {DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data. <em>NMETH</em>, <em>22</em>(9), 1846-1856. (<a href='https://doi.org/10.1038/s41592-025-02773-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations such as copy number alterations accumulate during cancer progression, driving intratumor heterogeneity that impacts therapy effectiveness. Understanding the characteristics and spatial distribution of genetically distinct subclones is essential for unraveling tumor evolution and improving cancer treatment. Here we present Clonalscope, a subclone detection method using copy number profiles, applicable to spatial transcriptomics and single-cell sequencing data. Clonalscope implements a nested Chinese Restaurant Process to identify de novo tumor subclones, which can incorporate prior information from matched bulk DNA sequencing data for improved subclone detection and malignant cell labeling. On single-cell RNA sequencing and single-cell assay for transposase-accessible chromatin using sequencing data from gastrointestinal tumors, Clonalscope successfully labeled malignant cells and identified genetically different subclones with thorough validations. On spatial transcriptomics data from various primary and metastasized tumors, Clonalscope labeled malignant spots, traced subclones and identified spatially segregated subclones with distinct differentiation levels and expression of genes associated with drug resistance and survival. Clonalscope is a method for cancer subclone detection leveraging copy number profiles estimated using spatial and single-cell sequencing data.},
  archive      = {J_NMETH},
  author       = {Wu, Chi-Yun and Rong, Jiazhen and Sathe, Anuja and Hess, Paul R. and Lau, Billy T. and Grimes, Susan M. and Huang, Sijia and Ji, Hanlee P. and Zhang, Nancy R.},
  doi          = {10.1038/s41592-025-02773-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1846-1856},
  shortjournal = {Nat. Meth.},
  title        = {Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scvi-hub: An actionable repository for model-driven single-cell analysis. <em>NMETH</em>, <em>22</em>(9), 1836-1845. (<a href='https://doi.org/10.1038/s41592-025-02799-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing availability of single-cell omics datasets presents new opportunities for reuse, while challenges in data transfer, normalization and integration remain a barrier. Here we present scvi-hub: a platform for efficiently sharing and accessing single-cell omics datasets using pretrained probabilistic models. It enables immediate execution of fundamental tasks like visualization, imputation, annotation and deconvolution on new query datasets using state-of-the-art methods, with massively reduced storage and compute requirements. We show that pretrained models support efficient analysis of large references, including the CZI CELLxGENE Discover Census. Scvi-hub is built within the scvi-tools open-source environment and integrated into scverse. Scvi-hub offers a scalable and user-friendly framework for accessing and contributing to a growing ecosystem of ready-to-use models and datasets, thus putting the power of atlas-level analysis at the fingertips of a broad community of users. Scvi-hub is a versatile and efficient platform for model-based analysis of single-cell sequencing studies with access to a diverse array of datasets and downstream analysis.},
  archive      = {J_NMETH},
  author       = {Ergen, Can and Pour Amiri, Valeh Valiollah and Kim, Martin and Kronfeld, Ori and Streets, Aaron and Gayoso, Adam and Yosef, Nir},
  doi          = {10.1038/s41592-025-02799-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1836-1845},
  shortjournal = {Nat. Meth.},
  title        = {Scvi-hub: An actionable repository for model-driven single-cell analysis},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues. <em>NMETH</em>, <em>22</em>(9), 1824-1835. (<a href='https://doi.org/10.1038/s41592-025-02774-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) are essential regulators of RNA fate and function. A long-standing challenge in studying RBP regulation has been mapping RNA interactomes within the dynamic transcriptomic landscape, especially in single-cell contexts and primary tissues. Here we introduce MAPIT-seq (modification added to RBP interacting transcript-sequencing), which uses an antibody-directed editing strategy to map genome-wide in situ RBP–RNA interactions and gene expression concurrently. We demonstrate MAPIT-seq’s robustness across multiple RBPs and systematically analyze RNA substrates associated with core polycomb repressive complex 2 (PRC2) components. MAPIT-seq is also applicable to frozen tissue sections, enabling the mapping of RBP roles during brain development. Importantly, we develop high-throughput single-cell MAPIT-seq (scMAPIT-seq) to reveal cell stage-specific RBP regulation. In summary, MAPIT-seq expands multi-omics profiling, providing an effective framework to study post-transcriptional regulation in dynamic biological processes and clinically relevant scenarios. This work presents MAPIT-seq, an antibody-guided RNA editing method that enables co-profiling of RBP-RNA interactions and gene expression at single-cell resolution and in tissue contexts.},
  archive      = {J_NMETH},
  author       = {Cheng, Qi-Xuan and Xie, Gang and Zhang, Xiangyu and Wang, Jie and Ding, Shuangjin and Wu, Yi-Xia and Shi, Ming and Duan, Fei-Fei and Wan, Zi-Li and Wei, Jing-Jia and Xiao, Junyu and Wang, Yangming},
  doi          = {10.1038/s41592-025-02774-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1824-1835},
  shortjournal = {Nat. Meth.},
  title        = {Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A realistic phantom dataset for benchmarking cryo-ET data annotation. <em>NMETH</em>, <em>22</em>(9), 1819-1823. (<a href='https://doi.org/10.1038/s41592-025-02800-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) is a powerful technique for imaging molecular complexes in their native cellular environments. However, identifying the vast majority of molecular species in cellular tomograms remains prohibitively difficult. Machine learning (ML) methods provide an opportunity to automate the annotation process, but algorithm development has been hindered by the lack of large, standardized datasets. Here we present an experimental phantom dataset with comprehensive ground-truth annotations for six molecular species to spur new algorithm development and benchmark existing tools. This annotated dataset is available on the CryoET Data Portal with infrastructure to streamline access for methods developers across fields. A standardized, realistic phantom dataset consisting of ground-truth annotations for six diverse molecular species is provided as a community resource for cryo-electron-tomography algorithm benchmarking.},
  archive      = {J_NMETH},
  author       = {Peck, Ariana and Yu, Yue and Schwartz, Jonathan and Cheng, Anchi and Ermel, Utz Heinrich and Hutchings, Joshua and Kandel, Saugat and Kimanius, Dari and Montabana, Elizabeth A. and Serwas, Daniel and Siems, Hannah and Wang, Feng and Zhao, Zhuowen and Zheng, Shawn and Haury, Matthias and Agard, David A. and Potter, Clinton S. and Carragher, Bridget and Harrington, Kyle and Paraan, Mohammadreza},
  doi          = {10.1038/s41592-025-02800-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1819-1823},
  shortjournal = {Nat. Meth.},
  title        = {A realistic phantom dataset for benchmarking cryo-ET data annotation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the physical hallmarks of cancer. <em>NMETH</em>, <em>22</em>(9), 1800-1818. (<a href='https://doi.org/10.1038/s41592-024-02564-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The physical microenvironment plays a crucial role in tumor development, progression, metastasis and treatment. Recently, we proposed four physical hallmarks of cancer, with distinct origins and consequences, to characterize abnormalities in the physical tumor microenvironment: (1) elevated compressive–tensile solid stresses, (2) elevated interstitial fluid pressure and the resulting interstitial fluid flow, (3) altered material properties (for example, increased tissue stiffness) and (4) altered physical micro-architecture. As this emerging field of physical oncology is being advanced by tumor biologists, cell and developmental biologists, engineers, physicists and oncologists, there is a critical need for model systems and measurement tools to mechanistically probe these physical hallmarks. Here, after briefly defining these physical hallmarks, we discuss the tools and model systems available for probing each hallmark in vitro, ex vivo, in vivo and in clinical settings. We finally review the unmet needs for mechanistic probing of the physical hallmarks of tumors and discuss the challenges and unanswered questions associated with each hallmark. This Review discusses the current challenges and the available methods to study the four physical hallmarks of cancer.},
  archive      = {J_NMETH},
  author       = {Nia, Hadi T. and Munn, Lance L. and Jain, Rakesh K.},
  doi          = {10.1038/s41592-024-02564-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1800-1818},
  shortjournal = {Nat. Meth.},
  title        = {Probing the physical hallmarks of cancer},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes. <em>NMETH</em>, <em>22</em>(9), 1788-1799. (<a href='https://doi.org/10.1038/s41592-025-02807-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbes within tumors have been recognized and experimentally related to oncogenesis, tumor growth, metastasis and therapeutic responsiveness. Studying the tumor microbiome presents difficulties, as early indications suggest that microbe populations are low in abundance, sparse and highly heterogeneous. Disparate results from computational profiling of the tumor microbiome have cast doubt on the premise of microbes in tumors. Yet decades of experimental evidence support the presence of tumor microbes, at least in a limited number of tumor types. In this Perspective, we discuss the importance of iteratively improving microbe-targeted sequencing techniques, established analytical pipelines, robust computational tools and solid validations to address current challenges and fill existing knowledge gaps. The vast amount of human tumor sequencing data available could greatly enhance systematic investigations of microbiome–tumor interactions with methods to quantify the composition of the tumor microbiome accurately. This Perspective explores the challenges and future directions in the world of human tumor microbiome research.},
  archive      = {J_NMETH},
  author       = {Li, Yingjie and Ma, Anjun and Johnson, Evan and Eng, Charis and De, Subhajyoti and Jiang, Sizun and Li, Zihai and Spakowicz, Daniel and Ma, Qin},
  doi          = {10.1038/s41592-025-02807-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1788-1799},
  shortjournal = {Nat. Meth.},
  title        = {The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research. <em>NMETH</em>, <em>22</em>(9), 1776-1787. (<a href='https://doi.org/10.1038/s41592-025-02769-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vitro tumor models are essential tools for cancer research, offering key insights into not only tumor biology but also therapeutic responses. The transition from traditional two-dimensional to three-dimensional organoid systems marks a paradigm shift in cancer modeling. Although two-dimensional models have been instrumental in elucidating fundamental molecular and genetic mechanisms, they fail to accurately replicate the intricate three-dimensional architecture and dynamic microenvironment characteristic of human tumors. Here we outline how advanced organoid technologies now enable more faithful recapitulation of tumor heterogeneity that better mimic native tissue mechanics and biochemistry. We discuss emerging methods, including air–liquid interface cultures, microfluidic tumor-on-a-chip devices and high-content imaging integrated with machine learning, which collectively address longstanding challenges such as matrix variability and the limited incorporation of immune and vascular elements. These innovations promise to enhance reproducibility and scalability while providing unprecedented insights into tumor biology, cancer progression and therapeutic strategies. This Perspective discusses recent progresses in development of in vitro tumor models and the current challenges in this field.},
  archive      = {J_NMETH},
  author       = {Rauner, Gat and Gupta, Piyush B. and Kuperwasser, Charlotte},
  doi          = {10.1038/s41592-025-02769-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1776-1787},
  shortjournal = {Nat. Meth.},
  title        = {From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible annotation of t cell subsets and activation states with gene expression programs. <em>NMETH</em>, <em>22</em>(9), 1774-1775. (<a href='https://doi.org/10.1038/s41592-025-02792-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created T-CellAnnoTator (TCAT), a computational method that helps to identify T cell subsets, activation states and functions. It does this using reproducible gene expression programs found across many disease contexts and tissues. TCAT outperforms conventional approaches for T cell subset prediction, is easy to use programmatically or through a website, and can be adapted for other cell types.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02792-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1774-1775},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible annotation of t cell subsets and activation states with gene expression programs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A miniature two-photon microscope detects multiple signals deep in the brains of moving mice. <em>NMETH</em>, <em>22</em>(9), 1772-1773. (<a href='https://doi.org/10.1038/s41592-025-02779-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A lightweight miniature two-photon microscope features multi-wavelength excitation, correction of aberrations, and interchangeable objectives for scalable fields of view. It enables multicolor, deep-brain and scalable neural imaging in freely moving mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02779-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1772-1773},
  shortjournal = {Nat. Meth.},
  title        = {A miniature two-photon microscope detects multiple signals deep in the brains of moving mice},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unravelling cellular interactions using flow cytometry. <em>NMETH</em>, <em>22</em>(9), 1770-1771. (<a href='https://doi.org/10.1038/s41592-025-02743-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a cost-effective ultra-high-throughput cytometry-based framework for the detection of physical interactions between cells, along with the characterization of complex cellular landscapes. Application of our approach can offer a systems-level understanding of immunity and facilitate study of the kinetics, mode of action and personalized response prediction of immunotherapies.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02743-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1770-1771},
  shortjournal = {Nat. Meth.},
  title        = {Unravelling cellular interactions using flow cytometry},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles. <em>NMETH</em>, <em>22</em>(9), 1768-1769. (<a href='https://doi.org/10.1038/s41592-025-02775-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present MAPIT-seq, a method that uses antibody-directed RNA editing to concurrently profile in situ RNA-binding protein (RBP)–RNA interactions and transcriptome-wide gene expression in limited input material, including single cells and frozen tissues. This dual-omic strategy streamlines mechanistic analyses of post-transcriptional regulation in dynamic biological processes and clinically relevant samples.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02775-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1768-1769},
  shortjournal = {Nat. Meth.},
  title        = {MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bridging histology and spatial gene expression across scales. <em>NMETH</em>, <em>22</em>(9), 1765-1767. (<a href='https://doi.org/10.1038/s41592-025-02806-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two deep-learning frameworks — GHIST and iSCALE — turn routine histology images into a rich molecular resource, and predict spatial gene expression at single-cell resolution (GHIST) and at super-resolution across large tissue sections (iSCALE), for scalable, data-driven tissue biology.},
  archive      = {J_NMETH},
  author       = {Ma, Ying},
  doi          = {10.1038/s41592-025-02806-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1765-1767},
  shortjournal = {Nat. Meth.},
  title        = {Bridging histology and spatial gene expression across scales},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Taming cancer evolution surprises. <em>NMETH</em>, <em>22</em>(9), 1760-1764. (<a href='https://doi.org/10.1038/s41592-025-02802-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a focus on the evolutionary forces that shape cancer cell behavior, some researchers work out ways to outmaneuver cancer.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02802-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1760-1764},
  shortjournal = {Nat. Meth.},
  title        = {Taming cancer evolution surprises},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SNAP-tag2 improves live-cell imaging. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02830-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02830-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {SNAP-tag2 improves live-cell imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optical pooled screens shed light on ebola infection. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02831-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kamenova, Ivanka},
  doi          = {10.1038/s41592-025-02831-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {Optical pooled screens shed light on ebola infection},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ViSOR puts the periphery in focus. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02828-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Willson, Joseph},
  doi          = {10.1038/s41592-025-02828-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {ViSOR puts the periphery in focus},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Benchmarking genomic language models. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02829-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-025-02829-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking genomic language models},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deciphering regulatory sequence grammar. <em>NMETH</em>, <em>22</em>(9), 1757. (<a href='https://doi.org/10.1038/s41592-025-02827-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leveraging data generated by UUATAC-seq, a method for cross-species chromatin accessibility profiling, the deep learning model NvwaCE deciphers cis-regulatory grammar from DNA sequence.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-025-02827-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1757},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering regulatory sequence grammar},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cichlid fishes. <em>NMETH</em>, <em>22</em>(9), 1755-1756. (<a href='https://doi.org/10.1038/s41592-025-02809-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cichlid fishes are a family of thousands of recently evolved species. As charismatic laboratory models, they are useful for studying anatomical, physiological and behavioral traits that vary across these closely related species.},
  archive      = {J_NMETH},
  author       = {Juntti, Scott A.},
  doi          = {10.1038/s41592-025-02809-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1755-1756},
  shortjournal = {Nat. Meth.},
  title        = {Cichlid fishes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Road trip home to start a lab. <em>NMETH</em>, <em>22</em>(9), 1754. (<a href='https://doi.org/10.1038/s41592-025-02801-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As a postdoctoral fellowship ends, for some, next is a trek to their new lab.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02801-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1754},
  shortjournal = {Nat. Meth.},
  title        = {Road trip home to start a lab},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A tale of tumors. <em>NMETH</em>, <em>22</em>(9), 1753. (<a href='https://doi.org/10.1038/s41592-025-02838-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advances for studying the biology of tumors have expanded our understanding of cancer.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02838-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1753},
  shortjournal = {Nat. Meth.},
  title        = {A tale of tumors},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nphys">NPHYS - 29</h2>
<ul>
<li><details>
<summary>
(2025). Precision without parity. <em>NPHYS</em>, <em>21</em>(9), 1498. (<a href='https://doi.org/10.1038/s41567-025-03018-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metrology has long been defined by its pursuit of reproducibility, accuracy and traceability. Yet, when it comes to the representation of women and other marginalized communities, precision has not been matched by parity, finds Solmaz Nadiri.},
  archive      = {J_NPHYS},
  author       = {Nadiri, Solmaz},
  doi          = {10.1038/s41567-025-03018-0},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1498},
  shortjournal = {Nat. Phys.},
  title        = {Precision without parity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: High-efficiency optical training of itinerant two-dimensional magnets. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03013-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Xie, Ti and Liang, Jierui and Bhattacharya, Dhritiman and Arachchige, Hasitha Suriya and Yakovenko, Victor M. and Mandrus, David G. and Qiu, Zi Qiang and Liu, Kai and Gong, Cheng},
  doi          = {10.1038/s41567-025-03013-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Author correction: High-efficiency optical training of itinerant two-dimensional magnets},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03054-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-03054-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient implementation of arbitrary two-qubit gates using unified control. <em>NPHYS</em>, <em>21</em>(9), 1489-1496. (<a href='https://doi.org/10.1038/s41567-025-02990-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The set of quantum logic gates that can be easily implemented is fundamental to the performance of quantum computers, as it governs the accuracy of basic quantum operations and dictates the complexity of implementing quantum algorithms. Traditional approaches to extending gate sets often require operating devices outside the ideal parameter regimes used to realize qubits, leading to increased control complexity while offering only a limited set of gates. Here we experimentally demonstrate a unified and versatile gate scheme capable of generating arbitrary two-qubit gates using only an exchange interaction and qubit driving on a superconducting quantum processor. We achieve high fidelities averaging 99.38% across a wide range of commonly used two-qubit unitaries, enabling precise multipartite entangled state preparation. Furthermore, we successfully produce a B gate, which efficiently synthesizes the entire family of two-qubit gates. Our results establish that fully exploiting the capabilities of the exchange interaction can yield a comprehensive and highly accurate gate set. With maximum expressivity, optimal gate time, demonstrated high fidelity and easy adaption to other quantum platforms, our unified control scheme offers the prospect of improved performance in quantum hardware and algorithm development. The efficiency of a quantum computer depends on which basic operations it can implement. Now a scheme that can implement any two-qubit logic gate has been demonstrated on a superconducting architecture.},
  archive      = {J_NPHYS},
  author       = {Chen, Zhen and Liu, Weiyang and Ma, Yanjun and Sun, Weijie and Wang, Ruixia and Wang, He and Xu, Huikai and Xue, Guangming and Yan, Haisheng and Yang, Zhen and Ding, Jiayu and Gao, Yang and Li, Feiyu and Zhang, Yujia and Zhang, Zikang and Jin, Yirong and Yu, Haifeng and Chen, Jianxin and Yan, Fei},
  doi          = {10.1038/s41567-025-02990-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1489-1496},
  shortjournal = {Nat. Phys.},
  title        = {Efficient implementation of arbitrary two-qubit gates using unified control},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1482-1488. (<a href='https://doi.org/10.1038/s41567-025-02989-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-fidelity quantum optomechanical control of a mechanical oscillator requires the ability to perform efficient, low-noise operations on long-lived phononic excitations. Microfabricated high-overtone bulk acoustic wave resonators (μHBARs) support high-frequency mechanical modes above 10 GHz with coherence times exceeding one millisecond. Here we demonstrate a μHBAR-based cavity optomechanical system that permits quantum optomechanical control of individual high-coherence phonon modes. We perform laser cooling of the phonon modes from an occupation of approximately 22 phonons to fewer than 0.4, corresponding to laser-based ground-state cooling of a mechanical object with a mass of 7.5 μg. During the cooling process we do not observe any absorption-induced heating, demonstrating the resilience of the HBAR optomechanical systems against parasitic heating. Our work demonstrates that μHBARs are promising as the basis for quantum optomechanical systems with robustness to decoherence that is necessary for efficient, low-noise photon–phonon conversion. A massive phonon mode in a high-overtone bulk acoustic wave resonator has been laser cooled close to its ground state. Its robustness to decoherence establishes the potential of these devices for quantum technologies.},
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-02989-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1482-1488},
  shortjournal = {Nat. Phys.},
  title        = {Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fast optical control of a coherent hole spin in a microcavity. <em>NPHYS</em>, <em>21</em>(9), 1475-1481. (<a href='https://doi.org/10.1038/s41567-025-02988-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many of the most promising quantum information platforms store quantum information in electronic or atomic spins. To incorporate these devices into quantum networks, a spin–photon interface is required. Currently, the best on-demand single-photon sources use a semiconductor quantum dot in an engineered photonic environment. However, it is difficult to achieve coherent spin control in a high-performance single-photon source, and spin coherence is limited by magnetic noise from nuclear spins in the semiconductor host material. Here we combine all-optical spin control with a quantum dot in an open microcavity. We demonstrate fast coherent rotations of a hole spin around an arbitrary axis of the Bloch sphere with a maximum π-pulse fidelity of 98.6%. To suppress the slow magnetic noise, we laser cool the nuclear spins using the hole as a central spin. This extends the hole spin free-induction-decay time by more than an order of magnitude. It becomes much larger than both rotation time and radiative recombination time of the spin, enabling the creation of many spin–photon pairs before the loss of spin coherence. Spin–photon interfaces provide a connection between quantum information stored in atomic or electronic spins and optical communications networks. A quantum photon emitter with long-lived, controllable coherent spin has now been demonstrated.},
  archive      = {J_NPHYS},
  author       = {Hogg, Mark R. and Antoniadis, Nadia O. and Marczak, Malwina A. and Nguyen, Giang N. and Baltisberger, Timon L. and Javadi, Alisa and Schott, Rüdiger and Valentin, Sascha R. and Wieck, Andreas D. and Ludwig, Arne and Warburton, Richard J.},
  doi          = {10.1038/s41567-025-02988-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1475-1481},
  shortjournal = {Nat. Phys.},
  title        = {Fast optical control of a coherent hole spin in a microcavity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A mechanical quantum memory for microwave photons. <em>NPHYS</em>, <em>21</em>(9), 1469-1474. (<a href='https://doi.org/10.1038/s41567-025-02975-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Superconducting qubits possess outstanding capabilities for processing quantum information in the microwave domain; however they have limited coherence times. An interface between photons and phonons could allow quantum information to be stored in long-lived mechanical oscillators. Here, we introduce a platform that relies on electrostatic forces in nanoscale structures to achieve strong coupling between a superconducting qubit and a nanomechanical oscillator with an energy decay time (T1) of approximately 25 ms, well beyond those achieved in integrated superconducting circuits. We use quantum operations in this system to investigate the microscopic origins of mechanical decoherence and mitigate its impact. By using two-pulse dynamical decoupling sequences, we can extend the coherence time (T2) from 64 μs to 1 ms. These findings establish that mechanical oscillators can act as quantum memories for superconducting devices, with potential future applications in quantum computing, sensing and transduction. Superconducting qubits, a leading platform for quantum information processing, suffer from decoherence. Interfacing them with nanomechanical oscillators allows quantum information to be stored in motional states with longer lifetimes.},
  archive      = {J_NPHYS},
  author       = {Bozkurt, Alkım B. and Golami, Omid and Yu, Yue and Tian, Hao and Mirhosseini, Mohammad},
  doi          = {10.1038/s41567-025-02975-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1469-1474},
  shortjournal = {Nat. Phys.},
  title        = {A mechanical quantum memory for microwave photons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics. <em>NPHYS</em>, <em>21</em>(9), 1462-1468. (<a href='https://doi.org/10.1038/s41567-025-02973-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Skyrmions are topologically stable quasiparticles that have been investigated in contexts including particle physics, quantum field theory, acoustics and condensed-matter physics. Quantum optical skyrmions with local topological textures are expected to reshape the landscape of quantum photonic technology, although their experimental implementation has not yet been demonstrated. Here we present experimental realizations of nanophotonic quantum skyrmions using a semiconductor cavity quantum electrodynamics system. By manipulating the photonic spin–orbit coupling in a Gaussian microcavity, we obtained a confined optical mode whose polarizations feature skyrmionic topologies. With pronounced cavity quantum electrodynamics effects, we generated and detected single-photon skyrmions from a solid-state quantum emitter deterministically coupled to the Gaussian microcavity. The polarity associated with single-photon skyrmions can be swapped by flipping the polarization of the quantum emitter through the Zeeman effect. We also investigated the topological protection of quantum optical skyrmions under different perturbations. Our work opens an unexplored aspect of quantum light–matter interactions in the nanoscale and might advance resilient photonic quantum technology with high-dimensional qubits and high-capacity quantum memories. Quantum optical skyrmions are promising for quantum photonic applications but have not been experimentally realized. Now nanophotonic quantum skyrmions are generated using a semiconductor quantum dot–Gaussian microcavity quantum electrodynamics system.},
  archive      = {J_NPHYS},
  author       = {Ma, Jiantao and Yang, Jiawei and Liu, Shunfa and Chen, Bo and Li, Xueshi and Song, Changkun and Qiu, Guixin and Zou, Kai and Hu, Xiaolong and Li, Feng and Yu, Ying and Liu, Jin},
  doi          = {10.1038/s41567-025-02973-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1462-1468},
  shortjournal = {Nat. Phys.},
  title        = {Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The actin cortex acts as a mechanical memory of morphology in confined migrating cells. <em>NPHYS</em>, <em>21</em>(9), 1451-1461. (<a href='https://doi.org/10.1038/s41567-025-02980-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell migration in narrow microenvironments occurs in numerous physiological processes. It involves successive cycles of confinement and release that drive important morphological changes. However, it remains unclear whether migrating cells can retain a memory of their past morphological states that could potentially facilitate their navigation through confined spaces. We demonstrate that local geometry governs a switch between two cell morphologies, thereby facilitating cell passage through long and narrow gaps. We combined cell migration assays on standardized microsystems with biophysical modelling and biochemical perturbations to show that migrating cells have a long-term memory of past confinement events. The morphological cell states correlate across transitions through actin cortex remodelling. These findings indicate that mechanical memory in migrating cells plays an active role in their migratory potential in confined environments. Cells often navigate through confined spaces. Now it is shown that cells retain a mechanical memory of previous confinement events, which makes them more efficient at migrating through narrow microenvironments.},
  archive      = {J_NPHYS},
  author       = {Kalukula, Yohalie and Luciano, Marine and Simanov, Gleb and Charras, Guillaume and Brückner, David B. and Gabriele, Sylvain},
  doi          = {10.1038/s41567-025-02980-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1451-1461},
  shortjournal = {Nat. Phys.},
  title        = {The actin cortex acts as a mechanical memory of morphology in confined migrating cells},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capillary interactions drive the self-organization of bacterial colonies. <em>NPHYS</em>, <em>21</em>(9), 1444-1450. (<a href='https://doi.org/10.1038/s41567-025-02965-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many bacteria inhabit thin water layers on solid surfaces. These thin films occur both naturally—in soils, on hosts and on textiles—and in the laboratory on agar hydrogels. In these environments, cells experience capillary forces, but it is unclear how these forces shape bacterial collective behaviour. Here we show that the water menisci formed around bacteria lead to capillary attraction between cells while still allowing them to slide past one another. We develop an experimental apparatus that allows us to control bacterial collective behaviour by varying the strength and range of capillary forces. Combining three-dimensional imaging and cell tracking with agent-based modelling, we demonstrate that capillary attraction organizes rod-shaped bacteria into densely packed nematic groups and influences their collective dynamics and morphologies. Our results suggest that capillary forces may be a ubiquitous physical ingredient in shaping microbial communities in partially hydrated environments. Bacteria tend to live in thin layers of water on surfaces. Now the capillary forces in these layers are shown to help organize the bacteria into dense packs.},
  archive      = {J_NPHYS},
  author       = {Black, Matthew E. and Fei, Chenyi and Alert, Ricard and Wingreen, Ned S. and Shaevitz, Joshua W.},
  doi          = {10.1038/s41567-025-02965-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1444-1450},
  shortjournal = {Nat. Phys.},
  title        = {Capillary interactions drive the self-organization of bacterial colonies},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion. <em>NPHYS</em>, <em>21</em>(9), 1431-1443. (<a href='https://doi.org/10.1038/s41567-025-02964-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells can sense and respond to their environment. Central to this process is the formation of molecular clutches, which are dynamic linkages between the extracellular matrix and the actin cytoskeleton mediated by integrins and adaptor proteins. Although it is well known that force-dependent interactions between molecular-clutch components are essential for sensing substrate rigidity, the influence of nonlinear adaptor-protein elasticity is poorly understood. Here we show that adaptor-protein elasticity and local interactions between molecular clutches and the extracellular matrix are key to cellular perception of substrate stiffness. We present a semi-analytical theory that integrates experimentally measured force responses of adaptor proteins to describe cell-adhesion sensing. We propose that molecular clutches probe local mechanical substrate properties and collectively function as a differential that allows a retrograde actin flow and substrate movement to occur at different rates while maintaining a stable mechanical coupling between them. Our model reproduces experimentally measured force-loading rates of individual molecular clutches and correctly predicts cell-adhesion behaviour for a range of substrate stiffnesses. The framework presented can be extended to study complex phenomena such as focal-adhesion maturation and cell-type-specific mechanosensing. The cytoskeleton and extracellular matrix are linked by molecular clutches that enable cells to sense their environment. Now it is shown that the elastic properties of the clutches underpin stiffness sensing.},
  archive      = {J_NPHYS},
  author       = {Liu, Ping and Wang, Qiuyu and Dai, Xin and Pei, Lingxuan and Wang, Junfan and Zhao, Weiyi and Johnson, Heath E. and Yao, Mingxi and Efremov, Artem K.},
  doi          = {10.1038/s41567-025-02964-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1431-1443},
  shortjournal = {Nat. Phys.},
  title        = {Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility. <em>NPHYS</em>, <em>21</em>(9), 1420-1430. (<a href='https://doi.org/10.1038/s41567-025-02983-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microtubules are hollow cylindrical cytoskeletal polymers of laterally associated protofilaments that contain head-to-tail aligned ɑ/β-tubulin heterodimers. Although the exposed microtubule exterior is readily accessible to proteins, the mechanism governing the accessibility of the confined microtubule lumen to luminal particles remains unknown. Here we show that certain tubulin family proteins (isotypes) facilitate luminal accessibility because of the mechanical properties and lateral interactions that they confer to the microtubules. We characterized the microtubules reconstituted from defined compositions of Caenorhabditis elegans tubulin isotypes. These tubulin isotypes form microtubules with comparable protofilament numbers but different luminal accessibility. We further revealed the role of tubulin isotypes in regulating the strength of inter-protofilament lateral interactions, which determines luminal accessibility through the mechanosensitivity of reversible protofilament separation. Deformation of the microtubule lattice, which generates stresses exceeding the strength of the lateral interactions, creates gaps between adjacent protofilaments, enhancing the accessibility of the lumen. Together, our findings uncovered the tubulin isotype-dependent mechanical plasticity that confers force sensitivity to the microtubule lattice and modulates the energy barrier for luminal proteins to access the lumen. Proteins interact with both the exterior and interior of microtubules. Here the relationship between microtubule building blocks and the accessibility of the microtubule interior to proteins is clarified.},
  archive      = {J_NPHYS},
  author       = {Ye, Yucheng and Hao, Zheng and Luo, Jingyi and Lam, Wai Hei and Liu, Zheng and Li, Xiang David and Zhai, Yuanliang and Lin, Yuan and Ti, Shih-Chieh},
  doi          = {10.1038/s41567-025-02983-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1420-1430},
  shortjournal = {Nat. Phys.},
  title        = {Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Control of collective activity to crystallize an oscillator gas. <em>NPHYS</em>, <em>21</em>(9), 1412-1419. (<a href='https://doi.org/10.1038/s41567-025-02957-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motility-induced phase separation occurs in assemblies of self-propelled units when activity is coupled negatively to density. By contrast, the consequences of a positive coupling between density and activity on the collective behaviour of active matter remain unexplored. Here we show that collective activity can emerge from such a positive coupling among non-motile building blocks. We perform experiments with self-sustained oscillators powered by contact-charge electrophoresis. Although the oscillators are non-motile by design, they spontaneously form an active gas when confined together. The super-elastic nature of collisions constitutes a positive density–activity coupling and underlies the active gas properties. Elucidating the origin of binary collisions allows us to precisely control the structure of the active gas and its eventual crystallization. Beyond considering the overlooked positive coupling between density and activity, our work suggests that rich collective properties can emerge not only from the symmetry of interactions between active building blocks but also from their adaptable and responsive behaviour. In active matter systems, increasing density usually reduces activity. Now, in a system where density enhances activity, collective motion is shown to arise from non-motile oscillators when they are confined.},
  archive      = {J_NPHYS},
  author       = {Le Blay, Marine and Saldi, Joshua H. K. and Morin, Alexandre},
  doi          = {10.1038/s41567-025-02957-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1412-1419},
  shortjournal = {Nat. Phys.},
  title        = {Control of collective activity to crystallize an oscillator gas},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multistable polar textures in geometrically frustrated nematic liquid crystals. <em>NPHYS</em>, <em>21</em>(9), 1404-1411. (<a href='https://doi.org/10.1038/s41567-025-02966-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to manipulate polar entities with multiple external fields could enable functionalities and applications in spin systems, photonics, metamaterials and soft matter. Liquid crystals that exhibit both a crystalline structure and liquid fluidity represent a promising platform for manipulating phases with polar molecular order, notably ferroelectric ones. However, achieving a polar symmetry is challenging with rod-shaped liquid crystal molecules, which form predominantly apolar nematic phases. Here we report an approach in which a geometric lattice confinement of nematic liquid crystals is used to induce planar polar order on the scale of a mesoscopic metamaterial. We confine the nematic liquid crystal in a micropillar array, forming topological defect–pillar pairs of elastic dipoles with a free top interface in contact with an immiscible fluid. The resulting dipole lattice configurations can be programmed rheologically by flowing the top fluid and maintained even after flow cessation, a phenomenon attributed to orientational multistability of the dipoles. This multimemory effect enables the encoding and reconfiguration of directional information. Overall, these results advance our understanding of topological dipoles under confinement and shear flow, enabling the detection, tracking and recording of flow profiles and could facilitate the development of stimuli-responsive materials. Programmable topological dipoles could be used in stimuli-responsive materials. Now it is shown that confinement of an apolar liquid crystal results in controllable polar arrangements of elastic dipoles.},
  archive      = {J_NPHYS},
  author       = {Kara, Ufuoma I. and Chen, Boyuan and Čopar, Simon and Li, Shucong and Mamtani, Rajdeep and Xu, Yang and Weible, Alan H. and Boerner, Eric C. and Yang, Zhan and Yao, Yuxing and Selinger, Robin L. B. and Tkalec, Uroš and Wang, Xiaoguang},
  doi          = {10.1038/s41567-025-02966-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1404-1411},
  shortjournal = {Nat. Phys.},
  title        = {Multistable polar textures in geometrically frustrated nematic liquid crystals},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability. <em>NPHYS</em>, <em>21</em>(9), 1398-1403. (<a href='https://doi.org/10.1038/s41567-025-02982-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Topological defects give rise to non-trivial excitations—such as solitons, vortices and skyrmions—characterized by integer-valued topological charges. However, their classification becomes difficult when singularities are present in the order parameter field, complicating the computation of these charges. Although exotic nonlinear excitations have been theoretically proposed in the superfluid 3He-A phase and spinor Bose–Einstein condensates, they have not been experimentally observed and their stability has not been investigated. Here we demonstrate the presence of a singular skyrmion that goes beyond the framework of topology in a ferromagnetic superfluid. These skyrmions emerge from anomalous symmetry-breaking associated with an eccentric spin singularity and carry half the elementary charge—a feature that distinguishes them from conventional skyrmions and merons. We realize the universal regime of the quantum Kelvin–Helmholtz instability, and we identify the unconventional fractional skyrmions produced by emission from a magnetic domain wall and the spontaneous splitting of an integer skyrmion with spin singularities. The singular skyrmions are stable even after 2 s of hold time. Our results confirm the universality between classical and quantum Kelvin–Helmholtz instabilities and broaden our understanding of complex nonlinear dynamics for a non-trivial texture in topological quantum systems. The quantum Kelvin–Helmholtz instability can induce non-trivial topological excitations. Now a proof-of-principle experiment demonstrates the emergence of a stable singular fractional skyrmion spin texture originating from this quantum instability.},
  archive      = {J_NPHYS},
  author       = {Huh, SeungJung and Yun, Wooyoung and Yun, Gabin and Hwang, Samgyu and Kwon, Kiryang and Hur, Junhyeok and Lee, Seungho and Takeuchi, Hiromitsu and Kim, Se Kwon and Choi, Jae-yoon},
  doi          = {10.1038/s41567-025-02982-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1398-1403},
  shortjournal = {Nat. Phys.},
  title        = {Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Anyonic braiding in a chiral Mach–Zehnder interferometer. <em>NPHYS</em>, <em>21</em>(9), 1392-1397. (<a href='https://doi.org/10.1038/s41567-025-02960-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fractional quantum statistics are the defining characteristic of anyons. Measuring the phase generated by an exchange of anyons is challenging, as standard interferometry set-ups—such as the Fabry–Pérot interferometer—suffer from charging effects that obscure the interference signal. Here we present the observation of anyonic interference and exchange phases in an optical-like Mach–Zehnder interferometer based on co-propagating interface modes. By avoiding backscattering and deleterious charging effects, this set-up enables pristine and robust Aharonov–Bohm interference without any phase slips. At various fractional filling factors, the observed flux periodicities agree with the fundamental fractionally charged excitations that correspond to Jain states and depend only on the bulk topological order. To probe the anyonic statistics, we used a small, charged top gate in the interferometer bulk to induce localized quasiparticles without modifying the Aharonov–Bohm phase. The added quasiparticles introduce periodic phase slips. The sign and magnitude of the observed phase slips align with the expected value at filling 1/3, but their direction shows systematic deviations at fillings 2/5 and 3/7. Control over added individual quasiparticles in this design is essential for measuring the coveted non-abelian statistics in the future. An interferometer design allows systematic investigation of the anyonic statistics of bulk fractional quantum Hall states.},
  archive      = {J_NPHYS},
  author       = {Ghosh, Bikash and Labendik, Maria and Musina, Liliia and Umansky, Vladimir and Heiblum, Moty and Mross, David F.},
  doi          = {10.1038/s41567-025-02960-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1392-1397},
  shortjournal = {Nat. Phys.},
  title        = {Anyonic braiding in a chiral Mach–Zehnder interferometer},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Measurement of phonon angular momentum. <em>NPHYS</em>, <em>21</em>(9), 1387-1391. (<a href='https://doi.org/10.1038/s41567-025-02952-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In condensed matter, angular momentum is intimately related to the emergence of topological quantum states, including chiral superconductivity, quantum spin liquids and various chiral quasiparticles. Recently, it has been predicted that microscopic lattice excitations, known as phonons, can carry finite angular momentum, leading to specific physical properties of materials. However, phonon angular momentum has not yet been observed directly. Here we demonstrate that angular momentum conservation results in a macroscopic mechanical torque when applying a time-reversal symmetry-breaking thermal gradient along the chiral axis of single-crystal tellurium. We probe this torque using a cantilever-based device and establish that it changes sign by flipping the chirality or thermal gradient. This behavior disappears in polycrystalline samples that lack a preferred chirality. Our experimental results align well with theoretical calculations. We provide compelling evidence for phonon angular momentum, which might enable quantum states with potential applications in microelectronics. Theory predicts that phonons—quanta of lattice vibrations—can carry finite angular momentum and thus influence physical properties of materials. Now phonons with angular momentum have been seen in tellurium with a chiral crystal structure.},
  archive      = {J_NPHYS},
  author       = {Zhang, H. and Peshcherenko, N. and Yang, Fazhi and Ward, T. Z. and Raghuvanshi, P. and Lindsay, L. and Felser, Claudia and Zhang, Yang and Yan, J.-Q. and Miao, H.},
  doi          = {10.1038/s41567-025-02952-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1387-1391},
  shortjournal = {Nat. Phys.},
  title        = {Measurement of phonon angular momentum},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Strongly interacting hofstadter states in magic-angle twisted bilayer graphene. <em>NPHYS</em>, <em>21</em>(9), 1380-1386. (<a href='https://doi.org/10.1038/s41567-025-02997-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Magic-angle twisted bilayer graphene hosts a variety of strongly correlated states at partial fillings of its flat bands. In a magnetic field, these flat bands evolve into a Hofstadter spectrum renormalized by strong Coulomb interactions. Here we study the interacting Hofstadter states that spontaneously form within the topological magnetic sub-bands of an ultraclean magic-angle twisted bilayer graphene device, including symmetry-broken Chern insulator states and fractional quantum Hall states. The observed symmetry-broken Chern insulator states form a cascade, with their Chern numbers mimicking the main sequence of correlated Chern insulators. The fractional quantum Hall states form in a Jain sequence. However, they disappear at high magnetic field, in contrast to conventional fractional quantum Hall states that strengthen with increasing magnetic field. We reveal a magnetic-field-driven phase transition from composite fermion phases to a dissipative Fermi liquid. Our theoretical analysis of the magnetic sub-bands hosting the fractional quantum Hall states predicts non-uniform quantum geometric properties far from the lowest Landau level. This points towards a more natural interpretation of these states as in-field fractional Chern insulators of the magnetic sub-bands. Interacting electrons in Hofstadter bands can form symmetry-broken topological states. These are now revealed in magic-angle twisted bilayer graphene, and their properties are influenced by non-uniform quantum geometry.},
  archive      = {J_NPHYS},
  author       = {He, Minhao and Wang, Xiaoyu and Cai, Jiaqi and Herzog-Arbeitman, Jonah and Peng, Ran and Taniguchi, Takashi and Watanabe, Kenji and Stern, Ady and Bernevig, B. Andrei and Yankowitz, Matthew and Vafek, Oskar and Xu, Xiaodong},
  doi          = {10.1038/s41567-025-02997-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1380-1386},
  shortjournal = {Nat. Phys.},
  title        = {Strongly interacting hofstadter states in magic-angle twisted bilayer graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Universality in quantum critical flow of charge and heat in ultraclean graphene. <em>NPHYS</em>, <em>21</em>(9), 1374-1379. (<a href='https://doi.org/10.1038/s41567-025-02972-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Close to the Dirac point, graphene is expected to exist in a quantum critical Dirac fluid state, where the flow of both charge and heat can be described with a characteristic d.c. electrical conductivity and thermodynamic variables such as entropy and enthalpy densities. Although the fluid-like viscous flow of charge has been reported in state-of-the-art graphene devices, the value of conductivity, predicted to be quantized and determined only by the universality class of the critical point, has not been established experimentally so far. Here we have discerned the quantum critical universality in graphene transport by combining the electrical and thermal conductivities in very high-quality devices close to the Dirac point. We find that they are inversely related, as expected from relativistic hydrodynamics, and the characteristic conductivity converges to a quantized value. We also observe a giant violation of the Wiedemann–Franz law, where the Lorentz number exceeds the semiclassical value by more than 200 times close to the Dirac point at low temperatures. At high temperatures, the effective dynamic viscosity to entropy density ratio close to the Dirac point in the cleanest devices approaches that of a minimally viscous quantum fluid within a factor of four. Critical behaviour is expected in graphene when the carrier density is tuned to the Dirac point. Now, universality associated with the critical point is observed in electronic and thermal transport.},
  archive      = {J_NPHYS},
  author       = {Majumdar, Aniket and Chadha, Nisarg and Pal, Pritam and Gugnani, Akash and Ghawri, Bhaskar and Watanabe, Kenji and Taniguchi, Takashi and Mukerjee, Subroto and Ghosh, Arindam},
  doi          = {10.1038/s41567-025-02972-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1374-1379},
  shortjournal = {Nat. Phys.},
  title        = {Universality in quantum critical flow of charge and heat in ultraclean graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerator technologies for proton and ion beam therapy. <em>NPHYS</em>, <em>21</em>(9), 1363-1373. (<a href='https://doi.org/10.1038/s41567-025-02994-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past 75 years, proton beam therapy has emerged as a promising modality for cancer treatment, boasting precise targeting and reduced collateral damage to healthy tissue. Here we discuss the evolution of accelerator technology in proton therapy, examining advancements in cyclotron, synchrotron and linear accelerator technology, and their implications for modern treatment delivery. Additionally, we explore advances in delivering accelerated carbon or helium ions for therapeutic treatments. We also discuss the integration of advanced imaging modalities, such as multienergy X-ray, magnetic resonance imaging and ion-based imaging, for real-time monitoring and adaptive radiotherapy. These advancements position particle therapy to offer personalized and effective cancer treatment strategies, heralding improved patient outcomes. Proton and heavy-ion beam therapies are promising approaches for cancer treatment. This Review discusses the accelerator technologies that enable these therapies.},
  archive      = {J_NPHYS},
  author       = {Maradia, Vivek and Clasie, Benjamin and Snively, Emma and Parodi, Katia and Schwarz, Marco and Durante, Marco},
  doi          = {10.1038/s41567-025-02994-7},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1363-1373},
  shortjournal = {Nat. Phys.},
  title        = {Accelerator technologies for proton and ion beam therapy},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Extended native gate sets to unlock the performance of quantum processors. <em>NPHYS</em>, <em>21</em>(9), 1361-1362. (<a href='https://doi.org/10.1038/s41567-025-02991-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum operations are modelled as unitary matrices, yet experimental implementations have been restricted to only a few gate types. This fundamental limitation has now been overcome using a scheme that combines frequency control with microwave driving, enabling universal implementation of arbitrary two-qubit operations.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02991-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1361-1362},
  shortjournal = {Nat. Phys.},
  title        = {Extended native gate sets to unlock the performance of quantum processors},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How squeezed cells remember their shape to migrate efficiently. <em>NPHYS</em>, <em>21</em>(9), 1359-1360. (<a href='https://doi.org/10.1038/s41567-025-02984-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells migrating through narrow spaces in their environment undergo repeated shape changes to pass through tight constrictions. Epithelial cells retain a memory of past confinement, allowing them to maintain a polarized, compact morphology that enhances future migration through narrow gaps. This memory is mechanically encoded in the actin cortex.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02984-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1359-1360},
  shortjournal = {Nat. Phys.},
  title        = {How squeezed cells remember their shape to migrate efficiently},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantized limit of conductivity in near-ideal graphene. <em>NPHYS</em>, <em>21</em>(9), 1357-1358. (<a href='https://doi.org/10.1038/s41567-025-02974-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transport properties near the Dirac point in graphene are expected to be determined by quantum many-body interactions between relativistic electrons. Experiments now show that the flow of charge and heat in high-quality graphene close to charge neutrality can be described within a hydrodynamic framework, with universal intrinsic electrical conductivity that is quantized to a value close to the quantum of conductance.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02974-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1357-1358},
  shortjournal = {Nat. Phys.},
  title        = {Quantized limit of conductivity in near-ideal graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boron at the boundaries. <em>NPHYS</em>, <em>21</em>(9), 1356. (<a href='https://doi.org/10.1038/s41567-025-03041-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Brierley, Richard},
  doi          = {10.1038/s41567-025-03041-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1356},
  shortjournal = {Nat. Phys.},
  title        = {Boron at the boundaries},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Liquid dipole lattice. <em>NPHYS</em>, <em>21</em>(9), 1354-1355. (<a href='https://doi.org/10.1038/s41567-025-02992-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geometrically confined apolar nematic liquid crystals lose rotational symmetry and form a multifunctional polar lattice of fluid elastic dipoles with controllable orientation. The lattice is of interest from a fundamental science perspective as well as having potential applications.},
  archive      = {J_NPHYS},
  author       = {Lucchetti, Liana},
  doi          = {10.1038/s41567-025-02992-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1354-1355},
  shortjournal = {Nat. Phys.},
  title        = {Liquid dipole lattice},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How to twist anyons. <em>NPHYS</em>, <em>21</em>(9), 1352-1353. (<a href='https://doi.org/10.1038/s41567-025-03009-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The properties of quantum Hall states can be probed in experiments inspired by optical interferometers. Now, a device design that overcomes major drawbacks of existing interferometers allows the extraction of the quasiparticle statistics.},
  archive      = {J_NPHYS},
  author       = {Carrega, Matteo and Heun, Stefan},
  doi          = {10.1038/s41567-025-03009-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1352-1353},
  shortjournal = {Nat. Phys.},
  title        = {How to twist anyons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In plain sight. <em>NPHYS</em>, <em>21</em>(9), 1351. (<a href='https://doi.org/10.1038/s41567-025-03021-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Buchanan, Mark},
  doi          = {10.1038/s41567-025-03021-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1351},
  shortjournal = {Nat. Phys.},
  title        = {In plain sight},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Invest in science education research to make science open to all. <em>NPHYS</em>, <em>21</em>(9), 1350. (<a href='https://doi.org/10.1038/s41567-025-03019-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attacks on Diversity, Equity and Inclusion efforts expose the need for science education research.},
  archive      = {J_NPHYS},
  author       = {Holmes, N. G.},
  doi          = {10.1038/s41567-025-03019-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1350},
  shortjournal = {Nat. Phys.},
  title        = {Invest in science education research to make science open to all},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Serendipitous signal. <em>NPHYS</em>, <em>21</em>(9), 1349. (<a href='https://doi.org/10.1038/s41567-025-03052-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first direct observation of gravitational waves occurred on 14 September 2015. We look back at ten years of gravitational wave science.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-03052-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1349},
  shortjournal = {Nat. Phys.},
  title        = {Serendipitous signal},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
