<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>nature</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs">NATCS - 12</h2>
<ul>
<li><details>
<summary>
(2025). Privacy-preserving multicenter differential protein abundance analysis with FedProt. <em>NATCS</em>, <em>5</em>(8), 675-688. (<a href='https://doi.org/10.1038/s43588-025-00832-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative mass spectrometry has revolutionized proteomics by enabling simultaneous quantification of thousands of proteins. Pooling patient-derived data from multiple institutions enhances statistical power but raises serious privacy concerns. Here we introduce FedProt, the first privacy-preserving tool for collaborative differential protein abundance analysis of distributed data, which utilizes federated learning and additive secret sharing. In the absence of a multicenter patient-derived dataset for evaluation, we created two: one at five centers from E. coli experiments and one at three centers from human serum. Evaluations using these datasets confirm that FedProt achieves accuracy equivalent to the DEqMS method applied to pooled data, with completely negligible absolute differences no greater than 4 × 10−12. By contrast, −log10P computed by the most accurate meta-analysis methods diverged from the centralized analysis results by up to 25–26. In this Resource, the authors present FedProt, a tool that enables privacy-preserving, federated differential protein abundance analysis across multiple institutions. Its results match the results of centralized analysis, enabling secure, collaborative proteomics without sensitive data sharing.},
  archive      = {J_NATCS},
  author       = {Burankova, Yuliya and Abele, Miriam and Bakhtiari, Mohammad and von Toerne, Christine and Barth, Teresa K. and Schweizer, Lisa and Giesbertz, Pieter and Schmidt, Johannes R. and Kalkhof, Stefan and Müller-Deile, Janina and van Veelen, Peter A. and Mohammed, Yassene and Hammer, Elke and Arend, Lis and Adamowicz, Klaudia and Laske, Tanja and Hartebrodt, Anne and Frisch, Tobias and Meng, Chen and Matschinske, Julian and Späth, Julian and Röttger, Richard and Schwämmle, Veit and Hauck, Stefanie M. and Lichtenthaler, Stefan F. and Imhof, Axel and Mann, Matthias and Ludwig, Christina and Kuster, Bernhard and Baumbach, Jan and Zolotareva, Olga},
  doi          = {10.1038/s43588-025-00832-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {675-688},
  shortjournal = {Nat. Comput. Sci},
  title        = {Privacy-preserving multicenter differential protein abundance analysis with FedProt},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comprehensive mapping of zeolite–template chemical space. <em>NATCS</em>, <em>5</em>(8), 661-674. (<a href='https://doi.org/10.1038/s43588-025-00842-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zeolites are industrially important catalysts and adsorbents, typically synthesized using specific molecules known as organic structure-directing agents (OSDAs). The templating effect of the OSDAs is pivotal in determining the zeolite polymorph formed and its physicochemical properties. However, de novo design of selective OSDAs is challenging because of the diversity and size of the zeolite–OSDA chemical space. Here we present ZeoBind, a computational workflow powered by machine learning that enables an exhaustive exploration of the OSDA space. We design predictive tasks that capture zeolite–molecule matching, train predictive models for these tasks on hundreds of thousands of datapoints and curate a library of 2.3 million synthetically accessible, hypothetical OSDA-like molecules enumerated from commercially available precursors. We use ZeoBind to screen nearly 500 million zeolite–molecule pairs and identified and experimentally validated two new OSDAs that template zeolites with novel compositions. The scale of the OSDA library, along with the open-access tools and data, has the potential to accelerate OSDA design for zeolite synthesis. ZeoBind is developed for high-throughput molecule screening in zeolite synthesis. Here 2.3 million organic structure-directing agents are enumerated and predictive models for binding affinity are developed; the screening is experimentally validated for two zeolites.},
  archive      = {J_NATCS},
  author       = {Xie, Mingrou and Schwalbe-Koda, Daniel and Semanate-Esquivel, Yolanda Marcela and Bello-Jurado, Estefanía and Hoffman, Alexander and Santiago-Reyes, Omar and Paris, Cecilia and Moliner, Manuel and Gómez-Bombarelli, Rafael},
  doi          = {10.1038/s43588-025-00842-5},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {661-674},
  shortjournal = {Nat. Comput. Sci},
  title        = {A comprehensive mapping of zeolite–template chemical space},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). RNA–ligand interaction scoring via data perturbation and augmentation modeling. <em>NATCS</em>, <em>5</em>(8), 648-660. (<a href='https://doi.org/10.1038/s43588-025-00820-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent advances in RNA-targeting drug discovery, the development of data-driven deep learning models remains challenging owing to limited validated RNA–small molecule interaction data and scarce known RNA structures. In this context, we introduce RNAsmol, a sequence-based deep learning framework that incorporates data perturbation with augmentation, graph-based molecular feature representation and attention-based feature fusion modules to predict RNA–small molecule interactions. RNAsmol employs perturbation strategies to balance the bias between the true negative and unknown interaction space, thereby elucidating the intrinsic binding patterns between RNA and small molecules. The resulting model demonstrates accurate predictions of the binding between RNA and small molecules, outperforming other methods in ten-fold cross-validation, unseen evaluation and decoy evaluation. Moreover, we use case studies to visualize molecular binding profiles and the distribution of learned weights, providing interpretable insights into RNAsmol’s predictions. In particular, without requiring structural input, RNAsmol can generate reliable predictions and be adapted to various drug design scenarios. RNAsmol is a sequence-based deep learning framework for predicting RNA–small molecule interactions, integrating perturbation and augmentation to achieve robust predictions under limited data and adaptability to diverse drug discovery scenarios.},
  archive      = {J_NATCS},
  author       = {Ma, Hongli and Gao, Letian and Jin, Yunfan and Ma, Jianwei and Bai, Yilan and Liu, Xiaofan and Bao, Pengfei and Liu, Ke and Xu, Zhenjiang Zech and Lu, Zhi John},
  doi          = {10.1038/s43588-025-00820-x},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {648-660},
  shortjournal = {Nat. Comput. Sci},
  title        = {RNA–ligand interaction scoring via data perturbation and augmentation modeling},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Investigating the volume and diversity of data needed for generalizable antibody–antigen ΔΔG prediction. <em>NATCS</em>, <em>5</em>(8), 635-647. (<a href='https://doi.org/10.1038/s43588-025-00823-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antibody–antigen binding affinity lies at the heart of therapeutic antibody development: efficacy is guided by specific binding and control of affinity. Here we present Graphinity, an equivariant graph neural network architecture built directly from antibody–antigen structures that achieves test Pearson’s correlations of up to 0.87 on experimental change in binding affinity (ΔΔG) prediction. However, our model, like previous methods, appears to be overtraining on the few hundred experimental data points available and performance is not robust to train–test cut-offs. To investigate the amount and type of data required to generalizably predict ΔΔG, we built synthetic datasets of nearly 1 million FoldX-generated and &gt;20,000 Rosetta Flex ddG-generated ΔΔG values. Our results indicate that there are currently insufficient experimental data to accurately and robustly predict ΔΔG, with orders of magnitude more likely needed. Dataset size is not the only consideration; diversity is also an important factor for model predictiveness. These findings provide a lower bound on data requirements to inform future method development and data collection efforts. Predicting the effects of mutations on antibody–antigen binding is a key challenge in therapeutic antibody development. Orders of magnitude more data will be needed to unlock accurate, robust prediction.},
  archive      = {J_NATCS},
  author       = {Hummer, Alissa M. and Schneider, Constantin and Chinery, Lewis and Deane, Charlotte M.},
  doi          = {10.1038/s43588-025-00823-8},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {635-647},
  shortjournal = {Nat. Comput. Sci},
  title        = {Investigating the volume and diversity of data needed for generalizable antibody–antigen ΔΔG prediction},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A large-scale replication of scenario-based experiments in psychology and management using large language models. <em>NATCS</em>, <em>5</em>(8), 627-634. (<a href='https://doi.org/10.1038/s43588-025-00840-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted a large-scale study replicating 156 psychological experiments from top social science journals using three state-of-the-art large language models (LLMs). Our results reveal that, while LLMs demonstrated high replication rates for main effects (73–81%) and moderate to strong success with interaction effects (46–63%), they consistently produced larger effect sizes than human studies. Notably, LLMs showed significantly lower replication rates for studies involving socially sensitive topics such as race, gender and ethics. When original studies reported null findings, LLMs produced significant results at remarkably high rates (68–83%); while this could reflect cleaner data with less noise, it also suggests potential risks of effect size overestimation. Our results demonstrate both the promises and the challenges of LLMs in psychological research: while LLMs are efficient tools for pilot testing and rapid hypothesis validation, enriching rather than replacing traditional human-participant studies, they require more nuanced interpretation and human validation for complex social phenomena and culturally sensitive research questions. Researchers replicated 156 psychological experiments using three large language models (LLMs) instead of human participants. LLMs achieved 73–81% replication rates but showed amplified effect sizes and challenges with socially sensitive topics.},
  archive      = {J_NATCS},
  author       = {Cui, Ziyan and Li, Ning and Zhou, Huaikang},
  doi          = {10.1038/s43588-025-00840-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {627-634},
  shortjournal = {Nat. Comput. Sci},
  title        = {A large-scale replication of scenario-based experiments in psychology and management using large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A formally exact method for high-throughput absolute binding-free-energy calculations. <em>NATCS</em>, <em>5</em>(8), 621-626. (<a href='https://doi.org/10.1038/s43588-025-00821-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we introduce a high-throughput, formally exact method for absolute binding-free-energy calculations that enhances computational efficiency and accuracy. At the core of this method is a thermodynamic cycle that minimizes protein ligand relative motion, thereby reducing system perturbations and driving a fourfold gain in efficiency over the traditional double-decoupling method. By combining this strategy with double-wide sampling and hydrogen-mass repartitioning algorithms, the efficiency is further boosted to eightfold. The presented method is applied to 45 diverse protein–ligand complexes. For 34 complexes with validated force-field accuracy, our method achieves an average unsigned error of less than 1 kcal mol−1 and a hysteresis below 0.5 kcal mol−1, showcasing exceptional reliability. Moreover, it efficiently manages flexible peptide ligands through a potential-of-mean-force calculation, adding less than 5% extra simulation time. For 11 challenging cases, the presented method also shows an improvement compared with previously published results. Put together, this method has potential for advancing research in physical, biological and medicinal chemistry. A thermodynamic cycle is proposed that limits protein–ligand relative motion and boosts the efficiency of absolute binding-free-energy calculations four- to eightfold, without compromising theoretical rigor.},
  archive      = {J_NATCS},
  author       = {Bian, Hengwei and Shao, Xueguang and Chipot, Christophe and Cai, Wensheng and Fu, Haohao},
  doi          = {10.1038/s43588-025-00821-w},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {621-626},
  shortjournal = {Nat. Comput. Sci},
  title        = {A formally exact method for high-throughput absolute binding-free-energy calculations},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying batch effects for individual genes in single-cell data. <em>NATCS</em>, <em>5</em>(8), 612-620. (<a href='https://doi.org/10.1038/s43588-025-00824-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Batch effects substantially impede the comparison of multiple single-cell experiment batches. Existing methods for batch effect removal and quantification primarily emphasize cell alignment across batches, often overlooking gene-level batch effects. Here we introduce group technical effects (GTE)—a quantitative metric to assess batch effects on individual genes. Using GTE, we show that batch effects unevenly impact genes within the dataset. A portion of highly batch-sensitive genes (HBGs) differ between datasets and dominate the batch effects, whereas non-HBGs exhibit low batch effects. We demonstrate that as few as three HBGs are sufficient to introduce substantial batch effects. Our method also enables the assessment of cell-level batch effects, outperforming existing batch effect quantification methods. We also observe that biologically similar cell types undergo similar batch effects, informing the development of data integration strategies. The GTE method is versatile and applicable to various single-cell omics data types. Group technical effects, a metric to quantify gene-level batch effects in single-cell data, is proposed. The analysis based on group technical effects reveals the unbalanced batch effects across genes.},
  archive      = {J_NATCS},
  author       = {Zhou, Yang and Sheng, Qiongyu and Wang, Guohua and Xu, Li and Jin, Shuilin},
  doi          = {10.1038/s43588-025-00824-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {612-620},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying batch effects for individual genes in single-cell data},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unbalanced gene-level batch effects in single-cell data. <em>NATCS</em>, <em>5</em>(8), 610-611. (<a href='https://doi.org/10.1038/s43588-025-00829-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed group technical effects (GTE) as a quantitative metric for evaluating gene-level batch effects in single-cell data. It identifies highly batch-sensitive genes — the primary contributors to batch effects — that vary across datasets, and whose removal effectively mitigates the batch effects.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00829-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {610-611},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unbalanced gene-level batch effects in single-cell data},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diving deep into zeolite space. <em>NATCS</em>, <em>5</em>(8), 608-609. (<a href='https://doi.org/10.1038/s43588-025-00831-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposed ZeoBind, an AI-accelerated workflow enabling the discovery and experimental verification of hits within chemical spaces containing hundreds of millions of zeolites.},
  archive      = {J_NATCS},
  author       = {Balcells, David},
  doi          = {10.1038/s43588-025-00831-8},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {608-609},
  shortjournal = {Nat. Comput. Sci},
  title        = {Diving deep into zeolite space},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What’s so hard about RNA-targeting drug discovery?. <em>NATCS</em>, <em>5</em>(8), 606-607. (<a href='https://doi.org/10.1038/s43588-025-00853-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An artificial intelligence model leverages language models and molecular perturbations to tackle the data scarcity problem in the RNA drug world.},
  archive      = {J_NATCS},
  author       = {Oliver, Carlos and Waldispühl, Jérôme},
  doi          = {10.1038/s43588-025-00853-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {606-607},
  shortjournal = {Nat. Comput. Sci},
  title        = {What’s so hard about RNA-targeting drug discovery?},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Experimenting on AI can (sometimes) teach us about ourselves. <em>NATCS</em>, <em>5</em>(8), 604-605. (<a href='https://doi.org/10.1038/s43588-025-00838-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study sought to replicate published experimental research using large language models, finding that human behavior is replicated surprisingly well overall, but deviates in important ways that could lead social scientists astray.},
  archive      = {J_NATCS},
  author       = {van Loon, Austin and Heidenry, Zoe},
  doi          = {10.1038/s43588-025-00838-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {604-605},
  shortjournal = {Nat. Comput. Sci},
  title        = {Experimenting on AI can (sometimes) teach us about ourselves},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pushing the frontiers in computational chemistry. <em>NATCS</em>, <em>5</em>(8), 603. (<a href='https://doi.org/10.1038/s43588-025-00862-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We reflect on ongoing topics and discussions at conferences in the field of computational chemistry.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00862-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {603},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pushing the frontiers in computational chemistry},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natee">NATEE - 33</h2>
<ul>
<li><details>
<summary>
(2025). Citrus reticulata (mandarin). <em>NATEE</em>, <em>9</em>(9), 1752. (<a href='https://doi.org/10.1038/s41559-025-02822-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Qiang Xu has been captivated by citrus fruits since childhood.},
  archive      = {J_NATEE},
  author       = {Xu, Qiang},
  doi          = {10.1038/s41559-025-02822-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1752},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Citrus reticulata (mandarin)},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease. <em>NATEE</em>, <em>9</em>(9), 1739-1751. (<a href='https://doi.org/10.1038/s41559-025-02797-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More than 10 years following the onset of the sea star wasting disease (SSWD) epidemic, affecting over 20 asteroid species from Mexico to Alaska, the causative agent has been elusive. SSWD killed billions of the most susceptible species, sunflower sea stars (Pycnopodia helianthoides), initiating a trophic cascade involving unchecked urchin population growth and the widespread loss of kelp forests. Identifying the causative agent underpins the development of recovery strategies. Here we induced disease and subsequent mortality in exposure experiments using tissue extracts, coelomic fluid and effluent water from wasting sunflower sea stars, with no mortality in controls. Deep sequencing of diseased sea star coelomic fluid samples from experiments and field outbreaks revealed a dominant proportion of reads assigned to the bacterium Vibrio pectenicida. Fulfilling Koch’s postulates, V. pectenicida strain FHCF-3, cultured from the coelomic fluid of a diseased sunflower sea star, caused disease and mortality in exposed sunflower sea stars, demonstrating that it is a causative agent of SSWD. This discovery will enable recovery efforts for sea stars and the ecosystems affected by their decline by facilitating culture-based experimental research and broad-scale screening for pathogen presence and abundance in the laboratory and field. The causative agent of sea star wasting disease has been elusive. This study used genetic datasets and experimental exposures to demonstrate that a strain of the bacterium Vibrio pectenicida caused disease and mortality in sea stars.},
  archive      = {J_NATEE},
  author       = {Prentice, Melanie B. and Crandall, Grace A. and Chan, Amy M. and Davis, Katherine M. and Hershberger, Paul K. and Finke, Jan F. and Hodin, Jason and McCracken, Andrew and Kellogg, Colleen T. E. and Clemente-Carvalho, Rute B. G. and Prentice, Carolyn and Zhong, Kevin X. and Harvell, C. Drew and Suttle, Curtis A. and Gehman, Alyssa-Lois M.},
  doi          = {10.1038/s41559-025-02797-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1739-1751},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Vibrio pectenicida strain FHCF-3 is a causative agent of sea star wasting disease},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories. <em>NATEE</em>, <em>9</em>(9), 1731-1738. (<a href='https://doi.org/10.1038/s41559-025-02798-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Radiogenic strontium isotopes (87Sr/86Sr) and the alkaline earth ratios (AERs) Sr/Ca and Ba/Ca in fossil dental enamel can inform the habitat, residence and life histories of early hominins recovered from the Pleistocene cradle-of-humankind sites of Gauteng, South Africa. Key questions, which may be addressed with these indices, are the relative exploitation of wet versus dry botanic regimes and whether early hominins dispersed in a manner similar to that of chimpanzees (characterized by male philopatry and female dispersal at puberty) or to that of humans (who are not so characterized). Here we developed 28 new dental chronologies in 20 Paranthropus robustus teeth from Swartkrans and Kromdraai. Resulting geochemical time series demonstrate that, while maternal 87Sr/86Sr in earlier-forming teeth varies widely, third molar 87Sr/86Sr, derived from postweaning solid foods, progressively converges to 0.7306 ± 0.0035 (± 2 s.d.), which we express as the local isotopically delineated exploitation area (LIDEA). The spatial resolution of LIDEA is determined using a bioavailable 87Sr/86Sr isoscape. In this environmental context, we interpret LIDEA as a quantifiable signal indicating eurytopy (generalization), with some 30% of Sr deriving from riparian woodland habitats. With regard to residence, many individuals arrived at the site after second molar mineralization, while some matured locally, demonstrating both male and female dispersal as well as lifelong local residence. Analysis of both 87Sr/86Sr and the AERs further highlights concomitant patterns, as well as numerous periodicities that may be related to resource depletion, seasonality or lunar cycles. Strontium analysis of Paranthropus robustus teeth from two South African sites reveals that the Pleistocene hominin probably exploited both savanna and riparian woodland habitats, had a home-range size compatible with that of present-day hunter-gatherers and had periodic ranging patterns. While some individuals lived their lives in the same place, others dispersed during their lifetime.},
  archive      = {J_NATEE},
  author       = {Sillen, Andrew and Dean, Christopher and Balter, Vincent},
  doi          = {10.1038/s41559-025-02798-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1731-1738},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Geochemical chronologies in paranthropus robustus teeth inform habitat and life histories},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas. <em>NATEE</em>, <em>9</em>(9), 1719-1730. (<a href='https://doi.org/10.1038/s41559-025-02794-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most faunas from the Mesozoic era were dominated by sauropod dinosaurs, the largest terrestrial animals to ever exist. These megaherbivores were remarkably diverse and widely distributed. Here we study three Late Jurassic faunas from the USA, Portugal and Tanzania, each approximately 150 million years old, which are known for their extreme sauropod diversity. Whereas general taxonomic composition was similar in these three faunas, the major clades differed in relative abundance. Moreover, their depositional strata record distinct climatic regimes. Using dental microwear texture analysis, we investigated the impact of these climate regimes and the resulting food availability on the different sauropod taxa. Wear patterns in camarasaurid macronarians show minimal variation across different climate regimes, supporting previous studies suggesting that these animals migrated to follow their preferred climate niche and food source. North American camarasaurids show similar wear patterns to those of Portuguese turiasaurs, another broad-crowned taxon, which did not exist in the Jurassic of North America. By contrast, where camarasaurids and turiasaurs co-occurred in Portugal, their microwear patterns are distinct, suggesting niche differentiation to avoid ecological competition between these two clades. Flagellicaudatan diplodocoids display highly variable wear patterns, indicating limited migration (and therefore seasonal variation in diet), which aligns with observed biogeography patterns in the USA. Early-branching titanosauriforms show highly distinct wear patterns between different climate regimes, which can probably be attributed to different abrasive loads in the respective habitats. Our results demonstrate that dental microwear texture analysis not only records dietary preferences but also reveals behaviour such as competition and migration related to dietary niches in past ecosystems. The authors compare tooth wear patterns from three Late Jurassic sauropod faunas in North America, Portugal and Tanzania and find that different regions and climate regimes seem to have had distinctive impacts on dental wear of megaherbivores in these areas, revealing habitat and ecological niche information.},
  archive      = {J_NATEE},
  author       = {Winkler, Daniela E. and Tschopp, Emanuel and Saleiro, André and Wiesinger, Ria and Kaiser, Thomas M.},
  doi          = {10.1038/s41559-025-02794-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1719-1730},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dental microwear texture analysis reveals behavioural, ecological and habitat signals in late jurassic sauropod dinosaur faunas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enlargement of sternum traits facilitated the evolution of powered flight in birds. <em>NATEE</em>, <em>9</em>(9), 1705-1718. (<a href='https://doi.org/10.1038/s41559-025-02795-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An enlarged sternum with a prominent keel is a central feature of the flight apparatus of modern birds. However, sterna of near-bird dinosaurs (Pennaraptora) and early avialans are either substantially different from those of living birds or absent altogether, raising questions about how specialized sternal structures evolved in birds and how they are related to function. This remains poorly understood because of the fragmentary nature of the fossil record, and the challenges in inferring form and function from crushed fossils. We use ancestral character estimations to trace sternal trait acquisition through the bird stem group, and multivariate phylogenetic regressions to analyse relationships between sternum morphology, body mass and flight capabilities. We find that sternum evolution was episodic: basal members of Pennaraptora had proportionally small sterna, which became larger and more craniocaudally elongated in Avialae. This enlargement precedes the appearance of a midline ridge, a possible precursor of the sternal keel, in Pygostylia. Sternum size increased again in crownward Ornithuromorpha, alongside a fully formed sternal keel and enlarged caudal projections, both critical areas of flight muscle attachment. Sternal experimentation in relation to flight characteristics occurs several times throughout Pennaraptora, including within Paraves and Enantiornithes, indicating that powered flight may have evolved several times before proliferating in crown-group birds. The extant avian sternum is highly adapted to powered flight, but understanding of its early evolution is hindered by patchy fossil representation and the fact that where preserved, sterna in near-bird dinosaurs and early avialans are substantially different. To overcome this, the authors use ancestral character estimations and find that sternum evolution was episodic and experimental in relation to flight characteristics.},
  archive      = {J_NATEE},
  author       = {Lowi-Merri, Talia M. and Benson, Roger and Hu, Han and O’Connor, Jingmai and Claramunt, Santiago and Evans, David C.},
  doi          = {10.1038/s41559-025-02795-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1705-1718},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Enlargement of sternum traits facilitated the evolution of powered flight in birds},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The role of plasmid copy number and mutation rate in evolutionary outcomes. <em>NATEE</em>, <em>9</em>(9), 1694-1704. (<a href='https://doi.org/10.1038/s41559-025-02792-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multicopy plasmids are widespread in nature and compose a common strategy for spreading beneficial traits across microbes. However, the role of plasmids in supporting the evolution of encoded genes remains underexplored due to challenges in experimentally manipulating key parameters such as plasmid copy number and mutation rate. Here we developed a strategy for controlling copy number in the plasmid-based Saccharomyces cerevisiae continuous evolution system, OrthoRep, and used our resulting capabilities to investigate the evolution of a conditionally essential gene under varying copy number and mutation rate conditions. Our results show that low copy number facilitated the faster enrichment of beneficial alleles whereas high copy number promoted robustness through the maintenance of allelic diversity. High copy number also slowed the removal of deleterious mutations and increased the fraction of non-functional alleles that could hitchhike during evolution. This study highlights the nuanced relationships between plasmid copy number, mutation rate and evolutionary outcomes, providing insights into the adaptive dynamics of genes encoded on multicopy plasmids and nominating OrthoRep as a versatile tool for studying plasmid evolution. In an experimental multicopy plasmid evolution system that allows manipulation of mutation rate and copy number, low copy number promoted enrichment of beneficial alleles whereas high copy number maintained allelic diversity.},
  archive      = {J_NATEE},
  author       = {Pisera, Alexander ‘Olek’ and Liu, Chang C.},
  doi          = {10.1038/s41559-025-02792-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1694-1704},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The role of plasmid copy number and mutation rate in evolutionary outcomes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas. <em>NATEE</em>, <em>9</em>(9), 1685-1693. (<a href='https://doi.org/10.1038/s41559-025-02771-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis is a recently identified cause of Hansen’s disease, and is associated with the more severe and potentially lethal presentations of diffuse lepromatous leprosy and Lucio’s phenomenon. Detection of this infection has been limited to a small number of individuals, leaving much to be learned about its global distribution and transmissibility. Its discovery in wild rodent populations in the United Kingdom and Ireland also raises questions about its zoonotic potential. Here, we raise further awareness of this disease via analyses of two exceptionally well preserved M. lepromatosis genomes obtained from 4,000-year-old human remains of two adult males from the archaeological sites of El Cerrito and La Herradura in Northern Chile. This formed the basis of genomic comparisons between ancient and modern forms of the pathogen. We demonstrate an unexpected long history of M. lepromatosis in the Americas, which contrasts with the more recent Eurasian history of the closely related Mycobacterium leprae. We offer relevant perspectives on its evolution while providing an incentive for further disease monitoring in both humans and other potential reservoir species in the Americas and elsewhere. Recovery of ancient Mycobacterium lepromatosis genomes in 4,000-year-old human remains from Chile demonstrates that this causal agent of Hansen’s disease has a long history in the Americas, predating the colonial period by millennia.},
  archive      = {J_NATEE},
  author       = {Ramirez, Darío A. and Sitter, T. Lesley and Översti, Sanni and Herrera-Soto, María José and Pastor, Nicolás and Fontana-Silva, Oscar Eduardo and Kirkpatrick, Casey L. and Castelleti-Dellepiane, José and Nores, Rodrigo and Bos, Kirsten I.},
  doi          = {10.1038/s41559-025-02771-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1685-1693},
  shortjournal = {Nat. Ecol. Evol},
  title        = {4,000-year-old mycobacterium lepromatosis genomes from chile reveal long establishment of hansen’s disease in the americas},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pervasive but biome-dependent relationship between fragmentation and resilience in forests. <em>NATEE</em>, <em>9</em>(9), 1670-1684. (<a href='https://doi.org/10.1038/s41559-025-02776-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The relationship between landscape fragmentation and vegetation resilience is uncertain. Here we use multiple satellite-based tree cover data and vegetation indices to quantify the apparent effects of fragmentation on global forest resilience and potential mechanisms thereof. We measure fragmentation as edge density, patch density and mean patch area of tree cover patches, and measure resilience as one-lag temporal autocorrelation of vegetation indices. We find a statistically significant (P &lt; 0.05) fragmentation–resilience relationship in about 77% of fragmented forests, but the direction varies across biomes. In tropical and temperate forests, fragmentation is linked to increased local temperature and atmospheric dryness, resulting in a negative fragmentation–resilience relationship. Conversely, in boreal forests, fragmentation is associated with decreased atmospheric dryness and enhanced light resource, thereby increasing forest resilience. Our results reconcile competing hypotheses and highlight the importance of accounting for fragmentation when predicting shifts in ecosystem resilience under disturbances. These findings also suggest the necessity of biome-targeted forest management strategies for climate change mitigation and adaptation. Human-driven forest fragmentation could have major impacts on ecosystem functioning. This global analysis shows that the relationship between fragmentation and vegetation resilience in forests may differ depending on bioclimatic region and local environmental conditions.},
  archive      = {J_NATEE},
  author       = {Su, Yongxian and Zhang, Chaoqun and Cescatti, Alessandro and Yu, Kailiang and Ciais, Philippe and Smith, Taylor and Shang, Jiali and Carnicer, Jofre and Liu, Jane and Chen, Jing Ming and Green, Julia K. and Wu, Jianping and Ponce-Campos, Guillermo E. and Zhang, Yongguang and Zuo, Zhiyan and Liao, Jinbao and Lafortezza, Raffaele and Yan, Kai and Yang, Xueqin and Liu, Liyang and Ren, Jiashun and Yuan, Wenping and Chen, Xiuzhi and Wu, Chaoyang and Zhou, Weiqi},
  doi          = {10.1038/s41559-025-02776-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1670-1684},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pervasive but biome-dependent relationship between fragmentation and resilience in forests},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family. <em>NATEE</em>, <em>9</em>(9), 1656-1669. (<a href='https://doi.org/10.1038/s41559-025-02777-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations are more likely to produce some phenotypes than others, but the causal role of these production propensities in the evolution of phenotypic diversity remains unclear. There are two major challenges: it is difficult to separate the effect of the genotype–phenotype (GP) map from that of natural selection when analysing natural diversity, and most extant phenotypes evolved long ago in species whose GP maps cannot be recovered. Here, using two reconstructed ancestral transcription factors that are closely related but differ in function, we created libraries containing all possible amino acid combinations at historically variable sites in the proteins’ DNA binding interface (the genotypes) and measured their capacity to specifically bind DNA elements containing all possible combinations of nucleotides at historically variable sites (the phenotypes). The ancestral GP maps were strongly anisotropic (the distribution of phenotypes encoded by genotypes is highly non-uniform) and heterogeneous (the phenotypes accessible around each genotype vary greatly among genotypes), but the extent and direction of these properties differed substantially between the maps. In both cases, these properties steered evolution towards the lineage-specific phenotypes that evolved during history. Our findings establish that ancient properties of the GP relationship were causal factors in the evolutionary process that produced present-day patterns of functional conservation and diversity. A combination of ancestral reconstruction and deep mutational scanning to examine the genotype–phenotype maps of steroid receptors shows that the properties of ancestral genotype–phenotype maps determine the lineage-specific evolution of DNA specificity.},
  archive      = {J_NATEE},
  author       = {Herrera-Álvarez, Santiago and Patton, Jaeda E. J. and Thornton, Joseph W.},
  doi          = {10.1038/s41559-025-02777-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1656-1669},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The structure of an ancient genotype–phenotype map shaped the functional evolution of a protein family},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments. <em>NATEE</em>, <em>9</em>(9), 1643-1655. (<a href='https://doi.org/10.1038/s41559-025-02779-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human impacts on nature span vast spatial scales that transcend abiotic gradients and biogeographic barriers, yet estimates of biodiversity loss from land-use change overwhelmingly derive from local-scale studies. Using a field dataset of 971 bird species sampled in forest and cattle pasture across 13 biogeographic regions of Colombia, we quantify biodiversity losses from local to near-national scales. Losses are on average 60% worse at the pan-Colombian scale than in individual regions, with underestimation remaining until six to seven biogeographic regions are sampled. Regional losses greatly exceed local losses when beta-diversity is high due to reduced species turnover in pasture across geographic space and elevation. Extrapolation from local-scale studies causes major underestimation of biodiversity loss, emphasizing the need to incorporate spatial structure into measures of change. Analysis of data on 971 bird species in natural habitat and cattle pasture in Colombia finds that near-national-scale losses of bird diversity greatly exceed losses recorded at the local scale, suggesting that extrapolations from local studies will severely underestimate biodiversity losses.},
  archive      = {J_NATEE},
  author       = {Socolar, Jacob B. and Mills, Simon C. and Gilroy, James J. and Martínez-Revelo, Diego E. and Medina-Uribe, Claudia A. and Parra-Sanchez, Edicson and Ramirez-Gutierrez, Marcela and Sand Sæbø, Jørgen and Meneses, Henry S. and Pérez, Giovanny and Barlow, Jos and Ochoa Quintero, Jose M. and Freckleton, Robert P. and Haugaasen, Torbjørn and Edwards, David P.},
  doi          = {10.1038/s41559-025-02779-4},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1643-1655},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Tropical biodiversity loss from land-use change is severely underestimated by local-scale assessments},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development. <em>NATEE</em>, <em>9</em>(9), 1626-1642. (<a href='https://doi.org/10.1038/s41559-025-02780-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bats are the only mammals capable of self-powered flight, an evolutionary innovation based on the transformation of forelimbs into wings. The bat wing is characterized by an extreme elongation of the second to fifth digits with a wing membrane called the chiropatagium connecting them. Here we investigated the developmental and cellular origin of this structure by comparing bat and mouse limbs using omics tools and single-cell analyses. Despite the substantial morphological differences between the species, we observed an overall conservation of cell populations and gene expression patterns including interdigital apoptosis. Single-cell analyses of micro-dissected embryonic chiropatagium identified a specific fibroblast population, independent of apoptosis-associated interdigital cells, as the origin of this tissue. These distal cells express a conserved gene programme including the transcription factors MEIS2 and TBX3, which are commonly known to specify and pattern the early proximal limb. Transgenic ectopic expression of MEIS2 and TBX3 in mouse distal limb cells resulted in the activation of genes expressed during wing development and phenotypic changes related to wing morphology, such as the fusion of digits. Our results elucidate fundamental molecular mechanisms of bat wing development and illustrate how drastic morphological changes can be achieved through repurposing of existing developmental programmes during evolution. Single-cell comparison of developing bat and mouse limbs reveals conservation of cell populations and gene expression patterns, and suggests repurposing of genes involved in proximal limb development for wing evolution.},
  archive      = {J_NATEE},
  author       = {Schindler, Magdalena and Feregrino, Christian and Aldrovandi, Silvia and Lo, Bai-Wei and Monaco, Anna A. and Ringel, Alessa R. and Morales, Ariadna E. and Zehnder, Tobias and Behncke, Rose Yinghan and Glaser, Juliane and Barclay, Alexander and Andrey, Guillaume and Kragesteen, Bjørt K. and Hägerling, René and Haas, Stefan A. and Vingron, Martin and Ulitsky, Igor and Marti-Renom, Marc A. and Hechavarria, Julio and Fasel, Nicolas and Hiller, Michael and Lupiáñez, Darío G. and Mundlos, Stefan and Real, Francisca M.},
  doi          = {10.1038/s41559-025-02780-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1626-1642},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Comparative single-cell analyses reveal evolutionary repurposing of a conserved gene programme in bat wing development},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep learning reveals the complex genetic architecture of male guppy colouration. <em>NATEE</em>, <em>9</em>(9), 1614-1625. (<a href='https://doi.org/10.1038/s41559-025-02781-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The extraordinary variation in male guppy (Poecilia reticulata) colouration is a powerful model for studying the interplay of natural and sexual selection. However, the complexity of this variation has hampered the high-resolution characterization and determination of the genetic architecture underlying male guppy colour and clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male colour variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a genome-wide association study of colour traits. Our phenotypic and genomic results converge to show that colour patterning in guppies is a combination of many heritable features, each with a largely independent genetic architecture spanning the entire genome. Autosomally inherited ornaments are polygenic, with significant contributions from loci involved in neural crest cell migration. Unusually, the results of our genome-wide association study suggest that gene duplicates from the autosomes to the Y chromosome are responsible for much of the sex-linked variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait. Guppy colouration is a very diverse trait under natural and sexual selection. Using a combination of high-resolution phenotyping and genomic analysis, the authors reveal the genetic architecture of this complex trait.},
  archive      = {J_NATEE},
  author       = {van der Bijl, Wouter and Shu, Jacelyn J. and Goberdhan, Versara S. and Sherin, Linley M. and Jia, Changfu and Cortazar-Chinarro, Maria and Corral-Lopez, Alberto and Mank, Judith E.},
  doi          = {10.1038/s41559-025-02781-w},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1614-1625},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deep learning reveals the complex genetic architecture of male guppy colouration},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood. <em>NATEE</em>, <em>9</em>(9), 1599-1613. (<a href='https://doi.org/10.1038/s41559-025-02785-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deadwood is a major carbon source in forests, and yet the fate of this carbon remains a gap in our understanding of global carbon cycling. Lignin, the most recalcitrant biopolymer in wood, is mainly decayed through extracellular enzymatic and chemical processes initiated by white-rot fungi. However, the intracellular conversion of lignin decay products has been overlooked in the fungal kingdom. Here we integrate comparative genomic and phylogenetic analyses to understand the distribution and evolution of enzymes responsible for modifying lignin-related aromatic compounds—such as decarboxylases, hydroxylases, dioxygenases and other downstream ring-cleavage enzymes—that funnel carbon to central metabolism across the bacterial and the fungal kingdoms. We demonstrate that specific fungal lineages conserve these enzyme families, and that the abilities to enzymatically depolymerize lignin and catabolize lignin-related aromatic compounds are not necessarily coupled. Our analyses also reveal an expanded substrate specificity of aromatic ring-cleavage enzymes during fungal evolution, as well as a clade of extracellular enzymes among them, broadening the spatial range of these biochemical capabilities. Together, our results highlight a large diversity of fungal enzymes and hosts that warrant further investigation for inclusion into carbon cycling models and biotechnological applications for the conversion of aromatic compounds. This study uses comparative genomics and phylogenetics to analyse the distribution and evolution of key enzymes involved in the catabolism of lignin-related aromatic compounds in the bacterial and fungal kingdoms.},
  archive      = {J_NATEE},
  author       = {Kijpornyongpan, Teeratas and Kuatsjah, Eugene and Komp, Evan and Evans, James E. and Ruiz-Dueñas, Francisco Javier and Salvachúa, Davinia},
  doi          = {10.1038/s41559-025-02785-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1599-1613},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cross-kingdom comparative genomics reveal the metabolic potential of fungi for lignin turnover in deadwood},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife. <em>NATEE</em>, <em>9</em>(9), 1585-1598. (<a href='https://doi.org/10.1038/s41559-025-02786-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across the world, human (anthropophonic) sounds add to sounds of biological (biophonic) and geophysical (geophonic) origin, with human contributions including both speech and technophony (sounds of technological devices). To characterize society’s contribution to the global soundscapes, we used passive acoustic recorders at 139 sites across 6 continents, sampling both urban green spaces and nearby pristine sites continuously for 3 years in a paired design. Recordings were characterized by bird species richness and by 14 complementary acoustic indices. By relating each index to seasonal, diurnal, climatic and anthropogenic factors, we show here that latitude, time of day and day of year each predict a substantial proportion of variation in key metrics of biophony—whereas anthropophony (speech and traffic) show less predictable patterns. Compared to pristine sites, the soundscape of urban green spaces is more dominated by technophony and less diverse in terms of acoustic energy across frequencies and time steps, with less instances of quiet. We conclude that the global soundscape is formed from a highly predictable rhythm in biophony, with added noise from geophony and anthropophony. At urban sites, animals experience an increasingly noisy background of sound, which poses challenges to efficient communication. Analysis of soundscape data from 139 globally distributed sites reveals that sounds of biological origin exhibit predictable rhythms depending on location and season, whereas sounds of anthropogenic origin are less predictable. Comparisons between paired urban–rural sites show that urban green spaces are noisier and dominated by sounds of technological origin.},
  archive      = {J_NATEE},
  author       = {Somervuo, Panu and Roslin, Tomas and Fisher, Brian L. and Hardwick, Bess and Kerdraon, Deirdre and Raharinjanahary, Dimby and Rajoelison, Eric Tsiriniaina and Lauha, Patrik and Griem, Lukas and Lehikoinen, Petteri and Niittynen, Pekka and Piirainen, Esko and Lumme, Markus and Riihikoski, Ville-Matti and Acevedo-Charry, Orlando and Adalsteinsson, Solny A. and Ahmad, Maaz and Alcobia, Sandra and Aldará, Jón and Andrew, Nigel R. and Anslan, Sten and Antonelli, Alexandre and Arena, Julieta Soledad and Arroyo Almeida, Santiago and Aster, Ines and Autto, Hannu and Aviles Gamboa, Anahi and Baixeras, Joaquín and Baldauf, Mario and Balestrieri, Rosario and Banelyte, Gaia Giedre and Barrett, Adrian and Beja, Pedro and Berg, Thomas Olof and Bergerot, Benjamin and Biro, Elizabeth G. and Blendinger, Pedro G. and Bollache, Loïc and Bou Dagher Kharrat, Magda and Boyer, Stephane and Bridell, Erika and Brotherson, Martyn and Brown, Leslie Robert and Buckley, Hannah L. and Buscardo, Erika and Buthelezi, Nokuphila and Cagnolo, Luciano and Calvente, Alice and Capobianco, Giovanni and Carreón-Palau, Laura and Carriere, Suzanne and Case, Bradley S. and Chang, Jenyu and Chaparro, Juan Matías and Chen, Chi-Ling and Chicoine, Christine and Christensson, Madeleine and Collado Rosique, Francisco and Colom Montero, William and da Fonseca, Ricardo do Sacramento and Da Silva, Luís P. and Dal Molin, Anamaria and Dallas, Tad and de Francesco, Maria Carla and Del Ángel-Rodríguez, Jorge Arturo and Díaz-Delgado, Ricardo and Dirnböck, Thomas and Djukic, Ika and Dladla, Philile and Domínguez Masciale, Jeremías and Dorigo, Thiago and Douwes, Errol and Ekrem, Torbjørn and Enderskog, Helena and Erefur, Charlotta and Fahad, Muhammad and Falahati-Anbaran, Mohsen and Farrell, Arielle and Ferland, Gabriel and Ferrari, Emanuele and Figueiredo, Axa and Forero, Fernando and Freiberga, Inga and Frosch-Radivo, Andrea and Ganchozo Intriago, Luis Alberto and Garzoli, Laura and Giacomotti, Paola and Gianuca, Andros T. and Gilg, Olivier and Gilg, Vladimir and Gísladóttir, Fanney and Glowacki, Ryan and Gottsberger, Brigitte and Gregoire, Jocelyn and Groner, Elli and Guedes, Patrícia and Guile, Aimee Michelle and Haase, Peter and Hadi, Fazal and Haidegger, Magdalena and Hansen, Leivur Janus and Hansen, Lars Holst and Harrop, Reid and Havnås, Harald and Herrera Báez, David and Ho, Chris C. Y. and Hohenbühel, Denise and Houska Tahadlova, Marketa and Hänninen, Jari and Höglund, Linda and Í Haraldsstovu, Kolbrún and Imbeau, Elise and Inkinen, Jasmin and Ishihara, Masae Iwamoto and Jackson, Abigail C. and Jansson, Gunnar and Jha, Rohit and Kager, Gerald and Kahale, Rhea and Kalttopää, Oula and Karai, Elizabeth Wanjiru and Karlsson, Dave and Kaus-Thiel, Andrea and Khan, Asghar and Khan, Qaisar and Kimoto, Keishi and Kipngetich, Shadrack Chumo and Klante, Clemens and Klemedtsson, Leif and Klinth, Mårten and Koskinen, Janne and Kotakorpi, Matti and Kreiling, Agnes-Katharina and Krisai-Greilhuber, Irmgard and Kristensen, Erik and König, Sebastian and Langenheder, Silke and Laurila, Kalevi and Le Gouar, Pascaline and Lecomte, Nicolas and Lecomte, Erin and Leitman, Paula Moraes and León-Cortés, Jorge L. and Li, Daijiang and Loehr, John and Lopez-Vaamonde, Carlos and Makari, Mehsen and Mangini, Gabriela Giselle and Maroschek, Michael and Mata, Vanessa A. and Matsuoka, Shunsuke and Mazzafera, Thais and McDonald, Paul G. and Meinert, Laura and Meléndez-González, Mayra and Mendoza-Henao, Angela M. and Moreau, Sebastien and Moreau, Jérôme and Mosbacher, Jesper and Moyer, Esteban and Mrazova, Anna and Mteshane, Samantha and Mungai, Nancy Wangari and Muñoz Herraiz, Gema and Murillo-Vázquez, Andrea and Musazzi, Simona and Mutanen, Marko and Müller, Jörg and Navarro Canales, Rebeca and Ndlovu, Monica and Nicolai, Annegret and Niessner, Armin and Nordén, Jenni and Nowak, Paweł and O’Connell, Erin and Orru, Arianna and Pagnon, Thomas and Pantoja-Diaz, Yurani Nayive and Pentinsaari, Mikko and Pilloni, Sebastian and Pinder, Adrian and Pinheiro, Thiago A. and Põlme, Sergei and Powell, Luke L. and Pröll, Gisela and Pulido-Santacruz, Paola and Queralt, Enrique and Quilantang, Mark Tristan and Quinlan, Kirsty and Ramirez, Ricardo and Rankinen, Juha and Rasino, Micaela Del Valle and Rebelo, Rui and Remmers, Wolfram and Retz, Franziska and Reyes, Evelin and Rivas Torres, Gonzalo and Rogers, Hanna M. K. and Rosário, Inês T. and Rosário Da Rosàrio da Costa, Sidney and Rütting, Tobias and Sahlstén, Johannes and Saliba, Carole and Salmirinne, Teppo and Sam, Katerina and Santos, Douglas and Santos-Reis, Margarida and Sawan, Michel and Schattanek-Wiesmair, Benjamin and Schiestl-Aalto, Pauliina and Schmidt, Niels Martin and Seibold, Sebastian and Seidl, Rupert and Seifert, Linda and Sithole, Malibongwe and Sivault, Elise and Smart, Jessica and Smerczyński, Ireneusz and Soda, Ayaka and Sousa-Lima, Renata S. and Stanisci, Angela and Stanley, Margaret C. and Steenkamp, Daleen and Stengel, Elisa and Stoll, Stefan and Strauss, Willem Maartin and Stur, Elisabeth and Sujala, Maija and Sundell, Janne and Svavarsdóttir, Jónína and Tedersoo, Leho and Tepsa, Saana and Tikochinsky, Maor Tiko and Tuominen, Esa-Pekka and Tweraser, Stefanie and Ulloa Espinosa, Catalina and Uusitalo, Joni and Vallinmäki, Mikko and Vannier, Fabrice and Varela, Abigail and Vatka, Emma and Veikkolainen, Silja and Vernes, Karl and Watts, Phillip C. and Weslien, Per and Wirth, Ciara and Wisniewski, Jana Helga and Young, Amanda B. and Övergaard, Robyn and Ovaskainen, Otso},
  doi          = {10.1038/s41559-025-02786-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1585-1598},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Human contributions to global soundscapes are less predictable than the acoustic rhythms of wildlife},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency. <em>NATEE</em>, <em>9</em>(9), 1571-1584. (<a href='https://doi.org/10.1038/s41559-025-02761-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerous leaf-level experiments suggest that plant intrinsic water-use efficiency (iWUE) increases under elevated CO2 because of reduced stomatal conductance and enhanced photosynthesis. However, it remains elusive whether this response can be extrapolated to the ecosystem scale, because confounding factors and compensating feedbacks are often involved in ecosystem iWUE variations. Here we develop a machine learning-based framework to disentangle the ecosystem-scale CO2 effects on iWUE and its two components, canopy conductance (Gc) and gross primary productivity (GPP), based on global networks of long-term eddy covariance observations. Our results show widespread CO2-induced enhancement of iWUE across diverse ecosystems, driven predominantly by Gc reduction rather than GPP stimulation. Moreover, three divergent response types are identified across the studied ecosystems, based on the strength and significance of CO2-driven Gc reduction and GPP enhancement, indicating spatially non-uniform responses to rising CO2. Nutrient supply, water availability and biome types are found to be critical factors regulating this spatial heterogeneity. Overall, our study provides observational insights into ecosystem-scale CO2 fertilization effects. Such understandings are essential to inform terrestrial biosphere models for better projections of carbon and water cycles given the intensified changing climate in a CO2-rich future. A machine learning approach using long-term observations of eddy covariance finds that the increase in plant intrinsic water-use efficiency under higher CO2 levels, across diverse ecosystems, is driven primarily by reduction in canopy conductance, rather than by stimulation of gross primary productivity.},
  archive      = {J_NATEE},
  author       = {Zhan, Weiwei and Lian, Xu and Liu, Jiangong and Han, Jisu and Huang, Yu and Yang, Hao and Zhan, Chunhui and Winkler, Alexander J. and Gentine, Pierre},
  doi          = {10.1038/s41559-025-02761-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1571-1584},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reduced water loss rather than increased photosynthesis controls CO2-enhanced water-use efficiency},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change. <em>NATEE</em>, <em>9</em>(9), 1562-1570. (<a href='https://doi.org/10.1038/s41559-025-02788-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plankton models form the core of marine ecosystem simulators, with uses from regional resource and ecosystem management to climate change projections. In this Perspective, we suggest that stronger alignment of models with empirical knowledge about plankton physiology, diversity and trophic roles will improve model utility and the reliability of their outputs regarding biodiversity, ecophysiology, trophic dynamics and biogeochemistry. We recommend key steps to resolve the disconnect between empirical research and simulation models accounting for well-established plankton processes with an aim to increase the utility of such models for applied uses. A central challenge is characterizing the complexity of plankton diversity and activity in ways that are amenable to model incorporation. We argue that experts in empirical science are best placed to advise the development of next-generation models to address these challenges, and we propose a series of actions to achieve that engagement, including involvement of these experts in the design and exploitation of plankton digital twins. Plankton models have crucial applications in ecosystem management and climate change projections. This Perspective suggests that stronger alignment of plankton models with empirical knowledge is needed and recommends steps to close the gap between empirical research and modelling.},
  archive      = {J_NATEE},
  author       = {Flynn, Kevin J. and Atkinson, Angus and Beardall, John and Berges, John A. and Boersma, Maarten and Brunet, Christophe and Calbet, Albert and Caron, Dave A. and Dam, Hans G. and Glibert, Patricia M. and Hansen, Per Juel and Jin, Peng and Lønborg, Christian and Mayor, Daniel J. and Menden-Deuer, Susanne and Mock, Thomas and Mulholland, Margaret R. and Needham, David M. and Polimene, Luca and Poulton, Alex J. and Robinson, Carol and Rokitta, Sebastian D. and Rost, Björn and Saiz, Enric and Scanlan, David J. and Schmidt, Katrin and Sherr, Evelyn and Stoecker, Diane K. and Svensen, Camilla and Thiele, Stefan and Thingstad, Tron F. and Våge, Selina},
  doi          = {10.1038/s41559-025-02788-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1562-1570},
  shortjournal = {Nat. Ecol. Evol},
  title        = {More realistic plankton simulation models will improve projections of ocean ecosystem responses to global change},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bird sternum evolution facilitated the origins of powered flight. <em>NATEE</em>, <em>9</em>(9), 1560-1561. (<a href='https://doi.org/10.1038/s41559-025-02817-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An in-depth analysis of the breastbone (sternum) of bird ancestors reveals its importance in the origin and evolution of flight in dinosaurs as they evolved into birds.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02817-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1560-1561},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Bird sternum evolution facilitated the origins of powered flight},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus. <em>NATEE</em>, <em>9</em>(9), 1558-1559. (<a href='https://doi.org/10.1038/s41559-025-02799-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geochemical chronologies in surface increments and exposed cross-sections of naturally fractured hominin enamel from the South African sites of Swartkrans and Kromdraai indicate that Paranthropus robustus exploited both forest and grassland habitats, and that individuals did not move on the landscape in a manner analogous to extant African apes.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02799-0},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1558-1559},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Isotope analysis of fossil teeth provides insight into the habitat and life histories of the early hominin paranthropus robustus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolving a bat wing by reusing an existing gene program. <em>NATEE</em>, <em>9</em>(9), 1556-1557. (<a href='https://doi.org/10.1038/s41559-025-02782-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interspecies comparative analyses of single-cell transcriptomic data reveal that a proximal limb gene program is reactivated during bat wing formation. Our results illustrate how evolutionary innovations can arise from the reutilization of an existing gene set, activated through species-specific regulatory repertoires.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02782-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1556-1557},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolving a bat wing by reusing an existing gene program},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The next generation of colour pattern genomics. <em>NATEE</em>, <em>9</em>(9), 1554-1555. (<a href='https://doi.org/10.1038/s41559-025-02816-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Convolutional neural networks and genetic association analysis decode the evolution of colour pattern diversity and its underlying complex genetic architecture in the Trinidadian guppy.},
  archive      = {J_NATEE},
  author       = {Karagic, Nidal and Kratochwil, Claudius F.},
  doi          = {10.1038/s41559-025-02816-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1554-1555},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The next generation of colour pattern genomics},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sea star wasting disease mystery finally solved. <em>NATEE</em>, <em>9</em>(9), 1552-1553. (<a href='https://doi.org/10.1038/s41559-025-02789-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A decade after a marine epidemic killed off sea stars and triggered ecosystem-wide effects along the Pacific Coast of North America, researchers have identified the bacterial pathogen that is responsible for sea star wasting disease.},
  archive      = {J_NATEE},
  author       = {Lafferty, Kevin D.},
  doi          = {10.1038/s41559-025-02789-2},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1552-1553},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sea star wasting disease mystery finally solved},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leprosy in the americas before columbus. <em>NATEE</em>, <em>9</em>(9), 1550-1551. (<a href='https://doi.org/10.1038/s41559-025-02790-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium lepromatosis genomes associated with 4,000-year-old human skeletons in Chile establish an American origin for this causal agent of leprosy (also known as Hansen’s disease), and point to different evolutionary trajectories and transmission pathways for M. lepromatosis and its sister pathogen Mycobacterium leprae.},
  archive      = {J_NATEE},
  author       = {Roberts, Charlotte A.},
  doi          = {10.1038/s41559-025-02790-9},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1550-1551},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Leprosy in the americas before columbus},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Water-use efficiency driven by water savings. <em>NATEE</em>, <em>9</em>(9), 1548-1549. (<a href='https://doi.org/10.1038/s41559-025-02793-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of land–atmosphere water and CO2 fluxes suggests that reduced water use by vegetation, rather than increased carbon uptake, is the driving factor behind the well-documented increase in vegetation water-use efficiency in response to rising atmospheric CO2.},
  archive      = {J_NATEE},
  author       = {Knauer, Jürgen},
  doi          = {10.1038/s41559-025-02793-6},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1548-1549},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Water-use efficiency driven by water savings},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Independent origins of long snouts. <em>NATEE</em>, <em>9</em>(9), 1547. (<a href='https://doi.org/10.1038/s41559-025-02856-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-025-02856-8},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1547},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Independent origins of long snouts},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Seed dispersers wanted. <em>NATEE</em>, <em>9</em>(9), 1546. (<a href='https://doi.org/10.1038/s41559-025-02857-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02857-7},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1546},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Seed dispersers wanted},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fish without borders. <em>NATEE</em>, <em>9</em>(9), 1545. (<a href='https://doi.org/10.1038/s41559-025-02852-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02852-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1545},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fish without borders},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mines aren’t on the maps. <em>NATEE</em>, <em>9</em>(9), 1544. (<a href='https://doi.org/10.1038/s41559-025-02833-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-025-02833-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1544},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mines aren’t on the maps},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Museums in the age of extinction. <em>NATEE</em>, <em>9</em>(9), 1542-1543. (<a href='https://doi.org/10.1038/s41559-025-02772-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Browning, Claire},
  doi          = {10.1038/s41559-025-02772-x},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1542-1543},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Museums in the age of extinction},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding environmental impacts of plastic requires a palaeontological lens. <em>NATEE</em>, <em>9</em>(9), 1539-1541. (<a href='https://doi.org/10.1038/s41559-025-02813-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Growing evidence suggests that timescales for plastic degradation have been vastly underestimated. The fossil record of plastic-like biopolymers might provide a perspective on plastic fossilization in deep time.},
  archive      = {J_NATEE},
  author       = {Gabbott, Sarah and Boom, Arnoud and Zalasiewicz, Jan},
  doi          = {10.1038/s41559-025-02813-5},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1539-1541},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Understanding environmental impacts of plastic requires a palaeontological lens},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Restoration outcomes are compromised by a lack of diverse native seed supply. <em>NATEE</em>, <em>9</em>(9), 1537-1538. (<a href='https://doi.org/10.1038/s41559-025-02825-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Changes in seed banking practices and business incentives are needed to ensure restoration practitioners have access to plentiful and reliable supplies of diverse native seeds.},
  archive      = {J_NATEE},
  author       = {Gallagher, Rachael V.},
  doi          = {10.1038/s41559-025-02825-1},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1537-1538},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Restoration outcomes are compromised by a lack of diverse native seed supply},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Increase the regulation of biodiversity effects from private equity markets. <em>NATEE</em>, <em>9</em>(9), 1535-1536. (<a href='https://doi.org/10.1038/s41559-025-02828-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Carter, Harrison and Miles, Nell and Hawkins, Isobel and Hulett, Toscanie and zu Ermgassen, Sophus O. S. E.},
  doi          = {10.1038/s41559-025-02828-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1535-1536},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increase the regulation of biodiversity effects from private equity markets},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The amazon’s new oil frontier is a biodiversity bomb in the making. <em>NATEE</em>, <em>9</em>(9), 1532-1534. (<a href='https://doi.org/10.1038/s41559-025-02815-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Paletta, Julia and Callegari, Camila and Oliveira Fiorini, Ana Carolina and Dias, Filipe Gomes and Freitas Junior, Adirson and de Barros, Pedro Henrique Batista},
  doi          = {10.1038/s41559-025-02815-3},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1532-1534},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The amazon’s new oil frontier is a biodiversity bomb in the making},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sound science. <em>NATEE</em>, <em>9</em>(9), 1531. (<a href='https://doi.org/10.1038/s41559-025-02860-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Passive acoustic monitoring is opening up new opportunities for biodiversity monitoring, but there is still work to do to ensure that it represents a reliable method for biodiversity reporting.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02860-y},
  journal      = {Nature Ecology & Evolution},
  month        = {9},
  number       = {9},
  pages        = {1531},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sound science},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natmi">NATMI - 16</h2>
<ul>
<li><details>
<summary>
(2025). Electron-density-informed effective and reliable de novo molecular design and optimization with ED2Mol. <em>NATMI</em>, <em>7</em>(8), 1355-1368. (<a href='https://doi.org/10.1038/s42256-025-01095-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative drug design opens avenues for discovering novel compounds within the vast chemical space rather than conventional screening against limited libraries. However, the practical utility of the generated molecules is frequently constrained, as many designs prioritize a narrow range of pharmacological properties and neglect physical reliability, which hinders the success rate of subsequent wet-laboratory evaluations. Here, to address this, we propose ED2Mol, a deep learning-based approach that leverages fundamental electron density information to improve de novo molecular generation and optimization. The extensive evaluations across multiple benchmarks demonstrate that ED2Mol surpasses existing methods in terms of the generation success rate and &gt;97% physical reliability. It also facilitates automated hit optimization that is not fully implemented by other methods using fragment-based strategies. Furthermore, ED2Mol exhibits generalizability to more challenging, unseen allosteric pocket benchmarks, attaining consistent performance. More importantly, ED2Mol has been applied to various real-world essential targets, successfully identifying wet-laboratory-validated bioactive compounds, ranging from FGFR3 orthosteric inhibitors to CDC42 allosteric inhibitors, GCK and GPRC5A allosteric activators. The directly generated binding modes of these compounds are close to predictions through molecular docking and further validated via the X-ray co-crystal structure. All these results highlight ED2Mol’s potential as a useful tool in drug design with enhanced effectiveness, physical reliability and practical applicability. A deep generative model is developed for de novo molecular design and optimization by leveraging electron density. Wet-laboratory assays validated its reliability to generate diverse bioactive molecules—orthosteric and allosteric, inhibitors and activators.},
  archive      = {J_NATMI},
  author       = {Li, Mingyu and Song, Kun and He, Jixiao and Zhao, Mingzhu and You, Gengshu and Zhong, Jie and Zhao, Mengxi and Li, Arong and Chen, Yu and Li, Guobin and Kong, Ying and Wei, Jiacheng and Wang, Zhaofu and Zhou, Jiamin and Yang, Hongbing and Ma, Shichao and Zhang, Hailong and Mélita, Irakoze Loïca and Lin, Weidong and Lu, Yuhang and Yu, Zhengtian and Lu, Xun and Zhao, Yujun and Zhang, Jian},
  doi          = {10.1038/s42256-025-01095-7},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1355-1368},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Electron-density-informed effective and reliable de novo molecular design and optimization with ED2Mol},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Kolmogorov–Arnold graph neural networks for molecular property prediction. <em>NATMI</em>, <em>7</em>(8), 1346-1354. (<a href='https://doi.org/10.1038/s42256-025-01087-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Graph neural networks (GNNs) have shown remarkable success in molecular property prediction as key models in geometric deep learning. Meanwhile, Kolmogorov–Arnold networks (KANs) have emerged as powerful alternatives to multi-layer perceptrons, offering improved expressivity, parameter efficiency and interpretability. To combine the strengths of both frameworks, we propose Kolmogorov–Arnold GNNs (KA-GNNs), which integrate KAN modules into the three fundamental components of GNNs: node embedding, message passing and readout. We further introduce Fourier-series-based univariate functions within KAN to enhance function approximation and provide theoretical analysis to support their expressiveness. Two architectural variants, KA-graph convolutional networks and KA-augmented graph attention networks, are developed and evaluated across seven molecular benchmarks. Experimental results show that KA-GNNs consistently outperform conventional GNNs in terms of both prediction accuracy and computational efficiency. Moreover, our models exhibit improved interpretability by highlighting chemically meaningful substructures. These findings demonstrate that KA-GNNs offer a powerful and generalizable framework for molecular data modelling, drug discovery and beyond. Li et al. developed KA-GNNs, graph neural network architectures enhanced by Kolmogorov–Arnold networks, which improve accuracy and interpretability in molecular property prediction and extend geometric deep learning to scientific domains.},
  archive      = {J_NATMI},
  author       = {Li, Longlong and Zhang, Yipeng and Wang, Guanghui and Xia, Kelin},
  doi          = {10.1038/s42256-025-01087-7},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1346-1354},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Kolmogorov–Arnold graph neural networks for molecular property prediction},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires. <em>NATMI</em>, <em>7</em>(8), 1331-1345. (<a href='https://doi.org/10.1038/s42256-025-01085-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cell selection is a vital process in which precursor T cells mature into functional cells. Accurately modelling and quantifying T cell selection utilizing high-throughput T cell receptor (TCR) sequencing data presents an important computational challenge in immunology. Statistical modelling of TCR repertoires allows the assessment of selection force through the selection factor that bridges the pre- and post-selection distributions. Current tools derive the principles underlying this selection factor through weakly supervised learning, limiting the effective use of available data. To overcome this shortcoming, we introduce TCRsep, a deep learning framework designed to directly learn the selection factor in a supervised training context. The performance and advantage of TCRsep were extensively validated across various scenarios using both simulated and real datasets. By applying TCRsep to over 1,500 repertoire samples, we elucidate the correlation between selection and repertoire diversities in aging, explore the stability and individuality of selection over short time frames, investigate the role of selection in defining TCR sharing profiles and demonstrate its efficiency in identifying candidate-disease-associated TCRs based on their sharing profiles. In particular, these identified TCRs were further utilized for diagnosing cytomegalovirus infection, achieving high predictive accuracy. In conclusion, TCRsep substantially improves the selection factor prediction and serves as a valuable discovery tool for clinical applications. TCRsep, a deep learning model for predicting selection factors that quantifies the T cell selection process, is introduced. Also, various benchmarks are designed to evaluate the selection models, demonstrating that TCRsep outperforms state-of-the-art models.},
  archive      = {J_NATMI},
  author       = {Jiang, Yuepeng and Zhang, Pingping and Huo, Miaozhe and Li, Shuai Cheng},
  doi          = {10.1038/s42256-025-01085-9},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1331-1345},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards generalizable and interpretable three-dimensional tracking with inverse neural rendering. <em>NATMI</em>, <em>7</em>(8), 1322-1330. (<a href='https://doi.org/10.1038/s42256-025-01083-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Today, the most successful methods for image-understanding tasks rely on feed-forward neural networks. Although this approach offers empirical accuracy, efficiency and task adaptation through fine-tuning, it also comes with fundamental disadvantages. Existing networks often struggle to generalize across different datasets, even on the same task. By design, these networks ultimately reason about high-dimensional scene features, which are challenging to analyse. This is true especially when attempting to predict three-dimensional (3D) information based on two-dimensional images. We propose to recast vision problems with RGB inputs as an inverse rendering problem by optimizing through a differentiable rendering pipeline over the latent space of pretrained 3D object representations and retrieving latents that best represent object instances in a given input image. Specifically, we solve the task of 3D multi-object tracking by optimizing an image loss over generative latent spaces that inherently disentangle shape and appearance properties. Not only do we investigate an alternative take on tracking but our method also enables us to examine the generated objects, reason about failure situations and resolve ambiguous cases. We validate the generalization and scaling capabilities of our method by learning the generative prior exclusively from synthetic data and assessing camera-based 3D tracking on two large-scale autonomous robot datasets. Both datasets are completely unseen to our method and do not require fine-tuning. Ost et al. present a method that recasts vision problems with RGB inputs as an inverse rendering problem, optimizing over the latent variables of pretrained three-dimensional object models through a differentiable rendering pipeline.},
  archive      = {J_NATMI},
  author       = {Ost, Julian and Banerjee, Tanushree and Bijelic, Mario and Heide, Felix},
  doi          = {10.1038/s42256-025-01083-x},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1322-1330},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards generalizable and interpretable three-dimensional tracking with inverse neural rendering},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Protein–peptide docking with a rational and accurate diffusion generative model. <em>NATMI</em>, <em>7</em>(8), 1308-1321. (<a href='https://doi.org/10.1038/s42256-025-01077-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Therapeutic peptides represent the forefront of drug discovery, offering potent and safe alternatives to traditional small molecules. However, their weak and context-dependent nature complicates the efficient virtual screening and structural characterization of protein–peptide patterns. Here we introduce RAPiDock, a diffusion generative model designed for rational, accurate and rapid protein–peptide docking at an all-atomic level. RAPiDock efficiently reduces the sampling space by incorporating physical constraints and uses a bi-scale graph to effectively capture multidimensional structural information while balancing efficiency. In addition, the model uses a Clebsch–Gordan tensor product-based architecture to ensure physical symmetry. RAPiDock outperforms existing tools in prediction of protein–peptide-binding patterns, achieving a 93.7% success rate at top-25 predictions (13.4% higher than AlphaFold2-Multimer), with an execution speed of 0.35 seconds per complex (~270 times faster than AlphaFold2-Multimer). Extensive experiments demonstrate RAPiDock’s remarkable ability to handle 92 types of residue including posttranslational modifications, accurately predict subtle docking patterns, successfully identify multiple potential peptide-binding sites in global docking and serve as a powerful tool for high-throughput virtual screening with structural precision. All these push the boundaries of efficient protein–peptide docking in multiple real-application scenarios. Zhao et al. present RAPiDock, an all-atom diffusion model that predicts peptide–protein binding patterns across 92 amino acid types, enabling high-throughput virtual screening for advancing therapeutic peptide design.},
  archive      = {J_NATMI},
  author       = {Zhao, Huifeng and Zhang, Odin and Jiang, Dejun and Wu, Zhenxing and Du, Hongyan and Wang, Xiaorui and Zhao, Yihao and Huang, Yuansheng and Ge, Jingxuan and Hou, Tingjun and Kang, Yu},
  doi          = {10.1038/s42256-025-01077-9},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1308-1321},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Protein–peptide docking with a rational and accurate diffusion generative model},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Histopathology-based protein multiplex generation using deep learning. <em>NATMI</em>, <em>7</em>(8), 1292-1307. (<a href='https://doi.org/10.1038/s42256-025-01074-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed protein imaging offers valuable insights into interactions between tumours and their surrounding tumour microenvironment, but its widespread use is limited by cost, time and tissue availability. Here we present HistoPlexer, a deep learning framework that generates spatially resolved protein multiplexes directly from standard haematoxylin and eosin (H&amp;E) histopathology images. HistoPlexer jointly predicts multiple tumour and immune markers using a conditional generative adversarial architecture with custom loss functions designed to ensure pixel- and embedding-level similarity while mitigating slice-to-slice variations. A comprehensive evaluation of metastatic melanoma samples demonstrates that HistoPlexer-generated protein maps closely resemble real maps, as validated by expert assessment. They preserve crucial biological relationships by capturing spatial co-localization patterns among proteins. The spatial distribution of immune infiltration from HistoPlexer-generated protein multiplex enables stratification of tumours into immune subtypes. In an independent cohort, integration of HistoPlexer-derived features into predictive models enhances performance in survival prediction and immune subtype classification compared to models using H&amp;E features alone. To assess broader applicability, we benchmarked HistoPlexer on publicly available pixel-aligned datasets from different cancer types. In all settings, HistoPlexer consistently outperformed baseline methods, demonstrating robustness across diverse tissue types and imaging conditions. By enabling whole-slide protein multiplex generation from routine H&amp;E images, HistoPlexer offers a cost- and time-efficient approach to tumour microenvironment characterization with strong potential to advance precision oncology. HistoPlexer, a deep learning model, generates multiplexed protein expression maps from H&amp;E images, capturing tumour–immune cell interactions. It outperforms baselines, enhances immune subtyping and survival prediction and offers a cost-effective tool for precision oncology.},
  archive      = {J_NATMI},
  author       = {Andani, Sonali and Chen, Boqi and Ficek-Pascual, Joanna and Heinke, Simon and Casanova, Ruben and Hild, Bernard Friedrich and Sobottka, Bettina and Bodenmiller, Bernd and Koelzer, Viktor H. and Rätsch, Gunnar},
  doi          = {10.1038/s42256-025-01074-y},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1292-1307},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Histopathology-based protein multiplex generation using deep learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Type II mechanoreceptors and cuneate spiking neuronal network enable touch localization on a large-area e-skin. <em>NATMI</em>, <em>7</em>(8), 1278-1291. (<a href='https://doi.org/10.1038/s42256-025-01076-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The sense of touch is essential for humans to perceive, locate and react to physical stimuli. Notwithstanding the substantial advancements in e-skin research and related applications with collaborative robots and bionic prostheses, biomimetic intelligence remains a challenge in the attempt to understand and mimic somatosensory processing schemes. In this work, we present a large-area e-skin embedded with photonic fibre Bragg gratings, capable of decoding touch localization through a bioinspired two-layered spiking neuronal network. The implemented biomimicry of slowly adapting and fast-adapting type II primary afferents, cuneate neurons with overlapping receptive fields and neuroplasticity, enable unsupervised learning in localizing tactile stimuli with an error lower than 10 mm, and two-point discrimination thresholds matching human psychophysical thresholds in the forearm. These results align with biological findings and offer a promising step towards the development of bionic systems, opening new avenues for both practical applications and scientific explorations of somatosensation. Tactile sensing is essential for interacting with the environment. A bioinspired spiking neuronal network and large-area e-skin is presented, which enables unsupervised learning of touch localization and two-point discrimination.},
  archive      = {J_NATMI},
  author       = {Pereira Resende da Costa, Ana Clara and Filosa, Mariangela and Barbosa Soares, Alcimar and Oddo, Calogero Maria},
  doi          = {10.1038/s42256-025-01076-w},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1278-1291},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Type II mechanoreceptors and cuneate spiking neuronal network enable touch localization on a large-area e-skin},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep-learning-aided dismantling of interdependent networks. <em>NATMI</em>, <em>7</em>(8), 1266-1277. (<a href='https://doi.org/10.1038/s42256-025-01070-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying the minimal set of nodes whose removal breaks a complex network apart, also referred as the network dismantling problem, is a highly non-trivial task with applications in multiple domains. Whereas network dismantling has been extensively studied over the past decade, research has primarily focused on the optimization problem for single-layer networks, neglecting that many, if not all, real networks display multiple layers of interdependent interactions. In such networks, the optimization problem is fundamentally different as the effect of removing nodes propagates within and across layers in a way that can not be predicted using a single-layer perspective. Here we propose a dismantling algorithm named MultiDismantler, which leverages multiplex network representation and deep reinforcement learning to optimally dismantle multilayer interdependent networks. MultiDismantler is trained on small synthetic graphs; when applied to large, either real or synthetic, networks, it displays exceptional dismantling performance, clearly outperforming all existing benchmark algorithms. We show that MultiDismantler is effective in guiding strategies for the containment of diseases in social networks characterized by multiple layers of social interactions. Also, we show that MultiDismantler is useful in the design of protocols aimed at delaying the onset of cascading failures in interdependent critical infrastructures. The dismantling problem of removing the smallest set of nodes so that a given network breaks into disconnected components is hard to solve exactly. Gu and colleagues use deep reinforcement learning and a multiplex network representation to avoid the heavy computational cost.},
  archive      = {J_NATMI},
  author       = {Gu, Weiwei and Yang, Chen and Li, Lei and Hou, Jinqiang and Radicchi, Filippo},
  doi          = {10.1038/s42256-025-01070-2},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1266-1277},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep-learning-aided dismantling of interdependent networks},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Self-iterative multiple-instance learning enables the prediction of CD4+ t cell immunogenic epitopes. <em>NATMI</em>, <em>7</em>(8), 1250-1265. (<a href='https://doi.org/10.1038/s42256-025-01073-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate prediction of antigen presentation to CD4+ T cells and subsequent induction of immune response are fundamentally important for vaccine development, autoimmune disease treatment and cancer neoepitope discovery. In immunopeptidomics, single-allelic data offer high specificity but limited allele coverage, whereas multi-allelic data provide broader representation at the expense of weak labelling. Current computational approaches either overlook the abundance of multi-allelic data or suffer from label ambiguity due to inadequate modelling strategies. To address these limitations, we present ImmuScope, a weakly supervised deep learning framework that integrates major histocompatibility complex class II (MHC-II) antigen presentation, CD4+ T cell epitopes and immunogenicity assessment. ImmuScope leverages self-iterative multiple-instance learning with positive-anchor triplet loss to decipher peptide-MHC-II binding from weakly labelled multi-allelic data and high-confidence single-allelic data. The training dataset comprises over 600,000 ligands across 142 alleles. Additionally, ImmuScope enables the interpretation of MHC-II binding specificity and motif deconvolution of immunopeptidomics data. We successfully applied ImmuScope to identify melanoma neoantigens, uncovering mutation-driven variations in peptide-MHC-II binding and immunogenicity. Furthermore, we employed ImmuScope to evaluate the effects of SARS-CoV-2 epitope mutations associated with immune escape, with predictions well aligned with experimentally observed immune escape dynamics. Overall, by offering a unified solution for CD4+ T cell antigen recognition and immunogenicity assessment, ImmuScope holds substantial promise for accelerating vaccine design and advancing personalized immunotherapy. ImmuScope, a weakly supervised deep learning model capable of analysing multi- and single-allelic data, is introduced, facilitating interpretable neoantigen discovery and immune escape analysis.},
  archive      = {J_NATMI},
  author       = {Shen, Long-Chen and Zhang, Yumeng and Wang, Zhikang and Littler, Dene R. and Liu, Yan and Tang, Jinhui and Rossjohn, Jamie and Yu, Dong-Jun and Song, Jiangning},
  doi          = {10.1038/s42256-025-01073-z},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1250-1265},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Self-iterative multiple-instance learning enables the prediction of CD4+ t cell immunogenic epitopes},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A semantic-enhanced multi-modal remote sensing foundation model for earth observation. <em>NATMI</em>, <em>7</em>(8), 1235-1249. (<a href='https://doi.org/10.1038/s42256-025-01078-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Remote sensing foundation models, pretrained on massive remote sensing data, have shown impressive performance in several Earth observation (EO) tasks. These models usually use single-modal temporal data for pretraining, which is insufficient for multi-modal applications. Moreover, these models require a considerable number of samples for fine-tuning in downstream tasks, posing challenges in time-sensitive scenarios, such as rapid flood mapping. We present SkySense++, a multi-modal remote sensing foundation model for diverse EO tasks. SkySense++ has a factorized architecture to accommodate multi-modal images acquired by diverse sensors. We adopt progressive pretraining, which involves two stages, on meticulously curated datasets of 27 million multi-modal remote sensing images. The first representation-enhanced pretraining stage uses multi-granularity contrastive learning to obtain general representations. The second semantic-enhanced pretraining stage leverages masked semantic learning to learn semantically enriched representations, enabling few-shot capabilities. This ability allows the model to handle unseen tasks with minimal labelled data, alleviating the need for fine-tuning on extensive annotated data. SkySense++ demonstrates consistent improvements in classification, detection and segmentation over previous state-of-the-art models across 12 EO tasks in 7 domains: agriculture, forestry, oceanography, atmosphere, biology, land surveying and disaster management. This generalizability may lead to a new chapter of remote sensing foundation model applications for EO tasks at scale. Wu et al. developed SkySense++, a multi-modal remote sensing foundation model pretrained on 27 million multi-modal images, which achieved robust generalization and few-shot capabilities across several Earth observation tasks and domains, including agriculture and disaster management.},
  archive      = {J_NATMI},
  author       = {Wu, Kang and Zhang, Yingying and Ru, Lixiang and Dang, Bo and Lao, Jiangwei and Yu, Lei and Luo, Junwei and Zhu, Zifan and Sun, Yue and Zhang, Jiahao and Zhu, Qi and Wang, Jian and Yang, Ming and Chen, Jingdong and Zhang, Yongjun and Li, Yansheng},
  doi          = {10.1038/s42256-025-01078-8},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1235-1249},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A semantic-enhanced multi-modal remote sensing foundation model for earth observation},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-level visual representations in the human brain are aligned with large language models. <em>NATMI</em>, <em>7</em>(8), 1220-1234. (<a href='https://doi.org/10.1038/s42256-025-01072-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human brain extracts complex information from visual inputs, including objects, their spatial and semantic interrelations, and their interactions with the environment. However, a quantitative approach for studying this information remains elusive. Here we test whether the contextual information encoded in large language models (LLMs) is beneficial for modelling the complex visual information extracted by the brain from natural scenes. We show that LLM embeddings of scene captions successfully characterize brain activity evoked by viewing the natural scenes. This mapping captures selectivities of different brain areas and is sufficiently robust that accurate scene captions can be reconstructed from brain activity. Using carefully controlled model comparisons, we then proceed to show that the accuracy with which LLM representations match brain representations derives from the ability of LLMs to integrate complex information contained in scene captions beyond that conveyed by individual words. Finally, we train deep neural network models to transform image inputs into LLM representations. Remarkably, these networks learn representations that are better aligned with brain representations than a large number of state-of-the-art alternative models, despite being trained on orders-of-magnitude less data. Overall, our results suggest that LLM embeddings of scene captions provide a representational format that accounts for complex information extracted by the brain from visual inputs. Doerig, Kietzmann and colleagues show that the brain’s response to visual scenes can be modelled using language-based AI representations. By linking brain activity to caption-based embeddings from large language models, the study reveals a way to quantify complex visual understanding.},
  archive      = {J_NATMI},
  author       = {Doerig, Adrien and Kietzmann, Tim C. and Allen, Emily and Wu, Yihan and Naselaris, Thomas and Kay, Kendrick and Charest, Ian},
  doi          = {10.1038/s42256-025-01072-0},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1220-1234},
  shortjournal = {Nat. Mach. Intell.},
  title        = {High-level visual representations in the human brain are aligned with large language models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Training data composition determines machine learning generalization and biological rule discovery. <em>NATMI</em>, <em>7</em>(8), 1206-1219. (<a href='https://doi.org/10.1038/s42256-025-01089-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Supervised machine learning models depend on training datasets containing positive and negative examples: dataset composition directly impacts model performance and bias. Given the importance of machine learning for immunotherapeutic design, we examined how different negative class definitions affect model generalization and rule discovery for antibody–antigen binding. Using synthetic-structure-based binding data, we evaluated models trained with various definitions of negative sets. Our findings reveal that high out-of-distribution performance can be achieved when the negative dataset contains more similar samples to the positive dataset, despite lower in-distribution performance. Furthermore, by leveraging ground-truth information, we show that binding rules associated with positive data change based on the negative data used. Validation on experimental data supported simulation-based observations. This work underscores the role of dataset composition in creating robust, generalizable and biology-aware sequence-based ML models. Negative data composition critically shapes machine learning robustness in sequence-based biological tasks. Training data composition and its implications are investigated on biological rule discoveries.},
  archive      = {J_NATMI},
  author       = {Ursu, Eugen and Minnegalieva, Aygul and Rawat, Puneet and Chernigovskaya, Maria and Tacutu, Robi and Sandve, Geir Kjetil and Robert, Philippe A. and Greiff, Victor},
  doi          = {10.1038/s42256-025-01089-5},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1206-1219},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Training data composition determines machine learning generalization and biological rule discovery},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying artificial intelligence through algorithmic generalization. <em>NATMI</em>, <em>7</em>(8), 1195-1205. (<a href='https://doi.org/10.1038/s42256-025-01092-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid development of artificial intelligence (AI) systems has created an urgent need for their scientific quantification. While their fluency across a variety of domains is impressive, AI systems fall short on tests requiring algorithmic reasoning—a glaring limitation, given the necessity for interpretable and reliable technology. Despite a surge in reasoning benchmarks emerging from the academic community, no theoretical framework exists to quantify algorithmic reasoning in AI systems. Here we adopt a framework from computational complexity theory to quantify algorithmic generalization using algebraic expressions: algebraic circuit complexity. Algebraic circuit complexity theory—the study of algebraic expressions as circuit models—is a natural framework for studying the complexity of algorithmic computation. Algebraic circuit complexity enables the study of generalization by defining benchmarks in terms of the computational requirements for solving a problem. Moreover, algebraic circuits are generic mathematical objects; an arbitrarily large number of samples can be generated for a specified circuit, making it an ideal experimental sandbox for the data-hungry models that are used today. In this Perspective, we adopt tools from algebraic circuit complexity, apply them to formalize a science of algorithmic generalization, and address key challenges for its successful application to AI science. Despite impressive performances of current large AI models, symbolic and abstract reasoning tasks often elicit failure modes in these systems. In this Perspective, Ito et al. propose to make use of computational complexity theory, formulating algebraic problems as computable circuits to address the challenge of mathematical and symbolic reasoning in AI systems.},
  archive      = {J_NATMI},
  author       = {Ito, Takuya and Campbell, Murray and Horesh, Lior and Klinger, Tim and Ram, Parikshit},
  doi          = {10.1038/s42256-025-01092-w},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1195-1205},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Quantifying artificial intelligence through algorithmic generalization},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The importance of negative training data for robust antibody binding prediction. <em>NATMI</em>, <em>7</em>(8), 1192-1194. (<a href='https://doi.org/10.1038/s42256-025-01080-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thoughtfully designed negative training datasets may hold the key to more robust machine learning models. Ursu et al. reveal how negative training data composition shapes antibody prediction models and their generalizability. Sometimes, the best way to get better is to train harder.},
  archive      = {J_NATMI},
  author       = {Ta, Wesley and Stokes, Jonathan M.},
  doi          = {10.1038/s42256-025-01080-0},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1192-1194},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The importance of negative training data for robust antibody binding prediction},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping t helper cell targets with deep learning. <em>NATMI</em>, <em>7</em>(8), 1190-1191. (<a href='https://doi.org/10.1038/s42256-025-01081-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By integrating multi-dimensional data with deep learning, a new method known as ImmuScope predicts both major histocompatibility complex class II (MHC-II) presentation and T helper cell immunogenicity. ImmuScope shows potential to accelerate neoantigen discovery and vaccine design.},
  archive      = {J_NATMI},
  author       = {Liu, Yuan and Han, Leng},
  doi          = {10.1038/s42256-025-01081-z},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1190-1191},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Mapping t helper cell targets with deep learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards responsible geospatial foundation models. <em>NATMI</em>, <em>7</em>(8), 1189. (<a href='https://doi.org/10.1038/s42256-025-01106-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent years have seen a surge in geospatial artificial intelligence models, with promising applications in ecological and environmental monitoring tasks. Further work should also focus on the sustainable development of such models.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-025-01106-7},
  journal      = {Nature Machine Intelligence},
  month        = {8},
  number       = {8},
  pages        = {1189},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards responsible geospatial foundation models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nature">NATURE - 64</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Multiple oestradiol functions inhibit ferroptosis and acute kidney injury. <em>NATURE</em>, <em>645</em>(8081), E9. (<a href='https://doi.org/10.1038/s41586-025-09562-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Tonnus, Wulf and Maremonti, Francesca and Gavali, Shubhangi and Schlecht, Marlena Nastassja and Gembardt, Florian and Belavgeni, Alexia and Leinung, Nadja and Flade, Karolin and Bethe, Natalie and Traikov, Sofia and Haag, Anne and Schilling, Danny and Penkov, Sider and Mallais, Melodie and Gaillet, Christine and Meyer, Claudia and Katebi, Melika and Ray, Anushka and Gerhardt, Louisa M. S. and Brucker, Anne and Becker, Jorunn Naila and Tmava, Mirela and Schlicker, Lisa and Schulze, Almut and Himmerkus, Nina and Shevchenko, Andrej and Peitzsch, Mirko and Barayeu, Uladzimir and Nasi, Sonia and Putz, Juliane and Korach, Kenneth S. and Neugarten, Joel and Golestaneh, Ladan and Hugo, Christian and Becker, Jan Ulrich and Weinberg, Joel M. and Lorenz, Svenja and Proneth, Bettina and Conrad, Marcus and Wolf, Eckhard and Plietker, Bernd and Rodriguez, Raphaël and Pratt, Derek A. and Dick, Tobias P. and Fedorova, Maria and Bornstein, Stefan R. and Linkermann, Andreas},
  doi          = {10.1038/s41586-025-09562-2},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {E9},
  shortjournal = {Nature},
  title        = {Publisher correction: Multiple oestradiol functions inhibit ferroptosis and acute kidney injury},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Dual neuromodulatory dynamics underlie birdsong learning. <em>NATURE</em>, <em>645</em>(8081), E8. (<a href='https://doi.org/10.1038/s41586-025-09563-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Qi, Jiaxuan and Schreiner, Drew C. and Martinez, Miles and Pearson, John and Mooney, Richard},
  doi          = {10.1038/s41586-025-09563-1},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {E8},
  shortjournal = {Nature},
  title        = {Author correction: Dual neuromodulatory dynamics underlie birdsong learning},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Endophilin marks and controls a clathrin-independent endocytic pathway. <em>NATURE</em>, <em>645</em>(8081), E7. (<a href='https://doi.org/10.1038/s41586-025-09552-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Boucrot, Emmanuel and Ferreira, Antonio P. A. and Almeida-Souza, Leonardo and Debard, Sylvain and Vallis, Yvonne and Howard, Gillian and Bertot, Laetitia and Sauvonnet, Nathalie and McMahon, Harvey T.},
  doi          = {10.1038/s41586-025-09552-4},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {E7},
  shortjournal = {Nature},
  title        = {Author correction: Endophilin marks and controls a clathrin-independent endocytic pathway},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Complex genetic variation in nearly complete human genomes. <em>NATURE</em>, <em>645</em>(8081), E6. (<a href='https://doi.org/10.1038/s41586-025-09547-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Logsdon, Glennis A. and Ebert, Peter and Audano, Peter A. and Loftus, Mark and Porubsky, David and Ebler, Jana and Yilmaz, Feyza and Hallast, Pille and Prodanov, Timofey and Yoo, DongAhn and Paisie, Carolyn A. and Harvey, William T. and Zhao, Xuefang and Martino, Gianni V. and Henglin, Mir and Munson, Katherine M. and Rabbani, Keon and Chin, Chen-Shan and Gu, Bida and Ashraf, Hufsah and Scholz, Stephan and Austine-Orimoloye, Olanrewaju and Balachandran, Parithi and Bonder, Marc Jan and Cheng, Haoyu and Chong, Zechen and Crabtree, Jonathan and Gerstein, Mark and Guethlein, Lisbeth A. and Hasenfeld, Patrick and Hickey, Glenn and Hoekzema, Kendra and Hunt, Sarah E. and Jensen, Matthew and Jiang, Yunzhe and Koren, Sergey and Kwon, Youngjun and Li, Chong and Li, Heng and Li, Jiaqi and Norman, Paul J. and Oshima, Keisuke K. and Paten, Benedict and Phillippy, Adam M. and Pollock, Nicholas R. and Rausch, Tobias and Rautiainen, Mikko and Song, Yuwei and Söylev, Arda and Sulovari, Arvis and Surapaneni, Likhitha and Tsapalou, Vasiliki and Zhou, Weichen and Zhou, Ying and Zhu, Qihui and Zody, Michael C. and Mills, Ryan E. and Devine, Scott E. and Shi, Xinghua and Talkowski, Michael E. and Chaisson, Mark J. P. and Dilthey, Alexander T. and Konkel, Miriam K. and Korbel, Jan O. and Lee, Charles and Beck, Christine R. and Eichler, Evan E. and Marschall, Tobias},
  doi          = {10.1038/s41586-025-09547-1},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {E6},
  shortjournal = {Nature},
  title        = {Author correction: Complex genetic variation in nearly complete human genomes},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Disease-associated astrocyte epigenetic memory promotes CNS pathology. <em>NATURE</em>, <em>645</em>(8081), E5. (<a href='https://doi.org/10.1038/s41586-025-09546-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Lee, Hong-Gyun and Rone, Joseph M. and Li, Zhaorong and Akl, Camilo Faust and Shin, Seung Won and Lee, Joon-Hyuk and Flausino, Lucas E. and Pernin, Florian and Chao, Chun-Cheih and Kleemann, Kilian L. and Srun, Lena and Illouz, Tomer and Giovannoni, Federico and Charabati, Marc and Sanmarco, Liliana M. and Kenison, Jessica E. and Piester, Gavin and Zandee, Stephanie E. J. and Antel, Jack P. and Rothhammer, Veit and Wheeler, Michael A. and Prat, Alexandre and Clark, Iain C. and Quintana, Francisco J.},
  doi          = {10.1038/s41586-025-09546-2},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {E5},
  shortjournal = {Nature},
  title        = {Author correction: Disease-associated astrocyte epigenetic memory promotes CNS pathology},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Macrophages excite muscle spindles with glutamate to bolster locomotion. <em>NATURE</em>, <em>645</em>(8081), E4. (<a href='https://doi.org/10.1038/s41586-025-09550-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Yan, Yuyang and Antolin, Nuria and Zhou, Luming and Xu, Luyang and Vargas, Irene Lisa and Gomez, Carlos Daniel and Kong, Guiping and Palmisano, Ilaria and Yang, Yi and Chadwick, Jessica and Müller, Franziska and Bull, Anthony M. J. and Lo Celso, Cristina and Primiano, Guido and Servidei, Serenella and Perrier, Jean François and Bellardita, Carmelo and Di Giovanni, Simone},
  doi          = {10.1038/s41586-025-09550-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {E4},
  shortjournal = {Nature},
  title        = {Author correction: Macrophages excite muscle spindles with glutamate to bolster locomotion},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sweet like chocolate: Researching in the shade of a cacao tree. <em>NATURE</em>, <em>645</em>(8081), 814. (<a href='https://doi.org/10.1038/d41586-025-02923-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Naailah Ali aims to support cacao farmers in the West Indies by improving the bean-fermentation process. Naailah Ali aims to support cacao farmers in the West Indies by improving the bean-fermentation process.},
  archive      = {J_NATURE},
  author       = {Brazil, Rachel},
  doi          = {10.1038/d41586-025-02923-x},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {814},
  shortjournal = {Nature},
  title        = {Sweet like chocolate: Researching in the shade of a cacao tree},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ‘Lipstick on a pig’: How to fight back against a peer-review bully. <em>NATURE</em>, <em>645</em>(8081), 809-812. (<a href='https://doi.org/10.1038/d41586-025-02922-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientific societies, journals, editors and researchers are pushing back against mean-spirited peer reviews. Scientific societies, journals, editors and researchers are pushing back against mean-spirited peer reviews.},
  archive      = {J_NATURE},
  author       = {Zimmer, Katarina},
  doi          = {10.1038/d41586-025-02922-y},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {809-812},
  shortjournal = {Nature},
  title        = {‘Lipstick on a pig’: How to fight back against a peer-review bully},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural and functional characterization of human sweet taste receptor. <em>NATURE</em>, <em>645</em>(8081), 801-808. (<a href='https://doi.org/10.1038/s41586-025-09302-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sweet taste perception influences dietary choices and metabolic health. The human sweet taste receptor, a class C G-protein-coupled receptor (GPCR) heterodimer composed of TAS1R2 and TAS1R3 (refs. 1,2), senses a wide range of sweet compounds—including natural sugars, artificial sweeteners and sweet proteins—and affects metabolic regulation beyond taste. However, the lack of three-dimensional structures hinders our understanding of its precise working mechanism. Here we present cryo-electron microscopy structures of the full-length human sweet taste receptor in apo and sucralose-bound states. These structures reveal a distinct asymmetric heterodimer architecture, with sucralose binding exclusively to the Venus flytrap domain of TAS1R2. Combining mutagenesis and molecular dynamics simulations, this work delineates the sweetener-recognition modes in TAS1R2. Structural comparisons further uncover conformational changes upon ligand binding and a unique activation mechanism. These findings illuminate the signal transduction mechanisms of chemosensory receptors in the class C GPCR family and provide the molecular basis for the design of a new generation of sweeteners. Cryo-electron microscopy structures of the human sweet taste receptor in the apo and sucralose-bound states reveal its heterodimeric architecture and mechanisms of ligand binding and signal activation.},
  archive      = {J_NATURE},
  author       = {Shi, Zongjun and Xu, Weixiu and Wu, Lijie and Yue, Xiaolei and Liu, Shenhui and Ding, Wei and Zhang, Jinyi and Meng, Bing and Zhao, Lianghao and Liu, Xiaoyan and Liu, Junlin and Liu, Zhi-Jie and Hua, Tian},
  doi          = {10.1038/s41586-025-09302-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {801-808},
  shortjournal = {Nature},
  title        = {Structural and functional characterization of human sweet taste receptor},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Programmable protein ligation on cell surfaces. <em>NATURE</em>, <em>645</em>(8081), 793-800. (<a href='https://doi.org/10.1038/s41586-025-09287-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The surface landscapes of cells differ as a function of cell type and are frequently altered in disease contexts1–3. Exploiting such differences is key to many therapeutic strategies and is the basis for developing diagnostic and basic-science tools. State-of-the-art strategies typically target single surface antigens, but each individual receptor rarely defines the specific cell type4,5. The development of programmable molecular systems that integrate multiple cell-surface features to convert on-target inputs to user-defined outputs is therefore highly desirable. Here we describe an autonomous decision-making device driven by proximity-gated protein trans-splicing that allows local generation of an active protein from two otherwise inactive polypeptide fragments. We show that this protein-actuator platform can perform convergent protein ligation on designated cell surfaces, allowing highly selective generation of active proteins, which can either remain physically associated with the cell surface on which they were manufactured or be released into the surrounding milieu. Because of its intrinsic modularity and tunability, we demonstrate that the technology is compatible with different types of input, targeting modality and functional output, allowing for the localized interrogation or manipulation of cellular systems. A synthetic biology system called SMART has been developed that uses conditional protein splicing for the programmable ligation of functional proteins from previously defined molecular combinations on cell surfaces.},
  archive      = {J_NATURE},
  author       = {Kofoed, Christian and Erkalo, Girum and Tay, Nicholas E. S. and Ye, Xuanjia and Lin, Yutong and Muir, Tom W.},
  doi          = {10.1038/s41586-025-09287-2},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {793-800},
  shortjournal = {Nature},
  title        = {Programmable protein ligation on cell surfaces},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). RNA N-glycosylation enables immune evasion and homeostatic efferocytosis. <em>NATURE</em>, <em>645</em>(8081), 784-792. (<a href='https://doi.org/10.1038/s41586-025-09310-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glycosylation is central to the localization and function of biomolecules1. We recently discovered that small RNAs undergo N-glycosylation2 at the modified RNA base 3-(3-amino-3-carboxypropyl) uridine (acp3U)3. However, the functional significance of N-glycosylation of RNAs is unknown. Here we show that the N-glycans on glycoRNAs prevent innate immune sensing of endogenous small RNAs. We found that de-N-glycosylation of cell-culture-derived and circulating human and mouse glycoRNA elicited potent inflammatory responses including the production of type I interferons in a Toll-like receptor 3- and Toll-like receptor 7-dependent manner. Furthermore, we show that N-glycans on cell surface RNAs prevent apoptotic cells from triggering endosomal RNA sensors in efferocytes, thus facilitating the non-inflammatory clearance of dead cells. Mechanistically, N-glycans conceal the hypermodified uracil base acp3U, which we identified as immunostimulatory when exposed in RNA. Consistent with this, genetic deletion of an enzyme (DTWD2) that synthesizes acp3U abrogated innate immune activation by de-N-glycosylated small RNAs and apoptotic cells. Furthermore, synthetic acp3U-containing RNAs are sufficient to trigger innate immune responses. Thus, our study has uncovered a natural mechanism by which N-glycans block RNAs from inducing acp3U-dependent innate immune activation, demonstrating how glycoRNAs exist on the cell surface and in the endosomal network without inducing autoinflammatory responses. N-glycans on glycoRNAs prevent innate immune sensing of endogenous small RNAs, and the natural mechanism they use demonstrates how glycoRNAs exist on the cell surface and in the endosomal network without inducing autoinflammatory responses.},
  archive      = {J_NATURE},
  author       = {Graziano, Vincent R. and Porat, Jennifer and Ah Kioon, Marie Dominique and Mejdrová, Ivana and Matz, Alyssa J. and Lebedenko, Charlotta G. and Chai, Peiyuan and Pluvinage, John V. and Ricci-Azevedo, Rafael and Harrison, Andrew G. and Wright, Skylar S. and Wang, Xinzheng and Strine, Madison S. and Wang, Penghua and Wilson, Michael R. and Vanaja, Sivapriya Kailasan and Zhou, Beiyan and Barrat, Franck J. and Carell, Thomas and Flynn, Ryan A. and Rathinam, Vijay A.},
  doi          = {10.1038/s41586-025-09310-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {784-792},
  shortjournal = {Nature},
  title        = {RNA N-glycosylation enables immune evasion and homeostatic efferocytosis},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Excised DNA circles from V(D)J recombination promote relapsed leukaemia. <em>NATURE</em>, <em>645</em>(8081), 774-783. (<a href='https://doi.org/10.1038/s41586-025-09372-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extrachromosomal DNA amplification is associated with poor cancer prognoses1. Large numbers of excised signal circles (ESCs) are produced as by-products of antigen receptor rearrangement during&nbsp;V(D)J recombination2,3. However, current dogma states that ESCs are progressively lost through cell division4. Here we show that ESCs replicate and persist through many cell generations and share many properties in common&nbsp;with circular extrachromosomal DNAs. Increased ESC copy numbers at diagnosis of B cell precursor acute lymphoblastic leukaemia were highly correlated with subsequent relapse. By taking advantage of the matching recombination footprint that is formed upon the generation of each ESC, we measured ESC persistence and replication and found increased ESC replication in patients who later relapsed. This increased replication is controlled by cell-intrinsic factors and corresponds to increased expression of DNA replication- and repair-associated genes. Consistent with high ESC levels having a role in disease progression, the number of mutations typical of those caused by the V(D)J recombinase–ESC complex was significantly increased at diagnosis in patients who later relapsed. The number of such&nbsp;mutations in genes associated with relapse increased between diagnosis and relapse, and corresponded to clonal expansion of cells with high ESC copy numbers. These data demonstrate that the by-product of V(D)J recombination, when increased in abundance, potently associates with the&nbsp;V(D)J recombinase to cause adverse disease outcomes. Excised signal circles are circular DNA by-products of V(D)J recombination that form a complex with the V(D)J recombinase, and when increased in abundance, result in increased mutagenesis, causing adverse outcomes in cancer.},
  archive      = {J_NATURE},
  author       = {Gao, Zeqian and Scott, James N. F. and Edwards, Matthew P. and Casey, Dylan and Wang, Xiaoling and Gillen, Andrew D. and Ryan, Sarra and Russell, Lisa J. and Moorman, Anthony V. and de Tute, Ruth and Cargo, Catherine and Ford, Anthony M. and Westhead, David R. and Boyes, Joan},
  doi          = {10.1038/s41586-025-09372-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {774-783},
  shortjournal = {Nature},
  title        = {Excised DNA circles from V(D)J recombination promote relapsed leukaemia},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fluctuating DNA methylation tracks cancer evolution at clinical scale. <em>NATURE</em>, <em>645</em>(8081), 764-773. (<a href='https://doi.org/10.1038/s41586-025-09374-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour. Cancer evolutionary dynamics are quantitatively inferred using a method, EVOFLUx, applied to fluctuating DNA methylation.},
  archive      = {J_NATURE},
  author       = {Gabbutt, Calum and Duran-Ferrer, Martí and Grant, Heather E. and Mallo, Diego and Nadeu, Ferran and Househam, Jacob and Villamor, Neus and Müller, Madlen and Heath, Simon and Raineri, Emanuele and Krali, Olga and Nordlund, Jessica and Zenz, Thorsten and Gut, Ivo G. and Campo, Elias and Lopez-Guillermo, Armando and Fitzgibbon, Jude and Barnes, Chris P. and Shibata, Darryl and Martin-Subero, José I. and Graham, Trevor A.},
  doi          = {10.1038/s41586-025-09374-4},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {764-773},
  shortjournal = {Nature},
  title        = {Fluctuating DNA methylation tracks cancer evolution at clinical scale},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SuFEx-based antitubercular compound irreversibly inhibits pks13. <em>NATURE</em>, <em>645</em>(8081), 755-763. (<a href='https://doi.org/10.1038/s41586-025-09286-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mycobacterium tuberculosis (Mtb) remains the world’s deadliest bacterial pathogen1. There is an urgent medical need to develop new drugs that shorten the treatment duration to combat widespread multi-drug-resistant and extensive-drug-resistant Mtb. Here, we present a preclinical covalent compound, CMX410, that contains an aryl fluorosulfate (SuFEx)2 warhead and uniquely targets the acyltransferase domain of Pks13, an essential enzyme in cell-wall biosynthesis. CMX410 is equipotent against drug-sensitive and drug-resistant strains of Mtb and efficacious in multiple mouse models of infection. Inhibition by CMX410 is irreversible through a previously undescribed mechanism: CMX410 reacts with the catalytic serine of the AT domain of Pks13, rapidly and irreversibly disabling the active site by forming a β-lactam. CMX410 is highly selective for its target and thus demonstrates excellent pharmacological and safety profiles, including no adverse effects in a 14-day rat toxicity study up to 1,000 mg kg−1 per day. The distinctive mode of action from current drugs, high potency across all tested clinical isolates, oral bioavailability, favourable performance in drug combination testing and superior pharmacological and safety characteristics make CMX410 a promising first-in-class candidate to replace outdated cell-wall biosynthesis inhibitors, such as isoniazid and ethambutol, in tuberculosis regimens. A preclinical covalent compound, CMX410, contains a aryl fluorosulfate warhead that targets the acyltransferase domain of Mtb&nbsp;Pks13, an essential enzyme in cell-wall biosynthesis, making it a promising candidate for tuberculosis treatment regimens.},
  archive      = {J_NATURE},
  author       = {Krieger, Inna V. and Sukheja, Paridhi and Yang, Baiyuan and Tang, Su and Selle, Daniel and Woods, Ashley and Engelhart, Curtis and Kumar, Pradeep and Harbut, Michael B. and Liu, Dongdong and Tsuda, Brendan and Qin, Bo and Bare, Grant A. L. and Li, Gencheng and Chi, Victor and Gambacurta, Julian and Hvizdos, Janine and Reagan, Matthew and Jones, Isabelle L. and Massoudi, Lisa M. and Woolhiser, Lisa K. and Cascioferro, Alessandro and Kundrick, Erica and Singh, Parul and Reiley, William and Ioerger, Thomas R. and Kandula, Dilipkumar Reddy and McCabe, Jacob W. and Guo, Taijie and Alland, David and Boshoff, Helena I. and Schnappinger, Dirk and Robertson, Gregory T. and Mdluli, Khisi and Lee, Kyoung-Jin and Dong, Jiajia and Li, Shuangwei and Schultz, Peter G. and Joseph, Sean B. and Love, Melissa S. and Sharpless, K. Barry and Petrassi, H. Michael and Chatterjee, Arnab K. and Sacchettini, James C. and McNamara, Case W.},
  doi          = {10.1038/s41586-025-09286-3},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {755-763},
  shortjournal = {Nature},
  title        = {SuFEx-based antitubercular compound irreversibly inhibits pks13},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Driving a protective allele of the mosquito FREP1 gene to combat malaria. <em>NATURE</em>, <em>645</em>(8081), 746-754. (<a href='https://doi.org/10.1038/s41586-025-09283-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Malaria remains a substantial global health challenge, causing approximately half a million deaths each year1. The mosquito fibrinogen-related protein 1 (FREP1) is required for malaria parasites to infect the midgut epithelium2. The naturally occurring FREP1Q allele has been reported to prevent parasite infection, while supporting essential physiological functions in the mosquito3. Here we generate congenic strains of Anopheles stephensi, edited to carry either the parasite-susceptible FREP1L224 or the putative-refractory FREP1Q224 alleles. The FREP1Q224 allele confers robust resistance to infection by both human and rodent malaria parasites, with negligible fitness costs. The protective FREP1Q224 allele can be efficiently driven into FREP1L224 mosquito populations using a novel linked allelic-drive system that selectively replaces the L224 codon with the parasite-refractory Q224 allele, thereby rendering populations refractory to parasite infection. This antimalaria drive system provides a novel genetic approach to aid in malaria elimination efforts. Coupling of a FREP1 variant to a gene-drive system allows the spread of a host malaria resistance allele through the mosquito population.},
  archive      = {J_NATURE},
  author       = {Li, Zhiqian and Dong, Yuemei and You, Lang and Corder, Rodrigo M. and Arzobal, Jemariz and Yeun, Audrey and Yang, Lei and Marshall, John M. and Dimopoulos, George and Bier, Ethan},
  doi          = {10.1038/s41586-025-09283-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {746-754},
  shortjournal = {Nature},
  title        = {Driving a protective allele of the mosquito FREP1 gene to combat malaria},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Remodelling autoactive NLRs for broad-spectrum immunity in plants. <em>NATURE</em>, <em>645</em>(8081), 737-745. (<a href='https://doi.org/10.1038/s41586-025-09252-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Remodelling plant immune receptors has become a vital strategy for creating new disease resistance traits to combat the growing threat of plant pathogens to global food security and environmental sustainability1–17. However, current methods are constrained by the rapid evolution of plant pathogens and often lack broad-spectrum and durable protection. Here we report an innovative strategy to engineer broad-spectrum, durable and complete disease resistance in plants through expression of a chimeric protein containing a flexible polypeptide coupled with a single or dual conserved pathogen-originated protease cleavage site fused in frame to the N terminus of an autoactive nucleotide-binding and leucine-rich-repeat immune receptor (NLR) containing a coiled-coil or RESISTANCE TO POWDERY MILDEW 8-like coiled-coil domain. Following invasion, pathogen-originated specific proteases cleave the inactive chimeric protein to form free autoactive NLR, triggering broad-spectrum plant disease resistance. We demonstrate that a single engineered NLR can confer broad-spectrum and complete resistance against multiple potyviruses. Given that many pathogenic organisms across kingdoms encode proteases, this strategy has the potential to be exploited to control viruses, bacteria, oomycetes, fungi, nematodes and pests in plants. Cleavage by pathogen-derived proteases of an engineered chimeric protein activates its plant immune receptor component, enabling broad-spectrum resistance to pathogens in plants.},
  archive      = {J_NATURE},
  author       = {Wang, Junzhu and Chen, Tianyuan and Zhang, Zhendong and Song, Mengjie and Shen, Tianxin and Wang, Xin and Zheng, Xiyin and Wang, Yan and Song, Ke and Ge, Xiaoyang and Xu, Kai and Qi, Tiancong and Li, Fuguang and Hong, Yiguo and Liu, Yule},
  doi          = {10.1038/s41586-025-09252-z},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {737-745},
  shortjournal = {Nature},
  title        = {Remodelling autoactive NLRs for broad-spectrum immunity in plants},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A gut sense for a microbial pattern regulates feeding. <em>NATURE</em>, <em>645</em>(8081), 729-736. (<a href='https://doi.org/10.1038/s41586-025-09301-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To coexist with its resident microorganisms, the host must have a sense to adjust its behaviour in response to them. In the intestine, a sense for nutrients transduced to the brain through neuroepithelial circuits guides appetitive choices1–5. However, a sense that allows the host to respond in real time to stimuli arising from resident gut microorganisms remains to be uncovered. Here we show that in the mouse colon, the ubiquitous microbial pattern flagellin—a unifying feature across phyla6—stimulates Toll-like receptor 5 (TLR5) in peptide YY (PYY)-labelled colonic neuropod cells. This stimulation leads to PYY release onto NPY2R vagal nodose neurons to regulate feeding. Mice lacking TLR5 in these cells eat more and gain more weight than controls. We found that flagellin does not act on the nerve directly. Instead, flagellin stimulates neuropod cells from the colonic lumen to reduce feeding through a gut–brain sensory neural circuit. Moreover, flagellin reduces feeding independent of immune responses, metabolic changes or the presence of gut microbiota. This sense enables the host to adjust its behaviour in response to a molecular pattern from its resident microorganisms. We call this sense at the interface of the biota and the brain the neurobiotic sense7. A study reveals a gut–brain sensory pathway through which the microbial component flagellin activates neuropod&nbsp;cells in the colon to signal the brain and reduce feeding in mice.},
  archive      = {J_NATURE},
  author       = {Liu, Winston W. and Reicher, Naama and Alway, Emily and Rupprecht, Laura E. and Weng, Peter and Schaefgen, Chloe and Klein, Marguerita E. and Villalobos, Jorge A. and Puerto-Hernandez, Carlos and Kiesling Altún, Yolanda Graciela and Carbajal, Amanda and Aguayo-Guerrero, José Alfredo and Coss, Alam and Sahasrabudhe, Atharva and Anikeeva, Polina and de Araujo, Alan and Bali, Avnika and de Lartigue, Guillaume and Gil-Lievana, Elvi and Gutierrez, Ranier and Miao, Edward A. and Rawls, John F. and Kaelberer, M. Maya and Bohórquez, Diego V.},
  doi          = {10.1038/s41586-025-09301-7},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {729-736},
  shortjournal = {Nature},
  title        = {A gut sense for a microbial pattern regulates feeding},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mitochondrial origins of the pressure to sleep. <em>NATURE</em>, <em>645</em>(8081), 722-728. (<a href='https://doi.org/10.1038/s41586-025-09261-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To gain a comprehensive, unbiased perspective on molecular changes in the brain that may underlie the need for sleep, we have characterized the transcriptomes of single cells isolated from rested and sleep-deprived flies. Here we report that transcripts upregulated after sleep deprivation, in sleep-control neurons projecting to the dorsal fan-shaped body1,2 (dFBNs) but not ubiquitously in the brain, encode almost exclusively proteins with roles in mitochondrial respiration and ATP synthesis. These gene expression changes are accompanied by mitochondrial fragmentation, enhanced mitophagy and an increase in the number of contacts between mitochondria and the endoplasmic reticulum, creating conduits3,4 for the replenishment of peroxidized lipids5. The morphological changes are reversible after recovery sleep and blunted by the installation of an electron overflow6,7 in the respiratory chain. Inducing or preventing mitochondrial fission or fusion8–13 in dFBNs alters sleep and the electrical properties of sleep-control cells in opposite directions: hyperfused mitochondria increase, whereas fragmented mitochondria decrease, neuronal excitability and sleep. ATP concentrations in dFBNs rise after enforced waking because of diminished ATP consumption during the arousal-mediated inhibition of these neurons14, which augments their mitochondrial electron leak7. Consistent with this view, uncoupling electron flux from ATP synthesis15 relieves the pressure to sleep, while exacerbating mismatches between electron supply and ATP demand (by powering ATP synthesis with a light-driven proton pump16) precipitates sleep. Sleep, like ageing17,18, may be an inescapable consequence of aerobic metabolism. Research on Drosophila neurons&nbsp;shows links between the need to sleep and aerobic metabolism, indicating that the pressure to sleep may have a mitochondrial origin.},
  archive      = {J_NATURE},
  author       = {Sarnataro, Raffaele and Velasco, Cecilia D. and Monaco, Nicholas and Kempf, Anissa and Miesenböck, Gero},
  doi          = {10.1038/s41586-025-09261-y},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {722-728},
  shortjournal = {Nature},
  title        = {Mitochondrial origins of the pressure to sleep},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lithium deficiency and the onset of alzheimer’s disease. <em>NATURE</em>, <em>645</em>(8081), 712-721. (<a href='https://doi.org/10.1038/s41586-025-09335-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The earliest molecular changes in Alzheimer’s disease (AD) are poorly understood1–5. Here we show that endogenous lithium (Li) is dynamically regulated in the brain and contributes to cognitive preservation during ageing. Of the metals we analysed, Li was the only one that was significantly reduced in the brain in individuals with mild cognitive impairment (MCI), a precursor to AD. Li bioavailability was further reduced in AD by amyloid sequestration. We explored the role of endogenous Li in the brain by depleting it from the diet of wild-type and AD mouse models. Reducing endogenous cortical Li by approximately 50% markedly increased the deposition of amyloid-β and the accumulation of phospho-tau, and led to pro-inflammatory microglial activation, the loss of synapses, axons and myelin, and accelerated cognitive decline. These effects were mediated, at least in part, through activation of the kinase GSK3β. Single-nucleus RNA-seq showed that Li deficiency gives rise to transcriptome changes in multiple brain cell types that overlap with transcriptome changes in AD. Replacement therapy with lithium orotate, which is a Li salt with reduced amyloid binding, prevents pathological changes and memory loss in AD mouse models and ageing wild-type mice. These findings reveal physiological effects of endogenous Li in the brain and indicate that disruption of Li homeostasis may be an early event in the pathogenesis of AD. Li replacement with amyloid-evading salts is a potential approach to the prevention and treatment of AD. Lithium has an essential role in the brain and is deficient early in Alzheimer’s disease, which can be recapitulated in mice and treated with a novel lithium salt that restores the physiological level.},
  archive      = {J_NATURE},
  author       = {Aron, Liviu and Ngian, Zhen Kai and Qiu, Chenxi and Choi, Jaejoon and Liang, Marianna and Drake, Derek M. and Hamplova, Sara E. and Lacey, Ella K. and Roche, Perle and Yuan, Monlan and Hazaveh, Saba S. and Lee, Eunjung A. and Bennett, David A. and Yankner, Bruce A.},
  doi          = {10.1038/s41586-025-09335-x},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {712-721},
  shortjournal = {Nature},
  title        = {Lithium deficiency and the onset of alzheimer’s disease},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A generic non-invasive neuromotor interface for human-computer interaction. <em>NATURE</em>, <em>645</em>(8081), 702-711. (<a href='https://doi.org/10.1038/s41586-025-09255-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since the advent of computing, humans have sought computer input technologies that are expressive, intuitive and universal. While diverse modalities have been developed, including keyboards, mice and touchscreens, they require interaction with a device that can be limiting, especially in on-the-go scenarios. Gesture-based systems use cameras or inertial sensors to avoid an intermediary device, but tend to perform well only for unobscured movements. By contrast, brain–computer or neuromotor interfaces that directly interface with the body’s electrical signalling have been imagined to solve the interface problem1, but high-bandwidth communication has been demonstrated only using invasive interfaces with bespoke decoders designed for single individuals2–4. Here, we describe the development of a generic non-invasive neuromotor interface that enables computer input decoded from surface electromyography (sEMG). We developed a highly sensitive, easily donned sEMG wristband and a scalable infrastructure for collecting training data from thousands of consenting participants. Together, these data enabled us to develop generic sEMG decoding models that generalize across people. Test users demonstrate a closed-loop median performance of gesture decoding of 0.66 target acquisitions per second in a continuous navigation task, 0.88 gesture detections per second in a discrete-gesture task and handwriting at 20.9 words per minute. We demonstrate that the decoding performance of handwriting models can be further improved by 16% by personalizing sEMG decoding models. To our knowledge, this is the first high-bandwidth neuromotor interface with performant out-of-the-box generalization across people. A high-bandwidth neuromotor interface offers performant out-of-the-box generalization across people.},
  archive      = {J_NATURE},
  author       = {Kaifosh, Patrick and Reardon, Thomas R.},
  doi          = {10.1038/s41586-025-09255-w},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {702-711},
  shortjournal = {Nature},
  title        = {A generic non-invasive neuromotor interface for human-computer interaction},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Whole-genome sequencing of 490,640 UK biobank participants. <em>NATURE</em>, <em>645</em>(8081), 692-701. (<a href='https://doi.org/10.1038/s41586-025-09272-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome sequencing provides an unbiased and complete view of the human genome and enables the discovery of genetic variation without the technical limitations of other genotyping technologies. Here we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping effort1. This advance deepens our understanding of how genetics associates with disease biology and further enhances the value of this open resource for the study of human biology and health. Coupling this dataset with rich phenotypic data, we surveyed within- and cross-ancestry genomic associations and identified novel genetic and clinical insights. Although most associations with disease traits were primarily observed in individuals of European ancestries, strong or novel signals were also identified in individuals of African and Asian ancestries. With the improved ability to accurately genotype structural variants and exonic variation in both coding and UTR sequences, we strengthened and revealed novel insights relative to whole-exome sequencing2,3 analyses. This dataset, representing a large collection of whole-genome sequencing&nbsp;data that is available to the UK Biobank research community, will enable advances of our understanding of the human genome, facilitate the discovery of diagnostics and&nbsp;therapeutics with higher efficacy and improved safety profile, and enable precision medicine strategies with the potential to improve global health. A study reports whole-genome sequences for 490,640 participants from the UK Biobank and combines these data with phenotypic data to provide new insights into the relationship between human variation and sequence variation.},
  archive      = {J_NATURE},
  doi          = {10.1038/s41586-025-09272-9},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {692-701},
  shortjournal = {Nature},
  title        = {Whole-genome sequencing of 490,640 UK biobank participants},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Novel assembly of a head–trunk interface in the sister group of jawed vertebrates. <em>NATURE</em>, <em>645</em>(8081), 686-691. (<a href='https://doi.org/10.1038/s41586-025-09329-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The standard scenario for the origin of jawed vertebrates depicts a transition from benthic grazers to nektonic predators1–3, facilitated by a suite of anatomical innovations, including elaborate sensory systems, a high-flow heart and the integration of jaw-opening muscles with the craniothoracic hinge4–7. However, the lamprey-like internal anatomy8–13 reconstructed for osteostracans, the sister group of jawed vertebrates, seem to lack these gnathostome traits, implying a morphological gap despite phylogenetic proximity. Here, using synchrotron-based X-ray microtomography on the model osteostracan Norselaspis glacialis, we reveal derived gnathostome traits straddling a uniquely ossified head–trunk interface in this jawless fish. The inner ear of Norselaspis shows sensory elaborations (enlarged pars inferior and sinus superior) acquired well before the origin of jaws. As in crown gnathostomes, paired venous drainage channels blood into a high-volume cardiac tract. We also confirm a feature not yet demonstrated in any other vertebrate, to our knowledge: the most anterior trunk nerve&nbsp;extends its single trunk to the pectoral fin. In this respect, our reconstruction challenges the hypotheses14–16 that the gnathostome shoulder evolved from the gill apparatus. Our observations highlight Norselaspis as a prelude to the intercalation of the muscular neck and throat that would power the early jaw apparatus. Therefore, the vertebrate jaw—often considered the functional driver for ‘gnathostome’ innovations1–3—evolved instead as a follower to the sensory enhancement, increased cardiac output and greater locomotory control now inferred in the jawless sister group. Imaging of a Devonian jawless fish reveals advanced features previously thought to be&nbsp;exclusive to jawed vertebrates, challenging the idea that jaws were the primary driver for the evolution of derived traits in the&nbsp;vertebrate body plan.},
  archive      = {J_NATURE},
  author       = {Miyashita, Tetsuto and Janvier, Philippe and Tietjen, Kristen and Berenguer, Felisa and Schöder, Sebastian and Marone, Federica and Gueriau, Pierre and Coates, Michael I.},
  doi          = {10.1038/s41586-025-09329-9},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {686-691},
  shortjournal = {Nature},
  title        = {Novel assembly of a head–trunk interface in the sister group of jawed vertebrates},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flourishing chemosynthetic life at the greatest depths of hadal trenches. <em>NATURE</em>, <em>645</em>(8081), 679-685. (<a href='https://doi.org/10.1038/s41586-025-09317-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hadal trenches, some of the Earth’s least explored and understood environments, have long been proposed to harbour chemosynthesis-based communities1,2. Despite increasing attention, actual documentation of such communities has been exceptionally rare3,4. Here we report the discovery of the deepest and the most extensive chemosynthesis-based communities known to exist on Earth during an expedition to the Kuril–Kamchatka Trench and the western Aleutian Trench using the manned submersible Fendouzhe. The communities dominated by siboglinid Polychaeta and Bivalvia span a distance of 2,500 km at depths from 5,800 m to 9,533 m. These communities are sustained by hydrogen sulfide-rich and methane-rich fluids that are transported along faults traversing deep sediment layers in trenches, where methane is produced microbially from deposited organic matter, as indicated by isotopic analysis. Given geological similarities with other hadal trenches, such chemosynthesis-based communities might be more widespread than previously anticipated. These findings challenge current models of life at extreme limits and carbon cycling in the deep ocean. The discovery of chemosynthesis-based benthic communities at depths of 5,800 m to 9,533 m in the Kuril–Kamchatka and western Aleutian trenches challenges traditional perspectives on the energy sources sustaining hadal fauna.},
  archive      = {J_NATURE},
  author       = {Peng, Xiaotong and Du, Mengran and Gebruk, Andrey and Liu, Shuangquan and Gao, Zhaoming and Glud, Ronnie N. and Zhou, Peng and Wang, Ruoheng and Rowden, Ashley A. and Kamenev, Gennady M. and Maiorova, Anastassya S. and Papineau, Dominic and Chen, Shun and Gao, Jinwei and Liu, Helu and He, Yuan and Alalykina, Inna L. and Dolmatov, Igor Yu. and Zhang, Hanyu and Li, Xuegong and Malyutina, Marina V. and Dasgupta, Shamik and Saulenko, Anastasiia A. and Shilov, Vladimir A. and Liu, Shuting and Xie, Tongtong and Qu, Yuangao and Song, Xikun and Zhang, Haibin and Liu, Hao and Zhang, Weijia and Huang, Xiaoxia and Xu, Hongzhou and Xu, Wenjing and Mordukhovich, Vladimir V. and Adrianov, Andrey V.},
  doi          = {10.1038/s41586-025-09317-z},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {679-685},
  shortjournal = {Nature},
  title        = {Flourishing chemosynthetic life at the greatest depths of hadal trenches},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-range PM2.5 pollution and health impacts from the 2023 canadian wildfires. <em>NATURE</em>, <em>645</em>(8081), 672-678. (<a href='https://doi.org/10.1038/s41586-025-09482-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Smoke from extreme wildfires in Canada adversely affected air quality in many regions in 20231,2. Here we use satellite observations, machine learning and a chemical transport model to quantify global and regional PM2.5 (particulate matter less than 2.5 μm in diameter) exposure and human health impacts related to the 2023 Canadian wildfires. We find that the fires increased annual PM2.5 exposure worldwide by 0.17 μg m–3 (95% confidence interval, 0.09–0.26 μg m–3). North America had the largest increase in annual mean exposure (1.08 μg m–3; 0.82–1.34 μg m–3), but there were also increases in Europe (0.41 μg m–3; 0.32–0.50 μg m–3) owing to long-range transport. Annual mean PM2.5 exposure in Canada increased by 3.82 μg m–3 (3.00–4.64 μg m–3). In the USA, the contribution of the Canadian fires to increased PM2.5 was 1.49 μg m–3 (1.22–1.77 μg m–3), four times as large as the contribution from the 2023 wildfires originating in the USA. We find that 354 million (277–421 million) people in North America and Europe were exposed to daily PM2.5 air pollution caused by Canadian wildfires in 2023. We estimate that 5,400 (3,400–7,400) acute deaths in North America and 64,300 (37,800–90,900) chronic deaths in North America and Europe were attributable to PM2.5 exposure to the 2023 Canadian wildfires. Our results highlight the far-reaching PM2.5 pollution and health burden that large wildfires can have in a single year. A modelling study based on satellite observations, machine learning&nbsp;and a&nbsp;chemical transport model quantifies the global and regional exposure to particulate-matter pollution and the human health impacts related to the 2023 Canadian wildfires.},
  archive      = {J_NATURE},
  author       = {Zhang, Qiang and Wang, Yuexuanzi and Xiao, Qingyang and Geng, Guannan and Davis, Steven J. and Liu, Xiaodong and Yang, Jin and Liu, Jiajun and Huang, Wenyu and He, Changpei and Luo, Binhe and Martin, Randall V. and Brauer, Michael and Randerson, James T. and He, Kebin},
  doi          = {10.1038/s41586-025-09482-1},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {672-678},
  shortjournal = {Nature},
  title        = {Long-range PM2.5 pollution and health impacts from the 2023 canadian wildfires},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Two-billion-year transitional oxygenation of the earth’s surface. <em>NATURE</em>, <em>645</em>(8081), 665-671. (<a href='https://doi.org/10.1038/s41586-025-09471-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Earth’s surface underwent stepwise oxygenation before persistently reaching modern levels late in its history1–5, but the details of this transition remain unclear5–16. Here we present a high-resolution 2.5-Gyr record of mass-independent oxygen isotopes in sedimentary sulfate (Δ′17Osulfate), a proxy linked to the atmospheric partial pressure of O2 ( $${p}_{{{\rm{O}}}_{2}}$$ )17–19. This record, together with existing sedimentary Δ33S data20–22, demonstrates a 2-Gyr transition characterized by generally low, fluctuating $${p}_{{{\rm{O}}}_{2}}$$ between an O2-free state before 2.4 billion years ago (Ga) and a modern $${p}_{{{\rm{O}}}_{2}}$$ state after 0.41 Ga, with relatively elevated levels after 1.0 Ga. Our data also show coupled declines in Δ′17Osulfate and sulfate-δ34S during major negative carbonate-δ13C excursions in the Neoproterozoic. Quantitative biogeochemical modelling indicates that these isotopic couplings reflect the increasing $${p}_{{{\rm{O}}}_{2}}$$ , which may have driven episodic ocean oxygenation through an increased atmospheric O2 influx. This process intensified the oxidation of marine organics and reduced-sulfur species, while triggering temporary $${p}_{{{\rm{O}}}_{2}}$$ drawdowns as negative feedback15. These findings support a dynamic, lengthy co-oxygenation history for the atmosphere and oceans—marked by long-term positive coupling and short-term negative feedbacks—offering a coherent explanation for the anomalous Neoproterozoic carbon cycles23,24 and the protracted, episodic rise of complex life25–27. A 2.5-billion-year record of oxygen isotopes in sedimentary sulfate reveals the transitional oxygenation of the Earth’s surface and provides constraints on the dynamic, lengthy co-oxygenation of Earth’s atmosphere and oceans.},
  archive      = {J_NATURE},
  author       = {Wang, Haiyang and Li, Chao and Peng, Yongbo and Zhang, Junpeng and Cheng, Meng and Cao, Xiaobin and Qie, Wenkun and Zhang, Zihu and Dodd, Matthew S. and Hou, Mingcai and Wallace, Malcolm and v. S. Hood, Ashleigh and Lyons, Timothy W. and Bao, Huiming},
  doi          = {10.1038/s41586-025-09471-4},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {665-671},
  shortjournal = {Nature},
  title        = {Two-billion-year transitional oxygenation of the earth’s surface},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-density soft bioelectronic fibres for multimodal sensing and stimulation. <em>NATURE</em>, <em>645</em>(8081), 656-664. (<a href='https://doi.org/10.1038/s41586-025-09481-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is an increasing demand for multimodal sensing and stimulation bioelectronic fibres for both research and clinical applications1,2. However, existing fibres suffer from high rigidity, low component layout precision, limited functionality and low density of active components. These limitations arise from the challenge of integrating many components into one-dimensional fibre devices, especially owing to the incompatibility of conventional microfabrication methods (for example, photolithography) with curved, thin and long fibre structures2. As a result, limited applications have been demonstrated so far. Here we use ‘spiral transformation’ to convert two-dimensional thin films containing microfabricated devices into one-dimensional soft fibres. This approach allows for the fabrication of high-density multimodal soft bioelectronic fibres, termed Spiral-NeuroString (S-NeuroString), while enabling precise control on the longitudinal, angular and radial positioning and distribution of the functional components. Taking advantage of the biocompatibility of our soft fibres with the dynamic and soft gastrointestinal system, we proceed to show the feasibility of our S-NeuroString for post-operative multimodal continuous motility mapping and tissue stimulation in awake pigs. We further demonstrate multi-channel single-unit electrical recording in mouse brain for up to 4 months, and a fabrication capability to produce 1,280 channels within a 230-μm-diameter soft fibre. Our soft bioelectronic fibres offer a powerful platform for minimally invasive implantable electronics, where diverse sensing and stimulation functionalities can be effectively integrated. High-density multimodal soft bioelectronic fibres provide a platform for minimally invasive implantable electronics, where diverse sensing and stimulation functionalities can be effectively integrated.},
  archive      = {J_NATURE},
  author       = {Khatib, Muhammad and Zhao, Eric Tianjiao and Wei, Shiyuan and Park, Jaeho and Abramson, Alex and Bishop, Estelle Spear and Thomas, Anne-Laure and Chen, Chih-Hsin and Emengo, Pamela and Xu, Chengyi and Hamnett, Ryan and Root, Samuel E. and Yuan, Lei and Wurdack, Matthias J. and Zaluska, Tomasz and Lee, Yeongjun and Parkatzidis, Kostas and Yu, Weilai and Chakhtoura, Dorine and Kim, Kyun Kyu and Zhong, Donglai and Nishio, Yuya and Zhao, Chuanzhen and Wu, Can and Jiang, Yuanwen and Zhang, Anqi and Li, Jinxing and Wang, Weichen and Salimi-Jazi, Fereshteh and Rafeeqi, Talha A. and Hemed, Nofar Mintz and Tok, Jeffrey B.-H. and Qian, Xiang and Chen, Xiaoke and Kaltschmidt, Julia A. and Dunn, James C. Y. and Bao, Zhenan},
  doi          = {10.1038/s41586-025-09481-2},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {656-664},
  shortjournal = {Nature},
  title        = {High-density soft bioelectronic fibres for multimodal sensing and stimulation},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A movable long-term implantable soft microfibre for dynamic bioelectronics. <em>NATURE</em>, <em>645</em>(8081), 648-655. (<a href='https://doi.org/10.1038/s41586-025-09344-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-term implantable bioelectronics offer a powerful means to evaluate the function of the nervous system and serve as effective human–machine interfaces1–3. Here, inspired by earthworms, we introduce NeuroWorm—a soft, stretchable and movable fibre sensor designed for bioelectronic interface. Our approach involves rolling to transform 2D bioelectronic devices into 1D NeuroWorm, creating a multifunctional microfibre that houses longitudinally distributed electrode arrays for both bioelectrical and biomechanical monitoring. NeuroWorm effectively records high-quality spatio-temporal signals in situ while steerably advancing within the brain or on the muscle as needed. This allows for the dynamic targeting and shifting of desired monitoring sites. Implanted in muscle through a tiny incision, NeuroWorm provides stable bioelectrical monitoring in rats for more than 43 weeks. Even after 54 weeks of implantation in muscle, fibroblast encapsulation around the fibre remains negligible. Our NeuroWorm represents a platform that promotes a substantial advance in bioelectronics—from an immobile probe fixed in place to active, intelligent and living devices for long-term, minimally invasive and mobile evaluation of the nervous system. A soft, stretchable and freely movable fibre sensor called NeuroWorm, inspired by earthworms and that can be magnetically steered through soft tissue, is described as a new platform for dynamic bioelectrical and biomechanical monitoring.},
  archive      = {J_NATURE},
  author       = {Xie, Ruijie and Han, Fei and Yu, Qianhengyuan and Li, Dong and Han, Xu and Xu, Xiaolong and Yu, Huan and Huang, Jianping and Zhou, Xiaomeng and Zhao, Hang and Deng, Xinping and Tian, Qiong and Li, Qingsong and Li, Hanfei and Zhao, Yang and Ma, Guoyao and Li, Guanglin and Zheng, Hairong and Zhu, Meifang and Yan, Wei and Xu, Tiantian and Liu, Zhiyuan},
  doi          = {10.1038/s41586-025-09344-w},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {648-655},
  shortjournal = {Nature},
  title        = {A movable long-term implantable soft microfibre for dynamic bioelectronics},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Supervised learning in DNA neural networks. <em>NATURE</em>, <em>645</em>(8081), 639-647. (<a href='https://doi.org/10.1038/s41586-025-09479-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Learning enables biological organisms to begin life simple yet develop immensely diverse and complex behaviours. Understanding learning principles in engineered molecular systems could enable us to endow non-living physical systems with similar capabilities. Inspired by how the brain processes information, the principles of neural computation have been developed over the past 80 years1, forming the foundation of modern machine learning. More than four decades ago, connections between neural computation and physical systems were established2. More recently, synthetic molecular systems, including nucleic acid and protein circuits, have been investigated for their abilities to implement neural computation3–7. However, in these systems, learning of molecular parameters such as concentrations and reaction rates was performed in silico to generate desired input–output functions. Here we show that DNA molecules can be programmed to autonomously carry out supervised learning in vitro, with the system learning to perform pattern classification from molecular examples of inputs and desired responses. We demonstrate a DNA neural network trained to classify three different sets of 100-bit patterns, integrating training data directly into memories of molecular concentrations and using these memories to process subsequent test data. Our work suggests that molecular circuits can learn tasks more complex than simple adaptive behaviours. This opens the door to molecular machines capable of embedded learning and decision-making in a wide range of physical systems, from biomedicine to soft materials. DNA molecules can be programmed to autonomously carry out supervised learning in vitro, with the system learning to perform pattern classification from molecular examples of inputs and desired responses.},
  archive      = {J_NATURE},
  author       = {Cherry, Kevin M. and Qian, Lulu},
  doi          = {10.1038/s41586-025-09479-w},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {639-647},
  shortjournal = {Nature},
  title        = {Supervised learning in DNA neural networks},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DeepSeek-r1 incentivizes reasoning in LLMs through reinforcement learning. <em>NATURE</em>, <em>645</em>(8081), 633-638. (<a href='https://doi.org/10.1038/s41586-025-09422-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {General reasoning represents a long-standing and formidable challenge in artificial intelligence (AI). Recent breakthroughs, exemplified by large language models (LLMs)1,2 and chain-of-thought (CoT) prompting3, have achieved considerable success on foundational reasoning tasks. However, this success is heavily contingent on extensive human-annotated demonstrations and the capabilities of models are still insufficient for more complex problems. Here we show that the reasoning abilities of LLMs can be incentivized through pure reinforcement learning (RL), obviating the need for human-labelled reasoning trajectories. The proposed RL framework facilitates the emergent development of advanced reasoning patterns, such as self-reflection, verification and dynamic strategy adaptation. Consequently, the trained model achieves superior performance on verifiable tasks such as mathematics, coding competitions and STEM fields, surpassing its counterparts trained through conventional supervised learning on human demonstrations. Moreover, the emergent reasoning patterns exhibited by these large-scale models can be systematically used to guide and enhance the reasoning capabilities of smaller models. A new artificial intelligence model, DeepSeek-R1, is introduced, demonstrating that the reasoning abilities of large language models can be incentivized through pure reinforcement learning, removing the need for human-annotated demonstrations.},
  archive      = {J_NATURE},
  author       = {Guo, Daya and Yang, Dejian and Zhang, Haowei and Song, Junxiao and Wang, Peiyi and Zhu, Qihao and Xu, Runxin and Zhang, Ruoyu and Ma, Shirong and Bi, Xiao and Zhang, Xiaokang and Yu, Xingkai and Wu, Yu and Wu, Z. F. and Gou, Zhibin and Shao, Zhihong and Li, Zhuoshu and Gao, Ziyi and Liu, Aixin and Xue, Bing and Wang, Bingxuan and Wu, Bochao and Feng, Bei and Lu, Chengda and Zhao, Chenggang and Deng, Chengqi and Ruan, Chong and Dai, Damai and Chen, Deli and Ji, Dongjie and Li, Erhang and Lin, Fangyun and Dai, Fucong and Luo, Fuli and Hao, Guangbo and Chen, Guanting and Li, Guowei and Zhang, H. and Xu, Hanwei and Ding, Honghui and Gao, Huazuo and Qu, Hui and Li, Hui and Guo, Jianzhong and Li, Jiashi and Chen, Jingchang and Yuan, Jingyang and Tu, Jinhao and Qiu, Junjie and Li, Junlong and Cai, J. L. and Ni, Jiaqi and Liang, Jian and Chen, Jin and Dong, Kai and Hu, Kai and You, Kaichao and Gao, Kaige and Guan, Kang and Huang, Kexin and Yu, Kuai and Wang, Lean and Zhang, Lecong and Zhao, Liang and Wang, Litong and Zhang, Liyue and Xu, Lei and Xia, Leyi and Zhang, Mingchuan and Zhang, Minghua and Tang, Minghui and Zhou, Mingxu and Li, Meng and Wang, Miaojun and Li, Mingming and Tian, Ning and Huang, Panpan and Zhang, Peng and Wang, Qiancheng and Chen, Qinyu and Du, Qiushi and Ge, Ruiqi and Zhang, Ruisong and Pan, Ruizhe and Wang, Runji and Chen, R. J. and Jin, R. L. and Chen, Ruyi and Lu, Shanghao and Zhou, Shangyan and Chen, Shanhuang and Ye, Shengfeng and Wang, Shiyu and Yu, Shuiping and Zhou, Shunfeng and Pan, Shuting and Li, S. S. and Zhou, Shuang and Wu, Shaoqing and Yun, Tao and Pei, Tian and Sun, Tianyu and Wang, T. and Zeng, Wangding and Liu, Wen and Liang, Wenfeng and Gao, Wenjun and Yu, Wenqin and Zhang, Wentao and Xiao, W. L. and An, Wei and Liu, Xiaodong and Wang, Xiaohan and Chen, Xiaokang and Nie, Xiaotao and Cheng, Xin and Liu, Xin and Xie, Xin and Liu, Xingchao and Yang, Xinyu and Li, Xinyuan and Su, Xuecheng and Lin, Xuheng and Li, X. Q. and Jin, Xiangyue and Shen, Xiaojin and Chen, Xiaosha and Sun, Xiaowen and Wang, Xiaoxiang and Song, Xinnan and Zhou, Xinyi and Wang, Xianzu and Shan, Xinxia and Li, Y. K. and Wang, Y. Q. and Wei, Y. X. and Zhang, Yang and Xu, Yanhong and Li, Yao and Zhao, Yao and Sun, Yaofeng and Wang, Yaohui and Yu, Yi and Zhang, Yichao and Shi, Yifan and Xiong, Yiliang and He, Ying and Piao, Yishi and Wang, Yisong and Tan, Yixuan and Ma, Yiyang and Liu, Yiyuan and Guo, Yongqiang and Ou, Yuan and Wang, Yuduan and Gong, Yue and Zou, Yuheng and He, Yujia and Xiong, Yunfan and Luo, Yuxiang and You, Yuxiang and Liu, Yuxuan and Zhou, Yuyang and Zhu, Y. X. and Huang, Yanping and Li, Yaohui and Zheng, Yi and Zhu, Yuchen and Ma, Yunxian and Tang, Ying and Zha, Yukun and Yan, Yuting and Ren, Z. Z. and Ren, Zehui and Sha, Zhangli and Fu, Zhe and Xu, Zhean and Xie, Zhenda and Zhang, Zhengyan and Hao, Zhewen and Ma, Zhicheng and Yan, Zhigang and Wu, Zhiyu and Gu, Zihui and Zhu, Zijia and Liu, Zijun and Li, Zilin and Xie, Ziwei and Song, Ziyang and Pan, Zizheng and Huang, Zhen and Xu, Zhipeng and Zhang, Zhongyu and Zhang, Zhen},
  doi          = {10.1038/s41586-025-09422-z},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {633-638},
  shortjournal = {Nature},
  title        = {DeepSeek-r1 incentivizes reasoning in LLMs through reinforcement learning},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Topological prethermal strong zero modes on superconducting processors. <em>NATURE</em>, <em>645</em>(8081), 626-632. (<a href='https://doi.org/10.1038/s41586-025-09476-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Symmetry-protected topological phases1–4 cannot be described by any local order parameter and are beyond the conventional symmetry-breaking model5. They are characterized by topological boundary modes that remain stable under symmetry respecting perturbations1–4,6–8. In clean, gapped systems without disorder, the stability of these edge modes is restricted to the zero-temperature manifold; at finite temperatures, interactions with mobile thermal excitations lead to their decay9–11. Here we report the observation of a distinct type of topological edge mode12–14, which is protected by emergent symmetries and persists across the entire spectrum, in an array of 100 programmable superconducting qubits. Through digital quantum simulation of a one-dimensional disorder-free stabilizer Hamiltonian, we observe robust long-lived topological edge modes over up to 30 cycles for a wide range of initial states. We show that the interaction between these edge modes and bulk excitations can be suppressed by dimerizing the stabilizer strength, leading to an emergent U(1) × U(1) symmetry in the prethermal regime of the system. Furthermore, we exploit these topological edge modes as logical qubits and prepare a logical Bell state, which exhibits persistent coherence, despite the system being disorder-free and at finite temperature. Our results establish a viable digital simulation approach15–18 to experimentally study topological matter at finite temperature and demonstrate a potential route to construct long-lived, robust boundary qubits in disorder-free systems. Stable and robust topological edge modes are observed at finite temperatures in an array of 100 programmable superconducting qubits because of emergent symmetries present in the prethermal regime of this system.},
  archive      = {J_NATURE},
  author       = {Jin, Feitong and Jiang, Si and Zhu, Xuhao and Bao, Zehang and Shen, Fanhao and Wang, Ke and Zhu, Zitian and Xu, Shibo and Song, Zixuan and Chen, Jiachen and Tan, Ziqi and Wu, Yaozu and Zhang, Chuanyu and Gao, Yu and Wang, Ning and Zou, Yiren and Zhang, Aosai and Li, Tingting and Zhong, Jiarun and Cui, Zhengyi and Han, Yihang and He, Yiyang and Wang, Han and Yang, Jia-Nan and Wang, Yanzhe and Shen, Jiayuan and Liu, Gongyu and Deng, Jinfeng and Dong, Hang and Zhang, Pengfei and Li, Weikang and Yuan, Dong and Lu, Zhide and Sun, Zheng-Zhi and Li, Hekang and Zhang, Junxiang and Song, Chao and Wang, Zhen and Guo, Qiujiang and Machado, Francisco and Kemp, Jack and Iadecola, Thomas and Yao, Norman Y. and Wang, H. and Deng, Dong-Ling},
  doi          = {10.1038/s41586-025-09476-z},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {626-632},
  shortjournal = {Nature},
  title        = {Topological prethermal strong zero modes on superconducting processors},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Experimental demonstration of logical magic state distillation. <em>NATURE</em>, <em>645</em>(8081), 620-625. (<a href='https://doi.org/10.1038/s41586-025-09367-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Realizing universal fault-tolerant quantum computation is a key goal in quantum information science1–4. By encoding quantum information into logical qubits using quantum error correcting codes, physical errors can be detected and corrected, enabling a substantial reduction in logical error rates5–11. However, the set of logical operations that can be easily implemented on these encoded qubits is often constrained1,12, necessitating the use of special resource states known as ‘magic states’13 to implement universal, classically hard circuits14. A key method to prepare high-fidelity magic states is to perform ‘distillation’, creating them from multiple lower-fidelity inputs13,15. Here we present the experimental realization of magic state distillation with logical qubits on a neutral-atom quantum computer. Our approach uses a dynamically reconfigurable architecture8,16 to encode and perform quantum operations on many logical qubits in parallel. We demonstrate the distillation of magic states encoded in d = 3 and d = 5 colour codes, observing improvements in the logical fidelity of the output magic states compared with the input logical magic states. These experiments demonstrate a key building block of universal fault-tolerant quantum computation and represent an important step towards large-scale logical quantum processors. Magic state distillation is achieved with logical qubits on a neutral-atom quantum computer using a dynamically reconfigurable architecture for parallel quantum operations.},
  archive      = {J_NATURE},
  author       = {Sales Rodriguez, Pedro and Robinson, John M. and Jepsen, Paul Niklas and He, Zhiyang and Duckering, Casey and Zhao, Chen and Wu, Kai-Hsin and Campo, Joseph and Bagnall, Kevin and Kwon, Minho and Karolyshyn, Thomas and Weinberg, Phillip and Cain, Madelyn and Evered, Simon J. and Geim, Alexandra A. and Kalinowski, Marcin and Li, Sophie H. and Manovitz, Tom and Amato-Grill, Jesse and Basham, James I. and Bernstein, Liane and Braverman, Boris and Bylinskii, Alexei and Choukri, Adam and DeAngelo, Robert J. and Fang, Fang and Fieweger, Connor and Frederick, Paige and Haines, David and Hamdan, Majd and Hammett, Julian and Hsu, Ning and Hu, Ming-Guang and Huber, Florian and Jia, Ningyuan and Kedar, Dhruv and Kornjača, Milan and Liu, Fangli and Long, John and Lopatin, Jonathan and Lopes, Pedro L. S. and Luo, Xiu-Zhe and Macrì, Tommaso and Marković, Ognjen and Martínez-Martínez, Luis A. and Meng, Xianmei and Ostermann, Stefan and Ostroumov, Evgeny and Paquette, David and Qiang, Zexuan and Shofman, Vadim and Singh, Anshuman and Singh, Manuj and Sinha, Nandan and Thoreen, Henry and Wan, Noel and Wang, Yiping and Waxman-Lenz, Daniel and Wong, Tak and Wurtz, Jonathan and Zhdanov, Andrii and Zheng, Laurent and Greiner, Markus and Keesling, Alexander and Gemelke, Nathan and Vuletić, Vladan and Kitagawa, Takuya and Wang, Sheng-Tao and Bluvstein, Dolev and Lukin, Mikhail D. and Lukin, Alexander and Zhou, Hengyun and Cantú, Sergio H.},
  doi          = {10.1038/s41586-025-09367-3},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {620-625},
  shortjournal = {Nature},
  title        = {Experimental demonstration of logical magic state distillation},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scaling and logic in the colour code on a superconducting quantum processor. <em>NATURE</em>, <em>645</em>(8081), 614-619. (<a href='https://doi.org/10.1038/s41586-025-09061-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum error correction1–4 is essential for bridging the gap between the error rates of physical devices and the extremely low error rates required for quantum algorithms. Recent error-correction demonstrations on superconducting processors5–8 have focused primarily on the surface code9, which offers a high error threshold but poses limitations for logical operations. The colour code10 enables more efficient logic, but it requires more complex stabilizer measurements and decoding. Measuring these stabilizers in planar architectures such as superconducting qubits is challenging, and realizations of colour codes11–19 have not addressed performance scaling with code size on any platform. Here we present a comprehensive demonstration of the colour code on a superconducting processor8. Scaling the code distance from three to five suppresses logical errors by a factor of Λ3/5 = 1.56(4). Simulations indicate this performance is below the threshold of the colour code, and the colour code may become more efficient than the surface code following modest device improvements. We test transversal Clifford gates with logical randomized benchmarking20 and inject magic states21, a key resource for universal computation, achieving fidelities exceeding 99% with post-selection. Finally, we teleport logical states between colour codes using lattice surgery22. This work establishes the colour code as a compelling research direction to realize fault-tolerant quantum computation on superconducting processors in the near future. Colour code on a superconducting qubit quantum processor is demonstrated, reporting above-breakeven performance and logical error scaling with increased code size by a factor of 1.56 moving from distance-3 to distance-5 code.},
  archive      = {J_NATURE},
  author       = {Lacroix, N. and Bourassa, A. and Heras, F. J. H. and Zhang, L. M. and Bausch, J. and Senior, A. W. and Edlich, T. and Shutty, N. and Sivak, V. and Bengtsson, A. and McEwen, M. and Higgott, O. and Kafri, D. and Claes, J. and Morvan, A. and Chen, Z. and Zalcman, A. and Madhuk, S. and Acharya, R. and Aghababaie Beni, L. and Aigeldinger, G. and Alcaraz, R. and Andersen, T. I. and Ansmann, M. and Arute, F. and Arya, K. and Asfaw, A. and Atalaya, J. and Babbush, R. and Ballard, B. and Bardin, J. C. and Bilmes, A. and Blackwell, S. and Bovaird, J. and Bowers, D. and Brill, L. and Broughton, M. and Browne, D. A. and Buchea, B. and Buckley, B. B. and Burger, T. and Burkett, B. and Bushnell, N. and Cabrera, A. and Campero, J. and Chang, H.-S. and Chiaro, B. and Chih, L.-Y. and Cleland, A. Y. and Cogan, J. and Collins, R. and Conner, P. and Courtney, W. and Crook, A. L. and Curtin, B. and Das, S. and Demura, S. and De Lorenzo, L. and Di Paolo, A. and Donohoe, P. and Drozdov, I. and Dunsworth, A. and Eickbusch, A. and Elbag, A. Moshe and Elzouka, M. and Erickson, C. and Ferreira, V. S. and Flores Burgos, L. and Forati, E. and Fowler, A. G. and Foxen, B. and Ganjam, S. and Garcia, G. and Gasca, R. and Genois, É. and Giang, W. and Gilboa, D. and Gosula, R. and Grajales Dau, A. and Graumann, D. and Greene, A. and Gross, J. A. and Ha, T. and Habegger, S. and Hansen, M. and Harrigan, M. P. and Harrington, S. D. and Heslin, S. and Heu, P. and Hiltermann, R. and Hilton, J. and Hong, S. and Huang, H.-Y. and Huff, A. and Huggins, W. J. and Jeffrey, E. and Jiang, Z. and Jin, X. and Joshi, C. and Juhas, P. and Kabel, A. and Kang, H. and Karamlou, A. H. and Kechedzhi, K. and Khaire, T. and Khattar, T. and Khezri, M. and Kim, S. and Klimov, P. V. and Kobrin, B. and Korotkov, A. N. and Kostritsa, F. and Kreikebaum, J. Mark and Kurilovich, V. D. and Landhuis, D. and Lange-Dei, T. and Langley, B. W. and Laptev, P. and Lau, K.-M. and Ledford, J. and Lee, K. and Lester, B. J. and Le Guevel, L. and Li, W. Yan and Li, Y. and Lill, A. T. and Livingston, W. P. and Locharla, A. and Lucero, E. and Lundahl, D. and Lunt, A. and Maloney, A. and Mandrà, S. and Martin, L. S. and Martin, O. and Maxfield, C. and McClean, J. R. and Meeks, S. and Megrant, A. and Miao, K. C. and Molavi, R. and Molina, S. and Montazeri, S. and Movassagh, R. and Neill, C. and Newman, M. and Nguyen, A. and Nguyen, M. and Ni, C.-H. and Niu, M. Y. and Oas, L. and Oliver, W. D. and Orosco, R. and Ottosson, K. and Pizzuto, A. and Potter, R. and Pritchard, O. and Quintana, C. and Ramachandran, G. and Reagor, M. J. and Resnick, R. and Rhodes, D. M. and Roberts, G. and Rosenberg, E. and Rosenfeld, E. and Rossi, E. and Roushan, P. and Sankaragomathi, K. and Schurkus, H. F. and Shearn, M. J. and Shorter, A. and Shvarts, V. and Small, S. and Smith, W. Clarke and Springer, S. and Sterling, G. and Suchard, J. and Szasz, A. and Sztein, A. and Thor, D. and Tomita, E. and Torres, A. and Torunbalci, M. Mert and Vaishnav, A. and Vargas, J. and Vdovichev, S. and Vidal, G. and Vollgraff Heidweiller, C. and Waltman, S. and Waltz, J. and Wang, S. X. and Ware, B. and Weidel, T. and White, T. and Wong, K. and Woo, B. W. K. and Woodson, M. and Xing, C. and Yao, Z. Jamie and Yeh, P. and Ying, B. and Yoo, J. and Yosri, N. and Young, G. and Zhang, Y. and Zhu, N. and Zobrist, N. and Neven, H. and Kohli, P. and Davies, A. and Boixo, S. and Kelly, J. and Jones, C. and Gidney, C. and Satzinger, K. J.},
  doi          = {10.1038/s41586-025-09061-4},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {614-619},
  shortjournal = {Nature},
  title        = {Scaling and logic in the colour code on a superconducting quantum processor},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Addressing the safety of next-generation batteries. <em>NATURE</em>, <em>645</em>(8081), 603-613. (<a href='https://doi.org/10.1038/s41586-025-09358-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Owing to increasing demand for low-cost energy storage with secure material&nbsp;supply chains, the battery community is approaching a pivotal shift beyond&nbsp;conventional lithium-ion (Li-ion) towards next-generation cells. Technologies that&nbsp;include alkali-metal anodes, solid electrolytes and earth-abundant materials&nbsp;such as&nbsp;sodium (Na) and sulfur (S) are reaching commercialization in cells. The abuse tolerance and thermal runaway hazards of such technologies diverge from conventional Li-ion cells. Consequently, designing safe batteries with next-generation materials requires a holistic approach to characterize cells and to understand their responses to abuse conditions from the beginning to the end of life. Here we provide a Perspective on how the safety and abuse tolerance of cells are likely to change for up-and-coming technologies; challenges and opportunities for reimagining safe cell and battery designs; gaps in our knowledge; capabilities for understanding the hazards of thermal runaway and how to address them; how standard&nbsp;abuse tests may need to adapt to new challenges; and how research needs to support affected professionals, from pack designers to first responders, to manage hazards and ensure safe roll-out of next-generation cells into&nbsp;applications like electric vehicles (EVs). Finally, given the large number of next-generation technologies being explored, we encourage giving priority to safety-focused research in proportion to the rate of manufacturing scale-up of each specific technology. Next-generation batteries&nbsp;will present different risks to&nbsp;conventional lithium-ion cells, emphasizing the need for efforts towards characterizing&nbsp;the abuse tolerance and hazards associated with&nbsp;next-generation battery materials over their life cycle and providing the data in the context of supporting affected professionals.},
  archive      = {J_NATURE},
  author       = {Yang, Chuanbo and Singh, Avtar and Pu, Xiaofei and Mallarapu, Anudeep and Smith, Kandler and Keyser, Matt and Haberman, Michael R. and Khani, Hadi and Misztal, Pawel and Spray, Ryan and Ezekoye, Ofodike A. and Finegan, Donal P.},
  doi          = {10.1038/s41586-025-09358-4},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {603-613},
  shortjournal = {Nature},
  title        = {Addressing the safety of next-generation batteries},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bioelectronic implants built from rolled-up stretchy circuits. <em>NATURE</em>, <em>645</em>(8081), 600-601. (<a href='https://doi.org/10.1038/d41586-025-02704-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soft devices made by rolling up rubber-like sheets patterned with 2D electronics can monitor, stimulate and navigate deep tissue in animal models. Soft devices made by rolling up rubber-like sheets patterned with 2D electronics can monitor, stimulate and navigate deep tissue in animal models.},
  archive      = {J_NATURE},
  author       = {Lee, Hyunjin and Kim, Dae-Hyeong},
  doi          = {10.1038/d41586-025-02704-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {600-601},
  shortjournal = {Nature},
  title        = {Bioelectronic implants built from rolled-up stretchy circuits},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Epigenetic clues from cancer’s past foretell its future. <em>NATURE</em>, <em>645</em>(8081), 598-600. (<a href='https://doi.org/10.1038/d41586-025-02256-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An innovative method to trace tumour evolution using patterns of fluctuating DNA modifications could be implemented in the clinic to predict cancer progression. An innovative method to trace tumour evolution using patterns of fluctuating DNA modifications could be implemented in the clinic to predict cancer progression.},
  archive      = {J_NATURE},
  author       = {Lutsik, Pavlo and Manojlovic, Veselin and Vassiliou, George S.},
  doi          = {10.1038/d41586-025-02256-9},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {598-600},
  shortjournal = {Nature},
  title        = {Epigenetic clues from cancer’s past foretell its future},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gone with the wind: Deciphering how dandelions drive seed dispersal. <em>NATURE</em>, <em>645</em>(8081), 597. (<a href='https://doi.org/10.1038/d41586-025-02949-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of seed attachment and the forces needed for detachment reveal how dandelion plants have evolved to spread in the same direction that the wind blows. Analysis of seed attachment and the forces needed for detachment reveal how dandelion plants have evolved to spread in the same direction that the wind blows.},
  archive      = {J_NATURE},
  author       = {Abraham, Mary},
  doi          = {10.1038/d41586-025-02949-1},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {597},
  shortjournal = {Nature},
  title        = {Gone with the wind: Deciphering how dandelions drive seed dispersal},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High levels of circular DNA made as immune cells develop increases the risk of leukaemia relapse. <em>NATURE</em>, <em>645</em>(8081), 596-598. (<a href='https://doi.org/10.1038/d41586-025-02421-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA segments that are naturally excised during immune-cell development can replicate, drive damage at various genomic sites and lead to cancer recurrence. DNA segments that are naturally excised during immune-cell development can replicate, drive damage at various genomic sites and lead to cancer recurrence.},
  archive      = {J_NATURE},
  author       = {Rodgers, Karla K.},
  doi          = {10.1038/d41586-025-02421-0},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {596-598},
  shortjournal = {Nature},
  title        = {High levels of circular DNA made as immune cells develop increases the risk of leukaemia relapse},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI can learn to show its workings through trial and error. <em>NATURE</em>, <em>645</em>(8081), 594-595. (<a href='https://doi.org/10.1038/d41586-025-02703-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) are more accurate when they output intermediate steps. A strategy called reinforcement can teach them to do this without being told. Large language models (LLMs) are more accurate when they output intermediate steps. A strategy called reinforcement can teach them to do this without being told.},
  archive      = {J_NATURE},
  author       = {Ippolito, Daphne and Zhang, Yiming},
  doi          = {10.1038/d41586-025-02703-7},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {594-595},
  shortjournal = {Nature},
  title        = {AI can learn to show its workings through trial and error},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Does lithium deficiency contribute to alzheimer’s disease?. <em>NATURE</em>, <em>645</em>(8081), 593-594. (<a href='https://doi.org/10.1038/d41586-025-02255-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lithium in the brain has been found to protect against cognitive decline. Restoring lost lithium could be a new angle from which to tackle Alzheimer’s disease. Lithium in the brain has been found to protect against cognitive decline. Restoring lost lithium could be a new angle from which to tackle Alzheimer’s disease.},
  archive      = {J_NATURE},
  author       = {Bush, Ashley I.},
  doi          = {10.1038/d41586-025-02255-w},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {593-594},
  shortjournal = {Nature},
  title        = {Does lithium deficiency contribute to alzheimer’s disease?},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Impoundment of funds endangers US investment in science and medical research. <em>NATURE</em>, <em>645</em>(8081), 592. (<a href='https://doi.org/10.1038/d41586-025-03003-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Weitz, Joshua S. and Harris, Mallory J. and Sinclair, Alyssa H. and Berg, Jeremy M.},
  doi          = {10.1038/d41586-025-03003-w},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {592},
  shortjournal = {Nature},
  title        = {Impoundment of funds endangers US investment in science and medical research},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Coordinate to combat pakistan’s climate-driven disasters. <em>NATURE</em>, <em>645</em>(8081), 592. (<a href='https://doi.org/10.1038/d41586-025-03004-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Khan, Raees and Liu, Jie},
  doi          = {10.1038/d41586-025-03004-9},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {592},
  shortjournal = {Nature},
  title        = {Coordinate to combat pakistan’s climate-driven disasters},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fertility declines are no cause for concern, history shows. <em>NATURE</em>, <em>645</em>(8081), 592. (<a href='https://doi.org/10.1038/d41586-025-03005-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {MacInnes, John},
  doi          = {10.1038/d41586-025-03005-8},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {592},
  shortjournal = {Nature},
  title        = {Fertility declines are no cause for concern, history shows},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A population plunge could help to mitigate the global biodiversity crisis. <em>NATURE</em>, <em>645</em>(8081), 592. (<a href='https://doi.org/10.1038/d41586-025-03006-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Hayward, Matt W. and Williams, Brooke and McBain, Bonnie},
  doi          = {10.1038/d41586-025-03006-7},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {592},
  shortjournal = {Nature},
  title        = {A population plunge could help to mitigate the global biodiversity crisis},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mirror of the unknown: Should research on mirror-image molecular biology be stopped?. <em>NATURE</em>, <em>645</em>(8081), 588-591. (<a href='https://doi.org/10.1038/d41586-025-02912-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amid growing debates about the benefits and risks of studying looking-glass versions of life’s building blocks, there is an urgent need to bridge divergent views. Amid growing debates about the benefits and risks of studying looking-glass versions of life’s building blocks, there is an urgent need to bridge divergent views.},
  archive      = {J_NATURE},
  author       = {Zhu, Ting},
  doi          = {10.1038/d41586-025-02912-0},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {588-591},
  shortjournal = {Nature},
  title        = {Mirror of the unknown: Should research on mirror-image molecular biology be stopped?},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). GPS timekeeping is increasingly vulnerable: Here’s how to deliver future-proofed time. <em>NATURE</em>, <em>645</em>(8081), 585-588. (<a href='https://doi.org/10.1038/d41586-025-02921-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With crucial infrastructure increasingly dependent on accurate, resilient clocks, outages in the dominant system for global positioning highlight the need for other ways of distributing time information. With crucial infrastructure increasingly dependent on accurate, resilient clocks, outages in the dominant system for global positioning highlight the need for other ways of distributing time information.},
  archive      = {J_NATURE},
  author       = {Lobo, Leon and Paul, Douglas and Velu, Chander},
  doi          = {10.1038/d41586-025-02921-z},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {585-588},
  shortjournal = {Nature},
  title        = {GPS timekeeping is increasingly vulnerable: Here’s how to deliver future-proofed time},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How did assaults on science become the norm — And what can we do?. <em>NATURE</em>, <em>645</em>(8081), 583-584. (<a href='https://doi.org/10.1038/d41586-025-02920-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An insightful book explores attacks on science from a historical and a personal perspective. An insightful book explores attacks on science from a historical and a personal perspective.},
  archive      = {J_NATURE},
  author       = {Goldman, Gretchen T.},
  doi          = {10.1038/d41586-025-02920-0},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {583-584},
  shortjournal = {Nature},
  title        = {How did assaults on science become the norm — And what can we do?},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Heroes or hoarders? the strange brains of people who collect. <em>NATURE</em>, <em>645</em>(8081), 581-582. (<a href='https://doi.org/10.1038/d41586-025-02919-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Public museums and private collections are stuffed with scientific artefacts gathered by individuals — often with highly questionable obsessions and motivations. Public museums and private collections are stuffed with scientific artefacts gathered by individuals — often with highly questionable obsessions and motivations.},
  archive      = {J_NATURE},
  author       = {Robinson, Andrew},
  doi          = {10.1038/d41586-025-02919-7},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {581-582},
  shortjournal = {Nature},
  title        = {Heroes or hoarders? the strange brains of people who collect},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How billions of hacked mosquitoes and a vaccine could beat the deadly dengue virus. <em>NATURE</em>, <em>645</em>(8081), 578-580. (<a href='https://doi.org/10.1038/d41586-025-02918-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Outbreaks of dengue are killing thousands of people in South America each year and getting worse. Brazil hopes to turn the tide with a home-grown vaccine and an army of mosquitoes infected with Wolbachia bacteria. Outbreaks of dengue are killing thousands of people in South America each year and getting worse. Brazil hopes to turn the tide with a home-grown vaccine and an army of mosquitoes infected with Wolbachia bacteria.},
  archive      = {J_NATURE},
  author       = {Pinto, Lucila},
  doi          = {10.1038/d41586-025-02918-8},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {578-580},
  shortjournal = {Nature},
  title        = {How billions of hacked mosquitoes and a vaccine could beat the deadly dengue virus},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI is helping to decode animals’ speech. will it also let us talk with them?. <em>NATURE</em>, <em>645</em>(8081), 574-576. (<a href='https://doi.org/10.1038/d41586-025-02917-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The complexity of vocal communication in some primates, whales and birds might approach that of human language. The complexity of vocal communication in some primates, whales and birds might approach that of human language.},
  archive      = {J_NATURE},
  author       = {Fieldhouse, Rachel},
  doi          = {10.1038/d41586-025-02917-9},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {574-576},
  shortjournal = {Nature},
  title        = {AI is helping to decode animals’ speech. will it also let us talk with them?},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ‘Brain dial’ turns food consumption on or off in mice. <em>NATURE</em>, <em>645</em>(8081), 573. (<a href='https://doi.org/10.1038/d41586-025-02871-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Master-control area integrates information about hunger, a food’s tastiness and more — and can even drive intake of plastic pellets. Master-control area integrates information about hunger, a food’s tastiness and more — and can even drive intake of plastic pellets.},
  archive      = {J_NATURE},
  author       = {Heidt, Amanda},
  doi          = {10.1038/d41586-025-02871-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {573},
  shortjournal = {Nature},
  title        = {‘Brain dial’ turns food consumption on or off in mice},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Newfound immune cell in mice hints at why inflammation spikes with old age. <em>NATURE</em>, <em>645</em>(8081), 572-573. (<a href='https://doi.org/10.1038/d41586-025-02788-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pathogen-consuming cells found in fat tissue also play a part in lipid balance. Pathogen-consuming cells found in fat tissue also play a part in lipid balance.},
  archive      = {J_NATURE},
  author       = {Conroy, Gemma},
  doi          = {10.1038/d41586-025-02788-0},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {572-573},
  shortjournal = {Nature},
  title        = {Newfound immune cell in mice hints at why inflammation spikes with old age},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ‘Amazing feat’: US man still alive six months after pig kidney transplant. <em>NATURE</em>, <em>645</em>(8081), 571-572. (<a href='https://doi.org/10.1038/d41586-025-02851-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first six months after an organ transplant are the riskiest for recipients. The first six months after an organ transplant are the riskiest for recipients.},
  archive      = {J_NATURE},
  author       = {Fieldhouse, Rachel},
  doi          = {10.1038/d41586-025-02851-w},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {571-572},
  shortjournal = {Nature},
  title        = {‘Amazing feat’: US man still alive six months after pig kidney transplant},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chemists cram record nine metals into trendy 2D material. <em>NATURE</em>, <em>645</em>(8081), 570-571. (<a href='https://doi.org/10.1038/d41586-025-02839-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MXenes, distant cousins of graphene, reach new heights of complexity — opening the way for use in advanced batteries and more. MXenes, distant cousins of graphene, reach new heights of complexity — opening the way for use in advanced batteries and more.},
  archive      = {J_NATURE},
  author       = {Bourzac, Katherine},
  doi          = {10.1038/d41586-025-02839-6},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {570-571},
  shortjournal = {Nature},
  title        = {Chemists cram record nine metals into trendy 2D material},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Can researchers stop AI making up citations?. <em>NATURE</em>, <em>645</em>(8081), 569-570. (<a href='https://doi.org/10.1038/d41586-025-02853-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {OpenAI’s GPT-5 hallucinates less than previous models do, but cutting hallucination completely might prove impossible. OpenAI’s GPT-5 hallucinates less than previous models do, but cutting hallucination completely might prove impossible.},
  archive      = {J_NATURE},
  author       = {Gibney, Elizabeth},
  doi          = {10.1038/d41586-025-02853-8},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {569-570},
  shortjournal = {Nature},
  title        = {Can researchers stop AI making up citations?},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). These nations are wooing PhD students amid US funding uncertainties. <em>NATURE</em>, <em>645</em>(8081), 568-569. (<a href='https://doi.org/10.1038/d41586-025-02416-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pauses to some US graduate programmes have led universities in several countries to launch initiatives to entice would-be PhD students. Pauses to some US graduate programmes have led universities in several countries to launch initiatives to entice would-be PhD students.},
  archive      = {J_NATURE},
  author       = {Fieldhouse, Rachel},
  doi          = {10.1038/d41586-025-02416-x},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {568-569},
  shortjournal = {Nature},
  title        = {These nations are wooing PhD students amid US funding uncertainties},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Air pollution directly linked to increased dementia risk. <em>NATURE</em>, <em>645</em>(8081), 566-567. (<a href='https://doi.org/10.1038/d41586-025-02844-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-term exposure accelerates the development of Lewy body dementia and Parkinson’s disease with dementia in people who are predisposed to the conditions. Long-term exposure accelerates the development of Lewy body dementia and Parkinson’s disease with dementia in people who are predisposed to the conditions.},
  archive      = {J_NATURE},
  author       = {Fieldhouse, Rachel},
  doi          = {10.1038/d41586-025-02844-9},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {566-567},
  shortjournal = {Nature},
  title        = {Air pollution directly linked to increased dementia risk},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). First CRISPR horses spark controversy: What’s next for gene-edited animals?. <em>NATURE</em>, <em>645</em>(8081), 565-566. (<a href='https://doi.org/10.1038/d41586-025-02800-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Horses with genomic edits to make them run faster have been banned from polo, but a zoo of CRISPR-edited animals is gaining acceptance in agriculture. Horses with genomic edits to make them run faster have been banned from polo, but a zoo of CRISPR-edited animals is gaining acceptance in agriculture.},
  archive      = {J_NATURE},
  author       = {Kavanagh, Katie},
  doi          = {10.1038/d41586-025-02800-7},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {565-566},
  shortjournal = {Nature},
  title        = {First CRISPR horses spark controversy: What’s next for gene-edited animals?},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How to make a ‘laser’ of neutrinos. <em>NATURE</em>, <em>645</em>(8081), 563. (<a href='https://doi.org/10.1038/d41586-025-02840-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rubidium atoms that are chilled to extremely low temperatures could emit bursts of particles called neutrinos. Rubidium atoms that are chilled to extremely low temperatures could emit bursts of particles called neutrinos.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02840-z},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {563},
  shortjournal = {Nature},
  title        = {How to make a ‘laser’ of neutrinos},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How to help refugees thrive: Have local families host them. <em>NATURE</em>, <em>645</em>(8081), 563. (<a href='https://doi.org/10.1038/d41586-025-02873-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ukrainian refugees who moved into German households report greater social integration than those who lived in other settings. Ukrainian refugees who moved into German households report greater social integration than those who lived in other settings.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02873-4},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {563},
  shortjournal = {Nature},
  title        = {How to help refugees thrive: Have local families host them},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Songs of the striped mouse show who’s friend and who’s foe. <em>NATURE</em>, <em>645</em>(8081), 563. (<a href='https://doi.org/10.1038/d41586-025-02874-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultrasonic calls, too high for the human ear to detect, convey a wealth of information across the rodent’s territory. Ultrasonic calls, too high for the human ear to detect, convey a wealth of information across the rodent’s territory.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02874-3},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {563},
  shortjournal = {Nature},
  title        = {Songs of the striped mouse show who’s friend and who’s foe},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Jelly-filled garment keeps wearers cool when heat and humidity soar. <em>NATURE</em>, <em>645</em>(8081), 563. (<a href='https://doi.org/10.1038/d41586-025-02875-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lightweight, battery-powered shirt could maintain skin at a comfortable temperature for hours. Lightweight, battery-powered shirt could maintain skin at a comfortable temperature for hours.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02875-2},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {563},
  shortjournal = {Nature},
  title        = {Jelly-filled garment keeps wearers cool when heat and humidity soar},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Equity in science is a beautiful lie — And i’m done pretending. <em>NATURE</em>, <em>645</em>(8081), 561. (<a href='https://doi.org/10.1038/d41586-025-02916-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Science isn’t really moving towards equity; institutions are just perfecting the appearance of equity. We need to build an alternative system. Science isn’t really moving towards equity; institutions are just perfecting the appearance of equity. We need to build an alternative system.},
  archive      = {J_NATURE},
  author       = {Armenteras, Dolors},
  doi          = {10.1038/d41586-025-02916-w},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {561},
  shortjournal = {Nature},
  title        = {Equity in science is a beautiful lie — And i’m done pretending},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Make trains great again — For the sake of people and the planet. <em>NATURE</em>, <em>645</em>(8081), 560. (<a href='https://doi.org/10.1038/d41586-025-02980-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As railways enter their third century of service, research must support their renaissance for more-sustainable travel that supports human development. As railways enter their third century of service, research must support their renaissance for more-sustainable travel that supports human development.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02980-2},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {560},
  shortjournal = {Nature},
  title        = {Make trains great again — For the sake of people and the planet},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bring us your LLMs: Why peer review is good for AI models. <em>NATURE</em>, <em>645</em>(8081), 559. (<a href='https://doi.org/10.1038/d41586-025-02979-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deepseek’s R1 model has been peer reviewed. Others should follow the firm’s example. Deepseek’s R1 model has been peer reviewed. Others should follow the firm’s example.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-02979-9},
  journal      = {Nature},
  month        = {9},
  number       = {8081},
  pages        = {559},
  shortjournal = {Nature},
  title        = {Bring us your LLMs: Why peer review is good for AI models},
  volume       = {645},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nbt">NBT - 34</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(9), 1580. (<a href='https://doi.org/10.1038/s41587-025-02804-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02804-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry. <em>NBT</em>, <em>43</em>(9), 1577-1579. (<a href='https://doi.org/10.1038/s41587-025-02803-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breaking the cycle of institutional betrayal and the culture of silence requires decisive actions to promote transparency, accountability and supportive environments in both academia and industry.},
  archive      = {J_NBT},
  author       = {Vogelaar, Alison E. and Mahmoudi, Morteza},
  doi          = {10.1038/s41587-025-02803-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1577-1579},
  shortjournal = {Nature Biotech.},
  title        = {Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Global profiling of protein complex dynamics with an experimental library of protein interaction markers. <em>NBT</em>, <em>43</em>(9), 1562-1576. (<a href='https://doi.org/10.1038/s41587-024-02432-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to systematically monitor protein complex dynamics are needed. We introduce serial ultrafiltration combined with limited proteolysis-coupled mass spectrometry (FLiP–MS), a structural proteomics workflow that generates a library of peptide markers specific to changes in PPIs by probing differences in protease susceptibility between complex-bound and monomeric forms of proteins. The library includes markers mapping to protein-binding interfaces and markers reporting on structural changes that accompany PPI changes. Integrating the marker library with LiP–MS data allows for global profiling of protein–protein interactions (PPIs) from unfractionated lysates. We apply FLiP–MS to Saccharomyces cerevisiae and probe changes in protein complex dynamics after DNA replication stress, identifying links between Spt-Ada-Gcn5 acetyltransferase activity and the assembly state of several complexes. FLiP–MS enables protein complex dynamics to be probed on any perturbation, proteome-wide, at high throughput, with peptide-level structural resolution and informing on occupancy of binding interfaces, thus providing both global and molecular views of a system under study. FLiP–MS uses a library of protein–protein interaction markers to understand protein complex dynamics.},
  archive      = {J_NBT},
  author       = {Dörig, Christian and Marulli, Cathy and Peskett, Thomas and Volkmar, Norbert and Pantolini, Lorenzo and Studer, Gabriel and Paleari, Camilla and Frommelt, Fabian and Schwede, Torsten and de Souza, Natalie and Barral, Yves and Picotti, Paola},
  doi          = {10.1038/s41587-024-02432-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1562-1576},
  shortjournal = {Nature Biotech.},
  title        = {Global profiling of protein complex dynamics with an experimental library of protein interaction markers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A circularly permuted CasRx platform for efficient, site-specific RNA editing. <em>NBT</em>, <em>43</em>(9), 1548-1561. (<a href='https://doi.org/10.1038/s41587-024-02430-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inactive Cas13 orthologs have been fused to a mutant human ADAR2 deaminase domain at the C terminus to enable programmable adenosine-to-inosine (A-to-I) RNA editing in selected transcripts. Although promising, existing RNA-editing tools generally suffer from a trade-off between efficacy and specificity, and off-target editing remains an unsolved problem. Here we describe the development of an optimized RNA-editing platform by rational protein engineering, CasRx-based Programmable Editing of RNA Technology (xPERT). We demonstrate that the topological rearrangement of a CasRx K940L mutant by circular permutation results in a robust scaffold for the tethering of a deaminase domain. We benchmark our tool against the REPAIR system and show that xPERT exhibits strong on-target activity like REPAIRv1 but low off-target editing like REPAIRv2. Our xPERT platform can be used to alter RNA sequence information without risking genome damage, effect temporary cellular changes and customize protein function. Site-specific and efficient RNA editing is achieved with a CasRx editor.},
  archive      = {J_NBT},
  author       = {Wang, Yuanming and Liu, Kaiwen Ivy and Liu, Mengying Mandy and Ooi, Kean Hean and Nguyen, Tram Anh and Chee, Jiunn En and Teo, Shun Xiang Danny and He, Shan and Tay, Jie Wen Douglas and Teo, Seok Yee and Liew, Kai Shin and Ge, Xiao Yu and Ng, Zhi Jian and Avagyan, Hasmik and Liu, Hao and Yi, Zirong and Chang, Keziah and Kok, Eng Piew Louis and Chen, Runjia and Yau, Chun En and Koh, Jun Wei and Wan, Yue and Tan, Meng How},
  doi          = {10.1038/s41587-024-02430-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1548-1561},
  shortjournal = {Nature Biotech.},
  title        = {A circularly permuted CasRx platform for efficient, site-specific RNA editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One-step nanoscale expansion microscopy reveals individual protein shapes. <em>NBT</em>, <em>43</em>(9), 1539-1547. (<a href='https://doi.org/10.1038/s41587-024-02431-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The attainable resolution of fluorescence microscopy has reached the subnanometer range, but this technique still fails to image the morphology of single proteins or small molecular complexes. Here, we expand the specimens at least tenfold, label them with conventional fluorophores and image them with conventional light microscopes, acquiring videos in which we analyze fluorescence fluctuations. One-step nanoscale expansion (ONE) microscopy enables the visualization of the shapes of individual membrane and soluble proteins, achieving around 1-nm resolution. We show that conformational changes are readily observable, such as those undergone by the ~17-kDa protein calmodulin upon Ca2+ binding. ONE is also applied to clinical samples, analyzing the morphology of protein aggregates in cerebrospinal fluid from persons with Parkinson disease, potentially aiding disease diagnosis. This technology bridges the gap between high-resolution structural biology techniques and light microscopy, providing new avenues for discoveries in biology and medicine. Combining expansion microscopy with super-resolution radial fluctuations captures the morphology of single proteins.},
  archive      = {J_NBT},
  author       = {Shaib, Ali H. and Chouaib, Abed Alrahman and Chowdhury, Rajdeep and Altendorf, Jonas and Mihaylov, Daniel and Zhang, Chi and Krah, Donatus and Imani, Vanessa and Spencer, Russell K. W. and Georgiev, Svilen Veselinov and Mougios, Nikolaos and Monga, Mehar and Reshetniak, Sofiia and Mimoso, Tiago and Chen, Han and Fatehbasharzad, Parisa and Crzan, Dagmar and Saal, Kim-Ann and Alawieh, Mohamad Mahdi and Alawar, Nadia and Eilts, Janna and Kang, Jinyoung and Soleimani, Alireza and Müller, Marcus and Pape, Constantin and Alvarez, Luis and Trenkwalder, Claudia and Mollenhauer, Brit and Outeiro, Tiago F. and Köster, Sarah and Preobraschenski, Julia and Becherer, Ute and Moser, Tobias and Boyden, Edward S. and Aricescu, A. Radu and Sauer, Markus and Opazo, Felipe and Rizzoli, Silvio O.},
  doi          = {10.1038/s41587-024-02431-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1539-1547},
  shortjournal = {Nature Biotech.},
  title        = {One-step nanoscale expansion microscopy reveals individual protein shapes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of compact transcriptional effectors using high-throughput measurements in diverse contexts. <em>NBT</em>, <em>43</em>(9), 1525-1538. (<a href='https://doi.org/10.1038/s41587-024-02442-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional effectors are protein domains known to activate or repress gene expression; however, a systematic understanding of which effector domains regulate transcription across genomic, cell type and DNA-binding domain (DBD) contexts is lacking. Here we develop dCas9-mediated high-throughput recruitment (HT-recruit), a pooled screening method for quantifying effector function at endogenous target genes and test effector function for a library containing 5,092 nuclear protein Pfam domains across varied contexts. We also map context dependencies of effectors drawn from unannotated protein regions using a larger library tiling chromatin regulators and transcription factors. We find that many effectors depend on target and DBD contexts, such as HLH domains that can act as either activators or repressors. To enable efficient perturbations, we select context-robust domains, including ZNF705 KRAB, that improve CRISPRi tools to silence promoters and enhancers. We engineer a compact human activator called NFZ, by combining NCOA3, FOXO3 and ZNF473 domains, which enables efficient CRISPRa with better viral delivery and inducible control of chimeric antigen receptor T cells. Improved effectors for CRISPRi/CRISPRa are developed following high-throughput screening of transcriptional domains.},
  archive      = {J_NBT},
  author       = {Tycko, Josh and Van, Mike V. and Aradhana and DelRosso, Nicole and Ye, Hanrong and Yao, David and Valbuena, Raeline and Vaughan-Jackson, Alun and Xu, Xiaoshu and Ludwig, Connor and Spees, Kaitlyn and Liu, Katherine and Gu, Mingxin and Khare, Venya and Mukund, Adi Xiyal and Suzuki, Peter H. and Arana, Sophia and Zhang, Catherine and Du, Peter P. and Ornstein, Thea S. and Hess, Gaelen T. and Kamber, Roarke A. and Qi, Lei S. and Khalil, Ahmad S. and Bintu, Lacramioara and Bassik, Michael C.},
  doi          = {10.1038/s41587-024-02442-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1525-1538},
  shortjournal = {Nature Biotech.},
  title        = {Development of compact transcriptional effectors using high-throughput measurements in diverse contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations. <em>NBT</em>, <em>43</em>(9), 1510-1524. (<a href='https://doi.org/10.1038/s41587-024-02428-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To assist the translation of genetic findings to disease pathobiology and therapeutics discovery, we present an ensemble deep learning framework, termed PIONEER (Protein–protein InteractiOn iNtErfacE pRediction), that predicts protein-binding partner-specific interfaces for all known protein interactions in humans and seven other common model organisms to generate comprehensive structurally informed protein interactomes. We demonstrate that PIONEER outperforms existing state-of-the-art methods and experimentally validate its predictions. We show that disease-associated mutations are enriched in PIONEER-predicted protein–protein interfaces and explore their impact on disease prognosis and drug responses. We identify 586 significant protein–protein interactions (PPIs) enriched with PIONEER-predicted interface somatic mutations (termed oncoPPIs) from analysis of approximately 11,000 whole exomes across 33 cancer types and show significant associations of oncoPPIs with patient survival and drug responses. PIONEER, implemented as both a web server platform and a software package, identifies functional consequences of disease-associated alleles and offers a deep learning tool for precision medicine at multiscale interactome network levels. Protein–protein interactomes incorporating structural data predict the functional consequences of disease mutations.},
  archive      = {J_NBT},
  author       = {Xiong, Dapeng and Qiu, Yunguang and Zhao, Junfei and Zhou, Yadi and Lee, Dongjin and Gupta, Shobhita and Torres, Mateo and Lu, Weiqiang and Liang, Siqi and Kang, Jin Joo and Eng, Charis and Loscalzo, Joseph and Cheng, Feixiong and Yu, Haiyuan},
  doi          = {10.1038/s41587-024-02428-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1510-1524},
  shortjournal = {Nature Biotech.},
  title        = {A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors. <em>NBT</em>, <em>43</em>(9), 1496-1509. (<a href='https://doi.org/10.1038/s41587-024-02448-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies to enhance the anti-tumor immune response of stereotactic ablative radiotherapy (SABR) at primary tumors and abscopal sites are under intensive investigation. Here we report a metabolizable binary supracluster (BSCgal) that combines gold nanoclusters as radiosensitizing adjuvants with small interfering RNA (siRNA) targeting the immunosuppressive mediator galectin-1 (Gal-1). BSCgal comprises reversibly crosslinked cationic gold nanoclusters and siRNA complexes in a polymer matrix that biodegrades over weeks, facilitating clearance (90.3% in vivo clearance at 4 weeks) to reduce toxicity. The particle size well above the renal filtration threshold facilitates passive delivery to tumors. Using mouse models of head and neck cancer, we show that BSCgal augments the radiodynamic and immunotherapeutic effects of SABR at the primary and metastatic tumors by promoting tumor-inhibitory leukocytes, upregulating cytotoxic granzyme B and reducing immunosuppressive cell populations. It outperforms SABR plus Gal-1 antagonists, chemoradiation drug cisplatin or PD-1 inhibitor. This work presents a translatable strategy to converge focal radiosensitization with targeted immune checkpoint silencing for personalized radioimmunotherapy. Gold-siRNA clusters boost the immune response of radiotherapy against primary and distant tumors.},
  archive      = {J_NBT},
  author       = {Jiang, Yuyan and Cao, Hongbin and Deng, Huaping and Guan, Li and Langthasa, Jimpi and Colburg, Deana Rae Crystal and Melemenidis, Stavros and Cotton, Renee M. and Aleman, John and Wang, Xiao-Jing and Graves, Edward E. and Kalbasi, Anusha and Pu, Kanyi and Rao, Jianghong and Le, Quynh-Thu},
  doi          = {10.1038/s41587-024-02448-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1496-1509},
  shortjournal = {Nature Biotech.},
  title        = {Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE. <em>NBT</em>, <em>43</em>(9), 1485-1495. (<a href='https://doi.org/10.1038/s41587-024-02426-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing of messenger RNA (mRNA) found in extracellular vesicles (EVs) in liquid biopsies can provide clinical information such as somatic mutations, resistance profiles and tumor recurrence. Despite this, EV mRNA remains underused due to its low abundance in liquid biopsies, and large sample volumes or specialized techniques for analysis are required. Here we introduce Self-amplified and CRISPR-aided Operation to Profile EVs (SCOPE), a platform for EV mRNA detection. SCOPE leverages CRISPR-mediated recognition of target RNA using Cas13 to initiate replication and signal amplification, achieving a sub-attomolar detection limit while maintaining single-nucleotide resolution. As a proof of concept, we designed probes for key mutations in KRAS, BRAF, EGFR and IDH1 genes, optimized protocols for single-pot assays and implemented an automated device for multi-sample detection. We validated SCOPE’s ability to detect early-stage lung cancer in animal models, monitored tumor mutational burden in patients with colorectal cancer and stratified patients with glioblastoma. SCOPE can expedite readouts, augmenting the clinical use of EVs in precision oncology. SCOPE detects low-abundance extracellular vesicle mRNA with high sensitivity for clinical applications.},
  archive      = {J_NBT},
  author       = {Song, Jayeon and Cho, Mi Hyeon and Cho, Hayoung and Song, Younseong and Lee, Sung Woon and Nam, Ho Chul and Yoon, Tae Ho and Shin, Jong Cheol and Hong, Jae-Sang and Kim, Yejin and Ekanayake, Emil and Jeon, Jueun and You, Dong Gil and Im, Sung Gap and Choi, Gyu-Seog and Park, Jun Seok and Carter, Bob C. and Balaj, Leonora and Seo, An Na and Miller, Miles A. and Park, Soo Yeun and Kang, Taejoon and Castro, Cesar M. and Lee, Hakho},
  doi          = {10.1038/s41587-024-02426-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1485-1495},
  shortjournal = {Nature Biotech.},
  title        = {Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Saturation profiling of drug-resistant genetic variants using prime editing. <em>NBT</em>, <em>43</em>(9), 1471-1484. (<a href='https://doi.org/10.1038/s41587-024-02465-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to characterize the functional effects of genetic variants of uncertain significance (VUSs) have been limited by incomplete coverage of the mutational space. In clinical oncology, drug resistance arising from VUSs can prevent optimal treatment. Here we introduce PEER-seq, a high-throughput method based on prime editing that can evaluate the functional effects of single-nucleotide variants (SNVs). PEER-seq introduces both intended SNVs and synonymous marker mutations using prime editing and deep sequences the endogenous target regions to identify the introduced SNVs. We generate and functionally evaluate 2,476 SNVs in the epidermal growth factor receptor gene (EGFR), including 99% of all possible variants in the canonical tyrosine kinase domain. We determined resistance profiles of 95% of all possible EGFR protein variants encoded in the whole tyrosine kinase domain against the common tyrosine kinase inhibitors afatinib, osimertinib and osimertinib in the presence of the co-occurring substitution T790M, in PC-9 cells. Our study has the potential to substantially improve the precision of therapeutic choices in clinical settings. PEER-seq characterizes the drug resistance profiles of thousands of genetic variants.},
  archive      = {J_NBT},
  author       = {Kim, Younggwang and Oh, Hyeong-Cheol and Lee, Seungho and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-024-02465-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1471-1484},
  shortjournal = {Nature Biotech.},
  title        = {Saturation profiling of drug-resistant genetic variants using prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal scanning of genetic variants with base and prime editing. <em>NBT</em>, <em>43</em>(9), 1458-1470. (<a href='https://doi.org/10.1038/s41587-024-02439-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational scanning connects genetic variants to phenotype, enabling the interrogation of protein functions, interactions and variant pathogenicity. However, current methodologies cannot efficiently engineer customizable sets of diverse genetic variants in endogenous loci across cellular contexts in high throughput. Here, we combine cytosine and adenine base editors and a prime editor to assess the pathogenicity of a broad spectrum of variants in the epithelial growth factor receptor gene (EGFR). Using pooled base editing and prime editing guide RNA libraries, we install tens of thousands of variants spanning the full coding sequence of EGFR in multiple cell lines and assess the role of these variants in tumorigenesis and resistance to tyrosine kinase inhibitors. Our EGFR variant scan identifies important hits, supporting the robustness of the approach and revealing underappreciated routes to EGFR activation and drug response. We anticipate that multimodal precision mutational scanning can be applied broadly to characterize genetic variation in any genetic element of interest at high and single-nucleotide resolution. Thousands of rare oncogene variants are evaluated using multimodal gene editing screens.},
  archive      = {J_NBT},
  author       = {Belli, Olivier and Karava, Kyriaki and Farouni, Rick and Platt, Randall J.},
  doi          = {10.1038/s41587-024-02439-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1458-1470},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal scanning of genetic variants with base and prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein. <em>NBT</em>, <em>43</em>(9), 1445-1457. (<a href='https://doi.org/10.1038/s41587-024-02437-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticle (LNP) delivery of clustered regularly interspaced short palindromic repeat (CRISPR) ribonucleoproteins (RNPs) could enable high-efficiency, low-toxicity and scalable in vivo genome editing if efficacious RNP–LNP complexes can be reliably produced. Here we engineer a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) to generate iGeoCas9 variants capable of &gt;100× more genome editing of cells and organs compared with the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP–LNP complexes edit a variety of cell types and induce homology-directed repair in cells receiving codelivered single-stranded DNA templates. Using tissue-selective LNP formulations, we observe genome-editing levels of 16‒37% in the liver and lungs of reporter mice that receive single intravenous injections of iGeoCas9 RNP–LNPs. In addition, iGeoCas9 RNPs complexed to biodegradable LNPs edit the disease-causing SFTPC gene in lung tissue with 19% average efficiency, representing a major improvement over genome-editing levels observed previously using viral or nonviral delivery strategies. These results show that thermostable Cas9 RNP–LNP complexes can expand the therapeutic potential of genome editing. An engineered clustered regularly interspaced short palindromic repeat ribonucleoprotein delivered in lipid nanoparticles efficiently edits cells in vivo.},
  archive      = {J_NBT},
  author       = {Chen, Kai and Han, Hesong and Zhao, Sheng and Xu, Bryant and Yin, Boyan and Lawanprasert, Atip and Trinidad, Marena and Burgstone, Benjamin W. and Murthy, Niren and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-024-02437-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1445-1457},
  shortjournal = {Nature Biotech.},
  title        = {Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering b cells to treat and study human disease. <em>NBT</em>, <em>43</em>(9), 1431-1444. (<a href='https://doi.org/10.1038/s41587-025-02757-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern cell therapies are designed to harness natural biology to treat a range of complex diseases. The field of immunology has shown that B cells exhibit multiple unique features, including a natural propensity to interact with and regulate other immune cells, a high capacity to produce proteins, and a long cellular lifespan, which are being creatively applied in engineered B cell (eB cell) therapies. In recent years, advances in genome editing technologies and animal modeling have facilitated rapid progress in our ability to study eB cells and execute proof-of-concept studies, thus enabling the first clinical trials of eB cell therapies. In this review, we provide an overview of recent developments in eB cell therapies, including early clinical studies. We discuss challenges to clinical implementation, and promising directions for leveraging B cell biology in future applications for cancer and chronic disease. Recent developments in B cell engineering expand our ability to study human disease and drive progress toward new B cell therapies, balancing challenges in clinical implementation with emerging opportunities.},
  archive      = {J_NBT},
  author       = {Trivedi, Nikita and Pitner, Ragan A. and Rawlings, David J. and James, Richard G.},
  doi          = {10.1038/s41587-025-02757-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1431-1444},
  shortjournal = {Nature Biotech.},
  title        = {Engineering b cells to treat and study human disease},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A CRISPR-based method for detecting mRNA in extracellular vesicles. <em>NBT</em>, <em>43</em>(9), 1428-1429. (<a href='https://doi.org/10.1038/s41587-024-02440-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A CRISPR-based assay both recognizes and amplifies target mRNA, achieving sub-attomolar sensitivity with single-nucleotide resolution. This method enables the detection of low-abundance mRNA in extracellular vesicles, providing clinically relevant information for precision oncology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02440-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1428-1429},
  shortjournal = {Nature Biotech.},
  title        = {A CRISPR-based method for detecting mRNA in extracellular vesicles},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Bao and rogers. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02784-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02784-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bao and rogers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Ntziachristos and yao. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02785-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02785-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Ntziachristos and yao},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI designs de novo antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02822-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02822-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {AI designs de novo antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosted mitochondria reverse cognitive impairment in mice. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02823-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02823-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Boosted mitochondria reverse cognitive impairment in mice},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mining archaea for antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02824-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02824-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Mining archaea for antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MicroRNAs. <em>NBT</em>, <em>43</em>(9), 1424. (<a href='https://doi.org/10.1038/s41587-025-02800-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to pharmaceutical compositions and methods of treatment involving microRNAs.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02800-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1424},
  shortjournal = {Nature Biotech.},
  title        = {MicroRNAs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Intellectual property rights in the cultivated meat industry: Opportunities and challenges. <em>NBT</em>, <em>43</em>(9), 1419-1423. (<a href='https://doi.org/10.1038/s41587-025-02798-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivated meat faces escalating intellectual property (IP) issues as the industry expands, including conventional patent rights, the private IP rights of animal breeders and the Convention on Biological Diversity.},
  archive      = {J_NBT},
  author       = {Baek, Garam and Chung, Sukhyun and Yu, Peter and Park, Yooheon},
  doi          = {10.1038/s41587-025-02798-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1419-1423},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property rights in the cultivated meat industry: Opportunities and challenges},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining druggable targets with artificial intelligence. <em>NBT</em>, <em>43</em>(9), 1416-1418. (<a href='https://doi.org/10.1038/s41587-025-02770-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A vast landscape of ‘undruggable’ cancer targets remains beyond the reach of conventional therapeutic agents. Recent advances in artificial intelligence (AI), however, are challenging this paradigm. Synthesizing insights from a Cancer Moonshot workshop, we argue that systemically addressing the undruggable target space with AI requires a new conceptual framework. We highlight the failure of current target taxonomies and the need for benchmarking datasets, and re-evaluate clinical validation for novel AI-driven modalities.},
  archive      = {J_NBT},
  author       = {Akinsanya, Karen and AlQuraishi, Mohammed and Boija, Ann and Chodera, John and Cichońska, Anna and Ghassemi, Marzyeh and Head, Martha and Jin, Wengong and Kibbe, Warren A. and Krogan, Nevan and LeVine, Michael V. and Moult, John and Stuart, Lynda and Tourassi, Georgia and Zou, James and Barzilay, Regina and Elemento, Olivier},
  doi          = {10.1038/s41587-025-02770-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1416-1418},
  shortjournal = {Nature Biotech.},
  title        = {Redefining druggable targets with artificial intelligence},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Senolytics under scrutiny in the quest to slow aging. <em>NBT</em>, <em>43</em>(9), 1413-1415. (<a href='https://doi.org/10.1038/s41587-025-02740-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de Magalhães, João Pedro},
  doi          = {10.1038/s41587-025-02740-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1413-1415},
  shortjournal = {Nature Biotech.},
  title        = {Senolytics under scrutiny in the quest to slow aging},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Freya biosciences. <em>NBT</em>, <em>43</em>(9), 1411-1412. (<a href='https://doi.org/10.1038/s41587-025-02783-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Freya Biosciences is restoring balance in the vaginal microbiome to improve pregnancy outcomes.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02783-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1411-1412},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Freya biosciences},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Vilya therapeutics. <em>NBT</em>, <em>43</em>(9), 1409-1410. (<a href='https://doi.org/10.1038/s41587-025-02782-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Vilya Therapeutics is designing macrocycle drug candidates.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-025-02782-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1409-1410},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Vilya therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Seaport therapeutics. <em>NBT</em>, <em>43</em>(9), 1407-1408. (<a href='https://doi.org/10.1038/s41587-025-02792-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Seaport Therapeutics is modifying central nervous system drugs for delivery through the lymphatic system, bypassing first-pass liver metabolism.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-025-02792-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1407-1408},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Seaport therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(9), 1406. (<a href='https://doi.org/10.1038/s41587-025-02817-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02817-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1406},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blooms supersized. <em>NBT</em>, <em>43</em>(9), 1405. (<a href='https://doi.org/10.1038/s41587-025-02825-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02825-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1405},
  shortjournal = {Nature Biotech.},
  title        = {Blooms supersized},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wegovy approved for MASH. <em>NBT</em>, <em>43</em>(9), 1404. (<a href='https://doi.org/10.1038/s41587-025-02828-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02828-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1404},
  shortjournal = {Nature Biotech.},
  title        = {Wegovy approved for MASH},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA go-ahead for HPV immunotherapy. <em>NBT</em>, <em>43</em>(9), 1402. (<a href='https://doi.org/10.1038/s41587-025-02827-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02827-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1402},
  shortjournal = {Nature Biotech.},
  title        = {FDA go-ahead for HPV immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Regenerative cell therapy marches into corneal blindness. <em>NBT</em>, <em>43</em>(9), 1401-1404. (<a href='https://doi.org/10.1038/s41587-025-02808-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a newly marketed product in Japan and 10 years of clinical data, regenerative cell therapies promise new options for patients facing a choice between invasive surgery or blindness.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew},
  doi          = {10.1038/s41587-025-02808-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1401-1404},
  shortjournal = {Nature Biotech.},
  title        = {Regenerative cell therapy marches into corneal blindness},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). US government slams mRNA vaccine research. <em>NBT</em>, <em>43</em>(9), 1400. (<a href='https://doi.org/10.1038/s41587-025-02826-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02826-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1400},
  shortjournal = {Nature Biotech.},
  title        = {US government slams mRNA vaccine research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mitochondrial transplants make healthy babies. <em>NBT</em>, <em>43</em>(9), 1399-1400. (<a href='https://doi.org/10.1038/s41587-025-02799-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After decades of research and consultation with regulators, mitochondrial donation is taking off in the UK. Will other countries follow?},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-025-02799-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1399-1400},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial transplants make healthy babies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing antivirals. <em>NBT</em>, <em>43</em>(9), 1397-1398. (<a href='https://doi.org/10.1038/s41587-025-02829-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Funding and public support for vaccines in the United States is at an all-time low. Promising research is resulting in effective antivirals, but they should not take the place of vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02829-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1397-1398},
  shortjournal = {Nature Biotech.},
  title        = {Advancing antivirals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nchem">NCHEM - 22</h2>
<ul>
<li><details>
<summary>
(2025). Future-proofing calcium fluoride. <em>NCHEM</em>, <em>17</em>(9), 1434. (<a href='https://doi.org/10.1038/s41557-025-01922-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thomas Schlatzer and Véronique Gouverneur discuss why the chemistry of calcium fluoride is challenging, and how imaginative thinking based on fundamental principles can create positive change for the fluorochemical industry.},
  archive      = {J_NCHEM},
  author       = {Schlatzer, Thomas and Gouverneur, Véronique},
  doi          = {10.1038/s41557-025-01922-9},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1434},
  shortjournal = {Nat. Chem.},
  title        = {Future-proofing calcium fluoride},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Catalytic and stoichiometric stepwise conversion of side-on bound dinitrogen to ammonia mediated by a uranium complex. <em>NCHEM</em>, <em>17</em>(9), 1425-1433. (<a href='https://doi.org/10.1038/s41557-025-01867-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The well-defined catalytic conversion of dinitrogen (N2) to ammonia (NH3) by molecular complexes is of fundamental interest and important for providing an atomic-level understanding of reactivity that can be related to industrial and biological nitrogen-fixation processes. Molecular catalytic N2 to NH3 conversion currently involves the reduction and protonation of terminal or bridging end-on bound metal–N2 complexes. However, catalytic N2 to NH3 conversion by side-on bound metal–N2 molecular complexes is relevant to both the industrial and biological nitrogen-fixation processes. Here, using a uranium triamidoamine complex, we report catalytic N2 to NH3 conversion involving side-on bound N2 binding. Stoichiometric reactions reveal stepwise reduction of N2 from free N2 to bridging side-on bound forms and subsequently to bridging nitrides, uniquely accessing four different states of side-on bound N2 for the same molecular system. This reveals the roles of N2, N22−, N23−, N24− and N3− in the catalytic conversion of N2 to NH3 when involving side-on bridging N2. The catalytic conversion of nitrogen to ammonia by molecular complexes typically involves the reduction and protonation of end-on metal–N2 compounds. Now, a uranium complex has been shown to catalyse the conversion of N2 to ammonia through the formation of a side-on bound dinitrogen complex. Four different states of side-on bound dinitrogen and the cleavage to nitride have been observed.},
  archive      = {J_NCHEM},
  author       = {Batov, Mikhail S. and Partlow, Heather T. and Chatelain, Lucile and Seed, John A. and Scopelliti, Rosario and Zivkovic, Ivica and Adams, Ralph W. and Liddle, Stephen T. and Mazzanti, Marinella},
  doi          = {10.1038/s41557-025-01867-z},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1425-1433},
  shortjournal = {Nat. Chem.},
  title        = {Catalytic and stoichiometric stepwise conversion of side-on bound dinitrogen to ammonia mediated by a uranium complex},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Catalytic c(sp2) homologation of alkylboranes. <em>NCHEM</em>, <em>17</em>(9), 1418-1424. (<a href='https://doi.org/10.1038/s41557-025-01854-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoboron compounds are important intermediates in organic synthesis, commonly used in metal-catalysed cross-coupling reactions. Their unique reactivity also allows modifications of their carbon framework with preservation of the valuable boryl group. Traditionally, these homologation reactions have been confined to the formation of alkyl boron compounds via C(sp3) insertion into a C–B bond. However, recent advancements in C(sp2)-insertive homologation highlight the potential of these reactions in synthesizing complex alkenes, despite current limitations in scope and control of the alkene geometry. Here we demonstrate a catalytic C(sp2)-insertive homologation for the regio- and diastereoselective synthesis of complex trisubstituted diborylalkenes from simple alkylboranes and alkynyl boronic esters. Our work demonstrates a broad reaction scope and application of the resulting products in modular and stereoselective synthesis of highly substituted alkenes. Furthermore, we provide evidence supporting a unique mechanism responsible for the excellent stereoselectivity observed in the reaction. Homologation reactions of organoboron compounds, while common, have rarely been used in stereoselective synthesis of highly substituted alkenes. Now a catalytic diastereoselective homologation of alkylboranes into trisubstituted alkenes has been developed, with key catalytic intermediates elucidated in a detailed mechanistic study.},
  archive      = {J_NCHEM},
  author       = {Gardner, Bradley W. and Lalic, Gojko},
  doi          = {10.1038/s41557-025-01854-4},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1418-1424},
  shortjournal = {Nat. Chem.},
  title        = {Catalytic c(sp2) homologation of alkylboranes},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bright triplet and bright charge-separated singlet excitons in organic diradicals enable optical read-out and writing of spin states. <em>NCHEM</em>, <em>17</em>(9), 1410-1417. (<a href='https://doi.org/10.1038/s41557-025-01875-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical control of electron spin states is important for quantum sensing and computing applications, as developed with the diamond nitrogen vacancy centre. This requires electronic excitations, excitons, with net spin. Here we report a molecular diradical where two trityl radical groups are coupled via a meta-linked fluorene bridge. The singlet exciton is at lower energy than the triplet because electron transfer from one of the radical non-bonding orbitals to the other is spin allowed, set by the charging energy for the double occupancy of the non-bonding level, the Hubbard U. Both excitons give efficient photoluminescence at 640 and 700 nm with near unity efficiency. The ground state exchange energy is low, 60 µeV, allowing control of ground state spin populations. We demonstrate spin-selective intersystem crossing and show coherent microwave control. We report up to 8% photoluminescence contrast at microwave resonance. This tuning of the singlet Mott–Hubbard exciton against the ‘bandgap’ exciton provides a new design platform for spin–optical materials. Controlling electron spin states with light is vital for quantum technologies but requires electronic excitations with net spin. Now a molecular diradical with two trityl radical groups coupled via a meta-linked fluorene bridge has been developed that features luminescent singlet and triplet excitons. The spins in the ground state can be written and read using photons, giving rise to broad magnetic and microwave modulation of the photoluminescence.},
  archive      = {J_NCHEM},
  author       = {Chowdhury, Rituparno and Murto, Petri and Panjwani, Naitik A. and Sun, Yan and Ghosh, Pratyush and Boeije, Yorrick and Cordeiro, Chiara Delpiano and Derkach, Vadim and Woo, Seung-Je and Millington, Oliver and Congrave, Daniel G. and Fu, Yao and Mustafa, Tarig B. E. and Monteverde, Miguel and Cerdá, Jesús and Londi, Giacomo and Behrends, Jan and Rao, Akshay and Beljonne, David and Chepelianskii, Alexei and Bronstein, Hugo and Friend, Richard H.},
  doi          = {10.1038/s41557-025-01875-z},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1410-1417},
  shortjournal = {Nat. Chem.},
  title        = {Bright triplet and bright charge-separated singlet excitons in organic diradicals enable optical read-out and writing of spin states},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synthesis of zero-dimensional octahedral metal halides through solvent incorporation and their photophysical properties. <em>NCHEM</em>, <em>17</em>(9), 1401-1409. (<a href='https://doi.org/10.1038/s41557-025-01869-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zero-dimensional (0D) metal halides, which feature discrete metal halide octahedra interspersed with large organoammonium cations, are the building blocks of halide perovskites. The optical properties of these materials make them promising candidates in light-emitting devices. However, developing their general design principles remains challenging. Here we report an antisolvent incorporation approach that transforms a broad range of two-dimensional tin iodide perovskites into 0D structures. This approach accommodates diverse organic cations and antisolvent molecules and is extendable to germanium and lead analogues. We show that enhancing the structural rigidity in Sn-based octahedra—modulated by organic packing—increases the radiative recombination rate while decreasing the non-radiative recombination rate, achieving near-unity photoluminescence quantum yield. Ge- and Sn-based structures exhibit large Stokes shifts and microsecond-scale triplet lifetimes due to localized excited states, while their Pb-based counterpart shows faster recombination via triplet–singlet mixing, supported by theoretical calculations. This work offers a versatile synthetic platform for 0D metal halides and deepens our current understanding of excited-state dynamics in halide perovskites. Zero-dimensional metal halides exhibit intriguing photophysical properties; however, developing their general design principles remains challenging. Now it has been shown that two-dimensional perovskites can be converted into zero-dimensional octahedral metal halides through solvent incorporation. The resulting materials feature near-unity photoluminescence quantum yields. Structural rigidity and the choice of metal cation are key parameters to tune the exciton recombination dynamics.},
  archive      = {J_NCHEM},
  author       = {Zheng, Nanlong and Cao, Songqi and Zhang, Tianhao and Huo, Peihao and Hu, Conglan and Liu, Huanyu and Guo, Ruoyao and Yan, Wenchao and Zheng, Jiayin and Jiang, Xiaofan and Zhu, Zhenyu and Sun, Junliang and Jiang, Hong and Bian, Zuqiang and Fu, Yongping and Liu, Zhiwei},
  doi          = {10.1038/s41557-025-01869-x},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1401-1409},
  shortjournal = {Nat. Chem.},
  title        = {Synthesis of zero-dimensional octahedral metal halides through solvent incorporation and their photophysical properties},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Expedited access to polyunsaturated fatty acids and biofunctional analogues by full solid-phase synthesis. <em>NCHEM</em>, <em>17</em>(9), 1391-1400. (<a href='https://doi.org/10.1038/s41557-025-01853-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyunsaturated fatty acids (PUFAs) represent a fundamental and essential class of lipids that exhibit versatile biofunctions. Lipidomic analysis has identified a growing number of lipid species, including PUFAs with diverse structural variations and biofunctions, yet their structure–function relationships are still largely unknown. In this context, an efficient synthesis of PUFAs would be highly desirable. However, no practical methodology exists for their preparation, in contrast to peptides and nucleic acids, for which diverse molecules are accessible through a well-established solid-phase synthesis. To address this, we have developed an efficient and expedited method to access a wide array of PUFAs by full solid-phase synthesis. The method allows the synthesis of various PUFAs and analogues through rapid and facile operations. Moreover, within our PUFA library, we have discovered an artificial fatty acid, antiefin, that has a high anti-inflammatory effect in vivo. Therefore, our practical synthetic pathway to PUFAs, a crucial class of lipids, is expected to make an important contribution to lipid science. Polyunsaturated fatty acids (PUFAs) are attracting considerable interest because of their diverse structures and properties. Now a method to access a wide array of PUFAs by full solid-phase synthesis has been developed and used to discover an anti-inflammatory fatty acid.},
  archive      = {J_NCHEM},
  author       = {Saito, Yutaro and Akita, Mayuko and Saika, Azusa and Sano, Yusuke and Hotta, Masashi and Morimoto, Jumpei and Uwamizu, Akiharu and Aoki, Junken and Nagatake, Takahiro and Kunisawa, Jun and Sando, Shinsuke},
  doi          = {10.1038/s41557-025-01853-5},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1391-1400},
  shortjournal = {Nat. Chem.},
  title        = {Expedited access to polyunsaturated fatty acids and biofunctional analogues by full solid-phase synthesis},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Energy transfer-enabled enantioselective photocyclization using a privileged al–salen catalyst. <em>NCHEM</em>, <em>17</em>(9), 1383-1390. (<a href='https://doi.org/10.1038/s41557-025-01857-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chiral catalysts that can engage multiple substrates, via distinct ground-state activation modes, to deliver enantioenriched products with high levels of fidelity are often described as ‘privileged’. Achieving generality in excited-state processes remains challenging, and efforts to identify privileged chiral photocatalysts are being intensively pursued. Aluminium–salen complexes are emergent contenders on account of their well-defined photophysical properties. Here we report the development of an enantioselective energy transfer (EnT) catalysis-enabled photocyclization of acrylanilides to expand the activation repertoire of Al–salen photocatalysts. This approach allows reactivity and enantioselectivity to be simultaneously regulated by an inexpensive, commercial chiral Al–salen complex upon irradiation at λ = 400 nm. Diverse cyclic products can be forged with high levels of enantioselectivity (up to 96:4 e.r.). Establishing this dichotomy in excited-state activation modes serves to consolidate the privileged status of chiral Al–salen complexes in enantioselective photocatalysis and to complement their ubiquity in ground-state regimes. Photocatalysts that can engage a range of substrates and regulate reactivity and enantioselectivity are highly desirable. Now, an enantioselective energy transfer catalysis-enabled photocyclization of acrylanilides has been developed. Reactivity and enantioselectivity are simultaneously regulated by a commercial chiral Al–salen complex upon irradiation, enabling the formation of diverse cyclic products with high levels of enantioselectivity.},
  archive      = {J_NCHEM},
  author       = {Soika, Julia and Onneken, Carina and Wiegmann, Thorben and Stünkel, Timo and Morack, Tobias and Lindfeld, Leander and Hebenbrock, Marian and Mück-Lichtenfeld, Christian and Neugebauer, Johannes and Gilmour, Ryan},
  doi          = {10.1038/s41557-025-01857-1},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1383-1390},
  shortjournal = {Nat. Chem.},
  title        = {Energy transfer-enabled enantioselective photocyclization using a privileged al–salen catalyst},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered reactivity of a bacterial e1-like enzyme enables ATP-driven modification of protein and peptide c termini. <em>NCHEM</em>, <em>17</em>(9), 1371-1382. (<a href='https://doi.org/10.1038/s41557-025-01871-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In biological systems, adenosine triphosphate (ATP) provides an energetic driving force for peptide bond formation, but protein chemists lack tools that emulate this strategy. Here we develop an ATP-driven platform for C-terminal activation and peptide ligation based on MccB, a bacterial ancestor of ubiquitin-activating (E1) enzymes. We show that MccB can act on non-native substrates to generate an O-AMPylated electrophile that reacts with exogenous nucleophiles to form diverse C-terminal functional groups including thioesters, a versatile class of biological intermediates that have been exploited for protein C-terminal bioconjugation. By mining the natural diversity of the MccB family, we identify both epitope-specific and more promiscuous MccBs. We show that epitope-specific MccB activity can be directed toward specific proteins of interest to enable high-yield, ATP-driven protein bioconjugation, and promiscuous MccB activity can be deployed for the synthesis of peptide thioester substrates for bioconjugation. Our method mimics the chemical logic of biological peptide bond synthesis for high-yield in vitro manipulation of protein structure with molecular precision. In living systems, ATP provides an energetic driving force for protein synthesis and modification. Now, an engineered enzymatic tool has been developed for high-yield, ATP-driven protein bioconjugation through versatile thioester intermediates. This method mimics biological strategies for peptide bond formation to enable precision chemical modification of proteins in vitro.},
  archive      = {J_NCHEM},
  author       = {Frazier, Clara L. and Deb, Debashrito and Leiter, William E. and Mondal, Umasankar and Weeks, Amy M.},
  doi          = {10.1038/s41557-025-01871-3},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1371-1382},
  shortjournal = {Nat. Chem.},
  title        = {Engineered reactivity of a bacterial e1-like enzyme enables ATP-driven modification of protein and peptide c termini},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A lanthanide–carbon triple bond stabilized within a fullerene cage. <em>NCHEM</em>, <em>17</em>(9), 1364-1370. (<a href='https://doi.org/10.1038/s41557-025-01856-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metal–ligand multiple bonds are ubiquitous in coordination and organometallic chemistry. In contrast, lanthanide–carbon multiple bonds are difficult to form. The isolation of complexes containing lanthanide–carbon double bonds with terminal methyl carbene (=CH2) and lanthanide–carbon triple bonds remains challenging. Here we present the synthesis of a lanthanide–carbon triple bond contained inside an endohedral fullerene cage. More specifically, we report a cerium–carbide [Ce≡CSc2] cluster encapsulated inside a C80 fullerene cage. The molecular structure of Ce≡CSc2@C80 and the nature of the Ce≡C triple bond are studied through X-ray crystallography, spectroscopic analyses and quantum chemical calculations. Our data reveal a very short Ce≡C distance of 1.969(7) Å. Chemical bonding analysis suggests that the formation of the Ce≡C bond primarily arises from the stronger bonding affinity between carbon and cerium compared with scandium inside the encapsulated cluster. The fullerene cage plays a crucial role in stabilizing and protecting this trimetallic carbide cluster featuring a Ce≡C triple bond. The synthesis and isolation of compounds featuring lanthanide–carbon triple bonds remain challenging. Now a cluster featuring a cerium–carbon triple bond has been stabilized inside a C80 fullerene cage. The solid-state structure of the compound reveals a cerium–carbon bond distance of 1.969(7) Å.},
  archive      = {J_NCHEM},
  author       = {Jiang, Hongjie and Zhao, Jing and Meng, Qingyu and Zhao, Xiao-Kun and Guo, Min and Hu, Han-Shi and Li, Jun and Chen, Ning},
  doi          = {10.1038/s41557-025-01856-2},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1364-1370},
  shortjournal = {Nat. Chem.},
  title        = {A lanthanide–carbon triple bond stabilized within a fullerene cage},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Zigzag graphene nanoribbons with periodic porphyrin edge extensions. <em>NCHEM</em>, <em>17</em>(9), 1356-1363. (<a href='https://doi.org/10.1038/s41557-025-01887-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Graphene nanoribbons (GNRs) with zigzag edges are promising materials for spintronic devices due to tunable bandgaps and spin-polarized edge states. Porphyrins offer complementary benefits such as desirable optoelectronic properties. Here we combine these features in a hybrid system by means of the on-surface synthesis of zigzag-edge GNRs embedded with porphyrins laterally fused along the ribbon backbone. Using scanning probe methods, we show that this design achieves strong electronic coupling between the porphyrin and the GNR. For transition metal porphyrins, pronounced exchange coupling between distant metal centres is mediated by the π-electron system. Such a hybrid d and π electron ribbon system introduces spin–orbit coupling and magnetic anisotropy to carbon nanomaterials, and holds great promise for coherent electrical control of electron spins. Graphene nanoribbons with zigzag edges are key candidates for spintronic applications due to their tunable bandgaps and spin-polarized edge states. Now it has been shown that hybrid ribbons embedded with metalloporphyrins enable strong electronic and magnetic coupling between distant metal centres, positioning such hybrids as promising materials for quantum devices.},
  archive      = {J_NCHEM},
  author       = {Xiang, Feifei and Gu, Yanwei and Kinikar, Amogh and Bassi, Nicolò and Ortega-Guerrero, Andres and Qiu, Zijie and Gröning, Oliver and Ruffieux, Pascal and Pignedoli, Carlo A. and Müllen, Klaus and Fasel, Roman},
  doi          = {10.1038/s41557-025-01887-9},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1356-1363},
  shortjournal = {Nat. Chem.},
  title        = {Zigzag graphene nanoribbons with periodic porphyrin edge extensions},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Metal-centred states control carrier lifetimes in transition metal oxide photocatalysts. <em>NCHEM</em>, <em>17</em>(9), 1348-1355. (<a href='https://doi.org/10.1038/s41557-025-01868-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efficient sunlight-to-energy conversion requires materials that can generate long-lived charge carriers upon illumination. However, the targeted design of semiconductors possessing intrinsically long lifetimes remains a key challenge. Here using a series of transition metal oxides, we establish a link between carrier lifetime and electronic configuration in transition metal-based semiconductors. We identify a subpicosecond relaxation mechanism via metal-centred ligand field states that compromise quantum yields in open d-shell transition metal oxides (for example, Fe2O3, Co3O4, Cr2O3 and NiO), which is more reminiscent of molecular complexes than crystalline semiconductors. We found that materials with spin-forbidden ligand field transitions could partially mitigate this relaxation pathway, explaining why Fe2O3 achieves higher photoelectrochemical activity than other visible light-absorbing transition metal oxides. However, achieving high yields of long-lived charges requires transition metal oxides with d0 or d10 electronic configurations (for example, TiO2 and BiVO4), where ligand field states are absent. These trends translate to transition metal-containing semiconductors beyond oxides, enabling the design of photoabsorbers with better-controlled recombination channels in photovoltaics, photocatalysis and communication devices. Efficient sunlight-to-fuel conversion requires materials that produce long-lived charges, but increasing carrier lifetimes in visible-light-active photocatalysts has proven difficult. Now it has been shown that rapid charge deactivation via metal-centred states prevents long lifetimes and limits photocatalytic performance. These insights bridge solid-state and molecular photochemistry, informing strategies for improved photocatalyst design.},
  archive      = {J_NCHEM},
  author       = {Sachs, Michael and Harnett-Caulfield, Liam and Pastor, Ernest and Davies, Bernadette and Sowood, Daniel J. C. and Moss, Benjamin and Kafizas, Andreas and Nelson, Jenny and Walsh, Aron and Durrant, James R.},
  doi          = {10.1038/s41557-025-01868-y},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1348-1355},
  shortjournal = {Nat. Chem.},
  title        = {Metal-centred states control carrier lifetimes in transition metal oxide photocatalysts},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On-DNA C–H functionalization of electron-rich arenes for DNA-encoded libraries. <em>NCHEM</em>, <em>17</em>(9), 1340-1347. (<a href='https://doi.org/10.1038/s41557-025-01844-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA-encoded libraries (DELs) are useful for hit discovery in the pharmaceutical industry. Although a large number of individually coded molecules are accessible through DELs, their structural diversity is limited because few transformations are benign and chemoselective enough to be applied in the presence of DNA in aqueous environments. In particular, C–H functionalization chemistry that could be ideally suited to increase structural diversity through late-stage functionalization is currently absent from DEL synthesis. Here we present a general C–H functionalization of electron-rich arenes on DNA. The development of a selenoxide reagent is key to achieving the regio- and chemoselective formation of arylselenonium salts in aqueous media. The introduction of arylselenonium salts offers a versatile linchpin on DNA conjugates, which gives access to a multitude of analogues through diverse subsequent reactions, including transition-metal-mediated and photochemical transformations for the formation of C–C, C–I and C–S bonds. C–H functionalization is a key reaction in organic synthesis. Now a selenoxide reagent has been developed for the formation of arylselenonium salts via C–H functionalization of DNA conjugates with high selectivity. The arylselenonium salts participate in various transformations to forge new C–C and C–X bonds.},
  archive      = {J_NCHEM},
  author       = {de Pedro Beato, Eduardo and Torkowski, Luca and Hartmann, Philipp and Vogelsang, Lara and Dietz, Karl-Josef and Ritter, Tobias},
  doi          = {10.1038/s41557-025-01844-6},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1340-1347},
  shortjournal = {Nat. Chem.},
  title        = {On-DNA C–H functionalization of electron-rich arenes for DNA-encoded libraries},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A selenoxide for single-atom protein modification of tyrosine residues enabled by water-resistant chalcogen and hydrogen bonding. <em>NCHEM</em>, <em>17</em>(9), 1331-1339. (<a href='https://doi.org/10.1038/s41557-025-01842-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modifications such as phosphorylation and acetylation are often minor structural modifications that can have profound effects on protein structure and thus broaden protein functions. Nevertheless, studying these effects directly is often out of reach because no general chemistry exists to introduce small modifications selectively; either a large, stable linker structure is selectively installed on protein residues, or a small substituent is introduced at the risk of low selectivity due to the use of reactive, indiscriminate molecules. Here we report a C–H functionalization reaction of tyrosine residues to access peptides and proteins modified by small structural changes including single-atom substitutions. A rationally designed selenoxide introduces a versatile selenonium linchpin featuring a Ctyr–Se bond that can be used for further transformations at specific tyrosine residues. Key to the advance is the interplay of water-resistant, intramolecular chalcogen and hydrogen bonding of the selenoxide reagent, which allows chemo- and site-selective electrophilic aromatic substitution of tyrosine residues in aqueous solutions. Proteins with small structural modifications at specific sites are valuable, yet challenging to access by chemical methods. Now, tyrosine-selective single-atom modifications on proteins have been achieved by C–H functionalization using a rationally designed selenoxide to introduce a versatile selenonium linchpin for further transformations.},
  archive      = {J_NCHEM},
  author       = {Lin, Songyun and Hirao, Marina and Hartmann, Philipp and Leutzsch, Markus and Sterling, Marie Sophie and Vetere, Alessandro and Klimmek, Sandra and Hinrichs, Heike and Mengeler, Johanna Marie and Lehmann, Johannes and Samsonowicz-Gόrski, Jan and Berger, Florian and Ritter, Tobias},
  doi          = {10.1038/s41557-025-01842-8},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1331-1339},
  shortjournal = {Nat. Chem.},
  title        = {A selenoxide for single-atom protein modification of tyrosine residues enabled by water-resistant chalcogen and hydrogen bonding},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modular alkyl growth in amines via the selective insertion of alkynes into C–C bonds. <em>NCHEM</em>, <em>17</em>(9), 1323-1330. (<a href='https://doi.org/10.1038/s41557-025-01849-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The site-specific modification of amines has been a highly sought-after objective in organic synthesis. Despite the rapid advancement of carbon–hydrogen (C–H) bond functionalization methods, effective strategies for carbon–carbon (C–C) bond functionalization of amines remain elusive. Here we report a borane-catalysed method for the selective insertion of alkynes into alkyl C–C bonds of amines, resulting in the ring expansion of cyclic amines and chain elongation of acyclic amines. This approach begins with the cleavage of C–H bonds in amines, then transitioning to C–C bond functionalization upon reaction with alkynes. This method is effective with amines lacking an activating or leaving group and is suitable for late-stage functionalization of pharmaceuticals through C–C bond modification. Furthermore, by coupling this reaction with hydrolysis and hydrogenation steps, successive alkyne insertions are achieved, enabling modular and iterative alkyl growth of amines. The functionalization of carbon–carbon bonds in amines remains an important challenge in organic synthesis. Now a borane-catalysed method has been developed that enables the insertion of alkynes into the alkyl C–C bonds of amines, providing a versatile approach for the ring expansion of cyclic amines and chain elongation of acyclic amines.},
  archive      = {J_NCHEM},
  author       = {Zhou, Xin-Yue and Liu, Lu and Lyu, Hairong and Wang, Xiao-Chen},
  doi          = {10.1038/s41557-025-01849-1},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1323-1330},
  shortjournal = {Nat. Chem.},
  title        = {Modular alkyl growth in amines via the selective insertion of alkynes into C–C bonds},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Solid solvation structure design improves all-solid-state organic batteries. <em>NCHEM</em>, <em>17</em>(9), 1313-1322. (<a href='https://doi.org/10.1038/s41557-025-01866-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organic electrode materials offer a versatile, sustainable approach for next-generation lithium-ion batteries but are limited by low working voltages and poor cycling stability. Here we report a solid-solvation-structure design strategy to improve both the voltage and stability of organic electrode materials in all-solid-state batteries. As a proof of concept, we incorporate halide electrolytes as solid solutes and tetrachloro-o-benzoquinone as a solid solvent to form homogeneous solid cathode solutions. Systematic optimization of the inner solvation configuration enables tetrachloro-o-benzoquinone to achieve a high working voltage (3.6 V vs. Li+/Li) at room temperature within an asymmetric solid solvation sheath. Moreover, the equilibrium redox pathway and electrostatically driven self-healing interfaces revealed rapid redox kinetics and stable performance over 7,500 cycles in all-solid-state batteries under low stack pressures. This work demonstrates that organic electrode materials can serve as viable, durable and cost-effective alternatives to transition metal oxides in all-solid-state batteries. Organic electrode materials offer a versatile and sustainable route for lithium-ion batteries, but their application is hindered by low working voltages and poor cycling stability. Now, it has been shown that dissolving halide electrolytes into organic electrodes effectively tunes the working voltage and enhances the electrochemical performance of all-solid-state batteries.},
  archive      = {J_NCHEM},
  author       = {Hu, Yang and Su, Han and Fu, Jiamin and Luo, Jing and Yu, Qihang and Zhao, Feipeng and Li, Weihan and Deng, Sixu and Liu, Yu and Yuan, Yi and Gan, Yi and Wang, Yijia and Kim, Jung Tae and Chen, Ning and Shakouri, Mohsen and Hao, Xiaoge and Gao, Yingjie and Pang, Tianlu and Zhang, Nian and Jiang, Ming and Li, Xia and Zhao, Yang and Tu, Jiangping and Wang, Changhong and Sun, Xueliang},
  doi          = {10.1038/s41557-025-01866-0},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1313-1322},
  shortjournal = {Nat. Chem.},
  title        = {Solid solvation structure design improves all-solid-state organic batteries},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An enzyme for high-yield, ATP-driven C-terminal thioester generation. <em>NCHEM</em>, <em>17</em>(9), 1311-1312. (<a href='https://doi.org/10.1038/s41557-025-01872-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current tools for protein bioconjugation at the C-terminal have limited yields. Now, an enzymatic strategy for ATP-dependent activation of protein and peptide C termini has been developed. This versatile tool can be deployed for synthesis of C-terminal thioesters that enhance the yield and accessibility of diverse protein bioconjugation methods.},
  archive      = {J_NCHEM},
  doi          = {10.1038/s41557-025-01872-2},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1311-1312},
  shortjournal = {Nat. Chem.},
  title        = {An enzyme for high-yield, ATP-driven C-terminal thioester generation},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stabilizing a Ce≡C triple bond through fullerene confinement. <em>NCHEM</em>, <em>17</em>(9), 1309-1310. (<a href='https://doi.org/10.1038/s41557-025-01919-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The formation of lanthanide–carbon triple bonds is generally considered unfeasible. Now, a cerium–carbon triple bond is stabilized through encapsulation within an endohedral fullerene cage, suggesting a revision to the fundamental concepts of lanthanide bonding theory.},
  archive      = {J_NCHEM},
  doi          = {10.1038/s41557-025-01919-4},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1309-1310},
  shortjournal = {Nat. Chem.},
  title        = {Stabilizing a Ce≡C triple bond through fullerene confinement},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Merging d- and π-electron magnetism. <em>NCHEM</em>, <em>17</em>(9), 1307-1308. (<a href='https://doi.org/10.1038/s41557-025-01923-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of materials with tunable electronic and magnetic properties contributes to the advancement of spintronic technologies and optoelectronics. Now, the combination of π- and d-electron magnetism has been achieved through the lateral fusion of periodic metalloporphyrins along the zigzag edges of graphene nanoribbons.},
  archive      = {J_NCHEM},
  author       = {Song, Shaotang and Lu, Jiong},
  doi          = {10.1038/s41557-025-01923-8},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1307-1308},
  shortjournal = {Nat. Chem.},
  title        = {Merging d- and π-electron magnetism},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Selenoxides unlock C–H bonds of biomacromolecules. <em>NCHEM</em>, <em>17</em>(9), 1304-1306. (<a href='https://doi.org/10.1038/s41557-025-01921-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Direct C–H selenylation of biomacromolecules under biocompatible conditions has long posed a formidable challenge. Now, engineered selenoxide reagents have been shown to achieve site-specific incorporation of selenium into DNA and proteins within physiologically relevant environments.},
  archive      = {J_NCHEM},
  author       = {Fang, Xianfu and Li, Yizhou},
  doi          = {10.1038/s41557-025-01921-w},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1304-1306},
  shortjournal = {Nat. Chem.},
  title        = {Selenoxides unlock C–H bonds of biomacromolecules},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A rationale for charge carrier lifetimes in transition metal semiconductors. <em>NCHEM</em>, <em>17</em>(9), 1301-1303. (<a href='https://doi.org/10.1038/s41557-025-01913-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-lived charge carriers in semiconductors are essential for effective sunlight-to-energy conversion. Still, few studies have fundamentally rationalized the charge carrier lifetimes. Now, metal-centred ligand field states of transition metal oxides have been suggested to control carrier lifetimes, akin to those for molecular photosensitizers.},
  archive      = {J_NCHEM},
  author       = {Bourgois, Céline and Troian-Gautier, Ludovic},
  doi          = {10.1038/s41557-025-01913-w},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1301-1303},
  shortjournal = {Nat. Chem.},
  title        = {A rationale for charge carrier lifetimes in transition metal semiconductors},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ring expansion and chain extension of alkyl amines. <em>NCHEM</em>, <em>17</em>(9), 1299-1300. (<a href='https://doi.org/10.1038/s41557-025-01920-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amines are highly valued molecules, used to understand biological processes and in the discovery of new medicines. However, medium-sized cyclic and macrocyclic amines are extremely challenging to synthesize and are therefore underutilized. Now, up to 15-membered cyclic amines can be formed directly from smaller, more accessible rings via an organoborane-catalysed two-carbon-unit C–C bond insertion.},
  archive      = {J_NCHEM},
  author       = {Barysevich, Maryia and Pulis, Alexander P.},
  doi          = {10.1038/s41557-025-01920-x},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1299-1300},
  shortjournal = {Nat. Chem.},
  title        = {Ring expansion and chain extension of alkyl amines},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Agree to disagree. <em>NCHEM</em>, <em>17</em>(9), 1297-1298. (<a href='https://doi.org/10.1038/s41557-025-01918-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Science is about discovering new knowledge, so, logically, there will be disagreement. Shira Joudan contemplates how disagreements can be useful, and how to deal with them when they arise.},
  archive      = {J_NCHEM},
  author       = {Joudan, Shira},
  doi          = {10.1038/s41557-025-01918-5},
  journal      = {Nature Chemistry},
  month        = {9},
  number       = {9},
  pages        = {1297-1298},
  shortjournal = {Nat. Chem.},
  title        = {Agree to disagree},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="ng">NG - 34</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48. <em>NG</em>, <em>57</em>(9), 2340-2342. (<a href='https://doi.org/10.1038/s41588-025-02332-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Riazuddin, Saima and Belyantseva, Inna A. and Giese, Arnaud P. J. and Lee, Kwanghyuk and Indzhykulian, Artur A. and Nandamuri, Sri Pratima and Yousaf, Rizwan and Sinha, Ghanshyam P. and Lee, Sue and Terrell, David and Hegde, Rashmi S. and Ali, Rana A. and Anwar, Saima and Andrade-Elizondo, Paula B. and Sirmaci, Asli and Parise, Leslie V. and Basit, Sulman and Wali, Abdul and Ayub, Muhammad and Ansar, Muhammad and Ahmad, Wasim and Khan, Shaheen N. and Akram, Javed and Tekin, Mustafa and Riazuddin, Sheikh and Cook, Tiffany and Buschbeck, Elke K. and Frolenkov, Gregory I. and Leal, Suzanne M. and Friedman, Thomas B. and Ahmed, Zubair M.},
  doi          = {10.1038/s41588-025-02332-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2340-2342},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation. <em>NG</em>, <em>57</em>(9), 2339. (<a href='https://doi.org/10.1038/s41588-025-02328-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Uryu, Hidetaka and Saeki, Koichi and Haeno, Hiroshi and Kapadia, Chiraag Deepak and Furudate, Ken and Nangalia, Jyoti and Spencer Chapman, Michael and Zhang, Linda and Padilla, Jennifer and Zhao, Li and Hsu, Joanne I. and Zhao, Chong and Chen, Shujuan and Tanaka, Tomoyuki and Li, Zongrui and Ogata, Satoko and Hanache, Sarah and Yang, Hui and DiNardo, Courtney and Daver, Naval and Pemmaraju, Naveen and Jain, Nitin and Ravandi, Farhad and Zhang, Jianhua and Song, Xingzhi and Thompson, Erika and Tang, Hongli and Little, Latasha and Gumbs, Curtis and Orlowski, Robert Z. and Qazilbash, Muzaffar and Bhalla, Kapil and Colla, Simona and Kantarjian, Hagop and Kanagal-Shamanna, Rashmi and Bueso-Ramos, Carlos and Nakada, Daisuke and Al-Atrash, Gheath and Molldrem, Jeffery and Futreal, P. Andrew and Shpall, Elizabeth and Goodell, Margaret and Garcia-Manero, Guillermo and Takahashi, Koichi},
  doi          = {10.1038/s41588-025-02328-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2339},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma. <em>NG</em>, <em>57</em>(9), 2338. (<a href='https://doi.org/10.1038/s41588-025-02320-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gundem, Gunes and Levine, Max F. and Roberts, Stephen S. and Cheung, Irene Y. and Medina-Martínez, Juan S. and Feng, Yi and Arango-Ossa, Juan E. and Chadoutaud, Loic and Rita, Mathieu and Asimomitis, Georgios and Zhou, Joe and You, Daoqi and Bouvier, Nancy and Spitzer, Barbara and Solit, David B. and Dela Cruz, Filemon and LaQuaglia, Michael P. and Kushner, Brian H. and Modak, Shakeel and Shukla, Neerav and Iacobuzio-Donahue, Christine A. and Kung, Andrew L. and Cheung, Nai-Kong V. and Papaemmanuil, Elli},
  doi          = {10.1038/s41588-025-02320-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2338},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states. <em>NG</em>, <em>57</em>(9), 2323-2334. (<a href='https://doi.org/10.1038/s41588-025-02284-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell atlas projects have revealed that common cell types often comprise distinct, recurrent transcriptional states, but the function and regulation of these states remain poorly understood. Here, we show that systematic activation of transcription factors can recreate such states in vitro, providing tractable models for mechanistic and functional studies. Using a scalable CRISPR activation (CRISPRa) Perturb-seq platform, we activated 1,836 transcription factors in two cell types. CRISPRa induced gene expression within physiological ranges, with chromatin features predicting responsiveness. Comparisons with atlas datasets showed that transcription factor perturbations recapitulated key fibroblast states and identified their regulators, including KLF2 and KLF4 for a universal state present in many tissues, and PLAGL1 for a disease-associated inflammatory state. Inducing the universal state suppressed the inflammatory state, suggesting therapeutic potential. These findings position CRISPRa as a nuanced tool for perturbing differentiated cells and establish a general strategy for studying clinically relevant transcriptional states ex vivo. CRISPR activation of 1,836 human transcription factors recapitulates fibroblast transcriptional states observed in vivo and identifies regulators that can revert inflammatory states.},
  archive      = {J_NG},
  author       = {Southard, Kaden M. and Ardy, Rico C. and Tang, Anran and O’Sullivan, Deirdre D. and Metzner, Eli and Guruvayurappan, Karthik and Norman, Thomas M.},
  doi          = {10.1038/s41588-025-02284-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2323-2334},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize. <em>NG</em>, <em>57</em>(9), 2313-2322. (<a href='https://doi.org/10.1038/s41588-025-02246-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive maps of functional variation at transcription factor (TF) binding sites (cis-elements) are crucial for elucidating how genotype shapes phenotype. Here, we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped over 200,000 variants, genetic, epigenetic, or both (termed binding quantitative trait loci (bQTL)), linked to cis-element occupancy. Three lines of evidence support the functional significance of bQTL: (1) coincidence with causative loci that regulate traits, including vgt1, ZmTRE1 and the MITE transposon near ZmNAC111 under drought; (2) bQTL allelic bias is shared between inbred parents and matches chromatin immunoprecipitation sequencing results; and (3) partitioning genetic variation across genomic regions demonstrates that bQTL capture the majority of heritable trait variation across ~72% of 143 phenotypes. Our study provides an auspicious approach to make functional cis-variation accessible at scale for genetic studies and targeted engineering of complex traits. Pan-cistrome of the maize leaf under well-watered and drought conditions profiled by haplotype-specific MOA-seq highlights the relevance of transcription factor binding QTLs for understanding phenotypic diversity in maize.},
  archive      = {J_NG},
  author       = {Engelhorn, Julia and Snodgrass, Samantha J. and Kok, Amelie and Seetharam, Arun S. and Schneider, Michael and Kiwit, Tatjana and Singh, Ayush and Banf, Michael and Doan, Duong Thi Hai and Khaipho-Burch, Merritt and Runcie, Daniel E. and Sánchez-Camargo, Victor A. and Bader, Rechien and Vladimir Torres-Rodriguez, J. and Sun, Guangchao and Stam, Maike and Fiorani, Fabio and Beier, Sebastian and Schnable, James C. and Bass, Hank W. and Hufford, Matthew B. and Stich, Benjamin and Frommer, Wolf B. and Ross-Ibarra, Jeffrey and Hartwig, Thomas},
  doi          = {10.1038/s41588-025-02246-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2313-2322},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants. <em>NG</em>, <em>57</em>(9), 2302-2312. (<a href='https://doi.org/10.1038/s41588-025-02306-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clubroot disease, caused by the obligate intracellular rhizarian protist Plasmodiophora brassicae, is devastating to cruciferous crops worldwide. Widespread field P. brassicae pathotypes frequently overcome the pathotype-specific resistance of modern varieties, posing a challenge for durable control of this disease. Here a genome-wide association study of 3 years of data comprising field clubroot phenotyping of 244 genome-resequenced Brassica napus accessions identified a strong association of β-1,3-glucan synthase-like 5 (GSL5) with clubroot susceptibility. GSL5 was evolutionarily conserved, and inactivation of GSL5 by genome editing in Arabidopsis, B. napus, Brassica rapa and Brassica oleracea conferred broad-spectrum, high-level resistance to P. brassicae pathotypes without yield penalties in B. napus. GSL5 inactivation derepressed the jasmonic acid-mediated immunity during P. brassicae secondary infection, and this immune repression was possibly reinforced through stabilization of GSL5 by a P. brassicae effector, facilitating clubroot susceptibility. Our study provides durable resistance resources for cruciferous clubroot disease control and insights into plant resistance against intracellular eukaryotic phytopathogens. This study implicates GSL5 inactivation in high, broad-spectrum resistance to the clubroot pathogen Plasmodiophora brassicae in Arabidopsis thaliana, Brassica napus, Brassica oleracea and Brassica rapa.},
  archive      = {J_NG},
  author       = {Wu, Yupo and Zhao, Chuanji and Zhang, Yi and Shen, Cuicui and Zhang, Yuanyuan and Zhang, Xiong and Gao, Lixia and Zeng, Lingyi and Ke, Qinglin and Qin, Li and Liu, Fan and Huang, Junyan and Ren, Li and Liu, Yueying and Cheng, Hongtao and Tong, Chaobo and Hu, Qiong and Cheng, Xiaohui and Wei, Yangdou and Liu, Shengyi and Liu, Lijiang},
  doi          = {10.1038/s41588-025-02306-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2302-2312},
  shortjournal = {Nat. Genet.},
  title        = {Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism. <em>NG</em>, <em>57</em>(9), 2289-2301. (<a href='https://doi.org/10.1038/s41588-025-02293-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Making sense of whole-genome polymorphism data is challenging, but it is essential for overcoming the biases in SNP data. Here we analyze 27 genomes of Arabidopsis thaliana to illustrate these issues. Genome size variation is mostly due to tandem repeat regions that are difficult to assemble. However, while the rest of the genome varies little in length, it is full of structural variants, mostly due to transposon insertions. Because of this, the pangenome coordinate system grows rapidly with sample size and ultimately becomes 70% larger than the size of any single genome, even for n = 27. Finally, we show how short-read data are biased by read mapping. SNP calling is biased by the choice of reference genome, and both transcriptome and methylome profiling results are affected by mapping reads to a reference genome rather than to the genome of the assayed individual. New concepts for comparing the genomes of 27 naturally inbred Arabidopsis thaliana accessions provide essential insights into obtaining a less biased view of whole-genome polymorphism.},
  archive      = {J_NG},
  author       = {Igolkina, Anna A. and Vorbrugg, Sebastian and Rabanal, Fernando A. and Liu, Hai-Jun and Ashkenazy, Haim and Kornienko, Aleksandra E. and Fitz, Joffrey and Collenberg, Max and Kubica, Christian and Mollá Morales, Almudena and Jaegle, Benjamin and Wrightsman, Travis and Voloshin, Vitaly and Bezlepsky, Alexander D. and Llaca, Victor and Nizhynska, Viktoria and Reichardt, Ilka and Bezrukov, Ilja and Lanz, Christa and Bemm, Felix and Flood, Pádraic J. and Nemomissa, Sileshi and Hancock, Angela and Guo, Ya-Long and Kersey, Paul and Weigel, Detlef and Nordborg, Magnus},
  doi          = {10.1038/s41588-025-02293-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2289-2301},
  shortjournal = {Nat. Genet.},
  title        = {A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity. <em>NG</em>, <em>57</em>(9), 2276-2288. (<a href='https://doi.org/10.1038/s41588-025-02294-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common oat, belonging to the genus Avena with 30 recognized species, is a nutritionally important cereal crop and high-quality forage worldwide. Here, we construct a genus-level super-pangenome of Avena comprising 35 high-quality genomes from 14 cultivated oat accessions and 21 wild species. The fully resolved phylogenomic analysis unveils the origin and evolutionary scenario of Avena species, and the super-pangenome analysis identifies 26.62% and 59.93% specific genes and haplotypes in wild species. We delineate the landscape of structural variations (SVs) and the transcriptome profile based 1,401 RNA-sequencing (RNA-seq) samples from diverse abiotic stress treatments in oat. We highlight the crucial role of SVs in modulating gene expression and shaping adaptation to diverse stresses. Further combining SV-based genome-wide association studies (GWASs), we characterize 13 candidate genes associated with drought resistance such as AsARF7, validated by transgenic oat lines. Our study provides unprecedented genomic resources to facilitate genomic, evolution and molecular breeding research in oat. A genus-level super-pangenome of Avena comprising 35 high-quality genomes from 23 cultivated or wild oat species highlights the evolution of Avena and the landscape of structural variations related to abiotic stress resistance.},
  archive      = {J_NG},
  author       = {Zhang, Hongyu and Liu, Ningkun and Wang, Yaru and Zheng, Xinyuan and Li, Wei and Liu, Ze and Liu, Jianan and Wang, Yu and Xing, Longsheng and Li, Tao and Yun, Yange and Zhou, Qinghong and Wang, Meijia and Qin, Yujie and Yan, Jinjiang and Gong, Zhizhong and He, Qiang and Du, Huilong},
  doi          = {10.1038/s41588-025-02294-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2276-2288},
  shortjournal = {Nat. Genet.},
  title        = {Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts. <em>NG</em>, <em>57</em>(9), 2264-2275. (<a href='https://doi.org/10.1038/s41588-025-02330-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Locusts are characterized by a large genome size, polyphenism and an X0 sex determination system. Here we generated chromosome-level genomes for both desert and migratory locusts, as well as a comprehensive chromatin map for the latter. We found that genome enlargement is associated with an increased number of enhancers in expanded intronic regions. To explore the function of expanded enhancers, we identified a distal enhancer that contributes to behavioral differences between solitary and gregarious locusts. In the X0 sex system, H4K16ac enrichment and H4K20me1 depletion maintain balanced X-linked expression in male soma. Notably, the distance-dependent H4K16ac enrichment diminishes gradually in intergenic regions, revealing a special dosage compensation mechanism in large genomes. Furthermore, the distance-dependent H4K16ac results in a lag in the evolution of dosage compensation for the X-linked genes translocated recently from autosomes. Therefore, expanded intronic and intergenic regions shape a distinct chromatin regulation landscape in large genomes. Chromosome-level genome assemblies of migratory and desert locusts, coupled with epigenomic profiling of migratory locusts, reveal chromatin dynamics underlying polyphenism and X-linked dosage compensation following autosomal gene translocation.},
  archive      = {J_NG},
  author       = {Liu, Qing and Jiang, Feng and Li, Ran and Liu, Shanlin and Fang, Wenjing and Li, Xiao and Hu, Ting and Feng, Lianyun and Zhang, Jie and Liu, Zhikang and He, Jing and Guo, Wei and Wang, Xianhui and Sun, Zongyi and Li, Jingjing and Gao, Yunan and Yi, Jiacheng and Li, Qiye and Wang, Xiaoxiao and Wei, Liya and Kang, Le},
  doi          = {10.1038/s41588-025-02330-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2264-2275},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. <em>NG</em>, <em>57</em>(9), 2250-2263. (<a href='https://doi.org/10.1038/s41588-025-02305-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian switch/sucrose nonfermentable (mSWI/SNF) complex regulates chromatin accessibility and frequently shows alterations due to mutation in cancer and neurological diseases. Inadequate expression of mSWI/SNF in heterozygous mice can lead to developmental defects, indicating dosage-sensitive effects of mSWI/SNF. However, how its dosage affects function has remained unclear. Using a targeted protein degradation system, we investigated its dosage-sensitive effects by precisely controlling protein levels of BRG1, the ATPase subunit of the mSWI/SNF complex. We found that binding of BRG1 to chromatin exhibited a linear response to the BRG1 protein level. Although chromatin accessibility at most promoters and insulators was largely unaffected by BRG1 depletion, 44% of enhancers, including 84% of defined superenhancers, showed reduced accessibility. Notably, half of the BRG1-regulated enhancers, particularly superenhancers, exhibited a buffered response to BRG1 loss. Consistently, transcription exhibited a predominantly buffered response to changes in BRG1 levels. Collectively, our findings demonstrate a genomic feature-specific response to BRG1 dosage, shedding light on the dosage-sensitive effects of mSWI/SNF complex defects in cancer and other diseases. This study investigates the dosage-sensitive effects of BRG1, a core SWI/SNF subunit, on chromatin binding, accessibility and transcriptional regulation by precisely controlling its protein level.},
  archive      = {J_NG},
  author       = {Hagihara, Yota and Zhang, Chao and Zhang, Yi},
  doi          = {10.1038/s41588-025-02305-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2250-2263},
  shortjournal = {Nat. Genet.},
  title        = {Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue. <em>NG</em>, <em>57</em>(9), 2238-2249. (<a href='https://doi.org/10.1038/s41588-025-02300-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, TET1 and DNMT3A are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome. This multiomic study, including single-nucleus DNA methylation and chromatin conformation matched with single-nuclei RNA sequencing, provides insights into the epigenomic landscape of human subcutaneous adipose tissue.},
  archive      = {J_NG},
  author       = {Chen, Zeyuan Johnson and Das, Sankha Subhra and Kar, Asha and Lee, Seung Hyuk T. and Abuhanna, Kevin D. and Alvarez, Marcus and Sukhatme, Mihir G. and Wang, Zitian and Gelev, Kyla Z. and Heffel, Matthew G. and Zhang, Yi and Avram, Oren and Rahmani, Elior and Sankararaman, Sriram and Laakso, Markku and Heinonen, Sini and Peltoniemi, Hilkka and Halperin, Eran and Pietiläinen, Kirsi H. and Luo, Chongyuan and Pajukanta, Päivi},
  doi          = {10.1038/s41588-025-02300-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2238-2249},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution. <em>NG</em>, <em>57</em>(9), 2226-2237. (<a href='https://doi.org/10.1038/s41588-025-02307-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients. Integrated multi-omic analyses using samples from the TRACERx study highlight cross-talk between DNA hypermethylation and genomic lesions in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Gimeno-Valiente, Francisco and Castignani, Carla and Larose Cadieux, Elizabeth and Mensah, Nana E. and Liu, Xiaohong and Chen, Kezhong and Chervova, Olga and Karasaki, Takahiro and Weeden, Clare E. and Richard, Corentin and Lai, Siqi and Martínez-Ruiz, Carlos and Lim, Emilia L. and Frankell, Alexander M. and Watkins, Thomas B. K. and Stavrou, Georgia and Usaite, Ieva and Lu, Wei-Ting and Marinelli, Daniele and Saghafinia, Sadegh and Wilson, Gareth A. and Dhami, Pawan and Vaikkinen, Heli and Steif, Jonathan and Veeriah, Selvaraju and Hynds, Robert E. and Hirst, Martin and Hiley, Crispin and Feber, Andrew and Deniz, Özgen and Jamal-Hanjani, Mariam and McGranahan, Nicholas and Beck, Stephan and Demeulemeester, Jonas and Tanić, Miljana and Swanton, Charles and Van Loo, Peter and Kanu, Nnennaya},
  doi          = {10.1038/s41588-025-02307-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2226-2237},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis. <em>NG</em>, <em>57</em>(9), 2215-2225. (<a href='https://doi.org/10.1038/s41588-025-02296-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in PPM1D and the TERT gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers. This study explores the relationship between telomere length and clonal hematopoiesis. Splicing factor and PPM1D gene mutations are more frequent in people with genetically predicted shorter telomere lengths, suggesting that these mutations protect against the consequences of telomere attrition.},
  archive      = {J_NG},
  author       = {McLoughlin, Matthew A. and Cheloor Kovilakam, Sruthi and Dunn, William G. and Gu, Muxin and Tobin, Jake and Pershad, Yash and Williams, Nicholas and Leongamornlert, Daniel and Dawson, Kevin and Bond, Laura and Marando, Ludovica and Wen, Sean and Wilson, Rachael and Valenzano, Giampiero and Symeonidou, Vasiliki and Rak, Justyna and Damaskou, Aristi and Gozdecka, Malgorzata and Liu, Xiaoxuan and Barcena, Clea and Nomdedeu, Josep and Costeas, Paul and Dimitriou, Ioannis D. and Fiorillo, Edoardo and Orrù, Valeria and de Almeida, Jose Guilherme and McKerrell, Thomas and Cullen, Matthew and Mohorianu, Irina and Foukaneli, Theodora and Warren, Alan J. and Wong, Chi and Follows, George and Godfrey, Anna L. and Gudgin, Emma and Cucca, Francesco and McKinney, Eoin and Baxter, E. Joanna and Gerstung, Moritz and Mitchell, Jonathan and Wiseman, Daniel and Bick, Alexander G. and Fabre, Margarete and Quiros, Pedro M. and Nangalia, Jyoti and Kar, Siddhartha and Vassiliou, George S.},
  doi          = {10.1038/s41588-025-02296-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2215-2225},
  shortjournal = {Nat. Genet.},
  title        = {Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Temporal genomic dynamics shape clinical trajectory in multiple myeloma. <em>NG</em>, <em>57</em>(9), 2203-2214. (<a href='https://doi.org/10.1038/s41588-025-02292-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple myeloma evolution is characterized by the accumulation of genomic drivers over time. To unravel this timeline and its impact on clinical outcomes, we analyzed 421 whole-genome sequences from 382 patients. Using clock-like mutational signatures, we estimated a time lag of two to four decades between the initiation of events and diagnosis. We demonstrate that odd-numbered chromosome trisomies in patients with hyperdiploidy can be acquired simultaneously with other chromosomal gains (for example, 1q gain). We show that hyperdiploidy is acquired after immunoglobulin heavy chain translocation when both events co-occur. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (&gt;1 extra copy), but fared worse than those with late 1q gain. This finding underscores that the 1q gain prognostic impact depends more on the timing of acquisition than on the number of copies gained. Overall, this study contributes to a better understanding of the life history of myeloma and may have prognostic implications. This analysis of whole-genome sequencing data from 421 multiple myeloma samples elucidates the timing of key genomic events and shows associations between the timing of 1q gain and clinical outcome.},
  archive      = {J_NG},
  author       = {Maura, Francesco and Kaddoura, Marcella and Poos, Alexandra M. and Baughn, Linda B. and Ziccheddu, Bachisio and Bärtsch, Marc-Andrea and Cirrincione, Anthony and Maclachlan, Kylee and Chojnacka, Monika and Diamond, Benjamin and Papadimitriou, Marios and Blaney, Patrick and John, Lukas and Reichert, Philipp and Huhn, Stefanie and Gagler, Dylan and Zhang, Yanming and Dogan, Ahmet and Lesokhin, Alexander M. and Davies, Faith and Goldschmidt, Hartmut and Fenk, Roland and Weisel, Katja C. and Mai, Elias K. and Korde, Neha and Morgan, Gareth J. and Rajkumar, S. Vincent and Kumar, Shaji and Usmani, Saad and Landgren, Ola and Raab, Marc S. and Weinhold, Niels},
  doi          = {10.1038/s41588-025-02292-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2203-2214},
  shortjournal = {Nat. Genet.},
  title        = {Temporal genomic dynamics shape clinical trajectory in multiple myeloma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tamoxifen induces PI3K activation in uterine cancer. <em>NG</em>, <em>57</em>(9), 2192-2202. (<a href='https://doi.org/10.1038/s41588-025-02308-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutagenic processes and clonal selection contribute to the development of therapy-associated secondary neoplasms, a known complication of cancer treatment. The association between tamoxifen therapy and secondary uterine cancers is uncommon but well established; however, the genetic mechanisms underlying tamoxifen-driven tumorigenesis remain unclear. We find that oncogenic PIK3CA mutations, common in spontaneously arising estrogen-associated de novo uterine cancer, are significantly less frequent in tamoxifen-associated tumors. In vivo, tamoxifen-induced estrogen receptor stimulation activates phosphoinositide 3-kinase (PI3K) signaling in normal mouse uterine tissue, potentially eliminating the selective benefit of PI3K-activating mutations in tamoxifen-associated uterine cancer. Together, we present a unique pathway of therapy-associated carcinogenesis in which tamoxifen-induced activation of the PI3K pathway acts as a non-genetic driver event, contributing to the multistep model of uterine carcinogenesis. While this PI3K mechanism is specific to tamoxifen-associated uterine cancer, the concept of treatment-induced signaling events may have broader applicability to other routes of tumorigenesis. Sequencing analysis of tamoxifen-associated uterine cancers and further in vivo analyses suggest that the drug tamoxifen can activate the PI3K pathway in the absence of oncogenic mutations.},
  archive      = {J_NG},
  author       = {Kübler, Kirsten and Nardone, Agostina and Anand, Shankara and Gurevich, Daniel and Gao, Jianjiong and Droog, Marjolein and Hermida-Prado, Francisco and Akhshi, Tara and Feiglin, Ariel and Feit, Avery S. and Cohen Feit, Gabriella and Dackus, Gwen and Pun, Matthew and Kuang, Yanan and Cha, Justin and Miller, Mendy and Gregoricchio, Sebastian and Lanfermeijer, Mirthe and Cornelissen, Sten and Gibson, William J. and Paweletz, Cloud P. and Van Allen, Eliezer M. and van Leeuwen, Flora E. and Nederlof, Petra M. and Nguyen, Quang-Dé and Mourits, Marian J. E. and Radovich, Milan and Leshchiner, Ignaty and Stewart, Chip and Matulonis, Ursula A. and Zwart, Wilbert and Maruvka, Yosef E. and Getz, Gad and Jeselsohn, Rinath},
  doi          = {10.1038/s41588-025-02308-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2192-2202},
  shortjournal = {Nat. Genet.},
  title        = {Tamoxifen induces PI3K activation in uterine cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L. <em>NG</em>, <em>57</em>(9), 2177-2191. (<a href='https://doi.org/10.1038/s41588-025-02289-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the high prevalence of ERG transcription factor translocations in prostate cancer, the mechanism of tumorigenicity remains poorly understood. Using lineage tracing, we find the tumor-initiating activity of ERG resides in a subpopulation of murine basal cells that coexpress luminal genes (BasalLum) and not in the larger population of ERG+ luminal cells. Upon ERG activation, BasalLum cells give rise to highly proliferative intermediate (IM) cells with stem-like features that coexpress basal, luminal, hillock and club marker genes, before transitioning to Krt8+ luminal cells. Transcriptomic analysis of ERG+ human prostate cancers confirms the presence of rare ERG+ BasalLum cells, as well as IM cells whose presence is associated with a worse prognosis. Single-cell analysis revealed a chromatin state in ERG+ IM cells enriched for STAT3 transcription factor binding sites and elevated expression of the KMT2A/MLL1 and DOT1L, all three of which are essential for ERG-driven tumorigenicity in vivo. In addition to providing translational opportunities, this work illustrates how single-cell approaches combined with lineage tracing can identify cancer vulnerabilities not evident from bulk analysis. Lineage tracing in mice identifies a subpopulation of basal cells that express Tmprss2 and Nkx3 as the origin of ERG-driven prostate cancer. Upon expansion, these cells show an enrichment for STAT3 chromatin binding and elevated expression of KMT2A and DOT1L as dependencies for ERG oncogenicity.},
  archive      = {J_NG},
  author       = {Feng, Weiran and Ladewig, Erik and Lange, Matthew and Salsabeel, Nazifa and Zhao, Huiyong and Lee, Young Sun and Gopalan, Anuradha and Luo, Hanzhi and Kang, Wenfei and Fan, Ning and Rosiek, Eric and de Stanchina, Elisa and Chen, Yu and Carver, Brett S. and Leslie, Christina S. and Sawyers, Charles L.},
  doi          = {10.1038/s41588-025-02289-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2177-2191},
  shortjournal = {Nat. Genet.},
  title        = {ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutational landscape of triple-negative breast cancer in african american women. <em>NG</em>, <em>57</em>(9), 2166-2176. (<a href='https://doi.org/10.1038/s41588-025-02322-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {African American (AA) women have the highest incidence of triple-negative breast cancer (TNBC) among all racial groups, but are underrepresented in cancer genomic studies. In 462 AA women with TNBC, we characterized the tumor mutational landscape by whole-exome sequencing and RNA sequencing. We unveiled a high-resolution mutational portrait of TNBC in AA women reminiscent of that in Asian and non-Hispanic white women, with no evidence of associations of mutational features with African ancestry. We also made some distinctive discoveries, including an almost complete dominance of TP53 mutations, low frequency of PIK3CA mutations and mutational signature-based subtypes with etiologic and prognostic significance. These findings do not support major racial differences in TNBC biology at the level of somatic mutations. Our study contributes considerably to diversifying the knowledge base of breast cancer genomics and provides insights into the disease etiology, disparities and therapeutic vulnerability of TNBC in AA women. This study explores the genomic and transcriptomic landscapes of triple-negative breast cancer in African American women. The authors show that the mutational profile is broadly similar to that observed in European and East Asian ancestry women while highlighting some interesting differences.},
  archive      = {J_NG},
  author       = {Yao, Song and Wei, Lei and Hu, Qiang and Liu, Song and Manojlovic, Zarko and Fiorica, Peter N. and Long, Mark and Zirpoli, Gary R. and Cai, Qiuyin and Long, Jirong and Ping, Jie and Barnard, Mollie E. and Jin, Yuxin and Murakami, Mitsuko and Wang, Jianmin and Zhu, Qianqian and Davis, Warren and Chen, Jianhong and Ondracek, Rochelle P. and Khoury, Thaer and Gandhi, Shipra and Takabe, Kazuaki and Ko, Naomi and Sanderson, Maureen and Hong, Chi-Chen and Bandera, Elisa V. and Craig, David W. and Ambrosone, Christine B. and Palmer, Julie R. and Zheng, Wei and Carpten, John D.},
  doi          = {10.1038/s41588-025-02322-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2166-2176},
  shortjournal = {Nat. Genet.},
  title        = {Mutational landscape of triple-negative breast cancer in african american women},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions. <em>NG</em>, <em>57</em>(9), 2156-2165. (<a href='https://doi.org/10.1038/s41588-025-02291-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand shared and ancestry-specific genetic control of brain protein expression and its ramifications for disease, we mapped protein quantitative trait loci (pQTLs) in 1,362 brain proteomes from African American, Hispanic/Latin American and non-Hispanic white donors. Among the pQTLs that multiancestry fine-mapping MESuSiE confidently assigned as putative causal pQTLs in a specific population, most were shared across the three studied populations and are referred to as multiancestry causal pQTLs. These multiancestry causal pQTLs were enriched for exonic and promoter regions. To investigate their effects on disease, we modeled the 858 multiancestry causal pQTLs as instrumental variables using Mendelian randomization and genome-wide association study results for neurologic and psychiatric conditions (21 traits in participants with European ancestry, 10 in those with African ancestry and 4 in Hispanic participants). We identified 119 multiancestry pQTL–protein pairs consistent with a causal role in these conditions. Remarkably, 29% of the multiancestry pQTLs in these pairs were coding variants. These results lay an important foundation for the creation of new molecular models of neurologic and psychiatric conditions that are likely to be relevant to individuals across different genetic ancestries. Multiancestry fine-mapping of brain protein quantitative trait loci coupled with Mendelian randomization analyses identifies protein–trait pairs consistent with causal effects across neurological and psychiatric conditions.},
  archive      = {J_NG},
  author       = {Wingo, Aliza P. and Liu, Yue and Vattathil, Selina M. and Gerasimov, Ekaterina S. and Mei, Zhen and Ravindran, Suda Parimala and Liu, Jiaqi and Shantaraman, Ananth and Seifar, Fatemeh and Wang, Erming and Zhang, Bin and Reddy, Joseph and Allen, Mariet and Ertekin-Taner, Nilüfer and De Jager, Philip L. and Fox, Edward J. and Duong, Duc M. and Epstein, Michael P. and Cutler, David J. and Levey, Allan I. and Bennett, David A. and Seyfried, Nicholas T. and Wingo, Thomas S.},
  doi          = {10.1038/s41588-025-02291-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2156-2165},
  shortjournal = {Nat. Genet.},
  title        = {Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes. <em>NG</em>, <em>57</em>(9), 2146-2155. (<a href='https://doi.org/10.1038/s41588-025-02288-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks with whole-genome sequencing (WGS) now enable the association of noncoding rare variants with complex human traits. Given that &gt;98% of the genome is available for exploration, the selection of noncoding variants remains a critical yet unresolved challenge in these analyses. Here we leverage knowledge of blood gene regulation and deleteriousness scores to select noncoding variants pertinent for association with blood-related traits. Integrating WGS and 42 blood cell count and biomarker measurements for 166,740 UK Biobank samples, we perform variant collapsing tests, identifying hundreds of gene–trait associations involving noncoding variants. However, we demonstrate that most of these noncoding rare variant associations (1) reproduce associations known from previous studies and (2) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting noncoding rare variant association results. Noncoding rare variant analyses using whole-genome sequencing data from the UK Biobank identify gene–trait associations for 42 blood cell traits but find that most signals are driven by linkage disequilibrium between common and rare variants.},
  archive      = {J_NG},
  author       = {Ribeiro, Diogo M. and Hofmeister, Robin J. and Rubinacci, Simone and Delaneau, Olivier},
  doi          = {10.1038/s41588-025-02288-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2146-2155},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores. <em>NG</em>, <em>57</em>(9), 2136-2145. (<a href='https://doi.org/10.1038/s41588-025-02298-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic health record (EHR)-based phenotype risk scores (PheRS) leverage individuals’ health trajectories to estimate disease risk, similar to how polygenic scores (PGS) use genetic information. While PGS generalizability has been studied, less is known about PheRS generalizability across healthcare systems and whether PheRS are complementary to PGS. We trained elastic-net-based PheRS to predict the onset of 13 common diseases for 845,929 individuals (age = 32–70 years) from three biobank-based studies in Finland (FinnGen), the UK (UKB) and Estonia (EstB). All PheRS were statistically significantly associated with the diseases of interest and most generalized well without retraining when applied to other studies. PheRS and PGS were only moderately correlated and models including both predictors improved onset prediction compared to PGS alone for 8 of 13 diseases. Our results indicate that EHR-based risk scores can transfer well between EHRs, capture largely independent information from PGS, and provide additive benefits for disease risk prediction. Comparison of electronic health record-based phenotype risk scores (PheRS) and polygenic scores (PGS) across 13 common diseases and three biobank-based studies indicates that PheRS and PGS may provide additive benefits for risk prediction.},
  archive      = {J_NG},
  author       = {Detrois, Kira E. and Hartonen, Tuomo and Teder-Laving, Maris and Jermy, Bradley and Läll, Kristi and Yang, Zhiyu and Mägi, Reedik and Ripatti, Samuli and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02298-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2136-2145},
  shortjournal = {Nat. Genet.},
  title        = {Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes. <em>NG</em>, <em>57</em>(9), 2124-2135. (<a href='https://doi.org/10.1038/s41588-025-02317-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Ancestral Recombination Graph (ARG), which describes the genealogical history of a sample of genomes, is a vital tool in population genomics and biomedical research. Recent advancements have substantially increased ARG reconstruction scalability, but they rely on approximations that can reduce accuracy, especially under model misspecification. Moreover, they reconstruct only a single ARG topology and cannot quantify the considerable uncertainty associated with ARG inferences. Here, to address these challenges, we introduce SINGER (sampling and inferring of genealogies with recombination), a method that accelerates ARG sampling from the posterior distribution by two orders of magnitude, enabling accurate inference and uncertainty quantification for hundreds of whole-genome sequences. Through extensive simulations, we demonstrate SINGER’s enhanced accuracy and robustness to model misspecification compared to existing methods. We demonstrate the utility of SINGER by applying it to individuals of British and African descent within the 1000 Genomes Project, identifying signals of population differentiation, archaic introgression and strong support for ancient polymorphism in the human leukocyte antigen region shared across primates. SINGER is a method for creating ancestral recombination graphs to understand the genealogical history of genomes. The method has increased speed, and thus scalability, without sacrificing accuracy.},
  archive      = {J_NG},
  author       = {Deng, Yun and Nielsen, Rasmus and Song, Yun S.},
  doi          = {10.1038/s41588-025-02317-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2124-2135},
  shortjournal = {Nat. Genet.},
  title        = {Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. <em>NG</em>, <em>57</em>(9), 2116-2123. (<a href='https://doi.org/10.1038/s41588-025-02286-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mixed-model association analysis (MMAA) is the preferred tool for performing genome-wide association studies. However, existing MMAA tools often have long runtimes and high memory requirements. Here we present LDAK-KVIK, an MMAA tool for analysis of quantitative and binary phenotypes. LDAK-KVIK is computationally efficient, requiring less than 10 CPU hours and 5 Gb memory to analyze genome-wide data for 350,000 individuals. Using simulated phenotypes, we show that LDAK-KVIK produces well-calibrated test statistics for both homogeneous and heterogeneous datasets. When applied to real phenotypes, LDAK-KVIK has the highest power among all tools considered. For example, across 40 quantitative UK Biobank phenotypes (average sample size 349,000), LDAK-KVIK finds 16% more independent, genome-wide significant loci than classical linear regression, whereas BOLT-LMM and REGENIE find 15% and 11% more, respectively. LDAK-KVIK can also be used to perform gene-based tests; across the 40 quantitative UK Biobank phenotypes, LDAK-KVIK finds 18% more significant genes than the leading existing tool. Last, LDAK-KVIK produces state-of-the-art polygenic scores. LDAK-KVIK is a mixed-model association method for genome-wide studies that optimizes computational performance and power. LDAK-KVIK can also perform gene-based tests and produces state-of-the-art polygenic scores.},
  archive      = {J_NG},
  author       = {Hof, Jasper P. and Speed, Doug},
  doi          = {10.1038/s41588-025-02286-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2116-2123},
  shortjournal = {Nat. Genet.},
  title        = {LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward improving multiomic research in understudied cereals. <em>NG</em>, <em>57</em>(9), 2106-2115. (<a href='https://doi.org/10.1038/s41588-025-02245-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the looming challenges of climate change and population pressure, understudied cereals hold notably untapped potential for future global food security and sustainable agriculture due to their unique nutritional profiles and notable stress resilience. Multiomics holds great promise in realizing the value of understudied cereals by understanding the genetic basis and molecular mechanisms regulating their unique traits; however, multiomic studies of understudied cereals still lag behind those of major cereals, hindering their genetic improvement and wide adoption in modern agriculture. Here, we discuss the limitations and challenges in multiomic studies of understudied cereals and how they can be overcome to enhance utilization of the traits unique to understudied cereals in improving both themselves and major cereals. We also offer a database specific to understudied cereals, including multiomics data from 12 understudied cereals and wild relatives for some of these, to serve the community. Understudied cereals offer untapped potential for food security, yet multiomic research substantially lags behind that of major cereals. This Perspective discusses these challenges and introduces a dedicated database to serve the community.},
  archive      = {J_NG},
  author       = {Yang, Fanjing and Gong, Xiaojiao and Zhao, Pan and Cheng, Lingtong and Zhang, Yujie and Luo, Yingting and Zhu, Qian-Hao and Ye, Chu-Yu},
  doi          = {10.1038/s41588-025-02245-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2106-2115},
  shortjournal = {Nat. Genet.},
  title        = {Toward improving multiomic research in understudied cereals},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The LISTEN principles for genetic sequence data governance and database engineering. <em>NG</em>, <em>57</em>(9), 2099-2105. (<a href='https://doi.org/10.1038/s41588-025-02270-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several international legal agreements include an ‘access and benefit-sharing’ (ABS) mechanism that attaches obligations to the use of genetic sequence data. These agreements are frequently subject to critique on the grounds that ABS is either fundamentally incompatible with the principles of open science, or technically challenging to implement in open scientific databases. Here, we argue that these critiques arise from a misinterpretation of the principles of open science and that both considerations can be addressed by a set of simple principles that link database engineering and governance. We introduce a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, which can be readily adopted by both new and existing platforms participating in ABS systems. We also highlight how these principles can act in concert with familiar principles of open science, such as findable, accessible, interoperable and reusable (FAIR) data sharing. This Perspective proposes a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, aimed at ensuring access and benefit-sharing principles in open science.},
  archive      = {J_NG},
  author       = {Carlson, Colin J. and Granados, Monica and Phelan, Alexandra and Ramakrishnan, Nithin and Poisot, Timothée},
  doi          = {10.1038/s41588-025-02270-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2099-2105},
  shortjournal = {Nat. Genet.},
  title        = {The LISTEN principles for genetic sequence data governance and database engineering},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Governance of cross-border genomic data sharing through a human rights approach. <em>NG</em>, <em>57</em>(9), 2090-2098. (<a href='https://doi.org/10.1038/s41588-025-02252-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cross-border genomic data sharing is crucial for advancing research progress, understanding of genetic diversity and international public health. However, this practice also leads to concerns regarding biosecurity, individual privacy and regulatory conflicts. Scientific communities often advocate data sovereignty approaches to address these challenges, yet related frameworks are often inadequate in their ability to address such challenges and have an overemphasis on national control and protectionism. They also may not fully consider the inherently transnational nature of genomic data and the need for a governance system that reflects the shared human heritage of the data and global interconnectivity. Instead, we propose here a human rights-based approach that surpasses territorial boundaries to promote shared responsibility and equitable research practices. A balanced governance framework that incorporates extraterritorial obligations and aligns data policy with human rights principles can safeguard individual privacy and biosecurity, encourage international collaboration and support ethically responsible genomic research. This Perspective proposes a human rights-based governance framework for cross-border genomic data sharing, addressing limitations of data sovereignty approaches while balancing privacy, security and global research collaboration.},
  archive      = {J_NG},
  author       = {Yu, Liang and Feng, Ruohan and Sun, Youhai and Peng, Yaojin},
  doi          = {10.1038/s41588-025-02252-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2090-2098},
  shortjournal = {Nat. Genet.},
  title        = {Governance of cross-border genomic data sharing through a human rights approach},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recreating fibroblast diversity in vitro by activating transcription factors. <em>NG</em>, <em>57</em>(9), 2088-2089. (<a href='https://doi.org/10.1038/s41588-025-02283-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We combined CRISPR activation of all 1,836 known human transcription factors with high-throughput Perturb-seq to recreate the diverse transcriptional states occupied by fibroblasts in vivo. Our study revealed regulators of key states and showed that inducing normal states can, in some cases, suppress those linked to disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02283-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2088-2089},
  shortjournal = {Nat. Genet.},
  title        = {Recreating fibroblast diversity in vitro by activating transcription factors},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improving broad-spectrum resistance to clubroot disease with one gene. <em>NG</em>, <em>57</em>(9), 2086-2087. (<a href='https://doi.org/10.1038/s41588-025-02304-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GSL5, a glucan synthase, acts as a suppressor of jasmonic acid-mediated immunity in cruciferous plants. Inactivation of GSL5 by genome editing confers high-level and broad-spectrum resistance to pathogens that cause clubroot disease in four cruciferous species.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02304-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2086-2087},
  shortjournal = {Nat. Genet.},
  title        = {Improving broad-spectrum resistance to clubroot disease with one gene},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mystery of tamoxifen-associated uterine cancer uncovered. <em>NG</em>, <em>57</em>(9), 2084-2085. (<a href='https://doi.org/10.1038/s41588-025-02316-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tamoxifen is an essential drug in breast cancer therapy. Unlike prevailing models of therapy-related tumorigenesis, tamoxifen acts by directly activating the PI3K pathway, bypassing the need for mutations in one of the most common driver genes in sporadic uterine cancer. These findings open avenues for investigating similar mechanisms in other drugs.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02316-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2084-2085},
  shortjournal = {Nat. Genet.},
  title        = {Mystery of tamoxifen-associated uterine cancer uncovered},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A bivalent chromatin state regulates the function of composite transposons. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02337-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02337-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {A bivalent chromatin state regulates the function of composite transposons},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting obesity from childhood. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02338-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02338-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Predicting obesity from childhood},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Building immune tolerance in the thymus. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02339-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02339-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Building immune tolerance in the thymus},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI identifies promoter variants that alter gene expression. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02340-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02340-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {AI identifies promoter variants that alter gene expression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease. <em>NG</em>, <em>57</em>(9), 2079-2082. (<a href='https://doi.org/10.1038/s41588-025-02299-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite Africa’s vast genetic diversity, its populations are underrepresented in global genomic datasets. Here we describe the vision of the KidneyGenAfrica, a pan-African initiative launched to address this inequity, and call for more inclusive genomics research that recognizes Africa’s key role in genetic variation and potential to generate insights in chronic kidney disease.},
  archive      = {J_NG},
  author       = {Fatumo, Segun and Ojewunmi, Oyesola and Camenzuli, Rebecca and Kintu, Christopher and Kamiza, Abram and Brandenburg, Jean-Tristan and Kalungi, Allan and Kalyesubula, Robert and Salako, Babatunde and Nash, Oyekanmi and Crampin, Amelia and Adeyemo, Adebowale and Nyirenda, Moffat and Ramsay, Michele and Fabian, June},
  doi          = {10.1038/s41588-025-02299-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2079-2082},
  shortjournal = {Nat. Genet.},
  title        = {KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The future of polygenic risk scores in direct-to-consumer genomics. <em>NG</em>, <em>57</em>(9), 2077-2078. (<a href='https://doi.org/10.1038/s41588-025-02313-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Sun, Yuhao and Vines, John and Tenesa, Albert},
  doi          = {10.1038/s41588-025-02313-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2077-2078},
  shortjournal = {Nat. Genet.},
  title        = {The future of polygenic risk scores in direct-to-consumer genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmat">NMAT - 30</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Ab initio structure solutions from nanocrystalline powder diffraction data via diffusion models. <em>NMAT</em>, <em>24</em>(9), 1502. (<a href='https://doi.org/10.1038/s41563-025-02301-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMAT},
  author       = {Guo, Gabe and Saidi, Tristan Luca and Terban, Maxwell W. and Valsecchi, Michele and Billinge, Simon J. L. and Lipson, Hod},
  doi          = {10.1038/s41563-025-02301-y},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1502},
  shortjournal = {Nat. Mater.},
  title        = {Author correction: Ab initio structure solutions from nanocrystalline powder diffraction data via diffusion models},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hybrid hydrogel–extracellular matrix scaffolds identify biochemical and mechanical signatures of cardiac ageing. <em>NMAT</em>, <em>24</em>(9), 1489-1501. (<a href='https://doi.org/10.1038/s41563-025-02234-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extracellular matrix remodelling of cardiac tissue is a key contributor to age-related cardiovascular disease and dysfunction. Such remodelling is multifaceted including changes to the biochemical composition, architecture and mechanics, clouding our understanding of how and which extracellular matrix properties contribute to a dysfunctional state. Here we describe a decellularized extracellular matrix–synthetic hydrogel hybrid scaffold that independently confers two distinct matrix properties—ligand presentation and stiffness—to cultured cells in vitro, allowing for the identification of their specific roles in cardiac ageing. The hybrid scaffold maintains native matrix composition and organization of young or aged murine cardiac tissue, whereas its mechanical properties can be independently tuned to mimic young or aged tissue stiffness. Seeding these scaffolds with murine primary cardiac fibroblasts, we identify distinct age- and matrix-dependent mechanisms of cardiac fibroblast activation, matrix remodelling and senescence. Importantly, we show that the ligand presentation of a young extracellular matrix can outweigh the profibrotic stiffness cues typically present in an aged extracellular matrix in maintaining or driving cardiac fibroblast quiescence. Ultimately, these tunable scaffolds can enable the discovery of specific extracellular targets to prevent ageing dysfunction and promote rejuvenation. A hybrid extracellular matrix–hydrogel with tunable mechanical stiffness and biochemical composition of young or aged cardiac tissue is used to identify the specific contributions of extracellular matrix ligands and mechanics for fibroblast aging.},
  archive      = {J_NMAT},
  author       = {Sun, Avery Rui and Ramli, Md Faris H. and Shen, Xingyu and Kannivadi Ramakanth, Karthikbabu and Chen, Dixiao and Hu, Yang and Vidyasekar, Prasanna and Foo, Roger S. and Long, Yuchen and Zhu, Jin and Ackers-Johnson, Matthew and Young, Jennifer L.},
  doi          = {10.1038/s41563-025-02234-6},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1489-1501},
  shortjournal = {Nat. Mater.},
  title        = {Hybrid hydrogel–extracellular matrix scaffolds identify biochemical and mechanical signatures of cardiac ageing},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cytoplasmic anillin and ect2 promote RhoA/myosin II-dependent confined migration and invasion. <em>NMAT</em>, <em>24</em>(9), 1476-1488. (<a href='https://doi.org/10.1038/s41563-025-02269-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell migration in mechanically confined environments is a crucial step of metastatic cancer progression. Nonetheless, the molecular components and processes mediating such behaviour are still not fully understood. Here we demonstrate that a pool of the scaffolding protein anillin and its cofactor Ect2, which are both predominantly nuclear proteins and critical mediators of cytokinesis, is present in the cytoplasm of multiple interphase cell types that promote confined cell migration. Confined migration in biomimetic microfluidic models triggers the actomyosin-binding-dependent recruitment of anillin to the plasma membrane at the poles of migrating cells in a manner that scales with microenvironmental stiffness and confinement. The guanine nucleotide exchange activity of Ect2 is required for its RhoA-GTPase-mediated activation of myosin II at the cell poles, enhancing invasion, bleb-based migration and extravasation. Confinement-induced nuclear envelope rupture further amplifies this process due to the release of further anillin and Ect2 into the cytoplasm. Overall, these results show how Ect2 and anillin cooperate to mediate RhoA/ROCK/myosin II-dependent mechanoadaptation and invasive cancer progression. Cell migration in confined environments is initiated by a cytoplasmic pool of anillin and Ect2 that promotes RhoA/myosin II-mediated activation at the poles of migrating cells, in a process dependent on the extracellular environment stiffness.},
  archive      = {J_NMAT},
  author       = {Tran, Avery T. and Wisniewski, Emily O. and Mistriotis, Panagiotis and Stoletov, Konstantin and Parlani, Maria and Amitrano, Alice and Ifemembi, Brent and Lee, Se Jong and Bera, Kaustav and Zhang, Yuqi and Tuntithavornwat, Soontorn and Afthinos, Alexandros and Kiepas, Alexander and Agarwal, Bhawana and Nath, Sanjiban and Jamieson, John J. and Zuo, Yi and Habib, Daniel and Wu, Pei-Hsun and Martin, Stuart S. and Gerecht, Sharon and Gu, Luo and Lewis, John D. and Kalab, Petr and Friedl, Peter and Konstantopoulos, Konstantinos},
  doi          = {10.1038/s41563-025-02269-9},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1476-1488},
  shortjournal = {Nat. Mater.},
  title        = {Cytoplasmic anillin and ect2 promote RhoA/myosin II-dependent confined migration and invasion},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Adaptive peptide dispersions enable drying-induced biomolecule encapsulation. <em>NMAT</em>, <em>24</em>(9), 1465-1475. (<a href='https://doi.org/10.1038/s41563-025-02300-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peptides are promising building blocks of designer materials with wide-ranging applications. These materials are stabilized by directional hydrogen-bonding patterns, giving rise to one-dimensional or two-dimensional assembly. It remains a challenge to mimic biology’s context-adaptive and flexible structures. Here we introduce minimalistic tripeptide sequences that form highly soluble dynamic ensembles through multivalent side-chain interactions. We observe these supramolecular dispersions undergo drying-induced sequential liquid–liquid phase separation followed by solidification, resulting in the formation of films of stiff, densely packed and porous peptide microparticles that can be instantaneously redispersed upon the re-introduction of water. Air-drying of peptide dispersions in the presence of proteins or small-molecule payloads results in efficient encapsulation and the retention of protein stability after redispersion, showing promise for the emulsification, encapsulation, protection and storage of biomacromolecules. The mechanism resembles the protective strategies in natural systems during desiccation, which rely on liquid–liquid phase separation to survive extreme conditions. Here the authors design tripeptides that form dynamic soluble dispersions and undergo phase separation upon drying to assemble into porous particles. This evaporation-driven emulsification can be harnessed to encapsulate and stabilize biomolecules.},
  archive      = {J_NMAT},
  author       = {Dave, Dhwanit R. and Kassem, Salma and Coste, Maeva and Xu, Lele and Tayarani-Najjaran, Mona and Podbevšek, Darjan and Colon-De Leon, Paola and Zhang, Sheng and Ortuno Macias, Luis and Sementa, Deborah and Pérez-Ferreiro, María and Ayati, Nooshin Sadat and Choudhury, Muniyat A. and Veerasammy, Kelly and Doganata, Selma and Zhong, Tiffany and Weng, Cory and Morales, Jorge and Favaro, Denize C. and Marianski, Mateusz and Ahn, So Yeon and Obermeyer, Allie C. and Wang, Tong and Li, Tai-De and Chen, Xi and Tu, Raymond and He, Ye and Ulijn, Rein V.},
  doi          = {10.1038/s41563-025-02300-z},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1465-1475},
  shortjournal = {Nat. Mater.},
  title        = {Adaptive peptide dispersions enable drying-induced biomolecule encapsulation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unveiling high-mobility hot carriers in a two-dimensional conjugated coordination polymer. <em>NMAT</em>, <em>24</em>(9), 1457-1464. (<a href='https://doi.org/10.1038/s41563-025-02246-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hot carriers, inheriting excess kinetic energy from high-energy photons, drive numerous optoelectronic applications reliant on non-equilibrium transport processes. Although extensively studied in inorganic materials, their potential in organic-based systems remains largely unexplored. Here we demonstrate highly mobile hot carriers in crystalline two-dimensional conjugated coordination polymer Cu3BHT (BHT, benzenehexathiol) films. Leveraging a suite of ultrafast spectroscopic and imaging techniques, we map the microscopic charge transport landscape in Cu3BHT films following non-equilibrium photoexcitation across temporal, spatial and frequency domains, revealing two distinct high-mobility transport regimes. In the non-equilibrium regime, hot carriers achieve an ultrahigh mobility of ~2,000 cm2 V–1 s–1, traversing grain boundaries up to ~300 nm within a picosecond. In the quasi-equilibrium regime, free carriers exhibit Drude-type, band-like transport with a remarkable mobility of ~400 cm2 V–1 s–1 and an intrinsic diffusion length exceeding 1 μm. These findings position two-dimensional conjugated coordination polymers as versatile platforms for advancing organic-based hot carrier applications. Hot carrier transport in organic systems has remained elusive due to rapid energy relaxation and limited transport properties. Here highly mobile hot carriers and their relaxation dynamics are reported in a crystalline two-dimensional conjugated coordination polymer, revealing two distinct transport regimes.},
  archive      = {J_NMAT},
  author       = {Fu, Shuai and Huang, Xing and Gao, Guoquan and St. Petkov, Petko and Gao, Wenpei and Zhang, Jianjun and Gao, Lei and Zhang, Heng and Liu, Min and Hambsch, Mike and Zhang, Wenjie and Zhang, Jiaxu and Li, Keming and Kaiser, Ute and Parkin, Stuart S. P. and Mannsfeld, Stefan C. B. and Zhu, Tong and Wang, Hai I. and Wang, Zhiyong and Dong, Renhao and Feng, Xinliang and Bonn, Mischa},
  doi          = {10.1038/s41563-025-02246-2},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1457-1464},
  shortjournal = {Nat. Mater.},
  title        = {Unveiling high-mobility hot carriers in a two-dimensional conjugated coordination polymer},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise synthesis of advanced polyarylamines for efficient perovskite solar cells. <em>NMAT</em>, <em>24</em>(9), 1450-1456. (<a href='https://doi.org/10.1038/s41563-025-02199-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although being highly demanded in organic electronics, functional conjugated polymers face challenges on scalable synthesis with batch uniformities. Here a reactivity-regulated sequent cross-coupling carbon–nitrogen polycondensation method is developed to enable the precise synthesis of functional polyarylamines with excellent batch-to-batch uniformity. It is revealed that the stepwise regulation of intermediate reactivities is key to accomplish controllable polycondensation via two sequent palladium-promoted carbon–nitrogen coupling cycles, which is distinct to the unicyclic carbon–carbon coupling. A variety of polyarylamines are prepared to improve the material functionalities, where a ternary polymer consisting of polar substituents is shown to optimize the interfacial and bulk properties of perovskite layers fabricated on top. The corresponding inverted perovskite solar cells achieved remarkable power conversion efficiencies of 25.2% (active area, 5.97 mm2) and 23.2% (active area, 128 mm2), along with decent operational stabilities. Overall, this work provides an effective polymerization method for advanced conjugated polymers to enable high-performance optoelectronics. The precise synthesis of functional polyarylamines based on a reactivity-regulated sequent cross-coupling carbon–nitrogen polycondensation method has been reported, with excellent batch-to-batch uniformity for perovskite solar cells.},
  archive      = {J_NMAT},
  author       = {Shen, Ziqiu and Huang, Yanchun and Dong, Yuan and Yan, Kangrong and Chen, Hongzhen and Li, Chang-Zhi},
  doi          = {10.1038/s41563-025-02199-6},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1450-1456},
  shortjournal = {Nat. Mater.},
  title        = {Precise synthesis of advanced polyarylamines for efficient perovskite solar cells},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Solvent co-intercalation in layered cathode active materials for sodium-ion batteries. <em>NMAT</em>, <em>24</em>(9), 1441-1449. (<a href='https://doi.org/10.1038/s41563-025-02287-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Solvent co-intercalation, that is, the combined intercalation of ions and solvent molecules into electrode materials, is an additional but much less explored lever for modifying the properties of metal-ion battery electrodes (metal = Li, Na, Mg, etc.). Knowledge on solvent co-intercalation is relatively scarce and largely limited to graphite anodes, for which in sodium-ion batteries, the co-intercalation of glyme molecules is fast and highly reversible. The use of co-intercalation for cathode active materials (CAMs) remains much less explored. Here we investigate for a series of sodium-layered sulfide CAMs (NaxMS2, M = Ti, V, Cr and mixtures) under which conditions solvent co-intercalation occurs and how this process impacts the phase behaviour, electrode breathing, redox potential and cycle life compared to ‘Na+-only’ intercalation. Co-intercalation is a complex process that can, for example, cause opposing fluxes, meaning that solvents intercalate into the CAMs while sodium ions simultaneously deintercalate. Co-intercalation leads to layered structures that can include different amounts of confined solvated ions, ions and unbound solvent molecules. It is an approach to designing structurally diverse, layered materials with potential applications for batteries and beyond. Solvent co-intercalation into graphite anodes for sodium-ion batteries is common; however, intercalation into cathodes is much less explored. Here, using operando experiments as well as theory, solvent co-intercalation in a range of layered sulfides is investigated.},
  archive      = {J_NMAT},
  author       = {Sun, Yanan and Åvall, Gustav and Wu, Shu-Han and A. Ferrero, Guillermo and Freytag, Annica and Groszewicz, Pedro B. and Wang, Hui and Mazzio, Katherine A. and Bianchini, Matteo and Baran, Volodymyr and Risse, Sebastian and Adelhelm, Philipp},
  doi          = {10.1038/s41563-025-02287-7},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1441-1449},
  shortjournal = {Nat. Mater.},
  title        = {Solvent co-intercalation in layered cathode active materials for sodium-ion batteries},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electron ptychography reveals a ferroelectricity dominated by anion displacements. <em>NMAT</em>, <em>24</em>(9), 1433-1440. (<a href='https://doi.org/10.1038/s41563-025-02205-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sodium niobate, a lead-free ferroic material, hosts delicately balanced, competing order parameters, including ferroelectric states that can be stabilized by epitaxial strain. Here we show that the resulting macroscopic ferroelectricity exhibits an unconventional microscopic structure using multislice electron ptychography. This technique overcomes multiple scattering artefacts limiting conventional electron microscopy, enabling both lateral spatial resolution beyond the diffraction limit and recovery of three-dimensional structural information. These imaging capabilities allow us to separate the ferroelectric interior of the sample from the relaxed surface structure and identify the soft phonon mode and related structural distortions with picometre precision. Unlike conventional ferroelectric perovskites, we find that the polar distortion in this material involves minimal distortions of the cation sublattices and is instead dominated by anion displacements relative to the niobium sublattice. We establish limits on film thickness for interfacial octahedral rotation engineering and directly visualize a random octahedral rotation pattern, arising from the flat dispersion of the associated phonon mode. Limited by challenges in light-atom imaging, microscopic investigations of ferroelectricity have used cation–cation displacements as a proxy for the true cation–anion distortions. Using electron ptychography, oxygen anions can be tracked to observe an otherwise-hidden ferroelectric mechanism in thin-film NaNbO3, which would have appeared antiferroelectric from cations alone.},
  archive      = {J_NMAT},
  author       = {KP, Harikrishnan and Xu, Ruijuan and Patel, Kinnary and Crust, Kevin J. and Khandelwal, Aarushi and Zhang, Chenyu and Prosandeev, Sergey and Zhou, Hua and Shao, Yu-Tsun and Bellaiche, Laurent and Hwang, Harold Y. and Muller, David A.},
  doi          = {10.1038/s41563-025-02205-x},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1433-1440},
  shortjournal = {Nat. Mater.},
  title        = {Electron ptychography reveals a ferroelectricity dominated by anion displacements},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Skyrmion nanodomains in ferroelectric–antiferroelectric solid solutions. <em>NMAT</em>, <em>24</em>(9), 1424-1432. (<a href='https://doi.org/10.1038/s41563-025-02216-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polar skyrmions have demonstrated rich physics and exotic properties for developing novel functionalities. However, so far, skyrmion nanodomains exist only in a few material systems, such as ferroelectric/dielectric superlattices, free-standing PbTiO3/SrTiO3 epitaxial bilayers and ultrathin Pb(Zr,Ti)O3/SrTiO3/Pb(Zr,Ti)O3 sandwiches. These heterostructures are fabricated with elaborately designed boundary conditions to meet the delicate energy balance for stabilizing topological phases. This requirement limits the broad applications of skyrmions in electronic devices. Here we show widespread skyrmion nanodomains in ferroelectric–antiferroelectric solid solutions, composed of ferroelectric PbTiO3 and one antiferroelectric PbSnO3 (Pb(Ti1–xSnx)O3), PbHfO3 (Pb(Ti1–xHfx)O3) or PbZrO3 (Pb(Ti1–xZrx)O3). The skyrmionic textures are formed by engineering dipole–dipole and antiferrodistortive–dipole couplings in competition between ferroelectric and antiferroelectric polar orderings, allowing the stabilization of topological phases. A phase diagram is built for the three solid solution series, revealing the stabilization regions of skyrmion nanodomains. In addition, the non-trivial domains also exhibit improved switching character, reversible writing/erasure and long-term retention for the electrical manipulation of polar configurations. These findings open an avenue for the investigation and exploitation of polar skyrmions in ferroelectric-based materials, providing opportunities in topological electronics. Polar skyrmions are of interest for nanoelectronics due to their exotic properties. However, so far, these are metastable states requiring a delicate balance of boundary conditions to form in heterostructures. In ferroelectric and antiferroelectric solid solutions, skyrmionic textures are observed to form in both bulk and film.},
  archive      = {J_NMAT},
  author       = {Zheng, Weijie and Ma, Xingyue and Pang, Zhentao and Ren, Yifeng and Chen, Hongying and Xu, Jibo and Zheng, Chunyan and Liu, Jianyi and Liu, Xiaohui and Deng, Yu and Nie, Yuefeng and Wu, Di and Bellaiche, Laurent and Yang, Yurong and Wen, Zheng},
  doi          = {10.1038/s41563-025-02216-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1424-1432},
  shortjournal = {Nat. Mater.},
  title        = {Skyrmion nanodomains in ferroelectric–antiferroelectric solid solutions},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Configurable antiferromagnetic domains and lateral exchange bias in atomically thin CrPS4. <em>NMAT</em>, <em>24</em>(9), 1414-1423. (<a href='https://doi.org/10.1038/s41563-025-02259-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interfacial exchange coupling between antiferromagnets (AFMs) and ferromagnets (FMs) crucially makes it possible to shift the FM hysteresis, known as exchange bias, and to switch AFM states. Two-dimensional magnets unlock opportunities to combine AFM and FM materials; however, the buried AFM–FM interfaces obtained by stacking remains challenging to understand. Here we demonstrate interfacial control via intralayer exchange coupling in the layered AFM CrPS4, where connected even and odd layers realize pristine lateral interfaces between AFM-like and FM-like regions. We distinguish antiphase even-layer states by scanning nitrogen-vacancy centre magnetometry due to a weak surface magnetization. This surface magnetization enables control over the even-layer state, with different regions switching at distinct fields due to their own lateral couplings. We toggle three AFM domains adjacent to a FM-like region and demonstrate a tunable multilevel exchange bias. Our nanoscale visualization unveils the microscopic origins of exchange bias and advances single two-dimensional crystals for hybrid AFM–FM technologies. The authors use nitrogen-vacancy centre magnetometry to explore layer number and magnetic field evolution of ferromagnetic and antiferromagnetic domains in the A-type antiferromagnet CrPS4.},
  archive      = {J_NMAT},
  author       = {Wang, Yu-Xuan and Graham, Thomas K. M. and Rama-Eiroa, Ricardo and Islam, Md Ariful and Badarneh, Mohammad H. and Nunes Gontijo, Rafael and Tiwari, Ganesh Prasad and Adhikari, Tibendra and Zhang, Xin-Yue and Watanabe, Kenji and Taniguchi, Takashi and Besson, Claire and J. G. Santos, Elton and Lin, Zhong and Zhou, Brian B.},
  doi          = {10.1038/s41563-025-02259-x},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1414-1423},
  shortjournal = {Nat. Mater.},
  title        = {Configurable antiferromagnetic domains and lateral exchange bias in atomically thin CrPS4},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). All-electrical perpendicular switching of chiral antiferromagnetic order. <em>NMAT</em>, <em>24</em>(9), 1407-1413. (<a href='https://doi.org/10.1038/s41563-025-02228-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antiferromagnets as active components in devices have been attracting attention due to their negligible stray field and ultrafast magnetic dynamics, which are promising for high-density and fast memory devices. Although the switching of antiferromagnetic (AFM) order by current-induced spin–orbit torque (SOT) has already been realized, field-free SOT-induced bidirectional switching of AFM order in perpendicular geometry has been elusive. Here we experimentally demonstrate field-free perpendicular switching of the magnetic octupole moct in chiral AFM Mn3Sn by combining in-plane and out-of-plane SOTs generated by two-dimensional van der Vaals WTe2. The out-of-plane SOT breaks the in-plane inversion symmetry and leads to deterministic bidirectional switching of moct in polycrystalline Mn3Sn even without a fixed perpendicular moct easy axis. The switching ratio reaches up to 80% compared to 20–30% in polycrystalline Mn3Sn and the critical current density is reduced by an order of magnitude to around 1 MA cm−2. Our work promotes the development of chiral AFM devices toward practical application. By introducing an unconventional spin–orbit torque, the authors demonstrate an all-electrical way to bidirectionally switch the perpendicular chiral antiferromagnetic order in Mn3Sn, promoting the application of antiferromagnetic spintronics.},
  archive      = {J_NMAT},
  author       = {Zheng, Zhenyi and Jia, Lanxin and Zhang, Zhizhong and Shen, Qia and Zhou, Guowei and Cui, Zhiwei and Ren, Lizhu and Chen, Zhiteng and Jamaludin, Nur Fadilah and Zhao, Tieyang and Xiao, Rui and Zhang, Qihan and Du, Yi and Liu, Liang and Gradečak, Silvija and Novoselov, Kostya S. and Zhao, Weisheng and Xu, Xiaohong and Zhang, Yue and Chen, Jingsheng},
  doi          = {10.1038/s41563-025-02228-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1407-1413},
  shortjournal = {Nat. Mater.},
  title        = {All-electrical perpendicular switching of chiral antiferromagnetic order},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Superconducting magic-angle twisted trilayer graphene with competing magnetic order and moiré inhomogeneities. <em>NMAT</em>, <em>24</em>(9), 1400-1406. (<a href='https://doi.org/10.1038/s41563-025-02252-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The microscopic mechanism of unconventional superconductivity in magic-angle twisted trilayer graphene is poorly understood. We show direct evidence for an in-plane magnetic order competing with the superconducting state motivated by theoretical proposals. We use two complementary electrical transport measurements. First, in statistically significant switching events in the superconducting state of magic-angle twisted trilayer graphene, we observe non-monotonic and hysteretic responses in the switching distributions as a function of temperature and in-plane magnetic field. Additionally, the system behaves like a network of Josephson junctions due to lattice-relaxation-induced moiré inhomogeneity. Second, in normal regions doped slightly away from the superconducting regime, hysteretic and linear positive magnetoresistance with the in-plane magnetic field shows evidence for an in-plane magnetic order. Furthermore, we estimate superfluid stiffness Js ≈ 0.15 K with strong temperature dependence and show a broadened Berezinskii–Kosterlitz–Thouless transition. Our observations may constrain possible intervalley-coherent magnetic orders and the superconductivity arising from its fluctuations. Electrical transport measurements of magic-angle twisted trilayer graphene are presented.},
  archive      = {J_NMAT},
  author       = {Mukherjee, Ayshi and Layek, Surat and Sinha, Subhajit and Kundu, Ritajit and Marchawala, Alisha H. and Hingankar, Mahesh and Sarkar, Joydip and Sangani, L. D. Varma and Agarwal, Heena and Ghosh, Sanat and Tazi, Aya Batoul and Watanabe, Kenji and Taniguchi, Takashi and Pasupathy, Abhay N. and Kundu, Arijit and Deshmukh, Mandar M.},
  doi          = {10.1038/s41563-025-02252-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1400-1406},
  shortjournal = {Nat. Mater.},
  title        = {Superconducting magic-angle twisted trilayer graphene with competing magnetic order and moiré inhomogeneities},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Observation of non-hermitian topology from optical loss modulation. <em>NMAT</em>, <em>24</em>(9), 1393-1399. (<a href='https://doi.org/10.1038/s41563-025-02278-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the interplay of non-Hermiticity and topology is crucial given the intrinsic openness of most natural and engineered systems, and has important ramifications in topological lasers and sensors. Recently, it has been theoretically proposed that topological features could originate solely from a system’s non-Hermiticity in photonic platforms. Here we experimentally demonstrate the appearance of non-Hermitian topology exclusively from loss modulation in a photonic system that is topologically trivial in the absence of loss. We do this by implementing a non-Hermitian generalization of an Aubry–André–Harper model with purely imaginary potential in a programmable integrated photonics platform, which allows us to investigate different periodic and quasiperiodic configurations of the model. In both cases, we show the emergence of topological edge modes and explore their resilience to different kinds of disorder. Our work highlights loss engineering as a mechanism to generate topological properties. Using a reconfigurable integrated photonic platform, the authors reveal the appearance of non-Hermitian topology and the existence of edge modes emerging exclusively from optical loss modulation.},
  archive      = {J_NMAT},
  author       = {Hashemi, Amin and Pereira, Elizabeth Louis and Li, Hongwei and Lado, Jose L. and Blanco-Redondo, Andrea},
  doi          = {10.1038/s41563-025-02278-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1393-1399},
  shortjournal = {Nat. Mater.},
  title        = {Observation of non-hermitian topology from optical loss modulation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hyper-gap transparent conductor. <em>NMAT</em>, <em>24</em>(9), 1387-1392. (<a href='https://doi.org/10.1038/s41563-025-02248-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An elusive conductor with perfect optical transparency holds revolutionary potential for fields such as optoelectronics and nanophotonics. Such a hypothetical metal would possess a spectral gap1,2—a ‘hyper-gap’—in its absorption spectrum, separating the intraband and interband absorptions, in which optical losses could vanish. Currently, this property is achievable only within the bandgap of insulators. However, realizing such a hyper-gap metal demands an exotic electronic structure in which the conducting bands have a bandwidth narrower than their energy separations from the remaining electronic states. Here we present such a hyper-gap in a family of organic metals—the Fabre charge-transfer salts3—through first-principles predictions coupled with both electrical and optical measurements. A transparent window, spanning from red to near-infrared wavelengths, is identified in bulk single crystals that remain transmissive over a thickness of 30 µm. The corresponding absorption coefficient is the lowest among known stoichiometric metals, rivalling thin films of transparent conductive oxides. This finding introduces a path, beyond traditional doping strategies in insulators, to combine electronic conduction and optical transparency. A family of organic metals that behave as hyper-gap transparent conductors is discussed. Such an elusive combination of electronic conduction and optical transparency is highly attractive for plasmonics and photonics applications.},
  archive      = {J_NMAT},
  author       = {Wu, Zhengran and Li, Chunhong and Hu, Xiaolei and Chen, Kun and Guo, Xiang and Li, Yan and Lu, Ling},
  doi          = {10.1038/s41563-025-02248-0},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1387-1392},
  shortjournal = {Nat. Mater.},
  title        = {Hyper-gap transparent conductor},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-quality narrow black phosphorus nanoribbons with nearly atomically smooth edges and well-defined edge orientation. <em>NMAT</em>, <em>24</em>(9), 1375-1386. (<a href='https://doi.org/10.1038/s41563-025-02314-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Black phosphorus nanoribbons (BPNRs) with a tunable bandgap and intriguing electronic and optical properties hold strong potential for logic applications. However, efficiently producing high-quality BPNRs with precise control over their size and structure remains a great challenge. Here we achieved high-quality, narrow and clean BPNRs with nearly atomically smooth edges and well-defined edge orientation at high yield (up to ~95%) through the sonochemical exfoliation of the synthesized bulk BP crystals with a slightly enlarged lattice parameter along the armchair direction. The resulting BPNRs have widths centred at 32 nm and can be as narrow as 1.5 nm, with edges consistently aligned along the zigzag direction in measured BPNRs with widths ≤340 nm. The formation of one-dimensional BPNRs with zigzag edges is attributed to the introduction of pre-stress along the armchair direction of the grown bulk BP and the application of suitable sonication conditions. The BPNR bandgap increases as the BPNR width decreases from 83 nm to 13 nm, with a large bandgap of 0.64 eV for a 13-nm-wide BPNR. A typical graphene-contacted field-effect transistor fabricated with a 13-nm-wide and 10-nm-thick BPNR can achieve an on/off ratio of 1.7 × 106, mobility of 1,506 cm2 V−1 s−1 and on-state channel conductivity of 1,845 µS. The devices also exhibit excellent photodetection performance. Our method opens up a route to produce BPNRs with high material quality and defined edge chirality for fundamental studies and practical applications in electronic and optoelectronic fields. Through the sonochemical exfoliation of grown black phosphorus crystals, high-quality, narrow and clean black phosphorus nanoribbons with smooth edges and defined edge chirality are achieved, enabling the realization of high-performance transistors and photodetectors.},
  archive      = {J_NMAT},
  author       = {Zhang, Teng and Chen, Youxin and He, Zhiyan and Liu, Qinran and Chen, Le and Wang, Kai and Chen, Wenzhe and Wang, Kunchan and Ye, Xiaowo and He, Zhuoyang and Zhang, Yuyang and Zhang, Yanming and Liao, Chenghao and He, Yuan and Gao, Wenpei and Zhang, Tianyu and Hu, Bei and Jiang, Shenghao and Shi, Fangyuan and Gao, Shengguang and Li, Xinyue and Chen, Changxin},
  doi          = {10.1038/s41563-025-02314-7},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1375-1386},
  shortjournal = {Nat. Mater.},
  title        = {High-quality narrow black phosphorus nanoribbons with nearly atomically smooth edges and well-defined edge orientation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bioinstructive scaffolds enhance stem cell engraftment for functional tissue regeneration. <em>NMAT</em>, <em>24</em>(9), 1364-1374. (<a href='https://doi.org/10.1038/s41563-025-02212-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stem cell therapy is a promising approach for tissue regeneration after traumatic injury, yet current applications are limited by inadequate control over the fate of stem cells after transplantation. Here we introduce a bioconstruct engineered for the staged release of growth factors, tailored to direct different phases of muscle regeneration. The bioconstruct is composed of a decellularized extracellular matrix containing polymeric nanocapsules sequentially releasing basic fibroblast growth factor and insulin-like growth factor 1, which promote the proliferation and differentiation of muscle stem cells, respectively. When applied to a volumetric muscle loss defect in an animal model, the bioconstruct enhances myofibre formation, angiogenesis, innervation and functional restoration. Further, it promotes functional muscle formation with human or aged murine muscle stem cells, highlighting the translational potential of this bioconstruct. Overall, these results highlight the potential of bioconstructs with orchestrated growth factor release for stem cell therapies in traumatic injury. A decellularized extracellular matrix scaffold containing nanocapsules orchestrates timed growth factor release to promote muscle stem cell proliferation and differentiation and effective muscle regeneration after traumatic injury.},
  archive      = {J_NMAT},
  author       = {Wu, Di and Eugenis, Ioannis and Hu, Caroline and Kim, Soochi and Kanugovi, Abhijnya and Yue, Shouzheng and Wheeler, Joshua R. and Fathali, Iman and Feeley, Sonali and Shrager, Joseph B. and Huang, Ngan F. and Rando, Thomas A.},
  doi          = {10.1038/s41563-025-02212-y},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1364-1374},
  shortjournal = {Nat. Mater.},
  title        = {Bioinstructive scaffolds enhance stem cell engraftment for functional tissue regeneration},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vortices and antivortices in antiferroelectric PbZrO3. <em>NMAT</em>, <em>24</em>(9), 1359-1363. (<a href='https://doi.org/10.1038/s41563-025-02245-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ferroelectric materials are characterized by a parallel arrangement of electric dipoles, but at the nanoscale they can present vortices and other non-trivial topological structures1–9 that combine small size and topological protection, rendering them functionally attractive10–13. The driving force for the appearance of vortices in ferroelectrics is the need to minimize the depolarizing fields at interfaces3–5,14; by making the polarization rotate, depolarization fields vanish4,5,8,9. Antiferroelectrics, by contrast, are defined by an antiparallel arrangement of electric dipoles15. A priori, therefore, they lack the depolarization fields that drive the appearance of non-trivial topologies in ferroelectrics. At the atomic scale of the dipoles, however, we find that polar discontinuities can still happen, driving the appearance of topological singularities at ferroelastic domain walls. Topological phenomena in ferroelectrics such as vortices are of interest as they may be useful for high-density storage applications, but similar phenomena have not been seen in antiferroelectrics, which possess antipolar dipole arrangements. Here, using electron microscopy, topological antivortices and faint vortices are seen in the prototypical antiferroelectric PbZrO3.},
  archive      = {J_NMAT},
  author       = {Liu, Ying and Zhang, Huazhang and Shapovalov, Konstantin and Niu, Ranming and Cairney, Julie M. and Liao, Xiaozhou and Roleder, Krystian and Majchrowski, Andrzej and Arbiol, Jordi and Ghosez, Philippe and Catalan, Gustau},
  doi          = {10.1038/s41563-025-02245-3},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1359-1363},
  shortjournal = {Nat. Mater.},
  title        = {Vortices and antivortices in antiferroelectric PbZrO3},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanisms of resistive switching in two-dimensional monolayer and multilayer materials. <em>NMAT</em>, <em>24</em>(9), 1346-1358. (<a href='https://doi.org/10.1038/s41563-025-02170-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The power and energy consumption of resistive switching devices can be lowered by reducing the dimensions of their active layers. Efforts to push this low-energy switching property to its limits have led to the investigation of active regions made with two-dimensional (2D) layered materials. Despite their small dimensions, 2D layered materials exhibit a rich variety of switching mechanisms, each involving different types of atomic structure reconfiguration. In this Review, we highlight and classify the mechanisms of resistive switching in monolayer and bulk 2D layered materials, with a subsequent focus on those occurring in a monolayer and/or localized to point defects in the crystalline sheet. We discuss the complex energetics involved in these fundamentally defect-assisted processes, including the coexistence of multiple mechanisms and the effects of the contacts used. Examining the highly localized ‘atomristor’-type switching, we provide insights into atomic motions and electronic transport across the metal–2D interfaces underlying their operation. Finally, we discuss progress and our perspective on the challenges associated with the development of 2D resistive switching devices. Promising application areas and material systems are identified and suggested for further research. The authors review the mechanisms of resistive switching in monolayer and bulk forms of two-dimensional layered materials, providing insights into atomic motions and electronic transport across interfaces.},
  archive      = {J_NMAT},
  author       = {Kaniselvan, M. and Jeon, Y.-R. and Mladenović, M. and Luisier, M. and Akinwande, D.},
  doi          = {10.1038/s41563-025-02170-5},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1346-1358},
  shortjournal = {Nat. Mater.},
  title        = {Mechanisms of resistive switching in two-dimensional monolayer and multilayer materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modulation of optical loss leads to robust localization of light. <em>NMAT</em>, <em>24</em>(9), 1344-1345. (<a href='https://doi.org/10.1038/s41563-025-02298-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical loss usually leads to dissipation, but has been shown, counterintuitively, to create strong light localization. This demonstration — realized by implementing a photonic arrangement with a special topology in a programmable integrated photonic platform — highlights loss as a powerful control of the topological properties of electromagnetic modes.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02298-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1344-1345},
  shortjournal = {Nat. Mater.},
  title        = {Modulation of optical loss leads to robust localization of light},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Black phosphorus nanoribbons for high-performance transistors and photodetectors. <em>NMAT</em>, <em>24</em>(9), 1342-1343. (<a href='https://doi.org/10.1038/s41563-025-02313-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sonochemical exfoliation of black phosphorus crystals with a slightly enlarged lattice parameter along the armchair direction produces narrow black phosphorus nanoribbons (BPNRs) with well-defined edge chirality at a yield of up to 95%. The high performance of transistors and photodetectors fabricated with these BPNRs demonstrates the potential of BPNRs for electronic and optoelectronic applications.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02313-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1342-1343},
  shortjournal = {Nat. Mater.},
  title        = {Black phosphorus nanoribbons for high-performance transistors and photodetectors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Orchestrated growth factor release for muscle restoration. <em>NMAT</em>, <em>24</em>(9), 1340-1341. (<a href='https://doi.org/10.1038/s41563-025-02330-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Implantable bioconstructs with sequential release of growth factors promote enhanced myofibre formation, angiogenesis and innervation in a mouse model of volumetric muscle loss.},
  archive      = {J_NMAT},
  author       = {Fischer, Eliana O. and Tsukerman, Anna and Levenberg, Shulamit},
  doi          = {10.1038/s41563-025-02330-7},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1340-1341},
  shortjournal = {Nat. Mater.},
  title        = {Orchestrated growth factor release for muscle restoration},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Role of the extracellular matrix in cardiac ageing. <em>NMAT</em>, <em>24</em>(9), 1337-1339. (<a href='https://doi.org/10.1038/s41563-025-02325-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A hybrid decellularized extracellular matrix–polyacrylamide scaffold reveals how extracellular matrix properties influence cardiac ageing, offering insights for potential rejuvenation therapies.},
  archive      = {J_NMAT},
  author       = {Prakash, Aishwarya and Smith, Poppy O. and Iskratsch, Thomas},
  doi          = {10.1038/s41563-025-02325-4},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1337-1339},
  shortjournal = {Nat. Mater.},
  title        = {Role of the extracellular matrix in cardiac ageing},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tripeptide armour shields encapsulated proteins. <em>NMAT</em>, <em>24</em>(9), 1335-1336. (<a href='https://doi.org/10.1038/s41563-025-02302-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drying-driven assembly of tripeptides into porous, redispersible microparticles enables spontaneous protein encapsulation and protection.},
  archive      = {J_NMAT},
  author       = {Li, Xianbao and Li, Junbai},
  doi          = {10.1038/s41563-025-02302-x},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1335-1336},
  shortjournal = {Nat. Mater.},
  title        = {Tripeptide armour shields encapsulated proteins},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-lived hot carriers in a polymeric semiconductor. <em>NMAT</em>, <em>24</em>(9), 1333-1334. (<a href='https://doi.org/10.1038/s41563-025-02286-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hot carriers with high mobility and long diffusion length are observed in a two-dimensional conjugated coordination polymer.},
  archive      = {J_NMAT},
  author       = {Handa, Taketo and Zhu, Xiaoyang},
  doi          = {10.1038/s41563-025-02286-8},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1333-1334},
  shortjournal = {Nat. Mater.},
  title        = {Long-lived hot carriers in a polymeric semiconductor},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-performance polymers for perovskite photovoltaics. <em>NMAT</em>, <em>24</em>(9), 1331-1332. (<a href='https://doi.org/10.1038/s41563-025-02235-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A carbon–nitrogen cross-coupling polymerization method is developed to precisely synthesize functional polyarylamines at large scale, resulting in perovskite solar cells with promising efficiencies and stability.},
  archive      = {J_NMAT},
  author       = {Wakamiya, Atsushi and Murdey, Richard},
  doi          = {10.1038/s41563-025-02235-5},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1331-1332},
  shortjournal = {Nat. Mater.},
  title        = {High-performance polymers for perovskite photovoltaics},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unveiling ferroelectric 3D microstructures. <em>NMAT</em>, <em>24</em>(9), 1328-1330. (<a href='https://doi.org/10.1038/s41563-025-02206-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multislice electron ptychography reveals ferroelectric microstructures with sub-ångström lateral resolution and nanometre depth resolution, directly imaging a ferroelectricity generated by anion displacements relative to the Nb sublattice.},
  archive      = {J_NMAT},
  author       = {Gao, Xiaoyue and Gao, Peng},
  doi          = {10.1038/s41563-025-02206-w},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1328-1330},
  shortjournal = {Nat. Mater.},
  title        = {Unveiling ferroelectric 3D microstructures},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A tug-of-war recipe for nanoscale swirls. <em>NMAT</em>, <em>24</em>(9), 1326-1327. (<a href='https://doi.org/10.1038/s41563-025-02288-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mixed ceramics and thin films of well-known ferroelectric and antiferroelectric materials reveal dipolar skyrmions, providing a scalable platform to explore the properties and applications of non-collinear polarization textures.},
  archive      = {J_NMAT},
  author       = {Maksymovych, Petro},
  doi          = {10.1038/s41563-025-02288-6},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1326-1327},
  shortjournal = {Nat. Mater.},
  title        = {A tug-of-war recipe for nanoscale swirls},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reconfigurable magnetic order in 2D materials. <em>NMAT</em>, <em>24</em>(9), 1324-1325. (<a href='https://doi.org/10.1038/s41563-025-02304-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Layer-dependent magnetic behaviour is used to realize reconfigurable exchange bias in a 2D antiferromagnet.},
  archive      = {J_NMAT},
  author       = {Stevenson, Paul},
  doi          = {10.1038/s41563-025-02304-9},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1324-1325},
  shortjournal = {Nat. Mater.},
  title        = {Reconfigurable magnetic order in 2D materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Switching a chiral antiferromagnet. <em>NMAT</em>, <em>24</em>(9), 1322-1323. (<a href='https://doi.org/10.1038/s41563-025-02266-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unconventional spin–orbit torque generated by a low-symmetry topological semimetal enables purely electrical switching of the perpendicular magnetic octupole in a chiral antiferromagnet.},
  archive      = {J_NMAT},
  author       = {Singh, Simranjeet and Kao, I-Hsuan},
  doi          = {10.1038/s41563-025-02266-y},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1322-1323},
  shortjournal = {Nat. Mater.},
  title        = {Switching a chiral antiferromagnet},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The expanding world of topological ferroelectrics. <em>NMAT</em>, <em>24</em>(9), 1321. (<a href='https://doi.org/10.1038/s41563-025-02350-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue we present three papers on the topological properties or structure of ferroelectrics, which are interlinked.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02350-3},
  journal      = {Nature Materials},
  month        = {9},
  number       = {9},
  pages        = {1321},
  shortjournal = {Nat. Mater.},
  title        = {The expanding world of topological ferroelectrics},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmeth">NMETH - 35</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: IPipet: Sample handling using a tablet. <em>NMETH</em>, <em>22</em>(9), 1996. (<a href='https://doi.org/10.1038/s41592-025-02804-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zielinski, Dina and Gordon, Assaf and Zaks, Benjamin L. and Erlich, Yaniv},
  doi          = {10.1038/s41592-025-02804-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1996},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: IPipet: Sample handling using a tablet},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A universal language for finding mass spectrometry data patterns. <em>NMETH</em>, <em>22</em>(9), 1995. (<a href='https://doi.org/10.1038/s41592-025-02785-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Damiani, Tito and Jarmusch, Alan K. and Aron, Allegra T. and Petras, Daniel and Phelan, Vanessa V. and Zhao, Haoqi Nina and Bittremieux, Wout and Acharya, Deepa D. and Ahmed, Mohammed M. A. and Bauermeister, Anelize and Bertin, Matthew J. and Boudreau, Paul D. and Borges, Ricardo M. and Bowen, Benjamin P. and Brown, Christopher J. and Chagas, Fernanda O. and Clevenger, Kenneth D. and Correia, Mario S. P. and Crandall, William J. and Crüsemann, Max and Fahy, Eoin and Fiehn, Oliver and Garg, Neha and Gerwick, William H. and Gilbert, Jeffrey R. and Globisch, Daniel and Gomes, Paulo Wender P. and Heuckeroth, Steffen and James, C. Andrew and Jarmusch, Scott A. and Kakhkhorov, Sarvar A. and Kang, Kyo Bin and Kessler, Nikolas and Kersten, Roland D. and Kim, Hyunwoo and Kirk, Riley D. and Kohlbacher, Oliver and Kontou, Eftychia E. and Liu, Ken and Lizama-Chamu, Itzel and Luu, Gordon T. and Luzzatto Knaan, Tal and Mannochio-Russo, Helena and Marty, Michael T. and Matsuzawa, Yuki and McAvoy, Andrew C. and McCall, Laura-Isobel and Mohamed, Osama G. and Nahor, Omri and Neuweger, Heiko and Niedermeyer, Timo H. J. and Nishida, Kozo and Northen, Trent R. and Overdahl, Kirsten E. and Rainer, Johannes and Reher, Raphael and Rodriguez, Elys and Sachsenberg, Timo T. and Sanchez, Laura M. and Schmid, Robin and Stevens, Cole and Subramaniam, Shankar and Tian, Zhenyu and Tripathi, Ashootosh and Tsugawa, Hiroshi and van der Hooft, Justin J. J. and Vicini, Andrea and Walter, Axel and Weber, Tilmann and Xiong, Quanbo and Xu, Tao and Pluskal, Tomáš and Dorrestein, Pieter C. and Wang, Mingxun},
  doi          = {10.1038/s41592-025-02785-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1995},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A universal language for finding mass spectrometry data patterns},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unified mass imaging maps the lipidome of vertebrate development. <em>NMETH</em>, <em>22</em>(9), 1981-1994. (<a href='https://doi.org/10.1038/s41592-025-02771-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryo development entails the formation of anatomical structures with distinct biochemical compositions. Compared with the wealth of knowledge on gene regulation, our understanding of metabolic programs operating during embryogenesis is limited. Mass spectrometry imaging (MSI) has the potential to map the distribution of metabolites across embryo development. Here we established uMAIA, an analytical framework for the joint analysis of large MSI datasets, which enables the construction of multidimensional metabolomic atlases. Employing this framework, we mapped the four-dimensional (4D) distribution of over a hundred lipids at micrometric resolution in Danio rerio embryos. We discovered metabolic trajectories that unfold in concert with morphogenesis and revealed spatially organized biochemical coordination overlooked by bulk measurements. Interestingly, lipid mapping revealed unexpected distributions of sphingolipid and triglyceride species, suggesting their involvement in pattern establishment and organ development. Our approach empowers a new generation of whole-organism metabolomic atlases and enables the discovery of spatially organized metabolic circuits. uMAIA is an analytical framework designed to enable the construction of metabolic atlases at high resolution using mass spectrometry imaging data.},
  archive      = {J_NMETH},
  author       = {Schede, Halima Hannah and Alieh, Leila Haj Abdullah and Rohde, Laurel Ann and Herrera, Antonio and Schlaeppi, Anjalie and Valentin, Guillaume and Gargoori Motlagh, Alireza and Dominguez Mantes, Albert and Jollivet, Chloe and Paz-Montoya, Jonathan and Capolupo, Laura and Khven, Irina and Oates, Andrew C. and D’Angelo, Giovanni and La Manno, Gioele},
  doi          = {10.1038/s41592-025-02771-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1981-1994},
  shortjournal = {Nat. Meth.},
  title        = {Unified mass imaging maps the lipidome of vertebrate development},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions. <em>NMETH</em>, <em>22</em>(9), 1964-1980. (<a href='https://doi.org/10.1038/s41592-025-02793-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues. TCAT is a pipeline that can simultaneously capture gene expression programs related to T cell subsets and activation states for accurate T cell characterization.},
  archive      = {J_NMETH},
  author       = {Kotliar, Dylan and Curtis, Michelle and Agnew, Ryan and Weinand, Kathryn and Nathan, Aparna and Baglaenko, Yuriy and Slowikowski, Kamil and Zhao, Yu and Sabeti, Pardis C. and Rao, Deepak A. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41592-025-02793-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1964-1980},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible single-cell annotation of programs underlying t cell subsets, activation states and functions},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput profiling of chemical-induced gene expression across 93,644 perturbations. <em>NMETH</em>, <em>22</em>(9), 1954-1963. (<a href='https://doi.org/10.1038/s41592-025-02781-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this Resource, we present an extensive dataset of chemical-induced gene signatures (CIGS), encompassing expression patterns of 3,407 genes regulating key biological processes in 2 human cell lines exposed to 13,221 compounds across 93,664 perturbations. This dataset encompasses 319,045,108 gene expression events, generated through 2 high-throughput technologies: the previously documented high-throughput sequencing-based high-throughput screening (HTS2) and the newly developed highly multiplexed and parallel sequencing (HiMAP-seq). Our results show that HiMAP-seq is comparable to RNA sequencing, but can profile the expression of thousands of genes across thousands of samples in one single test by utilizing a pooled-sample strategy. We further illustrate CIGS’s utility in elucidating the mechanism of action of unannotated small molecules, like ligustroflavone and 2,4-dihydroxybenzaldehyde, and to identify perturbation-induced cell states, such as those resistant to ferroptosis. The full dataset is publicly accessible at https://cigs.iomicscloud.com/ . CIGS is a high-resolution dataset comprising chemical perturbation-induced 319,045,108 gene expression events.},
  archive      = {J_NMETH},
  author       = {Xiang, Lei and Wang, Yumei and Shao, Wei and Li, Qingzhou and Yu, Xiankuo and Wei, Mingming and Gui, Yu and Li, Shengrong and Qin, Pan and Hu, Chao and Zhang, Guochen and Zhang, Xianwen and Wang, Jiawen and Li, Yingying and An, Jun and Luo, Yan and Liao, Yile and Deng, Jinghong and Tai, Xinran and Xu, Richard Y. and Huang, Lijun and Guo, Dale and Zhang, Guanbin and Xie, Zhi and Deng, Yun and Xu, Junquan and Wang, Dong},
  doi          = {10.1038/s41592-025-02781-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1954-1963},
  shortjournal = {Nat. Meth.},
  title        = {High-throughput profiling of chemical-induced gene expression across 93,644 perturbations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI. <em>NMETH</em>, <em>22</em>(9), 1944-1953. (<a href='https://doi.org/10.1038/s41592-025-02798-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concurrent recording of electroencephalogram (EEG) and functional magnetic resonance imaging (fMRI) signals reveals cross-scale neurovascular dynamics crucial for explaining fundamental linkages between function and behaviors. However, MRI scanners generate artifacts for EEG detection. Despite existing denoising methods, cabled connections to EEG receivers are susceptible to environmental fluctuations inside MRI scanners, creating baseline drifts that complicate EEG signal retrieval from the noisy background. Here we show that a wireless integrated sensing detector for simultaneous EEG and MRI can encode fMRI and EEG signals on distinct sidebands of the detector’s oscillation wave for detection by a standard MRI console over the entire duration of the fMRI sequence. Local field potential and fMRI maps are retrieved through low-pass and high-pass filtering of frequency-demodulated signals. From optogenetically stimulated somatosensory cortex in ChR2-transfected Sprague Dawley rats, positive correlation between evoked local field potential and fMRI signals validates strong neurovascular coupling, enabling cross-scale brain mapping with this two-in-one transducer. WISDEM is a hybrid detector for simultaneous EEG and fMRI recordings without artifacts or crosstalk, which allows access to neural activity at high temporal and spatial resolution, respectively, as demonstrated in rats.},
  archive      = {J_NMETH},
  author       = {Chen, Yi and Qian, Wei and Razansky, Daniel and Yu, Xin and Qian, Chunqi},
  doi          = {10.1038/s41592-025-02798-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1944-1953},
  shortjournal = {Nat. Meth.},
  title        = {WISDEM: A hybrid wireless integrated sensing detector for simultaneous EEG and MRI},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A versatile miniature two-photon microscope enabling multicolor deep-brain imaging. <em>NMETH</em>, <em>22</em>(9), 1935-1943. (<a href='https://doi.org/10.1038/s41592-025-02780-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present FHIRM-TPM 3.0, a 2.6 g miniature two-photon microscope capable of multicolor deep-brain imaging in freely behaving mice. The system was integrated with a broadband anti-resonant hollow-core fiber featuring low transmission loss, minimal dispersion from 700 nm to 1,060 nm and high tolerance of laser power. By correcting chromatic and spherical aberrations and optimizing the fluorescence collection aperture, we achieved cortical neuronal imaging at depths exceeding 820 μm and, using a GRIN lens, hippocampal Ca2+ imaging at single dendritic spine resolution. Moreover, we engineered three interchangeable parfocal objectives, allowing for a tenfold scalable field of view up to 1 × 0.8 mm², with lateral resolutions ranging from 0.68 μm to 1.46 μm. By multicolor imaging at excitation wavelengths of 780 nm, 920 nm and 1,030 nm, we investigated mitochondrial and cytosolic Ca2+ activities relative to the deposition of amyloid plaques in the cortex of awake APP/PS1 transgenic mice. Thus, FHIRM-TPM 3.0 provides a versatile imaging system suitable for diverse brain imaging scenarios. FHIRM-TPM 3.0 is a miniature microscope for multicolor two-photon imaging in freely moving mice. In addition to the multicolor imaging abilities achieved with the help of a specially designed optical fiber, the microscope is also compatible with multiple lenses for a choice of field of view and resolution.},
  archive      = {J_NMETH},
  author       = {Wu, Runlong and Zhao, Chunzhu and Qiu, Shan and Zhu, Yufei and Zhang, Lifeng and Fu, Qiang and Hu, Yanhui and Wu, Dakun and Yu, Fei and Zhou, Fangxu and Huang, Haipeng and Zhang, Yunfeng and Wang, Xianhua and Wang, Aimin and Cheng, Heping},
  doi          = {10.1038/s41592-025-02780-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1935-1943},
  shortjournal = {Nat. Meth.},
  title        = {A versatile miniature two-photon microscope enabling multicolor deep-brain imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy. <em>NMETH</em>, <em>22</em>(9), 1923-1934. (<a href='https://doi.org/10.1038/s41592-025-02794-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volume correlative light and electron microscopy (vCLEM) is a powerful imaging technique that enables the visualization of fluorescently labeled proteins within their ultrastructural context. Currently, vCLEM alignment relies on time-consuming and subjective manual methods. This paper presents CLEM-Reg, an algorithm that automates the three-dimensional alignment of vCLEM datasets by leveraging probabilistic point cloud registration techniques. Point clouds are derived from segmentations of common structures in each modality, created by state-of-the-art open-source methods. CLEM-Reg drastically reduces the registration time of vCLEM datasets to a few minutes and achieves correlation of fluorescent signal to submicron target structures in electron microscopy on three newly acquired vCLEM benchmark datasets. CLEM-Reg was then used to automatically obtain vCLEM overlays to unambiguously identify TGN46-positive transport carriers involved in protein trafficking between the trans-Golgi network and plasma membrane. Datasets are available on EMPIAR and BioStudies, and a napari plugin is provided to aid end-user adoption. CLEM-Reg automates the three-dimensional alignment of volume correlative light and electron microscopy datasets by leveraging probabilistic point cloud registration techniques for fast and accurate results across diverse datasets.},
  archive      = {J_NMETH},
  author       = {Krentzel, Daniel and Elphick, Matouš and Domart, Marie-Charlotte and Peddie, Christopher J. and Laine, Romain F. and Shand, Cameron and Henriques, Ricardo and Collinson, Lucy M. and Jones, Martin L.},
  doi          = {10.1038/s41592-025-02794-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1923-1934},
  shortjournal = {Nat. Meth.},
  title        = {CLEM-reg: An automated point cloud-based registration algorithm for volume correlative light and electron microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE. <em>NMETH</em>, <em>22</em>(9), 1911-1922. (<a href='https://doi.org/10.1038/s41592-025-02770-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while preserving crucial spatial context within tissues. However, existing ST platforms are constrained by high costs, long turnaround times, low resolution, limited gene coverage and inherently small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that reconstructs large-scale, super-resolution gene expression landscapes and automatically annotates cellular-level tissue architecture in samples exceeding capture areas of current ST platforms. The performance of iSCALE was assessed by comprehensive evaluations involving benchmarking experiments, immunohistochemistry staining and manual annotations by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion-associated cellular characteristics undetectable by conventional ST experiments. Our results demonstrate the utility of iSCALE in analyzing large tissues by enabling unbiased annotation, resolving cell type composition, mapping cellular microenvironments and revealing spatial features beyond the reach of standard ST analysis or routine histopathological assessment. iSCALE leverages histology and spatial transcriptomics to infer gene expression at super resolution in large tissues.},
  archive      = {J_NMETH},
  author       = {Schroeder, Amelia and Loth, Melanie L. and Luo, Chunyu and Yao, Sicong and Yan, Hanying and Zhang, Daiwei and Piya, Sarbottam and Plowey, Edward and Hu, Wenxing and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Chan, Su Jing and Reynolds, Taylor L. and Carlile, Thomas and Cullen, Patrick and Sung, Ji-Youn and Tsai, Hui-Hsin and Park, Jeong Hwan and Hwang, Tae Hyun and Zhang, Baohong and Li, Mingyao},
  doi          = {10.1038/s41592-025-02770-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1911-1922},
  shortjournal = {Nat. Meth.},
  title        = {Scaling up spatial transcriptomics for large-sized tissues: Uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial gene expression at single-cell resolution from histology using deep learning with GHIST. <em>NMETH</em>, <em>22</em>(9), 1900-1910. (<a href='https://doi.org/10.1038/s41592-025-02795-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The increased use of spatially resolved transcriptomics provides new biological insights into disease mechanisms. However, the high cost and complexity of these methods are barriers to broader application. Consequently, methods have been created to predict spot-based gene expression from routinely collected histology images. Recent benchmarking showed that current methodologies have limited accuracy and spatial resolution, constraining translational capacity. Here, we introduce GHIST, a deep learning-based framework that predicts spatial gene expression at single-cell resolution by leveraging subcellular spatial transcriptomics and synergistic relationships between multiple layers of biological information. We validated GHIST using public datasets and The Cancer Genome Atlas data, demonstrating its flexibility across different spatial resolutions and superior performance. Our results underscore the utility of in silico generation of single-cell spatial gene expression measurements and the capacity to enrich existing datasets with a spatially resolved omics modality, paving the way for scalable multi-omics analysis and biomarker identification. GHIST is a deep learning-based method that can predict spatial gene expression at high resolution using histology data.},
  archive      = {J_NMETH},
  author       = {Fu, Xiaohang and Cao, Yue and Bian, Beilei and Wang, Chuhan and Graham, Dinny and Pathmanathan, Nirmala and Patrick, Ellis and Kim, Jinman and Yang, Jean Yee Hwa},
  doi          = {10.1038/s41592-025-02795-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1900-1910},
  shortjournal = {Nat. Meth.},
  title        = {Spatial gene expression at single-cell resolution from histology using deep learning with GHIST},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ultra-high-scale cytometry-based cellular interaction mapping. <em>NMETH</em>, <em>22</em>(9), 1887-1899. (<a href='https://doi.org/10.1038/s41592-025-02744-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular interactions are of fundamental importance, orchestrating organismal development, tissue homeostasis and immunity. Recently, powerful methods that use single-cell genomic technologies to dissect physically interacting cells have been developed. However, these approaches are characterized by low cellular throughput, long processing times and high costs and are typically restricted to predefined cell types. Here we introduce Interact-omics, a cytometry-based framework to accurately map cellular landscapes and cellular interactions across all immune cell types at ultra-high resolution and scale. We demonstrate the utility of our approach to study kinetics, mode of action and personalized response prediction of immunotherapies, and organism-wide shifts in cellular composition and cellular interaction dynamics following infection in vivo. Our scalable framework can be applied a posteriori to existing cytometry datasets or incorporated into newly designed cytometry-based studies to map cellular interactions with a broad range of applications from fundamental biology to applied biomedicine. Interact-omics, a high-throughput cytometry-based framework, resolves the cellular interaction landscape.},
  archive      = {J_NMETH},
  author       = {Vonficht, Dominik and Jopp-Saile, Lea and Yousefian, Schayan and Flore, Viktoria and Simó Vesperinas, Inés and Teuber, Ruth and Avanesyan, Bogdan and Luo, Yanjiang and Röthemeier, Caroline and Grünschläger, Florian and Fernandez-Vaquero, Mirian and Fregona, Vincent and Ordoñez-Rueda, Diana and Schmalbrock, Laura K. and Deininger, Luca and Yamachui Sitcheu, Angelo Jovin and Gu, Zuguang and Funk, Maja C. and Mikut, Ralf and Heikenwälder, Mathias and Eggert, Angelika and von Stackelberg, Arend and Kobold, Sebastian and Krönke, Jan and Keller, Ulrich and Trumpp, Andreas and Hegazy, Ahmed N. and Eckert, Cornelia and Hübschmann, Daniel and Haas, Simon},
  doi          = {10.1038/s41592-025-02744-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1887-1899},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-scale cytometry-based cellular interaction mapping},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Laser flash melting cryo-EM samples to overcome preferred orientation. <em>NMETH</em>, <em>22</em>(9), 1880-1886. (<a href='https://doi.org/10.1038/s41592-025-02796-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sample preparation remains a bottleneck for protein structure determination by cryo-electron microscopy. A frequently encountered issue is that proteins adsorb to the air–water interface of the sample in a limited number of orientations. This makes it challenging to obtain high-resolution reconstructions, or may even cause projects to fail altogether. We have previously observed that laser flash melting and revitrification of cryo-EM samples reduces preferred orientation for large, symmetric particles. Here we demonstrate that our method can in fact be used to scramble the orientation of proteins of a range of sizes and symmetries. The effect can be enhanced for some proteins by increasing the heating rate during flash melting or by depositing amorphous ice onto the sample prior to revitrification. This also allows us to shed light onto the underlying mechanism. Our experiments establish a set of tools for overcoming preferred orientation that can be easily integrated into existing workflows. Individual proteins tend to adopt preferred orientations when subjected to vitrification for cryo-electron microscopy analysis. A laser flash melting procedure followed by rapid revitrification provides a simple approach to mitigate this issue, reducing the number of micrographs required for successful structure determination at high-resolution.},
  archive      = {J_NMETH},
  author       = {Straub, Monique S. and Harder, Oliver F. and Mowry, Nathan J. and Barrass, Sarah V. and Hruby, Jakub and Drabbels, Marcel and Lorenz, Ulrich J.},
  doi          = {10.1038/s41592-025-02796-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1880-1886},
  shortjournal = {Nat. Meth.},
  title        = {Laser flash melting cryo-EM samples to overcome preferred orientation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biophysics-based protein language models for protein engineering. <em>NMETH</em>, <em>22</em>(9), 1868-1879. (<a href='https://doi.org/10.1038/s41592-025-02776-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models trained on evolutionary data have emerged as powerful tools for predictive problems involving protein sequence, structure and function. However, these models overlook decades of research into biophysical factors governing protein function. We propose mutational effect transfer learning (METL), a protein language model framework that unites advanced machine learning and biophysical modeling. Using the METL framework, we pretrain transformer-based neural networks on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics. We fine-tune METL on experimental sequence–function data to harness these biophysical signals and apply them when predicting protein properties like thermostability, catalytic activity and fluorescence. METL excels in challenging protein engineering tasks like generalizing from small training sets and position extrapolation, although existing methods that train on evolutionary signals remain powerful for many types of experimental assays. We demonstrate METL’s ability to design functional green fluorescent protein variants when trained on only 64 examples, showcasing the potential of biophysics-based protein language models for protein engineering. Mutational effect transfer learning (METL) is a protein language model framework that unites machine learning and biophysical modeling. Transformer-based neural networks are pretrained on biophysical simulation data to capture fundamental relationships between protein sequence, structure and energetics.},
  archive      = {J_NMETH},
  author       = {Gelman, Sam and Johnson, Bryce and Freschlin, Chase R. and Sharma, Arnav and D’Costa, Sameer and Peters, John and Gitter, Anthony and Romero, Philip A.},
  doi          = {10.1038/s41592-025-02776-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1868-1879},
  shortjournal = {Nat. Meth.},
  title        = {Biophysics-based protein language models for protein engineering},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations. <em>NMETH</em>, <em>22</em>(9), 1857-1867. (<a href='https://doi.org/10.1038/s41592-025-02797-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modifications (PTMs) are critical regulators of protein function, and their disruption is a key mechanism by which missense variants contribute to disease. Accurate PTM site prediction using deep learning can help identify PTM-altering variants, but progress has been limited by the lack of large, high-quality training datasets. Here, we introduce PTMAtlas, a curated compendium of 397,524 PTM sites generated through systematic reprocessing of 241 public mass-spectrometry datasets, and DeepMVP, a deep learning framework trained on PTMAtlas to predict PTM sites for phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation. DeepMVP substantially outperforms existing tools across all six PTM types. Its application to predicting PTM-altering missense variants shows strong concordance with experimental results, validated using literature-curated variants and cancer proteogenomic datasets. Together, PTMAtlas and DeepMVP provide a robust platform for PTM research and a scalable framework for assessing the functional consequences of coding variants through the lens of PTMs. DeepMVP is a deep learning framework for predicting PTM sites and variant-induced alterations across six modification types, including phosphorylation, acetylation, methylation, sumoylation, ubiquitination and N-glycosylation.},
  archive      = {J_NMETH},
  author       = {Wen, Bo and Wang, Chenwei and Li, Kai and Han, Ping and Holt, Matthew V. and Savage, Sara R. and Lei, Jonathan T. and Dou, Yongchao and Shi, Zhiao and Li, Yi and Zhang, Bing},
  doi          = {10.1038/s41592-025-02797-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1857-1867},
  shortjournal = {Nat. Meth.},
  title        = {DeepMVP: Deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data. <em>NMETH</em>, <em>22</em>(9), 1846-1856. (<a href='https://doi.org/10.1038/s41592-025-02773-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations such as copy number alterations accumulate during cancer progression, driving intratumor heterogeneity that impacts therapy effectiveness. Understanding the characteristics and spatial distribution of genetically distinct subclones is essential for unraveling tumor evolution and improving cancer treatment. Here we present Clonalscope, a subclone detection method using copy number profiles, applicable to spatial transcriptomics and single-cell sequencing data. Clonalscope implements a nested Chinese Restaurant Process to identify de novo tumor subclones, which can incorporate prior information from matched bulk DNA sequencing data for improved subclone detection and malignant cell labeling. On single-cell RNA sequencing and single-cell assay for transposase-accessible chromatin using sequencing data from gastrointestinal tumors, Clonalscope successfully labeled malignant cells and identified genetically different subclones with thorough validations. On spatial transcriptomics data from various primary and metastasized tumors, Clonalscope labeled malignant spots, traced subclones and identified spatially segregated subclones with distinct differentiation levels and expression of genes associated with drug resistance and survival. Clonalscope is a method for cancer subclone detection leveraging copy number profiles estimated using spatial and single-cell sequencing data.},
  archive      = {J_NMETH},
  author       = {Wu, Chi-Yun and Rong, Jiazhen and Sathe, Anuja and Hess, Paul R. and Lau, Billy T. and Grimes, Susan M. and Huang, Sijia and Ji, Hanlee P. and Zhang, Nancy R.},
  doi          = {10.1038/s41592-025-02773-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1846-1856},
  shortjournal = {Nat. Meth.},
  title        = {Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scvi-hub: An actionable repository for model-driven single-cell analysis. <em>NMETH</em>, <em>22</em>(9), 1836-1845. (<a href='https://doi.org/10.1038/s41592-025-02799-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing availability of single-cell omics datasets presents new opportunities for reuse, while challenges in data transfer, normalization and integration remain a barrier. Here we present scvi-hub: a platform for efficiently sharing and accessing single-cell omics datasets using pretrained probabilistic models. It enables immediate execution of fundamental tasks like visualization, imputation, annotation and deconvolution on new query datasets using state-of-the-art methods, with massively reduced storage and compute requirements. We show that pretrained models support efficient analysis of large references, including the CZI CELLxGENE Discover Census. Scvi-hub is built within the scvi-tools open-source environment and integrated into scverse. Scvi-hub offers a scalable and user-friendly framework for accessing and contributing to a growing ecosystem of ready-to-use models and datasets, thus putting the power of atlas-level analysis at the fingertips of a broad community of users. Scvi-hub is a versatile and efficient platform for model-based analysis of single-cell sequencing studies with access to a diverse array of datasets and downstream analysis.},
  archive      = {J_NMETH},
  author       = {Ergen, Can and Pour Amiri, Valeh Valiollah and Kim, Martin and Kronfeld, Ori and Streets, Aaron and Gayoso, Adam and Yosef, Nir},
  doi          = {10.1038/s41592-025-02799-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1836-1845},
  shortjournal = {Nat. Meth.},
  title        = {Scvi-hub: An actionable repository for model-driven single-cell analysis},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues. <em>NMETH</em>, <em>22</em>(9), 1824-1835. (<a href='https://doi.org/10.1038/s41592-025-02774-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) are essential regulators of RNA fate and function. A long-standing challenge in studying RBP regulation has been mapping RNA interactomes within the dynamic transcriptomic landscape, especially in single-cell contexts and primary tissues. Here we introduce MAPIT-seq (modification added to RBP interacting transcript-sequencing), which uses an antibody-directed editing strategy to map genome-wide in situ RBP–RNA interactions and gene expression concurrently. We demonstrate MAPIT-seq’s robustness across multiple RBPs and systematically analyze RNA substrates associated with core polycomb repressive complex 2 (PRC2) components. MAPIT-seq is also applicable to frozen tissue sections, enabling the mapping of RBP roles during brain development. Importantly, we develop high-throughput single-cell MAPIT-seq (scMAPIT-seq) to reveal cell stage-specific RBP regulation. In summary, MAPIT-seq expands multi-omics profiling, providing an effective framework to study post-transcriptional regulation in dynamic biological processes and clinically relevant scenarios. This work presents MAPIT-seq, an antibody-guided RNA editing method that enables co-profiling of RBP-RNA interactions and gene expression at single-cell resolution and in tissue contexts.},
  archive      = {J_NMETH},
  author       = {Cheng, Qi-Xuan and Xie, Gang and Zhang, Xiangyu and Wang, Jie and Ding, Shuangjin and Wu, Yi-Xia and Shi, Ming and Duan, Fei-Fei and Wan, Zi-Li and Wei, Jing-Jia and Xiao, Junyu and Wang, Yangming},
  doi          = {10.1038/s41592-025-02774-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1824-1835},
  shortjournal = {Nat. Meth.},
  title        = {Co-profiling of in situ RNA-protein interactions and transcriptome in single cells and tissues},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A realistic phantom dataset for benchmarking cryo-ET data annotation. <em>NMETH</em>, <em>22</em>(9), 1819-1823. (<a href='https://doi.org/10.1038/s41592-025-02800-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) is a powerful technique for imaging molecular complexes in their native cellular environments. However, identifying the vast majority of molecular species in cellular tomograms remains prohibitively difficult. Machine learning (ML) methods provide an opportunity to automate the annotation process, but algorithm development has been hindered by the lack of large, standardized datasets. Here we present an experimental phantom dataset with comprehensive ground-truth annotations for six molecular species to spur new algorithm development and benchmark existing tools. This annotated dataset is available on the CryoET Data Portal with infrastructure to streamline access for methods developers across fields. A standardized, realistic phantom dataset consisting of ground-truth annotations for six diverse molecular species is provided as a community resource for cryo-electron-tomography algorithm benchmarking.},
  archive      = {J_NMETH},
  author       = {Peck, Ariana and Yu, Yue and Schwartz, Jonathan and Cheng, Anchi and Ermel, Utz Heinrich and Hutchings, Joshua and Kandel, Saugat and Kimanius, Dari and Montabana, Elizabeth A. and Serwas, Daniel and Siems, Hannah and Wang, Feng and Zhao, Zhuowen and Zheng, Shawn and Haury, Matthias and Agard, David A. and Potter, Clinton S. and Carragher, Bridget and Harrington, Kyle and Paraan, Mohammadreza},
  doi          = {10.1038/s41592-025-02800-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1819-1823},
  shortjournal = {Nat. Meth.},
  title        = {A realistic phantom dataset for benchmarking cryo-ET data annotation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the physical hallmarks of cancer. <em>NMETH</em>, <em>22</em>(9), 1800-1818. (<a href='https://doi.org/10.1038/s41592-024-02564-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The physical microenvironment plays a crucial role in tumor development, progression, metastasis and treatment. Recently, we proposed four physical hallmarks of cancer, with distinct origins and consequences, to characterize abnormalities in the physical tumor microenvironment: (1) elevated compressive–tensile solid stresses, (2) elevated interstitial fluid pressure and the resulting interstitial fluid flow, (3) altered material properties (for example, increased tissue stiffness) and (4) altered physical micro-architecture. As this emerging field of physical oncology is being advanced by tumor biologists, cell and developmental biologists, engineers, physicists and oncologists, there is a critical need for model systems and measurement tools to mechanistically probe these physical hallmarks. Here, after briefly defining these physical hallmarks, we discuss the tools and model systems available for probing each hallmark in vitro, ex vivo, in vivo and in clinical settings. We finally review the unmet needs for mechanistic probing of the physical hallmarks of tumors and discuss the challenges and unanswered questions associated with each hallmark. This Review discusses the current challenges and the available methods to study the four physical hallmarks of cancer.},
  archive      = {J_NMETH},
  author       = {Nia, Hadi T. and Munn, Lance L. and Jain, Rakesh K.},
  doi          = {10.1038/s41592-024-02564-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1800-1818},
  shortjournal = {Nat. Meth.},
  title        = {Probing the physical hallmarks of cancer},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes. <em>NMETH</em>, <em>22</em>(9), 1788-1799. (<a href='https://doi.org/10.1038/s41592-025-02807-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbes within tumors have been recognized and experimentally related to oncogenesis, tumor growth, metastasis and therapeutic responsiveness. Studying the tumor microbiome presents difficulties, as early indications suggest that microbe populations are low in abundance, sparse and highly heterogeneous. Disparate results from computational profiling of the tumor microbiome have cast doubt on the premise of microbes in tumors. Yet decades of experimental evidence support the presence of tumor microbes, at least in a limited number of tumor types. In this Perspective, we discuss the importance of iteratively improving microbe-targeted sequencing techniques, established analytical pipelines, robust computational tools and solid validations to address current challenges and fill existing knowledge gaps. The vast amount of human tumor sequencing data available could greatly enhance systematic investigations of microbiome–tumor interactions with methods to quantify the composition of the tumor microbiome accurately. This Perspective explores the challenges and future directions in the world of human tumor microbiome research.},
  archive      = {J_NMETH},
  author       = {Li, Yingjie and Ma, Anjun and Johnson, Evan and Eng, Charis and De, Subhajyoti and Jiang, Sizun and Li, Zihai and Spakowicz, Daniel and Ma, Qin},
  doi          = {10.1038/s41592-025-02807-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1788-1799},
  shortjournal = {Nat. Meth.},
  title        = {The new microbiome on the block: Challenges and opportunities of using human tumor sequencing data to study microbes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research. <em>NMETH</em>, <em>22</em>(9), 1776-1787. (<a href='https://doi.org/10.1038/s41592-025-02769-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vitro tumor models are essential tools for cancer research, offering key insights into not only tumor biology but also therapeutic responses. The transition from traditional two-dimensional to three-dimensional organoid systems marks a paradigm shift in cancer modeling. Although two-dimensional models have been instrumental in elucidating fundamental molecular and genetic mechanisms, they fail to accurately replicate the intricate three-dimensional architecture and dynamic microenvironment characteristic of human tumors. Here we outline how advanced organoid technologies now enable more faithful recapitulation of tumor heterogeneity that better mimic native tissue mechanics and biochemistry. We discuss emerging methods, including air–liquid interface cultures, microfluidic tumor-on-a-chip devices and high-content imaging integrated with machine learning, which collectively address longstanding challenges such as matrix variability and the limited incorporation of immune and vascular elements. These innovations promise to enhance reproducibility and scalability while providing unprecedented insights into tumor biology, cancer progression and therapeutic strategies. This Perspective discusses recent progresses in development of in vitro tumor models and the current challenges in this field.},
  archive      = {J_NMETH},
  author       = {Rauner, Gat and Gupta, Piyush B. and Kuperwasser, Charlotte},
  doi          = {10.1038/s41592-025-02769-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1776-1787},
  shortjournal = {Nat. Meth.},
  title        = {From 2D to 3D and beyond: The evolution and impact of in vitro tumor models in cancer research},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproducible annotation of t cell subsets and activation states with gene expression programs. <em>NMETH</em>, <em>22</em>(9), 1774-1775. (<a href='https://doi.org/10.1038/s41592-025-02792-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created T-CellAnnoTator (TCAT), a computational method that helps to identify T cell subsets, activation states and functions. It does this using reproducible gene expression programs found across many disease contexts and tissues. TCAT outperforms conventional approaches for T cell subset prediction, is easy to use programmatically or through a website, and can be adapted for other cell types.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02792-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1774-1775},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible annotation of t cell subsets and activation states with gene expression programs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A miniature two-photon microscope detects multiple signals deep in the brains of moving mice. <em>NMETH</em>, <em>22</em>(9), 1772-1773. (<a href='https://doi.org/10.1038/s41592-025-02779-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A lightweight miniature two-photon microscope features multi-wavelength excitation, correction of aberrations, and interchangeable objectives for scalable fields of view. It enables multicolor, deep-brain and scalable neural imaging in freely moving mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02779-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1772-1773},
  shortjournal = {Nat. Meth.},
  title        = {A miniature two-photon microscope detects multiple signals deep in the brains of moving mice},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unravelling cellular interactions using flow cytometry. <em>NMETH</em>, <em>22</em>(9), 1770-1771. (<a href='https://doi.org/10.1038/s41592-025-02743-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a cost-effective ultra-high-throughput cytometry-based framework for the detection of physical interactions between cells, along with the characterization of complex cellular landscapes. Application of our approach can offer a systems-level understanding of immunity and facilitate study of the kinetics, mode of action and personalized response prediction of immunotherapies.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02743-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1770-1771},
  shortjournal = {Nat. Meth.},
  title        = {Unravelling cellular interactions using flow cytometry},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles. <em>NMETH</em>, <em>22</em>(9), 1768-1769. (<a href='https://doi.org/10.1038/s41592-025-02775-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present MAPIT-seq, a method that uses antibody-directed RNA editing to concurrently profile in situ RNA-binding protein (RBP)–RNA interactions and transcriptome-wide gene expression in limited input material, including single cells and frozen tissues. This dual-omic strategy streamlines mechanistic analyses of post-transcriptional regulation in dynamic biological processes and clinically relevant samples.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02775-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1768-1769},
  shortjournal = {Nat. Meth.},
  title        = {MAPIT-seq reveals both RBP targets and transcriptome-wide gene expression profiles},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bridging histology and spatial gene expression across scales. <em>NMETH</em>, <em>22</em>(9), 1765-1767. (<a href='https://doi.org/10.1038/s41592-025-02806-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two deep-learning frameworks — GHIST and iSCALE — turn routine histology images into a rich molecular resource, and predict spatial gene expression at single-cell resolution (GHIST) and at super-resolution across large tissue sections (iSCALE), for scalable, data-driven tissue biology.},
  archive      = {J_NMETH},
  author       = {Ma, Ying},
  doi          = {10.1038/s41592-025-02806-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1765-1767},
  shortjournal = {Nat. Meth.},
  title        = {Bridging histology and spatial gene expression across scales},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Taming cancer evolution surprises. <em>NMETH</em>, <em>22</em>(9), 1760-1764. (<a href='https://doi.org/10.1038/s41592-025-02802-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a focus on the evolutionary forces that shape cancer cell behavior, some researchers work out ways to outmaneuver cancer.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02802-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1760-1764},
  shortjournal = {Nat. Meth.},
  title        = {Taming cancer evolution surprises},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SNAP-tag2 improves live-cell imaging. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02830-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02830-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {SNAP-tag2 improves live-cell imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optical pooled screens shed light on ebola infection. <em>NMETH</em>, <em>22</em>(9), 1759. (<a href='https://doi.org/10.1038/s41592-025-02831-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kamenova, Ivanka},
  doi          = {10.1038/s41592-025-02831-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1759},
  shortjournal = {Nat. Meth.},
  title        = {Optical pooled screens shed light on ebola infection},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ViSOR puts the periphery in focus. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02828-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Willson, Joseph},
  doi          = {10.1038/s41592-025-02828-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {ViSOR puts the periphery in focus},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Benchmarking genomic language models. <em>NMETH</em>, <em>22</em>(9), 1758. (<a href='https://doi.org/10.1038/s41592-025-02829-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-025-02829-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1758},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking genomic language models},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deciphering regulatory sequence grammar. <em>NMETH</em>, <em>22</em>(9), 1757. (<a href='https://doi.org/10.1038/s41592-025-02827-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leveraging data generated by UUATAC-seq, a method for cross-species chromatin accessibility profiling, the deep learning model NvwaCE deciphers cis-regulatory grammar from DNA sequence.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-025-02827-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1757},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering regulatory sequence grammar},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cichlid fishes. <em>NMETH</em>, <em>22</em>(9), 1755-1756. (<a href='https://doi.org/10.1038/s41592-025-02809-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cichlid fishes are a family of thousands of recently evolved species. As charismatic laboratory models, they are useful for studying anatomical, physiological and behavioral traits that vary across these closely related species.},
  archive      = {J_NMETH},
  author       = {Juntti, Scott A.},
  doi          = {10.1038/s41592-025-02809-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1755-1756},
  shortjournal = {Nat. Meth.},
  title        = {Cichlid fishes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Road trip home to start a lab. <em>NMETH</em>, <em>22</em>(9), 1754. (<a href='https://doi.org/10.1038/s41592-025-02801-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As a postdoctoral fellowship ends, for some, next is a trek to their new lab.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02801-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1754},
  shortjournal = {Nat. Meth.},
  title        = {Road trip home to start a lab},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A tale of tumors. <em>NMETH</em>, <em>22</em>(9), 1753. (<a href='https://doi.org/10.1038/s41592-025-02838-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advances for studying the biology of tumors have expanded our understanding of cancer.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02838-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1753},
  shortjournal = {Nat. Meth.},
  title        = {A tale of tumors},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nphys">NPHYS - 29</h2>
<ul>
<li><details>
<summary>
(2025). Precision without parity. <em>NPHYS</em>, <em>21</em>(9), 1498. (<a href='https://doi.org/10.1038/s41567-025-03018-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metrology has long been defined by its pursuit of reproducibility, accuracy and traceability. Yet, when it comes to the representation of women and other marginalized communities, precision has not been matched by parity, finds Solmaz Nadiri.},
  archive      = {J_NPHYS},
  author       = {Nadiri, Solmaz},
  doi          = {10.1038/s41567-025-03018-0},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1498},
  shortjournal = {Nat. Phys.},
  title        = {Precision without parity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: High-efficiency optical training of itinerant two-dimensional magnets. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03013-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Xie, Ti and Liang, Jierui and Bhattacharya, Dhritiman and Arachchige, Hasitha Suriya and Yakovenko, Victor M. and Mandrus, David G. and Qiu, Zi Qiang and Liu, Kai and Gong, Cheng},
  doi          = {10.1038/s41567-025-03013-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Author correction: High-efficiency optical training of itinerant two-dimensional magnets},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03054-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-03054-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient implementation of arbitrary two-qubit gates using unified control. <em>NPHYS</em>, <em>21</em>(9), 1489-1496. (<a href='https://doi.org/10.1038/s41567-025-02990-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The set of quantum logic gates that can be easily implemented is fundamental to the performance of quantum computers, as it governs the accuracy of basic quantum operations and dictates the complexity of implementing quantum algorithms. Traditional approaches to extending gate sets often require operating devices outside the ideal parameter regimes used to realize qubits, leading to increased control complexity while offering only a limited set of gates. Here we experimentally demonstrate a unified and versatile gate scheme capable of generating arbitrary two-qubit gates using only an exchange interaction and qubit driving on a superconducting quantum processor. We achieve high fidelities averaging 99.38% across a wide range of commonly used two-qubit unitaries, enabling precise multipartite entangled state preparation. Furthermore, we successfully produce a B gate, which efficiently synthesizes the entire family of two-qubit gates. Our results establish that fully exploiting the capabilities of the exchange interaction can yield a comprehensive and highly accurate gate set. With maximum expressivity, optimal gate time, demonstrated high fidelity and easy adaption to other quantum platforms, our unified control scheme offers the prospect of improved performance in quantum hardware and algorithm development. The efficiency of a quantum computer depends on which basic operations it can implement. Now a scheme that can implement any two-qubit logic gate has been demonstrated on a superconducting architecture.},
  archive      = {J_NPHYS},
  author       = {Chen, Zhen and Liu, Weiyang and Ma, Yanjun and Sun, Weijie and Wang, Ruixia and Wang, He and Xu, Huikai and Xue, Guangming and Yan, Haisheng and Yang, Zhen and Ding, Jiayu and Gao, Yang and Li, Feiyu and Zhang, Yujia and Zhang, Zikang and Jin, Yirong and Yu, Haifeng and Chen, Jianxin and Yan, Fei},
  doi          = {10.1038/s41567-025-02990-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1489-1496},
  shortjournal = {Nat. Phys.},
  title        = {Efficient implementation of arbitrary two-qubit gates using unified control},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1482-1488. (<a href='https://doi.org/10.1038/s41567-025-02989-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-fidelity quantum optomechanical control of a mechanical oscillator requires the ability to perform efficient, low-noise operations on long-lived phononic excitations. Microfabricated high-overtone bulk acoustic wave resonators (μHBARs) support high-frequency mechanical modes above 10 GHz with coherence times exceeding one millisecond. Here we demonstrate a μHBAR-based cavity optomechanical system that permits quantum optomechanical control of individual high-coherence phonon modes. We perform laser cooling of the phonon modes from an occupation of approximately 22 phonons to fewer than 0.4, corresponding to laser-based ground-state cooling of a mechanical object with a mass of 7.5 μg. During the cooling process we do not observe any absorption-induced heating, demonstrating the resilience of the HBAR optomechanical systems against parasitic heating. Our work demonstrates that μHBARs are promising as the basis for quantum optomechanical systems with robustness to decoherence that is necessary for efficient, low-noise photon–phonon conversion. A massive phonon mode in a high-overtone bulk acoustic wave resonator has been laser cooled close to its ground state. Its robustness to decoherence establishes the potential of these devices for quantum technologies.},
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-02989-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1482-1488},
  shortjournal = {Nat. Phys.},
  title        = {Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fast optical control of a coherent hole spin in a microcavity. <em>NPHYS</em>, <em>21</em>(9), 1475-1481. (<a href='https://doi.org/10.1038/s41567-025-02988-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many of the most promising quantum information platforms store quantum information in electronic or atomic spins. To incorporate these devices into quantum networks, a spin–photon interface is required. Currently, the best on-demand single-photon sources use a semiconductor quantum dot in an engineered photonic environment. However, it is difficult to achieve coherent spin control in a high-performance single-photon source, and spin coherence is limited by magnetic noise from nuclear spins in the semiconductor host material. Here we combine all-optical spin control with a quantum dot in an open microcavity. We demonstrate fast coherent rotations of a hole spin around an arbitrary axis of the Bloch sphere with a maximum π-pulse fidelity of 98.6%. To suppress the slow magnetic noise, we laser cool the nuclear spins using the hole as a central spin. This extends the hole spin free-induction-decay time by more than an order of magnitude. It becomes much larger than both rotation time and radiative recombination time of the spin, enabling the creation of many spin–photon pairs before the loss of spin coherence. Spin–photon interfaces provide a connection between quantum information stored in atomic or electronic spins and optical communications networks. A quantum photon emitter with long-lived, controllable coherent spin has now been demonstrated.},
  archive      = {J_NPHYS},
  author       = {Hogg, Mark R. and Antoniadis, Nadia O. and Marczak, Malwina A. and Nguyen, Giang N. and Baltisberger, Timon L. and Javadi, Alisa and Schott, Rüdiger and Valentin, Sascha R. and Wieck, Andreas D. and Ludwig, Arne and Warburton, Richard J.},
  doi          = {10.1038/s41567-025-02988-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1475-1481},
  shortjournal = {Nat. Phys.},
  title        = {Fast optical control of a coherent hole spin in a microcavity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A mechanical quantum memory for microwave photons. <em>NPHYS</em>, <em>21</em>(9), 1469-1474. (<a href='https://doi.org/10.1038/s41567-025-02975-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Superconducting qubits possess outstanding capabilities for processing quantum information in the microwave domain; however they have limited coherence times. An interface between photons and phonons could allow quantum information to be stored in long-lived mechanical oscillators. Here, we introduce a platform that relies on electrostatic forces in nanoscale structures to achieve strong coupling between a superconducting qubit and a nanomechanical oscillator with an energy decay time (T1) of approximately 25 ms, well beyond those achieved in integrated superconducting circuits. We use quantum operations in this system to investigate the microscopic origins of mechanical decoherence and mitigate its impact. By using two-pulse dynamical decoupling sequences, we can extend the coherence time (T2) from 64 μs to 1 ms. These findings establish that mechanical oscillators can act as quantum memories for superconducting devices, with potential future applications in quantum computing, sensing and transduction. Superconducting qubits, a leading platform for quantum information processing, suffer from decoherence. Interfacing them with nanomechanical oscillators allows quantum information to be stored in motional states with longer lifetimes.},
  archive      = {J_NPHYS},
  author       = {Bozkurt, Alkım B. and Golami, Omid and Yu, Yue and Tian, Hao and Mirhosseini, Mohammad},
  doi          = {10.1038/s41567-025-02975-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1469-1474},
  shortjournal = {Nat. Phys.},
  title        = {A mechanical quantum memory for microwave photons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics. <em>NPHYS</em>, <em>21</em>(9), 1462-1468. (<a href='https://doi.org/10.1038/s41567-025-02973-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Skyrmions are topologically stable quasiparticles that have been investigated in contexts including particle physics, quantum field theory, acoustics and condensed-matter physics. Quantum optical skyrmions with local topological textures are expected to reshape the landscape of quantum photonic technology, although their experimental implementation has not yet been demonstrated. Here we present experimental realizations of nanophotonic quantum skyrmions using a semiconductor cavity quantum electrodynamics system. By manipulating the photonic spin–orbit coupling in a Gaussian microcavity, we obtained a confined optical mode whose polarizations feature skyrmionic topologies. With pronounced cavity quantum electrodynamics effects, we generated and detected single-photon skyrmions from a solid-state quantum emitter deterministically coupled to the Gaussian microcavity. The polarity associated with single-photon skyrmions can be swapped by flipping the polarization of the quantum emitter through the Zeeman effect. We also investigated the topological protection of quantum optical skyrmions under different perturbations. Our work opens an unexplored aspect of quantum light–matter interactions in the nanoscale and might advance resilient photonic quantum technology with high-dimensional qubits and high-capacity quantum memories. Quantum optical skyrmions are promising for quantum photonic applications but have not been experimentally realized. Now nanophotonic quantum skyrmions are generated using a semiconductor quantum dot–Gaussian microcavity quantum electrodynamics system.},
  archive      = {J_NPHYS},
  author       = {Ma, Jiantao and Yang, Jiawei and Liu, Shunfa and Chen, Bo and Li, Xueshi and Song, Changkun and Qiu, Guixin and Zou, Kai and Hu, Xiaolong and Li, Feng and Yu, Ying and Liu, Jin},
  doi          = {10.1038/s41567-025-02973-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1462-1468},
  shortjournal = {Nat. Phys.},
  title        = {Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The actin cortex acts as a mechanical memory of morphology in confined migrating cells. <em>NPHYS</em>, <em>21</em>(9), 1451-1461. (<a href='https://doi.org/10.1038/s41567-025-02980-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell migration in narrow microenvironments occurs in numerous physiological processes. It involves successive cycles of confinement and release that drive important morphological changes. However, it remains unclear whether migrating cells can retain a memory of their past morphological states that could potentially facilitate their navigation through confined spaces. We demonstrate that local geometry governs a switch between two cell morphologies, thereby facilitating cell passage through long and narrow gaps. We combined cell migration assays on standardized microsystems with biophysical modelling and biochemical perturbations to show that migrating cells have a long-term memory of past confinement events. The morphological cell states correlate across transitions through actin cortex remodelling. These findings indicate that mechanical memory in migrating cells plays an active role in their migratory potential in confined environments. Cells often navigate through confined spaces. Now it is shown that cells retain a mechanical memory of previous confinement events, which makes them more efficient at migrating through narrow microenvironments.},
  archive      = {J_NPHYS},
  author       = {Kalukula, Yohalie and Luciano, Marine and Simanov, Gleb and Charras, Guillaume and Brückner, David B. and Gabriele, Sylvain},
  doi          = {10.1038/s41567-025-02980-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1451-1461},
  shortjournal = {Nat. Phys.},
  title        = {The actin cortex acts as a mechanical memory of morphology in confined migrating cells},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capillary interactions drive the self-organization of bacterial colonies. <em>NPHYS</em>, <em>21</em>(9), 1444-1450. (<a href='https://doi.org/10.1038/s41567-025-02965-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many bacteria inhabit thin water layers on solid surfaces. These thin films occur both naturally—in soils, on hosts and on textiles—and in the laboratory on agar hydrogels. In these environments, cells experience capillary forces, but it is unclear how these forces shape bacterial collective behaviour. Here we show that the water menisci formed around bacteria lead to capillary attraction between cells while still allowing them to slide past one another. We develop an experimental apparatus that allows us to control bacterial collective behaviour by varying the strength and range of capillary forces. Combining three-dimensional imaging and cell tracking with agent-based modelling, we demonstrate that capillary attraction organizes rod-shaped bacteria into densely packed nematic groups and influences their collective dynamics and morphologies. Our results suggest that capillary forces may be a ubiquitous physical ingredient in shaping microbial communities in partially hydrated environments. Bacteria tend to live in thin layers of water on surfaces. Now the capillary forces in these layers are shown to help organize the bacteria into dense packs.},
  archive      = {J_NPHYS},
  author       = {Black, Matthew E. and Fei, Chenyi and Alert, Ricard and Wingreen, Ned S. and Shaevitz, Joshua W.},
  doi          = {10.1038/s41567-025-02965-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1444-1450},
  shortjournal = {Nat. Phys.},
  title        = {Capillary interactions drive the self-organization of bacterial colonies},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion. <em>NPHYS</em>, <em>21</em>(9), 1431-1443. (<a href='https://doi.org/10.1038/s41567-025-02964-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells can sense and respond to their environment. Central to this process is the formation of molecular clutches, which are dynamic linkages between the extracellular matrix and the actin cytoskeleton mediated by integrins and adaptor proteins. Although it is well known that force-dependent interactions between molecular-clutch components are essential for sensing substrate rigidity, the influence of nonlinear adaptor-protein elasticity is poorly understood. Here we show that adaptor-protein elasticity and local interactions between molecular clutches and the extracellular matrix are key to cellular perception of substrate stiffness. We present a semi-analytical theory that integrates experimentally measured force responses of adaptor proteins to describe cell-adhesion sensing. We propose that molecular clutches probe local mechanical substrate properties and collectively function as a differential that allows a retrograde actin flow and substrate movement to occur at different rates while maintaining a stable mechanical coupling between them. Our model reproduces experimentally measured force-loading rates of individual molecular clutches and correctly predicts cell-adhesion behaviour for a range of substrate stiffnesses. The framework presented can be extended to study complex phenomena such as focal-adhesion maturation and cell-type-specific mechanosensing. The cytoskeleton and extracellular matrix are linked by molecular clutches that enable cells to sense their environment. Now it is shown that the elastic properties of the clutches underpin stiffness sensing.},
  archive      = {J_NPHYS},
  author       = {Liu, Ping and Wang, Qiuyu and Dai, Xin and Pei, Lingxuan and Wang, Junfan and Zhao, Weiyi and Johnson, Heath E. and Yao, Mingxi and Efremov, Artem K.},
  doi          = {10.1038/s41567-025-02964-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1431-1443},
  shortjournal = {Nat. Phys.},
  title        = {Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility. <em>NPHYS</em>, <em>21</em>(9), 1420-1430. (<a href='https://doi.org/10.1038/s41567-025-02983-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microtubules are hollow cylindrical cytoskeletal polymers of laterally associated protofilaments that contain head-to-tail aligned ɑ/β-tubulin heterodimers. Although the exposed microtubule exterior is readily accessible to proteins, the mechanism governing the accessibility of the confined microtubule lumen to luminal particles remains unknown. Here we show that certain tubulin family proteins (isotypes) facilitate luminal accessibility because of the mechanical properties and lateral interactions that they confer to the microtubules. We characterized the microtubules reconstituted from defined compositions of Caenorhabditis elegans tubulin isotypes. These tubulin isotypes form microtubules with comparable protofilament numbers but different luminal accessibility. We further revealed the role of tubulin isotypes in regulating the strength of inter-protofilament lateral interactions, which determines luminal accessibility through the mechanosensitivity of reversible protofilament separation. Deformation of the microtubule lattice, which generates stresses exceeding the strength of the lateral interactions, creates gaps between adjacent protofilaments, enhancing the accessibility of the lumen. Together, our findings uncovered the tubulin isotype-dependent mechanical plasticity that confers force sensitivity to the microtubule lattice and modulates the energy barrier for luminal proteins to access the lumen. Proteins interact with both the exterior and interior of microtubules. Here the relationship between microtubule building blocks and the accessibility of the microtubule interior to proteins is clarified.},
  archive      = {J_NPHYS},
  author       = {Ye, Yucheng and Hao, Zheng and Luo, Jingyi and Lam, Wai Hei and Liu, Zheng and Li, Xiang David and Zhai, Yuanliang and Lin, Yuan and Ti, Shih-Chieh},
  doi          = {10.1038/s41567-025-02983-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1420-1430},
  shortjournal = {Nat. Phys.},
  title        = {Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Control of collective activity to crystallize an oscillator gas. <em>NPHYS</em>, <em>21</em>(9), 1412-1419. (<a href='https://doi.org/10.1038/s41567-025-02957-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motility-induced phase separation occurs in assemblies of self-propelled units when activity is coupled negatively to density. By contrast, the consequences of a positive coupling between density and activity on the collective behaviour of active matter remain unexplored. Here we show that collective activity can emerge from such a positive coupling among non-motile building blocks. We perform experiments with self-sustained oscillators powered by contact-charge electrophoresis. Although the oscillators are non-motile by design, they spontaneously form an active gas when confined together. The super-elastic nature of collisions constitutes a positive density–activity coupling and underlies the active gas properties. Elucidating the origin of binary collisions allows us to precisely control the structure of the active gas and its eventual crystallization. Beyond considering the overlooked positive coupling between density and activity, our work suggests that rich collective properties can emerge not only from the symmetry of interactions between active building blocks but also from their adaptable and responsive behaviour. In active matter systems, increasing density usually reduces activity. Now, in a system where density enhances activity, collective motion is shown to arise from non-motile oscillators when they are confined.},
  archive      = {J_NPHYS},
  author       = {Le Blay, Marine and Saldi, Joshua H. K. and Morin, Alexandre},
  doi          = {10.1038/s41567-025-02957-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1412-1419},
  shortjournal = {Nat. Phys.},
  title        = {Control of collective activity to crystallize an oscillator gas},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multistable polar textures in geometrically frustrated nematic liquid crystals. <em>NPHYS</em>, <em>21</em>(9), 1404-1411. (<a href='https://doi.org/10.1038/s41567-025-02966-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to manipulate polar entities with multiple external fields could enable functionalities and applications in spin systems, photonics, metamaterials and soft matter. Liquid crystals that exhibit both a crystalline structure and liquid fluidity represent a promising platform for manipulating phases with polar molecular order, notably ferroelectric ones. However, achieving a polar symmetry is challenging with rod-shaped liquid crystal molecules, which form predominantly apolar nematic phases. Here we report an approach in which a geometric lattice confinement of nematic liquid crystals is used to induce planar polar order on the scale of a mesoscopic metamaterial. We confine the nematic liquid crystal in a micropillar array, forming topological defect–pillar pairs of elastic dipoles with a free top interface in contact with an immiscible fluid. The resulting dipole lattice configurations can be programmed rheologically by flowing the top fluid and maintained even after flow cessation, a phenomenon attributed to orientational multistability of the dipoles. This multimemory effect enables the encoding and reconfiguration of directional information. Overall, these results advance our understanding of topological dipoles under confinement and shear flow, enabling the detection, tracking and recording of flow profiles and could facilitate the development of stimuli-responsive materials. Programmable topological dipoles could be used in stimuli-responsive materials. Now it is shown that confinement of an apolar liquid crystal results in controllable polar arrangements of elastic dipoles.},
  archive      = {J_NPHYS},
  author       = {Kara, Ufuoma I. and Chen, Boyuan and Čopar, Simon and Li, Shucong and Mamtani, Rajdeep and Xu, Yang and Weible, Alan H. and Boerner, Eric C. and Yang, Zhan and Yao, Yuxing and Selinger, Robin L. B. and Tkalec, Uroš and Wang, Xiaoguang},
  doi          = {10.1038/s41567-025-02966-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1404-1411},
  shortjournal = {Nat. Phys.},
  title        = {Multistable polar textures in geometrically frustrated nematic liquid crystals},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability. <em>NPHYS</em>, <em>21</em>(9), 1398-1403. (<a href='https://doi.org/10.1038/s41567-025-02982-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Topological defects give rise to non-trivial excitations—such as solitons, vortices and skyrmions—characterized by integer-valued topological charges. However, their classification becomes difficult when singularities are present in the order parameter field, complicating the computation of these charges. Although exotic nonlinear excitations have been theoretically proposed in the superfluid 3He-A phase and spinor Bose–Einstein condensates, they have not been experimentally observed and their stability has not been investigated. Here we demonstrate the presence of a singular skyrmion that goes beyond the framework of topology in a ferromagnetic superfluid. These skyrmions emerge from anomalous symmetry-breaking associated with an eccentric spin singularity and carry half the elementary charge—a feature that distinguishes them from conventional skyrmions and merons. We realize the universal regime of the quantum Kelvin–Helmholtz instability, and we identify the unconventional fractional skyrmions produced by emission from a magnetic domain wall and the spontaneous splitting of an integer skyrmion with spin singularities. The singular skyrmions are stable even after 2 s of hold time. Our results confirm the universality between classical and quantum Kelvin–Helmholtz instabilities and broaden our understanding of complex nonlinear dynamics for a non-trivial texture in topological quantum systems. The quantum Kelvin–Helmholtz instability can induce non-trivial topological excitations. Now a proof-of-principle experiment demonstrates the emergence of a stable singular fractional skyrmion spin texture originating from this quantum instability.},
  archive      = {J_NPHYS},
  author       = {Huh, SeungJung and Yun, Wooyoung and Yun, Gabin and Hwang, Samgyu and Kwon, Kiryang and Hur, Junhyeok and Lee, Seungho and Takeuchi, Hiromitsu and Kim, Se Kwon and Choi, Jae-yoon},
  doi          = {10.1038/s41567-025-02982-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1398-1403},
  shortjournal = {Nat. Phys.},
  title        = {Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Anyonic braiding in a chiral Mach–Zehnder interferometer. <em>NPHYS</em>, <em>21</em>(9), 1392-1397. (<a href='https://doi.org/10.1038/s41567-025-02960-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fractional quantum statistics are the defining characteristic of anyons. Measuring the phase generated by an exchange of anyons is challenging, as standard interferometry set-ups—such as the Fabry–Pérot interferometer—suffer from charging effects that obscure the interference signal. Here we present the observation of anyonic interference and exchange phases in an optical-like Mach–Zehnder interferometer based on co-propagating interface modes. By avoiding backscattering and deleterious charging effects, this set-up enables pristine and robust Aharonov–Bohm interference without any phase slips. At various fractional filling factors, the observed flux periodicities agree with the fundamental fractionally charged excitations that correspond to Jain states and depend only on the bulk topological order. To probe the anyonic statistics, we used a small, charged top gate in the interferometer bulk to induce localized quasiparticles without modifying the Aharonov–Bohm phase. The added quasiparticles introduce periodic phase slips. The sign and magnitude of the observed phase slips align with the expected value at filling 1/3, but their direction shows systematic deviations at fillings 2/5 and 3/7. Control over added individual quasiparticles in this design is essential for measuring the coveted non-abelian statistics in the future. An interferometer design allows systematic investigation of the anyonic statistics of bulk fractional quantum Hall states.},
  archive      = {J_NPHYS},
  author       = {Ghosh, Bikash and Labendik, Maria and Musina, Liliia and Umansky, Vladimir and Heiblum, Moty and Mross, David F.},
  doi          = {10.1038/s41567-025-02960-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1392-1397},
  shortjournal = {Nat. Phys.},
  title        = {Anyonic braiding in a chiral Mach–Zehnder interferometer},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Measurement of phonon angular momentum. <em>NPHYS</em>, <em>21</em>(9), 1387-1391. (<a href='https://doi.org/10.1038/s41567-025-02952-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In condensed matter, angular momentum is intimately related to the emergence of topological quantum states, including chiral superconductivity, quantum spin liquids and various chiral quasiparticles. Recently, it has been predicted that microscopic lattice excitations, known as phonons, can carry finite angular momentum, leading to specific physical properties of materials. However, phonon angular momentum has not yet been observed directly. Here we demonstrate that angular momentum conservation results in a macroscopic mechanical torque when applying a time-reversal symmetry-breaking thermal gradient along the chiral axis of single-crystal tellurium. We probe this torque using a cantilever-based device and establish that it changes sign by flipping the chirality or thermal gradient. This behavior disappears in polycrystalline samples that lack a preferred chirality. Our experimental results align well with theoretical calculations. We provide compelling evidence for phonon angular momentum, which might enable quantum states with potential applications in microelectronics. Theory predicts that phonons—quanta of lattice vibrations—can carry finite angular momentum and thus influence physical properties of materials. Now phonons with angular momentum have been seen in tellurium with a chiral crystal structure.},
  archive      = {J_NPHYS},
  author       = {Zhang, H. and Peshcherenko, N. and Yang, Fazhi and Ward, T. Z. and Raghuvanshi, P. and Lindsay, L. and Felser, Claudia and Zhang, Yang and Yan, J.-Q. and Miao, H.},
  doi          = {10.1038/s41567-025-02952-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1387-1391},
  shortjournal = {Nat. Phys.},
  title        = {Measurement of phonon angular momentum},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Strongly interacting hofstadter states in magic-angle twisted bilayer graphene. <em>NPHYS</em>, <em>21</em>(9), 1380-1386. (<a href='https://doi.org/10.1038/s41567-025-02997-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Magic-angle twisted bilayer graphene hosts a variety of strongly correlated states at partial fillings of its flat bands. In a magnetic field, these flat bands evolve into a Hofstadter spectrum renormalized by strong Coulomb interactions. Here we study the interacting Hofstadter states that spontaneously form within the topological magnetic sub-bands of an ultraclean magic-angle twisted bilayer graphene device, including symmetry-broken Chern insulator states and fractional quantum Hall states. The observed symmetry-broken Chern insulator states form a cascade, with their Chern numbers mimicking the main sequence of correlated Chern insulators. The fractional quantum Hall states form in a Jain sequence. However, they disappear at high magnetic field, in contrast to conventional fractional quantum Hall states that strengthen with increasing magnetic field. We reveal a magnetic-field-driven phase transition from composite fermion phases to a dissipative Fermi liquid. Our theoretical analysis of the magnetic sub-bands hosting the fractional quantum Hall states predicts non-uniform quantum geometric properties far from the lowest Landau level. This points towards a more natural interpretation of these states as in-field fractional Chern insulators of the magnetic sub-bands. Interacting electrons in Hofstadter bands can form symmetry-broken topological states. These are now revealed in magic-angle twisted bilayer graphene, and their properties are influenced by non-uniform quantum geometry.},
  archive      = {J_NPHYS},
  author       = {He, Minhao and Wang, Xiaoyu and Cai, Jiaqi and Herzog-Arbeitman, Jonah and Peng, Ran and Taniguchi, Takashi and Watanabe, Kenji and Stern, Ady and Bernevig, B. Andrei and Yankowitz, Matthew and Vafek, Oskar and Xu, Xiaodong},
  doi          = {10.1038/s41567-025-02997-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1380-1386},
  shortjournal = {Nat. Phys.},
  title        = {Strongly interacting hofstadter states in magic-angle twisted bilayer graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Universality in quantum critical flow of charge and heat in ultraclean graphene. <em>NPHYS</em>, <em>21</em>(9), 1374-1379. (<a href='https://doi.org/10.1038/s41567-025-02972-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Close to the Dirac point, graphene is expected to exist in a quantum critical Dirac fluid state, where the flow of both charge and heat can be described with a characteristic d.c. electrical conductivity and thermodynamic variables such as entropy and enthalpy densities. Although the fluid-like viscous flow of charge has been reported in state-of-the-art graphene devices, the value of conductivity, predicted to be quantized and determined only by the universality class of the critical point, has not been established experimentally so far. Here we have discerned the quantum critical universality in graphene transport by combining the electrical and thermal conductivities in very high-quality devices close to the Dirac point. We find that they are inversely related, as expected from relativistic hydrodynamics, and the characteristic conductivity converges to a quantized value. We also observe a giant violation of the Wiedemann–Franz law, where the Lorentz number exceeds the semiclassical value by more than 200 times close to the Dirac point at low temperatures. At high temperatures, the effective dynamic viscosity to entropy density ratio close to the Dirac point in the cleanest devices approaches that of a minimally viscous quantum fluid within a factor of four. Critical behaviour is expected in graphene when the carrier density is tuned to the Dirac point. Now, universality associated with the critical point is observed in electronic and thermal transport.},
  archive      = {J_NPHYS},
  author       = {Majumdar, Aniket and Chadha, Nisarg and Pal, Pritam and Gugnani, Akash and Ghawri, Bhaskar and Watanabe, Kenji and Taniguchi, Takashi and Mukerjee, Subroto and Ghosh, Arindam},
  doi          = {10.1038/s41567-025-02972-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1374-1379},
  shortjournal = {Nat. Phys.},
  title        = {Universality in quantum critical flow of charge and heat in ultraclean graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerator technologies for proton and ion beam therapy. <em>NPHYS</em>, <em>21</em>(9), 1363-1373. (<a href='https://doi.org/10.1038/s41567-025-02994-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past 75 years, proton beam therapy has emerged as a promising modality for cancer treatment, boasting precise targeting and reduced collateral damage to healthy tissue. Here we discuss the evolution of accelerator technology in proton therapy, examining advancements in cyclotron, synchrotron and linear accelerator technology, and their implications for modern treatment delivery. Additionally, we explore advances in delivering accelerated carbon or helium ions for therapeutic treatments. We also discuss the integration of advanced imaging modalities, such as multienergy X-ray, magnetic resonance imaging and ion-based imaging, for real-time monitoring and adaptive radiotherapy. These advancements position particle therapy to offer personalized and effective cancer treatment strategies, heralding improved patient outcomes. Proton and heavy-ion beam therapies are promising approaches for cancer treatment. This Review discusses the accelerator technologies that enable these therapies.},
  archive      = {J_NPHYS},
  author       = {Maradia, Vivek and Clasie, Benjamin and Snively, Emma and Parodi, Katia and Schwarz, Marco and Durante, Marco},
  doi          = {10.1038/s41567-025-02994-7},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1363-1373},
  shortjournal = {Nat. Phys.},
  title        = {Accelerator technologies for proton and ion beam therapy},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Extended native gate sets to unlock the performance of quantum processors. <em>NPHYS</em>, <em>21</em>(9), 1361-1362. (<a href='https://doi.org/10.1038/s41567-025-02991-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum operations are modelled as unitary matrices, yet experimental implementations have been restricted to only a few gate types. This fundamental limitation has now been overcome using a scheme that combines frequency control with microwave driving, enabling universal implementation of arbitrary two-qubit operations.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02991-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1361-1362},
  shortjournal = {Nat. Phys.},
  title        = {Extended native gate sets to unlock the performance of quantum processors},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How squeezed cells remember their shape to migrate efficiently. <em>NPHYS</em>, <em>21</em>(9), 1359-1360. (<a href='https://doi.org/10.1038/s41567-025-02984-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells migrating through narrow spaces in their environment undergo repeated shape changes to pass through tight constrictions. Epithelial cells retain a memory of past confinement, allowing them to maintain a polarized, compact morphology that enhances future migration through narrow gaps. This memory is mechanically encoded in the actin cortex.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02984-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1359-1360},
  shortjournal = {Nat. Phys.},
  title        = {How squeezed cells remember their shape to migrate efficiently},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantized limit of conductivity in near-ideal graphene. <em>NPHYS</em>, <em>21</em>(9), 1357-1358. (<a href='https://doi.org/10.1038/s41567-025-02974-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transport properties near the Dirac point in graphene are expected to be determined by quantum many-body interactions between relativistic electrons. Experiments now show that the flow of charge and heat in high-quality graphene close to charge neutrality can be described within a hydrodynamic framework, with universal intrinsic electrical conductivity that is quantized to a value close to the quantum of conductance.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02974-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1357-1358},
  shortjournal = {Nat. Phys.},
  title        = {Quantized limit of conductivity in near-ideal graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boron at the boundaries. <em>NPHYS</em>, <em>21</em>(9), 1356. (<a href='https://doi.org/10.1038/s41567-025-03041-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Brierley, Richard},
  doi          = {10.1038/s41567-025-03041-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1356},
  shortjournal = {Nat. Phys.},
  title        = {Boron at the boundaries},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Liquid dipole lattice. <em>NPHYS</em>, <em>21</em>(9), 1354-1355. (<a href='https://doi.org/10.1038/s41567-025-02992-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geometrically confined apolar nematic liquid crystals lose rotational symmetry and form a multifunctional polar lattice of fluid elastic dipoles with controllable orientation. The lattice is of interest from a fundamental science perspective as well as having potential applications.},
  archive      = {J_NPHYS},
  author       = {Lucchetti, Liana},
  doi          = {10.1038/s41567-025-02992-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1354-1355},
  shortjournal = {Nat. Phys.},
  title        = {Liquid dipole lattice},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How to twist anyons. <em>NPHYS</em>, <em>21</em>(9), 1352-1353. (<a href='https://doi.org/10.1038/s41567-025-03009-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The properties of quantum Hall states can be probed in experiments inspired by optical interferometers. Now, a device design that overcomes major drawbacks of existing interferometers allows the extraction of the quasiparticle statistics.},
  archive      = {J_NPHYS},
  author       = {Carrega, Matteo and Heun, Stefan},
  doi          = {10.1038/s41567-025-03009-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1352-1353},
  shortjournal = {Nat. Phys.},
  title        = {How to twist anyons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In plain sight. <em>NPHYS</em>, <em>21</em>(9), 1351. (<a href='https://doi.org/10.1038/s41567-025-03021-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Buchanan, Mark},
  doi          = {10.1038/s41567-025-03021-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1351},
  shortjournal = {Nat. Phys.},
  title        = {In plain sight},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Invest in science education research to make science open to all. <em>NPHYS</em>, <em>21</em>(9), 1350. (<a href='https://doi.org/10.1038/s41567-025-03019-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attacks on Diversity, Equity and Inclusion efforts expose the need for science education research.},
  archive      = {J_NPHYS},
  author       = {Holmes, N. G.},
  doi          = {10.1038/s41567-025-03019-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1350},
  shortjournal = {Nat. Phys.},
  title        = {Invest in science education research to make science open to all},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Serendipitous signal. <em>NPHYS</em>, <em>21</em>(9), 1349. (<a href='https://doi.org/10.1038/s41567-025-03052-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first direct observation of gravitational waves occurred on 14 September 2015. We look back at ten years of gravitational wave science.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-03052-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1349},
  shortjournal = {Nat. Phys.},
  title        = {Serendipitous signal},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
