<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>nature</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs">NATCS - 18</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Probing the limitations of multimodal language models for chemistry and materials research. <em>NATCS</em>, <em>5</em>(10), 973. (<a href='https://doi.org/10.1038/s43588-025-00869-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Alampara, Nawaf and Schilling-Wilhelmi, Mara and Ríos-García, Martiño and Mandal, Indrajeet and Khetarpal, Pranav and Grover, Hargun Singh and Krishnan, N. M. Anoop and Jablonka, Kevin Maik},
  doi          = {10.1038/s43588-025-00869-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {973},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Probing the limitations of multimodal language models for chemistry and materials research},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SciToolAgent: A knowledge-graph-driven scientific agent for multitool integration. <em>NATCS</em>, <em>5</em>(10), 962-972. (<a href='https://doi.org/10.1038/s43588-025-00849-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientific research increasingly relies on specialized computational tools, yet effectively utilizing these tools requires substantial domain expertise. While large language models show promise in tool automation, they struggle to seamlessly integrate and orchestrate multiple tools for complex scientific workflows. Here we present SciToolAgent, a large language model-powered agent that automates hundreds of scientific tools across biology, chemistry and materials science. At its core, SciToolAgent leverages a scientific tool knowledge graph that enables intelligent tool selection and execution through graph-based retrieval-augmented generation. The agent also incorporates a comprehensive safety-checking module to ensure responsible and ethical tool usage. Extensive evaluations on a curated benchmark demonstrate that SciToolAgent outperforms existing approaches. Case studies in protein engineering, chemical reactivity prediction, chemical synthesis and metal–organic framework screening further demonstrate SciToolAgent’s capability to automate complex scientific workflows, making advanced research tools accessible to both experts and nonexperts. This study presents SciToolAgent, a large language model-based agent that orchestrates scientific tools via a knowledge graph, enabling automated and effective execution of scientific research workflows.},
  archive      = {J_NATCS},
  author       = {Ding, Keyan and Yu, Jing and Huang, Junjie and Yang, Yuchen and Zhang, Qiang and Chen, Huajun},
  doi          = {10.1038/s43588-025-00849-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {962-972},
  shortjournal = {Nat. Comput. Sci},
  title        = {SciToolAgent: A knowledge-graph-driven scientific agent for multitool integration},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the limitations of multimodal language models for chemistry and materials research. <em>NATCS</em>, <em>5</em>(10), 952-961. (<a href='https://doi.org/10.1038/s43588-025-00836-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advancements in artificial intelligence have sparked interest in scientific assistants that could support researchers across the full spectrum of scientific workflows, from literature review to experimental design and data analysis. A key capability for such systems is the ability to process and reason about scientific information in both visual and textual forms—from interpreting spectroscopic data to understanding laboratory set-ups. Here we introduce MaCBench, a comprehensive benchmark for evaluating how vision language models handle real-world chemistry and materials science tasks across three core aspects: data extraction, experimental execution and results interpretation. Through a systematic evaluation of leading models, we find that although these systems show promising capabilities in basic perception tasks—achieving near-perfect performance in equipment identification and standardized data extraction—they exhibit fundamental limitations in spatial reasoning, cross-modal information synthesis and multi-step logical inference. Our insights have implications beyond chemistry and materials science, suggesting that developing reliable multimodal AI scientific assistants may require advances in curating suitable training data and approaches to training those models. A comprehensive benchmark, called MaCBench, is developed to evaluate how vision language models handle different aspects of real-world chemistry and materials science tasks.},
  archive      = {J_NATCS},
  author       = {Alampara, Nawaf and Schilling-Wilhelmi, Mara and Ríos-García, Martiño and Mandal, Indrajeet and Khetarpal, Pranav and Grover, Hargun Singh and Krishnan, N. M. Anoop and Jablonka, Kevin Maik},
  doi          = {10.1038/s43588-025-00836-3},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {952-961},
  shortjournal = {Nat. Comput. Sci},
  title        = {Probing the limitations of multimodal language models for chemistry and materials research},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Real-time raw signal genomic analysis using fully integrated memristor hardware. <em>NATCS</em>, <em>5</em>(10), 940-951. (<a href='https://doi.org/10.1038/s43588-025-00867-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in third-generation sequencing have enabled portable and real-time genomic sequencing, but real-time data processing remains a bottleneck, hampering on-site genomic analysis. These technologies generate noisy analog signals that traditionally require basecalling and read mapping, both demanding costly data movement on von Neumann hardware. Here, to overcome this, we present a memristor-based hardware–software codesign that processes raw sequencer signals directly in analog memory, combining the two separated steps. By exploiting intrinsic device noise for locality-sensitive hashing and implementing parallel approximate searches in content-addressable memory, we experimentally showcase on-site applications, including infectious disease detection and metagenomic classification on a fully integrated memristor chip. Our experimentally validated analysis confirms the effectiveness of this approach on real-world tasks, achieving a 97.15% F1 score in virus raw signal mapping, with 51× speed-up and 477× energy saving over an application-specific integrated circuit. These results demonstrate that in-memory computing hardware provides a viable solution for integration with portable sequencers, enabling real-time and on-site genomic analysis. The authors report a memristor-based system that analyzes raw analog signals from a genomic sequencer directly in memory. By bypassing slow data conversion, the system achieves substantial improvements in speed and efficiency, enabling real-time, on-site genomic analysis.},
  archive      = {J_NATCS},
  author       = {He, Peiyi and Wang, Shengbo and Mao, Ruibin and Jiang, Mingrui and Siegel, Sebastian and Pedretti, Giacomo and Ignowski, Jim and Strachan, John Paul and Luo, Ruibang and Li, Can},
  doi          = {10.1038/s43588-025-00867-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {940-951},
  shortjournal = {Nat. Comput. Sci},
  title        = {Real-time raw signal genomic analysis using fully integrated memristor hardware},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes. <em>NATCS</em>, <em>5</em>(10), 927-939. (<a href='https://doi.org/10.1038/s43588-025-00878-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Noncoding mutations play a critical role in regulating gene expression, yet predicting their effects across diverse tissues and cell types remains a challenge. Here we present EMO, a transformer-based model that integrates DNA sequence with chromatin accessibility data (assay for transposase-accessible chromatin with sequencing) to predict the regulatory impact of noncoding single nucleotide polymorphisms on gene expression. A key component of EMO is its ability to incorporate personalized functional genomic profiles, enabling individual-level and disease-contextual predictions and addressing critical limitations of current approaches. EMO generalizes across tissues and cell types by modeling both short- and long-range regulatory interactions and capturing dynamic gene expression changes associated with disease progression. In benchmark evaluations, the pretraining-based EMO framework outperformed existing models, with fine-tuning small-sample tissues enhancing the model’s ability to fit target tissues. In single-cell contexts, EMO accurately identified cell-type-specific regulatory patterns and successfully captured the effects of disease-associated single nucleotide polymorphisms in conditions, linking genetic variation to disease-relevant pathways. EMO integrates DNA sequence and chromatin accessibility data to predict how noncoding variants regulate gene expression across tissues and single cells, enabling context-aware personalized insights into genetic effects for precision medicine.},
  archive      = {J_NATCS},
  author       = {Liu, Zhe and Bao, Yihang and Gu, An and Song, Weichen and Lin, Guan Ning},
  doi          = {10.1038/s43588-025-00878-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {927-939},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rhythm-based hierarchical predictive computations support acoustic−semantic transformation in speech processing. <em>NATCS</em>, <em>5</em>(10), 915-926. (<a href='https://doi.org/10.1038/s43588-025-00876-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unraveling how humans understand speech despite distortions has long intrigued researchers. A prominent hypothesis highlights the role of multiple endogenous brain rhythms in forming the computational context to predict speech structure and content. Yet how neural processes may implement rhythm-based context formation remains unclear. Here we propose the brain rhythm-based inference model (BRyBI) as a possible neural implementation of speech processing in the auditory cortex based on the interaction of endogenous brain rhythms in a predictive coding framework. BRyBI encodes key rhythmic processes for parsing spectro-temporal representations of the speech signal into phoneme sequences and to govern the formation of the phrasal context. BRyBI matches patterns of human performance in speech recognition tasks and explains contradictory experimental observations of rhythms during speech listening and their dependence on the informational aspect of speech (uncertainty and surprise). This work highlights the computational role of multiscale brain rhythms in predictive speech processing. This study presents a brain rhythm-based inference model (BRyBI) for speech processing in the auditory cortex. BRyBI shows how rhythmic neural activity enables robust speech processing by dynamically predicting context and elucidates mechanistic principles that allow robust speech parsing in the brain.},
  archive      = {J_NATCS},
  author       = {Dogonasheva, Olesia and Doelling, Keith B. and Zakharov, Denis and Giraud, Anne-Lise and Gutkin, Boris},
  doi          = {10.1038/s43588-025-00876-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {915-926},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rhythm-based hierarchical predictive computations support acoustic−semantic transformation in speech processing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pharmacophore-oriented 3D molecular generation toward efficient feature-customized drug discovery. <em>NATCS</em>, <em>5</em>(10), 898-914. (<a href='https://doi.org/10.1038/s43588-025-00850-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular generation is a cutting-edge technology with the potential to revolutionize intelligent drug discovery. However, currently reported ligand-based or structure-based molecular generation methods remain unpractical for real-world drug discovery. Here we propose an explicit pharmacophore-oriented 3D molecular generation method, termed PhoreGen. PhoreGen employs asynchronous perturbations and updates on both atomic and bond information, coupled with a message-passing mechanism that incorporates prior knowledge of ligand–pharmacophore mapping during the diffusion–denoising process. Evaluations revealed that PhoreGen efficiently generates 3D molecules well aligned with pharmacophores, maintaining good chemical reasonability, diversity, drug-likeness and binding affinity and, importantly, produces feature-customized molecules at high frequency. By using PhoreGen, we successfully identified new bicyclic boronate inhibitors of evolved metallo-β-lactamase and serine-β-lactamases, which potentiate meropenem against clinically isolated superbugs. Moreover, we identified inhibitors of metallo-nicotinamidases, emerging targets for insecticides. This work explores an explicitly constrained mode for molecular generation and demonstrates its potential in feature-customized drug discovery. A pharmacophore-oriented 3D molecular generation method, which efficiently generates diverse, drug-like molecules is proposed. It successfully identifies new inhibitors in real-world drug discovery.},
  archive      = {J_NATCS},
  author       = {Peng, Jian and Yu, Jun-Lin and Yang, Zeng-Bao and Chen, Yi-Ting and Wei, Si-Qi and Meng, Fan-Bo and Wang, Yao-Geng and Huang, Xiao-Tian and Li, Guo-Bo},
  doi          = {10.1038/s43588-025-00850-5},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {898-914},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pharmacophore-oriented 3D molecular generation toward efficient feature-customized drug discovery},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MetaKSSD: Boosting the scalability of the reference taxonomic marker database and the performance of metagenomic profiling using sketch operations. <em>NATCS</em>, <em>5</em>(10), 884-897. (<a href='https://doi.org/10.1038/s43588-025-00855-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The performance of metagenomic profiling is constrained by the diversity of taxa present in the reference taxonomic marker database (MarkerDB) used. However, continually updating MarkerDB to include newly determined taxa using existing approaches faces increasing difficulties and will soon become impractical. Here we introduce MetaKSSD, which redefines MarkerDB construction and metagenomic profiling using sketch operations, enhancing MarkerDB scalability and profiling performance. MetaKSSD encompasses 85,202 species in its MarkerDB using just 0.17 GB of storage and profiles 10 GB of data within seconds. Leveraging its comprehensive MarkerDB, MetaKSSD substantially improves profiling results. In a microbiome–phenotype association study, MetaKSSD identified more effective associations than MetaPhlAn4. We profiled 382,016 metagenomic runs using MetaKSSD, conducted extensive sample clustering analyses and suggested potential yet-to-be-discovered niches. MetaKSSD offers functionality for instantaneous searching of similar profiles. It enables the swift transmission of metagenome sketches over the network and real-time online metagenomic analysis, facilitating use by non-expert users. MetaKSSD accelerates metagenomic profiling by combining k-mer sketching with marker-based methods, enabling real-time online metagenomic profiling and scalable analysis of massive microbial datasets without sacrificing accuracy.},
  archive      = {J_NATCS},
  author       = {Yi, Huiguang and Lu, Xiaoxin and Chang, Qing},
  doi          = {10.1038/s43588-025-00855-0},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {884-897},
  shortjournal = {Nat. Comput. Sci},
  title        = {MetaKSSD: Boosting the scalability of the reference taxonomic marker database and the performance of metagenomic profiling using sketch operations},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering morphogenesis of cell clusters with differentiable programming. <em>NATCS</em>, <em>5</em>(10), 875-883. (<a href='https://doi.org/10.1038/s43588-025-00851-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the fundamental rules of organismal development is a central, unsolved problem in biology. These rules dictate how individual cellular actions coordinate over macroscopic numbers of cells to grow complex structures with exquisite functionality. We use recent advances in automatic differentiation to discover local interaction rules and genetic networks that yield emergent, systems-level characteristics in a model of development. We consider a growing tissue with cellular interactions mediated by morphogen diffusion, cell adhesion and mechanical stress. Each cell has an internal genetic network that is used to make decisions based on the cell’s local environment. Here we show that one can learn the parameters governing cell interactions in the form of interpretable genetic networks for complex developmental scenarios. When combined with recent experimental advances measuring spatio-temporal dynamics and gene expression of cells in a growing tissue, the methodology outlined here offers a promising path to unraveling the cellular bases of development. This work uses differentiable simulations and reinforcement learning to design interpretable genetic networks, enabling simulated cells to self-organize into emergent developmental patterns by responding to local chemical and mechanical cues.},
  archive      = {J_NATCS},
  author       = {Deshpande, Ramya and Mottes, Francesco and Vlad, Ariana-Dalia and Brenner, Michael P. and Dal Co, Alma},
  doi          = {10.1038/s43588-025-00851-4},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {875-883},
  shortjournal = {Nat. Comput. Sci},
  title        = {Engineering morphogenesis of cell clusters with differentiable programming},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards privacy-aware mental health AI models. <em>NATCS</em>, <em>5</em>(10), 863-874. (<a href='https://doi.org/10.1038/s43588-025-00875-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mental health disorders create profound personal and societal burdens, yet conventional diagnostics are resource-intensive and limit accessibility. Recent advances in artificial intelligence, particularly natural language processing and multimodal methods, offer promise for detecting and addressing mental disorders. However, these innovations also introduce privacy concerns. Here we examine these challenges and propose solutions, including anonymization, synthetic data and privacy-preserving training, while outlining frameworks for privacy–utility trade-offs, aiming to advance reliable, privacy-aware artificial-intelligence tools that support clinical decision-making and improve mental health outcomes. In this Perspective, the authors examine privacy risks in mental health AI, and explore solutions and evaluation frameworks to balance privacy–utility trade-offs. They suggest a pipeline for developing privacy-aware mental health AI systems.},
  archive      = {J_NATCS},
  author       = {Mandal, Aishik and Chakraborty, Tanmoy and Gurevych, Iryna},
  doi          = {10.1038/s43588-025-00875-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {863-874},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards privacy-aware mental health AI models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational and ethical considerations for using large language models in psychotherapy. <em>NATCS</em>, <em>5</em>(10), 854-862. (<a href='https://doi.org/10.1038/s43588-025-00874-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) hold great potential for augmenting psychotherapy by enhancing accessibility, personalization and engagement. However, a systematic understanding of the roles that LLMs can play in psychotherapy remains underexplored. In this Perspective, we propose a taxonomy of LLM roles in psychotherapy that delineates six specific roles of LLMs across two key dimensions: artificial intelligence autonomy and emotional engagement. We discuss key computational and ethical challenges, such as emotion recognition, memory retention, privacy and emotional dependency, and offer recommendations to address these challenges. Large language models (LLMs) offer promising ways to enhance psychotherapy through greater accessibility, personalization and engagement. This Perspective introduces a typology that categorizes the roles of LLMs in psychotherapy along two critical dimensions: autonomy and emotional engagement.},
  archive      = {J_NATCS},
  author       = {Zhang, Renwen and Meng, Han and Neubronner, Marion and Lee, Yi-Chieh},
  doi          = {10.1038/s43588-025-00874-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {854-862},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational and ethical considerations for using large language models in psychotherapy},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vision language models excel at perception but struggles with scientific reasoning. <em>NATCS</em>, <em>5</em>(10), 852-853. (<a href='https://doi.org/10.1038/s43588-025-00871-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A benchmark — MaCBench — is developed for evaluating the scientific knowledge of vision language models (VLMs). Evaluation of leading VLMs reveals that they excel at basic scientific tasks such as equipment identification, but struggle with spatial reasoning and multistep analysis — a limitation for autonomous scientific discovery.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00871-0},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {852-853},
  shortjournal = {Nat. Comput. Sci},
  title        = {Vision language models excel at perception but struggles with scientific reasoning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pioneering real-time genomic analysis by in-memory computing. <em>NATCS</em>, <em>5</em>(10), 850-851. (<a href='https://doi.org/10.1038/s43588-025-00883-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid identification of pathogenic viruses remains a critical challenge. A recent study advances this frontier by demonstrating a fully integrated memristor-based hardware system that accelerates genomic analysis by a factor of 51, while reducing energy consumption to just 0.2% of that required by conventional computational methods.},
  archive      = {J_NATCS},
  author       = {Zhu, Kaichen and Lanza, Mario},
  doi          = {10.1038/s43588-025-00883-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {850-851},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pioneering real-time genomic analysis by in-memory computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How neural rhythms can guide word recognition. <em>NATCS</em>, <em>5</em>(10), 848-849. (<a href='https://doi.org/10.1038/s43588-025-00888-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recent computational model ‘BRyBI’ proposes that gamma, theta, and delta neural oscillations can guide the process of word recognition by providing temporal windows for the integration of bottom-up input with top-down information.},
  archive      = {J_NATCS},
  author       = {Slaats, Sophie},
  doi          = {10.1038/s43588-025-00888-5},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {848-849},
  shortjournal = {Nat. Comput. Sci},
  title        = {How neural rhythms can guide word recognition},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transforming psychiatry with computational and brain-based methods. <em>NATCS</em>, <em>5</em>(10), 844-847. (<a href='https://doi.org/10.1038/s43588-025-00884-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrating computational methods with brain-based data presents a path to precision psychiatry by capturing individual neurobiological variation, improving diagnosis, prognosis, and personalized care. This Viewpoint highlights advances in normative and foundation models, the importance of clinically grounded principles, and the role of robust measurement and interpretability in progressing mental health care.},
  archive      = {J_NATCS},
  author       = {Akiki, Teddy J. and Williams, Leanne M. and Wolfers, Thomas and Yang, Yanwu and Stahl, Daniel and Gillan, Claire M.},
  doi          = {10.1038/s43588-025-00884-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {844-847},
  shortjournal = {Nat. Comput. Sci},
  title        = {Transforming psychiatry with computational and brain-based methods},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trials for computational psychiatry. <em>NATCS</em>, <em>5</em>(10), 841-843. (<a href='https://doi.org/10.1038/s43588-025-00879-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational psychiatry is increasingly delivering causal evidence by focusing on interventions research and clinical trials. Causal evidence could improve patient outcomes through improved precision, repurposing, novel interventions, scaling of psychotherapy and better translation to the clinic.},
  archive      = {J_NATCS},
  author       = {Huys, Quentin J. M. and Browning, Michael},
  doi          = {10.1038/s43588-025-00879-6},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {841-843},
  shortjournal = {Nat. Comput. Sci},
  title        = {Trials for computational psychiatry},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Developing mental health AI tools that improve care across different groups and contexts. <em>NATCS</em>, <em>5</em>(10), 839-840. (<a href='https://doi.org/10.1038/s43588-025-00882-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In order to realize the potential of mental health AI applications to deliver improved care, a multipronged approach is needed, including representative AI datasets, research practices that reflect and anticipate potential sources of bias, stakeholder engagement, and equitable design practices.},
  archive      = {J_NATCS},
  author       = {Martinez-Martin, Nicole},
  doi          = {10.1038/s43588-025-00882-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {839-840},
  shortjournal = {Nat. Comput. Sci},
  title        = {Developing mental health AI tools that improve care across different groups and contexts},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rethinking mental illness through a computational lens. <em>NATCS</em>, <em>5</em>(10), 837-838. (<a href='https://doi.org/10.1038/s43588-025-00894-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Computational Science presents a Focus that explores the field of computational psychiatry and its key challenges, from privacy concerns to the ethical use of artificial intelligence, offering new insights into the future of mental health care.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00894-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {837-838},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rethinking mental illness through a computational lens},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natee">NATEE - 28</h2>
<ul>
<li><details>
<summary>
(2025). Heliconia tortuosa. <em>NATEE</em>, <em>9</em>(10), 1963. (<a href='https://doi.org/10.1038/s41559-025-02850-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What being outsmarted by a tropical plant taught Matthew Betts about science.},
  archive      = {J_NATEE},
  author       = {Betts, Matthew G.},
  doi          = {10.1038/s41559-025-02850-0},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1963},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Heliconia tortuosa},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Inferring DNA methylation in non-skeletal tissues of ancient specimens. <em>NATEE</em>, <em>9</em>(10), 1962. (<a href='https://doi.org/10.1038/s41559-025-02865-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Mathov, Yoav and Nissim-Rafinia, Malka and Leibson, Chen and Galun, Nir and Marques-Bonet, Tomas and Kandel, Arye and Liebergall, Meir and Meshorer, Eran and Carmel, Liran},
  doi          = {10.1038/s41559-025-02865-7},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1962},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Inferring DNA methylation in non-skeletal tissues of ancient specimens},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). EdgeHOG: A method for fine-grained ancestral gene order inference at large scale. <em>NATEE</em>, <em>9</em>(10), 1951-1961. (<a href='https://doi.org/10.1038/s41559-025-02818-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ancestral genomes are essential for studying the diversification of life from the last universal common ancestor to modern organisms. Methods have been proposed to infer ancestral gene order, but they lack scalability, limiting the depth to which gene neighbourhood evolution can be traced back. Here we introduce edgeHOG, a tool designed for accurate ancestral gene order inference with linear time complexity. We validated edgeHOG on various benchmarks and applied it to the entire OMA orthology database, encompassing 2,845 extant genomes across all domains of life. We reconstructed ancestral gene order for 1,133 ancestral genomes, including ancestral contigs for the last common ancestor of eukaryotes, dating back around 1.8 billion years, and observed significant functional association among neighbouring genes. EdgeHOG also dates gene adjacencies, allowing the detection of both conserved gene clusters and chromosomal rearrangements. A new method to reconstruct ancestral genomes is used to estimate contigs of the last common ancestor of eukaryotes and to infer features such as the age of gene adjacencies and chromosome rearrangements.},
  archive      = {J_NATEE},
  author       = {Bernard, Charles and Nevers, Yannis and Karampudi, Naga Bhushana Rao and Gilbert, Kimberly J. and Train, Clément and Warwick Vesztrocy, Alex and Glover, Natasha and Altenhoff, Adrian and Dessimoz, Christophe},
  doi          = {10.1038/s41559-025-02818-0},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1951-1961},
  shortjournal = {Nat. Ecol. Evol},
  title        = {EdgeHOG: A method for fine-grained ancestral gene order inference at large scale},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Widespread temporal niche partitioning in an adaptive radiation of cichlid fishes. <em>NATEE</em>, <em>9</em>(10), 1938-1950. (<a href='https://doi.org/10.1038/s41559-025-02819-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The partitioning of ecological niches is a fundamental component of species diversification in adaptive radiations. However, it is currently unknown if and how such bursts of organismal diversity are influenced by temporal niche partitioning, wherein species avoid competition by being active or sleeping during different time windows. Here we address this question through profiling temporal activity patterns in the exceptionally diverse fauna of cichlid fishes from the African Lake Tanganyika. By integrating week-long longitudinal behavioural recordings of over 500 individuals from 60 species with eco-morphological and genomic information, we provide two lines of evidence that temporal niche partitioning occurs in this massive adaptive radiation. First, Tanganyikan cichlids exhibit all known circadian temporal activity patterns (diurnal, nocturnal, crepuscular and cathemeral) and display substantial interspecific variation in daily amounts of locomotion. Second, many species with similar habitat and diet niches occupy distinct temporal niches. Moreover, our results suggest that shifts between diurnal and nocturnal activity patterns are facilitated by a crepuscular intermediate state. Genome-wide association studies indicate that the genetics underlying activity patterns is complex, with different clades associated with different combinations of variants. The identified variants were not associated with core circadian clock genes but with genes implicated in synapse function. These observations indicate that temporal niche partitioning may have contributed to adaptive radiation in cichlids and that many genes are associated with the diversity and evolution of temporal activity patterns. An analysis of behavioural, eco-morphological and genomic data in 60 species of Lake Tanganyika cichlids reveals a remarkable diversity of temporal activity patterns across species, suggesting that temporal niche partitioning may have played a role in the adaptive diversification of this group.},
  archive      = {J_NATEE},
  author       = {Nichols, Annika L. A. and Shafer, Maxwell E. R. and Indermaur, Adrian and Rüegg, Attila and Gonzalez-Dominguez, Rita and Malinsky, Milan and Sommer-Trembo, Carolin and Fritschi, Laura and Mesich, Amelia and Abdalla-Wyse, Ayasha and Salzburger, Walter and Schier, Alexander F.},
  doi          = {10.1038/s41559-025-02819-z},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1938-1950},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Widespread temporal niche partitioning in an adaptive radiation of cichlid fishes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicted deleterious mutations reveal the genetic architecture of male reproductive success in a lekking bird. <em>NATEE</em>, <em>9</em>(10), 1924-1937. (<a href='https://doi.org/10.1038/s41559-025-02802-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deleterious mutations are ubiquitous in natural populations and, when expressed, reduce fitness. However, the specific nature of these mutations and the ways in which they impact fitness remain poorly understood. We exploited recent advances in genomics to predict deleterious mutations in the black grouse (Lyrurus tetrix), an iconic lekking species. Analysis of 190 whole genomes alongside comprehensive life-history data including repeated measures of behavioural, ornamental and fitness traits revealed that deleterious mutations identified through evolutionary conservation and functional prediction are associated with reduced male lifetime mating success. Both homozygous and heterozygous deleterious mutations reduce fitness, indicating that fully and partially recessive mutations contribute towards an individual’s realized mutation load. Notably, deleterious mutations in promotors have disproportionally negative fitness effects, suggesting that they impair an individual’s ability to dynamically adjust gene expression to meet context-dependent functional demands. Finally, deleterious mutations impact male mating success by reducing lek attendance rather than by altering the expression of ornamental traits, suggesting that behaviour serves as an honest indicator of genetic quality. These findings offer insights into the genetic architecture of male fitness and illuminate the complex interplay between genetic variation and phenotypic expression. Analysis of whole genomes and life-history data of male black grouse shows that deleterious mutations, especially those in promotors, reduce lifetime reproductive success.},
  archive      = {J_NATEE},
  author       = {Chen, R. S. and Soulsbury, C. D. and Hench, K. and van Oers, K. and Hoffman, J. I.},
  doi          = {10.1038/s41559-025-02802-8},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1924-1937},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Predicted deleterious mutations reveal the genetic architecture of male reproductive success in a lekking bird},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep-sea-floor diversity in asteroidea is shaped by competing processes across different latitudes and oceans. <em>NATEE</em>, <em>9</em>(10), 1910-1923. (<a href='https://doi.org/10.1038/s41559-025-02808-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The occurrence, shape and drivers of global distributional trends in species richness throughout the deep sea are poorly explored. Here we present a spatial description of the global, bathymetric and taxonomic extent of the benthic marine class Asteroidea using a compiled dataset of ~200,000 species-level occurrence records. We used these data to produce comparisons of sea-floor richness between hemispheres and oceans. We show that species richness is significantly correlated with temperature and nutrient flux despite markedly different distributional patterns across oceans and latitudes that suggest further influence from a combination of additional geographic, taxonomic and environmental factors. The relative importance of temperature and nutrient levels also varies greatly with depth. Species richness peaks in the shallow-water tropics, closely matching sea-floor temperature variation, but at bathyal and abyssal depths it is higher at temperate latitudes, where nutrient flux levels are of greater importance. We show that richness in the deep benthos is restricted below ~1.5 °C, with this strong thermal threshold consistent among oceans irrespective of other factors. Global trends in species richness in the deep sea are poorly explored. This study uses occurrence records of Asteroidea to show that species richness in the deep benthos is restricted below ~1.5 °C but correlated with temperature and nutrient flux, despite different patterns across oceans and latitudes.},
  archive      = {J_NATEE},
  author       = {Carter, H. F. and Bribiesca-Contreras, G. and Williams, S. T.},
  doi          = {10.1038/s41559-025-02808-2},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1910-1923},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deep-sea-floor diversity in asteroidea is shaped by competing processes across different latitudes and oceans},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large reductions in tropical bird abundance attributable to heat extreme intensification. <em>NATEE</em>, <em>9</em>(10), 1897-1909. (<a href='https://doi.org/10.1038/s41559-025-02811-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although species exhibit widespread sensitivity to environmental conditions, the extent to which human-driven climate change may have already altered their abundance remains unclear. Here we quantify the impact of climate change on bird populations from across the world by combining models of their response to environmental conditions with a climate attribution framework. We identify a dominant role of intensified heat extremes compared to changes in average temperature and precipitation. Increased interannual exposure to hot extremes reduces annual abundance growth rates most strongly in lower-latitude tropical regions, with effects robust when controlling for changing human industrial pressure and other long-term drivers. Compared to a counterfactual without human-driven climate change, the historical intensification of heat extremes has caused a 25–38% reduction in the level of abundance of tropical birds, which has accumulated from 1950 to 2020. Across observed tropical bird populations, impacts of climate change have typically been larger than direct human pressure, the opposite across sub-tropical regions. Overall, these results showcase how human-driven climate change is already reshaping biodiversity globally and may explain reported declines of birds in undisturbed tropical habitats. Climate change poses a growing threat to biodiversity, but disentangling its overall impact from other anthropogenic stressors is challenging. Here the authors use a data-driven climate attribution framework to identify a key role of heat extremes in causing changes in bird populations globally.},
  archive      = {J_NATEE},
  author       = {Kotz, Maximilian and Amano, Tatsuya and Watson, James E. M.},
  doi          = {10.1038/s41559-025-02811-7},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1897-1909},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Large reductions in tropical bird abundance attributable to heat extreme intensification},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Emerging threats to antarctic conservation. <em>NATEE</em>, <em>9</em>(10), 1885-1896. (<a href='https://doi.org/10.1038/s41559-025-02814-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antarctica, long considered an environmental sanctuary, now confronts accelerating, complex and inter-related conservation challenges. The vast size and remote location of the continent introduce substantial uncertainty in understanding and predicting these threats. Here, using strategic foresight techniques, we synthesized insights from a global horizon scan with 131 experts from 42 countries. We identified ten emerging conservation challenges across six thematic categories. Key issues included extreme precipitation, emerging animal pathogens, human pandemics, security threats, reduced cooperation among Antarctic Treaty parties and potential agricultural expansion. Several of these challenges stem from persistent underlying drivers, revealing how longstanding processes are giving rise to new and increasingly acute conservation concerns. Others, driven by global disruptions, have no historical precedent in the region but increasingly constrain decision-making and international coordination. This horizon scan reveals substantial limitations in the ability of the Antarctic Treaty system to address these challenges, underscoring the need to reassess existing governance mechanisms to protect the unique ecosystems of Antarctica and its vital role in the global climate system. A horizon scan of international respondents identifies and discusses ten developing challenges in Antarctic conservation, revealing an increased emphasis on challenges related to governance, geopolitics and economics compared to a similar scan from 2012.},
  archive      = {J_NATEE},
  author       = {Carter, Zachary T. and Bode, Michael and Chown, Steven L. and Burrows, Joanna L. and Shaw, Justine D. and Walsh, Jessica C. and Burgman, Mark A. and Cassey, Phillip and Wilson, Kerrie A.},
  doi          = {10.1038/s41559-025-02814-4},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1885-1896},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Emerging threats to antarctic conservation},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Archived natural DNA samplers reveal four decades of biodiversity change across the tree of life. <em>NATEE</em>, <em>9</em>(10), 1873-1884. (<a href='https://doi.org/10.1038/s41559-025-02812-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detecting the imprints of global environmental change on biological communities is a paramount task for ecological research. But a lack of standardized long-term biomonitoring data prevents a deeper understanding of biodiversity change in the Anthropocene. Novel sources of data for analysing biodiversity change across time and space are urgently needed. By metabarcoding highly standardized biota samples from a long-term pollution monitoring archive in Germany, we here analyse four decades of community diversity for tens of thousands of species across the tree of life. The archived samples—tree leaves, marine macroalgae, and marine and limnic mussels—represent natural community DNA samplers, preserving a taxonomically diverse imprint of their associated biodiversity at the time of collection. We find no evidence for universal diversity declines at the local scale. Instead, a gradual compositional turnover emerges as a universal pattern of temporal biodiversity change in Germany’s terrestrial and aquatic ecosystems. This turnover results in biotic homogenization in most terrestrial and marine communities. Limnic communities, in contrast, rather differentiate across space, probably due to the immigration of different invasive species into different sites. Our study highlights the immense promise of alternative sample sources to provide standardized time series data of biodiversity change in the Anthropocene. This study explores four decades of biodiversity change across the tree of life in Germany’s terrestrial and aquatic ecosystems by sequencing standardized biota samples from a long-term pollution monitoring archive.},
  archive      = {J_NATEE},
  author       = {Junk, Isabelle and Hans, Julian and Perez-Lamarque, Benoît and Stothut, Manuel and Weber, Sven and Gold, Elisabeth and Schubert, Caroline and Schumacher, Alice and Schmitt, Nina and Melcher, Anja and Paulus, Martin and Klein, Roland and Teubner, Diana and Koschorreck, Jan and Kennedy, Susan and Morlon, Hélène and Krehenwinkel, Henrik},
  doi          = {10.1038/s41559-025-02812-6},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1873-1884},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Archived natural DNA samplers reveal four decades of biodiversity change across the tree of life},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Two decades of improved wetland carbon sequestration in northern mid-to-high latitudes are offset by tropical and southern declines. <em>NATEE</em>, <em>9</em>(10), 1861-1872. (<a href='https://doi.org/10.1038/s41559-025-02809-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Terrestrial carbon (C) sink has long been recognized as trending upwards, yet its recent slowdown raises concerns about accelerating climate change. Variations in wetland C sequestration are hypothesized to play a key role in this shift. Here we mapped annual water levels in global wetlands from 2000 to 2020 using 2,295 field-based measurements and predicted the spatiotemporal pattern of wetland net ecosystem production (NEP) in conjunction with other environmental factors. By compiling 934 in situ observations, we estimated a global mean wetland NEP of 56.4 (44.0‒68.8) gC m−2 yr−1. Integrating the NEP dataset with environmental datasets and machine-learning models, we estimated the mean annual global wetland C sequestration between 2000 and 2020 to be 1,004 (961‒1,047) TgC, 70% of which originated from tropical wetlands. We observed a decline in global wetland C sinks until 2005, followed by an increase thereafter. Overall, wetland C sequestration was roughly stable during 2000‒2020, as gains in northern mid-to-high latitudes were fully overwhelmed by declines in the tropics and southern mid-to-high latitudes. Our findings highlight hydrological change as a dominant driver of increasing regional variability in wetland C sinks, while intensifying hydrological extremes under climate change may undermine the resilience of wetland C sinks and the ecosystem services they support. Here the authors identify hotspots for wetland carbon uptake and regions where wetland carbon sinks are most susceptible to hydrological shifts. They show that two decades of improved carbon sinks in northern mid- to high-latitude wetlands are offset by declining efficiencies elsewhere.},
  archive      = {J_NATEE},
  author       = {Li, Junjie and Yuan, Junji and Ciais, Philippe and Kang, Hojeong and Freeman, Chris and Huang, Yuanyuan and Dong, Yanhong and Liu, Deyan and Li, Ye and Ding, Weixin},
  doi          = {10.1038/s41559-025-02809-1},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1861-1872},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Two decades of improved wetland carbon sequestration in northern mid-to-high latitudes are offset by tropical and southern declines},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Global covariation of forest age transitions with the net carbon balance. <em>NATEE</em>, <em>9</em>(10), 1848-1860. (<a href='https://doi.org/10.1038/s41559-025-02821-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Forest age transitions are critical in shaping the global carbon balance, yet their influence on carbon stocks and fluxes remains poorly quantified. Here we analyse global forest age dynamics from 2010 to 2020 using the Global Age Mapping Integration v2.0 dataset, alongside satellite-derived aboveground carbon (AGC) and atmospheric inversion-derived net CO2 flux data. We reveal widespread declines in forest age across the Amazon, Congo Basin, Southeast Asia and parts of Siberia, primarily driven by stand-replacing disturbances such as fire and harvest, leading to the replacement of older forests by younger stands. Meanwhile, forests in China, Europe and North America experienced net ageing. Globally, stand replacement resulted in substantial AGC losses, with old forests (&gt;200 years, ~98.0 MgC ha−1) transitioning to younger, carbon-poor stands (&lt;20 years, ~43.5 MgC ha−1), leading to a net AGC loss of ~0.14 PgC per year. Despite this, regions with high rates of young stands replacing old forests exhibited a temporary strengthening of the carbon sink, driven by the rapid regrowth of these young stands. Crucially, these young forests do not compensate for the long-term carbon storage of old forests. Our findings underscore the importance of protecting old forests while optimizing forest management strategies to maximize carbon gains and enhance climate mitigation. A global analysis reveals regional trends of net forest ageing but also that widespread stand-replacing disturbances, such as fire and harvest, are driving declining forest age in many areas, often accompanied by substantial losses in aboveground carbon stocks and shifts in carbon sink dynamics.},
  archive      = {J_NATEE},
  author       = {Besnard, Simon and Heinrich, Viola H. A. and Carvalhais, Nuno and Ciais, Philippe and Herold, Martin and Luijkx, Ingrid and Peters, Wouter and Requena Suarez, Daniela and Santoro, Maurizio and Yang, Hui},
  doi          = {10.1038/s41559-025-02821-5},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1848-1860},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Global covariation of forest age transitions with the net carbon balance},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stabilizing effects of biodiversity arise from species-specific dynamics rather than interspecific interactions in grasslands. <em>NATEE</em>, <em>9</em>(10), 1837-1847. (<a href='https://doi.org/10.1038/s41559-025-02787-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although numerous studies have demonstrated a positive relationship between biodiversity and ecosystem stability, that is low temporal variability in ecosystem functions, the role of interspecific interactions in driving this relationship remains elusive. Here we develop a partitioning framework to disentangle the effects of interspecific interactions on ecosystem variability from those of interaction-independent processes. Applying this framework to competition models and biodiversity experiments in European and North American grasslands, we find that species-specific dynamics observable in monocultures (for example, environmental responses or demographic stochasticity), rather than interspecific interactions observable only in mixtures, explain much of the stabilizing effects of biodiversity on ecosystems. The weak net effect of interspecific interactions results from two counter-balancing effects: an interaction-driven increase in average species variability and an interaction-driven decrease in average species synchrony. We also find that interspecific interactions contributing to higher ecosystem functioning tend to reduce ecosystem variability. Our study provides a systematic empirical assessment of the role of interspecific interactions in shaping grassland ecosystem stability, challenging traditional assumptions about their importance and highlighting species-specific dynamics as the primary mechanism underlying the pervasive biodiversity–stability relationship. The role of interspecific interactions in biodiversity–ecosystem stability relationships is unclear. Here the authors develop a theoretical approach and show that empirical diversity–stability relationships in grasslands are best explained by species-specific dynamics rather than by interspecific interactions.},
  archive      = {J_NATEE},
  author       = {Meng, Bo and Luo, Mingyu and Loreau, Michel and Hong, Pubin and Craven, Dylan and Eisenhauer, Nico and Isbell, Forest and Liang, Maowei and Reuman, Daniel and Wilsey, Brian and van Ruijven, Jasper and Zhao, Lei and Wang, Shaopeng},
  doi          = {10.1038/s41559-025-02787-4},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1837-1847},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Stabilizing effects of biodiversity arise from species-specific dynamics rather than interspecific interactions in grasslands},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Limited plastic responses in safety traits support greater hydraulic risk under drier conditions. <em>NATEE</em>, <em>9</em>(10), 1825-1836. (<a href='https://doi.org/10.1038/s41559-025-02830-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how plants adjust their hydraulic system to the environment is essential to predict how these organisms will respond to global change. Here we compiled a dataset and performed meta-analysis on 223 studies on plastic and evolutionary adjustments of hydraulic traits to air temperature, CO2 concentration, irradiance, soil nutrient and water availability. On average, species plastically increased embolism resistance and sapwood area per leaf area under drier conditions, with a decrease in stem-specific hydraulic conductivity and water potential at the turgor loss point, which are consistent with adaptive responses. However, the average increased embolism resistance was not sufficient to compensate the reduction in the minimum water potential, implying a lower safety margin from lethal hydraulic failure under drought. These results point towards a general critical increase in the risk of hydraulic failure in future drier environments. Plastic responses to increased soil nutrient content and irradiance did not always align with those to drought, highlighting the potential for changes in light and nutrient conditions to modify plant hydraulic responses to climate-change-driven droughts. Plants across many ecosystems are increasingly exposed to dryness stress. Using meta-analysis, the authors show that plants can adjust their hydraulic traits in response to drought and other global change factors, but not equally across traits and not enough to prevent lethal hydraulic failure.},
  archive      = {J_NATEE},
  author       = {Ramírez-Valiente, José A. and Poyatos, Rafael and Blackman, Chris J. and Cabon, Antoine and Castells, Eva and Cochard, Hervé and Creek, Danielle and Delzon, Sylvain and García-Valdés, Raúl and Limousin, Jean-Marc and López, Rosana and Martin-StPaul, Nicolas and Moreno, Myriam and Rowland, Lucy and Santiago, Louis S. and Schuldt, Bernhard and Torres-Ruiz, José M. and Valade, Aude and Martínez-Vilalta, Jordi and Mencuccini, Maurizio},
  doi          = {10.1038/s41559-025-02830-4},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1825-1836},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Limited plastic responses in safety traits support greater hydraulic risk under drier conditions},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Neighbourhood diversity increases tree growth in experimental forests more in wetter climates but not in wetter years. <em>NATEE</em>, <em>9</em>(10), 1812-1824. (<a href='https://doi.org/10.1038/s41559-025-02805-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tree diversity often increases stand-level growth, but whether neighbourhood diversity effects on individual tree growth change with climatic conditions remains unclear. Here, using 852,170 records of 113,701 individuals from 129 species in 15 tree diversity experiments across four biomes, we address this knowledge gap with a synthesis of tree growth data spanning a broad climate gradient. We examine how neighbourhood-scale (defined as a focal tree and the adjacent trees) taxonomic and functional diversity effects on tree growth vary with climate, both spatially (across sites) and temporally (within sites). Increasing species richness and trait dissimilarity from monospecific to high-diversity neighbourhoods enhanced individual tree growth by 7–13% on average. The positive diversity effect increased from dry to wet climates, contrasting with most prior studies, but was unaffected by interannual climatic variation within sites. Given that tree–tree interactions are ubiquitous and likely to interact with climate in both young and old forests, our findings suggest incorporating neighbourhood diversity as a management tool to enhance forest productivity, while considering underlying mechanisms and interactions with climate, thereby facilitating targeted and site-specific climate and biodiversity benefits. How tree diversity effects on ecosystem functioning vary along climatic gradients is unclear. Here, analysing data from 15 experimental forest sites, the authors show that tree growth responses to neighbourhood species diversity are stronger in wetter climates but are unaffected by interannual climatic variation within sites.},
  archive      = {J_NATEE},
  author       = {Zheng, Liting and Ibáñez, Inés and Williams, Laura J. and Zhu, Kai and Serrano-León, Hernán and Jensen, Joel and Eisenhauer, Nico and Verheyen, Kris and Scherer-Lorenzen, Michael and Schnabel, Florian and Kreft, Holger and Guerrero-Ramírez, Nathaly R. and Hölscher, Dirk and Paterno, Gustavo B. and Irawan, Bambang and Ponette, Quentin and Messier, Christian and Paquette, Alain and Stefanski, Artur and Mereu, Simone and Bauhus, Jürgen and Hajek, Peter and Nock, Charles A. and Cavender-Bares, Jeannine and Parker, William C. and Quosh, Julius and Ferlian, Olga and Auge, Harald and Potvin, Catherine and Yan, Enrong and Yang, Baiyu and Zhang, Li and Zhao, Zhao and Sinacore, Katherine and Hall, Jefferson S. and Guillemot, Joannès and Robin, Agnès and Brancalion, Pedro H. S. and Sundawati, Leti and Reich, Peter B.},
  doi          = {10.1038/s41559-025-02805-5},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1812-1824},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Neighbourhood diversity increases tree growth in experimental forests more in wetter climates but not in wetter years},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lagged precipitation effects on plant production across terrestrial biomes. <em>NATEE</em>, <em>9</em>(10), 1800-1811. (<a href='https://doi.org/10.1038/s41559-025-02806-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precipitation effects on plant carbon uptake extend beyond immediate timeframes, reflecting temporal lags between rainfall and plant growth. Mechanisms and relative importance of such lagged effects are expected to vary across ecosystems. Here we draw on an extensive collections of productivity proxies from long-term ground measurements, satellite observations and model simulations to show that preceding-year precipitation exerts a comparable influence on plant productivity to current-year precipitation. Statistically supported lagged precipitation effects are detected in 13.4%–19.7% of the grids depending on the data source. In these sites, preceding-year precipitation positively controls current-year plant productivity in water-limited areas, while negative effects occur in some wet regions, such as tropical forests. While aridity emerges as the main driver of this spatial variability, machine learning-based spatial attribution also indicates interactions among plant traits, climatic conditions and soil properties. We also show that soil water dynamics, plant phenology and foliar structure might mediate lagged precipitation effects across time. Our findings highlight the role of preceding-year precipitation in global plant productivity. Precipitation is a key driver of plant growth. Here the authors integrate ground-based observations, remote sensing and process-based models to disentangle the relative contribution of preceding-year and current-year precipitation on plant productivity and identify its predictors across biomes.},
  archive      = {J_NATEE},
  author       = {He, Lei and Wang, Jian and Peltier, Drew M. P. and Ritter, François and Ciais, Philippe and Peñuelas, Josep and Xiao, Jingfeng and Crowther, Thomas W. and Li, Xing and Ye, Jian-Sheng and Sasaki, Takehiro and Zhou, Chenghu and Li, Zhao-Liang},
  doi          = {10.1038/s41559-025-02806-4},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1800-1811},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Lagged precipitation effects on plant production across terrestrial biomes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ecological and hydroclimatic determinants of vegetation water-use strategies. <em>NATEE</em>, <em>9</em>(10), 1791-1799. (<a href='https://doi.org/10.1038/s41559-025-02810-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vegetation responses to soil moisture limitation play a key role in land–atmosphere interactions and are a major source of uncertainty in future projections of the global water and carbon cycles. Vegetation water-use strategies—that is, how plants regulate transpiration rates as the soil dries—are highly dynamic across space and time, presenting a major challenge to inferring ecosystem responses to water limitation. Here we show that, when aggregated globally, water-use strategies derived from point-based soil moisture observations exhibit emergent patterns across and within climates and vegetation types along a spectrum of aggressive to conservative responses to water limitation. Water use becomes more conservative, declining more rapidly as the soil dries, as mean annual precipitation increases and as woody cover increases from grasslands to savannahs to forests. We embed this empirical synthesis within an ecohydrological framework to show that key ecological (leaf area) and hydroclimatic (aridity) factors driving demand for water explain up to 77% of the variance in water-use strategies within ecosystem types. All biomes respond to ecological and hydroclimatic demand by shifting towards more aggressive water-use strategies. However, woodlands reach a threshold beyond which water use becomes increasingly conservative, probably reflecting the greater hydraulic risk and cost of tissue damage associated with sustaining high transpiration rates under water limitation for trees than grasses. These findings highlight the importance of characterizing the dynamic nature of vegetation water-use strategies to improve predictions of ecosystem responses to climate change. Vegetation responses to water limitation are difficult to predict due to large variation across space and time. A new analysis of global soil moisture dynamics reveals that water-use strategies vary systematically by vegetation type in response to ecological and climatic conditions.},
  archive      = {J_NATEE},
  author       = {Morgan, Bryn E. and Araki, Ryoko and Trugman, Anna T. and Caylor, Kelly K.},
  doi          = {10.1038/s41559-025-02810-8},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1791-1799},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecological and hydroclimatic determinants of vegetation water-use strategies},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hominin glacial-stage occupation 712,000 to 424,000 years ago at fordwich pit, old park (Canterbury, UK). <em>NATEE</em>, <em>9</em>(10), 1781-1790. (<a href='https://doi.org/10.1038/s41559-025-02829-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Few high-latitude archaeological contexts are older than marine isotope stage (MIS) 15 and even fewer provide evidence of early human occupation during a glacial period. New discoveries at Old Park, Canterbury (UK), provide evidence of both the oldest accessible artefact-bearing sediment in northern Europe and cold-stage adaptation. Radiometric and palaeomagnetic dating places the earliest suggested occupation of this site between 773 thousand years ago (ka) and 607 ka, with hominin presence inferred during MIS 17–16. Two additional artefact-bearing stratigraphic units, dated to around 542 ka and 437 ka, strongly align with the MIS 14 and 12 cold stages, respectively. The latter unit contains convincing evidence of glacial-stage occupation by Acheulean hominins; fresh, unabraded flakes (including biface-thinning) between clearly defined glacial-aged sediments displaying mixed grassland palaeoenvironmental evidence. An historically collected assemblage of more than 330 handaxes is argued to be derived from both the MIS 17–16 and MIS 12 sediments, providing evidence of the earliest known Acheulean bifaces in northern Europe, and re-occupation by Acheulean populations 200,000 years later. Together, Old Park provides evidence for Lower Palaeolithic hominins reoccupying a location over several mid-Pleistocene MIS cycles, early human presence above 51° latitude during a glacial stage and handaxe production in northern Europe from MIS 17 to 16. The authors report Acheulean hominin occupation of eastern Britain during glacial marine isotope stages 17–16 and again in glacial marine isotope stage 12 via stone tools in sediments dated to 712,000 to 424,000 years ago.},
  archive      = {J_NATEE},
  author       = {Key, Alastair and Clark, James and Lauer, Tobias and Bates, Jennifer and Sier, Mark-Jan and Nichols, Claire and Martín-Ramos, Carmen and Cebeiro, Adela and Williams, Eleanor and Kim, Sunghui and Stileman, Finn and Mika, Anna and Pope, Matthew and Bridgland, David and Redhouse, David and Leonardi, Michela and Smith, Geoff M. and Proffitt, Tomos},
  doi          = {10.1038/s41559-025-02829-x},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1781-1790},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Hominin glacial-stage occupation 712,000 to 424,000 years ago at fordwich pit, old park (Canterbury, UK)},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Applying evolutionary theory to understand host–microbiome evolution. <em>NATEE</em>, <em>9</em>(10), 1769-1780. (<a href='https://doi.org/10.1038/s41559-025-02846-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {All plants and animals are host to a community of microorganisms, their microbiotas, which have crucial influences on the life history and performance of their hosts. Despite the importance of such host–microbiota relationships, relatively little is known about the role microbiotas have in mediating evolution of the host and entire host–microbe assemblages. This knowledge gap is partly due to the lack of theoretical frameworks that generate testable predictions on the evolutionary dynamics of host–microbiota systems. In this Perspective, we argue that the foundation for such frameworks exists in evolutionary theory. We highlight four examples of theoretical models—niche construction, indirect genetic effects, maternal effects and multilevel selection—that capture important aspects of host–microbiome evolution. We outline how each of these frameworks can provide key insights into the evolution of host–microbiota systems while also suggesting expansions of current theory to incorporate processes unique to host–microbe assemblages; for instance, focusing on nuances in microbiota transmission and ecological microbial community dynamics. Expanding evolutionary theory to accommodate host–microbiota associations is key for a more integrative understanding of evolution, and the approach outlined here can guide future empirical research on the function and evolution of these ubiquitous interactions. This Perspective discusses how well-established theoretical models of evolution can be adapted to study and generate testable predictions about the evolutionary dynamics of host–microbiota associations.},
  archive      = {J_NATEE},
  author       = {Week, Bob and Russell, Shelbi L. and Schulenburg, Hinrich and Bohannan, Brendan J. M. and Bruijning, Marjolein},
  doi          = {10.1038/s41559-025-02846-w},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1769-1780},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Applying evolutionary theory to understand host–microbiome evolution},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lingering effects of precipitation on plant productivity. <em>NATEE</em>, <em>9</em>(10), 1767-1768. (<a href='https://doi.org/10.1038/s41559-025-02824-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Climatic memory complicates interpretation of the relationship between plant productivity and precipitation. We find that the influence of preceding-year precipitation on plant productivity is comparable to that of current-year precipitation. The lagged precipitation effect varies with aridity and exerts a positive influence in dry regions and a negative influence in wet regions.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02824-2},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1767-1768},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Lingering effects of precipitation on plant productivity},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Plant responses to water stress revealed from soil moisture dynamics. <em>NATEE</em>, <em>9</em>(10), 1765-1766. (<a href='https://doi.org/10.1038/s41559-025-02826-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vegetation responses to water limitation are difficult to predict owing to large variation across space and time. Our global analysis of soil moisture dynamics reveals that plant water-use strategies vary systematically by ecosystem type in response to recent ecological and climatic conditions.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02826-0},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1765-1766},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Plant responses to water stress revealed from soil moisture dynamics},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lake tanganyika cichlids partition habitat by time. <em>NATEE</em>, <em>9</em>(10), 1763-1764. (<a href='https://doi.org/10.1038/s41559-025-02847-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New experimental evidence suggests that cichlids in Lake Tanganyika exhibit diverse activity patterns, and these differences are associated with variation in unexpected genetic loci.},
  archive      = {J_NATEE},
  author       = {Conith, Andrew J.},
  doi          = {10.1038/s41559-025-02847-9},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1763-1764},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Lake tanganyika cichlids partition habitat by time},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Partitioning how biodiversity brings stability. <em>NATEE</em>, <em>9</em>(10), 1761-1762. (<a href='https://doi.org/10.1038/s41559-025-02803-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A theoretical approach quantifies the drivers of community variability in simulated and natural plant communities, which sheds light on the mechanisms that underlie biodiversity–ecosystem stability relationships.},
  archive      = {J_NATEE},
  author       = {Pennekamp, Frank},
  doi          = {10.1038/s41559-025-02803-7},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1761-1762},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Partitioning how biodiversity brings stability},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Uphill urbanization. <em>NATEE</em>, <em>9</em>(10), 1760. (<a href='https://doi.org/10.1038/s41559-025-02878-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Dickson, Iain},
  doi          = {10.1038/s41559-025-02878-2},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1760},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Uphill urbanization},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hot butterflies. <em>NATEE</em>, <em>9</em>(10), 1759. (<a href='https://doi.org/10.1038/s41559-025-02877-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02877-3},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1759},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Hot butterflies},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Speciating swallows. <em>NATEE</em>, <em>9</em>(10), 1758. (<a href='https://doi.org/10.1038/s41559-025-02875-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-025-02875-5},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1758},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Speciating swallows},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ebony and ivory. <em>NATEE</em>, <em>9</em>(10), 1757. (<a href='https://doi.org/10.1038/s41559-025-02874-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-025-02874-6},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1757},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ebony and ivory},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying coral reef–ocean interactions is critical for predicting reef futures under climate change. <em>NATEE</em>, <em>9</em>(10), 1754-1756. (<a href='https://doi.org/10.1038/s41559-025-02839-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coral reefs are inextricably linked to their surrounding seascape, ecologically shaped by ocean circulation patterns and dependent on upwelled nutrients and planktonic subsidies. To better predict coral reef futures, we must more effectively quantify and incorporate these fundamental biophysical interactions.},
  archive      = {J_NATEE},
  author       = {Richardson, Laura E. and Williams, Gareth J. and Dunne, Aislinn and Jackson-Bué, Tim and Green, J. A. Mattias and Morrison, Tiffany H. and Fox, Michael D.},
  doi          = {10.1038/s41559-025-02839-9},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1754-1756},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Quantifying coral reef–ocean interactions is critical for predicting reef futures under climate change},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Water, water, everywhere. <em>NATEE</em>, <em>9</em>(10), 1753. (<a href='https://doi.org/10.1038/s41559-025-02876-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how terrestrial plants and ecosystems cope with shifts of water availability in space and time requires multiple approaches and collaborative efforts.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02876-4},
  journal      = {Nature Ecology & Evolution},
  month        = {10},
  number       = {10},
  pages        = {1753},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Water, water, everywhere},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="natmi">NATMI - 21</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires. <em>NATMI</em>, <em>7</em>(9), 1587. (<a href='https://doi.org/10.1038/s42256-025-01128-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Jiang, Yuepeng and Zhang, Pingping and Huo, Miaozhe and Li, Shuai Cheng},
  doi          = {10.1038/s42256-025-01128-1},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1587},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Deep learning-based prediction of the selection factors for quantifying selection in immune receptor repertoires},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A framework to evaluate machine learning crystal stability predictions. <em>NATMI</em>, <em>7</em>(9), 1586. (<a href='https://doi.org/10.1038/s42256-025-01117-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Riebesell, Janosh and Goodall, Rhys E. A. and Benner, Philipp and Chiang, Yuan and Deng, Bowen and Ceder, Gerbrand and Asta, Mark and Lee, Alpha A. and Jain, Anubhav and Persson, Kristin A.},
  doi          = {10.1038/s42256-025-01117-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1586},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: A framework to evaluate machine learning crystal stability predictions},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanistic understanding and validation of large AI models with SemanticLens. <em>NATMI</em>, <em>7</em>(9), 1572-1585. (<a href='https://doi.org/10.1038/s42256-025-01084-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike human-engineered systems, such as aeroplanes, for which the role and dependencies of each component are well understood, the inner workings of artificial intelligence models remain largely opaque, which hinders verifiability and undermines trust. Current approaches to neural network interpretability, including input attribution methods, probe-based analysis and activation visualization techniques, typically provide limited insights about the role of individual components or require extensive manual interpretation that cannot scale with model complexity. This paper introduces SemanticLens, a universal explanation method for neural networks that maps hidden knowledge encoded by components (for example, individual neurons) into the semantically structured, multimodal space of a foundation model such as CLIP. In this space, unique operations become possible, including (1) textual searches to identify neurons encoding specific concepts, (2) systematic analysis and comparison of model representations, (3) automated labelling of neurons and explanation of their functional roles, and (4) audits to validate decision-making against requirements. Fully scalable and operating without human input, SemanticLens is shown to be effective for debugging and validation, summarizing model knowledge, aligning reasoning with expectations (for example, adherence to the ABCDE rule in melanoma classification) and detecting components tied to spurious correlations and their associated training data. By enabling component-level understanding and validation, the proposed approach helps mitigate the opacity that limits confidence in artificial intelligence systems compared to traditional engineered systems, enabling more reliable deployment in critical applications. SemanticLens is a tool that embeds artificial intelligence model components (such as neurons) into a searchable, human-understandable space. This enables automated auditing, validation of decisions and detection of problematic behaviours with minimal human oversight.},
  archive      = {J_NATMI},
  author       = {Dreyer, Maximilian and Berend, Jim and Labarta, Tobias and Vielhaben, Johanna and Wiegand, Thomas and Lapuschkin, Sebastian and Samek, Wojciech},
  doi          = {10.1038/s42256-025-01084-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1572-1585},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Mechanistic understanding and validation of large AI models with SemanticLens},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning. <em>NATMI</em>, <em>7</em>(9), 1561-1571. (<a href='https://doi.org/10.1038/s42256-025-01098-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence has transformed the field of precise organic synthesis. Data-driven methods, including machine learning and deep learning, have shown great promise in predicting reaction performance and synthesis planning. However, the inherent methodological divergence between numerical regression-driven reaction performance prediction and sequence generation-based synthesis planning creates formidable challenges in constructing a unified deep learning architecture. Here we present RXNGraphormer, a framework to jointly address these tasks through a unified pre-training approach. By synergizing graph neural networks for intramolecular pattern recognition with Transformer-based models for intermolecular interaction modelling, and training on 13 million reactions via a carefully designed strategy, RXNGraphormer achieves state-of-the-art performance across eight benchmark datasets for reactivity or selectivity prediction and forward-synthesis or retrosynthesis planning, as well as three external realistic datasets for reactivity and selectivity prediction. Notably, the model generates chemically meaningful embeddings that spontaneously cluster reactions by type without explicit supervision. This work bridges the critical gap between performance prediction and synthesis planning tasks in chemical AI, offering a versatile tool for accurate reaction prediction and synthesis design. Xu et al. present RXNGraphormer, a pre-trained model that learns bond transformation patterns from over 13 million reactions, achieving state-of-the-art accuracy in reaction performance prediction and synthesis planning.},
  archive      = {J_NATMI},
  author       = {Xu, Li-Cheng and Tang, Miao-Jiong and An, Junyi and Cao, Fenglei and Qi, Yuan},
  doi          = {10.1038/s42256-025-01098-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1561-1571},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A unified pre-trained deep learning framework for cross-task reaction performance prediction and synthesis planning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces. <em>NATMI</em>, <em>7</em>(9), 1555-1560. (<a href='https://doi.org/10.1038/s42256-025-01091-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular docking plays a crucial role in structure-based drug discovery, enabling the prediction of how small molecules interact with protein targets. Traditional docking methods rely on scoring functions and search heuristics, whereas recent generative approaches, such as DiffDock, leverage deep learning for pose prediction. However, blind-diffusion-based docking often struggles with binding site localization and pose accuracy, particularly in complex protein–ligand systems. This work introduces GeoDirDock (GDD), a guided diffusion approach to molecular docking that enhances the accuracy and physical plausibility of ligand docking predictions. GDD guides the denoising process of a diffusion model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. Our method leverages expert knowledge to direct the generative modelling process, specifically targeting desired protein–ligand interaction regions. We demonstrate that GDD outperforms existing blind docking methods in terms of root mean squared distance accuracy and physicochemical pose realism. Our results indicate that incorporating domain expertise into the diffusion process leads to more biologically relevant docking predictions. Additionally, we explore the potential of GDD as a template-based modelling tool for lead optimization in drug discovery through angle transfer in maximum common substructure docking, showcasing its capability to accurately predict ligand orientations for chemically similar compounds. Future applications in real-world drug discovery campaigns will naturally continue to refine and extend the utility of prior-informed diffusion docking methods. GeoDirDock is a framework that guides the denoising process of a generative diffusion docking model along geodesic paths within multiple spaces representing translational, rotational and torsional degrees of freedom. This approach enhances the accuracy and physical plausibility of ligand docking predictions.},
  archive      = {J_NATMI},
  author       = {Miñán, Raúl and Gallardo, Javier and Ciudad, Álvaro and Molina, Alexis},
  doi          = {10.1038/s42256-025-01091-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1555-1560},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Informed protein–ligand docking via geodesic guidance in translational, rotational and torsional spaces},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Error-controlled non-additive interaction discovery in machine learning models. <em>NATMI</em>, <em>7</em>(9), 1541-1554. (<a href='https://doi.org/10.1038/s42256-025-01086-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) models are powerful tools for detecting complex patterns, yet their ‘black-box’ nature limits their interpretability, hindering their use in critical domains like healthcare and finance. Interpretable ML methods aim to explain how features influence model predictions but often focus on univariate feature importance, overlooking complex feature interactions. Although recent efforts extend interpretability to feature interactions, existing approaches struggle with robustness and error control, especially under data perturbations. In this study, we introduce Diamond, a method for trustworthy feature interaction discovery. Diamond uniquely integrates the model-X knockoffs framework to control the false discovery rate, ensuring a low proportion of falsely detected interactions. Diamond includes a non-additivity distillation procedure that refines existing interaction importance measures to isolate non-additive interaction effects and preserve false discovery rate control. This approach addresses the limitations of off-the-shelf interaction measures, which, when used naively, can lead to inaccurate discoveries. Diamond’s applicability spans a broad class of ML models, including deep neural networks, transformers, tree-based models and factorization-based models. Empirical evaluations on both simulated and real datasets across various biomedical studies demonstrate its utility in enabling reliable data-driven scientific discoveries. Diamond represents a significant step forward in leveraging ML for scientific innovation and hypothesis generation. Diamond, a statistically rigorous method, is capable of finding meaningful feature interactions within machine learning models, making black-box models more interpretable for science and medicine.},
  archive      = {J_NATMI},
  author       = {Chen, Winston and Jiang, Yifan and Noble, William Stafford and Lu, Yang Young},
  doi          = {10.1038/s42256-025-01086-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1541-1554},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Error-controlled non-additive interaction discovery in machine learning models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks. <em>NATMI</em>, <em>7</em>(9), 1524-1540. (<a href='https://doi.org/10.1038/s42256-025-01082-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering novel drug candidate molecules is a fundamental step in drug development. Generative deep learning models can sample new molecular structures from learned probability distributions; however, their practical use in drug discovery hinges on generating compounds tailored to a specific target molecule. Here we introduce DrugGEN, an end-to-end generative system for the de novo design of drug candidate molecules that interact with a selected protein. The proposed method represents molecules as graphs and processes them using a generative adversarial network that comprises graph transformer layers. Trained on large datasets of drug-like compounds and target-specific bioactive molecules, DrugGEN designed candidate inhibitors for AKT1, a kinase crucial in many cancers. Docking and molecular dynamics simulations suggest that the generated compounds effectively bind to AKT1, and attention maps provide insights into the model’s reasoning. Furthermore, selected de novo molecules were synthesized and shown to inhibit AKT1 at low micromolar concentrations in the context of in vitro enzymatic assays. These results demonstrate the potential of DrugGEN for designing target-specific molecules. Using the open-access DrugGEN codebase, researchers can retrain the model for other druggable proteins, provided a dataset of known bioactive molecules is available. Inhibiting AKT1 kinase can have potentially positive uses against many types of cancer. To find novel molecules targeting this protein, a graph adversarial network is trained as a generative model.},
  archive      = {J_NATMI},
  author       = {Ünlü, Atabey and Çevrim, Elif and Yiğit, Melih Gökay and Sarıgün, Ahmet and Çelikbilek, Hayriye and Bayram, Osman and Kahraman, Deniz Cansen and Olğaç, Abdurrahman and Rifaioglu, Ahmet Sureyya and Banoğlu, Erden and Doğan, Tunca},
  doi          = {10.1038/s42256-025-01082-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1524-1540},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Target-specific de novo design of drug candidate molecules with graph-transformer-based generative adversarial networks},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Brain–computer interface control with artificial intelligence copilots. <em>NATMI</em>, <em>7</em>(9), 1510-1523. (<a href='https://doi.org/10.1038/s42256-025-01090-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motor brain–computer interfaces (BCIs) decode neural signals to help people with paralysis move and communicate. Even with important advances in the past two decades, BCIs face a key obstacle to clinical viability: BCI performance should strongly outweigh costs and risks. To significantly increase the BCI performance, we use shared autonomy, where artificial intelligence (AI) copilots collaborate with BCI users to achieve task goals. We demonstrate this AI-BCI in a non-invasive BCI system decoding electroencephalography signals. We first contribute a hybrid adaptive decoding approach using a convolutional neural network and ReFIT-like Kalman filter, enabling healthy users and a participant with paralysis to control computer cursors and robotic arms via decoded electroencephalography signals. We then design two AI copilots to aid BCI users in a cursor control task and a robotic arm pick-and-place task. We demonstrate AI-BCIs that enable a participant with paralysis to achieve 3.9-times-higher performance in target hit rate during cursor control and control a robotic arm to sequentially move random blocks to random locations, a task they could not do without an AI copilot. As AI copilots improve, BCIs designed with shared autonomy may achieve higher performance. AI copilots are integrated into brain–computer interfaces, enabling a paralysed participant to achieve improved control of computer cursors and robotic arms. This shared autonomy approach offers a promising path to increase BCI performance and clinical viability.},
  archive      = {J_NATMI},
  author       = {Lee, Johannes Y. and Lee, Sangjoon and Mishra, Abhishek and Yan, Xu and McMahan, Brandon and Gaisford, Brent and Kobashigawa, Charles and Qu, Mike and Xie, Chang and Kao, Jonathan C.},
  doi          = {10.1038/s42256-025-01090-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1510-1523},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Brain–computer interface control with artificial intelligence copilots},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Conditional generation of real antigen-specific t cell receptor sequences. <em>NATMI</em>, <em>7</em>(9), 1494-1509. (<a href='https://doi.org/10.1038/s42256-025-01096-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent advances in T cell receptor (TCR) engineering, designing functional TCRs against arbitrary targets remains challenging due to complex rules governing cross-reactivity and limited paired data. Here we present TCR-TRANSLATE, a sequence-to-sequence framework that adapts low-resource machine translation techniques to generate antigen-specific TCR sequences against unseen epitopes. By evaluating 12 model variants of the BART and T5 model architectures, we identified key factors affecting performance and utility, revealing discordances between these objectives. Our flagship model, TCRT5, outperforms existing approaches on computational benchmarks, prioritizing functionally relevant sequences at higher ranks. Most significantly, we experimentally validated a computationally designed TCR against Wilms’ tumour antigen, a therapeutically relevant target in leukaemia, excluded from our training and validation sets. Although the identified TCR shows cross-reactivity with pathogen-derived peptides, highlighting limitations in specificity, our work represents the successful computational design of a functional TCR construct against a non-viral epitope from the target sequence alone. Our findings establish a foundation for computational TCR design and reveal current limitations in data availability and methodology, providing a framework for accelerating personalized immunotherapy by reducing the search space for novel targets. TCR-TRANSLATE, a deep learning framework adapting machine translation to immune design, demonstrates the successful generation of a functional T cell receptor sequence for a cancer epitope from the target sequence alone.},
  archive      = {J_NATMI},
  author       = {Karthikeyan, Dhuvarakesh and Bennett, Sarah N. and Reynolds, Amy G. and Vincent, Benjamin G. and Rubinsteyn, Alex},
  doi          = {10.1038/s42256-025-01096-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1494-1509},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Conditional generation of real antigen-specific t cell receptor sequences},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Modelling neural coding in the auditory midbrain with high resolution and accuracy. <em>NATMI</em>, <em>7</em>(9), 1478-1493. (<a href='https://doi.org/10.1038/s42256-025-01104-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational models of auditory processing can be valuable tools for research and technology development. Models of the cochlea are highly accurate and widely used, but models of the auditory brain lag far behind in both performance and penetration. Here we present ICNet, a convolutional encoder–decoder model of neural coding in the inferior colliculus. We developed ICNet using large-scale intracranial recordings from anaesthetized gerbils, addressing three key modelling challenges that are common across all sensory systems: capturing the full statistical structure of neuronal response patterns; accounting for physiological and experimental non-stationarity; and extracting features of sensory processing that are shared across different brains. ICNet provides highly accurate simulation of multi-unit neural responses to a wide range of complex sounds, including near-perfect responses to speech. It also reproduces key neurophysiological phenomena such as forward masking and dynamic range adaptation. ICNet can be used to simulate activity from thousands of neural units or to provide a compact representation of early central auditory processing through its latent dynamics, facilitating a wide range of hearing and audio applications. It can also serve as a foundation core, providing a baseline neural representation for models of active listening or higher-level auditory processing. Drakopoulos et al. present a model that captures the transformation from sound waves to neural activity patterns underlying early auditory processing. The model reproduces neural responses to a range of complex sounds and key neurophysiological phenomena.},
  archive      = {J_NATMI},
  author       = {Drakopoulos, Fotios and Pellatt, Lloyd and Sabesan, Shievanie and Xia, Yiqing and Fragner, Andreas and Lesica, Nicholas A.},
  doi          = {10.1038/s42256-025-01104-9},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1478-1493},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Modelling neural coding in the auditory midbrain with high resolution and accuracy},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping. <em>NATMI</em>, <em>7</em>(9), 1461-1477. (<a href='https://doi.org/10.1038/s42256-025-01099-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) deciphers genome-wide chromatin accessibility, providing profound insights into gene regulation mechanisms. With the rapid advance of sequencing technologies, scATAC-seq data typically encompass numerous samples from various conditions, resulting in complex batch effects, thus necessitating reliable integration tools. While numerous batch integration tools exist for single-cell RNA sequencing data, inherent data characteristic differences limit their effectiveness on scATAC-seq data. Existing integration methods for scATAC-seq data suffer from several fundamental limitations, such as disrupting the biological heterogeneity and focusing solely on low-dimensional correction, which may distort data and hinder downstream analysis. Here we propose Fountain, a deep learning framework for scATAC-seq data integration via rigorous barycentric mapping. Barycentric mapping transforms one data distribution to another in a principled and effective manner through optimal transport. By regularizing barycentric mapping with geometric data information, Fountain achieves accurate batch alignment while preserving biological heterogeneity. Comprehensive experiments across diverse real-world datasets demonstrate the advantages of Fountain over existing methods in batch correction and biological conservation. In addition, the trained Fountain model can integrate data from new batches alongside already integrated data without retraining, enabling continuous online data integration. Moreover, Fountain’s reconstruction strategy generates batch-corrected ATAC profiles, improving the capture of cellular heterogeneity and revealing cell-type-specific implications such as expression enrichment analysis and partitioned heritability analysis. Zhu, Hua and Chen propose Fountain, a deep learning framework for batch integration of scATAC-seq data that utilizes regularized barycentric mapping. It preserves biological heterogeneity, enabling online and original dimensionality integration.},
  archive      = {J_NATMI},
  author       = {Zhu, Shuchen and Hua, Heyang and Chen, Shengquan},
  doi          = {10.1038/s42256-025-01099-3},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1461-1477},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerating protein engineering with fitness landscape modelling and reinforcement learning. <em>NATMI</em>, <em>7</em>(9), 1446-1460. (<a href='https://doi.org/10.1038/s42256-025-01103-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein engineering holds substantial promise for designing proteins with customized functions, yet the vast landscape of potential mutations versus limited laboratory capacity constrains the discovery of optimal sequences. Here, to address this, we present the μProtein framework, which accelerates protein engineering by combining μFormer, a deep learning model for accurate mutational effect prediction, with μSearch, a reinforcement learning algorithm designed to efficiently navigate the protein fitness landscape using μFormer as an oracle. μProtein leverages single-mutation data to predict optimal sequences with complex, multi-amino-acid mutations through its modelling of epistatic interactions and a multi-step search strategy. In addition to strong performance on benchmark datasets, μProtein identified high-gain-of-function multi-point mutants for the enzyme β-lactamase, surpassing one of the highest-known activity levels, in wet laboratory, trained solely on single-mutation data. These results demonstrate μProtein’s capability to discover impactful mutations across the vast protein sequence space, offering a robust, efficient approach for protein optimization. μProtein, combining deep learning and reinforcement learning, is developed to design high-function proteins. This framework, trained only on single-mutation data, discovers multi-site β-lactamase mutants with up to 2,000× growth rates.},
  archive      = {J_NATMI},
  author       = {Sun, Haoran and He, Liang and Deng, Pan and Liu, Guoqing and Zhao, Zhiyu and Jiang, Yuliang and Cao, Chuan and Ju, Fusong and Wu, Lijun and Liu, Haiguang and Qin, Tao and Liu, Tie-Yan},
  doi          = {10.1038/s42256-025-01103-w},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1446-1460},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accelerating protein engineering with fitness landscape modelling and reinforcement learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient protein structure generation with sparse denoising models. <em>NATMI</em>, <em>7</em>(9), 1429-1445. (<a href='https://doi.org/10.1038/s42256-025-01100-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteins play diverse roles in all domains of life and are extensively harnessed as biomolecules in biotechnology, with applications spanning from fundamental research to biomedicine. Therefore, there is considerable interest in computationally designing proteins with specified properties. Protein structure generative models provide a means to design protein structures in a controllable manner and have been successfully applied to address various protein design tasks. Such models are paired with protein sequence and structure predictors to produce and select protein sequences for experimental testing. However, current protein structure generators face important limitations for proteins with more than 400 amino acids and require retraining for protein design tasks unseen during model training. To address the first issue, we introduce salad, a family of sparse all-atom denoising models for protein structure generation. Our models are smaller and faster than the state of the art and matching or improving design quality, successfully generating structures for protein lengths up to 1,000 amino acids. To address the second issue, we combine salad with structure editing, a sampling strategy for expanding the capability of protein denoising models to unseen tasks. We apply our approach to a variety of challenging protein design tasks, from generating protein scaffolds containing functional protein motifs (motif scaffolding) to designing proteins capable of adopting multiple distinct folds under different conditions (multi-state protein design), demonstrating the flexibility of salad and structure editing. A small and fast diffusion model is presented, which is able to efficiently generate long protein backbones.},
  archive      = {J_NATMI},
  author       = {Jendrusch, Michael and Korbel, Jan O.},
  doi          = {10.1038/s42256-025-01100-z},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1429-1445},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Efficient protein structure generation with sparse denoising models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics. <em>NATMI</em>, <em>7</em>(9), 1414-1428. (<a href='https://doi.org/10.1038/s42256-025-01097-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Self-supervised learning (SSL) has emerged as a powerful approach for learning meaningful representations from large-scale unlabelled datasets in single-cell genomics. Richter et al. evaluated SSL pretext tasks on modelling single-cell RNA sequencing (scRNA-seq) data, demonstrating the effective use of SSL models. However, the transferability of these pretrained SSL models to the spatial transcriptomics domain remains unexplored. Here we assess the performance of three SSL models (random mask, gene programme mask and Barlow Twins) pretrained on scRNA-seq data with spatial transcriptomics datasets, focusing on cell-type prediction and spatial clustering. Our experiments demonstrate that the SSL model with random mask strategy exhibits the best overall performance among evaluated SSL models. Moreover, the models trained from scratch on spatial transcriptomics data outperform the fine-tuned SSL models on cell-type prediction, highlighting a domain gap between scRNA-seq and spatial transcriptomics data whose underlying causes remain an open question. Through expanded analyses of multiple imputation methods and data degradation scenarios, we demonstrate that gene imputation would degrade SSL model performance on cell-type prediction, an effect that is exacerbated by increasing data sparsity. Finally, integrating zero-shot random mask embeddings into chosen spatial clustering methods significantly enhanced their accuracy. Overall, our findings provide valuable insights into the limitations and potential of transferring SSL models to spatial transcriptomics and offer practical guidance for researchers leveraging pretrained models for spatial transcriptomics data analysis. Self-supervised learning models for single-cell RNA sequencing data exhibit poor transferability to spatial transcriptomics for cell-type prediction, although their learned features may enhance spatial analysis.},
  archive      = {J_NATMI},
  author       = {Han, Chuangyi and Lin, Senlin and Wang, Zhikang and Cui, Yan and Zou, Qi and Yuan, Zhiyuan},
  doi          = {10.1038/s42256-025-01097-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1414-1428},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosting the predictive power of protein representations with a corpus of text annotations. <em>NATMI</em>, <em>7</em>(9), 1403-1413. (<a href='https://doi.org/10.1038/s42256-025-01088-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models are trained to predict amino acid sequences from vast protein databases and learn to represent proteins as feature vectors. These vector representations have enabled impressive applications, from predicting mutation effects to protein folding. One of the reasons offered for the success of these models is that conserved sequence motifs tend to be important for protein fitness. Yet, the relationship between sequence conservation and fitness can be confounded by the evolutionary and environmental context. Should we, therefore, look to other data sources that may contain more direct functional information? In this work, we conduct a comprehensive study examining the effects of training protein models to predict 19 types of text annotation from UniProt. Our results show that fine-tuning protein models on a subset of these annotations enhances the models’ predictive capabilities on a variety of function prediction tasks. In particular, when evaluated on our tasks, our model outperforms the basic local alignment search tool, which none of the pretrained protein models accomplished. Our results suggest that a much wider array of data modalities, such as text annotations, may be tapped to improve protein language models. Although protein language models have enabled major advances, they often rely on indirect signals that may not fully capture functional relevance. Fine-tuning these models on textual annotations is shown to improve their performance on function prediction tasks.},
  archive      = {J_NATMI},
  author       = {Duan, Haonan and Skreta, Marta and Cotta, Leonardo and Rajaonson, Ella Miray and Dhawan, Nikita and Aspuru-Guzik, Alán and Maddison, Chris J.},
  doi          = {10.1038/s42256-025-01088-6},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1403-1413},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Boosting the predictive power of protein representations with a corpus of text annotations},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury. <em>NATMI</em>, <em>7</em>(9), 1390-1402. (<a href='https://doi.org/10.1038/s42256-025-01105-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spinal cord injury (SCI) disrupts neuromuscular control, severely affecting independence and quality of life. Although upper limb wearable robots hold considerable promise for functional restoration, most existing prototypes have been validated minimally in people with SCI and target almost exclusively hand opening and closing. We introduce a lightweight, modular assistive soft exosuit that simultaneously and automatically supports shoulder abduction and elbow flexion or extension movements using lightweight fabric-based pneumatic actuators, controlled through inertial sensors. The individual elbow modules were first validated in 11 healthy volunteers, and subsequently tested, together with the shoulder module, in 15 individuals with cervical SCI (C4–C7, AIS A–D). In the SCI participants, exosuits assistance resulted in increased static endurance time (by more than 250%), and lower activity of the primary muscles involved in dynamic tasks (by up to 50%). The two SCI participants retaining prehensile capability also improved their scores in the box and block test when assisted. Moreover, the soft actuation provided a safe, comfortable and easy-to-use solution that was positively appreciated by the participants. Collectively, these results provide encouraging evidence that exosuits can augment upper limb motor performance, and may ultimately translate into greater functional independence and quality of life for the SCI population. A lightweight, modular assistive soft exosuit is introduced, which supports shoulder and elbow movement in individuals with cervical spinal cord injury. The device enhances endurance and range of motion, reduces muscle effort and improves clinical test scores.},
  archive      = {J_NATMI},
  author       = {Ferroni, Roberto and D’Avola, Gaetano and Sciarrone, Giorgia and Righi, Gabriele and De Santis, Claudia and Carpaneto, Jacopo and Gandolla, Marta and Del Popolo, Giulio and Micera, Silvestro and Proietti, Tommaso},
  doi          = {10.1038/s42256-025-01105-8},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1390-1402},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A multi-joint soft exosuit improves shoulder and elbow motor functions in individuals with spinal cord injury},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Aligning generalization between humans and machines. <em>NATMI</em>, <em>7</em>(9), 1378-1389. (<a href='https://doi.org/10.1038/s42256-025-01109-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in artificial intelligence (AI)—including generative approaches—have resulted in technology that can support humans in scientific discovery and forming decisions, but may also disrupt democracies and target individuals. The responsible use of AI and its participation in human–AI teams increasingly shows the need for AI alignment, that is, to make AI systems act according to our preferences. A crucial yet often overlooked aspect of these interactions is the different ways in which humans and machines generalize. In cognitive science, human generalization commonly involves abstraction and concept learning. By contrast, AI generalization encompasses out-of-domain generalization in machine learning, rule-based reasoning in symbolic AI, and abstraction in neurosymbolic AI. Here we combine insights from AI and cognitive science to identify key commonalities and differences across three dimensions: notions of, methods for, and evaluation of generalization. We map the different conceptualizations of generalization in AI and cognitive science along these three dimensions and consider their role for alignment in human–AI teaming. This results in interdisciplinary challenges across AI and cognitive science that must be tackled to support effective and cognitively supported alignment in human–AI teaming scenarios. Ilievski et al. examine differences and similarities in the various ways human and AI systems generalize. The insights are important for effectively supporting alignment in human–AI teams.},
  archive      = {J_NATMI},
  author       = {Ilievski, Filip and Hammer, Barbara and van Harmelen, Frank and Paassen, Benjamin and Saralajew, Sascha and Schmid, Ute and Biehl, Michael and Bolognesi, Marianna and Dong, Xin Luna and Gashteovski, Kiril and Hitzler, Pascal and Marra, Giuseppe and Minervini, Pasquale and Mundt, Martin and Ngomo, Axel-Cyrille Ngonga and Oltramari, Alessandro and Pasi, Gabriella and Saribatur, Zeynep G. and Serafini, Luciano and Shawe-Taylor, John and Shwartz, Vered and Skitalinskaya, Gabriella and Stachl, Clemens and van de Ven, Gido M. and Villmann, Thomas},
  doi          = {10.1038/s42256-025-01109-4},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1378-1389},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Aligning generalization between humans and machines},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Real-world validation of a structure-aware pipeline for molecular design. <em>NATMI</em>, <em>7</em>(9), 1376-1377. (<a href='https://doi.org/10.1038/s42256-025-01102-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The next major challenge for artificial intelligence in drug development lies in proving its value in real-world settings. A new technology not only supports the generation of novel chemical entities but also accelerates a range of real-world molecular design tasks.},
  archive      = {J_NATMI},
  author       = {Dias, Ana Laura and Rodrigues, Tiago},
  doi          = {10.1038/s42256-025-01102-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1376-1377},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Real-world validation of a structure-aware pipeline for molecular design},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards agentic science for advancing scientific discovery. <em>NATMI</em>, <em>7</em>(9), 1373-1375. (<a href='https://doi.org/10.1038/s42256-025-01110-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence is transforming scientific discovery through (semi-)autonomous agents capable of reasoning, planning, and interacting with digital and physical environments. This Comment explores the foundations and frontiers of agentic science, outlining its emerging directions, current limitations, and the pathways for responsible integration into scientific practice.},
  archive      = {J_NATMI},
  author       = {Xin, Hongliang and Kitchin, John R. and Kulik, Heather J.},
  doi          = {10.1038/s42256-025-01110-x},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1373-1375},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards agentic science for advancing scientific discovery},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LLMs as all-in-one tools to easily generate publication-ready citation diversity reports. <em>NATMI</em>, <em>7</em>(9), 1371-1372. (<a href='https://doi.org/10.1038/s42256-025-01101-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Cantú, Melissa S. and King, Michael R.},
  doi          = {10.1038/s42256-025-01101-y},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1371-1372},
  shortjournal = {Nat. Mach. Intell.},
  title        = {LLMs as all-in-one tools to easily generate publication-ready citation diversity reports},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Applying genomic AI to combat antibiotic resistance in low-income countries. <em>NATMI</em>, <em>7</em>(9), 1369-1370. (<a href='https://doi.org/10.1038/s42256-025-01108-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Aruhomukama, Dickson},
  doi          = {10.1038/s42256-025-01108-5},
  journal      = {Nature Machine Intelligence},
  month        = {9},
  number       = {9},
  pages        = {1369-1370},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Applying genomic AI to combat antibiotic resistance in low-income countries},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nature">NATURE - 49</h2>
<ul>
<li><details>
<summary>
(2025). Connecting the dots: I trace the history of the east kolkata wetlands. <em>NATURE</em>, <em>646</em>(8084), 508. (<a href='https://doi.org/10.1038/d41586-025-03231-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Environmental-humanities researcher Jenia Mukherjee uncovers the complex relationship between people, canals and wetlands in Kolkata, India. Environmental-humanities researcher Jenia Mukherjee uncovers the complex relationship between people, canals and wetlands in Kolkata, India.},
  archive      = {J_NATURE},
  author       = {Forrester, Nikki},
  doi          = {10.1038/d41586-025-03231-0},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {508},
  shortjournal = {Nature},
  title        = {Connecting the dots: I trace the history of the east kolkata wetlands},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biothreat hunters catch dangerous DNA before it gets made. <em>NATURE</em>, <em>646</em>(8084), 503-505. (<a href='https://doi.org/10.1038/d41586-025-03230-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microsoft researchers found a biological zero-day vulnerability — and created a software patch. Microsoft researchers found a biological zero-day vulnerability — and created a software patch.},
  archive      = {J_NATURE},
  author       = {Hutson, Matthew},
  doi          = {10.1038/d41586-025-03230-1},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {503-505},
  shortjournal = {Nature},
  title        = {Biothreat hunters catch dangerous DNA before it gets made},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 27 things we wish we’d known when we started our PhDs. <em>NATURE</em>, <em>646</em>(8084), 501-502. (<a href='https://doi.org/10.1038/d41586-025-03229-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature’s survey of PhD candidates reveals hard-won wisdom on choosing supervisors, managing mental health and surviving academic culture. Nature’s survey of PhD candidates reveals hard-won wisdom on choosing supervisors, managing mental health and surviving academic culture.},
  archive      = {J_NATURE},
  author       = {Nordling, Linda},
  doi          = {10.1038/d41586-025-03229-8},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {501-502},
  shortjournal = {Nature},
  title        = {27 things we wish we’d known when we started our PhDs},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural basis for the dynamic regulation of mTORC1 by amino acids. <em>NATURE</em>, <em>646</em>(8084), 493-500. (<a href='https://doi.org/10.1038/s41586-025-09428-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanistic target of rapamycin complex 1 (mTORC1) anchors a conserved signalling pathway that regulates growth in response to nutrient availability1–5. Amino acids activate mTORC1 through the Rag GTPases, which are regulated by GATOR, a supercomplex consisting of GATOR1, KICSTOR and the nutrient-sensing hub GATOR2 (refs. 6–9). GATOR2 forms an octagonal cage, with its distinct WD40 domain β-propellers interacting with GATOR1 and the leucine sensors Sestrin1&nbsp;and Sestrin2&nbsp;(SESN1 and SESN2) and the arginine sensor&nbsp;CASTOR1 (ref. 10). The mechanisms through which these sensors regulate GATOR2 and how they detach from it upon binding their cognate amino acids remain unknown. Here, using cryo-electron microscopy, we determined the structures of a stabilized GATOR2 bound to either Sestrin2 or CASTOR1. The sensors occupy distinct and non-overlapping binding sites, disruption of which selectively impairs the ability of mTORC1 to sense individual amino acids. We also resolved the apo (leucine-free) structure of Sestrin2 and characterized the amino acid-induced structural rearrangements within Sestrin2 and CASTOR1 that trigger their dissociation from GATOR2. Binding of either sensor restricts the dynamic WDR24 β-propeller of GATOR2, a domain essential for nutrient-dependent mTORC1 activation. These findings reveal the allosteric mechanisms that convey amino acid sufficiency to GATOR2 and the ensuing structural changes that lead to mTORC1 activation. Using cryo-electron microscopy, the interactions of Sestrin2 and CASTOR1 with GATOR2 were resolved, revealing how mTORC1 activation is regulated and how nutrient availability triggers signalling for cellular growth.},
  archive      = {J_NATURE},
  author       = {Valenstein, Max L. and Wranik, Maximilian and Lalgudi, Pranav V. and Linde-Garelli, Karen Y. and Choi, Yuri and Chivukula, Raghu R. and Sabatini, David M. and Rogala, Kacper B.},
  doi          = {10.1038/s41586-025-09428-7},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {493-500},
  shortjournal = {Nature},
  title        = {Structural basis for the dynamic regulation of mTORC1 by amino acids},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One-shot design of functional protein binders with BindCraft. <em>NATURE</em>, <em>646</em>(8084), 483-492. (<a href='https://doi.org/10.1038/s41586-025-09429-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein–protein interactions are at the core of all key biological processes. However, the complexity of the structural features that determine protein–protein interactions makes their design challenging. Here we present BindCraft, an open-source and automated pipeline for de novo protein binder design with experimental success rates of 10–100%. BindCraft leverages the weights of AlphaFold2 (ref. 1) to generate binders with nanomolar affinity without the need for high-throughput screening or experimental optimization, even in the absence of known binding sites. We successfully designed binders against a diverse set of challenging targets, including cell-surface receptors, common allergens, de novo designed proteins and multi-domain nucleases, such as CRISPR–Cas9. We showcase the functional and therapeutic potential of designed binders by reducing IgE binding to birch allergen in patient-derived samples, modulating Cas9 gene editing activity and reducing the cytotoxicity of a foodborne bacterial enterotoxin. Last, we use cell-surface-receptor-specific binders to redirect adeno-associated virus capsids for targeted gene delivery. This work represents a significant advancement towards a ‘one design-one binder’ approach in computational design, with immense potential in therapeutics, diagnostics and biotechnology. BindCraft, an open-source, automated pipeline for de novo protein binder design with experimental success rates of 10–100%, leverages AlphaFold2 weights to generate binders with nanomolar affinity without the need for high-throughput screening.},
  archive      = {J_NATURE},
  author       = {Pacesa, Martin and Nickel, Lennart and Schellhaas, Christian and Schmidt, Joseph and Pyatova, Ekaterina and Kissling, Lucas and Barendse, Patrick and Choudhury, Jagrity and Kapoor, Srajan and Alcaraz-Serna, Ana and Cho, Yehlin and Ghamary, Kourosh H. and Vinué, Laura and Yachnin, Brahm J. and Wollacott, Andrew M. and Buckley, Stephen and Westphal, Adrie H. and Lindhoud, Simon and Georgeon, Sandrine and Goverde, Casper A. and Hatzopoulos, Georgios N. and Gönczy, Pierre and Muller, Yannick D. and Schwank, Gerald and Swarts, Daan C. and Vecchio, Alex J. and Schneider, Bernard L. and Ovchinnikov, Sergey and Correia, Bruno E.},
  doi          = {10.1038/s41586-025-09429-6},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {483-492},
  shortjournal = {Nature},
  title        = {One-shot design of functional protein binders with BindCraft},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantitative imaging of lipid transport in mammalian cells. <em>NATURE</em>, <em>646</em>(8084), 474-482. (<a href='https://doi.org/10.1038/s41586-025-09432-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Eukaryotic cells produce over 1,000 different lipid species that tune organelle membrane properties, control signalling and store energy1,2. How lipid species are selectively sorted between organelles to maintain specific membrane identities is largely unclear, owing to the difficulty of imaging lipid transport in cells3. Here we measured the&nbsp;retrograde transport and metabolism of individual lipid species in mammalian cells using time-resolved fluorescence imaging of bifunctional lipid probes in combination with ultra-high-resolution mass spectrometry and mathematical modelling. Quantification of lipid flux between organelles revealed that directional, non-vesicular lipid transport is responsible for fast, species-selective lipid sorting, in contrast to the slow, unspecific vesicular membrane trafficking. Using genetic perturbations, we found that coupling between energy-dependent lipid flipping and non-vesicular transport is a mechanism for directional lipid transport. Comparison of metabolic conversion and transport rates showed that non-vesicular transport dominates the organelle distribution of lipids, while species-specific phospholipid metabolism controls neutral lipid accumulation. Our results provide the first quantitative map of retrograde lipid flux in cells4. We anticipate that our pipeline for mapping of lipid flux through physical and chemical space in cells will boost our understanding of lipids in cell biology and disease. Directional, non-vesicular lipid transport is responsible for fast, species-selective lipid sorting&nbsp;into organelle membranes.},
  archive      = {J_NATURE},
  author       = {Iglesias-Artola, Juan M. and Böhlig, Kristin and Schuhmann, Kai and Cook, Katelyn C. and Lennartz, H. Mathilda and Schuhmacher, Milena and Barahtjan, Pavel and Jiménez López, Cristina and Šachl, Radek and Garikapati, Vannuruswamy and Pombo-Garcia, Karina and Lohmann, Annett and Riegerová, Petra and Hof, Martin and Drobot, Björn and Shevchenko, Andrej and Honigmann, Alf and Nadler, André},
  doi          = {10.1038/s41586-025-09432-x},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {474-482},
  shortjournal = {Nature},
  title        = {Quantitative imaging of lipid transport in mammalian cells},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer-induced nerve injury promotes resistance to anti-PD-1 therapy. <em>NATURE</em>, <em>646</em>(8084), 462-473. (<a href='https://doi.org/10.1038/s41586-025-09370-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Perineural invasion (PNI) is a well-established factor of poor prognosis in multiple cancer types1, yet its mechanism remains unclear. Here we provide clinical and mechanistic insights into the role of PNI and cancer-induced nerve injury (CINI) in resistance to anti-PD-1 therapy. Our study demonstrates that PNI and CINI of tumour-associated nerves are associated with poor response to anti-PD-1 therapy among patients with cutaneous squamous cell carcinoma, melanoma and gastric cancer. Electron microscopy and electrical conduction analyses reveal that cancer cells degrade the nerve fibre myelin sheets. The injured neurons respond by autonomously initiating IL-6- and type I interferon-mediated&nbsp;inflammation to promote nerve healing and regeneration. As the tumour grows, the CINI burden increases, and&nbsp;its associated inflammation becomes chronic and skews the general immune tone within the tumour microenvironment into a suppressive and exhaustive state. The CINI-driven anti-PD-1 resistance can be reversed by targeting multiple steps in the CINI signalling process: denervating the tumour, conditional knockout of the transcription factor mediating the injury signal within neurons (Atf3), knockout of interferon-α receptor signalling (Ifnar1−/−) or by combining anti-PD-1 and anti-IL-6-receptor blockade. Our findings demonstrate the direct immunoregulatory roles of CINI and its therapeutic potential. Perineural invasion and cancer-induced nerve injury of tumour-associated nerves are associated with poor response to anti-PD-1 therapy, which can be reversed by combining anti-PD-1 therapy with&nbsp;anti-inflammatory interventions.},
  archive      = {J_NATURE},
  author       = {Baruch, Erez N. and Gleber-Netto, Frederico O. and Nagarajan, Priyadharsini and Rao, Xiayu and Akhter, Shamima and Eichwald, Tuany and Xie, Tongxin and Balood, Mohammad and Adewale, Adebayo and Naara, Shorook and Sathishkumar, Hinduja N. and Islam, Shajedul and McCarthy, William and Mattson, Brandi J. and Ferrarotto, Renata and Wong, Michael K. and Davies, Michael A. and Jindal, Sonali and Basu, Sreyashi and Roversi, Karine and Nikpoor, Amin Reza and Ahmadi, Maryam and Ahmadi, Ali and Harwood, Catherine and Leigh, Irene and Gong, Dennis and Tallón de Lara, Paulino and Tao, Derrick L. and Davidson, Tara M. and Ajami, Nadim J. and Futreal, Andrew and Rai, Kunal and Kochat, Veena and Castillo, Micah and Gunaratne, Preethi and Goepfert, Ryan P. and Hernandez, Sharia D. and Khushalani, Nikhil I. and Wang, Jing and Watowich, Stephanie S. and Calin, George A. and Migden, Michael R. and Yuan, Mona and Liu, Naijiang and Ye, Yi and Hwang, William L. and Vermeer, Paola D. and D’Silva, Nisha J. and Bunimovich, Yuri L. and Yaniv, Dan and Burks, Jared K. and Gomez, Javier and Dougherty, Patrick M. and Tsai, Kenneth Y. and Allison, James P. and Sharma, Padmanee and Wargo, Jennifer A. and Myers, Jeffrey N. and Talbot, Sebastien and Gross, Neil D. and Amit, Moran},
  doi          = {10.1038/s41586-025-09370-8},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {462-473},
  shortjournal = {Nature},
  title        = {Cancer-induced nerve injury promotes resistance to anti-PD-1 therapy},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Axonal injury is a targetable driver of glioblastoma progression. <em>NATURE</em>, <em>646</em>(8084), 452-461. (<a href='https://doi.org/10.1038/s41586-025-09411-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glioblastoma (GBM) is an aggressive and highly therapy-resistant brain tumour1,2. Although advanced disease has been intensely investigated, the mechanisms that underpin the earlier, likely more tractable, stages of GBM development remain poorly understood. Here we identify axonal injury as a key driver of GBM progression, which we find is induced in white matter by early tumour cells preferentially expanding in this region. Mechanistically, axonal injury promotes gliomagenesis by triggering Wallerian degeneration, a targetable active programme of axonal death3, which we show increases neuroinflammation and tumour proliferation. Inactivation of SARM1, the key enzyme activated in response to injury that mediates Wallerian degeneration4, was sufficient to break this tumour-promoting feedforward loop, leading to the development of less advanced terminal tumours and prolonged survival in mice. Thus, targeting the tumour-induced injury microenvironment may supress progression from latent to advanced disease, thereby providing a potential strategy for GBM interception and control. Axonal injury, induced in the white matter by expansion of early tumour cells, is a key driver of glioblastoma progression.},
  archive      = {J_NATURE},
  author       = {Clements, Melanie and Tang, Wenhao and Florjanic Baronik, Zan and Simpson Ragdale, Holly and Oria, Roger and Volteras, Dimitrios and White, Ian J. and Beattie, Gordon and Uddin, Imran and Lenn, Tchern and Lindsay, Rachel and Castro Devesa, Sara and Karamched, Saketh R. and Lythgoe, Mark F. and Shahrezaei, Vahid and Weaver, Valerie M. and Sugisawa, Ryoichi and Roncaroli, Federico and Marguerat, Samuel and Hill, Ciaran S. and Parrinello, Simona},
  doi          = {10.1038/s41586-025-09411-2},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {452-461},
  shortjournal = {Nature},
  title        = {Axonal injury is a targetable driver of glioblastoma progression},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). TCF1 and LEF1 promote B-1a cell homeostasis and regulatory function. <em>NATURE</em>, <em>646</em>(8084), 442-451. (<a href='https://doi.org/10.1038/s41586-025-09421-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {B-1 cells are innate-like immune cells abundant in serosal cavities with antibodies enriched in bacterial recognition, yet their existence in humans has been controversial1–3. The CD5+ B-1a subset expresses anti-inflammatory molecules including IL-10, PDL1 and CTLA4 and can be immunoregulatory4–6. Unlike conventional B cells that are continuously replenished, B-1a cells are produced early in life and maintained through self-renewal7. Here we show that the transcription factors TCF1 and LEF1 are critical regulators of B-1a cells. LEF1 expression is highest in fetal and bone marrow B-1 progenitors, whereas the levels of TCF1 are higher in splenic and peritoneal B-1 cells than in B-1 progenitors. TCF1–LEF1 double deficient mice have reduced B-1a cells and defective B-1a cell maintenance. These transcription factors promote MYC-dependent metabolic pathways and induce a stem-like population upon activation, partly via IL-10 production. In the absence of TCF1 and LEF1, B-1 cells proliferate excessively and acquire an exhausted phenotype with reduced IL-10 and PDL1 expression. Furthermore, adoptive transfer of B-1 cells lacking TCF1 and LEF1 fails to suppress brain inflammation. These transcription factors are also expressed in human chronic lymphocytic leukaemia B cells and in a B-1-like population that is abundant in pleural fluid and circulation of some patients with pleural infection. Our findings define a TCF1–LEF1-driven transcriptional program that integrates stemness and regulatory function in B-1a cells. The transcription factors TCF1 and LEF1 promote self-renewal and regulatory functions in B-1a cells.},
  archive      = {J_NATURE},
  author       = {Shen, Qian and Wang, Hao and Roco, Jonathan A. and Meng, Xiangpeng and Bosticardo, Marita and Hodges, Marie and Battaglia, Michael and Feng, Zhi-Ping and Talks, Benjamin James and Powell, Jason and Shanmugiah, Vijaya Baskar Mahalingam and Chu, Julia and Rahman, Najib M. and Elsheikh, Alguili and Chakravarty, Probir and Grenov, Amalie and Emmerich, Max and Delmonte, Ottavia M. and Freeman, Alexandra F. and Keller, Michael D. and Belaid, Brahim and Papa, Ilenia and Lee, James C. and Cañete, Pablo F. and Gonzalez-Figueroa, Paula and Zhang, Yaoyuan and Xue, Hai-Hui and Turajlic, Samra and Notarangelo, Luigi D. and Haniffa, Muzlifah and Garrett-Sinha, Lee Ann and Parry, Helen M. and Kanellakis, Nikolaos I. and Vinuesa, Carola G.},
  doi          = {10.1038/s41586-025-09421-0},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {442-451},
  shortjournal = {Nature},
  title        = {TCF1 and LEF1 promote B-1a cell homeostasis and regulatory function},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A nanobody specific to prefusion glycoprotein b neutralizes HSV-1 and HSV-2. <em>NATURE</em>, <em>646</em>(8084), 433-441. (<a href='https://doi.org/10.1038/s41586-025-09438-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The nine human herpesviruses, including herpes simplex virus 1 and 2, human cytomegalovirus and Epstein–Barr virus, present a significant burden to global public health1. Their envelopes contain at least ten different glycoproteins, which are necessary for host cell tropism, attachment and entry2. The best conserved among them, glycoprotein B (gB), is essential as it performs membrane fusion by undergoing extensive rearrangements from a prefusion to postfusion conformation. At present, there are no antiviral drugs targeting gB or neutralizing antibodies directed against its prefusion form, because of the difficulty in structurally determining and using this metastable conformation. Here we show the isolation of prefusion-specific nanobodies, one of which exhibits strong neutralizing and cross-species activity. By mutational stabilization we solved the herpes simplex virus 1 gB full-length prefusion structure, which allowed the bound epitope to be determined. Our analyses show the membrane-embedded regions of gB and previously unresolved structural features3,4, including a new fusion loop arrangement, providing insights into the initial conformational changes required for membrane fusion. Binding an epitope spanning three domains, proximal only in the prefusion state, the nanobody keeps wild-type HSV-2 gB in this conformation and enabled its native prefusion structure to be determined. This also indicates the mode of neutralization and an attractive avenue for antiviral interventions. A neutralizing nanobody specific to the prefusion conformation of herpes simplex virus glycoprotein B has cross-species activity and offers insights into virus neutralization, possible immunogens and an attractive avenue for antiviral interventions.},
  archive      = {J_NATURE},
  author       = {Vollmer, Benjamin and Ebel, Henriette and Rees, Renate and Nentwig, Julia and Mulvaney, Thomas and Schünemann, Jürgen and Krull, Jens and Topf, Maya and Görlich, Dirk and Grünewald, Kay},
  doi          = {10.1038/s41586-025-09438-5},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {433-441},
  shortjournal = {Nature},
  title        = {A nanobody specific to prefusion glycoprotein b neutralizes HSV-1 and HSV-2},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Discovery of a widespread chemical signalling pathway in the bacteroidota. <em>NATURE</em>, <em>646</em>(8084), 423-432. (<a href='https://doi.org/10.1038/s41586-025-09418-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Considerable advances have been made in characterizing bioactive molecules secreted by bacteria, yet the regulatory elements controlling their production remain largely understudied. Here we identify and characterize the N-acyl-cyclolysine (ACL) system—a cell-density-dependent chemical signalling system specific to and widespread in the phylum Bacteroidota (formerly Bacteroidetes)—and show that it regulates the expression of co-localized operons encoding diverse secreted molecules. Using genetic and biochemical analyses, combined with structural studies of a key biosynthetic enzyme, AclA, we elucidate the molecular structure of various ACLs and their complete biosynthetic pathway involving l-lysine acylation and ATP-dependent cyclization. Furthermore, we find that secreted ACLs are sensed by a dedicated transcription factor, AclR, resulting in the expression of associated operons and the autoinduction of ACL biosynthesis. Moreover, we show that different Bacteroidota strains produce structurally diverse ACLs and encode transcription factors with varying ligand specificities. Finally, we find that the acl circuit is widely distributed and transcribed in human gut and oral microbiome samples, with clear evidence for an active role in regulating associated operons under host colonization conditions. Understanding the function of the ACL system in different contexts has the potential to reveal details about the biology, ecology and chemistry of the Bacteroidota and how members of this phylum interact with their environments and hosts. The N-acyl-cyclolysine system regulates the expression of co-localized operons encoding diverse secreted molecules in Bacteroidota.},
  archive      = {J_NATURE},
  author       = {Linares-Otoya, Luis and Shirkey, Jaden D. and Chhetri, Bhuwan Khatri and Mira, Amira and Biswas, Abhishek and Neff, Samuel L. and Linares-Otoya, Maria V. and Chen, Ye and Campos-Florian, Julio V. and Ganoza-Yupanqui, Mayar L. and Jeffrey, Philip D. and Hughson, Frederick M. and Donia, Mohamed S.},
  doi          = {10.1038/s41586-025-09418-9},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {423-432},
  shortjournal = {Nature},
  title        = {Discovery of a widespread chemical signalling pathway in the bacteroidota},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rewiring of cortical glucose metabolism fuels human brain cancer growth. <em>NATURE</em>, <em>646</em>(8084), 413-422. (<a href='https://doi.org/10.1038/s41586-025-09460-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The brain avidly consumes glucose to fuel neurophysiology1. Cancers of the brain, such as glioblastoma, relinquish physiological integrity and gain the ability to proliferate and invade healthy tissue2. How brain cancers rewire glucose use to drive aggressive growth remains unclear. Here we infused 13C-labelled glucose into patients and mice with brain cancer, coupled with quantitative metabolic flux analysis, to map the fates of glucose-derived carbon in tumour versus cortex. Through direct and comprehensive measurements of carbon and nitrogen labelling in both cortex and glioma tissues, we identify profound metabolic transformations. In the human cortex, glucose carbons fuel essential physiological processes, including tricarboxylic acid cycle oxidation and neurotransmitter synthesis. Conversely, gliomas downregulate these processes and scavenge alternative carbon sources such as amino acids from the environment, repurposing glucose-derived carbons to generate molecules needed for proliferation and invasion. Targeting this metabolic rewiring in mice through dietary amino acid modulation selectively alters glioblastoma metabolism, slows tumour growth and augments the efficacy of standard-of-care treatments. These findings illuminate how aggressive brain tumours exploit glucose to suppress normal physiological activity in favour of malignant expansion and offer potential therapeutic strategies to enhance treatment outcomes. The cortex fuels essential physiological processes with glucose-derived carbon, while gliomas fuel their aggressiveness by rerouting glucose carbon pathways and scavenging alternative carbon sources such as&nbsp;environmental amino acids, providing&nbsp;a potential therapeutic target.},
  archive      = {J_NATURE},
  author       = {Scott, Andrew J. and Mittal, Anjali and Meghdadi, Baharan and O’Brien, Alexandra and Bailleul, Justine and Sravya, Palavalasa and Achreja, Abhinav and Zhou, Weihua and Xu, Jie and Lin, Angelica and Wilder-Romans, Kari and Liang, Ningning and Kothari, Ayesha U. and Korimerla, Navyateja and Edwards, Donna M. and Wu, Zhe and Feng, Jiane and Su, Sophia and Zhang, Li and Sajjakulnukit, Peter and Andren, Anthony C. and Park, Junyoung O. and ten Hoeve, Johanna and Tarnal, Vijay and Redic, Kimberly A. and Qi, Nathan R. and Fischer, Joshua L. and Yang, Ethan and Regan, Michael S. and Stopka, Sylwia A. and Baquer, Gerard and Suresh, Krithika and Sarkaria, Jann N. and Lawrence, Theodore S. and Venneti, Sriram and Agar, Nathalie Y. R. and Vlashi, Erina and Lyssiotis, Costas A. and Al-Holou, Wajd N. and Nagrath, Deepak and Wahl, Daniel R.},
  doi          = {10.1038/s41586-025-09460-7},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {413-422},
  shortjournal = {Nature},
  title        = {Rewiring of cortical glucose metabolism fuels human brain cancer growth},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SLC45A4 is a pain gene encoding a neuronal polyamine transporter. <em>NATURE</em>, <em>646</em>(8084), 404-412. (<a href='https://doi.org/10.1038/s41586-025-09326-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyamines are regulatory metabolites with key roles in transcription, translation, cell signalling and autophagy1. They are implicated in multiple neurological disorders, including stroke, epilepsy and neurodegeneration, and can regulate neuronal excitability through interactions with ion channels2. Polyamines have been linked to pain, showing altered levels in human persistent pain states and modulation of pain behaviour in animal models3. However, the systems governing polyamine transport within the nervous system remain unclear. Here, undertaking a genome-wide association study (GWAS)&nbsp;of chronic pain intensity in the UK&nbsp;Biobank&nbsp;(UKB), we found a significant association between pain intensity and variants mapping to the SLC45A4 gene locus. In the mouse nervous system, Slc45a4 expression is enriched in all sensory neuron subtypes within the dorsal root ganglion, including nociceptors. Cell-based assays show that SLC45A4 is a selective plasma membrane polyamine transporter, and the cryo-electron microscopy (cryo-EM) structure reveals a regulatory domain and basis for polyamine recognition. Mice lacking SLC45A4 show normal mechanosensitivity but reduced sensitivity to noxious heat- and algogen-induced tonic pain that is associated with reduced excitability of C-polymodal nociceptors. Our findings therefore establish a role for neuronal polyamine transport in pain perception and identify a target for therapeutic intervention in pain treatment. The SLC45A4 gene encodes a neuronal polyamine transporter and is linked to pain response in humans and mice.},
  archive      = {J_NATURE},
  author       = {Middleton, Steven J. and Markússon, Sigurbjörn and Åkerlund, Mikael and Deme, Justin C. and Tseng, Mandy and Li, Wenqianglong and Zuberi, Sana R. and Kuteyi, Gabriel and Sarkies, Peter and Baskozos, Georgios and Perez-Sanchez, Jimena and Farah, Adham and Hébert, Harry L. and Toikumo, Sylvanus and Yu, Zhanru and Maxwell, Susan and Dong, Yin Y. and Kessler, Benedikt M. and Kranzler, Henry R. and Linley, John E. and Smith, Blair H. and Lea, Susan M. and Parker, Joanne L. and Lyssenko, Valeriya and Newstead, Simon and Bennett, David L.},
  doi          = {10.1038/s41586-025-09326-y},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {404-412},
  shortjournal = {Nature},
  title        = {SLC45A4 is a pain gene encoding a neuronal polyamine transporter},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A circuit that integrates drive state and social contact to gate mating. <em>NATURE</em>, <em>646</em>(8084), 394-403. (<a href='https://doi.org/10.1038/s41586-025-09327-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Internal motive states, such as sexual arousal, drive behaviour in response to social cues. However, little is known about how internal states and external cues are integrated to release appropriate behaviours at the correct moment during a social interaction, such as the transition from the appetitive to the consummatory phases of mating1,2. Here we identify a neural circuit in male mice that gates the onset of consummatory reproductive behaviours on contact with a mating partner. Stimulating MPOAEsr1∩Vgat hypothalamic neurons promotes mounting of conspecifics and three-dimensional dummy objects3. We find that such mounting depends on mechanosensory but not visual cues. Through a large-scale electrophysiological screen, we identify neurons in the subparafascicular thalamic nucleus that nonlinearly integrate medial preoptic area of the hypothalamus (MPOA) and mechanosensory input to encode contact with a potential mate. Circuit tracing and perturbations demonstrated that this conjunctive coding occurs by means of convergent disinhibition from MPOA and excitation from the spinal trigeminal nucleus. Functional manipulations and calcium recordings showed these social-contact neurons, marked by parathyroid hormone 2, were essential for and able to promote mounting. These data indicate that subparafascicular thalamic nucleus-parathyroid hormone 2 neurons integrate internal drive with social touch to trigger mounting at opportune moments during mating. More generally, our findings uncover a brain mechanism whereby an internal state can attribute a social quality to a generic touch to initiate purposeful reproductive actions. Analyses of consummatory reproductive behaviours in male mice uncover a brain mechanism whereby an internal state can attribute a social quality to a generic touch to initiate purposeful reproductive actions.},
  archive      = {J_NATURE},
  author       = {Salay, Lindsey D. and Tsao, Doris Y. and Anderson, David J.},
  doi          = {10.1038/s41586-025-09327-x},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {394-403},
  shortjournal = {Nature},
  title        = {A circuit that integrates drive state and social contact to gate mating},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ancient DNA connects large-scale migration with the spread of slavs. <em>NATURE</em>, <em>646</em>(8084), 384-393. (<a href='https://doi.org/10.1038/s41586-025-09437-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The second half of the first millennium ce in Central and Eastern Europe was accompanied by fundamental cultural and political transformations. This period of change is commonly associated with the appearance of the Slavs, which is supported by textual evidence1,2 and coincides with the emergence of similar archaeological horizons3–6. However, so far there has been no consensus on whether this archaeological horizon spread by migration, Slavicisation or a combination of both. Genetic data remain sparse, especially owing to the widespread practice of cremation in the early phase of the Slavic settlement. Here we present genome-wide data from 555 ancient individuals, including 359 samples from Slavic contexts from as early as the seventh century ce. Our data demonstrate large-scale population movement from Eastern Europe during the sixth to eighth centuries, replacing more than 80% of the local gene pool in Eastern Germany, Poland and Croatia. Yet, we also show substantial regional heterogeneity as well as a lack of sex-biased admixture, indicating varying degrees of cultural assimilation of the autochthonous populations. Comparing archaeological and genetic evidence, we find that the change in ancestry in Eastern Germany coincided with a change in social organization, characterized by an intensification of inter- and intra-site genetic relatedness and patrilocality. On the European scale, it appears plausible that the changes in material culture and language between the sixth and eighth centuries were connected to these large-scale population movements. Analyses of ancient human DNA show that cultural and political transformations in Central Europe during the second half of the first millennium ce were associated with movements of Slavic populations into Germany, Poland and Croatia.},
  archive      = {J_NATURE},
  author       = {Gretzinger, Joscha and Biermann, Felix and Mager, Hellen and King, Benedict and Zlámalová, Denisa and Traverso, Luca and Gnecchi Ruscone, Guido A. and Peltola, Sanni and Salmela, Elina and Neumann, Gunnar U. and Radzeviciute, Rita and Ingrová, Pavlína and Liwoch, Radosław and Wronka, Iwona and Jurić, Radomir and Hyrchała, Anna and Niezabitowska-Wiśniewska, Barbara and Bartecki, Bartłomiej and Borowska, Beata and Dzieńkowski, Tomasz and Wołoszyn, Marcin and Wojenka, Michał and Wilczyński, Jarosław and Kot, Małgorzata and Müller, Eric and Orschiedt, Jörg and Zariņa, Gunita and Onkamo, Päivi and Daim, Falko and Muhl, Arnold and Schwarz, Ralf and Majer, Marek and McCormick, Michael and Květina, Jan and Vida, Tivadar and Geary, Patrick J. and Macháček, Jiří and Šlaus, Mario and Meller, Harald and Pohl, Walter and Hofmanová, Zuzana and Krause, Johannes},
  doi          = {10.1038/s41586-025-09437-6},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {384-393},
  shortjournal = {Nature},
  title        = {Ancient DNA connects large-scale migration with the spread of slavs},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hominins on sulawesi during the early pleistocene. <em>NATURE</em>, <em>646</em>(8084), 378-383. (<a href='https://doi.org/10.1038/s41586-025-09348-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dispersal of archaic hominins beyond mainland Southeast Asia (Sunda)1 represents the earliest evidence for humans crossing ocean barriers to reach isolated landmasses2–4. Previously, the oldest indication of hominins in Wallacea, the oceanic island zone east of Sunda, comprised flaked stone artefacts deposited at least 1.02 ± 0.02 million years ago (Ma) at Wolo Sege on Flores5. Early hominins were also established on the oceanic island of Luzon (Philippines), as indicated by both stone artefacts and cut marks on faunal remains dating to between 777 and 631 thousand years ago (ka) at Kalinga6. Moreover, fossils of extinct, small-bodied hominins occur on Flores (Homo floresiensis)7–12 and Luzon (Homo luzonensis)13. On Sulawesi, the largest Wallacean island, previous excavations revealed stone artefacts with a minimum age of 194 ka at the open site of Talepu in the Walanae Depression14, long preceding the earliest known presence of modern humans (Homo sapiens) in the region (73–63 ka in Sunda)15. Here we show that stone artefacts also occur at the nearby site of Calio in fossiliferous layers dated to at least 1.04 Ma and possibly up to 1.48 Ma, using palaeomagnetic dating of sedimentary rocks and coupled Uranium-series (U-series) and electron-spin resonance (US–ESR) dating of fossil teeth. The discovery of Early Pleistocene artefacts at Calio suggests that Sulawesi was populated by hominins at around the same time as Flores, if not earlier. Early Pleistocene artefacts at Calio suggest that Sulawesi was populated by hominins at around the same time as Flores, if not earlier.},
  archive      = {J_NATURE},
  author       = {Hakim, Budianto and Wibowo, Unggul Prasetyo and van den Bergh, Gerrit D. and Yurnaldi, Dida and Joannes-Boyau, Renaud and Duli, Akin and Suryatman and Sardi, Ratno and Nurani, Indah Asikin and Puspaningrum, Mika Rizki and Mahmud, Irfan and Haris, Afdalah and Anshari, Khairun Al and Saiful, Andi Muhammad and Arman Bungaran, P. and Adhityatama, Shinatria and Muhammad, Putra Hudlinas and Akib, Anwar and Somba, Nani and Fakhri and Burhan, Basran and Mas’ud, Zubair and Moore, Mark W. and Perston, Yinika L. and Yu, Wenjing and Aubert, Maxime and Brumm, Adam},
  doi          = {10.1038/s41586-025-09348-6},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {378-383},
  shortjournal = {Nature},
  title        = {Hominins on sulawesi during the early pleistocene},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One mother for two species via obligate cross-species cloning in ants. <em>NATURE</em>, <em>646</em>(8084), 372-377. (<a href='https://doi.org/10.1038/s41586-025-09425-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Living organisms are assumed to produce same-species offspring1,2. Here, we report a shift from this norm in Messor ibericus, an ant that lays individuals from two distinct species. In this life cycle, females must clone males of another species because they require their sperm to produce the worker caste. As a result, males from the same mother exhibit distinct genomes and morphologies, as they belong to species that diverged over 5 million years ago. The evolutionary history of this system appears as sexual parasitism3 that evolved into a natural case of cross-species cloning4,5, resulting in the maintenance of a male-only lineage cloned through distinct species’ ova. We term females exhibiting this reproductive mode as xenoparous, meaning they give birth to other species as part of their life cycle. In a case of obligate cross-species cloning, female ants of Messor ibericus need to clone males of Messor structor to obtain sperm for producing the worker caste, resulting in males from the same mother having distinct genomes and morphologies.},
  archive      = {J_NATURE},
  author       = {Juvé, Y. and Lutrat, C. and Ha, A. and Weyna, A. and Lauroua, E. and Afonso Silva, A. C. and Roux, C. and Schifani, E. and Galkowski, C. and Lebas, C. and Allio, R. and Stoyanov, I. and Galtier, N. and Schlick-Steiner, B. C. and Steiner, F. M. and Baas, D. and Kaufmann, B. and Romiguier, J.},
  doi          = {10.1038/s41586-025-09425-w},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {372-377},
  shortjournal = {Nature},
  title        = {One mother for two species via obligate cross-species cloning in ants},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered yeast provides rare but essential pollen sterols for honeybees. <em>NATURE</em>, <em>646</em>(8084), 365-371. (<a href='https://doi.org/10.1038/s41586-025-09431-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Honeybees are important crop pollinators, but&nbsp;they increasingly face pollen starvation as a result of agricultural intensification and climate change1. Frequent flowering dearth periods and high-density rearing conditions weaken colonies, which often leads to their demise2. Beekeepers provide colonies with pollen substitutes, but these feeds do not sustain brood production because they lack essential sterols found in pollen3,4. Here we describe a technological advance in honeybee nutrition with wide-reaching impacts on global food security. We first measured the quantity and proportion of sterols present in honeybee tissues. Using this information, we genetically engineered a strain of the oleaginous yeast Yarrowia lipolytica to produce a mixture of essential sterols for bees and incorporated this&nbsp;yeast strain into an otherwise nutritionally complete diet. Colonies exclusively fed with this diet reared brood for significantly longer than those fed diets without suitable sterols. The use of this method to incorporate sterol supplements into pollen substitutes will enable honeybee colonies to produce brood in the absence of floral pollen. Optimized diets created using this yeast strain could also reduce competition between bee species for access to natural floral resources and stem the decline in wild bee populations. Genetic engineering of a yeast strain provides rare but essential pollen sterols for honeybee colonies.},
  archive      = {J_NATURE},
  author       = {Moore, Elynor and de Sousa, Raquel T. and Felsinger, Stella and Arnesen, Jonathan A. and Dyekjær, Jane D. and Farman, Dudley I. and Gonçalves, Rui F. S. and Stevenson, Philip C. and Borodina, Irina and Wright, Geraldine A.},
  doi          = {10.1038/s41586-025-09431-y},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {365-371},
  shortjournal = {Nature},
  title        = {Engineered yeast provides rare but essential pollen sterols for honeybees},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Epidemiology models explain rumour spreading during france’s great fear of 1789. <em>NATURE</em>, <em>646</em>(8084), 358-364. (<a href='https://doi.org/10.1038/s41586-025-09392-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Great Fear of 1789, a wave of panic and unrest in rural France fuelled by the spreading of rumours, was an important moment at the onset of the French Revolution, marking the collapse of feudalism and the rise of the new regime1. The Great Fear provides a vivid example of the role the spreading of rumours has in driving political changes that might be relevant today2,3. Here, we collect existing historical records related to the Great Fear and use epidemiology tools and models4 to reconstruct the network of its transmission from town to town. In this way, we quantify the spatiotemporal spread of the rumours and compute key epidemiological parameters, such as the basic reproduction number. Exploiting information on the structure of the road network in eighteenth century France5, we estimate the most probable diffusion paths of the Great Fear and quantify the distribution of spreading velocities. By endowing the nodes in our reconstructed network with indicators related to the institutional, demographic and socio-economic conditions of the time6, including literacy, population size, political participation, wheat prices7,8, income and ownership laws9, and the unequal distribution of land ownership, we compute factors associated with spread of the Great Fear. Our analysis sheds light on unresolved historiographic issues on the significance of the Great Fear for the French Revolution, providing a quantitative answer to the unresolved debate between the role of emotions and rationality in explaining its diffusion. Epidemiological methods are used to show that the Great Fear of 1789, a series of peasant insurrections in rural revolutionary France, was driven by deliberate political action rather than spontaneous fear.},
  archive      = {J_NATURE},
  author       = {Zapperi, Stefano and Varlet-Bertrand, Constant and Bastidon, Cécile and La Porta, Caterina A. M. and Parent, Antoine},
  doi          = {10.1038/s41586-025-09392-2},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {358-364},
  shortjournal = {Nature},
  title        = {Epidemiology models explain rumour spreading during france’s great fear of 1789},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scalable total synthesis of saxitoxin and related natural products. <em>NATURE</em>, <em>646</em>(8084), 351-357. (<a href='https://doi.org/10.1038/s41586-025-09551-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Saxitoxin (STX, 1), a potent neurotoxin from shellfish, first isolated in 1957 (ref. 1), offers immense pharmaceutical potential owing to its interaction with voltage-gated sodium channels2, which are ubiquitously present in all excitable cells of the central and peripheral nervous system3. Hundreds of studies towards its synthesis have been disclosed so far, yet a fully modular and scalable approach to the family remains elusive4–12. Here we show how a tactical combination of radical retrosynthesis, biocatalysis and C–H functionalization logic can be used to solve this problem, resulting in a scalable approach to the STX family in fewer than ten steps, including the first total synthesis of neosaxitoxin (neoSTX, 4), a hydroxylated naturally occurring STX analogue previously under clinical investigation13. The modular nature of the synthesis enables access to diverse analogues that were previously inaccessible and have now been evaluated through electrophysiological assays for biological activity. Using a combination of radical retrosynthesis, biocatalysis and C–H functionalization logic, a concise and scalable approach to the synthesis of saxitoxin and derivatives thereof in fewer than ten steps is presented.},
  archive      = {J_NATURE},
  author       = {Guo, Yinliang and Li, Yiheng and Chen, Sihan and Wu, Yige and Poll, Oscar and Ren, Zhouyang and Liu, Zhonglin and Vlkolinsky, Roman and Bajo, Michal and Prier, Christopher K. and Xiao, Kai-Jiong and Cravatt, Benjamin F. and Roberto, Marisa and Baran, Phil S.},
  doi          = {10.1038/s41586-025-09551-5},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {351-357},
  shortjournal = {Nature},
  title        = {Scalable total synthesis of saxitoxin and related natural products},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tailoring polymer electrolyte solvation for 600 wh kg−1 lithium batteries. <em>NATURE</em>, <em>646</em>(8084), 343-350. (<a href='https://doi.org/10.1038/s41586-025-09565-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polymer electrolytes paired with lithium-rich manganese-based layered oxide (LRMO) cathodes and anode-free cell design are considered one of the most promising high-energy-density and high-safety systems1–4. However, the unstable anode morphological changes and the irreversible anionic reactions at the electrolyte–cathode interfaces induce oxygen escape and catalytic decomposition of polymer electrolytes, resulting in severe interfacial degradation and poor cycling stability. Here we design an in-built fluoropolyether-based polymer electrolyte composed of strongly solvating polyether and weakly solvating fluorohydrocarbon pendants, creating an anion-rich solvation structure and thus anion-derived fluorine-rich interfacial layers on the cathode and anode to resist interfacial issues. The LRMO cathode exhibits improved oxygen redox reversibility with substantially reduced oxygen-involving interfacial side reactions. This quasi-solid-state polymer electrolyte with 30 wt% trimethyl phosphate enables an LRMO cathode with a reversible high-areal-capacity cycling (&gt;8 mAh cm−2) in pouch cells&nbsp;and long-term stability (&gt;500 cycles at 25 °C)&nbsp;in coin cells, respectively. The pouch cells exhibit an energy density of 604 Wh kg−1 (1,027 Wh l−1) and excellent safety under a nail penetration at a fully charged condition. Our work, therefore, provides a promising direction for creating practical high-energy-density and high-safety lithium batteries. An in-built fluoropolyether-based quasi-solid-state polymer electrolyte enables high-capacity lithium-rich manganese-based layered oxide cathodes with stable interfaces, achieving 604 Wh kg−1 pouch-cell energy density.},
  archive      = {J_NATURE},
  author       = {Huang, Xue-Yan and Zhao, Chen-Zi and Kong, Wei-Jin and Yao, Nan and Shuang, Zong-Yao and Xu, Pan and Sun, Shuo and Lu, Yang and Huang, Wen-Ze and Li, Jin-Liang and Shen, Liang and Chen, Xiang and Huang, Jia-Qi and Archer, Lynden A. and Zhang, Qiang},
  doi          = {10.1038/s41586-025-09565-z},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {343-350},
  shortjournal = {Nature},
  title        = {Tailoring polymer electrolyte solvation for 600 wh kg−1 lithium batteries},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A room temperature rechargeable all-solid-state hydride ion battery. <em>NATURE</em>, <em>646</em>(8084), 338-342. (<a href='https://doi.org/10.1038/s41586-025-09561-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As a negative charge carrier, the hydride ion (H−) is more energetic, polarizable and reactive than cations1. An&nbsp;H−-mediated electrochemical process is fundamentally different from existing systems and enables the development of innovative electrochemical devices, such as rechargeable batteries, fuel cells, electrolysis cells and gas separation membranes2. Here we developed a core-shell hydride 3CeH3@BaH2, which exhibits fast H− conduction at ambient temperature and becomes a superionic conductor above 60 °C. This hydride allows us to construct an all-solid-state rechargeable H− battery CeH2|3CeH3@BaH2|NaAlH4, which operates at ambient conditions using NaAlH4 and CeH2 as cathode and anode materials, respectively. This battery has an initial specific capacity of 984 mAh g−1 and retains 402 mAh g−1 after 20 cycles. Using hydrogen as charge carriers can avoid the formation of detrimental metal dendrites, in principle, which creates new research avenues for clean energy storage and conversion. A core-shell hydride 3CeH3@BaH2, which exhibits fast H− conduction at ambient temperature and becomes a superionic conductor above 60 °C, is used to construct an all-solid-state rechargeable H− battery.},
  archive      = {J_NATURE},
  author       = {Cui, Jirong and Zou, Ren and Zhang, Weijin and Wen, Hong and Liu, Jinyao and Wang, Shangshang and Liu, Shukun and Chen, Hetong and Liu, Wei and Ju, Xiaohua and Wang, Weiwei and Gan, Tao and Li, Jiong and Guo, Jianping and He, Teng and Cao, Hujun and Chen, Ping},
  doi          = {10.1038/s41586-025-09561-3},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {338-342},
  shortjournal = {Nature},
  title        = {A room temperature rechargeable all-solid-state hydride ion battery},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A ductile chromium–molybdenum alloy resistant to high-temperature oxidation. <em>NATURE</em>, <em>646</em>(8084), 331-337. (<a href='https://doi.org/10.1038/s41586-025-09516-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Even with the rapid development of renewable energy sources, improving the efficiency of energy conversion from fossil or synthetic fuels remains a challenge because, for example, combustion engines in long-range aircraft will still be needed in the upcoming decades1. Increasing their operating temperatures (1,050–1,150 °C (refs. 2–4)) is one option. This requires replacing single-crystalline Ni-based superalloys in the hottest sections of turbines by refractory-element-based materials, which exhibit much higher solidus temperatures beyond 2,000 °C (refs. 5–7). Here we introduce a single-phase Cr-36.1Mo-3Si (at.%) alloy that meets, for the first time, to our knowledge, the most important critical requirements for refractory-element-based materials: (1) relevant resistance against pesting, nitridation and scale spallation at elevated temperatures, minimum up to 1,100 °C, and (2) sufficient compression ductility at room temperature. Although strength and creep resistance in such alloys were already superior to Ni-based superalloys in several cases, oxidation/corrosion resistance, mandatory to withstand the combustion atmosphere, and ductility/toughness, needed for damage tolerance and device setting, still pose barriers for the development or application of refractory-element-based candidate materials. Any previous successful attempts to address the otherwise catastrophic oxidation of Mo and nitridation of Cr during oxidation suffered from a loss in ductility at ambient temperatures. A single-phase chromium–molybdenum–silicon alloy is described that exhibits compression ductility at room temperature as well as resistance to oxidation, pesting, nitridation and scale spallation at temperatures up to at least 1,100 °C.},
  archive      = {J_NATURE},
  author       = {Hinrichs, Frauke and Winkens, Georg and Kramer, Lena Katharina and Falcão, Gabriely and Hahn, Ewa M. and Schliephake, Daniel and Eusterholz, Michael Konrad and Sen, Sandipan and Galetz, Mathias Christian and Inui, Haruyuki and Kauffmann, Alexander and Heilmaier, Martin},
  doi          = {10.1038/s41586-025-09516-8},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {331-337},
  shortjournal = {Nature},
  title        = {A ductile chromium–molybdenum alloy resistant to high-temperature oxidation},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Isothermal solidification for high-entropy alloy synthesis. <em>NATURE</em>, <em>646</em>(8084), 323-330. (<a href='https://doi.org/10.1038/s41586-025-09530-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Kinetically trapping the high-temperature states through rapid cooling solidification is widely used for the synthesis of high-entropy alloys (HEAs), especially those with intrinsically immiscible elemental combinations1–4. However, strategies need to be developed to overcome the fundamental limitations of rapid cooling solidification in controlling the crystallinity, structure and morphology of HEAs. Here we introduce an isothermal solidification strategy for the synthesis of HEAs by rapidly altering the metal alloy composition through liquid–liquid interface reactions at low temperatures, for example, from 25 °C to 80 °C. We use gallium (Ga)-based metal as the sacrificial reagent and mixing medium. By directing the reactions to the interfaces between the Ga-based liquid metal and an aqueous metal ion solution, the foreign metal ions can be reduced at the interfaces and incorporated into the liquid metal quickly. HEAs with various crystallinity (single crystal, mesocrystal, polycrystal and amorphous), morphology (zero, two and three dimensions) and compositions can be achieved through the isothermal solidification. Ga can be completely consumed, resulting in Ga-free HEAs. If desired, Ga can be one of the metal elements in the final products. In situ liquid phase transmission electron microscopy (TEM) studies and theoretical analysis show the isothermal solidification mechanisms. Our direct observations show the enhanced mixing of liquid metal elements and the solidification process with fluctuating nucleation dynamics. The isothermal solidification marks a powerful strategy for HEA synthesis through an unexplored pathway of kinetically trapping the high-entropy states. Gallium is used as a sacrificial agent and mixing medium for the isothermal solidification synthesis of high-entropy alloy nanomaterials with diverse crystallinities and morphologies.},
  archive      = {J_NATURE},
  author       = {Zhang, Qiubo and Gallant, Max C. and Chen, Yi and Song, Zhigang and Liu, Yang and Zheng, Qi and Chen, Linfeng and Bustillo, Karen. C. and Huang, Yu and Persson, Kristin A. and Zheng, Haimei},
  doi          = {10.1038/s41586-025-09530-w},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {323-330},
  shortjournal = {Nature},
  title        = {Isothermal solidification for high-entropy alloy synthesis},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Heat-rechargeable computation in DNA logic circuits and neural networks. <em>NATURE</em>, <em>646</em>(8084), 315-322. (<a href='https://doi.org/10.1038/s41586-025-09570-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metabolism enables life to sustain dynamics and to repeatedly interact with the environment by storing and consuming chemical energy. A major challenge for artificial molecular machines is to find a universal energy source akin to ATP for biological organisms and electricity for electromechanical machines. More than 20 years ago, DNA was first used as fuel to drive nanomechanical devices1,2 and catalytic reactions3. However, each system requires distinct fuel sequences, preventing DNA alone from becoming a universal energy source. Despite extensive efforts4, we still lack an ATP-like or electricity-like power supply to sustain diverse molecular machines. Here we show that heat can restore enzyme-free DNA circuits from equilibrium to out-of-equilibrium states. During heating and cooling, nucleic acids with strong secondary structures reach kinetically trapped states5,6, providing energy for subsequent computation. We demonstrate that complex logic circuits and neural networks, involving more than 200 distinct molecular species, can respond to a temperature ramp and recharge within minutes, allowing at least 16 rounds of computation with varying sequential inputs. Our strategy enables diverse systems to be powered by the same energy source without problematic waste build-up, thereby ensuring consistent performance over time. This scalable approach supports the sustained operation of enzyme-free molecular circuits and opens opportunities for advanced autonomous behaviours, such as iterative computation and unsupervised learning in artificial chemical systems. Heat recharges enzyme-free DNA circuits, enabling complex logic operations and neural networks to perform multiple computations, offering a universal energy source for molecular machines and advancing autonomous behaviours in artificial chemical systems.},
  archive      = {J_NATURE},
  author       = {Song, Tianqi and Qian, Lulu},
  doi          = {10.1038/s41586-025-09570-2},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {315-322},
  shortjournal = {Nature},
  title        = {Heat-rechargeable computation in DNA logic circuits and neural networks},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantum-amplified global-phase spectroscopy on an optical clock transition. <em>NATURE</em>, <em>646</em>(8084), 309-314. (<a href='https://doi.org/10.1038/s41586-025-09578-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical lattice clocks are at the forefront of precision metrology1–6, operating near a standard quantum limit set by quantum noise4,7. Harnessing quantum entanglement offers a promising route to surpass this limit8–15; however, there are practical difficulties in terms of scalability and measurement resolution requirements16,17. Here we adapt the holonomic quantum gate concept18 to develop a new Rabi-type ‘global-phase spectroscopy’ that uses the detuning-sensitive global Aharonov–Anandan phase19. With this approach, we can demonstrate quantum-amplified time-reversal spectroscopy on an optical clock transition that achieves directly measured 2.4(7) dB metrological gain and 4.0(8) dB improvement in laser noise sensitivity beyond the standard quantum limit. To this end, we introduce rotary echo to protect the dynamics from inhomogeneities in light–atom coupling and implement a laser-noise-cancelling differential measurement through symmetric phase encoding in two nuclear spin states. Our technique is not limited by measurement resolution, scales easily because of the global nature of entangling interaction and exhibits high resilience to typical experimental imperfections. We expect it to be broadly applicable to next-generation atomic clocks and other quantum sensors approaching the fundamental quantum precision limits20–22. The precision of determining the phase of an optical atomic clock is increased above the standard quantum limit in a new spectroscopic method.},
  archive      = {J_NATURE},
  author       = {Zaporski, Leon and Liu, Qi and Velez, Gustavo and Radzihovsky, Matthew and Li, Zeyang and Colombo, Simone and Pedrozo-Peñafiel, Edwin and Vuletić, Vladan},
  doi          = {10.1038/s41586-025-09578-8},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {309-314},
  shortjournal = {Nature},
  title        = {Quantum-amplified global-phase spectroscopy on an optical clock transition},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Low-overhead transversal fault tolerance for universal quantum computation. <em>NATURE</em>, <em>646</em>(8084), 303-308. (<a href='https://doi.org/10.1038/s41586-025-09543-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fast, reliable logical operations are essential for realizing useful quantum computers1–3. By redundantly encoding logical qubits into many physical qubits and using syndrome measurements to detect and correct errors, we can achieve low logical error rates. However, for many practical quantum error correction codes such as the surface code, owing to syndrome measurement errors, standard constructions require multiple extraction rounds—of the order of the code distance d—for fault-tolerant computation, particularly considering fault-tolerant state preparation4–12. Here we show that logical operations can be performed fault-tolerantly with only a constant number of extraction rounds for a broad class of quantum error correction codes, including the surface code with magic state inputs and feedforward, to achieve ‘transversal algorithmic fault tolerance’. Through the combination of transversal operations7 and new strategies for correlated decoding13, despite only having access to partial syndrome information, we prove that the deviation from the ideal logical measurement distribution can be made exponentially small in the distance, even if the instantaneous quantum state cannot be made close to a logical codeword because of measurement errors. We supplement this proof with circuit-level simulations in a range of relevant settings, demonstrating the fault tolerance and competitive performance of our approach. Our work sheds new light on the theory of quantum fault tolerance and has the potential to reduce the space–time cost of practical fault-tolerant quantum computation by over an order of magnitude. Logical operations can be performed fault-tolerantly with only a constant number of syndrome extraction rounds for a broad class of quantum error correction codes, including the surface code with magic state inputs and feedforward, to achieve ‘transversal algorithmic fault tolerance’.},
  archive      = {J_NATURE},
  author       = {Zhou, Hengyun and Zhao, Chen and Cain, Madelyn and Bluvstein, Dolev and Maskara, Nishad and Duckering, Casey and Hu, Hong-Ye and Wang, Sheng-Tao and Kubica, Aleksander and Lukin, Mikhail D.},
  doi          = {10.1038/s41586-025-09543-5},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {303-308},
  shortjournal = {Nature},
  title        = {Low-overhead transversal fault tolerance for universal quantum computation},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Southward impact excavated magma ocean at the lunar south Pole–Aitken basin. <em>NATURE</em>, <em>646</em>(8084), 297-302. (<a href='https://doi.org/10.1038/s41586-025-09582-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ancient South Pole–Aitken impact basin provides a key data point for our understanding of the evolution of the Moon, as it formed during the earliest pre-Nectarian epoch of lunar history1, excavated more deeply than any other known impact basin2,3 and is found on the lunar far side, about which less is known than the well-explored near side. Here we show that the tapering of the basin outline and the more gradual topographic and crustal thickness transition towards the south support a southward impact trajectory, opposite of that commonly assumed. A broad thorium-rich and iron-rich ejecta deposit southwest of the basin is consistent with partial excavation of late-stage magma ocean liquids. These observations indicate that thorium-rich magma ocean liquids persisted only beneath the southwestern half of the basin at the time of impact, matching predictions for the transition from a global magma ocean to a local enrichment of potassium, rare-earth elements&nbsp;and phosphorus (KREEP) in the&nbsp;near-side Procellarum KREEP Terrane. These results have important implications for the upcoming human exploration of the lunar south pole by Artemis, as proposed landing sites are now recognized to sit on the downrange rim and thorium-rich impact ejecta of the basin. Observations of the shape, topography, crustal thickness&nbsp;and surface composition of the South Pole–Aitken impact basin on the Moon suggest a southward impact trajectory and the excavation of a discontinuous remnant magma ocean from beneath the crust.},
  archive      = {J_NATURE},
  author       = {Andrews-Hanna, Jeffrey C. and Bottke, William F. and Broquet, Adrien and Evans, Alexander J. and Gowman, Gabriel and Johnson, Brandon C. and Keane, James T. and Levin, Janette N. and Mallik, Ananya and Marchi, Simone and Moruzzi, Samantha A. and Roy, Arkadeep and Wakita, Shigeru},
  doi          = {10.1038/s41586-025-09582-y},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {297-302},
  shortjournal = {Nature},
  title        = {Southward impact excavated magma ocean at the lunar south Pole–Aitken basin},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ancient DNA clarifies the early history of the slavs. <em>NATURE</em>, <em>646</em>(8084), 295-296. (<a href='https://doi.org/10.1038/d41586-025-02525-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study examines the emergence of a group that sparked a fundamental change in lifestyle in Central and Eastern Europe in the sixth century. A study examines the emergence of a group that sparked a fundamental change in lifestyle in Central and Eastern Europe in the sixth century.},
  archive      = {J_NATURE},
  author       = {Patzold, Steffen},
  doi          = {10.1038/d41586-025-02525-7},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {295-296},
  shortjournal = {Nature},
  title        = {Ancient DNA clarifies the early history of the slavs},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sacrificial metal speeds up synthesis of nano-alloys from multiple elements. <em>NATURE</em>, <em>646</em>(8084), 293-295. (<a href='https://doi.org/10.1038/d41586-025-02973-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An approach for making nanoscale high-entropy alloys using liquid metal provides much-needed control over the structure and composition of these remarkable materials. An approach for making nanoscale high-entropy alloys using liquid metal provides much-needed control over the structure and composition of these remarkable materials.},
  archive      = {J_NATURE},
  author       = {Kang, Yunqing and Yamauchi, Yusuke},
  doi          = {10.1038/d41586-025-02973-1},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {293-295},
  shortjournal = {Nature},
  title        = {Sacrificial metal speeds up synthesis of nano-alloys from multiple elements},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Depriving brain tumours of an amino acid could enhance chemotherapy. <em>NATURE</em>, <em>646</em>(8084), 292-293. (<a href='https://doi.org/10.1038/d41586-025-02531-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glioblastoma, an aggressive brain cancer, depends on the amino acid serine as a fuel source. A serine-deficient diet has been shown to slow tumour growth in mice. Glioblastoma, an aggressive brain cancer, depends on the amino acid serine as a fuel source. A serine-deficient diet has been shown to slow tumour growth in mice.},
  archive      = {J_NATURE},
  author       = {Mobilio, Daniel and Singh, Sheila},
  doi          = {10.1038/d41586-025-02531-9},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {292-293},
  shortjournal = {Nature},
  title        = {Depriving brain tumours of an amino acid could enhance chemotherapy},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Protests are infectious: Mapping rural unrest in revolutionary france. <em>NATURE</em>, <em>646</em>(8084), 290-291. (<a href='https://doi.org/10.1038/d41586-025-02526-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers have used epidemiological models to determine whether the wave of riots in 1789 known as the Great Fear spread through irrational panic or rational protest. Researchers have used epidemiological models to determine whether the wave of riots in 1789 known as the Great Fear spread through irrational panic or rational protest.},
  archive      = {J_NATURE},
  author       = {Goldstone, Jack A.},
  doi          = {10.1038/d41586-025-02526-6},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {290-291},
  shortjournal = {Nature},
  title        = {Protests are infectious: Mapping rural unrest in revolutionary france},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ant queens produce sons of two distinct species. <em>NATURE</em>, <em>646</em>(8084), 289-290. (<a href='https://doi.org/10.1038/d41586-025-02524-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery of an unusual reproductive system for one ant species solves a long-standing puzzle about a missing population of another ant species. The discovery of an unusual reproductive system for one ant species solves a long-standing puzzle about a missing population of another ant species.},
  archive      = {J_NATURE},
  author       = {Purcell, Jessica},
  doi          = {10.1038/d41586-025-02524-8},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {289-290},
  shortjournal = {Nature},
  title        = {Ant queens produce sons of two distinct species},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Insect biodiversity monitoring must be expanded. <em>NATURE</em>, <em>646</em>(8084), 288. (<a href='https://doi.org/10.1038/d41586-025-03107-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Wyckhuys, Kris A. G. and Basset, Yves and Lewis, Owen T.},
  doi          = {10.1038/d41586-025-03107-3},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {288},
  shortjournal = {Nature},
  title        = {Insect biodiversity monitoring must be expanded},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Incentives accelerate progress on open access in spain. <em>NATURE</em>, <em>646</em>(8084), 288. (<a href='https://doi.org/10.1038/d41586-025-03276-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Arroyo-Machado, Wenceslao and Torres-Salinas, Daniel},
  doi          = {10.1038/d41586-025-03276-1},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {288},
  shortjournal = {Nature},
  title        = {Incentives accelerate progress on open access in spain},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stop destructive fishing in marine protected areas. <em>NATURE</em>, <em>646</em>(8084), 288. (<a href='https://doi.org/10.1038/d41586-025-03277-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Sullivan-Stack, Jenna and Estradivari and Pike, Elizabeth P. and Stephenson, Fabrice},
  doi          = {10.1038/d41586-025-03277-0},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {288},
  shortjournal = {Nature},
  title        = {Stop destructive fishing in marine protected areas},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Microbe discovery holds promise for sustainable biotechnology. <em>NATURE</em>, <em>646</em>(8084), 288. (<a href='https://doi.org/10.1038/d41586-025-03278-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Li, Jie and Liu, Shuwen and Song, Hao and Ding, Shiming},
  doi          = {10.1038/d41586-025-03278-z},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {288},
  shortjournal = {Nature},
  title        = {Microbe discovery holds promise for sustainable biotechnology},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Customizable AI systems that anyone can adapt bring big opportunities — And even bigger risks. <em>NATURE</em>, <em>646</em>(8084), 286-287. (<a href='https://doi.org/10.1038/d41586-025-03228-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Open and adaptable artificial-intelligence models are crucial for scientific progress, but robust safeguards against their misuse are still nascent. Open and adaptable artificial-intelligence models are crucial for scientific progress, but robust safeguards against their misuse are still nascent.},
  archive      = {J_NATURE},
  author       = {Gal, Yarin and Casper, Stephen},
  doi          = {10.1038/d41586-025-03228-9},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {286-287},
  shortjournal = {Nature},
  title        = {Customizable AI systems that anyone can adapt bring big opportunities — And even bigger risks},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stop treating code like an afterthought: Record, share and value it. <em>NATURE</em>, <em>646</em>(8084), 284-286. (<a href='https://doi.org/10.1038/d41586-025-03196-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists, research institutions, funders, libraries and publishers must all improve software practices. Scientists, research institutions, funders, libraries and publishers must all improve software practices.},
  archive      = {J_NATURE},
  author       = {Di Cosmo, Roberto and Granger, Sabrina and Hinsen, Konrad and Jullien, Nicolas and Le Berre, Daniel and Louvet, Violaine and Maumet, Camille and Maurice, Clémentine and Monat, Raphaël and Rougier, Nicolas P.},
  doi          = {10.1038/d41586-025-03196-0},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {284-286},
  shortjournal = {Nature},
  title        = {Stop treating code like an afterthought: Record, share and value it},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Violin therapy: Books in brief. <em>NATURE</em>, <em>646</em>(8084), 283. (<a href='https://doi.org/10.1038/d41586-025-03226-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Andrew Robinson reviews five of the best science picks. Andrew Robinson reviews five of the best science picks.},
  archive      = {J_NATURE},
  author       = {Robinson, Andrew},
  doi          = {10.1038/d41586-025-03226-x},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {283},
  shortjournal = {Nature},
  title        = {Violin therapy: Books in brief},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The cutting-edge medical approaches that could transform ageing. <em>NATURE</em>, <em>646</em>(8084), 281-283. (<a href='https://doi.org/10.1038/d41586-025-03225-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An optimistic take on the state of the longevity field provides hope for treating a range of ageing-related diseases — but feels at odds with the current shape of US research. An optimistic take on the state of the longevity field provides hope for treating a range of ageing-related diseases — but feels at odds with the current shape of US research.},
  archive      = {J_NATURE},
  author       = {Murphy, Coleen T.},
  doi          = {10.1038/d41586-025-03225-y},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {281-283},
  shortjournal = {Nature},
  title        = {The cutting-edge medical approaches that could transform ageing},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). From fossil fuels to ‘green capitalism’: The dilemmas of a just energy transition. <em>NATURE</em>, <em>646</em>(8084), 279-280. (<a href='https://doi.org/10.1038/d41586-025-03224-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Greening the global economy does not have to be done through unjust fuel extraction; societies can choose fairer paths to net-zero emissions. Greening the global economy does not have to be done through unjust fuel extraction; societies can choose fairer paths to net-zero emissions.},
  archive      = {J_NATURE},
  author       = {Kalantzakos, Sophia},
  doi          = {10.1038/d41586-025-03224-z},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {279-280},
  shortjournal = {Nature},
  title        = {From fossil fuels to ‘green capitalism’: The dilemmas of a just energy transition},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Will AI ever win its own nobel? some predict a prize-worthy science discovery soon. <em>NATURE</em>, <em>646</em>(8084), 276-278. (<a href='https://doi.org/10.1038/d41586-025-03223-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Other researchers question whether autonomous AI scientists are possible or even desirable. Other researchers question whether autonomous AI scientists are possible or even desirable.},
  archive      = {J_NATURE},
  author       = {Ahart, Jenna},
  doi          = {10.1038/d41586-025-03223-0},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {276-278},
  shortjournal = {Nature},
  title        = {Will AI ever win its own nobel? some predict a prize-worthy science discovery soon},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI models that lie, cheat and plot murder: How dangerous are LLMs really?. <em>NATURE</em>, <em>646</em>(8084), 272-275. (<a href='https://doi.org/10.1038/d41586-025-03222-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tests of large language models reveal that they can behave in deceptive and potentially harmful ways. What does this mean for the future? Tests of large language models reveal that they can behave in deceptive and potentially harmful ways. What does this mean for the future?},
  archive      = {J_NATURE},
  author       = {Hutson, Matthew},
  doi          = {10.1038/d41586-025-03222-1},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {272-275},
  shortjournal = {Nature},
  title        = {AI models that lie, cheat and plot murder: How dangerous are LLMs really?},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Will your study change the world? this AI tool predicts the impact of your research. <em>NATURE</em>, <em>646</em>(8084), 268-269. (<a href='https://doi.org/10.1038/d41586-025-03120-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A tool called Funding the Frontier visualizes all the downstream impacts of funding — and predicts which studies will have the biggest societal impact. A tool called Funding the Frontier visualizes all the downstream impacts of funding — and predicts which studies will have the biggest societal impact.},
  archive      = {J_NATURE},
  author       = {Owens, Brian},
  doi          = {10.1038/d41586-025-03120-6},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {268-269},
  shortjournal = {Nature},
  title        = {Will your study change the world? this AI tool predicts the impact of your research},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trees of the amazon are becoming even mightier. <em>NATURE</em>, <em>646</em>(8084), 262. (<a href='https://doi.org/10.1038/d41586-025-03137-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Forest giants are becoming bigger and more common than they once were. Forest giants are becoming bigger and more common than they once were.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-03137-x},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {262},
  shortjournal = {Nature},
  title        = {Trees of the amazon are becoming even mightier},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Prizes must recognize machine contributions to discovery. <em>NATURE</em>, <em>646</em>(8084), 261. (<a href='https://doi.org/10.1038/d41586-025-03217-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The future of science will be written by humans and machines together. Awards should reflect that reality. The future of science will be written by humans and machines together. Awards should reflect that reality.},
  archive      = {J_NATURE},
  author       = {Wang, Dashun},
  doi          = {10.1038/d41586-025-03217-y},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {261},
  shortjournal = {Nature},
  title        = {Prizes must recognize machine contributions to discovery},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The EU’s horizon research fund needs to stay a defence-free zone. <em>NATURE</em>, <em>646</em>(8084), 260. (<a href='https://doi.org/10.1038/d41586-025-03221-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is time to sound the alarm as the EU’s research priorities face an unprecedented degree of political interference. It is time to sound the alarm as the EU’s research priorities face an unprecedented degree of political interference.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-03221-2},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {260},
  shortjournal = {Nature},
  title        = {The EU’s horizon research fund needs to stay a defence-free zone},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Weaponizing uncertainty in science and in public health puts people in harm’s way. <em>NATURE</em>, <em>646</em>(8084), 259-260. (<a href='https://doi.org/10.1038/d41586-025-03167-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Those who cite scientific studies to support policies should take care to tell the whole story, especially when it’s complex. Those who cite scientific studies to support policies should take care to tell the whole story, especially when it’s complex.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-03167-5},
  journal      = {Nature},
  month        = {10},
  number       = {8084},
  pages        = {259-260},
  shortjournal = {Nature},
  title        = {Weaponizing uncertainty in science and in public health puts people in harm’s way},
  volume       = {646},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nbt">NBT - 34</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(9), 1580. (<a href='https://doi.org/10.1038/s41587-025-02804-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02804-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry. <em>NBT</em>, <em>43</em>(9), 1577-1579. (<a href='https://doi.org/10.1038/s41587-025-02803-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breaking the cycle of institutional betrayal and the culture of silence requires decisive actions to promote transparency, accountability and supportive environments in both academia and industry.},
  archive      = {J_NBT},
  author       = {Vogelaar, Alison E. and Mahmoudi, Morteza},
  doi          = {10.1038/s41587-025-02803-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1577-1579},
  shortjournal = {Nature Biotech.},
  title        = {Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Global profiling of protein complex dynamics with an experimental library of protein interaction markers. <em>NBT</em>, <em>43</em>(9), 1562-1576. (<a href='https://doi.org/10.1038/s41587-024-02432-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to systematically monitor protein complex dynamics are needed. We introduce serial ultrafiltration combined with limited proteolysis-coupled mass spectrometry (FLiP–MS), a structural proteomics workflow that generates a library of peptide markers specific to changes in PPIs by probing differences in protease susceptibility between complex-bound and monomeric forms of proteins. The library includes markers mapping to protein-binding interfaces and markers reporting on structural changes that accompany PPI changes. Integrating the marker library with LiP–MS data allows for global profiling of protein–protein interactions (PPIs) from unfractionated lysates. We apply FLiP–MS to Saccharomyces cerevisiae and probe changes in protein complex dynamics after DNA replication stress, identifying links between Spt-Ada-Gcn5 acetyltransferase activity and the assembly state of several complexes. FLiP–MS enables protein complex dynamics to be probed on any perturbation, proteome-wide, at high throughput, with peptide-level structural resolution and informing on occupancy of binding interfaces, thus providing both global and molecular views of a system under study. FLiP–MS uses a library of protein–protein interaction markers to understand protein complex dynamics.},
  archive      = {J_NBT},
  author       = {Dörig, Christian and Marulli, Cathy and Peskett, Thomas and Volkmar, Norbert and Pantolini, Lorenzo and Studer, Gabriel and Paleari, Camilla and Frommelt, Fabian and Schwede, Torsten and de Souza, Natalie and Barral, Yves and Picotti, Paola},
  doi          = {10.1038/s41587-024-02432-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1562-1576},
  shortjournal = {Nature Biotech.},
  title        = {Global profiling of protein complex dynamics with an experimental library of protein interaction markers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A circularly permuted CasRx platform for efficient, site-specific RNA editing. <em>NBT</em>, <em>43</em>(9), 1548-1561. (<a href='https://doi.org/10.1038/s41587-024-02430-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inactive Cas13 orthologs have been fused to a mutant human ADAR2 deaminase domain at the C terminus to enable programmable adenosine-to-inosine (A-to-I) RNA editing in selected transcripts. Although promising, existing RNA-editing tools generally suffer from a trade-off between efficacy and specificity, and off-target editing remains an unsolved problem. Here we describe the development of an optimized RNA-editing platform by rational protein engineering, CasRx-based Programmable Editing of RNA Technology (xPERT). We demonstrate that the topological rearrangement of a CasRx K940L mutant by circular permutation results in a robust scaffold for the tethering of a deaminase domain. We benchmark our tool against the REPAIR system and show that xPERT exhibits strong on-target activity like REPAIRv1 but low off-target editing like REPAIRv2. Our xPERT platform can be used to alter RNA sequence information without risking genome damage, effect temporary cellular changes and customize protein function. Site-specific and efficient RNA editing is achieved with a CasRx editor.},
  archive      = {J_NBT},
  author       = {Wang, Yuanming and Liu, Kaiwen Ivy and Liu, Mengying Mandy and Ooi, Kean Hean and Nguyen, Tram Anh and Chee, Jiunn En and Teo, Shun Xiang Danny and He, Shan and Tay, Jie Wen Douglas and Teo, Seok Yee and Liew, Kai Shin and Ge, Xiao Yu and Ng, Zhi Jian and Avagyan, Hasmik and Liu, Hao and Yi, Zirong and Chang, Keziah and Kok, Eng Piew Louis and Chen, Runjia and Yau, Chun En and Koh, Jun Wei and Wan, Yue and Tan, Meng How},
  doi          = {10.1038/s41587-024-02430-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1548-1561},
  shortjournal = {Nature Biotech.},
  title        = {A circularly permuted CasRx platform for efficient, site-specific RNA editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One-step nanoscale expansion microscopy reveals individual protein shapes. <em>NBT</em>, <em>43</em>(9), 1539-1547. (<a href='https://doi.org/10.1038/s41587-024-02431-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The attainable resolution of fluorescence microscopy has reached the subnanometer range, but this technique still fails to image the morphology of single proteins or small molecular complexes. Here, we expand the specimens at least tenfold, label them with conventional fluorophores and image them with conventional light microscopes, acquiring videos in which we analyze fluorescence fluctuations. One-step nanoscale expansion (ONE) microscopy enables the visualization of the shapes of individual membrane and soluble proteins, achieving around 1-nm resolution. We show that conformational changes are readily observable, such as those undergone by the ~17-kDa protein calmodulin upon Ca2+ binding. ONE is also applied to clinical samples, analyzing the morphology of protein aggregates in cerebrospinal fluid from persons with Parkinson disease, potentially aiding disease diagnosis. This technology bridges the gap between high-resolution structural biology techniques and light microscopy, providing new avenues for discoveries in biology and medicine. Combining expansion microscopy with super-resolution radial fluctuations captures the morphology of single proteins.},
  archive      = {J_NBT},
  author       = {Shaib, Ali H. and Chouaib, Abed Alrahman and Chowdhury, Rajdeep and Altendorf, Jonas and Mihaylov, Daniel and Zhang, Chi and Krah, Donatus and Imani, Vanessa and Spencer, Russell K. W. and Georgiev, Svilen Veselinov and Mougios, Nikolaos and Monga, Mehar and Reshetniak, Sofiia and Mimoso, Tiago and Chen, Han and Fatehbasharzad, Parisa and Crzan, Dagmar and Saal, Kim-Ann and Alawieh, Mohamad Mahdi and Alawar, Nadia and Eilts, Janna and Kang, Jinyoung and Soleimani, Alireza and Müller, Marcus and Pape, Constantin and Alvarez, Luis and Trenkwalder, Claudia and Mollenhauer, Brit and Outeiro, Tiago F. and Köster, Sarah and Preobraschenski, Julia and Becherer, Ute and Moser, Tobias and Boyden, Edward S. and Aricescu, A. Radu and Sauer, Markus and Opazo, Felipe and Rizzoli, Silvio O.},
  doi          = {10.1038/s41587-024-02431-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1539-1547},
  shortjournal = {Nature Biotech.},
  title        = {One-step nanoscale expansion microscopy reveals individual protein shapes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of compact transcriptional effectors using high-throughput measurements in diverse contexts. <em>NBT</em>, <em>43</em>(9), 1525-1538. (<a href='https://doi.org/10.1038/s41587-024-02442-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional effectors are protein domains known to activate or repress gene expression; however, a systematic understanding of which effector domains regulate transcription across genomic, cell type and DNA-binding domain (DBD) contexts is lacking. Here we develop dCas9-mediated high-throughput recruitment (HT-recruit), a pooled screening method for quantifying effector function at endogenous target genes and test effector function for a library containing 5,092 nuclear protein Pfam domains across varied contexts. We also map context dependencies of effectors drawn from unannotated protein regions using a larger library tiling chromatin regulators and transcription factors. We find that many effectors depend on target and DBD contexts, such as HLH domains that can act as either activators or repressors. To enable efficient perturbations, we select context-robust domains, including ZNF705 KRAB, that improve CRISPRi tools to silence promoters and enhancers. We engineer a compact human activator called NFZ, by combining NCOA3, FOXO3 and ZNF473 domains, which enables efficient CRISPRa with better viral delivery and inducible control of chimeric antigen receptor T cells. Improved effectors for CRISPRi/CRISPRa are developed following high-throughput screening of transcriptional domains.},
  archive      = {J_NBT},
  author       = {Tycko, Josh and Van, Mike V. and Aradhana and DelRosso, Nicole and Ye, Hanrong and Yao, David and Valbuena, Raeline and Vaughan-Jackson, Alun and Xu, Xiaoshu and Ludwig, Connor and Spees, Kaitlyn and Liu, Katherine and Gu, Mingxin and Khare, Venya and Mukund, Adi Xiyal and Suzuki, Peter H. and Arana, Sophia and Zhang, Catherine and Du, Peter P. and Ornstein, Thea S. and Hess, Gaelen T. and Kamber, Roarke A. and Qi, Lei S. and Khalil, Ahmad S. and Bintu, Lacramioara and Bassik, Michael C.},
  doi          = {10.1038/s41587-024-02442-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1525-1538},
  shortjournal = {Nature Biotech.},
  title        = {Development of compact transcriptional effectors using high-throughput measurements in diverse contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations. <em>NBT</em>, <em>43</em>(9), 1510-1524. (<a href='https://doi.org/10.1038/s41587-024-02428-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To assist the translation of genetic findings to disease pathobiology and therapeutics discovery, we present an ensemble deep learning framework, termed PIONEER (Protein–protein InteractiOn iNtErfacE pRediction), that predicts protein-binding partner-specific interfaces for all known protein interactions in humans and seven other common model organisms to generate comprehensive structurally informed protein interactomes. We demonstrate that PIONEER outperforms existing state-of-the-art methods and experimentally validate its predictions. We show that disease-associated mutations are enriched in PIONEER-predicted protein–protein interfaces and explore their impact on disease prognosis and drug responses. We identify 586 significant protein–protein interactions (PPIs) enriched with PIONEER-predicted interface somatic mutations (termed oncoPPIs) from analysis of approximately 11,000 whole exomes across 33 cancer types and show significant associations of oncoPPIs with patient survival and drug responses. PIONEER, implemented as both a web server platform and a software package, identifies functional consequences of disease-associated alleles and offers a deep learning tool for precision medicine at multiscale interactome network levels. Protein–protein interactomes incorporating structural data predict the functional consequences of disease mutations.},
  archive      = {J_NBT},
  author       = {Xiong, Dapeng and Qiu, Yunguang and Zhao, Junfei and Zhou, Yadi and Lee, Dongjin and Gupta, Shobhita and Torres, Mateo and Lu, Weiqiang and Liang, Siqi and Kang, Jin Joo and Eng, Charis and Loscalzo, Joseph and Cheng, Feixiong and Yu, Haiyuan},
  doi          = {10.1038/s41587-024-02428-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1510-1524},
  shortjournal = {Nature Biotech.},
  title        = {A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors. <em>NBT</em>, <em>43</em>(9), 1496-1509. (<a href='https://doi.org/10.1038/s41587-024-02448-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies to enhance the anti-tumor immune response of stereotactic ablative radiotherapy (SABR) at primary tumors and abscopal sites are under intensive investigation. Here we report a metabolizable binary supracluster (BSCgal) that combines gold nanoclusters as radiosensitizing adjuvants with small interfering RNA (siRNA) targeting the immunosuppressive mediator galectin-1 (Gal-1). BSCgal comprises reversibly crosslinked cationic gold nanoclusters and siRNA complexes in a polymer matrix that biodegrades over weeks, facilitating clearance (90.3% in vivo clearance at 4 weeks) to reduce toxicity. The particle size well above the renal filtration threshold facilitates passive delivery to tumors. Using mouse models of head and neck cancer, we show that BSCgal augments the radiodynamic and immunotherapeutic effects of SABR at the primary and metastatic tumors by promoting tumor-inhibitory leukocytes, upregulating cytotoxic granzyme B and reducing immunosuppressive cell populations. It outperforms SABR plus Gal-1 antagonists, chemoradiation drug cisplatin or PD-1 inhibitor. This work presents a translatable strategy to converge focal radiosensitization with targeted immune checkpoint silencing for personalized radioimmunotherapy. Gold-siRNA clusters boost the immune response of radiotherapy against primary and distant tumors.},
  archive      = {J_NBT},
  author       = {Jiang, Yuyan and Cao, Hongbin and Deng, Huaping and Guan, Li and Langthasa, Jimpi and Colburg, Deana Rae Crystal and Melemenidis, Stavros and Cotton, Renee M. and Aleman, John and Wang, Xiao-Jing and Graves, Edward E. and Kalbasi, Anusha and Pu, Kanyi and Rao, Jianghong and Le, Quynh-Thu},
  doi          = {10.1038/s41587-024-02448-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1496-1509},
  shortjournal = {Nature Biotech.},
  title        = {Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE. <em>NBT</em>, <em>43</em>(9), 1485-1495. (<a href='https://doi.org/10.1038/s41587-024-02426-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing of messenger RNA (mRNA) found in extracellular vesicles (EVs) in liquid biopsies can provide clinical information such as somatic mutations, resistance profiles and tumor recurrence. Despite this, EV mRNA remains underused due to its low abundance in liquid biopsies, and large sample volumes or specialized techniques for analysis are required. Here we introduce Self-amplified and CRISPR-aided Operation to Profile EVs (SCOPE), a platform for EV mRNA detection. SCOPE leverages CRISPR-mediated recognition of target RNA using Cas13 to initiate replication and signal amplification, achieving a sub-attomolar detection limit while maintaining single-nucleotide resolution. As a proof of concept, we designed probes for key mutations in KRAS, BRAF, EGFR and IDH1 genes, optimized protocols for single-pot assays and implemented an automated device for multi-sample detection. We validated SCOPE’s ability to detect early-stage lung cancer in animal models, monitored tumor mutational burden in patients with colorectal cancer and stratified patients with glioblastoma. SCOPE can expedite readouts, augmenting the clinical use of EVs in precision oncology. SCOPE detects low-abundance extracellular vesicle mRNA with high sensitivity for clinical applications.},
  archive      = {J_NBT},
  author       = {Song, Jayeon and Cho, Mi Hyeon and Cho, Hayoung and Song, Younseong and Lee, Sung Woon and Nam, Ho Chul and Yoon, Tae Ho and Shin, Jong Cheol and Hong, Jae-Sang and Kim, Yejin and Ekanayake, Emil and Jeon, Jueun and You, Dong Gil and Im, Sung Gap and Choi, Gyu-Seog and Park, Jun Seok and Carter, Bob C. and Balaj, Leonora and Seo, An Na and Miller, Miles A. and Park, Soo Yeun and Kang, Taejoon and Castro, Cesar M. and Lee, Hakho},
  doi          = {10.1038/s41587-024-02426-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1485-1495},
  shortjournal = {Nature Biotech.},
  title        = {Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Saturation profiling of drug-resistant genetic variants using prime editing. <em>NBT</em>, <em>43</em>(9), 1471-1484. (<a href='https://doi.org/10.1038/s41587-024-02465-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to characterize the functional effects of genetic variants of uncertain significance (VUSs) have been limited by incomplete coverage of the mutational space. In clinical oncology, drug resistance arising from VUSs can prevent optimal treatment. Here we introduce PEER-seq, a high-throughput method based on prime editing that can evaluate the functional effects of single-nucleotide variants (SNVs). PEER-seq introduces both intended SNVs and synonymous marker mutations using prime editing and deep sequences the endogenous target regions to identify the introduced SNVs. We generate and functionally evaluate 2,476 SNVs in the epidermal growth factor receptor gene (EGFR), including 99% of all possible variants in the canonical tyrosine kinase domain. We determined resistance profiles of 95% of all possible EGFR protein variants encoded in the whole tyrosine kinase domain against the common tyrosine kinase inhibitors afatinib, osimertinib and osimertinib in the presence of the co-occurring substitution T790M, in PC-9 cells. Our study has the potential to substantially improve the precision of therapeutic choices in clinical settings. PEER-seq characterizes the drug resistance profiles of thousands of genetic variants.},
  archive      = {J_NBT},
  author       = {Kim, Younggwang and Oh, Hyeong-Cheol and Lee, Seungho and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-024-02465-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1471-1484},
  shortjournal = {Nature Biotech.},
  title        = {Saturation profiling of drug-resistant genetic variants using prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal scanning of genetic variants with base and prime editing. <em>NBT</em>, <em>43</em>(9), 1458-1470. (<a href='https://doi.org/10.1038/s41587-024-02439-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational scanning connects genetic variants to phenotype, enabling the interrogation of protein functions, interactions and variant pathogenicity. However, current methodologies cannot efficiently engineer customizable sets of diverse genetic variants in endogenous loci across cellular contexts in high throughput. Here, we combine cytosine and adenine base editors and a prime editor to assess the pathogenicity of a broad spectrum of variants in the epithelial growth factor receptor gene (EGFR). Using pooled base editing and prime editing guide RNA libraries, we install tens of thousands of variants spanning the full coding sequence of EGFR in multiple cell lines and assess the role of these variants in tumorigenesis and resistance to tyrosine kinase inhibitors. Our EGFR variant scan identifies important hits, supporting the robustness of the approach and revealing underappreciated routes to EGFR activation and drug response. We anticipate that multimodal precision mutational scanning can be applied broadly to characterize genetic variation in any genetic element of interest at high and single-nucleotide resolution. Thousands of rare oncogene variants are evaluated using multimodal gene editing screens.},
  archive      = {J_NBT},
  author       = {Belli, Olivier and Karava, Kyriaki and Farouni, Rick and Platt, Randall J.},
  doi          = {10.1038/s41587-024-02439-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1458-1470},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal scanning of genetic variants with base and prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein. <em>NBT</em>, <em>43</em>(9), 1445-1457. (<a href='https://doi.org/10.1038/s41587-024-02437-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticle (LNP) delivery of clustered regularly interspaced short palindromic repeat (CRISPR) ribonucleoproteins (RNPs) could enable high-efficiency, low-toxicity and scalable in vivo genome editing if efficacious RNP–LNP complexes can be reliably produced. Here we engineer a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) to generate iGeoCas9 variants capable of &gt;100× more genome editing of cells and organs compared with the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP–LNP complexes edit a variety of cell types and induce homology-directed repair in cells receiving codelivered single-stranded DNA templates. Using tissue-selective LNP formulations, we observe genome-editing levels of 16‒37% in the liver and lungs of reporter mice that receive single intravenous injections of iGeoCas9 RNP–LNPs. In addition, iGeoCas9 RNPs complexed to biodegradable LNPs edit the disease-causing SFTPC gene in lung tissue with 19% average efficiency, representing a major improvement over genome-editing levels observed previously using viral or nonviral delivery strategies. These results show that thermostable Cas9 RNP–LNP complexes can expand the therapeutic potential of genome editing. An engineered clustered regularly interspaced short palindromic repeat ribonucleoprotein delivered in lipid nanoparticles efficiently edits cells in vivo.},
  archive      = {J_NBT},
  author       = {Chen, Kai and Han, Hesong and Zhao, Sheng and Xu, Bryant and Yin, Boyan and Lawanprasert, Atip and Trinidad, Marena and Burgstone, Benjamin W. and Murthy, Niren and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-024-02437-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1445-1457},
  shortjournal = {Nature Biotech.},
  title        = {Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering b cells to treat and study human disease. <em>NBT</em>, <em>43</em>(9), 1431-1444. (<a href='https://doi.org/10.1038/s41587-025-02757-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern cell therapies are designed to harness natural biology to treat a range of complex diseases. The field of immunology has shown that B cells exhibit multiple unique features, including a natural propensity to interact with and regulate other immune cells, a high capacity to produce proteins, and a long cellular lifespan, which are being creatively applied in engineered B cell (eB cell) therapies. In recent years, advances in genome editing technologies and animal modeling have facilitated rapid progress in our ability to study eB cells and execute proof-of-concept studies, thus enabling the first clinical trials of eB cell therapies. In this review, we provide an overview of recent developments in eB cell therapies, including early clinical studies. We discuss challenges to clinical implementation, and promising directions for leveraging B cell biology in future applications for cancer and chronic disease. Recent developments in B cell engineering expand our ability to study human disease and drive progress toward new B cell therapies, balancing challenges in clinical implementation with emerging opportunities.},
  archive      = {J_NBT},
  author       = {Trivedi, Nikita and Pitner, Ragan A. and Rawlings, David J. and James, Richard G.},
  doi          = {10.1038/s41587-025-02757-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1431-1444},
  shortjournal = {Nature Biotech.},
  title        = {Engineering b cells to treat and study human disease},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A CRISPR-based method for detecting mRNA in extracellular vesicles. <em>NBT</em>, <em>43</em>(9), 1428-1429. (<a href='https://doi.org/10.1038/s41587-024-02440-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A CRISPR-based assay both recognizes and amplifies target mRNA, achieving sub-attomolar sensitivity with single-nucleotide resolution. This method enables the detection of low-abundance mRNA in extracellular vesicles, providing clinically relevant information for precision oncology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02440-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1428-1429},
  shortjournal = {Nature Biotech.},
  title        = {A CRISPR-based method for detecting mRNA in extracellular vesicles},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Bao and rogers. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02784-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02784-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bao and rogers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Ntziachristos and yao. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02785-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02785-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Ntziachristos and yao},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI designs de novo antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02822-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02822-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {AI designs de novo antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosted mitochondria reverse cognitive impairment in mice. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02823-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02823-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Boosted mitochondria reverse cognitive impairment in mice},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mining archaea for antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02824-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02824-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Mining archaea for antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MicroRNAs. <em>NBT</em>, <em>43</em>(9), 1424. (<a href='https://doi.org/10.1038/s41587-025-02800-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to pharmaceutical compositions and methods of treatment involving microRNAs.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02800-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1424},
  shortjournal = {Nature Biotech.},
  title        = {MicroRNAs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Intellectual property rights in the cultivated meat industry: Opportunities and challenges. <em>NBT</em>, <em>43</em>(9), 1419-1423. (<a href='https://doi.org/10.1038/s41587-025-02798-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivated meat faces escalating intellectual property (IP) issues as the industry expands, including conventional patent rights, the private IP rights of animal breeders and the Convention on Biological Diversity.},
  archive      = {J_NBT},
  author       = {Baek, Garam and Chung, Sukhyun and Yu, Peter and Park, Yooheon},
  doi          = {10.1038/s41587-025-02798-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1419-1423},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property rights in the cultivated meat industry: Opportunities and challenges},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining druggable targets with artificial intelligence. <em>NBT</em>, <em>43</em>(9), 1416-1418. (<a href='https://doi.org/10.1038/s41587-025-02770-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A vast landscape of ‘undruggable’ cancer targets remains beyond the reach of conventional therapeutic agents. Recent advances in artificial intelligence (AI), however, are challenging this paradigm. Synthesizing insights from a Cancer Moonshot workshop, we argue that systemically addressing the undruggable target space with AI requires a new conceptual framework. We highlight the failure of current target taxonomies and the need for benchmarking datasets, and re-evaluate clinical validation for novel AI-driven modalities.},
  archive      = {J_NBT},
  author       = {Akinsanya, Karen and AlQuraishi, Mohammed and Boija, Ann and Chodera, John and Cichońska, Anna and Ghassemi, Marzyeh and Head, Martha and Jin, Wengong and Kibbe, Warren A. and Krogan, Nevan and LeVine, Michael V. and Moult, John and Stuart, Lynda and Tourassi, Georgia and Zou, James and Barzilay, Regina and Elemento, Olivier},
  doi          = {10.1038/s41587-025-02770-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1416-1418},
  shortjournal = {Nature Biotech.},
  title        = {Redefining druggable targets with artificial intelligence},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Senolytics under scrutiny in the quest to slow aging. <em>NBT</em>, <em>43</em>(9), 1413-1415. (<a href='https://doi.org/10.1038/s41587-025-02740-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de Magalhães, João Pedro},
  doi          = {10.1038/s41587-025-02740-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1413-1415},
  shortjournal = {Nature Biotech.},
  title        = {Senolytics under scrutiny in the quest to slow aging},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Freya biosciences. <em>NBT</em>, <em>43</em>(9), 1411-1412. (<a href='https://doi.org/10.1038/s41587-025-02783-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Freya Biosciences is restoring balance in the vaginal microbiome to improve pregnancy outcomes.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02783-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1411-1412},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Freya biosciences},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Vilya therapeutics. <em>NBT</em>, <em>43</em>(9), 1409-1410. (<a href='https://doi.org/10.1038/s41587-025-02782-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Vilya Therapeutics is designing macrocycle drug candidates.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-025-02782-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1409-1410},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Vilya therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Seaport therapeutics. <em>NBT</em>, <em>43</em>(9), 1407-1408. (<a href='https://doi.org/10.1038/s41587-025-02792-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Seaport Therapeutics is modifying central nervous system drugs for delivery through the lymphatic system, bypassing first-pass liver metabolism.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-025-02792-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1407-1408},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Seaport therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(9), 1406. (<a href='https://doi.org/10.1038/s41587-025-02817-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02817-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1406},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blooms supersized. <em>NBT</em>, <em>43</em>(9), 1405. (<a href='https://doi.org/10.1038/s41587-025-02825-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02825-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1405},
  shortjournal = {Nature Biotech.},
  title        = {Blooms supersized},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wegovy approved for MASH. <em>NBT</em>, <em>43</em>(9), 1404. (<a href='https://doi.org/10.1038/s41587-025-02828-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02828-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1404},
  shortjournal = {Nature Biotech.},
  title        = {Wegovy approved for MASH},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA go-ahead for HPV immunotherapy. <em>NBT</em>, <em>43</em>(9), 1402. (<a href='https://doi.org/10.1038/s41587-025-02827-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02827-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1402},
  shortjournal = {Nature Biotech.},
  title        = {FDA go-ahead for HPV immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Regenerative cell therapy marches into corneal blindness. <em>NBT</em>, <em>43</em>(9), 1401-1404. (<a href='https://doi.org/10.1038/s41587-025-02808-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a newly marketed product in Japan and 10 years of clinical data, regenerative cell therapies promise new options for patients facing a choice between invasive surgery or blindness.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew},
  doi          = {10.1038/s41587-025-02808-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1401-1404},
  shortjournal = {Nature Biotech.},
  title        = {Regenerative cell therapy marches into corneal blindness},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). US government slams mRNA vaccine research. <em>NBT</em>, <em>43</em>(9), 1400. (<a href='https://doi.org/10.1038/s41587-025-02826-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02826-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1400},
  shortjournal = {Nature Biotech.},
  title        = {US government slams mRNA vaccine research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mitochondrial transplants make healthy babies. <em>NBT</em>, <em>43</em>(9), 1399-1400. (<a href='https://doi.org/10.1038/s41587-025-02799-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After decades of research and consultation with regulators, mitochondrial donation is taking off in the UK. Will other countries follow?},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-025-02799-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1399-1400},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial transplants make healthy babies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing antivirals. <em>NBT</em>, <em>43</em>(9), 1397-1398. (<a href='https://doi.org/10.1038/s41587-025-02829-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Funding and public support for vaccines in the United States is at an all-time low. Promising research is resulting in effective antivirals, but they should not take the place of vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02829-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1397-1398},
  shortjournal = {Nature Biotech.},
  title        = {Advancing antivirals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nchem">NCHEM - 24</h2>
<ul>
<li><details>
<summary>
(2025). Harnessing light the retinal way. <em>NCHEM</em>, <em>17</em>(10), 1618. (<a href='https://doi.org/10.1038/s41557-025-01924-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Yasuhisa Mizutani marvels at how effectively nature utilizes the π-conjugated system of retinal in proteins.},
  archive      = {J_NCHEM},
  author       = {Mizutani, Yasuhisa},
  doi          = {10.1038/s41557-025-01924-7},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1618},
  shortjournal = {Nat. Chem.},
  title        = {Harnessing light the retinal way},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lattice O–O ligands in fe-incorporated hydroxides enhance water oxidation electrocatalysis. <em>NCHEM</em>, <em>17</em>(10), 1607-1614. (<a href='https://doi.org/10.1038/s41557-025-01898-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the structural dynamics of ligands and their interaction with catalytic centres under reaction conditions remains a fundamental challenge, yet it is essential for catalyst design. Here we reveal an in situ transformation of Ni–Fe hydroxide into a stable superoxo-hydroxide phase, which is accompanied by the formation of lattice O–O (Olatt–Olatt) ligands, as demonstrated using operando 18O-labelling spectroelectrochemistry and machine-learning-assisted global optimization. By correlating the intrinsic activity of Fe with the Olatt–Olatt concentration across a series of Fe-incorporated transition-metal hydroxides and oxides, we demonstrate that Olatt–Olatt triggers Fe activation for oxygen evolution electrocatalysis—a finding further supported by first-principles calculations. Oxygen production proceeds via an adsorbate evolution mechanism, and the enhanced reaction kinetics stem from the lowered activation energy at surface Fe sites in the newly formed superoxo-hydroxide structure. This work offers a strategic framework for designing high-performance Fe-incorporated electrocatalysts and underscores the pivotal role of ligand dynamics in activating catalytic centres. While the ligand coordination microenvironment surrounding catalytic centres influences reactivity, dynamic oxygen reconstruction during water oxidation electrocatalysis complicates structure-based mechanistic insights. Now the in situ formation of lattice O–O ligands has been shown to activate Fe centres in metal oxides and hydroxides, thereby enhancing their oxygen evolution reaction activity.},
  archive      = {J_NCHEM},
  author       = {Shi, Guoshuai and Li, Jili and Lu, Tingyu and Yang, Chunlei and Xu, Qinshang and Gu, Huoliang and Tong, Haonan and Zhao, Siwen and Zhu, Chenyuan and Shen, Yuluo and Wu, Jing and Lao, Xianzhuo and Chen, Peng-Cheng and Li, Jiong and Zhang, Shuo and Shi, Jueli and Zhang, Kelvin H. L. and Li, Ye-Fei and Liu, Zhi-Pan and Zhang, Liming},
  doi          = {10.1038/s41557-025-01898-6},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1607-1614},
  shortjournal = {Nat. Chem.},
  title        = {Lattice O–O ligands in fe-incorporated hydroxides enhance water oxidation electrocatalysis},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Immobilized acyl-transfer molecular reactors enable the solid-phase synthesis of sterically hindered peptides. <em>NCHEM</em>, <em>17</em>(10), 1596-1606. (<a href='https://doi.org/10.1038/s41557-025-01896-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Incorporating unnatural amino acids, such as hindered N-methylated or α,α-disubstituted amino acid(s), into peptides can improve their properties for application in the pharmaceutical and biomedical fields. However, the current solid-phase peptide synthesis (SPPS) faces sluggish reaction rates and low yields when incorporating sterically hindered amino acids, owing to the poor kinetics of the two-phase acyl-transfer process from solution to solid. Here we introduce an immobilized ribosome-mimicking molecular reactor to facilitate on-resin proximity-induced intra(inter)-reactor acyl transfers. This strategy bypasses the two-phase acyl-transfer mechanism in conventional SPPS and boosts coupling efficiency in the solid-phase synthesis of N-methylated and/or α,α-disubstituted amino acid(s)-containing sterically hindered peptides, including cyclosporin A and alamethicin F analogues. The ribosome-mimicking molecular reactor SPPS can be integrated into existing SPPS platforms using commercially available resins and reagents, and displays high compatibility with standard synthesizers, enabling the automated synthesis of pharmaceutically important, sterically hindered difficult peptides. Current solid-phase peptide synthesis methods struggle to incorporate sterically hindered amino acids. Now researchers have developed a ribosome-mimicking molecular reactor that bypasses two-phase acyl transfer to boost the coupling efficiency of peptides containing N-methylated and/or α,α-disubstituted amino acids.},
  archive      = {J_NCHEM},
  author       = {Wei, Siyuan and Zhang, Xuchun and Yang, Xiaoliang and Liu, Fa and Yao, Zhu-Jun},
  doi          = {10.1038/s41557-025-01896-8},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1596-1606},
  shortjournal = {Nat. Chem.},
  title        = {Immobilized acyl-transfer molecular reactors enable the solid-phase synthesis of sterically hindered peptides},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stepwise and reversible assembly of [2Fe–2S] rhombs to [8Fe–8S] clusters and their topological interconversions. <em>NCHEM</em>, <em>17</em>(10), 1586-1595. (<a href='https://doi.org/10.1038/s41557-025-01895-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Among all enzymatic metallocofactors, those found in nitrogenases, the P and L or M clusters, stand out for their intricate structures. They are assembled by proteins of the Nif gene cluster from Fe2S2 rhombs—the smallest building blocks in FeS cluster chemistry—through a sequence of reactions constructing a Fe8S8 precursor. To advance our understanding of how enzymes selectively build such elaborate inorganic molecules, here we parallel the biosynthetic pathway by reporting the rational stepwise assembly of [Fe8S8]m+ (m = 2, 4, 6) clusters from [Fe2S2]2+ rhombs within an extensive cyclic synthetic network. A [Fe8S8]4+ cluster of unique topology is identified, for which we coin the term ‘interlocked’ double cubane. As a molecular analogue of the NifB K cluster, a proposed precursor to both the P and L or M clusters, its preparation and the characterization of all related intermediates, offers fundamental insights into the molecular mechanisms governing the assembly of both biogenic and synthetic FeS clusters. Elucidating the nature of the metallocofactors in nitrogenase enzymes, and preparing synthetic analogues of these clusters, is a classic target for bioinorganic chemists. Now the transformation of [Fe2S2]2+ rhombs to [Fe8S8]n+ clusters has been achieved through a series of redox- and ligand-substitution reactions.},
  archive      = {J_NCHEM},
  author       = {Grunwald, Liam and Weber, Micha L. and Seng, Henrik and Clémancey, Martin and Dubourdeaux, Patrick and Wang, Hongxin and Wörle, Michael and Yoda, Yoshitaka and Cramer, Stephen P. and Blondin, Geneviève and Mougel, Victor},
  doi          = {10.1038/s41557-025-01895-9},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1586-1595},
  shortjournal = {Nat. Chem.},
  title        = {Stepwise and reversible assembly of [2Fe–2S] rhombs to [8Fe–8S] clusters and their topological interconversions},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unlocking azole chemical space via modular and regioselective N-alkylation. <em>NCHEM</em>, <em>17</em>(10), 1576-1585. (<a href='https://doi.org/10.1038/s41557-025-01891-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Azoles are important synthetic targets due to their diverse applications in areas ranging from human health to food security. Accordingly, access to N-functionalized azoles is an essential goal in modern synthetic chemistry. Surprisingly, however, the relied-upon azole N-alkylation strategies fundamentally limit the structural diversity of these important compounds that can be synthesized and studied. Here we introduce an approach to prepare a broad array of important but difficult-to-access N-alkyl azole compounds. We accomplish this through the introduction of a base-catalysed hydroazolation of readily accessible alkenylthianthrenium electrophiles. This strategy circumvents the classical challenge of azole alkylation regiocontrol through an unusual reversible C–N-bond-forming step that exploits the thermodynamic differences between azole N-alkylation isomers. This reaction furnishes a class of versatile azolothianthrenium building blocks that provides a general platform to investigate diverse N-alkyl azole molecules. More broadly, the distinctive approach outlined through this project is poised to impact the design and development of diverse regioselective alkylation reactions. Azoles are important synthetic targets due to their diverse applications in areas ranging from human health to food security. Now it has been shown that the hydroazolation of alkenylthianthrenium salts provides a modular platform to access diverse, densely functionalized N-alkyl azole compounds with high N-regioselectivity.},
  archive      = {J_NCHEM},
  author       = {Dorval, Céline and Matthews, Adrian D. and Targos, Karina and Alektiar, Sara N. and Holst, Dylan E. and Tan, Zhifeng and Muuronen, Mikko and Diccianni, Justin B. and Gómez, José Enrique and Sanders, Kyana M. and Guzei, Ilia A. and Wickens, Zachary K.},
  doi          = {10.1038/s41557-025-01891-z},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1576-1585},
  shortjournal = {Nat. Chem.},
  title        = {Unlocking azole chemical space via modular and regioselective N-alkylation},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput discovery of fluoroprobes that recognize amyloid fibril polymorphs. <em>NCHEM</em>, <em>17</em>(10), 1565-1575. (<a href='https://doi.org/10.1038/s41557-025-01889-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aggregation of microtubule-associated protein tau into conformationally distinct fibrils underpins neurodegenerative tauopathies. Fluorescent probes (fluoroprobes) such as thioflavin T have been essential tools for studying tau aggregation; however, most of them do not discriminate between amyloid fibril conformations (polymorphs). This gap is due, in part, to a lack of high-throughput methods for screening large, diverse chemical collections. Here we leverage advances in protein-adaptive differential scanning fluorimetry to screen the Aurora collection of 300+ fluoroprobes against multiple synthetic fibril polymorphs, including those formed from tau, α-synuclein and islet amyloid polypeptide. This screen—coupled with excitation-multiplexed bright-emission recording (EMBER) imaging and orthogonal secondary assays—revealed pan-fibril-binding chemotypes, as well as fluoroprobes selective for fibril subsets. One fluoroprobe recognized tau pathology in ex vivo brain slices from Alzheimer’s disease and rodent models. We propose that these scaffolds represent entry points for developing fibril-selective ligands. Amyloid fibrils can adopt a range of distinct conformations, yet it is challenging to rapidly discriminate between these polymorphs. Now methods have been developed to screen large, diverse libraries of turn-on fluorescent dyes to rapidly identify probes that recognize fibril subsets.},
  archive      = {J_NCHEM},
  author       = {Carroll, Emma C. and Yang, Hyunjun and Powell, Wyatt C. and Charvat, Annemarie F. and Oehler, Abby and Jones, Julia G. and Montgomery, Kelly M. and Yung, Anthony and Millbern, Zoe and Taylor, Alexander I. P. and Wilkinson, Martin and Ranson, Neil A. and Radford, Sheena E. and Vinueza, Nelson R. and DeGrado, William F. and Mordes, Daniel A. and Condello, Carlo and Gestwicki, Jason E.},
  doi          = {10.1038/s41557-025-01889-7},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1565-1575},
  shortjournal = {Nat. Chem.},
  title        = {High-throughput discovery of fluoroprobes that recognize amyloid fibril polymorphs},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deaminative giese-type reaction. <em>NCHEM</em>, <em>17</em>(10), 1556-1564. (<a href='https://doi.org/10.1038/s41557-025-01888-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Primary aliphatic amines are essential components in numerous functional molecules and rank among the most readily available commercial building blocks. Although commonly utilized as nitrogen nucleophiles, their application as alkyl sources for constructing (sp3)C–C(sp3) bonds remains a notable challenge. Here we present an approach integrating nitrogen-atom deletion into the aza-Michael reaction, thereby redirecting the classical pathway from (sp3)C–N bond formation to (sp3)C–C(sp3) bond construction. Leveraging commercially available O-diphenylphosphinylhydroxylamine as an efficient nitrogen-deletion reagent, this method enables a wide variety of primary aliphatic amines to serve as alkyl sources in couplings with structurally diverse electron-deficient olefins. This Giese-type reaction proceeds under mild conditions, achieves completion within 10 min and exhibits broad functional-group compatibility. By bridging two foundational transformations—the aza-Michael reaction and the Giese-type reaction—this approach interlinks their product spaces through a unified precursor library, substantially enhancing synthetic utility. Primary aliphatic amines are abundant building blocks but underutilized as alkyl sources in C–C bond formation. Now it has been shown that integrating nitrogen-atom deletion into the aza-Michael reaction redirects the classical pathway from (sp3)C–N bond formation to Giese-type (sp3)C–C(sp3) bond construction.},
  archive      = {J_NCHEM},
  author       = {Ma, Panpan and Cui, Zhangkai and Lu, Hongjian},
  doi          = {10.1038/s41557-025-01888-8},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1556-1564},
  shortjournal = {Nat. Chem.},
  title        = {Deaminative giese-type reaction},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ultrafast charging of two-dimensional polymer cathodes enabled by cross-flow structure design. <em>NCHEM</em>, <em>17</em>(10), 1546-1555. (<a href='https://doi.org/10.1038/s41557-025-01899-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While ultrafast charging and discharging is highly desired for energy storage, the densely packed crystalline inorganic electrodes suffer from sluggish ion transport that limits this capability. Here we report two-dimensional vertical ladder polymer cathode materials that feature layered nanosheets with rich intralayer pores and structural defects alongside weak interlayer interactions. This structural design allows lithium ions to migrate vertically across the intrinsic pores and/or defects accompanied by horizontal intercalation, thus establishing a cross-flow pathway for lithium storage. Such an effective ion-transport method enables flash charging of an ultrahigh-power polymer cathode to ~70% state-of-charge within 30 s at a high current density. Even operated at −50 °C, the polymer cathode achieves 3-min charging to ~55% state-of-charge. Furthermore, we propose an organic–inorganic hybrid strategy that overall improves the electrode-level specific energy at high rates for meeting practical metrics. This work demonstrates the potential of organic electrodes for high-power output under extreme operational conditions. Electrochemical energy-storage devices require rapid charge–discharge profiles, but the dense electrode packing required to achieve high energy densities results in sluggish ion-transport kinetics. Now a two-dimensional vertical ladder polymer cathode has been shown to achieve promising performance and tolerance by using a cross-flow lithium migration pathway.},
  archive      = {J_NCHEM},
  author       = {Deng, Xianming and Liu, Li and Zhang, Shuchao and Chen, Qingxuan and Zhang, Xinming and Wu, Huimin and Chen, Yongqi and Hou, Tingzheng and Liu, Ming and Zhou, Dong and Peng, Lele and Chen, Zhen},
  doi          = {10.1038/s41557-025-01899-5},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1546-1555},
  shortjournal = {Nat. Chem.},
  title        = {Ultrafast charging of two-dimensional polymer cathodes enabled by cross-flow structure design},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantitative profiling of lipid transport between organelles enabled by subcellular photocatalytic labelling. <em>NCHEM</em>, <em>17</em>(10), 1534-1545. (<a href='https://doi.org/10.1038/s41557-025-01886-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Subcellular lipid composition and transport substantially influence the physiological and pathological functions of both cells and organelles. However, lipid transport and turnover between organelles remain poorly understood due to a lack of methods for selectively labelling lipids in organelles. Here we develop a subcellular photocatalytic labelling strategy that enables organelle-selective lipid analysis by mass spectrometry and the quantitative profiling of lipid transport between organelles. We use this approach to quantitatively characterize fatty-acyl-dependent transport of phosphatidylethanolamine and phosphatidylserine lipids between the endoplasmic reticulum and mitochondria, the nucleus or lysosomes. Further experiments revealed the relative contributions of various biosynthesis pathways to the phosphatidylethanolamine and phosphatidylserine lipid compositions in the mitochondria, nucleus and lysosomes. Lysosome-specific photocatalytic labelling revealed the impact of the mTOR kinase pathway on lysosomal lipid metabolism. Together, this subcellularly localized photocatalytic labelling of lipids quantitatively deciphers the subcellular lipid composition and transport, enhancing our understanding of lipid metabolism in living organisms. Subcellular lipid transport between organelles and turnover remain poorly explored due to the technical challenges associated with selective lipid labelling. Now a subcellular photocatalytic labelling strategy has been developed that allows organelle-specific lipid analysis and quantitative profiling of lipid transport.},
  archive      = {J_NCHEM},
  author       = {Chen, Xi and He, Ru and Xiong, Haolin and Wang, Ruohong and Yin, Yandong and Chen, Yiyun and Zhu, Zheng-Jiang},
  doi          = {10.1038/s41557-025-01886-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1534-1545},
  shortjournal = {Nat. Chem.},
  title        = {Quantitative profiling of lipid transport between organelles enabled by subcellular photocatalytic labelling},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diastereodivergent synthesis of multi-substituted cycloalkanes. <em>NCHEM</em>, <em>17</em>(10), 1524-1533. (<a href='https://doi.org/10.1038/s41557-025-01885-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The escape from flat aromatic architectures towards saturated structures enriched with an increased proportion of sp3 atoms (Fsp3) has been harnessed as a drug design strategy. Chemical synthesis of drug-like saturated cycloalkanes, particularly in terms of the precise control of substituents either on the same or opposing faces of the ring plane, poses greater complexities than the assembly of their two-dimensional aromatic analogues. Reliable and predictable divergent synthesis of all isomers of multi-substituted cycloalkanes continues to pose a formidable challenge. Here we show a cobalt-catalysed process for achieving diastereodivergent synthesis. By strategically manipulating ligands, we accomplished diastereodivergent hydroalkylation of methylenecyclohexanes. This cobalt-catalysed system manifests exceptional resilience and versatility in synthesizing all isomers of disubstituted cyclohexanes and piperidines, as well as multi-substituted cyclohexanes, transcending inherent limited selective tendencies in other traditional synthesis strategies. The synthesis of drug-like saturated cycloalkanes is more complex than their two-dimensional aromatic analogues. A formidable challenge lies in synthesizing all isomers of multi-substituted cycloalkanes. Now a cobalt-catalysed system enables diastereodivergent hydroalkylation of substituted methylenecyclohexanes with exceptional versatility. Strategic manipulation of ligands provides access to all isomers of disubstituted cyclohexanes, piperidines and multi-substituted cyclohexanes.},
  archive      = {J_NCHEM},
  author       = {Li, Zhen and Liu, Deguang and Gao, Gen-Wei and Chen, Pei-Wen and Li, Yan and Zeng, Xiongzhi and Li, Zhenyu and Lu, Xi and Fu, Yao},
  doi          = {10.1038/s41557-025-01885-x},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1524-1533},
  shortjournal = {Nat. Chem.},
  title        = {Diastereodivergent synthesis of multi-substituted cycloalkanes},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Introducing metal–sulfur active sites in metal–organic frameworks via post-synthetic modification for hydrogenation catalysis. <em>NCHEM</em>, <em>17</em>(10), 1514-1523. (<a href='https://doi.org/10.1038/s41557-025-01876-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metal–sulfur active sites play a central role in catalytic processes such as hydrogenation and dehydrogenation, yet the majority of active sites in these compounds reside on the surfaces and edges of catalyst particles, limiting overall efficiency. Here we present a strategy to embed metal–sulfur active sites into metal–organic frameworks (MOFs) by converting bridging or terminal chloride ligands into hydroxide and subsequently into sulfide groups through post-synthetic modification. We apply this method to two representative MOF families: one featuring one-dimensional metal–chloride chains and another containing discrete multinuclear metal clusters. Crystallographic and spectroscopic analyses confirm structural integrity and sulfide incorporation, and the transformation is monitored by in situ total scattering methods. The sulfided MOFs display enhanced catalytic activity in the selective hydrogenation of nitroarenes using molecular hydrogen. Density functional theory calculations indicate that sulfur incorporation promotes homolytic metal–ligand bond cleavage and facilitates H2 activation. This work establishes an approach to construct MOFs featuring accessible metal–sulfide sites. Metal–sulfur motifs are commonly found in enzymatic active sites and heterogeneous catalysis, but they remain underexplored in porous solids. Now, sulfur-based ligands have been incorporated into metal–organic frameworks through post-synthetic modifications. The resulting sulfide MOFs exhibit enhanced catalytic performance in the selective hydrogenation of nitroarenes compared with their parent MOFs containing terminal or bridging chloride and hydroxyl groups.},
  archive      = {J_NCHEM},
  author       = {Xie, Haomiao and Khoshooei, Milad Ahmadi and Mandal, Mukunda and Vornholt, Simon M. and Hofmann, Jan and Tufaro, Luke M. and Kirlikovali, Kent O. and Grimes, Dawson A. and Lee, Seryeong and Su, Shengyi and Reischauer, Susanne and Sengupta, Debabrata and Fahy, Kira and Ma, Kaikai and Wang, Xiaoliang and Sha, Fanrui and Gong, Wei and Chen, Yongwei and Vitillo, Jenny G. and Anderson, John S. and Notestein, Justin M. and Chapman, Karena W. and Gagliardi, Laura and Farha, Omar K.},
  doi          = {10.1038/s41557-025-01876-y},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1514-1523},
  shortjournal = {Nat. Chem.},
  title        = {Introducing metal–sulfur active sites in metal–organic frameworks via post-synthetic modification for hydrogenation catalysis},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Magnetic activation of spherical nucleic acids enables the remote control of synthetic cells. <em>NCHEM</em>, <em>17</em>(10), 1505-1513. (<a href='https://doi.org/10.1038/s41557-025-01909-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The flexible and modular design of synthetic cells, comprising lipid vesicles capable of imitating the structure and function of living cells, facilitates their application as drug delivery devices. The ability to control the synthesis of biomolecules within synthetic cells using a tissue-penetrating stimulus opens up additional levels of functionality that has the potential to improve biological potency and circumvent drug leakage from preloaded vesicles. To this end, we have designed spherical nucleic acids comprising DNA promoter sequences decorating magnetic nanoparticle cores. These spherical nucleic acids allowed us to harness the heat dissipated from magnetic hyperthermia (a clinically approved anticancer therapy) to regulate cell-free protein synthesis and release cargo on demand. Furthermore, this magnetic regulation of biosynthesis was achieved using clinically tolerable magnetic field strengths and frequencies. We then deployed an opaque blocking material that is impenetrable by current activation methods to highlight the potential of this technology for targeting and controlling the in situ synthesis of biomolecules using tissue-penetrating magnetic fields deep within the body. The programmability of synthetic cells, comprising lipid vesicles that are capable of imitating the structure and function of living cells, facilitates their application as drug delivery devices. Now, magnetic hyperthermia has been used to control the on-demand synthesis and release of biomolecules from within synthetic cells.},
  archive      = {J_NCHEM},
  author       = {Parkes, Ellen and Al Samad, Assala and Mazzotti, Giacomo and Newell, Charlie and Ng, Brian and Radford, Amy and Booth, Michael J.},
  doi          = {10.1038/s41557-025-01909-6},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1505-1513},
  shortjournal = {Nat. Chem.},
  title        = {Magnetic activation of spherical nucleic acids enables the remote control of synthetic cells},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flame synthesis achieves compositionally tailorable high-entropy metal-containing nanomaterials. <em>NCHEM</em>, <em>17</em>(10), 1497-1504. (<a href='https://doi.org/10.1038/s41557-025-01894-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-entropy metal-containing nanomaterials have garnered interest in diverse fields such as electrocatalysis and energy conversion. Their synthesis typically requires high temperatures (&gt;1,000 K) to facilitate homogeneous mixing and rapid transformation of metal precursors. However, current state-of-the-art approaches typically involve complex reaction environments and require specialized equipment and operations. Herein we demonstrate a versatile flame synthesis process to fabricate high-entropy metallic single atoms and/or nanoparticles supported on soot-like carbon via blending organometallic precursors into fuel (namely, paraffin wax) and subsequent burning. The high flame temperature (~1,800 K) enables strong metal–carbon association with tailorable chemistry and homogeneous bonding between dissimilar metallic elements (up to 25 metals studied), regardless of their thermodynamic compatibility. Additionally, we show high-performance electrosynthesis of hydrogen peroxide to highlight this approach as a promising method for electrocatalyst development. High-entropy nanomaterials, which incorporate multiple homogeneously dispersed elements, are promising for energy conversion, but typically require complex, high-temperature synthesis. Now a scalable flame synthesis method has been shown to produce tunable, high-entropy nanomaterials under facile conditions, enabling the design of advanced electrocatalysts for hydrogen peroxide production and other relevant applications.},
  archive      = {J_NCHEM},
  author       = {Liu, Zhen and Goudeli, Eirini and Guo, Rui and Xie, Houting and Luo, Qinjian and Zhao, Laichun and Xu, Wanjun and Richardson, Joseph J. and Xu, Weijian and Caruso, Frank and Pan, Shuaijun},
  doi          = {10.1038/s41557-025-01894-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1497-1504},
  shortjournal = {Nat. Chem.},
  title        = {Flame synthesis achieves compositionally tailorable high-entropy metal-containing nanomaterials},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stable single-site organonickel catalyst preferentially hydrogenolyses branched polyolefin C–C bonds. <em>NCHEM</em>, <em>17</em>(10), 1488-1496. (<a href='https://doi.org/10.1038/s41557-025-01892-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods of processing accumulated polyolefin waste typically require harsh conditions, precious metals or high metal loadings to achieve appreciable activities. Here we examined supported, single-site organonickel catalysts for polyolefin upcycling. Chemisorption of Ni(COD)2 (COD, 1,5-cyclooctadiene) onto Brønsted acidic sulfated alumina (AlS) yields a highly electrophilic Ni(I) precatalyst, AlS/Ni(COD)2, which is converted under H2 to the active AlS/NiIIH catalyst. This single-site system exhibits unique hydrogenolysis selectivity that favours cleaving branched polyolefin C–C linkages, enabling the hydrogenolytic separation of polyethylene and isotactic polypropylene (iPP) mixtures. Moreover, AlS/NiIIH remains highly selective and active for hydrogenolysis of iPP admixed with polyvinyl chloride, and the spent catalyst can be repeatedly regenerated by AlEt3 treatment. Experimental mechanistic analysis and density functional theory modelling reveal a turnover-limiting C–C scission pathway featuring β-alkyl transfer and strong olefin binding. These results highlight the potential of nickel-based systems for the selective upcycling of complex plastic waste streams. Selective chemical upcycling of polyolefin mixtures remains challenging due to the structural similarity of their backbones. Now it has been shown that a single-site nickel catalyst can preferentially and efficiently cleave branched C–C bonds, enabling the hydrogenolytic separation of isotactic polypropylene from mixtures containing both isotactic polypropylene and polyethylene.},
  archive      = {J_NCHEM},
  author       = {Lai, Qingheng and Zhang, Xinrui and Jiang, Shan and Krzyaniak, Matthew D. and Alayoglu, Selim and Agarwal, Amol and Liu, Yukun and Edenfield, Wilson C. and Kobayashi, Takeshi and Wang, Yuyang and Dravid, Vinayak and Wasielewski, Michael R. and Miller, Jeffery T. and Kratish, Yosi and Marks, Tobin J.},
  doi          = {10.1038/s41557-025-01892-y},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1488-1496},
  shortjournal = {Nat. Chem.},
  title        = {Stable single-site organonickel catalyst preferentially hydrogenolyses branched polyolefin C–C bonds},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanochemical pathway for converting fluoropolymers to fluorochemicals. <em>NCHEM</em>, <em>17</em>(10), 1480-1487. (<a href='https://doi.org/10.1038/s41557-025-01855-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing demand for fluorochemicals has increased the production of toxic hydrogen fluoride (HF)—a widely used fluorination agent—from mineral fluorspar (CaF2). Although potassium fluoride (KF) is a safer alternative fluorination agent, its production still involves HF as an intermediate. Simultaneously, the disposal of fluoropolymer waste, especially polyvinylidene fluoride (PVDF), poses environmental challenges. We present a mechanochemical protocol that decomposes PVDF to generate KF as a nucleophilic fluorinating agent. As an advantage over traditional batch reactions, mechanochemical protocols can be performed quickly, and with or without modest volumes of solvents. We show that KF produced using this method can be used to efficiently construct S–F, C(sp2)–F and C(sp3)–F bonds, providing a sustainable solution to fluorine resource depletion and hazardous HF usage. Our approach also addresses fluoropolymer waste disposal and reduces reliance on natural fluorspar resources, marking a notable advancement in fluorine chemistry and paving the way for safer and eco-friendly fluorination techniques. The production of fluorination agents generally involves the formation of toxic hydrogen fluoride. Now, a mechanochemical protocol to decompose the fluoropolymer polyvinylidene fluoride has been developed, generating potassium fluoride as a fluorinating agent. This approach provides a safer and sustainable fluorination strategy for efficient S–F, C(sp2)–F and C(sp3)–F bond formation.},
  archive      = {J_NCHEM},
  author       = {Hattori, Masashi and Saha, Debarshi and Bacho, Muhamad Zulfaqar and Shibata, Norio},
  doi          = {10.1038/s41557-025-01855-3},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1480-1487},
  shortjournal = {Nat. Chem.},
  title        = {Mechanochemical pathway for converting fluoropolymers to fluorochemicals},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tunable cobalt-catalysed hydrogenation of allenes enabled by multiple metal–ligand cooperative functionalities. <em>NCHEM</em>, <em>17</em>(10), 1469-1479. (<a href='https://doi.org/10.1038/s41557-025-01945-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Catalytic hydrogenation is essential in synthetic chemistry, with ongoing innovations aimed at enhancing the selectivity and efficiency of these reactions. The hydrogenation of multisubstituted allenes, however, presents a long-standing challenge due to the difficulty of controlling multiple selectivity factors simultaneously. Here we introduce a series of chiral pincer cobalt catalysts featuring multiple metal–ligand cooperative functionalities. These catalysts feature an ‘N–H’ moiety as an outer-sphere binding site and an N-heterocycle group as a hemilabile basic site, enabling the use of structurally diverse tridentate ligands for selective hydrogenation of functionalized allenes. This design liberates a coordination site for H2 activation and enhances selectivity control through the structural tuning of the N-heterocycle group. The catalysts exhibit exceptional chemo-, regio-, enantio- and Z/E-selectivities, along with broad functional group tolerance, enabling access to all possible semihydrogenation products of multisubstituted allenes. Mechanistic studies uncover a distinctive redox-neutral Co(I) catalytic cycle that facilitates heterolytic cleavage of H2, assisted by the basic N-heterocycle on the ligand. The hydrogenation of multisubstituted allenes presents a long-standing challenge due to the difficulty in simultaneously controlling multiple selectivity parameters. Now a class of pincer cobalt catalysts featuring multiple metal–ligand cooperation functionalities has been developed that enables the regio- and stereodivergent hydrogenation of allenes to provide all possible multisubstituted alkene products.},
  archive      = {J_NCHEM},
  author       = {Rong, Xianle and Ren, Yunjuan and Chen, Yongshun and Li, Chongyao and Wang, Jie and Chen, Zhaoqiang and Tao, Lizhi and Wei, Zhihong and Ma, Shengming and Liu, Qiang},
  doi          = {10.1038/s41557-025-01945-2},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1469-1479},
  shortjournal = {Nat. Chem.},
  title        = {Tunable cobalt-catalysed hydrogenation of allenes enabled by multiple metal–ligand cooperative functionalities},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dual dynamic helical poly(disulfide)s with conformational adaptivity and configurational recyclability. <em>NCHEM</em>, <em>17</em>(10), 1462-1468. (<a href='https://doi.org/10.1038/s41557-025-01947-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The structural dynamics of biopolymers, such as proteins and DNA, feature the unique combination of simultaneous conformational adaptivity and configurational reversibility. However, it remains challenging to design a synthetic system featuring dynamic covalent bonds with secondary structural folding and full recycling capabilities. Here we present a synthetic covalent polymer, constructed from biologically relevant units (amino acids and disulfides), that can reversibly adapt between disordered and helical conformations and can recycle itself to the original small molecules. This dual dynamic feature arises from the subtle synergy of noncovalent bonds (H bonds) and dynamic covalent bonds within a coupled chemical equilibrium, of which thermodynamics is found to follow a nonlinear van’t Hoff equation with nonzero heat capacity change. The covalent polymers show conformational dynamics between structured helices and disordered coils and the ability to convert back to the original monomers. This system demonstrates how dual dynamic function can control helical and recyclable polymer formation. Covalent polymers with helical conformations offer an adaptive scaffold for smart materials, but polymer-to-monomer deconstruction is inhibited by the covalent backbone. Now it has been shown that poly(disulfide)s can be folded into helices driven by side-chain hydrogen-bonding self-assembly, resulting in a synthetic helical polymer that can be fully recycled.},
  archive      = {J_NCHEM},
  author       = {Zhang, Qi and Nicu, Valentin P. and Buma, Wybren Jan and Tian, He and Qu, Da-Hui and Feringa, Ben L.},
  doi          = {10.1038/s41557-025-01947-0},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1462-1468},
  shortjournal = {Nat. Chem.},
  title        = {Dual dynamic helical poly(disulfide)s with conformational adaptivity and configurational recyclability},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding single-molecule reactions using nanopore-based techniques. <em>NCHEM</em>, <em>17</em>(10), 1450-1461. (<a href='https://doi.org/10.1038/s41557-025-01905-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The formation and cleavage of chemical bonds are two fundamental processes in chemistry, and the nature of these bonds determines the physical and chemical properties of a molecule. Real-time observation of chemical bonding at the single-molecule level offers insights into transient intermediates that are normally inaccessible via ensemble measurements. Protein nanopores, with their unique geometries, can be tailored into nanoreactors. Molecular bond-making and -cleavage at the reactive site of a protein nanopore’s interior wall can be visualized by monitoring ionic current changes. Therefore, nanopore-based techniques can enhance the understanding of complex binding kinetics and reaction mechanisms. In this Review we summarize recent advances in using biological nanopores as both single-molecule nanoreactors and single-molecule biosensors. The discussion covers the kinetics of single-molecule reactions under nanopore confinement, the strategies for designing biological nanopores and the latest progress in revealing reaction intermediates and pathways at the single-molecule level. Finally, we emphasize unresolved challenges and anticipate future developments in this rapidly evolving field. Real-time single-molecule observation provides insights into transient intermediates that are normally inaccessible using ensemble measurements. This Review discusses the use of nanopores as both single-molecule nanoreactors and single-molecule sensors for elucidating reaction pathways and kinetics, while highlighting their potential to control covalent reactions under confinement.},
  archive      = {J_NCHEM},
  author       = {Ying, Yi-Lun and Yang, Chao-Nan and Liu, Wei and Wu, Xue-Yuan and Long, Yi-Tao},
  doi          = {10.1038/s41557-025-01905-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1450-1461},
  shortjournal = {Nat. Chem.},
  title        = {Understanding single-molecule reactions using nanopore-based techniques},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Repurposing magnetic hyperthermia therapy to control biosynthesis in synthetic cells. <em>NCHEM</em>, <em>17</em>(10), 1448-1449. (<a href='https://doi.org/10.1038/s41557-025-01910-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The programmability of synthetic cells facilitates their application as drug delivery devices, but on-demand biosynthesis and release of cargo using a tissue-penetrating stimuli remains a challenge. Now, the heat generated from magnetic hyperthermia — a clinically approved anticancer therapy — is used to control the in situ synthesis and release of biomolecules from within synthetic cells.},
  archive      = {J_NCHEM},
  doi          = {10.1038/s41557-025-01910-z},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1448-1449},
  shortjournal = {Nat. Chem.},
  title        = {Repurposing magnetic hyperthermia therapy to control biosynthesis in synthetic cells},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-entropy nanomaterials by candlelight. <em>NCHEM</em>, <em>17</em>(10), 1445-1447. (<a href='https://doi.org/10.1038/s41557-025-01959-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-entropy materials offer an exciting realm of possibilities for material discovery and application; however, their nanoscale synthesis is limited by the need for specialized equipment. Now, a simple synthetic strategy employing wax candles enables production of these compositionally complex nanomaterials.},
  archive      = {J_NCHEM},
  author       = {Liu, Shuo and Dun, Chaochao and Swihart, Mark T.},
  doi          = {10.1038/s41557-025-01959-w},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1445-1447},
  shortjournal = {Nat. Chem.},
  title        = {High-entropy nanomaterials by candlelight},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-site catalyst cuts branched polyolefins to pieces. <em>NCHEM</em>, <em>17</em>(10), 1442-1444. (<a href='https://doi.org/10.1038/s41557-025-01927-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical upcycling of waste plastic is challenging because of its heterogeneous nature. Now, a newly designed single-site organo-nickel catalyst cuts through the mix by selectively snipping branched C–C linkages, allowing for the separation of polyolefin mixtures.},
  archive      = {J_NCHEM},
  author       = {Brody, Leo and Rorrer, Julie},
  doi          = {10.1038/s41557-025-01927-4},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1442-1444},
  shortjournal = {Nat. Chem.},
  title        = {Single-site catalyst cuts branched polyolefins to pieces},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanochemical upcycling technologies for sustainable circular fluorine economy. <em>NCHEM</em>, <em>17</em>(10), 1439-1441. (<a href='https://doi.org/10.1038/s41557-025-01956-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Per- and polyfluoroalkyl substances are widely utilized across various fields such as energy and electronics industries, but concerns have been raised about their environmental accumulation. Now, recent studies have revealed methods to decompose these compounds into reusable fluorine sources, enabling a circular economy.},
  archive      = {J_NCHEM},
  author       = {Matsunami, Asuka and Okazoe, Takashi},
  doi          = {10.1038/s41557-025-01956-z},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1439-1441},
  shortjournal = {Nat. Chem.},
  title        = {Mechanochemical upcycling technologies for sustainable circular fluorine economy},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cooperative ligands enable cobalt-catalysed hydrogenation. <em>NCHEM</em>, <em>17</em>(10), 1437-1438. (<a href='https://doi.org/10.1038/s41557-025-01944-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The partial hydrogenation of multi-substituted allenes is a highly useful reaction but impeded by difficult selectivity control. Now, pincer cobalt catalysts enable this transformation through metal–ligand cooperative interactions, achieving ligand-controlled selective allene semihydrogenation in a divergent manner.},
  archive      = {J_NCHEM},
  author       = {Zhong, Hongyu},
  doi          = {10.1038/s41557-025-01944-3},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1437-1438},
  shortjournal = {Nat. Chem.},
  title        = {Cooperative ligands enable cobalt-catalysed hydrogenation},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Opinionated science. <em>NCHEM</em>, <em>17</em>(10), 1435-1436. (<a href='https://doi.org/10.1038/s41557-025-01957-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When the facts can’t speak for themselves, scientists can give them a voice, argues Michelle Francl.},
  archive      = {J_NCHEM},
  author       = {Francl, Michelle},
  doi          = {10.1038/s41557-025-01957-y},
  journal      = {Nature Chemistry},
  month        = {10},
  number       = {10},
  pages        = {1435-1436},
  shortjournal = {Nat. Chem.},
  title        = {Opinionated science},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="ng">NG - 34</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes. <em>NG</em>, <em>57</em>(10), 2604. (<a href='https://doi.org/10.1038/s41588-025-02383-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {van der Laan, Camiel M. and Ip, Hill F. and Schipper, Marijn and Hottenga, Jouke-Jan and St Pourcain, Beate and Zayats, Tetyana and Pool, René and Krapohl, Eva M. L. and Brikell, Isabell and Soler Artigas, María and Cabana-Domínguez, Judit and Llonga, Natalia and Nolte, Ilja M. and Bolhuis, Koen and Palviainen, Teemu and Zafarmand, Hadi and Gordon, Scott and Aliev, Fazil and Burt, S. Alexandra and Wang, Carol A. and Saunders, Gretchen and Karhunen, Ville and Adkins, Daniel E. and Border, Richard and Peterson, Roseann E. and Prinz, Joseph A. and Thiering, Elisabeth and Vilor-Tejedor, Natàlia and Ahluwalia, Tarunveer S. and Allegrini, Andrea and Rimfeld, Kaili and Chen, Qi and Lu, Yi and Martin, Joanna and Bosch, Rosa and Ramos-Quiroga, Josep Antoni and Neumann, Alexander and Ensink, Judith and Grasby, Katrina L. and Morosoli, José J. and Tong, Xiaoran and Marrington, Shelby and Scott, James G. and Shabalin, Andrey A. and Corley, Robin and Evans, Luke M. and Sugden, Karen and Alemany, Silvia and Sass, Lærke and Vinding, Rebecca and Ehli, Erik A. and Hagenbeek, Fiona A. and Derks, Eske M. and Larsson, Henrik and Snieder, Harold and Cecil, Charlotte and Whipp, Alyce M. and Korhonen, Tellervo and Vuoksimaa, Eero and Rose, Richard J. and Uitterlinden, André G. and Haavik, Jan and Harris, Jennifer R. and Helgeland, Øyvind and Johansson, Stefan and Knudsen, Gun Peggy S. and Njolstad, Pal Rasmus and Lu, Qing and Rodriguez, Alina and Henders, Anjali K. and Mamun, Abdullah and Najman, Jackob M. and Brown, Sandy and Hopfer, Christian and Krauter, Kenneth and Reynolds, Chandra A. and Smolen, Andrew and Stallings, Michael and Wadsworth, Sally and Wall, Tamara L. and Eaves, Lindon and Silberg, Judy L. and Miller, Allison and Havdahl, Alexandra and Llop, Sabrina and Lopez-Espinosa, Maria-Jose and Bønnelykke, Klaus and Sunyer, Jordi and Arseneault, Louise and Standl, Marie and Heinrich, Joachim and Boden, Joseph and Pearson, John and Horwood, John and Kennedy, Martin and Poulton, Richie and Maes, Hermine H. and Hewitt, John and Copeland, William E. and Middeldorp, Christel M. and Williams, Gail M. and Wray, Naomi and Järvelin, Marjo-Riitta and McGue, Matt and Iacono, William and Caspi, Avshalom and Moffitt, Terrie E. and Whitehouse, Andrew J. O. and Pennell, Craig E. and Klump, Kelly L. and Jiang, Chang and Dick, Danielle M. and Reichborn-Kjennerud, Ted and Martin, Nicholas G. and Medland, Sarah E. and Vrijkotte, Tanja and Kaprio, Jaakko and Tiemeier, Henning and Davey Smith, George and Hartman, Catharina A. and Oldehinkel, Albertine J. and Casas, Miquel and Ribasés, Marta and Lichtenstein, Paul and Lundström, Sebastian and Plomin, Robert and Bartels, Meike and Nivard, Michel G. and Boomsma, Dorret I.},
  doi          = {10.1038/s41588-025-02383-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2604},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars. <em>NG</em>, <em>57</em>(10), 2603. (<a href='https://doi.org/10.1038/s41588-025-02356-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Yano, Ryoichi and Li, Feng and Hiraga, Susumu and Takeshima, Ryoma and Kobayashi, Michie and Toda, Kyoko and Umehara, Yosuke and Kajiya-Kanegae, Hiromi and Iwata, Hiroyoshi and Kaga, Akito and Ishimoto, Masao},
  doi          = {10.1038/s41588-025-02356-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2603},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Why genomic diversity should not be framed by census alone. <em>NG</em>, <em>57</em>(10), 2603. (<a href='https://doi.org/10.1038/s41588-025-02357-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Corpas, Manuel and Guio, Heinner and Lopez-Correa, Catalina and Fatumo, Segun},
  doi          = {10.1038/s41588-025-02357-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2603},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Why genomic diversity should not be framed by census alone},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nucleotide dependency analysis of genomic language models detects functional elements. <em>NG</em>, <em>57</em>(10), 2589-2602. (<a href='https://doi.org/10.1038/s41588-025-02347-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering how nucleotides in genomes encode regulatory instructions and molecular machines is a long-standing goal. Genomic language models (gLMs) implicitly capture functional elements and their organization from genomic sequences alone by modeling probabilities of each nucleotide given its sequence context. However, discovering functional genomic elements from gLMs has been challenging due to the lack of interpretable methods. Here we introduce nucleotide dependencies, which quantify how nucleotide substitutions at one genomic position affect the probabilities of nucleotides at other positions. We demonstrate that nucleotide dependencies are more effective at indicating the deleteriousness of genetic variants than alignment-based conservation and gLM reconstruction. Dependency analysis accurately detects regulatory motifs and highlights bases in contact within RNAs, including pseudoknots and tertiary structure contacts, revealing new, experimentally validated RNA structures. Finally, we leverage dependency maps to reveal critical limitations of several gLM architectures and training strategies. Altogether, nucleotide dependency analysis opens a new avenue for discovering and studying functional elements and their interactions in genomes. Mapping pairwise nucleotide dependencies by leveraging genomic language models highlights functional genomic elements and predicts deleterious genetic variants more effectively than alignment-based conservation metrics.},
  archive      = {J_NG},
  author       = {Tomaz da Silva, Pedro and Karollus, Alexander and Hingerl, Johannes and Galindez, Gihanna Sta. Teresa and Wagner, Nils and Hernandez-Alias, Xavier and Incarnato, Danny and Gagneur, Julien},
  doi          = {10.1038/s41588-025-02347-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2589-2602},
  shortjournal = {Nat. Genet.},
  title        = {Nucleotide dependency analysis of genomic language models detects functional elements},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variants affecting RNA stability influence complex traits and disease risk. <em>NG</em>, <em>57</em>(10), 2578-2588. (<a href='https://doi.org/10.1038/s41588-025-02326-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression is modulated jointly by transcriptional regulation and messenger RNA stability, yet the latter is often overlooked in studies on genetic variants. Here, leveraging metabolic labeling data (Bru/BruChase-seq) and a new computational pipeline, RNAtracker, we categorize genes as allele-specific RNA stability (asRS) or allele-specific RNA transcription events. We identify more than 5,000 asRS variants among 665 genes across a panel of 11 human cell lines. These variants directly overlap conserved microRNA target regions and allele-specific RNA-binding protein sites, illuminating mechanisms through which stability is mediated. Furthermore, we identified causal asRS variants using a massively parallel screen (MapUTR) for variants that affect post-transcriptional mRNA abundance, as well as through CRISPR prime editing approaches. Notably, asRS genes were enriched significantly among a multitude of immune-related pathways and contribute to the risk of several immune system diseases. This work highlights RNA stability as a critical, yet understudied mechanism linking genetic variation and disease. RNAtracker is a computational pipeline that distinguishes variants associated with allele-specific RNA stability from those associated with allele-specific RNA transcription. Variants affecting RNA stability are enriched in immune-related genes and contribute to disease risk.},
  archive      = {J_NG},
  author       = {Huang, Elaine and Fu, Ting and Zhang, Ling and Yan, Guanao and Yamamoto, Ryo and Terrazas, Sari and Nguyen, Thuy Linh and Gonzalez-Figueroa, Carlos and Khanbabaei, Armen and Bahn, Jae Hoon and Varada, Rajagopal and Amoah, Kofi and Hervoso, Jonatan and Paulsen, Michelle T. and Magnuson, Brian and Ljungman, Mats and Li, Jingyi Jessica and Xiao, Xinshu},
  doi          = {10.1038/s41588-025-02326-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2578-2588},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variants affecting RNA stability influence complex traits and disease risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerated bayesian inference of population size history from recombining sequence data. <em>NG</em>, <em>57</em>(10), 2570-2577. (<a href='https://doi.org/10.1038/s41588-025-02323-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study introduces population history learning by averaging sampled histories (PHLASH), a new method for inferring population history from whole-genome sequence data. It works by drawing random, low-dimensional projections of the coalescent intensity function from the posterior distribution of a pairwise sequentially Markovian coalescent-like model and averaging them together to form an accurate and adaptive estimator. On simulated data, PHLASH tends to be faster and have lower error than several competing methods, including SMC++, MSMC2 and FITCOAL. Moreover, it provides automatic uncertainty quantification and leads to new Bayesian testing procedures for detecting population structure and ancient bottlenecks. The key technical advance is a new algorithm for computing the score function (gradient of the log likelihood) of a coalescent hidden Markov model, which has the same computational cost as evaluating the log likelihood. PHLASH has been released as an easy-to-use Python software package and leverages graphics processing unit acceleration when available. Population history learning by averaging sampled histories is a new Bayesian method for estimating historical effective population size from recombining sequence data that offers improved speed and accuracy and enables uncertainty quantification and testing for ancient bottlenecks.},
  archive      = {J_NG},
  author       = {Terhorst, Jonathan},
  doi          = {10.1038/s41588-025-02323-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2570-2577},
  shortjournal = {Nat. Genet.},
  title        = {Accelerated bayesian inference of population size history from recombining sequence data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bryophytes hold a larger gene family space than vascular plants. <em>NG</em>, <em>57</em>(10), 2562-2569. (<a href='https://doi.org/10.1038/s41588-025-02325-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After 500 million years of evolution, extant land plants compose the following two sister groups: the bryophytes and the vascular plants. Despite their small size and simple structure, bryophytes thrive in a wide variety of habitats, including extreme conditions. However, the genetic basis for their ecological adaptability and long-term survival is not well understood. A comprehensive super-pangenome analysis, incorporating 123 newly sequenced bryophyte genomes, reveals that bryophytes possess a substantially greater diversity of gene families than vascular plants. This includes a higher number of unique and lineage-specific gene families, originating from extensive new gene formation and continuous horizontal transfer of microbial genes over their long evolutionary history. The evolution of bryophytes’ rich and diverse genetic toolkit, which includes new physiological innovations like unique immune receptors, likely facilitated their spread across different biomes. These newly sequenced bryophyte genomes offer a valuable resource for exploring alternative evolutionary strategies for terrestrial success. A super-pangenome analysis incorporating 123 newly sequenced bryophyte genomes reveals that bryophytes exhibit a larger number of unique and lineage-specific gene families than vascular plants.},
  archive      = {J_NG},
  author       = {Dong, Shanshan and Wang, Sibo and Li, Linzhou and Yu, Jin and Zhang, Yongxia and Xue, Jia-Yu and Chen, Hengchi and Ma, Jianchao and Zeng, Yuying and Cai, Yuqing and Huang, Wei and Zhou, Xuping and Wu, Jiayi and Li, Jianyou and Yao, Yifeng and Hu, Ruoyang and Zhao, Tao and Villarreal A, Juan Carlos and Dirick, Leon and Liu, Li and Ignatov, Michael and Jin, Minghui and Ruan, Jue and He, Yikun and Wang, Haifeng and Xu, Bo and Rozzi, Ricardo and Wegrzyn, Jill and Stevenson, Dennis William and Renzaglia, Karen S. and Chen, Hongfeng and Zhang, Li and Zhang, Shouzhou and Mackenzie, Roy and Moreno, Javier E. and Melkonian, Michael and Wei, Tong and Gu, Ying and Xu, Xun and Rensing, Stefan A. and Huang, Jinling and Long, Manyuan and Goffinet, Bernard and Bowman, John L. and Van de Peer, Yves and Liu, Huan and Liu, Yang},
  doi          = {10.1038/s41588-025-02325-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2562-2569},
  shortjournal = {Nat. Genet.},
  title        = {Bryophytes hold a larger gene family space than vascular plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multi-tissue single-cell expression atlas in cattle. <em>NG</em>, <em>57</em>(10), 2546-2561. (<a href='https://doi.org/10.1038/s41588-025-02329-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Systematic characterization of the molecular states of cells in livestock tissues is essential for understanding the cellular and genetic mechanisms underlying economically and ecologically important physiological traits. Here, as part of the Farm Animal Genotype-Tissue Expression (FarmGTEx) project, we describe a comprehensive reference map including 1,793,854 cells from 59 bovine tissues in calves and adult cattle, spanning both sexes, which reveals intra-tissue and inter-tissue cellular heterogeneity in gene expression, transcription factor regulation and intercellular communication. Integrative analysis with genetic variants that underpin bovine monogenic and complex traits uncovers cell types of relevance, such as spermatocytes, responsible for sperm motility and excitatory neurons for milk fat yield. Comparative analysis reveals similarities in gene expression between cattle and humans, allowing for the detection of relevant cell types to study human complex phenotypes. This Cattle Cell Atlas will serve as a key resource for cattle genetics and genomics, selective breeding and comparative biology. This study introduces the Cattle Cell Atlas, a single-cell expression resource including 1,793,854 cells from 59 tissues. Integrative analyses leveraging this atlas provide insights into the biology underlying bovine monogenic and complex traits.},
  archive      = {J_NG},
  author       = {Han, Bo and Li, Houcheng and Zheng, Weijie and Zhang, Qi and Chen, Ao and Zhu, Senlin and Shi, Tao and Wang, Fei and Zou, Dong and Song, Yu and Ye, Wen and Du, Aixia and Fu, Yihan and Jia, Minghui and Bai, Zhonghao and Yuan, Zhixiang and Liu, Wansheng and Tuo, Wenbin and Hope, Jayne C. and MacHugh, David E. and O’Grady, John F. and Madsen, Ole and Sahana, Goutam and Luo, Yonglun and Lin, Lin and Li, Congjun and Cai, Zexi and Li, Bingjie and Huang, Jinming and Liu, Lin and Zhang, Zhang and Ma, Zhu and Hou, Yali and Liu, George E. and Jiang, Yu and Sun, Hui-zeng and Fang, Lingzhao and Sun, Dongxiao},
  doi          = {10.1038/s41588-025-02329-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2546-2561},
  shortjournal = {Nat. Genet.},
  title        = {A multi-tissue single-cell expression atlas in cattle},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic and epigenetic screens in primary human t cells link candidate causal autoimmune variants to t cell networks. <em>NG</em>, <em>57</em>(10), 2536-2545. (<a href='https://doi.org/10.1038/s41588-025-02301-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variants associated with autoimmune diseases are highly enriched within putative cis-regulatory regions of CD4+ T cells, suggesting that they could alter disease risk through changes in gene regulation. However, very few genetic variants have been shown to affect T cell gene expression or function. Here we tested &gt;18,000 autoimmune disease-associated variants for allele-specific effects on expression using massively parallel reporter assays in primary human CD4+ T cells. We find 545 variants that modulate expression in an allele-specific manner (emVars). Primary T cell emVars greatly enrich for likely causal variants, are mediated by common upstream pathways and their putative target genes are highly enriched within a lymphocyte activation network. Using bulk and single-cell CRISPR-interference screens, we confirm that emVar-containing T cell cis-regulatory elements modulate both known and previously unappreciated target genes that regulate T cell proliferation, providing plausible mechanisms by which these variants alter autoimmune disease risk. Massively parallel reporter assay in primary human CD4+ T cells and bulk and single-cell CRISPR-interference screens identify candidate causal variants linked to autoimmune disease risk that modulate T cell gene expression and proliferation.},
  archive      = {J_NG},
  author       = {Ho, Ching-Huang and Dippel, Maxwell A. and McQuade, Meghan S. and Nguyen, LeAnn P. and Mishra, Arpit and Pribitzer, Stephan and Hardy, Samantha and Chandok, Harshpreet and Chardon, Florence M. and McDiarmid, Troy A. and DeBerg, Hannah A. and Buckner, Jane H. and Shendure, Jay and de Boer, Carl G. and Guo, Michael H. and Tewhey, Ryan and Ray, John P.},
  doi          = {10.1038/s41588-025-02301-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2536-2545},
  shortjournal = {Nat. Genet.},
  title        = {Genetic and epigenetic screens in primary human t cells link candidate causal autoimmune variants to t cell networks},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq. <em>NG</em>, <em>57</em>(10), 2522-2535. (<a href='https://doi.org/10.1038/s41588-025-02343-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies. This study introduces single-cell transcription factor (TF) sequencing, a single-cell barcoded and doxycycline-inducible TF overexpression approach that reveals dose-sensitive functional classes of TFs and cellular heterogeneity by mapping TF dose-dependent transcriptomic changes during the reprogramming of mouse embryonic multipotent stromal cells.},
  archive      = {J_NG},
  author       = {Liu, Wangjie and Saelens, Wouter and Rainer, Pernille and Biočanin, Marjan and Gardeux, Vincent and Gralak, Antoni Jakub and van Mierlo, Guido and Gebhart, Angelika and Russeil, Julie and Liu, Tingdang and Chen, Wanze and Deplancke, Bart},
  doi          = {10.1038/s41588-025-02343-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2522-2535},
  shortjournal = {Nat. Genet.},
  title        = {Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation influences human centromere positioning and function. <em>NG</em>, <em>57</em>(10), 2509-2521. (<a href='https://doi.org/10.1038/s41588-025-02324-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maintaining the epigenetic identity of centromeres is essential to prevent genome instability. Centromeres are epigenetically defined by the histone H3 variant CENP-A. Prior work in human centromeres has shown that CENP-A is associated with regions of hypomethylated DNA located within large arrays of hypermethylated repeats, but the functional importance of these DNA methylation (DNAme) patterns remains poorly understood. To address this, we developed tools to perturb centromeric DNAme, revealing that it causally influences CENP-A positioning. We show that rapid loss of methylation results in increased binding of centromeric proteins and alterations in centromere architecture, leading to aneuploidy and reduced cell viability. We also demonstrate that gradual centromeric DNA demethylation prompts a process of cellular adaptation. Altogether, we find that DNAme causally influences CENP-A localization and centromere function, offering mechanistic insights into pathological alterations of centromeric DNAme. Genome-wide and targeted perturbation of DNA methylation at centromeres affects CENP-A positioning and centromere structure, resulting in aneuploidy and reduced cell viability.},
  archive      = {J_NG},
  author       = {Salinas-Luypaert, Catalina and Dubocanin, Danilo and Lee, Rosa Jooyoung and Andrade Ruiz, Lorena and Gamba, Riccardo and Grison, Marine and Velikovsky, Leonid and Angrisani, Annapaola and Scelfo, Andrea and Xu, Yuan and Dumont, Marie and Barra, Viviana and Wilhelm, Therese and Velasco, Guillaume and Losito, Marialucrezia and Wardenaar, René and Francastel, Claire and Foijer, Floris and Kops, Geert J. P. L. and Miga, Karen H. and Altemose, Nicolas and Fachinetti, Daniele},
  doi          = {10.1038/s41588-025-02324-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2509-2521},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation influences human centromere positioning and function},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell multi-omic and spatial profiling of esophageal squamous cell carcinoma reveals the immunosuppressive role of GPR116+ pericytes in cancer metastasis. <em>NG</em>, <em>57</em>(10), 2494-2508. (<a href='https://doi.org/10.1038/s41588-025-02341-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor metastasis leads to most cancer deaths. However, how cellular diversity and dynamic cooperation within the tumor microenvironment contribute to metastasis remains poorly understood. Here we leverage single-cell multi-omics (16 samples, 117,169 cells) and spatial transcriptomics (five samples, 195,366 cells) to uncover the cellular and spatial architecture of esophageal squamous cell carcinoma (ESCC), and characterize an immunosuppressive GPR116+ pericyte subset promoting tumor metastasis and immunotherapy resistance. GPR116+ pericyte enrichment is transcriptionally regulated by PRRX1, evidenced by pericyte-specific Prrx1 knockout mice. Mechanistically, GPR116+ pericytes secrete EGFL6 to bind integrin β1 on cancer cells, activating the NF-κB pathway to facilitate metastasis. Serum EGFL6 serves as a noninvasive biomarker for the diagnosis and prognosis of several tumors. Blocking integrin β1 suppresses metastasis and improves immunotherapy response in animal models of ESCC. Collectively, we provide a spatially resolved landscape of the prometastatic tumor microenvironment in ESCC and highlight the biological and clinical importance of GPR116+ pericytes, proposing potential innovative therapeutic strategies for metastatic cancers. This study provides a single-cell and spatial atlas of the prometastatic tumor microenvironment in esophageal squamous cell carcinoma and characterizes the immunosuppressive function of GPR116+ pericytes in cancer metastasis and immune evasion.},
  archive      = {J_NG},
  author       = {Pei, Xiaoya and Liu, Zhichao and Tang, Lin and Zhang, Junpeng and He, Yan and Zhuang, Xiaowan and Song, Yuxiang and Peng, Shaocong and Zhao, Yahui and Huang, Xuanzhang and Xu, Yiwei and Li, Junyi and Peng, Liang and Shi, Xingyuan and Zhang, Fan and Liu, Chong and He, Mingliang and Dai, Wei and Wu, Liang and Law, Simon and Liu, Jinbao and Wang, Zhenning and Li, Zhigang and Xu, Wenwen and Liu, Zhihua and Li, Bin},
  doi          = {10.1038/s41588-025-02341-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2494-2508},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-omic and spatial profiling of esophageal squamous cell carcinoma reveals the immunosuppressive role of GPR116+ pericytes in cancer metastasis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial signatures for predicting immunotherapy outcomes using multi-omics in non-small cell lung cancer. <em>NG</em>, <em>57</em>(10), 2482-2493. (<a href='https://doi.org/10.1038/s41588-025-02351-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Non-small cell lung cancer (NSCLC) shows variable responses to immunotherapy, highlighting the need for biomarkers to guide patient selection. We applied a spatial multi-omics approach to 234 advanced NSCLC patients treated with programmed death 1-based immunotherapy across three cohorts to identify biomarkers associated with outcome. Spatial proteomics (n = 67) and spatial compartment-based transcriptomics (n = 131) enabled profiling of the tumor immune microenvironment (TIME). Using spatial proteomics, we identified a resistance cell-type signature including proliferating tumor cells, granulocytes, vessels (hazard ratio (HR) = 3.8, P = 0.004) and a response signature, including M1/M2 macrophages and CD4 T cells (HR = 0.4, P = 0.019). We then generated a cell-to-gene resistance signature using spatial transcriptomics, which was predictive of poor outcomes (HR = 5.3, 2.2, 1.7 across Yale, University of Queensland and University of Athens cohorts), while a cell-to-gene response signature predicted favorable outcomes (HR = 0.22, 0.38 and 0.56, respectively). This framework enables robust TIME modeling and identifies biomarkers to support precision immunotherapy in NSCLC. This study uses spatial proteomic and spatial compartment-based whole-transcriptome profiling to develop predictive models for immunotherapy outcomes in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Aung, Thazin N. and Monkman, James and Warrell, Jonathan and Vathiotis, Ioannis and Bates, Katherine M. and Gavrielatou, Niki and Trontzas, Ioannis P. and Tan, Chin Wee and Fernandez, Aileen I. and Moutafi, Myrto and O’ Byrne, Ken and Schalper, Kurt A. and Syrigos, Konstantinos and Herbst, Roy S. and Kulasinghe, Arutha and Rimm, David L.},
  doi          = {10.1038/s41588-025-02351-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2482-2493},
  shortjournal = {Nat. Genet.},
  title        = {Spatial signatures for predicting immunotherapy outcomes using multi-omics in non-small cell lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Targeting histone H2B acetylated enhanceosomes via p300/CBP degradation in prostate cancer. <em>NG</em>, <em>57</em>(10), 2468-2481. (<a href='https://doi.org/10.1038/s41588-025-02336-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prostate cancer is driven by oncogenic transcription factor enhanceosomes comprising chromatin and epigenetic regulators. The lysine acetyltransferases p300 and CREB-binding protein (CBP) are key cofactors that activate enhancers through histone acetylation. Here we identify p300/CBP-mediated multisite histone H2B N-terminal acetylation (H2BNTac) as a defining feature of oncogenic enhanceosomes in androgen receptor (AR)-positive prostate cancer. p300/CBP are essential for AR and ETS transcription factor ERG transcriptional activity, and their dual degradation eliminates H2BNTac and histone H3 lysine 27 acetylation at hyperactive enhancers, leading to stronger suppression of oncogenic transcription than targeting either paralog or bromodomain alone. Cytotoxicity profiling across &gt;900 cell lines revealed that tumors with high H2BNTac, including AR-positive prostate cancer, are selectively dependent on p300/CBP. In preclinical models, systemic p300/CBP degradation inhibited tumor growth, synergized with AR antagonists and showed no evident toxicity. These findings position H2BNTac as an epigenetic marker of enhancer addiction and establish dual p300/CBP degradation as a promising therapeutic strategy for enhancer-driven cancers. This study shows that p300/CBP-dependent H2B acetylation is crucial for maintaining transcription of oncogenic gene programs in androgen receptor-driven prostate cancer.},
  archive      = {J_NG},
  author       = {Luo, Jie and Chen, Zhixiang and Qiao, Yuanyuan and Tien, Jean Ching-Yi and Young, Eleanor and Mannan, Rahul and Mahapatra, Somnath and Bhattacharyya, Rupam and Xiao, Lanbo and He, Tongchen and Eyunni, Sanjana and Zhang, Yuping and Zheng, Yang and Su, Fengyun and Cao, Xuhong and Wang, Rui and Cheng, Yunhui and Seri, Rithvik and George, James and Shahine, Miriam and Miner, Stephanie J. and Rees, Matthew G. and Ronan, Melissa M. and Roth, Jennifer A. and Vaishampayan, Ulka and Wang, Mi and Wang, Shaomeng and Parolia, Abhijit and Chinnaiyan, Arul M.},
  doi          = {10.1038/s41588-025-02336-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2468-2481},
  shortjournal = {Nat. Genet.},
  title        = {Targeting histone H2B acetylated enhanceosomes via p300/CBP degradation in prostate cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid epigenomic classification of acute leukemia. <em>NG</em>, <em>57</em>(10), 2456-2467. (<a href='https://doi.org/10.1038/s41588-025-02321-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acute leukemia requires precise molecular classification and urgent treatment. However, standard-of-care diagnostic tests are time-intensive and do not capture the full spectrum of acute leukemia heterogeneity. Here, we developed a framework to classify acute leukemia using genome-wide DNA methylation profiling. We first assembled a comprehensive reference cohort (n = 2,540 samples) and defined 38 methylation classes. Methylation-based classification matched standard-pathology lineage classification in most cases and revealed heterogeneity in addition to that captured by genetic categories. Using this reference, we developed a neural network (MARLIN; methylation- and AI-guided rapid leukemia subtype inference) for acute leukemia classification from sparse DNA methylation profiles. In retrospective cohorts profiled by nanopore sequencing, high-confidence predictions were concordant with conventional diagnoses in 25 out of 26 cases. Real-time MARLIN classification in patients with suspected acute leukemia provided accurate predictions in five out of five cases, which were typically generated within 2 h of sample receipt. In summary, we present a framework for rapid acute leukemia classification that complements and enhances standard-of-care diagnostics. The authors present a molecular classification of acute leukemia using 5-methylcytosine signatures, together with a neural network-based classifier for clinical use.},
  archive      = {J_NG},
  author       = {Steinicke, Til L. and Benfatto, Salvatore and Capilla-Guerra, Maria R. and Monteleone, Andre B. and Young, Jonathan H. and Shankar, Subha and Michaels, Phillip D. and Tsai, Harrison K. and Good, Jonathan D. and Kreso, Antonia and van Galen, Peter and Schliemann, Christoph and Chen, Evan C. and Griffin, Gabriel K. and Hovestadt, Volker},
  doi          = {10.1038/s41588-025-02321-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2456-2467},
  shortjournal = {Nat. Genet.},
  title        = {Rapid epigenomic classification of acute leukemia},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A genetic map of human metabolism across the allele frequency spectrum. <em>NG</em>, <em>57</em>(10), 2445-2455. (<a href='https://doi.org/10.1038/s41588-025-02355-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies of human metabolism have been limited in scale and allelic breadth. Here we provide a data-driven map of the genetic regulation of circulating small molecules and lipoprotein characteristics (249 traits) measured using proton nuclear magnetic resonance spectroscopy across the allele frequency spectrum in ~450,000 individuals. Trans-ancestral meta-analyses identify 29,824 locus–metabolite associations mapping to 753 regions with effects largely consistent between men and women and large ancestral groups represented in UK Biobank. We observe and classify extreme genetic pleiotropy, identify regulators of lipid metabolism, and assign effector genes at &gt;100 loci through rare-to-common allelic series. We propose roles for genes less established in metabolic control (for example, SIDT2), genes characterized by phenotypic heterogeneity (for example, APOA1) and genes with specific disease relevance (for example, VEGFA). Our study demonstrates the value of broad, large-scale metabolomic phenotyping to identify and characterize regulators of human metabolism. Genome-wide association analyses of 249 circulating small molecules and lipoprotein characteristics across the allele frequency spectrum in UK Biobank advance understanding of genes and pathways regulating human metabolism.},
  archive      = {J_NG},
  author       = {Zoodsma, Martijn and Beuchel, Carl and Yasmeen, Summaira and Kohleick, Leonhard and Nepal, Aakash and Koprulu, Mine and Kronenberg, Florian and Mayr, Manuel and Williamson, Alice and Pietzner, Maik and Langenberg, Claudia},
  doi          = {10.1038/s41588-025-02355-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2445-2455},
  shortjournal = {Nat. Genet.},
  title        = {A genetic map of human metabolism across the allele frequency spectrum},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Population-scale gene-based analysis of whole-genome sequencing provides insights into metabolic health. <em>NG</em>, <em>57</em>(10), 2436-2444. (<a href='https://doi.org/10.1038/s41588-025-02364-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In addition to its coverage of the noncoding genome, whole-genome sequencing (WGS) may better capture the coding genome than exome sequencing. Here we sought to exploit this and identify new rare, protein-coding variants associated with metabolic health in WGS data (n = 708,956) from the UK Biobank and All of Us studies. Identified genes highlight new biological mechanisms, including protein-truncating variants (PTVs) in the DNA double-strand break repair gene RIF1 that have a substantial effect on body mass index (2.66 kg m−2, s.e. 0.43, P = 3.7 × 10−10). UBR3 is an intriguing example where PTVs independently increase body mass index and type 2 diabetes risk. Furthermore, PTVs in IRS2 have a substantial effect on type 2 diabetes (odds ratio 6.4 (3.7–11.3), P = 9.9 × 10−14, 34% case prevalence among carriers) and were also associated with chronic kidney disease independent of diabetes status, suggesting an important role for IRS2 in maintaining renal health. Our study demonstrates that large-scale WGS provides new mechanistic insights into human metabolic phenotypes through improved capture of coding sequences. Analyses of whole-genome sequencing data from UK Biobank and All of Us identify rare variant burden signals associated with metabolic health, including effects of protein-truncating variants in IRS2 on type 2 diabetes and chronic kidney disease risk.},
  archive      = {J_NG},
  author       = {Zhao, Yajie and Lockhart, Sam and Liu, Jimmy and Li, Xihao and Cortes, Adrian and Hua, Xing and Gardner, Eugene J. and Kentistou, Katherine A. and Cañadas-Garre, Marisa and Fabian, Laurie and Ho, Karen and Timpson, Nicholas and Lo, Yancy and Davitte, Jonathan and Savage, David B. and Buser-Doepner, Carolyn and Ong, Ken K. and Zhang, Haoyu and Scott, Robert and O’Rahilly, Stephen and Perry, John R. B.},
  doi          = {10.1038/s41588-025-02364-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2436-2444},
  shortjournal = {Nat. Genet.},
  title        = {Population-scale gene-based analysis of whole-genome sequencing provides insights into metabolic health},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes. <em>NG</em>, <em>57</em>(10), 2427-2435. (<a href='https://doi.org/10.1038/s41588-025-02295-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We performed a genome-wide association meta-analysis (GWAMA) of 290,134 attention-deficit/hyperactivity disorder (ADHD) symptom measures of 70,953 unique individuals from multiple raters, ages and instruments (ADHDSYMP). Next, we meta-analyzed the results with a study of ADHD diagnosis (ADHDOVERALL). ADHDSYMP returned no genome-wide significant variants. We show that the combined ADHDOVERALL GWAMA identified 39 independent loci, of which 17 were new. Using a recently developed gene-mapping method, Fine-mapped Locus Assessment Model of Effector genes, we identified 22 potential ADHD effector genes implicating several new biological processes and pathways. Moderate negative genetic correlations (rg &lt; −0.40) were observed with multiple cognitive traits. In three cohorts, polygenic scores (PGSs) based on ADHDOVERALL outperformed PGSs based on ADHD symptoms and diagnosis alone. Our findings support the notion that clinical ADHD is at the extreme end of a continuous liability that is indexed by ADHD symptoms. We show that including ADHD symptom counts helps to identify new genes implicated in ADHD. Genome-wide association meta-analyses of attention-deficit/hyperactivity disorder symptom measures and clinical diagnoses identify new risk loci, highlight putative effector genes and yield improved polygenic risk scores.},
  archive      = {J_NG},
  author       = {van der Laan, Camiel M. and Ip, Hill F. and Schipper, Marijn and Hottenga, Jouke-Jan and St Pourcain, Beate and Zayats, Tetyana and Pool, René and Krapohl, Eva M. L. and Brikell, Isabell and Soler Artigas, María and Cabana-Domínguez, Judit and Llonga, Natalia and Nolte, Ilja M. and Bolhuis, Koen and Palviainen, Teemu and Zafarmand, Hadi and Gordon, Scott and Aliev, Fazil and Burt, S. Alexandra and Wang, Carol A. and Saunders, Gretchen and Karhunen, Ville and Adkins, Daniel E. and Border, Richard and Peterson, Roseann E. and Prinz, Joseph A. and Thiering, Elisabeth and Vilor-Tejedor, Natàlia and Ahluwalia, Tarunveer S. and Allegrini, Andrea and Rimfeld, Kaili and Chen, Qi and Lu, Yi and Martin, Joanna and Bosch, Rosa and Ramos-Quiroga, Josep Antoni and Neumann, Alexander and Ensink, Judith and Grasby, Katrina L. and Morosoli, José J. and Tong, Xiaoran and Marrington, Shelby and Scott, James G. and Shabalin, Andrey A. and Corley, Robin and Evans, Luke M. and Sugden, Karen and Alemany, Silvia and Sass, Lærke and Vinding, Rebecca and Ehli, Erik A. and Hagenbeek, Fiona A. and Derks, Eske M. and Larsson, Henrik and Snieder, Harold and Cecil, Charlotte and Whipp, Alyce M. and Korhonen, Tellervo and Vuoksimaa, Eero and Rose, Richard J. and Uitterlinden, André G. and Haavik, Jan and Harris, Jennifer R. and Helgeland, Øyvind and Johansson, Stefan and Knudsen, Gun Peggy S. and Njolstad, Pal Rasmus and Lu, Qing and Rodriguez, Alina and Henders, Anjali K. and Mamun, Abdullah and Najman, Jackob M. and Brown, Sandy and Hopfer, Christian and Krauter, Kenneth and Reynolds, Chandra A. and Smolen, Andrew and Stallings, Michael and Wadsworth, Sally and Wall, Tamara L. and Eaves, Lindon and Silberg, Judy L. and Miller, Allison and Havdahl, Alexandra and Llop, Sabrina and Lopez-Espinosa, Maria-Jose and Bønnelykke, Klaus and Sunyer, Jordi and Arseneault, Louise and Standl, Marie and Heinrich, Joachim and Boden, Joseph and Pearson, John and Horwood, John and Kennedy, Martin and Poulton, Richie and Maes, Hermine H. and Hewitt, John and Copeland, William E. and Middeldorp, Christel M. and Williams, Gail M. and Wray, Naomi and Järvelin, Marjo-Riitta and McGue, Matt and Iacono, William and Caspi, Avshalom and Moffitt, Terrie E. and Whitehouse, Andrew J. O. and Pennell, Craig E. and Klump, Kelly L. and Jiang, Chang and Dick, Danielle M. and Reichborn-Kjennerud, Ted and Martin, Nicholas G. and Medland, Sarah E. and Vrijkotte, Tanja and Kaprio, Jaakko and Tiemeier, Henning and Davey Smith, George and Hartman, Catharina A. and Oldehinkel, Albertine J. and Casas, Miquel and Ribasés, Marta and Lichtenstein, Paul and Lundström, Sebastian and Plomin, Robert and Bartels, Meike and Nivard, Michel G. and Boomsma, Dorret I.},
  doi          = {10.1038/s41588-025-02295-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2427-2435},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Limited overlap between genetic effects on disease susceptibility and disease survival. <em>NG</em>, <em>57</em>(10), 2418-2426. (<a href='https://doi.org/10.1038/s41588-025-02342-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding disease progression is of high biological and clinical interest. Unlike disease susceptibility, whose genetic basis has been abundantly studied, less is known about the genetics of disease progression and its overlap with disease susceptibility. Considering nine common diseases (ncases ranging from 11,980 to 124,682) across seven biobanks, we systematically compared genetic architectures of susceptibility and progression, defined as disease-specific mortality. We identified only one locus substantially associated with disease-specific mortality and showed that, at a similar sample size, more genome-wide significant loci can be identified in a genome-wide association study of disease susceptibility. Variants substantially affecting disease susceptibility were weakly or not associated with disease-specific mortality. Moreover, susceptibility polygenic scores (PGSs) were weak predictors of disease-specific mortality, while a PGS for general lifespan was substantially associated with disease-specific mortality for seven of nine diseases. We explored alternative definitions of disease progression and found that genetic signals for macrovascular complications in type 2 diabetes overlap with similar phenotypes in the general population; however, these effects are attenuated. Overall, our findings indicate limited similarity in genetic effects between disease susceptibility and disease-specific mortality, suggesting that larger sample sizes, different measures of progression, or the integration of related phenotypes from the general population is needed to identify the genetic underpinnings of disease progression. Systematic comparison of genome-wide association results for disease risk and disease-specific mortality for nine common diseases across seven biobanks finds limited overlap between genetic effects on disease susceptibility and survival.},
  archive      = {J_NG},
  author       = {Yang, Zhiyu and Pajuste, Fanny-Dhelia and Zguro, Kristina and Cheng, Yipeng and Kurant, Danielle E. and Eoli, Andrea and Wanner, Julian and Jermy, Bradley and Rämö, Joel and Kanoni, Stavroula and van Heel, David A. and Hayward, Caroline and Marioni, Riccardo E. and McCartney, Daniel L. and Renieri, Alessandra and Furini, Simone and Mägi, Reedik and Gusev, Alexander and Drineas, Petros and Paschou, Peristera and Heyne, Henrike and Ripatti, Samuli and Mars, Nina and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02342-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2418-2426},
  shortjournal = {Nat. Genet.},
  title        = {Limited overlap between genetic effects on disease susceptibility and disease survival},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pan-UK biobank genome-wide association analyses enhance discovery and resolution of ancestry-enriched effects. <em>NG</em>, <em>57</em>(10), 2408-2417. (<a href='https://doi.org/10.1038/s41588-025-02335-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks, such as the UK Biobank (UKB), enable massive phenome by genome-wide association studies that elucidate genetic etiology of complex traits. However, people from diverse genetic ancestry groups are often excluded from association analyses due to concerns about population structure introducing false positive associations. Here we generate mixed model associations and meta-analyses across genetic ancestry groups, inclusive of a larger fraction of the UK Biobank than previous efforts, to produce freely available summary statistics for 7,266 traits. We build a quality control and analysis framework informed by genetic architecture. Overall, we identify 14,676 significant loci (P &lt; 5 × 10−8) in the meta-analysis that were not found in the EUR genetic ancestry group alone, including new associations, for example between CAMK2D and triglycerides. We also highlight associations from ancestry-enriched variation, including a known pleiotropic missense variant in G6PD associated with several biomarker traits. We release these results publicly alongside frequently asked questions that describe caveats for interpretation of results, enhancing available resources for interpretation of risk variants across diverse populations. Genome-wide analyses for 7,266 traits leveraging data from several genetic ancestry groups in UK Biobank identify new associations and enhance resources for interpreting risk variants across diverse populations.},
  archive      = {J_NG},
  author       = {Karczewski, Konrad J. and Gupta, Rahul and Kanai, Masahiro and Lu, Wenhan and Tsuo, Kristin and Wang, Ying and Walters, Raymond K. and Turley, Patrick and Callier, Shawneequa and Shah, Nirav N. and Baya, Nikolas and Palmer, Duncan S. and Goldstein, Jacqueline I. and Sarma, Gopal and Solomonson, Matthew and Cheng, Nathan and Bryant, Sam and Churchhouse, Claire and Cusick, Caroline M. and Poterba, Timothy and Compitello, John and King, Daniel and Zhou, Wei and Seed, Cotton and Finucane, Hilary K. and Daly, Mark J. and Neale, Benjamin M. and Atkinson, Elizabeth G. and Martin, Alicia R.},
  doi          = {10.1038/s41588-025-02335-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2408-2417},
  shortjournal = {Nat. Genet.},
  title        = {Pan-UK biobank genome-wide association analyses enhance discovery and resolution of ancestry-enriched effects},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Non-CG DNA methylation in animal genomes. <em>NG</em>, <em>57</em>(10), 2395-2407. (<a href='https://doi.org/10.1038/s41588-025-02303-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine DNA methylation is widespread in animal genomes and occurs predominantly at CG dinucleotides (mCG). While the roles of mCG, such as in genomic imprinting and genome stability, are well established, non-CG DNA methylation (mCH) remains poorly understood. In most vertebrate tissues, roughly 80% of CGs are methylated, whereas mCH levels are generally low, typically ranging from 1% to 3%. In vertebrates, mCH is most prevalent in neural tissue, oocytes and embryonic stem cells and has been linked to neurodevelopmental disorders. Moreover, mCH appears to have a conserved role in regulating vertebrate neural genomes, and recent studies suggest that it has functions in the embryogenesis of teleost fish. Overall, mCH represents an intriguing emerging aspect of gene regulation with potential implications for cellular identity, repeat silencing and neural function. In this Review, we provide a critical overview of the patterning, mechanisms and functional implications of mCH in animals. This Review discusses noncanonical DNA methylation (mCH) in animal genomes and highlights the remaining need to clarify whether mCH represents a conserved regulatory layer or a lineage-specific epigenetic feature with distinct biological roles.},
  archive      = {J_NG},
  author       = {Brethouwer, Thirsa and de Mendoza, Alex and Bogdanovic, Ozren},
  doi          = {10.1038/s41588-025-02303-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2395-2407},
  shortjournal = {Nat. Genet.},
  title        = {Non-CG DNA methylation in animal genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing biological understanding of cellular senescence with computational multiomics. <em>NG</em>, <em>57</em>(10), 2381-2394. (<a href='https://doi.org/10.1038/s41588-025-02314-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular senescence is a complex biological process that plays a pathophysiological role in aging and age-related diseases. The biological understanding of senescence at the cellular and tissue levels remains incomplete due to the lack of specific biomarkers as well as the relative rarity of senescent cells, their phenotypic heterogeneity and dynamic features. This Review provides a comprehensive overview of multiomic approaches for the characterization and biological understanding of cellular senescence. The technical capability and challenges of each approach are discussed, and practical guidelines are provided for selecting tools for identifying, characterizing and spatially mapping senescent cells. The importance of computational analyses in multiomics research, including senescent cell identification, signature detection and interactions of senescent cells with microenvironments, is highlighted. Moreover, tissue-specific case studies and experimental design considerations for individual organs are presented. Finally, future directions and the potential impact of multiomic approaches on the biological understanding of cellular senescence are discussed. This Review discusses multiomic approaches for the characterization and biological understanding of cellular senescence, including detailed case studies on skeletal muscle and adipose tissue that highlight current outstanding issues in the field.},
  archive      = {J_NG},
  author       = {Li, Sheng and Agudelo Garcia, Paula A. and Aliferis, Constantin and Becich, Michael J. and Calyeca, Jazmin and Cosgrove, Benjamin D. and Elisseeff, Jennifer and Farzad, Negin and Fertig, Elana J. and Glass, Carolyn and Gu, Liangcai and Hu, Qianjiang and Ji, Zhicheng and Königshoff, Melanie and LeBrasseur, Nathan K. and Li, Dongmei and Ma, Anjun and Ma, Qin and Menon, Vilas and Mitchell, Jacob T. and Mora, Ana L. and Nagaraj, Sushma and Nelson, Andrew C. and Niedernhofer, Laura J. and Rojas, Mauricio and Taha, Hash Brown and Wang, Jinhua and Wang, Siyuan and Wu, Pei-Hsun and Xie, Jichun and Xu, Ming and Yu, Miao and Zhang, Xu and Zhao, Yue and Adams, Peter D. and Aguayo-Mazzucato, Cristina and Baker, Darren J. and Benz, Christopher and Bernlohr, David A. and Bueno, Marta and Chen, Jin and Childs, Bennett G. and Chuang, Jeffrey H. and Chung, Dongjun and Dileepan, Mythili and Ding, Li and Dong, Mingze and Duncan, Francesca and Enninful, Archibald and Flynn, William F. and Franco, Ana Catarina and Furman, David and Garovic, Vesna and Halene, Stephanie and Herman, Allison B. and Hertzel, Ann V. and Iwasaki, Kanako and Jeon, Hyeongseon and Kang, Jeon Woong and Karmakar, Shilpita and Kirkland, James L. and Korstanje, Ron and Kummerfeld, Erich and Lee, Jun Hee and Liu, Yang and Lu, Yao and Lugo-Martinez, Jose and Martini, Helene and Melov, Simon and Musi, Nicolas and Passos, João F. and Peters, Samuel T. and Rahman, Irfan and Ramasamy, Ramalakshmi and Rindone, Alexandra N. and Robbins, Paul D. and Robson, Paul and Rodriguez-Lopez, Jhonny and Rosas, Lorena and Rosenthal, Nadia and Schafer, Marissa J. and Schilling, Birgit and Schmidt, Elizabeth L. and Schneider, Kevin and Sengupta, Kaustav and Shu, Jian and So, Peter T. C. and Sun, Li and Tchkonia, Tamar and Teneche, Marcos G. and Vanegas, Natalia and Wang, Julia and Xie, Juan and Yin, Shanshan and Zhang, Ke and Zhu, Quan and Fan, Rong},
  doi          = {10.1038/s41588-025-02314-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2381-2394},
  shortjournal = {Nat. Genet.},
  title        = {Advancing biological understanding of cellular senescence with computational multiomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A standardized framework for circulating blood proteomics. <em>NG</em>, <em>57</em>(10), 2371-2380. (<a href='https://doi.org/10.1038/s41588-025-02319-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The circulating blood proteome holds immense potential for biomarker discovery and understanding disease mechanisms. Notable advances in mass spectrometry and affinity-based technologies have been made, but data integration across studies and platforms is hindered by the absence of unified analytical standards. This limitation impedes comprehensive exploration of human biology across diverse phenotypes and cohorts as well as the translation of findings into clinical applications. The disparities between datasets, stemming from a combination of factors related to differences in sample collection, pre-analytical handling, measurement methods and instrumentation, further complicate data integration. In this Perspective, we outline key challenges in blood-based proteomics and propose actionable strategies. Central to our recommendations are high-quality, technology-agnostic reference samples, which can bridge disparate datasets and enable robust cross-study comparisons. By fostering interconnected investigations across proteomic technologies, blood sample collections, clinical phenotypes and different populations, these references will accelerate the field and its translation. The field of blood proteomics faces an upsurge of data with the challenge of cross-study comparisons. This Perspective offers an in-depth analysis and proposes reference materials to enhance data integration and accelerate clinical translation.},
  archive      = {J_NG},
  author       = {Cai, Xue and Geyer, Philipp E. and Perez-Riverol, Yasset and Omenn, Gilbert S. and Dong, Lianhua and Winkler, Robert and Ahadi, Sara and Lössl, Philip and Yu, Xiaobo and Chang, Cheng and Ralser, Markus and Jimenez, Connie R. and Zhao, Yang and Chen, Yu-Ju and Poon, Terence C. W. and Bache, Nicolai and Shi, Leming and Dai, Xinhua and Wang, Ziyue and Zhu, Yi and Fang, Xiang and Schwenk, Jochen M. and Van Eyk, Jennifer E. and Völker, Uwe and Guo, Tiannan},
  doi          = {10.1038/s41588-025-02319-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2371-2380},
  shortjournal = {Nat. Genet.},
  title        = {A standardized framework for circulating blood proteomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The solve-RD solvathons as a pan-european interdisciplinary collaboration to diagnose patients with rare disease. <em>NG</em>, <em>57</em>(10), 2361-2370. (<a href='https://doi.org/10.1038/s41588-025-02290-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in genomic diagnostics, the majority of individuals with rare diseases remain without a confirmed genetic diagnosis. The rapid emergence of advanced omics technologies, such as long-read genome sequencing, optical genome mapping and multiomic profiling, has improved diagnostic yield but also substantially increased analytical and interpretational complexity. Addressing this complexity requires systematic multidisciplinary collaboration, as recently demonstrated by targeted diagnostic workshops. Here, we highlight the experience of the Solve-RD consortium, a pan-European initiative, in implementing four structured workshops, termed ‘Solvathons’, as a regular and effective component of its operational workflow. We provide actionable insights, best practices and lessons learned for successful data integration, expert training and scalable collaborative diagnostics within large research consortia. This Perspective presents the Solve-RD Solvathon model, an innovative, pan-European framework uniting clinical and bioinformatics experts to diagnose rare diseases through integrative multiomics analysis and structured collaboration.},
  archive      = {J_NG},
  author       = {Yépez, Vicente A. and Demidov, German and Ellwanger, Kornelia and Laurie, Steven and Luknárová, Rebeka and Joseph Maran, Midhuna Immaculate and Hentrich, Thomas and Sagath, Lydia and van der Sanden, Bart and Astuti, Galuh and Neveling, Kornelia and Batlle-Masó, Laura and Beijer, Danique and Brechtmann, Felix and Caballero-Oteyza, Andrés and Dabad, Marc and Denommé-Pichon, Anne-Sophie and Doornbos, Cenna and Eddafir, Zakaria and Estévez-Arias, Berta and Kilicarslan, Ozge Aksel and Kolen, Ingrid H. M. and Kraß, Leon and Lohmann, Katja and Londhe, Shubhankar and López-Martín, Estrella and Maassen, Kars and Macken, William and Martínez-Delgado, Beatriz and Mei, Davide and Mertes, Christian and Minardi, Raffaella and Morsy, Heba and Mueller, Juliane S. and Natera-de Benito, Daniel and Nelson, Isabelle and Oud, Machteld M. and Paramonov, Ida and Picó, Daniel and Piscia, Davide and Polavarapu, Kiran and Raineri, Emanuele and Savarese, Marco and Smal, Noor and Steehouwer, Marloes and Steyaert, Wouter and Swertz, Morris A. and Thomsen, Mirja and Töpf, Ana and Van de Vondel, Liedewei and van der Vries, Gerben and Vitobello, Antonio and Wilke, Carlo and Zurek, Birte and t’ Hoen, Peter-Bram and Matalonga, Leslie and Vissers, Lisenka E. L. M. and Gilissen, Christian and Schulze-Hentrich, Julia and Beltran, Sergi and Esteve-Codina, Anna and Hoischen, Alexander and Gagneur, Julien and Graessner, Holm},
  doi          = {10.1038/s41588-025-02290-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2361-2370},
  shortjournal = {Nat. Genet.},
  title        = {The solve-RD solvathons as a pan-european interdisciplinary collaboration to diagnose patients with rare disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic perturbations link autoimmune risk variants to t cell function. <em>NG</em>, <em>57</em>(10), 2359-2360. (<a href='https://doi.org/10.1038/s41588-025-02302-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We used a massively parallel reporter assay (MPRA) to test thousands of autoimmune disease-associated genetic variants for allele-specific effects on gene expression in primary human T cells. Variants that altered gene expression within our MPRA were then linked to T cell regulatory networks and proliferation using bulk and single-cell CRISPR-interference screens.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02302-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2359-2360},
  shortjournal = {Nat. Genet.},
  title        = {Genetic perturbations link autoimmune risk variants to t cell function},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cracking the histone code for prostate cancer therapy. <em>NG</em>, <em>57</em>(10), 2357-2358. (<a href='https://doi.org/10.1038/s41588-025-02354-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, we uncover the critical role of p300/CBP-mediated histone H2B N terminus multisite lysine acetylation (H2BNTac) in defining oncogenic enhanceosomes in prostate cancer. Degradation of p300/CBP rapidly disables H2BNTac-marked oncogenic enhancers and represents a promising therapeutic strategy for enhancer-driven malignancies, including prostate cancer.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02354-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2357-2358},
  shortjournal = {Nat. Genet.},
  title        = {Cracking the histone code for prostate cancer therapy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The dawn of bryophyte genomics is here. <em>NG</em>, <em>57</em>(10), 2355-2356. (<a href='https://doi.org/10.1038/s41588-025-02240-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bryophytes are a widespread group of land plants that occupy nearly all biomes, yet their genetics and evolutionary history have long remained underexplored. Now, a study that generates extensive genomic data for bryophytes highlights de novo gene formation and horizontal gene transfer as key forces that shape bryophyte diversity and adaptation.},
  archive      = {J_NG},
  author       = {Potente, Giacomo and Yue, Yuling and Szövényi, Péter},
  doi          = {10.1038/s41588-025-02240-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2355-2356},
  shortjournal = {Nat. Genet.},
  title        = {The dawn of bryophyte genomics is here},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dipping into the function of DNA methylation at centromeres. <em>NG</em>, <em>57</em>(10), 2352-2354. (<a href='https://doi.org/10.1038/s41588-025-02334-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human centromeres contain a small chromatin region with low levels of DNA cytosine methylation that resides with CENP-A. Salinas-Luypaert et al. find a role of DNA methylation in maintaining the size and function of centromeres by controlling the binding affinity of key centromere components.},
  archive      = {J_NG},
  author       = {Erhardt, Sylvia},
  doi          = {10.1038/s41588-025-02334-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2352-2354},
  shortjournal = {Nat. Genet.},
  title        = {Dipping into the function of DNA methylation at centromeres},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A high-power spatial, single-cell view of pericytes in esophageal cancer metastasis. <em>NG</em>, <em>57</em>(10), 2350-2351. (<a href='https://doi.org/10.1038/s41588-025-02318-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using integrated single-cell multi-omics and spatial analysis, this study identifies a PRRX1-driven GPR116⁺ pericyte subpopulation that promotes tumor metastasis and immune evasion in esophageal squamous cell carcinoma, offering a diagnostic biomarker and potential therapeutic strategies.},
  archive      = {J_NG},
  author       = {Sinha, Uttam K. and Park, Young Min and Lin, De-Chen},
  doi          = {10.1038/s41588-025-02318-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2350-2351},
  shortjournal = {Nat. Genet.},
  title        = {A high-power spatial, single-cell view of pericytes in esophageal cancer metastasis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A plasma proteomic signature for ALS. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02384-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02384-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {A plasma proteomic signature for ALS},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recycling enhancers. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02385-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02385-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Recycling enhancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Detecting parent-of-origin genetic effects without parental genomes. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02386-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02386-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Detecting parent-of-origin genetic effects without parental genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Basal cells drive small cell lung cancer plasticity. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02387-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02387-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Basal cells drive small cell lung cancer plasticity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Realizing the full potential of our future health through data linkage and trans-biobank efforts. <em>NG</em>, <em>57</em>(10), 2341-2348. (<a href='https://doi.org/10.1038/s41588-025-02327-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scale and population coverage of Our Future Health, alongside other next-generation biobanks, offers unique opportunities to advance genomic medicine. Focusing on the UK context, we provide a researcher’s perspective of how this new resource could reach its full potential in a way that is impactful, user-friendly and informs related global efforts.},
  archive      = {J_NG},
  author       = {Straub, Vincent J. and Benonisdottir, Stefania and Kong, Augustine and Mills, Melinda C.},
  doi          = {10.1038/s41588-025-02327-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2341-2348},
  shortjournal = {Nat. Genet.},
  title        = {Realizing the full potential of our future health through data linkage and trans-biobank efforts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmat">NMAT - 28</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Highly reversible extrinsic electrocaloric effects over a wide temperature range in epitaxially strained SrTiO3 films. <em>NMAT</em>, <em>24</em>(10), 1664. (<a href='https://doi.org/10.1038/s41563-025-02321-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMAT},
  author       = {Zhang, S. and Deliyore-Ramírez, J. and Deng, S. and Nair, B. and Pesquera, D. and Jing, Q. and Vickers, M. E. and Crossley, S. and Ghidini, M. and Kar-Narayan, S. and Guzmán-Verri, G. G. and Moya, X. and Mathur, N. D.},
  doi          = {10.1038/s41563-025-02321-8},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1664},
  shortjournal = {Nat. Mater.},
  title        = {Publisher correction: Highly reversible extrinsic electrocaloric effects over a wide temperature range in epitaxially strained SrTiO3 films},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blood–brain-barrier-crossing lipid nanoparticles for mRNA delivery to the central nervous system. <em>NMAT</em>, <em>24</em>(10), 1653-1663. (<a href='https://doi.org/10.1038/s41563-024-02114-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The systemic delivery of mRNA molecules to the central nervous system is challenging as they need to cross the blood–brain barrier (BBB) to reach into the brain. Here we design and synthesize 72 BBB-crossing lipids fabricated by conjugating BBB-crossing modules and amino lipids, and use them to assemble BBB-crossing lipid nanoparticles for mRNA delivery. Screening and structure optimization studies resulted in a lead formulation that has substantially higher mRNA delivery efficiency into the brain than those exhibited by FDA-approved lipid nanoparticles. Studies in distinct mouse models show that these BBB-crossing lipid nanoparticles can transfect neurons and astrocytes of the whole brain after intravenous injections, being well tolerated across several dosage regimens. Moreover, these nanoparticles can deliver mRNA to human brain ex vivo samples. Overall, these BBB-crossing lipid nanoparticles deliver mRNA to neurons and astrocytes in broad brain regions, thereby being a promising platform to treat a range of central nervous system diseases. A lipid-nanoparticle-based formulation incorporating engineered lipids containing a blood–brain-barrier-crossing moiety effectively delivers mRNA to neurons and astrocytes following systemic administration in mice.},
  archive      = {J_NMAT},
  author       = {Wang, Chang and Xue, Yonger and Markovic, Tamara and Li, Haoyuan and Wang, Siyu and Zhong, Yichen and Du, Shi and Zhang, Yuebao and Hou, Xucheng and Yu, Yang and Liu, Zhengwei and Tian, Meng and Kang, Diana D. and Wang, Leiming and Guo, Kaiyuan and Cao, Dinglingge and Yan, Jingyue and Deng, Binbin and McComb, David W. and Parsons, Ramon E. and Minier-Toribio, Angelica M. and Holt, Leanne M. and Pan, Jiayi and Hashemi, Alice and Kopell, Brian H. and Charney, Alexander W. and Nestler, Eric J. and Peng, Paul C. and Dong, Yizhou},
  doi          = {10.1038/s41563-024-02114-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1653-1663},
  shortjournal = {Nat. Mater.},
  title        = {Blood–brain-barrier-crossing lipid nanoparticles for mRNA delivery to the central nervous system},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bond-centric modular design of protein assemblies. <em>NMAT</em>, <em>24</em>(10), 1644-1652. (<a href='https://doi.org/10.1038/s41563-025-02297-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directional interactions that generate regular coordination geometries are a powerful means of guiding molecular and colloidal self-assembly, but implementing such high-level interactions with proteins remains challenging due to their complex shapes and intricate interface properties. Here we describe a modular approach to protein nanomaterial design inspired by the rich chemical diversity that can be generated from the small number of atomic valencies. We design protein building blocks using deep learning-based generative tools, incorporating regular coordination geometries and tailorable bonding interactions that enable the assembly of diverse closed and open architectures guided by simple geometric principles. Experimental characterization confirms the successful formation of more than 20 multicomponent polyhedral protein cages, two-dimensional arrays and three-dimensional protein lattices, with a high (10%–50%) success rate and electron microscopy data closely matching the corresponding design models. Due to modularity, individual building blocks can assemble with different partners to generate distinct regular assemblies, resulting in an economy of parts and enabling the construction of reconfigurable networks for designer nanomaterials. Deep learning-based generative tools are used to design protein building blocks with well-defined directional bonding interactions, allowing the generation of a variety of scalable protein assemblies from a small set of reusable subunits.},
  archive      = {J_NMAT},
  author       = {Wang, Shunzhi and Favor, Andrew and Kibler, Ryan D. and Lubner, Joshua M. and Borst, Andrew J. and Coudray, Nicolas and Redler, Rachel L. and Chiang, Huat Thart and Sheffler, William and Hsia, Yang and Bethel, Neville P. and Li, Zhe and Ekiert, Damian C. and Bhabha, Gira and Pozzo, Lilo D. and Baker, David},
  doi          = {10.1038/s41563-025-02297-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1644-1652},
  shortjournal = {Nat. Mater.},
  title        = {Bond-centric modular design of protein assemblies},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational design of bifaceted protein nanomaterials. <em>NMAT</em>, <em>24</em>(10), 1635-1643. (<a href='https://doi.org/10.1038/s41563-025-02295-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in computational methods have led to considerable progress in the design of protein nanomaterials. However, nearly all nanoparticles designed so far exhibit strict point group symmetry, which limits structural diversity and precludes anisotropic functionalization. Here we describe a computational strategy for designing multicomponent bifaceted protein nanomaterials with two distinctly addressable sides. The method centres on docking pseudosymmetric hetero-oligomeric building blocks in architectures with dihedral symmetry and designing an asymmetric protein–protein interface between them. We obtain an initial 30-subunit assembly with pseudo-D5 symmetry and generate variants in which we alter the size and morphology of the bifaceted nanoparticles by designing extensions to one of the subunits. Functionalization of the two nanoparticle faces with protein minibinders enables the specific colocalization of two populations of polystyrene microparticles coated with the target protein receptors. The ability to accurately design anisotropic protein nanoparticles could be broadly useful in applications requiring the colocalization of distinct target moieties. Predictive protein design and experiments are combined to develop anisotropic bifaceted protein nanomaterials using pseudosymmetric hetero-oligomeric building blocks.},
  archive      = {J_NMAT},
  author       = {Rankovic, Sanela and Carr, Kenneth D. and Decarreau, Justin and Skotheim, Rebecca and Kibler, Ryan D. and Ols, Sebastian and Lee, Sangmin and Chun, Jung-Ho and Tooley, Marti R. and Dauparas, Justas and Eisenach, Helen E. and Glögl, Matthias and Weidle, Connor and Borst, Andrew J. and Baker, David and King, Neil P.},
  doi          = {10.1038/s41563-025-02295-7},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1635-1643},
  shortjournal = {Nat. Mater.},
  title        = {Computational design of bifaceted protein nanomaterials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Organic solar cells with 21% efficiency enabled by a hybrid interfacial layer with dual-component synergy. <em>NMAT</em>, <em>24</em>(10), 1626-1634. (<a href='https://doi.org/10.1038/s41563-025-02305-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cathode interfacial layer (CIL) critically influences electron extraction and charge recombination, thereby playing a pivotal role in organic solar cells (OSCs). However, most state-of-the-art CILs are constrained by limited conductivity, high recombination and poor morphology, which collectively hinder device efficiency and stability. Here we report an inorganic–organic hybrid CIL (AZnO-F3N), developed by a dual-component synergy strategy, which integrates organic material PNDIT-F3N with two-dimensional amorphous zinc oxide. This design leverages the synergistic interactions between two-dimensional amorphous zinc oxide and PNDIT-F3N, resulting in reduced interfacial defect, enhanced conductivity and improved film uniformity. OSCs incorporating the AZnO-F3N CIL exhibit more efficient charge extraction and transport, along with reduced recombination. Consequently, a D18:L8-BO-based binary OSC achieves an efficiency of 20.6%. The introduction of BTP-eC9 as the third component further elevates the efficiency to 21.0% (certified as 20.8%). Moreover, the CIL demonstrates versatility across various active layers, thick-film configuration and flexible devices, underscoring its great potential to advance OSC technology. A hybrid cathode interfacial layer (AZnO-F3N) is developed, delivering 21.0% efficiency along with excellent stability, mechanical robustness and broad versatility, highlighting its potential to advance organic solar cells.},
  archive      = {J_NMAT},
  author       = {Li, Congqi and Cai, Yunhao and Hu, Pengfei and Liu, Tao and Zhu, Lei and Zeng, Rui and Han, Fei and Zhang, Ming and Zhang, Meng and Lv, Jikai and Ma, Yuanxin and Han, Dexia and Lin, Qijie and Xu, Jingwen and Yu, Na and Qiao, Jiawei and Wang, Jiarui and Zhang, Xin and Xia, Jianlong and Tang, Zheng and Ye, Long and Li, Xiaoyi and Xu, Zihao and Hao, Xiaotao and Peng, Qian and Liu, Feng and Guo, Lin and Huang, Hui},
  doi          = {10.1038/s41563-025-02305-8},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1626-1634},
  shortjournal = {Nat. Mater.},
  title        = {Organic solar cells with 21% efficiency enabled by a hybrid interfacial layer with dual-component synergy},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanoscale phonon dynamics in self-assembled nanoparticle lattices. <em>NMAT</em>, <em>24</em>(10), 1616-1625. (<a href='https://doi.org/10.1038/s41563-025-02253-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geometry and topology endow mechanical frames with unusual properties from shape morphing to phonon wave manipulation, enabling emerging technologies. Despite important advances in macroscopic frames, the realization and phonon imaging of nanoscale mechanical metamaterials has remained challenging. Here we extend the principle of topologically engineered mechanical frames to self-assembled nanoparticle lattices, resolving phonon dynamics using liquid-phase transmission electron microscopy. The vibrations of nanoparticles in Maxwell lattices are used to measure properties that have been difficult to obtain, such as phonon band structures, nanoscale spring constants and nonlinear lattice deformation paths. Studies of five different lattices reveal that these properties are modulated by nanoscale colloidal interactions. Our discrete mechanical model and simulations capture these interactions and the critical role of effects beyond nearest neighbours, bridging mechanical metamaterials with nanoparticle self-assembly. Our study provides opportunities for understanding and manufacturing self-assembled nanostructures for phonon manipulation, offering solution processability, transformability and emergent functions at underexplored scales of length, frequency and energy density. The realization and phonon imaging of nanoscale mechanical metamaterials has remained challenging. Here the authors resolve the phonon dynamics and band structures of five different self-assembled nanoparticle lattices, revealing the role of nanoscale colloidal interactions in modulating the lattice properties.},
  archive      = {J_NMAT},
  author       = {Qian, Chang and Stanifer, Ethan and Ma, Zhan and Yao, Lehan and Luo, Binbin and Liu, Chang and Li, Jiahui and Pan, Puquan and Pan, Wenxiao and Mao, Xiaoming and Chen, Qian},
  doi          = {10.1038/s41563-025-02253-3},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1616-1625},
  shortjournal = {Nat. Mater.},
  title        = {Nanoscale phonon dynamics in self-assembled nanoparticle lattices},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flexible fibre-shaped fuel cells with gel-mediated internal pressure encapsulation. <em>NMAT</em>, <em>24</em>(10), 1608-1615. (<a href='https://doi.org/10.1038/s41563-025-02319-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of flexible fuel cells has been hindered by the rigid components and stringent requirements for pressure encapsulation and fuel sealing. Here we report an adaptive internal pressure encapsulation strategy that leverages the dynamic swelling behaviour of woven cotton fibres enclosed in a gel matrix in methanol. This strategy achieves simultaneous interfacial self-reinforcement and pressure modulation, enabling the fabrication of fibre-shaped direct methanol fuel cells. These flexible fuel cells operate across a broad temperature range, from –22 °C to 70 °C, showcasing cuttability, water resistance and fast refuelling capabilities, with full refuelling being achieved within 1 min. Furthermore, the fuel cells maintain consistent discharge performance, even after enduring 2,000 continuous flexing cycles. With an energy density of 161.36 Wh kg−1, these fibre-shaped direct methanol fuel cells surpass the energy densities of typical fibre-based power systems. This technology mitigates many of the technical challenges related to the lightweight and flexible application of fuel cells or fuel cell stacks for powering high-energy flexible devices. Cotton yarns encapsulated by a gel layer show dynamic swelling in methanol, creating interfacial self-reinforcement and pressure modulation mechanisms that enable the fabrication of fibre-shaped methanol fuel cells.},
  archive      = {J_NMAT},
  author       = {Yuan, Yongjiang and Liu, Ziyang and Zou, Xiuyang and Fang, Pengda and Zhang, Jiale and Zhang, Qiuhuan and Zhang, Hao and Yu, Qikun and Zhou, Tao and Li, Weizheng and Zheng, Sijie and Yang, Mingchen and Sun, Zhe and Zhu, Meifang and Yan, Feng},
  doi          = {10.1038/s41563-025-02319-2},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1608-1615},
  shortjournal = {Nat. Mater.},
  title        = {Flexible fibre-shaped fuel cells with gel-mediated internal pressure encapsulation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An amorphous Li–V–O–F cathode with tetrahedral coordination and O–O formal redox at low voltage. <em>NMAT</em>, <em>24</em>(10), 1600-1607. (<a href='https://doi.org/10.1038/s41563-025-02293-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ever-increasing demand for lithium-ion batteries has necessitated the development of high-performance cathode materials. However, previous studies have predominantly focused on crystal cathodes comprising the octahedral coordination of metal atoms and a well-ordered layered topology. This omits other cathode materials with other structures or coordination that could potentially surpass conventional counterparts in terms of performance. Here, using X-ray diffraction, resonant inelastic X-ray scattering and X-ray absorption near-edge spectra experiments, we investigated an amorphous Li–V–O–F cathode (a-LVOF) with tetrahedral coordination and elucidated an O–O formal redox mechanism at a moderate voltage of 4.1 V, without a conventional octahedral Li–O–Li configuration. The electrochemically amorphized material fosters randomly distributed VO4 units and scattered dangling oxygen bonds, which facilitated O–O binding. Moreover, a-LVOF demonstrates a high capacity of 230 mAh g−1. Our findings reveal a low-voltage O–O formal redox mechanism in an amorphized cathode material. Cathode materials for lithium-ion batteries typically possess octahedral coordination, which may exclude other possible solutions to degradation during deep cycling. A series of tetrahedral-framework-based amorphous Li–V–O–F materials are investigated, and shown to demonstrate O–O formal redox at 4.1 V.},
  archive      = {J_NMAT},
  author       = {Zhang, Kun and Yang, Tonghuan and Chen, Tao and Yang, Yali and Jiang, Zewen and Gao, Chuan and Zuo, Yuxuan and Xiao, Wukun and Xia, Dingguo},
  doi          = {10.1038/s41563-025-02293-9},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1600-1607},
  shortjournal = {Nat. Mater.},
  title        = {An amorphous Li–V–O–F cathode with tetrahedral coordination and O–O formal redox at low voltage},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Signatures of magnetism in zigzag graphene nanoribbons embedded in a hexagonal boron nitride lattice. <em>NMAT</em>, <em>24</em>(10), 1592-1599. (<a href='https://doi.org/10.1038/s41563-025-02317-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zigzag edges of graphene are predicted to host magnetic electronic states, critical for spintronics, but an experimental confirmation of these magnetic conduction channels remains elusive. Here we report the signatures of magnetism in zigzag graphene nanoribbons (zGNRs) embedded in hexagonal boron nitride. Hexagonal boron nitride provides crucial edge stabilization, enabling the direct probing of this intrinsic magnetism. Scanning nitrogen-vacancy-centre microscopy initially confirmed magnetism in zGNR. Subsequently, an ~9-nm-wide zGNR transistor was fabricated with a sub-50-nm channel length. Magnetotransport measurements at 4 K revealed distinct Fabry–Pérot-like interference patterns, indicating coherent transport. A large, anisotropic magnetoresistance (~175 Ω, ~1.3%) was observed, persisting well above room temperature. These findings strongly corroborate the existence of robust magnetic ordering in the zGNR edge state. This hexagonal-boron-nitride-embedded zGNR system offers an effective platform for future graphene-based spintronic devices. Nitrogen-vacancy-centre microscopy and anisotropic magnetoresistance measurements reveal the evidence of magnetism in zigzag graphene nanoribbons embedded in hexagonal boron nitride.},
  archive      = {J_NMAT},
  author       = {Jiang, Chengxin and Wang, Hui Shan and Liu, Chenxi and Chen, Chen and Chen, Lingxiu and Wang, Xiujun and Wang, Yibo and Kong, Ziqiang and Feng, Yuhan and Liu, Yixin and Feng, Yu and Zhang, Yu and Wei, Zhipeng and Guo, Maosen and Tong, Aomei and Mu, Gang and Yang, Yumeng and Watanabe, Kenji and Taniguchi, Takashi and Shi, Wangzhou and Wang, Haomin},
  doi          = {10.1038/s41563-025-02317-4},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1592-1599},
  shortjournal = {Nat. Mater.},
  title        = {Signatures of magnetism in zigzag graphene nanoribbons embedded in a hexagonal boron nitride lattice},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Symmetry-protected electronic metastability in an optically driven cuprate ladder. <em>NMAT</em>, <em>24</em>(10), 1584-1591. (<a href='https://doi.org/10.1038/s41563-025-02254-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optically excited quantum materials exhibit non-equilibrium states with remarkable emergent properties, but these phenomena are usually transient, decaying on picosecond timescales and limiting practical applications. Advancing the design and control of non-equilibrium phases requires the development of targeted strategies to achieve long-lived, metastable phases. Here we report the discovery of symmetry-protected electronic metastability in the model cuprate ladder Sr14Cu24O41. Using femtosecond resonant X-ray scattering and spectroscopy, we show that this metastability is driven by a transfer of holes from chain-like charge reservoirs into the ladders. This ultrafast charge redistribution arises from the optical dressing and activation of a hopping pathway that is forbidden by symmetry at equilibrium. Relaxation back to the ground state is, hence, suppressed after the pump coherence dissipates. Our findings highlight how dressing materials with electromagnetic fields can dynamically activate terms in the electronic Hamiltonian, and provide a rational design strategy for non-equilibrium phases of matter. A long-lived photoinduced metastable state in a quasi-one-dimensional cuprate, Sr14Cu24O41, is reported.},
  archive      = {J_NMAT},
  author       = {Padma, Hari and Glerean, Filippo and TenHuisen, Sophia F. R. and Shen, Zecheng and Wang, Haoxin and Xu, Luogen and Elliott, Joshua D. and Homes, Christopher C. and Skoropata, Elizabeth and Ueda, Hiroki and Liu, Biaolong and Paris, Eugenio and Romaguera, Arnau and Lee, Byungjune and He, Wei and Wang, Yu and Lee, Seng Huat and Choi, Hyeongi and Park, Sang-Youn and Mao, Zhiqiang and Calandra, Matteo and Jang, Hoyoung and Razzoli, Elia and Dean, Mark P. M. and Wang, Yao and Mitrano, Matteo},
  doi          = {10.1038/s41563-025-02254-2},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1584-1591},
  shortjournal = {Nat. Mater.},
  title        = {Symmetry-protected electronic metastability in an optically driven cuprate ladder},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Achieving small singlet–triplet energy gaps in polycyclic heteroaromatic emitters. <em>NMAT</em>, <em>24</em>(10), 1576-1583. (<a href='https://doi.org/10.1038/s41563-025-02309-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In polycyclic heteroaromatic (PHA) emitters, which possess great potential for application in ultrahigh-definition displays, the origin of a small singlet–triplet energy gap (ΔES1T1) and its relationship with the molecular structure still remain poorly established. Here we derive an effective expression for ΔES1T1, in which ΔES1T1 positively depends on 2KHL (where KHL is the exchange energy between the highest occupied molecular orbital and the lowest unoccupied molecular orbital (LUMO)) and on the energy gap between LUMO and LUMO + 1 (ΔELUMO–LUMO+1). This expression for ΔES1T1 is validated over a series of 100 reported PHA emitters. It allows us to easily identify various molecular design approaches for managing ΔES1T1 by synergistically regulating 2KHL and ΔELUMO–LUMO+1. The proof-of-concept PHA molecules were synthesized and characterized to further confirm the validity of this expression for ΔES1T1. Overall, our work provides a physical picture to not only modulate ΔES1T1 in emerging PHA emitters but also design and screen such materials with small ΔES1T1. Insight into the singlet–triplet energy gap and molecular structure relationship in polycyclic heteroaromatic molecules enables organic light-emitting diodes based on multiresonance thermally activated delayed fluorescence, with improved efficiency and roll-off.},
  archive      = {J_NMAT},
  author       = {Walia, Rajat and Xiong, Xin and Fan, Xiao-Chun and Chen, Ting-Feng and Wang, Hui and Wang, Kai and Shi, Yi-Zhong and Tang, Xun and Bredas, Jean-Luc and Adachi, Chihaya and Chen, Xian-Kai and Zhang, Xiao-Hong},
  doi          = {10.1038/s41563-025-02309-4},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1576-1583},
  shortjournal = {Nat. Mater.},
  title        = {Achieving small singlet–triplet energy gaps in polycyclic heteroaromatic emitters},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flatland wakes based on leaky hyperbolic polaritons. <em>NMAT</em>, <em>24</em>(10), 1569-1575. (<a href='https://doi.org/10.1038/s41563-025-02280-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hyperbolic polaritons facilitate nanoscale light manipulation, but strong field confinement limits their transmission across interfaces. Conversely, leaky waves can convert radiation from confined sources towards the far field. Here we combine hyperbolic polaritons and leaky wave radiation to demonstrate flatland leaky polaritonic wakes. We employ a mixed-dimensional van der Waals heterostructure consisting of a nanoscale waveguide strip on a van der Waals film. The waveguide mode, confined inside the hyperbolic light cone of the background film, enables efficient directional in-plane emission of fast phonon polaritons. The constructive interference of these leaky polaritons generates highly directional polaritonic wakes. Their spatial symmetry can be tailored through the orientation of the heterostructure with respect to the hyperbolic film dispersion. Leveraging van der Waals stacking, we also demonstrate effective acceleration and deceleration of polaritonic wakes by locally tailoring the leaky nano-waveguide dispersion through gradient thickness design. Our findings demonstrate that polaritonic wakes hold promise for integrated nanophotonic circuits. The authors combine the strong confinement of hyperbolic polaritons with leaky nano-waveguides to demonstrate directional in-plane emission of fast phonon polaritons and their acceleration and deceleration by tailoring waveguide dispersion.},
  archive      = {J_NMAT},
  author       = {Chen, Na and Teng, Hanchao and Hu, Hai and Liu, Min and Jiang, Chengyu and Xue, Zhuoxin and Zhu, Hualong and Gui, Jiayi and Li, Peining and Alù, Andrea and Dai, Qing},
  doi          = {10.1038/s41563-025-02280-0},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1569-1575},
  shortjournal = {Nat. Mater.},
  title        = {Flatland wakes based on leaky hyperbolic polaritons},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Twist-assisted intrinsic toughening in two-dimensional transition metal dichalcogenides. <em>NMAT</em>, <em>24</em>(10), 1561-1568. (<a href='https://doi.org/10.1038/s41563-025-02193-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Material fractures are typically irreversible, marking a one-time event leading to failure. Great efforts have been made to enhance both strength and fracture toughness of bulk materials for engineering applications, such as by introducing self-recovery and secondary breaking behaviours. In low-dimensional structures, two-dimensional materials often exhibit exceptional strength but accompanied by extreme brittleness. Here we discover that the toughness of two-dimensional materials can be enhanced without sacrificing strength—by simply twisting the layers. Through in situ scanning transmission electron microscopy, supported by nanoindentation and theoretical analysis, we reveal that twisted bilayer structures enable sequential fracture events: initial cracks heal to form stable grain boundaries, which then shield subsequent fracture tips from stress concentration. This process consumes additional energy compared with conventional fracture, with toughness enhancement tunable through twist angle adjustment. The intrinsic toughening mechanism via twisting, along with the emerging electronic properties of twistronics that are currently attracting substantial attention, presents an exciting opportunity for future devices. Intrinsic toughening in two-dimensional transition metal dichalcogenides can be achieved simply by twisting the layers. This twisting promotes cross-layer healing and grain boundary formation, which shield fracture tips from stress concentration.},
  archive      = {J_NMAT},
  author       = {Zheng, Xiaodong and Feng, Shizhe and Tsang, Chi Shing and Thi, Quoc Huy and Han, Wei and Wong, Lok Wing and Liu, Haijun and Lee, Chun-Sing and Lau, Shu Ping and Ly, Thuc Hue and Xu, Zhiping and Zhao, Jiong},
  doi          = {10.1038/s41563-025-02193-y},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1561-1568},
  shortjournal = {Nat. Mater.},
  title        = {Twist-assisted intrinsic toughening in two-dimensional transition metal dichalcogenides},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flaw-size-dependent mechanical interlayer coupling and edge-reconstruction embrittlement in van der waals materials. <em>NMAT</em>, <em>24</em>(10), 1554-1560. (<a href='https://doi.org/10.1038/s41563-025-02194-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Van der Waals (vdW) materials consisting of two-dimensional (2D) building blocks have strong in-plane covalent bonding and weak interlayer interactions. While monolayer 2D materials exhibit impressive fracture resistance, as demonstrated in hexagonal boron nitride (h-BN), preserving these remarkable properties in vdW materials remains a challenge. Here we reveal an anomalous mechanical interlayer coupling that involves interlayer-friction toughening and edge-reconstruction embrittlement during the fracture of multilayer h-BN. Both asynchronous and synchronous fracture modes and their flaw-size dependence are identified. Edge reconstruction in the synchronous fracture mode can eliminate a toughening mechanism induced by lattice asymmetry in monolayer h-BN, leading to embrittlement of the multilayer h-BN, while the asynchronous fracture mode results in greater fracture resistance. Such findings will provide fundamental guidelines for engineering interlayer interactions in vdW materials including heterostructures and layered architectures for better mechanical and functional performances. Fracture behaviours in multilayer h-BN, involving interlayer-friction toughening and edge-reconstruction embrittlement, are identified through in situ experiments and theoretical analyses.},
  archive      = {J_NMAT},
  author       = {Song, Zhigong and Zhang, Boyu and Yang, Yingchao and Gao, Guanhui and Tang, Daiming and Fang, Qiyi and Zhang, Youtian and Shin, Bongki and Steinbach, Doug and Ai, Qing and Zhao, Xuan and Han, Yimo and Padture, Nitin P. and Sheldon, Brian W. and Taniguchi, Takashi and Watanabe, Kenji and Gao, Huajian and Lou, Jun},
  doi          = {10.1038/s41563-025-02194-x},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1554-1560},
  shortjournal = {Nat. Mater.},
  title        = {Flaw-size-dependent mechanical interlayer coupling and edge-reconstruction embrittlement in van der waals materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Iterative sublattice amorphization facilitates exceptional processability in inorganic semiconductors. <em>NMAT</em>, <em>24</em>(10), 1545-1553. (<a href='https://doi.org/10.1038/s41563-024-02112-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cold-forming processing is a crucial means for the cost-effective production of metal and alloy products. However, this process often results in catastrophic fracture when applied to most inorganic semiconductors owing to their inherent brittleness. Here we report the unique room-temperature plastic deformation mechanism involving sublattice amorphization coupled with Ag-ion diffusion in inorganic semiconductors Ag2Te1–xSx (0.3 ≤ x ≤ 0.6), and an ultrahigh extensibility of up to 10,150%. Once subject to external stress, the crystalline Te/S sublattice undergoes a uniform transformation into an amorphous state, whereas the Ag cations continuously bond with Te/S anions, endowing bulk Ag2Te1–xSx with exceptional plastic deformability. Remarkably, even slight polishing can induce sublattice amorphization in the surface layers. Furthermore, this sublattice amorphization can be reversed to crystals through simple annealing, enlightening the iterative sublattice amorphization strategy, with which metal-like wire drawing, curving, forging and ultrahigh ductility have been obtained in bulk Ag2Te1–xSx at room temperature. These results highlight sublattice amorphization as a critical plastic deformation mechanism in silver chalcogenide inorganic semiconductors, which will facilitate their applications in flexible electronics and drive further exploration of more plastic inorganic semiconductors. Sublattice amorphization is revealed as the deformation mechanism of Ag2Te1–xSx (0.3 ≤ x ≤ 0.6), based on which an iterative crystalline–amorphous transition strategy is proposed to enable these bulk inorganic semiconductors with metal-like processability.},
  archive      = {J_NMAT},
  author       = {Wang, Yuechu and Li, Airan and Hong, Youran and Deng, Tianqi and Deng, Pan and Huang, Yi and Liu, Kai and Wang, Jiangwei and Fu, Chenguang and Zhu, Tiejun},
  doi          = {10.1038/s41563-024-02112-7},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1545-1553},
  shortjournal = {Nat. Mater.},
  title        = {Iterative sublattice amorphization facilitates exceptional processability in inorganic semiconductors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Warm metalworking for plastic manufacturing in brittle semiconductors. <em>NMAT</em>, <em>24</em>(10), 1538-1544. (<a href='https://doi.org/10.1038/s41563-025-02223-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Semiconductors are the core of modern electronics1. Because of their brittleness, semiconductors are usually processed by the complicated techniques of sputtering or deposition2–4, instead of the effective and versatile metalworking methods like rolling, extrusion and pressing used with metals5. Here we show that brittle semiconductors can be plastically manufactured with an extensibility as large as ~3,000% using warm metalworking, that is, plastic manufacturing at slightly elevated temperatures (empirically below 500 K). Many bulk brittle semiconductors, such as Cu2Se, Ag2Se and Bi90Sb10, can be processed like metals below 400–500 K into free-standing, large and high-quality films with a thickness from the macro-scale to the micrometre scale. A model based on temperature-dependent collective atomic displacement and thermal vibration is proposed to explain the superior plasticity. The warm-metalworked films can retain the excellent and tunable physical properties of the bulk versions, such as a high carrier mobility up to ~5,000 cm2 V−1 s−1 and tunable electrical conductivities over six orders of magnitude by adjusting the chemical composition. A case study in film thermoelectric devices demonstrates ultra-high normalized output power densities of 43–54 μW cm−2 K−2. This work suggests that brittle semiconductors can be manufactured by warm metalworking for applications in various electronics. The authors show that bulk brittle semiconductors can be plastically manufactured like metals by warm metalworking into free-standing, metre-scale films with decent physical properties.},
  archive      = {J_NMAT},
  author       = {Gao, Zhiqiang and Yang, Shiqi and Ma, Yupeng and Wei, Tian-Ran and Chen, Xiaohui and Zheng, Wenwen and Qiu, Pengfei and Zeng, Xiaoqin and Chen, Lidong and Shi, Xun},
  doi          = {10.1038/s41563-025-02223-9},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1538-1544},
  shortjournal = {Nat. Mater.},
  title        = {Warm metalworking for plastic manufacturing in brittle semiconductors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Streaming flexoelectricity in saline ice. <em>NMAT</em>, <em>24</em>(10), 1533-1537. (<a href='https://doi.org/10.1038/s41563-025-02332-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite 10% of the Earth’s surface being covered by ice, ice power remains untapped. Although ice is known to generate electricity upon bending via flexoelectricity1, the generated electric polarization per curvature, that is, the flexoelectric coefficient, is too small (~1–10 nC m−1) to be utilized for electromechanical devices. Here we demonstrate that doping ice with NaCl can enhance its flexoelectric coefficient 1,000-fold, to ~1–10 μC m−1. We find that this enhancement is due to the bending-induced streaming current along ice grain boundaries. On the basis of this mechanism, we fabricated flexural devices with an effective piezoelectric coefficient of ~4,000 pC N−1, which is comparable to that of the best piezoelectric materials. The high flexoelectricity of saline ice brings the vision of harnessing ice power one step closer to reality, and may also be relevant to the electrical activity of ice-covered terrestrial regions and icy ocean worlds such as Europa or Enceladus. In addition, the model for coupling between strain gradients and streaming currents is not limited to ice and provides a general framework for extracting electromechanical activity from liquid-infused porous solids. By doping ice with NaCl, it is shown that a flexoelectric coefficient of up to 10 μC m−1 is generated, enabling effective piezoelectric coefficients that are comparable to those of ceramics. This arises from the streaming current of quasi-liquid flow through grain boundaries from one side of the sample to the other.},
  archive      = {J_NMAT},
  author       = {Wen, X. and Ma, Q. and Liu, J. and Saeed, U. and Shen, S. and Catalan, G.},
  doi          = {10.1038/s41563-025-02332-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1533-1537},
  shortjournal = {Nat. Mater.},
  title        = {Streaming flexoelectricity in saline ice},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerated data-driven materials science with the materials project. <em>NMAT</em>, <em>24</em>(10), 1522-1532. (<a href='https://doi.org/10.1038/s41563-025-02272-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Materials Project was launched formally in 2011 to drive materials discovery forwards through high-throughput computation and open data. More than a decade later, the Materials Project has become an indispensable tool used by more than 600,000 materials researchers around the world. This Perspective describes how the Materials Project, as a data platform and a software ecosystem, has helped to shape research in data-driven materials science. We cover how sustainable software and computational methods have accelerated materials design while becoming more open source and collaborative in nature. Next, we present cases where the Materials Project was used to understand and discover functional materials. We then describe our efforts to meet the needs of an expanding user base, through technical infrastructure updates ranging from data architecture and cloud resources to interactive web applications. Finally, we discuss opportunities to better aid the research community, with the vision that more accessible and easy-to-understand materials data will result in democratized materials knowledge and an increasingly collaborative community. Materials design and informatics have become increasingly prominent over the past several decades. Using the Materials Project as an example, this Perspective discusses how properties are calculated and curated, how this knowledge can be used for materials discovery, and the challenges in modelling complex material systems or managing software architecture.},
  archive      = {J_NMAT},
  author       = {Horton, Matthew K. and Huck, Patrick and Yang, Ruo Xi and Munro, Jason M. and Dwaraknath, Shyam and Ganose, Alex M. and Kingsbury, Ryan S. and Wen, Mingjian and Shen, Jimmy X. and Mathis, Tyler S. and Kaplan, Aaron D. and Berket, Karlo and Riebesell, Janosh and George, Janine and Rosen, Andrew S. and Spotte-Smith, Evan W. C. and McDermott, Matthew J. and Cohen, Orion A. and Dunn, Alex and Kuner, Matthew C. and Rignanese, Gian-Marco and Petretto, Guido and Waroquiers, David and Griffin, Sinead M. and Neaton, Jeffrey B. and Chrzan, Daryl C. and Asta, Mark and Hautier, Geoffroy and Cholia, Shreyas and Ceder, Gerbrand and Ong, Shyue Ping and Jain, Anubhav and Persson, Kristin A.},
  doi          = {10.1038/s41563-025-02272-0},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1522-1532},
  shortjournal = {Nat. Mater.},
  title        = {Accelerated data-driven materials science with the materials project},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerating development of polycyclic heteroaromatic emitters with small singlet–triplet energy gaps. <em>NMAT</em>, <em>24</em>(10), 1520-1521. (<a href='https://doi.org/10.1038/s41563-025-02352-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A simple physical model based on electronic interactions between excited configurations enables the quick and reliable prediction of singlet–triplet energy gaps in polycyclic heteroaromatic emitters. Guided by this model, organic emitters with small singlet–triplet energy gaps can be designed for applications in organic light-emitting diodes with high efficiency and colour purity.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02352-1},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1520-1521},
  shortjournal = {Nat. Mater.},
  title        = {Accelerating development of polycyclic heteroaromatic emitters with small singlet–triplet energy gaps},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Designer lipids for delivering mRNA to the brain. <em>NMAT</em>, <em>24</em>(10), 1518-1519. (<a href='https://doi.org/10.1038/s41563-025-02184-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticles formulated with ionizable lipids inspired by brain-targeting small molecules facilitate the delivery of mRNA past the blood–brain barrier and into the brain.},
  archive      = {J_NMAT},
  author       = {Han, Emily L. and Safford, Hannah C. and Mitchell, Michael J.},
  doi          = {10.1038/s41563-025-02184-z},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1518-1519},
  shortjournal = {Nat. Mater.},
  title        = {Designer lipids for delivering mRNA to the brain},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Entering the age of designer protein materials. <em>NMAT</em>, <em>24</em>(10), 1515-1517. (<a href='https://doi.org/10.1038/s41563-025-02372-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning-based generative tools are used to design protein building blocks with well-defined directional protein bonding interactions, allowing for the generation of a variety of scalable protein assemblies from a small set of reusable subunits.},
  archive      = {J_NMAT},
  author       = {Zhang, Shuguang},
  doi          = {10.1038/s41563-025-02372-x},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1515-1517},
  shortjournal = {Nat. Mater.},
  title        = {Entering the age of designer protein materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capturing the dance of nanoparticles in crystals. <em>NMAT</em>, <em>24</em>(10), 1513-1514. (<a href='https://doi.org/10.1038/s41563-025-02342-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phonon dynamics of self-assembled two-dimensional nanoparticle crystals is resolved by liquid-phase transmission electron microscopy.},
  archive      = {J_NMAT},
  author       = {Shen, Eric and Gang, Oleg},
  doi          = {10.1038/s41563-025-02342-3},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1513-1514},
  shortjournal = {Nat. Mater.},
  title        = {Capturing the dance of nanoparticles in crystals},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Salt turns ice into a powerful flexoelectric material. <em>NMAT</em>, <em>24</em>(10), 1511-1512. (<a href='https://doi.org/10.1038/s41563-025-02328-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The brine network in common sea ice boosts its flexoelectric response by three orders of magnitude, matching high-quality piezoelectrics and pointing to cost-effective energy harvesters for cold environments.},
  archive      = {J_NMAT},
  author       = {Lee, Daesu},
  doi          = {10.1038/s41563-025-02328-1},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1511-1512},
  shortjournal = {Nat. Mater.},
  title        = {Salt turns ice into a powerful flexoelectric material},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering a light-induced metastable state. <em>NMAT</em>, <em>24</em>(10), 1509-1510. (<a href='https://doi.org/10.1038/s41563-025-02303-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Investigation of a long-lived metallic state in a low-dimensional cuprate demonstrates a strategy for on-demand light-driven metastability.},
  archive      = {J_NMAT},
  author       = {Lee, Wei-Sheng},
  doi          = {10.1038/s41563-025-02303-w},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1509-1510},
  shortjournal = {Nat. Mater.},
  title        = {Engineering a light-induced metastable state},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Interlayer mechanical coupling in van der waals materials. <em>NMAT</em>, <em>24</em>(10), 1507-1508. (<a href='https://doi.org/10.1038/s41563-025-02239-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fracture behaviours and related atomic-scale phenomena of van der Waals materials are elucidated through the use of electron microscopy, which highlights the important role of interlayer coupling and re-bonding, providing valuable insights for engineering mechanical properties of layered materials and structures.},
  archive      = {J_NMAT},
  author       = {Kim, Kangsik and Lee, Zonghoon},
  doi          = {10.1038/s41563-025-02239-1},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1507-1508},
  shortjournal = {Nat. Mater.},
  title        = {Interlayer mechanical coupling in van der waals materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Plastic processing of bulk semiconductors. <em>NMAT</em>, <em>24</em>(10), 1505-1506. (<a href='https://doi.org/10.1038/s41563-025-02283-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bulk inorganic semiconductors can show remarkable plasticity and extensibility, defying their inherent brittleness and enabling opportunities in advanced semiconductor manufacturing and processing.},
  archive      = {J_NMAT},
  author       = {Nakamura, Atsutomo and Li, Yan},
  doi          = {10.1038/s41563-025-02283-x},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1505-1506},
  shortjournal = {Nat. Mater.},
  title        = {Plastic processing of bulk semiconductors},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mikhail eremets (1949–2024). <em>NMAT</em>, <em>24</em>(10), 1504. (<a href='https://doi.org/10.1038/s41563-025-02333-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pioneer of high-pressure physics and superconductivity.},
  archive      = {J_NMAT},
  author       = {Boeri, Lilia and Drozdov, Alexander P. and Ksenofontov, Vadim and Du, Feng and Minkov, Vasily S. and Pöschl, Ulrich},
  doi          = {10.1038/s41563-025-02333-4},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1504},
  shortjournal = {Nat. Mater.},
  title        = {Mikhail eremets (1949–2024)},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanical insights from functional materials. <em>NMAT</em>, <em>24</em>(10), 1503. (<a href='https://doi.org/10.1038/s41563-025-02378-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our knowledge of deformation behaviour in functional materials is pushing the frontiers of mechanics, informing design strategies and enabling scalable manufacturing.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02378-5},
  journal      = {Nature Materials},
  month        = {10},
  number       = {10},
  pages        = {1503},
  shortjournal = {Nat. Mater.},
  title        = {Mechanical insights from functional materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nmeth">NMETH - 33</h2>
<ul>
<li><details>
<summary>
(2025). HippoMaps: Multiscale cartography of human hippocampal organization. <em>NMETH</em>, <em>22</em>(10), 2211-2222. (<a href='https://doi.org/10.1038/s41592-025-02783-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The hippocampus has a specialized microarchitecture, is situated at the nexus of multiple macroscale functional networks, contributes to numerous cognitive as well as affective processes and is highly susceptible to brain pathology across common disorders. These features make the hippocampus a model to understand how brain structure covaries with function, in both health and disease. Here we introduce HippoMaps, an open access toolbox and online data warehouse for the mapping and contextualization of subregional hippocampal data in the human brain ( http://hippomaps.readthedocs.io ). HippoMaps capitalizes on a unified hippocampal unfolding approach as well as shape intrinsic registration capabilities to allow for cross-participant and cross-modal data aggregation. We initialize this repository with a combination of hippocampal data spanning three-dimensional ex vivo histology, ex vivo 9.4-Tesla magnetic resonance imaging (MRI), as well as in vivo structural MRI and resting-state functional MRI obtained at 3 Tesla and 7 Tesla, together with intracranial encephalography recordings in patients with epilepsy. All code, data and tools are openly available online, with the aim of fostering further community contributions. HippoMaps provides an open-source resource for studying the human hippocampus at different scales and with different modalities such as histology, fMRI, structural MRI and EEG.},
  archive      = {J_NMETH},
  author       = {DeKraker, Jordan and Cabalo, Donna Gift and Royer, Jessica and Ngo, Alexander and Khan, Ali R. and Karat, Bradley G. and Benkarim, Oualid and Rodriguez-Cruces, Raul and Frauscher, Birgit and Pana, Raluca and Hansen, Justine Y. and Misic, Bratislav and Valk, Sofie L. and Lau, Jonathan C. and Kirschner, Matthias and Bernasconi, Andrea and Bernasconi, Neda and Muenzing, Sascha E. A. and Axer, Markus and Amunts, Katrin and Evans, Alan C. and Bernhardt, Boris C.},
  doi          = {10.1038/s41592-025-02783-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2211-2222},
  shortjournal = {Nat. Meth.},
  title        = {HippoMaps: Multiscale cartography of human hippocampal organization},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatiotemporal focusing enables all-optical in situ histology of heterogeneous tissue. <em>NMETH</em>, <em>22</em>(10), 2205-2210. (<a href='https://doi.org/10.1038/s41592-025-02849-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Living systems embody heterogeneous tissues with complex opto-mechanical properties. Achieving organ-scale, diffraction-limited volumetric imaging that faithfully captures in vivo architecture requires minimizing sample deformation and preserving vascular and neuronal continuity across delicate tissue interfaces. As a solution to this problem, we developed a robotic nonlinear optical system for iterative multiphoton microscopy and opto-micromachining. Adaptive control enabled days-long autonomous operation, while spatiotemporal line-focused ablation increased the machining efficiency by 100-fold over prior configurations. Using the intact murine craniocerebral system as a test bed, our approach demonstrates the potential for whole-body submicrometer resolution imaging and anatomical reconstruction. Laser ablation of imaged surface layers under adaptive control enables volumetric imaging of samples consisting of heterogeneous tissues, such as the skull and brain.},
  archive      = {J_NMETH},
  author       = {Ji, Xiang and Huang, Sincheng and Friedman, Beth and Kleinfeld, David},
  doi          = {10.1038/s41592-025-02849-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2205-2210},
  shortjournal = {Nat. Meth.},
  title        = {Spatiotemporal focusing enables all-optical in situ histology of heterogeneous tissue},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Kilohertz volumetric imaging of in vivo dynamics using squeezed light field microscopy. <em>NMETH</em>, <em>22</em>(10), 2194-2204. (<a href='https://doi.org/10.1038/s41592-025-02843-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volumetric functional imaging of transient cellular signaling and motion dynamics is often limited by hardware bandwidth and the scarcity of photons under short exposures. To overcome these challenges, we introduce squeezed light field microscopy (SLIM), a computational imaging approach that rapidly captures high-resolution three-dimensional light signals using only a single, low-format camera sensor. SLIM records over 1,000 volumes per second across a 550-µm diameter field of view and 300-µm depth, achieving 3.6-µm lateral and 6-µm axial resolution. Here we demonstrate its utility in blood cell velocimetry within the embryonic zebrafish brain and in freely moving tails undergoing high-frequency swings. Millisecond-scale temporal resolution further enables precise voltage imaging of neural membrane potentials in the leech ganglion and hippocampus of behaving mice. Together, these results establish SLIM as a versatile and robust tool for high-speed volumetric microscopy across diverse biological systems. Squeezed light field microscopy (SLIM) combines ideas from tomography and compressed sensing with light field microscopy to enable volumetric imaging at kilohertz rates, as demonstrated in blood flow imaging in zebrafish and voltage imaging in leeches and mice.},
  archive      = {J_NMETH},
  author       = {Wang, Zhaoqiang and Zhao, Ruixuan and Wagenaar, Daniel A. and Espino, Diego and Sheintuch, Liron and Benshlomo, Ohr and Kang, Wenjun and Zhu, Enbo and Lee, Calvin K. and Schmidt, William C. and Pammar, Aryan and Wang, Jing and Wong, Gerard C. L. and Liang, Rongguang and Golshani, Peyman and Hsiai, Tzung K. and Gao, Liang},
  doi          = {10.1038/s41592-025-02843-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2194-2204},
  shortjournal = {Nat. Meth.},
  title        = {Kilohertz volumetric imaging of in vivo dynamics using squeezed light field microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Highly multiplexed 3D profiling of cell states and immune niches in human tumors. <em>NMETH</em>, <em>22</em>(10), 2180-2193. (<a href='https://doi.org/10.1038/s41592-025-02824-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diseases such as cancer involve alterations in cell proportions, states and interactions, as well as complex changes in tissue morphology and architecture. Histopathological diagnosis of disease and most multiplexed spatial profiling relies on inspecting thin (4–5 µm) specimens. Here we describe a high-plex cyclic immunofluorescence method for three-dimensional tissue imaging and use it to show that few, if any, cells are intact in conventional thin tissue sections, reducing the accuracy of cell phenotyping and interaction analysis. However, three-dimensional cyclic immunofluorescence of sections eightfold to tenfold thicker enables accurate morphological assessment of diverse protein markers in intact tumor, immune and stromal cells. Moreover, the high resolution of this confocal approach generates images of cells in a preserved tissue environment at a level of detail previously limited to cell culture. Precise imaging of cell membranes also makes it possible to detect and map cell–cell contacts and juxtracrine signaling complexes in immune cell niches. Confocal microscopy enables high-resolution, high-plex 3D cyclic immunofluorescence of 30- to 50-µm-thick tissue sections. The approach allows for rich phenotypic assessments of intact cells and intercellular interactions with subcellular resolution.},
  archive      = {J_NMETH},
  author       = {Yapp, Clarence and Nirmal, Ajit J. and Zhou, Felix and Wong, Alex Y. H. and Tefft, Juliann B. and Lu, Yi Daniel and Shang, Zhiguo and Maliga, Zoltan and Llopis, Paula Montero and Murphy, George F. and Lian, Christine G. and Danuser, Gaudenz and Santagata, Sandro and Sorger, Peter K.},
  doi          = {10.1038/s41592-025-02824-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2180-2193},
  shortjournal = {Nat. Meth.},
  title        = {Highly multiplexed 3D profiling of cell states and immune niches in human tumors},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fourier-based three-dimensional multistage transformer for aberration correction in multicellular specimens. <em>NMETH</em>, <em>22</em>(10), 2171-2179. (<a href='https://doi.org/10.1038/s41592-025-02844-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-resolution tissue imaging is often compromised by sample-induced optical aberrations that degrade resolution and contrast. Although wavefront sensor-based adaptive optics (AO) can measure these aberrations, such hardware solutions are typically complex, expensive to implement and slow when serially mapping spatially varying aberrations across large fields of view. Here we introduce AOViFT (adaptive optical vision Fourier transformer)—a machine learning-based aberration sensing framework built around a three-dimensional multistage vision transformer that operates on Fourier domain embeddings. AOViFT infers aberrations and restores diffraction-limited performance in puncta-labeled specimens with substantially reduced computational cost, training time and memory footprint compared to conventional architectures or real-space networks. We validated AOViFT on live gene-edited zebrafish embryos, demonstrating its ability to correct spatially varying aberrations using either a deformable mirror or postacquisition deconvolution. By eliminating the need for the guide star and wavefront sensing hardware and simplifying the experimental workflow, AOViFT lowers technical barriers for high-resolution volumetric microscopy across diverse biological samples. Adaptive optical vision Fourier transformer (AOViFT) is a machine learning-based framework for accurately inferring aberrations and restoring diffraction-limited performance in diverse biological specimens.},
  archive      = {J_NMETH},
  author       = {Alshaabi, Thayer and Milkie, Daniel E. and Liu, Gaoxiang and Shirazinejad, Cyna and Hong, Jason L. and Achour, Kemal and Görlitz, Frederik and Milunovic-Jevtic, Ana and Simmons, Cat and Abuzahriyeh, Ibrahim S. and Hong, Erin and Williams, Samara Erin and Harrison, Nathanael and Huang, Evan and Bae, Eun Seok and Killilea, Alison N. and Swinburne, Ian A. and Drubin, David G. and Upadhyayula, Srigokul and Betzig, Eric},
  doi          = {10.1038/s41592-025-02844-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2171-2179},
  shortjournal = {Nat. Meth.},
  title        = {Fourier-based three-dimensional multistage transformer for aberration correction in multicellular specimens},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Automated classification of cellular expression in multiplexed imaging data with nimbus. <em>NMETH</em>, <em>22</em>(10), 2161-2170. (<a href='https://doi.org/10.1038/s41592-025-02826-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed imaging offers a powerful approach to characterize the spatial topography of tissues in both health and disease. To analyze such data, the specific combination of markers that are present in each cell must be enumerated to enable accurate phenotyping, a process that often relies on unsupervised clustering. We constructed the Pan-Multiplex (Pan-M) dataset containing 197 million distinct annotations of marker expression across 15 different cell types. We used Pan-M to create Nimbus, a deep learning model to predict marker positivity from multiplexed image data. Nimbus is a pretrained model that uses the underlying images to classify marker expression of individual cells as positive or negative across distinct cell types, from different tissues, acquired using different microscope platforms, without requiring any retraining. We demonstrate that Nimbus predictions capture the underlying staining patterns of the full diversity of markers present in Pan-M, and that Nimbus matches or exceeds the accuracy of previous approaches that must be retrained on each dataset. We then show how Nimbus predictions can be integrated with downstream clustering algorithms to robustly identify cell subtypes in image data. We have open-sourced Nimbus and Pan-M to enable community use at https://github.com/angelolab/Nimbus-Inference . Nimbus, a deep learning model, uses a large multiplexed imaging dataset to predict the likelihood of marker positivity in single cells.},
  archive      = {J_NMETH},
  author       = {Rumberger, Josef Lorenz and Greenwald, Noah F. and Ranek, Jolene S. and Boonrat, Potchara and Walker, Cameron and Franzen, Jannik and Varra, Sricharan Reddy and Kong, Alex and Sowers, Cameron and Liu, Candace C. and Averbukh, Inna and Piyadasa, Hadeesha and Vanguri, Rami and Nederlof, Iris and Wang, Xuefei Julie and Van Valen, David and Kok, Marleen and Bendall, Sean C. and Hollmann, Travis J. and Kainmueller, Dagmar and Angelo, Michael},
  doi          = {10.1038/s41592-025-02826-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2161-2170},
  shortjournal = {Nat. Meth.},
  title        = {Automated classification of cellular expression in multiplexed imaging data with nimbus},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Foundation model for efficient biological discovery in single-molecule time traces. <em>NMETH</em>, <em>22</em>(10), 2149-2160. (<a href='https://doi.org/10.1038/s41592-025-02839-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule fluorescence microscopy (SMFM) can reveal important biological insights. However, uncovering rare but critical intermediates often demands manual inspection of time traces and iterative ad hoc approaches. To facilitate systematic and efficient discovery from SMFM time traces, we introduce META-SiM, a transformer-based foundation model pretrained on diverse SMFM analysis tasks. META-SiM rivals best-in-class algorithms on a broad range of tasks including trace classification, segmentation, idealization and stepwise photobleaching analysis. Additionally, the model produces embeddings that encapsulate detailed information about each trace, which the web-based META-SiM Projector ( https://www.simol-projector.org ) casts into lower-dimensional space for efficient whole-dataset visualization, labeling, comparison and sharing. Combining this Projector with the objective metric of local Shannon entropy enables rapid identification of condition-specific behaviors, even if rare or subtle. Applying META-SiM to an existing single-molecule Förster resonance energy transfer dataset, we discover a previously undetected intermediate state in pre-mRNA splicing. META-SiM removes bottlenecks, improves objectivity and both systematizes and accelerates biological discovery in single-molecule data. META-SiM brings foundation model power to single-molecule time traces, excelling across diverse analysis tasks. Paired with the web-based META-SiM Projector and entropy mapping, it rapidly reveals hidden molecular behaviors inaccessible by other means.},
  archive      = {J_NMETH},
  author       = {Li, Jieming and Zhang, Leyou and Johnson-Buck, Alexander and Walter, Nils G.},
  doi          = {10.1038/s41592-025-02839-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2149-2160},
  shortjournal = {Nat. Meth.},
  title        = {Foundation model for efficient biological discovery in single-molecule time traces},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dose-efficient cryo-electron microscopy for thick samples using tilt- corrected scanning transmission electron microscopy. <em>NMETH</em>, <em>22</em>(10), 2138-2148. (<a href='https://doi.org/10.1038/s41592-025-02834-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron microscopy is a powerful tool in structural biology. In thick specimens, challenges arise as an exponentially larger fraction of the transmitted electrons lose energy from inelastic scattering and can no longer be properly focused as a result of chromatic aberrations in the post-specimen optics. Rather than filtering out the inelastic scattering at the price of reducing potential signal, as is done in energy-filtered transmission electron microscopy, we show how a dose-efficient and unfiltered image can be rapidly obtained using tilt-corrected bright-field scanning transmission electron microscopy data collected on a pixelated detector. Enhanced contrast and a 3–5× improvement in dose efficiency are observed for two-dimensional images of intact bacterial cells and large organelles using tilt-corrected bright-field scanning transmission electron microscopy compared to energy-filtered transmission electron microscopy for thicknesses beyond 500 nm. As a proof of concept for the technique’s performance in structural determination, we present a single-particle analysis map at sub-nanometer resolution for a highly symmetric virus-like particle determined from 789 particles. Tilt-corrected bright-field scanning transmission electron microscopy offers enhanced cryogenic electron microscopy contrast and substantial improvement in dose efficiency for thick samples such as bacterial cells and large organelles, while still being able to perform single-particle analysis.},
  archive      = {J_NMETH},
  author       = {Yu, Yue and Spoth, Katherine A. and Colletta, Michael and Nguyen, Kayla X. and Zeltmann, Steven E. and Zhang, Xiyue S. and Paraan, Mohammadreza and Kopylov, Mykhailo and Dubbeldam, Charlie and Serwas, Daniel and Siems, Hannah and Muller, David A. and Kourkoutis, Lena F.},
  doi          = {10.1038/s41592-025-02834-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2138-2148},
  shortjournal = {Nat. Meth.},
  title        = {Dose-efficient cryo-electron microscopy for thick samples using tilt- corrected scanning transmission electron microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Uncovering hidden protein modifications with native top-down mass spectrometry. <em>NMETH</em>, <em>22</em>(10), 2127-2137. (<a href='https://doi.org/10.1038/s41592-025-02846-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein modifications drive dynamic cellular processes by modulating biomolecular interactions, yet capturing these modifications within their native structural context remains a significant challenge. Native top-down mass spectrometry promises to preserve the critical link between modifications and interactions. However, current methods often fail to detect uncharacterized or low-abundance modifications, limiting insights into proteoform diversity. To address this gap, we introduce precise and accurate Identification Of Native proteoforms (precisION), an interactive end-to-end software package that leverages a robust, data-driven fragment-level open search to detect, localize and quantify ‘hidden’ modifications within intact protein complexes. Applying precisION to four therapeutically relevant targets—PDE6, ACE2, osteopontin (SPP1) and a GABA transporter (GAT1)—we discover undocumented phosphorylation, glycosylation and lipidation, and resolve previously uninterpretable density in an electron cryo-microscopy map of GAT1. As an open-source software package, precisION offers an intuitive means for interpreting complex protein fragmentation data. This tool will empower the community to unlock the potential of native top-down mass spectrometry, advancing integrative structural biology, molecular pathology and drug development. precisION discovers, localizes and quantifies protein modifications within complex proteoform assemblies through data-driven analysis of native top-down mass spectra.},
  archive      = {J_NMETH},
  author       = {Bennett, Jack L. and El-Baba, Tarick J. and Zouboulis, Konstantin C. and Kirschbaum, Carla and Song, Haigang and Butroid, Frances I. and Benesch, Justin L. P. and Lutomski, Corinne A. and Robinson, Carol V.},
  doi          = {10.1038/s41592-025-02846-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2127-2137},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering hidden protein modifications with native top-down mass spectrometry},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Merging conformational landscapes in a single consensus space with FlexConsensus algorithm. <em>NMETH</em>, <em>22</em>(10), 2118-2126. (<a href='https://doi.org/10.1038/s41592-025-02841-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural heterogeneity analysis in cryogenic electron microscopy is experiencing a breakthrough in estimating more accurate, richer and interpretable conformational landscapes derived from experimental data. The emergence of new methods designed to tackle the heterogeneity challenge reflects this new paradigm, enabling users to gain a better understanding of protein dynamics. However, the question of how intrinsically different heterogeneity algorithms compare remains unsolved, which is crucial for determining the reliability, stability and correctness of the estimated conformational landscapes. Here, to overcome the previous challenge, we introduce FlexConsenus: a multi-autoencoder neural network able to learn the commonalities and differences among several conformational landscapes, enabling them to be placed in a shared consensus space with enhanced reliability. The consensus space enables the measurement of reproducibility in heterogeneity estimations, allowing users to either focus their analysis on particles with a stable estimation of their structural variability or concentrate on specific particle subsets detected by only certain methods. FlexConsensus is a multi-autoencoder-based algorithm for merging different conformational landscapes from cryogenic electron microscopy heterogeneity analysis into a common latent space for the identification of similarities and differences among various methods. This helps in the validation of estimated conformational landscape and provides tools to streamline the heterogeneity workflow.},
  archive      = {J_NMETH},
  author       = {Herreros, David and Perez Mata, Carlos and Sanchez Sorzano, Carlos Oscar and Carazo, Jose Maria},
  doi          = {10.1038/s41592-025-02841-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2118-2126},
  shortjournal = {Nat. Meth.},
  title        = {Merging conformational landscapes in a single consensus space with FlexConsensus algorithm},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). InterPLM: Discovering interpretable features in protein language models via sparse autoencoders. <em>NMETH</em>, <em>22</em>(10), 2107-2117. (<a href='https://doi.org/10.1038/s41592-025-02836-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite their success in protein modeling and design, the internal mechanisms of protein language models (PLMs) are poorly understood. Here we present a systematic framework to extract and analyze interpretable features from PLMs using sparse autoencoders. Training sparse autoencoders on ESM-2 embeddings, we identify thousands of interpretable features highlighting biological concepts including binding sites, structural motifs and functional domains. Individual neurons show considerably less conceptual alignment, suggesting PLMs store concepts in superposition. This superposition persists across model scales and larger PLMs capture more interpretable concepts. Beyond known annotations, ESM-2 learns coherent patterns across evolutionarily distinct protein families. To systematically analyze these numerous features, we developed an automated interpretation approach using large language models for feature description and validation. As practical applications, these features can accurately identify missing database annotations and enable targeted steering of sequence generation. Our results show PLM representations can be decomposed into interpretable components, demonstrating the feasibility and utility of mechanistically interpreting these models. InterPLM is a computational framework to extract and analyze interpretable features from protein language models using sparse autoencoders. By training sparse autoencoders on ESM-2 embeddings, this study identifies thousands of interpretable biological features learned by the different layers of the ESM-2 model.},
  archive      = {J_NMETH},
  author       = {Simon, Elana and Zou, James},
  doi          = {10.1038/s41592-025-02836-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2107-2117},
  shortjournal = {Nat. Meth.},
  title        = {InterPLM: Discovering interpretable features in protein language models via sparse autoencoders},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). All-at-once RNA folding with 3D motif prediction framed by evolutionary information. <em>NMETH</em>, <em>22</em>(10), 2094-2106. (<a href='https://doi.org/10.1038/s41592-025-02833-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural RNAs exhibit a vast array of recurrent short three-dimensional (3D) elements found in loop regions involving non-Watson–Crick interactions that help arrange canonical double helices into tertiary structures. Here we present CaCoFold-R3D, a probabilistic grammar that predicts these RNA 3D motifs (also termed modules) jointly with RNA secondary structure over a sequence or alignment. CaCoFold-R3D uses evolutionary information present in an RNA alignment to reliably identify canonical helices (including pseudoknots) by covariation. Here we further introduce the R3D grammars, which also exploit helix covariation that constrains the positioning of the mostly noncovarying RNA 3D motifs. Our method runs predictions over an almost-exhaustive list of over 50 known RNA motifs (‘everything’). Motifs can appear in any nonhelical loop region (including three-way, four-way and higher junctions) (‘everywhere’). All structural motifs as well as the canonical helices are arranged into one single structure predicted by one single joint probabilistic grammar (‘all-at-once’). Our results demonstrate that CaCoFold-R3D is a valid alternative for predicting the all-residue interactions present in a RNA 3D structure. CaCoFold-R3D is fast and easily customizable for novel motif discovery and shows promising value both as a strong input for deep learning approaches to all-atom structure prediction as well as toward guiding RNA design as drug targets for therapeutic small molecules. CaCoFold-R3D is a probabilistic model that simultaneously predicts the RNA 3D motifs jointly with the secondary structure in a structural RNA using evolutionary information.},
  archive      = {J_NMETH},
  author       = {Karan, Aayush and Rivas, Elena},
  doi          = {10.1038/s41592-025-02833-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2094-2106},
  shortjournal = {Nat. Meth.},
  title        = {All-at-once RNA folding with 3D motif prediction framed by evolutionary information},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Coupling CRISPR scanning with targeted chromatin accessibility profiling using a double-stranded DNA deaminase. <em>NMETH</em>, <em>22</em>(10), 2083-2093. (<a href='https://doi.org/10.1038/s41592-025-02811-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing enables sequence-function profiling of endogenous cis-regulatory elements, driving understanding of their mechanisms. However, these approaches lack direct, scalable readouts of chromatin accessibility across long single-molecule chromatin fibers. Here we leverage double-stranded DNA cytidine deaminases to profile chromatin accessibility at endogenous loci of interest through targeted PCR and long-read sequencing, a method we term targeted deaminase-accessible chromatin sequencing (TDAC-seq). With high sequence coverage at targeted loci, TDAC-seq can be integrated with CRISPR perturbations to link genetic edits and their effects on chromatin accessibility on the same single chromatin fiber at single-nucleotide resolution. We employed TDAC-seq to parse CRISPR edits that activate fetal hemoglobin in human CD34+ hematopoietic stem and progenitor cells (HSPCs) during erythroid differentiation as well as in pooled CRISPR and base-editing screens tiling an enhancer controlling the globin locus. We further scaled the method to interrogate 947 variants in a GFI1B-linked enhancer associated with myeloproliferative neoplasm risk in a single pooled CRISPR experiment in CD34+ HSPCs. Together, TDAC-seq enables high-resolution sequence-function mapping of single-molecule chromatin fibers by genome editing. This paper presents TDAC-seq, a targeted chromatin-accessibility-profiling method using cytidine deaminases and long-read sequencing, to resolve the effects of CRISPR edits on single chromatin fibers.},
  archive      = {J_NMETH},
  author       = {Roh, Heejin and Shen, Simon P. and Hu, Yan and Kwok, Hui Si and Siegenfeld, Allison P. and Lee, Ceejay and Zepeda, Marcanthony U. and Guo, Chun-Jie and Roseman, Shelby A. and Comenho, Caroline and Sankaran, Vijay G. and Buenrostro, Jason D. and Liau, Brian B.},
  doi          = {10.1038/s41592-025-02811-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2083-2093},
  shortjournal = {Nat. Meth.},
  title        = {Coupling CRISPR scanning with targeted chromatin accessibility profiling using a double-stranded DNA deaminase},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Common to rare transfer learning (CORAL) enables inference and prediction for a quarter million rare malagasy arthropods. <em>NMETH</em>, <em>22</em>(10), 2074-2082. (<a href='https://doi.org/10.1038/s41592-025-02823-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA-based biodiversity surveys result in massive-scale data, including up to millions of species—of which, most are rare. Making the most of such data for inference and prediction requires modeling approaches that can relate species occurrences to environmental and spatial predictors, while incorporating information about their taxonomic or phylogenetic placement. Even if the scalability of joint species distribution models to large communities has greatly advanced, incorporating hundreds of thousands of species has not been feasible to date, leading to compromised analyses. Here we present a ‘common to rare transfer learning’ (CORAL) approach, based on borrowing information from the common species to enable statistically and computationally efficient modeling of both common and rare species. We illustrate that CORAL leads to much improved prediction and inference in the context of DNA metabarcoding data from Madagascar, comprising 255,188 arthropod species detected in 2,874 samples. CORAL can infer the occurrence of rare species based on common species, using DNA metabarcoding data or other high-dimensional biodiversity data. The approach is illustrated on a large-scale biodiversity survey from Madagascar.},
  archive      = {J_NMETH},
  author       = {Ovaskainen, Otso and Winter, Steven and Tikhonov, Gleb and Abrego, Nerea and Anslan, Sten and deWaard, Jeremy R. and deWaard, Stephanie L. and Fisher, Brian L. and Furneaux, Brendan and Hardwick, Bess and Kerdraon, Deirdre and Pentinsaari, Mikko and Raharinjanahary, Dimby and Rajoelison, Eric Tsiriniaina and Ratnasingham, Sujeevan and Somervuo, Panu and Sones, Jayme E. and Zakharov, Evgeny V. and Hebert, Paul D. N. and Roslin, Tomas and Dunson, David},
  doi          = {10.1038/s41592-025-02823-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2074-2082},
  shortjournal = {Nat. Meth.},
  title        = {Common to rare transfer learning (CORAL) enables inference and prediction for a quarter million rare malagasy arthropods},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). De novo discovery of conserved gene clusters in microbial genomes with spacedust. <em>NMETH</em>, <em>22</em>(10), 2065-2073. (<a href='https://doi.org/10.1038/s41592-025-02816-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metagenomics has revolutionized environmental and human-associated microbiome studies. However, the limited fraction of proteins with known biological processes and molecular functions presents a major bottleneck. In prokaryotes and viruses, evolution favors keeping genes participating in the same biological processes colocalized as conserved gene clusters. Conversely, conservation of gene neighborhood indicates functional association. Here we present Spacedust, a tool for systematic, de novo discovery of conserved gene clusters. To find homologous protein matches, Spacedust uses fast and sensitive structure comparison with Foldseek. Partially conserved clusters are detected using novel clustering and order conservation P values. We demonstrate Spacedust’s sensitivity with an all-versus-all analysis of 1,308 bacterial genomes, identifying 72,843 conserved gene clusters containing 58% of the 4.2 million genes. It recovered 95% of antiviral defense system clusters annotated by the specialized tool PADLOC. Spacedust’s high sensitivity and speed will facilitate the annotation of large numbers of sequenced bacterial, archaeal and viral genomes. This work presents Spacedust, a tool for de novo identification of conserved gene clusters from metagenomic data.},
  archive      = {J_NMETH},
  author       = {Zhang, Ruoshi and Mirdita, Milot and Söding, Johannes},
  doi          = {10.1038/s41592-025-02816-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2065-2073},
  shortjournal = {Nat. Meth.},
  title        = {De novo discovery of conserved gene clusters in microbial genomes with spacedust},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Giotto suite: A multiscale and technology-agnostic spatial multiomics analysis ecosystem. <em>NMETH</em>, <em>22</em>(10), 2052-2064. (<a href='https://doi.org/10.1038/s41592-025-02817-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emerging spatial multiomics technologies provide an increasingly large amount of information content at multiple scales. However, it remains challenging to efficiently represent and harmonize diverse spatial datasets. Here we present Giotto Suite, a suite of modular packages that provides scalable and extensible end-to-end solutions for multiscale and multiomic data analysis, integration and visualization. At its core, Giotto Suite is centered around an innovative data framework, allowing the representation and integration of spatial omics data in a technology-agnostic manner. Giotto Suite integrates molecular, morphology, spatial and annotated feature information to create a responsive and flexible workflow, as demonstrated by applications to several state-of-the-art spatial technologies. Furthermore, Giotto Suite builds upon interoperable interfaces and data structures that bridge the established fields of genomics and spatial data science in R, thereby enabling independent developers to create custom-engineered pipelines. As such, Giotto Suite creates an immersive and multiscale ecosystem for spatial multiomic data analysis. Giotto Suite provides a comprehensive, flexible and scalable platform for technology-agnostic spatial omics analysis using R.},
  archive      = {J_NMETH},
  author       = {Chen, Jiaji G. and Chávez-Fuentes, Joselyn C. and O’Brien, Matthew and Xu, Junxiang and Ruiz, Edward C. and Wang, Wen and Amin, Iqra and Sheridan, Jeffrey P. and Shin, Sujung C. and Hasyagar, Sanjana V. and Sarfraz, Irzam and Guckhool, Pratishtha and Sistig, Adriana and Jarzabek, Veronica and Yuan, Guo-Cheng and Dries, Ruben},
  doi          = {10.1038/s41592-025-02817-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2052-2064},
  shortjournal = {Nat. Meth.},
  title        = {Giotto suite: A multiscale and technology-agnostic spatial multiomics analysis ecosystem},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell multi-omic detection of DNA methylation and histone modifications reconstructs the dynamics of epigenomic maintenance. <em>NMETH</em>, <em>22</em>(10), 2042-2051. (<a href='https://doi.org/10.1038/s41592-025-02847-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA methylation and histone modifications encode epigenetic information. Recently, major progress was made to measure either mark at a single-cell resolution; however, a method for simultaneous detection is lacking, preventing study of their interactions. Here, to bridge this gap, we developed scEpi2-seq. Our technique provides a readout of histone modifications and DNA methylation at the single-cell and single-molecule level. Application in a cell line with the FUCCI cell cycle reporter system reveals how DNA methylation maintenance is influenced by the local chromatin context. In addition, profiling of H3K27me3 and DNA methylation in the mouse intestine yields insights into epigenetic interactions during cell type specification. Differentially methylated regions also demonstrated independent cell-type regulation in addition to H3K27me3 regulation, which reinforces that CpG methylation acts as an additional layer of control in facultative heterochromatin. This work presents scEpi2-seq, a method for simultaneous single-cell profiling of DNA methylation and histone modifications, enabling direct investigation of the interplay between these two epigenomic marks.},
  archive      = {J_NMETH},
  author       = {Geisenberger, Christoph and van den Berg, Jeroen and van Batenburg, Vincent and de Barbanson, Buys and Lyubimova, Anna and Verity-Legg, Joe and Chen, Xiufei and Liu, Yibin and Song, Chun-Xiao and de Ridder, Jeroen and van Oudenaarden, Alexander},
  doi          = {10.1038/s41592-025-02847-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2042-2051},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell multi-omic detection of DNA methylation and histone modifications reconstructs the dynamics of epigenomic maintenance},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing. <em>NMETH</em>, <em>22</em>(10), 2032-2041. (<a href='https://doi.org/10.1038/s41592-025-02805-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variants (both coding and noncoding) can impact gene function and expression, driving disease mechanisms such as cancer progression. The systematic study of endogenous genetic variants is hindered by inefficient precision editing tools, combined with technical limitations in confidently linking genotypes to gene expression at single-cell resolution. We developed single-cell DNA–RNA sequencing (SDR-seq) to simultaneously profile up to 480 genomic DNA loci and genes in thousands of single cells, enabling accurate determination of coding and noncoding variant zygosity alongside associated gene expression changes. Using SDR-seq, we associate coding and noncoding variants with distinct gene expression in human induced pluripotent stem cells. Furthermore, we demonstrate that in primary B cell lymphoma samples, cells with a higher mutational burden exhibit elevated B cell receptor signaling and tumorigenic gene expression. SDR-seq provides a powerful platform to dissect regulatory mechanisms encoded by genetic variants, advancing our understanding of gene expression regulation and its implications for disease. This study introduces SDR-seq, a droplet-based single-cell DNA–RNA sequencing platform, enabling the study of gene expression profiles linked to both noncoding and coding variants.},
  archive      = {J_NMETH},
  author       = {Lindenhofer, Dominik and Bauman, Julia R. and Hawkins, John A. and Fitzgerald, Donnacha and Yildiz, Umut and Jung, Haeyeon and Korosteleva, Anastasiia and Marttinen, Mikael and Kueblbeck, Moritz and Zaugg, Judith B. and Noh, Kyung-Min and Dietrich, Sascha and Huber, Wolfgang and Stegle, Oliver and Steinmetz, Lars M.},
  doi          = {10.1038/s41592-025-02805-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2032-2041},
  shortjournal = {Nat. Meth.},
  title        = {Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MSnLib: Efficient generation of open multi-stage fragmentation mass spectral libraries. <em>NMETH</em>, <em>22</em>(10), 2028-2031. (<a href='https://doi.org/10.1038/s41592-025-02813-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Untargeted high-resolution mass spectrometry is a key tool in clinical metabolomics, natural product discovery and exposomics, with compound identification remaining the major bottleneck. Currently, the standard workflow applies spectral library matching against tandem mass spectrometry (MS2) fragmentation data. Multi-stage fragmentation (MSn) yields more profound insights into substructures, enabling validation of fragmentation pathways; however, the community lacks open MSn reference data of diverse natural products and other chemicals. Here we describe MSnLib, a machine learning-ready open resource of &gt;2 million spectra in MSn trees of 30,008 unique small molecules, built with a high-throughput data acquisition and processing pipeline in the open-source software mzmine. MSnLib is a large-scale, open MSn spectral library featuring &gt;2.3 million MSn and &gt;357,000 MS2 spectra for 30,008 unique small molecules.},
  archive      = {J_NMETH},
  author       = {Brungs, Corinna and Schmid, Robin and Heuckeroth, Steffen and Mazumdar, Aninda and Drexler, Matúš and Šácha, Pavel and Dorrestein, Pieter C. and Petras, Daniel and Nothias, Louis-Felix and Veverka, Václav and Nencka, Radim and Kameník, Zdeněk and Pluskal, Tomáš},
  doi          = {10.1038/s41592-025-02813-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2028-2031},
  shortjournal = {Nat. Meth.},
  title        = {MSnLib: Efficient generation of open multi-stage fragmentation mass spectral libraries},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). GPU-accelerated homology search with MMseqs2. <em>NMETH</em>, <em>22</em>(10), 2024-2027. (<a href='https://doi.org/10.1038/s41592-025-02819-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapidly growing protein databases demand faster sensitive search tools. Here the graphics processing unit (GPU)-accelerated MMseqs2 delivers 6× faster single-protein searches than CPU methods on 2 × 64 cores, speeds previously requiring large protein batches. For larger query batches, it is the most cost-effective solution, outperforming the fastest alternative method by 2.4-fold with eight GPUs. It accelerates protein structure prediction with ColabFold 31.8× over the standard AlphaFold2 pipeline and protein structure search with Foldseek by 4–27×. MMseqs2-GPU is available under an open-source license at https://mmseqs.com/ . Graphics processing unit-accelerated MMseqs2 offers tremendous speedups for homology retrieval from metagenomic databases, query-centered multiple sequence alignment generation for structure prediction, and structural searches with Foldseek.},
  archive      = {J_NMETH},
  author       = {Kallenborn, Felix and Chacon, Alejandro and Hundt, Christian and Sirelkhatim, Hassan and Didi, Kieran and Cha, Sooyoung and Dallago, Christian and Mirdita, Milot and Schmidt, Bertil and Steinegger, Martin},
  doi          = {10.1038/s41592-025-02819-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2024-2027},
  shortjournal = {Nat. Meth.},
  title        = {GPU-accelerated homology search with MMseqs2},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Machine learning for accelerating discovery from single-molecule data. <em>NMETH</em>, <em>22</em>(10), 2022-2023. (<a href='https://doi.org/10.1038/s41592-025-02840-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Manual analysis of single-molecule time traces is slow and subjective. Now, a transformer-based foundation model — META-SiM —automates key analysis tasks across diverse datasets and enables rapid, systematic discovery of subtle single-molecule behaviors. Application of this approach reveals a previously undetected pre-mRNA splicing intermediate, highlighting its potential to streamline biological discovery.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02840-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2022-2023},
  shortjournal = {Nat. Meth.},
  title        = {Machine learning for accelerating discovery from single-molecule data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Joint profiling of chromatin accessibility and CRISPR edits via double-stranded DNA deaminases. <em>NMETH</em>, <em>22</em>(10), 2020-2021. (<a href='https://doi.org/10.1038/s41592-025-02812-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted deaminase-accessible chromatin sequencing (TDAC-seq) measures chromatin accessibility across long chromatin fibers at targeted loci using double-stranded DNA cytidine deaminases. When combined with pooled CRISPR mutational screening, TDAC-seq enables the high-throughput detection of changes in chromatin accessibility following CRISPR perturbations, allowing fine mapping of sequence–function relationships within endogenous cis-regulatory elements.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02812-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2020-2021},
  shortjournal = {Nat. Meth.},
  title        = {Joint profiling of chromatin accessibility and CRISPR edits via double-stranded DNA deaminases},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What does a language model know about proteins?. <em>NMETH</em>, <em>22</em>(10), 2017-2019. (<a href='https://doi.org/10.1038/s41592-025-02837-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new approach sheds light on the biological features learned by protein language models, promising greater interpretability for unsupervised sequence learning.},
  archive      = {J_NMETH},
  author       = {Ruffolo, Jeffrey A.},
  doi          = {10.1038/s41592-025-02837-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2017-2019},
  shortjournal = {Nat. Meth.},
  title        = {What does a language model know about proteins?},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cryogenic STEM of thick biological specimens. <em>NMETH</em>, <em>22</em>(10), 2015-2016. (<a href='https://doi.org/10.1038/s41592-025-02818-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A cryogenic scanning transmission electron microscopy (STEM) approach for analyzing thick biological specimens expands the reach of cryo-electron microscopy.},
  archive      = {J_NMETH},
  author       = {Sachse, Carsten},
  doi          = {10.1038/s41592-025-02818-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2015-2016},
  shortjournal = {Nat. Meth.},
  title        = {Cryogenic STEM of thick biological specimens},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Choose your human genome reference wisely. <em>NMETH</em>, <em>22</em>(10), 2010-2014. (<a href='https://doi.org/10.1038/s41592-025-02850-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists can choose between multiple human genome references, and a pangenome reference is coming. Deciding what to use when is not quite straightforward.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02850-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2010-2014},
  shortjournal = {Nat. Meth.},
  title        = {Choose your human genome reference wisely},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep-sea imaging of octopus locomotion. <em>NMETH</em>, <em>22</em>(10), 2009. (<a href='https://doi.org/10.1038/s41592-025-02875-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Economou, Andrew},
  doi          = {10.1038/s41592-025-02875-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2009},
  shortjournal = {Nat. Meth.},
  title        = {Deep-sea imaging of octopus locomotion},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Retracing tumor evolution. <em>NMETH</em>, <em>22</em>(10), 2009. (<a href='https://doi.org/10.1038/s41592-025-02876-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-025-02876-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2009},
  shortjournal = {Nat. Meth.},
  title        = {Retracing tumor evolution},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ionic liquids clear tissue without distortion. <em>NMETH</em>, <em>22</em>(10), 2008. (<a href='https://doi.org/10.1038/s41592-025-02873-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Whalley, Katherine},
  doi          = {10.1038/s41592-025-02873-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2008},
  shortjournal = {Nat. Meth.},
  title        = {Ionic liquids clear tissue without distortion},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). BioEmu is a biomolecular emulator for sampling protein structure ensembles. <em>NMETH</em>, <em>22</em>(10), 2008. (<a href='https://doi.org/10.1038/s41592-025-02874-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02874-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2008},
  shortjournal = {Nat. Meth.},
  title        = {BioEmu is a biomolecular emulator for sampling protein structure ensembles},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancing prime editing via AI-guided binder design. <em>NMETH</em>, <em>22</em>(10), 2007. (<a href='https://doi.org/10.1038/s41592-025-02872-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AI-designed binders can enhance prime editing performance by inhibiting the DNA mismatch repair pathway.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-025-02872-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2007},
  shortjournal = {Nat. Meth.},
  title        = {Enhancing prime editing via AI-guided binder design},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Translon: A single term for translated regions. <em>NMETH</em>, <em>22</em>(10), 2002-2006. (<a href='https://doi.org/10.1038/s41592-025-02810-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Świrski, Michał I. and Tierney, Jack A. S. and Albà, M. Mar and Andreev, Dmitry E. and Aspden, Julie L. and Atkins, John F. and Bassani-Sternberg, Michal and Berry, Marla J. and Biffo, Stefano and Boris-Lawrie, Kathleen and Borodovsky, Mark and Brierley, Ian and Brook, Matthew and Brunet, Marie A. and Bujnicki, Janusz M. and Caliskan, Neva and Calviello, Lorenzo and Carvunis, Anne-Ruxandra and Cate, Jamie H. D. and Cenik, Can and Chang, Kung Yao and Chen, Yiwen and Chothani, Sonia and Choudhary, Jyoti S. and Clark, Patricia L. and Clauwaert, Jim and Cooley, Lynn and Dassi, Erik and Dean, Kellie and Diaz, Jean-Jacques and Dieterich, Christoph and Dikstein, Rivka and Dinman, Jonathan D. and Dmitriev, Sergey E. and Dontsova, Olga A. and Dunham, Christine M. and Eswarappa, Sandeep M. and Farabaugh, Philip J. and Faridi, Pouya and Fierro-Monti, Ivo and Firth, Andrew E. and Gatfield, David and Gebauer, Fátima and Gelfand, Mikhail S. and Gray, Nicola K. and Green, Rachel and Hill, Chris H. and Hou, Ya-Ming and Hübner, Norbert and Ignatova, Zoya and Ivanov, Pavel and Iwasaki, Shintaro and Johnson, Rory and Jomaa, Ahmad and Jovanovic, Marko and Jungreis, Irwin and Kellis, Manolis and Kieft, Jeffrey S. and Kochetov, Alex V. and Koonin, Eugene V. and Korostelev, Andrei A. and Kufel, Joanna and Kulakovskiy, Ivan V. and Kurian, Leo and Lafontaine, Denis L. J. and Larsson, Ola and Loughran, Gary and Lukeš, Julius and Mariotti, Marco and Martens-Uzunova, Elena S. and Martinez, Thomas F. and Matsumoto, Akinobu and McManus, Joel and Medenbach, Jan and Melnikov, Sergey V. and Menschaert, Gerben and Merchante, Catharina and Mikl, Martin and Miller, W. Allen and Mühlemann, Oliver and Namy, Olivier and Nedialkova, Danny D. and Nosek, Jozef and Orchard, Sandra and Ozretić, Petar and Pertea, Mihaela and Pervouchine, Dmitri D. and Romão, Luísa and Ron, David and Roucou, Xavier and Rubtsova, Maria P. and Ruiz-Orera, Jorge and Saghatelian, Alan and Salzberg, Steven L. and Seale, Lucia A. and Seoighe, Cathal and Sergiev, Petr V. and Shah, Premal and Shirokikh, Nikolay and Slavoff, Sarah A. and Sonenberg, Nahum and Stasevich, Timothy J. and Szczesny, Roman J. and Tamm, Tiina and Tchórzewski, Marek and Topisirovic, Ivan and Tremblay, Michel L. and Tuller, Tamir and Ulitsky, Igor and Valášek, Leoš Shivaya and Van Damme, Petra and Viero, Gabriella and Vizcaino, Juan Antonio and Vogel, Christine and Wallace, Edward W. J. and Weissman, Jonathan S. and Westhof, Eric and Whiffin, Nicola and Wilson, Daniel N. and Xie, Zhi and Yewdell, Jonathan W. and Yordanova, Martina M. and Yu, Chien-Hung and Yurchenko, Vyacheslav and Zagrovic, Bojan and Valen, Eivind and Baranov, Pavel V.},
  doi          = {10.1038/s41592-025-02810-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {2002-2006},
  shortjournal = {Nat. Meth.},
  title        = {Translon: A single term for translated regions},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mouse lemurs. <em>NMETH</em>, <em>22</em>(10), 1999-2001. (<a href='https://doi.org/10.1038/s41592-025-02852-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studying the gray mouse lemur (Microcebus murinus), one of the world’s smallest primates, in its natural habitat and in the laboratory provides unique perspectives on primate brain evolution, cognition, aging and neurodegenerative diseases, on an accelerated timescale and at a substantially lower cost as compared with larger primate models.},
  archive      = {J_NMETH},
  author       = {Fichtel, Claudia and Huber, Daniel},
  doi          = {10.1038/s41592-025-02852-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1999-2001},
  shortjournal = {Nat. Meth.},
  title        = {Mouse lemurs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What’s new at nature methods. <em>NMETH</em>, <em>22</em>(10), 1997. (<a href='https://doi.org/10.1038/s41592-025-02883-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Team news, editorial projects and initiatives, plus a preview of what’s to come in 2026.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02883-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1997},
  shortjournal = {Nat. Meth.},
  title        = {What’s new at nature methods},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

<h2 id="nphys">NPHYS - 29</h2>
<ul>
<li><details>
<summary>
(2025). Precision without parity. <em>NPHYS</em>, <em>21</em>(9), 1498. (<a href='https://doi.org/10.1038/s41567-025-03018-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metrology has long been defined by its pursuit of reproducibility, accuracy and traceability. Yet, when it comes to the representation of women and other marginalized communities, precision has not been matched by parity, finds Solmaz Nadiri.},
  archive      = {J_NPHYS},
  author       = {Nadiri, Solmaz},
  doi          = {10.1038/s41567-025-03018-0},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1498},
  shortjournal = {Nat. Phys.},
  title        = {Precision without parity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: High-efficiency optical training of itinerant two-dimensional magnets. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03013-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Xie, Ti and Liang, Jierui and Bhattacharya, Dhritiman and Arachchige, Hasitha Suriya and Yakovenko, Victor M. and Mandrus, David G. and Qiu, Zi Qiang and Liu, Kai and Gong, Cheng},
  doi          = {10.1038/s41567-025-03013-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Author correction: High-efficiency optical training of itinerant two-dimensional magnets},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1497. (<a href='https://doi.org/10.1038/s41567-025-03054-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-03054-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1497},
  shortjournal = {Nat. Phys.},
  title        = {Publisher correction: Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient implementation of arbitrary two-qubit gates using unified control. <em>NPHYS</em>, <em>21</em>(9), 1489-1496. (<a href='https://doi.org/10.1038/s41567-025-02990-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The set of quantum logic gates that can be easily implemented is fundamental to the performance of quantum computers, as it governs the accuracy of basic quantum operations and dictates the complexity of implementing quantum algorithms. Traditional approaches to extending gate sets often require operating devices outside the ideal parameter regimes used to realize qubits, leading to increased control complexity while offering only a limited set of gates. Here we experimentally demonstrate a unified and versatile gate scheme capable of generating arbitrary two-qubit gates using only an exchange interaction and qubit driving on a superconducting quantum processor. We achieve high fidelities averaging 99.38% across a wide range of commonly used two-qubit unitaries, enabling precise multipartite entangled state preparation. Furthermore, we successfully produce a B gate, which efficiently synthesizes the entire family of two-qubit gates. Our results establish that fully exploiting the capabilities of the exchange interaction can yield a comprehensive and highly accurate gate set. With maximum expressivity, optimal gate time, demonstrated high fidelity and easy adaption to other quantum platforms, our unified control scheme offers the prospect of improved performance in quantum hardware and algorithm development. The efficiency of a quantum computer depends on which basic operations it can implement. Now a scheme that can implement any two-qubit logic gate has been demonstrated on a superconducting architecture.},
  archive      = {J_NPHYS},
  author       = {Chen, Zhen and Liu, Weiyang and Ma, Yanjun and Sun, Weijie and Wang, Ruixia and Wang, He and Xu, Huikai and Xue, Guangming and Yan, Haisheng and Yang, Zhen and Ding, Jiayu and Gao, Yang and Li, Feiyu and Zhang, Yujia and Zhang, Zikang and Jin, Yirong and Yu, Haifeng and Chen, Jianxin and Yan, Fei},
  doi          = {10.1038/s41567-025-02990-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1489-1496},
  shortjournal = {Nat. Phys.},
  title        = {Efficient implementation of arbitrary two-qubit gates using unified control},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optomechanical control of long-lived bulk acoustic phonons in the quantum regime. <em>NPHYS</em>, <em>21</em>(9), 1482-1488. (<a href='https://doi.org/10.1038/s41567-025-02989-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-fidelity quantum optomechanical control of a mechanical oscillator requires the ability to perform efficient, low-noise operations on long-lived phononic excitations. Microfabricated high-overtone bulk acoustic wave resonators (μHBARs) support high-frequency mechanical modes above 10 GHz with coherence times exceeding one millisecond. Here we demonstrate a μHBAR-based cavity optomechanical system that permits quantum optomechanical control of individual high-coherence phonon modes. We perform laser cooling of the phonon modes from an occupation of approximately 22 phonons to fewer than 0.4, corresponding to laser-based ground-state cooling of a mechanical object with a mass of 7.5 μg. During the cooling process we do not observe any absorption-induced heating, demonstrating the resilience of the HBAR optomechanical systems against parasitic heating. Our work demonstrates that μHBARs are promising as the basis for quantum optomechanical systems with robustness to decoherence that is necessary for efficient, low-noise photon–phonon conversion. A massive phonon mode in a high-overtone bulk acoustic wave resonator has been laser cooled close to its ground state. Its robustness to decoherence establishes the potential of these devices for quantum technologies.},
  archive      = {J_NPHYS},
  author       = {Diamandi, Hilel Hagai and Luo, Yizhi and Mason, David and Kanmaz, Tevfik Bulent and Ghosh, Sayan and Pavlovich, Margaret and Yoon, Taekwan and Behunin, Ryan and Puri, Shruti and Harris, Jack G. E. and Rakich, Peter T.},
  doi          = {10.1038/s41567-025-02989-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1482-1488},
  shortjournal = {Nat. Phys.},
  title        = {Optomechanical control of long-lived bulk acoustic phonons in the quantum regime},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fast optical control of a coherent hole spin in a microcavity. <em>NPHYS</em>, <em>21</em>(9), 1475-1481. (<a href='https://doi.org/10.1038/s41567-025-02988-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many of the most promising quantum information platforms store quantum information in electronic or atomic spins. To incorporate these devices into quantum networks, a spin–photon interface is required. Currently, the best on-demand single-photon sources use a semiconductor quantum dot in an engineered photonic environment. However, it is difficult to achieve coherent spin control in a high-performance single-photon source, and spin coherence is limited by magnetic noise from nuclear spins in the semiconductor host material. Here we combine all-optical spin control with a quantum dot in an open microcavity. We demonstrate fast coherent rotations of a hole spin around an arbitrary axis of the Bloch sphere with a maximum π-pulse fidelity of 98.6%. To suppress the slow magnetic noise, we laser cool the nuclear spins using the hole as a central spin. This extends the hole spin free-induction-decay time by more than an order of magnitude. It becomes much larger than both rotation time and radiative recombination time of the spin, enabling the creation of many spin–photon pairs before the loss of spin coherence. Spin–photon interfaces provide a connection between quantum information stored in atomic or electronic spins and optical communications networks. A quantum photon emitter with long-lived, controllable coherent spin has now been demonstrated.},
  archive      = {J_NPHYS},
  author       = {Hogg, Mark R. and Antoniadis, Nadia O. and Marczak, Malwina A. and Nguyen, Giang N. and Baltisberger, Timon L. and Javadi, Alisa and Schott, Rüdiger and Valentin, Sascha R. and Wieck, Andreas D. and Ludwig, Arne and Warburton, Richard J.},
  doi          = {10.1038/s41567-025-02988-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1475-1481},
  shortjournal = {Nat. Phys.},
  title        = {Fast optical control of a coherent hole spin in a microcavity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A mechanical quantum memory for microwave photons. <em>NPHYS</em>, <em>21</em>(9), 1469-1474. (<a href='https://doi.org/10.1038/s41567-025-02975-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Superconducting qubits possess outstanding capabilities for processing quantum information in the microwave domain; however they have limited coherence times. An interface between photons and phonons could allow quantum information to be stored in long-lived mechanical oscillators. Here, we introduce a platform that relies on electrostatic forces in nanoscale structures to achieve strong coupling between a superconducting qubit and a nanomechanical oscillator with an energy decay time (T1) of approximately 25 ms, well beyond those achieved in integrated superconducting circuits. We use quantum operations in this system to investigate the microscopic origins of mechanical decoherence and mitigate its impact. By using two-pulse dynamical decoupling sequences, we can extend the coherence time (T2) from 64 μs to 1 ms. These findings establish that mechanical oscillators can act as quantum memories for superconducting devices, with potential future applications in quantum computing, sensing and transduction. Superconducting qubits, a leading platform for quantum information processing, suffer from decoherence. Interfacing them with nanomechanical oscillators allows quantum information to be stored in motional states with longer lifetimes.},
  archive      = {J_NPHYS},
  author       = {Bozkurt, Alkım B. and Golami, Omid and Yu, Yue and Tian, Hao and Mirhosseini, Mohammad},
  doi          = {10.1038/s41567-025-02975-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1469-1474},
  shortjournal = {Nat. Phys.},
  title        = {A mechanical quantum memory for microwave photons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics. <em>NPHYS</em>, <em>21</em>(9), 1462-1468. (<a href='https://doi.org/10.1038/s41567-025-02973-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Skyrmions are topologically stable quasiparticles that have been investigated in contexts including particle physics, quantum field theory, acoustics and condensed-matter physics. Quantum optical skyrmions with local topological textures are expected to reshape the landscape of quantum photonic technology, although their experimental implementation has not yet been demonstrated. Here we present experimental realizations of nanophotonic quantum skyrmions using a semiconductor cavity quantum electrodynamics system. By manipulating the photonic spin–orbit coupling in a Gaussian microcavity, we obtained a confined optical mode whose polarizations feature skyrmionic topologies. With pronounced cavity quantum electrodynamics effects, we generated and detected single-photon skyrmions from a solid-state quantum emitter deterministically coupled to the Gaussian microcavity. The polarity associated with single-photon skyrmions can be swapped by flipping the polarization of the quantum emitter through the Zeeman effect. We also investigated the topological protection of quantum optical skyrmions under different perturbations. Our work opens an unexplored aspect of quantum light–matter interactions in the nanoscale and might advance resilient photonic quantum technology with high-dimensional qubits and high-capacity quantum memories. Quantum optical skyrmions are promising for quantum photonic applications but have not been experimentally realized. Now nanophotonic quantum skyrmions are generated using a semiconductor quantum dot–Gaussian microcavity quantum electrodynamics system.},
  archive      = {J_NPHYS},
  author       = {Ma, Jiantao and Yang, Jiawei and Liu, Shunfa and Chen, Bo and Li, Xueshi and Song, Changkun and Qiu, Guixin and Zou, Kai and Hu, Xiaolong and Li, Feng and Yu, Ying and Liu, Jin},
  doi          = {10.1038/s41567-025-02973-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1462-1468},
  shortjournal = {Nat. Phys.},
  title        = {Nanophotonic quantum skyrmions enabled by semiconductor cavity quantum electrodynamics},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The actin cortex acts as a mechanical memory of morphology in confined migrating cells. <em>NPHYS</em>, <em>21</em>(9), 1451-1461. (<a href='https://doi.org/10.1038/s41567-025-02980-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell migration in narrow microenvironments occurs in numerous physiological processes. It involves successive cycles of confinement and release that drive important morphological changes. However, it remains unclear whether migrating cells can retain a memory of their past morphological states that could potentially facilitate their navigation through confined spaces. We demonstrate that local geometry governs a switch between two cell morphologies, thereby facilitating cell passage through long and narrow gaps. We combined cell migration assays on standardized microsystems with biophysical modelling and biochemical perturbations to show that migrating cells have a long-term memory of past confinement events. The morphological cell states correlate across transitions through actin cortex remodelling. These findings indicate that mechanical memory in migrating cells plays an active role in their migratory potential in confined environments. Cells often navigate through confined spaces. Now it is shown that cells retain a mechanical memory of previous confinement events, which makes them more efficient at migrating through narrow microenvironments.},
  archive      = {J_NPHYS},
  author       = {Kalukula, Yohalie and Luciano, Marine and Simanov, Gleb and Charras, Guillaume and Brückner, David B. and Gabriele, Sylvain},
  doi          = {10.1038/s41567-025-02980-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1451-1461},
  shortjournal = {Nat. Phys.},
  title        = {The actin cortex acts as a mechanical memory of morphology in confined migrating cells},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capillary interactions drive the self-organization of bacterial colonies. <em>NPHYS</em>, <em>21</em>(9), 1444-1450. (<a href='https://doi.org/10.1038/s41567-025-02965-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many bacteria inhabit thin water layers on solid surfaces. These thin films occur both naturally—in soils, on hosts and on textiles—and in the laboratory on agar hydrogels. In these environments, cells experience capillary forces, but it is unclear how these forces shape bacterial collective behaviour. Here we show that the water menisci formed around bacteria lead to capillary attraction between cells while still allowing them to slide past one another. We develop an experimental apparatus that allows us to control bacterial collective behaviour by varying the strength and range of capillary forces. Combining three-dimensional imaging and cell tracking with agent-based modelling, we demonstrate that capillary attraction organizes rod-shaped bacteria into densely packed nematic groups and influences their collective dynamics and morphologies. Our results suggest that capillary forces may be a ubiquitous physical ingredient in shaping microbial communities in partially hydrated environments. Bacteria tend to live in thin layers of water on surfaces. Now the capillary forces in these layers are shown to help organize the bacteria into dense packs.},
  archive      = {J_NPHYS},
  author       = {Black, Matthew E. and Fei, Chenyi and Alert, Ricard and Wingreen, Ned S. and Shaevitz, Joshua W.},
  doi          = {10.1038/s41567-025-02965-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1444-1450},
  shortjournal = {Nat. Phys.},
  title        = {Capillary interactions drive the self-organization of bacterial colonies},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion. <em>NPHYS</em>, <em>21</em>(9), 1431-1443. (<a href='https://doi.org/10.1038/s41567-025-02964-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells can sense and respond to their environment. Central to this process is the formation of molecular clutches, which are dynamic linkages between the extracellular matrix and the actin cytoskeleton mediated by integrins and adaptor proteins. Although it is well known that force-dependent interactions between molecular-clutch components are essential for sensing substrate rigidity, the influence of nonlinear adaptor-protein elasticity is poorly understood. Here we show that adaptor-protein elasticity and local interactions between molecular clutches and the extracellular matrix are key to cellular perception of substrate stiffness. We present a semi-analytical theory that integrates experimentally measured force responses of adaptor proteins to describe cell-adhesion sensing. We propose that molecular clutches probe local mechanical substrate properties and collectively function as a differential that allows a retrograde actin flow and substrate movement to occur at different rates while maintaining a stable mechanical coupling between them. Our model reproduces experimentally measured force-loading rates of individual molecular clutches and correctly predicts cell-adhesion behaviour for a range of substrate stiffnesses. The framework presented can be extended to study complex phenomena such as focal-adhesion maturation and cell-type-specific mechanosensing. The cytoskeleton and extracellular matrix are linked by molecular clutches that enable cells to sense their environment. Now it is shown that the elastic properties of the clutches underpin stiffness sensing.},
  archive      = {J_NPHYS},
  author       = {Liu, Ping and Wang, Qiuyu and Dai, Xin and Pei, Lingxuan and Wang, Junfan and Zhao, Weiyi and Johnson, Heath E. and Yao, Mingxi and Efremov, Artem K.},
  doi          = {10.1038/s41567-025-02964-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1431-1443},
  shortjournal = {Nat. Phys.},
  title        = {Elastic properties of force-transmitting linkages determine multistable mechanosensitive behaviour of cell adhesion},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility. <em>NPHYS</em>, <em>21</em>(9), 1420-1430. (<a href='https://doi.org/10.1038/s41567-025-02983-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microtubules are hollow cylindrical cytoskeletal polymers of laterally associated protofilaments that contain head-to-tail aligned ɑ/β-tubulin heterodimers. Although the exposed microtubule exterior is readily accessible to proteins, the mechanism governing the accessibility of the confined microtubule lumen to luminal particles remains unknown. Here we show that certain tubulin family proteins (isotypes) facilitate luminal accessibility because of the mechanical properties and lateral interactions that they confer to the microtubules. We characterized the microtubules reconstituted from defined compositions of Caenorhabditis elegans tubulin isotypes. These tubulin isotypes form microtubules with comparable protofilament numbers but different luminal accessibility. We further revealed the role of tubulin isotypes in regulating the strength of inter-protofilament lateral interactions, which determines luminal accessibility through the mechanosensitivity of reversible protofilament separation. Deformation of the microtubule lattice, which generates stresses exceeding the strength of the lateral interactions, creates gaps between adjacent protofilaments, enhancing the accessibility of the lumen. Together, our findings uncovered the tubulin isotype-dependent mechanical plasticity that confers force sensitivity to the microtubule lattice and modulates the energy barrier for luminal proteins to access the lumen. Proteins interact with both the exterior and interior of microtubules. Here the relationship between microtubule building blocks and the accessibility of the microtubule interior to proteins is clarified.},
  archive      = {J_NPHYS},
  author       = {Ye, Yucheng and Hao, Zheng and Luo, Jingyi and Lam, Wai Hei and Liu, Zheng and Li, Xiang David and Zhai, Yuanliang and Lin, Yuan and Ti, Shih-Chieh},
  doi          = {10.1038/s41567-025-02983-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1420-1430},
  shortjournal = {Nat. Phys.},
  title        = {Tubulin isotypes of c. elegans harness the mechanosensitivity of the lattice for microtubule luminal accessibility},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Control of collective activity to crystallize an oscillator gas. <em>NPHYS</em>, <em>21</em>(9), 1412-1419. (<a href='https://doi.org/10.1038/s41567-025-02957-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motility-induced phase separation occurs in assemblies of self-propelled units when activity is coupled negatively to density. By contrast, the consequences of a positive coupling between density and activity on the collective behaviour of active matter remain unexplored. Here we show that collective activity can emerge from such a positive coupling among non-motile building blocks. We perform experiments with self-sustained oscillators powered by contact-charge electrophoresis. Although the oscillators are non-motile by design, they spontaneously form an active gas when confined together. The super-elastic nature of collisions constitutes a positive density–activity coupling and underlies the active gas properties. Elucidating the origin of binary collisions allows us to precisely control the structure of the active gas and its eventual crystallization. Beyond considering the overlooked positive coupling between density and activity, our work suggests that rich collective properties can emerge not only from the symmetry of interactions between active building blocks but also from their adaptable and responsive behaviour. In active matter systems, increasing density usually reduces activity. Now, in a system where density enhances activity, collective motion is shown to arise from non-motile oscillators when they are confined.},
  archive      = {J_NPHYS},
  author       = {Le Blay, Marine and Saldi, Joshua H. K. and Morin, Alexandre},
  doi          = {10.1038/s41567-025-02957-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1412-1419},
  shortjournal = {Nat. Phys.},
  title        = {Control of collective activity to crystallize an oscillator gas},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multistable polar textures in geometrically frustrated nematic liquid crystals. <em>NPHYS</em>, <em>21</em>(9), 1404-1411. (<a href='https://doi.org/10.1038/s41567-025-02966-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to manipulate polar entities with multiple external fields could enable functionalities and applications in spin systems, photonics, metamaterials and soft matter. Liquid crystals that exhibit both a crystalline structure and liquid fluidity represent a promising platform for manipulating phases with polar molecular order, notably ferroelectric ones. However, achieving a polar symmetry is challenging with rod-shaped liquid crystal molecules, which form predominantly apolar nematic phases. Here we report an approach in which a geometric lattice confinement of nematic liquid crystals is used to induce planar polar order on the scale of a mesoscopic metamaterial. We confine the nematic liquid crystal in a micropillar array, forming topological defect–pillar pairs of elastic dipoles with a free top interface in contact with an immiscible fluid. The resulting dipole lattice configurations can be programmed rheologically by flowing the top fluid and maintained even after flow cessation, a phenomenon attributed to orientational multistability of the dipoles. This multimemory effect enables the encoding and reconfiguration of directional information. Overall, these results advance our understanding of topological dipoles under confinement and shear flow, enabling the detection, tracking and recording of flow profiles and could facilitate the development of stimuli-responsive materials. Programmable topological dipoles could be used in stimuli-responsive materials. Now it is shown that confinement of an apolar liquid crystal results in controllable polar arrangements of elastic dipoles.},
  archive      = {J_NPHYS},
  author       = {Kara, Ufuoma I. and Chen, Boyuan and Čopar, Simon and Li, Shucong and Mamtani, Rajdeep and Xu, Yang and Weible, Alan H. and Boerner, Eric C. and Yang, Zhan and Yao, Yuxing and Selinger, Robin L. B. and Tkalec, Uroš and Wang, Xiaoguang},
  doi          = {10.1038/s41567-025-02966-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1404-1411},
  shortjournal = {Nat. Phys.},
  title        = {Multistable polar textures in geometrically frustrated nematic liquid crystals},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability. <em>NPHYS</em>, <em>21</em>(9), 1398-1403. (<a href='https://doi.org/10.1038/s41567-025-02982-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Topological defects give rise to non-trivial excitations—such as solitons, vortices and skyrmions—characterized by integer-valued topological charges. However, their classification becomes difficult when singularities are present in the order parameter field, complicating the computation of these charges. Although exotic nonlinear excitations have been theoretically proposed in the superfluid 3He-A phase and spinor Bose–Einstein condensates, they have not been experimentally observed and their stability has not been investigated. Here we demonstrate the presence of a singular skyrmion that goes beyond the framework of topology in a ferromagnetic superfluid. These skyrmions emerge from anomalous symmetry-breaking associated with an eccentric spin singularity and carry half the elementary charge—a feature that distinguishes them from conventional skyrmions and merons. We realize the universal regime of the quantum Kelvin–Helmholtz instability, and we identify the unconventional fractional skyrmions produced by emission from a magnetic domain wall and the spontaneous splitting of an integer skyrmion with spin singularities. The singular skyrmions are stable even after 2 s of hold time. Our results confirm the universality between classical and quantum Kelvin–Helmholtz instabilities and broaden our understanding of complex nonlinear dynamics for a non-trivial texture in topological quantum systems. The quantum Kelvin–Helmholtz instability can induce non-trivial topological excitations. Now a proof-of-principle experiment demonstrates the emergence of a stable singular fractional skyrmion spin texture originating from this quantum instability.},
  archive      = {J_NPHYS},
  author       = {Huh, SeungJung and Yun, Wooyoung and Yun, Gabin and Hwang, Samgyu and Kwon, Kiryang and Hur, Junhyeok and Lee, Seungho and Takeuchi, Hiromitsu and Kim, Se Kwon and Choi, Jae-yoon},
  doi          = {10.1038/s41567-025-02982-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1398-1403},
  shortjournal = {Nat. Phys.},
  title        = {Stable singular fractional skyrmion spin texture from the quantum Kelvin–Helmholtz instability},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Anyonic braiding in a chiral Mach–Zehnder interferometer. <em>NPHYS</em>, <em>21</em>(9), 1392-1397. (<a href='https://doi.org/10.1038/s41567-025-02960-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fractional quantum statistics are the defining characteristic of anyons. Measuring the phase generated by an exchange of anyons is challenging, as standard interferometry set-ups—such as the Fabry–Pérot interferometer—suffer from charging effects that obscure the interference signal. Here we present the observation of anyonic interference and exchange phases in an optical-like Mach–Zehnder interferometer based on co-propagating interface modes. By avoiding backscattering and deleterious charging effects, this set-up enables pristine and robust Aharonov–Bohm interference without any phase slips. At various fractional filling factors, the observed flux periodicities agree with the fundamental fractionally charged excitations that correspond to Jain states and depend only on the bulk topological order. To probe the anyonic statistics, we used a small, charged top gate in the interferometer bulk to induce localized quasiparticles without modifying the Aharonov–Bohm phase. The added quasiparticles introduce periodic phase slips. The sign and magnitude of the observed phase slips align with the expected value at filling 1/3, but their direction shows systematic deviations at fillings 2/5 and 3/7. Control over added individual quasiparticles in this design is essential for measuring the coveted non-abelian statistics in the future. An interferometer design allows systematic investigation of the anyonic statistics of bulk fractional quantum Hall states.},
  archive      = {J_NPHYS},
  author       = {Ghosh, Bikash and Labendik, Maria and Musina, Liliia and Umansky, Vladimir and Heiblum, Moty and Mross, David F.},
  doi          = {10.1038/s41567-025-02960-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1392-1397},
  shortjournal = {Nat. Phys.},
  title        = {Anyonic braiding in a chiral Mach–Zehnder interferometer},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Measurement of phonon angular momentum. <em>NPHYS</em>, <em>21</em>(9), 1387-1391. (<a href='https://doi.org/10.1038/s41567-025-02952-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In condensed matter, angular momentum is intimately related to the emergence of topological quantum states, including chiral superconductivity, quantum spin liquids and various chiral quasiparticles. Recently, it has been predicted that microscopic lattice excitations, known as phonons, can carry finite angular momentum, leading to specific physical properties of materials. However, phonon angular momentum has not yet been observed directly. Here we demonstrate that angular momentum conservation results in a macroscopic mechanical torque when applying a time-reversal symmetry-breaking thermal gradient along the chiral axis of single-crystal tellurium. We probe this torque using a cantilever-based device and establish that it changes sign by flipping the chirality or thermal gradient. This behavior disappears in polycrystalline samples that lack a preferred chirality. Our experimental results align well with theoretical calculations. We provide compelling evidence for phonon angular momentum, which might enable quantum states with potential applications in microelectronics. Theory predicts that phonons—quanta of lattice vibrations—can carry finite angular momentum and thus influence physical properties of materials. Now phonons with angular momentum have been seen in tellurium with a chiral crystal structure.},
  archive      = {J_NPHYS},
  author       = {Zhang, H. and Peshcherenko, N. and Yang, Fazhi and Ward, T. Z. and Raghuvanshi, P. and Lindsay, L. and Felser, Claudia and Zhang, Yang and Yan, J.-Q. and Miao, H.},
  doi          = {10.1038/s41567-025-02952-3},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1387-1391},
  shortjournal = {Nat. Phys.},
  title        = {Measurement of phonon angular momentum},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Strongly interacting hofstadter states in magic-angle twisted bilayer graphene. <em>NPHYS</em>, <em>21</em>(9), 1380-1386. (<a href='https://doi.org/10.1038/s41567-025-02997-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Magic-angle twisted bilayer graphene hosts a variety of strongly correlated states at partial fillings of its flat bands. In a magnetic field, these flat bands evolve into a Hofstadter spectrum renormalized by strong Coulomb interactions. Here we study the interacting Hofstadter states that spontaneously form within the topological magnetic sub-bands of an ultraclean magic-angle twisted bilayer graphene device, including symmetry-broken Chern insulator states and fractional quantum Hall states. The observed symmetry-broken Chern insulator states form a cascade, with their Chern numbers mimicking the main sequence of correlated Chern insulators. The fractional quantum Hall states form in a Jain sequence. However, they disappear at high magnetic field, in contrast to conventional fractional quantum Hall states that strengthen with increasing magnetic field. We reveal a magnetic-field-driven phase transition from composite fermion phases to a dissipative Fermi liquid. Our theoretical analysis of the magnetic sub-bands hosting the fractional quantum Hall states predicts non-uniform quantum geometric properties far from the lowest Landau level. This points towards a more natural interpretation of these states as in-field fractional Chern insulators of the magnetic sub-bands. Interacting electrons in Hofstadter bands can form symmetry-broken topological states. These are now revealed in magic-angle twisted bilayer graphene, and their properties are influenced by non-uniform quantum geometry.},
  archive      = {J_NPHYS},
  author       = {He, Minhao and Wang, Xiaoyu and Cai, Jiaqi and Herzog-Arbeitman, Jonah and Peng, Ran and Taniguchi, Takashi and Watanabe, Kenji and Stern, Ady and Bernevig, B. Andrei and Yankowitz, Matthew and Vafek, Oskar and Xu, Xiaodong},
  doi          = {10.1038/s41567-025-02997-4},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1380-1386},
  shortjournal = {Nat. Phys.},
  title        = {Strongly interacting hofstadter states in magic-angle twisted bilayer graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Universality in quantum critical flow of charge and heat in ultraclean graphene. <em>NPHYS</em>, <em>21</em>(9), 1374-1379. (<a href='https://doi.org/10.1038/s41567-025-02972-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Close to the Dirac point, graphene is expected to exist in a quantum critical Dirac fluid state, where the flow of both charge and heat can be described with a characteristic d.c. electrical conductivity and thermodynamic variables such as entropy and enthalpy densities. Although the fluid-like viscous flow of charge has been reported in state-of-the-art graphene devices, the value of conductivity, predicted to be quantized and determined only by the universality class of the critical point, has not been established experimentally so far. Here we have discerned the quantum critical universality in graphene transport by combining the electrical and thermal conductivities in very high-quality devices close to the Dirac point. We find that they are inversely related, as expected from relativistic hydrodynamics, and the characteristic conductivity converges to a quantized value. We also observe a giant violation of the Wiedemann–Franz law, where the Lorentz number exceeds the semiclassical value by more than 200 times close to the Dirac point at low temperatures. At high temperatures, the effective dynamic viscosity to entropy density ratio close to the Dirac point in the cleanest devices approaches that of a minimally viscous quantum fluid within a factor of four. Critical behaviour is expected in graphene when the carrier density is tuned to the Dirac point. Now, universality associated with the critical point is observed in electronic and thermal transport.},
  archive      = {J_NPHYS},
  author       = {Majumdar, Aniket and Chadha, Nisarg and Pal, Pritam and Gugnani, Akash and Ghawri, Bhaskar and Watanabe, Kenji and Taniguchi, Takashi and Mukerjee, Subroto and Ghosh, Arindam},
  doi          = {10.1038/s41567-025-02972-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1374-1379},
  shortjournal = {Nat. Phys.},
  title        = {Universality in quantum critical flow of charge and heat in ultraclean graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerator technologies for proton and ion beam therapy. <em>NPHYS</em>, <em>21</em>(9), 1363-1373. (<a href='https://doi.org/10.1038/s41567-025-02994-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past 75 years, proton beam therapy has emerged as a promising modality for cancer treatment, boasting precise targeting and reduced collateral damage to healthy tissue. Here we discuss the evolution of accelerator technology in proton therapy, examining advancements in cyclotron, synchrotron and linear accelerator technology, and their implications for modern treatment delivery. Additionally, we explore advances in delivering accelerated carbon or helium ions for therapeutic treatments. We also discuss the integration of advanced imaging modalities, such as multienergy X-ray, magnetic resonance imaging and ion-based imaging, for real-time monitoring and adaptive radiotherapy. These advancements position particle therapy to offer personalized and effective cancer treatment strategies, heralding improved patient outcomes. Proton and heavy-ion beam therapies are promising approaches for cancer treatment. This Review discusses the accelerator technologies that enable these therapies.},
  archive      = {J_NPHYS},
  author       = {Maradia, Vivek and Clasie, Benjamin and Snively, Emma and Parodi, Katia and Schwarz, Marco and Durante, Marco},
  doi          = {10.1038/s41567-025-02994-7},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1363-1373},
  shortjournal = {Nat. Phys.},
  title        = {Accelerator technologies for proton and ion beam therapy},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Extended native gate sets to unlock the performance of quantum processors. <em>NPHYS</em>, <em>21</em>(9), 1361-1362. (<a href='https://doi.org/10.1038/s41567-025-02991-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum operations are modelled as unitary matrices, yet experimental implementations have been restricted to only a few gate types. This fundamental limitation has now been overcome using a scheme that combines frequency control with microwave driving, enabling universal implementation of arbitrary two-qubit operations.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02991-w},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1361-1362},
  shortjournal = {Nat. Phys.},
  title        = {Extended native gate sets to unlock the performance of quantum processors},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How squeezed cells remember their shape to migrate efficiently. <em>NPHYS</em>, <em>21</em>(9), 1359-1360. (<a href='https://doi.org/10.1038/s41567-025-02984-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells migrating through narrow spaces in their environment undergo repeated shape changes to pass through tight constrictions. Epithelial cells retain a memory of past confinement, allowing them to maintain a polarized, compact morphology that enhances future migration through narrow gaps. This memory is mechanically encoded in the actin cortex.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02984-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1359-1360},
  shortjournal = {Nat. Phys.},
  title        = {How squeezed cells remember their shape to migrate efficiently},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantized limit of conductivity in near-ideal graphene. <em>NPHYS</em>, <em>21</em>(9), 1357-1358. (<a href='https://doi.org/10.1038/s41567-025-02974-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transport properties near the Dirac point in graphene are expected to be determined by quantum many-body interactions between relativistic electrons. Experiments now show that the flow of charge and heat in high-quality graphene close to charge neutrality can be described within a hydrodynamic framework, with universal intrinsic electrical conductivity that is quantized to a value close to the quantum of conductance.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02974-x},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1357-1358},
  shortjournal = {Nat. Phys.},
  title        = {Quantized limit of conductivity in near-ideal graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boron at the boundaries. <em>NPHYS</em>, <em>21</em>(9), 1356. (<a href='https://doi.org/10.1038/s41567-025-03041-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Brierley, Richard},
  doi          = {10.1038/s41567-025-03041-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1356},
  shortjournal = {Nat. Phys.},
  title        = {Boron at the boundaries},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Liquid dipole lattice. <em>NPHYS</em>, <em>21</em>(9), 1354-1355. (<a href='https://doi.org/10.1038/s41567-025-02992-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geometrically confined apolar nematic liquid crystals lose rotational symmetry and form a multifunctional polar lattice of fluid elastic dipoles with controllable orientation. The lattice is of interest from a fundamental science perspective as well as having potential applications.},
  archive      = {J_NPHYS},
  author       = {Lucchetti, Liana},
  doi          = {10.1038/s41567-025-02992-9},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1354-1355},
  shortjournal = {Nat. Phys.},
  title        = {Liquid dipole lattice},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How to twist anyons. <em>NPHYS</em>, <em>21</em>(9), 1352-1353. (<a href='https://doi.org/10.1038/s41567-025-03009-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The properties of quantum Hall states can be probed in experiments inspired by optical interferometers. Now, a device design that overcomes major drawbacks of existing interferometers allows the extraction of the quasiparticle statistics.},
  archive      = {J_NPHYS},
  author       = {Carrega, Matteo and Heun, Stefan},
  doi          = {10.1038/s41567-025-03009-1},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1352-1353},
  shortjournal = {Nat. Phys.},
  title        = {How to twist anyons},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In plain sight. <em>NPHYS</em>, <em>21</em>(9), 1351. (<a href='https://doi.org/10.1038/s41567-025-03021-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Buchanan, Mark},
  doi          = {10.1038/s41567-025-03021-5},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1351},
  shortjournal = {Nat. Phys.},
  title        = {In plain sight},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Invest in science education research to make science open to all. <em>NPHYS</em>, <em>21</em>(9), 1350. (<a href='https://doi.org/10.1038/s41567-025-03019-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attacks on Diversity, Equity and Inclusion efforts expose the need for science education research.},
  archive      = {J_NPHYS},
  author       = {Holmes, N. G.},
  doi          = {10.1038/s41567-025-03019-z},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1350},
  shortjournal = {Nat. Phys.},
  title        = {Invest in science education research to make science open to all},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Serendipitous signal. <em>NPHYS</em>, <em>21</em>(9), 1349. (<a href='https://doi.org/10.1038/s41567-025-03052-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first direct observation of gravitational waves occurred on 14 September 2015. We look back at ten years of gravitational wave science.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-03052-y},
  journal      = {Nature Physics},
  month        = {9},
  number       = {9},
  pages        = {1349},
  shortjournal = {Nat. Phys.},
  title        = {Serendipitous signal},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
