<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt">NBT - 34</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(9), 1580. (<a href='https://doi.org/10.1038/s41587-025-02804-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02804-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry. <em>NBT</em>, <em>43</em>(9), 1577-1579. (<a href='https://doi.org/10.1038/s41587-025-02803-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breaking the cycle of institutional betrayal and the culture of silence requires decisive actions to promote transparency, accountability and supportive environments in both academia and industry.},
  archive      = {J_NBT},
  author       = {Vogelaar, Alison E. and Mahmoudi, Morteza},
  doi          = {10.1038/s41587-025-02803-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1577-1579},
  shortjournal = {Nature Biotech.},
  title        = {Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Global profiling of protein complex dynamics with an experimental library of protein interaction markers. <em>NBT</em>, <em>43</em>(9), 1562-1576. (<a href='https://doi.org/10.1038/s41587-024-02432-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to systematically monitor protein complex dynamics are needed. We introduce serial ultrafiltration combined with limited proteolysis-coupled mass spectrometry (FLiP–MS), a structural proteomics workflow that generates a library of peptide markers specific to changes in PPIs by probing differences in protease susceptibility between complex-bound and monomeric forms of proteins. The library includes markers mapping to protein-binding interfaces and markers reporting on structural changes that accompany PPI changes. Integrating the marker library with LiP–MS data allows for global profiling of protein–protein interactions (PPIs) from unfractionated lysates. We apply FLiP–MS to Saccharomyces cerevisiae and probe changes in protein complex dynamics after DNA replication stress, identifying links between Spt-Ada-Gcn5 acetyltransferase activity and the assembly state of several complexes. FLiP–MS enables protein complex dynamics to be probed on any perturbation, proteome-wide, at high throughput, with peptide-level structural resolution and informing on occupancy of binding interfaces, thus providing both global and molecular views of a system under study. FLiP–MS uses a library of protein–protein interaction markers to understand protein complex dynamics.},
  archive      = {J_NBT},
  author       = {Dörig, Christian and Marulli, Cathy and Peskett, Thomas and Volkmar, Norbert and Pantolini, Lorenzo and Studer, Gabriel and Paleari, Camilla and Frommelt, Fabian and Schwede, Torsten and de Souza, Natalie and Barral, Yves and Picotti, Paola},
  doi          = {10.1038/s41587-024-02432-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1562-1576},
  shortjournal = {Nature Biotech.},
  title        = {Global profiling of protein complex dynamics with an experimental library of protein interaction markers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A circularly permuted CasRx platform for efficient, site-specific RNA editing. <em>NBT</em>, <em>43</em>(9), 1548-1561. (<a href='https://doi.org/10.1038/s41587-024-02430-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inactive Cas13 orthologs have been fused to a mutant human ADAR2 deaminase domain at the C terminus to enable programmable adenosine-to-inosine (A-to-I) RNA editing in selected transcripts. Although promising, existing RNA-editing tools generally suffer from a trade-off between efficacy and specificity, and off-target editing remains an unsolved problem. Here we describe the development of an optimized RNA-editing platform by rational protein engineering, CasRx-based Programmable Editing of RNA Technology (xPERT). We demonstrate that the topological rearrangement of a CasRx K940L mutant by circular permutation results in a robust scaffold for the tethering of a deaminase domain. We benchmark our tool against the REPAIR system and show that xPERT exhibits strong on-target activity like REPAIRv1 but low off-target editing like REPAIRv2. Our xPERT platform can be used to alter RNA sequence information without risking genome damage, effect temporary cellular changes and customize protein function. Site-specific and efficient RNA editing is achieved with a CasRx editor.},
  archive      = {J_NBT},
  author       = {Wang, Yuanming and Liu, Kaiwen Ivy and Liu, Mengying Mandy and Ooi, Kean Hean and Nguyen, Tram Anh and Chee, Jiunn En and Teo, Shun Xiang Danny and He, Shan and Tay, Jie Wen Douglas and Teo, Seok Yee and Liew, Kai Shin and Ge, Xiao Yu and Ng, Zhi Jian and Avagyan, Hasmik and Liu, Hao and Yi, Zirong and Chang, Keziah and Kok, Eng Piew Louis and Chen, Runjia and Yau, Chun En and Koh, Jun Wei and Wan, Yue and Tan, Meng How},
  doi          = {10.1038/s41587-024-02430-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1548-1561},
  shortjournal = {Nature Biotech.},
  title        = {A circularly permuted CasRx platform for efficient, site-specific RNA editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One-step nanoscale expansion microscopy reveals individual protein shapes. <em>NBT</em>, <em>43</em>(9), 1539-1547. (<a href='https://doi.org/10.1038/s41587-024-02431-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The attainable resolution of fluorescence microscopy has reached the subnanometer range, but this technique still fails to image the morphology of single proteins or small molecular complexes. Here, we expand the specimens at least tenfold, label them with conventional fluorophores and image them with conventional light microscopes, acquiring videos in which we analyze fluorescence fluctuations. One-step nanoscale expansion (ONE) microscopy enables the visualization of the shapes of individual membrane and soluble proteins, achieving around 1-nm resolution. We show that conformational changes are readily observable, such as those undergone by the ~17-kDa protein calmodulin upon Ca2+ binding. ONE is also applied to clinical samples, analyzing the morphology of protein aggregates in cerebrospinal fluid from persons with Parkinson disease, potentially aiding disease diagnosis. This technology bridges the gap between high-resolution structural biology techniques and light microscopy, providing new avenues for discoveries in biology and medicine. Combining expansion microscopy with super-resolution radial fluctuations captures the morphology of single proteins.},
  archive      = {J_NBT},
  author       = {Shaib, Ali H. and Chouaib, Abed Alrahman and Chowdhury, Rajdeep and Altendorf, Jonas and Mihaylov, Daniel and Zhang, Chi and Krah, Donatus and Imani, Vanessa and Spencer, Russell K. W. and Georgiev, Svilen Veselinov and Mougios, Nikolaos and Monga, Mehar and Reshetniak, Sofiia and Mimoso, Tiago and Chen, Han and Fatehbasharzad, Parisa and Crzan, Dagmar and Saal, Kim-Ann and Alawieh, Mohamad Mahdi and Alawar, Nadia and Eilts, Janna and Kang, Jinyoung and Soleimani, Alireza and Müller, Marcus and Pape, Constantin and Alvarez, Luis and Trenkwalder, Claudia and Mollenhauer, Brit and Outeiro, Tiago F. and Köster, Sarah and Preobraschenski, Julia and Becherer, Ute and Moser, Tobias and Boyden, Edward S. and Aricescu, A. Radu and Sauer, Markus and Opazo, Felipe and Rizzoli, Silvio O.},
  doi          = {10.1038/s41587-024-02431-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1539-1547},
  shortjournal = {Nature Biotech.},
  title        = {One-step nanoscale expansion microscopy reveals individual protein shapes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of compact transcriptional effectors using high-throughput measurements in diverse contexts. <em>NBT</em>, <em>43</em>(9), 1525-1538. (<a href='https://doi.org/10.1038/s41587-024-02442-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional effectors are protein domains known to activate or repress gene expression; however, a systematic understanding of which effector domains regulate transcription across genomic, cell type and DNA-binding domain (DBD) contexts is lacking. Here we develop dCas9-mediated high-throughput recruitment (HT-recruit), a pooled screening method for quantifying effector function at endogenous target genes and test effector function for a library containing 5,092 nuclear protein Pfam domains across varied contexts. We also map context dependencies of effectors drawn from unannotated protein regions using a larger library tiling chromatin regulators and transcription factors. We find that many effectors depend on target and DBD contexts, such as HLH domains that can act as either activators or repressors. To enable efficient perturbations, we select context-robust domains, including ZNF705 KRAB, that improve CRISPRi tools to silence promoters and enhancers. We engineer a compact human activator called NFZ, by combining NCOA3, FOXO3 and ZNF473 domains, which enables efficient CRISPRa with better viral delivery and inducible control of chimeric antigen receptor T cells. Improved effectors for CRISPRi/CRISPRa are developed following high-throughput screening of transcriptional domains.},
  archive      = {J_NBT},
  author       = {Tycko, Josh and Van, Mike V. and Aradhana and DelRosso, Nicole and Ye, Hanrong and Yao, David and Valbuena, Raeline and Vaughan-Jackson, Alun and Xu, Xiaoshu and Ludwig, Connor and Spees, Kaitlyn and Liu, Katherine and Gu, Mingxin and Khare, Venya and Mukund, Adi Xiyal and Suzuki, Peter H. and Arana, Sophia and Zhang, Catherine and Du, Peter P. and Ornstein, Thea S. and Hess, Gaelen T. and Kamber, Roarke A. and Qi, Lei S. and Khalil, Ahmad S. and Bintu, Lacramioara and Bassik, Michael C.},
  doi          = {10.1038/s41587-024-02442-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1525-1538},
  shortjournal = {Nature Biotech.},
  title        = {Development of compact transcriptional effectors using high-throughput measurements in diverse contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations. <em>NBT</em>, <em>43</em>(9), 1510-1524. (<a href='https://doi.org/10.1038/s41587-024-02428-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To assist the translation of genetic findings to disease pathobiology and therapeutics discovery, we present an ensemble deep learning framework, termed PIONEER (Protein–protein InteractiOn iNtErfacE pRediction), that predicts protein-binding partner-specific interfaces for all known protein interactions in humans and seven other common model organisms to generate comprehensive structurally informed protein interactomes. We demonstrate that PIONEER outperforms existing state-of-the-art methods and experimentally validate its predictions. We show that disease-associated mutations are enriched in PIONEER-predicted protein–protein interfaces and explore their impact on disease prognosis and drug responses. We identify 586 significant protein–protein interactions (PPIs) enriched with PIONEER-predicted interface somatic mutations (termed oncoPPIs) from analysis of approximately 11,000 whole exomes across 33 cancer types and show significant associations of oncoPPIs with patient survival and drug responses. PIONEER, implemented as both a web server platform and a software package, identifies functional consequences of disease-associated alleles and offers a deep learning tool for precision medicine at multiscale interactome network levels. Protein–protein interactomes incorporating structural data predict the functional consequences of disease mutations.},
  archive      = {J_NBT},
  author       = {Xiong, Dapeng and Qiu, Yunguang and Zhao, Junfei and Zhou, Yadi and Lee, Dongjin and Gupta, Shobhita and Torres, Mateo and Lu, Weiqiang and Liang, Siqi and Kang, Jin Joo and Eng, Charis and Loscalzo, Joseph and Cheng, Feixiong and Yu, Haiyuan},
  doi          = {10.1038/s41587-024-02428-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1510-1524},
  shortjournal = {Nature Biotech.},
  title        = {A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors. <em>NBT</em>, <em>43</em>(9), 1496-1509. (<a href='https://doi.org/10.1038/s41587-024-02448-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies to enhance the anti-tumor immune response of stereotactic ablative radiotherapy (SABR) at primary tumors and abscopal sites are under intensive investigation. Here we report a metabolizable binary supracluster (BSCgal) that combines gold nanoclusters as radiosensitizing adjuvants with small interfering RNA (siRNA) targeting the immunosuppressive mediator galectin-1 (Gal-1). BSCgal comprises reversibly crosslinked cationic gold nanoclusters and siRNA complexes in a polymer matrix that biodegrades over weeks, facilitating clearance (90.3% in vivo clearance at 4 weeks) to reduce toxicity. The particle size well above the renal filtration threshold facilitates passive delivery to tumors. Using mouse models of head and neck cancer, we show that BSCgal augments the radiodynamic and immunotherapeutic effects of SABR at the primary and metastatic tumors by promoting tumor-inhibitory leukocytes, upregulating cytotoxic granzyme B and reducing immunosuppressive cell populations. It outperforms SABR plus Gal-1 antagonists, chemoradiation drug cisplatin or PD-1 inhibitor. This work presents a translatable strategy to converge focal radiosensitization with targeted immune checkpoint silencing for personalized radioimmunotherapy. Gold-siRNA clusters boost the immune response of radiotherapy against primary and distant tumors.},
  archive      = {J_NBT},
  author       = {Jiang, Yuyan and Cao, Hongbin and Deng, Huaping and Guan, Li and Langthasa, Jimpi and Colburg, Deana Rae Crystal and Melemenidis, Stavros and Cotton, Renee M. and Aleman, John and Wang, Xiao-Jing and Graves, Edward E. and Kalbasi, Anusha and Pu, Kanyi and Rao, Jianghong and Le, Quynh-Thu},
  doi          = {10.1038/s41587-024-02448-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1496-1509},
  shortjournal = {Nature Biotech.},
  title        = {Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE. <em>NBT</em>, <em>43</em>(9), 1485-1495. (<a href='https://doi.org/10.1038/s41587-024-02426-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing of messenger RNA (mRNA) found in extracellular vesicles (EVs) in liquid biopsies can provide clinical information such as somatic mutations, resistance profiles and tumor recurrence. Despite this, EV mRNA remains underused due to its low abundance in liquid biopsies, and large sample volumes or specialized techniques for analysis are required. Here we introduce Self-amplified and CRISPR-aided Operation to Profile EVs (SCOPE), a platform for EV mRNA detection. SCOPE leverages CRISPR-mediated recognition of target RNA using Cas13 to initiate replication and signal amplification, achieving a sub-attomolar detection limit while maintaining single-nucleotide resolution. As a proof of concept, we designed probes for key mutations in KRAS, BRAF, EGFR and IDH1 genes, optimized protocols for single-pot assays and implemented an automated device for multi-sample detection. We validated SCOPE’s ability to detect early-stage lung cancer in animal models, monitored tumor mutational burden in patients with colorectal cancer and stratified patients with glioblastoma. SCOPE can expedite readouts, augmenting the clinical use of EVs in precision oncology. SCOPE detects low-abundance extracellular vesicle mRNA with high sensitivity for clinical applications.},
  archive      = {J_NBT},
  author       = {Song, Jayeon and Cho, Mi Hyeon and Cho, Hayoung and Song, Younseong and Lee, Sung Woon and Nam, Ho Chul and Yoon, Tae Ho and Shin, Jong Cheol and Hong, Jae-Sang and Kim, Yejin and Ekanayake, Emil and Jeon, Jueun and You, Dong Gil and Im, Sung Gap and Choi, Gyu-Seog and Park, Jun Seok and Carter, Bob C. and Balaj, Leonora and Seo, An Na and Miller, Miles A. and Park, Soo Yeun and Kang, Taejoon and Castro, Cesar M. and Lee, Hakho},
  doi          = {10.1038/s41587-024-02426-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1485-1495},
  shortjournal = {Nature Biotech.},
  title        = {Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Saturation profiling of drug-resistant genetic variants using prime editing. <em>NBT</em>, <em>43</em>(9), 1471-1484. (<a href='https://doi.org/10.1038/s41587-024-02465-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to characterize the functional effects of genetic variants of uncertain significance (VUSs) have been limited by incomplete coverage of the mutational space. In clinical oncology, drug resistance arising from VUSs can prevent optimal treatment. Here we introduce PEER-seq, a high-throughput method based on prime editing that can evaluate the functional effects of single-nucleotide variants (SNVs). PEER-seq introduces both intended SNVs and synonymous marker mutations using prime editing and deep sequences the endogenous target regions to identify the introduced SNVs. We generate and functionally evaluate 2,476 SNVs in the epidermal growth factor receptor gene (EGFR), including 99% of all possible variants in the canonical tyrosine kinase domain. We determined resistance profiles of 95% of all possible EGFR protein variants encoded in the whole tyrosine kinase domain against the common tyrosine kinase inhibitors afatinib, osimertinib and osimertinib in the presence of the co-occurring substitution T790M, in PC-9 cells. Our study has the potential to substantially improve the precision of therapeutic choices in clinical settings. PEER-seq characterizes the drug resistance profiles of thousands of genetic variants.},
  archive      = {J_NBT},
  author       = {Kim, Younggwang and Oh, Hyeong-Cheol and Lee, Seungho and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-024-02465-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1471-1484},
  shortjournal = {Nature Biotech.},
  title        = {Saturation profiling of drug-resistant genetic variants using prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal scanning of genetic variants with base and prime editing. <em>NBT</em>, <em>43</em>(9), 1458-1470. (<a href='https://doi.org/10.1038/s41587-024-02439-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational scanning connects genetic variants to phenotype, enabling the interrogation of protein functions, interactions and variant pathogenicity. However, current methodologies cannot efficiently engineer customizable sets of diverse genetic variants in endogenous loci across cellular contexts in high throughput. Here, we combine cytosine and adenine base editors and a prime editor to assess the pathogenicity of a broad spectrum of variants in the epithelial growth factor receptor gene (EGFR). Using pooled base editing and prime editing guide RNA libraries, we install tens of thousands of variants spanning the full coding sequence of EGFR in multiple cell lines and assess the role of these variants in tumorigenesis and resistance to tyrosine kinase inhibitors. Our EGFR variant scan identifies important hits, supporting the robustness of the approach and revealing underappreciated routes to EGFR activation and drug response. We anticipate that multimodal precision mutational scanning can be applied broadly to characterize genetic variation in any genetic element of interest at high and single-nucleotide resolution. Thousands of rare oncogene variants are evaluated using multimodal gene editing screens.},
  archive      = {J_NBT},
  author       = {Belli, Olivier and Karava, Kyriaki and Farouni, Rick and Platt, Randall J.},
  doi          = {10.1038/s41587-024-02439-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1458-1470},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal scanning of genetic variants with base and prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein. <em>NBT</em>, <em>43</em>(9), 1445-1457. (<a href='https://doi.org/10.1038/s41587-024-02437-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticle (LNP) delivery of clustered regularly interspaced short palindromic repeat (CRISPR) ribonucleoproteins (RNPs) could enable high-efficiency, low-toxicity and scalable in vivo genome editing if efficacious RNP–LNP complexes can be reliably produced. Here we engineer a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) to generate iGeoCas9 variants capable of &gt;100× more genome editing of cells and organs compared with the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP–LNP complexes edit a variety of cell types and induce homology-directed repair in cells receiving codelivered single-stranded DNA templates. Using tissue-selective LNP formulations, we observe genome-editing levels of 16‒37% in the liver and lungs of reporter mice that receive single intravenous injections of iGeoCas9 RNP–LNPs. In addition, iGeoCas9 RNPs complexed to biodegradable LNPs edit the disease-causing SFTPC gene in lung tissue with 19% average efficiency, representing a major improvement over genome-editing levels observed previously using viral or nonviral delivery strategies. These results show that thermostable Cas9 RNP–LNP complexes can expand the therapeutic potential of genome editing. An engineered clustered regularly interspaced short palindromic repeat ribonucleoprotein delivered in lipid nanoparticles efficiently edits cells in vivo.},
  archive      = {J_NBT},
  author       = {Chen, Kai and Han, Hesong and Zhao, Sheng and Xu, Bryant and Yin, Boyan and Lawanprasert, Atip and Trinidad, Marena and Burgstone, Benjamin W. and Murthy, Niren and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-024-02437-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1445-1457},
  shortjournal = {Nature Biotech.},
  title        = {Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering b cells to treat and study human disease. <em>NBT</em>, <em>43</em>(9), 1431-1444. (<a href='https://doi.org/10.1038/s41587-025-02757-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern cell therapies are designed to harness natural biology to treat a range of complex diseases. The field of immunology has shown that B cells exhibit multiple unique features, including a natural propensity to interact with and regulate other immune cells, a high capacity to produce proteins, and a long cellular lifespan, which are being creatively applied in engineered B cell (eB cell) therapies. In recent years, advances in genome editing technologies and animal modeling have facilitated rapid progress in our ability to study eB cells and execute proof-of-concept studies, thus enabling the first clinical trials of eB cell therapies. In this review, we provide an overview of recent developments in eB cell therapies, including early clinical studies. We discuss challenges to clinical implementation, and promising directions for leveraging B cell biology in future applications for cancer and chronic disease. Recent developments in B cell engineering expand our ability to study human disease and drive progress toward new B cell therapies, balancing challenges in clinical implementation with emerging opportunities.},
  archive      = {J_NBT},
  author       = {Trivedi, Nikita and Pitner, Ragan A. and Rawlings, David J. and James, Richard G.},
  doi          = {10.1038/s41587-025-02757-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1431-1444},
  shortjournal = {Nature Biotech.},
  title        = {Engineering b cells to treat and study human disease},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A CRISPR-based method for detecting mRNA in extracellular vesicles. <em>NBT</em>, <em>43</em>(9), 1428-1429. (<a href='https://doi.org/10.1038/s41587-024-02440-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A CRISPR-based assay both recognizes and amplifies target mRNA, achieving sub-attomolar sensitivity with single-nucleotide resolution. This method enables the detection of low-abundance mRNA in extracellular vesicles, providing clinically relevant information for precision oncology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02440-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1428-1429},
  shortjournal = {Nature Biotech.},
  title        = {A CRISPR-based method for detecting mRNA in extracellular vesicles},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Bao and rogers. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02784-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02784-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bao and rogers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Ntziachristos and yao. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02785-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02785-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Ntziachristos and yao},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI designs de novo antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02822-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02822-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {AI designs de novo antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosted mitochondria reverse cognitive impairment in mice. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02823-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02823-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Boosted mitochondria reverse cognitive impairment in mice},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mining archaea for antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02824-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02824-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Mining archaea for antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MicroRNAs. <em>NBT</em>, <em>43</em>(9), 1424. (<a href='https://doi.org/10.1038/s41587-025-02800-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to pharmaceutical compositions and methods of treatment involving microRNAs.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02800-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1424},
  shortjournal = {Nature Biotech.},
  title        = {MicroRNAs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Intellectual property rights in the cultivated meat industry: Opportunities and challenges. <em>NBT</em>, <em>43</em>(9), 1419-1423. (<a href='https://doi.org/10.1038/s41587-025-02798-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivated meat faces escalating intellectual property (IP) issues as the industry expands, including conventional patent rights, the private IP rights of animal breeders and the Convention on Biological Diversity.},
  archive      = {J_NBT},
  author       = {Baek, Garam and Chung, Sukhyun and Yu, Peter and Park, Yooheon},
  doi          = {10.1038/s41587-025-02798-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1419-1423},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property rights in the cultivated meat industry: Opportunities and challenges},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining druggable targets with artificial intelligence. <em>NBT</em>, <em>43</em>(9), 1416-1418. (<a href='https://doi.org/10.1038/s41587-025-02770-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A vast landscape of ‘undruggable’ cancer targets remains beyond the reach of conventional therapeutic agents. Recent advances in artificial intelligence (AI), however, are challenging this paradigm. Synthesizing insights from a Cancer Moonshot workshop, we argue that systemically addressing the undruggable target space with AI requires a new conceptual framework. We highlight the failure of current target taxonomies and the need for benchmarking datasets, and re-evaluate clinical validation for novel AI-driven modalities.},
  archive      = {J_NBT},
  author       = {Akinsanya, Karen and AlQuraishi, Mohammed and Boija, Ann and Chodera, John and Cichońska, Anna and Ghassemi, Marzyeh and Head, Martha and Jin, Wengong and Kibbe, Warren A. and Krogan, Nevan and LeVine, Michael V. and Moult, John and Stuart, Lynda and Tourassi, Georgia and Zou, James and Barzilay, Regina and Elemento, Olivier},
  doi          = {10.1038/s41587-025-02770-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1416-1418},
  shortjournal = {Nature Biotech.},
  title        = {Redefining druggable targets with artificial intelligence},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Senolytics under scrutiny in the quest to slow aging. <em>NBT</em>, <em>43</em>(9), 1413-1415. (<a href='https://doi.org/10.1038/s41587-025-02740-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de Magalhães, João Pedro},
  doi          = {10.1038/s41587-025-02740-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1413-1415},
  shortjournal = {Nature Biotech.},
  title        = {Senolytics under scrutiny in the quest to slow aging},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Freya biosciences. <em>NBT</em>, <em>43</em>(9), 1411-1412. (<a href='https://doi.org/10.1038/s41587-025-02783-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Freya Biosciences is restoring balance in the vaginal microbiome to improve pregnancy outcomes.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02783-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1411-1412},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Freya biosciences},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Vilya therapeutics. <em>NBT</em>, <em>43</em>(9), 1409-1410. (<a href='https://doi.org/10.1038/s41587-025-02782-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Vilya Therapeutics is designing macrocycle drug candidates.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-025-02782-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1409-1410},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Vilya therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Seaport therapeutics. <em>NBT</em>, <em>43</em>(9), 1407-1408. (<a href='https://doi.org/10.1038/s41587-025-02792-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Seaport Therapeutics is modifying central nervous system drugs for delivery through the lymphatic system, bypassing first-pass liver metabolism.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-025-02792-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1407-1408},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Seaport therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(9), 1406. (<a href='https://doi.org/10.1038/s41587-025-02817-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02817-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1406},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blooms supersized. <em>NBT</em>, <em>43</em>(9), 1405. (<a href='https://doi.org/10.1038/s41587-025-02825-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02825-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1405},
  shortjournal = {Nature Biotech.},
  title        = {Blooms supersized},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wegovy approved for MASH. <em>NBT</em>, <em>43</em>(9), 1404. (<a href='https://doi.org/10.1038/s41587-025-02828-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02828-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1404},
  shortjournal = {Nature Biotech.},
  title        = {Wegovy approved for MASH},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA go-ahead for HPV immunotherapy. <em>NBT</em>, <em>43</em>(9), 1402. (<a href='https://doi.org/10.1038/s41587-025-02827-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02827-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1402},
  shortjournal = {Nature Biotech.},
  title        = {FDA go-ahead for HPV immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Regenerative cell therapy marches into corneal blindness. <em>NBT</em>, <em>43</em>(9), 1401-1404. (<a href='https://doi.org/10.1038/s41587-025-02808-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a newly marketed product in Japan and 10 years of clinical data, regenerative cell therapies promise new options for patients facing a choice between invasive surgery or blindness.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew},
  doi          = {10.1038/s41587-025-02808-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1401-1404},
  shortjournal = {Nature Biotech.},
  title        = {Regenerative cell therapy marches into corneal blindness},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). US government slams mRNA vaccine research. <em>NBT</em>, <em>43</em>(9), 1400. (<a href='https://doi.org/10.1038/s41587-025-02826-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02826-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1400},
  shortjournal = {Nature Biotech.},
  title        = {US government slams mRNA vaccine research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mitochondrial transplants make healthy babies. <em>NBT</em>, <em>43</em>(9), 1399-1400. (<a href='https://doi.org/10.1038/s41587-025-02799-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After decades of research and consultation with regulators, mitochondrial donation is taking off in the UK. Will other countries follow?},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-025-02799-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1399-1400},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial transplants make healthy babies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing antivirals. <em>NBT</em>, <em>43</em>(9), 1397-1398. (<a href='https://doi.org/10.1038/s41587-025-02829-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Funding and public support for vaccines in the United States is at an all-time low. Promising research is resulting in effective antivirals, but they should not take the place of vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02829-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1397-1398},
  shortjournal = {Nature Biotech.},
  title        = {Advancing antivirals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
